<html class="pb-page" data-request-id="85a64a4a-3fdd-4893-9216-408e0df3cbba" lang="en"><head data-pb-dropzone="head"><script type="text/javascript" src="https://m.addthis.com/live/red_lojson/300lo.json?si=62719c727c671af5&amp;bkl=0&amp;bl=1&amp;pdt=144&amp;sid=62719c727c671af5&amp;pub=xa-4faab26f2cff13a7&amp;rev=v8.28.8-wp&amp;ln=en&amp;pc=men&amp;cb=0&amp;ab=-&amp;dp=pubs.acs.org&amp;fp=doi%2F10.1021%2Facscentsci.1c01108&amp;fr=&amp;of=0&amp;pd=0&amp;irt=1&amp;vcl=1&amp;md=0&amp;ct=1&amp;tct=0&amp;abt=0&amp;cdn=0&amp;pi=1&amp;rb=0&amp;gen=100&amp;chr=UTF-8&amp;colc=1651612786739&amp;jsl=1&amp;uvs=62719c6ea575c8b7002&amp;skipb=1&amp;callback=addthis.cbs.jsonp__39891072185846190"></script><script type="text/javascript" src="https://v1.addthisedge.com/live/boost/xa-4faab26f2cff13a7/_ate.track.config_resp"></script><script type="text/javascript" src="https://z.moatads.com/addthismoatframe568911941483/moatframe.js"></script><meta name="pbContext" content=";page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/acscii.2022.8.issue-2;wgroup:string:ACHS website Group;requestedJournal:journal:acscii;website:website:acspubs;journal:journal:acscii;pageGroup:string:Publication Pages;article:article:10.1021/acscentsci.1c01108;subPage:string:Full Text">




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="ACS Central Science"><meta name="dc.Title" content="NP Analyst: An Open Online Platform for Compound Activity Mapping"><meta name="dc.Creator" content="Sanghoon  Lee"><meta name="dc.Creator" content="Jeffrey A.  van Santen"><meta name="dc.Creator" content="Nima  Farzaneh"><meta name="dc.Creator" content="Dennis Y.  Liu"><meta name="dc.Creator" content="Cameron R.  Pye"><meta name="dc.Creator" content="Tim U. H.  Baumeister"><meta name="dc.Creator" content="Weng Ruh  Wong"><meta name="dc.Creator" content="Roger G.  Linington"><meta name="dc.Description" content="Few tools exist in natural products discovery to integrate biological screening and untargeted mass spectrometry data at the library scale. Previously, we reported Compound Activity Mapping as a st..."><meta name="Description" content="Few tools exist in natural products discovery to integrate biological screening and untargeted mass spectrometry data at the library scale. Previously, we reported Compound Activity Mapping as a st..."><meta name="dc.Description" content="NP Analyst is a free, open access online platform for the integration of untargeted metabolomics data with biological screening data for bioactive natural products prioritization"><meta name="Description" content="NP Analyst is a free, open access online platform for the integration of untargeted metabolomics data with biological screening data for bioactive natural products prioritization"><meta name="dc.Publisher" content="American Chemical Society"><meta name="dc.Date" scheme="WTN8601" content="January 24, 2022"><meta name="dc.Type" content="research-article"><meta name="dc.Format" content="text/HTML"><meta name="dc.Identifier" scheme="doi" content="10.1021/acscentsci.1c01108"><meta name="dc.Language" content="en"><meta name="dc.Coverage" content="world"><meta name="dc.Rights" content="Â© 2022 The Authors. Published by American Chemical Society"><meta name="citation_fulltext_world_readable" content="">
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facscentsci.1c01108">
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facscentsci.1c01108">
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facscentsci.1c01108">
        
    
    

<title>NP Analyst: An Open Online Platform for Compound Activity Mapping | ACS Central Science</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive">





    






        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acscentsci.1c01108"><meta property="og:title" content="NP Analyst: An Open Online Platform for Compound Activity Mapping"><meta property="og:image" content="https://pubs.acs.org/cms/10.1021/acscentsci.1c01108/asset/images/acscentsci.1c01108.social.jpeg_v03"><meta property="og:description" content="Few tools exist in natural products discovery to integrate biological screening and untargeted mass spectrometry data at the library scale. Previously, we reported Compound Activity Mapping as a strategy for predicting compound bioactivity profiles directly from primary screening results on extract libraries. We now present NP Analyst, an open online platform for Compound Activity Mapping that accepts bioassay data of almost any type, and is compatible with mass spectrometry data from major instrument manufacturers via the mzML format. In addition, NP Analyst will accept processed mass spectrometry data from the MZmine 2 and GNPS open-source platforms, making it a versatile tool for integration with existing discovery workflows. We demonstrate the utility of this new tool for both the dereplication of known compounds and the discovery of novel bioactive natural products using a challenging low-resolution antimicrobial bioassay data set. This new platform is available at www.npanalyst.org."><meta property="og:type" content="Article"><meta property="og:site_name" content="ACS Publications"><meta name="twitter:card" content="summary_large_image"><meta name="twitter:site" content="@ACSPublications"><meta name="twitter:image" content="https://pubs.acs.org/cms/10.1021/acscentsci.1c01108/asset/images/acscentsci.1c01108.social.jpeg_v03">
    





    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0">















    

    <meta name="dc.identifier" content="10.1021/acscentsci.1c01108">



<meta name="format-detection" content="telephone=no">




<link rel="stylesheet" type="text/css" href="/wro/l6j1~product.css">

<link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-c7d2282eca032afdc856.css"><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css">




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1640088552890.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1648583148410.css">
<script type="text/javascript" async="" src="//img.en25.com/i/elqCfg.min.js"></script><script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script><script async="" src="//www.google-analytics.com/analytics.js"></script><script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer="" src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon="{&quot;rayId&quot;:&quot;5200035e6e4d5180&quot;,&quot;startTime&quot;:1570117212010}"></script>

            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    , 'auto'
                    );
                
                ga('send', 'pageview');
            </script>
            











    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acscentsci.1c01108">
            
        







    <!-- loading Javascript for using the persistence Service -->
    <script src="/templates/jsp/pb2/js/plugin/thiss-ds.js"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            alt_context: window.location.origin,
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if (ms == undefined || ms === '') {
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            if (results.length == 0) {
                                persistService.entities(self.alt_context).then((res) => {
                                    return resolve(res.data);
                                }, function (err) {
                                    // failed
                                    reject('failed to retrieve entity from local storage (publisher domain): ' + err);
                                });
                            } else {
                                return resolve(results);
                            }
                        }, function (err) {
                            // failed
                            reject('failed to retrieve entity from local storage: ' + err);
                        });
                    // Set up the timeout
                    setTimeout(function () {
                        reject('Promise timed out ' + ms + ' ms');
                    }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                    persistService.update(self.alt_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage (publisher domain) duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            var results = res.data;
                            if (!results) {
                                persistService.remove(self.alt_context, entityId).then((res) => {
                                    return resolve(res);
                                }, function (err) {
                                    // failed
                                    console.log('failed to remove from local storage (publisher domain) duo to: ' + err);
                                });
                            }
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId, entityTitle) {
                return new Promise(function (resolve, reject) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {
                            doi: doi,
                            entityid: entityId,
                            entityTitle: entityTitle,
                            eventGroupKey: 'e8c2a95f-7162-412e-9c6f-b7f195da79b9'
                        }

                    }).done(function (status) {
                        return resolve(status);
                    }).fail(function (status) {
                        return reject(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID, ssoInstitution.entity.title)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + ssoInstitution.entity.entityID,err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            }).catch(function (err) {
                                reject(err);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();

                    if ($('.institution-preference-userconsent-checkbox').length > 0) {
                        let institutionPreferenceUserConsentCheckbox = $('.institution-preference-userconsent-checkbox').get(0);
                        if (!$(institutionPreferenceUserConsentCheckbox).is(":checked")) {
                            return;
                        }
                    }

                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err) {
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>







    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY">


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html">

<link rel="shortcut icon" href="/pb-assets/ux3/favicon-1635812676023.ico"><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><meta http-equiv="origin-trial" content="AxujKG9INjsZ8/gUq8+dTruNvk7RjZQ1oFhhgQbcTJKDnZfbzSTE81wvC2Hzaf3TW4avA76LTZEMdiedF1vIbA4AAABueyJvcmlnaW4iOiJodHRwczovL2ltYXNkay5nb29nbGVhcGlzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzVGhpcmRQYXJ0eSI6dHJ1ZX0="><meta http-equiv="origin-trial" content="Azuce85ORtSnWe1MZDTv68qpaW3iHyfL9YbLRy0cwcCZwVnePnOmkUJlG8HGikmOwhZU22dElCcfrfX2HhrBPAkAAAB7eyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A16nvcdeoOAqrJcmjLRpl1I6f3McDD8EfofAYTt/P/H4/AWwB99nxiPp6kA0fXoiZav908Z8etuL16laFPUdfQsAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="AxBHdr0J44vFBQtZUqX9sjiqf5yWZ/OcHRcRMN3H9TH+t90V/j3ENW6C8+igBZFXMJ7G3Pr8Dd13632aLng42wgAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A88BWHFjcawUfKU3lIejLoryXoyjooBXLgWmGh+hNcqMK44cugvsI5YZbNarYvi3roc1fYbHA1AVbhAtuHZflgEAAAB2eyJvcmlnaW4iOiJodHRwczovL2dvb2dsZS5jb206NDQzIiwiZmVhdHVyZSI6IlRydXN0VG9rZW5zIiwiZXhwaXJ5IjoxNjUyNzc0NDAwLCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="AzoawhTRDevLR66Y6MROu167EDncFPBvcKOaQispTo9ouEt5LvcBjnRFqiAByRT+2cDHG1Yj4dXwpLeIhc98/gIAAACFeyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A6+nc62kbJgC46ypOwRsNW6RkDn2x7tgRh0wp7jb3DtFF7oEhu1hhm4rdZHZ6zXvnKZLlYcBlQUImC4d3kKihAcAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A/9La288e7MDEU2ifusFnMg1C2Ij6uoa/Z/ylwJIXSsWfK37oESIPbxbt4IU86OGqDEPnNVruUiMjfKo65H/CQwAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_2022050201.js?cb=31067382" async=""></script><script src="https://widgets.figshare.com/static/figshare.js?_=1651612786093"></script><script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/CcdcJsmolUtilities.min.js?_=1651612786094"></script><script src="https://common.ccdc.cam.ac.uk/jsmol/14.2.7/JSmol.min.nojq.js?_=1651612786095"></script><link type="text/css" rel="stylesheet" href="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/CcdcWidget.min.css"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'central_science';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/central_science/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/central_science/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
					console.log("ad display: " + event.creativeId + " " + event.size[0] + "x" + event.size[1]);
				}
				else {
				    console.log("no ad found");
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>

<style>
.articleHeaderDropzone5 .advertisement { display: none; }
.article_header .advertisement {
  margin: -25px auto 25px auto;
}
</style>




        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&amp;EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Indiana University Libraries - Bloomington</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"></a>




        
        <div class="pj-badge"></div><a href="/journal1/acscii" title="ACS Central Science" class="header_article-logo"><img src="/cms/10.1021/acscii/asset/title.png" alt="ACS Central Science"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="acscii" name="SeriesKey" checked="checked" class="all-content"><span>ACS Cent. Sci.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"><select name="quickLinkJournal" class="quick-search_journals-select"><option value="acscii">ACS Central Science</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science &amp; Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials &amp; Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio &amp; Med Chem Au</option><option value="abseba">ACS Biomaterials Science &amp; Engineering</option><option value="accacs">ACS Catalysis</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health &amp; Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&amp;T Engineering</option><option value="aewcaa">ACS ES&amp;T Water</option><option value="afsthl">ACS Food Science &amp; Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic &amp; Inorganic Au</option><option value="aptsfn">ACS Pharmacology &amp; Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry &amp; Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&amp;EN Global Enterprise</option><option value="cenear">Chemical &amp; Engineering News Archive</option><option value="chlseg">Chemical Health &amp; Safety</option><option value="chlseg0">Chemical Health &amp; Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth &amp; Design</option><option value="enfuem">Energy &amp; Fuels</option><option value="esthag">Environmental Science &amp; Technology</option><option value="estlcu">Environmental Science &amp; Technology Letters</option><option value="iepra6.2">I&amp;EC Product Research and Development</option><option value="iechad">Industrial &amp; Engineering Chemistry</option><option value="iecac0">Industrial &amp; Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series</option><option value="iecfa7">Industrial &amp; Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial &amp; Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial &amp; Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial &amp; Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical &amp; Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health &amp; Safety</option><option value="jchsc20">Journal of Chemical Health &amp; Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial &amp; Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research &amp; Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&amp;D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autocomplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autocomplete="false" placeholder="Page" required="required" class="quick-search_page-input"><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden="âtrueâ"></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/acscentsci.1c01024">Bone-Specific Enhancement of Antibody Therapy for Breast Cancer Metastasis to Bone</a></li><li><a href="/doi/full/10.1021/acscentsci.1c01382">An Orthogonal Protection Strategy for Synthesizing Scaffold-Modifiable Dendrons and Their Application in Drug Delivery</a></li></ul></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="âtrueâ"></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="âtrueâ"></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden="âtrueâ"></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors &amp; Reviewers');">Authors &amp;
                        Reviewers</a></li>
                <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
				<li><a href="https://pubs.acs.org/page/virtual-collections.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Virtual Issues');">Virtual Issues</a></li>
                <li><a href="https://preferences.acs.org/journals" onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS &amp; Mobile');">RSS &amp; Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products &amp; Services');">Products &amp; Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos &amp; Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#" onclick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');" class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/who-we-serve/librarians/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians &amp; Account Managers');">For
                        Librarians &amp; Account Managers</a></li>
            </ul>
        </li>
        <li class="menu-pair-section hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a></li>
                
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS &amp; Open Access');">ACS &amp; Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="/events" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!-- Types--><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><!-- Subjects--><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!--div.pub-modal_right-cover-gallery.hidden-lg.hidden-md//a(href="/coverGallery") Browse Our Cover Art Gallery
--><!-- Publications list--><!-- 'pub-modal_right--inner' adds styling required for IE--><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><!-- The widget data--><input type="hidden" id="ajaxUrl" name="ajaxUrl" value="/pb/widgets/achs-publication-list?widgetId=cabcb15c-6f7a-494b-a19c-eba4fafaee5c&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Facscii.2022.8.issue-2%3Bwgroup%3Astring%3AACHS+website+Group%3BrequestedJournal%3Ajournal%3Aacscii%3Bwebsite%3Awebsite%3Aacspubs%3Bjournal%3Ajournal%3Aacscii%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facscentsci.1c01108%3BsubPage%3Astring%3AFull+Text&amp;key=c7ca46a97ce077568c7371697bcd489c0&amp;ajaxRequest=true"><input type="hidden" name="pubModalData" value=""></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facscentsci.1c01108&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facscentsci.1c01108&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acscentsci.1c01108&amp;href=/doi/10.1021/acscentsci.1c01108" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>ACS Cent. Sci.</i></span><span class="cit-fg-year"> 2022</span><span class="cit-fg-volume">, 8</span><span class="cit-fg-issue">, 2</span><span class="cit-fg-pageRange">, 223-234</span></span></span></div></div></div></div></div><input type="hidden" name="acscii" id="copyRights" value="/pb-assets/json/journal-info-1641590078917.json"><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/acscii/8/2" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acscentsci.1c01090" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Research Article</span><a href="/doi/10.1021/acscentsci.1c01127" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/acscii" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/cms/10.1021/acscii/asset/title.png" alt="Journal Logo"></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">NP Analyst: An Open Online Platform for Compound Activity Mapping</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Sanghoon Lee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sanghoon Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Simon Fraser University, 8888 University Drive, Burnaby, British Columbia V5A 1S6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sanghoon++Lee">Sanghoon Lee</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Jeffrey A. van Santen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeffrey A. van Santen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Simon Fraser University, 8888 University Drive, Burnaby, British Columbia V5A 1S6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeffrey+A.++van+Santen">Jeffrey A. van Santen</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Nima Farzaneh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nima Farzaneh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Simon Fraser University, 8888 University Drive, Burnaby, British Columbia V5A 1S6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nima++Farzaneh">Nima Farzaneh</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Dennis Y. Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dennis Y. Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Simon Fraser University, 8888 University Drive, Burnaby, British Columbia V5A 1S6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dennis+Y.++Liu">Dennis Y. Liu</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Cameron R. Pye</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cameron R. Pye</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Unnatural Products Inc., 2161 Delaware Avenue Suite A, Santa Cruz, California 95060, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cameron+R.++Pye">Cameron R. Pye</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Tim U. H. Baumeister</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tim U. H. Baumeister</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Simon Fraser University, 8888 University Drive, Burnaby, British Columbia V5A 1S6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tim+U.+H.++Baumeister">Tim U. H. Baumeister</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Weng Ruh Wong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Weng Ruh Wong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Biochemistry, University of California, Santa Cruz, Santa Cruz, California 95064, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Weng+Ruh++Wong">Weng Ruh Wong</a></span></div></div></span></li><span class="comma-separator">,&nbsp;and&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Roger G. Linington</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roger G. Linington</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Simon Fraser University, 8888 University Drive, Burnaby, British Columbia V5A 1S6, Canada</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="mailto:rliningt@sfu.ca">rliningt@sfu.ca</a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roger+G.++Linington">Roger G. Linington</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="https://orcid.org/0000-0003-1818-4971" title="Orcid link">https://orcid.org/0000-0003-1818-4971</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="acscii"><input type="hidden" name="journalNameForjhpLink" value="ACS Central Science"><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facscentsci.1c01108&amp;href=/doi/10.1021%2Facscentsci.1c01108" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>ACS Cent. Sci.</i></span> <span class="cit-year-info"><span>2022</span></span><span class="cit-volume">, 8</span><span class="cit-issue">, 2</span><span class="cit-pageRange">, 223â234</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 24, 2022</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>10 September 2021</li><li><span class="item_label"><b>Published</b> online</span>24 January 2022</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 23 February 2022</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acscentsci.1c01108" title="DOI URL">https://doi.org/10.1021/acscentsci.1c01108</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2022 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS &amp; PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="https://acsopenscience.org/open-access/licensing-options/"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice"><span>ACS AuthorChoice</span></a><span class="article_header-cc-by-nc"><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_cc-1612971093887.svg" alt="CC: Creative Commons"><span><strong>CC: </strong>Creative Commons</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_by-1612971093257.svg" alt="BY: Credit must be given to the creator"><span><strong>BY: </strong>Credit must be given to the creator</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nc-1612971093307.svg" alt="NC: Only noncommercial uses of the work are permitted"><span><strong>NC: </strong>Only noncommercial uses of the work are permitted</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nd-1612971093637.svg" alt="ND: No derivatives or adaptations of the work are permitted"><span><strong>ND: </strong>No derivatives or adaptations of the work are permitted</span></a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acscentsci.1c01108"><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4578</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" class="doi" value="10.1021/acscentsci.1c01108"><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;NP Analyst: An Open Online Platform for Compound Activity Mapping&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sanghoon&quot;,&quot;last_name&quot;:&quot;Lee&quot;},{&quot;first_name&quot;:&quot;Jeffrey&quot;,&quot;last_name&quot;:&quot;A. van Santen&quot;},{&quot;first_name&quot;:&quot;Nima&quot;,&quot;last_name&quot;:&quot;Farzaneh&quot;},{&quot;first_name&quot;:&quot;Dennis&quot;,&quot;last_name&quot;:&quot;Y. Liu&quot;},{&quot;first_name&quot;:&quot;Cameron&quot;,&quot;last_name&quot;:&quot;R. Pye&quot;},{&quot;first_name&quot;:&quot;Tim&quot;,&quot;last_name&quot;:&quot;U. H. Baumeister&quot;},{&quot;first_name&quot;:&quot;Weng&quot;,&quot;last_name&quot;:&quot;Ruh Wong&quot;},{&quot;first_name&quot;:&quot;Roger&quot;,&quot;last_name&quot;:&quot;G. Linington&quot;}],&quot;source&quot;:&quot;ACS Central Science&quot;,&quot;year&quot;:&quot;2022&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;24&quot;,&quot;issue&quot;:&quot;2&quot;,&quot;volume&quot;:&quot;8&quot;,&quot;pages&quot;:&quot;223-234&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acscentsci.1c01108&quot;},&quot;abstract&quot;:&quot;Few tools exist in natural products discovery to integrate biological screening and untargeted mass spectrometry data at the library scale. Previously, we reported Compound Activity Mapping as a strategy for predicting compound bioactivity profiles directly from primary screening results on extract libraries. We now present NP Analyst, an open online platform for Compound Activity Mapping that accepts bioassay data of almost any type, and is compatible with mass spectrometry data from major instrument manufacturers via the mzML format. In addition, NP Analyst will accept processed mass spectrometry data from the MZmine 2 and GNPS open-source platforms, making it a versatile tool for integration with existing discovery workflows. We demonstrate the utility of this new tool for both the dereplication of known compounds and the discovery of novel bioactive natural products using a challenging low-resolution antimicrobial bioassay data set. This new platform is available at www.npanalyst.org.&quot;}" class="meta-data"><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"><input type="hidden" name="pub-access" value="free" class="pub-access"><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"><input type="hidden" name="selected-share-option" class="selected-share-option"></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facscentsci.1c01108&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.1c01108" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facscentsci.1c01108&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.1c01108" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facscentsci.1c01108&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.1c01108" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facscentsci.1c01108&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.1c01108&amp;href=/doi/10.1021/acscentsci.1c01108" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acscentsci.1c01108"><a href="/doi/pdf/10.1021/acscentsci.1c01108" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acscentsci.1c01108&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fid%3Ddoi%3A10.1021%252Facscentsci.1c01108%26sid%3Dliteratum%253Aachs%26genre%3Darticle%26aulast%3DLee%26date%3D2022%26atitle%3DNP%2BAnalyst%253A%2BAn%2BOpen%2BOnline%2BPlatform%2Bfor%2BCompound%2BActivity%2BMapping%26jtitle%3DACS%2BCentral%2BScience%26title%3DACS%2BCentral%2BScience%26volume%3D8%26issue%3D2%26spage%3D223%26epage%3D234%26issn%3D2374-7943" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="acscii" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/acscii?ifr=1"></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollduration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><!-- Empty dropzone --></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/acscii/8/2" title="Go to ACS Central Science "><img src="/cms/10.1021/acscii.2022.8.issue-2/asset/acscii.2022.8.issue-2.largecover.jpg" alt="Go to ACS Central Science "></a></div></div>




        
        <!--Article Page E-Alerts-->
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','acscii');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','acscii');
	  }
  });
</script>
<!--End Article Script-->




        
        <div class="pj-badge"><input type="hidden" id="partnerJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json"></div>




        
        <div class="aJhp_link"></div>
<input type="hidden" id="aJhpJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json">
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=acscii" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0008.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Few tools exist in natural products discovery to integrate biological screening and untargeted mass spectrometry data at the library scale. Previously, we reported Compound Activity Mapping as a strategy for predicting compound bioactivity profiles directly from primary screening results on extract libraries. We now present NP Analyst, an open online platform for Compound Activity Mapping that accepts bioassay data of almost any type, and is compatible with mass spectrometry data from major instrument manufacturers via the mzML format. In addition, NP Analyst will accept processed mass spectrometry data from the MZmine 2 and GNPS open-source platforms, making it a versatile tool for integration with existing discovery workflows. We demonstrate the utility of this new tool for both the dereplication of known compounds and the discovery of novel bioactive natural products using a challenging low-resolution antimicrobial bioassay data set. This new platform is available at <a href="http://www.npanalyst.org" class="extLink">www.npanalyst.org</a>.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="hlFld-Abstract synopsis"><h2>Synopsis</h2><span><p class="articleBody_abstractText">NP Analyst is a free, open access online platform for the integration of untargeted metabolomics data with biological screening data for bioactive natural products prioritization</p></span></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05230" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05230" class="dropBlock__holder"></div></div></div></div></div><hr><div class="NLM_p">Traditionally, natural products discovery has been a linear endeavor, with projects completed on a sample-by-sample basis. This approach has been successful in discovering some of our most valuable therapeutics such as Taxol, rapamycin, and cyclosporine A, <a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1â3)</a> but is becoming increasingly inefficient as rates of rediscovery rise. <a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The development of accurate and accessible omics technologies, including genome sequencing, untargeted metabolomics, and high-throughput screening, is changing the discovery landscape for this field. Omics-data integration between these platforms has the potential to improve both the speed and accuracy of compound discovery by leveraging information from orthogonal data types at the system level, rather than evaluating individual samples sequentially. <a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> Currently, however, there are few open access tools that integrate metabolomics and bioassay data sets, limiting applications of these approaches in the natural products community. <a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p">Several approaches exist for integrating biological activity data with untargeted metabolomics data. Bioactivity-based molecular networking predicts the bioactivity of each MS feature by calculating the Pearson correlation between activity profiles and intensity profiles for each feature in the sample set using a combination of open-source tools and custom R scripts. <a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Ory et al. developed a method termed FInd BIoactive COmpounds (FiBiCo) which combines results from four different statistical models (Spearman, F-PCA, PLS, PLS-DA) to prioritize MS features that correlate positively with bioactivity profiles. <a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> This tool is written in R and is available from the Supporting Information of the original article. Finally, Olivon et al. have developed a strategy that incorporates metabolomic, taxonomic, and bioactivity data into a single data matrix for bioactive compound prioritization. <a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> This approach color codes molecular networks based on the biological activities of active fractions to permit manual prioritization of MS features. Several of these methods are labor-intensive to implement and require tailored and customized workflows; none are available as stand-alone platforms for data integration that include both data analysis and data visualization components.</div><div class="NLM_p last">Previously we developed an approach, termed Compound Activity Mapping, to directly predict bioactive constituents and modes of action from complex mixtures using a combination of image-based screening and untargeted metabolomics. <a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> We now present NP Analyst, a versatile, open access platform for Compound Activity Mapping (<a href="http://www.npanalyst.org" class="extLink">www.npanalyst.org</a>). Importantly, NP Analyst is designed to work with biological data from any assay platform and accepts mass spectrometry in several common data formats including the standard open mzML format, <a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> output peak lists from MZmine 2, <a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and network files from the Global Natural Products Social molecular networking platform (GNPS). <a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The inclusion of these input formats makes NP Analyst compatible with bioassay data from most bioassay types and MS data from all of the major instrument manufacturers. We have developed new strategies for processing metabolomics data and integrating these results with bioactivity profiles, and have packaged these tools in an open access online environment. NP Analyst provides the research community with a new discovery platform designed to accelerate compound dereplication, highlight priority metabolites for isolation, and generate global network views of biologically active chemical space for large extract libraries.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16137" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16137" class="dropBlock__holder"></div></div></div></div></div><hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Platform Design</h3><div class="NLM_p">Originally, Compound Activity Mapping was designed to work with one specific biological assay (cytological profiling) and relied on highly customized in-house scripts for data integration. <a onclick="showRef(event, 'ref11 ref15'); return false;" href="javascript:void(0);" class="ref ref11 ref15">(11,15)</a> In designing the NP Analyst platform (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure 1</a>), we identified three key attributes required for adoption by the natural products community:<ul class="NLM_list-list_type-bullet"><li><p class="inline">an open, freely accessible online interface</p></li><li><p class="inline">a workflow capable of accepting data from any biological assay</p></li><li><p class="inline">mass spectrometry data import functions compatible with both open data formats for raw data and common mass spectrometry data processing packages for processed data</p></li></ul></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0001.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0001.jpeg" id="gr1" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of the NP Analyst platform. Users input both mass spectrometry files and biological activity data for sample sets, which are scored to prioritize compounds with strong predicted biological activities for isolation and secondary screening.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Developing NP Analyst as an online resource offered several advantages over desktop deployment. It eliminated the need to support multiple operating systems and allows dynamic allocation of storage and processing resources, making the platform both responsive and scalable. In addition, updates and upgrades can be made immediately available without the need for software updates, improving the user experience.</div><div class="NLM_p">The online interface includes a suite of data validation and quality control checks to assist users with data upload and formatting. This feedback identifies issues with input data and allows users to make corrections before to job submission, reducing job failures related to data format issues. Each results file is assigned a unique job number which is used to create a permanent hyperlink to the results pages, providing a facile mechanism for collaborators to retrieve and share data.</div><div class="NLM_p">Compound Activity Mapping works by comparing the distribution of mass spectrometry features in the sample set against the biological signatures of each of these samples. This requires processing of mass spectrometry data to generate a complete list of all unique mass spectrometry features (<i>m</i>/<i>z</i> vs retention time pairs) in the sample set and the distribution of these features among the samples in the set. To ensure that NP Analyst would work for the broadest cross section of users we created a new data processing pipeline that requires only MS<sup>1</sup> level MS data in the standard mzML open data format. This strategy is sufficient to describe the vast majority of unique mass spectrometry features in the data set without requiring MS<sup>2</sup> data of a specific format from one of the myriad possible data acquisition modes. This approach also enables laboratories without MS<sup>2</sup> capability (e.g., UPLC-TOF instruments) to use the NP Analyst platform.</div><div class="NLM_p last">To extend this approach beyond our original study using image-based screening data, we have developed a new algorithm for data integration that is agnostic to bioassay data format. Unlike the previous strategy, which required normalized data sets with values between â1 and +1 and a precise number of features, this new approach will accept data sets with any number of biological features and accepts different assay data formats (inhibition/no inhibition, percent growth, etc.). This eliminates the requirement for users to have access to sophisticated screening infrastructure (e.g., high-content imaging microscope) and permits the use of existing biological screening data in the NP Analyst platform. The only requirement is that the bioactivity file contains multiple biological features to create biological fingerprints for each sample. These can either be multiple readouts from a single assay (e.g., gene expression profiles) or single readouts from multiple assays (e.g., activity across a panel of bacterial pathogens). These data should be provided as a flat CSV file containing one row for each sample in the set and one column for each bioassay readout. For full instructions on bioassay file formatting requirements see the online documentation at <a href="https://liningtonlab.github.io/npanalyst_documentation/NPAnalyst/file-import/" class="extLink">https://liningtonlab.github.io/npanalyst_documentation/NPAnalyst/file-import/</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Algorithms</h3><div class="NLM_p">NP Analyst contains three main functions; an optional step to process the mass spectrometry data to create a list of unique MS features, feature score determination based on bioactivity profiles, and generation of output files for visualization.</div><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6"> Mass Spectrometry Data Processing</h4><div class="NLM_p">Mass spectrometry data can be uploaded in one of three formats: as individual mzML files for each sample, as a comprehensive peak list obtained through third-party software (e.g., MZmine 2), or as a GNPS-derived graphML network file. In the first case (mzML), NP Analyst will perform replicate comparison and feature alignment on the individual files as discussed below. In the latter two cases (MZmine 2 and GNPS), data processing and alignment are performed in these external platforms, and only the final aligned peak list is supplied to NP Analyst. This allows users to either process their data in GNPS or MZmine 2 using existing workflows or use the alignment and replicate comparison tools built into the NP Analyst platform.</div><div class="NLM_p">For the mzML input, files are uploaded as peak picked centroided data. If technical replicates are available, then samples are first compared to identify signals that are present consistently between replicates. For <i>n</i> replicates, the default requirement is present in <i>n</i> â 1 samples. Because both retention times (rt) and <i>m</i>/<i>z</i> values can vary between analytical runs, a processing method to align replicate signals is required. We designed an alignment method based on R-trees <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> that dynamically groups signals between samples into groups that conform to allowed tolerances of rt and <i>m</i>/<i>z</i> values (<a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>). This approach ensures that groups remain limited to the defined errors in both dimensions by subdividing groups that grow to include members with too wide a range of rt of <i>m</i>/<i>z</i> values. An advantage of this R-trees approach is that it is easily extensible to additional dimensions. This permits future incorporation of additional data types, such as drift times/CCS values from ion mobility spectroscopy. Following replicate comparison, consensus signals from each sample are aligned using the same R-trees method. This analysis yields a single file containing all the unique mass spectral features in the sample set, describing these signalsâ distribution within the set.</div><div class="NLM_p last">For MZmine 2 data, preprocessing, replicate comparison, and alignment are performed using in-built tools in the MZmine package, and a single aligned peak list is exported into NP Analyst without further manipulation. For GNPS data the standard GNPS graphML network file is imported and reformatted by the NP Analyst package to generate a list of unique <i>m</i>/<i>z</i> features and sample distributions compatible with the NP Analyst pipeline. Instructions for both of these export methods are provided in the software documentation (<a href="https://liningtonlab.github.io/npanalyst_documentation/" class="extLink">https://liningtonlab.github.io/npanalyst_documentation/</a>).</div></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7"> Bioactivity Profile Integration</h4><div class="NLM_p">MS features in the unique feature list are scored for the strength and consistency of their predicted biological profiles using the Activity Score and Cluster Score metrics employed in our previous study. <a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> NP Analyst has been configured to automatically adjust to the dimensionality of the bioassay data file (e.g., 5 bioassay features vs 250) and is capable of handling data from most assay readouts (IC<sub>50</sub>, percent inhibition etc.). In addition, Boolean data types (e.g., True/False, Inhibition/no inhibition, Yes/No) are also accepted. The only requirements are that each data column must contain a consistent data type (i.e., no columns containing a mixture of numerical and Boolean data) and that missing data must be represented by empty cells rather than ânot testedâ or other text entries.</div><div class="NLM_p">Activity Score is a measure of the strength of the phenotype for a given mass spectrometry feature. It is determined by calculating the sum of the mean of the squares of the bioactivity values for each extract containing a given mass spectrometric feature. For example, if a feature is present in three samples, the Activity score is calculated by averaging the squares of the values in each column of the bioassay data and summing the resulting values.</div><div class="NLM_p">Cluster Score is a measure of the consistency of the biological fingerprints between all of the samples that contain a given mass spectrometric feature. It is determined by taking the average Pearson similarity scores between the biological fingerprints of all extracts containing a given mass spectrometric feature.</div><div class="NLM_p last">Users can define Activity and Cluster Score cutoffs during the job submission process. The analysis workflow calculates both scores for every MS feature and retains MS features that meet the minimum values for both parameters. Increasing the cutoff values for either score reduces the complexity of the results files by removing MS features that do not have strong bioactivity profiles and/or do not correlate with specific biological phenotypes. These two parameters can be used in concert to select either MS features with strong biological profiles (Activity Score) or features with consistent activity profiles independent of spectrum of activity (Cluster Score). The range of Activity Score values depends on the bioassay data format. For bioassay data that has been normalized to a scale of 0 to 1, the maximum Activity Score is equal to the number of bioassay parameters. By contrast, Cluster Score values always range from â1 to +1. In practice, setting low positive cutoff values (0.3 for Activity Score, 0.1 for Cluster Score) is sufficient to remove many of the inactive features, simplifying both data visualization and data interpretation. The definition of âhighâ scores is dependent on the type of bioassay data used, but as a general guide values greater than 30% of maximum for Activity Score and 50% of maximum for Cluster Score can be considered âhighâ.</div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Data Visualization</h3><div id="sec2_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Network Visualization</h4><div class="NLM_p">NP Analyst provides three complementary data visualization options: network, scatter plot, and community visualizations. The network view (<a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Figure S1</a>) displays samples and associated mass spectrometry features, distributed by chemical relatedness. Extract nodes (squares) are connected by edges to their associated mass spectrometry features (circles), with the network containing all MS features that pass the Activity and Cluster Score filters. The value of this representation is that samples are grouped together in the network based on the presence of shared bioactive mass spectrometry features. MS features that possess high interconnectivity within each group derive from molecules predicted to be responsible for the observed biological phenotypes. These can include different adducts and in-source fragments from the same molecule or features from groups of related molecules with similar biological profiles. Therefore, the network view is valuable for describing the range and breadth of predicted bioactive metabolites within the sample set and how they are distributed between samples.</div><div class="NLM_p last">As part of the analysis pipeline the network is divided into communities using the Louvain method for community detection. Each community is given a unique color and community ID used in the Community view (<a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Figure S2</a>). These communities contain highly interrelated nodes indicative of shared biologically active MS features and are a helpful resource for selecting priority extracts for further investigation. Switching the âColor by Communityâ toggle in the network view changes the network color scheme from activity color-coding to node colors based on community membership, which is valuable for understanding how the full sample set is distributed between communities and how those communities are interconnected. Finally, clicking on any node highlights that node and the nodes to which it is directly connected, providing a mechanism for exploring the network. The graph is interactive, permitting zoom and pan functions, as well as autolabeling based on zoom level.</div></div><div id="sec2_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> Scatter Plot Visualization</h4><div class="NLM_p last">The scatter plot visualization (<a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Figure S3</a>) presents a plot of the bioactive MS features (retention time vs <i>m</i>/<i>z</i> ratio), providing a more targeted, compound-centric data representation of the full data set. This plot can be filtered by Activity Score and Cluster Score to retain only features with strong bioactivity predictions. In addition, users can select samples of interest from the interactive list below the plot to retain only MS features from a defined subset of samples. This provides an interactive interface for assessing the predicted bioactive mass spectrometry features for any sample or set of samples in the data set and is particularly valuable when combined with information from the Community view.</div></div><div id="sec2_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Community Visualization</h4><div class="NLM_p">The community visualization provides data on bioactive MS features for specific communities from the Louvain community detection algorithm (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure 2</a>). Each community page contains a network view of the extract and MS feature nodes in that community, two scatter plots (rt vs <i>m</i>/<i>z</i> and Cluster Score vs Activity Score), and a bioactivity heatmap for the extracts in the community. Together these plots allow users to assess the biological similarities between samples within each community, select bioactive MS features that interconnect these samples, and determine the rt and <i>m</i>/<i>z</i> values for these features for subsequent dereplication and compound isolation.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0002.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0002.jpeg" id="gr2" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. NP Analyst community view (A) including network diagram (B), plot of retention time (<i>x</i>-axis) vs <i>m</i>/<i>z</i> ratio (<i>y</i>-axis) for bioactive MS features (C), plot of Cluster Score (<i>x</i>-axis) vs Activity Score (<i>y</i>-axis) for bioactive MS features (D), and activity profiles for extracts in community. Rows contain bioactivity data for each sample. Columns contain bioassay values for each bioassay readout (E). In plots BâD, extracts are represented by square gray nodes, while colored circular nodes represent MS features. The color of MS feature nodes is defined by Cluster Score (blue to red, â1 to +1). The diameter of these nodes is defined by the Activity Score (normalized scale from minimum to maximum Activity Score values). The original webpage can be accessed from <a href="http://www.npanalyst.org" class="extLink">www.npanalyst.org</a> by selecting the âOpen Sample Outputâ button, choosing the âCommunitiesâ tab, and selecting community 15 from the dropdown menu at the top of the page.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> User Interface</h3><div class="NLM_p">The user interface is designed to assist users with data import, quality control, and visualization, while prioritizing ease of use. The platform requires no registration or login details and is W3C compliant to ensure functionality on all major operating systems and browsers. To start a new analysis, users optionally enter an email address (in order to receive notifications on job status) and then upload a comma separated value (CSV) file containing the biological activity data. Because mass spectrometry files are typically large and uploads are therefore often slow, NP Analyst performs several key validation steps on the bioassay data file prior to MS data upload. This reduces failure rates and improves user experience by correcting errors early in the submission pipeline. These validation steps include verification that sample names and column headers are unique and that results columns contain exclusively numerical values or allowed Boolean terms. In addition, a warning is raised if null values are detected.</div><div class="NLM_p">Following bioassay data validation, users select the mass spectrometry data type they wish to analyze and upload the file(s). If the mzML type is selected, then files are first reviewed to ensure that every sample has the same number of replicates. Filenames are displayed in an interactive âdrag and dropâ layout that allows users to correct errors with file selection, naming, replicate assignment, etc. Once the correct replicate files are associated with each sample, MS files are reviewed to ensure that sample names align between the bioassay and the mass spectrometry data. Mismatches between bioassay and MS file sample names raises a warning listing the missing sample names.</div><div class="NLM_p">Once all validation steps are passed, the submit button is enabled. Clicking submit initializes the upload of mass spectrometry files, generates a unique job number for the experiment, and starts the data analysis pipeline. This job number is used to access the data in the visualization section of the Web site. It is also part of the unique URL that is sent to users who opt to supply an email address, providing a convenient mechanism to share results between collaborators.</div><div class="NLM_p">Upon completion, the results section is displayed, including interactive tabs for scatter plot, network, and community views, as well as a downloads page to export the results files. Export options include a graphML network file for use in network visualization tools (Cytoscape, Gephi) and CSV for use in spreadsheets and graph plotting tools (e.g., Excel, Tableau, Jmp, Spotfire).</div><div class="NLM_p last">For advanced users, the underlying processing algorithms are freely available as a command line tool via a Docker container at <a href="https://github.com/liningtonlab/npanalyst" class="extLink">https://github.com/liningtonlab/npanalyst</a>. The advantage of local installation is that it eliminates the slow step of uploading mass spectrometry data to the online server; a particular issue with mzML files. The disadvantages are that (i) programming experience is required to deploy the Docker container, (ii) some of the client-side quality control steps for individual files built into the web interface are not included in the command line tool, and (iii) the online interface cannot be used for data visualization.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Bioactive Compound Discovery</h3><div class="NLM_p">In order to evaluate the value of NP Analyst for bioactive compound discovery, we analyzed a set of 925 prefractions from our in-house marine Actinobacterial strain library. For biological profiling we deliberately selected a low-density bioassay data set that would test the performance limits of the platform. Data from our previously developed BioMAP antibacterial profiling platform <a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> were combined to generate a data set containing inhibition/no inhibition results against 15 bacterial pathogens (6 Gram + ve, 9 Gram â ve) for all 925 prefractions. This bioassay offers a maximum of 32,768 unique phenotypes (2<sup>15</sup>). However, antibacterial compounds tend to have broad activity against either Gram + ve or Gram â ve strains (or both), meaning that the effective number of phenotypes is significantly lower. Therefore, the BioMAP biological profiles provided a valuable test case for the NP Analyst platform due to their coarse-grained nature.</div><div class="NLM_p">Submission of these data and the associated MS data for all prefractions (UPLC-ESI-qTOF, positive mode, three replicates, mzML format) yielded the NP Analyst network shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure 3</a>. Examination of the bioactive features in each community on the Communities results page highlighted several communities (Communities 12, 15, and 16) containing MS features with high predicted biological properties that were prioritized for further analysis.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0003.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0003.jpeg" id="gr3" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. NP Analyst network for 925 microbial natural products prefractions. Square gray nodes represent Prefractions. Circular nodes represent MS features. The color of MS feature nodes is defined by Cluster Score (blue to red, â1 to +1). The diameter of these nodes is defined by Activity Score (normalized scale from minimum to maximum Activity Score values). A high-resolution version of this figure, including text annotations for each node and a color scale bar, is available in the <a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Supporting Information</a> (Figure S1).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Discovery of Dracolactam C</h4><div class="NLM_p">Community 12 contained 23 prefractions with very similar biological profiles (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure 4</a>A). Sixteen of the 23 prefractions in this community were connected by a single MS feature with an <i>m</i>/<i>z</i> of 452.2788 and a retention time of 3.12 min. The molecule responsible for this MS feature was therefore prioritized for isolation. Refermentation of the producing organism followed by mass-guided purification yielded a compound with a precursor [M + H]<sup>+</sup> peak at 470.2888 and a prominent [M â H<sub>2</sub>O + H]<sup>+</sup> peak at 452.2791. Dereplication against the Natural Products Atlas database <a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and comparison of the <sup>1</sup>H NMR data against the published literature identified this compound as the polyene macrolactam micromonolactam (<b>1</b>). <a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> However, during the isolation of this metabolite we identified two compounds from this fraction with similar MS features, one of which was isobaric with micromonolactam. To determine the identity of the active species unequivocally, we isolated these two additional metabolites and identified them using extensive 1D and 2D NMR experiments. One of these compounds was the known metabolite dracolactam A (<b>2</b>), <a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> which is proposed to derive from the intramolecular DielsâAlder cyclization of micromonolactam (<a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Scheme S1</a>). The second was a new compound, dracolactam C (<b>3</b>), which was identified as a different intramolecular cyclization product of micromonolactam (<a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Scheme S1</a>). For a full description of the structure elucidation of this new compound, see the <a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Supporting Information and Figure S4</a>. Screening of all three compounds (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure 4</a>B) in the BioMAP panel revealed that micromonolactam possessed a similar antibacterial profile to the profile for community 12, while dracolactams A and C were largely inactive (<a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">Table 1</a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0004.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0004.jpeg" id="gr4" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Community 12 network. Prefractions illustrated as square gray nodes. Predicted bioactive MS features illustrated as circular red nodes. <i>m</i>/<i>z</i> values for priority MS feature node annotated on the network. Peripheral nodes removed for clarity. Full community presented in <a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Figure S2</a>. (B) Structures of isolated compounds. (C) Extracted ion chromatogram (EIC) traces for predicted bioactive MS feature (top), micromonolactam (middle), and dracolactam C (bottom) illustrating alignment between the predicted bioactive constituent and the purified bioactive metabolite (micromonolactam) and the absence of predicted bioactivity for the inactive isomer dracolactam C.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Comparison of BioMAP Screening Data for Selected Communities<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0007.gif" alt="" id="fx1"><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last"><sup>1</sup>Each color (HEX code) indicates activity against the test organism in the BioMAP screening panel. (1) Red (#ff0000) â <i>A. baumannii</i>, (2) Orange (#ff6600) â <i>B. subtilis</i>, (3) Tangerine Yellow (#ffcc00) â <i>K. aerogenes</i>, (4) Fluorescent Yellow (#ccff00) â <i>E. coli</i>, (5) Bright Green (#66ff00) â <i>E. faecium</i>, (6) Lime Green (#00ff00) â <i>L. ivanovii</i>, (7) Spring Green (#00ff66) â MRSA, (8) Bright Turquoise (#00ffcc) â <i>O. anthropi</i>, (9) Deep Sky Blue (#00ccff) â <i>P. aeruginosa</i>, (10) Navy Blue (#0066ff) â <i>P. alcalifaciens</i>, (11) Blue (#0000ff) â <i>S. aureus</i>, (12) Electric Indigo (#6600ff) â <i>S. epidermidis</i>, (13) Electric Purple (#cc00ff) â <i>S. enterica</i>, (14) Hot Magenta (#ff00 cm<sup>3</sup>) â <i>V. cholerae</i>, (15) Vivid Raspberry (#ff0066) â <i>Y. pseudotuberculosis</i>. <sup>2</sup>No screening results available for collismycin A or amychelin C against <i>S. epidermidis</i>.</p></div></div><div></div></div><div class="NLM_p last">These bioassay results align with the prediction from NP Analyst, which prioritized a single MS feature (<i>m</i>/<i>z</i> 452.2788; rt 3.12 min). Inspection of the original UPLC-HRMS metabolomics data for these fractions and comparison against the retention times for compounds <b>1</b>â<b>3</b> using the same analytical method revealed that all three compounds were present in multiple prefractions in the library. However, only the distribution of micromonolactam aligned with the activity profile for community 12 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure 4</a>C), explaining why only one of these metabolites was predicted as the active component. This represents the first instance of reported biological activity for micromonolactam.</div></div><div id="sec2_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Dereplication of Collismycin Compound Family</h4><div class="NLM_p">Community 15 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure 5</a>A) presented a more complicated scenario. The community contained three prefractions with broad-spectrum antibiotic activities and a large number of candidate bioactive features. However, six of these features possessed strong Activity Scores (large diameter red nodes connecting prefractions RLUS-2108C and RLUS-2108D in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure 5</a>A, highlighted rows in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure 5</a>E). A review of the retention times for these features in the scatter plot view (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure 5</a>B) highlighted three sets of features with related retention times (2.64, 3.42, and 3.97 min) that were consistent with adducts and fragments from three separate molecules (<a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Figure S5</a>). This was supported by the peak shapes for extracted ion chromatograms (EICs) for these features in the original MS data, which were grouped into three sets (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure 5</a>D).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0005.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0005.jpeg" id="gr5" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Community 15 network. Prefractions illustrated as square gray nodes. Predicted bioactive MS features illustrated as circular red nodes. The color of MS feature nodes is defined by Cluster Score (blue to red, â1 to +1). The diameter of these nodes is defined by Activity Score (normalized scale from minimum to maximum Activity Score values). (B) Scatter plot (Activity Score â¥ 0.3) illustrating retention time alignment for bioactive MS features. (C) Structures of isolated compounds. (D) Alignment of EICs for bioactive MS features from community 15 illustrating retention time and peak shape alignment for three major components in the mixture. (E) Results table for all bioactive features from the community, highlighting features related to compounds in panel C (green = collismycin A, blue = collismycin B, red = SF2738D).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Isolation and characterization of two of these metabolites by 1D and 2D NMR experiments (<a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Figures S22âS25</a>) identified the related bipyridyl compounds collismycin A (<b>4</b>) and SF2738D (<b>5</b>) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure 5</a>C). <a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> The third metabolite was not present in sufficient quantity for analysis by NMR but was isobaric with collismycin A and possessed very similar MS<sup>2</sup> fragmentation and UV absorbance spectra (<a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Figure S6</a>), consistent with the known stereoisomer collismycin B. <a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Surprisingly, screening of collismycin A and SF2738D in the BioMAP assay revealed that, while collismycin A possessed antibacterial activity against a range of strains, SF2738D was completely inactive. This result is in line with previously published screening data for these compounds, which did not identify any antimicrobial activity for SF2738D (<a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">Table 1</a>). <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div><div class="NLM_p last">This result highlights one of the limitations of the NP Analyst platform. In cases where inactive compounds are always coproduced with bioactive molecules in microbial cultures it is not possible to differentiate between the activities of the two metabolites. The central premise of the method is that metabolites that are present in both active and inactive fractions will have low Activity and Cluster Scores. However, inactive metabolites that are only present in active fractions will have high Activity and Cluster Scores, provided that the active fractions they are present in all have similar biological profiles. This issue is particularly acute for small communities with low numbers of samples (&lt;10), as the chance of coexpression of active and inactive metabolites is higher if the number of samples is low. Therefore, users should take care to consider the values for Cluster and Activity Scores when selecting priority MS features to isolate from small communities containing multiple candidate masses. Specifically, users should examine the connections between MS features and samples, and the activity profiles of those samples on the Communities page. This information should be used in combination with the relative magnitudes of the Activity and Cluster Scores to select priority molecules for downstream isolation and biological evaluation. It is recommended that users prioritize features with the highest interconnectivity between samples with similar activity profiles, and that MS features with additional connections to inactive samples are given lower priority.</div></div><div id="sec2_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Discovery of Amychelin C</h4><div class="NLM_p">Finally, community 16 contained 5 prefractions, 2 of which possessed very similar bioactivity profiles and were connected by 3 mass spectrometry features. Evaluation of the scatter plot and EIC traces for these prefractions indicated the presence of two related molecules, one of which contained both the precursor [M + H]<sup>+</sup> feature at <i>m</i>/<i>z</i> 753.3243 and a prominent mass fragment at <i>m/</i>z 623.2319 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure 6</a>A). Searching the Natural Products Atlas database did not yield any candidate structures for this compound pair, so this community was prioritized for further investigation.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0006.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0006.jpeg" id="gr6" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Community 16 network. Prefractions illustrated as square gray nodes. Predicted bioactive MS features illustrated as circular red nodes. Circle diameter proportional to Activity Score. Peripheral nodes removed for clarity. Full community presented in <a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Figure S2</a>. (B) Substructures determined from NMR data. (C) Comparison of MS fragmentation patterns for unlabeled and labeled amychelin C, illustrating the position of labeled [1,2,3-<sup>13</sup>C]-(<span class="smallcaps smallerCapital">l</span>)-serine (red moiety) and <sup>13</sup>C-labeled carbons (blue circles). (D) Structures of amychelin C (<b>6</b>) and amychelin (<b>7</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Fermentation of the producing organism, followed by mass-guided isolation by HPLC-MS, yielded 1.18 mg of an off-white solid with an <i>m</i>/<i>z</i> of 753.3053. 1D and 2D NMR analyses, coupled with high-resolution mass spectrometry, suggested a formula for this new metabolite of C<sub>31</sub>H<sub>45</sub>N<sub>8</sub>O<sub>14</sub> (<i>m</i>/<i>z</i> 753.3053 [M + H]<sup>+</sup> (calcd. 753.3050)). Examination of the gCOSY, gHSQC, and gHMBC spectra identified three subunits (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure 6</a>B). Signals for subunit B were broad and asymmetric, and possessed integrations in multiples of three, suggesting three repeating motifs. Examination of the MS<sup>2</sup> spectrum revealed sequential neutral losses consistent with the loss of the amino acid subunits proposed in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure 6</a>C. This analysis, coupled with key gHMBC correlations between subunits (<a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Figure S7A</a>), afforded the planar structure for this new metabolite, amychelin C (<b>6</b>; <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure 6</a>C). Amychelin C is related to a previously reported siderophore, amychelin (<b>7</b>), <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> but differs by the inclusion of a methyl-oxazoline moiety in place of the oxazoline subunit present in amychelin. To determine the absolute configurations of the amino acid-derived stereocenters we performed Marfeyâs analysis (<a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Figure S8</a>), which identified the presence of <span class="smallcaps smallerCapital">l</span>-threonine, <span class="smallcaps smallerCapital">l</span>-ornithine, and a 2:1 ratio of <span class="smallcaps smallerCapital">d</span>- and <span class="smallcaps smallerCapital">l</span>-serine.</div><div class="NLM_p last">Because amychelin C contained both <span class="smallcaps smallerCapital">d</span>- and <span class="smallcaps smallerCapital">l</span>-serine., an additional experiment was required to determine the position of the <span class="smallcaps smallerCapital">l</span>-serine residue. Refermentation of the producing organism in the presence of a 1:1 mixture of [1,2,3-<sup>13</sup>C]-(<span class="smallcaps smallerCapital">l</span>)-serine and [1,2,3-<sup>12</sup>C]-(<span class="smallcaps smallerCapital">d</span>)-serine yielded an isotopically labeled version of amychelin C with an increased [M + H]<sup>+</sup> signal of 3 Da. Interpretation of the MS fragmentation data for this isotopically labeled derivative (<a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Figure S9</a>) identified the position of isotopic labeling (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure 6</a>C), completing the full absolute configurational analysis. Interestingly, amychelin C is enantiomeric to amychelin at all shared chiral centers (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure 6</a>D). This finding supports a recent study on the evolutionary origins of this compound class which noted several examples with opposite configurations at some or all positions. <a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Screening of purified amychelin C in the BioMAP assay recapitulated the spectrum of activity predicted from the original community, confirming this molecule as the predominant active component in this community (<a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">Table 1</a>).</div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82470" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82470" class="dropBlock__holder"></div></div></div></div></div><hr><div class="NLM_p">Prioritization of bioactive constituents from complex mixtures has been a longstanding challenge for the field of natural products. <a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> This is often the rate-limiting step for bioactive compound discovery and frequently leads to the rediscovery of known molecules. <a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> NP Analyst offers a target-agnostic platform for predicting biological activities of MS features directly from complex mixtures that addresses this bottleneck. Importantly, this platform is not tailored to a particular biological assay or target class, making it suitable for use with any numerical or Boolean assay readout. This extends its utility beyond drug discovery to other scientific areas. For example, this platform could be used to identify molecules related to behavioral phenotypes in chemical ecology studies, given response data for extract libraries in ecologically relevant assay systems.</div><div class="NLM_p">In principle, every natural product will have a defined profile across all of the biological space, <a onclick="showRef(event, 'ref4 ref28'); return false;" href="javascript:void(0);" class="ref ref4 ref28">(4,28)</a> meaning that NP Analyst is not restricted to multiparametric assays but can also be used with individual results from a panel of different assay platforms. For example, laboratories may have results for the same extract library against a range of different bioassay targets (cancer cell lines, bacterial or fungal pathogens, protozoan parasites, viruses, etc.). Although acquired as individual results in each screen, these data are suitable for assembly into biological profiles for use in NP Analyst, provided that the data are normalized so that each column uses the same scale. Detailed instructions for data normalization are included in the Web site documentation (<a href="https://liningtonlab.github.io/npanalyst_documentation/" class="extLink">https://liningtonlab.github.io/npanalyst_documentation/</a>). This tool is therefore immediately applicable in laboratories that have legacy bioassay results, provided that companion mass spectrometry data also exist for these samples.</div><div class="NLM_p">For optimal performance, users should carefully consider the quality of the input MS data that they import. While it is possible to run NP Analyst without any data preprocessing, this typically increases the number of candidate bioactive MS features, because raw mass lists can include many âjunkâ MS features. <a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29â31)</a> Replicate comparison can significantly reduce noise peaks in untargeted metabolomics data sets <a onclick="showRef(event, 'ref29 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref29 ref31 ref32">(29,31,32)</a> and is strongly recommended where possible. In addition, it is useful to consider the likely limit of detection of biological assays and to select an appropriate signal intensity cutoff for the MS data. While it is tempting to select the minimum possible cutoff value so as not to miss any potentially important features, if the assay is insensitive this may dramatically increase the complexity of the resulting networks without including any additional biologically informative MS features.</div><div class="NLM_p">An advantage of the NP Analyst approach is that communities are created based on bioactive MS feature distribution, rather than sample activity profiles. Therefore, molecules with different structures but similar biological profiles will form separate communities, even if the samples that contain them have indistinguishable biological fingerprints. For example, the prefractions containing micromonolactam are part of a large group of 79 prefractions with activity against the same 6 pathogens (<i>S. aureus</i>, MRSA, <i>P. alcalifaciens</i>, <i>B. subtilis</i>, <i>E. faecium</i>, <i>L. ivanovii</i>). However, the micromonolactam-containing cluster includes just 23 prefractions. The remaining prefractions with this biological profile are distributed across other communities in the network, including several (such as community 3) that have clear candidate MS features for future development that are distinct from micromonolactam.</div><div class="NLM_p">In cases where communities do not have clear individual bioactive MS features, an effective strategy for prioritizing bioactive molecules is to identify sets of active MS features with the same retention time in the plot of retention time vs <i>m</i>/<i>z</i> ratio in the community view (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure 2</a>C). It is widely recognized that many compounds form multiple MS features during the ionization process (adducts, fragments, and multiply charged species). <a onclick="showRef(event, 'ref29 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref31">(29,31)</a> Because NP Analyst scores each MS feature independently, all MS features from a given bioactive molecule will have similar Activity and Cluster Scores but different <i>m</i>/<i>z</i> ratios. These appear as a vertical âstripeâ of features with the same retention time in the retention time vs <i>m</i>/<i>z</i> ratio plot (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure 2</a>C), which provides retention time and MS properties for bioactive molecules for direct isolation without the need for further bioassay-guided fractionation. Examples of this phenomenon are communities 3 and 15, both of which contain multiple MS features for individual priority molecules.</div><div class="NLM_p last">Overall, NP Analyst offers a powerful suite of data visualizations for exploring the bioactive component of extract libraries. The scatter plot, network, and community views provide three complementary viewpoints that should be used interdependently to identify priority MS features for further chemical and biological evaluation. Provision of the raw data from all three visualizations in the downloads page allows users to manipulate these data in other third-party data visualization tools (e.g., Tableau), enabling the development of tailored data analysis workflows as required.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41258" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41258" class="dropBlock__holder"></div></div></div></div></div><hr><div class="NLM_p">A new open online platform has been developed for the direct prediction of metabolite bioactivity profiles from complex mixtures. This platform accepts a wide range of bioassay data types and is compatible with both the mzML mass spectrometry open data format and output files from two commonly used open-source mass spectrometry data processing platforms (GNPS and MZmine 2). This platform is available at <a href="http://www.npanalyst.org" class="extLink">www.npanalyst.org</a>.</div><div class="NLM_p last">We validated this new platform by analyzing a âlow-resolutionâ antimicrobial bioassay data set for 925 natural product prefractions. From these results, three communities were selected for further study, leading to the isolation of three classes of bioactive metabolites including two new compounds (dracolactam C (<b>3</b>) and amychelin C (<b>6</b>)). In addition, this analysis afforded accurate predictions of biological activities for all three compound classes, including the first reported biological activity for the polyene macrolactam micromonolactam (<b>1</b>). Together, these results demonstrate the utility of this new platform as a rapid, accurate, and flexible strategy for the discovery of novel bioactive natural products from complex mixtures.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01057" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01057" class="dropBlock__holder"></div></div></div></div></div><hr><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Experimental Procedures</h3><div class="NLM_p last">Optical rotations were measured on a Model 341 Polarimeter (PerkinElmer). Ultraviolet absorption spectra were recorded on a Cary 300 UVâvis spectrophotometer (Agilent Technologies). HR-ESI-MS and MS<sup>2</sup> fragmentation spectra were recorded on a SYNAPT UPLC-ESI-qTOF (Waters). NMR spectra were measured on an AVANCE II 600 MHz spectrometer equipped with a 5 mm QNP cryoprobe (Bruker). MPLC (CombiFlash, Teledyne ISCO) was carried out on RediSep Rf solid load cartridge (5 g, Teledyne ISCO). HPLC separations were performed on either a Waters autopurification system equipped with a SQ Detector 2 quadrupole MS detector or an Agilent 1200 series HPLC equipped with a binary pump and a diode array detector using either Synergi Fusion-RP or Kinetex XB-C18 columns (Phenomenex).</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Collection of Samples</h3><div class="NLM_p last">Sediment samples were collected into sterile 15 mL Falcon tubes by SCUBA. Sample RL09-219 was collected from Crowbar Canyon, CA. Samples RL12-176 and RL12-145 were collected from Bell Point and Dinner Island, respectively, under permit number 12-034 from the Washington Department of Fish and Wildlife.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Isolation of Bacteria</h3><div class="NLM_p last">Sediment samples were plated onto Actinobacteria-specific isolation media with added antifungal and Gram-negative antibacterial agents (cycloheximide and nalidixic acid; 50 mg/L each) by radial stamping with sterile cotton swabs. Morphologically distinct colonies were picked and replated on Difco marine broth agar plates repeatedly until pure isolates were obtained. Isolated colonies of Actinobacteria were subjected to liquid medium culturing using our standard fermentation conditions <a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and cryopreserved as glycerol stock solutions at â80 Â°C.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Isolate Fermentation, Extraction, and Prefractionation</h3><div class="NLM_p">Bacterial frozen stocks were inoculated on solid media (10.0 g of glucose, 5.0 g of NZ-amine, 1.0 g of CaCO<sub>3</sub>, 20.0 g of starch, 5.0 g of yeast extract, 20.0 g of agarose, and 1.0 L of water) and incubated at room temperature until discrete colonies became visible.</div><div class="NLM_p last">The colonies were inoculated into 40.0 mL culture tubes containing 7.0 mL of liquid media (31.2 g of Instant Ocean, 10.0 g of soluble starch, 4.0 g of peptone, 2.0 g of yeast extract per 1 L of water). Liquid cultures were incubated at room temperature and shaken at 200 rpm. After 3 days, 3.0 mL of the small-scale cultures were used to inoculate 60.0 mL of the same media in 250.0 mL wide-neck Erlenmeyer flasks with a metal spring and milk filter top. After 5 days, 50.0 mL of the medium scale cultures were inoculated into 1.0 L of media in 2.8 L wide-mouth Fernbach flasks containing a large spring with 20.0 g of Amberlite XAD-16 adsorbent resin. Large-scale cultures were fermented for 7 days, and the cells and resin were filtered using Whatman glass microfiber filters and washed with sterile water. The filtered cells and resin with the filter paper were extracted with 1:1 dichloromethane (CH<sub>2</sub>Cl<sub>2</sub>) and methanol (MeOH). The organic extracts were removed from the cells and resin by vacuum filtration, and the extracts were evaporated under vacuum. The dried extracts were prefractionated by MPLC using a MeOH/H<sub>2</sub>O step gradient system (10% MeOH wash (discarded), 20% MeOH, 40% MeOH, 60% MeOH, 80% MeOH, 100% MeOH, and 100% EtOAc) to afford six prefractions (AâF). Growth, extract, and prefractionation of each bacterium (RL09-219-HVF-D (RLUS-2105), RL12-176-HVF-A (RLUS-2108), and RL12-145-NTF-A (RLUS-2152)) were processed in the same conditions as described above.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Compound Purification</h3><div class="NLM_p">Micromonolactam (<b>1</b>) and dracolactams A (<b>2</b>) and C (<b>3</b>) were isolated from prefractions RLUS-2152C and RLUS-2152D. RLUS-2152D was separated by a RP-HPLC (Phenomenex Synergi Fusion-RP 80A, 250 mm Ã 10.0 mm, 10 Î¼m) and eluted using a gradient of 5% MeCN to 95% MeCN with 0.02% formic acid (0â22 min) at a flow rate of 8.0 mL/min to afford two subfractions (RLUS-2152D-1 and RLUS-2152D-2). RLUS-2152D-1 was purified by HPLC on a Phenomonex Kinetex XB-C18 column (100 mm Ã 4.6 mm, 26 Î¼m, 1.0 mL/min) using an isocratic elution profile (25% MeCN with 0.02% formic acid) to afford micromonolactam (<b>1</b>) and dracolactam C (<b>3</b>). RLUS-2152C was subjected to an ODS HPLC (Phenomonex Kinetex XB-C<sub>18</sub>, 100 mm Ã 4.6 mm, 26 Î¼m) using an isocratic elution system (20% MeCN + 0.02% formic acid, 1.0 mL/min) to furnish dracolactam A (<b>2</b>).</div><div class="NLM_p">RLUS-2108C was loaded on a RP-HPLC (Phenomenex Synergi Fusion-RP, 250 mm Ã 10.0 mm, 10 Î¼m) using a gradient of 5% MeCN to 95% MeCN with 0.02% formic acid at a flow rate of 8.0 mL/min gave collismycin A (<b>4</b>, RLUS-2108C-1) and two fractions (RLUS-2108C-2 and RLUS-2108C-3). The RLUS-2108C-2 fraction was purified by a RP-HPLC (Phenomonex Kinetex XB-C<sub>18</sub>, 100 mm Ã 4.6 mm, 26 Î¼m) using an isocratic separation (50% MeCN + 0.02% formic acid, 1.0 mL/min) to afford SF2738D (<b>5</b>).</div><div class="NLM_p last">Amychelin C (<b>6</b>) was purified from RLUS-2105D by a RP-HPLC (Phenomenex Synergi Fusion-RP, 250 mm Ã 10.0 mm, 10 Î¼m) using a gradient elution profile (5% MeCN to 95% MeCN + 0.02% formic acid, 8.0 mL/min).</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Antimicrobial Screening Methods and Data</h3><div class="NLM_p last">Antimicrobial susceptibility tests were performed using a miniaturized high-throughput assay adapted from the broth microdilution method outlined by the Clinical and Laboratory Standards Institute (CLSI). Bacterial test strains were individually grown on fresh Nutrient Broth (NB, ATCC Medium 3) agar, Tryptic Soy Broth (TSB, ATCC Medium 18) agar, or Brain Heart Infusion (BHI, ATCC Medium 44) agar, respectively (<a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Table S4</a>), as recommended by the American Type Culture Collection (ATCC) cultivation protocol. Individual colonies were used to inoculate 3 mL of sterile NB, TSB, or BHI media and grown overnight with shaking (200 rpm; 37 Â°C). <i>Listeria ivanovii</i> (ATCC BAA-139) and <i>Streptococcus pneumoniae</i> (ATCC 49619) were incubated overnight but not shaken (37 Â°C; 5% CO<sub>2</sub>). Saturated overnight cultures were diluted in Cation-Adjusted Mueller-Hinton Broth (CAMHB, BBL BD) or 1:1 CAMHB/BHI media according to turbidity to achieve approximately 5 Ã 10<sup>5</sup> CFU of final inoculum density and dispensed into sterile clear polystyrene 384-well microplates (Thermo Scientific 265202) with a final screening volume of 30 Î¼L. DMSO solutions of test compounds and antibiotic controls were prepared as 1:1 dilution series and pinned into each assay plate (200 nL) using a high-throughput pinning robot (Tecan Freedom EVO 100) to achieve final screening concentrations ranging from 128 Î¼M to 3.91 nM per compound. In each 384-well plate, lane 1 was reserved for DMSO vehicle and culture medium; lane 2 reserved for DMSO vehicle, culture medium, and target bacteria; lanes 23 and 24 reserved for antibiotic controls, DMSO vehicle, culture medium, and target bacteria. After compound pinning, assay plates were read as <i>T</i><sub>0</sub> at OD<sub>600</sub> using an automated plate reader system (Thermo Scientific Spinnaker Microplate Robot; BioTek Synergy Neo2 plate reader) and then every hour for 20 hours (<i>T</i><sub>1</sub>â<i>T</i><sub>20</sub>), while incubating in an ambient room temperature carousel (25 Â°C). Resulting growth curves from the dilution series of each compound and control were used to determine their minimum inhibitory concentration (MIC) values following standard procedures.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Safety Statement</h3><div class="NLM_p last"><b>Caution!</b> Biological agents <i>Staphylococcus aureus</i>, methicillin-resistant <i>Staphylococcus aureus</i> (MRSA), <i>Vibrio cholerae</i>, <i>Salmonella entericayphimurium</i>ser. Typhimurium, <i>Pseudomonas aeruginosa</i>, and <i>Yersinia pseudotuberculosis</i> were handled following BSL-2 protocols. No other unexpected or unusually high safety hazards were encountered in the work reported.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01045" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01045" class="dropBlock__holder"></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acscentsci.1c01108?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acscentsci.1c01108</a>.</p><ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Structure elucidation descriptions and NMR data for compounds <b>3</b> and <b>6</b>, chemical characterization data for compounds <b>1</b>â<b>6</b>, experimental methods and results for antibacterial screening, and high-resolution version of the full NP Analyst network presented in Figure S1 (<a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf">oc1c01108_si_001.pdf (11.12 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acscentsci.1c01108" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms &amp; Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08321" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08321" class="dropBlock__holder"></div></div></div></div></div><hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roger G. Linington</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Simon Fraser University, 8888 University Drive, Burnaby, British Columbia V5A 1S6, Canada</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="https://orcid.org/0000-0003-1818-4971" title="Orcid link">https://orcid.org/0000-0003-1818-4971</a>;&nbsp;
    </span><span class="author-email">
                Email: <a href="mailto:rliningt@sfu.ca">rliningt@sfu.ca</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sanghoon Lee</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Simon Fraser University, 8888 University Drive, Burnaby, British Columbia V5A 1S6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeffrey A. van Santen</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Simon Fraser University, 8888 University Drive, Burnaby, British Columbia V5A 1S6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nima Farzaneh</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Simon Fraser University, 8888 University Drive, Burnaby, British Columbia V5A 1S6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dennis Y. Liu</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Simon Fraser University, 8888 University Drive, Burnaby, British Columbia V5A 1S6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cameron R. Pye</span> - <span class="hlFld-Affiliation affiliation">Unnatural
Products Inc., 2161 Delaware
Avenue Suite A, Santa Cruz, California 95060, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tim U. H. Baumeister</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Simon Fraser University, 8888 University Drive, Burnaby, British Columbia V5A 1S6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weng Ruh Wong</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Biochemistry, University
of California, Santa Cruz, Santa
Cruz, California 95064, United States</span>;&nbsp;
    <span>Present Address:
                        Microchemistry, Proteomics &amp; Lipidomics Department, Genentech Inc., 470 E Grand Ave., South San Francisco, CA 94080</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported by NIH U41-AT008718 and NSERC Discovery.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): JAvS is a consultant for Unnatural Products (a peptide-based biotech startup). CRP is the CTO of Unnatural Products (listed in affiliations).<br></div></li></ul></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28658" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28658" class="dropBlock__holder"></div></div></div></div></div><hr></div><p class="references-count">
            This article references 32 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wani, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPhail, A. T.</span></span> <span> </span><span class="NLM_article-title">Plant Antitumor Agents. VI. Isolation and Structure of Taxol, a Novel Antileukemic and Antitumor Agent from Taxus Brevifolia</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1971</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">2325</span>â <span class="NLM_lpage">2327</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja00738a045</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00738a045" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADyaE3MXktVKisLc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=93&amp;publication_year=1971&amp;pages=2325-2327&amp;author=M.+C.+Waniauthor=H.+L.+Taylorauthor=M.+E.+Wallauthor=P.+Coggonauthor=A.+T.+McPhail&amp;title=Plant+Antitumor+Agents.+VI.+Isolation+and+Structure+of+Taxol%2C+a+Novel+Antileukemic+and+Antitumor+Agent+from+Taxus+Brevifolia&amp;doi=10.1021%2Fja00738a045" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Plant antitumor agents.  VI.  Isolation and structure of taxol, a novel antileukemic and antitumor agent from Taxus brevifolia</span></div><div class="casAuthors">Wani, Mansukhlal C.; Taylor, Harold Lawrence; Wall, Monroe E.; Coggon, Philip; McPhail, Andrew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2325-7</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Taxol (I), a complex ester with a taxane ring system exhibiting potent antileukemic and antitumor properties, was isolated from Taxus brevifolia (yew).  BzNHCH(Ph)CH(OH)CO2Me and 5Î²,16-epoxy-1Î²,-2Î±,4Î±,7Î²,10Î²,13Î±-hexahydroxytax-11-en-9-one 4Î±-acetate 2Î±-benzoate are obtained by methanolysis of I.  Based on chem. evidence, I is 5Î²,16-epoxy-1Î²,2Î±,4Î±,7Î²,10Î²,13Î±-hexahydroxytax-11-en-9-one 4Î±,10Î²-diacetate 2Î±-benzoate 13Î±-(3-benzamido-2-hydroxy-3-phenylpropionate).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpexINv4qMTjbVg90H21EOLACvtfcHk0lgAZ-UlL4o8lA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaE3MXktVKisLc%253D&amp;md5=50c59ef3bf2ae493408bed2a76ca0c2e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fja00738a045&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00738a045%26sid%3Dliteratum%253Aachs%26aulast%3DWani%26aufirst%3DM.%2BC.%26aulast%3DTaylor%26aufirst%3DH.%2BL.%26aulast%3DWall%26aufirst%3DM.%2BE.%26aulast%3DCoggon%26aufirst%3DP.%26aulast%3DMcPhail%26aufirst%3DA.%2BT.%26atitle%3DPlant%2520Antitumor%2520Agents.%2520VI.%2520Isolation%2520and%2520Structure%2520of%2520Taxol%252C%2520a%2520Novel%2520Antileukemic%2520and%2520Antitumor%2520Agent%2520from%2520Taxus%2520Brevifolia%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1971%26volume%3D93%26spage%3D2325%26epage%3D2327%26doi%3D10.1021%2Fja00738a045" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VÃ©zina, C.</span></span> <span> </span><span class="NLM_article-title">Rapamycin (Ay-22,989), a New Antifungal Antibiotic. II. Fermentation, Isolation and Characterization</span>. <i>J. Antibiot. (Tokyo)</i> <span class="NLM_year" style="font-weight: bold;">1975</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">727</span>â <span class="NLM_lpage">732</span>, <span class="refDoi">&nbsp;DOI: 10.7164/antibiotics.28.727</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.7164%2Fantibiotics.28.727" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1102509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A280%3ADyaE28%252Fks1Snsw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=28&amp;publication_year=1975&amp;pages=727-732&amp;author=S.+N.+Sehgalauthor=H.+Bakerauthor=C.+V%C3%A9zina&amp;title=Rapamycin+%28Ay-22%2C989%29%2C+a+New+Antifungal+Antibiotic.+II.+Fermentation%2C+Isolation+and+Characterization&amp;doi=10.7164%2Fantibiotics.28.727" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Rapamycin (AY-22,989), a new antifungal antibiotic. II. Fermentation, isolation and characterization</span></div><div class="casAuthors">Sehgal S N; Baker H; Vezina C</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of antibiotics</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">727-32</span>
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Rapamycin is a new antifungal antibiotic produced by Streptomyces hygroscopicus NRRL 5491.  It was isolated from the mycelium by solvent extraction, purified by silica gel column chromatography and crystallized as a colorless solid which melts at 183 approximately to 185 degrees C and has the empirical formula C56H89NO14.  From its characteristic ultraviolet absorption spectrum rapamycin can be classified as a triene.  It is highly active against various Candida species, especially Candida albicans.  Its activity is compared with that of amphotericin B, candicidin and nystatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSI9b1_oP_x4o3yKPtYViAGfW6udTcc2ebafGBwRNTdKrntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADyaE28%252Fks1Snsw%253D%253D&amp;md5=dc47c0f6a250c0db2710d7371a5c43c4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.28.727&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.28.727%26sid%3Dliteratum%253Aachs%26aulast%3DSehgal%26aufirst%3DS.%2BN.%26aulast%3DBaker%26aufirst%3DH.%26aulast%3DV%25C3%25A9zina%26aufirst%3DC.%26atitle%3DRapamycin%2520%2528Ay-22%252C989%2529%252C%2520a%2520New%2520Antifungal%2520Antibiotic.%2520II.%2520Fermentation%252C%2520Isolation%2520and%2520Characterization%26jtitle%3DJ.%2520Antibiot.%2520%2528Tokyo%2529%26date%3D1975%26volume%3D28%26spage%3D727%26epage%3D732%26doi%3D10.7164%2Fantibiotics.28.727" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">RÃ¼egger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loosli, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huguenin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quiquerez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Wartburg, A.</span></span> <span> </span><span class="NLM_article-title">Cyclosporin A, a Peptide Metabolite from Trichoderma Polysporum (LINK Ex PERS.) Rifai, with a Remarkable Immunosuppressive Activity</span>. <i>Helvetica</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1075</span>â <span class="NLM_lpage">1092</span>, <span class="refDoi">&nbsp;DOI: 10.1002/hlca.19760590412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1002%2Fhlca.19760590412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=950308" title="">PubMed</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=59&amp;publication_year=1976&amp;pages=1075-1092&amp;author=A.+R%C3%BCeggerauthor=M.+Kuhnauthor=H.+Lichtiauthor=H.-R.+Loosliauthor=R.+Hugueninauthor=C.+Quiquerezauthor=A.+von+Wartburg&amp;title=Cyclosporin+A%2C+a+Peptide+Metabolite+from+Trichoderma+Polysporum+%28LINK+Ex+PERS.%29+Rifai%2C+with+a+Remarkable+Immunosuppressive+Activity&amp;doi=10.1002%2Fhlca.19760590412" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fhlca.19760590412&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhlca.19760590412%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25BCegger%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DLichti%26aufirst%3DH.%26aulast%3DLoosli%26aufirst%3DH.-R.%26aulast%3DHuguenin%26aufirst%3DR.%26aulast%3DQuiquerez%26aufirst%3DC.%26aulast%3Dvon%2BWartburg%26aufirst%3DA.%26atitle%3DCyclosporin%2520A%252C%2520a%2520Peptide%2520Metabolite%2520from%2520Trichoderma%2520Polysporum%2520%2528LINK%2520Ex%2520PERS.%2529%2520Rifai%252C%2520with%2520a%2520Remarkable%2520Immunosuppressive%2520Activity%26jtitle%3DHelvetica%26date%3D1976%26volume%3D59%26spage%3D1075%26epage%3D1092%26doi%3D10.1002%2Fhlca.19760590412" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerwick, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span> <span> </span><span class="NLM_article-title">Retrospective Analysis of Natural Products Provides Insights for Future Discovery Trends</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">5601</span>â <span class="NLM_lpage">5606</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1614680114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1073%2Fpnas.1614680114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=28461474" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFSgsro%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=114&amp;publication_year=2017&amp;pages=5601-5606&amp;author=C.+R.+Pyeauthor=M.+J.+Bertinauthor=R.+S.+Lokeyauthor=W.+H.+Gerwickauthor=R.+G.+Linington&amp;title=Retrospective+Analysis+of+Natural+Products+Provides+Insights+for+Future+Discovery+Trends&amp;doi=10.1073%2Fpnas.1614680114" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Retrospective analysis of natural products provides insights for future discovery trends</span></div><div class="casAuthors">Pye, Cameron R.; Bertin, Matthew J.; Lokey, R. Scott; Gerwick, William H.; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5601-5606</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Understanding of the capacity of the natural world to produce secondary metabolites is important to a broad range of fields, including drug discovery, ecol., biosynthesis, and chem. biol., among others.  Both the abs. no. and the rate of discovery of natural products have increased significantly in recent years.  However, there is a perception and concern that the fundamental novelty of these discoveries is decreasing relative to previously known natural products.  This study presents a quant. examn. of the field from the perspective of both no. of compds. and compd. novelty using a dataset of all published microbial and marine-derived natural products.  This anal. aimed to explore a no. of key questions, such as how the rate of discovery of new natural products has changed over the past decades, how the av. natural product structural novelty has changed as a function of time, whether exploring novel taxonomic space affords an advantage in terms of novel compd. discovery, and whether it is possible to est. how close we are to having described all of the chem. space covered by natural products.  Our analyses demonstrate that most natural products being published today bear structural similarity to previously published compds., and that the range of scaffolds readily accessible from nature is limited.  However, the anal. also shows that the field continues to discover appreciable nos. of natural products with no structural precedent.  Together, these results suggest that the development of innovative discovery methods will continue to yield compds. with unique structural and biol. properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3OY35vWvx_bVg90H21EOLACvtfcHk0lg6mwAEjJ4iHg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXmvFSgsro%253D&amp;md5=44da23318031b09694c94b0bfdf1d718</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1614680114&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1614680114%26sid%3Dliteratum%253Aachs%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DBertin%26aufirst%3DM.%2BJ.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26aulast%3DGerwick%26aufirst%3DW.%2BH.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26atitle%3DRetrospective%2520Analysis%2520of%2520Natural%2520Products%2520Provides%2520Insights%2520for%2520Future%2520Discovery%2520Trends%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D5601%26epage%3D5606%26doi%3D10.1073%2Fpnas.1614680114" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Louwen, J. J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Hooft, J. J. J.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Large-Scale Integrative Analysis of Omics Data to Accelerate Specialized Metabolite Discovery</span>. <i>mSystems</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">e000726</span>â <span class="NLM_lpage">21</span>, <span class="refDoi">&nbsp;DOI: 10.1128/mSystems.00726-21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1128%2FmSystems.00726-21" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2021&amp;pages=e000726-21&amp;author=J.+J.+R.+Louwenauthor=J.+J.+J.+van+der+Hooft&amp;title=Comprehensive+Large-Scale+Integrative+Analysis+of+Omics+Data+to+Accelerate+Specialized+Metabolite+Discovery&amp;doi=10.1128%2FmSystems.00726-21" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FmSystems.00726-21&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmSystems.00726-21%26sid%3Dliteratum%253Aachs%26aulast%3DLouwen%26aufirst%3DJ.%2BJ.%2BR.%26aulast%3Dvan%2Bder%2BHooft%26aufirst%3DJ.%2BJ.%2BJ.%26atitle%3DComprehensive%2520Large-Scale%2520Integrative%2520Analysis%2520of%2520Omics%2520Data%2520to%2520Accelerate%2520Specialized%2520Metabolite%2520Discovery%26jtitle%3DmSystems%26date%3D2021%26volume%3D6%26spage%3De000726%26epage%3D21%26doi%3D10.1128%2FmSystems.00726-21" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schorn, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoeven, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ridder, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aksenov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aleti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moghaddam, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aron, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aziz, S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">A Community Resource for Paired Genomic and Metabolomic Data Mining</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">363</span>â <span class="NLM_lpage">368</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41589-020-00724-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1038%2Fs41589-020-00724-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=33589842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BB3MXktF2gt7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=2021&amp;pages=363-368&amp;author=M.+A.+Schornauthor=S.+Verhoevenauthor=L.+Ridderauthor=F.+Huberauthor=D.+D.+Acharyaauthor=A.+A.+Aksenovauthor=G.+Aletiauthor=J.+A.+Moghaddamauthor=A.+T.+Aronauthor=S.+Aziz&amp;title=A+Community+Resource+for+Paired+Genomic+and+Metabolomic+Data+Mining&amp;doi=10.1038%2Fs41589-020-00724-z" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A community resource for paired genomic and metabolomic data mining</span></div><div class="casAuthors">Schorn, Michelle A.; Verhoeven, Stefan; Ridder, Lars; Huber, Florian; Acharya, Deepa D.; Aksenov, Alexander A.; Aleti, Gajender; Moghaddam, Jamshid Amiri; Aron, Allegra T.; Aziz, Saefuddin; Bauermeister, Anelize; Bauman, Katherine D.; Baunach, Martin; Beemelmanns, Christine; Beman, J. Michael; Berlanga-Clavero, Maria Victoria; Blacutt, Alex A.; Bode, Helge B.; Boullie, Anne; Brejnrod, Asker; Bugni, Tim S.; Calteau, Alexandra; Cao, Liu; Carrion, Victor J.; Castelo-Branco, Raquel; Chanana, Shaurya; Chase, Alexander B.; Chevrette, Marc G.; Costa-Lotufo, Leticia V.; Crawford, Jason M.; Currie, Cameron R.; Cuypers, Bart; Dang, Tam; de Rond, Tristan; Demko, Alyssa M.; Dittmann, Elke; Du, Chao; Drozd, Christopher; Dujardin, Jean-Claude; Dutton, Rachel J.; Edlund, Anna; Fewer, David P.; Garg, Neha; Gauglitz, Julia M.; Gentry, Emily C.; Gerwick, Lena; Glukhov, Evgenia; Gross, Harald; Gugger, Muriel; Guillen Matus, Dulce G.; Helfrich, Eric J. N.; Hempel, Benjamin-Florian; Hur, Jae-Seoun; Iorio, Marianna; Jensen, Paul R.; Kang, Kyo Bin; Kaysser, Leonard; Kelleher, Neil L.; Kim, Chung Sub; Kim, Ki Hyun; Koester, Irina; Konig, Gabriele M.; Leao, Tiago; Lee, Seoung Rak; Lee, Yi-Yuan; Li, Xuanji; Little, Jessica C.; Maloney, Katherine N.; Mannle, Daniel; Martin H., Christian; McAvoy, Andrew C.; Metcalf, Willam W.; Mohimani, Hosein; Molina-Santiago, Carlos; Moore, Bradley S.; Mullowney, Michael W.; Muskat, Mitchell; Nothias, Louis-Felix; O'Neill, Ellis C.; Parkinson, Elizabeth I.; Petras, Daniel; Piel, Jorn; Pierce, Emily C.; Pires, Karine; Reher, Raphael; Romero, Diego; Roper, M. Caroline; Rust, Michael; Saad, Hamada; Saenz, Carmen; Sanchez, Laura M.; Soerensen, Soeren Johannes; Sosio, Margherita; Sussmuth, Roderich D.; Sweeney, Douglas; Tahlan, Kapil; Thomson, Regan J.; Tobias, Nicholas J.; Trindade-Silva, Amaro E.; van Wezel, Gilles P.; Wang, Mingxun; Weldon, Kelly C.; Zhang, Fan; Ziemert, Nadine; Duncan, Katherine R.; Crusemann, Max; Rogers, Simon; Dorrestein, Pieter C.; Medema, Marnix H.; van der Hooft, Justin J. J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">363-368</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Genomics and metabolomics are widely used to explore specialized metabolite diversity.  The Paired Omics Data Platform is a community initiative to systematically document links between metabolome and (meta)genome data, aiding identification of natural product biosynthetic origins and metabolite structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlNF_EKMtzsrVg90H21EOLACvtfcHk0lg6mwAEjJ4iHg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXktF2gt7Y%253D&amp;md5=3da222f9acc0f30cd629e0e5f628ae53</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-00724-z&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-00724-z%26sid%3Dliteratum%253Aachs%26aulast%3DSchorn%26aufirst%3DM.%2BA.%26aulast%3DVerhoeven%26aufirst%3DS.%26aulast%3DRidder%26aufirst%3DL.%26aulast%3DHuber%26aufirst%3DF.%26aulast%3DAcharya%26aufirst%3DD.%2BD.%26aulast%3DAksenov%26aufirst%3DA.%2BA.%26aulast%3DAleti%26aufirst%3DG.%26aulast%3DMoghaddam%26aufirst%3DJ.%2BA.%26aulast%3DAron%26aufirst%3DA.%2BT.%26aulast%3DAziz%26aufirst%3DS.%26atitle%3DA%2520Community%2520Resource%2520for%2520Paired%2520Genomic%2520and%2520Metabolomic%2520Data%2520Mining%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2021%26volume%3D17%26spage%3D363%26epage%3D368%26doi%3D10.1038%2Fs41589-020-00724-z" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolfender, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litaudon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touboul, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queiroz, E. F.</span></span> <span> </span><span class="NLM_article-title">Innovative Omics-Based Approaches for Prioritisation and Targeted Isolation of Natural Products â New Strategies for Drug Discovery</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">855</span>â <span class="NLM_lpage">868</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C9NP00004F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1039%2FC9NP00004F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=31073562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC1MXps1SqtL8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=2019&amp;pages=855-868&amp;author=J.-L.+Wolfenderauthor=M.+Litaudonauthor=D.+Touboulauthor=E.+F.+Queiroz&amp;title=Innovative+Omics-Based+Approaches+for+Prioritisation+and+Targeted+Isolation+of+Natural+Products+%E2%80%93+New+Strategies+for+Drug+Discovery&amp;doi=10.1039%2FC9NP00004F" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Innovative omics-based approaches for prioritisation and targeted isolation of natural products - new strategies for drug discovery</span></div><div class="casAuthors">Wolfender, Jean-Luc; Litaudon, Marc; Touboul, David; Queiroz, Emerson Ferreira</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">855-868</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Covering: 2013 to 2019The exploration of the chem. diversity of exts. from various biol. sources has led to major drug discoveries.  Over the past two decades, despite the introduction of advanced methodologies for natural product (NP) research (e.g., dereplication and high content screening), successful accounts of the validation of NPs as lead therapeutic candidates have been limited.  In this context, one of the main challenges faced is related to working with crude natural exts. because of their complex compn. and the inadequacies of classical bioguided isolation studies given the pace of high-throughput screening campaigns.  In line with the development of metabolomics, genomics and chemometrics, significant advances in metabolite profiling have been achieved and have generated high-quality massive genome and metabolome data on natural exts.  The unambiguous identification of each individual NP in an ext. using generic methods remains challenging.  However, the establishment of structural links among NPs via mol. network anal. and the detn. of common features of ext. compn. have provided invaluable information to the scientific community.  In this context, new multi-informational-based profiling approaches integrating taxonomic and/or bioactivity data can hold promise for the discovery and development of new bioactive compds. and return NPs back to an exciting era of development.  In this article, we examine recent studies that have the potential to improve the efficiency of NP prioritisation and to accelerate the targeted isolation of key NPs.  Perspectives on the field's evolution are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNC23cJr_lj7Vg90H21EOLACvtfcHk0lih3SERvu4GAw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXps1SqtL8%253D&amp;md5=72cac76e7b42521a7dfc82af3d8bb513</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1039%2FC9NP00004F&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9NP00004F%26sid%3Dliteratum%253Aachs%26aulast%3DWolfender%26aufirst%3DJ.-L.%26aulast%3DLitaudon%26aufirst%3DM.%26aulast%3DTouboul%26aufirst%3DD.%26aulast%3DQueiroz%26aufirst%3DE.%2BF.%26atitle%3DInnovative%2520Omics-Based%2520Approaches%2520for%2520Prioritisation%2520and%2520Targeted%2520Isolation%2520of%2520Natural%2520Products%2520%25E2%2580%2593%2520New%2520Strategies%2520for%2520Drug%2520Discovery%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2019%26volume%3D36%26spage%3D855%26epage%3D868%26doi%3D10.1039%2FC9NP00004F" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nothias, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nothias-Esposito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">da Silva, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protsyuk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarvepalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touboul, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Bioactivity-Based Molecular Networking for the Discovery of Drug Leads in Natural Product Bioassay-Guided Fractionation</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">758</span>â <span class="NLM_lpage">767</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.jnatprod.7b00737</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.7b00737" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjs1Snt7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=81&amp;publication_year=2018&amp;pages=758-767&amp;author=L.+F.+Nothiasauthor=M.+Nothias-Espositoauthor=R.+da+Silvaauthor=M.+Wangauthor=I.+Protsyukauthor=Z.+Zhangauthor=A.+Sarvepalliauthor=P.+Leyssenauthor=D.+Touboulauthor=J.+Costa&amp;title=Bioactivity-Based+Molecular+Networking+for+the+Discovery+of+Drug+Leads+in+Natural+Product+Bioassay-Guided+Fractionation&amp;doi=10.1021%2Facs.jnatprod.7b00737" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactivity-Based Molecular Networking for the Discovery of Drug Leads in Natural Product Bioassay-Guided Fractionation</span></div><div class="casAuthors">Nothias, Louis-Felix; Nothias-Esposito, Melissa; da Silva, Ricardo; Wang, Mingxun; Protsyuk, Ivan; Zhang, Zheng; Sarvepalli, Abi; Leyssen, Pieter; Touboul, David; Costa, Jean; Paolini, Julien; Alexandrov, Theodore; Litaudon, Marc; Dorrestein, Pieter C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">758-767</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">It is a common problem in natural product therapeutic lead discovery programs that despite good bioassay results in the initial ext., the active compd.(s) may not be isolated during subsequent bioassay-guided purifn.  Herein, we present the concept of bioactive mol. networking to find candidate active mols. directly from fractionated bioactive exts.  By employing tandem mass spectrometry, it is possible to accelerate the dereplication of mols. using mol. networking prior to subsequent isolation of the compds., and it is also possible to expose potentially bioactive mols. using bioactivity score prediction.  Indeed, bioactivity score prediction can be calcd. with the relative abundance of a mol. in fractions and the bioactivity level of each fraction.  For that reason, we have developed a bioinformatic workflow able to map bioactivity score in mol. networks and applied it for discovery of antiviral compds. from a previously investigated ext. of Euphorbia dendroides where the bioactive candidate mols. were not discovered following a classical bioassay-guided fractionation procedure.  It can be expected that this approach will be implemented as a systematic strategy, not only in current and future bioactive lead discovery from natural ext. collections but also for the reinvestigation of the untapped reservoir of bioactive analogs in previous bioassay-guided fractionation efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq34cZPrBsZ67Vg90H21EOLACvtfcHk0lih3SERvu4GAw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXjs1Snt7g%253D&amp;md5=3a945a4d97b5407ac5b22fce75acf50e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.7b00737&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.7b00737%26sid%3Dliteratum%253Aachs%26aulast%3DNothias%26aufirst%3DL.%2BF.%26aulast%3DNothias-Esposito%26aufirst%3DM.%26aulast%3Dda%2BSilva%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DProtsyuk%26aufirst%3DI.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DSarvepalli%26aufirst%3DA.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DTouboul%26aufirst%3DD.%26aulast%3DCosta%26aufirst%3DJ.%26atitle%3DBioactivity-Based%2520Molecular%2520Networking%2520for%2520the%2520Discovery%2520of%2520Drug%2520Leads%2520in%2520Natural%2520Product%2520Bioassay-Guided%2520Fractionation%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2018%26volume%3D81%26spage%3D758%26epage%3D767%26doi%3D10.1021%2Facs.jnatprod.7b00737" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ory, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazih, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daoud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mocquard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourjot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margueritte, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delsuc, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bard, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pouchus, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Targeting Bioactive Compounds in Natural Extracts - Development of a Comprehensive Workflow Combining Chemical and Biological Data</span>. <i>Anal. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1070</i></span>,  <span class="NLM_fpage">29</span>â <span class="NLM_lpage">42</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.aca.2019.04.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1016%2Fj.aca.2019.04.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=31103165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovVOmsLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1070&amp;publication_year=2019&amp;pages=29-42&amp;author=L.+Oryauthor=E.+H.+Nazihauthor=S.+Daoudauthor=J.+Mocquardauthor=M.+Bourjotauthor=L.+Margueritteauthor=M.+A.+Delsucauthor=J.+M.+Bardauthor=Y.+F.+Pouchusauthor=S.+Bertrand&amp;title=Targeting+Bioactive+Compounds+in+Natural+Extracts+-+Development+of+a+Comprehensive+Workflow+Combining+Chemical+and+Biological+Data&amp;doi=10.1016%2Fj.aca.2019.04.038" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bioactive compounds in natural extracts - Development of a comprehensive workflow combining chemical and biological data</span></div><div class="casAuthors">Ory, Lucie; Nazih, El-Hassane; Daoud, Sahar; Mocquard, Julia; Bourjot, Melanie; Margueritte, Laure; Delsuc, Marc-Andre; Bard, Jean-Marie; Pouchus, Yves Francois; Bertrand, Samuel; Roullier, Catherine</div><div class="citationInfo"><span class="NLM_cas:title">Analytica Chimica Acta</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1070</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">29-42</span>CODEN:
                <span class="NLM_cas:coden">ACACAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2670</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In natural product drug discovery, several strategies have emerged to highlight specifically bioactive compd.(s) within complex mixts. (fractions or crude exts.) using metabolomics tools.  In this area, a great deal of interest has raised among the scientific community on strategies to link chem. profiles and assocd. biol. data, leading to the new field called "biochemometrics".  This article falls into this emerging research by proposing a complete workflow, which was divided into three major steps.  The first one consists in the fractionation of the same ext. using four different chromatog. stationary phases and appropriated elution conditions to obtain five fractions for each column.  The second step corresponds to the acquisition of chem. profiles using HPLC-HRMS anal., and the biol. evaluation of each fraction.  The last step evaluates the links between the relative abundances of mols. present in fractions (peak area) and the global bioactivity level obsd. for each fraction.  To this purpose, an original bioinformatics script (encoded with R Studio software) using the combination of four statistical models (Spearman, F-PCA, PLS, PLS-DA) was here developed giving a "Super list" of potential bioactive compds. together with a predictive score.  This strategy was validated by its application on a marine-derived Penicillium chrysogenum ext. exhibiting antiproliferative activity on breast cancer cells (MCF-7 cells).  After the three steps of the workflow, one main compd. was highlighted as responsible for the bioactivity and identified as ergosterol.  Its antiproliferative activity was confirmed with an IC50 of 0.10 Î¼M on MCF-7 cells.  The script efficiency was further demonstrated by comparing the results obtained with a different recently described approach based on NMR profiling and by virtually modifying the data to evaluate the computational tool behavior.  This approach represents a new and efficient tool to tackle some of the bottlenecks in natural product drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA_ZskSnWKW7Vg90H21EOLACvtfcHk0lg89UpDxVCN1w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXovVOmsLw%253D&amp;md5=af32e028880d6391d11fa24c3971f8aa</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.aca.2019.04.038&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.aca.2019.04.038%26sid%3Dliteratum%253Aachs%26aulast%3DOry%26aufirst%3DL.%26aulast%3DNazih%26aufirst%3DE.%2BH.%26aulast%3DDaoud%26aufirst%3DS.%26aulast%3DMocquard%26aufirst%3DJ.%26aulast%3DBourjot%26aufirst%3DM.%26aulast%3DMargueritte%26aufirst%3DL.%26aulast%3DDelsuc%26aufirst%3DM.%2BA.%26aulast%3DBard%26aufirst%3DJ.%2BM.%26aulast%3DPouchus%26aufirst%3DY.%2BF.%26aulast%3DBertrand%26aufirst%3DS.%26atitle%3DTargeting%2520Bioactive%2520Compounds%2520in%2520Natural%2520Extracts%2520-%2520Development%2520of%2520a%2520Comprehensive%2520Workflow%2520Combining%2520Chemical%2520and%2520Biological%2520Data%26jtitle%3DAnal.%2520Chim.%2520Acta%26date%3D2019%26volume%3D1070%26spage%3D29%26epage%3D42%26doi%3D10.1016%2Fj.aca.2019.04.038" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olivon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allard, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koval, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Righi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genta-Jouve, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannecouque, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nothias, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cachet, X.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Bioactive Natural Products Prioritization Using Massive Multi-Informational Molecular Networks</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2644</span>â <span class="NLM_lpage">2651</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acschembio.7b00413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKntr%252FP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2017&amp;pages=2644-2651&amp;author=F.+Olivonauthor=P.+M.+Allardauthor=A.+Kovalauthor=D.+Righiauthor=G.+Genta-Jouveauthor=J.+Neytsauthor=C.+Apelauthor=C.+Pannecouqueauthor=L.+F.+Nothiasauthor=X.+Cachet&amp;title=Bioactive+Natural+Products+Prioritization+Using+Massive+Multi-Informational+Molecular+Networks&amp;doi=10.1021%2Facschembio.7b00413" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactive Natural Products Prioritization Using Massive Multi-informational Molecular Networks</span></div><div class="casAuthors">Olivon, Florent; Allard, Pierre-Marie; Koval, Alexey; Righi, Davide; Genta-Jouve, Gregory; Neyts, Johan; Apel, Cecile; Pannecouque, Christophe; Nothias, Louis-Felix; Cachet, Xavier; Marcourt, Laurence; Roussi, Fanny; Katanaev, Vladimir L.; Touboul, David; Wolfender, Jean-Luc; Litaudon, Marc</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2644-2651</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Natural products represent an inexhaustible source of novel therapeutic agents.  Their complex and constrained three-dimensional structures endow these mols. with exceptional biol. properties, thereby giving them a major role in drug discovery programs.  However, the search for new bioactive metabolites is hampered by the chem. complexity of the biol. matrixes in which they are found.  The purifn. of single constituents from such matrixes requires such a significant amt. of work that it should be ideally performed only on mols. of high potential value (i.e., chem. novelty and biol. activity).  Recent bioinformatics approaches based on mass spectrometry metabolite profiling methods are beginning to address the complex task of compd. identification within complex mixts.  However, in parallel to these developments, methods providing information on the bioactivity potential of natural products prior to their isolation are still lacking and are of key interest to target the isolation of valuable natural products only.  In the present investigation, we propose an integrated anal. strategy for bioactive natural products prioritization.  Our approach uses massive mol. networks embedding various informational layers (bioactivity and taxonomical data) to highlight potentially bioactive scaffolds within the chem. diversity of crude exts. collections.  We exemplify this workflow by targeting the isolation of predicted active and nonactive metabolites from two botanical sources (Bocquillonia nervosa and Neoguillauminia cleopatra) against two biol. targets (Wnt signaling pathway and chikungunya virus replication).  Eventually, the detection and isolation processes of a daphnane diterpene orthoester and four 12-deoxyphorbols inhibiting the Wnt signaling pathway and exhibiting potent antiviral activities against the CHIKV virus are detailed.  Combined with efficient metabolite annotation tools, this bioactive natural products prioritization pipeline proves to be efficient.  Implementation of this approach in drug discovery programs based on natural ext. screening should speed up and rationalize the isolation of bioactive natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsOBSoAGjVzLVg90H21EOLACvtfcHk0lg89UpDxVCN1w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKntr%252FP&amp;md5=edb6828f781687eb1a6f9ab83497b24b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00413&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00413%26sid%3Dliteratum%253Aachs%26aulast%3DOlivon%26aufirst%3DF.%26aulast%3DAllard%26aufirst%3DP.%2BM.%26aulast%3DKoval%26aufirst%3DA.%26aulast%3DRighi%26aufirst%3DD.%26aulast%3DGenta-Jouve%26aufirst%3DG.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DApel%26aufirst%3DC.%26aulast%3DPannecouque%26aufirst%3DC.%26aulast%3DNothias%26aufirst%3DL.%2BF.%26aulast%3DCachet%26aufirst%3DX.%26atitle%3DBioactive%2520Natural%2520Products%2520Prioritization%2520Using%2520Massive%2520Multi-Informational%2520Molecular%2520Networks%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2644%26epage%3D2651%26doi%3D10.1021%2Facschembio.7b00413" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kurita, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glassey, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span> <span> </span><span class="NLM_article-title">Integration of High-Content Screening and Untargeted Metabolomics for Comprehensive Functional Annotation of Natural Product Libraries</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">11999</span>â <span class="NLM_lpage">12004</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1507743112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1073%2Fpnas.1507743112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=26371303" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2ntLnO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=112&amp;publication_year=2015&amp;pages=11999-12004&amp;author=K.+L.+Kuritaauthor=E.+Glasseyauthor=R.+G.+Linington&amp;title=Integration+of+High-Content+Screening+and+Untargeted+Metabolomics+for+Comprehensive+Functional+Annotation+of+Natural+Product+Libraries&amp;doi=10.1073%2Fpnas.1507743112" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Integration of high-content screening and untargeted metabolomics for comprehensive functional annotation of natural product libraries</span></div><div class="casAuthors">Kurita, Kenji L.; Glassey, Emerson; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">11999-12004</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Traditional natural products discovery using a combination of live/dead screening followed by iterative bioassay-guided fractionation affords no information about compd. structure or mode of action until late in the discovery process.  This leads to high rates of rediscovery and low probabilities of finding compds. with unique biol. and/or chem. properties.  By integrating image-based phenotypic screening in HeLa cells with high-resoln. untargeted metabolomics anal., we have developed a new platform, termed Compd. Activity Mapping, that is capable of directly predicting the identities and modes of action of bioactive constituents for any complex natural product ext. library.  This new tool can be used to rapidly identify novel bioactive constituents and provide predictions of compd. modes of action directly from primary screening data.  This approach inverts the natural products discovery process from the existing "grind and find" model to a targeted, hypothesis-driven discovery model where the chem. features and biol. function of bioactive metabolites are known early in the screening workflow, and lead compds. can be rationally selected based on biol. and/or chem. novelty.  We demonstrate the utility of the Compd. Activity Mapping platform by combining 10,977 mass spectral features and 58,032 biol. measurements from a library of 234 natural products exts. and integrating these two datasets to identify 13 clusters of fractions contg. 11 known compd. families and four new compds.  Using Compd. Activity Mapping we discovered the quinocinnolinomycins, a new family of natural products with a unique carbon skeleton that cause endoplasmic reticulum stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2SjVGolhWsLVg90H21EOLACvtfcHk0liU6s6EyPtGVg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2ntLnO&amp;md5=36b4c261ec51d891a3818ba99dd0af6e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1507743112&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1507743112%26sid%3Dliteratum%253Aachs%26aulast%3DKurita%26aufirst%3DK.%2BL.%26aulast%3DGlassey%26aufirst%3DE.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26atitle%3DIntegration%2520of%2520High-Content%2520Screening%2520and%2520Untargeted%2520Metabolomics%2520for%2520Comprehensive%2520Functional%2520Annotation%2520of%2520Natural%2520Product%2520Libraries%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D11999%26epage%3D12004%26doi%3D10.1073%2Fpnas.1507743112" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levander, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shofstahl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RÃ¶mpp, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pizarro, A. D.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">MzML - A Community Standard for Mass Spectrometry Data</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">R110.000133</span>, <span class="refDoi">&nbsp;DOI: 10.1074/mcp.R110.000133</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1074%2Fmcp.R110.000133" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=20716697" title="">PubMed</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2011&amp;pages=R110.000133&amp;author=L.+Martensauthor=M.+Chambersauthor=M.+Sturmauthor=D.+Kessnerauthor=F.+Levanderauthor=J.+Shofstahlauthor=W.+H.+Tangauthor=A.+R%C3%B6mppauthor=S.+Neumannauthor=A.+D.+Pizarro&amp;title=MzML+-+A+Community+Standard+for+Mass+Spectrometry+Data&amp;doi=10.1074%2Fmcp.R110.000133" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fmcp.R110.000133&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.R110.000133%26sid%3Dliteratum%253Aachs%26aulast%3DMartens%26aufirst%3DL.%26aulast%3DChambers%26aufirst%3DM.%26aulast%3DSturm%26aufirst%3DM.%26aulast%3DKessner%26aufirst%3DD.%26aulast%3DLevander%26aufirst%3DF.%26aulast%3DShofstahl%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DW.%2BH.%26aulast%3DR%25C3%25B6mpp%26aufirst%3DA.%26aulast%3DNeumann%26aufirst%3DS.%26aulast%3DPizarro%26aufirst%3DA.%2BD.%26atitle%3DMzML%2520-%2520A%2520Community%2520Standard%2520for%2520Mass%2520Spectrometry%2520Data%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2011%26volume%3D10%26spage%3DR110.000133%26doi%3D10.1074%2Fmcp.R110.000133" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pluskal, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castillo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villar-Briones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OreÅ¡iÄ, M.</span></span> <span> </span><span class="NLM_article-title">MZmine 2: Modular Framework for Processing, Visualizing, and Analyzing Mass Spectrometry-Based Molecular Profile Data</span>. <i>BMC Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">395</span>, <span class="refDoi">&nbsp;DOI: 10.1186/1471-2105-11-395</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1186%2F1471-2105-11-395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=20650010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A280%3ADC%252BC3cjjsVymsA%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2010&amp;pages=395&amp;author=T.+Pluskalauthor=S.+Castilloauthor=A.+Villar-Brionesauthor=M.+Ore%C5%A1i%C4%8D&amp;title=MZmine+2%3A+Modular+Framework+for+Processing%2C+Visualizing%2C+and+Analyzing+Mass+Spectrometry-Based+Molecular+Profile+Data&amp;doi=10.1186%2F1471-2105-11-395" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">MZmine 2: modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data</span></div><div class="casAuthors">Pluskal Tomas; Castillo Sandra; Villar-Briones Alejandro; Oresic Matej</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">395</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Mass spectrometry (MS) coupled with online separation methods is commonly applied for differential and quantitative profiling of biological samples in metabolomic as well as proteomic research.  Such approaches are used for systems biology, functional genomics, and biomarker discovery, among others.  An ongoing challenge of these molecular profiling approaches, however, is the development of better data processing methods.  Here we introduce a new generation of a popular open-source data processing toolbox, MZmine 2.  RESULTS:  A key concept of the MZmine 2 software design is the strict separation of core functionality and data processing modules, with emphasis on easy usability and support for high-resolution spectra processing.  Data processing modules take advantage of embedded visualization tools, allowing for immediate previews of parameter settings.  Newly introduced functionality includes the identification of peaks using online databases, MSn data support, improved isotope pattern support, scatter plot visualization, and a new method for peak list alignment based on the random sample consensus (RANSAC) algorithm.  The performance of the RANSAC alignment was evaluated using synthetic datasets as well as actual experimental data, and the results were compared to those obtained using other alignment algorithms.  CONCLUSIONS:  MZmine 2 is freely available under a GNU GPL license and can be obtained from the project website at: http://mzmine.sourceforge.net/.  The current version of MZmine 2 is suitable for processing large batches of data and has been applied to both targeted and non-targeted metabolomic analyses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSPbblHwbnggjUE-swuRqsHfW6udTcc2ebMWbAWJsuXmLntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC3cjjsVymsA%253D%253D&amp;md5=e6e2ac996767f8526daccbdb7f4929e0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-11-395&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-11-395%26sid%3Dliteratum%253Aachs%26aulast%3DPluskal%26aufirst%3DT.%26aulast%3DCastillo%26aufirst%3DS.%26aulast%3DVillar-Briones%26aufirst%3DA.%26aulast%3DOre%25C5%25A1i%25C4%258D%26aufirst%3DM.%26atitle%3DMZmine%25202%253A%2520Modular%2520Framework%2520for%2520Processing%252C%2520Visualizing%252C%2520and%2520Analyzing%2520Mass%2520Spectrometry-Based%2520Molecular%2520Profile%2520Data%26jtitle%3DBMC%2520Bioinformatics%26date%3D2010%26volume%3D11%26spage%3D395%26doi%3D10.1186%2F1471-2105-11-395" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carver, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phelan, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watrous, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapono, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luzzatto-Knaan, T.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Sharing and Community Curation of Mass Spectrometry Data with Global Natural Products Social Molecular Networking</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">828</span>â <span class="NLM_lpage">837</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nbt.3597</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1038%2Fnbt.3597" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=27504778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlaitLnE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=2016&amp;pages=828-837&amp;author=M.+Wangauthor=J.+J.+Carverauthor=V.+V.+Phelanauthor=L.+M.+Sanchezauthor=N.+Gargauthor=Y.+Pengauthor=D.+D.+Nguyenauthor=J.+Watrousauthor=C.+A.+Kaponoauthor=T.+Luzzatto-Knaan&amp;title=Sharing+and+Community+Curation+of+Mass+Spectrometry+Data+with+Global+Natural+Products+Social+Molecular+Networking&amp;doi=10.1038%2Fnbt.3597" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking</span></div><div class="casAuthors">Wang, Mingxun; Carver, Jeremy J.; Phelan, Vanessa V.; Sanchez, Laura M.; Garg, Neha; Peng, Yao; Nguyen, Don Duy; Watrous, Jeramie; Kapono, Clifford A.; Luzzatto-Knaan, Tal; Porto, Carla; Bouslimani, Amina; Melnik, Alexey V.; Meehan, Michael J.; Liu, Wei-Ting; Crusemann, Max; Boudreau, Paul D.; Esquenazi, Eduardo; Sandoval-Calderon, Mario; Kersten, Roland D.; Pace, Laura A.; Quinn, Robert A.; Duncan, Katherine R.; Hsu, Cheng-Chih; Floros, Dimitrios J.; Gavilan, Ronnie G.; Kleigrewe, Karin; Northen, Trent; Dutton, Rachel J.; Parrot, Delphine; Carlson, Erin E.; Aigle, Bertrand; Michelsen, Charlotte F.; Jelsbak, Lars; Sohlenkamp, Christian; Pevzner, Pavel; Edlund, Anna; McLean, Jeffrey; Piel, Jorn; Murphy, Brian T.; Gerwick, Lena; Liaw, Chih-Chuang; Yang, Yu-Liang; Humpf, Hans-Ulrich; Maansson, Maria; Keyzers, Robert A.; Sims, Amy C.; Johnson, Andrew R.; Sidebottom, Ashley M.; Sedio, Brian E.; Klitgaard, Andreas; Larson, Charles B.; Boya P, Cristopher A.; Torres-Mendoza, Daniel; Gonzalez, David J.; Silva, Denise B.; Marques, Lucas M.; Demarque, Daniel P.; Pociute, Egle; O'Neill, Ellis C.; Briand, Enora; Helfrich, Eric J. N.; Granatosky, Eve A.; Glukhov, Evgenia; Ryffel, Florian; Houson, Hailey; Mohimani, Hosein; Kharbush, Jenan J.; Zeng, Yi; Vorholt, Julia A.; Kurita, Kenji L.; Charusanti, Pep; McPhail, Kerry L.; Nielsen, Kristian Fog; Vuong, Lisa; Elfeki, Maryam; Traxler, Matthew F.; Engene, Niclas; Koyama, Nobuhiro; Vining, Oliver B.; Baric, Ralph; Silva, Ricardo R.; Mascuch, Samantha J.; Tomasi, Sophie; Jenkins, Stefan; Macherla, Venkat; Hoffman, Thomas; Agarwal, Vinayak; Williams, Philip G.; Dai, Jingqui; Neupane, Ram; Gurr, Joshua; Rodriguez, Andres M. C.; Lamsa, Anne; Zhang, Chen; Dorrestein, Kathleen; Duggan, Brendan M.; Almaliti, Jehad; Allard, Pierre-Marie; Phapale, Prasad; Nothias, Louis-Felix; Alexandrov, Theodore; Litaudon, Marc; Wolfender, Jean-Luc; Kyle, Jennifer E.; Metz, Thomas O.; Peryea, Tyler; Nguyen, Dac-Trung; Van Leer, Danielle; Shinn, Paul; Jadhav, Ajit; Muller, Rolf; Waters, Katrina M.; Shi, Wenyuan; Liu, Xueting; Zhang, Lixin; Knight, Rob; Jensen, Paul R.; Palsson, Bernhard O.; Pogliano, Kit; Linington, Roger G.; Gutierrez, Marcelino; Lopes, Norberto P.; Gerwick, William H.; Moore, Bradley S.; Dorrestein, Pieter C.; Bandeira, Nuno</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">828-837</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential of the diverse chemistries present in natural products (NP) for biotechnol. and medicine remains untapped because NP databases are not searchable with raw data and the NP community has no way to share data other than in published papers.  Although mass spectrometry (MS) techniques are well-suited to high-throughput characterization of NP, there is a pressing need for an infrastructure to enable sharing and curation of data.  We present Global Natural Products Social Mol. Networking (GNPS; http://gnps.ucsd.edu), an open-access knowledge base for community-wide organization and sharing of raw, processed or identified tandem mass (MS/MS) spectrometry data.  In GNPS, crowdsourced curation of freely available community-wide ref. MS libraries will underpin improved annotations.  Data-driven social-networking should facilitate identification of spectra and foster collaborations.  We also introduce the concept of 'living data' through continuous reanal. of deposited data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptdRCove5agrVg90H21EOLACvtfcHk0liU6s6EyPtGVg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XhtlaitLnE&amp;md5=e6ca23ede2d85dd1460a5d73da542444</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3597&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3597%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%26aulast%3DCarver%26aufirst%3DJ.%2BJ.%26aulast%3DPhelan%26aufirst%3DV.%2BV.%26aulast%3DSanchez%26aufirst%3DL.%2BM.%26aulast%3DGarg%26aufirst%3DN.%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DNguyen%26aufirst%3DD.%2BD.%26aulast%3DWatrous%26aufirst%3DJ.%26aulast%3DKapono%26aufirst%3DC.%2BA.%26aulast%3DLuzzatto-Knaan%26aufirst%3DT.%26atitle%3DSharing%2520and%2520Community%2520Curation%2520of%2520Mass%2520Spectrometry%2520Data%2520with%2520Global%2520Natural%2520Products%2520Social%2520Molecular%2520Networking%26jtitle%3DNat.%2520Biotechnol.%26date%3D2016%26volume%3D34%26spage%3D828%26epage%3D837%26doi%3D10.1038%2Fnbt.3597" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woerhmann, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span> <span> </span><span class="NLM_article-title">Function-Firstâ Lead Discovery: Mode of Action Profiling of Natural Product Libraries Using Image-Based Screening</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">285</span>â <span class="NLM_lpage">295</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.chembiol.2012.12.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1016%2Fj.chembiol.2012.12.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=23438757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFSlu7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2013&amp;pages=285-295&amp;author=C.+J.+Schulzeauthor=W.+M.+Brayauthor=M.+H.+Woerhmannauthor=J.+Stuartauthor=R.+S.+Lokeyauthor=R.+G.+Linington&amp;title=Function-First%E2%80%9D+Lead+Discovery%3A+Mode+of+Action+Profiling+of+Natural+Product+Libraries+Using+Image-Based+Screening&amp;doi=10.1016%2Fj.chembiol.2012.12.007" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">"Function-First" Lead Discovery: Mode of Action Profiling of Natural Product Libraries Using Image-Based Screening</span></div><div class="casAuthors">Schulze, Christopher J.; Bray, Walter M.; Woerhmann, Marcos H.; Stuart, Joshua; Lokey, R. Scott; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry &amp; Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">285-295</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cytol. profiling is a high-content image-based screening technol. that provides insight into the mode of action (MOA) for test compds. by directly measuring hundreds of phenotypic cellular features.  We have extended this recently reported technol. to the mechanistic characterization of unknown natural products libraries for the direct prediction of compd. MOAs at the primary screening stage.  By analyzing a training set of com. compds. of known mechanism and comparing these profiles to those obtained from natural product library members, we have successfully annotated exts. based on MOA, dereplicated known compds. based on biol. similarity to the training set, and identified and predicted the MOA of a unique family of iron siderophores.  Coupled with traditional anal. techniques, cytol. profiling provides an avenue for the creation of "function-first" approaches to natural products discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmxEmwpkTYBbVg90H21EOLACvtfcHk0liVp3s8dE5SPw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXjtFSlu7g%253D&amp;md5=8bec654bcd3ae6b1a93156a8935f841f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.12.007&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.12.007%26sid%3Dliteratum%253Aachs%26aulast%3DSchulze%26aufirst%3DC.%2BJ.%26aulast%3DBray%26aufirst%3DW.%2BM.%26aulast%3DWoerhmann%26aufirst%3DM.%2BH.%26aulast%3DStuart%26aufirst%3DJ.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26atitle%3DFunction-First%25E2%2580%259D%2520Lead%2520Discovery%253A%2520Mode%2520of%2520Action%2520Profiling%2520of%2520Natural%2520Product%2520Libraries%2520Using%2520Image-Based%2520Screening%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D285%26epage%3D295%26doi%3D10.1016%2Fj.chembiol.2012.12.007" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guttman, A.</span></span> <span> </span><span class="NLM_article-title">R-Trees: A Dynamic Index Structure for Spatial Searching</span>. <i>SIGMOD Rec.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">47</span>â <span class="NLM_lpage">57</span>, <span class="refDoi">&nbsp;DOI: 10.1145/971697.602266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1145%2F971697.602266" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=1984&amp;pages=47-57&amp;author=A.+Guttman&amp;title=R-Trees%3A+A+Dynamic+Index+Structure+for+Spatial+Searching&amp;doi=10.1145%2F971697.602266" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1145%2F971697.602266&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1145%252F971697.602266%26sid%3Dliteratum%253Aachs%26aulast%3DGuttman%26aufirst%3DA.%26atitle%3DR-Trees%253A%2520A%2520Dynamic%2520Index%2520Structure%2520for%2520Spatial%2520Searching%26jtitle%3DSIGMOD%2520Rec.%26date%3D1984%26volume%3D14%26spage%3D47%26epage%3D57%26doi%3D10.1145%2F971697.602266" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span> <span> </span><span class="NLM_article-title">Development of Antibiotic Activity Profile Screening for the Classification and Discovery of Natural Product Antibiotics</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1483</span>â <span class="NLM_lpage">1495</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.chembiol.2012.09.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1016%2Fj.chembiol.2012.09.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=23177202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslejur%252FP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2012&amp;pages=1483-1495&amp;author=W.+R.+Wongauthor=A.+G.+Oliverauthor=R.+G.+Linington&amp;title=Development+of+Antibiotic+Activity+Profile+Screening+for+the+Classification+and+Discovery+of+Natural+Product+Antibiotics&amp;doi=10.1016%2Fj.chembiol.2012.09.014" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Antibiotic Activity Profile Screening for the Classification and Discovery of Natural Product Antibiotics</span></div><div class="casAuthors">Wong, Weng Ruh; Oliver, Allen G.; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry &amp; Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1483-1495</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Despite recognition of the looming antibiotic crisis by healthcare professionals, the no. of new antibiotics reaching the clinic continues to decline sharply.  This study aimed to establish an antibiotic profiling strategy using a panel of clin. relevant bacterial strains to create unique biol. fingerprints for all major classes of antibiotics.  Antibiotic mode of action profile (BioMAP) screening has been shown to effectively cluster antibiotics by structural class based on these fingerprints.  Using this approach, we have accurately predicted the presence of known antibiotics in natural product exts. and have discovered a naphthoquinone-based antibiotic from our marine natural product library that possesses a unique carbon skeleton.  We have demonstrated that bioactivity fingerprinting is a successful strategy for profiling antibiotic lead compds. and that BioMAP can be applied to the discovery of new natural product antibiotics leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9yIDXE5qmhLVg90H21EOLACvtfcHk0liVp3s8dE5SPw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38Xhslejur%252FP&amp;md5=7817766457640524ce86f1d8476a3156</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.09.014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.09.014%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DW.%2BR.%26aulast%3DOliver%26aufirst%3DA.%2BG.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26atitle%3DDevelopment%2520of%2520Antibiotic%2520Activity%2520Profile%2520Screening%2520for%2520the%2520Classification%2520and%2520Discovery%2520of%2520Natural%2520Product%2520Antibiotics%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D1483%26epage%3D1495%26doi%3D10.1016%2Fj.chembiol.2012.09.014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Santen, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aniebok, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balunas, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunsko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neto, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">CastaÃ±o-Espriu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T. N.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">The Natural Products Atlas: An Open Access Knowledge Base for Microbial Natural Products Discovery</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1824</span>â <span class="NLM_lpage">1833</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscentsci.9b00806</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.9b00806" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCit73P" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2019&amp;pages=1824-1833&amp;author=J.+A.+van%0ASantenauthor=G.+Jacobauthor=A.+L.+Singhauthor=V.+Aniebokauthor=M.+J.+Balunasauthor=D.+Bunskoauthor=F.+C.+Netoauthor=L.+Casta%C3%B1o-Espriuauthor=C.+Changauthor=T.+N.+Clark&amp;title=The+Natural+Products+Atlas%3A+An+Open+Access+Knowledge+Base+for+Microbial+Natural+Products+Discovery&amp;doi=10.1021%2Facscentsci.9b00806" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The Natural Products Atlas: An Open Access Knowledge Base for Microbial Natural Products Discovery</span></div><div class="casAuthors">van Santen, Jeffrey A.; Jacob, Gregoire; Singh, Amrit Leen; Aniebok, Victor; Balunas, Marcy J.; Bunsko, Derek; Neto, Fausto Carnevale; Castano-Espriu, Laia; Chang, Chen; Clark, Trevor N.; Cleary Little, Jessica L.; Delgadillo, David A.; Dorrestein, Pieter C.; Duncan, Katherine R.; Egan, Joseph M.; Galey, Melissa M.; Haeckl, F. P. Jake; Hua, Alex; Hughes, Alison H.; Iskakova, Dasha; Khadilkar, Aswad; Lee, Jung-Ho; Lee, Sanghoon; LeGrow, Nicole; Liu, Dennis Y.; Macho, Jocelyn M.; McCaughey, Catherine S.; Medema, Marnix H.; Neupane, Ram P.; ODonnell, Timothy J.; Paula, Jasmine S.; Sanchez, Laura M.; Shaikh, Anam F.; Soldatou, Sylvia; Terlouw, Barbara R.; Tran, Tuan Anh; Valentine, Mercia; van der Hooft, Justin J. J.; Vo, Duy A.; Wang, Mingxun; Wilson, Darryl; Zink, Katherine E.; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1824-1833</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Despite rapid evolution in the area of microbial natural products chem., there is currently no open access database contg. all microbially produced natural product structures.  Lack of availability of these data is preventing the implementation of new technologies in natural products science.  Specifically, development of new computational strategies for compd. characterization and identification are being hampered by the lack of a comprehensive database of known compds. against which to compare exptl. data.  The creation of an open access, community-maintained database of microbial natural product structures would enable the development of new technologies in natural products discovery, and improve the interoperability of existing natural products data resources.  However, these data are spread unevenly throughout the historical scientific literature, including both journal articles and international patents.  These documents have no std. format, are often not digitized as machine readable text, and are not publicly available.  Further, none of these documents have assocd. structure files (e.g., MOL, InChI, or SMILES), instead contg. images of structures.  This makes extn. and formatting of relevant natural products data a formidable challenge.  Using a combination of manual curation and automated data mining approaches we have created a database of microbial natural products (The Natural Products Atlas, www.npatlas.org) that includes 24,594 compds. and contains referenced data for structure, compd. names, source organisms, isolation refs., total syntheses and instances of structural reassignment.  This database is accompanied by an interactive web portal that permits searching by structure, substructure, and phys. properties.  The Web site also provides mechanisms for visualizing natural products chem. space, and dashboards for displaying author and discovery timeline data.  These interactive tools offer a powerful knowledge base for natural products discovery with a central interface for structure and property-based searching, and presents new viewpoints on structural diversity in natural products.  The Natural Products Atlas has been developed under FAIR principles (Findable, Accessible, Interoperable, and Reusable) and is integrated with other emerging natural product databases, including the Min. Information About a Biosynthetic Gene Cluster (MIBiG) repository, and the Global Natural Products Social Mol. Networking (GNPS) platform.  It is designed as a community-supported resource to provide a central repository for known natural product structures from microorganisms, and is the 1st comprehensive, open access resource of this type.  It is expected that the Natural Products Atlas will enable the development of new natural products discovery modalities and accelerate the process of structural characterization for complex natural products libraries.  The Natural Products Atlas is a new online database of microbially derived natural product structures, designed as a comprehensive open access repository for the scientific community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv6naGjGM-w7Vg90H21EOLACvtfcHk0ljXJN99A2BYDQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXitFCit73P&amp;md5=1ebc951ca96d93dd36d65dd8b3b3ad67</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.9b00806&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.9b00806%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BSanten%26aufirst%3DJ.%2BA.%26aulast%3DJacob%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DA.%2BL.%26aulast%3DAniebok%26aufirst%3DV.%26aulast%3DBalunas%26aufirst%3DM.%2BJ.%26aulast%3DBunsko%26aufirst%3DD.%26aulast%3DNeto%26aufirst%3DF.%2BC.%26aulast%3DCasta%25C3%25B1o-Espriu%26aufirst%3DL.%26aulast%3DChang%26aufirst%3DC.%26aulast%3DClark%26aufirst%3DT.%2BN.%26atitle%3DThe%2520Natural%2520Products%2520Atlas%253A%2520An%2520Open%2520Access%2520Knowledge%2520Base%2520for%2520Microbial%2520Natural%2520Products%2520Discovery%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2019%26volume%3D5%26spage%3D1824%26epage%3D1833%26doi%3D10.1021%2Facscentsci.9b00806" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skellam, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strangman, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. L. C.</span></span> <span> </span><span class="NLM_article-title">Identification of Micromonolactam, a New Polyene Macrocyclic Lactam from Two Marine Micromonospora Strains Using Chemical and Molecular Methods: Clarification of the Biosynthetic Pathway from a Glutamate Starter Unit</span>. <i>J. Antibiot. (Tokyo)</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">431</span>â <span class="NLM_lpage">441</span>, <span class="refDoi">&nbsp;DOI: 10.1038/ja.2013.34</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1038%2Fja.2013.34" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=23677034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A280%3ADC%252BC3snktVejsg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2013&amp;pages=431-441&amp;author=E.+J.+Skellamauthor=A.+K.+Stewartauthor=W.+K.+Strangmanauthor=J.+L.+C.+Wright&amp;title=Identification+of+Micromonolactam%2C+a+New+Polyene+Macrocyclic+Lactam+from+Two+Marine+Micromonospora+Strains+Using+Chemical+and+Molecular+Methods%3A+Clarification+of+the+Biosynthetic+Pathway+from+a+Glutamate+Starter+Unit&amp;doi=10.1038%2Fja.2013.34" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of micromonolactam, a new polyene macrocyclic lactam from two marine Micromonospora strains using chemical and molecular methods: clarification of the biosynthetic pathway from a glutamate starter unit</span></div><div class="casAuthors">Skellam Elizabeth Julia; Stewart Allison Kathleen; Strangman Wendy Karen; Wright Jeffrey Lawson Cameron</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of antibiotics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">431-41</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Through a combination of chemical and molecular analysis, a new polyene macrolactam named micromonolactam was obtained from two marine-derived Micromonospora species.  This new polyene metabolite is a constitutional isomer of salinilactam A but contains a different polyene pattern and one cis double bond, in contrast to the all trans structure reported for salinilactam A.  The molecular analysis data also established that micromonolactam is a hybrid polyketide derived from 11 polyketide units and a modified glutamate starter unit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQC-9nbPIe_2TToTC3aBaesfW6udTcc2eZwQBsVnjtXQLntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC3snktVejsg%253D%253D&amp;md5=8b07f2b229caebef239fd2966a368125</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fja.2013.34&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2013.34%26sid%3Dliteratum%253Aachs%26aulast%3DSkellam%26aufirst%3DE.%2BJ.%26aulast%3DStewart%26aufirst%3DA.%2BK.%26aulast%3DStrangman%26aufirst%3DW.%2BK.%26aulast%3DWright%26aufirst%3DJ.%2BL.%2BC.%26atitle%3DIdentification%2520of%2520Micromonolactam%252C%2520a%2520New%2520Polyene%2520Macrocyclic%2520Lactam%2520from%2520Two%2520Marine%2520Micromonospora%2520Strains%2520Using%2520Chemical%2520and%2520Molecular%2520Methods%253A%2520Clarification%2520of%2520the%2520Biosynthetic%2520Pathway%2520from%2520a%2520Glutamate%2520Starter%2520Unit%26jtitle%3DJ.%2520Antibiot.%2520%2528Tokyo%2529%26date%3D2013%26volume%3D66%26spage%3D431%26epage%3D441%26doi%3D10.1038%2Fja.2013.34" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoshino, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awakawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asamizu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, I.</span></span> <span> </span><span class="NLM_article-title">Mycolic Acid Containing Bacterium Stimulates Tandem Cyclization of Polyene Macrolactam in a Lake Sediment Derived Rare Actinomycete</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4992</span>â <span class="NLM_lpage">4995</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.orglett.7b02508</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.7b02508" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKmsrzI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2017&amp;pages=4992-4995&amp;author=S.+Hoshinoauthor=M.+Okadaauthor=T.+Awakawaauthor=S.+Asamizuauthor=H.+Onakaauthor=I.+Abe&amp;title=Mycolic+Acid+Containing+Bacterium+Stimulates+Tandem+Cyclization+of+Polyene+Macrolactam+in+a+Lake+Sediment+Derived+Rare+Actinomycete&amp;doi=10.1021%2Facs.orglett.7b02508" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Mycolic Acid Containing Bacterium Stimulates Tandem Cyclization of Polyene Macrolactam in a Lake Sediment Derived Rare Actinomycete</span></div><div class="casAuthors">Hoshino, Shotaro; Okada, Masahiro; Awakawa, Takayoshi; Asamizu, Shumpei; Onaka, Hiroyasu; Abe, Ikuro</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4992-4995</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two novel macrolactams, dracolactams A and B, were identified from a combined-culture of Micromonospora species and a mycolic-acid contg. bacterium (MACB).  Their structures and stereochemistries were completely assigned, based on spectroscopic analyses and chem. derivatization.  Both dracolactams were probably generated from a common macrolactam precursor produced by the Micromonospora species.  In this combined-culture system, MACB is likely to activate cryptic oxidase genes in the Micromonospora species and induce the downstream polyene macrolactam cyclization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphxcpmQ0K137Vg90H21EOLACvtfcHk0ljXJN99A2BYDQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKmsrzI&amp;md5=e75ba729cd49df12aeef4bf87ecf9f8e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.7b02508&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.7b02508%26sid%3Dliteratum%253Aachs%26aulast%3DHoshino%26aufirst%3DS.%26aulast%3DOkada%26aufirst%3DM.%26aulast%3DAwakawa%26aufirst%3DT.%26aulast%3DAsamizu%26aufirst%3DS.%26aulast%3DOnaka%26aufirst%3DH.%26aulast%3DAbe%26aufirst%3DI.%26atitle%3DMycolic%2520Acid%2520Containing%2520Bacterium%2520Stimulates%2520Tandem%2520Cyclization%2520of%2520Polyene%2520Macrolactam%2520in%2520a%2520Lake%2520Sediment%2520Derived%2520Rare%2520Actinomycete%26jtitle%3DOrg.%2520Lett.%26date%3D2017%26volume%3D19%26spage%3D4992%26epage%3D4995%26doi%3D10.1021%2Facs.orglett.7b02508" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyadoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodama, Y.</span></span> <span> </span><span class="NLM_article-title">Novel Antibiotics SF2738A, B and C, and Their Analogs Produced by Streptomyces Sp</span>. <i>J. Antibiot. (Tokyo)</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1385</span>â <span class="NLM_lpage">1394</span>, <span class="refDoi">&nbsp;DOI: 10.7164/antibiotics.47.1385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.7164%2Fantibiotics.47.1385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=7844033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A280%3ADyaK2M7ks1CitQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=1994&amp;pages=1385-1394&amp;author=S.+Gomiauthor=S.+Amanoauthor=E.+Satoauthor=S.+Miyadohauthor=Y.+Kodama&amp;title=Novel+Antibiotics+SF2738A%2C+B+and+C%2C+and+Their+Analogs+Produced+by+Streptomyces+Sp&amp;doi=10.7164%2Fantibiotics.47.1385" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Novel antibiotics SF2738A, B and C, and their analogs produced by Streptomyces sp</span></div><div class="casAuthors">Gomi S; Amano S; Sato E; Miyadoh S; Kodama Y</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of antibiotics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1385-94</span>
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Three new antibiotics SF2738A, B and C, and their analogs were isolated from the culture broth of Streptomyces sp.  The antibiotics are active against Gram-positive bacteria, Gram-negative bacteria and fungi, and exhibited cytotoxic activity against P388 murine leukemia cells with IC50 values of 0.08, 0.25 and 7.5 micrograms/ml, respectively.  Their structures were determined by spectral analyses and chemical conversion.  Especially, the structure of SF2738A was confirmed to be (E)-((4-methoxy-5-methylthio-2-(2-pyridyl)pyridin-6-yl)methylene)azan ol by X-ray crystallographic analysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRU_1c4u7jGETEwO064Z95rfW6udTcc2eaMMLrMfeYv2bntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADyaK2M7ks1CitQ%253D%253D&amp;md5=a5abd0d5ae3537e4f845b6a4af557dfc</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.47.1385&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.47.1385%26sid%3Dliteratum%253Aachs%26aulast%3DGomi%26aufirst%3DS.%26aulast%3DAmano%26aufirst%3DS.%26aulast%3DSato%26aufirst%3DE.%26aulast%3DMiyadoh%26aufirst%3DS.%26aulast%3DKodama%26aufirst%3DY.%26atitle%3DNovel%2520Antibiotics%2520SF2738A%252C%2520B%2520and%2520C%252C%2520and%2520Their%2520Analogs%2520Produced%2520by%2520Streptomyces%2520Sp%26jtitle%3DJ.%2520Antibiot.%2520%2528Tokyo%2529%26date%3D1994%26volume%3D47%26spage%3D1385%26epage%3D1394%26doi%3D10.7164%2Fantibiotics.47.1385" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shindo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamagishi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, H.</span></span> <span> </span><span class="NLM_article-title">Collismycins A and B, Novel Non-Steroidal Inhibitors of Dexamethasone-Glucocorticoid Receptor Binding</span>. <i>J. Antibiot. (Tokyo)</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1072</span>â <span class="NLM_lpage">1074</span>, <span class="refDoi">&nbsp;DOI: 10.7164/antibiotics.47.1072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.7164%2Fantibiotics.47.1072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=7928696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A280%3ADyaK2M%252FhtlOhtQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=1994&amp;pages=1072-1074&amp;author=K.+Shindoauthor=Y.+Yamagishiauthor=Y.+Okadaauthor=H.+Kawai&amp;title=Collismycins+A+and+B%2C+Novel+Non-Steroidal+Inhibitors+of+Dexamethasone-Glucocorticoid+Receptor+Binding&amp;doi=10.7164%2Fantibiotics.47.1072" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Collismycins A and B, novel non-steroidal inhibitors of dexamethasone-glucocorticoid receptor binding</span></div><div class="casAuthors">Shindo K; Yamagishi Y; Okada Y; Kawai H</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of antibiotics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1072-4</span>
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcShmd3Kb6P_ycve_zCu79I8fW6udTcc2eaMMLrMfeYv2bntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADyaK2M%252FhtlOhtQ%253D%253D&amp;md5=b1a9e19078279c9df17ab825fedfd45f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.47.1072&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.47.1072%26sid%3Dliteratum%253Aachs%26aulast%3DShindo%26aufirst%3DK.%26aulast%3DYamagishi%26aufirst%3DY.%26aulast%3DOkada%26aufirst%3DY.%26aulast%3DKawai%26aufirst%3DH.%26atitle%3DCollismycins%2520A%2520and%2520B%252C%2520Novel%2520Non-Steroidal%2520Inhibitors%2520of%2520Dexamethasone-Glucocorticoid%2520Receptor%2520Binding%26jtitle%3DJ.%2520Antibiot.%2520%2528Tokyo%2529%26date%3D1994%26volume%3D47%26spage%3D1072%26epage%3D1074%26doi%3D10.7164%2Fantibiotics.47.1072" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seyedsayamdost, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clardy, J.</span></span> <span> </span><span class="NLM_article-title">Structure and Biosynthesis of Amychelin, an Unusual Mixed-Ligand Siderophore from Amycolatopsis Sp. AA4</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">11434</span>â <span class="NLM_lpage">11437</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja203577e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja203577e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFykt78%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=133&amp;publication_year=2011&amp;pages=11434-11437&amp;author=M.+R.+Seyedsayamdostauthor=M.+F.+Traxlerauthor=S.+L.+Zhengauthor=R.+Kolterauthor=J.+Clardy&amp;title=Structure+and+Biosynthesis+of+Amychelin%2C+an+Unusual+Mixed-Ligand+Siderophore+from+Amycolatopsis+Sp.+AA4&amp;doi=10.1021%2Fja203577e" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Biosynthesis of Amychelin, an Unusual Mixed-Ligand Siderophore from Amycolatopsis sp. AA4</span></div><div class="casAuthors">Seyedsayamdost, Mohammad R.; Traxler, Matthew F.; Zheng, Shao-Liang; Kolter, Roberto; Clardy, Jon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">11434-11437</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Actinobacteria generate a large no. of structurally diverse small mols. with potential therapeutic value.  Genomic analyses of this productive group of bacteria show that their genetic potential to manuf. small mols. exceeds their obsd. ability by roughly an order of magnitude, and this revelation has prompted a no. of studies to identify members of the unknown majority.  As a potential window into this cryptic secondary metabolome, pairwise assays for developmental interactions within a set of 20 sequenced actinomycetes were carried out.  These assays revealed that Amycolatopsis sp. AA4, a so-called "rare" actinomycete, produces a novel siderophore, amychelin, which alters the developmental processes of several neighboring streptomycetes.  Using this phenotype as an assay, we isolated amychelin and solved its structure by NMR and MS methods coupled with an X-ray crystallog. anal. of its Fe-complex.  The iron binding affinity of amychelin was detd. using EDTA competition assays, and a biosynthetic cluster was identified and annotated to provide a tentative biosynthetic scheme for amychelin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPHzELqJtDVbVg90H21EOLACvtfcHk0lj04CADCFieRw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXosFykt78%253D&amp;md5=ad591145f788ab92316e44443d548898</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fja203577e&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja203577e%26sid%3Dliteratum%253Aachs%26aulast%3DSeyedsayamdost%26aufirst%3DM.%2BR.%26aulast%3DTraxler%26aufirst%3DM.%2BF.%26aulast%3DZheng%26aufirst%3DS.%2BL.%26aulast%3DKolter%26aufirst%3DR.%26aulast%3DClardy%26aufirst%3DJ.%26atitle%3DStructure%2520and%2520Biosynthesis%2520of%2520Amychelin%252C%2520an%2520Unusual%2520Mixed-Ligand%2520Siderophore%2520from%2520Amycolatopsis%2520Sp.%2520AA4%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D11434%26epage%3D11437%26doi%3D10.1021%2Fja203577e" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, T. T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helfrich, E. J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mevers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melo, W. G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Arnam, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andes, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currie, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pupo, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clardy, J.</span></span> <span> </span><span class="NLM_article-title">Specialized Metabolites Reveal Evolutionary History and Geographic Dispersion of a Multilateral Symbiosis</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">292</span>â <span class="NLM_lpage">299</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscentsci.0c00978</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.0c00978" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhsVKjsrc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2021&amp;pages=292-299&amp;author=T.+T.+H.+Fukudaauthor=E.+J.+N.+Helfrichauthor=E.+Meversauthor=W.+G.+P.+Meloauthor=E.+B.+van+Arnamauthor=D.+R.+Andesauthor=C.+R.+Currieauthor=M.+T.+Pupoauthor=J.+Clardy&amp;title=Specialized+Metabolites+Reveal+Evolutionary+History+and+Geographic+Dispersion+of+a+Multilateral+Symbiosis&amp;doi=10.1021%2Facscentsci.0c00978" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Specialized Metabolites Reveal Evolutionary History and Geographic Dispersion of a Multilateral Symbiosis</span></div><div class="casAuthors">Fukuda, Taise T. H.; Helfrich, Eric J. N.; Mevers, Emily; Melo, Weilan G. P.; Van Arnam, Ethan B.; Andes, David R.; Currie, Cameron R.; Pupo, Monica T.; Clardy, Jon</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">292-299</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fungus-growing ants engage in a multilateral symbiosis: they cultivate a fungal garden as their primary food source and host symbiotic actinobacteria (Pseudonocardia spp.) that provide chem. defenses.  The bacterial symbionts produce small specialized metabolites that protect the fungal garden from specific fungal pathogens (Escovopsis spp.), and in return, they are fed by the ant hosts.  Multiple studies on the mols. underlying this symbiotic system have led to the discovery of a large no. of structurally diverse antifungal mols., but somewhat surprisingly no shared structural theme emerged from these studies.  A large systematic study of Brazilian nests led to the discovery of the widespread prodn. of a potent but overlooked antifungal agent, which we named attinimicin, by nearly two-thirds of all Pseudonocardia strains from multiple sites in Brazil.  Here we report the structure of attinimicin, its putative biosynthetic gene cluster, and the evolutionary relationship between attinimicin and two related peptides, oxachelin A and cahuitamycin A.  All three nonribosomal peptides are structural isomers with different primary peptide sequences.  Attinimicin shows iron-dependent antifungal activity against specific environmental fungal parasites but no activity against the fungal cultivar.  Attinimicin showed potent in vivo activity in a mouse Candida albicans infection model comparable to clin. used azole-contg. antifungals.  In situ detection of attinimicin in both ant nests and on worker ants supports an ecol. role for attinimicin in protecting the fungal cultivar from pathogens.  The geog. spread of the attinimicin biosynthetic gene cluster in Brazilian Pseudonocardia spp. marks attinimicin as the first specialized metabolite from ant-assocd. bacteria with broad geog. distribution.  Attinimicin, the product of an evolutionarily conserved combinatorial three-module nonribosomal peptide biosynthetic pathway, provides iron-dependent chem. defenses to fungus-growing ants over a broad geog. swath.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhdoGNFcL-xrVg90H21EOLACvtfcHk0lgmPAHAB_gUnQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXhsVKjsrc%253D&amp;md5=1b46fe0b72ed5157ea5de2ca957226c6</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.0c00978&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.0c00978%26sid%3Dliteratum%253Aachs%26aulast%3DFukuda%26aufirst%3DT.%2BT.%2BH.%26aulast%3DHelfrich%26aufirst%3DE.%2BJ.%2BN.%26aulast%3DMevers%26aufirst%3DE.%26aulast%3DMelo%26aufirst%3DW.%2BG.%2BP.%26aulast%3Dvan%2BArnam%26aufirst%3DE.%2BB.%26aulast%3DAndes%26aufirst%3DD.%2BR.%26aulast%3DCurrie%26aufirst%3DC.%2BR.%26aulast%3DPupo%26aufirst%3DM.%2BT.%26aulast%3DClardy%26aufirst%3DJ.%26atitle%3DSpecialized%2520Metabolites%2520Reveal%2520Evolutionary%2520History%2520and%2520Geographic%2520Dispersion%2520of%2520a%2520Multilateral%2520Symbiosis%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2021%26volume%3D7%26spage%3D292%26epage%3D299%26doi%3D10.1021%2Facscentsci.0c00978" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potterat, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamburger, M.</span></span> <span> </span><span class="NLM_article-title">Concepts and Technologies for Tracking Bioactive Compounds in Natural Product Extracts: Generation of Libraries, and Hyphenation of Analytical Processes with Bioassays</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">546</span>â <span class="NLM_lpage">564</span>, <span class="refDoi">&nbsp;DOI: 10.1039/c3np20094a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1039%2Fc3np20094a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=23459784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjvFWnsbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=30&amp;publication_year=2013&amp;pages=546-564&amp;author=O.+Potteratauthor=M.+Hamburger&amp;title=Concepts+and+Technologies+for+Tracking+Bioactive+Compounds+in+Natural+Product+Extracts%3A+Generation+of+Libraries%2C+and+Hyphenation+of+Analytical+Processes+with+Bioassays&amp;doi=10.1039%2Fc3np20094a" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Concepts and technologies for tracking bioactive compounds in natural product extracts: generation of libraries, and hyphenation of analytical processes with bioassays</span></div><div class="casAuthors">Potterat, Olivier; Hamburger, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">546-564</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Covering: up to 2012Since the advent of high-throughput screening (HTS) in the early 1990s, a wealth of innovative technologies have been proposed and implemented for the effective localization and characterization of bioactive constituents in complex matrixes.  The latest developments in this field are reviewed under the perspective of their applicability to natural product-based drug discovery.  The approaches discussed here include TLC-based bioautog., HPLC-based assays with online, at-line and off-line detection, as well as affinity-based methods, such as frontal affinity chromatog., pulsed ultrafiltration mass spectrometry, imprinted polymers, and affinity capillary electrophoresis.  Selected practical examples are given to illustrate the strengths and limitations of these approaches in contemporary natural product lead discovery.  In addn., compatibility issues of natural product exts. and HTS are addressed, and selected protocols for the generation of high quality libraries are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRygOdG_tP17Vg90H21EOLACvtfcHk0lgmPAHAB_gUnQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXjvFWnsbc%253D&amp;md5=5043cccfd966157eec78e395abb129d6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1039%2Fc3np20094a&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3np20094a%26sid%3Dliteratum%253Aachs%26aulast%3DPotterat%26aufirst%3DO.%26aulast%3DHamburger%26aufirst%3DM.%26atitle%3DConcepts%2520and%2520Technologies%2520for%2520Tracking%2520Bioactive%2520Compounds%2520in%2520Natural%2520Product%2520Extracts%253A%2520Generation%2520of%2520Libraries%252C%2520and%2520Hyphenation%2520of%2520Analytical%2520Processes%2520with%2520Bioassays%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2013%26volume%3D30%26spage%3D546%26epage%3D564%26doi%3D10.1039%2Fc3np20094a" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atanasov, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zotchev, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dirsch, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orhan, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rollinger, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barreca, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weckwerth, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayer, E. A.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Natural Products in Drug Discovery: Advances and Opportunities</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">200</span>â <span class="NLM_lpage">216</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41573-020-00114-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1038%2Fs41573-020-00114-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=33510482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BB3MXisV2ht74%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2021&amp;pages=200-216&amp;author=A.+G.+Atanasovauthor=S.+B.+Zotchevauthor=V.+M.+Dirschauthor=I.+E.+Orhanauthor=M.+Banachauthor=J.+M.+Rollingerauthor=D.+Barrecaauthor=W.+Weckwerthauthor=R.+Bauerauthor=E.+A.+Bayer&amp;title=Natural+Products+in+Drug+Discovery%3A+Advances+and+Opportunities&amp;doi=10.1038%2Fs41573-020-00114-z" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Natural products in drug discovery: advances and opportunities</span></div><div class="casAuthors">Atanasov, Atanas G.; Zotchev, Sergey B.; Dirsch, Verena M.; the International Natural Product Sciences Taskforce; Supuran, Claudiu T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">200-216</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Natural products and their structural analogs have historically made a major contribution to pharmacotherapy, esp. for cancer and infectious diseases.  Nevertheless, natural products also present challenges for drug discovery, such as tech. barriers to screening, isolation, characterization and optimization, which contributed to a decline in their pursuit by the pharmaceutical industry from the 1990s onwards.  In recent years, several technol. and scientific developments - including improved anal. tools, genome mining and engineering strategies, and microbial culturing advances - are addressing such challenges and opening up new opportunities.  Consequently, interest in natural products as drug leads is being revitalized, particularly for tackling antimicrobial resistance.  Here, we summarize recent technol. developments that are enabling natural product-based drug discovery, highlight selected applications and discuss key opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXOyA9Np8_q7Vg90H21EOLACvtfcHk0lhEThD-uUR_pg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXisV2ht74%253D&amp;md5=8c026ca4ea44eca1b0517ecbf687caac</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41573-020-00114-z&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-020-00114-z%26sid%3Dliteratum%253Aachs%26aulast%3DAtanasov%26aufirst%3DA.%2BG.%26aulast%3DZotchev%26aufirst%3DS.%2BB.%26aulast%3DDirsch%26aufirst%3DV.%2BM.%26aulast%3DOrhan%26aufirst%3DI.%2BE.%26aulast%3DBanach%26aufirst%3DM.%26aulast%3DRollinger%26aufirst%3DJ.%2BM.%26aulast%3DBarreca%26aufirst%3DD.%26aulast%3DWeckwerth%26aufirst%3DW.%26aulast%3DBauer%26aufirst%3DR.%26aulast%3DBayer%26aufirst%3DE.%2BA.%26atitle%3DNatural%2520Products%2520in%2520Drug%2520Discovery%253A%2520Advances%2520and%2520Opportunities%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2021%26volume%3D20%26spage%3D200%26epage%3D216%26doi%3D10.1038%2Fs41573-020-00114-z" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. D.</span></span> <span> </span><span class="NLM_article-title">Opportunities for Natural Products in 21st Century Antibiotic Discovery</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">694</span>â <span class="NLM_lpage">701</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C7NP00019G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1039%2FC7NP00019G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=28569300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC2sXptFCmsLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=2017&amp;pages=694-701&amp;author=G.+D.+Wright&amp;title=Opportunities+for+Natural+Products+in+21st+Century+Antibiotic+Discovery&amp;doi=10.1039%2FC7NP00019G" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities for natural products in 21st century antibiotic discovery</span></div><div class="casAuthors">Wright, Gerard D.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">694-701</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Natural products and their derivs. are mainstays of our antibiotic drugs, but they are increasingly in peril.  The combination of widespread multidrug resistance in once susceptible bacterial pathogens, disenchantment with natural products as sources of new drugs, lack of success using synthetic compds. and target-based discovery methods, along with shifting economic and regulatory issues, conspire to move investment in research and development away from the antibiotics arena.  The result is a growing crisis in antibiotic drug discovery that threatens modern medicine.  21St century natural product research is perfectly positioned to fill the antibiotic discovery gap and bring new drug candidates to the clinic.  Innovations in genomics and techniques to explore new sources of antimicrobial chem. matter are revealing new chem.  Increasing appreciation of the value of narrow-spectrum drugs and re-examn. of once discarded chem. scaffolds coupled with synthetic biol. methods to generate new compds. and improve yields offer new strategies to revitalize once moribund natural product programs.  The increasing awareness that the combination of antibiotics with adjuvants, non-antibiotic compds. that overcome resistance and enhance drug activity, can rescue older chem. scaffolds, and concepts such as blocking pathogen virulence present orthogonal strategies to traditional antibiotics.  In all these areas, natural products offer chem. matter, shaped by natural selection, that is privileged in this therapeutic area.  Natural product research is poised to regain prominence in delivering new drugs to solve the antibiotic crisis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHEYEudGcHELVg90H21EOLACvtfcHk0lhEThD-uUR_pg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXptFCmsLg%253D&amp;md5=a0df60088a52f943922e460e9d0e38be</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC7NP00019G&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7NP00019G%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DG.%2BD.%26atitle%3DOpportunities%2520for%2520Natural%2520Products%2520in%252021st%2520Century%2520Antibiotic%2520Discovery%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2017%26volume%3D34%26spage%3D694%26epage%3D701%26doi%3D10.1039%2FC7NP00019G" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prieto-MartÃ­nez, F. D.</span>; <span class="NLM_string-name">Norinder, U.</span>; <span class="NLM_string-name">Medina-Franco, J. L.</span></span> <span> </span><span class="NLM_article-title">Cheminformatics Explorations of Natural Products</span>. In  <i>Progress in the Chemistry of Organic Natural Products 110: Cheminformatics in Natural Product Research</i>, <span class="NLM_contrib-group"><span class="NLM_string-name">Kinghorn, A. D.</span>; <span class="NLM_string-name">Falk, H.</span>; <span class="NLM_string-name">Gibbons, S.</span>; <span class="NLM_string-name">Kobayashi, J.</span>; <span class="NLM_string-name">Asakawa, Y.</span>; <span class="NLM_string-name">Liu, J.-K.</span></span>, Eds.; <span class="NLM_publisher-name">Springer International Publishing</span>: <span class="NLM_publisher-loc">Cham</span>, <span class="NLM_year">2019</span>; pp  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">35</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1007%2F978-3-030-14632-0_1" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2019&amp;pages=1-35&amp;author=F.+D.+Prieto-Mart%C3%ADnez&amp;author=U.+Norinder&amp;author=J.+L.+Medina-Francoauthor=A.+D.+Kinghorn&amp;author=H.+Falk&amp;author=S.+Gibbons&amp;author=J.+Kobayashi&amp;author=Y.+Asakawa&amp;author=J.-K.+Liu&amp;title=Progress+in+the+Chemistry+of+Organic+Natural+Products+110%3A+Cheminformatics+in+Natural+Product+Research" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2F978-3-030-14632-0_1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-030-14632-0_1%26sid%3Dliteratum%253Aachs%26aulast%3DPrieto-Mart%25C3%25ADnez%26aufirst%3DF.%2BD.%26atitle%3DCheminformatics%2520Explorations%2520of%2520Natural%2520Products%26btitle%3DProgress%2520in%2520the%2520Chemistry%2520of%2520Organic%2520Natural%2520Products%2520110%253A%2520Cheminformatics%2520in%2520Natural%2520Product%2520Research%26aulast%3DKinghorn%26aufirst%3DA.%2BD.%26pub%3DSpringer%2520International%2520Publishing%26date%3D2019%26spage%3D1%26epage%3D35" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houriet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidar, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellogg, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todd, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cech, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span> <span> </span><span class="NLM_article-title">Interlaboratory Comparison of Untargeted Mass Spectrometry Data Uncovers Underlying Causes for Variability</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">824</span>â <span class="NLM_lpage">835</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.jnatprod.0c01376</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.0c01376" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BB3MXls1Crs7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=84&amp;publication_year=2021&amp;pages=824-835&amp;author=T.+N.+Clarkauthor=J.+Hourietauthor=W.+S.+Vidarauthor=J.+J.+Kelloggauthor=D.+A.+Toddauthor=N.+B.+Cechauthor=R.+G.+Linington&amp;title=Interlaboratory+Comparison+of+Untargeted+Mass+Spectrometry+Data+Uncovers+Underlying+Causes+for+Variability&amp;doi=10.1021%2Facs.jnatprod.0c01376" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Interlaboratory Comparison of Untargeted Mass Spectrometry Data Uncovers Underlying Causes for Variability</span></div><div class="casAuthors">Clark, Trevor N.; Houriet, Joelle; Vidar, Warren S.; Kellogg, Joshua J.; Todd, Daniel A.; Cech, Nadja B.; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">824-835</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Despite the value of mass spectrometry in modern natural products discovery workflows, it remains very difficult to compare data sets between labs.  In this study we compared mass spectrometry data for the same sample set from two different labs. (quadrupole time-of-flight and quadrupole-Orbitrap) and evaluated the similarity between these two data sets in terms of both mass spectrometry features and their ability to describe the chem. compn. of the sample set.  Somewhat surprisingly, the two data sets, collected with appropriate controls and replication, had very low feature overlap (25.7% of Lab. A features overlapping 21.8% of Lab. B features).  Our data clearly demonstrate that differences in fragmentation, charge state, and adduct formation in the ionization source are a major underlying cause for these differences.  Consistent with other recent literature, these findings challenge the conventional wisdom that electrospray ionization mass spectrometry (ESI-MS) yields a simple one-to-one correspondence between analytes in soln. and features in the data set.  Importantly, despite low overlap in feature lists, principal component anal. (PCA) generated qual. similar PCA plots.  Overall, our findings demonstrate that comparing untargeted metabolomics data between labs. is challenging, but that data sets with low feature overlap can yield the same qual. description of a sample set using PCA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm7C-Gmm7P5bVg90H21EOLACvtfcHk0lhEThD-uUR_pg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXls1Crs7w%253D&amp;md5=cd93a585f05b819287b4f052b6baa21f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.0c01376&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.0c01376%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DT.%2BN.%26aulast%3DHouriet%26aufirst%3DJ.%26aulast%3DVidar%26aufirst%3DW.%2BS.%26aulast%3DKellogg%26aufirst%3DJ.%2BJ.%26aulast%3DTodd%26aufirst%3DD.%2BA.%26aulast%3DCech%26aufirst%3DN.%2BB.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26atitle%3DInterlaboratory%2520Comparison%2520of%2520Untargeted%2520Mass%2520Spectrometry%2520Data%2520Uncovers%2520Underlying%2520Causes%2520for%2520Variability%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2021%26volume%3D84%26spage%3D824%26epage%3D835%26doi%3D10.1021%2Facs.jnatprod.0c01376" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hohrenk, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itzel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baetz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuerk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vosough, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, T. C.</span></span> <span> </span><span class="NLM_article-title">Comparison of Software Tools for Liquid Chromatography-High-Resolution Mass Spectrometry Data Processing in Nontarget Screening of Environmental Samples</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1898</span>â <span class="NLM_lpage">1907</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.analchem.9b04095</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.9b04095" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitl2htL%252FP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=92&amp;publication_year=2020&amp;pages=1898-1907&amp;author=L.+L.+Hohrenkauthor=F.+Itzelauthor=N.+Baetzauthor=J.+Tuerkauthor=M.+Vosoughauthor=T.+C.+Schmidt&amp;title=Comparison+of+Software+Tools+for+Liquid+Chromatography-High-Resolution+Mass+Spectrometry+Data+Processing+in+Nontarget+Screening+of+Environmental+Samples&amp;doi=10.1021%2Facs.analchem.9b04095" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Software Tools for Liquid Chromatography-High-Resolution Mass Spectrometry Data Processing in Nontarget Screening of Environmental Samples</span></div><div class="casAuthors">Hohrenk, Lotta L.; Itzel, Fabian; Baetz, Nicolai; Tuerk, Jochen; Vosough, Maryam; Schmidt, Torsten C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1898-1907</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The field of high-resoln. mass spectrometry has undergone rapid progress in the last years due to instrumental improvements leading to a higher sensitivity and selectivity of instruments.  A variety of qual. screening approaches, summarized as nontarget screening, have been introduced and have successfully extended the environmental monitoring of org. micropollutants.  Several automated data processing workflows have been developed to handle the immense amt. of data that are recorded in short time frames by these methods.  Most data processing workflows include similar steps, but underlying algorithms and implementation of different processing steps vary.  In this study the consistency of data processing with different software tools was investigated.  For this purpose, the same raw data files were processed with the software packages MZmine2, enviMass, Compd. Discoverer, and XCMS online and resulting feature lists were compared.  Results show a low coherence between different processing tools, as overlap of features between all four programs was around 10%, and for each software between 40% and 55% of features did not match with any other program.  The implementation of replicate and blank filter was identified as one of the sources of obsd. divergences.  However, there is a need for a better understanding and user instructions on the influence of different algorithms and settings on feature extn. and following filtering steps.  In future studies it would be of interest to investigate how final data interpretation is influenced by different processing software.  With this work we want to encourage more awareness on data processing as a crucial step in the workflow of nontarget screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY6ad0_f0cZLVg90H21EOLACvtfcHk0ljCoVsASgX6ag"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXitl2htL%252FP&amp;md5=cbb2700b3716c265e660d73d250ec787</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.9b04095&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.9b04095%26sid%3Dliteratum%253Aachs%26aulast%3DHohrenk%26aufirst%3DL.%2BL.%26aulast%3DItzel%26aufirst%3DF.%26aulast%3DBaetz%26aufirst%3DN.%26aulast%3DTuerk%26aufirst%3DJ.%26aulast%3DVosough%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DT.%2BC.%26atitle%3DComparison%2520of%2520Software%2520Tools%2520for%2520Liquid%2520Chromatography-High-Resolution%2520Mass%2520Spectrometry%2520Data%2520Processing%2520in%2520Nontarget%2520Screening%2520of%2520Environmental%2520Samples%26jtitle%3DAnal.%2520Chem.%26date%3D2020%26volume%3D92%26spage%3D1898%26epage%3D1907%26doi%3D10.1021%2Facs.analchem.9b04095" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahieu, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patti, G. J.</span></span> <span> </span><span class="NLM_article-title">Systems-Level Annotation of a Metabolomics Data Set Reduces 25â¯000 Features to Fewer than 1000 Unique Metabolites</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">10397</span>â <span class="NLM_lpage">10406</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.analchem.7b02380</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.7b02380" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsV2iurnL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=89&amp;publication_year=2017&amp;pages=10397-10406&amp;author=N.+G.+Mahieuauthor=G.+J.+Patti&amp;title=Systems-Level+Annotation+of+a+Metabolomics+Data+Set+Reduces+25%E2%80%AF000+Features+to+Fewer+than+1000+Unique+Metabolites&amp;doi=10.1021%2Facs.analchem.7b02380" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Systems-Level Annotation of a Metabolomics Data Set Reduces 25,000 Features to Fewer than 1000 Unique Metabolites</span></div><div class="casAuthors">Mahieu, Nathaniel G.; Patti, Gary J.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10397-10406</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">When using liq. chromatog./mass spectrometry (LC/MS) to perform untargeted metabolomics, it is now routine to detect tens of thousands of features from biol. samples.  Poor understanding of the data, however, has complicated interpretation and masked the no. of unique metabolites actually being measured in an expt.  Here the authors place an upper bound on the no. of unique metabolites detected in Escherichia coli samples analyzed with one untargeted metabolomic method.  The authors first group multiple features arising from the same analyte, which the authors call "degenerate features", using a context-driven annotation approach.  Surprisingly, this anal. revealed thousands of previously unreported degeneracies that reduced the no. of unique analytes to â¼2961.  The authors then applied an orthogonal approach to remove nonbiol. features from the data using the 13C-based credentialing technol.  This further reduced the no. of unique analytes to less than 1000.  The 90% redn. in data is 5-fold greater than any previously published studies.  On the basis of the results, the authors propose an alternative approach to untargeted metabolomics that relies on thoroughly annotated ref. data sets.  To this end, the authors introduce the creDBle database (http://creDBle.wustl.edu), which contains accurate mass, retention time, and MS/MS fragmentation data as well as annotations of all credentialed features.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpujHD0I2nPpbVg90H21EOLACvtfcHk0ljCoVsASgX6ag"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhsV2iurnL&amp;md5=d83c63dfa6be1da7a6bb35466544f6e7</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.7b02380&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.7b02380%26sid%3Dliteratum%253Aachs%26aulast%3DMahieu%26aufirst%3DN.%2BG.%26aulast%3DPatti%26aufirst%3DG.%2BJ.%26atitle%3DSystems-Level%2520Annotation%2520of%2520a%2520Metabolomics%2520Data%2520Set%2520Reduces%252025%25E2%2580%25AF000%2520Features%2520to%2520Fewer%2520than%25201000%2520Unique%2520Metabolites%26jtitle%3DAnal.%2520Chem.%26date%3D2017%26volume%3D89%26spage%3D10397%26epage%3D10406%26doi%3D10.1021%2Facs.analchem.7b02380" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krug, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zurek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, R.</span></span> <span> </span><span class="NLM_article-title">Efficient Mining of Myxobacterial Metabolite Profiles Enabled by Liquid Chromatography-Electrospray Ionisation-Time-of-Flight Mass Spectrometry and Compound-Based Principal Component Analysis</span>. <i>Anal. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>624</i></span>,  <span class="NLM_fpage">97</span>â <span class="NLM_lpage">106</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.aca.2008.06.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1016%2Fj.aca.2008.06.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=18706314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVWksbzI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=624&amp;publication_year=2008&amp;pages=97-106&amp;author=D.+Krugauthor=G.+Zurekauthor=B.+Schneiderauthor=R.+Garciaauthor=R.+M%C3%BCller&amp;title=Efficient+Mining+of+Myxobacterial+Metabolite+Profiles+Enabled+by+Liquid+Chromatography-Electrospray+Ionisation-Time-of-Flight+Mass+Spectrometry+and+Compound-Based+Principal+Component+Analysis&amp;doi=10.1016%2Fj.aca.2008.06.036" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient mining of myxobacterial metabolite profiles enabled by liquid chromatography-electrospray ionization-time-of-flight mass spectrometry and compound-based principal component analysis</span></div><div class="casAuthors">Krug, Daniel; Zurek, Gabriela; Schneider, Birgit; Garcia, Ronald; Mueller, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Analytica Chimica Acta</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">624</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">97-106</span>CODEN:
                <span class="NLM_cas:coden">ACACAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2670</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bacteria producing secondary metabolites are an important source of natural products with highly diverse structures and biol. activities.  Developing methods to efficiently mine procaryotic secondary metabolomes for the presence of potentially novel natural products is therefore of considerable interest.  Modern mass spectrometry-coupled liq. chromatog. can effectively capture microbial metabolic diversity with ever improving sensitivity and accuracy.  In addn., computational and statistical tools increasingly enable the targeted anal. and exploration of information-rich LC-MS datasets.  In this article, the authors describe the use of such techniques for the characterization of myxobacterial secondary metabolomes.  Using accurate mass data from high-resoln. ESI-TOF measurements, target screening has facilitated the rapid identification of known myxobacterial metabolites in exts. from nine Myxococcus species.  Furthermore, principal component anal. (PCA), implementing an advanced compd.-based bucketing approach, readily revealed the presence of further compds. which contribute to variation among the metabolite profiles under investigation.  The generation of mol. formulas for putative novel compds. with high confidence due to evaluation of both exact mass position and isotopic pattern, is exemplified as an important key for de-replication and prioritization of candidates for further characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ0E82nT40mrVg90H21EOLACvtfcHk0ljCoVsASgX6ag"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXhtVWksbzI&amp;md5=6bef2ebc2a2fce50fa495f0a6e8dd007</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.aca.2008.06.036&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.aca.2008.06.036%26sid%3Dliteratum%253Aachs%26aulast%3DKrug%26aufirst%3DD.%26aulast%3DZurek%26aufirst%3DG.%26aulast%3DSchneider%26aufirst%3DB.%26aulast%3DGarcia%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DR.%26atitle%3DEfficient%2520Mining%2520of%2520Myxobacterial%2520Metabolite%2520Profiles%2520Enabled%2520by%2520Liquid%2520Chromatography-Electrospray%2520Ionisation-Time-of-Flight%2520Mass%2520Spectrometry%2520and%2520Compound-Based%2520Principal%2520Component%2520Analysis%26jtitle%3DAnal.%2520Chim.%2520Acta%26date%3D2008%26volume%3D624%26spage%3D97%26epage%3D106%26doi%3D10.1016%2Fj.aca.2008.06.036" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></div><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0008.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0001.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of the NP Analyst platform. Users input both mass spectrometry files and biological activity data for sample sets, which are scored to prioritize compounds with strong predicted biological activities for isolation and secondary screening.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0002.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. NP Analyst community view (A) including network diagram (B), plot of retention time (<i>x</i>-axis) vs <i>m</i>/<i>z</i> ratio (<i>y</i>-axis) for bioactive MS features (C), plot of Cluster Score (<i>x</i>-axis) vs Activity Score (<i>y</i>-axis) for bioactive MS features (D), and activity profiles for extracts in community. Rows contain bioactivity data for each sample. Columns contain bioassay values for each bioassay readout (E). In plots BâD, extracts are represented by square gray nodes, while colored circular nodes represent MS features. The color of MS feature nodes is defined by Cluster Score (blue to red, â1 to +1). The diameter of these nodes is defined by the Activity Score (normalized scale from minimum to maximum Activity Score values). The original webpage can be accessed from <a href="http://www.npanalyst.org" class="extLink">www.npanalyst.org</a> by selecting the âOpen Sample Outputâ button, choosing the âCommunitiesâ tab, and selecting community 15 from the dropdown menu at the top of the page.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0003.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. NP Analyst network for 925 microbial natural products prefractions. Square gray nodes represent Prefractions. Circular nodes represent MS features. The color of MS feature nodes is defined by Cluster Score (blue to red, â1 to +1). The diameter of these nodes is defined by Activity Score (normalized scale from minimum to maximum Activity Score values). A high-resolution version of this figure, including text annotations for each node and a color scale bar, is available in the <a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Supporting Information</a> (Figure S1).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0004.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Community 12 network. Prefractions illustrated as square gray nodes. Predicted bioactive MS features illustrated as circular red nodes. <i>m</i>/<i>z</i> values for priority MS feature node annotated on the network. Peripheral nodes removed for clarity. Full community presented in <a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Figure S2</a>. (B) Structures of isolated compounds. (C) Extracted ion chromatogram (EIC) traces for predicted bioactive MS feature (top), micromonolactam (middle), and dracolactam C (bottom) illustrating alignment between the predicted bioactive constituent and the purified bioactive metabolite (micromonolactam) and the absence of predicted bioactivity for the inactive isomer dracolactam C.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0005.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Community 15 network. Prefractions illustrated as square gray nodes. Predicted bioactive MS features illustrated as circular red nodes. The color of MS feature nodes is defined by Cluster Score (blue to red, â1 to +1). The diameter of these nodes is defined by Activity Score (normalized scale from minimum to maximum Activity Score values). (B) Scatter plot (Activity Score â¥ 0.3) illustrating retention time alignment for bioactive MS features. (C) Structures of isolated compounds. (D) Alignment of EICs for bioactive MS features from community 15 illustrating retention time and peak shape alignment for three major components in the mixture. (E) Results table for all bioactive features from the community, highlighting features related to compounds in panel C (green = collismycin A, blue = collismycin B, red = SF2738D).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/cms/10.1021/acscentsci.1c01108/asset/images/medium/oc1c01108_0006.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Community 16 network. Prefractions illustrated as square gray nodes. Predicted bioactive MS features illustrated as circular red nodes. Circle diameter proportional to Activity Score. Peripheral nodes removed for clarity. Full community presented in <a href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf" class="ext-link">Figure S2</a>. (B) Substructures determined from NMR data. (C) Comparison of MS fragmentation patterns for unlabeled and labeled amychelin C, illustrating the position of labeled [1,2,3-<sup>13</sup>C]-(<span class="smallcaps smallerCapital">l</span>)-serine (red moiety) and <sup>13</sup>C-labeled carbons (blue circles). (D) Structures of amychelin C (<b>6</b>) and amychelin (<b>7</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c01108/asset/images/large/oc1c01108_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c01108&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11738" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11738" class="dropBlock__holder"></div></div></div></div></div><hr></div><p class="rightTab-references-count">
            This article references 32 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wani, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPhail, A. T.</span></span> <span> </span><span class="NLM_article-title">Plant Antitumor Agents. VI. Isolation and Structure of Taxol, a Novel Antileukemic and Antitumor Agent from Taxus Brevifolia</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1971</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">2325</span>â <span class="NLM_lpage">2327</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja00738a045</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00738a045" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADyaE3MXktVKisLc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=93&amp;publication_year=1971&amp;pages=2325-2327&amp;author=M.+C.+Waniauthor=H.+L.+Taylorauthor=M.+E.+Wallauthor=P.+Coggonauthor=A.+T.+McPhail&amp;title=Plant+Antitumor+Agents.+VI.+Isolation+and+Structure+of+Taxol%2C+a+Novel+Antileukemic+and+Antitumor+Agent+from+Taxus+Brevifolia&amp;doi=10.1021%2Fja00738a045" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Plant antitumor agents.  VI.  Isolation and structure of taxol, a novel antileukemic and antitumor agent from Taxus brevifolia</span></div><div class="casAuthors">Wani, Mansukhlal C.; Taylor, Harold Lawrence; Wall, Monroe E.; Coggon, Philip; McPhail, Andrew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2325-7</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Taxol (I), a complex ester with a taxane ring system exhibiting potent antileukemic and antitumor properties, was isolated from Taxus brevifolia (yew).  BzNHCH(Ph)CH(OH)CO2Me and 5Î²,16-epoxy-1Î²,-2Î±,4Î±,7Î²,10Î²,13Î±-hexahydroxytax-11-en-9-one 4Î±-acetate 2Î±-benzoate are obtained by methanolysis of I.  Based on chem. evidence, I is 5Î²,16-epoxy-1Î²,2Î±,4Î±,7Î²,10Î²,13Î±-hexahydroxytax-11-en-9-one 4Î±,10Î²-diacetate 2Î±-benzoate 13Î±-(3-benzamido-2-hydroxy-3-phenylpropionate).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpexINv4qMTjbVg90H21EOLACvtfcHk0lh_e6wmPe5Xdw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaE3MXktVKisLc%253D&amp;md5=50c59ef3bf2ae493408bed2a76ca0c2e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fja00738a045&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00738a045%26sid%3Dliteratum%253Aachs%26aulast%3DWani%26aufirst%3DM.%2BC.%26aulast%3DTaylor%26aufirst%3DH.%2BL.%26aulast%3DWall%26aufirst%3DM.%2BE.%26aulast%3DCoggon%26aufirst%3DP.%26aulast%3DMcPhail%26aufirst%3DA.%2BT.%26atitle%3DPlant%2520Antitumor%2520Agents.%2520VI.%2520Isolation%2520and%2520Structure%2520of%2520Taxol%252C%2520a%2520Novel%2520Antileukemic%2520and%2520Antitumor%2520Agent%2520from%2520Taxus%2520Brevifolia%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1971%26volume%3D93%26spage%3D2325%26epage%3D2327%26doi%3D10.1021%2Fja00738a045" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VÃ©zina, C.</span></span> <span> </span><span class="NLM_article-title">Rapamycin (Ay-22,989), a New Antifungal Antibiotic. II. Fermentation, Isolation and Characterization</span>. <i>J. Antibiot. (Tokyo)</i> <span class="NLM_year" style="font-weight: bold;">1975</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">727</span>â <span class="NLM_lpage">732</span>, <span class="refDoi">&nbsp;DOI: 10.7164/antibiotics.28.727</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.7164%2Fantibiotics.28.727" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1102509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A280%3ADyaE28%252Fks1Snsw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=28&amp;publication_year=1975&amp;pages=727-732&amp;author=S.+N.+Sehgalauthor=H.+Bakerauthor=C.+V%C3%A9zina&amp;title=Rapamycin+%28Ay-22%2C989%29%2C+a+New+Antifungal+Antibiotic.+II.+Fermentation%2C+Isolation+and+Characterization&amp;doi=10.7164%2Fantibiotics.28.727" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Rapamycin (AY-22,989), a new antifungal antibiotic. II. Fermentation, isolation and characterization</span></div><div class="casAuthors">Sehgal S N; Baker H; Vezina C</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of antibiotics</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">727-32</span>
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Rapamycin is a new antifungal antibiotic produced by Streptomyces hygroscopicus NRRL 5491.  It was isolated from the mycelium by solvent extraction, purified by silica gel column chromatography and crystallized as a colorless solid which melts at 183 approximately to 185 degrees C and has the empirical formula C56H89NO14.  From its characteristic ultraviolet absorption spectrum rapamycin can be classified as a triene.  It is highly active against various Candida species, especially Candida albicans.  Its activity is compared with that of amphotericin B, candicidin and nystatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSI9b1_oP_x4o3yKPtYViAGfW6udTcc2eaBuLZu7NIl9rntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADyaE28%252Fks1Snsw%253D%253D&amp;md5=dc47c0f6a250c0db2710d7371a5c43c4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.28.727&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.28.727%26sid%3Dliteratum%253Aachs%26aulast%3DSehgal%26aufirst%3DS.%2BN.%26aulast%3DBaker%26aufirst%3DH.%26aulast%3DV%25C3%25A9zina%26aufirst%3DC.%26atitle%3DRapamycin%2520%2528Ay-22%252C989%2529%252C%2520a%2520New%2520Antifungal%2520Antibiotic.%2520II.%2520Fermentation%252C%2520Isolation%2520and%2520Characterization%26jtitle%3DJ.%2520Antibiot.%2520%2528Tokyo%2529%26date%3D1975%26volume%3D28%26spage%3D727%26epage%3D732%26doi%3D10.7164%2Fantibiotics.28.727" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">RÃ¼egger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichti, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loosli, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huguenin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quiquerez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Wartburg, A.</span></span> <span> </span><span class="NLM_article-title">Cyclosporin A, a Peptide Metabolite from Trichoderma Polysporum (LINK Ex PERS.) Rifai, with a Remarkable Immunosuppressive Activity</span>. <i>Helvetica</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1075</span>â <span class="NLM_lpage">1092</span>, <span class="refDoi">&nbsp;DOI: 10.1002/hlca.19760590412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1002%2Fhlca.19760590412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=950308" title="">PubMed</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=59&amp;publication_year=1976&amp;pages=1075-1092&amp;author=A.+R%C3%BCeggerauthor=M.+Kuhnauthor=H.+Lichtiauthor=H.-R.+Loosliauthor=R.+Hugueninauthor=C.+Quiquerezauthor=A.+von+Wartburg&amp;title=Cyclosporin+A%2C+a+Peptide+Metabolite+from+Trichoderma+Polysporum+%28LINK+Ex+PERS.%29+Rifai%2C+with+a+Remarkable+Immunosuppressive+Activity&amp;doi=10.1002%2Fhlca.19760590412" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fhlca.19760590412&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhlca.19760590412%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25BCegger%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DLichti%26aufirst%3DH.%26aulast%3DLoosli%26aufirst%3DH.-R.%26aulast%3DHuguenin%26aufirst%3DR.%26aulast%3DQuiquerez%26aufirst%3DC.%26aulast%3Dvon%2BWartburg%26aufirst%3DA.%26atitle%3DCyclosporin%2520A%252C%2520a%2520Peptide%2520Metabolite%2520from%2520Trichoderma%2520Polysporum%2520%2528LINK%2520Ex%2520PERS.%2529%2520Rifai%252C%2520with%2520a%2520Remarkable%2520Immunosuppressive%2520Activity%26jtitle%3DHelvetica%26date%3D1976%26volume%3D59%26spage%3D1075%26epage%3D1092%26doi%3D10.1002%2Fhlca.19760590412" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerwick, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span> <span> </span><span class="NLM_article-title">Retrospective Analysis of Natural Products Provides Insights for Future Discovery Trends</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">5601</span>â <span class="NLM_lpage">5606</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1614680114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1073%2Fpnas.1614680114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=28461474" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFSgsro%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=114&amp;publication_year=2017&amp;pages=5601-5606&amp;author=C.+R.+Pyeauthor=M.+J.+Bertinauthor=R.+S.+Lokeyauthor=W.+H.+Gerwickauthor=R.+G.+Linington&amp;title=Retrospective+Analysis+of+Natural+Products+Provides+Insights+for+Future+Discovery+Trends&amp;doi=10.1073%2Fpnas.1614680114" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Retrospective analysis of natural products provides insights for future discovery trends</span></div><div class="casAuthors">Pye, Cameron R.; Bertin, Matthew J.; Lokey, R. Scott; Gerwick, William H.; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5601-5606</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Understanding of the capacity of the natural world to produce secondary metabolites is important to a broad range of fields, including drug discovery, ecol., biosynthesis, and chem. biol., among others.  Both the abs. no. and the rate of discovery of natural products have increased significantly in recent years.  However, there is a perception and concern that the fundamental novelty of these discoveries is decreasing relative to previously known natural products.  This study presents a quant. examn. of the field from the perspective of both no. of compds. and compd. novelty using a dataset of all published microbial and marine-derived natural products.  This anal. aimed to explore a no. of key questions, such as how the rate of discovery of new natural products has changed over the past decades, how the av. natural product structural novelty has changed as a function of time, whether exploring novel taxonomic space affords an advantage in terms of novel compd. discovery, and whether it is possible to est. how close we are to having described all of the chem. space covered by natural products.  Our analyses demonstrate that most natural products being published today bear structural similarity to previously published compds., and that the range of scaffolds readily accessible from nature is limited.  However, the anal. also shows that the field continues to discover appreciable nos. of natural products with no structural precedent.  Together, these results suggest that the development of innovative discovery methods will continue to yield compds. with unique structural and biol. properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3OY35vWvx_bVg90H21EOLACvtfcHk0ligZZfAQb8iWg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXmvFSgsro%253D&amp;md5=44da23318031b09694c94b0bfdf1d718</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1614680114&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1614680114%26sid%3Dliteratum%253Aachs%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DBertin%26aufirst%3DM.%2BJ.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26aulast%3DGerwick%26aufirst%3DW.%2BH.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26atitle%3DRetrospective%2520Analysis%2520of%2520Natural%2520Products%2520Provides%2520Insights%2520for%2520Future%2520Discovery%2520Trends%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D5601%26epage%3D5606%26doi%3D10.1073%2Fpnas.1614680114" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Louwen, J. J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Hooft, J. J. J.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Large-Scale Integrative Analysis of Omics Data to Accelerate Specialized Metabolite Discovery</span>. <i>mSystems</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">e000726</span>â <span class="NLM_lpage">21</span>, <span class="refDoi">&nbsp;DOI: 10.1128/mSystems.00726-21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1128%2FmSystems.00726-21" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2021&amp;pages=e000726-21&amp;author=J.+J.+R.+Louwenauthor=J.+J.+J.+van+der+Hooft&amp;title=Comprehensive+Large-Scale+Integrative+Analysis+of+Omics+Data+to+Accelerate+Specialized+Metabolite+Discovery&amp;doi=10.1128%2FmSystems.00726-21" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1128%2FmSystems.00726-21&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FmSystems.00726-21%26sid%3Dliteratum%253Aachs%26aulast%3DLouwen%26aufirst%3DJ.%2BJ.%2BR.%26aulast%3Dvan%2Bder%2BHooft%26aufirst%3DJ.%2BJ.%2BJ.%26atitle%3DComprehensive%2520Large-Scale%2520Integrative%2520Analysis%2520of%2520Omics%2520Data%2520to%2520Accelerate%2520Specialized%2520Metabolite%2520Discovery%26jtitle%3DmSystems%26date%3D2021%26volume%3D6%26spage%3De000726%26epage%3D21%26doi%3D10.1128%2FmSystems.00726-21" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schorn, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoeven, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ridder, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aksenov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aleti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moghaddam, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aron, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aziz, S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">A Community Resource for Paired Genomic and Metabolomic Data Mining</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">363</span>â <span class="NLM_lpage">368</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41589-020-00724-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1038%2Fs41589-020-00724-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=33589842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BB3MXktF2gt7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=2021&amp;pages=363-368&amp;author=M.+A.+Schornauthor=S.+Verhoevenauthor=L.+Ridderauthor=F.+Huberauthor=D.+D.+Acharyaauthor=A.+A.+Aksenovauthor=G.+Aletiauthor=J.+A.+Moghaddamauthor=A.+T.+Aronauthor=S.+Aziz&amp;title=A+Community+Resource+for+Paired+Genomic+and+Metabolomic+Data+Mining&amp;doi=10.1038%2Fs41589-020-00724-z" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A community resource for paired genomic and metabolomic data mining</span></div><div class="casAuthors">Schorn, Michelle A.; Verhoeven, Stefan; Ridder, Lars; Huber, Florian; Acharya, Deepa D.; Aksenov, Alexander A.; Aleti, Gajender; Moghaddam, Jamshid Amiri; Aron, Allegra T.; Aziz, Saefuddin; Bauermeister, Anelize; Bauman, Katherine D.; Baunach, Martin; Beemelmanns, Christine; Beman, J. Michael; Berlanga-Clavero, Maria Victoria; Blacutt, Alex A.; Bode, Helge B.; Boullie, Anne; Brejnrod, Asker; Bugni, Tim S.; Calteau, Alexandra; Cao, Liu; Carrion, Victor J.; Castelo-Branco, Raquel; Chanana, Shaurya; Chase, Alexander B.; Chevrette, Marc G.; Costa-Lotufo, Leticia V.; Crawford, Jason M.; Currie, Cameron R.; Cuypers, Bart; Dang, Tam; de Rond, Tristan; Demko, Alyssa M.; Dittmann, Elke; Du, Chao; Drozd, Christopher; Dujardin, Jean-Claude; Dutton, Rachel J.; Edlund, Anna; Fewer, David P.; Garg, Neha; Gauglitz, Julia M.; Gentry, Emily C.; Gerwick, Lena; Glukhov, Evgenia; Gross, Harald; Gugger, Muriel; Guillen Matus, Dulce G.; Helfrich, Eric J. N.; Hempel, Benjamin-Florian; Hur, Jae-Seoun; Iorio, Marianna; Jensen, Paul R.; Kang, Kyo Bin; Kaysser, Leonard; Kelleher, Neil L.; Kim, Chung Sub; Kim, Ki Hyun; Koester, Irina; Konig, Gabriele M.; Leao, Tiago; Lee, Seoung Rak; Lee, Yi-Yuan; Li, Xuanji; Little, Jessica C.; Maloney, Katherine N.; Mannle, Daniel; Martin H., Christian; McAvoy, Andrew C.; Metcalf, Willam W.; Mohimani, Hosein; Molina-Santiago, Carlos; Moore, Bradley S.; Mullowney, Michael W.; Muskat, Mitchell; Nothias, Louis-Felix; O'Neill, Ellis C.; Parkinson, Elizabeth I.; Petras, Daniel; Piel, Jorn; Pierce, Emily C.; Pires, Karine; Reher, Raphael; Romero, Diego; Roper, M. Caroline; Rust, Michael; Saad, Hamada; Saenz, Carmen; Sanchez, Laura M.; Soerensen, Soeren Johannes; Sosio, Margherita; Sussmuth, Roderich D.; Sweeney, Douglas; Tahlan, Kapil; Thomson, Regan J.; Tobias, Nicholas J.; Trindade-Silva, Amaro E.; van Wezel, Gilles P.; Wang, Mingxun; Weldon, Kelly C.; Zhang, Fan; Ziemert, Nadine; Duncan, Katherine R.; Crusemann, Max; Rogers, Simon; Dorrestein, Pieter C.; Medema, Marnix H.; van der Hooft, Justin J. J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">363-368</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Genomics and metabolomics are widely used to explore specialized metabolite diversity.  The Paired Omics Data Platform is a community initiative to systematically document links between metabolome and (meta)genome data, aiding identification of natural product biosynthetic origins and metabolite structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlNF_EKMtzsrVg90H21EOLACvtfcHk0ligZZfAQb8iWg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXktF2gt7Y%253D&amp;md5=3da222f9acc0f30cd629e0e5f628ae53</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-00724-z&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-00724-z%26sid%3Dliteratum%253Aachs%26aulast%3DSchorn%26aufirst%3DM.%2BA.%26aulast%3DVerhoeven%26aufirst%3DS.%26aulast%3DRidder%26aufirst%3DL.%26aulast%3DHuber%26aufirst%3DF.%26aulast%3DAcharya%26aufirst%3DD.%2BD.%26aulast%3DAksenov%26aufirst%3DA.%2BA.%26aulast%3DAleti%26aufirst%3DG.%26aulast%3DMoghaddam%26aufirst%3DJ.%2BA.%26aulast%3DAron%26aufirst%3DA.%2BT.%26aulast%3DAziz%26aufirst%3DS.%26atitle%3DA%2520Community%2520Resource%2520for%2520Paired%2520Genomic%2520and%2520Metabolomic%2520Data%2520Mining%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2021%26volume%3D17%26spage%3D363%26epage%3D368%26doi%3D10.1038%2Fs41589-020-00724-z" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolfender, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litaudon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touboul, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queiroz, E. F.</span></span> <span> </span><span class="NLM_article-title">Innovative Omics-Based Approaches for Prioritisation and Targeted Isolation of Natural Products â New Strategies for Drug Discovery</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">855</span>â <span class="NLM_lpage">868</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C9NP00004F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1039%2FC9NP00004F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=31073562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC1MXps1SqtL8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=2019&amp;pages=855-868&amp;author=J.-L.+Wolfenderauthor=M.+Litaudonauthor=D.+Touboulauthor=E.+F.+Queiroz&amp;title=Innovative+Omics-Based+Approaches+for+Prioritisation+and+Targeted+Isolation+of+Natural+Products+%E2%80%93+New+Strategies+for+Drug+Discovery&amp;doi=10.1039%2FC9NP00004F" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Innovative omics-based approaches for prioritisation and targeted isolation of natural products - new strategies for drug discovery</span></div><div class="casAuthors">Wolfender, Jean-Luc; Litaudon, Marc; Touboul, David; Queiroz, Emerson Ferreira</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">855-868</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Covering: 2013 to 2019The exploration of the chem. diversity of exts. from various biol. sources has led to major drug discoveries.  Over the past two decades, despite the introduction of advanced methodologies for natural product (NP) research (e.g., dereplication and high content screening), successful accounts of the validation of NPs as lead therapeutic candidates have been limited.  In this context, one of the main challenges faced is related to working with crude natural exts. because of their complex compn. and the inadequacies of classical bioguided isolation studies given the pace of high-throughput screening campaigns.  In line with the development of metabolomics, genomics and chemometrics, significant advances in metabolite profiling have been achieved and have generated high-quality massive genome and metabolome data on natural exts.  The unambiguous identification of each individual NP in an ext. using generic methods remains challenging.  However, the establishment of structural links among NPs via mol. network anal. and the detn. of common features of ext. compn. have provided invaluable information to the scientific community.  In this context, new multi-informational-based profiling approaches integrating taxonomic and/or bioactivity data can hold promise for the discovery and development of new bioactive compds. and return NPs back to an exciting era of development.  In this article, we examine recent studies that have the potential to improve the efficiency of NP prioritisation and to accelerate the targeted isolation of key NPs.  Perspectives on the field's evolution are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNC23cJr_lj7Vg90H21EOLACvtfcHk0liT_NhekdIFbQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXps1SqtL8%253D&amp;md5=72cac76e7b42521a7dfc82af3d8bb513</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1039%2FC9NP00004F&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9NP00004F%26sid%3Dliteratum%253Aachs%26aulast%3DWolfender%26aufirst%3DJ.-L.%26aulast%3DLitaudon%26aufirst%3DM.%26aulast%3DTouboul%26aufirst%3DD.%26aulast%3DQueiroz%26aufirst%3DE.%2BF.%26atitle%3DInnovative%2520Omics-Based%2520Approaches%2520for%2520Prioritisation%2520and%2520Targeted%2520Isolation%2520of%2520Natural%2520Products%2520%25E2%2580%2593%2520New%2520Strategies%2520for%2520Drug%2520Discovery%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2019%26volume%3D36%26spage%3D855%26epage%3D868%26doi%3D10.1039%2FC9NP00004F" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nothias, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nothias-Esposito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">da Silva, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protsyuk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarvepalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touboul, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Bioactivity-Based Molecular Networking for the Discovery of Drug Leads in Natural Product Bioassay-Guided Fractionation</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">758</span>â <span class="NLM_lpage">767</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.jnatprod.7b00737</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.7b00737" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjs1Snt7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=81&amp;publication_year=2018&amp;pages=758-767&amp;author=L.+F.+Nothiasauthor=M.+Nothias-Espositoauthor=R.+da+Silvaauthor=M.+Wangauthor=I.+Protsyukauthor=Z.+Zhangauthor=A.+Sarvepalliauthor=P.+Leyssenauthor=D.+Touboulauthor=J.+Costa&amp;title=Bioactivity-Based+Molecular+Networking+for+the+Discovery+of+Drug+Leads+in+Natural+Product+Bioassay-Guided+Fractionation&amp;doi=10.1021%2Facs.jnatprod.7b00737" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactivity-Based Molecular Networking for the Discovery of Drug Leads in Natural Product Bioassay-Guided Fractionation</span></div><div class="casAuthors">Nothias, Louis-Felix; Nothias-Esposito, Melissa; da Silva, Ricardo; Wang, Mingxun; Protsyuk, Ivan; Zhang, Zheng; Sarvepalli, Abi; Leyssen, Pieter; Touboul, David; Costa, Jean; Paolini, Julien; Alexandrov, Theodore; Litaudon, Marc; Dorrestein, Pieter C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">758-767</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">It is a common problem in natural product therapeutic lead discovery programs that despite good bioassay results in the initial ext., the active compd.(s) may not be isolated during subsequent bioassay-guided purifn.  Herein, we present the concept of bioactive mol. networking to find candidate active mols. directly from fractionated bioactive exts.  By employing tandem mass spectrometry, it is possible to accelerate the dereplication of mols. using mol. networking prior to subsequent isolation of the compds., and it is also possible to expose potentially bioactive mols. using bioactivity score prediction.  Indeed, bioactivity score prediction can be calcd. with the relative abundance of a mol. in fractions and the bioactivity level of each fraction.  For that reason, we have developed a bioinformatic workflow able to map bioactivity score in mol. networks and applied it for discovery of antiviral compds. from a previously investigated ext. of Euphorbia dendroides where the bioactive candidate mols. were not discovered following a classical bioassay-guided fractionation procedure.  It can be expected that this approach will be implemented as a systematic strategy, not only in current and future bioactive lead discovery from natural ext. collections but also for the reinvestigation of the untapped reservoir of bioactive analogs in previous bioassay-guided fractionation efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq34cZPrBsZ67Vg90H21EOLACvtfcHk0liT_NhekdIFbQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXjs1Snt7g%253D&amp;md5=3a945a4d97b5407ac5b22fce75acf50e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.7b00737&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.7b00737%26sid%3Dliteratum%253Aachs%26aulast%3DNothias%26aufirst%3DL.%2BF.%26aulast%3DNothias-Esposito%26aufirst%3DM.%26aulast%3Dda%2BSilva%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DProtsyuk%26aufirst%3DI.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DSarvepalli%26aufirst%3DA.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DTouboul%26aufirst%3DD.%26aulast%3DCosta%26aufirst%3DJ.%26atitle%3DBioactivity-Based%2520Molecular%2520Networking%2520for%2520the%2520Discovery%2520of%2520Drug%2520Leads%2520in%2520Natural%2520Product%2520Bioassay-Guided%2520Fractionation%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2018%26volume%3D81%26spage%3D758%26epage%3D767%26doi%3D10.1021%2Facs.jnatprod.7b00737" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ory, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nazih, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daoud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mocquard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourjot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margueritte, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delsuc, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bard, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pouchus, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Targeting Bioactive Compounds in Natural Extracts - Development of a Comprehensive Workflow Combining Chemical and Biological Data</span>. <i>Anal. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1070</i></span>,  <span class="NLM_fpage">29</span>â <span class="NLM_lpage">42</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.aca.2019.04.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1016%2Fj.aca.2019.04.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=31103165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovVOmsLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1070&amp;publication_year=2019&amp;pages=29-42&amp;author=L.+Oryauthor=E.+H.+Nazihauthor=S.+Daoudauthor=J.+Mocquardauthor=M.+Bourjotauthor=L.+Margueritteauthor=M.+A.+Delsucauthor=J.+M.+Bardauthor=Y.+F.+Pouchusauthor=S.+Bertrand&amp;title=Targeting+Bioactive+Compounds+in+Natural+Extracts+-+Development+of+a+Comprehensive+Workflow+Combining+Chemical+and+Biological+Data&amp;doi=10.1016%2Fj.aca.2019.04.038" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bioactive compounds in natural extracts - Development of a comprehensive workflow combining chemical and biological data</span></div><div class="casAuthors">Ory, Lucie; Nazih, El-Hassane; Daoud, Sahar; Mocquard, Julia; Bourjot, Melanie; Margueritte, Laure; Delsuc, Marc-Andre; Bard, Jean-Marie; Pouchus, Yves Francois; Bertrand, Samuel; Roullier, Catherine</div><div class="citationInfo"><span class="NLM_cas:title">Analytica Chimica Acta</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1070</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">29-42</span>CODEN:
                <span class="NLM_cas:coden">ACACAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2670</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In natural product drug discovery, several strategies have emerged to highlight specifically bioactive compd.(s) within complex mixts. (fractions or crude exts.) using metabolomics tools.  In this area, a great deal of interest has raised among the scientific community on strategies to link chem. profiles and assocd. biol. data, leading to the new field called "biochemometrics".  This article falls into this emerging research by proposing a complete workflow, which was divided into three major steps.  The first one consists in the fractionation of the same ext. using four different chromatog. stationary phases and appropriated elution conditions to obtain five fractions for each column.  The second step corresponds to the acquisition of chem. profiles using HPLC-HRMS anal., and the biol. evaluation of each fraction.  The last step evaluates the links between the relative abundances of mols. present in fractions (peak area) and the global bioactivity level obsd. for each fraction.  To this purpose, an original bioinformatics script (encoded with R Studio software) using the combination of four statistical models (Spearman, F-PCA, PLS, PLS-DA) was here developed giving a "Super list" of potential bioactive compds. together with a predictive score.  This strategy was validated by its application on a marine-derived Penicillium chrysogenum ext. exhibiting antiproliferative activity on breast cancer cells (MCF-7 cells).  After the three steps of the workflow, one main compd. was highlighted as responsible for the bioactivity and identified as ergosterol.  Its antiproliferative activity was confirmed with an IC50 of 0.10 Î¼M on MCF-7 cells.  The script efficiency was further demonstrated by comparing the results obtained with a different recently described approach based on NMR profiling and by virtually modifying the data to evaluate the computational tool behavior.  This approach represents a new and efficient tool to tackle some of the bottlenecks in natural product drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA_ZskSnWKW7Vg90H21EOLACvtfcHk0liT_NhekdIFbQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXovVOmsLw%253D&amp;md5=af32e028880d6391d11fa24c3971f8aa</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.aca.2019.04.038&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.aca.2019.04.038%26sid%3Dliteratum%253Aachs%26aulast%3DOry%26aufirst%3DL.%26aulast%3DNazih%26aufirst%3DE.%2BH.%26aulast%3DDaoud%26aufirst%3DS.%26aulast%3DMocquard%26aufirst%3DJ.%26aulast%3DBourjot%26aufirst%3DM.%26aulast%3DMargueritte%26aufirst%3DL.%26aulast%3DDelsuc%26aufirst%3DM.%2BA.%26aulast%3DBard%26aufirst%3DJ.%2BM.%26aulast%3DPouchus%26aufirst%3DY.%2BF.%26aulast%3DBertrand%26aufirst%3DS.%26atitle%3DTargeting%2520Bioactive%2520Compounds%2520in%2520Natural%2520Extracts%2520-%2520Development%2520of%2520a%2520Comprehensive%2520Workflow%2520Combining%2520Chemical%2520and%2520Biological%2520Data%26jtitle%3DAnal.%2520Chim.%2520Acta%26date%3D2019%26volume%3D1070%26spage%3D29%26epage%3D42%26doi%3D10.1016%2Fj.aca.2019.04.038" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olivon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allard, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koval, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Righi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genta-Jouve, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannecouque, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nothias, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cachet, X.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Bioactive Natural Products Prioritization Using Massive Multi-Informational Molecular Networks</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2644</span>â <span class="NLM_lpage">2651</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acschembio.7b00413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKntr%252FP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2017&amp;pages=2644-2651&amp;author=F.+Olivonauthor=P.+M.+Allardauthor=A.+Kovalauthor=D.+Righiauthor=G.+Genta-Jouveauthor=J.+Neytsauthor=C.+Apelauthor=C.+Pannecouqueauthor=L.+F.+Nothiasauthor=X.+Cachet&amp;title=Bioactive+Natural+Products+Prioritization+Using+Massive+Multi-Informational+Molecular+Networks&amp;doi=10.1021%2Facschembio.7b00413" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactive Natural Products Prioritization Using Massive Multi-informational Molecular Networks</span></div><div class="casAuthors">Olivon, Florent; Allard, Pierre-Marie; Koval, Alexey; Righi, Davide; Genta-Jouve, Gregory; Neyts, Johan; Apel, Cecile; Pannecouque, Christophe; Nothias, Louis-Felix; Cachet, Xavier; Marcourt, Laurence; Roussi, Fanny; Katanaev, Vladimir L.; Touboul, David; Wolfender, Jean-Luc; Litaudon, Marc</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2644-2651</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Natural products represent an inexhaustible source of novel therapeutic agents.  Their complex and constrained three-dimensional structures endow these mols. with exceptional biol. properties, thereby giving them a major role in drug discovery programs.  However, the search for new bioactive metabolites is hampered by the chem. complexity of the biol. matrixes in which they are found.  The purifn. of single constituents from such matrixes requires such a significant amt. of work that it should be ideally performed only on mols. of high potential value (i.e., chem. novelty and biol. activity).  Recent bioinformatics approaches based on mass spectrometry metabolite profiling methods are beginning to address the complex task of compd. identification within complex mixts.  However, in parallel to these developments, methods providing information on the bioactivity potential of natural products prior to their isolation are still lacking and are of key interest to target the isolation of valuable natural products only.  In the present investigation, we propose an integrated anal. strategy for bioactive natural products prioritization.  Our approach uses massive mol. networks embedding various informational layers (bioactivity and taxonomical data) to highlight potentially bioactive scaffolds within the chem. diversity of crude exts. collections.  We exemplify this workflow by targeting the isolation of predicted active and nonactive metabolites from two botanical sources (Bocquillonia nervosa and Neoguillauminia cleopatra) against two biol. targets (Wnt signaling pathway and chikungunya virus replication).  Eventually, the detection and isolation processes of a daphnane diterpene orthoester and four 12-deoxyphorbols inhibiting the Wnt signaling pathway and exhibiting potent antiviral activities against the CHIKV virus are detailed.  Combined with efficient metabolite annotation tools, this bioactive natural products prioritization pipeline proves to be efficient.  Implementation of this approach in drug discovery programs based on natural ext. screening should speed up and rationalize the isolation of bioactive natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsOBSoAGjVzLVg90H21EOLACvtfcHk0ljWVlN2uUhHyw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKntr%252FP&amp;md5=edb6828f781687eb1a6f9ab83497b24b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00413&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00413%26sid%3Dliteratum%253Aachs%26aulast%3DOlivon%26aufirst%3DF.%26aulast%3DAllard%26aufirst%3DP.%2BM.%26aulast%3DKoval%26aufirst%3DA.%26aulast%3DRighi%26aufirst%3DD.%26aulast%3DGenta-Jouve%26aufirst%3DG.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DApel%26aufirst%3DC.%26aulast%3DPannecouque%26aufirst%3DC.%26aulast%3DNothias%26aufirst%3DL.%2BF.%26aulast%3DCachet%26aufirst%3DX.%26atitle%3DBioactive%2520Natural%2520Products%2520Prioritization%2520Using%2520Massive%2520Multi-Informational%2520Molecular%2520Networks%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2644%26epage%3D2651%26doi%3D10.1021%2Facschembio.7b00413" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kurita, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glassey, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span> <span> </span><span class="NLM_article-title">Integration of High-Content Screening and Untargeted Metabolomics for Comprehensive Functional Annotation of Natural Product Libraries</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">11999</span>â <span class="NLM_lpage">12004</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1507743112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1073%2Fpnas.1507743112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=26371303" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2ntLnO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=112&amp;publication_year=2015&amp;pages=11999-12004&amp;author=K.+L.+Kuritaauthor=E.+Glasseyauthor=R.+G.+Linington&amp;title=Integration+of+High-Content+Screening+and+Untargeted+Metabolomics+for+Comprehensive+Functional+Annotation+of+Natural+Product+Libraries&amp;doi=10.1073%2Fpnas.1507743112" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Integration of high-content screening and untargeted metabolomics for comprehensive functional annotation of natural product libraries</span></div><div class="casAuthors">Kurita, Kenji L.; Glassey, Emerson; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">11999-12004</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Traditional natural products discovery using a combination of live/dead screening followed by iterative bioassay-guided fractionation affords no information about compd. structure or mode of action until late in the discovery process.  This leads to high rates of rediscovery and low probabilities of finding compds. with unique biol. and/or chem. properties.  By integrating image-based phenotypic screening in HeLa cells with high-resoln. untargeted metabolomics anal., we have developed a new platform, termed Compd. Activity Mapping, that is capable of directly predicting the identities and modes of action of bioactive constituents for any complex natural product ext. library.  This new tool can be used to rapidly identify novel bioactive constituents and provide predictions of compd. modes of action directly from primary screening data.  This approach inverts the natural products discovery process from the existing "grind and find" model to a targeted, hypothesis-driven discovery model where the chem. features and biol. function of bioactive metabolites are known early in the screening workflow, and lead compds. can be rationally selected based on biol. and/or chem. novelty.  We demonstrate the utility of the Compd. Activity Mapping platform by combining 10,977 mass spectral features and 58,032 biol. measurements from a library of 234 natural products exts. and integrating these two datasets to identify 13 clusters of fractions contg. 11 known compd. families and four new compds.  Using Compd. Activity Mapping we discovered the quinocinnolinomycins, a new family of natural products with a unique carbon skeleton that cause endoplasmic reticulum stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2SjVGolhWsLVg90H21EOLACvtfcHk0ljWVlN2uUhHyw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2ntLnO&amp;md5=36b4c261ec51d891a3818ba99dd0af6e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1507743112&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1507743112%26sid%3Dliteratum%253Aachs%26aulast%3DKurita%26aufirst%3DK.%2BL.%26aulast%3DGlassey%26aufirst%3DE.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26atitle%3DIntegration%2520of%2520High-Content%2520Screening%2520and%2520Untargeted%2520Metabolomics%2520for%2520Comprehensive%2520Functional%2520Annotation%2520of%2520Natural%2520Product%2520Libraries%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D11999%26epage%3D12004%26doi%3D10.1073%2Fpnas.1507743112" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levander, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shofstahl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RÃ¶mpp, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pizarro, A. D.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">MzML - A Community Standard for Mass Spectrometry Data</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">R110.000133</span>, <span class="refDoi">&nbsp;DOI: 10.1074/mcp.R110.000133</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1074%2Fmcp.R110.000133" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=20716697" title="">PubMed</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2011&amp;pages=R110.000133&amp;author=L.+Martensauthor=M.+Chambersauthor=M.+Sturmauthor=D.+Kessnerauthor=F.+Levanderauthor=J.+Shofstahlauthor=W.+H.+Tangauthor=A.+R%C3%B6mppauthor=S.+Neumannauthor=A.+D.+Pizarro&amp;title=MzML+-+A+Community+Standard+for+Mass+Spectrometry+Data&amp;doi=10.1074%2Fmcp.R110.000133" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fmcp.R110.000133&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.R110.000133%26sid%3Dliteratum%253Aachs%26aulast%3DMartens%26aufirst%3DL.%26aulast%3DChambers%26aufirst%3DM.%26aulast%3DSturm%26aufirst%3DM.%26aulast%3DKessner%26aufirst%3DD.%26aulast%3DLevander%26aufirst%3DF.%26aulast%3DShofstahl%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DW.%2BH.%26aulast%3DR%25C3%25B6mpp%26aufirst%3DA.%26aulast%3DNeumann%26aufirst%3DS.%26aulast%3DPizarro%26aufirst%3DA.%2BD.%26atitle%3DMzML%2520-%2520A%2520Community%2520Standard%2520for%2520Mass%2520Spectrometry%2520Data%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2011%26volume%3D10%26spage%3DR110.000133%26doi%3D10.1074%2Fmcp.R110.000133" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pluskal, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castillo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villar-Briones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OreÅ¡iÄ, M.</span></span> <span> </span><span class="NLM_article-title">MZmine 2: Modular Framework for Processing, Visualizing, and Analyzing Mass Spectrometry-Based Molecular Profile Data</span>. <i>BMC Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">395</span>, <span class="refDoi">&nbsp;DOI: 10.1186/1471-2105-11-395</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1186%2F1471-2105-11-395" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=20650010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A280%3ADC%252BC3cjjsVymsA%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2010&amp;pages=395&amp;author=T.+Pluskalauthor=S.+Castilloauthor=A.+Villar-Brionesauthor=M.+Ore%C5%A1i%C4%8D&amp;title=MZmine+2%3A+Modular+Framework+for+Processing%2C+Visualizing%2C+and+Analyzing+Mass+Spectrometry-Based+Molecular+Profile+Data&amp;doi=10.1186%2F1471-2105-11-395" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">MZmine 2: modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data</span></div><div class="casAuthors">Pluskal Tomas; Castillo Sandra; Villar-Briones Alejandro; Oresic Matej</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">395</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Mass spectrometry (MS) coupled with online separation methods is commonly applied for differential and quantitative profiling of biological samples in metabolomic as well as proteomic research.  Such approaches are used for systems biology, functional genomics, and biomarker discovery, among others.  An ongoing challenge of these molecular profiling approaches, however, is the development of better data processing methods.  Here we introduce a new generation of a popular open-source data processing toolbox, MZmine 2.  RESULTS:  A key concept of the MZmine 2 software design is the strict separation of core functionality and data processing modules, with emphasis on easy usability and support for high-resolution spectra processing.  Data processing modules take advantage of embedded visualization tools, allowing for immediate previews of parameter settings.  Newly introduced functionality includes the identification of peaks using online databases, MSn data support, improved isotope pattern support, scatter plot visualization, and a new method for peak list alignment based on the random sample consensus (RANSAC) algorithm.  The performance of the RANSAC alignment was evaluated using synthetic datasets as well as actual experimental data, and the results were compared to those obtained using other alignment algorithms.  CONCLUSIONS:  MZmine 2 is freely available under a GNU GPL license and can be obtained from the project website at: http://mzmine.sourceforge.net/.  The current version of MZmine 2 is suitable for processing large batches of data and has been applied to both targeted and non-targeted metabolomic analyses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSPbblHwbnggjUE-swuRqsHfW6udTcc2ebOz8wbGxunG7ntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC3cjjsVymsA%253D%253D&amp;md5=e6e2ac996767f8526daccbdb7f4929e0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-11-395&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-11-395%26sid%3Dliteratum%253Aachs%26aulast%3DPluskal%26aufirst%3DT.%26aulast%3DCastillo%26aufirst%3DS.%26aulast%3DVillar-Briones%26aufirst%3DA.%26aulast%3DOre%25C5%25A1i%25C4%258D%26aufirst%3DM.%26atitle%3DMZmine%25202%253A%2520Modular%2520Framework%2520for%2520Processing%252C%2520Visualizing%252C%2520and%2520Analyzing%2520Mass%2520Spectrometry-Based%2520Molecular%2520Profile%2520Data%26jtitle%3DBMC%2520Bioinformatics%26date%3D2010%26volume%3D11%26spage%3D395%26doi%3D10.1186%2F1471-2105-11-395" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carver, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phelan, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watrous, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapono, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luzzatto-Knaan, T.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Sharing and Community Curation of Mass Spectrometry Data with Global Natural Products Social Molecular Networking</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">828</span>â <span class="NLM_lpage">837</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nbt.3597</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1038%2Fnbt.3597" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=27504778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlaitLnE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=2016&amp;pages=828-837&amp;author=M.+Wangauthor=J.+J.+Carverauthor=V.+V.+Phelanauthor=L.+M.+Sanchezauthor=N.+Gargauthor=Y.+Pengauthor=D.+D.+Nguyenauthor=J.+Watrousauthor=C.+A.+Kaponoauthor=T.+Luzzatto-Knaan&amp;title=Sharing+and+Community+Curation+of+Mass+Spectrometry+Data+with+Global+Natural+Products+Social+Molecular+Networking&amp;doi=10.1038%2Fnbt.3597" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking</span></div><div class="casAuthors">Wang, Mingxun; Carver, Jeremy J.; Phelan, Vanessa V.; Sanchez, Laura M.; Garg, Neha; Peng, Yao; Nguyen, Don Duy; Watrous, Jeramie; Kapono, Clifford A.; Luzzatto-Knaan, Tal; Porto, Carla; Bouslimani, Amina; Melnik, Alexey V.; Meehan, Michael J.; Liu, Wei-Ting; Crusemann, Max; Boudreau, Paul D.; Esquenazi, Eduardo; Sandoval-Calderon, Mario; Kersten, Roland D.; Pace, Laura A.; Quinn, Robert A.; Duncan, Katherine R.; Hsu, Cheng-Chih; Floros, Dimitrios J.; Gavilan, Ronnie G.; Kleigrewe, Karin; Northen, Trent; Dutton, Rachel J.; Parrot, Delphine; Carlson, Erin E.; Aigle, Bertrand; Michelsen, Charlotte F.; Jelsbak, Lars; Sohlenkamp, Christian; Pevzner, Pavel; Edlund, Anna; McLean, Jeffrey; Piel, Jorn; Murphy, Brian T.; Gerwick, Lena; Liaw, Chih-Chuang; Yang, Yu-Liang; Humpf, Hans-Ulrich; Maansson, Maria; Keyzers, Robert A.; Sims, Amy C.; Johnson, Andrew R.; Sidebottom, Ashley M.; Sedio, Brian E.; Klitgaard, Andreas; Larson, Charles B.; Boya P, Cristopher A.; Torres-Mendoza, Daniel; Gonzalez, David J.; Silva, Denise B.; Marques, Lucas M.; Demarque, Daniel P.; Pociute, Egle; O'Neill, Ellis C.; Briand, Enora; Helfrich, Eric J. N.; Granatosky, Eve A.; Glukhov, Evgenia; Ryffel, Florian; Houson, Hailey; Mohimani, Hosein; Kharbush, Jenan J.; Zeng, Yi; Vorholt, Julia A.; Kurita, Kenji L.; Charusanti, Pep; McPhail, Kerry L.; Nielsen, Kristian Fog; Vuong, Lisa; Elfeki, Maryam; Traxler, Matthew F.; Engene, Niclas; Koyama, Nobuhiro; Vining, Oliver B.; Baric, Ralph; Silva, Ricardo R.; Mascuch, Samantha J.; Tomasi, Sophie; Jenkins, Stefan; Macherla, Venkat; Hoffman, Thomas; Agarwal, Vinayak; Williams, Philip G.; Dai, Jingqui; Neupane, Ram; Gurr, Joshua; Rodriguez, Andres M. C.; Lamsa, Anne; Zhang, Chen; Dorrestein, Kathleen; Duggan, Brendan M.; Almaliti, Jehad; Allard, Pierre-Marie; Phapale, Prasad; Nothias, Louis-Felix; Alexandrov, Theodore; Litaudon, Marc; Wolfender, Jean-Luc; Kyle, Jennifer E.; Metz, Thomas O.; Peryea, Tyler; Nguyen, Dac-Trung; Van Leer, Danielle; Shinn, Paul; Jadhav, Ajit; Muller, Rolf; Waters, Katrina M.; Shi, Wenyuan; Liu, Xueting; Zhang, Lixin; Knight, Rob; Jensen, Paul R.; Palsson, Bernhard O.; Pogliano, Kit; Linington, Roger G.; Gutierrez, Marcelino; Lopes, Norberto P.; Gerwick, William H.; Moore, Bradley S.; Dorrestein, Pieter C.; Bandeira, Nuno</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">828-837</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential of the diverse chemistries present in natural products (NP) for biotechnol. and medicine remains untapped because NP databases are not searchable with raw data and the NP community has no way to share data other than in published papers.  Although mass spectrometry (MS) techniques are well-suited to high-throughput characterization of NP, there is a pressing need for an infrastructure to enable sharing and curation of data.  We present Global Natural Products Social Mol. Networking (GNPS; http://gnps.ucsd.edu), an open-access knowledge base for community-wide organization and sharing of raw, processed or identified tandem mass (MS/MS) spectrometry data.  In GNPS, crowdsourced curation of freely available community-wide ref. MS libraries will underpin improved annotations.  Data-driven social-networking should facilitate identification of spectra and foster collaborations.  We also introduce the concept of 'living data' through continuous reanal. of deposited data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptdRCove5agrVg90H21EOLACvtfcHk0lgOhqLYRg8DFA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XhtlaitLnE&amp;md5=e6ca23ede2d85dd1460a5d73da542444</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3597&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3597%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%26aulast%3DCarver%26aufirst%3DJ.%2BJ.%26aulast%3DPhelan%26aufirst%3DV.%2BV.%26aulast%3DSanchez%26aufirst%3DL.%2BM.%26aulast%3DGarg%26aufirst%3DN.%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DNguyen%26aufirst%3DD.%2BD.%26aulast%3DWatrous%26aufirst%3DJ.%26aulast%3DKapono%26aufirst%3DC.%2BA.%26aulast%3DLuzzatto-Knaan%26aufirst%3DT.%26atitle%3DSharing%2520and%2520Community%2520Curation%2520of%2520Mass%2520Spectrometry%2520Data%2520with%2520Global%2520Natural%2520Products%2520Social%2520Molecular%2520Networking%26jtitle%3DNat.%2520Biotechnol.%26date%3D2016%26volume%3D34%26spage%3D828%26epage%3D837%26doi%3D10.1038%2Fnbt.3597" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schulze, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woerhmann, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span> <span> </span><span class="NLM_article-title">Function-Firstâ Lead Discovery: Mode of Action Profiling of Natural Product Libraries Using Image-Based Screening</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">285</span>â <span class="NLM_lpage">295</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.chembiol.2012.12.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1016%2Fj.chembiol.2012.12.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=23438757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFSlu7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2013&amp;pages=285-295&amp;author=C.+J.+Schulzeauthor=W.+M.+Brayauthor=M.+H.+Woerhmannauthor=J.+Stuartauthor=R.+S.+Lokeyauthor=R.+G.+Linington&amp;title=Function-First%E2%80%9D+Lead+Discovery%3A+Mode+of+Action+Profiling+of+Natural+Product+Libraries+Using+Image-Based+Screening&amp;doi=10.1016%2Fj.chembiol.2012.12.007" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">"Function-First" Lead Discovery: Mode of Action Profiling of Natural Product Libraries Using Image-Based Screening</span></div><div class="casAuthors">Schulze, Christopher J.; Bray, Walter M.; Woerhmann, Marcos H.; Stuart, Joshua; Lokey, R. Scott; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry &amp; Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">285-295</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cytol. profiling is a high-content image-based screening technol. that provides insight into the mode of action (MOA) for test compds. by directly measuring hundreds of phenotypic cellular features.  We have extended this recently reported technol. to the mechanistic characterization of unknown natural products libraries for the direct prediction of compd. MOAs at the primary screening stage.  By analyzing a training set of com. compds. of known mechanism and comparing these profiles to those obtained from natural product library members, we have successfully annotated exts. based on MOA, dereplicated known compds. based on biol. similarity to the training set, and identified and predicted the MOA of a unique family of iron siderophores.  Coupled with traditional anal. techniques, cytol. profiling provides an avenue for the creation of "function-first" approaches to natural products discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmxEmwpkTYBbVg90H21EOLACvtfcHk0lgOhqLYRg8DFA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXjtFSlu7g%253D&amp;md5=8bec654bcd3ae6b1a93156a8935f841f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.12.007&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.12.007%26sid%3Dliteratum%253Aachs%26aulast%3DSchulze%26aufirst%3DC.%2BJ.%26aulast%3DBray%26aufirst%3DW.%2BM.%26aulast%3DWoerhmann%26aufirst%3DM.%2BH.%26aulast%3DStuart%26aufirst%3DJ.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26atitle%3DFunction-First%25E2%2580%259D%2520Lead%2520Discovery%253A%2520Mode%2520of%2520Action%2520Profiling%2520of%2520Natural%2520Product%2520Libraries%2520Using%2520Image-Based%2520Screening%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D285%26epage%3D295%26doi%3D10.1016%2Fj.chembiol.2012.12.007" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guttman, A.</span></span> <span> </span><span class="NLM_article-title">R-Trees: A Dynamic Index Structure for Spatial Searching</span>. <i>SIGMOD Rec.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">47</span>â <span class="NLM_lpage">57</span>, <span class="refDoi">&nbsp;DOI: 10.1145/971697.602266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1145%2F971697.602266" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=1984&amp;pages=47-57&amp;author=A.+Guttman&amp;title=R-Trees%3A+A+Dynamic+Index+Structure+for+Spatial+Searching&amp;doi=10.1145%2F971697.602266" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1145%2F971697.602266&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1145%252F971697.602266%26sid%3Dliteratum%253Aachs%26aulast%3DGuttman%26aufirst%3DA.%26atitle%3DR-Trees%253A%2520A%2520Dynamic%2520Index%2520Structure%2520for%2520Spatial%2520Searching%26jtitle%3DSIGMOD%2520Rec.%26date%3D1984%26volume%3D14%26spage%3D47%26epage%3D57%26doi%3D10.1145%2F971697.602266" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span> <span> </span><span class="NLM_article-title">Development of Antibiotic Activity Profile Screening for the Classification and Discovery of Natural Product Antibiotics</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1483</span>â <span class="NLM_lpage">1495</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.chembiol.2012.09.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1016%2Fj.chembiol.2012.09.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=23177202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslejur%252FP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2012&amp;pages=1483-1495&amp;author=W.+R.+Wongauthor=A.+G.+Oliverauthor=R.+G.+Linington&amp;title=Development+of+Antibiotic+Activity+Profile+Screening+for+the+Classification+and+Discovery+of+Natural+Product+Antibiotics&amp;doi=10.1016%2Fj.chembiol.2012.09.014" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Antibiotic Activity Profile Screening for the Classification and Discovery of Natural Product Antibiotics</span></div><div class="casAuthors">Wong, Weng Ruh; Oliver, Allen G.; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry &amp; Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1483-1495</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Despite recognition of the looming antibiotic crisis by healthcare professionals, the no. of new antibiotics reaching the clinic continues to decline sharply.  This study aimed to establish an antibiotic profiling strategy using a panel of clin. relevant bacterial strains to create unique biol. fingerprints for all major classes of antibiotics.  Antibiotic mode of action profile (BioMAP) screening has been shown to effectively cluster antibiotics by structural class based on these fingerprints.  Using this approach, we have accurately predicted the presence of known antibiotics in natural product exts. and have discovered a naphthoquinone-based antibiotic from our marine natural product library that possesses a unique carbon skeleton.  We have demonstrated that bioactivity fingerprinting is a successful strategy for profiling antibiotic lead compds. and that BioMAP can be applied to the discovery of new natural product antibiotics leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9yIDXE5qmhLVg90H21EOLACvtfcHk0ljTNzXFyLKxjw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38Xhslejur%252FP&amp;md5=7817766457640524ce86f1d8476a3156</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.09.014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.09.014%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DW.%2BR.%26aulast%3DOliver%26aufirst%3DA.%2BG.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26atitle%3DDevelopment%2520of%2520Antibiotic%2520Activity%2520Profile%2520Screening%2520for%2520the%2520Classification%2520and%2520Discovery%2520of%2520Natural%2520Product%2520Antibiotics%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D1483%26epage%3D1495%26doi%3D10.1016%2Fj.chembiol.2012.09.014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Santen, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aniebok, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balunas, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunsko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neto, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">CastaÃ±o-Espriu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T. N.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">The Natural Products Atlas: An Open Access Knowledge Base for Microbial Natural Products Discovery</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1824</span>â <span class="NLM_lpage">1833</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscentsci.9b00806</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.9b00806" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCit73P" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2019&amp;pages=1824-1833&amp;author=J.+A.+van%0ASantenauthor=G.+Jacobauthor=A.+L.+Singhauthor=V.+Aniebokauthor=M.+J.+Balunasauthor=D.+Bunskoauthor=F.+C.+Netoauthor=L.+Casta%C3%B1o-Espriuauthor=C.+Changauthor=T.+N.+Clark&amp;title=The+Natural+Products+Atlas%3A+An+Open+Access+Knowledge+Base+for+Microbial+Natural+Products+Discovery&amp;doi=10.1021%2Facscentsci.9b00806" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The Natural Products Atlas: An Open Access Knowledge Base for Microbial Natural Products Discovery</span></div><div class="casAuthors">van Santen, Jeffrey A.; Jacob, Gregoire; Singh, Amrit Leen; Aniebok, Victor; Balunas, Marcy J.; Bunsko, Derek; Neto, Fausto Carnevale; Castano-Espriu, Laia; Chang, Chen; Clark, Trevor N.; Cleary Little, Jessica L.; Delgadillo, David A.; Dorrestein, Pieter C.; Duncan, Katherine R.; Egan, Joseph M.; Galey, Melissa M.; Haeckl, F. P. Jake; Hua, Alex; Hughes, Alison H.; Iskakova, Dasha; Khadilkar, Aswad; Lee, Jung-Ho; Lee, Sanghoon; LeGrow, Nicole; Liu, Dennis Y.; Macho, Jocelyn M.; McCaughey, Catherine S.; Medema, Marnix H.; Neupane, Ram P.; ODonnell, Timothy J.; Paula, Jasmine S.; Sanchez, Laura M.; Shaikh, Anam F.; Soldatou, Sylvia; Terlouw, Barbara R.; Tran, Tuan Anh; Valentine, Mercia; van der Hooft, Justin J. J.; Vo, Duy A.; Wang, Mingxun; Wilson, Darryl; Zink, Katherine E.; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1824-1833</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Despite rapid evolution in the area of microbial natural products chem., there is currently no open access database contg. all microbially produced natural product structures.  Lack of availability of these data is preventing the implementation of new technologies in natural products science.  Specifically, development of new computational strategies for compd. characterization and identification are being hampered by the lack of a comprehensive database of known compds. against which to compare exptl. data.  The creation of an open access, community-maintained database of microbial natural product structures would enable the development of new technologies in natural products discovery, and improve the interoperability of existing natural products data resources.  However, these data are spread unevenly throughout the historical scientific literature, including both journal articles and international patents.  These documents have no std. format, are often not digitized as machine readable text, and are not publicly available.  Further, none of these documents have assocd. structure files (e.g., MOL, InChI, or SMILES), instead contg. images of structures.  This makes extn. and formatting of relevant natural products data a formidable challenge.  Using a combination of manual curation and automated data mining approaches we have created a database of microbial natural products (The Natural Products Atlas, www.npatlas.org) that includes 24,594 compds. and contains referenced data for structure, compd. names, source organisms, isolation refs., total syntheses and instances of structural reassignment.  This database is accompanied by an interactive web portal that permits searching by structure, substructure, and phys. properties.  The Web site also provides mechanisms for visualizing natural products chem. space, and dashboards for displaying author and discovery timeline data.  These interactive tools offer a powerful knowledge base for natural products discovery with a central interface for structure and property-based searching, and presents new viewpoints on structural diversity in natural products.  The Natural Products Atlas has been developed under FAIR principles (Findable, Accessible, Interoperable, and Reusable) and is integrated with other emerging natural product databases, including the Min. Information About a Biosynthetic Gene Cluster (MIBiG) repository, and the Global Natural Products Social Mol. Networking (GNPS) platform.  It is designed as a community-supported resource to provide a central repository for known natural product structures from microorganisms, and is the 1st comprehensive, open access resource of this type.  It is expected that the Natural Products Atlas will enable the development of new natural products discovery modalities and accelerate the process of structural characterization for complex natural products libraries.  The Natural Products Atlas is a new online database of microbially derived natural product structures, designed as a comprehensive open access repository for the scientific community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv6naGjGM-w7Vg90H21EOLACvtfcHk0ljTNzXFyLKxjw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXitFCit73P&amp;md5=1ebc951ca96d93dd36d65dd8b3b3ad67</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.9b00806&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.9b00806%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BSanten%26aufirst%3DJ.%2BA.%26aulast%3DJacob%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DA.%2BL.%26aulast%3DAniebok%26aufirst%3DV.%26aulast%3DBalunas%26aufirst%3DM.%2BJ.%26aulast%3DBunsko%26aufirst%3DD.%26aulast%3DNeto%26aufirst%3DF.%2BC.%26aulast%3DCasta%25C3%25B1o-Espriu%26aufirst%3DL.%26aulast%3DChang%26aufirst%3DC.%26aulast%3DClark%26aufirst%3DT.%2BN.%26atitle%3DThe%2520Natural%2520Products%2520Atlas%253A%2520An%2520Open%2520Access%2520Knowledge%2520Base%2520for%2520Microbial%2520Natural%2520Products%2520Discovery%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2019%26volume%3D5%26spage%3D1824%26epage%3D1833%26doi%3D10.1021%2Facscentsci.9b00806" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skellam, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strangman, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. L. C.</span></span> <span> </span><span class="NLM_article-title">Identification of Micromonolactam, a New Polyene Macrocyclic Lactam from Two Marine Micromonospora Strains Using Chemical and Molecular Methods: Clarification of the Biosynthetic Pathway from a Glutamate Starter Unit</span>. <i>J. Antibiot. (Tokyo)</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">431</span>â <span class="NLM_lpage">441</span>, <span class="refDoi">&nbsp;DOI: 10.1038/ja.2013.34</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1038%2Fja.2013.34" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=23677034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A280%3ADC%252BC3snktVejsg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2013&amp;pages=431-441&amp;author=E.+J.+Skellamauthor=A.+K.+Stewartauthor=W.+K.+Strangmanauthor=J.+L.+C.+Wright&amp;title=Identification+of+Micromonolactam%2C+a+New+Polyene+Macrocyclic+Lactam+from+Two+Marine+Micromonospora+Strains+Using+Chemical+and+Molecular+Methods%3A+Clarification+of+the+Biosynthetic+Pathway+from+a+Glutamate+Starter+Unit&amp;doi=10.1038%2Fja.2013.34" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of micromonolactam, a new polyene macrocyclic lactam from two marine Micromonospora strains using chemical and molecular methods: clarification of the biosynthetic pathway from a glutamate starter unit</span></div><div class="casAuthors">Skellam Elizabeth Julia; Stewart Allison Kathleen; Strangman Wendy Karen; Wright Jeffrey Lawson Cameron</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of antibiotics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">431-41</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Through a combination of chemical and molecular analysis, a new polyene macrolactam named micromonolactam was obtained from two marine-derived Micromonospora species.  This new polyene metabolite is a constitutional isomer of salinilactam A but contains a different polyene pattern and one cis double bond, in contrast to the all trans structure reported for salinilactam A.  The molecular analysis data also established that micromonolactam is a hybrid polyketide derived from 11 polyketide units and a modified glutamate starter unit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQC-9nbPIe_2TToTC3aBaesfW6udTcc2eZbQ8wnkc7b0rntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC3snktVejsg%253D%253D&amp;md5=8b07f2b229caebef239fd2966a368125</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fja.2013.34&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2013.34%26sid%3Dliteratum%253Aachs%26aulast%3DSkellam%26aufirst%3DE.%2BJ.%26aulast%3DStewart%26aufirst%3DA.%2BK.%26aulast%3DStrangman%26aufirst%3DW.%2BK.%26aulast%3DWright%26aufirst%3DJ.%2BL.%2BC.%26atitle%3DIdentification%2520of%2520Micromonolactam%252C%2520a%2520New%2520Polyene%2520Macrocyclic%2520Lactam%2520from%2520Two%2520Marine%2520Micromonospora%2520Strains%2520Using%2520Chemical%2520and%2520Molecular%2520Methods%253A%2520Clarification%2520of%2520the%2520Biosynthetic%2520Pathway%2520from%2520a%2520Glutamate%2520Starter%2520Unit%26jtitle%3DJ.%2520Antibiot.%2520%2528Tokyo%2529%26date%3D2013%26volume%3D66%26spage%3D431%26epage%3D441%26doi%3D10.1038%2Fja.2013.34" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoshino, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awakawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asamizu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, I.</span></span> <span> </span><span class="NLM_article-title">Mycolic Acid Containing Bacterium Stimulates Tandem Cyclization of Polyene Macrolactam in a Lake Sediment Derived Rare Actinomycete</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4992</span>â <span class="NLM_lpage">4995</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.orglett.7b02508</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.7b02508" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKmsrzI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2017&amp;pages=4992-4995&amp;author=S.+Hoshinoauthor=M.+Okadaauthor=T.+Awakawaauthor=S.+Asamizuauthor=H.+Onakaauthor=I.+Abe&amp;title=Mycolic+Acid+Containing+Bacterium+Stimulates+Tandem+Cyclization+of+Polyene+Macrolactam+in+a+Lake+Sediment+Derived+Rare+Actinomycete&amp;doi=10.1021%2Facs.orglett.7b02508" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Mycolic Acid Containing Bacterium Stimulates Tandem Cyclization of Polyene Macrolactam in a Lake Sediment Derived Rare Actinomycete</span></div><div class="casAuthors">Hoshino, Shotaro; Okada, Masahiro; Awakawa, Takayoshi; Asamizu, Shumpei; Onaka, Hiroyasu; Abe, Ikuro</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4992-4995</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two novel macrolactams, dracolactams A and B, were identified from a combined-culture of Micromonospora species and a mycolic-acid contg. bacterium (MACB).  Their structures and stereochemistries were completely assigned, based on spectroscopic analyses and chem. derivatization.  Both dracolactams were probably generated from a common macrolactam precursor produced by the Micromonospora species.  In this combined-culture system, MACB is likely to activate cryptic oxidase genes in the Micromonospora species and induce the downstream polyene macrolactam cyclization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphxcpmQ0K137Vg90H21EOLACvtfcHk0ljPTKVzBEXtxw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKmsrzI&amp;md5=e75ba729cd49df12aeef4bf87ecf9f8e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.7b02508&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.7b02508%26sid%3Dliteratum%253Aachs%26aulast%3DHoshino%26aufirst%3DS.%26aulast%3DOkada%26aufirst%3DM.%26aulast%3DAwakawa%26aufirst%3DT.%26aulast%3DAsamizu%26aufirst%3DS.%26aulast%3DOnaka%26aufirst%3DH.%26aulast%3DAbe%26aufirst%3DI.%26atitle%3DMycolic%2520Acid%2520Containing%2520Bacterium%2520Stimulates%2520Tandem%2520Cyclization%2520of%2520Polyene%2520Macrolactam%2520in%2520a%2520Lake%2520Sediment%2520Derived%2520Rare%2520Actinomycete%26jtitle%3DOrg.%2520Lett.%26date%3D2017%26volume%3D19%26spage%3D4992%26epage%3D4995%26doi%3D10.1021%2Facs.orglett.7b02508" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyadoh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodama, Y.</span></span> <span> </span><span class="NLM_article-title">Novel Antibiotics SF2738A, B and C, and Their Analogs Produced by Streptomyces Sp</span>. <i>J. Antibiot. (Tokyo)</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1385</span>â <span class="NLM_lpage">1394</span>, <span class="refDoi">&nbsp;DOI: 10.7164/antibiotics.47.1385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.7164%2Fantibiotics.47.1385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=7844033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A280%3ADyaK2M7ks1CitQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=1994&amp;pages=1385-1394&amp;author=S.+Gomiauthor=S.+Amanoauthor=E.+Satoauthor=S.+Miyadohauthor=Y.+Kodama&amp;title=Novel+Antibiotics+SF2738A%2C+B+and+C%2C+and+Their+Analogs+Produced+by+Streptomyces+Sp&amp;doi=10.7164%2Fantibiotics.47.1385" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Novel antibiotics SF2738A, B and C, and their analogs produced by Streptomyces sp</span></div><div class="casAuthors">Gomi S; Amano S; Sato E; Miyadoh S; Kodama Y</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of antibiotics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1385-94</span>
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Three new antibiotics SF2738A, B and C, and their analogs were isolated from the culture broth of Streptomyces sp.  The antibiotics are active against Gram-positive bacteria, Gram-negative bacteria and fungi, and exhibited cytotoxic activity against P388 murine leukemia cells with IC50 values of 0.08, 0.25 and 7.5 micrograms/ml, respectively.  Their structures were determined by spectral analyses and chemical conversion.  Especially, the structure of SF2738A was confirmed to be (E)-((4-methoxy-5-methylthio-2-(2-pyridyl)pyridin-6-yl)methylene)azan ol by X-ray crystallographic analysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRU_1c4u7jGETEwO064Z95rfW6udTcc2eZbQ8wnkc7b0rntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADyaK2M7ks1CitQ%253D%253D&amp;md5=a5abd0d5ae3537e4f845b6a4af557dfc</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.47.1385&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.47.1385%26sid%3Dliteratum%253Aachs%26aulast%3DGomi%26aufirst%3DS.%26aulast%3DAmano%26aufirst%3DS.%26aulast%3DSato%26aufirst%3DE.%26aulast%3DMiyadoh%26aufirst%3DS.%26aulast%3DKodama%26aufirst%3DY.%26atitle%3DNovel%2520Antibiotics%2520SF2738A%252C%2520B%2520and%2520C%252C%2520and%2520Their%2520Analogs%2520Produced%2520by%2520Streptomyces%2520Sp%26jtitle%3DJ.%2520Antibiot.%2520%2528Tokyo%2529%26date%3D1994%26volume%3D47%26spage%3D1385%26epage%3D1394%26doi%3D10.7164%2Fantibiotics.47.1385" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shindo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamagishi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, H.</span></span> <span> </span><span class="NLM_article-title">Collismycins A and B, Novel Non-Steroidal Inhibitors of Dexamethasone-Glucocorticoid Receptor Binding</span>. <i>J. Antibiot. (Tokyo)</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1072</span>â <span class="NLM_lpage">1074</span>, <span class="refDoi">&nbsp;DOI: 10.7164/antibiotics.47.1072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.7164%2Fantibiotics.47.1072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=7928696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A280%3ADyaK2M%252FhtlOhtQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=1994&amp;pages=1072-1074&amp;author=K.+Shindoauthor=Y.+Yamagishiauthor=Y.+Okadaauthor=H.+Kawai&amp;title=Collismycins+A+and+B%2C+Novel+Non-Steroidal+Inhibitors+of+Dexamethasone-Glucocorticoid+Receptor+Binding&amp;doi=10.7164%2Fantibiotics.47.1072" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Collismycins A and B, novel non-steroidal inhibitors of dexamethasone-glucocorticoid receptor binding</span></div><div class="casAuthors">Shindo K; Yamagishi Y; Okada Y; Kawai H</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of antibiotics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1072-4</span>
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcShmd3Kb6P_ycve_zCu79I8fW6udTcc2eYglUEyt-_vk7ntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADyaK2M%252FhtlOhtQ%253D%253D&amp;md5=b1a9e19078279c9df17ab825fedfd45f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.47.1072&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.47.1072%26sid%3Dliteratum%253Aachs%26aulast%3DShindo%26aufirst%3DK.%26aulast%3DYamagishi%26aufirst%3DY.%26aulast%3DOkada%26aufirst%3DY.%26aulast%3DKawai%26aufirst%3DH.%26atitle%3DCollismycins%2520A%2520and%2520B%252C%2520Novel%2520Non-Steroidal%2520Inhibitors%2520of%2520Dexamethasone-Glucocorticoid%2520Receptor%2520Binding%26jtitle%3DJ.%2520Antibiot.%2520%2528Tokyo%2529%26date%3D1994%26volume%3D47%26spage%3D1072%26epage%3D1074%26doi%3D10.7164%2Fantibiotics.47.1072" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seyedsayamdost, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clardy, J.</span></span> <span> </span><span class="NLM_article-title">Structure and Biosynthesis of Amychelin, an Unusual Mixed-Ligand Siderophore from Amycolatopsis Sp. AA4</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">11434</span>â <span class="NLM_lpage">11437</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja203577e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja203577e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFykt78%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=133&amp;publication_year=2011&amp;pages=11434-11437&amp;author=M.+R.+Seyedsayamdostauthor=M.+F.+Traxlerauthor=S.+L.+Zhengauthor=R.+Kolterauthor=J.+Clardy&amp;title=Structure+and+Biosynthesis+of+Amychelin%2C+an+Unusual+Mixed-Ligand+Siderophore+from+Amycolatopsis+Sp.+AA4&amp;doi=10.1021%2Fja203577e" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Biosynthesis of Amychelin, an Unusual Mixed-Ligand Siderophore from Amycolatopsis sp. AA4</span></div><div class="casAuthors">Seyedsayamdost, Mohammad R.; Traxler, Matthew F.; Zheng, Shao-Liang; Kolter, Roberto; Clardy, Jon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">11434-11437</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Actinobacteria generate a large no. of structurally diverse small mols. with potential therapeutic value.  Genomic analyses of this productive group of bacteria show that their genetic potential to manuf. small mols. exceeds their obsd. ability by roughly an order of magnitude, and this revelation has prompted a no. of studies to identify members of the unknown majority.  As a potential window into this cryptic secondary metabolome, pairwise assays for developmental interactions within a set of 20 sequenced actinomycetes were carried out.  These assays revealed that Amycolatopsis sp. AA4, a so-called "rare" actinomycete, produces a novel siderophore, amychelin, which alters the developmental processes of several neighboring streptomycetes.  Using this phenotype as an assay, we isolated amychelin and solved its structure by NMR and MS methods coupled with an X-ray crystallog. anal. of its Fe-complex.  The iron binding affinity of amychelin was detd. using EDTA competition assays, and a biosynthetic cluster was identified and annotated to provide a tentative biosynthetic scheme for amychelin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPHzELqJtDVbVg90H21EOLACvtfcHk0ljvTiZhJQssWQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXosFykt78%253D&amp;md5=ad591145f788ab92316e44443d548898</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fja203577e&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja203577e%26sid%3Dliteratum%253Aachs%26aulast%3DSeyedsayamdost%26aufirst%3DM.%2BR.%26aulast%3DTraxler%26aufirst%3DM.%2BF.%26aulast%3DZheng%26aufirst%3DS.%2BL.%26aulast%3DKolter%26aufirst%3DR.%26aulast%3DClardy%26aufirst%3DJ.%26atitle%3DStructure%2520and%2520Biosynthesis%2520of%2520Amychelin%252C%2520an%2520Unusual%2520Mixed-Ligand%2520Siderophore%2520from%2520Amycolatopsis%2520Sp.%2520AA4%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D11434%26epage%3D11437%26doi%3D10.1021%2Fja203577e" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukuda, T. T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helfrich, E. J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mevers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melo, W. G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Arnam, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andes, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currie, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pupo, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clardy, J.</span></span> <span> </span><span class="NLM_article-title">Specialized Metabolites Reveal Evolutionary History and Geographic Dispersion of a Multilateral Symbiosis</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">292</span>â <span class="NLM_lpage">299</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscentsci.0c00978</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.0c00978" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhsVKjsrc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2021&amp;pages=292-299&amp;author=T.+T.+H.+Fukudaauthor=E.+J.+N.+Helfrichauthor=E.+Meversauthor=W.+G.+P.+Meloauthor=E.+B.+van+Arnamauthor=D.+R.+Andesauthor=C.+R.+Currieauthor=M.+T.+Pupoauthor=J.+Clardy&amp;title=Specialized+Metabolites+Reveal+Evolutionary+History+and+Geographic+Dispersion+of+a+Multilateral+Symbiosis&amp;doi=10.1021%2Facscentsci.0c00978" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Specialized Metabolites Reveal Evolutionary History and Geographic Dispersion of a Multilateral Symbiosis</span></div><div class="casAuthors">Fukuda, Taise T. H.; Helfrich, Eric J. N.; Mevers, Emily; Melo, Weilan G. P.; Van Arnam, Ethan B.; Andes, David R.; Currie, Cameron R.; Pupo, Monica T.; Clardy, Jon</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">292-299</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fungus-growing ants engage in a multilateral symbiosis: they cultivate a fungal garden as their primary food source and host symbiotic actinobacteria (Pseudonocardia spp.) that provide chem. defenses.  The bacterial symbionts produce small specialized metabolites that protect the fungal garden from specific fungal pathogens (Escovopsis spp.), and in return, they are fed by the ant hosts.  Multiple studies on the mols. underlying this symbiotic system have led to the discovery of a large no. of structurally diverse antifungal mols., but somewhat surprisingly no shared structural theme emerged from these studies.  A large systematic study of Brazilian nests led to the discovery of the widespread prodn. of a potent but overlooked antifungal agent, which we named attinimicin, by nearly two-thirds of all Pseudonocardia strains from multiple sites in Brazil.  Here we report the structure of attinimicin, its putative biosynthetic gene cluster, and the evolutionary relationship between attinimicin and two related peptides, oxachelin A and cahuitamycin A.  All three nonribosomal peptides are structural isomers with different primary peptide sequences.  Attinimicin shows iron-dependent antifungal activity against specific environmental fungal parasites but no activity against the fungal cultivar.  Attinimicin showed potent in vivo activity in a mouse Candida albicans infection model comparable to clin. used azole-contg. antifungals.  In situ detection of attinimicin in both ant nests and on worker ants supports an ecol. role for attinimicin in protecting the fungal cultivar from pathogens.  The geog. spread of the attinimicin biosynthetic gene cluster in Brazilian Pseudonocardia spp. marks attinimicin as the first specialized metabolite from ant-assocd. bacteria with broad geog. distribution.  Attinimicin, the product of an evolutionarily conserved combinatorial three-module nonribosomal peptide biosynthetic pathway, provides iron-dependent chem. defenses to fungus-growing ants over a broad geog. swath.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhdoGNFcL-xrVg90H21EOLACvtfcHk0ljvTiZhJQssWQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXhsVKjsrc%253D&amp;md5=1b46fe0b72ed5157ea5de2ca957226c6</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.0c00978&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.0c00978%26sid%3Dliteratum%253Aachs%26aulast%3DFukuda%26aufirst%3DT.%2BT.%2BH.%26aulast%3DHelfrich%26aufirst%3DE.%2BJ.%2BN.%26aulast%3DMevers%26aufirst%3DE.%26aulast%3DMelo%26aufirst%3DW.%2BG.%2BP.%26aulast%3Dvan%2BArnam%26aufirst%3DE.%2BB.%26aulast%3DAndes%26aufirst%3DD.%2BR.%26aulast%3DCurrie%26aufirst%3DC.%2BR.%26aulast%3DPupo%26aufirst%3DM.%2BT.%26aulast%3DClardy%26aufirst%3DJ.%26atitle%3DSpecialized%2520Metabolites%2520Reveal%2520Evolutionary%2520History%2520and%2520Geographic%2520Dispersion%2520of%2520a%2520Multilateral%2520Symbiosis%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2021%26volume%3D7%26spage%3D292%26epage%3D299%26doi%3D10.1021%2Facscentsci.0c00978" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potterat, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamburger, M.</span></span> <span> </span><span class="NLM_article-title">Concepts and Technologies for Tracking Bioactive Compounds in Natural Product Extracts: Generation of Libraries, and Hyphenation of Analytical Processes with Bioassays</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">546</span>â <span class="NLM_lpage">564</span>, <span class="refDoi">&nbsp;DOI: 10.1039/c3np20094a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1039%2Fc3np20094a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=23459784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjvFWnsbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=30&amp;publication_year=2013&amp;pages=546-564&amp;author=O.+Potteratauthor=M.+Hamburger&amp;title=Concepts+and+Technologies+for+Tracking+Bioactive+Compounds+in+Natural+Product+Extracts%3A+Generation+of+Libraries%2C+and+Hyphenation+of+Analytical+Processes+with+Bioassays&amp;doi=10.1039%2Fc3np20094a" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Concepts and technologies for tracking bioactive compounds in natural product extracts: generation of libraries, and hyphenation of analytical processes with bioassays</span></div><div class="casAuthors">Potterat, Olivier; Hamburger, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">546-564</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Covering: up to 2012Since the advent of high-throughput screening (HTS) in the early 1990s, a wealth of innovative technologies have been proposed and implemented for the effective localization and characterization of bioactive constituents in complex matrixes.  The latest developments in this field are reviewed under the perspective of their applicability to natural product-based drug discovery.  The approaches discussed here include TLC-based bioautog., HPLC-based assays with online, at-line and off-line detection, as well as affinity-based methods, such as frontal affinity chromatog., pulsed ultrafiltration mass spectrometry, imprinted polymers, and affinity capillary electrophoresis.  Selected practical examples are given to illustrate the strengths and limitations of these approaches in contemporary natural product lead discovery.  In addn., compatibility issues of natural product exts. and HTS are addressed, and selected protocols for the generation of high quality libraries are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRygOdG_tP17Vg90H21EOLACvtfcHk0lirsTU8Y4y6iA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXjvFWnsbc%253D&amp;md5=5043cccfd966157eec78e395abb129d6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1039%2Fc3np20094a&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3np20094a%26sid%3Dliteratum%253Aachs%26aulast%3DPotterat%26aufirst%3DO.%26aulast%3DHamburger%26aufirst%3DM.%26atitle%3DConcepts%2520and%2520Technologies%2520for%2520Tracking%2520Bioactive%2520Compounds%2520in%2520Natural%2520Product%2520Extracts%253A%2520Generation%2520of%2520Libraries%252C%2520and%2520Hyphenation%2520of%2520Analytical%2520Processes%2520with%2520Bioassays%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2013%26volume%3D30%26spage%3D546%26epage%3D564%26doi%3D10.1039%2Fc3np20094a" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atanasov, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zotchev, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dirsch, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orhan, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rollinger, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barreca, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weckwerth, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayer, E. A.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Natural Products in Drug Discovery: Advances and Opportunities</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">200</span>â <span class="NLM_lpage">216</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41573-020-00114-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1038%2Fs41573-020-00114-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=33510482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BB3MXisV2ht74%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2021&amp;pages=200-216&amp;author=A.+G.+Atanasovauthor=S.+B.+Zotchevauthor=V.+M.+Dirschauthor=I.+E.+Orhanauthor=M.+Banachauthor=J.+M.+Rollingerauthor=D.+Barrecaauthor=W.+Weckwerthauthor=R.+Bauerauthor=E.+A.+Bayer&amp;title=Natural+Products+in+Drug+Discovery%3A+Advances+and+Opportunities&amp;doi=10.1038%2Fs41573-020-00114-z" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Natural products in drug discovery: advances and opportunities</span></div><div class="casAuthors">Atanasov, Atanas G.; Zotchev, Sergey B.; Dirsch, Verena M.; the International Natural Product Sciences Taskforce; Supuran, Claudiu T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">200-216</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Natural products and their structural analogs have historically made a major contribution to pharmacotherapy, esp. for cancer and infectious diseases.  Nevertheless, natural products also present challenges for drug discovery, such as tech. barriers to screening, isolation, characterization and optimization, which contributed to a decline in their pursuit by the pharmaceutical industry from the 1990s onwards.  In recent years, several technol. and scientific developments - including improved anal. tools, genome mining and engineering strategies, and microbial culturing advances - are addressing such challenges and opening up new opportunities.  Consequently, interest in natural products as drug leads is being revitalized, particularly for tackling antimicrobial resistance.  Here, we summarize recent technol. developments that are enabling natural product-based drug discovery, highlight selected applications and discuss key opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXOyA9Np8_q7Vg90H21EOLACvtfcHk0lirsTU8Y4y6iA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXisV2ht74%253D&amp;md5=8c026ca4ea44eca1b0517ecbf687caac</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41573-020-00114-z&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-020-00114-z%26sid%3Dliteratum%253Aachs%26aulast%3DAtanasov%26aufirst%3DA.%2BG.%26aulast%3DZotchev%26aufirst%3DS.%2BB.%26aulast%3DDirsch%26aufirst%3DV.%2BM.%26aulast%3DOrhan%26aufirst%3DI.%2BE.%26aulast%3DBanach%26aufirst%3DM.%26aulast%3DRollinger%26aufirst%3DJ.%2BM.%26aulast%3DBarreca%26aufirst%3DD.%26aulast%3DWeckwerth%26aufirst%3DW.%26aulast%3DBauer%26aufirst%3DR.%26aulast%3DBayer%26aufirst%3DE.%2BA.%26atitle%3DNatural%2520Products%2520in%2520Drug%2520Discovery%253A%2520Advances%2520and%2520Opportunities%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2021%26volume%3D20%26spage%3D200%26epage%3D216%26doi%3D10.1038%2Fs41573-020-00114-z" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. D.</span></span> <span> </span><span class="NLM_article-title">Opportunities for Natural Products in 21st Century Antibiotic Discovery</span>. <i>Nat. Prod. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">694</span>â <span class="NLM_lpage">701</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C7NP00019G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1039%2FC7NP00019G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=28569300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC2sXptFCmsLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=2017&amp;pages=694-701&amp;author=G.+D.+Wright&amp;title=Opportunities+for+Natural+Products+in+21st+Century+Antibiotic+Discovery&amp;doi=10.1039%2FC7NP00019G" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities for natural products in 21st century antibiotic discovery</span></div><div class="casAuthors">Wright, Gerard D.</div><div class="citationInfo"><span class="NLM_cas:title">Natural Product Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">694-701</span>CODEN:
                <span class="NLM_cas:coden">NPRRDF</span>;
        ISSN:<span class="NLM_cas:issn">0265-0568</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Natural products and their derivs. are mainstays of our antibiotic drugs, but they are increasingly in peril.  The combination of widespread multidrug resistance in once susceptible bacterial pathogens, disenchantment with natural products as sources of new drugs, lack of success using synthetic compds. and target-based discovery methods, along with shifting economic and regulatory issues, conspire to move investment in research and development away from the antibiotics arena.  The result is a growing crisis in antibiotic drug discovery that threatens modern medicine.  21St century natural product research is perfectly positioned to fill the antibiotic discovery gap and bring new drug candidates to the clinic.  Innovations in genomics and techniques to explore new sources of antimicrobial chem. matter are revealing new chem.  Increasing appreciation of the value of narrow-spectrum drugs and re-examn. of once discarded chem. scaffolds coupled with synthetic biol. methods to generate new compds. and improve yields offer new strategies to revitalize once moribund natural product programs.  The increasing awareness that the combination of antibiotics with adjuvants, non-antibiotic compds. that overcome resistance and enhance drug activity, can rescue older chem. scaffolds, and concepts such as blocking pathogen virulence present orthogonal strategies to traditional antibiotics.  In all these areas, natural products offer chem. matter, shaped by natural selection, that is privileged in this therapeutic area.  Natural product research is poised to regain prominence in delivering new drugs to solve the antibiotic crisis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHEYEudGcHELVg90H21EOLACvtfcHk0lirsTU8Y4y6iA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXptFCmsLg%253D&amp;md5=a0df60088a52f943922e460e9d0e38be</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC7NP00019G&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7NP00019G%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DG.%2BD.%26atitle%3DOpportunities%2520for%2520Natural%2520Products%2520in%252021st%2520Century%2520Antibiotic%2520Discovery%26jtitle%3DNat.%2520Prod.%2520Rep.%26date%3D2017%26volume%3D34%26spage%3D694%26epage%3D701%26doi%3D10.1039%2FC7NP00019G" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prieto-MartÃ­nez, F. D.</span>; <span class="NLM_string-name">Norinder, U.</span>; <span class="NLM_string-name">Medina-Franco, J. L.</span></span> <span> </span><span class="NLM_article-title">Cheminformatics Explorations of Natural Products</span>. In  <i>Progress in the Chemistry of Organic Natural Products 110: Cheminformatics in Natural Product Research</i>, <span class="NLM_contrib-group"><span class="NLM_string-name">Kinghorn, A. D.</span>; <span class="NLM_string-name">Falk, H.</span>; <span class="NLM_string-name">Gibbons, S.</span>; <span class="NLM_string-name">Kobayashi, J.</span>; <span class="NLM_string-name">Asakawa, Y.</span>; <span class="NLM_string-name">Liu, J.-K.</span></span>, Eds.; <span class="NLM_publisher-name">Springer International Publishing</span>: <span class="NLM_publisher-loc">Cham</span>, <span class="NLM_year">2019</span>; pp  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">35</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1007%2F978-3-030-14632-0_1" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2019&amp;pages=1-35&amp;author=F.+D.+Prieto-Mart%C3%ADnez&amp;author=U.+Norinder&amp;author=J.+L.+Medina-Francoauthor=A.+D.+Kinghorn&amp;author=H.+Falk&amp;author=S.+Gibbons&amp;author=J.+Kobayashi&amp;author=Y.+Asakawa&amp;author=J.-K.+Liu&amp;title=Progress+in+the+Chemistry+of+Organic+Natural+Products+110%3A+Cheminformatics+in+Natural+Product+Research" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2F978-3-030-14632-0_1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-030-14632-0_1%26sid%3Dliteratum%253Aachs%26aulast%3DPrieto-Mart%25C3%25ADnez%26aufirst%3DF.%2BD.%26atitle%3DCheminformatics%2520Explorations%2520of%2520Natural%2520Products%26btitle%3DProgress%2520in%2520the%2520Chemistry%2520of%2520Organic%2520Natural%2520Products%2520110%253A%2520Cheminformatics%2520in%2520Natural%2520Product%2520Research%26aulast%3DKinghorn%26aufirst%3DA.%2BD.%26pub%3DSpringer%2520International%2520Publishing%26date%3D2019%26spage%3D1%26epage%3D35" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houriet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidar, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellogg, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todd, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cech, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span> <span> </span><span class="NLM_article-title">Interlaboratory Comparison of Untargeted Mass Spectrometry Data Uncovers Underlying Causes for Variability</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">824</span>â <span class="NLM_lpage">835</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.jnatprod.0c01376</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.0c01376" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BB3MXls1Crs7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=84&amp;publication_year=2021&amp;pages=824-835&amp;author=T.+N.+Clarkauthor=J.+Hourietauthor=W.+S.+Vidarauthor=J.+J.+Kelloggauthor=D.+A.+Toddauthor=N.+B.+Cechauthor=R.+G.+Linington&amp;title=Interlaboratory+Comparison+of+Untargeted+Mass+Spectrometry+Data+Uncovers+Underlying+Causes+for+Variability&amp;doi=10.1021%2Facs.jnatprod.0c01376" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Interlaboratory Comparison of Untargeted Mass Spectrometry Data Uncovers Underlying Causes for Variability</span></div><div class="casAuthors">Clark, Trevor N.; Houriet, Joelle; Vidar, Warren S.; Kellogg, Joshua J.; Todd, Daniel A.; Cech, Nadja B.; Linington, Roger G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">824-835</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">Despite the value of mass spectrometry in modern natural products discovery workflows, it remains very difficult to compare data sets between labs.  In this study we compared mass spectrometry data for the same sample set from two different labs. (quadrupole time-of-flight and quadrupole-Orbitrap) and evaluated the similarity between these two data sets in terms of both mass spectrometry features and their ability to describe the chem. compn. of the sample set.  Somewhat surprisingly, the two data sets, collected with appropriate controls and replication, had very low feature overlap (25.7% of Lab. A features overlapping 21.8% of Lab. B features).  Our data clearly demonstrate that differences in fragmentation, charge state, and adduct formation in the ionization source are a major underlying cause for these differences.  Consistent with other recent literature, these findings challenge the conventional wisdom that electrospray ionization mass spectrometry (ESI-MS) yields a simple one-to-one correspondence between analytes in soln. and features in the data set.  Importantly, despite low overlap in feature lists, principal component anal. (PCA) generated qual. similar PCA plots.  Overall, our findings demonstrate that comparing untargeted metabolomics data between labs. is challenging, but that data sets with low feature overlap can yield the same qual. description of a sample set using PCA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm7C-Gmm7P5bVg90H21EOLACvtfcHk0lgUAf6sM-iI-w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXls1Crs7w%253D&amp;md5=cd93a585f05b819287b4f052b6baa21f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.0c01376&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.0c01376%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DT.%2BN.%26aulast%3DHouriet%26aufirst%3DJ.%26aulast%3DVidar%26aufirst%3DW.%2BS.%26aulast%3DKellogg%26aufirst%3DJ.%2BJ.%26aulast%3DTodd%26aufirst%3DD.%2BA.%26aulast%3DCech%26aufirst%3DN.%2BB.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26atitle%3DInterlaboratory%2520Comparison%2520of%2520Untargeted%2520Mass%2520Spectrometry%2520Data%2520Uncovers%2520Underlying%2520Causes%2520for%2520Variability%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2021%26volume%3D84%26spage%3D824%26epage%3D835%26doi%3D10.1021%2Facs.jnatprod.0c01376" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hohrenk, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itzel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baetz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuerk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vosough, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, T. C.</span></span> <span> </span><span class="NLM_article-title">Comparison of Software Tools for Liquid Chromatography-High-Resolution Mass Spectrometry Data Processing in Nontarget Screening of Environmental Samples</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1898</span>â <span class="NLM_lpage">1907</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.analchem.9b04095</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.9b04095" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitl2htL%252FP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=92&amp;publication_year=2020&amp;pages=1898-1907&amp;author=L.+L.+Hohrenkauthor=F.+Itzelauthor=N.+Baetzauthor=J.+Tuerkauthor=M.+Vosoughauthor=T.+C.+Schmidt&amp;title=Comparison+of+Software+Tools+for+Liquid+Chromatography-High-Resolution+Mass+Spectrometry+Data+Processing+in+Nontarget+Screening+of+Environmental+Samples&amp;doi=10.1021%2Facs.analchem.9b04095" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Software Tools for Liquid Chromatography-High-Resolution Mass Spectrometry Data Processing in Nontarget Screening of Environmental Samples</span></div><div class="casAuthors">Hohrenk, Lotta L.; Itzel, Fabian; Baetz, Nicolai; Tuerk, Jochen; Vosough, Maryam; Schmidt, Torsten C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1898-1907</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The field of high-resoln. mass spectrometry has undergone rapid progress in the last years due to instrumental improvements leading to a higher sensitivity and selectivity of instruments.  A variety of qual. screening approaches, summarized as nontarget screening, have been introduced and have successfully extended the environmental monitoring of org. micropollutants.  Several automated data processing workflows have been developed to handle the immense amt. of data that are recorded in short time frames by these methods.  Most data processing workflows include similar steps, but underlying algorithms and implementation of different processing steps vary.  In this study the consistency of data processing with different software tools was investigated.  For this purpose, the same raw data files were processed with the software packages MZmine2, enviMass, Compd. Discoverer, and XCMS online and resulting feature lists were compared.  Results show a low coherence between different processing tools, as overlap of features between all four programs was around 10%, and for each software between 40% and 55% of features did not match with any other program.  The implementation of replicate and blank filter was identified as one of the sources of obsd. divergences.  However, there is a need for a better understanding and user instructions on the influence of different algorithms and settings on feature extn. and following filtering steps.  In future studies it would be of interest to investigate how final data interpretation is influenced by different processing software.  With this work we want to encourage more awareness on data processing as a crucial step in the workflow of nontarget screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY6ad0_f0cZLVg90H21EOLACvtfcHk0lgUAf6sM-iI-w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXitl2htL%252FP&amp;md5=cbb2700b3716c265e660d73d250ec787</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.9b04095&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.9b04095%26sid%3Dliteratum%253Aachs%26aulast%3DHohrenk%26aufirst%3DL.%2BL.%26aulast%3DItzel%26aufirst%3DF.%26aulast%3DBaetz%26aufirst%3DN.%26aulast%3DTuerk%26aufirst%3DJ.%26aulast%3DVosough%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DT.%2BC.%26atitle%3DComparison%2520of%2520Software%2520Tools%2520for%2520Liquid%2520Chromatography-High-Resolution%2520Mass%2520Spectrometry%2520Data%2520Processing%2520in%2520Nontarget%2520Screening%2520of%2520Environmental%2520Samples%26jtitle%3DAnal.%2520Chem.%26date%3D2020%26volume%3D92%26spage%3D1898%26epage%3D1907%26doi%3D10.1021%2Facs.analchem.9b04095" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahieu, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patti, G. J.</span></span> <span> </span><span class="NLM_article-title">Systems-Level Annotation of a Metabolomics Data Set Reduces 25â¯000 Features to Fewer than 1000 Unique Metabolites</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">10397</span>â <span class="NLM_lpage">10406</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.analchem.7b02380</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.7b02380" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsV2iurnL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=89&amp;publication_year=2017&amp;pages=10397-10406&amp;author=N.+G.+Mahieuauthor=G.+J.+Patti&amp;title=Systems-Level+Annotation+of+a+Metabolomics+Data+Set+Reduces+25%E2%80%AF000+Features+to+Fewer+than+1000+Unique+Metabolites&amp;doi=10.1021%2Facs.analchem.7b02380" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Systems-Level Annotation of a Metabolomics Data Set Reduces 25,000 Features to Fewer than 1000 Unique Metabolites</span></div><div class="casAuthors">Mahieu, Nathaniel G.; Patti, Gary J.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10397-10406</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">When using liq. chromatog./mass spectrometry (LC/MS) to perform untargeted metabolomics, it is now routine to detect tens of thousands of features from biol. samples.  Poor understanding of the data, however, has complicated interpretation and masked the no. of unique metabolites actually being measured in an expt.  Here the authors place an upper bound on the no. of unique metabolites detected in Escherichia coli samples analyzed with one untargeted metabolomic method.  The authors first group multiple features arising from the same analyte, which the authors call "degenerate features", using a context-driven annotation approach.  Surprisingly, this anal. revealed thousands of previously unreported degeneracies that reduced the no. of unique analytes to â¼2961.  The authors then applied an orthogonal approach to remove nonbiol. features from the data using the 13C-based credentialing technol.  This further reduced the no. of unique analytes to less than 1000.  The 90% redn. in data is 5-fold greater than any previously published studies.  On the basis of the results, the authors propose an alternative approach to untargeted metabolomics that relies on thoroughly annotated ref. data sets.  To this end, the authors introduce the creDBle database (http://creDBle.wustl.edu), which contains accurate mass, retention time, and MS/MS fragmentation data as well as annotations of all credentialed features.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpujHD0I2nPpbVg90H21EOLACvtfcHk0lgUAf6sM-iI-w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhsV2iurnL&amp;md5=d83c63dfa6be1da7a6bb35466544f6e7</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.7b02380&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.7b02380%26sid%3Dliteratum%253Aachs%26aulast%3DMahieu%26aufirst%3DN.%2BG.%26aulast%3DPatti%26aufirst%3DG.%2BJ.%26atitle%3DSystems-Level%2520Annotation%2520of%2520a%2520Metabolomics%2520Data%2520Set%2520Reduces%252025%25E2%2580%25AF000%2520Features%2520to%2520Fewer%2520than%25201000%2520Unique%2520Metabolites%26jtitle%3DAnal.%2520Chem.%26date%3D2017%26volume%3D89%26spage%3D10397%26epage%3D10406%26doi%3D10.1021%2Facs.analchem.7b02380" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krug, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zurek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, R.</span></span> <span> </span><span class="NLM_article-title">Efficient Mining of Myxobacterial Metabolite Profiles Enabled by Liquid Chromatography-Electrospray Ionisation-Time-of-Flight Mass Spectrometry and Compound-Based Principal Component Analysis</span>. <i>Anal. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>624</i></span>,  <span class="NLM_fpage">97</span>â <span class="NLM_lpage">106</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.aca.2008.06.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=10.1016%2Fj.aca.2008.06.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=18706314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c01108&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVWksbzI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=624&amp;publication_year=2008&amp;pages=97-106&amp;author=D.+Krugauthor=G.+Zurekauthor=B.+Schneiderauthor=R.+Garciaauthor=R.+M%C3%BCller&amp;title=Efficient+Mining+of+Myxobacterial+Metabolite+Profiles+Enabled+by+Liquid+Chromatography-Electrospray+Ionisation-Time-of-Flight+Mass+Spectrometry+and+Compound-Based+Principal+Component+Analysis&amp;doi=10.1016%2Fj.aca.2008.06.036" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient mining of myxobacterial metabolite profiles enabled by liquid chromatography-electrospray ionization-time-of-flight mass spectrometry and compound-based principal component analysis</span></div><div class="casAuthors">Krug, Daniel; Zurek, Gabriela; Schneider, Birgit; Garcia, Ronald; Mueller, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Analytica Chimica Acta</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">624</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">97-106</span>CODEN:
                <span class="NLM_cas:coden">ACACAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2670</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bacteria producing secondary metabolites are an important source of natural products with highly diverse structures and biol. activities.  Developing methods to efficiently mine procaryotic secondary metabolomes for the presence of potentially novel natural products is therefore of considerable interest.  Modern mass spectrometry-coupled liq. chromatog. can effectively capture microbial metabolic diversity with ever improving sensitivity and accuracy.  In addn., computational and statistical tools increasingly enable the targeted anal. and exploration of information-rich LC-MS datasets.  In this article, the authors describe the use of such techniques for the characterization of myxobacterial secondary metabolomes.  Using accurate mass data from high-resoln. ESI-TOF measurements, target screening has facilitated the rapid identification of known myxobacterial metabolites in exts. from nine Myxococcus species.  Furthermore, principal component anal. (PCA), implementing an advanced compd.-based bucketing approach, readily revealed the presence of further compds. which contribute to variation among the metabolite profiles under investigation.  The generation of mol. formulas for putative novel compds. with high confidence due to evaluation of both exact mass position and isotopic pattern, is exemplified as an important key for de-replication and prioritization of candidates for further characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ0E82nT40mrVg90H21EOLACvtfcHk0lgUAf6sM-iI-w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXhtVWksbzI&amp;md5=6bef2ebc2a2fce50fa495f0a6e8dd007</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.aca.2008.06.036&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.aca.2008.06.036%26sid%3Dliteratum%253Aachs%26aulast%3DKrug%26aufirst%3DD.%26aulast%3DZurek%26aufirst%3DG.%26aulast%3DSchneider%26aufirst%3DB.%26aulast%3DGarcia%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DR.%26atitle%3DEfficient%2520Mining%2520of%2520Myxobacterial%2520Metabolite%2520Profiles%2520Enabled%2520by%2520Liquid%2520Chromatography-Electrospray%2520Ionisation-Time-of-Flight%2520Mass%2520Spectrometry%2520and%2520Compound-Based%2520Principal%2520Component%2520Analysis%26jtitle%3DAnal.%2520Chim.%2520Acta%26date%3D2008%26volume%3D624%26spage%3D97%26epage%3D106%26doi%3D10.1016%2Fj.aca.2008.06.036" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i28">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_18184"></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acscentsci.1c01108?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acscentsci.1c01108</a>.</p><ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Structure elucidation descriptions and NMR data for compounds <b>3</b> and <b>6</b>, chemical characterization data for compounds <b>1</b>â<b>6</b>, experimental methods and results for antibacterial screening, and high-resolution version of the full NP Analyst network presented in Figure S1 (<a class="ext-link" href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf">PDF</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.1c01108/suppl_file/oc1c01108_si_001.pdf">oc1c01108_si_001.pdf (11.12 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms &amp; Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facscentsci.1c01108&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Facscii.2022.8.issue-2%3Bwgroup%3Astring%3AACHS+website+Group%3BrequestedJournal%3Ajournal%3Aacscii%3Bwebsite%3Awebsite%3Aacspubs%3Bjournal%3Ajournal%3Aacscii%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facscentsci.1c01108%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acscentsci.1c01108" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2022<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS &amp; Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support &amp; Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"><input type="hidden" name="appID" value="literatum" class="appID"><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onclick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        <!-- <div id="gdpr-message-container">
			<a id="closebtn" href="#"><i class="icon-times-circle-o"></i></a> -->
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        <!-- </div> -->
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        <!-- AdRoll Pixel -->
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>

<!-- Twitter universal website tag code -->
<!-- Commented out for ACHS-6152
<script>
    !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe.apply(s,arguments):s.queue.push(arguments);
    },s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',
    a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');
    // Insert Twitter Pixel ID and Standard Event data below
    twq('init','o3njp');
    twq('track','PageView');
</script>
-->
<!-- Facebook Pixel Code -->
<!-- Commented out for ACHS-6152
<script>
    !function(f,b,e,v,n,t,s)
    {if(f.fbq)return;n=f.fbq=function(){n.callMethod?
    n.callMethod.apply(n,arguments):n.queue.push(arguments)};
    if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
    n.queue=[];t=b.createElement(e);t.async=!0;
    t.src=v;s=b.getElementsByTagName(e)[0];
    s.parentNode.insertBefore(t,s)}(window, document,'script',
    'https://connect.facebook.net/en_US/fbevents.js');
    fbq('init', '529840281033320');
    fbq('track', 'PageView');
</script>
<noscript><img height="1" width="1" style="display:none" src="https://www.facebook.com/tr?id=529840281033320&amp;ev=PageView&amp;noscript=1"
/></noscript>
-->





        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
var currentJournal = "ACS Central Science";
$(".citationLinks > a").click(function(e){
    if($(this).text() === "ACS Full Text ") {
        ga('send', 'event', 'Reference _Click', 'ACS Full Text', currentJournal);
    }
    else if($(this).text() === "Crossref") {
        ga('send', 'event', 'Reference _Click', 'Crossref', currentJournal);
    }
    else if($(this).text() === "CAS") {
        ga('send', 'event', 'Reference _Click', 'CAS', currentJournal);
    }
    else if($(this).text() === "PubMed") {
        ga('send', 'event', 'Reference _Click', 'PubMed', currentJournal);
    }
    else if($(this).text() === "Google Scholar") {
        ga('send', 'event', 'Reference _Click', 'Google Scholar', currentJournal);
    }
    else if($(this).hasClass("issue-item_open-url") === true) {
        ga('send', 'event', 'Reference _Click', 'OpenURL', currentJournal);
    }
});
</script>






        
        



        
    





        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        <div tabindex="0" class="showMenuAlertMsg-popup"><div class="pairing-affiliations-popup-content"><div class="pairing-affiliations-header"><button type="button" class="close paringAffClose"><i aria-hidden="true" class="icon-close_thin"></i></button></div><div class="responsrMsg"></div></div></div>




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	dartInstitutionType = 'education';
</script>
        </div>
    





        
        



        
    





        
        <script>
// LIT-455776 workaround
// Moving the popup before the header content for tab accessibility
document.addEventListener('DOMContentLoaded', () => {
    const headerNode = $('.header_content');
    const popupNode = $('.pairing-affiliations-popup');

    if (!headerNode.length || !popupNode.length)
        return;

    popupNode.insertBefore(headerNode);  
});
</script> 
        </div>
    </div>





	
    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-d93b9088814c7fcd2f1c.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-9dee82777d6af4ace557.js"></script>
<script type="text/javascript" src="/wro/l6j1~product.js"></script>

















<script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/v652eace1692a40cfa3763df669d7439c1639079717194" integrity="sha512-Gi7xpJR8tSkrpF7aordPZQlW2DLtzUlZcumS8dMQjwDHEnw9I7ZLyiOj/6tZStRBGtGgN6ceN6cMH8z7etPGlw==" data-cf-beacon="{&quot;rayId&quot;:&quot;705c096318e220e5&quot;,&quot;token&quot;:&quot;0562ec7bcba14d1cae3543aa43d6e749&quot;,&quot;version&quot;:&quot;2021.12.0&quot;,&quot;si&quot;:100}" crossorigin="anonymous"></script>


<div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh864" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.47702646771490775&amp;iit=1651612786729&amp;tmr=load%3D1651612786363%26core%3D1651612786442%26main%3D1651612786722%26ifr%3D1651612786734&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=pubs.acs.org&amp;dr=&amp;du=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facscentsci.1c01108&amp;href=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facscentsci.1c01108&amp;dt=NP%20Analyst%3A%20An%20Open%20Online%20Platform%20for%20Compound%20Activity%20Mapping&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Ctype%3DArticle%2Cdescription%2Cimage%2Ctitle%2Curl&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=62719c727c671af5&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=url%3Dhttps%253A%252F%252Fpubs.acs.org%252Fdoi%252Ffull%252F10.1021%252Facscentsci.1c01108%26title%3DNP%2520Analyst%253A%2520An%2520Open%2520Online%2520Platform%2520for%2520Compound%2520Activity%2520Mapping%26image%3Dhttps%253A%252F%252Fpubs.acs.org%252Fcms%252F10.1021%252Facscentsci.1c01108%252Fasset%252Fimages%252Facscentsci.1c01108.social.jpeg_v03%26description%3DFew%2520tools%2520exist%2520in%2520natural%2520products%2520discovery%2520to%2520integrate%2520biological%2520screening%2520and%2520untargeted%2520mass%2520spectrometry%2520data%2520at%2520the%2520library%2520scale.%2520Previously%252C%2520we%2520reported%2520Compound%2520Activity%2520Mapping%2520as%2520a%2520strategy%2520for%2520predicting%2520compound%2520bioactivity%2520profiles%2520directly%2520from%2520primary%2520screening%2520results%2520on%2520extract%2520libraries.%2520We%2520now%2520present%2520NP%2520Analyst%252C%2520an%2520open%2520online%2520platform%2520for%2520Compound%2520Activity%2520Mapping%2520that%2520accepts%2520bioassay%2520data%2520of%2520almost%2520any%2520type%252C%2520and%2520is%2520compatible%2520with%2520mass%2520spectrometry%2520data%2520from%2520major%2520instrument%2520manufacturers%2520via%2520the%2520mzML%2520format.%2520In%2520addition%252C%2520NP%2520Analyst%2520will%2520accept%2520processed%2520mass%2520spectrometry%2520data%2520from%2520the%2520MZmine%25202%2520and%2520GNPS%2520open-source%2520platforms%252C%2520making%2520it%2520a%2520versatile%2520tool%2520for%2520integration%2520with%2520existing%2520discovery%2520workflows.%2520We%2520demonstrate%2520the%2520utility%2520of%2520this%2520new%2520tool%2520for%2520both%2520the%2520dereplication%2520of%2520known%2520compounds%2520and%2520the%2520discovery%2520of%2520novel%2520bioactive%2520natural%2520products%2520using%2520a%2520challenging%2520low-resolution%2520antimicrobial%2520bioassay%2520data%2520set.%2520This%2520new%2520platform%2520is%2520available%2520at%2520www.npanalyst.org.%26type%3DArticle%26site_name%3DACS%2520Publications&amp;csi=undefined&amp;rev=v8.28.8-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style></body></html>