<html class="pb-page" data-request-id="10785866-df4e-4051-baef-92eca1c2aa5d" lang="en"><head data-pb-dropzone="head"><script type="text/javascript" src="https://m.addthis.com/live/red_lojson/300lo.json?si=6269cd9e5e2b76f5&amp;bkl=0&amp;bl=1&amp;pdt=471&amp;sid=6269cd9e5e2b76f5&amp;pub=xa-4faab26f2cff13a7&amp;rev=v8.28.8-wp&amp;ln=en&amp;pc=men&amp;cb=0&amp;ab=-&amp;dp=pubs.acs.org&amp;fp=doi%2F10.1021%2Fjacs.8b10992&amp;fr=&amp;of=2&amp;pd=0&amp;irt=1&amp;vcl=1&amp;md=0&amp;ct=1&amp;tct=0&amp;abt=0&amp;cdn=0&amp;pi=1&amp;rb=0&amp;gen=100&amp;chr=UTF-8&amp;colc=1651101087306&amp;jsl=1&amp;uvs=6269cd2a519da309036&amp;skipb=1&amp;callback=addthis.cbs.jsonp__76327131022054250"></script><script type="text/javascript" src="https://v1.addthisedge.com/live/boost/xa-4faab26f2cff13a7/_ate.track.config_resp"></script><script src="https://widgets.figshare.com/static/figshare.js?_=1651101085997"></script><script type="text/javascript" src="https://z.moatads.com/addthismoatframe568911941483/moatframe.js"></script><meta name="pbContext" content=";page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jacsat.2019.141.issue-4;wgroup:string:ACHS website Group;requestedJournal:journal:jacsat;website:website:acspubs;article:article:10.1021/jacs.8b10992;pageGroup:string:Publication Pages;subPage:string:Full Text;journal:journal:jacsat">




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of the American Chemical Society"><meta name="dc.Title" content="Biophysical Characterization of a Disabled Double Mutant of Soybean Lipoxygenase: The “Undoing” of Precise Substrate Positioning Relative to Metal Cofactor and an Identified Dynamical Network"><meta name="dc.Creator" content="Shenshen  Hu"><meta name="dc.Creator" content="Adam R.  Offenbacher"><meta name="dc.Creator" content="Erin M.  Thompson"><meta name="dc.Creator" content="Christine L.  Gee"><meta name="dc.Creator" content="Jarett  Wilcoxen"><meta name="dc.Creator" content="Cody A. M.  Carr"><meta name="dc.Creator" content="Daniil M.  Prigozhin"><meta name="dc.Creator" content="Vanessa  Yang"><meta name="dc.Creator" content="Tom  Alber"><meta name="dc.Creator" content="R. David  Britt"><meta name="dc.Creator" content="James S.  Fraser"><meta name="dc.Creator" content="Judith P.  Klinman"><meta name="dc.Description" content="Soybean lipoxygenase (SLO) has served as a prototype for understanding the molecular origin of enzymatic rate accelerations. The double mutant (DM) L546A/L754A is considered a dramatic outlier, due..."><meta name="Description" content="Soybean lipoxygenase (SLO) has served as a prototype for understanding the molecular origin of enzymatic rate accelerations. The double mutant (DM) L546A/L754A is considered a dramatic outlier, due..."><meta name="dc.Publisher" content="American Chemical Society"><meta name="dc.Date" scheme="WTN8601" content="January 15, 2019"><meta name="dc.Type" content="research-article"><meta name="dc.Format" content="text/HTML"><meta name="dc.Identifier" scheme="doi" content="10.1021/jacs.8b10992"><meta name="dc.Language" content="en"><meta name="dc.Coverage" content="world"><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society">
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjacs.8b10992">
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjacs.8b10992">
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjacs.8b10992">
        
    
    

<title>Biophysical Characterization of a Disabled Double Mutant of Soybean Lipoxygenase: The “Undoing” of Precise Substrate Positioning Relative to Metal Cofactor and an Identified Dynamical Network | Journal of the American Chemical Society</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive">





    






        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jacs.8b10992"><meta property="og:title" content="Biophysical Characterization of a Disabled Double Mutant of Soybean Lipoxygenase: The “Undoing” of Precise Substrate Positioning Relative to Metal Cofactor and an Identified Dynamical Network"><meta property="og:image" content="https://pubs.acs.org/cms/10.1021/jacs.8b10992/asset/images/jacs.8b10992.social.jpeg_v03"><meta property="og:description" content="Soybean lipoxygenase (SLO) has served as a prototype for understanding the molecular origin of enzymatic rate accelerations. The double mutant (DM) L546A/L754A is considered a dramatic outlier, due to the unprecedented size and near temperature-independence of its primary kinetic isotope effect, low catalytic efficiency, and elevated enthalpy of activation. To uncover the physical basis of these features, we herein apply three structural probes: hydrogen–deuterium exchange mass spectrometry, room-temperature X-ray crystallography and EPR spectroscopy on four SLO variants (wild-type (WT) enzyme, DM, and the two parental single mutants, L546A and L754A). DM is found to incorporate features of each parent, with the perturbation at position 546 predominantly influencing thermally activated motions that connect the active site to a protein–solvent interface, while mutation at position 754 disrupts the ligand field and solvation near the cofactor iron. However, the expanded active site in DM leads to more active site water molecules and their associated hydrogen bond network, and the individual features from L546A and L754A alone cannot explain the aggregate kinetic properties for DM. Using recently published QM/MM-derived ground-state SLO-substrate complexes for WT and DM, together with the thorough structural analyses presented herein, we propose that the impairment of DM is the combined result of a repositioning of the reactive carbon of linoleic acid substrate with regard to both the iron cofactor and a catalytically linked dynamic region of protein."><meta property="og:type" content="Article"><meta property="og:site_name" content="ACS Publications"><meta name="twitter:card" content="summary_large_image"><meta name="twitter:site" content="@ACSPublications"><meta name="twitter:image" content="https://pubs.acs.org/cms/10.1021/jacs.8b10992/asset/images/jacs.8b10992.social.jpeg_v03">
    





    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0">















    

    <meta name="dc.identifier" content="10.1021/jacs.8b10992">



<meta name="format-detection" content="telephone=no">




<link rel="stylesheet" type="text/css" href="/wro/l6j1~product.css">

<link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-c7d2282eca032afdc856.css"><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css">




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1640088552890.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1648583148410.css">
<script type="text/javascript" async="" src="//img.en25.com/i/elqCfg.min.js"></script><script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script><script async="" src="//www.google-analytics.com/analytics.js"></script><script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer="" src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon="{&quot;rayId&quot;:&quot;5200035e6e4d5180&quot;,&quot;startTime&quot;:1570117212010}"></script>

            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    , 'auto'
                    );
                
                ga('send', 'pageview');
            </script>
            











    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jacs.8b10992">
            
        







    <!-- loading Javascript for using the persistence Service -->
    <script src="/templates/jsp/pb2/js/plugin/thiss-ds.js"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            alt_context: window.location.origin,
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if (ms == undefined || ms === '') {
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            if (results.length == 0) {
                                persistService.entities(self.alt_context).then((res) => {
                                    return resolve(res.data);
                                }, function (err) {
                                    // failed
                                    reject('failed to retrieve entity from local storage (publisher domain): ' + err);
                                });
                            } else {
                                return resolve(results);
                            }
                        }, function (err) {
                            // failed
                            reject('failed to retrieve entity from local storage: ' + err);
                        });
                    // Set up the timeout
                    setTimeout(function () {
                        reject('Promise timed out ' + ms + ' ms');
                    }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                    persistService.update(self.alt_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage (publisher domain) duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            var results = res.data;
                            if (!results) {
                                persistService.remove(self.alt_context, entityId).then((res) => {
                                    return resolve(res);
                                }, function (err) {
                                    // failed
                                    console.log('failed to remove from local storage (publisher domain) duo to: ' + err);
                                });
                            }
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId, entityTitle) {
                return new Promise(function (resolve, reject) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {
                            doi: doi,
                            entityid: entityId,
                            entityTitle: entityTitle,
                            eventGroupKey: '45371114-e4bb-458c-8f9f-8faf4edd7a4f'
                        }

                    }).done(function (status) {
                        return resolve(status);
                    }).fail(function (status) {
                        return reject(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID, ssoInstitution.entity.title)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + ssoInstitution.entity.entityID,err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            }).catch(function (err) {
                                reject(err);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();

                    if ($('.institution-preference-userconsent-checkbox').length > 0) {
                        let institutionPreferenceUserConsentCheckbox = $('.institution-preference-userconsent-checkbox').get(0);
                        if (!$(institutionPreferenceUserConsentCheckbox).is(":checked")) {
                            return;
                        }
                    }

                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err) {
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>







    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY">


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html">

<link rel="shortcut icon" href="/pb-assets/ux3/favicon-1635812676023.ico"><link type="text/css" rel="stylesheet" href="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/CcdcWidget.min.css"><style></style><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><meta http-equiv="origin-trial" content="AxujKG9INjsZ8/gUq8+dTruNvk7RjZQ1oFhhgQbcTJKDnZfbzSTE81wvC2Hzaf3TW4avA76LTZEMdiedF1vIbA4AAABueyJvcmlnaW4iOiJodHRwczovL2ltYXNkay5nb29nbGVhcGlzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzVGhpcmRQYXJ0eSI6dHJ1ZX0="><meta http-equiv="origin-trial" content="Azuce85ORtSnWe1MZDTv68qpaW3iHyfL9YbLRy0cwcCZwVnePnOmkUJlG8HGikmOwhZU22dElCcfrfX2HhrBPAkAAAB7eyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A16nvcdeoOAqrJcmjLRpl1I6f3McDD8EfofAYTt/P/H4/AWwB99nxiPp6kA0fXoiZav908Z8etuL16laFPUdfQsAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="AxBHdr0J44vFBQtZUqX9sjiqf5yWZ/OcHRcRMN3H9TH+t90V/j3ENW6C8+igBZFXMJ7G3Pr8Dd13632aLng42wgAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A88BWHFjcawUfKU3lIejLoryXoyjooBXLgWmGh+hNcqMK44cugvsI5YZbNarYvi3roc1fYbHA1AVbhAtuHZflgEAAAB2eyJvcmlnaW4iOiJodHRwczovL2dvb2dsZS5jb206NDQzIiwiZmVhdHVyZSI6IlRydXN0VG9rZW5zIiwiZXhwaXJ5IjoxNjUyNzc0NDAwLCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="AzoawhTRDevLR66Y6MROu167EDncFPBvcKOaQispTo9ouEt5LvcBjnRFqiAByRT+2cDHG1Yj4dXwpLeIhc98/gIAAACFeyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A6+nc62kbJgC46ypOwRsNW6RkDn2x7tgRh0wp7jb3DtFF7oEhu1hhm4rdZHZ6zXvnKZLlYcBlQUImC4d3kKihAcAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A/9La288e7MDEU2ifusFnMg1C2Ij6uoa/Z/ylwJIXSsWfK37oESIPbxbt4IU86OGqDEPnNVruUiMjfKo65H/CQwAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_2022042101.js?cb=31067286" async=""></script><script async="true" type="text/javascript" src="https://s.adroll.com/j/roundtrip.js"></script></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'JACS';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/JACS/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/JACS/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
					console.log("ad display: " + event.creativeId + " " + event.size[0] + "x" + event.size[1]);
				}
				else {
				    console.log("no ad found");
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>

<style>
.articleHeaderDropzone5 .advertisement { display: none; }
.article_header .advertisement {
  margin: -25px auto 25px auto;
}
</style>




        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&amp;EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Indiana University Libraries - Bloomington</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"></a>




        
        <div class="pj-badge"></div><a href="/journal/jacsat" title="Journal of the American Chemical Society" class="header_article-logo"><img src="/cms/10.1021/jacsat/asset/title.png" alt="Journal of the American Chemical Society"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="/doi/10.1021/jacs.8b10336" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span><span class="hlFld-Title">Biophysical Characterization of a Disabled Double Mutant of Soybean Lipoxygenase: The “Undoing” of Precise Substrate Positioning Relative to Metal Cofactor and an Identified Dynamical Network</span></span>
        </div>
         <a href="/doi/10.1021/jacs.8b10932" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete ui-autocomplete-input"><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jacsat" name="SeriesKey" checked="checked" class="all-content"><span>J. Am. Chem. Soc.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"><span>All Publications/Website</span></label></div><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" style="display: none;"></ul></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jacsat">Journal of the American Chemical Society</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science &amp; Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials &amp; Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio &amp; Med Chem Au</option><option value="abseba">ACS Biomaterials Science &amp; Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health &amp; Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&amp;T Engineering</option><option value="aewcaa">ACS ES&amp;T Water</option><option value="afsthl">ACS Food Science &amp; Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic &amp; Inorganic Au</option><option value="aptsfn">ACS Pharmacology &amp; Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry &amp; Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&amp;EN Global Enterprise</option><option value="cenear">Chemical &amp; Engineering News Archive</option><option value="chlseg">Chemical Health &amp; Safety</option><option value="chlseg0">Chemical Health &amp; Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth &amp; Design</option><option value="enfuem">Energy &amp; Fuels</option><option value="esthag">Environmental Science &amp; Technology</option><option value="estlcu">Environmental Science &amp; Technology Letters</option><option value="iepra6.2">I&amp;EC Product Research and Development</option><option value="iechad">Industrial &amp; Engineering Chemistry</option><option value="iecac0">Industrial &amp; Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series</option><option value="iecfa7">Industrial &amp; Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial &amp; Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial &amp; Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial &amp; Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical &amp; Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health &amp; Safety</option><option value="jchsc20">Journal of Chemical Health &amp; Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial &amp; Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research &amp; Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&amp;D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autocomplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autocomplete="false" placeholder="Page" required="required" class="quick-search_page-input"><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden="”true”"></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/jacs.8b10990">Solid-State NMR and MD Study of the Structure of the Statherin Mutant SNa15 on Mineral Surfaces</a></li><li><a href="/doi/full/10.1021/jacs.8b10932">Molecular Upconversion in Water in Heteropolynuclear Supramolecular Tb/Yb Assemblies</a></li><li><a href="/doi/full/10.1021/jacs.8b12124">Acid Responsive Hydrogen-Bonded Organic Frameworks</a></li><li><a href="/doi/full/10.1021/jacs.8b11259">Germanium Fluoride Nanocages as Optically Transparent n-Type Materials and Their Endohedral Metallofullerene Derivatives</a></li><li><a href="/doi/full/10.1021/jacs.8b10083">Flexible Polymer-Assisted Mesoscale Self-Assembly of Colloidal CsPbBr3 Perovskite Nanocrystals into Higher Order Superstructures with Strong Inter-Nanocrystal Electronic Coupling</a></li></ul></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="”true”"></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="”true”"></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden="”true”"></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors &amp; Reviewers');">Authors &amp;
                        Reviewers</a></li>
                <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
				<li><a href="https://pubs.acs.org/page/virtual-collections.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Virtual Issues');">Virtual Issues</a></li>
                <li><a href="https://preferences.acs.org/journals" onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS &amp; Mobile');">RSS &amp; Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products &amp; Services');">Products &amp; Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos &amp; Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#" onclick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');" class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/who-we-serve/librarians/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians &amp; Account Managers');">For
                        Librarians &amp; Account Managers</a></li>
            </ul>
        </li>
        <li class="menu-pair-section hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a></li>
                
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage" style="display: block;">Login</a>
            <a class="header_burger-menu_logout-link" href="#" style="display: none;">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS &amp; Open Access');">ACS &amp; Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="/events" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i><div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/jacs.8b10990">Solid-State NMR and MD Study of the Structure of the Statherin Mutant SNa15 on Mineral Surfaces</a></li><li><a href="/doi/full/10.1021/jacs.8b10932">Molecular Upconversion in Water in Heteropolynuclear Supramolecular Tb/Yb Assemblies</a></li><li><a href="/doi/full/10.1021/jacs.8b12124">Acid Responsive Hydrogen-Bonded Organic Frameworks</a></li><li><a href="/doi/full/10.1021/jacs.8b11259">Germanium Fluoride Nanocages as Optically Transparent n-Type Materials and Their Endohedral Metallofullerene Derivatives</a></li><li><a href="/doi/full/10.1021/jacs.8b10083">Flexible Polymer-Assisted Mesoscale Self-Assembly of Colloidal CsPbBr3 Perovskite Nanocrystals into Higher Order Superstructures with Strong Inter-Nanocrystal Electronic Coupling</a></li></ul></div></div>


</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!-- Types--><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><!-- Subjects--><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!--div.pub-modal_right-cover-gallery.hidden-lg.hidden-md//a(href="/coverGallery") Browse Our Cover Art Gallery
--><!-- Publications list--><!-- 'pub-modal_right--inner' adds styling required for IE--><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><!-- The widget data--><input type="hidden" id="ajaxUrl" name="ajaxUrl" value="/pb/widgets/achs-publication-list?widgetId=cabcb15c-6f7a-494b-a19c-eba4fafaee5c&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjacsat.2019.141.issue-4%3Bwgroup%3Astring%3AACHS+website+Group%3BrequestedJournal%3Ajournal%3Ajacsat%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Fjacs.8b10992%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bjournal%3Ajournal%3Ajacsat&amp;key=c7ca46a97ce077568c7371697bcd489c0&amp;ajaxRequest=true"><input type="hidden" name="pubModalData" value=""></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article" style="min-height: 754px;"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjacs.8b10992&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjacs.8b10992&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right" style="display: inline;"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left" style="display: inline;"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jacs.8b10992&amp;href=/doi/10.1021/jacs.8b10992" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Am. Chem. Soc.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 141</span><span class="cit-fg-issue">, 4</span><span class="cit-fg-pageRange">, 1555-1567</span></span></span></div></div></div></div></div><input type="hidden" name="jacsat" id="copyRights" value="/pb-assets/json/journal-info-1641590078917.json"><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jacsat/141/4" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jacs.8b10336" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jacs.8b10932" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jacsat" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/cms/10.1021/jacsat/asset/title.png" alt="Journal Logo"></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="jacsat" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info" href="#"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/jacsat?ifr=1"></div></div></div><h1 class="article_header-title"><span class="hlFld-Title">Biophysical Characterization of a Disabled Double Mutant of Soybean Lipoxygenase: The “Undoing” of Precise Substrate Positioning Relative to Metal Cofactor and an Identified Dynamical Network</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Shenshen Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shenshen Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">California Institute for Quantitative Biosciences,&nbsp; Department of Chemistry &nbsp;and&nbsp; Department of Molecular and Cell Biology, University of California, Berkeley, Berkeley, California 94720, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shenshen++Hu">Shenshen Hu</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Adam R. Offenbacher</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Adam R. Offenbacher</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">California Institute for Quantitative Biosciences,&nbsp; Department of Chemistry &nbsp;and&nbsp; Department of Molecular and Cell Biology, University of California, Berkeley, Berkeley, California 94720, United States</div><div class="loa-info-affiliations-info">Department of Chemistry, East Carolina University, Greenville, North Carolina 27858, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Adam+R.++Offenbacher">Adam R. Offenbacher</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Erin M. Thompson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Erin M. Thompson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Bioengineering and Therapeutic Science, University of California, San Francisco, San Francisco, California 94158, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Erin+M.++Thompson">Erin M. Thompson</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Christine L. Gee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christine L. Gee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">California Institute for Quantitative Biosciences &nbsp;and&nbsp; Department of Molecular and Cell Biology, University of California, Berkeley, Berkeley, California 94720, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christine+L.++Gee">Christine L. Gee</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Jarett Wilcoxen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jarett Wilcoxen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of California, Davis, Davis, California 95695, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jarett++Wilcoxen">Jarett Wilcoxen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0003-2220-0815" title="Orcid link">http://orcid.org/0000-0003-2220-0815</a></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Cody A. M. Carr</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cody A. M. Carr</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">California Institute for Quantitative Biosciences,&nbsp; Department of Chemistry &nbsp;and&nbsp; Department of Molecular and Cell Biology, University of California, Berkeley, Berkeley, California 94720, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cody+A.+M.++Carr">Cody A. M. Carr</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0002-5887-4507" title="Orcid link">http://orcid.org/0000-0002-5887-4507</a></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Daniil M. Prigozhin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniil M. Prigozhin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">California Institute for Quantitative Biosciences &nbsp;and&nbsp; Department of Molecular and Cell Biology, University of California, Berkeley, Berkeley, California 94720, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniil+M.++Prigozhin">Daniil M. Prigozhin</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Vanessa Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vanessa Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of California, Berkeley, Berkeley, California 94720, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vanessa++Yang">Vanessa Yang</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Tom Alber</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tom Alber</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">California Institute for Quantitative Biosciences &nbsp;and&nbsp; Department of Molecular and Cell Biology, University of California, Berkeley, Berkeley, California 94720, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tom++Alber">Tom Alber</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">R. David Britt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">R. David Britt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of California, Davis, Davis, California 95695, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=R.+David++Britt">R. David Britt</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0003-0889-8436" title="Orcid link">http://orcid.org/0000-0003-0889-8436</a></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">James S. Fraser</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James S. Fraser</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Bioengineering and Therapeutic Science, University of California, San Francisco, San Francisco, California 94158, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+S.++Fraser">James S. Fraser</a></span></div></div></span></li><span class="comma-separator">,&nbsp;and&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Judith P. Klinman</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Judith P. Klinman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">California Institute for Quantitative Biosciences,&nbsp; Department of Chemistry &nbsp;and&nbsp; Department of Molecular and Cell Biology, University of California, Berkeley, Berkeley, California 94720, United States</div></div><span class="conrtib-corresp"><strong>*</strong><a href="mailto:klinman@berkeley.edu">klinman@berkeley.edu</a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Judith+P.++Klinman">Judith P. Klinman</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-5734-2843" title="Orcid link">http://orcid.org/0000-0001-5734-2843</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jacsat"><input type="hidden" name="journalNameForjhpLink" value="Journal of the American Chemical Society"><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjacs.8b10992&amp;href=/doi/10.1021%2Fjacs.8b10992" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Am. Chem. Soc.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 141</span><span class="cit-issue">, 4</span><span class="cit-pageRange">, 1555–1567</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 15, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 October 2018</li><li><span class="item_label"><b>Published</b> online</span>15 January 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 30 January 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jacs.8b10992" title="DOI URL">https://doi.org/10.1021/jacs.8b10992</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1555%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DShenshen%2BHu%252C%2BAdam%2BR.%2BOffenbacher%252C%2BErin%2BM.%2BThompson%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D141%26issueNum%3D4%26contentID%3Djacs.8b10992%26title%3DBiophysical%2BCharacterization%2Bof%2Ba%2BDisabled%2BDouble%2BMutant%2Bof%2BSoybean%2BLipoxygenase%253A%2BThe%2B%25E2%2580%259CUndoing%25E2%2580%259D%2Bof%2BPrecise%2BSubstrate%2BPositioning%2BRelative%2Bto%2BMetal%2BCofactor%2Band%2Ban%2BIdentified%2BDynamical%2BNetwork%26numPages%3D13%26pa%3D%26issn%3D0002-7863%26publisherName%3Dacs%26publication%3Djacsat%26rpt%3Dn%26endPage%3D1567%26publicationDate%3DJanuary%2B2019">RIGHTS &amp; PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access"></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jacs.8b10992"><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1491</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">15</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">11</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" class="doi" value="10.1021/jacs.8b10992"><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="add-full-document achs-mendeley_button"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Biophysical Characterization of a Disabled Double Mutant of Soybean Lipoxygenase: The “Undoing” of Precise Substrate Positioning Relative to Metal Cofactor and an Identified Dynamical Network&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Shenshen&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Adam&quot;,&quot;last_name&quot;:&quot;R. Offenbacher&quot;},{&quot;first_name&quot;:&quot;Erin&quot;,&quot;last_name&quot;:&quot;M. Thompson&quot;},{&quot;first_name&quot;:&quot;Christine&quot;,&quot;last_name&quot;:&quot;L. Gee&quot;},{&quot;first_name&quot;:&quot;Jarett&quot;,&quot;last_name&quot;:&quot;Wilcoxen&quot;},{&quot;first_name&quot;:&quot;Cody&quot;,&quot;last_name&quot;:&quot;A. M. Carr&quot;},{&quot;first_name&quot;:&quot;Daniil&quot;,&quot;last_name&quot;:&quot;M. Prigozhin&quot;},{&quot;first_name&quot;:&quot;Vanessa&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Tom&quot;,&quot;last_name&quot;:&quot;Alber&quot;},{&quot;first_name&quot;:&quot;R.&quot;,&quot;last_name&quot;:&quot;David Britt&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;S. Fraser&quot;},{&quot;first_name&quot;:&quot;Judith&quot;,&quot;last_name&quot;:&quot;P. Klinman&quot;}],&quot;source&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;volume&quot;:&quot;141&quot;,&quot;pages&quot;:&quot;1555-1567&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jacs.8b10992&quot;},&quot;abstract&quot;:&quot;Soybean lipoxygenase (SLO) has served as a prototype for understanding the molecular origin of enzymatic rate accelerations. The double mutant (DM) L546A/L754A is considered a dramatic outlier, due to the unprecedented size and near temperature-independence of its primary kinetic isotope effect, low catalytic efficiency, and elevated enthalpy of activation. To uncover the physical basis of these features, we herein apply three structural probes: hydrogen–deuterium exchange mass spectrometry, room-temperature X-ray crystallography and EPR spectroscopy on four SLO variants (wild-type (WT) enzyme, DM, and the two parental single mutants, L546A and L754A). DM is found to incorporate features of each parent, with the perturbation at position 546 predominantly influencing thermally activated motions that connect the active site to a protein–solvent interface, while mutation at position 754 disrupts the ligand field and solvation near the cofactor iron. However, the expanded active site in DM leads to more active si&quot;}" class="meta-data"><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"><input type="hidden" name="pub-access" value="full" class="pub-access"><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"><input type="hidden" name="selected-share-option" class="selected-share-option"></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjacs.8b10992&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jacs.8b10992" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjacs.8b10992&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jacs.8b10992" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjacs.8b10992&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jacs.8b10992" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjacs.8b10992&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jacs.8b10992&amp;href=/doi/10.1021/jacs.8b10992" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jacs.8b10992"><a href="/doi/pdf/10.1021/jacs.8b10992" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jacs.8b10992&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fid%3Ddoi%3A10.1021%252Fjacs.8b10992%26sid%3Dliteratum%253Aachs%26genre%3Darticle%26aulast%3DHu%26date%3D2019%26atitle%3DBiophysical%2BCharacterization%2Bof%2Ba%2BDisabled%2BDouble%2BMutant%2Bof%2BSoybean%2BLipoxygenase%253A%2BThe%2B%25E2%2580%259CUndoing%25E2%2580%259D%2Bof%2BPrecise%2BSubstrate%2BPositioning%2BRelative%2Bto%2BMetal%2BCofactor%2Band%2Ban%2BIdentified%2BDynamical%2BNetwork%26jtitle%3DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26title%3DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26volume%3D141%26issue%3D4%26spage%3D1555%26epage%3D1567%26issn%3D0002-7863" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="jacsat" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info" href="#"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/jacsat?ifr=1"></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollduration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy" style="padding-left: 523.167px;"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291719" title="Redox reactions">Redox reactions</a>,</li><li><a href="/action/doSearch?ConceptID=291707" title="Computer simulations">Computer simulations</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="javascript:void(0)" class="read-more" aria-expanded="false"><i class="icon-angle-down"></i></a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="external-alerts-management" style="margin-left: -59px;"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="jacsat" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info" href="#"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/jacsat?ifr=1"></div><div class="cover-image"><div class="cover-image__image"><a href="/toc/jacsat/141/4" title="Go to Journal of the American Chemical Society "><img src="/cms/10.1021/jacsat.2019.141.issue-4/asset/jacsat.2019.141.issue-4.largecover.jpg" alt="Go to Journal of the American Chemical Society "></a></div></div>




        
        <!--Article Page E-Alerts-->
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jacsat');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jacsat');
	  }
  });
</script>
<!--End Article Script-->




        
        <div class="pj-badge"><input type="hidden" id="partnerJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json"></div>




        
        <div class="aJhp_link"><a href="https://pubs.acs.org/jacs">Journal of the American Chemical Society</a></div>
<input type="hidden" id="aJhpJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json">
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jacsat" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0009.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Soybean lipoxygenase (SLO) has served as a prototype for understanding the molecular origin of enzymatic rate accelerations. The double mutant (DM) L546A/L754A is considered a dramatic outlier, due to the unprecedented size and near temperature-independence of its primary kinetic isotope effect, low catalytic efficiency, and elevated enthalpy of activation. To uncover the physical basis of these features, we herein apply three structural probes: hydrogen–deuterium exchange mass spectrometry, room-temperature X-ray crystallography and EPR spectroscopy on four SLO variants (wild-type (WT) enzyme, DM, and the two parental single mutants, L546A and L754A). DM is found to incorporate features of each parent, with the perturbation at position 546 predominantly influencing thermally activated motions that connect the active site to a protein–solvent interface, while mutation at position 754 disrupts the ligand field and solvation near the cofactor iron. However, the expanded active site in DM leads to more active site water molecules and their associated hydrogen bond network, and the individual features from L546A and L754A alone cannot explain the aggregate kinetic properties for DM. Using recently published QM/MM-derived ground-state SLO-substrate complexes for WT and DM, together with the thorough structural analyses presented herein, we propose that the impairment of DM is the combined result of a repositioning of the reactive carbon of linoleic acid substrate with regard to both the iron cofactor and a catalytically linked dynamic region of protein.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left ui-resizable" style="width: 730px;"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30444" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30444" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i4" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Conclusion"><span>Conclusion</span></a></li><li><a href="#_i16" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i24" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div class="NLM_p">Enzymes catalyze chemical reactions with enormous rate enhancements nearing 10<sup>26</sup>-fold that are accompanied by high regio- and stereospecificities relative to their uncatalyzed counterparts. <a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> The physical bases for such properties are of considerable ongoing interest and investigation. <a onclick="showRef(event, 'ref3 ref4 ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5 ref6 ref7 ref8 ref9">(3−9)</a><i>De novo</i> designed protein catalysts, based on structural information of active sites and the principles of transition state theory, have generally achieved only moderate rate accelerations in the range of 10<sup>3</sup>–10<sup>9</sup>-fold, even after further functional optimization via directed evolution. <a onclick="showRef(event, 'ref10 ref11 ref12 ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13 ref14 ref15">(10−15)</a> The large gap between naturally occurring catalysts and designed systems underscores the need for increased understanding that can be used to inform greater success in designed catalysts. <a onclick="showRef(event, 'ref3 ref4 ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5 ref6 ref7 ref8 ref9">(3−9)</a> Soybean lipoxygenase (SLO) is one of several paradigmatic systems for studying the properties of enzymatic room-temperature hydrogen tunneling and their relevance to general, multidimensional models for the origins of enzymatic rate acceleration. <a onclick="showRef(event, 'ref5 ref9 ref16'); return false;" href="javascript:void(0);" class="ref ref5 ref9 ref16">(5,9,16)</a> SLO catalyzes the stereospecific oxidation of linoleic acid through a rate-limiting abstraction of a pro-<i>S</i> hydrogen atom from the C11 of the substrate linoleic acid (LA); this occurs via a proton coupled electron transfer (PCET) reaction in which a proton is transferred to the active site ferric-bound hydroxide and an electron reduces the ferric center to ferrous (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). <a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> The initially measured and, at the time, unprecedented kinetic features of wild type (WT) enzyme, <a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19−21)</a> included a large (ca. 80) and nearly temperature-independent (Δ<i>E</i><sub>a</sub> = 0.9 kcal/mol) hydrogen/deuterium kinetic isotope effect and a small enthalpy of activation (<i>E</i><sub>a(H)</sub> = 2 kcal/mol) on <i>k</i><sub>cat</sub>. These features have stimulated more than two decades of experimental and theoretical efforts to establish an appropriate room temperature catalytic model. <a onclick="showRef(event, 'ref17 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref17 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29">(17,22−29)</a> The use of site specific mutagenesis has played an important role in building and developing a “deep-tunneling” model. Early studies were primarily focused on six single site mutants that reduce the size of native, bulky hydrophobic side chain, at positions either adjacent to the active carbon C-11 of the substrate (residues L546 and L754), or one helix away (residue I553). <a onclick="showRef(event, 'ref21 ref27'); return false;" href="javascript:void(0);" class="ref ref21 ref27">(21,27)</a> While these mutations produce a range of catalytic impairment on <i>k</i><sub>cat</sub>, five of the six mutants studied retain a primary kinetic isotope effect (KIE, = <i>k</i><sub>cat-H</sub>/<i>k</i><sub>cat-D</sub>) at 30 °C similar to WT (with only a single variant, I553G, showing an increase in the KIE to ca. 180). A compelling feature of this series of mutations is an increase in the temperature-dependence of the KIE, as assessed as Δ<i>E</i><sub>a</sub> = <i>E</i><sub>a(D)</sub> – <i>E</i><sub>a(H)</sub>. (<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Table S1</a>).</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0007.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0007.jpeg" id="gr1" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Reaction Mechanism for the First and Rate Limiting Step of the Stereospecific Oxidation of Linoleic Acid Catalyzed by SLO, Where the Hydrogen Atom Transfer from Substrate LA to Ferric Hydroxide Cofactor Occurs by a PCET Process</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Significantly, the magnitude and temperature-dependence of the experimental KIEs for both WT SLO and each of the single mutants can be well reproduced and interpreted within the framework of vibronically nonadiabatic PCET models. <a onclick="showRef(event, 'ref17 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref17 ref21 ref22 ref23">(17,21−23)</a> Within these deep tunneling models, the observed energy barrier (<i>E</i><sub>a(H)</sub>) of the C–H bond activation originates entirely from the heavy atom motions of the protein and substrate. When conformational sampling is optimal, as occurs in the WT enzyme, a compact and precisely positioned active site leads to a small energy difference for the H- vs D-transfer (i.e., Δ<i>E</i><sub>a</sub> 0). <a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> By contrast, the introduction of packing defects via the reduction in size of native hydrophobic side chains leads to an elongated donor–acceptor distance (DAD) for hydrogen transfer and, concomitantly, a more flexible active site structure that enables the restoration of the DAD to one that resembles native enzyme. Deuterium, with its less diffusive wave function, requires more sampling energy to recover WT-like function, generating a enthalpy of activation for D-transfer that can greatly exceed that for H-transfer (increased Δ<i>E</i><sub>a</sub>). <a onclick="showRef(event, 'ref23 ref27 ref30'); return false;" href="javascript:void(0);" class="ref ref23 ref27 ref30">(23,27,30)</a> This “deep-tunneling” feature of SLO has allowed robust correlations to be made between active site motions and the kinetic parameter Δ<i>E</i><sub>a</sub>.</div><div class="NLM_p">More recent investigations of SLOs have assisted in forming important correlations between protein motions and the energy of activation for catalysis. Differences between experimental KIEs during the biomolecular reaction of enzyme with substrate (<i>k</i><sub>cat</sub>/<i>K</i><sub>m</sub>) and the unimolecular turnover rate (<i>k</i><sub>cat</sub>) led to the proposal of perturbed conformational landscapes in selected mutants. <a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The further application of various biophysical approaches that include studies of increased hydrostatic pressure, <a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> hydrogen–deuterium exchange mass spectrometry (HDXMS), <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and <sup>13</sup>C ENDOR spectroscopy, <a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> has both enhanced our understanding of the structural origin of perturbed conformational landscapes and expanded the “deep-tunneling” model to include both local DAD sampling and more global conformational sampling coordinates. <a onclick="showRef(event, 'ref9 ref35'); return false;" href="javascript:void(0);" class="ref ref9 ref35">(9,35)</a> For the majority of enzymes that have been studied (e.g., refs  <a onclick="showRef(event, 'ref32 ref36 ref37 ref38 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref32 ref36 ref37 ref38 ref39 ref40">(32,36−40)</a>), perturbations produce an elevated energy of activation for C–H cleavage that can be ascribed to an altered global conformational landscape with an increase in inactive (or poorly active) enzyme–substrate (E-S) substates. For native enzyme, one of the primary functions of conformational sampling is the transient achievement of hydrogen donor and acceptor distances that are sufficiently reduced from those of initial van der Waals complexes to enable productive wave function overlap.</div><div class="NLM_p">Studies of a series of double mutants of SLO have further “pushed the envelope” in the ability of theoretical models to rationalize behavior. <a onclick="showRef(event, 'ref25 ref28 ref29 ref31'); return false;" href="javascript:void(0);" class="ref ref25 ref28 ref29 ref31">(25,28,29,31)</a> While the majority of double mutants show behavior similar to single mutants, i.e., analogous KIE values at RT that are accompanied by an increase in Δ<i>E</i><sub>a</sub> (<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Table S1</a>), a single dramatic outlier, L546A/L754A (DM), emerged with a <i>k</i><sub>H</sub>/<i>k</i><sub>D</sub> of 537 ± 55 under presteady single turnover anaerobic measurements <a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and an average <i>k</i><sub>H</sub>/<i>k</i><sub>D</sub> of 661 ± 27 via steady-state measurement across six temperatures. <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Additionally, in contrast to the KIEs of other active site single and double mutants that show temperature sensitivity, the KIEs in DM change little if at all with temperature (<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Table S1</a>). These properties of DM SLO can be well-reproduced using the developed vibronically nonadiabatic PCET model, where donor−acceptor distance sampling within the active site appears to have become rigidified compared to WT and the other SLO variants, <a onclick="showRef(event, 'ref22 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref22 ref28 ref29">(22,28,29)</a> preventing DM from recovering the tunneling-ready distance characteristic of native enzyme and other well characterized mutants. A related temperature–pressure perturbation study of DM SLO also highlights the distinctive features of this variant. <a onclick="showRef(event, 'ref28 ref32'); return false;" href="javascript:void(0);" class="ref ref28 ref32">(28,32)</a> Unlike the pressure-induced shifted conformational landscape in WT and three single mutants (L546A, L754A and I553V), the conformational landscape of the enzyme–substrate complex of DM is more resistant to perturbations from elevated pressure, showing a consistently high energy barrier (<i>E</i><sub>a</sub> = 8–9.9 kcal/mol) and slow catalytic efficiency (10<sup>4</sup>-fold decrease) relative to WT. At this juncture, a major unanswered question is the structural origin underlying the aggregated features of DM SLO.</div><div class="NLM_p last">In this study we compare a suite of biophysical probes applied to DM, the two components of DM (L546A and L754A) and WT. These probes include HDXMS as a function of temperature, EPR spectroscopy and room temperature X-ray crystallography. The primary results show that DM adopts features from both parent single mutants, exhibiting the same HDXMS behavior as L546A and showing structural changes similar to L754A. Significantly, in no instance has evidence been found for enhanced structural rigidification in the resting form of enzyme, in contrast to the earlier inference of rigidification within the donor–acceptor distance sampling modes. As we discuss, the aggregate data indicate a disruption in the positioning of the reactive carbon of substrate with regard to the catalytic iron center that is further incapacitated via an impairment of its communication with a dynamical network that has been identified to extend from the active site to a remote protein/solvent interface. <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41827" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41827" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i4" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Conclusion"><span>Conclusion</span></a></li><li><a href="#_i16" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i24" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Hydrogen–Deuterium Exchange</h3><div class="NLM_p">The time and temperature dependent analysis of HDXMS was carried out with DM for comparison to the previously characterized WT SLO. <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Using the same set of nonoverlapping peptides selected for WT SLO, 78% of the protein sequence for the DM catalytic domain (C-terminal domain) was covered (compared to 89% for WT). The apparent averaged rate constants for exchange, <i>k</i><sub>HDX(avg)</sub>, from exponential fits to the HDXMS traces for WT and DM are presented in <a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Table S2</a> and represent the protein in the absence of substrate and exchanged in buffer pL (L = H, D) of 8, conditions where EX-2 conditions have been demonstrated. EX-2 conditions enable us to extract thermodynamic information regarding regional peptide flexibility. The majority of DM-derived peptides display nearly identical exchange behavior to WT across the 5 temperatures (10, 20, 25, 30, and 40 °C) analyzed (cf. <a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Figure S1</a>). In a previous study, peptide 317–334, a solvent-exposed loop (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A), was shown to produce significant alterations in <i>E</i><sub>a</sub> for <i>k</i><sub>HDX(avg)</sub> following mutation at positions 553 (I553G) and 546 (L546A). <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> This peptide exhibited trends in the <i>E</i><sub>a</sub> for <i>k</i><sub>HDX(avg)</sub>(<i>E</i><sub>aHDX(avg)</sub>) that correlate with trends in the <i>E</i><sub>a</sub> for catalytic PCET (solid line in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>C). <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> In the current HDX analysis of peptide 317–334 in DM, we observe a value for <i>E</i><sub>aHDX(avg)</sub>, again distinct from WT (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B, <a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Table S3</a>); however, while DM SLO displays the highest enthalpic barrier of catalysis among all SLO variants (<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Table S1</a>), the value of <i>E</i><sub>aHDX(avg)</sub> associated with DM is 14 kcal/mol, almost identical to the behavior of L546A (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>C, top red arrow). This initially surprising result, of comparable <i>E</i><sub>aHDX(avg)</sub> values in DM and L546A, implies that the added active site packing defect that arises from L754A in DM does not impact the thermally activated surface motions that correlate with catalysis.</div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0001.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0001.jpeg" id="gr2" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Peptide 317–334 (in orange) remote from the active site exhibits mutation-induced changes in the enthapic barrier for HDX rates (<i>E</i><sub>aHDX(avg)</sub>) in L546A, I553G and DM but not in L754A. (A) The SLO structure with highlighted active site residues L546 (in green), L754 (in blue) and I553 (in red). In panel B, Arrhenius-like plots of the weighted average exchange rate, (ln(<i>k</i><sub>HDX(avg)</sub>) for WT, I553G, L546A, L754A and DM are compared. In panel C, the relationship between <i>E</i><sub>aHDX(avg)</sub> and the enthalpic barrier for hydrogen tunneling (<i>E</i><sub>a</sub>(<i>k</i><sub>cat</sub>)) is represented. The solid line is the trend among I553G, WT and L546A previously reported. <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The bottom arrow indicates that L754A shows no difference in <i>E</i><sub>aHDX(avg)</sub> relative to WT. The top arrow shows that there is no further difference in <i>E</i><sub>aHDX(avg)</sub> when L754A is further introduced into L546A.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further substantiate the lack of an impact of L754 on the long-range protein motional network, the single variant L754A was examined with regard to the time and temperature dependence of its HDXMS. The L754A demonstrates an elevated enthalpic barrier of catalysis (<i>E</i><sub>a(H)</sub> = 4.1 kcal/mol) and an even more impaired catalytic rate constant than L546A (<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Table S1</a>); <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> however, the <i>k</i><sub>HDX(avg)</sub> and <i>E</i><sub>aHDX(avg)</sub> obtained from the HDXMS profiles of all L754A derived peptide across the full temperature range are within experimental error of WT (<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Table S2, S3</a>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>C, bottom red arrow). The absence of a linkage between the <i>E</i><sub>a(H)</sub> and the <i>E</i><sub>aHDX(avg)</sub> in L754A supports an uncoupling between the packing defect at position 754 and motions at the protein–solvent interface, such that the observed rate impairment with this mutant must arise from other features (see below). We note that there were no noticeable changes in the extent of exchange as a function of temperature for L546A, L754A and DM (cf. full data sets in <a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Figure S1</a>), in marked contrast to the previously studied I553X series where the primary impact of mutation was seen as an increase in the extent of exchange in the 541–565 region of protein. <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div><div class="NLM_p last">These new comparisons between HDXMS profiles and catalytic parameters highlight the combinative yet distinct contributions from the residues L546 and L754 in defining the behavior of DM. As earlier noted, L546 is located within the identified network of structural communication between the surface loop (317–334) and active site residues (<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Figure S2</a>), <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> with a packing defect at position 546 influencing motions at the protein–solvent interface that are reflected both in <i>E</i><sub>a(H)</sub> and <i>E</i><sub>aHDX(avg)</sub>. This impact from L546A appears well preserved in DM. In contrast, L754 resides near the iron cofactor and spatially away from the identified network (<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Figure S2</a>), suggesting that changes in the microenvironment around the metal, such as the degree of hydration, ligand field, and/or redox potential may be the major cause of the observed change in properties for L754A. These possibilities are explored below.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Room-Temperature X-ray Crystallography of the SLOs</h3><div class="NLM_p">To systematically reveal structural changes associated with the size-reduction at position 546 and/or position 754, room temperature X-ray crystallography was adopted for WT, L546A, L754A and DM. Such room temperature structures can reveal a multiplicity of protein conformers that are undetectable under cryo-conditions. <a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> Additionally, in the context of visualizing solvent water, difference maps calculated from two room temperature structures are more reliable than cryogenic maps. <a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> High resolution (1.70–1.85 Å) data were obtained for L546A, L754A and DM, revealing practically superimposable protein backbones and largely unaltered side-chain conformations in comparison to the room temperature X-ray structure of WT. <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The most dramatic change in these three mutants is the expanded active site that is associated with changes in solvation. The size reduction of the side chain on position 546 and/or position 754 leads to a gradually enlarged active site. Defining a more specific active site pocket than in the earlier study, <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> the cavity in WT is calculated to be ≤21 Å<sup>3</sup>. Using this new value as a frame of reference, the variants described in this study indicate increases of 170% in L546A, 76% in L754A, and over 270% in DM. The isomorphous difference map was further applied to individual mutants using phases from the room temperature WT model. The difference map between WT and DM indicates an increased electron density in the active site of DM (<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Figure S3A</a>), resembling most closely the difference map between WT and L754A (<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Figure S3B</a>). These density changes imply increased bound (or diffuse) water molecules near the iron cofactor. The other single mutant L546A demonstrates a slightly positive density, which also implicates the presence of extra water molecules (<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Figure S3C</a>). However, the number of water molecules visualized is clearly less than for DM and L754A, and restricted to a position near 546 and H499. It is important to note that changes in solvation have not always been associated with mutation at hydrophobic side chains in SLOs: the I553X series failed to indicate X-ray evidence for the presence of any additional water molecules at low temperature and room temperature, albeit with a significantly enlarged volume near the mutated residue. <a onclick="showRef(event, 'ref26 ref33'); return false;" href="javascript:void(0);" class="ref ref26 ref33">(26,33)</a></div><div class="NLM_p">To further substantiate possible microenvironment changes around the iron cofactor, we turned to a detailed inspection of the impact of mutation on the active-site geometries for WT, L546A, L754A and DM. <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A–D indicates no significant changes in the coordination geometry between the iron and first/second sphere residues in all SLOs studied. The major difference within the active site is the newly entered water molecules in each variant that may interact with other coordinating residues through hydrogen bond networks. In both L546A and L754A, one additional water molecule is seen to occupy the packing defect created near position 546 and 754, respectively. However, the newly entered water molecule in L754A forms hydrogen bonds with two first sphere coordinating residues (iron-bound water molecule and H499) as well as a second sphere coordinating residue Q495, while the extra water in L546A is far from the second sphere coordinating residue Q495, interacting dominantly with H499 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B,C). The water molecules occupying these single site packing defects and their associated hydrogen bond network are retained in DM. Beyond these two water molecules, we further observe two additional water molecules in DM that are adjacent to position 754 and form hydrogen bond network to the first sphere coordinating residues I839 as well as Q697.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0002.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0002.jpeg" id="gr3" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Coordination geometry of the iron center of (A) WT SLO (PDB: <a href="http://doi.org/10.2210/pdb5T5V/pdb">5T5V</a>); (B) L546A (PDB: <a href="http://doi.org/10.2210/pdb5TQN/pdb">5TQN</a>); (C) L754A (PDB: <a href="http://doi.org/10.2210/pdb5TR0/pdb">5TR0</a>); and (D) DM (PDB: <a href="http://doi.org/10.2210/pdb5TQO/pdb">5TQO</a>). Oxygen atoms are colored in red, nitrogens in blue, iron center in orange and water in cyan. Carbons in first sphere ligands are colored in yellow, second-sphere ligands in pink, side chains on position 546 and 754 in gray. Bonds between iron and first-sphere ligands (yellow) are drawn as dashed gray lines, and hydrogen-bonding interactions between newly added water in the active site with the first sphere and second sphere ligands are shown as solid gray lines. There are no significant differences in side chain conformers for SLO variants based on the electron densities analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> EPR Spectra of Fe(II) and Fe(III) SLOs</h3><div class="NLM_p">In order to further explore changes in the microenvironment around the nonheme iron center of SLO that result from L546A, L754A and DM we employed EPR spectroscopy. The WT, L546A, L754A and DM isolated from <i>E. coli</i> all contain EPR-silent Fe(II), as shown in <a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Figure S4</a>, consistent with previous studies on SLOs and other lipoxygenases. <a onclick="showRef(event, 'ref44 ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46 ref47">(44−47)</a> The oxidized (“yellow”) Fe(III) SLOs were prepared through the addition of 2 equiv of substrate, linoleic acid (LA), to the Fe(II) SLOs as described in <a class="ref internalNav" href="#sec5" aria-label="Materials and Methods">Materials and Methods</a> section. The Fe(III) WT and L546A both demonstrate a complex multicomponent EPR signals with a pronounced feature around <i>g</i> = 7.3 and 6.2 and an additional weak feature at <i>g</i> = 4.3, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>. The previous EPR analysis of WT attributes the two groups of peaks to different geometric structures around the high-spin iron with the <i>g</i> = 6.2 signal representing axial asymmetry, <i>g</i> ≈ 4.3 arising from a rhombic species, and <i>g</i> = 7.3 as an intermediate between the above two species. <a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> These features indicate that the ferric form of WT and L546A both contain the above-mentioned three species with axial asymmetry of the zero field splitting as the dominant feature.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0003.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0003.jpeg" id="gr4" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. EPR spectra (X-band, 6.5K) of oxidized WT, L546A, L754A and DM SLO (140 μM) in 0.1 M potassium phosphate pH = 7.0 buffer. The peak intensities of L546A, L754A and DM have been corrected for the iron content of each mutant: WT (0.9), L546A (0.50), L754A (0.70), DM (0.70). A 10× amplified EPR spectrum of Fe(III) DM SLO is presented for comparison. Expanded EPR spectra are shown in <a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Figure S5</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Remarkably, once again the L754A shows distinct patterns from WT and L546A. The EPR spectrum of L754A shows dramatically increased intensity at the <i>g</i> ≈ 4.3 peak and a weak feature at <i>g</i> = 7.3 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>), indicating that the ferric active site in L754A contains a primarily rhombic species. The interconversion between axial to rhombic species zero field splitting in lipoxygenases indicates the alteration in the active site geometry, which is usually seen to arise as the result of either product coordination to the iron center or substitution of the iron ligands that impacts the ligand field strengths. <a onclick="showRef(event, 'ref44 ref47'); return false;" href="javascript:void(0);" class="ref ref44 ref47">(44,47)</a> We propose that the unique EPR signals in L754A are directly related to the changes in microenvironment around the iron center, including the additional water molecule near the position 754 and its interaction with other iron ligands as shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C.</div><div class="NLM_p">Unlike any previously reported high-spin ferric iron, the EPR studies of oxidized DM demonstrate a dramatically decreased EPR signal intensity (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). However, the increase in UV/vis Fe(III) absorption between 330 and 350 nm for DM, representing the formation of Fe(III) following incubation of enzyme with linoleic acid, appears almost identical to WT after correction for their respective iron levels (<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Figure S6</a>); this indicates that the active site in DM SLO does contain an oxidized, Fe(III) species. The presence of Fe(III) in DM SLO is further validated by the strong peak at <i>g</i> = 4.3 after addition of 4-nitrocatechol to free enzyme (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>D, top two EPR spectra). As shown previously in spectroscopic, kinetic and structural analyses, <a onclick="showRef(event, 'ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref48 ref49 ref50">(48−50)</a>4-nitrocatechol can bind to the active site of SLO and form complexes with Fe(III) SLO in the absence of a redox reaction at pH = 7.0. The dramatically increased intensity in the 4-nitrocatechol complex is, thus, attributed to a zero field splitting change caused by displacement of first sphere side chain ligands with catechol oxygens within the DM-4-nitrocatechol complex. <a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> As has been reported for SLO and homologues of lipoxygenase, the Fe(III) spectra can have a range of zero field splitting parameters (<i>D</i> = −0.3 to +1.7 MHz, <i>E</i>/<i>D</i> = 0.02 to 0.33) which is influenced by the microenvironment of the Fe and will greatly affect the intensity and shape of the EPR spectrum. <a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a> We note that the 10× EPR spectra of oxidized DM SLO displays features of both WT/L546A and L754A (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0004.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0004.jpeg" id="gr5" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. EPR spectra after Fe(III)-SLOs are anaerobically incubated with catechol ligands in 0.1 M potassium phosphate pH = 7.0 buffer at 4 °C for 12 h: (A) WT; (B) L546A; (C) L754A; (D) DM. The EPR spectra of oxidized Fe(III) SLO vs Fe(II) SLO are shown at the top and bottom of each panel for comparison.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Estimation of Redox Potential in SLOs</h3><div class="NLM_p">Room temperature X-ray structures, together with the above EPR analyses, indicate significant disruption in both the microenvironment and electronic structures of the iron sites in L754A and DM. Such changes could impact the redox potential of the active site iron, which would further impact both the reaction free energy Δ<i>G</i>° and the corresponding rate constant <i>k</i><sub>cat</sub>. A previously developed method by Nelson demonstrated that formation of complexes between oxidized WT Fe(III) SLO and a series of substituted catechols enables an estimate of the redox potentials of the active site irons. <a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Specifically, conditions were established for the SLO-catechol complexes, in which less reducing catechols do not transfer an electron to the active site Fe(III), while the more reducing catechols reduce the ferric iron to EPR silent Fe(II), establishing a redox ruler for the active site iron of SLO.</div><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A shows a series of EPR spectra of the WT SLO-catechol complexes after an incubation time of 12 h in phosphate buffer (pH = 7.0). Samples of complexes of WT with 4-nitrocatechol or 3,4-dihydroxybenzonitirile both show sharp EPR signals at <i>g</i> ≈ 4.3, which suggest a conversion from the dominant axial component of Fe(III) to a rhombic component of Fe(III) in the complexes. The EPR signals at <i>g</i> ≈ 4.3 are also seen in the complex of ferric WT SLO with 3,4-dihydroxybenzaldehyde albeit with largely decreased intensity, as evidence for a partially reduction of Fe(III) to Fe(II). In the case of the other five SLO-catechol complexes, the EPR spectra all demonstrate very minor peaks in the same region, indicating almost fully reduced Fe(II) in those complexes. The remaining very small peak have been suggested to arise from some portion of denatured protein that releases the free Fe(III). <a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The trends of the reaction between active site iron and catechols at pH = 7.0 are consistent with Nelson’s previous observations, and an assignment of a redox potential for iron in WT at pH = 7.0 between the range of the redox potentials for the test ligands (0.57 to 0.68 V). <a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div><div class="NLM_p">We proceeded to measure the redox potential of iron in L546A, L754A and DM in a similar fashion, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B–D. In the cases of L546A and DM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B,D), the trends in the EPR with 4-nitrocatechol and the more reducing catechol are similar to WT, implicating redox potentials for iron in these two mutants within the 0.57 to 0.68 V. The slight differences in the reducing ability of catechols in the middle range (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A,B,D, from 3,4-dihydroxybenzonitrile to caffeic acid) further imply a trend of redox potential that is DM &gt; WT &gt; L546A. Finally, in the case of L754A, the observation of a reduction of the ferric center by the least reducing reagent in this series, 4-nitrocatechol, indicates a potential that is somewhat above 0.68 V, for a full trend in redox potential of L754A &gt; DM &gt; WT &gt; L546A.</div><div class="NLM_p last">Even though the adopted method can only give an estimated range and relative trend of redox potentials for SLO variants, we can conclude that the redox potential at the iron center does not directly control C–H activation rate constants. This is supported first, by only minor alternations in the range of redox potential (less than 0.1 V) for changes in rate constants of 10<sup>2</sup>–10<sup>4</sup>-fold. Second, the trend of the redox potential change (L754A &gt; DM &gt; WT &gt; L546A) does not align with the trend of observed rate constants (WT &gt; L546A &gt; L754A &gt; DM). Clearly, other factors dominate the enormous impact of the DM (and other variants) on the rate impairment of SLO.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28032" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28032" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i4" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Conclusion"><span>Conclusion</span></a></li><li><a href="#_i16" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i24" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div class="NLM_p">Hydrogen tunneling offers a unique window into the role of protein conformational motions in facilitating optimal active site alignment that accounts for the rate enhancements in enzyme catalyzed reactions. SLO has served as an ideal model for understanding the role of protein structure and dynamics in generating highly efficient tunneling rates and properties relative to small molecules in condensed phase. Kinetic analyses of this system over decades have led to a predictive multidimensional kinetic model that incorporates a hierarchy of protein motions. <a onclick="showRef(event, 'ref5 ref9 ref16 ref35'); return false;" href="javascript:void(0);" class="ref ref5 ref9 ref16 ref35">(5,9,16,35)</a> Further, recent biophysical studies of WT versus single site SLO mutants, <a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> have begun to reveal the molecular basis of catalysis-linked protein motions. The unique kinetic features emerging from a nearly fatal double mutant, DM, earlier led to the proposal of an elongation of the hydrogen transfer distance within its E-S complex that is accompanied by an apparent rigidification of the enzyme active site that prevents recovery to a tunneling DAD that approximates WT. These parameters include a greatly reduced <i>k</i><sub>cat</sub> together with an enlarged temperature dependence on <i>k</i><sub>cat</sub> (<i>E</i><sub>a</sub> ∼ 10 kcal/mol) and an enormous KIE (<sup>D</sup><i>k</i><sub>cat</sub> ∼ 500–700) that is nearly independent of temperature (Δ<i>E</i><sub>a</sub> ∼ 0 kcal/mol). <a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> The present work addresses the structural origin of the enormous KIE and other aberrant properties of DM SLO via the interrogation of a range of biophysical and structural probes that include HDXMS, EPR and room temperature X-ray crystallography.</div><div class="NLM_p">Among the kinetic parameters characteristic of DM, <i>k</i><sub>cat</sub> is found to be reduced 10<sup>4</sup>-fold relative to WT. Previously, it had been shown that altering the ligands to the metal in SLO both reduced rates and decreased reduction potentials of the metal, implicating an altered chemical driving force, Δ<i>G</i>°, in catalytic rate impairment. <a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> To test this possibility for the present series of SLO mutants, we examined relative reduction potentials using an established EPR method involving analysis of Fe(III) SLO complexes with various catechol derivatives. As shown, the DM undergoes only a very modest variation in its redox potential, with no systematic trend in the magnitude of its <i>k</i><sub>cat</sub> and estimated <i>E</i>° relative to WT (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). These experiments serve as an important frame of reference in ruling out significant alterations in the chemical driving force as the source of the impaired activity of DM (see below).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0005.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0005.jpeg" id="gr6" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Rate constants for enzyme catalyzed nonadiabatic hydrogen transfer can be formalized as <i>k</i><sub>obs</sub> = <i>K</i><sub>eq</sub>·<i>k</i><sub>PCET</sub>. <a onclick="showRef(event, 'ref9 ref28 ref30'); return false;" href="javascript:void(0);" class="ref ref9 ref28 ref30">(9,28,30)</a> (A) <i>K</i><sub>eq</sub> (≪1) represents a stochastic ground state search through inactive conformations (<i>b</i>) to reach catalytically active E-S complexes (<i>a</i> conformer). Subsequent thermal sampling further reduces the distance between the H-donor and acceptor as seen in the pretunneling <i>a*</i> conformer. (<b>B</b>-<b>D</b>) illustrate contributions from <i>k</i><sub>PCET</sub>: (B) Heavy atom protein motions produce transiently degenerate energies for the reactant and product wells, a prerequisite for wave function overlap at the tunneling ready state (TRS) (C). (D) The effective potential along the DAD sampling coordinate varies, starting with Δ<i>E</i><sub>a</sub> ≅ 0 for native enzyme (top), becoming Δ<i>E</i><sub>a</sub> ≫ 0 when the DAD is elongated following single mutations (middle) and finally arriving at the catastrophic DM scenario in which the DAD is elongated, but coupled with a rigid DAD sampling potential (bottom). In panel D, light purple refers to the shorter wave function distribution for deuterium, and dark purple represents the more distributive protium wave function.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Origins of the Elevated Enthalpic Barriers in SLO Mutants</h3><div class="NLM_p">Because wave function overlap from the rate-limiting C–H bond cleavage step in SLO catalysis is, by definition, barrier-less, the observed enthalpic barriers are derived from the conformational sampling of the enzyme–substrate complexes. We first examined the extent to which the aggregate behavior of DM could be rationalized from the two parent mutants, L546A and L754A. Though these single mutants have the same enthalpic barriers (<i>E</i><sub>aH</sub> = 4 kcal/mol), the origins of the increased <i>E</i><sub>a</sub> values relative to the native enzyme (<i>E</i><sub>aH</sub> = 2 kcal/mol) are now concluded to be different, arising either from impairment of a catalytically linked dynamic network in SLO (L546A <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a>) or an alteration in the active site microenvironment (L754A, this study).</div><div class="NLM_p">HDXMS previously showed that reduction in side chain volume at L546 or I553 produces changes in the enthalpic barrier for HDXMS rate constants within an identified region of solvated protein (317–334) that correlate with trends in the enthalpic barrier for the catalytic PCET process. <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The elevated catalytic <i>E</i><sub>aH</sub> for L546A was, thus, assigned to an impact of this mutation on a catalytically linked protein dynamical network that traverses a distance of ≥20 Å from the active site to the protein–solvent interface. By contrast, the current comparison of HDXMS profiles for L754A to WT shows these to be practically identical for every peptide including 317–334, ruling out perturbations to the dynamical network as the primary origin of the altered properties of L754A.</div><div class="NLM_p">The present studies point instead toward a perturbation of the microenvironment surrounding the iron cofactor as the source of changes in catalysis for L754A. The estimated relative redox potential for L754A (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>) is seen to exhibit a modest increase relative to the <i>E</i>°of WT (Δ<i>E</i>° &lt; 100 mV), predicting an increase in rate, thereby making a change in redox potential an improbable source of its 10<sup>3</sup> lower <i>k</i><sub>cat</sub> and elevated <i>E</i><sub>aH</sub>. The position of the additional, structured water that infiltrates the expanded active site in L754A (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) may be a particularly significant feature that influences both the kinetic parameters and EPR spectrum of apo-L754A; the latter is clearly divergent from both WT and the L546A variant (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>).</div><div class="NLM_p">Buried water molecules have been assigned to important roles in biological function, including ligand binding, protein stability and flexibility. <a onclick="showRef(event, 'ref53 ref54 ref55 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref53 ref54 ref55 ref56 ref57">(53−57)</a> In the case of buried waters that become displaced upon ligand binding, <a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> these can provide an important thermodynamic driving force for enhanced substrate affinity (see for examples, refs  <a onclick="showRef(event, 'ref55 ref56 ref57 ref59'); return false;" href="javascript:void(0);" class="ref ref55 ref56 ref57 ref59">(55−57,59)</a>). WT SLO has at least five X-ray-resolved water molecules that line the putative ligand binding pocket <a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and most, if not all, are likely to be expelled upon substrate acquisition. Because of the unique positioning of the new water introduced upon mutation at position 754, it is possible that this will remain bound in the E-S complex, leading to changes in the active site dielectric that impact both <i>k</i><sub>cat</sub> and the accompanying <i>E</i><sub>a</sub>. While X-ray crystallography of complexes of LA with SLO could, in principle, resolve any ambiguity regarding the degree to which active site waters are retained in the E-S complex (WT or otherwise), decades of efforts to obtain such data have thus far been unsuccessful. We note that X-ray structures of the apo- form of I553G provide a reasonable control for the impact of L754A. While the 553G variant shows an enlarged active site cavity volume similar to 754A, it lacks any detectable added water molecules by either cryo- or room temperature X-ray studies <a onclick="showRef(event, 'ref26 ref33'); return false;" href="javascript:void(0);" class="ref ref26 ref33">(26,33)</a> and exhibits a turnover number much closer to WT. <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><div class="NLM_p last">The combination of impaired dynamics and altered active site hydration that are distinctive of L546A and L754A, respectively, begin to rationalize the greatly increased activation energy (∼10 kcal/mol) and decreased rate of DM relative to WT. However, DM also shows two highly deviant experimental properties, absent in either L546A or L754A: these are an enormous primary deuterium room temperature isotope effect of 661 ± 27 and its very low sensitivity to temperature (Δ<i>E</i><sub>a</sub> ∼ 0). <a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28−30)</a></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> DM as an Outlier: Enormous Temperature Independent KIEs Can Be Rationalized by Modifications in Both the Hydrogenic Donor–Acceptor Distance (DAD) and the Frequency of DAD Sampling</h3><div class="NLM_p">Based on the nonadiabatic PCET model that fits the temperature independent KIEs in many native enzymes, <a onclick="showRef(event, 'ref54 ref60 ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref54 ref60 ref61 ref62">(54,60−62)</a> the observed rate constants of SLO are determined by a conformational sampling coordinate and PCET reaction coordinate, which can be visualized in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> and formalized as <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>: <a onclick="showRef(event, 'ref9 ref28 ref33'); return false;" href="javascript:void(0);" class="ref ref9 ref28 ref33">(9,28,33)</a><span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_m001.gif" alt="" id="_i12"><span class="labelSpan">(1)</span></span>where <i>K</i><sub>eq</sub> &lt; 1 represents the small fraction of active enzyme–substrate protein substates that arise from stochastic sampling processes (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A), multipled by <i>k</i><sub>PCET</sub>, the nonadiabatic rate constant of the H-tunneling process (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B–D). The <i>k</i><sub>PCET</sub> is derived from a formulation that resembles the Marcus analysis for electron tunneling (including reorganization energy, λ, and driving force Δ<i>G</i>° parameters), in which the barrier-less hydrogenic wave function overlap coordinate depends on a temperature-dependent coordinate for achieving transient degenerate energy levels between reactant and product (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B,C). The addition of a thermally averaged DAD sampling mode completes the analysis (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>D): <a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> The latter has emerged as an important ruler for the degree of active site optimization and can be quantitated by the magnitude of Δ<i>E</i><sub>a</sub>.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0006.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0006.jpeg" id="gr7" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Comparison between the ground state substrate binding mode in WT (blue) and DM (yellow). The reactive carbon C11 is colored as black, iron colored as orange.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In naturally evolved (i.e., WT) enzymes, active sites for enzymatic C–H activation reactions are finely tuned by way of conformational sampling, to achieve highly compacted DADs in pretunneling configurations, labeled conformer (<i>a*</i>) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A). When this occurs, the coordinate in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B produces the tunneling ready state (TRS) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>C) via thermal excitation that is largely independent of the labeling of substrate with deuterium; in this instance, further optimization of the reaction coordinate via DAD sampling (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>D) is either absent or minor. The DAD sampling coordinate becomes increasingly important for impaired enzymes systems, leading to an incremental increase for <i>E</i><sub>a(D)</sub> relative to <i>E</i><sub>a(H)</sub>, and is easily detected from an increase in Δ<i>E</i><sub>a</sub> away from a WT value that is generally small, close to zero. The isotope dependence of the DAD sampling coordinate, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>D, is often perceived as puzzling, but can be readily understood in the context of the smaller, less diffuse wavelength for deuterium that introduces a requirement for shorter DADs; this can become an important source of the increased energetic barrier for achieving deuterium wave function overlap when the active site environment has been perturbed away from optimal alignment. The fact that mutation most generally also leads to softer active site potential energy surfaces enhances access to the DAD sampling process.</div><div class="NLM_p last">Historically, in the case of all catalytically impairing single mutants of SLO that include L546A and L754A, the Δ<i>E</i><sub>a</sub> increases relative to WT. However, the DM kinetic isotope effect is almost completely temperature independent (Δ<i>E</i><sub>a</sub> ∼ 0). While this would at first glance suggest a finely tuned active site, the significantly decreased <i>k</i><sub>cat</sub> and enormous primary <sup>D</sup><i>k</i><sub>cat</sub> of ∼700 of DM can only be ascribed to an elongated, nonoptimal DAD in the pretunneling conformer that prevents subsequent DAD sampling. When seen in this context, the collective kinetic data indicate that DM is simply unable to recover from a nearly catastrophically elongated DAD. Though the multiple biophysical probes in the present study were initially pursued with the goal of detecting and defining which aspects of protein structure/dynamics would be unique to DM and account for an implied rigidification, we have found no evidence for impairment to the dynamical network in L754A, which looks identical to WT in this regard. While a dynamical impairment does arise in the other component of DM, L546A, it alone is not serious enough to prevent DAD sampling and allows recovery to an active site configuration and KIE similar to WT. In the context of the behavior of these two parent mutants the most direct interpretation of the origin of the impairment in DM is seen as a combination of a serious mispositioning of substrate (due to L754A) that has simultaneously made it much more difficult to rearrange itself into a configuration that permits tunneling (due to the additional impairment of the protein dynamical network from L546A).</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> DM as an Outlier: Mispositioning of the Reactive Carbon of Substrate into a Region of SLO That Lies Outside of the Native Protein Dynamical Network</h3><div class="NLM_p">In light of the current findings, results from other methods were sought as a source of structural insight into the DM’s unique behavior. ENDOR studies have shown themselves to be a powerful probe for E-S complexes of SLO, enabling a direct interrogation of the dominant ground state H-donor and H-acceptor distances. <a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Completed studies of WT show two such ground state structures, with one configuration (<i>a</i>) showing van der Waals distance for the hydrogen donor and acceptor and a second, inactive configuration (<i>b</i>) with a DAD almost 1 Å longer. Proficient wave function overlap in WT enzyme is proposed to arise via thermal sampling within the potential energy well of the “<i>a</i>” conformer to form DADs reduced from van der Waals (<i>a*</i>) (cf. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>).</div><div class="NLM_p">Importantly, ENDOR of the DM reveals only a single ground state “<i>b</i>” configuration that corresponds to the more extended, catalytically inactive form of E-S, consistent with the sluggish rate of this variant. In the same experiment, a single “<i>b</i>” configuration was also obtained with I553G, a mutation that like DM is predicted to exhibit an elongated DAD, but unlike DM, has a catalytic activity only 5-fold reduced from WT; the latter is accompanied by a highly temperature dependent KIE (Δ<i>E</i><sub>a</sub> ∼ 5 kcal/mol) that reflects active dynamical sampling back to a WT-like configuration, indicating that ENDOR alone cannot provide the elusive source of impairment for DM SLO. MD simulations were initiated for the ground state E-S structures of WT, DM and I553G using the metal to C11 and metal to C10 distances restrained by ENDOR measurements; significantly these studies suggested differences in the positioning of substrate within the DM form of SLO relative to WT and I553G.</div><div class="NLM_p">Pursuing this possibility, a recent QM/MM study of the E-S complexes for WT and DM SLO provides a potential glimpse into the molecular origins for the collective impairment of mutations at both L546 and L754. <a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> QM/MM models have the advantage over the restrained MD simulations, because these calculations do not require any input from experimental findings to generate active site conformations. Corresponding ground state models of the active site structures for WT and DM derived from these computations show an almost superimposable C1–C7 region of LA that subsequently diverges in the region of substrate that encompasses the reactive carbon and its flanking double bonds (C9–C13). Importantly, the computed ground state DADs between C11 of LA substrate and metal-bound oxygen, of 3.2 Å for WT and 3.6 Å for DM, are remarkably consistent with the distances estimated from ENDOR experiments of 3.1 and 3.8 Å, respectively. <a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> It has been noted that the two altered side chains in DM, L546 and L754, are highly conserved and act to clamp the C11 of LA in position for reaction with the adjacent Fe(III)–OH. <a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> It is, therefore, of considerable interest that simultaneous replacement of both leucine side chains for alanine in SLO leads to a new QM/MM-based configuration for C11 of substrate, which has moved into the space normally filled by L754 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>).</div><div class="NLM_p">A second observation in support of substrate mispositioning in DM comes from the accompanying QM/MM computation of Δ<i>G</i>° for conversion of bound substrate to product. For DM, this Δ<i>G</i>° is calculated to be more uphill by 10 kcal/mol relative to WT, in the same direction as the experimentally observed increase in the energy of activation of ∼10 kcal/mol for DM. Given an improbable ability of the SLO active site to alter the innate bond dissociation energy of bound substrate, together with our failure to detect significant changes in redox potential of the Fe cofactor in DM, a change in chemical driving force is ruled out as the source of the increased Δ<i>G</i>° for DM. The most reasonable interpretation is a stabilization of substrate within a binding mode that is highly compromised with regard to its distance and ability to promote wave function overlap from the C11 of substrate to the Fe(III)–OH acceptor. One caveat regarding the referenced QM/MM study is its inability to capture the experimental evidence for irreversible formation of pentadienyl radical under single turnover, anaerobic conditions. <a onclick="showRef(event, 'ref20 ref29'); return false;" href="javascript:void(0);" class="ref ref20 ref29">(20,29)</a><a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> suggests a way to rationalize the discrepancy between computation (Δ<i>G</i>° &gt; 0) and experimental findings (Δ<i>G</i>° &lt; 0). As shown, completion of the DM reaction will entail a return of the initially formed radical intermediate (illustrated in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) to a less active but thermodynamically more stable binding position (E-I). This leads to the net exergonic process observed experimentally for the conversion of E-S to E-I. In this context, the trajectory captured by QM/MM for DM represents only a portion of the reaction in which E-S has been converted to E-I′ via an uphill process.</div><figure id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0008.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0008.jpeg" id="gr8" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Free Energy Profile for WT (red) Represented by a Two State Thermodynamic Model Involving E-S and the Enzyme-Bound Pentadienyl Intermediate, E-I<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"></p><p class="last"><span class="fn-label"><sup>a</sup></span>In the case of DM, the most stable E-S complex (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>) must first isomerize to a pretunneling configuration (E-S′) that converts via PCET to the mispositioned pentadienyl intermediate (E-I′) with a driving force similar to WT. The return of the bound intermediate E-I′ to a more stable binding position completes the reaction. The dashed blue line has been added as a possible model for the QM/MM computation.</p><p></p></figure></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10730" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10730" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i4" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Conclusion"><span>Conclusion</span></a></li><li><a href="#_i16" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i24" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div class="NLM_p last">The paper presents the biophysical characterization of a kinetically impaired double mutant of soybean lipoxygenase L546A/L754A and its parent single mutants L546A and L754A. Beyond adopting features from both parent single mutants, DM demonstrates a distinctive substrate binding mode that mispositions the substrate into configurations that are unfavorable for hydrogen tunneling. In the WT SLO, L546 resides within a protein motional network communicating between the surface loop and active site (<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Figure S2</a>). <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> It is of considerable interest that while L546A alone disrupts the protein motional network connecting the catalytically relevant surface loop and the active site, <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> the enzyme is still able to undergo DAD sampling to achieve a KIE similar to WT. <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The movement of C–H of substrate into the “hole” created by the second mutation of L756 in DM has the additional effect of removing the reactive carbon of substrate from access to such thermal activation. This combination of a new substrate binding mode that is removed from access to thermally activated protein modes provides an explanation for the divergent features of DM that include its greatly impaired <i>k</i><sub>cat</sub>, excessively elevated energy of activationand its enormous and close to temperature independent KIE. The current study, thus, provides a critical examination and insightful structural example of how relatively minor alterations in the placement of the reactive bond(s) of a bound substrate can cause dramatic alterations in the efficiency of enzymatic rate acceleration. In this context, the findings are highly relevant to the generic origins of substrate specificity in enzymes, whereby changes in the volume of active site side chains may either activate or impair an intended reaction course.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78347" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78347" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i4" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Conclusion"><span>Conclusion</span></a></li><li><a href="#_i16" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i24" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Materials</h3><div class="NLM_p last">All reagents were purchased from commercial sources and used without further purification unless otherwise indicated. Wild-type and mutant SLOs were prepared, expressed and purified according to the previous report. <a onclick="showRef(event, 'ref28 ref29 ref31'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref31">(28,29,31)</a></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> EPR Sample Preparation</h3><div class="NLM_p last">The EPR samples were prepared according to the previous protocols <a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> with several modifications. Each sample contained 140 μM SLO and 1 equiv of respective catechols in 0.1 M potassium phosphate (pH = 7.0). The ferric (yellow) lipoxygenase was first prepared from the mixing purified ferrous lipoxygenase and linoleic acid. The concentrated, purified ferrous lipoxygenase (300 μM) was titrated with linoleic acid (10 mM) at 4 °C, 0.1 M borate buffer (pH = 9.0) while monitoring the optical spectrum between 300 and 800 nm for development of the 330 nm peak characteristic of the ferric lipoxygenase. For WT and mutants, 2 equiv of linoleic acid was required to fully oxidize the protein. The ferric lipoxygenase (ca. 280 μM) was removed from the cuvette and dialyzed overnight in 0.1 M potassium phosphate buffer (pH = 7.0) at 4 °C. The sample was then stored at −80 °C until use. A known amount of catechols was transferred to an anaerobic glovebox, then dissolved in deoxygenated 0.1 M potassium phosphate with a final concentration of 280 μM. The catechols solution (100 μL) was mixed with deoxygenated ferric lipoxygenase (100 μL) and transferred into a suitable EPR tube. The tube was sealed with rubber stopper and kept at 4 °C. After 12 h, the tube was removed from the glovebox, and frozen in liquid nitrogen within 1 min of its removal. The samples were stored at liquid nitrogen temperature until data acquisition. For each SLO variant, two control EPR samples were prepared as described above but with minor variations: One only contains 140 μM ferric lipoxygenase; another only contains 140 μM ferrous lipoxygenase.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> EPR Spectroscopy</h3><div class="NLM_p last">Samples for X-band (∼9.4 GHz) EPR spectroscopy were measured at the CalEPR center at the University of California, Davis. Continuous wave (CW) spectra were collected using a Bruker Instruments EleXsys-II E500 CW EPR spectrometer (Bruker Corporation, Billerica, MA) equipped with an Oxford Instruments ESR900 liquid helium cryostat and an Oxford Instruments ITC503 temperature and gas-flow controller. Samples were measured under nonsaturating slow-passage conditions using a Super-High Q resonator (ER 4122SHQE). Typical acquisition conditions were: <i>T</i> = 6.5 K; 9.38 GHz microwave frequency; 10G modulation amplitude; 0.6325 mW microwave power.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> HDX Sample Preparation</h3><div class="NLM_p last">Purified aliquots of wild-type and variant SLO were thawed and diluted to 100 μM. Protein aliquots from this stock sample were diluted 10-fold in D<sub>2</sub>O buffer 10 mM HEPES pD 7.4; corrected pD = pHread +0.4(65) to a pL of 8.0 (confirmed with pH electrode). Samples were incubated randomly at a specified temperature (temperatures studied: 10, 20, 30 and 40 °C; bath stability ±0.1 °C), using a NESLAB RTE-111 water bath, for 14 time points (0, 10, 30, 45, 60, 180, 600, 1200, 1800, 2700, 3600, 7200, 10 800, 14 400 s) at each temperature. The same conditions were used in characterizing each mutant. Upon completion of the designated incubation time, samples were rapidly cooled (−20 °C bath) and acid quenched (to pH 2.4, confirmed with pH electrode, with 0.32 M citric acid stock solution at 0 °C). SLO derived peptides were generated from 2.5 min incubations with pepsin after addition of guanidine HCl (ca. 0.5 M final concentration). The peptide fragments were removed from the immobilized pepsin with spin cups and flash frozen immediately in liquid N2. Samples were stored at −80 °C until data collection.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Liquid Chromatography-Mass Spectrometry for Hydrogen/Deuterium Exchange Measurements</h3><div class="NLM_p last">Peptide assignments (from the digestion in water) were previously determined for WT SLO. <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Deuterated, pepsin-digested samples of SLO from hydrogen/deuterium exchange experiments were analyzed using an Agilent 1200 LC (Santa Clara, CA) that was connected in-line with the LTQ Orbitrap XL mass spectrometer (Thermo). The LC was equipped with a reversed-phase analytical column (Viva C8, 30 mm length × 1.0 mm inner diameter, 5 μm particles, Restek, Bellefonte, PA) and guard precolumn (C8, Restek). Solvent A was 99.9% water/0.1% formic acid and solvent B was 99.9% acetonitrile/0.1% formic acid (v/v). Each sample was thawed immediately prior to injection onto the column. The elution program consisted of a linear gradient from 5% to 10% B over 1 min, a linear gradient to 40% B over 5 min, a linear gradient to 100% B over 4 min, isocratic conditions at 100% B for 3 min, a linear gradient to 5% B over 0.5 min, and isocratic conditions at 5% B for 5.5 min, at a flow rate of 300 μL/min. The column compartment was maintained at 4 °C and lined with towels to absorb atmospheric moisture condensation. The column exit was connected to the ESI source of the mass spectrometer using PEEK tubing (0.005″ inner diameter × 1/16″ outer diameter, Agilent). Mass spectra were acquired in the positive ion mode over the range <i>m</i>/<i>z</i> = 350 to 1800 using the Orbitrap mass analyzer, in profile format, with a mass resolution setting of 100 000 (at <i>m</i>/<i>z</i> = 400). Data acquisition was controlled using Xcalibur software (version 2.0.7, Thermo).</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> HDX Data Analysis</h3><div class="NLM_p last">Mass spectral data acquired for HDX measurements were analyzed using the software, HDX WorkBench. The percent deuterium incorporation was calculated for each of these peptides, taking into account the number of amide linkages (excluding proline residues) and the calculated number of deuterons incorporated. The values were normalized for 100% D<sub>2</sub>O and corrected for peptide-specific back-exchange, as determined previously. <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The data were plotted as deuterium exchange versus time using Igor Pro software. The rates and extents of exchange (at 4 h) were determined from one- or two-exponential fits to the analyzed time-resolved HDX data. The weighted average rates of HDX were determined from the individual rate constants of these two exponentials. Apparent enthalpies of activation from HDXMS rates (<i>E</i><sub>aHDX(avg)</sub>) were determined for each peptide and mutation from the linear fits to the Arrhenius plots of the HDXMS weighted average exchange rates versus inverse temperature. These <i>E</i><sub>aHDX(avg)</sub> values represent the mean ± s.d. from the five temperatures studied here.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Room-Temperature Crystallography</h3><div class="NLM_p last">All additional protein purification and crystallization steps were performed as previously described. <a onclick="showRef(event, 'ref33 ref67'); return false;" href="javascript:void(0);" class="ref ref33 ref67">(33,67)</a> Diffraction data were collected at room temperature at Beamline 8.3.1 of the Advanced Light Source at Lawrence Berkeley National Laboratory. Data were processed by the ELVES program, <a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> with integration performed in MOSFLM <a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> and scaling and merging in SCALA, POINTLESS and TRUNCATE, <a onclick="showRef(event, 'ref70 ref71'); return false;" href="javascript:void(0);" class="ref ref70 ref71">(70,71)</a> or using XDS <a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> with scaling and merging using AIMLESS, <a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> POINTLESS and TRUNCATE in the CCP4 suite <a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> provided through SBGRID. <a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> An initial solution was found by molecular replacement using Phaser <a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> with PDB <a href="http://doi.org/10.2210/pdb3PZW/pdb">3PZW</a> as the search model. Manual refinement was performed in Coot <a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> and automated refinement using the PHENIX suite. <a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> Alternate conformers were modeled using qFit. <a onclick="showRef(event, 'ref79 ref80'); return false;" href="javascript:void(0);" class="ref ref79 ref80">(79,80)</a> We manually deleted waters that did not fit into the Fo-Fc map density and edited alternate side chain conformations as indicated in the positive density of the Fo-Fc map. Active site volumes were calculated using CASTp. <a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> The relative size of the pockets in each mutant was based on analysis of the most populated conformer of the residues lining the pocket and included residues Gln495, Leu496 His499, Trp500, His504, Ile538, Ala542, Leu546, Ile547, Ile553, Phe557, Val564, Ser567, Val750, Ile751, Leu754 and Ile839. See <a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Table S4</a> for data processing and refinement statistics.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26324" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26324" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i4" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Conclusion"><span>Conclusion</span></a></li><li><a href="#_i16" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i24" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/jacs.8b10992" class="ext-link">10.1021/jacs.8b10992</a>.</p><ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">(<a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf">ja8b10992_si_001.pdf (4.98 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jacs.8b10992" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget" style="width: 876.3px; display: block;"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms &amp; Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86467" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86467" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i4" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Conclusion"><span>Conclusion</span></a></li><li><a href="#_i16" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i24" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Judith P. Klinman</span> - <span class="hlFld-Affiliation affiliation">California
Institute for Quantitative Biosciences,&nbsp;
    Department of Chemistry
        &nbsp;and&nbsp;
    Department of Molecular and Cell
Biology, University of California, Berkeley, Berkeley, California 94720, United States</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-5734-2843" title="Orcid link">http://orcid.org/0000-0001-5734-2843</a>;&nbsp;
    </span><span class="author-email">
                Email: <a href="mailto:klinman@berkeley.edu">klinman@berkeley.edu</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shenshen Hu</span> - <span class="hlFld-Affiliation affiliation">California
Institute for Quantitative Biosciences,&nbsp;
    Department of Chemistry
        &nbsp;and&nbsp;
    Department of Molecular and Cell
Biology, University of California, Berkeley, Berkeley, California 94720, United States</span>;&nbsp;
    <span>Present Address:
                        S.H.: Corvus Pharmaceuticals, Inc., 863 Mitten Road #102, Burlingame, CA 94010, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adam R. Offenbacher</span> - <span class="hlFld-Affiliation affiliation">California
Institute for Quantitative Biosciences,&nbsp;
    Department of Chemistry
        &nbsp;and&nbsp;
    Department of Molecular and Cell
Biology, University of California, Berkeley, Berkeley, California 94720, United States</span>;&nbsp;
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, East Carolina University, Greenville, North Carolina 27858, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Erin M. Thompson</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering and Therapeutic Science, University of California, San Francisco, San Francisco, California 94158, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christine L. Gee</span> - <span class="hlFld-Affiliation affiliation">California
Institute for Quantitative Biosciences
        &nbsp;and&nbsp;
    Department of Molecular and Cell
Biology, University of California, Berkeley, Berkeley, California 94720, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jarett Wilcoxen</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of California,
Davis, Davis, California 95695, United States</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0003-2220-0815" title="Orcid link">http://orcid.org/0000-0003-2220-0815</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cody A. M. Carr</span> - <span class="hlFld-Affiliation affiliation">California
Institute for Quantitative Biosciences,&nbsp;
    Department of Chemistry
        &nbsp;and&nbsp;
    Department of Molecular and Cell
Biology, University of California, Berkeley, Berkeley, California 94720, United States</span>;&nbsp;
    <span>Present Address:
                        C.A.M.C.: Riffyn, Oakland, CA 94612, United States</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0002-5887-4507" title="Orcid link">http://orcid.org/0000-0002-5887-4507</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniil M. Prigozhin</span> - <span class="hlFld-Affiliation affiliation">California
Institute for Quantitative Biosciences
        &nbsp;and&nbsp;
    Department of Molecular and Cell
Biology, University of California, Berkeley, Berkeley, California 94720, United States</span>;&nbsp;
    <span>Present Address:
                        D.M.P.: Department of Medicine, School of Clinical Medicine, Cambridge University, Cambridge CB2 0SP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vanessa Yang</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of California, Berkeley, Berkeley, California 94720, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tom Alber</span> - <span class="hlFld-Affiliation affiliation">California
Institute for Quantitative Biosciences
        &nbsp;and&nbsp;
    Department of Molecular and Cell
Biology, University of California, Berkeley, Berkeley, California 94720, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">R. David Britt</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of California,
Davis, Davis, California 95695, United States</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0003-0889-8436" title="Orcid link">http://orcid.org/0000-0003-0889-8436</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James S. Fraser</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering and Therapeutic Science, University of California, San Francisco, San Francisco, California 94158, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br><sup></sup>T.A.: Deceased<br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63361" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63361" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i4" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Conclusion"><span>Conclusion</span></a></li><li><a href="#_i16" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i24" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><p class="last">We thank Prof. John Kuriyan, UC Berkeley for his provision of personnel and facilities in the X-ray crystallographic portion of this investigation. We also thank Prof. Sharon Hammes-Schiffer and Dr. Pengfei Li for assistance in the preparation of Figure 6, which was derived from their QM/MM analysis of the WT and DM forms of SLO (cf., ref  <a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a>). This study was supported in part by funding from the following sources: National Institutes of Health grants to J.P.K. (GM118117), A.R.O. (GM113432 (F32)), R.D.B. (1R35GM126961-01) and J.S.F. (GM123159 and GM124149), National Science Foundation grant to J.S.F. (STC-1231306), University of California Office of the President Laboratory Fees Research Program to J.S.F. (LFR-17-476732), the David and Lucile Packard Foundation Packard Fellowship to J.S.F., and institutional support from the Howard Hughes Medical Institute to C.L.G. Data collection at BL831 at the Advanced Light Sources is supported by the Director, Office of Science, Office of Basic Energy Sciences, of the U.S. Department of Energy under Contract No. DE-AC02-05CH11231, UC Office of the President, Multicampus Research Programs and Initiatives grant MR 15 32859, the Program Breakthrough Biomedical Research, which is partially funded by the Sandler Foundation, and the NIH P30GM124169.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47858" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47858" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i4" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Conclusion"><span>Conclusion</span></a></li><li><a href="#_i16" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i24" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr></div><p class="references-count">
            This article references 81 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Radzicka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfenden, R.</span></span> <span> </span><span class="NLM_article-title">A Proficient Enzyme</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">90</span>– <span class="NLM_lpage">93</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.7809611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1126%2Fscience.7809611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=7809611" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK2MXivF2iuro%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=267&amp;publication_year=1995&amp;pages=90-93&amp;author=A.+Radzickaauthor=R.+Wolfenden&amp;title=A+Proficient+Enzyme&amp;doi=10.1126%2Fscience.7809611" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A proficient enzyme</span></div><div class="casAuthors">Radzicka, Anna; Wolfenden, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">5194</span>),
    <span class="NLM_cas:pages">90-3</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review and discussion, with 37 refs.  Orotic acid is decarboxylated with a half-time (t1/2) of 78 million years in neutral aq. soln. at room temp., as indicated by reactions in quartz tubes at elevated temps.  Spontaneous hydrolysis of phosphodiester bonds, such as those present in the backbone of DNA, proceeds even more slowly at high temps., but the heat of activation is less pos., so that di-Me phosphate is hydrolyzed with the t1/2 of 130,000 yr in neutral soln. at room temp.  These values extend the known range of spontaneous rate consts. for reactions that are also susceptible to catalysis by enzymes to more than 14 orders or magnitude.  Values of the second-order rate const. kcat/Km for the corresponding enzyme reactions are confined to a range of only 600-fold, in contrast, orotidine 5'-phosphate decarboxylase, an extremely proficient enzyme, enhances the rate of reaction by a factor of 1017 and is estd. to bind the altered substrate in the transition state with a dissocn. const. of less than 5×10-24 M.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLxicA2pBfXrVg90H21EOLACvtfcHk0lidy6qdq8z_eA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXivF2iuro%253D&amp;md5=03a6da852b507155b6d6c5308a83201d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.7809611&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7809611%26sid%3Dliteratum%253Aachs%26aulast%3DRadzicka%26aufirst%3DA.%26aulast%3DWolfenden%26aufirst%3DR.%26atitle%3DA%2520Proficient%2520Enzyme%26jtitle%3DScience%26date%3D1995%26volume%3D267%26spage%3D90%26epage%3D93%26doi%3D10.1126%2Fscience.7809611" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfenden, R.</span></span> <span> </span><span class="NLM_article-title">Catalytic Proficiency: The Extreme Case of S-O Cleaving Sulfatases</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">531</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja208827q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja208827q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=134&amp;publication_year=2012&amp;pages=525-531&amp;author=D.+R.+Edwardsauthor=D.+C.+Lohmanauthor=R.+Wolfenden&amp;title=Catalytic+Proficiency%3A+The+Extreme+Case+of+S-O+Cleaving+Sulfatases&amp;doi=10.1021%2Fja208827q" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fja208827q&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja208827q%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DD.%2BR.%26aulast%3DLohman%26aufirst%3DD.%2BC.%26aulast%3DWolfenden%26aufirst%3DR.%26atitle%3DCatalytic%2520Proficiency%253A%2520The%2520Extreme%2520Case%2520of%2520S-O%2520Cleaving%2520Sulfatases%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D525%26epage%3D531%26doi%3D10.1021%2Fja208827q" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruice, T. C.</span></span> <span> </span><span class="NLM_article-title">A View at the Millennium: The Efficiency of Enzymatic Catalysis</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">139</span>– <span class="NLM_lpage">148</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar0001665</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar0001665" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD38XksVentw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=35&amp;publication_year=2002&amp;pages=139-148&amp;author=T.+C.+Bruice&amp;title=A+View+at+the+Millennium%3A+The+Efficiency+of+Enzymatic+Catalysis&amp;doi=10.1021%2Far0001665" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A view at the millennium: the efficiency of enzymatic catalysis</span></div><div class="casAuthors">Bruice, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">139-148</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review and discussion with 58 refs.  L. Pauling (1946) proposed that the active site of an enzyme (E) binds the transition state (TS) in preference to the substrate (S), and by doing so, stabilizes the TS and lowers the activation energy.  E binding of TS in preference to S and increasing the TΔS⧧ by freezing out motions in E·S and E·TS have been accepted as the driving forces in enzymic catalysis.  However, the smaller value of ΔG⧧ for a 1-substrate enzymic reaction, as compared to its nonenzymic counterpart, is generally the result of a smaller value of ΔH⧧.  The TS in an enzymic reaction is reached through ground-state conformers that closely resemble the TS (near-attack conformers or NACs).  E·NACs are in thermal equil. with all other E·S conformers and are turnstiles through which substrate mols. must pass to arrive at the lowest-energy TS.  The TS in E·TS may or may not be bound tighter than the NAC is in E·NAC.  Thus, the belief that enzymic reactions owe their facility to TS binding or to an increase in ΔStmo requires modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkAreKITPLB7Vg90H21EOLACvtfcHk0lhjqorDRbC5tw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD38XksVentw%253D%253D&amp;md5=2ae856f67a7524cc0a9ba10ffe650fc4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Far0001665&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far0001665%26sid%3Dliteratum%253Aachs%26aulast%3DBruice%26aufirst%3DT.%2BC.%26atitle%3DA%2520View%2520at%2520the%2520Millennium%253A%2520The%2520Efficiency%2520of%2520Enzymatic%2520Catalysis%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2002%26volume%3D35%26spage%3D139%26epage%3D148%26doi%3D10.1021%2Far0001665" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benkovic, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">A Perspective on Enzyme Catalysis</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>301</i></span>,  <span class="NLM_fpage">1196</span>– <span class="NLM_lpage">1202</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.1085515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1126%2Fscience.1085515" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=12947189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmslShsL0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=301&amp;publication_year=2003&amp;pages=1196-1202&amp;author=S.+J.+Benkovicauthor=S.+Hammes-Schiffer&amp;title=A+Perspective+on+Enzyme+Catalysis&amp;doi=10.1126%2Fscience.1085515" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A perspective on enzyme catalysis</span></div><div class="casAuthors">Benkovic, Stephen J.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">301</span>
        (<span class="NLM_cas:issue">5637</span>),
    <span class="NLM_cas:pages">1196-1202</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  The seminal hypotheses proposed over the years for enzymic catalysis have been scrutinized.  The historical record has been explored from both biochem. and theor. perspectives.  Particular attention is given to the impact of mol. motions within a protein on an enzyme's catalytic properties.  A case study is dihydrofolate reductase, which provides evidence for coupled networks of predominantly conserved residues that influence protein structure and motion.  Such coupled networks have important implications for the origin and evolution of enzymes, as well as for protein engineering.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokLxa7e9sqC7Vg90H21EOLACvtfcHk0lhjqorDRbC5tw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3sXmslShsL0%253D&amp;md5=d97a8aa302a623866f98a4ff9c744f98</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.1085515&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1085515%26sid%3Dliteratum%253Aachs%26aulast%3DBenkovic%26aufirst%3DS.%2BJ.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DA%2520Perspective%2520on%2520Enzyme%2520Catalysis%26jtitle%3DScience%26date%3D2003%26volume%3D301%26spage%3D1196%26epage%3D1202%26doi%3D10.1126%2Fscience.1085515" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, Z. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">A 21st Century Revisionist’s View at a Turning Point in Enzymology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">543</span>– <span class="NLM_lpage">550</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2Fnchembio.204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=19620995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD1MXoslentrY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2009&amp;pages=543-550&amp;author=Z.+D.+Nagelauthor=J.+P.+Klinman&amp;title=A+21st+Century+Revisionist%E2%80%99s+View+at+a+Turning+Point+in+Enzymology&amp;doi=10.1038%2Fnchembio.204" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A 21st century revisionist's view at a turning point in enzymology</span></div><div class="casAuthors">Nagel, Zachary D.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">543-550</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the fact that the no. of publications assocd. with the keyword 'enzyme' increases every year, the precise origin of enzyme catalysis has remained unresolved.  Because of sustained intensive research efforts from an increasing no. of labs., detailed information regarding the physics, chem., and kinetics of enzymes is accumulating rapidly.  The growing body of data contains many examples of kinetic behavior that are incompatible with a static view of enzyme catalysis.  As a result, numerous labs. are approaching the consensus that protein motion plays an essential role in enzyme catalysis.  A model that incorporates nuclear quantum tunneling together with 2 classes of protein motion, termed conformational sampling (pre-organization) and reorganization, is recommended as a means of understanding the large body of data for enzyme-catalyzed H-transfers.  It should also serve as a vehicle for future efforts in the development of potent enzyme inhibitors and the de novo design of all classifications of enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxj2lQthYBt7Vg90H21EOLACvtfcHk0lj7KLvpH8o18g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXoslentrY%253D&amp;md5=5722cbad9de292231d410b357f9797bd</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.204&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.204%26sid%3Dliteratum%253Aachs%26aulast%3DNagel%26aufirst%3DZ.%2BD.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DA%252021st%2520Century%2520Revisionist%25E2%2580%2599s%2520View%2520at%2520a%2520Turning%2520Point%2520in%2520Enzymology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D543%26epage%3D550%26doi%3D10.1038%2Fnchembio.204" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span> <span> </span><span class="NLM_article-title">Enzymatic Transition States and Dynamic Motion in Barrier Crossing</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">551</span>– <span class="NLM_lpage">558</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2Fnchembio.202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=19620996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD1MXoslenu7k%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2009&amp;pages=551-558&amp;author=S.+D.+Schwartzauthor=V.+L.+Schramm&amp;title=Enzymatic+Transition+States+and+Dynamic+Motion+in+Barrier+Crossing&amp;doi=10.1038%2Fnchembio.202" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Enzymic transition states and dynamic motion in barrier crossing</span></div><div class="casAuthors">Schwartz, Steven D.; Schramm, Vern L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">551-558</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  What are the at. motions at enzymic catalytic sites on the timescale of chem. change.  Combined exptl. and computational chem. approaches take advantage of transition-state analogs to reveal dynamic motions linked to transition-state formation.  QM/MM transition path sampling from reactive complexes provides both temporal and dynamic information for barrier crossing.  Fast (femtosecond to picosecond) dynamic motions provide essential links to enzymic barrier crossing by local or promoting-mode dynamic searches through bond-vibrational space.  Transition-state lifetimes are within the femtosecond timescales of bond vibrations and show no manifestations of stabilized, equilibrated complexes.  The slow binding and protein conformational changes (microsecond to millisecond) also required for catalysis are temporally decoupled from the fast dynamic motions forming the transition state.  According to this view of enzymic catalysis, transition states are formed by fast, coincident dynamic excursions of catalytic site elements, while the binding of transition-state analogs is the conversion of the dynamic excursions to equilibrated states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6kNG4U5WwFbVg90H21EOLACvtfcHk0lj7KLvpH8o18g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXoslenu7k%253D&amp;md5=6e8267248f4674b9be11fb9c16cea2fe</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.202&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.202%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DS.%2BD.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26atitle%3DEnzymatic%2520Transition%2520States%2520and%2520Dynamic%2520Motion%2520in%2520Barrier%2520Crossing%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D551%26epage%3D558%26doi%3D10.1038%2Fnchembio.202" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glowacki, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, A. J.</span></span> <span> </span><span class="NLM_article-title">Taking Ockham’s Razor to Enzyme Dynamics and Catalysis</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">176</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.1244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2Fnchem.1244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=22354430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyit7o%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2012&amp;pages=169-176&amp;author=D.+R.+Glowackiauthor=J.+N.+Harveyauthor=A.+J.+Mulholland&amp;title=Taking+Ockham%E2%80%99s+Razor+to+Enzyme+Dynamics+and+Catalysis&amp;doi=10.1038%2Fnchem.1244" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Taking Ockham's razor to enzyme dynamics and catalysis</span></div><div class="casAuthors">Glowacki, David R.; Harvey, Jeremy N.; Mulholland, Adrian J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">169-176</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The role of protein dynamics in enzyme catalysis is a matter of intense current debate.  Enzyme-catalyzed reactions that involve significant quantum tunneling can give rise to exptl. kinetic isotope effects with complex temp. dependences, and it has been suggested that std. statistical rate theories, such as transition-state theory, are inadequate for their explanation.  Here, the authors have introduced aspects of transition-state theory relevant to the study of enzyme reactivity, taking cues from chem. kinetics and dynamics studies of small mols. in the gas phase and in soln., where breakdowns of statistical theories have received significant attention and their origins are relatively better understood.  The authors discuss recent theor. approaches to understanding enzyme activity and then show how exptl. observations for a no. of enzymes may be reproduced using a transition-state theory framework with phys. reasonable parameters.  Essential to this simple model is the inclusion of multiple conformations with different reactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzlnwn-OYN8rVg90H21EOLACvtfcHk0lj7KLvpH8o18g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhsVyit7o%253D&amp;md5=a917798b4751dc4898a50e6e3f76647a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1244&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1244%26sid%3Dliteratum%253Aachs%26aulast%3DGlowacki%26aufirst%3DD.%2BR.%26aulast%3DHarvey%26aufirst%3DJ.%2BN.%26aulast%3DMulholland%26aufirst%3DA.%2BJ.%26atitle%3DTaking%2520Ockham%25E2%2580%2599s%2520Razor%2520to%2520Enzyme%2520Dynamics%2520and%2520Catalysis%26jtitle%3DNat.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D169%26epage%3D176%26doi%3D10.1038%2Fnchem.1244" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scrutton, N. S.</span></span> <span> </span><span class="NLM_article-title">Good Vibrations in Enzyme-Catalysed Reactions</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">168</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.1223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2Fnchem.1223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=22354429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyit70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2012&amp;pages=161-168&amp;author=S.+Hayauthor=N.+S.+Scrutton&amp;title=Good+Vibrations+in+Enzyme-Catalysed+Reactions&amp;doi=10.1038%2Fnchem.1223" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Good vibrations in enzyme-catalyzed reactions</span></div><div class="casAuthors">Hay, Sam; Scrutton, Nigel S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">161-168</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fast motions (femtosecond-to-picosecond) and their potential involvement during enzyme-catalyzed reactions have ignited considerable interest in recent years.  Their influence on reaction chem. has been inferred indirectly from studies of the anomalous temp. dependence of kinetic isotope effects and computational simulations.  But, can such motions reduce the width and height of energy barriers along the reaction coordinate, and contribute to quantum mech. and/or classical nuclear-transfer chem.  Here, the authors discuss contemporary ideas for enzymic reactions invoking a role for fast 'promoting' (or 'compressive') motions that, in principle, can aid H-transfer reactions.  Of key importance is the direct demonstration of a role for compressive motions and the ability to understand in at. detail the structural origin of these fast motions, but so far this has not been achieved.  Here, the authors discuss both indirect exptl. evidence that supports a role for compressive motion and the addnl. insight gained from computational simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-AwOb4hlFN7Vg90H21EOLACvtfcHk0lj7KLvpH8o18g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhsVyit70%253D&amp;md5=b159a2bbce7ea193f8cc51d28840ddd2</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1223&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1223%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DS.%26aulast%3DScrutton%26aufirst%3DN.%2BS.%26atitle%3DGood%2520Vibrations%2520in%2520Enzyme-Catalysed%2520Reactions%26jtitle%3DNat.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D161%26epage%3D168%26doi%3D10.1038%2Fnchem.1223" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offenbacher, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span> <span> </span><span class="NLM_article-title">Origins of Enzyme Catalysis: Experimental Findings for C–H Activation, New Models, and Their Relevance to Prevailing Theoretical Constructs</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">18409</span>– <span class="NLM_lpage">18427</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.7b08418</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b08418" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOls7fO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=139&amp;publication_year=2017&amp;pages=18409-18427&amp;author=J.+P.+Klinmanauthor=A.+R.+Offenbacherauthor=S.+Hu&amp;title=Origins+of+Enzyme+Catalysis%3A+Experimental+Findings+for+C%E2%80%93H+Activation%2C+New+Models%2C+and+Their+Relevance+to+Prevailing+Theoretical+Constructs&amp;doi=10.1021%2Fjacs.7b08418" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Origins of Enzyme Catalysis: Experimental Findings for C-H Activation, New Models, and Their Relevance to Prevailing Theoretical Constructs</span></div><div class="casAuthors">Klinman, Judith P.; Offenbacher, Adam R.; Hu, Shenshen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">18409-18427</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The phys. basis for enzymic rate accelerations is a subject of great fundamental interest and of direct relevance to areas that include the de novo design of green catalysts and the pursuit of new drug regimens.  Extensive investigations of C-H activating systems have provided considerable insight into the relationship between an enzyme's overall structure and the catalytic chem. at its active site.  This Perspective highlights recent exptl. data for two members of distinct, yet iconic C-H activation enzyme classes, lipoxygenases and prokaryotic alc. dehydrogenases.  The data necessitate a reformulation of the dominant textbook definition of biol. catalysis.  A multidimensional model emerges that incorporates a range of protein motions that can be parsed into a combination of global stochastic conformational thermal fluctuations and local donor-acceptor distance sampling.  These motions are needed to achieve a high degree of precision with regard to internuclear distances, geometries, and charges within the active site.  The available model also suggests a phys. framework for understanding the empirical enthalpic barrier in enzyme-catalyzed processes.  We conclude by addressing the often conflicting interface between computational and exptl. chemists, emphasizing the need for computation to predict exptl. results in advance of their measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOOb4Grn56bLVg90H21EOLACvtfcHk0lgUbEOTo93Adg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOls7fO&amp;md5=842f40425e13f7398e5b9cc43e88bcfa</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b08418&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b08418%26sid%3Dliteratum%253Aachs%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26aulast%3DOffenbacher%26aufirst%3DA.%2BR.%26aulast%3DHu%26aufirst%3DS.%26atitle%3DOrigins%2520of%2520Enzyme%2520Catalysis%253A%2520Experimental%2520Findings%2520for%2520C%25E2%2580%2593H%2520Activation%252C%2520New%2520Models%252C%2520and%2520Their%2520Relevance%2520to%2520Prevailing%2520Theoretical%2520Constructs%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D18409%26epage%3D18427%26doi%3D10.1021%2Fjacs.7b08418" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burton, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolfson, D. N.</span></span> <span> </span><span class="NLM_article-title">Installing Hydrolytic Activity into a Completely De Novo Protein Framework</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">837</span>– <span class="NLM_lpage">844</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.2555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2Fnchem.2555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=27554410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFWjs7%252FN" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2016&amp;pages=837-844&amp;author=A.+J.+Burtonauthor=A.+R.+Thomsonauthor=W.+M.+Dawsonauthor=R.+L.+Bradyauthor=D.+N.+Woolfson&amp;title=Installing+Hydrolytic+Activity+into+a+Completely+De+Novo+Protein+Framework&amp;doi=10.1038%2Fnchem.2555" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Installing hydrolytic activity into a completely de novo protein framework</span></div><div class="casAuthors">Burton, Antony J.; Thomson, Andrew R.; Dawson, William M.; Brady, R. Leo; Woolfson, Derek N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">837-844</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design of enzyme-like catalysts tests the understanding of sequence-to-structure/function relations in proteins.  Here, the authors installed hydrolytic activity predictably into a completely de novo and thermostable α-helical barrel, which comprised 7 helixes arranged around an accessible channel.  The authors showed that the lumen of the barrel accepted 21 mutations to functional polar residues.  The resulting variant, which had Cys-His-Glu triads on each helix, hydrolyzed p-nitrophenyl acetate with catalytic efficiencies that matched the most-efficient redesigned hydrolases based on natural protein scaffolds.  This is the 1st report of a functional catalytic triad engineered into a de novo protein framework.  The flexibility of this system also allowed the facile incorporation of unnatural side-chains to improve activity and probe the catalytic mechanism.  Such a predictable and robust construction of truly de novo biocatalysts holds promise for applications in chem. and biochem. synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPhsHmLnqYRbVg90H21EOLACvtfcHk0lgUbEOTo93Adg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XhtFWjs7%252FN&amp;md5=502ed91d1528dab9147614353f9a081f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2555&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2555%26sid%3Dliteratum%253Aachs%26aulast%3DBurton%26aufirst%3DA.%2BJ.%26aulast%3DThomson%26aufirst%3DA.%2BR.%26aulast%3DDawson%26aufirst%3DW.%2BM.%26aulast%3DBrady%26aufirst%3DR.%2BL.%26aulast%3DWoolfson%26aufirst%3DD.%2BN.%26atitle%3DInstalling%2520Hydrolytic%2520Activity%2520into%2520a%2520Completely%2520De%2520Novo%2520Protein%2520Framework%26jtitle%3DNat.%2520Chem.%26date%3D2016%26volume%3D8%26spage%3D837%26epage%3D844%26doi%3D10.1038%2Fnchem.2555" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilvert, D.</span></span> <span> </span><span class="NLM_article-title">Design of Protein Catalysts</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">470</span>, <span class="refDoi">&nbsp;DOI: 10.1146/annurev-biochem-072611-101825</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1146%2Fannurev-biochem-072611-101825" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=23746259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOqsr3P" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=82&amp;publication_year=2013&amp;pages=447-470&amp;author=D.+Hilvert&amp;title=Design+of+Protein+Catalysts&amp;doi=10.1146%2Fannurev-biochem-072611-101825" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Design of protein catalysts</span></div><div class="casAuthors">Hilvert, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">447-470</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  Diverse engineering strategies have been developed to create enzymes with novel catalytic activities.  Among these, computational approaches hold particular promise.  Enzymes have been computationally designed to promote several nonbiol. reactions, including a Diels-Alder cycloaddn., proton transfer, multistep retroaldol transformations, and metal-dependent hydrolysis of phosphotriesters.  Although their efficiencies (kcat/KM = 0.1-100 M-1 s-1) are typically low compared with those of the best natural enzymes (106-108 M-1 s-1), these catalysts are excellent starting points for lab. evolution.  This review surveys recent progress in combining computational and evolutionary approaches to enzyme design, together with insights into enzyme function gained from studies of the engineered catalysts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtEa8xP2cUlrVg90H21EOLACvtfcHk0lgUbEOTo93Adg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOqsr3P&amp;md5=a31af0495a8aca01d71a2c8c280882d8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-072611-101825&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-072611-101825%26sid%3Dliteratum%253Aachs%26aulast%3DHilvert%26aufirst%3DD.%26atitle%3DDesign%2520of%2520Protein%2520Catalysts%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2013%26volume%3D82%26spage%3D447%26epage%3D470%26doi%3D10.1146%2Fannurev-biochem-072611-101825" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obexer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrabou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittl, P. R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilvert, D.</span></span> <span> </span><span class="NLM_article-title">Emergence of a Catalytic Tetrad During Evolution of a Highly Active Artificial Aldolase</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">50</span>– <span class="NLM_lpage">56</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.2596</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2Fnchem.2596" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=27995916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVChurbF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2017&amp;pages=50-56&amp;author=R.+Obexerauthor=A.+Godinaauthor=X.+Garrabouauthor=P.+R.+E.+Mittlauthor=D.+Bakerauthor=A.+D.+Griffithsauthor=D.+Hilvert&amp;title=Emergence+of+a+Catalytic+Tetrad+During+Evolution+of+a+Highly+Active+Artificial+Aldolase&amp;doi=10.1038%2Fnchem.2596" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Emergence of a catalytic tetrad during evolution of a highly active artificial aldolase</span></div><div class="casAuthors">Obexer, Richard; Godina, Alexei; Garrabou, Xavier; Mittl, Peer R. E.; Baker, David; Griffiths, Andrew D.; Hilvert, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-56</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Designing catalysts that achieve the rates and selectivities of natural enzymes is a long-standing goal in protein chem.  Here, the authors show that an ultrahigh-throughput droplet-based microfluidic screening platform can be used to improve a previously optimized artificial aldolase by an addnl. factor of 30 to give a &gt;109 rate enhancement that rivals the efficiency of class I aldolases.  The resulting enzyme catalyzes a reversible aldol reaction with high stereoselectivity and tolerates a broad range of substrates.  Biochem. and structural studies show that catalysis depends on a Lys-Tyr-Asn-Tyr tetrad that emerged adjacent to a computationally designed hydrophobic pocket during directed evolution.  This constellation of residues is poised to activate the substrate by Schiff base formation, promote mechanistically important proton transfers, and stabilize multiple transition states along a complex reaction coordinate.  The emergence of such a sophisticated catalytic center shows that there is nothing magical about the catalytic activities or mechanisms of naturally occurring enzymes, or the evolutionary process that gave rise to them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEGhgSJ6Hl_rVg90H21EOLACvtfcHk0livR2KhjjLpaA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XhsVChurbF&amp;md5=907ce8bfe60665728bf3b3a035e906f9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2596&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2596%26sid%3Dliteratum%253Aachs%26aulast%3DObexer%26aufirst%3DR.%26aulast%3DGodina%26aufirst%3DA.%26aulast%3DGarrabou%26aufirst%3DX.%26aulast%3DMittl%26aufirst%3DP.%2BR.%2BE.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DGriffiths%26aufirst%3DA.%2BD.%26aulast%3DHilvert%26aufirst%3DD.%26atitle%3DEmergence%2520of%2520a%2520Catalytic%2520Tetrad%2520During%2520Evolution%2520of%2520a%2520Highly%2520Active%2520Artificial%2520Aldolase%26jtitle%3DNat.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D50%26epage%3D56%26doi%3D10.1038%2Fnchem.2596" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P.
G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">From Molecular Diversity to Catalysis - Lessons from the Immune-System</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>269</i></span>,  <span class="NLM_fpage">1835</span>– <span class="NLM_lpage">1842</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.7569920</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1126%2Fscience.7569920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=7569920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK2MXotlKqt7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=269&amp;publication_year=1995&amp;pages=1835-1842&amp;author=P.%0AG.+Schultzauthor=R.+A.+Lerner&amp;title=From+Molecular+Diversity+to+Catalysis+-+Lessons+from+the+Immune-System&amp;doi=10.1126%2Fscience.7569920" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">From molecular diversity to catalysis: lessons from the immune system</span></div><div class="casAuthors">Schultz, Peter G.; Lerner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue">5232</span>),
    <span class="NLM_cas:pages">1835-42</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review, with 75 refs.  By combining the enormous mol. diversity of the immune system with basic mechanistic principles of chem., one can produce catalytic antibodies that allow control of reactions in ways heretofore not possible.  Mechanistic and structural studies of these antibodies are also providing insights into important aspects of enzymic catalysis and the evolution of catalytic function.  Moreover the ability to harness and rationally direct the immune response to generate selective catalysts for reactions ranging from pericyclic and redox reactions to cationic rearrangement reactions underscores the chem. potential of this and other large combinatorial libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8dPAe2FNgSbVg90H21EOLACvtfcHk0livR2KhjjLpaA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXotlKqt7c%253D&amp;md5=4bcfe80a46e64fb6244605a0356bda1f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1126%2Fscience.7569920&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7569920%26sid%3Dliteratum%253Aachs%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DFrom%2520Molecular%2520Diversity%2520to%2520Catalysis%2520-%2520Lessons%2520from%2520the%2520Immune-System%26jtitle%3DScience%26date%3D1995%26volume%3D269%26spage%3D1835%26epage%3D1842%26doi%3D10.1126%2Fscience.7569920" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">The Chemistry of the Antibody Molecule</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">4427</span>– <span class="NLM_lpage">4437</span>, <span class="refDoi">&nbsp;DOI: 10.1002/1521-3773(20021202)41:23&lt;4427::AID-ANIE4427&gt;3.0.CO;2-K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1002%2F1521-3773%2820021202%2941%3A23%3C4427%3A%3AAID-ANIE4427%3E3.0.CO%3B2-K" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=41&amp;publication_year=2002&amp;pages=4427-4437&amp;author=P.+G.+Schultzauthor=J.+Yinauthor=R.+A.+Lerner&amp;title=The+Chemistry+of+the+Antibody+Molecule&amp;doi=10.1002%2F1521-3773%2820021202%2941%3A23%3C4427%3A%3AAID-ANIE4427%3E3.0.CO%3B2-K" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820021202%2941%3A23%3C4427%3A%3AAID-ANIE4427%3E3.0.CO%3B2-K&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820021202%252941%253A23%253C4427%253A%253AAID-ANIE4427%253E3.0.CO%253B2-K%26sid%3Dliteratum%253Aachs%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DThe%2520Chemistry%2520of%2520the%2520Antibody%2520Molecule%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2002%26volume%3D41%26spage%3D4427%26epage%3D4437%26doi%3D10.1002%2F1521-3773%2820021202%2941%3A23%3C4427%3A%3AAID-ANIE4427%3E3.0.CO%3B2-K" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kries, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomberg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilvert, D.</span></span> <span> </span><span class="NLM_article-title">De Novo Enzymes by Computational Design</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">228</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cbpa.2013.02.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1016%2Fj.cbpa.2013.02.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=23498973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjvFCnsrY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=2013&amp;pages=221-228&amp;author=H.+Kriesauthor=R.+Blombergauthor=D.+Hilvert&amp;title=De+Novo+Enzymes+by+Computational+Design&amp;doi=10.1016%2Fj.cbpa.2013.02.012" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">De novo enzymes by computational design</span></div><div class="casAuthors">Kries, Hajo; Blomberg, Rebecca; Hilvert, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">221-228</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Computational enzyme design has emerged as a promising tool for generating made-to-order biocatalysts.  In addn. to improving the reliability of the design cycle, current efforts in this area are focusing on expanding the set of catalyzed reactions and investigating the structure and mechanism of individual designs.  Although the activities of de novo enzymes are typically low, they can be significantly increased by directed evolution.  Anal. of their evolutionary trajectories provides valuable feedback for the design algorithms and can enhance our understanding of natural protein evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBIsxuc3Rt4LVg90H21EOLACvtfcHk0livR2KhjjLpaA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXjvFCnsrY%253D&amp;md5=68e6e72cff643028e7b28ef71303b559</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2013.02.012&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2013.02.012%26sid%3Dliteratum%253Aachs%26aulast%3DKries%26aufirst%3DH.%26aulast%3DBlomberg%26aufirst%3DR.%26aulast%3DHilvert%26aufirst%3DD.%26atitle%3DDe%2520Novo%2520Enzymes%2520by%2520Computational%2520Design%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D17%26spage%3D221%26epage%3D228%26doi%3D10.1016%2Fj.cbpa.2013.02.012" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Importance of Protein Dynamics During Enzymatic C-H Bond Cleavage Catalysis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2068</span>– <span class="NLM_lpage">2077</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi301504m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi301504m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVyntrs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=2013&amp;pages=2068-2077&amp;author=J.+P.+Klinman&amp;title=Importance+of+Protein+Dynamics+During+Enzymatic+C-H+Bond+Cleavage+Catalysis&amp;doi=10.1021%2Fbi301504m" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Importance of protein dynamics during enzymatic C-H bond cleavage catalysis</span></div><div class="casAuthors">Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2068-2077</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Quantum tunneling and protein dynamics have emerged as important components of enzyme function.  Here, the author focuses on soybean lipoxygenase-1, to illustrate how the properties of enzymic C-H bond activation link protein motions to fundamental bond-making and -breaking processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDcr_ue4eoubVg90H21EOLACvtfcHk0livR2KhjjLpaA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhvVyntrs%253D&amp;md5=e157f1a61f384157c7e4a24498d08855</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fbi301504m&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi301504m%26sid%3Dliteratum%253Aachs%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DImportance%2520of%2520Protein%2520Dynamics%2520During%2520Enzymatic%2520C-H%2520Bond%2520Cleavage%2520Catalysis%26jtitle%3DBiochemistry%26date%3D2013%26volume%3D52%26spage%3D2068%26epage%3D2077%26doi%3D10.1021%2Fbi301504m" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Proton-Coupled Electron Transfer in Soybean Lipoxygenase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">5763</span>– <span class="NLM_lpage">5775</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja039606o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja039606o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjt1Ghur8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=126&amp;publication_year=2004&amp;pages=5763-5775&amp;author=E.+Hatcherauthor=A.+V.+Soudackovauthor=S.+Hammes-Schiffer&amp;title=Proton-Coupled+Electron+Transfer+in+Soybean+Lipoxygenase&amp;doi=10.1021%2Fja039606o" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Proton-Coupled Electron Transfer in Soybean Lipoxygenase</span></div><div class="casAuthors">Hatcher, Elizabeth; Soudackov, Alexander V.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5763-5775</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The proton-coupled electron transfer reaction catalyzed by soybean lipoxygenase-1 is studied with a multistate continuum theory that represents the transferring hydrogen nucleus as a quantum mech. wave function.  The inner-sphere reorganization energy of the iron cofactor is calcd. with d. functional theory, and the outer-sphere reorganization energy of the protein is calcd. with the frequency-resolved cavity model for conformations obtained with docking simulations.  Both classical and quantum mech. treatments of the proton donor-acceptor vibrational motion are presented.  The temp. dependence of the calcd. rates and kinetic isotope effects is in agreement with the exptl. data.  The weak temp. dependence of the rates is due to the relatively small free energy barrier arising from a balance between the reorganization energy and the reaction free energy.  The unusually high deuterium kinetic isotope effect of 81 is due to the small overlap of the reactant and product proton vibrational wave functions and the dominance of the lowest energy reactant and product vibronic states in the tunneling process.  The temp. dependence of the kinetic isotope effect is strongly influenced by the proton donor-acceptor distance with the dominant contribution to the overall rate.  This dominant proton donor-acceptor distance is significantly smaller than the equil. donor-acceptor distance and is detd. by a balance between the larger coupling and the smaller Boltzmann probability as the distance decreases.  Thus, the proton donor-acceptor vibrational motion plays a vital role in decreasing the dominant donor-acceptor distance relative to its equil. value to facilitate the proton-coupled electron transfer reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsrMbjRwFRkrVg90H21EOLACvtfcHk0liOJFe03l8IZg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2cXjt1Ghur8%253D&amp;md5=c3de26943ff2aa6ef66769aba01d26ca</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fja039606o&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja039606o%26sid%3Dliteratum%253Aachs%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DProton-Coupled%2520Electron%2520Transfer%2520in%2520Soybean%2520Lipoxygenase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D5763%26epage%3D5775%26doi%3D10.1021%2Fja039606o" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glickman, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Nature of Rate-Limiting Steps in the Soybean Lipoxygenase-1 Reaction</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">14077</span>– <span class="NLM_lpage">14092</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00043a013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00043a013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK2MXos1Wms7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=1995&amp;pages=14077-14092&amp;author=M.+H.+Glickmanauthor=J.+P.+Klinman&amp;title=Nature+of+Rate-Limiting+Steps+in+the+Soybean+Lipoxygenase-1+Reaction&amp;doi=10.1021%2Fbi00043a013" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Nature of Rate-Limiting Steps in the Soybean Lipoxygenase-1 Reaction</span></div><div class="casAuthors">Glickman, Michael H.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">14077-92</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of kinetic isotope effect expts. were performed with the goal of understanding the nature of rate-limiting steps in the soybean lipoxygenase-1 (SBL-1) reaction.  SBL-1 was reacted with linoleic acid (LA) and deuterated linoleic acid (D-LA) under a variety of exptl. conditions involving changes in temp., pH, viscosity, and replacement of H2O with D2O.  The extrapolated intrinsic primary H/D isotope effect can be estd. to be possibly as large as 80.  This value is probably the largest effect published for an enzymic reaction, and much larger than that predicted from semiclassical models.  Due to this large primary isotope effect, the C-D bond cleavage fully limits the rate of reaction under all conditions tested.  In the case of protonated linoleic acid, a no. of steps are partially rate-limiting at room temp.; three distinct mechanistic steps which include substrate binding, an H2O/D2O sensitive step, and C-H bond cleavage have been characterized.  Use of glucose as a solvent viscosogen demonstrates that substrate binding is approx. 48% rate-limiting for LA at 20°.  SBL-1 is one of the few enzymes that fit the definition of a "perfect enzyme", in the sense that further optimization of any single step at room temp. will not significantly increase the overall rate.  At lower temps., the step sensitive to solvent deuteration begins to dominate the reaction, whereas at higher temps., the hydrogen abstraction step is rate-limiting.  The pH dependence of kcat/Km for SBL-1 can be explained as arising from two pKa's, one controlling substrate binding and the other substrate release.  Below pH 7.8, the rate of substrate release increases, thus decreasing the commitment to catalysis and unmasking the large intrinsic isotope effect on the subsequent hydrogen abstraction.  An abnormally high pKa, in the range of 7-8, has been detd. for LA in the concn. range employed in these studies.  The authors propose that the neg. charged form of LA, predominating above pH 7.8, is the preferred substrate with larger commitments to catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH4-SiOjfKPbVg90H21EOLACvtfcHk0liOJFe03l8IZg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXos1Wms7c%253D&amp;md5=1bce6fa8366f7a36f5196031d68f0e1a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fbi00043a013&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00043a013%26sid%3Dliteratum%253Aachs%26aulast%3DGlickman%26aufirst%3DM.%2BH.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DNature%2520of%2520Rate-Limiting%2520Steps%2520in%2520the%2520Soybean%2520Lipoxygenase-1%2520Reaction%26jtitle%3DBiochemistry%26date%3D1995%26volume%3D34%26spage%3D14077%26epage%3D14092%26doi%3D10.1021%2Fbi00043a013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glickman, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Extremely Large Isotope Effects in the Soybean Lipoxygenase-Linoleic Acid Reaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">793</span>– <span class="NLM_lpage">794</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja00081a060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00081a060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK2cXnt1aksg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=116&amp;publication_year=1994&amp;pages=793-794&amp;author=M.+H.+Glickmanauthor=J.+S.+Wisemanauthor=J.+P.+Klinman&amp;title=Extremely+Large+Isotope+Effects+in+the+Soybean+Lipoxygenase-Linoleic+Acid+Reaction&amp;doi=10.1021%2Fja00081a060" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Extremely Large Isotope Effects in the Soybean Lipoxygenase-Linoleic Acid Reaction</span></div><div class="casAuthors">Glickman, Michael H.; Wiseman, Jeffrey S.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">793-4</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Very large primary deuterium isotope effects are obsd. for the oxidn. of linoleic acid by lipoxygenase.  Primary kinetic (kH/kD) isotope effects were measured both competitively and noncompetitively, with substrates synthesized by different methods and under different conditions.  At room temp. and pH 9, comparing linoleic acid to [11,11-2H2]linoleic acid shows a D(V) = 43.4 (±1.9) and D(V/K) = 27.4 (±3.8).  In all cases an upper limit of ca. 60 was obtained for the isotope effects by changing reaction conditions.  These seem to be the largest published primary (kH/kD) isotope effects in an enzymic system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJzH6GqLdVX7Vg90H21EOLACvtfcHk0liOJFe03l8IZg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2cXnt1aksg%253D%253D&amp;md5=b59c096f56bfa2d1102bebb63c5c8258</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fja00081a060&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00081a060%26sid%3Dliteratum%253Aachs%26aulast%3DGlickman%26aufirst%3DM.%2BH.%26aulast%3DWiseman%26aufirst%3DJ.%2BS.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DExtremely%2520Large%2520Isotope%2520Effects%2520in%2520the%2520Soybean%2520Lipoxygenase-Linoleic%2520Acid%2520Reaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1994%26volume%3D116%26spage%3D793%26epage%3D794%26doi%3D10.1021%2Fja00081a060" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jonsson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glickman, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Experimental Evidence for Extensive Tunneling of Hydrogen in the Lipoxygenase Reaction: Implications for Enzyme Catalysis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">10319</span>– <span class="NLM_lpage">10320</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja961827p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja961827p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK28XmtFKks7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=118&amp;publication_year=1996&amp;pages=10319-10320&amp;author=T.+Jonssonauthor=M.+H.+Glickmanauthor=S.+J.+Sunauthor=J.+P.+Klinman&amp;title=Experimental+Evidence+for+Extensive+Tunneling+of+Hydrogen+in+the+Lipoxygenase+Reaction%3A+Implications+for+Enzyme+Catalysis&amp;doi=10.1021%2Fja961827p" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental Evidence for Extensive Tunneling of Hydrogen in the Lipoxygenase Reaction: Implications for Enzyme Catalysis</span></div><div class="casAuthors">Jonsson, Thorlakur; Glickman, Michael H.; Sun, Shujun; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">10319-10320</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogen tunneling has recently has been shown to take place in the enzyme-catalyzed hydrogen transfer reactions of yeast alc. dehydrogenase, horse liver alc. dehydrogenase, bovine serum amine oxidase, and monoamine oxidase B.  In all cases, the obsd. behavior is consistent with the view of a tunneling correction to the reaction rate, such that the lighter isotopes of hydrogen undergo tunneling to a greater extent than the heavier ones.  Recent investigations of steady state kinetic isotope effects in the soybean lipoxygenase reaction have indicated the largest deuterium isotope effect seen for a biol. catalyst (kH/kD = 48 at 25°).  Extension of these measurements to single turnover conditions supports the view that the enormous rate discrimination between hydrogen and deuterium occurs in the C-H bond activation step.  Measurement of the temp. dependence for C-H(D) cleavage under both pre-steady state and steady state conditions reveals a nearly activationless process.  The combined observation of very small enthalpies of activation and very large, nearly temp.-independent isotope effects leads us to conclude that soybean lipoxygenase facilitates the loss of hydrogen from substrate near room temp. by invoking extensive tunneling.  This type of behavior has been seen with small mols., but generally only at very low temps.  A diagram is presented to encompass the observe range of enzymic behavior, in which tunneling varies from moderate to extensive.  As predicted, a correlation can be obsd. between enzymic enthalpies of activation and the magnitude of isotope effects on Arrhenius prefactors.  These results indicate that optimal catalytic efficiency in enzyme reactions requires a modulation of reaction barrier width as well as height.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru7wKNDte5FbVg90H21EOLACvtfcHk0lgs-2Zvg252ug"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK28XmtFKks7w%253D&amp;md5=2d3e621a9c0194942cab4f8ec21c67f3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fja961827p&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja961827p%26sid%3Dliteratum%253Aachs%26aulast%3DJonsson%26aufirst%3DT.%26aulast%3DGlickman%26aufirst%3DM.%2BH.%26aulast%3DSun%26aufirst%3DS.%2BJ.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DExperimental%2520Evidence%2520for%2520Extensive%2520Tunneling%2520of%2520Hydrogen%2520in%2520the%2520Lipoxygenase%2520Reaction%253A%2520Implications%2520for%2520Enzyme%2520Catalysis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D10319%26epage%3D10320%26doi%3D10.1021%2Fja961827p" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Temperature-Dependent Isotope Effects in Soybean Lipoxygenase-1: Correlating Hydrogen Tunneling with Protein Dynamics</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">3865</span>– <span class="NLM_lpage">3874</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja012205t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja012205t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD38XitVKqurg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=124&amp;publication_year=2002&amp;pages=3865-3874&amp;author=M.+J.+Knappauthor=K.+Rickertauthor=J.+P.+Klinman&amp;title=Temperature-Dependent+Isotope+Effects+in+Soybean+Lipoxygenase-1%3A+Correlating+Hydrogen+Tunneling+with+Protein+Dynamics&amp;doi=10.1021%2Fja012205t" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Temperature-Dependent Isotope Effects in Soybean Lipoxygenase-1: Correlating Hydrogen Tunneling with Protein Dynamics</span></div><div class="casAuthors">Knapp, Michael J.; Rickert, Keith; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3865-3874</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The hydrogen-atom transfer in soybean lipoxygenase-1 (SLO) exhibits a large kinetic isotope effect on kcat (KIE = 81) near room temp. and a very weak temp. dependence (Eact = 2.1 kcal/mol).  These properties are consistent with H· transfer that occurs entirely by a tunneling event.  Mutants of SLO were prepd., and the temp. dependence of the KIE was measured, to test for alterations in the tunneling behavior.  All mutants studied exhibit KIEs of similar, large magnitude at 30°, despite an up to 3 orders of magnitude change in kcat.  Eact for two of the mutants (Leu754 → Ala, Leu546 → Ala) is larger than for wild-type (WT), and the KIE becomes slightly more temp. dependent.  In contrast, Ile553 → Ala exhibits kcat and Eact parameters similar to wild-type soybean lipoxygenase-1 (WT-SLO) for protiated substrate; however, the KIE is markedly temp. dependent.  The behavior of the former two mutants could reflect increased reorganization energies (λ), but the behavior of the latter mutant is inconsistent with this description.  We have invoked a full H· tunneling model (Kuznetsov, A. M.; Ulstrup, J. Can. J. Chem. 1999, 77, 1085-1096) to explain the temp. dependence of the KIE, which is indicative of the extent to which distance sampling (gating) modulates hydrogen transfer.  WT-SLO exhibits a very small Eact and a nearly temp.-independent KIE, which was modeled as arising from a compressed hydrogen transfer distance with little modulation of the hydrogen transfer distance.  The observations on the Leu754 → Ala and Leu546 → Ala mutants were modeled as arising from a slightly less compressed active site with greater modulation of the hydrogen transfer distance by environmental dynamics.  Finally, the obsd. behavior of the Ile553 → Ala mutant indicates a relaxed active site with extensive involvement of gating to facilitate hydrogen transfer.  We conclude that WT-SLO has an active site structure that is well organized to support hydrogen tunneling and that mutations perturb structural elements that support hydrogen tunneling.  Modest alterations in active site residues increase λ and/or increase the hydrogen transfer distance, thereby affecting the probability that tunneling can occur.  These studies allow the detection and characterization of a protein-gating mode in catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVlq9ISuo2KrVg90H21EOLACvtfcHk0lgs-2Zvg252ug"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD38XitVKqurg%253D&amp;md5=e63570c9ed25cd5d141ecee52583487b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fja012205t&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja012205t%26sid%3Dliteratum%253Aachs%26aulast%3DKnapp%26aufirst%3DM.%2BJ.%26aulast%3DRickert%26aufirst%3DK.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DTemperature-Dependent%2520Isotope%2520Effects%2520in%2520Soybean%2520Lipoxygenase-1%253A%2520Correlating%2520Hydrogen%2520Tunneling%2520with%2520Protein%2520Dynamics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2002%26volume%3D124%26spage%3D3865%26epage%3D3874%26doi%3D10.1021%2Fja012205t" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Proton-Coupled Electron Transfer Reactions: Analytical Rate Constants and Case Study of Kinetic Isotope Effects in Lipoxygenase</span>. <i>Faraday Discuss.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>195</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">189</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C6FD00122J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1039%2FC6FD00122J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=27735009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsVOlsb0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=195&amp;publication_year=2016&amp;pages=171-189&amp;author=A.+V.+Soudackovauthor=S.+Hammes-Schiffer&amp;title=Proton-Coupled+Electron+Transfer+Reactions%3A+Analytical+Rate+Constants+and+Case+Study+of+Kinetic+Isotope+Effects+in+Lipoxygenase&amp;doi=10.1039%2FC6FD00122J" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Proton-coupled electron transfer reactions: analytical rate constants and case study of kinetic isotope effects in lipoxygenase</span></div><div class="casAuthors">Soudackov, Alexander V.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Faraday Discussions</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue">Reaction Rate Theory</span>),
    <span class="NLM_cas:pages">171-189</span>CODEN:
                <span class="NLM_cas:coden">FDISE6</span>;
        ISSN:<span class="NLM_cas:issn">1359-6640</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A general theory has been developed for proton-coupled electron transfer (PCET), which is vital to a wide range of chem. and biol. processes.  This theory describes PCET reactions in terms of nonadiabatic transitions between reactant and product electron-proton vibronic states and includes the effects of thermal fluctuations of the solvent or protein environment, as well as the proton donor-acceptor motion.  Within the framework of this general PCET theory, a series of anal. rate const. expressions has been derived for PCET reactions in well-defined regimes.  Herein, the application of this theory to PCET in the enzyme soybean lipoxygenase illustrates the regimes of validity for the various rate const. expressions and elucidates the fundamental phys. principles dictating PCET reactions.  Such theor. studies provide significant phys. insights that guide the interpretation of exptl. data and lead to exptl. testable predictions.  A combination of theor. treatments with at.-level simulations is essential to understanding PCET.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKY2SDFI5vRLVg90H21EOLACvtfcHk0lgs-2Zvg252ug"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XpsVOlsb0%253D&amp;md5=103341468cac8348e786883eee5b1a72</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1039%2FC6FD00122J&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6FD00122J%26sid%3Dliteratum%253Aachs%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DProton-Coupled%2520Electron%2520Transfer%2520Reactions%253A%2520Analytical%2520Rate%2520Constants%2520and%2520Case%2520Study%2520of%2520Kinetic%2520Isotope%2520Effects%2520in%2520Lipoxygenase%26jtitle%3DFaraday%2520Discuss.%26date%3D2016%26volume%3D195%26spage%3D171%26epage%3D189%26doi%3D10.1039%2FC6FD00122J" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Proton-Coupled Electron Transfer in Soybean Lipoxygenase: Dynamical Behavior and Temperature Dependence of Kinetic Isotope Effects</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">196</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja0667211</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0667211" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlaju73N" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=129&amp;publication_year=2007&amp;pages=187-196&amp;author=E.+Hatcherauthor=A.+V.+Soudackovauthor=S.+Hammes-Schiffer&amp;title=Proton-Coupled+Electron+Transfer+in+Soybean+Lipoxygenase%3A+Dynamical+Behavior+and+Temperature+Dependence+of+Kinetic+Isotope+Effects&amp;doi=10.1021%2Fja0667211" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Proton-Coupled Electron Transfer in Soybean Lipoxygenase: Dynamical Behavior and Temperature Dependence of Kinetic Isotope Effects</span></div><div class="casAuthors">Hatcher, Elizabeth; Soudackov, Alexander V.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">187-196</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dynamical behavior and the temp. dependence of the kinetic isotope effects (KIEs) are examd. for the proton-coupled electron transfer reaction catalyzed by the enzyme soybean lipoxygenase.  The calcns. are based on a vibronically nonadiabatic formulation that includes the quantum mech. effects of the active electrons and the transferring proton, as well as the motions of all atoms in the complete solvated enzyme system.  The rate const. is represented by the time integral of a probability flux correlation function that depends on the vibronic coupling and on time correlation functions of the energy gap and the proton donor-acceptor mode, which can be calcd. from classical mol. dynamics simulations of the entire system.  The dynamical behavior of the probability flux correlation function is dominated by the equil. protein and solvent motions and is not significantly influenced by the proton donor-acceptor motion.  The magnitude of the overall rate is strongly influenced by the proton donor-acceptor frequency, the vibronic coupling, and the protein/solvent reorganization energy.  The calcns. reproduce the exptl. obsd. magnitude and temp. dependence of the KIE for the soybean lipoxygenase reaction without fitting any parameters directly to the exptl. kinetic data.  The temp. dependence of the KIE is detd. predominantly by the proton donor-acceptor frequency and the distance dependence of the vibronic couplings for hydrogen and deuterium.  The ratio of the overlaps of the hydrogen and deuterium vibrational wavefunctions strongly impacts the magnitude of the KIE but does not significantly influence its temp. dependence.  For this enzyme reaction, the large magnitude of the KIE arises mainly from the dominance of tunneling between the ground vibronic states and the relatively large ratio of the overlaps between the corresponding hydrogen and deuterium vibrational wavefunctions.  The weak temp. dependence of the KIE is due in part to the dominance of the local component of the proton donor-acceptor motion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNgQiCJB5OY7Vg90H21EOLACvtfcHk0lgs-2Zvg252ug"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28Xhtlaju73N&amp;md5=63db97f6cd817adf179e0a9cc71fc3c3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fja0667211&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0667211%26sid%3Dliteratum%253Aachs%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DProton-Coupled%2520Electron%2520Transfer%2520in%2520Soybean%2520Lipoxygenase%253A%2520Dynamical%2520Behavior%2520and%2520Temperature%2520Dependence%2520of%2520Kinetic%2520Isotope%2520Effects%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D187%26epage%3D196%26doi%3D10.1021%2Fja0667211" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Impact of Distal Mutation on Hydrogen Transfer Interface and Substrate Conformation in Soybean Lipoxygenase</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">6653</span>– <span class="NLM_lpage">6660</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jp100133p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp100133p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1WrtLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=114&amp;publication_year=2010&amp;pages=6653-6660&amp;author=S.+J.+Edwardsauthor=A.+V.+Soudackovauthor=S.+Hammes-Schiffer&amp;title=Impact+of+Distal+Mutation+on+Hydrogen+Transfer+Interface+and+Substrate+Conformation+in+Soybean+Lipoxygenase&amp;doi=10.1021%2Fjp100133p" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Distal Mutation on Hydrogen Transfer Interface and Substrate Conformation in Soybean Lipoxygenase</span></div><div class="casAuthors">Edwards, Sarah J.; Soudackov, Alexander V.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6653-6660</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The impact of distal mutation on the hydrogen transfer interface properties and on the substrate mobility, conformation, and orientation in soybean lipoxygenase-1 (SLO) is examd.  SLO catalyzes a hydrogen abstraction reaction that occurs by a proton-coupled electron transfer mechanism.  Mutation of isoleucine 553 to less bulky residues has been found exptl. to increase the magnitude and temp. dependence of the kinetic isotope effect for this reaction.  This residue borders the linoleic acid substrate but is ∼15 Å from the active site iron.  In the present study, we model these exptl. data with a vibronically nonadiabatic theory and perform all-atom mol. dynamics simulations on the complete solvated wild-type and mutant enzymes.  Our calcns. indicate that the proton transfer equil. distance increases and the assocd. frequency decreases as residue 553 becomes less bulky.  The mol. dynamics simulations illustrate that this mutation impacts the mobility, geometrical conformation, and orientation of the linoleic acid within the active site.  In turn, these effects alter the proton donor-acceptor equil. distance and frequency, leading to the exptl. obsd. changes in the magnitude and temp. dependence of the kinetic isotope effect.  This study provides insight into how the effects of distal mutations may be transmitted in enzymes to ultimately impact the catalytic rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCwBTFxq69I7Vg90H21EOLACvtfcHk0ljRJigQNR63dw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXlt1WrtLw%253D&amp;md5=72b2f9a34f8c590693d305d3ea488c64</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjp100133p&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp100133p%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DS.%2BJ.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DImpact%2520of%2520Distal%2520Mutation%2520on%2520Hydrogen%2520Transfer%2520Interface%2520and%2520Substrate%2520Conformation%2520in%2520Soybean%2520Lipoxygenase%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2010%26volume%3D114%26spage%3D6653%26epage%3D6660%26doi%3D10.1021%2Fjp100133p" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Experimental Evidence for Hydrogen Tunneling When the Isotopic Arrhenius Prefactor (a(H)/a(D)) Is Unity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">17632</span>– <span class="NLM_lpage">17633</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja806354w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja806354w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=130&amp;publication_year=2008&amp;pages=17632-17633&amp;author=S.+C.+Sharmaauthor=J.+P.+Klinman&amp;title=Experimental+Evidence+for+Hydrogen+Tunneling+When+the+Isotopic+Arrhenius+Prefactor+%28a%28H%29%2Fa%28D%29%29+Is+Unity&amp;doi=10.1021%2Fja806354w" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja806354w&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja806354w%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BC.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DExperimental%2520Evidence%2520for%2520Hydrogen%2520Tunneling%2520When%2520the%2520Isotopic%2520Arrhenius%2520Prefactor%2520%2528a%2528H%2529%252Fa%2528D%2529%2529%2520Is%2520Unity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D17632%26epage%3D17633%26doi%3D10.1021%2Fja806354w" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomchick, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Enzyme Structure and Dynamics Affect Hydrogen Tunneling: The Impact of a Remote Side Chain (1553) in Soybean Lipoxygenase-1</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">1146</span>– <span class="NLM_lpage">1151</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.0710643105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1073%2Fpnas.0710643105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=18216254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhs1KmtLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=105&amp;publication_year=2008&amp;pages=1146-1151&amp;author=M.+P.+Meyerauthor=D.+R.+Tomchickauthor=J.+P.+Klinman&amp;title=Enzyme+Structure+and+Dynamics+Affect+Hydrogen+Tunneling%3A+The+Impact+of+a+Remote+Side+Chain+%281553%29+in+Soybean+Lipoxygenase-1&amp;doi=10.1073%2Fpnas.0710643105" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme structure and dynamics affect hydrogen tunneling: The impact of a remote side chain (I553) in soybean lipoxygenase-1</span></div><div class="casAuthors">Meyer, Matthew P.; Tomchick, Diana R.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1146-1151, S1146/1-S1146/12</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">This study examines the impact of a series of mutations at position 553 on the kinetic and structural properties of soybean lipoxygenase-1 (SLO-1).  The previously uncharacterized mutants reported herein are I553L, I553V, and I553G.  High-resoln. x-ray studies of these mutants, together with the earlier studied I553A, show almost no structural change in relation to the WT-enzyme.  By contrast, a progression in kinetic behavior occurs in which the decrease in the size of the side chain at position 553 leads to an increased importance of donor-acceptor distance sampling in the course of the hydrogen transfer process.  These dynamical changes in behavior are interpreted in the context of two general classes of protein motions, preorganization and reorganization, with the latter including the distance sampling modes.  The aggregate data for SLO-1 show how judicious placement of hydrophobic side chains can influence enzyme catalysis via enhanced donor-acceptor hydro-genic wave function overlap.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBbF8U0rjNi7Vg90H21EOLACvtfcHk0ljRJigQNR63dw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXhs1KmtLw%253D&amp;md5=9a5a789879adba5216bab4822f4ad9a4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0710643105&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0710643105%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DM.%2BP.%26aulast%3DTomchick%26aufirst%3DD.%2BR.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DEnzyme%2520Structure%2520and%2520Dynamics%2520Affect%2520Hydrogen%2520Tunneling%253A%2520The%2520Impact%2520of%2520a%2520Remote%2520Side%2520Chain%2520%25281553%2529%2520in%2520Soybean%2520Lipoxygenase-1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D1146%26epage%3D1151%26doi%3D10.1073%2Fpnas.0710643105" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Modeling Temperature Dependent Kinetic Isotope Effects for Hydrogen Transfer in a Series of Soybean Lipoxygenase Mutants: The Effect of Anharmonicity Upon Transfer Distance</span>. <i>Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>319</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">296</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.chemphys.2005.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1016%2Fj.chemphys.2005.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Ort7zL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=319&amp;publication_year=2005&amp;pages=283-296&amp;author=M.+P.+Meyerauthor=J.+P.+Klinman&amp;title=Modeling+Temperature+Dependent+Kinetic+Isotope+Effects+for+Hydrogen+Transfer+in+a+Series+of+Soybean+Lipoxygenase+Mutants%3A+The+Effect+of+Anharmonicity+Upon+Transfer+Distance&amp;doi=10.1016%2Fj.chemphys.2005.05.017" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling temperature dependent kinetic isotope effects for hydrogen transfer in a series of soybean lipoxygenase mutants: The effect of anharmonicity upon transfer distance</span></div><div class="casAuthors">Meyer, Matthew P.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">319</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">283-296</span>CODEN:
                <span class="NLM_cas:coden">CMPHC2</span>;
        ISSN:<span class="NLM_cas:issn">0301-0104</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Soybean lipoxygenase-1 (SLO) catalyzes the oxidn. of linoleic acid.  The rate-limiting step in this transformation is the net abstraction of the pro-S hydrogen atom from the center of the 1,5-pentadienyl moiety in linoleic acid.  The large deuterium kinetic isotope effect (KIE) for this step appears in the first order rate const. (Dkcat = 81±5 at T = 25°).  Furthermore, the KIE and the rate for protium abstraction are weakly temp. dependent (EA,D - EA,H = 0.9±0.2 kcal/mol and EA,H = 2.1±0.2 kcal/mol, resp.).  Mutations at a hydrophobic site about 13 Å from the active site Fe(III), Ile553, induce a marked temp. dependence that varies roughly in accordance with the degree to which the residue is changed in bulk from the wild type Ile.  While the temp. dependence for these mutants varies from the wild type enzyme, the magnitude of the KIE at 25° is on the same order of magnitude.  A hydrogen tunneling model [Kuznetsov, A. M., Ulstrup, J. Can. J. Chem. 77 (1999) 1085-1096] is utilized to model the KIE temp. profiles for the wild type SLO and each Ile553 mutant.  Hydrogenic wavefunctions are modeled using harmonic oscillators and Morse oscillators in order to explore the effects of anharmonicity upon computed kinetic observables used to characterize this hydrogen transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0punSHBQzTLVg90H21EOLACvtfcHk0ljRJigQNR63dw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXht1Ort7zL&amp;md5=fc46ef4a3f7b45c2c5ec1dd44ac3c2ca</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.chemphys.2005.05.017&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chemphys.2005.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DM.%2BP.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DModeling%2520Temperature%2520Dependent%2520Kinetic%2520Isotope%2520Effects%2520for%2520Hydrogen%2520Transfer%2520in%2520a%2520Series%2520of%2520Soybean%2520Lipoxygenase%2520Mutants%253A%2520The%2520Effect%2520of%2520Anharmonicity%2520Upon%2520Transfer%2520Distance%26jtitle%3DChem.%2520Phys.%26date%3D2005%26volume%3D319%26spage%3D283%26epage%3D296%26doi%3D10.1016%2Fj.chemphys.2005.05.017" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Enhanced Rigidification within a Double Mutant of Soybean Lipoxygenase Provides Experimental Support for Vibronically Nonadiabatic Proton-Coupled Electron Transfer Models</span>. <i>ACS Catal.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3569</span>– <span class="NLM_lpage">3574</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscatal.7b00688</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscatal.7b00688" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmt1Kqtrg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2017&amp;pages=3569-3574&amp;author=S.+Huauthor=A.+V.+Soudackovauthor=S.+Hammes-Schifferauthor=J.+P.+Klinman&amp;title=Enhanced+Rigidification+within+a+Double+Mutant+of+Soybean+Lipoxygenase+Provides+Experimental+Support+for+Vibronically+Nonadiabatic+Proton-Coupled+Electron+Transfer+Models&amp;doi=10.1021%2Facscatal.7b00688" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced rigidification within a double mutant of soybean lipoxygenase provides experimental support for vibronically nonadiabatic proton-coupled electron transfer models</span></div><div class="casAuthors">Hu, Shenshen; Soudackov, Alexander V.; Hammes-Schiffer, Sharon; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Catalysis</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3569-3574</span>CODEN:
                <span class="NLM_cas:coden">ACCACS</span>;
        ISSN:<span class="NLM_cas:issn">2155-5435</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Soybean lipoxygenase (SLO) is a prototype for nonadiabatic hydrogen tunneling reactions and, as such, has served as the subject of numerous theor. studies.  Here, we report a nearly temp.-independent kinetic isotope effect (KIE) with an av. KIE value of 661 ± 27 for a double mutant (DM) of SLO at 6 temps.  The data were well-reproduced within a vibronically nonadiabatic proton-coupled electron transfer model in which the active site became rigidified compared to the wild-type enzyme and single-site mutants.  A combined temp.-pressure perturbation further showed that temp.-dependent global motions within DM-SLO were more resistant to perturbation by elevated pressure.  These findings provided strong exptl. support for the model of hydrogen tunneling in SLO, where optimization of both local protein and ligand motions and distal conformational rearrangements is a prerequisite for effective proton vibrational wave function overlap between the substrate and the active site Fe cofactor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBJL8MFFez27Vg90H21EOLACvtfcHk0lhF6lNMG4tJPQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXmt1Kqtrg%253D&amp;md5=8595deec42ff84a7ab9122606914d275</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facscatal.7b00688&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscatal.7b00688%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DS.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DEnhanced%2520Rigidification%2520within%2520a%2520Double%2520Mutant%2520of%2520Soybean%2520Lipoxygenase%2520Provides%2520Experimental%2520Support%2520for%2520Vibronically%2520Nonadiabatic%2520Proton-Coupled%2520Electron%2520Transfer%2520Models%26jtitle%3DACS%2520Catal.%26date%3D2017%26volume%3D7%26spage%3D3569%26epage%3D3574%26doi%3D10.1021%2Facscatal.7b00688" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scouras, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, C. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alber, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Extremely Elevated Room-Temperature Kinetic Isotope Effects Quantify the Critical Role of Barrier Width in Enzymatic C-H Activation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">8157</span>– <span class="NLM_lpage">8160</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja502726s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja502726s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptVaht7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=136&amp;publication_year=2014&amp;pages=8157-8160&amp;author=S.+Huauthor=S.+C.+Sharmaauthor=A.+D.+Scourasauthor=A.+V.+Soudackovauthor=C.+A.+M.+Carrauthor=S.+Hammes-Schifferauthor=T.+Alberauthor=J.+P.+Klinman&amp;title=Extremely+Elevated+Room-Temperature+Kinetic+Isotope+Effects+Quantify+the+Critical+Role+of+Barrier+Width+in+Enzymatic+C-H+Activation&amp;doi=10.1021%2Fja502726s" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Extremely elevated room-temperature kinetic isotope effects quantify the critical role of barrier width in enzymatic C-H activation</span></div><div class="casAuthors">Hu, Shenshen; Sharma, Sudhir C.; Scouras, Alexander D.; Soudackov, Alexander V.; Carr, Cody A. Marcus; Hammes-Schiffer, Sharon; Alber, Tom; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8157-8160</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Soybean lipoxygenase (I) has served as a prototype for H-tunneling reactions, as a result of its unusual kinetic isotope effects (KIEs) and their temp. dependencies.  Here, using a synergy of kinetic, structural, and theor. studies, the authors show how the interplay between donor-acceptor distance and active site flexibility leads to catalytic behavior previously predicted by quantum tunneling theory.  Modification of the size of 2 hydrophobic residues by site-specific mutagenesis in I reduced the reaction rate 104-fold and was accompanied by an enormous and unprecedented room-temp. KIE.  Fitting of the kinetic data to a non-adiabatic model implicated an expansion of the active site that could not be compensated by donor-acceptor distance sampling.  A 1.7 Å resoln. x-ray structure of the double mutant further indicated an unaltered backbone conformation, almost identical side-chain conformations, and a significantly enlarged active site cavity.  These findings show the compelling property of room-temp. H-tunneling within a biol. context and demonstrate the very high sensitivity of such tunneling to barrier width.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKqfdvn6JAErVg90H21EOLACvtfcHk0lhF6lNMG4tJPQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXptVaht7Y%253D&amp;md5=8f826611721093d5b32fc8f5b847aeef</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fja502726s&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja502726s%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DS.%26aulast%3DSharma%26aufirst%3DS.%2BC.%26aulast%3DScouras%26aufirst%3DA.%2BD.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DCarr%26aufirst%3DC.%2BA.%2BM.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26aulast%3DAlber%26aufirst%3DT.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DExtremely%2520Elevated%2520Room-Temperature%2520Kinetic%2520Isotope%2520Effects%2520Quantify%2520the%2520Critical%2520Role%2520of%2520Barrier%2520Width%2520in%2520Enzymatic%2520C-H%2520Activation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D8157%26epage%3D8160%26doi%3D10.1021%2Fja502726s" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offenbacher, A. R.</span></span> <span> </span><span class="NLM_article-title">Understanding Biological Hydrogen Transfer through the Lens of Temperature Dependent Kinetic Isotope Effects</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1974</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.accounts.8b00226</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.8b00226" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsF2jtbnP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=51&amp;publication_year=2018&amp;pages=1966-1974&amp;author=J.+K.+Klinmanauthor=A.+R.+Offenbacher&amp;title=Understanding+Biological+Hydrogen+Transfer+through+the+Lens+of+Temperature+Dependent+Kinetic+Isotope+Effects&amp;doi=10.1021%2Facs.accounts.8b00226" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding Biological Hydrogen Transfer Through the Lens of Temperature Dependent Kinetic Isotope Effects</span></div><div class="casAuthors">Klinman, Judith P.; Offenbacher, Adam R.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1966-1974</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogen atom transfer (HAT) is a salient feature of many enzymic C-H cleavage mechanisms.  In systems where kinetic isolation of HAT is achieved, selective labeling of substrate with hydrogen isotopes, such as deuterium, enables the detn. of intrinsic kinetic isotope effects (KIEs).  While the magnitude of the KIE is itself informative, ultimately the size of the temp. dependence of the KIE, ΔEa = Ea(D)-Ea(H), serves as a crit., and often misinterpreted (or even ignored) descriptor of the reaction coordinate.  As will be highlighted in this Accounts, ΔEa is one of the most robust parameters to emerge from studies of enzyme catalyzed hydrogen transfer.  Kinetic parameters for C-H reactions via HAT can appear consistent with either classical "over-the-barrier" or "Bell-like tunneling correction" models.  However, neither of these models is able to explain the observation of near-zero ΔEa values with many native enzymes that increase upon extrinsic or intrinsic perturbations to function.  Instead, a full tunneling model has been developed that can account for the aggregate trends in the temp. dependence of the KIE.  This model is reminiscent of Marcus-like theory for electron tunneling, with the addnl. incorporation of an H atom donor-acceptor distance (DAD) sampling term for effective wave function overlap; the role of the latter term is manifested in the exptl. detd. ΔEa.  Three enzyme systems from this lab. that illustrate different aspects of HAT are presented: taurine dioxygenase, the dual copper β-monooxygenases, and soybean lipoxygenase (SLO).  The latter provides a particularly compelling system for understanding the properties of hydrogen tunneling, showing systematic increases in ΔEa upon redn. in the size of hydrophobic residues both proximal and distal from the active site iron cofactor.  Of note, recent ENDOR-based studies of enzyme-substrate complexes with SLO indicate an increase in DAD for mutants with increased ΔEa, observations that are inconsistent with "Bell-like correction" models.  Overall, the surmounting kinetic and biophys. evidence corroborates a multidimensional approach for understanding HAT, offering a robust mechanistic explanation for the magnitude and trends of the KIE and ΔEa.  Recent DFT and QM/MM computations on SLO are compared to the developed nonadiabatic anal. constructs, providing considerable insight into ground state structures and reactivity.  However, QM/MM is unable to readily reproduce the small ΔEa values characteristic of native enzymes.  Future theor. developments to capture these exptl. observations may necessitate a parsing of protein motions for local, substrate deuteration-sensitive modes from the larger conformational landscape, in the process providing deeper understanding of how native enzymes have evolved to transiently optimize their active site configurations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP7XMNQ6YPIbVg90H21EOLACvtfcHk0lhF6lNMG4tJPQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhsF2jtbnP&amp;md5=fa35a3eabc0c60f7131f911866f35e9d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.8b00226&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.8b00226%26sid%3Dliteratum%253Aachs%26aulast%3DKlinman%26aufirst%3DJ.%2BK.%26aulast%3DOffenbacher%26aufirst%3DA.%2BR.%26atitle%3DUnderstanding%2520Biological%2520Hydrogen%2520Transfer%2520through%2520the%2520Lens%2520of%2520Temperature%2520Dependent%2520Kinetic%2520Isotope%2520Effects%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2018%26volume%3D51%26spage%3D1966%26epage%3D1974%26doi%3D10.1021%2Facs.accounts.8b00226" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Kinetic Detection of Orthogonal Protein and Chemical Coordinates in Enzyme Catalysis: Double Mutants of Soybean Lipoxygenase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">5447</span>– <span class="NLM_lpage">5456</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.biochem.5b00374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.5b00374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFejtbzE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=54&amp;publication_year=2015&amp;pages=5447-5456&amp;author=S.+C.+Sharmaauthor=J.+P.+Klinman&amp;title=Kinetic+Detection+of+Orthogonal+Protein+and+Chemical+Coordinates+in+Enzyme+Catalysis%3A+Double+Mutants+of+Soybean+Lipoxygenase&amp;doi=10.1021%2Facs.biochem.5b00374" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Detection of Orthogonal Protein and Chemical Coordinates in Enzyme Catalysis: Double Mutants of Soybean Lipoxygenase</span></div><div class="casAuthors">Sharma, Sudhir C.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">5447-5456</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Soybean lipoxygenase-1 (SLO-1) is a paradigmatic enzyme system for studying the contribution of hydrogen tunneling to enzymic proton-coupled electron transfer processes.  In this study, the impact of pairs of double mutants on the properties of SLO-1 is presented.  Steady-state rates and their deuterium kinetic isotope effects (KIEs) have been measured for the bimol. reaction of enzyme with free substrate (kcat/Km) and compared to the unimol. rate const., kcat.  A key kinetic finding is that the competitive KIEs on the second-order rate const. (kcat/Km) are all reduced from Dkcat and, despite large changes in rate and activation parameters, remain essentially unaltered under a variety of conditions.  These data implicate a protein reaction coordinate that is orthogonal to the chem. reaction coordinate and controls the concn. of the active enzyme.  This study introduces a new means to interrogate the alteration of conformational landscapes that can occur following site-specific mutagenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwwbzEqN7MtLVg90H21EOLACvtfcHk0lit5uw8ih-Hrg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXhtFejtbzE&amp;md5=c6188bedfa5918ce1313d7e3df13e987</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.5b00374&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.5b00374%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BC.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DKinetic%2520Detection%2520of%2520Orthogonal%2520Protein%2520and%2520Chemical%2520Coordinates%2520in%2520Enzyme%2520Catalysis%253A%2520Double%2520Mutants%2520of%2520Soybean%2520Lipoxygenase%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26spage%3D5447%26epage%3D5456%26doi%3D10.1021%2Facs.biochem.5b00374" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cattin-Ortola, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munos, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Hydrostatic Pressure Studies Distinguish Global from Local Protein Motions in C-H Activation by Soybean Lipoxygenase-1</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9361</span>– <span class="NLM_lpage">9364</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.201603592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1002%2Fanie.201603592" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=55&amp;publication_year=2016&amp;pages=9361-9364&amp;author=S.+Huauthor=J.+Cattin-Ortolaauthor=J.+W.+Munosauthor=J.+P.+Klinman&amp;title=Hydrostatic+Pressure+Studies+Distinguish+Global+from+Local+Protein+Motions+in+C-H+Activation+by+Soybean+Lipoxygenase-1&amp;doi=10.1002%2Fanie.201603592" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603592&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603592%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DS.%26aulast%3DCattin-Ortola%26aufirst%3DJ.%26aulast%3DMunos%26aufirst%3DJ.%2BW.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DHydrostatic%2520Pressure%2520Studies%2520Distinguish%2520Global%2520from%2520Local%2520Protein%2520Motions%2520in%2520C-H%2520Activation%2520by%2520Soybean%2520Lipoxygenase-1%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D9361%26epage%3D9364%26doi%3D10.1002%2Fanie.201603592" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Offenbacher, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poss, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, C. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scouras, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prigozhin, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iavarone, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alber, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Hydrogen–Deuterium Exchange of Lipoxygenase Uncovers a Relationship between Distal, Solvent Exposed Protein Motions and the Thermal Activation Barrier for Catalytic Proton-Coupled Electron Tunneling</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">570</span>– <span class="NLM_lpage">579</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscentsci.7b00142</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.7b00142" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpsFShu70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2017&amp;pages=570-579&amp;author=A.+R.+Offenbacherauthor=S.+Huauthor=E.+M.+Possauthor=C.+A.+M.+Carrauthor=A.+D.+Scourasauthor=D.+M.+Prigozhinauthor=A.+T.+Iavaroneauthor=A.+Pallaauthor=T.+Alberauthor=J.+S.+Fraserauthor=J.+P.+Klinman&amp;title=Hydrogen%E2%80%93Deuterium+Exchange+of+Lipoxygenase+Uncovers+a+Relationship+between+Distal%2C+Solvent+Exposed+Protein+Motions+and+the+Thermal+Activation+Barrier+for+Catalytic+Proton-Coupled+Electron+Tunneling&amp;doi=10.1021%2Facscentsci.7b00142" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen-Deuterium Exchange of Lipoxygenase Uncovers a Relationship between Distal, Solvent Exposed Protein Motions and the Thermal Activation Barrier for Catalytic Proton-Coupled Electron Tunneling</span></div><div class="casAuthors">Offenbacher, Adam R.; Hu, Shenshen; Poss, Erin M.; Carr, Cody A. M.; Scouras, Alexander D.; Prigozhin, Daniil M.; Iavarone, Anthony T.; Palla, Ali; Alber, Tom; Fraser, James S.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">570-579</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Defining specific pathways for efficient heat transfer from protein-solvent interfaces to their active sites represents one of the compelling and timely challenges in the quest for a phys. description of the origins of enzyme catalysis.  Enzymic hydrogen tunneling reactions constitute excellent systems in which to validate exptl. approaches to this important question, given the inherent temp. independence of quantum mech. wave function overlap.  Here, we present the application of H-D-exchange coupled to mass spectrometry toward the spatial resoln. of protein motions that can be related to enzymic catalytic parameters.  Employing the proton-coupled electron transfer reaction of soybean lipoxygenase as proof of principle, we 1st corroborated the impact of active site mutation on increased local flexibility and 2nd, uncovered a solvent-exposed loop, 15-34 Å from the reactive ferric center whose temp.-dependent motions were demonstrated to mirror the enthalpic barrier for catalytic C-H bond cleavage.  A network that connects this surface loop to the active site was structurally identified and supported by changes in kinetic parameters that resulted from site-specific mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNZ9uHrEh-gbVg90H21EOLACvtfcHk0lit5uw8ih-Hrg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXpsFShu70%253D&amp;md5=8881a7c990b8ce62dc0525656f09e563</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.7b00142&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.7b00142%26sid%3Dliteratum%253Aachs%26aulast%3DOffenbacher%26aufirst%3DA.%2BR.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DPoss%26aufirst%3DE.%2BM.%26aulast%3DCarr%26aufirst%3DC.%2BA.%2BM.%26aulast%3DScouras%26aufirst%3DA.%2BD.%26aulast%3DPrigozhin%26aufirst%3DD.%2BM.%26aulast%3DIavarone%26aufirst%3DA.%2BT.%26aulast%3DPalla%26aufirst%3DA.%26aulast%3DAlber%26aufirst%3DT.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DHydrogen%25E2%2580%2593Deuterium%2520Exchange%2520of%2520Lipoxygenase%2520Uncovers%2520a%2520Relationship%2520between%2520Distal%252C%2520Solvent%2520Exposed%2520Protein%2520Motions%2520and%2520the%2520Thermal%2520Activation%2520Barrier%2520for%2520Catalytic%2520Proton-Coupled%2520Electron%2520Tunneling%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2017%26volume%3D3%26spage%3D570%26epage%3D579%26doi%3D10.1021%2Facscentsci.7b00142" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horitani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offenbacher, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, C. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeke, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutsail, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, B. M.</span></span> <span> </span><span class="NLM_article-title">C-13 Endor Spectroscopy of Lipoxygenase-Substrate Complexes Reveals the Structural Basis for C-H Activation by Tunneling</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">1984</span>– <span class="NLM_lpage">1997</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.6b11856</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b11856" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVCisrg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=139&amp;publication_year=2017&amp;pages=1984-1997&amp;author=M.+Horitaniauthor=A.+R.+Offenbacherauthor=C.+A.+M.+Carrauthor=T.+Yuauthor=V.+Hoekeauthor=G.+E.+Cutsailauthor=S.+Hammes-Schifferauthor=J.+P.+Klinmanauthor=B.+M.+Hoffman&amp;title=C-13+Endor+Spectroscopy+of+Lipoxygenase-Substrate+Complexes+Reveals+the+Structural+Basis+for+C-H+Activation+by+Tunneling&amp;doi=10.1021%2Fjacs.6b11856" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">13C ENDOR Spectroscopy of Lipoxygenase-Substrate Complexes Reveals the Structural Basis for C-H Activation by Tunneling</span></div><div class="casAuthors">Horitani, Masaki; Offenbacher, Adam R.; Carr, Cody A. Marcus; Yu, Tao; Hoeke, Veronika; Cutsail, George E., III; Hammes-Schiffer, Sharon; Klinman, Judith P.; Hoffman, Brian M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1984-1997</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In enzymic C-H activation by H-tunneling, reduced barrier width is important for efficient H-wave-function overlap during catalysis.  For native enzymes displaying non-adiabatic tunneling, the dominant reactive H donor-acceptor distance (DAD) is typically ∼2.7 Å, considerably shorter than normal van der Waals distances.  Without a ground state substrate-bound structure for the prototypical nonadiabatic tunneling system, soybean lipoxygenase (SLO), it has remained unclear whether the requisite close tunneling distance occurs through an unusual ground-state active site arrangement or by thermally sampling conformational substates.  Here, the authors introduced Mn2+ as a spin-probe surrogate for the SLO Fe-ion; x-ray diffraction showed that Mn-SLO was structurally faithful to native enzyme.  13C-ENDOR then revealed the locations of 13C10 and reactive 13C11 of linoleic acid relative to the metal; 1H-ENDOR and mol. dynamics simulations of the fully solvated SLO model using ENDOR-derived restraints gave addnl. metrical information.  The resulting 3-dimensional representation of the SLO active site ground state contained a reactive conformer with a H DAD of ∼3.1 Å, approx. van der Waals contact, plus an inactive conformer with an even longer DAD, establishing that stochastic conformational sampling was required to achieve reactive tunneling geometries.  Tunneling-impaired SLO variants showed increased DADs and variations in substrate positioning and rigidity, confirming previous kinetic and theor. predictions of such behavior.  Overall, this investigation highlighted the (1) predictive power of non-adiabatic quantum treatments of proton-coupled electron transfer in SLO and (2) sensitivity of ENDOR probes to test, detect, and corroborate kinetically predicted trends in active site reactivity, and to reveal unexpected features of active site architecture.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl7b4GGYIScbVg90H21EOLACvtfcHk0ljIognF8V20pg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhsVCisrg%253D&amp;md5=899c3e1bdbd63671367a417ad6f28c19</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b11856&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b11856%26sid%3Dliteratum%253Aachs%26aulast%3DHoritani%26aufirst%3DM.%26aulast%3DOffenbacher%26aufirst%3DA.%2BR.%26aulast%3DCarr%26aufirst%3DC.%2BA.%2BM.%26aulast%3DYu%26aufirst%3DT.%26aulast%3DHoeke%26aufirst%3DV.%26aulast%3DCutsail%26aufirst%3DG.%2BE.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26aulast%3DHoffman%26aufirst%3DB.%2BM.%26atitle%3DC-13%2520Endor%2520Spectroscopy%2520of%2520Lipoxygenase-Substrate%2520Complexes%2520Reveals%2520the%2520Structural%2520Basis%2520for%2520C-H%2520Activation%2520by%2520Tunneling%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D1984%26epage%3D1997%26doi%3D10.1021%2Fjacs.6b11856" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohen, A.</span></span> <span> </span><span class="NLM_article-title">Hydrogen Tunneling Links Protein Dynamics to Enzyme Catalysis</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">496</span>, <span class="refDoi">&nbsp;DOI: 10.1146/annurev-biochem-051710-133623</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1146%2Fannurev-biochem-051710-133623" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=23746260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOqsr3I" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=82&amp;publication_year=2013&amp;pages=471-496&amp;author=J.+P.+Klinmanauthor=A.+Kohen&amp;title=Hydrogen+Tunneling+Links+Protein+Dynamics+to+Enzyme+Catalysis&amp;doi=10.1146%2Fannurev-biochem-051710-133623" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen tunneling links protein dynamics to enzyme catalysis</span></div><div class="casAuthors">Klinman, Judith P.; Kohen, Amnon</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">471-496</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The relationship between protein dynamics and function is a subject of considerable contemporary interest.  Although protein motions are frequently obsd. during ligand binding and release steps, the contribution of protein motions to the catalysis of bond making/breaking processes is more difficult to probe and verify.  Here, we show how the quantum mech. hydrogen tunneling assocd. with enzymic C-H bond cleavage provides a unique window into the necessity of protein dynamics for achieving optimal catalysis.  Exptl. findings support a hierarchy of thermodynamically equilibrated motions that control the H-donor and -acceptor distance and active-site electrostatics, creating an ensemble of conformations suitable for H-tunneling.  A possible extension of this view to Me transfer and other catalyzed reactions is also presented.  The impact of understanding these dynamics on the conceptual framework for enzyme activity, inhibitor/drug design, and biomimetic catalyst design is likely to be substantial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJuUE_QVx_NrVg90H21EOLACvtfcHk0ljIognF8V20pg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOqsr3I&amp;md5=6e24730783d3737dc346d51c2a27d473</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-051710-133623&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-051710-133623%26sid%3Dliteratum%253Aachs%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26aulast%3DKohen%26aufirst%3DA.%26atitle%3DHydrogen%2520Tunneling%2520Links%2520Protein%2520Dynamics%2520to%2520Enzyme%2520Catalysis%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2013%26volume%3D82%26spage%3D471%26epage%3D496%26doi%3D10.1146%2Fannurev-biochem-051710-133623" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kohen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartolucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Enzyme Dynamics and Hydrogen Tunnelling in a Thermophilic Alcohol Dehydrogenase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>399</i></span>,  <span class="NLM_fpage">496</span>– <span class="NLM_lpage">499</span>, <span class="refDoi">&nbsp;DOI: 10.1038/20981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2F20981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10365965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=399&amp;publication_year=1999&amp;pages=496-499&amp;author=A.+Kohenauthor=R.+Cannioauthor=S.+Bartolucciauthor=J.+P.+Klinman&amp;title=Enzyme+Dynamics+and+Hydrogen+Tunnelling+in+a+Thermophilic+Alcohol+Dehydrogenase&amp;doi=10.1038%2F20981" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme dynamics and hydrogen tunnelling in a thermophilic alcohol dehydrogenase</span></div><div class="casAuthors">Kohen, Amnon; Cannio, Raffaele; Bartolucci, Simonetta; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">496-499</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Biol. catalysts (enzymes) speed up reactions by many orders of magnitude using fundamental phys. processes to increase chem. reactivity.  Hydrogen tunnelling has increasingly been found to contribute to enzyme reactions at room temp.  Tunnelling is the phenomenon by which a particle transfers through a reaction barrier as a result of its wave-like property.  In reactions involving small mols., the relative importance of tunnelling increases as the temp. is reduced.  We have now investigated whether hydrogen tunnelling occurs at elevated temps. in a biol. system that functions physiol. under such conditions.  Using a thermophilic alc. dehydrogenase (ADH), we find that hydrogen tunnelling makes a significant contribution at 65°C; this is analogous to previous findings with mesophilic ADH at 25°C.  Contrary to predictions for tunnelling through a rigid barrier, the tunnelling with the thermophilic ADH decreases at and below room temp.  These findings provide exptl. evidence for a role of thermally excited enzyme fluctuations in modulating enzyme-catalyzed bond cleavage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0tQQNwnXApLVg90H21EOLACvtfcHk0ljIognF8V20pg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLg%253D&amp;md5=8515907854e2ac0724ce23b545beb9a3</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2F20981&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20981%26sid%3Dliteratum%253Aachs%26aulast%3DKohen%26aufirst%3DA.%26aulast%3DCannio%26aufirst%3DR.%26aulast%3DBartolucci%26aufirst%3DS.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DEnzyme%2520Dynamics%2520and%2520Hydrogen%2520Tunnelling%2520in%2520a%2520Thermophilic%2520Alcohol%2520Dehydrogenase%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D496%26epage%3D499%26doi%3D10.1038%2F20981" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, Z. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahnson, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Impaired Protein Conformational Landscapes as Revealed in Anomalous Arrhenius Prefactors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">10520</span>– <span class="NLM_lpage">10525</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1104989108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1073%2Fpnas.1104989108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=21670258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovFentLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=108&amp;publication_year=2011&amp;pages=10520-10525&amp;author=Z.+D.+Nagelauthor=M.+Dongauthor=B.+J.+Bahnsonauthor=J.+P.+Klinman&amp;title=Impaired+Protein+Conformational+Landscapes+as+Revealed+in+Anomalous+Arrhenius+Prefactors&amp;doi=10.1073%2Fpnas.1104989108" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Impaired protein conformational landscapes as revealed in anomalous Arrhenius prefactors</span></div><div class="casAuthors">Nagel, Zachary D.; Dong, Ming; Bahnson, Brian J.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10520-10525, S10520/1-S10520/3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A growing body of data supports a role for protein motion in enzyme catalysis.  In particular, the ability of enzymes to sample catalytically relevant conformational substates has been invoked to model kinetic and spectroscopic data.  However, direct exptl. links between rapidly interconverting conformations and the chem. steps of catalysis remain rare.  We report here on the kinetic anal. and characterization of the hydride transfer step catalyzed by a series of mutant thermophilic alc. dehydrogenases (ht-ADH), presenting evidence for Arrhenius prefactor values that become enormously elevated above an expected value of approx. 1013 s-1 when the enzyme operates below its optimal temp. range.  Restoration of normal Arrhenius behavior in the ht-ADH reaction occurs at elevated temps.  A simple model, in which reduced temp. alters the ability of the ht-ADH variants to sample the catalytically relevant region of conformational space, can reproduce the available data.  These findings indicate an impaired landscape that has been generated by the combined condition of reduced temp. and mutation at a single, active-site hydrophobic side chain.  The broader implication is that optimal enzyme function requires the maintenance of a relatively smooth landscape that minimizes low energy traps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVxw5hY0gxsbVg90H21EOLACvtfcHk0lhOv9lbyc-z1Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXovFentLg%253D&amp;md5=74b719a1094871a0de2ad383a0a9dcf3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1104989108&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1104989108%26sid%3Dliteratum%253Aachs%26aulast%3DNagel%26aufirst%3DZ.%2BD.%26aulast%3DDong%26aufirst%3DM.%26aulast%3DBahnson%26aufirst%3DB.%2BJ.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DImpaired%2520Protein%2520Conformational%2520Landscapes%2520as%2520Revealed%2520in%2520Anomalous%2520Arrhenius%2520Prefactors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D10520%26epage%3D10525%26doi%3D10.1073%2Fpnas.1104989108" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohen, A.</span></span> <span> </span><span class="NLM_article-title">Extension and Limits of the Network of Coupled Motions Correlated to Hydride Transfer in Dihydrofolate Reductase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">2575</span>– <span class="NLM_lpage">2582</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja411998h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja411998h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1eitbk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=136&amp;publication_year=2014&amp;pages=2575-2582&amp;author=P.+Singhauthor=A.+Senauthor=K.+Francisauthor=A.+Kohen&amp;title=Extension+and+Limits+of+the+Network+of+Coupled+Motions+Correlated+to+Hydride+Transfer+in+Dihydrofolate+Reductase&amp;doi=10.1021%2Fja411998h" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Extension and limits of the network of coupled motions correlated to hydride transfer in dihydrofolate reductase</span></div><div class="casAuthors">Singh, Priyanka; Sen, Arundhuti; Francis, Kevin; Kohen, Amnon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2575-2582</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzyme catalysis has been studied extensively, but the role of enzyme dynamics in the catalyzed chem. conversion is still an enigma.  Dihydrofolate reductase (DHFR) is often used as a model system to assess a network of coupled motions across the protein that may affect the catalyzed chem. transformation.  Mol. dynamics simulations, quantum mech./mol. mech. studies, and bioinformatics studies have suggested the presence of a "global dynamic network" of residues in DHFR.  Earlier studies of 2 DHFR distal mutants, G121V and M42W, indicated that these residues affect the chem. step synergistically.  While this finding was in accordance with the concept of a network of functional motions across the protein, 2 residues do not constitute a network.  To better define the extent and limits of the proposed network, the current work studied 2 remote residues predicted to be part of the same network: Trp-133 and Phe-125.  The effect of mutations in these residues on the nature of the chem. step was examd. via measurements of the temp.-dependence of the intrinsic kinetic isotope effects (KIEs) and other kinetic parameters, and double mutants were used to tie the findings to Gly-121 and Met-42.  The findings indicated that residue Phe-125, which was implicated by both calcns. and bioinformatic methods, is a part of the same global dynamic network as Gly-121 and Met-42, while Trp-133, implicated only by bioinformatics, is not.  These findings extend the understanding of the proposed network and the relations between functional and genomic couplings.  Delineating that network illuminates the need to consider remote residues and protein structural dynamics in the rational design of drugs and of biomimetic catalysts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopv3iI2c9-4bVg90H21EOLACvtfcHk0lhOv9lbyc-z1Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXht1eitbk%253D&amp;md5=818cc6c8b59ac5f3804922bae67cdf4e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja411998h&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja411998h%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DP.%26aulast%3DSen%26aufirst%3DA.%26aulast%3DFrancis%26aufirst%3DK.%26aulast%3DKohen%26aufirst%3DA.%26atitle%3DExtension%2520and%2520Limits%2520of%2520the%2520Network%2520of%2520Coupled%2520Motions%2520Correlated%2520to%2520Hydride%2520Transfer%2520in%2520Dihydrofolate%2520Reductase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D2575%26epage%3D2582%26doi%3D10.1021%2Fja411998h" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Francis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohen, A.</span></span> <span> </span><span class="NLM_article-title">Protein Motions and the Activation of the Ch Bond Catalyzed by Dihydrofolate Reductase</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">24</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cbpa.2014.03.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1016%2Fj.cbpa.2014.03.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=24742825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1KqtrzP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2014&amp;pages=19-24&amp;author=K.+Francisauthor=A.+Kohen&amp;title=Protein+Motions+and+the+Activation+of+the+Ch+Bond+Catalyzed+by+Dihydrofolate+Reductase&amp;doi=10.1016%2Fj.cbpa.2014.03.009" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Protein motions and the activation of the C-H bond catalyzed by dihydrofolate reductase</span></div><div class="casAuthors">Francis, Kevin; Kohen, Amnon</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">19-24</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The role of protein motions in enzymic C-H → C transfer is an area of great contemporary debate.  An effective tool in probing such a role is the temp. dependence of the intrinsic kinetic isotope effects for the enzyme-catalyzed reaction.  The outcome of those expts. is interpreted within the context of phenomenol. Marcus-like models of hydrogen tunneling.  Here, the authors focus on recent studies of dihydrofolate reductase (DHFR) and how the role of protein motions in the catalyzed reaction has been demonstrated.  The motions in DHFR are controlled by local effects of active site residues, global effects involving remote residues across the enzyme, and appear to be preserved during the evolution of the enzyme from bacteria to humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtBmNL987ZDLVg90H21EOLACvtfcHk0lhOv9lbyc-z1Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXht1KqtrzP&amp;md5=204e9deea02d9ebdcab2d91ff37eb219</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2014.03.009&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2014.03.009%26sid%3Dliteratum%253Aachs%26aulast%3DFrancis%26aufirst%3DK.%26aulast%3DKohen%26aufirst%3DA.%26atitle%3DProtein%2520Motions%2520and%2520the%2520Activation%2520of%2520the%2520Ch%2520Bond%2520Catalyzed%2520by%2520Dihydrofolate%2520Reductase%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D19%26epage%3D24%26doi%3D10.1016%2Fj.cbpa.2014.03.009" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loveridge, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tey, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behiry, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunther, U. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crump, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allemann, R. K.</span></span> <span> </span><span class="NLM_article-title">The Role of Large-Scale Motions in Catalysis by Dihydrofolate Reductase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">20561</span>– <span class="NLM_lpage">20570</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja208844j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja208844j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVaisLnF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=133&amp;publication_year=2011&amp;pages=20561-20570&amp;author=E.+J.+Loveridgeauthor=L.+H.+Teyauthor=E.+M.+Behiryauthor=W.+M.+Dawsonauthor=R.+M.+Evansauthor=S.+B.+Whittakerauthor=U.+L.+Guntherauthor=C.+Williamsauthor=M.+P.+Crumpauthor=R.+K.+Allemann&amp;title=The+Role+of+Large-Scale+Motions+in+Catalysis+by+Dihydrofolate+Reductase&amp;doi=10.1021%2Fja208844j" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Large-Scale Motions in Catalysis by Dihydrofolate Reductase</span></div><div class="casAuthors">Loveridge, E. Joel; Tey, Lai-Hock; Behiry, Enas M.; Dawson, William M.; Evans, Rhiannon M.; Whittaker, Sara B.-M.; Gunther, Ulrich L.; Williams, Christopher; Crump, Matthew P.; Allemann, Rudolf K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">20561-20570</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dihydrofolate reductase has long been used as a model system to study the coupling of protein motions to enzymic hydride transfer.  By studying environmental effects on hydride transfer in dihydrofolate reductase (DHFR) from the cold-adapted bacterium Moritella profunda (MpDHFR) and comparing the flexibility of this enzyme to that of DHFR from Escherichia coli (EcDHFR), we demonstrate that factors that affect large-scale (i.e., long-range, but not necessarily large amplitude) protein motions have no effect on the kinetic isotope effect on hydride transfer or its temp. dependence, although the rates of the catalyzed reaction are affected.  Hydrogen/deuterium exchange studies by NMR-spectroscopy show that MpDHFR is a more flexible enzyme than EcDHFR.  NMR expts. with EcDHFR in the presence of cosolvents suggest differences in the conformational ensemble of the enzyme.  The fact that enzymes from different environmental niches and with different flexibilities display the same behavior of the kinetic isotope effect on hydride transfer strongly suggests that, while protein motions are important to generate the reaction ready conformation, an optimal conformation with the correct electrostatics and geometry for the reaction to occur, they do not influence the nature of the chem. step itself; large-scale motions do not couple directly to hydride transfer proper in DHFR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5qS2SlKFZ2rVg90H21EOLACvtfcHk0lhOv9lbyc-z1Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXhsVaisLnF&amp;md5=d97e59da403072e939b6dc97ef5955c4</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fja208844j&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja208844j%26sid%3Dliteratum%253Aachs%26aulast%3DLoveridge%26aufirst%3DE.%2BJ.%26aulast%3DTey%26aufirst%3DL.%2BH.%26aulast%3DBehiry%26aufirst%3DE.%2BM.%26aulast%3DDawson%26aufirst%3DW.%2BM.%26aulast%3DEvans%26aufirst%3DR.%2BM.%26aulast%3DWhittaker%26aufirst%3DS.%2BB.%26aulast%3DGunther%26aufirst%3DU.%2BL.%26aulast%3DWilliams%26aufirst%3DC.%26aulast%3DCrump%26aufirst%3DM.%2BP.%26aulast%3DAllemann%26aufirst%3DR.%2BK.%26atitle%3DThe%2520Role%2520of%2520Large-Scale%2520Motions%2520in%2520Catalysis%2520by%2520Dihydrofolate%2520Reductase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D20561%26epage%3D20570%26doi%3D10.1021%2Fja208844j" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Bedem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samelson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alber, T.</span></span> <span> </span><span class="NLM_article-title">Accessing Protein Conformational Ensembles Using Room-Temperature X-Ray Crystallography</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">16247</span>– <span class="NLM_lpage">16252</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1111325108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1073%2Fpnas.1111325108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=21918110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Kmtb%252FO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=108&amp;publication_year=2011&amp;pages=16247-16252&amp;author=J.+S.+Fraserauthor=H.+van+den+Bedemauthor=A.+J.+Samelsonauthor=P.+T.+Langauthor=J.+M.+Holtonauthor=N.+Echolsauthor=T.+Alber&amp;title=Accessing+Protein+Conformational+Ensembles+Using+Room-Temperature+X-Ray+Crystallography&amp;doi=10.1073%2Fpnas.1111325108" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Accessing protein conformational ensembles using room-temperature X-ray crystallography</span></div><div class="casAuthors">Fraser, James S.; van den Bedemb, H. Enry; Samelson, Avi J.; Lang, P. Therese; Holton, James M.; Echols, Nathaniel; Alber, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">16247-16252, S16247/1-S16247/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Modern protein crystal structures are based nearly exclusively on X-ray data collected at cryogenic temps. (generally 100 K).  The cooling process is thought to introduce little bias in the functional interpretation of structural results, because cryogenic temps. minimally perturb the overall protein backbone fold.  In contrast, here we show that flash cooling biases previously hidden structural ensembles in protein crystals.  By analyzing available data for 30 different proteins using new computational tools for electron-d. sampling, model refinement, and mol. packing anal., we found that crystal cryocooling remodels the conformational distributions of more than 35% of side chains and eliminates packing defects necessary for functional motions.  In the signaling switch protein, H-Ras, an allosteric network consistent with fluctuations detected in soln. by NMR was uncovered in the room-temp., but not the cryogenic, electron-d. maps.  These results expose a bias in structural databases toward smaller, overpacked, and unrealistically unique models.  Monitoring room-temp. conformational ensembles by X-ray crystallog. can reveal motions crucial for catalysis, ligand binding, and allosteric regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCOxoYI1n2J7Vg90H21EOLACvtfcHk0li8hTy_KL-3Ow"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXht1Kmtb%252FO&amp;md5=3208af982595fe975437459890bcb436</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1111325108&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1111325108%26sid%3Dliteratum%253Aachs%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3Dvan%2Bden%2BBedem%26aufirst%3DH.%26aulast%3DSamelson%26aufirst%3DA.%2BJ.%26aulast%3DLang%26aufirst%3DP.%2BT.%26aulast%3DHolton%26aufirst%3DJ.%2BM.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DAlber%26aufirst%3DT.%26atitle%3DAccessing%2520Protein%2520Conformational%2520Ensembles%2520Using%2520Room-Temperature%2520X-Ray%2520Crystallography%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D16247%26epage%3D16252%26doi%3D10.1073%2Fpnas.1111325108" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span>; <span class="NLM_string-name">Kenner, L. R.</span>; <span class="NLM_string-name">Warkentin, M.</span>; <span class="NLM_string-name">Woldeyes, R. A.</span>; <span class="NLM_string-name">Hopkins, J. B.</span>; <span class="NLM_string-name">Thompson, M. C.</span>; <span class="NLM_string-name">Brewster, A. S.</span>; <span class="NLM_string-name">Van Benschoten, A. H.</span>; <span class="NLM_string-name">Baxter, E. L.</span>; <span class="NLM_string-name">Uervirojnangkoorn, M.</span>; <span class="NLM_string-name">McPhillips, S. E.</span>; <span class="NLM_string-name">Song, J.</span>; <span class="NLM_string-name">Alonso-Mori, R.</span>; <span class="NLM_string-name">Holton, J. M.</span>; <span class="NLM_string-name">Weis, W. I.</span>; <span class="NLM_string-name">Brunger, A. T.</span>; <span class="NLM_string-name">Soltis, S. M.</span>; <span class="NLM_string-name">Lemke, H.</span>; <span class="NLM_string-name">Gonzalez, A.</span>; <span class="NLM_string-name">Sauter, N. K.</span>; <span class="NLM_string-name">Cohen, A. E.</span>; <span class="NLM_string-name">van den Bedem, H.</span>; <span class="NLM_string-name">Thorne, R. E.</span>; <span class="NLM_string-name">Fraser, J. S.</span></span> <span> </span><span class="NLM_article-title">Mapping the Conformational Landscape of a Dynamic Enzyme by Multitemperature and XFEL Crystallography</span>.  <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume">4</span>,<span class="refDoi">&nbsp;DOI: 10.7554/eLife.07574</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.7554%2FeLife.07574" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2015&amp;author=D.+A.+Keedy&amp;author=L.+R.+Kenner&amp;author=M.+Warkentin&amp;author=R.+A.+Woldeyes&amp;author=J.+B.+Hopkins&amp;author=M.+C.+Thompson&amp;author=A.+S.+Brewster&amp;author=A.+H.+Van+Benschoten&amp;author=E.+L.+Baxter&amp;author=M.+Uervirojnangkoorn&amp;author=S.+E.+McPhillips&amp;author=J.+Song&amp;author=R.+Alonso-Mori&amp;author=J.+M.+Holton&amp;author=W.+I.+Weis&amp;author=A.+T.+Brunger&amp;author=S.+M.+Soltis&amp;author=H.+Lemke&amp;author=A.+Gonzalez&amp;author=N.+K.+Sauter&amp;author=A.+E.+Cohen&amp;author=H.+van+den+Bedem&amp;author=R.+E.+Thorne&amp;author=J.+S.+Fraser&amp;title=Mapping+the+Conformational+Landscape+of+a+Dynamic+Enzyme+by+Multitemperature+and+XFEL+Crystallography&amp;doi=10.7554%2FeLife.07574" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.7554%2FeLife.07574&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.07574%26sid%3Dliteratum%253Aachs%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26atitle%3DMapping%2520the%2520Conformational%2520Landscape%2520of%2520a%2520Dynamic%2520Enzyme%2520by%2520Multitemperature%2520and%2520XFEL%2520Crystallography%26jtitle%3DeLife%26date%3D2015%26volume%3D4%26doi%3D10.7554%2FeLife.07574" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Bedem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivak, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petsko, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. S.</span></span> <span> </span><span class="NLM_article-title">Crystal Cryocooling Distorts Conformational Heterogeneity in a Model Michaelis Complex of Dhfr</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">899</span>– <span class="NLM_lpage">910</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.str.2014.04.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1016%2Fj.str.2014.04.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=24882744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslCgsb4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=2014&amp;pages=899-910&amp;author=D.+A.+Keedyauthor=H.+van+den+Bedemauthor=D.+A.+Sivakauthor=G.+A.+Petskoauthor=D.+Ringeauthor=M.+A.+Wilsonauthor=J.+S.+Fraser&amp;title=Crystal+Cryocooling+Distorts+Conformational+Heterogeneity+in+a+Model+Michaelis+Complex+of+Dhfr&amp;doi=10.1016%2Fj.str.2014.04.016" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal cryocooling distorts conformational heterogeneity in a model Michaelis complex of DHFR</span></div><div class="casAuthors">Keedy, Daniel A.; van den Bedem, Henry; Sivak, David A.; Petsko, Gregory A.; Ringe, Dagmar; Wilson, Mark A.; Fraser, James S.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">899-910</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Most macromol. X-ray structures are detd. from cryocooled crystals, but it is unclear whether cryocooling distorts functionally relevant flexibility.  Here we compare independently acquired pairs of high-resoln. data sets of a model Michaelis complex of dihydrofolate reductase (DHFR), collected by sep. groups at both room and cryogenic temps.  These data sets allow us to isolate the differences between exptl. procedures and between temps.  Our analyses of multiconformer models and time-averaged ensembles suggest that cryocooling suppresses and otherwise modifies side-chain and main-chain conformational heterogeneity, quenching dynamic contact networks.  Despite some idiosyncratic differences, most changes from room temp. to cryogenic temp. are conserved and likely reflect temp.-dependent solvent remodeling.  Both cryogenic data sets point to addnl. conformations not evident in the corresponding room temp. data sets, suggesting that cryocooling does not merely trap preexisting conformational heterogeneity.  Our results demonstrate that crystal cryocooling consistently distorts the energy landscape of DHFR, a paragon for understanding functional protein dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU-JIrWe57irVg90H21EOLACvtfcHk0li8hTy_KL-3Ow"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXpslCgsb4%253D&amp;md5=33df728ed5ba360d47dd06e46332b132</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.04.016&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.04.016%26sid%3Dliteratum%253Aachs%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3Dvan%2Bden%2BBedem%26aufirst%3DH.%26aulast%3DSivak%26aufirst%3DD.%2BA.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DM.%2BA.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26atitle%3DCrystal%2520Cryocooling%2520Distorts%2520Conformational%2520Heterogeneity%2520in%2520a%2520Model%2520Michaelis%2520Complex%2520of%2520Dhfr%26jtitle%3DStructure%26date%3D2014%26volume%3D22%26spage%3D899%26epage%3D910%26doi%3D10.1016%2Fj.str.2014.04.016" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slappendel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veldink, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vliegenthart, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aasa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malmstrom, B. G.</span></span> <span> </span><span class="NLM_article-title">Epr Spectroscopy of Soybean Lipoxygenase-1. Description and Quantification of the High-Spin Fe(Iii) Signals</span>. <i>Biochim. Biophys. Acta, Protein Struct.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>667</i></span>,  <span class="NLM_fpage">77</span>– <span class="NLM_lpage">86</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0005-2795(81)90068-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1016%2F0005-2795%2881%2990068-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=6260216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaL3MXnvFehtg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=667&amp;publication_year=1981&amp;pages=77-86&amp;author=S.+Slappendelauthor=G.+A.+Veldinkauthor=J.+F.+Vliegenthartauthor=R.+Aasaauthor=B.+G.+Malmstrom&amp;title=Epr+Spectroscopy+of+Soybean+Lipoxygenase-1.+Description+and+Quantification+of+the+High-Spin+Fe%28Iii%29+Signals&amp;doi=10.1016%2F0005-2795%2881%2990068-4" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">EPR spectroscopy of soybean lipoxygenase-1.  Description and quantification of the high-spin iron(III) signals</span></div><div class="casAuthors">Slappendel, Steven; Veldink, Gerrit A.; Vliegenthart, Johannes F. G.; Aasa, Roland; Malmstroem, Bo G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Protein Structure</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">667</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">77-86</span>CODEN:
                <span class="NLM_cas:coden">BBPTBH</span>;
        ISSN:<span class="NLM_cas:issn">0005-2795</span>.
    </div><div class="casAbstract">The interaction of soybean lipoxygenase-1 with 13-LS-hydroperoxy-9-cis,11-trans-octadecadienoic acid, the product of the enzymic dioxygenation of linoleic acid, results in the formation of either a yellow or purple enzyme, depending on the amt. of product used.  The compn. of the high-spin Fe(III) signals in the ESR spectra of both enzyme forms was studied and the amt. of EPR-visible Fe detd. by integration and simulation.  Sets of g values of the species building up the high-spin Fe(III) signal around g 6 are derived from both 3rd-order perturbation calcn. and exact numerical diagonalization of the spin Hamiltonian describing the system.  The Fe in the native, colorless enzyme is almost ESR-nondetectable.  The yellow form of the enzyme shows a complex ESR signal around g 6 which consists of contributions of ≥3 species with different ligand symmetry.  The signal corresponds to ∼75% of the total Fe content.  The g 6 signal of the purple Fe(III) enzyme corresponds roughly to the same amt. of Fe but the ratio between the different species is not the same as in the yellow enzyme.  This enzyme form also shows an addnl. g 4.3 signal with a large amplitude but a relatively low integrated intensity (∼10% of the total Fe content).  The results are consistent with the suggested mechanism of the catalytic function of Fe in lipoxygenase based on qual. ESR results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-yeBSG7_tULVg90H21EOLACvtfcHk0li8hTy_KL-3Ow"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL3MXnvFehtg%253D%253D&amp;md5=2cb37f135de8328515e280ea89bb809a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2F0005-2795%2881%2990068-4&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2795%252881%252990068-4%26sid%3Dliteratum%253Aachs%26aulast%3DSlappendel%26aufirst%3DS.%26aulast%3DVeldink%26aufirst%3DG.%2BA.%26aulast%3DVliegenthart%26aufirst%3DJ.%2BF.%26aulast%3DAasa%26aufirst%3DR.%26aulast%3DMalmstrom%26aufirst%3DB.%2BG.%26atitle%3DEpr%2520Spectroscopy%2520of%2520Soybean%2520Lipoxygenase-1.%2520Description%2520and%2520Quantification%2520of%2520the%2520High-Spin%2520Fe%2528Iii%2529%2520Signals%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Protein%2520Struct.%26date%3D1981%26volume%3D667%26spage%3D77%26epage%3D86%26doi%3D10.1016%2F0005-2795%2881%2990068-4" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chasteen, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grady, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skorey, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neden, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riendeau, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Percival, M. D.</span></span> <span> </span><span class="NLM_article-title">Characterization of the Non-Heme Iron Center of Human 5-Lipoxygenase by Electron Paramagnetic Resonance, Fluorescence, and Ultraviolet-Visible Spectroscopy: Redox Cycling between Ferrous and Ferric States</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">9763</span>– <span class="NLM_lpage">9771</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00088a031</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00088a031" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK3sXlvFyntb4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=32&amp;publication_year=1993&amp;pages=9763-9771&amp;author=N.+D.+Chasteenauthor=J.+K.+Gradyauthor=K.+I.+Skoreyauthor=K.+J.+Nedenauthor=D.+Riendeauauthor=M.+D.+Percival&amp;title=Characterization+of+the+Non-Heme+Iron+Center+of+Human+5-Lipoxygenase+by+Electron+Paramagnetic+Resonance%2C+Fluorescence%2C+and+Ultraviolet-Visible+Spectroscopy%3A+Redox+Cycling+between+Ferrous+and+Ferric+States&amp;doi=10.1021%2Fbi00088a031" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the non-heme iron center of human 5-lipoxygenase by electron paramagnetic resonance, fluorescence, and ultraviolet-visible spectroscopy: Redox cycling between ferrous and ferric states</span></div><div class="casAuthors">Chasteen, N. Dennis; Grady, John K.; Skorey, Kathryn I.; Neden, Kevin J.; Riendeau, Denis; Percival, M. David</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">9763-71</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Purified human 5-lipoxygenase, a non-heme iron contg. enzyme, has been characterized by ESR, UV-visible and fluorescence spectroscopy.  As isolated, the enzyme is largely in the ferrous state and shows a weak X-band EPR signal extending from 0 to 700 G at 15 K, tentatively ascribed to integer spin Fe(II).  Titrn. of the protein with 13-HPOD (13-hydroperoxyoctadecadienoic acid) generates a strong multicomponent EPR signal in the g' ≈ 6 region, a yellow color assocd. with an increased absorption between 310 and 450 nm (ε330nm = 2400 M-1 cm-), and a 17% decrease in the intrinsic protein fluorescence.  The multiple component nature of the g' ≈ 6 signal indicates that the metal center in its oxidized state exists in more than one but related forms.  The g' ≈ 6 EPR signal and the yellow color reach a max. when approx. 1 mol of 13-HPOD is added/mol of iron; the resultant EPR spectrum accounts quant. for all of the iron in the protein with a signal at g' = 4.3 representing less than 3% of the total iron in the majority of samples.  Addn. of a hydroxyurea reducing agent abolished the g' ≈ 6 signal and yellow color of the protein and also reversed the decrease in fluorescence caused by the oxidant 13-HPOD.  The results indicate that the g' ≈ 6 EPR signal, the yellow color, and the decreased fluorescence are assocd. with the formation of the Fe(III) form of the enzyme.  The spectroscopic data demonstrate that the Fe(III) center of 5-lipoxygenase can be redox-cycled, consistent with its postulated role in the mechanism of oxidn. of unsatd. fatty acids to hydroperoxides.  Studies using the irreversible inhibitor 4-nitrocatechol, which binds to the ferric form of the enzyme, are also consistent with the redox cycling of the iron and demonstrate that the oxidn. state of the enzyme influences both its functional and phys. properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXVHQPqeif2rVg90H21EOLACvtfcHk0lhUAoa_lyHm6w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3sXlvFyntb4%253D&amp;md5=f1dab32ec0534608700a512fda59b3d5</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fbi00088a031&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00088a031%26sid%3Dliteratum%253Aachs%26aulast%3DChasteen%26aufirst%3DN.%2BD.%26aulast%3DGrady%26aufirst%3DJ.%2BK.%26aulast%3DSkorey%26aufirst%3DK.%2BI.%26aulast%3DNeden%26aufirst%3DK.%2BJ.%26aulast%3DRiendeau%26aufirst%3DD.%26aulast%3DPercival%26aufirst%3DM.%2BD.%26atitle%3DCharacterization%2520of%2520the%2520Non-Heme%2520Iron%2520Center%2520of%2520Human%25205-Lipoxygenase%2520by%2520Electron%2520Paramagnetic%2520Resonance%252C%2520Fluorescence%252C%2520and%2520Ultraviolet-Visible%2520Spectroscopy%253A%2520Redox%2520Cycling%2520between%2520Ferrous%2520and%2520Ferric%2520States%26jtitle%3DBiochemistry%26date%3D1993%26volume%3D32%26spage%3D9763%26epage%3D9771%26doi%3D10.1021%2Fbi00088a031" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mavrophilipos, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, K. S.</span></span> <span> </span><span class="NLM_article-title">Access of Ligands to the Ferric Center in Lipoxygenase-1</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">773</span>– <span class="NLM_lpage">783</span>, <span class="refDoi">&nbsp;DOI: 10.1016/S0006-3495(93)81438-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1016%2FS0006-3495%2893%2981438-3" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=64&amp;publication_year=1993&amp;pages=773-783&amp;author=B.+J.+Gaffneyauthor=D.+V.+Mavrophiliposauthor=K.+S.+Doctor&amp;title=Access+of+Ligands+to+the+Ferric+Center+in+Lipoxygenase-1&amp;doi=10.1016%2FS0006-3495%2893%2981438-3" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2FS0006-3495%2893%2981438-3&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-3495%252893%252981438-3%26sid%3Dliteratum%253Aachs%26aulast%3DGaffney%26aufirst%3DB.%2BJ.%26aulast%3DMavrophilipos%26aufirst%3DD.%2BV.%26aulast%3DDoctor%26aufirst%3DK.%2BS.%26atitle%3DAccess%2520of%2520Ligands%2520to%2520the%2520Ferric%2520Center%2520in%2520Lipoxygenase-1%26jtitle%3DBiophys.%2520J.%26date%3D1993%26volume%3D64%26spage%3D773%26epage%3D783%26doi%3D10.1016%2FS0006-3495%2893%2981438-3" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, Q.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavel, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigal, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, E. I.</span></span> <span> </span><span class="NLM_article-title">Epr Definition of the Non-Heme Ferric Active Sites of Mammalian 15-Lipoxygenases: Major Spectral Differences Relative to Human 5-Lipoxygenases and Plant Lipoxygenases and Their Ligand Field Origin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">7422</span>– <span class="NLM_lpage">7427</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja00133a015</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00133a015" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=117&amp;publication_year=1995&amp;pages=7422-7427&amp;author=Y.+Zhangauthor=Q.-F.+Ganauthor=E.+G.+Pavelauthor=E.+Sigalauthor=E.+I.+Solomon&amp;title=Epr+Definition+of+the+Non-Heme+Ferric+Active+Sites+of+Mammalian+15-Lipoxygenases%3A+Major+Spectral+Differences+Relative+to+Human+5-Lipoxygenases+and+Plant+Lipoxygenases+and+Their+Ligand+Field+Origin&amp;doi=10.1021%2Fja00133a015" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fja00133a015&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00133a015%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DGan%26aufirst%3DQ.-F.%26aulast%3DPavel%26aufirst%3DE.%2BG.%26aulast%3DSigal%26aufirst%3DE.%26aulast%3DSolomon%26aufirst%3DE.%2BI.%26atitle%3DEpr%2520Definition%2520of%2520the%2520Non-Heme%2520Ferric%2520Active%2520Sites%2520of%2520Mammalian%252015-Lipoxygenases%253A%2520Major%2520Spectral%2520Differences%2520Relative%2520to%2520Human%25205-Lipoxygenases%2520and%2520Plant%2520Lipoxygenases%2520and%2520Their%2520Ligand%2520Field%2520Origin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1995%26volume%3D117%26spage%3D7422%26epage%3D7427%26doi%3D10.1021%2Fja00133a015" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowling, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarrow, R. C.</span></span> <span> </span><span class="NLM_article-title">Structure and Kinetics of Formation of Catechol Complexes of Ferric Soybean Lipoxygenase-1</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">15219</span>– <span class="NLM_lpage">15229</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00046a030</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00046a030" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK2MXovFOjtbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=1995&amp;pages=15219-15229&amp;author=M.+J.+Nelsonauthor=B.+A.+Brennanauthor=D.+B.+Chaseauthor=R.+A.+Cowlingauthor=G.+N.+Groveauthor=R.+C.+Scarrow&amp;title=Structure+and+Kinetics+of+Formation+of+Catechol+Complexes+of+Ferric+Soybean+Lipoxygenase-1&amp;doi=10.1021%2Fbi00046a030" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Kinetics of Formation of Catechol Complexes of Ferric Soybean Lipoxygenase-1</span></div><div class="casAuthors">Nelson, Mark J.; Brennan, Bridget A.; Chase, D. Bruce; Cowling, Rebecca A.; Grove, Geoffrey N.; Scarrow, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">15219-29</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ferric soybean lipoxygenase forms stable complexes with 4-substituted catechols.  The structure of the complex between the enzyme and 3,4-dihydroxybenzonitrile has been studied by resonance Raman, ESR, visible, and x-ray spectroscopies.  It is a bidentate iron-catecholate complex with at least one water ligand.  The kinetics of formation of complexes between lipoxygenase and 3,4-dihydroxybenzonitrile and 3,4-dihydroxyacetophenone have been studied by stopped-flow spectroscopy.  The data are consistent with two kinetically distinct, reversible steps.  The pH dependence of the first step suggests that the substrate for the reaction is the catechol monoanion.  When these results are combined, plausible mechanisms for the complexation reaction are suggested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF6XlDfC4WJrVg90H21EOLACvtfcHk0lhUAoa_lyHm6w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXovFOjtbc%253D&amp;md5=87c812d60f08b04d97e80f7833b9acb9</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fbi00046a030&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00046a030%26sid%3Dliteratum%253Aachs%26aulast%3DNelson%26aufirst%3DM.%2BJ.%26aulast%3DBrennan%26aufirst%3DB.%2BA.%26aulast%3DChase%26aufirst%3DD.%2BB.%26aulast%3DCowling%26aufirst%3DR.%2BA.%26aulast%3DGrove%26aufirst%3DG.%2BN.%26aulast%3DScarrow%26aufirst%3DR.%2BC.%26atitle%3DStructure%2520and%2520Kinetics%2520of%2520Formation%2520of%2520Catechol%2520Complexes%2520of%2520Ferric%2520Soybean%2520Lipoxygenase-1%26jtitle%3DBiochemistry%26date%3D1995%26volume%3D34%26spage%3D15219%26epage%3D15229%26doi%3D10.1021%2Fbi00046a030" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pham, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jankun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skrzypczak-Jankun, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flowers, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, M. O.</span></span> <span> </span><span class="NLM_article-title">Structural and Thermochemical Characterization of Lipoxygenase-Catechol Complexes</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">17952</span>– <span class="NLM_lpage">17957</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi981989t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi981989t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK1cXns1Ckt7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=37&amp;publication_year=1998&amp;pages=17952-17957&amp;author=C.+Phamauthor=J.+Jankunauthor=E.+Skrzypczak-Jankunauthor=R.+A.+Flowersauthor=M.+O.+Funk&amp;title=Structural+and+Thermochemical+Characterization+of+Lipoxygenase-Catechol+Complexes&amp;doi=10.1021%2Fbi981989t" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Thermochemical Characterization of Lipoxygenase-Catechol Complexes</span></div><div class="casAuthors">Pham, Chau; Jankun, Jerzy; Skrzypczak-Jankun, Ewa; Flowers, Robert A., II; Funk, Max O. Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">17952-17957</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A complex between native, iron(II) soybean lipoxygenase 3 and 4-nitrocatechol, a known inhibitor of the enzyme, has been detected by isothermal titrn. calorimetry and characterized by X-ray crystallog.  The compd. moors in the central cavity of the protein close to the essential iron atom, but not in a bonding arrangement with it.  The iron ligands experience a significant rearrangement upon formation of the complex relative to their positions in the native enzyme; a water mol. becomes bound to iron in the complex, and one histidine ligand moves away from the iron to become involved in a hydrogen bonding interaction with the catechol.  These changes in position result in a trigonal pyramid coordination geometry for iron in the complex.  Mol. modeling and force field calcns. predict more than one stable complex between 4-nitrocatechol and the central cavity of lipoxygenase 3, but the interaction having the small mol. in the same orientation as the one found in the crystal structure was the most favorable.  These observations reveal specific details of the interaction between lipoxygenase and a small mol. and raise the possibility that changes in the ligand environment of the iron atom could be a feature of the product activation reaction or the catalytic mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHfShk3q3eKbVg90H21EOLACvtfcHk0lgr6_abkkXjxg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1cXns1Ckt7w%253D&amp;md5=56a972ac677763307d6cc4cb4dc4bf0b</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fbi981989t&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi981989t%26sid%3Dliteratum%253Aachs%26aulast%3DPham%26aufirst%3DC.%26aulast%3DJankun%26aufirst%3DJ.%26aulast%3DSkrzypczak-Jankun%26aufirst%3DE.%26aulast%3DFlowers%26aufirst%3DR.%2BA.%26aulast%3DFunk%26aufirst%3DM.%2BO.%26atitle%3DStructural%2520and%2520Thermochemical%2520Characterization%2520of%2520Lipoxygenase-Catechol%2520Complexes%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D17952%26epage%3D17957%26doi%3D10.1021%2Fbi981989t" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, M. J.</span></span> <span> </span><span class="NLM_article-title">Catecholate Complexes of Ferric Soybean Lipoxygenase-1</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">4273</span>– <span class="NLM_lpage">4278</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00412a011</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00412a011" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaL1cXktF2gs7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=27&amp;publication_year=1988&amp;pages=4273-4278&amp;author=M.+J.+Nelson&amp;title=Catecholate+Complexes+of+Ferric+Soybean+Lipoxygenase-1&amp;doi=10.1021%2Fbi00412a011" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Catecholate complexes of ferric soybean lipoxygenase 1</span></div><div class="casAuthors">Nelson, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4273-8</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Lipoxygenases are non-heme Fe-contg. enzymes that catalyze the peroxidn. of unsatd. fatty acids contg. 1,4-dienes to yield 1,3-diene-5-hydroperoxides.  The mechanism is thought to include a ferrous-bis(allyl) radical complex as an intermediate.  The complexes formed between ferric soybean lipoxygenase 1 and a series of 4-substituted catechols were used to study the Fe environment.  At pH 7, substituted catechols with sufficiently electron-withdrawing substituents (e.g., 3,4-dihydroxybenzonitrile, 3,4-dihydroxybenzaldehyde) formed stable complexes that were best viewed as Fe3+-catechol2- on the basis of the positions and no. of charge-transfer bands in their visible spectra.  Catecholes with less electron-withdrawing substituents (e.g., catechol, 2,3-dihydroxynaphthalene) efficiently reduced the Fe.  More of the substituted catechols were competent to reduce the Fe at pH 8, and at pH 9 all of the catechols tested reduced ferric lipoxygenase.  Ests. of the pH-dependent oxidn. potentials of the catechols suggested that the redn. potential of Fe(III) in lipoxygenase was approx. +0.6 V (relative to the normal H electrode) at pH 7-9.  These catechols were slow-binding inhibitors of lipoxygenase at pH 7, with final Ki values of 6-100 μM.  The magnitude of Ki did not correlate with the electron-withdrawing character of the substituents; presumably, other effects, such as charge and polarity, are more important.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdc5KXci4es7Vg90H21EOLACvtfcHk0lgr6_abkkXjxg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL1cXktF2gs7g%253D&amp;md5=e5105f78692146d428acb7eca9a4c310</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fbi00412a011&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00412a011%26sid%3Dliteratum%253Aachs%26aulast%3DNelson%26aufirst%3DM.%2BJ.%26atitle%3DCatecholate%2520Complexes%2520of%2520Ferric%2520Soybean%2520Lipoxygenase-1%26jtitle%3DBiochemistry%26date%3D1988%26volume%3D27%26spage%3D4273%26epage%3D4278%26doi%3D10.1021%2Fbi00412a011" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kemal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis-Flamberg, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krupinski-Olsen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorter, A. L.</span></span> <span> </span><span class="NLM_article-title">Reductive Inactivation of Soybean Lipoxygenase 1 by Catechols: A Possible Mechanism for Regulation of Lipoxygenase Activity</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">7064</span>– <span class="NLM_lpage">7072</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00396a031</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00396a031" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaL2sXlvFeqsLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=1987&amp;pages=7064-7072&amp;author=C.+Kemalauthor=P.+Louis-Flambergauthor=R.+Krupinski-Olsenauthor=A.+L.+Shorter&amp;title=Reductive+Inactivation+of+Soybean+Lipoxygenase+1+by+Catechols%3A+A+Possible+Mechanism+for+Regulation+of+Lipoxygenase+Activity&amp;doi=10.1021%2Fbi00396a031" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Reductive inactivation of soybean lipoxygenase 1 by catechols:  a possible mechanism for regulation of lipoxygenase activity</span></div><div class="casAuthors">Kemal, Cemal; Louis-Flamberg, Pearl; Krupinski-Olsen, Rosita; Shorter, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7064-72</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Nordihydroguaiaretic acid (NDGA), one of the most efficient inhibitors of lipoxygenases, is shown, by EPR, CD, and fluorescence studies, to reduce the catalytically active ferric soybean lipoxygenase 1 (Eox) to the inactive ferrous form (Ered).  In decreasing order of reactivity, the following also reduced Eox: catechol &gt; hydroquinone &gt; esculetin ≈ norepinephrine &gt; 2,6-di-tert-butyl-4-methylphenol &gt; caffeic acid &gt; α-tocopherol &gt; dithiothreitol.  The redn. of Eox by NDGA (k = 8.1 × 106 M-1 s-1, pH 9.0, 25°) was almost as fast as the Eox-catalyzed conversion of linoleate (LH) to 13(S)-hydroperoxy-9(Z),11(E)-octadecadienoate (LOOH) and the oxidn. of Ered by LOOH to give Eox.  Thus, NDGA can efficiently inhibit the Eox-catalyzed conversion of LH to LOOH by reducing Eox to the inactive Ered, thereby diminishing the turnover rate.  Lipoxygenase catalyzed the oxidn. of NDGA by LOOH at a rate that was consistent with the independently detd. rate const. for the redn. of Eox by NDGA.  All 4 reducing equiv from the 2 catechol groups in NDGA could be utilized in the redn. of Eox, leading to the consumption of 4 mol LOOH/mol NDGA initially present.  Because the catalytically inactive Ered was oxidized by fatty acid hydroperoxides (e.g., LOOH) to give the active Eox, reducing agents such as NDGA were most effective as lipoxygenase inhibitors at low hydroperoxide concns.  The results suggested that in vivo, where lipid hydroperoxides are maintained at low steady-state levels, redn. of lipoxygenase from the ferric to ferrous state may be important in the regulation of lipoxygenase activity and hence leukotriene biosynthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9wPoiujbK87Vg90H21EOLACvtfcHk0lgr6_abkkXjxg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL2sXlvFeqsLw%253D&amp;md5=fdf018c23827865699dfc617044d7e54</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fbi00396a031&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00396a031%26sid%3Dliteratum%253Aachs%26aulast%3DKemal%26aufirst%3DC.%26aulast%3DLouis-Flamberg%26aufirst%3DP.%26aulast%3DKrupinski-Olsen%26aufirst%3DR.%26aulast%3DShorter%26aufirst%3DA.%2BL.%26atitle%3DReductive%2520Inactivation%2520of%2520Soybean%2520Lipoxygenase%25201%2520by%2520Catechols%253A%2520A%2520Possible%2520Mechanism%2520for%2520Regulation%2520of%2520Lipoxygenase%2520Activity%26jtitle%3DBiochemistry%26date%3D1987%26volume%3D26%26spage%3D7064%26epage%3D7072%26doi%3D10.1021%2Fbi00396a031" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holman, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, E. I.</span></span> <span> </span><span class="NLM_article-title">Spectroscopic and Functional Characterization of a Ligand Coordination Mutant of Soybean Lipoxygenase-1: First Coordination Sphere Analogue of Human 15-Lipoxygenase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">12564</span>– <span class="NLM_lpage">12572</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja982844c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja982844c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=120&amp;publication_year=1998&amp;pages=12564-12572&amp;author=T.+R.+Holmanauthor=J.+Zhouauthor=E.+I.+Solomon&amp;title=Spectroscopic+and+Functional+Characterization+of+a+Ligand+Coordination+Mutant+of+Soybean+Lipoxygenase-1%3A+First+Coordination+Sphere+Analogue+of+Human+15-Lipoxygenase&amp;doi=10.1021%2Fja982844c" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fja982844c&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja982844c%26sid%3Dliteratum%253Aachs%26aulast%3DHolman%26aufirst%3DT.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DSolomon%26aufirst%3DE.%2BI.%26atitle%3DSpectroscopic%2520and%2520Functional%2520Characterization%2520of%2520a%2520Ligand%2520Coordination%2520Mutant%2520of%2520Soybean%2520Lipoxygenase-1%253A%2520First%2520Coordination%2520Sphere%2520Analogue%2520of%2520Human%252015-Lipoxygenase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D12564%26epage%3D12572%26doi%3D10.1021%2Fja982844c" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bellissent-Funel, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassanali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Havenith, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henchman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterpone, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A. E.</span></span> <span> </span><span class="NLM_article-title">Water Determines the Structure and Dynamics of Proteins</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">7673</span>– <span class="NLM_lpage">97</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.chemrev.5b00664</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00664" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFeqsrw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=116&amp;publication_year=2016&amp;pages=7673-97&amp;author=M.+C.+Bellissent-Funelauthor=A.+Hassanaliauthor=M.+Havenithauthor=R.+Henchmanauthor=P.+Pohlauthor=F.+Sterponeauthor=D.+van+der+Spoelauthor=Y.+Xuauthor=A.+E.+Garcia&amp;title=Water+Determines+the+Structure+and+Dynamics+of+Proteins&amp;doi=10.1021%2Facs.chemrev.5b00664" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Water Determines the Structure and Dynamics of Proteins</span></div><div class="casAuthors">Bellissent-Funel, Marie-Claire; Hassanali, Ali; Havenith, Martina; Henchman, Richard; Pohl, Peter; Sterpone, Fabio; van der Spoel, David; Xu, Yao; Garcia, Angel E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">7673-7697</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water is an essential participant in the stability, structure, dynamics, and function of proteins and other biomols.  Thermodynamically, changes in the aq. environment affect the stability of biomols.  Structurally, water participates chem. in the catalytic function of proteins and nucleic acids and phys. in the collapse of the protein chain during folding through hydrophobic collapse and mediates binding through the hydrogen bond in complex formation.  Water is a partner that slaves the dynamics of proteins, and water interaction with proteins affect their dynamics.  Here we provide a review of the exptl. and computational advances over the past decade in understanding the role of water in the dynamics, structure, and function of proteins.  We focus on the combination of X-ray and neutron crystallog., NMR, terahertz spectroscopy, mass spectroscopy, thermodn., and computer simulations to reveal how water assist proteins in their function.  The recent advances in computer simulations and the enhanced sensitivity of exptl. tools promise major advances in the understanding of protein dynamics, and water surely will be a protagonist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5TmeeJWenT7Vg90H21EOLACvtfcHk0lgUkbVu0et1dw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XnvFeqsrw%253D&amp;md5=365764ba66b0f0b2f94d1d401ee4d691</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00664&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00664%26sid%3Dliteratum%253Aachs%26aulast%3DBellissent-Funel%26aufirst%3DM.%2BC.%26aulast%3DHassanali%26aufirst%3DA.%26aulast%3DHavenith%26aufirst%3DM.%26aulast%3DHenchman%26aufirst%3DR.%26aulast%3DPohl%26aufirst%3DP.%26aulast%3DSterpone%26aufirst%3DF.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DGarcia%26aufirst%3DA.%2BE.%26atitle%3DWater%2520Determines%2520the%2520Structure%2520and%2520Dynamics%2520of%2520Proteins%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D7673%26epage%3D97%26doi%3D10.1021%2Facs.chemrev.5b00664" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abad, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camps, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabrias, G.</span></span> <span> </span><span class="NLM_article-title">Is Hydrogen Tunneling Involved in Acylcoa Desaturase Reactions? The Case of a Delta(9) Desaturase That Transforms (E)-11-Tetradecenoic Acid into (Z,E)-9,11-Tetradecadienoic Acid</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">3279</span>– <span class="NLM_lpage">3281</span>, <span class="refDoi">&nbsp;DOI: 10.1002/1521-3773(20000915)39:18&lt;3279::AID-ANIE3279&gt;3.0.CO;2-G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1002%2F1521-3773%2820000915%2939%3A18%3C3279%3A%3AAID-ANIE3279%3E3.0.CO%3B2-G" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=2000&amp;pages=3279-3281&amp;author=J.+L.+Abadauthor=F.+Campsauthor=G.+Fabrias&amp;title=Is+Hydrogen+Tunneling+Involved+in+Acylcoa+Desaturase+Reactions%3F+The+Case+of+a+Delta%289%29+Desaturase+That+Transforms+%28E%29-11-Tetradecenoic+Acid+into+%28Z%2CE%29-9%2C11-Tetradecadienoic+Acid&amp;doi=10.1002%2F1521-3773%2820000915%2939%3A18%3C3279%3A%3AAID-ANIE3279%3E3.0.CO%3B2-G" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820000915%2939%3A18%3C3279%3A%3AAID-ANIE3279%3E3.0.CO%3B2-G&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820000915%252939%253A18%253C3279%253A%253AAID-ANIE3279%253E3.0.CO%253B2-G%26sid%3Dliteratum%253Aachs%26aulast%3DAbad%26aufirst%3DJ.%2BL.%26aulast%3DCamps%26aufirst%3DF.%26aulast%3DFabrias%26aufirst%3DG.%26atitle%3DIs%2520Hydrogen%2520Tunneling%2520Involved%2520in%2520Acylcoa%2520Desaturase%2520Reactions%253F%2520The%2520Case%2520of%2520a%2520Delta%25289%2529%2520Desaturase%2520That%2520Transforms%2520%2528E%2529-11-Tetradecenoic%2520Acid%2520into%2520%2528Z%252CE%2529-9%252C11-Tetradecadienoic%2520Acid%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2000%26volume%3D39%26spage%3D3279%26epage%3D3281%26doi%3D10.1002%2F1521-3773%2820000915%2939%3A18%3C3279%3A%3AAID-ANIE3279%3E3.0.CO%3B2-G" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biela, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasief, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Dissecting the Hydrophobic Effect on the Molecular Level: The Role of Water, Enthalpy, and Entropy in Ligand Binding to Thermolysin</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1822</span>– <span class="NLM_lpage">1828</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.201208561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1002%2Fanie.201208561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFSitw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=2013&amp;pages=1822-1828&amp;author=A.+Bielaauthor=N.+N.+Nasiefauthor=M.+Betzauthor=A.+Heineauthor=D.+Hangauerauthor=G.+Klebe&amp;title=Dissecting+the+Hydrophobic+Effect+on+the+Molecular+Level%3A+The+Role+of+Water%2C+Enthalpy%2C+and+Entropy+in+Ligand+Binding+to+Thermolysin&amp;doi=10.1002%2Fanie.201208561" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Dissecting the Hydrophobic Effect on the Molecular Level: The Role of Water, Enthalpy, and Entropy in Ligand Binding to Thermolysin</span></div><div class="casAuthors">Biela, Adam; Nasief, Nader N.; Betz, Michael; Heine, Andreas; Hangauer, David; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1822-1828</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH &amp; Co. KGaA</span>)
        </div><div class="casAbstract">The hydrophobic effect is assocd. with the successive replacement of water mols. in the binding site of a protein by hydrophobic groups of the ligand.  Although the hydrophobic effect is assumed to be entropy-driven, large changes in enthalpy and entropy are obsd. with the model system thermolysin.  Structural changes in the binding features of the water mols. ultimately det. the thermodn. of the hydrophobic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYVn8gd7ufubVg90H21EOLACvtfcHk0lgUkbVu0et1dw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXjtFSitw%253D%253D&amp;md5=d9fa6a9a10b9f4ed78e68436bfc50c45</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fanie.201208561&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201208561%26sid%3Dliteratum%253Aachs%26aulast%3DBiela%26aufirst%3DA.%26aulast%3DNasief%26aufirst%3DN.%2BN.%26aulast%3DBetz%26aufirst%3DM.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DHangauer%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DDissecting%2520the%2520Hydrophobic%2520Effect%2520on%2520the%2520Molecular%2520Level%253A%2520The%2520Role%2520of%2520Water%252C%2520Enthalpy%252C%2520and%2520Entropy%2520in%2520Ligand%2520Binding%2520to%2520Thermolysin%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D1822%26epage%3D1828%26doi%3D10.1002%2Fanie.201208561" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mecinovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moustakas, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, S. W.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mack, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockett, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heroux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitesides, G. M.</span></span> <span> </span><span class="NLM_article-title">Mechanism of the Hydrophobic Effect in the Biomolecular Recognition of Arylsulfonamides by Carbonic Anhydrase</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">17889</span>– <span class="NLM_lpage">17894</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1114107108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1073%2Fpnas.1114107108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=22011572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOktLvK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=108&amp;publication_year=2011&amp;pages=17889-17894&amp;author=P.+W.+Snyderauthor=J.+Mecinovicauthor=D.+T.+Moustakasauthor=S.+W.+Thomasauthor=M.+Harderauthor=E.+T.+Mackauthor=M.+R.+Lockettauthor=A.+Herouxauthor=W.+Shermanauthor=G.+M.+Whitesides&amp;title=Mechanism+of+the+Hydrophobic+Effect+in+the+Biomolecular+Recognition+of+Arylsulfonamides+by+Carbonic+Anhydrase&amp;doi=10.1073%2Fpnas.1114107108" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of the hydrophobic effect in the biomolecular recognition of arylsulfonamides by carbonic anhydrase</span></div><div class="casAuthors">Snyder, Phillip W.; Mecinoviv, Jasmin; Moustakas, Demetri T.; Thomas, Samuel W., III; Harder, Michael; Mack, Eric T.; Lockett, Matthew R.; Heroux, Annie; Sherman, Woody; Whitesides, George M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17889-17894, S17889/1-S17889/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The hydrophobic effect - a rationalization of the insoly. of nonpolar mols. in water-is centrally important to biomol. recognition.  Despite extensive research devoted to the hydrophobic effect, its mol. mechanisms remain controversial, and there are still no reliably predictive models for its role in protein-ligand binding.  Here we describe a particularly well-defined system of protein and ligands - carbonic anhydrase and a series of structurally homologous heterocyclic arom. sulfonamides - that we use to characterize hydrophobic interactions thermodynamically and structurally.  In binding to this structurally rigid protein, a set of ligands (also defined to be structurally rigid) shows the expected gain in binding free energy as hydrophobic surface area is added.  Isothermal titrn. calorimetry demonstrates that enthalpy dets. these increases in binding affinity, and that changes in the heat capacity of binding are. neg.  X-ray crystallog. and mol. dynamics simulations are compatible with the proposal that the differences in binding between the homologous ligands stem from changes in the no. and organization of water mols. localized in the active site in the bound complexes, rather than (or perhaps in addn. to) release of structured water from the apposed hydrophobic surfaces.  These results support the hypothesis that structured water mols. - including both the mols. of water displaced by the ligands and those reorganized upon ligand binding - det. the thermodn. of binding of these ligands at the active site of the protein.  Hydrophobic effects in various contexts have different structural and thermodn. origins, although all may be manifestations of the differences in characteristics of bulk water and water close to hydrophobic surfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdykl8V0IikrVg90H21EOLACvtfcHk0lgUkbVu0et1dw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOktLvK&amp;md5=ba721fce93bd15d5028c8426c017897d</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1114107108&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1114107108%26sid%3Dliteratum%253Aachs%26aulast%3DSnyder%26aufirst%3DP.%2BW.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DMoustakas%26aufirst%3DD.%2BT.%26aulast%3DThomas%26aufirst%3DS.%2BW.%26aulast%3DHarder%26aufirst%3DM.%26aulast%3DMack%26aufirst%3DE.%2BT.%26aulast%3DLockett%26aufirst%3DM.%2BR.%26aulast%3DHeroux%26aufirst%3DA.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DWhitesides%26aufirst%3DG.%2BM.%26atitle%3DMechanism%2520of%2520the%2520Hydrophobic%2520Effect%2520in%2520the%2520Biomolecular%2520Recognition%2520of%2520Arylsulfonamides%2520by%2520Carbonic%2520Anhydrase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D17889%26epage%3D17894%26doi%3D10.1073%2Fpnas.1114107108" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chervenak, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toone, E. J.</span></span> <span> </span><span class="NLM_article-title">A Direct Measure of the Contribution of Solvent Reorganization to the Enthalpy of Binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">10533</span>– <span class="NLM_lpage">10539</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja00102a021</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00102a021" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK2MXhslyisrs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=116&amp;publication_year=1994&amp;pages=10533-10539&amp;author=M.+C.+Chervenakauthor=E.+J.+Toone&amp;title=A+Direct+Measure+of+the+Contribution+of+Solvent+Reorganization+to+the+Enthalpy+of+Binding&amp;doi=10.1021%2Fja00102a021" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A Direct Measure of the Contribution of Solvent Reorganization to the Enthalpy of Binding</span></div><div class="casAuthors">Chervenak, Mary C.; Toone, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10533-9</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">The thermodn. of assocn. of several binding systems, including protein-carbohydrate, small mol.-small mol., protein-peptide, and protein-nucleic acid, were evaluated calorimetrically in light and heavy water.  In every case, the enthalpy of binding in D2O was decreased relative to that in H2O: the differences range from 400 to 1800 cal mol-∞1.  A compensating change in ΔS left the free energy of binding virtually unchanged in each case.  A strong correlation between the differential enthalpy of binding and ΔCp for binding was obsd., with a slope of 5 K.  An anal. of the obsd. effect utilizing a Born-Haber thermodn. cycle shows that the measured decrease in enthalpy represents approx. 10% of the binding enthalpy arising from solvent reorganization.  For the range of systems investigated, solvent reorganization provides 25-100% of the obsd. enthalpy of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxen1jg8NXYLVg90H21EOLACvtfcHk0lhUIPdywmMp8Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXhslyisrs%253D&amp;md5=fc6a558ad9e76aa7b7c28ac930f58510</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fja00102a021&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00102a021%26sid%3Dliteratum%253Aachs%26aulast%3DChervenak%26aufirst%3DM.%2BC.%26aulast%3DToone%26aufirst%3DE.%2BJ.%26atitle%3DA%2520Direct%2520Measure%2520of%2520the%2520Contribution%2520of%2520Solvent%2520Reorganization%2520to%2520the%2520Enthalpy%2520of%2520Binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1994%26volume%3D116%26spage%3D10533%26epage%3D10539%26doi%3D10.1021%2Fja00102a021" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barillari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span> <span> </span><span class="NLM_article-title">Classification of Water Molecules in Protein Binding Sites</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">2577</span>– <span class="NLM_lpage">2587</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja066980q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja066980q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsFGjt7o%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=129&amp;publication_year=2007&amp;pages=2577-2587&amp;author=C.+Barillariauthor=J.+Taylorauthor=R.+Vinerauthor=J.+W.+Essex&amp;title=Classification+of+Water+Molecules+in+Protein+Binding+Sites&amp;doi=10.1021%2Fja066980q" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of Water Molecules in Protein Binding Sites</span></div><div class="casAuthors">Barillari, Caterina; Taylor, Justine; Viner, Russell; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2577-2587</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water mols. play a crucial role in mediating the interaction between a ligand and a macromol. receptor.  An understanding of the nature and role of each water mol. in the active site of a protein could greatly increase the efficiency of rational drug design approaches: if the propensity of a water mol. for displacement can be detd., then synthetic effort may be most profitably applied to the design of specific ligands with the displacement of this water mol. in mind.  In this paper, a thermodn. anal. of water mols. in the binding sites of six proteins, each complexed with a no. of inhibitors, is presented.  Two classes of water mols. were identified: those conserved and not displaced by any of the ligands, and those that are displaced by some ligands.  The abs. binding free energies of 54 water mols. were calcd. using the double decoupling method, with replica exchange thermodn. integration in Monte Carlo simulations.  It was found that conserved water mols. are on av. more tightly bound than displaced water mols.  In addn., Bayesian statistics is used to calc. the probability that a particular water mol. may be displaced by an appropriately designed ligand, given the calcd. binding free energy of the water mol.  This approach therefore allows the numerical assessment of whether or not a given water mol. should be targeted for displacement as part of a rational drug design strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr38ETxUkD-NrVg90H21EOLACvtfcHk0lhUIPdywmMp8Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXhsFGjt7o%253D&amp;md5=e74664f19993a9e9c087dbb9c1d54c3e</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fja066980q&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja066980q%26sid%3Dliteratum%253Aachs%26aulast%3DBarillari%26aufirst%3DC.%26aulast%3DTaylor%26aufirst%3DJ.%26aulast%3DViner%26aufirst%3DR.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DClassification%2520of%2520Water%2520Molecules%2520in%2520Protein%2520Binding%2520Sites%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D2577%26epage%3D2587%26doi%3D10.1021%2Fja066980q" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2817</span>– <span class="NLM_lpage">2831</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=130&amp;publication_year=2008&amp;pages=2817-2831&amp;author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&amp;title=Role+of+the+Active-Site+Solvent+in+the+Thermodynamics+of+Factor+Xa+Ligand+Binding&amp;doi=10.1021%2Fja0771033" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span></div><div class="casAuthors">Abel, Robert; Young, Tom; Farid, Ramy; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2817-2831</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the underlying physics of the binding of small-mol. ligands to protein active sites is a key objective of computational chem. and biol.  It is widely believed that displacement of water mols. from the active site by the ligand is a principal (if not the dominant) source of binding free energy.  Although continuum theories of hydration are routinely used to describe the contributions of the solvent to the binding affinity of the complex, it is still an unsettled question as to whether or not these continuum solvation theories describe the underlying mol. physics with sufficient accuracy to reliably rank the binding affinities of a set of ligands for a given protein.  Here we develop a novel, computationally efficient descriptor of the contribution of the solvent to the binding free energy of a small mol. and its assocd. receptor that captures the effects of the ligand displacing the solvent from the protein active site with at. detail.  This descriptor quant. predicts (R2 = 0.81) the binding free energy differences between congeneric ligand pairs for the test system factor Xa, elucidates phys. properties of the active site solvent that appear to be missing in most continuum theories of hydration, and identifies several features of the hydration of the factor Xa active site relevant to the structure-activity relationship of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS42r59nNXH7Vg90H21EOLACvtfcHk0lhUIPdywmMp8Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D&amp;md5=8eefe76547a82bccaea38f2c84e6cd5f</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DRole%2520of%2520the%2520Active-Site%2520Solvent%2520in%2520the%2520Thermodynamics%2520of%2520Factor%2520Xa%2520Ligand%2520Binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Basran, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutcliffe, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scrutton, N. S.</span></span> <span> </span><span class="NLM_article-title">Enzymatic H-Transfer Requires Vibration-Driven Extreme Tunneling</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">3218</span>– <span class="NLM_lpage">3222</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi982719d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi982719d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK1MXhtFClurc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=38&amp;publication_year=1999&amp;pages=3218-3222&amp;author=J.+Basranauthor=M.+J.+Sutcliffeauthor=N.+S.+Scrutton&amp;title=Enzymatic+H-Transfer+Requires+Vibration-Driven+Extreme+Tunneling&amp;doi=10.1021%2Fbi982719d" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Enzymic H-Transfer Requires Vibration-Driven Extreme Tunneling</span></div><div class="casAuthors">Basran, Jaswir; Sutcliffe, Michael J.; Scrutton, Nigel S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3218-3222</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymic breakage of the substrate C-H bond by Methylophilus methyltrophus (sp. W3A1) methylamine dehydrogenase (MADH) has been studied by stopped-flow spectroscopy.  The rate of redn. of the tryptophan tryptophylquinone (TTQ) cofactor has a large kinetic isotope effect (KIE = 16.8 ± 0.5), and the KIE is independent of temp.  Anal. of the temp. dependence of C-H bond breakage revealed that extreme (ground state) quantum tunneling is responsible for the transfer of the hydrogen nucleus.  Reaction rates are strongly dependent on temp., indicating thermally induced, vibrational motion drives the H-transfer reaction.  The data provide direct exptl. evidence for enzymic bond breakage by extreme tunneling driven by vibrational motion of the protein scaffold.  The results demonstrate that classical transition state theory and its tunneling derivs. do not adequately describe this enzymic reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry9BaMERdHkrVg90H21EOLACvtfcHk0lhUIPdywmMp8Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1MXhtFClurc%253D&amp;md5=0eafc6b49552f43d1af2ef6968cea7b9</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fbi982719d&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi982719d%26sid%3Dliteratum%253Aachs%26aulast%3DBasran%26aufirst%3DJ.%26aulast%3DSutcliffe%26aufirst%3DM.%2BJ.%26aulast%3DScrutton%26aufirst%3DN.%2BS.%26atitle%3DEnzymatic%2520H-Transfer%2520Requires%2520Vibration-Driven%2520Extreme%2520Tunneling%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D3218%26epage%3D3222%26doi%3D10.1021%2Fbi982719d" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Francisco, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Hydrogen Tunneling in Peptidylglycine Alpha-Hydroxylating Monooxygenase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">8194</span>– <span class="NLM_lpage">8195</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja025758s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja025758s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD38XksFOqsr8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=124&amp;publication_year=2002&amp;pages=8194-8195&amp;author=W.+A.+Franciscoauthor=M.+J.+Knappauthor=N.+J.+Blackburnauthor=J.+P.+Klinman&amp;title=Hydrogen+Tunneling+in+Peptidylglycine+Alpha-Hydroxylating+Monooxygenase&amp;doi=10.1021%2Fja025758s" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen Tunneling in Peptidylglycine α-Hydroxylating Monooxygenase</span></div><div class="casAuthors">Francisco, Wilson A.; Knapp, Michael J.; Blackburn, Ninian J.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">8194-8195</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The temp. dependence of the primary and secondary intrinsic isotope effects for the C-H bond cleavage catalyzed by peptidylglycine α-hydroxylating monooxygenase has been detd.  Anal. of the magnitude and Arrhenius behavior of the intrinsic isotope effects provides strong evidence for the use of tunneling as a primary catalytic strategy for this enzyme.  Modeling of the isotope effect data allows for a comparison to the hydrogen transfer catalyzed by soybean lipoxygenase in terms of environmental reorganization energy and frequency of the protein vibration that controls the hydrogen transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCf6jrSC5J57Vg90H21EOLACvtfcHk0li0YGGbNJSg5g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD38XksFOqsr8%253D&amp;md5=33cc4af7f3fb7ce66abea707ecb28326</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fja025758s&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja025758s%26sid%3Dliteratum%253Aachs%26aulast%3DFrancisco%26aufirst%3DW.%2BA.%26aulast%3DKnapp%26aufirst%3DM.%2BJ.%26aulast%3DBlackburn%26aufirst%3DN.%2BJ.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DHydrogen%2520Tunneling%2520in%2520Peptidylglycine%2520Alpha-Hydroxylating%2520Monooxygenase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2002%26volume%3D124%26spage%3D8194%26epage%3D8195%26doi%3D10.1021%2Fja025758s" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sikorski, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markham, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopalan, P. T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benkovic, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohen, A.</span></span> <span> </span><span class="NLM_article-title">Tunneling and Coupled Motion in the Escherichia Coli Dihydrofolate Reductase Catalysis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">4778</span>– <span class="NLM_lpage">4779</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja031683w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja031683w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisVygsLY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=126&amp;publication_year=2004&amp;pages=4778-4779&amp;author=R.+S.+Sikorskiauthor=L.+Wangauthor=K.+A.+Markhamauthor=P.+T.+R.+Rajagopalanauthor=S.+J.+Benkovicauthor=A.+Kohen&amp;title=Tunneling+and+Coupled+Motion+in+the+Escherichia+Coli+Dihydrofolate+Reductase+Catalysis&amp;doi=10.1021%2Fja031683w" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Tunneling and Coupled Motion in the Escherichia coli Dihydrofolate Reductase Catalysis</span></div><div class="casAuthors">Sikorski, R. Steven; Wang, Lin; Markham, Kelli A.; Rajagopalan, P. T. Ravi; Benkovic, Stephen J.; Kohen, Amnon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4778-4779</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">H-transfer was studied in the complex kinetic cascade of dihydrofolate reductase.  Intrinsic kinetic isotope effects, their temp. dependence, and other temp.-dependent parameters indicated H-tunneling, but no 1° to 2° coupled motion.  The data also suggested environmentally coupled tunneling and commitment to catalysis on pre-steady-state isotope effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfUWlPsgr2t7Vg90H21EOLACvtfcHk0li0YGGbNJSg5g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2cXisVygsLY%253D&amp;md5=99cf286374e540ecbc8cfc0e2f195a4b</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fja031683w&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja031683w%26sid%3Dliteratum%253Aachs%26aulast%3DSikorski%26aufirst%3DR.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMarkham%26aufirst%3DK.%2BA.%26aulast%3DRajagopalan%26aufirst%3DP.%2BT.%2BR.%26aulast%3DBenkovic%26aufirst%3DS.%2BJ.%26aulast%3DKohen%26aufirst%3DA.%26atitle%3DTunneling%2520and%2520Coupled%2520Motion%2520in%2520the%2520Escherichia%2520Coli%2520Dihydrofolate%2520Reductase%2520Catalysis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D4778%26epage%3D4779%26doi%3D10.1021%2Fja031683w" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Hydrogen Tunneling and Protein Motion in Enzyme Reactions</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">100</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar040199a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar040199a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVCrsLY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=2006&amp;pages=93-100&amp;author=S.+Hammes-Schiffer&amp;title=Hydrogen+Tunneling+and+Protein+Motion+in+Enzyme+Reactions&amp;doi=10.1021%2Far040199a" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen tunneling and protein motion in enzyme reactions</span></div><div class="casAuthors">Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">93-100</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Theor. perspectives on H-transfer reactions in enzymes are presented.  The proton-coupled electron transfer reaction catalyzed by soybean lipoxygenase and the hydride transfer reaction catalyzed by dihydrofolate reductase are discussed.  The 1st reaction is nonadiabatic and involves 2 distinct electronic states, whereas the 2nd reaction is predominantly adiabatic and occurs on the electronic ground state.  Theor. studies indicate that H-tunneling and protein motion play significant roles in both reactions.  In both cases, the proton donor-acceptor distance decreases relative to its equil. value to enable efficient H-tunneling.  Equil. thermal motions of the protein lead to conformational changes that facilitate H-transfer, but the nonequil. dynamical aspects of these motions have negligible impact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGordjIcMLgKErVg90H21EOLACvtfcHk0li0YGGbNJSg5g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXmvVCrsLY%253D&amp;md5=4eb45fbf92dd7b07f84bd9e34f9de972</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Far040199a&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far040199a%26sid%3Dliteratum%253Aachs%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DHydrogen%2520Tunneling%2520and%2520Protein%2520Motion%2520in%2520Enzyme%2520Reactions%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2006%26volume%3D39%26spage%3D93%26epage%3D100%26doi%3D10.1021%2Far040199a" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Fundamental Insights into Proton-Coupled Electron Transfer in Soybean Lipoxygenase from Quantum Mechanical/Molecular Mechanical Free Energy Simulations</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">3068</span>– <span class="NLM_lpage">3076</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.7b13642</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b13642" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Wisbo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=140&amp;publication_year=2018&amp;pages=3068-3076&amp;author=P.+Liauthor=A.+V.+Soudackovauthor=S.+Hammes-Schiffer&amp;title=Fundamental+Insights+into+Proton-Coupled+Electron+Transfer+in+Soybean+Lipoxygenase+from+Quantum+Mechanical%2FMolecular+Mechanical+Free+Energy+Simulations&amp;doi=10.1021%2Fjacs.7b13642" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Fundamental Insights into Proton-Coupled Electron Transfer in Soybean Lipoxygenase from Quantum Mechanical/Molecular Mechanical Free Energy Simulations</span></div><div class="casAuthors">Li, Pengfei; Soudackov, Alexander V.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3068-3076</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The proton-coupled electron transfer (PCET) reaction catalyzed by soybean lipoxygenase has served as a prototype for understanding hydrogen tunneling in enzymes.  Herein this PCET reaction is studied with mixed quantum mech./mol. mech. (QM/MM) free energy simulations.  The free energy surfaces are computed as functions of the proton donor-acceptor (C-O) distance and the proton coordinate, and the potential of mean force is computed as a function of the C-O distance, inherently including anharmonicity.  The simulation results are used to calc. the kinetic isotope effects for the wild-type enzyme (WT) and the L546A/L754A double mutant (DM), which have been measured exptl. to be ∼80 and ∼700, resp.  The PCET reaction is found to be exoergic for WT and slightly endoergic for the DM, and the equil. C-O distance for the reactant is found to be ∼0.2 Å greater for the DM than for WT.  The larger equil. distance for the DM, which is due mainly to less optimal substrate binding in the expanded binding cavity, is primarily responsible for its higher kinetic isotope effect.  The calcd. potentials of mean force are anharmonic and relatively soft at shorter C-O distances, allowing efficient thermal sampling of the shorter distances required for effective hydrogen tunneling.  The primarily local electrostatic field at the transferring hydrogen is ∼100 MV/cm in the direction to facilitate proton transfer and increases dramatically as the C-O distance decreases.  These simulations suggest that the overall protein environment is important for conformational sampling of active substrate configurations aligned for proton transfer, but the PCET reaction is influenced primarily by local electrostatic effects that facilitate conformational sampling of shorter proton donor-acceptor distances required for effective hydrogen tunneling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_8mb6abhxLbVg90H21EOLACvtfcHk0lhrM6UOnFg3-g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXit1Wisbo%253D&amp;md5=7004dbdc16b392d9421fc9af6b2e6a95</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b13642&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b13642%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DP.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DFundamental%2520Insights%2520into%2520Proton-Coupled%2520Electron%2520Transfer%2520in%2520Soybean%2520Lipoxygenase%2520from%2520Quantum%2520Mechanical%252FMolecular%2520Mechanical%2520Free%2520Energy%2520Simulations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D3068%26epage%3D3076%26doi%3D10.1021%2Fjacs.7b13642" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Fundamental Insights into Proton-Coupled Electron Transfer in Soybean Lipoxygenase from Quantum Mechanical/Molecular Mechanical Free Energy Simulations</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">3068</span>– <span class="NLM_lpage">3076</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.7b13642</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b13642" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Wisbo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=140&amp;publication_year=2018&amp;pages=3068-3076&amp;author=P.+Liauthor=A.+V.+Soudackovauthor=S.+Hammes-Schiffer&amp;title=Fundamental+Insights+into+Proton-Coupled+Electron+Transfer+in+Soybean+Lipoxygenase+from+Quantum+Mechanical%2FMolecular+Mechanical+Free+Energy+Simulations&amp;doi=10.1021%2Fjacs.7b13642" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Fundamental Insights into Proton-Coupled Electron Transfer in Soybean Lipoxygenase from Quantum Mechanical/Molecular Mechanical Free Energy Simulations</span></div><div class="casAuthors">Li, Pengfei; Soudackov, Alexander V.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3068-3076</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The proton-coupled electron transfer (PCET) reaction catalyzed by soybean lipoxygenase has served as a prototype for understanding hydrogen tunneling in enzymes.  Herein this PCET reaction is studied with mixed quantum mech./mol. mech. (QM/MM) free energy simulations.  The free energy surfaces are computed as functions of the proton donor-acceptor (C-O) distance and the proton coordinate, and the potential of mean force is computed as a function of the C-O distance, inherently including anharmonicity.  The simulation results are used to calc. the kinetic isotope effects for the wild-type enzyme (WT) and the L546A/L754A double mutant (DM), which have been measured exptl. to be ∼80 and ∼700, resp.  The PCET reaction is found to be exoergic for WT and slightly endoergic for the DM, and the equil. C-O distance for the reactant is found to be ∼0.2 Å greater for the DM than for WT.  The larger equil. distance for the DM, which is due mainly to less optimal substrate binding in the expanded binding cavity, is primarily responsible for its higher kinetic isotope effect.  The calcd. potentials of mean force are anharmonic and relatively soft at shorter C-O distances, allowing efficient thermal sampling of the shorter distances required for effective hydrogen tunneling.  The primarily local electrostatic field at the transferring hydrogen is ∼100 MV/cm in the direction to facilitate proton transfer and increases dramatically as the C-O distance decreases.  These simulations suggest that the overall protein environment is important for conformational sampling of active substrate configurations aligned for proton transfer, but the PCET reaction is influenced primarily by local electrostatic effects that facilitate conformational sampling of shorter proton donor-acceptor distances required for effective hydrogen tunneling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_8mb6abhxLbVg90H21EOLACvtfcHk0lhrM6UOnFg3-g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXit1Wisbo%253D&amp;md5=7004dbdc16b392d9421fc9af6b2e6a95</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b13642&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b13642%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DP.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DFundamental%2520Insights%2520into%2520Proton-Coupled%2520Electron%2520Transfer%2520in%2520Soybean%2520Lipoxygenase%2520from%2520Quantum%2520Mechanical%252FMolecular%2520Mechanical%2520Free%2520Energy%2520Simulations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D3068%26epage%3D3076%26doi%3D10.1021%2Fjacs.7b13642" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newcomer, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brash, A. R.</span></span> <span> </span><span class="NLM_article-title">The Structural Basis for Specificity in Lipoxygenase Catalysis</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">309</span>, <span class="refDoi">&nbsp;DOI: 10.1002/pro.2626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1002%2Fpro.2626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=25524168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtFaksbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=24&amp;publication_year=2015&amp;pages=298-309&amp;author=M.+E.+Newcomerauthor=A.+R.+Brash&amp;title=The+Structural+Basis+for+Specificity+in+Lipoxygenase+Catalysis&amp;doi=10.1002%2Fpro.2626" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis for specificity in lipoxygenase catalysis</span></div><div class="casAuthors">Newcomer, Marcia E.; Brash, Alan R.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">298-309</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Many intriguing facets of lipoxygenase (LOX) catalysis are open to a detailed structural anal.  Polyunsatd. fatty acids with two to six double bonds are oxygenated precisely on a particular carbon, typically forming a single chiral fatty acid hydroperoxide product.  Mol. oxygen is not bound or liganded during catalysis, yet it is directed precisely to one position and one stereo configuration on the reacting fatty acid.  The transformations proceed upon exposure of substrate to enzyme in the presence of O2 (RH + O2 → ROOH), so it has proved challenging to capture the precise mode of substrate binding in the LOX active site.  Beginning with crystal structures with bound inhibitors or surrogate substrates, and most recently arachidonic acid bound under anaerobic conditions, a picture is consolidating of catalysis in a U-shaped fatty acid binding channel in which individual LOX enzymes use distinct amino acids to control the head-to-tail orientation of the fatty acid and register of the selected pentadiene opposite the non-heme iron, suitably positioned for the initial stereoselective hydrogen abstraction and subsequent reaction with O2.  Drawing on the crystal structures available currently, this review features the roles of the N-terminal β-barrel (C2-like, or PLAT domain) in substrate acquisition and sensitivity to cellular calcium, and the α-helical catalytic domain in fatty acid binding and reactions with O2 that produce hydroperoxide products with regio and stereospecificity.  LOX structures combine to explain how similar enzymes with conserved catalytic machinery differ in product, but not substrate, specificities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQAtaUuCTtmrVg90H21EOLACvtfcHk0lhrM6UOnFg3-g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXjtFaksbw%253D&amp;md5=201690b22d2eebd4a08d1cf92302147c</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2Fpro.2626&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2626%26sid%3Dliteratum%253Aachs%26aulast%3DNewcomer%26aufirst%3DM.%2BE.%26aulast%3DBrash%26aufirst%3DA.%2BR.%26atitle%3DThe%2520Structural%2520Basis%2520for%2520Specificity%2520in%2520Lipoxygenase%2520Catalysis%26jtitle%3DProtein%2520Sci.%26date%3D2015%26volume%3D24%26spage%3D298%26epage%3D309%26doi%3D10.1002%2Fpro.2626" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carr, C. A. M.</span></span> <span> </span><span class="NLM_article-title">On the Role of Enzyme Structure and Structural Dynamics in Lipoxygenase Catalysis</span>.  <i>Doctoral Dissertation</i>, <span class="NLM_publisher-name">University of California</span>, <span class="NLM_publisher-loc">Berkeley</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2013&amp;author=C.+A.+M.+Carr&amp;title=On+the+Role+of+Enzyme+Structure+and+Structural+Dynamics+in+Lipoxygenase+Catalysis" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DC.%2BA.%2BM.%26atitle%3DOn%2520the%2520Role%2520of%2520Enzyme%2520Structure%2520and%2520Structural%2520Dynamics%2520in%2520Lipoxygenase%2520Catalysis%26jtitle%3DDoctoral%2520Dissertation%26pub%3DUniversity%2520of%2520California%26date%3D2013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alber, T.</span></span> <span> </span><span class="NLM_article-title">Automated Protein Crystal Structure Determination Using ELVES</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">1537</span>– <span class="NLM_lpage">1542</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.0306241101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1073%2Fpnas.0306241101" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=101&amp;publication_year=2004&amp;pages=1537-1542&amp;author=J.+Holtonauthor=T.+Alber&amp;title=Automated+Protein+Crystal+Structure+Determination+Using+ELVES&amp;doi=10.1073%2Fpnas.0306241101" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0306241101&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0306241101%26sid%3Dliteratum%253Aachs%26aulast%3DHolton%26aufirst%3DJ.%26aulast%3DAlber%26aufirst%3DT.%26atitle%3DAutomated%2520Protein%2520Crystal%2520Structure%2520Determination%2520Using%2520ELVES%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D1537%26epage%3D1542%26doi%3D10.1073%2Fpnas.0306241101" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span>; <span class="NLM_string-name">Powell, H. R.</span></span> <span> </span><span class="NLM_article-title">Processing Diffraction Data with Mosflm</span>. In  <i>Evolving Methods for Macromolecular Crystallography</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span>, <span class="NLM_string-name">Sussman, J. L.</span></span>, Eds.; NATO Science Series; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Dordrecht</span>, <span class="NLM_year">2007</span>; Vol.  <span class="NLM_volume">245</span>, pp  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">51</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1007%2F978-1-4020-6316-9_4" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2007&amp;pages=41-51&amp;author=A.+G.+W.+Leslie&amp;author=H.+R.+Powellauthor=R.+J.+Read&amp;author=J.+L.+Sussman&amp;title=Evolving+Methods+for+Macromolecular+Crystallography" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1007%2F978-1-4020-6316-9_4&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4020-6316-9_4%26sid%3Dliteratum%253Aachs%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DProcessing%2520Diffraction%2520Data%2520with%2520Mosflm%26btitle%3DEvolving%2520Methods%2520for%2520Macromolecular%2520Crystallography%26aulast%3DRead%26aufirst%3DR.%2BJ.%26pub%3DSpringer%26date%3D2007%26volume%3D245%26spage%3D41%26epage%3D51" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Single-Crystal X-Ray Diffraction Data from a Position-Sensitive Detector</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">916</span>– <span class="NLM_lpage">924</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0021889888007903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0021889888007903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaL1MXhtVGmu7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=1988&amp;pages=916-924&amp;author=W.+Kabsch&amp;title=Evaluation+of+Single-Crystal+X-Ray+Diffraction+Data+from+a+Position-Sensitive+Detector&amp;doi=10.1107%2FS0021889888007903" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of single-crystal x-ray diffraction data from a position-sensitive detector</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">916-24</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">A Fortran program was developed for the redn. of single-crystal diffraction data from a sequence of adjacent rotation pictures recorded at a fixed x-ray wavelength by an electronic area detector.  The electronic pictures (data frames) covering the first 5° of crystal rotation are used to locate strong diffraction spots and to est. the background.  The orientation of the crystal is derived automatically from the list of obsd. spots and all parameters describing the diffraction pattern are refined.  The only input required from the user is the specification of the space group, approx. cell dimensions, and detector setting.  When the initialization step is finished the program goes back to the first picture and evaluates all data frames in the order they arrive from the measurement.  Each element of an electronic picture (pixel) is labeled by the indexes of the nearest reflection using the current refined parameters describing the diffraction geometry.  If the pixel is close to its nearest reflection the counts contribute to the 3-dimensional profile; otherwise the counts are used to update the background.  Each profile is represented as if the reflection had followed the shortest path through the Ewald sphere and had been recorded on the surface of the sphere.  Reflections close to the Ewald sphere are kept in a hash table to allow rapid access for updating the profiles.  Reflections which have completely passed through the Ewald sphere are removed from the table and saved for further processing.  Intensities are estd. by fitting their profiles to an av. shape learned from strong neighboring reflections.  Smoothly varying scaling factors are applied to the Lp-cor. intensities which minimize discrepancies between symmetry-related reflections and fit to a ref. data set if available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtr9BhTPWbOLVg90H21EOLACvtfcHk0lj8LHw70rJ5wQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL1MXhtVGmu7c%253D&amp;md5=f8d8c37891ec0d60758000879e7be9d6</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1107%2FS0021889888007903&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889888007903%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DEvaluation%2520of%2520Single-Crystal%2520X-Ray%2520Diffraction%2520Data%2520from%2520a%2520Position-Sensitive%2520Detector%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1988%26volume%3D21%26spage%3D916%26epage%3D924%26doi%3D10.1107%2FS0021889888007903" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Scaling and Assessment of Data Quality</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">82</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=62&amp;publication_year=2006&amp;pages=72-82&amp;author=P.+Evans&amp;title=Scaling+and+Assessment+of+Data+Quality&amp;doi=10.1107%2FS0907444905036693" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0lj8LHw70rJ5wQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&amp;md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1107%2FS0907444905036693&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520Assessment%2520of%2520Data%2520Quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2FS0907444905036693" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2010&amp;pages=125-132&amp;author=W.+Kabsch&amp;title=Xds&amp;doi=10.1107%2FS0907444909047337" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lj8LHw70rJ5wQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&amp;md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How Good Are My Data and What Is the Resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=69&amp;publication_year=2013&amp;pages=1204-1214&amp;author=P.+R.+Evansauthor=G.+N.+Murshudov&amp;title=How+Good+Are+My+Data+and+What+Is+the+Resolution%3F&amp;doi=10.1107%2FS0907444913000061" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lhvJahRUFriSA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&amp;md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520Good%2520Are%2520My%2520Data%2520and%2520What%2520Is%2520the%2520Resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the Ccp4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=67&amp;publication_year=2011&amp;pages=235-242&amp;author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&amp;title=Overview+of+the+Ccp4+Suite+and+Current+Developments&amp;doi=10.1107%2FS0907444910045749" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhvJahRUFriSA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&amp;md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520Ccp4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenbraun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Key, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timony, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottaviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliz, P.</span></span> <span> </span><span class="NLM_article-title">Cutting Edge: Collaboration Gets the Most out of Software</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e01456</span> <span class="refDoi">&nbsp;DOI: 10.7554/eLife.01456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.7554%2FeLife.01456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=24040512" title="">PubMed</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2&amp;publication_year=2013&amp;author=A.+Morinauthor=B.+Eisenbraunauthor=J.+Keyauthor=P.+C.+Sanschagrinauthor=M.+A.+Timonyauthor=M.+Ottavianoauthor=P.+Sliz&amp;title=Cutting+Edge%3A+Collaboration+Gets+the+Most+out+of+Software&amp;doi=10.7554%2FeLife.01456" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.7554%2FeLife.01456&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.01456%26sid%3Dliteratum%253Aachs%26aulast%3DMorin%26aufirst%3DA.%26aulast%3DEisenbraun%26aufirst%3DB.%26aulast%3DKey%26aufirst%3DJ.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DTimony%26aufirst%3DM.%2BA.%26aulast%3DOttaviano%26aufirst%3DM.%26aulast%3DSliz%26aufirst%3DP.%26atitle%3DCutting%2520Edge%253A%2520Collaboration%2520Gets%2520the%2520Most%2520out%2520of%2520Software%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26doi%3D10.7554%2FeLife.01456" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=40&amp;publication_year=2007&amp;pages=658-674&amp;author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&amp;title=Phaser+Crystallographic+Software&amp;doi=10.1107%2FS0021889807021206" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhvJahRUFriSA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&amp;md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2010&amp;pages=486-501&amp;author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&amp;title=Features+and+Development+of+Coot&amp;doi=10.1107%2FS0907444910007493" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhSzgqqSalTsg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&amp;md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">Phenix: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2010&amp;pages=213-221&amp;author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&amp;title=Phenix%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&amp;doi=10.1107%2FS0907444909052925" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhSzgqqSalTsg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&amp;md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPhenix%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
den Bedem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latombe, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deacon, A. M.</span></span> <span> </span><span class="NLM_article-title">Modeling Discrete Heterogeneity in X-Ray Diffraction Data by Fitting Multi-Conformers</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1107</span>– <span class="NLM_lpage">1117</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444909030613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444909030613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=19770508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=65&amp;publication_year=2009&amp;pages=1107-1117&amp;author=H.+van%0Aden+Bedemauthor=A.+Dhanikauthor=J.-C.+Latombeauthor=A.+M.+Deacon&amp;title=Modeling+Discrete+Heterogeneity+in+X-Ray+Diffraction+Data+by+Fitting+Multi-Conformers&amp;doi=10.1107%2FS0907444909030613" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling discrete heterogeneity in X-ray diffraction data by fitting multi-conformers</span></div><div class="casAuthors">van den Bedem, Henry; Dhanik, Ankur; Latombe, Jean Claude; Deacon, Ashley M.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1107-1117</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The native state of a protein is regarded to be an ensemble of conformers, which allows assocn. with binding partners.  While some of this structural heterogeneity is retained upon crystn., reliably extg. heterogeneous features from diffraction data has remained a challenge.  In this study, a new algorithm for the automatic modeling of discrete heterogeneity is presented.  At high resoln., the authors' single multi-conformer model, with correlated structural features to represent heterogeneity, shows improved agreement with the diffraction data compared with a single-conformer model.  The model appears to be representative of the set of structures present in the crystal.  In contrast, below 2 Å resoln. representing ambiguous electron d. by correlated multi-conformers in a single model does not yield better agreement with the exptl. data.  Consistent with previous studies, this suggests that variability in multi-conformer models at lower resoln. levels reflects uncertainty more than coordinated motion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcZSvF9cJhcLVg90H21EOLACvtfcHk0lhSzgqqSalTsg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnJ&amp;md5=bd39757c428ec3105d1e6affe77ae830</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1107%2FS0907444909030613&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909030613%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bden%2BBedem%26aufirst%3DH.%26aulast%3DDhanik%26aufirst%3DA.%26aulast%3DLatombe%26aufirst%3DJ.-C.%26aulast%3DDeacon%26aufirst%3DA.%2BM.%26atitle%3DModeling%2520Discrete%2520Heterogeneity%2520in%2520X-Ray%2520Diffraction%2520Data%2520by%2520Fitting%2520Multi-Conformers%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1107%26epage%3D1117%26doi%3D10.1107%2FS0907444909030613" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den
Bedem, H.</span></span> <span> </span><span class="NLM_article-title">Exposing Hidden Alternative Backbone Conformations in X-Ray Crystallography Using Qfit</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e1004507</span> <span class="refDoi">&nbsp;DOI: 10.1371/journal.pcbi.1004507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1371%2Fjournal.pcbi.1004507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=26506617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvVKhtL4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2015&amp;author=D.+A.+Keedyauthor=J.+S.+Fraserauthor=H.+van+den%0ABedem&amp;title=Exposing+Hidden+Alternative+Backbone+Conformations+in+X-Ray+Crystallography+Using+Qfit&amp;doi=10.1371%2Fjournal.pcbi.1004507" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Exposing hidden alternative backbone conformations in X-ray crystallography using qFit</span></div><div class="casAuthors">Keedy, Daniel A.; Fraser, James S.; van den Bedem, Henry</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1004507/1-e1004507/22</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Proteins must move between different conformations of their native ensemble to perform their functions.  Crystal structures obtained from high-resoln. X-ray diffraction data reflect this heterogeneity as a spatial and temporal conformational av.  Although movement between natively populated alternative conformations can be crit. for characterizing mol. mechanisms, it is challenging to identify these conformations within electron d. maps.  Alternative side chain conformations are generally well sepd. into distinct rotameric conformations, but alternative backbone conformations can overlap at several at. positions.  Our model building program qFit uses mixed integer quadratic programming (MIQP) to evaluate an extremely large no. of combinations of sidechain conformers and backbone fragments to locally explain the electron d.  Here, we describe two major modeling enhancements to qFit: peptide flips and alternative glycine conformations.  We find that peptide flips fall into four stereotypical clusters and are enriched in glycine residues at the n + 1 position.  The potential for insights uncovered by new peptide flips and glycine conformations is exemplified by HIV protease, where different inhibitors are assocd. with peptide flips in the "flap" regions adjacent to the inhibitor binding site.  Our results paint a picture of peptide flips as conformational switches, often enabled by glycine flexibility, that result in dramatic local rearrangements.  Our results furthermore demonstrate the power of large-scale computational anal. to provide new insights into conformational heterogeneity.  Overall, improved modeling of backbone heterogeneity with high-resoln. X-ray data will connect dynamics to the structure-function relationship and help drive new design strategies for inhibitors of biomedically important systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1GkxACgHTjbVg90H21EOLACvtfcHk0lg7IZSnHHL7ug"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XkvVKhtL4%253D&amp;md5=dfb3faeb27a16df94cc9df4e9d26229d</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1004507&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1004507%26sid%3Dliteratum%253Aachs%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3Dvan%2Bden%2BBedem%26aufirst%3DH.%26atitle%3DExposing%2520Hidden%2520Alternative%2520Backbone%2520Conformations%2520in%2520X-Ray%2520Crystallography%2520Using%2520Qfit%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2015%26volume%3D11%26doi%3D10.1371%2Fjournal.pcbi.1004507" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dundas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binkowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turpaz, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span> <span> </span><span class="NLM_article-title">Castp: Computed Atlas of Surface Topography of Proteins with Structural and Topographical Mapping of Functionally Annotated Residues</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">W116</span>– <span class="NLM_lpage">W118</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gkl282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1093%2Fnar%2Fgkl282" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=16844972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1yhtrw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=2006&amp;pages=W116-W118&amp;author=J.+Dundasauthor=Z.+Ouyangauthor=J.+Tsengauthor=A.+Binkowskiauthor=Y.+Turpazauthor=J.+Liang&amp;title=Castp%3A+Computed+Atlas+of+Surface+Topography+of+Proteins+with+Structural+and+Topographical+Mapping+of+Functionally+Annotated+Residues&amp;doi=10.1093%2Fnar%2Fgkl282" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">CASTp: computed atlas of surface topography of proteins with structural and topographical mapping of functionally annotated residues</span></div><div class="casAuthors">Dundas, Joe; Ouyang, Zheng; Tseng, Jeffery; Binkowski, Andrew; Turpaz, Yaron; Liang, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W116-W118</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Cavities on a proteins surface as well as specific amino acid positioning within it create the physicochem. properties needed for a protein to perform its function.  CASTp (http://cast.engr.uic.edu) is an online tool that locates and measures pockets and voids on 3D protein structures.  This new version of CASTp includes annotated functional information of specific residues on the protein structure.  The annotations are derived from the Protein Data Bank (PDB), Swiss-Prot, as well as Online Mendelian Inheritance in Man (OMIM), the latter contains information on the variant single nucleotide polymorphisms (SNPs) that are known to cause disease.  These annotated residues are mapped to surface pockets, interior voids or other regions of the PDB structures.  The authors use a semi-global pair-wise sequence alignment method to obtain sequence mapping between entries in Swiss-Prot, OMIM and entries in PDB.  The updated CASTp web server can be used to study surface features, functional regions and specific roles of key residues of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocC-dI1GB6RrVg90H21EOLACvtfcHk0lg7IZSnHHL7ug"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28Xps1yhtrw%253D&amp;md5=a5766c6a710f0b80633f68599134227f</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl282&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl282%26sid%3Dliteratum%253Aachs%26aulast%3DDundas%26aufirst%3DJ.%26aulast%3DOuyang%26aufirst%3DZ.%26aulast%3DTseng%26aufirst%3DJ.%26aulast%3DBinkowski%26aufirst%3DA.%26aulast%3DTurpaz%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DJ.%26atitle%3DCastp%253A%2520Computed%2520Atlas%2520of%2520Surface%2520Topography%2520of%2520Proteins%2520with%2520Structural%2520and%2520Topographical%2520Mapping%2520of%2520Functionally%2520Annotated%2520Residues%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3DW116%26epage%3DW118%26doi%3D10.1093%2Fnar%2Fgkl282" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></div><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 11 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Ellie I. James, Taylor A. Murphree, Clint Vorauer, John R. Engen, <span class="NLM_string-name hlFld-ContribAuthor">Miklos Guttman</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in Hydrogen/Deuterium Exchange Mass Spectrometry and the Pursuit of Challenging Biological Systems. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2022,</strong> <em>122 </em>
                                    (8)
                                     , 7562-7623. <a href="https://doi.org/10.1021/acs.chemrev.1c00279" title="DOI URL">https://doi.org/10.1021/acs.chemrev.1c00279</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.1c00279&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.1c00279%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DAdvances%252Bin%252BHydrogen%25252FDeuterium%252BExchange%252BMass%252BSpectrometry%252Band%252Bthe%252BPursuit%252Bof%252BChallenging%252BBiological%252BSystems%26aulast%3DJames%26aufirst%3DEllie%2BI.%26date%3D2022%26date%3D2021%26date%3D2021%26date%3D02042021%26date%3D07092021%26volume%3D122%26issue%3D8%26spage%3D7562%26epage%3D7623" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Wan-Chen Tsai, Ansari Mukhtar Aleem, Chris Whittington, Wilian A. Cortopassi, Chakrapani Kalyanaraman, Angel Baroz, Anthony T. Iavarone, Ewa Skrzypczak-Jankun, Matthew P. Jacobson, Adam R. Offenbacher, <span class="NLM_string-name hlFld-ContribAuthor">Theodore Holman</span>. </span><span class="cited-content_cbyCitation_article-title">Mutagenesis, Hydrogen–Deuterium Exchange, and Molecular Docking Investigations Establish the Dimeric Interface of Human Platelet-Type 12-Lipoxygenase. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2021,</strong> <em>60 </em>
                                    (10)
                                     , 802-812. <a href="https://doi.org/10.1021/acs.biochem.1c00053" title="DOI URL">https://doi.org/10.1021/acs.biochem.1c00053</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.1c00053&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.1c00053%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DMutagenesis%25252C%252BHydrogen%2525E2%252580%252593Deuterium%252BExchange%25252C%252Band%252BMolecular%252BDocking%252BInvestigations%252BEstablish%252Bthe%252BDimeric%252BInterface%252Bof%252BHuman%252BPlatelet-Type%252B12-Lipoxygenase%26aulast%3DTsai%26aufirst%3DWan-Chen%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D19012021%26date%3D12022021%26date%3D26022021%26volume%3D60%26issue%3D10%26spage%3D802%26epage%3D812" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter Maness, Shailendra Koirala, Pratichhya Adhikari, Nasim Salimraftar, <span class="NLM_string-name hlFld-ContribAuthor">Yun Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Substituent Effects on Temperature Dependence of Kinetic Isotope Effects in Hydride-Transfer Reactions of NADH/NAD+ Analogues in Solution: Reaction Center Rigidity Is the Key. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2020,</strong> <em>22 </em>
                                    (15)
                                     , 5963-5967. <a href="https://doi.org/10.1021/acs.orglett.0c02049" title="DOI URL">https://doi.org/10.1021/acs.orglett.0c02049</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.0c02049&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.0c02049%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DSubstituent%252BEffects%252Bon%252BTemperature%252BDependence%252Bof%252BKinetic%252BIsotope%252BEffects%252Bin%252BHydride-Transfer%252BReactions%252Bof%252BNADH%25252FNAD%25252B%252BAnalogues%252Bin%252BSolution%25253A%252BReaction%252BCenter%252BRigidity%252BIs%252Bthe%252BKey%26aulast%3DManess%26aufirst%3DPeter%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D21062020%26date%3D14072020%26volume%3D22%26issue%3D15%26spage%3D5963%26epage%3D5967" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Adam R. Offenbacher, Ajay Sharma, Peter E. Doan, Judith P. Klinman, <span class="NLM_string-name hlFld-ContribAuthor">Brian M. Hoffman</span>. </span><span class="cited-content_cbyCitation_article-title">The Soybean Lipoxygenase–Substrate Complex: Correlation between the Properties of Tunneling-Ready States and ENDOR-Detected Structures of Ground States. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2020,</strong> <em>59 </em>
                                    (7)
                                     , 901-910. <a href="https://doi.org/10.1021/acs.biochem.9b00861" title="DOI URL">https://doi.org/10.1021/acs.biochem.9b00861</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.9b00861&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.9b00861%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DThe%252BSoybean%252BLipoxygenase%2525E2%252580%252593Substrate%252BComplex%25253A%252BCorrelation%252Bbetween%252Bthe%252BProperties%252Bof%252BTunneling-Ready%252BStates%252Band%252BENDOR-Detected%252BStructures%252Bof%252BGround%252BStates%26aulast%3DOffenbacher%26aufirst%3DAdam%2BR.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D2020%26date%3D18092019%26date%3D22012020%26date%3D05022020%26volume%3D59%26issue%3D7%26spage%3D901%26epage%3D910" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">D.A.  Mokhtari</span>, <span class="hlFld-ContribAuthor ">M.J.  Appel</span>, <span class="hlFld-ContribAuthor ">P.M.  Fordyce</span>, <span class="hlFld-ContribAuthor ">D.  Herschlag</span>. </span><span class="cited-content_cbyCitation_article-title">High throughput and quantitative enzymology in the genomic era. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2021,</strong> <em>71 </em>, 259-273. <a href="https://doi.org/10.1016/j.sbi.2021.07.010" title="DOI URL">https://doi.org/10.1016/j.sbi.2021.07.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2021.07.010&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2021.07.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DHigh%252Bthroughput%252Band%252Bquantitative%252Benzymology%252Bin%252Bthe%252Bgenomic%252Bera%26aulast%3DMokhtari%26aufirst%3DD.A.%26date%3D2021%26volume%3D71%26spage%3D259%26epage%3D273" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zahra  Ouaray</span>, <span class="hlFld-ContribAuthor ">Steven A.  Benner</span>, <span class="hlFld-ContribAuthor ">Millie M.  Georgiadis</span>, <span class="hlFld-ContribAuthor ">Nigel G.J.  Richards</span>. </span><span class="cited-content_cbyCitation_article-title">Building better polymerases: Engineering the replication of expanded genetic alphabets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2020,</strong> <em>295 </em>
                                    (50)
                                     , 17046-17059. <a href="https://doi.org/10.1074/jbc.REV120.013745" title="DOI URL">https://doi.org/10.1074/jbc.REV120.013745</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.REV120.013745&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.REV120.013745%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DBuilding%252Bbetter%252Bpolymerases%25253A%252BEngineering%252Bthe%252Breplication%252Bof%252Bexpanded%252Bgenetic%252Balphabets%26aulast%3DOuaray%26aufirst%3DZahra%26date%3D2020%26volume%3D295%26issue%3D50%26spage%3D17046%26epage%3D17059" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam R.  Offenbacher</span>, <span class="hlFld-ContribAuthor ">Theodore R.  Holman</span>. </span><span class="cited-content_cbyCitation_article-title">Fatty Acid Allosteric Regulation of C-H Activation in Plant and Animal Lipoxygenases. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (15)
                                     , 3374. <a href="https://doi.org/10.3390/molecules25153374" title="DOI URL">https://doi.org/10.3390/molecules25153374</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25153374&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25153374%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DFatty%252BAcid%252BAllosteric%252BRegulation%252Bof%252BC-H%252BActivation%252Bin%252BPlant%252Band%252BAnimal%252BLipoxygenases%26aulast%3DOffenbacher%26aufirst%3DAdam%2BR.%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D15%26spage%3D3374" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Betty J.  Gaffney</span>. </span><span class="cited-content_cbyCitation_article-title">EPR Spectroscopic Studies of Lipoxygenases. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – An Asian Journal</span><span> <strong>2020,</strong> <em>15 </em>
                                    (1)
                                     , 42-50. <a href="https://doi.org/10.1002/asia.201901461" title="DOI URL">https://doi.org/10.1002/asia.201901461</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/asia.201901461&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fasia.201901461%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520An%2520Asian%2520Journal%26atitle%3DEPR%252BSpectroscopic%252BStudies%252Bof%252BLipoxygenases%26aulast%3DGaffney%26aufirst%3DBetty%2BJ.%26date%3D2020%26date%3D2019%26volume%3D15%26issue%3D1%26spage%3D42%26epage%3D50" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yisong  Guo</span>, <span class="hlFld-ContribAuthor ">Wei-chen  Chang</span>, <span class="hlFld-ContribAuthor ">Jikun  Li</span>, <span class="hlFld-ContribAuthor ">Madison  Davidson</span>. </span><span class="cited-content_cbyCitation_article-title">Non-heme Mono-Iron Enzymes: Co-Substrate-Independent Dioxygen Activation. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,<a href="https://doi.org/10.1016/B978-0-12-409547-2.14905-4" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.14905-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.14905-4&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.14905-4%26sid%3Dliteratum%253Aachs%26atitle%3DNon-heme%252BMono-Iron%252BEnzymes%25253A%252BCo-Substrate-Independent%252BDioxygen%252BActivation%26aulast%3DGuo%26aufirst%3DYisong%26date%3D2020%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BChemistry%25252C%252BMolecular%252BSciences%252Band%252BChemical%252BEngineering%26date%3D2020%26volume%3D100" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nicole  DeGregorio</span>, <span class="hlFld-ContribAuthor ">Srinivasan S.  Iyengar</span>. </span><span class="cited-content_cbyCitation_article-title">Challenges in constructing accurate methods for hydrogen transfer reactions in large biological assemblies: rare events sampling for mechanistic discovery and tensor networks for quantum nuclear effects. </span><span class="cited-content_cbyCitation_journal-name">Faraday Discussions</span><span> <strong>2020,</strong> <em>221 </em>, 379-405. <a href="https://doi.org/10.1039/C9FD00071B" title="DOI URL">https://doi.org/10.1039/C9FD00071B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9FD00071B&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9FD00071B%26sid%3Dliteratum%253Aachs%26jtitle%3DFaraday%2520Discussions%26atitle%3DChallenges%252Bin%252Bconstructing%252Baccurate%252Bmethods%252Bfor%252Bhydrogen%252Btransfer%252Breactions%252Bin%252Blarge%252Bbiological%252Bassemblies%25253A%252Brare%252Bevents%252Bsampling%252Bfor%252Bmechanistic%252Bdiscovery%252Band%252Btensor%252Bnetworks%252Bfor%252Bquantum%252Bnuclear%252Beffects%26aulast%3DDeGregorio%26aufirst%3DNicole%26date%3D2020%26date%3D2020%26volume%3D221%26spage%3D379%26epage%3D405" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shenshen  Hu</span>, <span class="hlFld-ContribAuthor ">Adam R.  Offenbacher</span>, <span class="hlFld-ContribAuthor ">Edbert D.  Lu</span>, <span class="hlFld-ContribAuthor ">Judith P.  Klinman</span>. </span><span class="cited-content_cbyCitation_article-title">Comparative kinetic isotope effects on first- and second-order rate constants of soybean lipoxygenase variants uncover a substrate-binding network. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2019,</strong> <em>294 </em>
                                    (48)
                                     , 18069-18076. <a href="https://doi.org/10.1074/jbc.RA119.010826" title="DOI URL">https://doi.org/10.1074/jbc.RA119.010826</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.RA119.010826&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.RA119.010826%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DComparative%252Bkinetic%252Bisotope%252Beffects%252Bon%252Bfirst-%252Band%252Bsecond-order%252Brate%252Bconstants%252Bof%252Bsoybean%252Blipoxygenase%252Bvariants%252Buncover%252Ba%252Bsubstrate-binding%252Bnetwork%26aulast%3DHu%26aufirst%3DShenshen%26date%3D2019%26volume%3D294%26issue%3D48%26spage%3D18069%26epage%3D18076" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li></ol></div></div></div></div><div class="ui-resizable-handle ui-resizable-e" style="z-index: 90;"></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active" aria-label="Figures"><i class="icon-image"></i><span style="display: none;">Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class=""><i class="icon-link"></i><span style="display: none;">References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="tab__nav-suppInfo"><i class="icon-supporting-info"></i><span style="display: none;">Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0009.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0007.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Reaction Mechanism for the First and Rate Limiting Step of the Stereospecific Oxidation of Linoleic Acid Catalyzed by SLO, Where the Hydrogen Atom Transfer from Substrate LA to Ferric Hydroxide Cofactor Occurs by a PCET Process</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0001.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Peptide 317–334 (in orange) remote from the active site exhibits mutation-induced changes in the enthapic barrier for HDX rates (<i>E</i><sub>aHDX(avg)</sub>) in L546A, I553G and DM but not in L754A. (A) The SLO structure with highlighted active site residues L546 (in green), L754 (in blue) and I553 (in red). In panel B, Arrhenius-like plots of the weighted average exchange rate, (ln(<i>k</i><sub>HDX(avg)</sub>) for WT, I553G, L546A, L754A and DM are compared. In panel C, the relationship between <i>E</i><sub>aHDX(avg)</sub> and the enthalpic barrier for hydrogen tunneling (<i>E</i><sub>a</sub>(<i>k</i><sub>cat</sub>)) is represented. The solid line is the trend among I553G, WT and L546A previously reported. <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The bottom arrow indicates that L754A shows no difference in <i>E</i><sub>aHDX(avg)</sub> relative to WT. The top arrow shows that there is no further difference in <i>E</i><sub>aHDX(avg)</sub> when L754A is further introduced into L546A.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0002.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Coordination geometry of the iron center of (A) WT SLO (PDB: <a href="http://doi.org/10.2210/pdb5T5V/pdb">5T5V</a>); (B) L546A (PDB: <a href="http://doi.org/10.2210/pdb5TQN/pdb">5TQN</a>); (C) L754A (PDB: <a href="http://doi.org/10.2210/pdb5TR0/pdb">5TR0</a>); and (D) DM (PDB: <a href="http://doi.org/10.2210/pdb5TQO/pdb">5TQO</a>). Oxygen atoms are colored in red, nitrogens in blue, iron center in orange and water in cyan. Carbons in first sphere ligands are colored in yellow, second-sphere ligands in pink, side chains on position 546 and 754 in gray. Bonds between iron and first-sphere ligands (yellow) are drawn as dashed gray lines, and hydrogen-bonding interactions between newly added water in the active site with the first sphere and second sphere ligands are shown as solid gray lines. There are no significant differences in side chain conformers for SLO variants based on the electron densities analysis.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0003.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. EPR spectra (X-band, 6.5K) of oxidized WT, L546A, L754A and DM SLO (140 μM) in 0.1 M potassium phosphate pH = 7.0 buffer. The peak intensities of L546A, L754A and DM have been corrected for the iron content of each mutant: WT (0.9), L546A (0.50), L754A (0.70), DM (0.70). A 10× amplified EPR spectrum of Fe(III) DM SLO is presented for comparison. Expanded EPR spectra are shown in <a href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf" class="ext-link">Figure S5</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0004.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. EPR spectra after Fe(III)-SLOs are anaerobically incubated with catechol ligands in 0.1 M potassium phosphate pH = 7.0 buffer at 4 °C for 12 h: (A) WT; (B) L546A; (C) L754A; (D) DM. The EPR spectra of oxidized Fe(III) SLO vs Fe(II) SLO are shown at the top and bottom of each panel for comparison.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0005.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Rate constants for enzyme catalyzed nonadiabatic hydrogen transfer can be formalized as <i>k</i><sub>obs</sub> = <i>K</i><sub>eq</sub>·<i>k</i><sub>PCET</sub>. <a onclick="showRef(event, 'ref9 ref28 ref30'); return false;" href="javascript:void(0);" class="ref ref9 ref28 ref30">(9,28,30)</a> (A) <i>K</i><sub>eq</sub> (≪1) represents a stochastic ground state search through inactive conformations (<i>b</i>) to reach catalytically active E-S complexes (<i>a</i> conformer). Subsequent thermal sampling further reduces the distance between the H-donor and acceptor as seen in the pretunneling <i>a*</i> conformer. (<b>B</b>-<b>D</b>) illustrate contributions from <i>k</i><sub>PCET</sub>: (B) Heavy atom protein motions produce transiently degenerate energies for the reactant and product wells, a prerequisite for wave function overlap at the tunneling ready state (TRS) (C). (D) The effective potential along the DAD sampling coordinate varies, starting with Δ<i>E</i><sub>a</sub> ≅ 0 for native enzyme (top), becoming Δ<i>E</i><sub>a</sub> ≫ 0 when the DAD is elongated following single mutations (middle) and finally arriving at the catastrophic DM scenario in which the DAD is elongated, but coupled with a rigid DAD sampling potential (bottom). In panel D, light purple refers to the shorter wave function distribution for deuterium, and dark purple represents the more distributive protium wave function.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0006.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Comparison between the ground state substrate binding mode in WT (blue) and DM (yellow). The reactive carbon C11 is colored as black, iron colored as orange.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/cms/10.1021/jacs.8b10992/asset/images/medium/ja-2018-10992n_0008.gif" alt="" data-lg-src="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Free Energy Profile for WT (red) Represented by a Two State Thermodynamic Model Involving E-S and the Enzyme-Bound Pentadienyl Intermediate, E-I<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/jacs.8b10992/asset/images/large/ja-2018-10992n_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jacs.8b10992&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"></p><p class="last"><span class="fn-label"><sup>a</sup></span>In the case of DM, the most stable E-S complex (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>) must first isomerize to a pretunneling configuration (E-S′) that converts via PCET to the mispositioned pentadienyl intermediate (E-I′) with a driving force similar to WT. The return of the bound intermediate E-I′ to a more stable binding position completes the reaction. The dashed blue line has been added as a possible model for the QM/MM computation.</p><p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i30">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88034" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88034" class="dropBlock__holder"></div></div></div></div></div><hr></div><p class="rightTab-references-count">
            This article references 81 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Radzicka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfenden, R.</span></span> <span> </span><span class="NLM_article-title">A Proficient Enzyme</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">90</span>– <span class="NLM_lpage">93</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.7809611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1126%2Fscience.7809611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=7809611" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK2MXivF2iuro%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=267&amp;publication_year=1995&amp;pages=90-93&amp;author=A.+Radzickaauthor=R.+Wolfenden&amp;title=A+Proficient+Enzyme&amp;doi=10.1126%2Fscience.7809611" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A proficient enzyme</span></div><div class="casAuthors">Radzicka, Anna; Wolfenden, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">5194</span>),
    <span class="NLM_cas:pages">90-3</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review and discussion, with 37 refs.  Orotic acid is decarboxylated with a half-time (t1/2) of 78 million years in neutral aq. soln. at room temp., as indicated by reactions in quartz tubes at elevated temps.  Spontaneous hydrolysis of phosphodiester bonds, such as those present in the backbone of DNA, proceeds even more slowly at high temps., but the heat of activation is less pos., so that di-Me phosphate is hydrolyzed with the t1/2 of 130,000 yr in neutral soln. at room temp.  These values extend the known range of spontaneous rate consts. for reactions that are also susceptible to catalysis by enzymes to more than 14 orders or magnitude.  Values of the second-order rate const. kcat/Km for the corresponding enzyme reactions are confined to a range of only 600-fold, in contrast, orotidine 5'-phosphate decarboxylase, an extremely proficient enzyme, enhances the rate of reaction by a factor of 1017 and is estd. to bind the altered substrate in the transition state with a dissocn. const. of less than 5×10-24 M.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLxicA2pBfXrVg90H21EOLACvtfcHk0ljOi35QZhMzig"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXivF2iuro%253D&amp;md5=03a6da852b507155b6d6c5308a83201d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.7809611&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7809611%26sid%3Dliteratum%253Aachs%26aulast%3DRadzicka%26aufirst%3DA.%26aulast%3DWolfenden%26aufirst%3DR.%26atitle%3DA%2520Proficient%2520Enzyme%26jtitle%3DScience%26date%3D1995%26volume%3D267%26spage%3D90%26epage%3D93%26doi%3D10.1126%2Fscience.7809611" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfenden, R.</span></span> <span> </span><span class="NLM_article-title">Catalytic Proficiency: The Extreme Case of S-O Cleaving Sulfatases</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">531</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja208827q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja208827q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=134&amp;publication_year=2012&amp;pages=525-531&amp;author=D.+R.+Edwardsauthor=D.+C.+Lohmanauthor=R.+Wolfenden&amp;title=Catalytic+Proficiency%3A+The+Extreme+Case+of+S-O+Cleaving+Sulfatases&amp;doi=10.1021%2Fja208827q" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fja208827q&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja208827q%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DD.%2BR.%26aulast%3DLohman%26aufirst%3DD.%2BC.%26aulast%3DWolfenden%26aufirst%3DR.%26atitle%3DCatalytic%2520Proficiency%253A%2520The%2520Extreme%2520Case%2520of%2520S-O%2520Cleaving%2520Sulfatases%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D525%26epage%3D531%26doi%3D10.1021%2Fja208827q" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruice, T. C.</span></span> <span> </span><span class="NLM_article-title">A View at the Millennium: The Efficiency of Enzymatic Catalysis</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">139</span>– <span class="NLM_lpage">148</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar0001665</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar0001665" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD38XksVentw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=35&amp;publication_year=2002&amp;pages=139-148&amp;author=T.+C.+Bruice&amp;title=A+View+at+the+Millennium%3A+The+Efficiency+of+Enzymatic+Catalysis&amp;doi=10.1021%2Far0001665" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A view at the millennium: the efficiency of enzymatic catalysis</span></div><div class="casAuthors">Bruice, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">139-148</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review and discussion with 58 refs.  L. Pauling (1946) proposed that the active site of an enzyme (E) binds the transition state (TS) in preference to the substrate (S), and by doing so, stabilizes the TS and lowers the activation energy.  E binding of TS in preference to S and increasing the TΔS⧧ by freezing out motions in E·S and E·TS have been accepted as the driving forces in enzymic catalysis.  However, the smaller value of ΔG⧧ for a 1-substrate enzymic reaction, as compared to its nonenzymic counterpart, is generally the result of a smaller value of ΔH⧧.  The TS in an enzymic reaction is reached through ground-state conformers that closely resemble the TS (near-attack conformers or NACs).  E·NACs are in thermal equil. with all other E·S conformers and are turnstiles through which substrate mols. must pass to arrive at the lowest-energy TS.  The TS in E·TS may or may not be bound tighter than the NAC is in E·NAC.  Thus, the belief that enzymic reactions owe their facility to TS binding or to an increase in ΔStmo requires modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkAreKITPLB7Vg90H21EOLACvtfcHk0lhigvauZed7UA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD38XksVentw%253D%253D&amp;md5=2ae856f67a7524cc0a9ba10ffe650fc4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Far0001665&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far0001665%26sid%3Dliteratum%253Aachs%26aulast%3DBruice%26aufirst%3DT.%2BC.%26atitle%3DA%2520View%2520at%2520the%2520Millennium%253A%2520The%2520Efficiency%2520of%2520Enzymatic%2520Catalysis%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2002%26volume%3D35%26spage%3D139%26epage%3D148%26doi%3D10.1021%2Far0001665" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benkovic, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">A Perspective on Enzyme Catalysis</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>301</i></span>,  <span class="NLM_fpage">1196</span>– <span class="NLM_lpage">1202</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.1085515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1126%2Fscience.1085515" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=12947189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmslShsL0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=301&amp;publication_year=2003&amp;pages=1196-1202&amp;author=S.+J.+Benkovicauthor=S.+Hammes-Schiffer&amp;title=A+Perspective+on+Enzyme+Catalysis&amp;doi=10.1126%2Fscience.1085515" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A perspective on enzyme catalysis</span></div><div class="casAuthors">Benkovic, Stephen J.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">301</span>
        (<span class="NLM_cas:issue">5637</span>),
    <span class="NLM_cas:pages">1196-1202</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  The seminal hypotheses proposed over the years for enzymic catalysis have been scrutinized.  The historical record has been explored from both biochem. and theor. perspectives.  Particular attention is given to the impact of mol. motions within a protein on an enzyme's catalytic properties.  A case study is dihydrofolate reductase, which provides evidence for coupled networks of predominantly conserved residues that influence protein structure and motion.  Such coupled networks have important implications for the origin and evolution of enzymes, as well as for protein engineering.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokLxa7e9sqC7Vg90H21EOLACvtfcHk0lhigvauZed7UA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3sXmslShsL0%253D&amp;md5=d97a8aa302a623866f98a4ff9c744f98</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.1085515&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1085515%26sid%3Dliteratum%253Aachs%26aulast%3DBenkovic%26aufirst%3DS.%2BJ.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DA%2520Perspective%2520on%2520Enzyme%2520Catalysis%26jtitle%3DScience%26date%3D2003%26volume%3D301%26spage%3D1196%26epage%3D1202%26doi%3D10.1126%2Fscience.1085515" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, Z. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">A 21st Century Revisionist’s View at a Turning Point in Enzymology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">543</span>– <span class="NLM_lpage">550</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2Fnchembio.204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=19620995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD1MXoslentrY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2009&amp;pages=543-550&amp;author=Z.+D.+Nagelauthor=J.+P.+Klinman&amp;title=A+21st+Century+Revisionist%E2%80%99s+View+at+a+Turning+Point+in+Enzymology&amp;doi=10.1038%2Fnchembio.204" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A 21st century revisionist's view at a turning point in enzymology</span></div><div class="casAuthors">Nagel, Zachary D.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">543-550</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the fact that the no. of publications assocd. with the keyword 'enzyme' increases every year, the precise origin of enzyme catalysis has remained unresolved.  Because of sustained intensive research efforts from an increasing no. of labs., detailed information regarding the physics, chem., and kinetics of enzymes is accumulating rapidly.  The growing body of data contains many examples of kinetic behavior that are incompatible with a static view of enzyme catalysis.  As a result, numerous labs. are approaching the consensus that protein motion plays an essential role in enzyme catalysis.  A model that incorporates nuclear quantum tunneling together with 2 classes of protein motion, termed conformational sampling (pre-organization) and reorganization, is recommended as a means of understanding the large body of data for enzyme-catalyzed H-transfers.  It should also serve as a vehicle for future efforts in the development of potent enzyme inhibitors and the de novo design of all classifications of enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxj2lQthYBt7Vg90H21EOLACvtfcHk0lhigvauZed7UA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXoslentrY%253D&amp;md5=5722cbad9de292231d410b357f9797bd</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.204&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.204%26sid%3Dliteratum%253Aachs%26aulast%3DNagel%26aufirst%3DZ.%2BD.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DA%252021st%2520Century%2520Revisionist%25E2%2580%2599s%2520View%2520at%2520a%2520Turning%2520Point%2520in%2520Enzymology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D543%26epage%3D550%26doi%3D10.1038%2Fnchembio.204" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span> <span> </span><span class="NLM_article-title">Enzymatic Transition States and Dynamic Motion in Barrier Crossing</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">551</span>– <span class="NLM_lpage">558</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2Fnchembio.202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=19620996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD1MXoslenu7k%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2009&amp;pages=551-558&amp;author=S.+D.+Schwartzauthor=V.+L.+Schramm&amp;title=Enzymatic+Transition+States+and+Dynamic+Motion+in+Barrier+Crossing&amp;doi=10.1038%2Fnchembio.202" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Enzymic transition states and dynamic motion in barrier crossing</span></div><div class="casAuthors">Schwartz, Steven D.; Schramm, Vern L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">551-558</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  What are the at. motions at enzymic catalytic sites on the timescale of chem. change.  Combined exptl. and computational chem. approaches take advantage of transition-state analogs to reveal dynamic motions linked to transition-state formation.  QM/MM transition path sampling from reactive complexes provides both temporal and dynamic information for barrier crossing.  Fast (femtosecond to picosecond) dynamic motions provide essential links to enzymic barrier crossing by local or promoting-mode dynamic searches through bond-vibrational space.  Transition-state lifetimes are within the femtosecond timescales of bond vibrations and show no manifestations of stabilized, equilibrated complexes.  The slow binding and protein conformational changes (microsecond to millisecond) also required for catalysis are temporally decoupled from the fast dynamic motions forming the transition state.  According to this view of enzymic catalysis, transition states are formed by fast, coincident dynamic excursions of catalytic site elements, while the binding of transition-state analogs is the conversion of the dynamic excursions to equilibrated states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6kNG4U5WwFbVg90H21EOLACvtfcHk0lgdiypHz1zrpQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXoslenu7k%253D&amp;md5=6e8267248f4674b9be11fb9c16cea2fe</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.202&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.202%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DS.%2BD.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26atitle%3DEnzymatic%2520Transition%2520States%2520and%2520Dynamic%2520Motion%2520in%2520Barrier%2520Crossing%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D551%26epage%3D558%26doi%3D10.1038%2Fnchembio.202" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glowacki, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, A. J.</span></span> <span> </span><span class="NLM_article-title">Taking Ockham’s Razor to Enzyme Dynamics and Catalysis</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">176</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.1244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2Fnchem.1244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=22354430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyit7o%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2012&amp;pages=169-176&amp;author=D.+R.+Glowackiauthor=J.+N.+Harveyauthor=A.+J.+Mulholland&amp;title=Taking+Ockham%E2%80%99s+Razor+to+Enzyme+Dynamics+and+Catalysis&amp;doi=10.1038%2Fnchem.1244" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Taking Ockham's razor to enzyme dynamics and catalysis</span></div><div class="casAuthors">Glowacki, David R.; Harvey, Jeremy N.; Mulholland, Adrian J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">169-176</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The role of protein dynamics in enzyme catalysis is a matter of intense current debate.  Enzyme-catalyzed reactions that involve significant quantum tunneling can give rise to exptl. kinetic isotope effects with complex temp. dependences, and it has been suggested that std. statistical rate theories, such as transition-state theory, are inadequate for their explanation.  Here, the authors have introduced aspects of transition-state theory relevant to the study of enzyme reactivity, taking cues from chem. kinetics and dynamics studies of small mols. in the gas phase and in soln., where breakdowns of statistical theories have received significant attention and their origins are relatively better understood.  The authors discuss recent theor. approaches to understanding enzyme activity and then show how exptl. observations for a no. of enzymes may be reproduced using a transition-state theory framework with phys. reasonable parameters.  Essential to this simple model is the inclusion of multiple conformations with different reactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzlnwn-OYN8rVg90H21EOLACvtfcHk0lgdiypHz1zrpQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhsVyit7o%253D&amp;md5=a917798b4751dc4898a50e6e3f76647a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1244&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1244%26sid%3Dliteratum%253Aachs%26aulast%3DGlowacki%26aufirst%3DD.%2BR.%26aulast%3DHarvey%26aufirst%3DJ.%2BN.%26aulast%3DMulholland%26aufirst%3DA.%2BJ.%26atitle%3DTaking%2520Ockham%25E2%2580%2599s%2520Razor%2520to%2520Enzyme%2520Dynamics%2520and%2520Catalysis%26jtitle%3DNat.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D169%26epage%3D176%26doi%3D10.1038%2Fnchem.1244" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scrutton, N. S.</span></span> <span> </span><span class="NLM_article-title">Good Vibrations in Enzyme-Catalysed Reactions</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">168</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.1223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2Fnchem.1223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=22354429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyit70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2012&amp;pages=161-168&amp;author=S.+Hayauthor=N.+S.+Scrutton&amp;title=Good+Vibrations+in+Enzyme-Catalysed+Reactions&amp;doi=10.1038%2Fnchem.1223" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Good vibrations in enzyme-catalyzed reactions</span></div><div class="casAuthors">Hay, Sam; Scrutton, Nigel S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">161-168</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fast motions (femtosecond-to-picosecond) and their potential involvement during enzyme-catalyzed reactions have ignited considerable interest in recent years.  Their influence on reaction chem. has been inferred indirectly from studies of the anomalous temp. dependence of kinetic isotope effects and computational simulations.  But, can such motions reduce the width and height of energy barriers along the reaction coordinate, and contribute to quantum mech. and/or classical nuclear-transfer chem.  Here, the authors discuss contemporary ideas for enzymic reactions invoking a role for fast 'promoting' (or 'compressive') motions that, in principle, can aid H-transfer reactions.  Of key importance is the direct demonstration of a role for compressive motions and the ability to understand in at. detail the structural origin of these fast motions, but so far this has not been achieved.  Here, the authors discuss both indirect exptl. evidence that supports a role for compressive motion and the addnl. insight gained from computational simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-AwOb4hlFN7Vg90H21EOLACvtfcHk0lgdiypHz1zrpQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhsVyit70%253D&amp;md5=b159a2bbce7ea193f8cc51d28840ddd2</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1223&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1223%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DS.%26aulast%3DScrutton%26aufirst%3DN.%2BS.%26atitle%3DGood%2520Vibrations%2520in%2520Enzyme-Catalysed%2520Reactions%26jtitle%3DNat.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D161%26epage%3D168%26doi%3D10.1038%2Fnchem.1223" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offenbacher, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span> <span> </span><span class="NLM_article-title">Origins of Enzyme Catalysis: Experimental Findings for C–H Activation, New Models, and Their Relevance to Prevailing Theoretical Constructs</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">18409</span>– <span class="NLM_lpage">18427</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.7b08418</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b08418" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOls7fO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=139&amp;publication_year=2017&amp;pages=18409-18427&amp;author=J.+P.+Klinmanauthor=A.+R.+Offenbacherauthor=S.+Hu&amp;title=Origins+of+Enzyme+Catalysis%3A+Experimental+Findings+for+C%E2%80%93H+Activation%2C+New+Models%2C+and+Their+Relevance+to+Prevailing+Theoretical+Constructs&amp;doi=10.1021%2Fjacs.7b08418" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Origins of Enzyme Catalysis: Experimental Findings for C-H Activation, New Models, and Their Relevance to Prevailing Theoretical Constructs</span></div><div class="casAuthors">Klinman, Judith P.; Offenbacher, Adam R.; Hu, Shenshen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">18409-18427</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The phys. basis for enzymic rate accelerations is a subject of great fundamental interest and of direct relevance to areas that include the de novo design of green catalysts and the pursuit of new drug regimens.  Extensive investigations of C-H activating systems have provided considerable insight into the relationship between an enzyme's overall structure and the catalytic chem. at its active site.  This Perspective highlights recent exptl. data for two members of distinct, yet iconic C-H activation enzyme classes, lipoxygenases and prokaryotic alc. dehydrogenases.  The data necessitate a reformulation of the dominant textbook definition of biol. catalysis.  A multidimensional model emerges that incorporates a range of protein motions that can be parsed into a combination of global stochastic conformational thermal fluctuations and local donor-acceptor distance sampling.  These motions are needed to achieve a high degree of precision with regard to internuclear distances, geometries, and charges within the active site.  The available model also suggests a phys. framework for understanding the empirical enthalpic barrier in enzyme-catalyzed processes.  We conclude by addressing the often conflicting interface between computational and exptl. chemists, emphasizing the need for computation to predict exptl. results in advance of their measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOOb4Grn56bLVg90H21EOLACvtfcHk0lhXz0EyiuHOvg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOls7fO&amp;md5=842f40425e13f7398e5b9cc43e88bcfa</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b08418&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b08418%26sid%3Dliteratum%253Aachs%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26aulast%3DOffenbacher%26aufirst%3DA.%2BR.%26aulast%3DHu%26aufirst%3DS.%26atitle%3DOrigins%2520of%2520Enzyme%2520Catalysis%253A%2520Experimental%2520Findings%2520for%2520C%25E2%2580%2593H%2520Activation%252C%2520New%2520Models%252C%2520and%2520Their%2520Relevance%2520to%2520Prevailing%2520Theoretical%2520Constructs%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D18409%26epage%3D18427%26doi%3D10.1021%2Fjacs.7b08418" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burton, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolfson, D. N.</span></span> <span> </span><span class="NLM_article-title">Installing Hydrolytic Activity into a Completely De Novo Protein Framework</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">837</span>– <span class="NLM_lpage">844</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.2555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2Fnchem.2555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=27554410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFWjs7%252FN" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2016&amp;pages=837-844&amp;author=A.+J.+Burtonauthor=A.+R.+Thomsonauthor=W.+M.+Dawsonauthor=R.+L.+Bradyauthor=D.+N.+Woolfson&amp;title=Installing+Hydrolytic+Activity+into+a+Completely+De+Novo+Protein+Framework&amp;doi=10.1038%2Fnchem.2555" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Installing hydrolytic activity into a completely de novo protein framework</span></div><div class="casAuthors">Burton, Antony J.; Thomson, Andrew R.; Dawson, William M.; Brady, R. Leo; Woolfson, Derek N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">837-844</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design of enzyme-like catalysts tests the understanding of sequence-to-structure/function relations in proteins.  Here, the authors installed hydrolytic activity predictably into a completely de novo and thermostable α-helical barrel, which comprised 7 helixes arranged around an accessible channel.  The authors showed that the lumen of the barrel accepted 21 mutations to functional polar residues.  The resulting variant, which had Cys-His-Glu triads on each helix, hydrolyzed p-nitrophenyl acetate with catalytic efficiencies that matched the most-efficient redesigned hydrolases based on natural protein scaffolds.  This is the 1st report of a functional catalytic triad engineered into a de novo protein framework.  The flexibility of this system also allowed the facile incorporation of unnatural side-chains to improve activity and probe the catalytic mechanism.  Such a predictable and robust construction of truly de novo biocatalysts holds promise for applications in chem. and biochem. synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPhsHmLnqYRbVg90H21EOLACvtfcHk0lhXz0EyiuHOvg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XhtFWjs7%252FN&amp;md5=502ed91d1528dab9147614353f9a081f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2555&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2555%26sid%3Dliteratum%253Aachs%26aulast%3DBurton%26aufirst%3DA.%2BJ.%26aulast%3DThomson%26aufirst%3DA.%2BR.%26aulast%3DDawson%26aufirst%3DW.%2BM.%26aulast%3DBrady%26aufirst%3DR.%2BL.%26aulast%3DWoolfson%26aufirst%3DD.%2BN.%26atitle%3DInstalling%2520Hydrolytic%2520Activity%2520into%2520a%2520Completely%2520De%2520Novo%2520Protein%2520Framework%26jtitle%3DNat.%2520Chem.%26date%3D2016%26volume%3D8%26spage%3D837%26epage%3D844%26doi%3D10.1038%2Fnchem.2555" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilvert, D.</span></span> <span> </span><span class="NLM_article-title">Design of Protein Catalysts</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">470</span>, <span class="refDoi">&nbsp;DOI: 10.1146/annurev-biochem-072611-101825</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1146%2Fannurev-biochem-072611-101825" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=23746259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOqsr3P" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=82&amp;publication_year=2013&amp;pages=447-470&amp;author=D.+Hilvert&amp;title=Design+of+Protein+Catalysts&amp;doi=10.1146%2Fannurev-biochem-072611-101825" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Design of protein catalysts</span></div><div class="casAuthors">Hilvert, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">447-470</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  Diverse engineering strategies have been developed to create enzymes with novel catalytic activities.  Among these, computational approaches hold particular promise.  Enzymes have been computationally designed to promote several nonbiol. reactions, including a Diels-Alder cycloaddn., proton transfer, multistep retroaldol transformations, and metal-dependent hydrolysis of phosphotriesters.  Although their efficiencies (kcat/KM = 0.1-100 M-1 s-1) are typically low compared with those of the best natural enzymes (106-108 M-1 s-1), these catalysts are excellent starting points for lab. evolution.  This review surveys recent progress in combining computational and evolutionary approaches to enzyme design, together with insights into enzyme function gained from studies of the engineered catalysts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtEa8xP2cUlrVg90H21EOLACvtfcHk0lhXz0EyiuHOvg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOqsr3P&amp;md5=a31af0495a8aca01d71a2c8c280882d8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-072611-101825&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-072611-101825%26sid%3Dliteratum%253Aachs%26aulast%3DHilvert%26aufirst%3DD.%26atitle%3DDesign%2520of%2520Protein%2520Catalysts%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2013%26volume%3D82%26spage%3D447%26epage%3D470%26doi%3D10.1146%2Fannurev-biochem-072611-101825" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obexer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrabou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittl, P. R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilvert, D.</span></span> <span> </span><span class="NLM_article-title">Emergence of a Catalytic Tetrad During Evolution of a Highly Active Artificial Aldolase</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">50</span>– <span class="NLM_lpage">56</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.2596</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2Fnchem.2596" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=27995916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVChurbF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2017&amp;pages=50-56&amp;author=R.+Obexerauthor=A.+Godinaauthor=X.+Garrabouauthor=P.+R.+E.+Mittlauthor=D.+Bakerauthor=A.+D.+Griffithsauthor=D.+Hilvert&amp;title=Emergence+of+a+Catalytic+Tetrad+During+Evolution+of+a+Highly+Active+Artificial+Aldolase&amp;doi=10.1038%2Fnchem.2596" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Emergence of a catalytic tetrad during evolution of a highly active artificial aldolase</span></div><div class="casAuthors">Obexer, Richard; Godina, Alexei; Garrabou, Xavier; Mittl, Peer R. E.; Baker, David; Griffiths, Andrew D.; Hilvert, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-56</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Designing catalysts that achieve the rates and selectivities of natural enzymes is a long-standing goal in protein chem.  Here, the authors show that an ultrahigh-throughput droplet-based microfluidic screening platform can be used to improve a previously optimized artificial aldolase by an addnl. factor of 30 to give a &gt;109 rate enhancement that rivals the efficiency of class I aldolases.  The resulting enzyme catalyzes a reversible aldol reaction with high stereoselectivity and tolerates a broad range of substrates.  Biochem. and structural studies show that catalysis depends on a Lys-Tyr-Asn-Tyr tetrad that emerged adjacent to a computationally designed hydrophobic pocket during directed evolution.  This constellation of residues is poised to activate the substrate by Schiff base formation, promote mechanistically important proton transfers, and stabilize multiple transition states along a complex reaction coordinate.  The emergence of such a sophisticated catalytic center shows that there is nothing magical about the catalytic activities or mechanisms of naturally occurring enzymes, or the evolutionary process that gave rise to them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEGhgSJ6Hl_rVg90H21EOLACvtfcHk0lga_n97GRALXQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XhsVChurbF&amp;md5=907ce8bfe60665728bf3b3a035e906f9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2596&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2596%26sid%3Dliteratum%253Aachs%26aulast%3DObexer%26aufirst%3DR.%26aulast%3DGodina%26aufirst%3DA.%26aulast%3DGarrabou%26aufirst%3DX.%26aulast%3DMittl%26aufirst%3DP.%2BR.%2BE.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DGriffiths%26aufirst%3DA.%2BD.%26aulast%3DHilvert%26aufirst%3DD.%26atitle%3DEmergence%2520of%2520a%2520Catalytic%2520Tetrad%2520During%2520Evolution%2520of%2520a%2520Highly%2520Active%2520Artificial%2520Aldolase%26jtitle%3DNat.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D50%26epage%3D56%26doi%3D10.1038%2Fnchem.2596" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P.
G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">From Molecular Diversity to Catalysis - Lessons from the Immune-System</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>269</i></span>,  <span class="NLM_fpage">1835</span>– <span class="NLM_lpage">1842</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.7569920</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1126%2Fscience.7569920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=7569920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK2MXotlKqt7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=269&amp;publication_year=1995&amp;pages=1835-1842&amp;author=P.%0AG.+Schultzauthor=R.+A.+Lerner&amp;title=From+Molecular+Diversity+to+Catalysis+-+Lessons+from+the+Immune-System&amp;doi=10.1126%2Fscience.7569920" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">From molecular diversity to catalysis: lessons from the immune system</span></div><div class="casAuthors">Schultz, Peter G.; Lerner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue">5232</span>),
    <span class="NLM_cas:pages">1835-42</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review, with 75 refs.  By combining the enormous mol. diversity of the immune system with basic mechanistic principles of chem., one can produce catalytic antibodies that allow control of reactions in ways heretofore not possible.  Mechanistic and structural studies of these antibodies are also providing insights into important aspects of enzymic catalysis and the evolution of catalytic function.  Moreover the ability to harness and rationally direct the immune response to generate selective catalysts for reactions ranging from pericyclic and redox reactions to cationic rearrangement reactions underscores the chem. potential of this and other large combinatorial libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8dPAe2FNgSbVg90H21EOLACvtfcHk0lga_n97GRALXQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXotlKqt7c%253D&amp;md5=4bcfe80a46e64fb6244605a0356bda1f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1126%2Fscience.7569920&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7569920%26sid%3Dliteratum%253Aachs%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DFrom%2520Molecular%2520Diversity%2520to%2520Catalysis%2520-%2520Lessons%2520from%2520the%2520Immune-System%26jtitle%3DScience%26date%3D1995%26volume%3D269%26spage%3D1835%26epage%3D1842%26doi%3D10.1126%2Fscience.7569920" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">The Chemistry of the Antibody Molecule</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">4427</span>– <span class="NLM_lpage">4437</span>, <span class="refDoi">&nbsp;DOI: 10.1002/1521-3773(20021202)41:23&lt;4427::AID-ANIE4427&gt;3.0.CO;2-K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1002%2F1521-3773%2820021202%2941%3A23%3C4427%3A%3AAID-ANIE4427%3E3.0.CO%3B2-K" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=41&amp;publication_year=2002&amp;pages=4427-4437&amp;author=P.+G.+Schultzauthor=J.+Yinauthor=R.+A.+Lerner&amp;title=The+Chemistry+of+the+Antibody+Molecule&amp;doi=10.1002%2F1521-3773%2820021202%2941%3A23%3C4427%3A%3AAID-ANIE4427%3E3.0.CO%3B2-K" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820021202%2941%3A23%3C4427%3A%3AAID-ANIE4427%3E3.0.CO%3B2-K&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820021202%252941%253A23%253C4427%253A%253AAID-ANIE4427%253E3.0.CO%253B2-K%26sid%3Dliteratum%253Aachs%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DThe%2520Chemistry%2520of%2520the%2520Antibody%2520Molecule%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2002%26volume%3D41%26spage%3D4427%26epage%3D4437%26doi%3D10.1002%2F1521-3773%2820021202%2941%3A23%3C4427%3A%3AAID-ANIE4427%3E3.0.CO%3B2-K" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kries, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomberg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilvert, D.</span></span> <span> </span><span class="NLM_article-title">De Novo Enzymes by Computational Design</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">228</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cbpa.2013.02.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1016%2Fj.cbpa.2013.02.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=23498973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjvFCnsrY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=2013&amp;pages=221-228&amp;author=H.+Kriesauthor=R.+Blombergauthor=D.+Hilvert&amp;title=De+Novo+Enzymes+by+Computational+Design&amp;doi=10.1016%2Fj.cbpa.2013.02.012" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">De novo enzymes by computational design</span></div><div class="casAuthors">Kries, Hajo; Blomberg, Rebecca; Hilvert, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">221-228</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Computational enzyme design has emerged as a promising tool for generating made-to-order biocatalysts.  In addn. to improving the reliability of the design cycle, current efforts in this area are focusing on expanding the set of catalyzed reactions and investigating the structure and mechanism of individual designs.  Although the activities of de novo enzymes are typically low, they can be significantly increased by directed evolution.  Anal. of their evolutionary trajectories provides valuable feedback for the design algorithms and can enhance our understanding of natural protein evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBIsxuc3Rt4LVg90H21EOLACvtfcHk0lga_n97GRALXQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXjvFCnsrY%253D&amp;md5=68e6e72cff643028e7b28ef71303b559</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2013.02.012&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2013.02.012%26sid%3Dliteratum%253Aachs%26aulast%3DKries%26aufirst%3DH.%26aulast%3DBlomberg%26aufirst%3DR.%26aulast%3DHilvert%26aufirst%3DD.%26atitle%3DDe%2520Novo%2520Enzymes%2520by%2520Computational%2520Design%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D17%26spage%3D221%26epage%3D228%26doi%3D10.1016%2Fj.cbpa.2013.02.012" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Importance of Protein Dynamics During Enzymatic C-H Bond Cleavage Catalysis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2068</span>– <span class="NLM_lpage">2077</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi301504m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi301504m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVyntrs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=2013&amp;pages=2068-2077&amp;author=J.+P.+Klinman&amp;title=Importance+of+Protein+Dynamics+During+Enzymatic+C-H+Bond+Cleavage+Catalysis&amp;doi=10.1021%2Fbi301504m" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Importance of protein dynamics during enzymatic C-H bond cleavage catalysis</span></div><div class="casAuthors">Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2068-2077</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Quantum tunneling and protein dynamics have emerged as important components of enzyme function.  Here, the author focuses on soybean lipoxygenase-1, to illustrate how the properties of enzymic C-H bond activation link protein motions to fundamental bond-making and -breaking processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDcr_ue4eoubVg90H21EOLACvtfcHk0lhD398p645ixA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhvVyntrs%253D&amp;md5=e157f1a61f384157c7e4a24498d08855</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fbi301504m&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi301504m%26sid%3Dliteratum%253Aachs%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DImportance%2520of%2520Protein%2520Dynamics%2520During%2520Enzymatic%2520C-H%2520Bond%2520Cleavage%2520Catalysis%26jtitle%3DBiochemistry%26date%3D2013%26volume%3D52%26spage%3D2068%26epage%3D2077%26doi%3D10.1021%2Fbi301504m" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Proton-Coupled Electron Transfer in Soybean Lipoxygenase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">5763</span>– <span class="NLM_lpage">5775</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja039606o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja039606o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjt1Ghur8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=126&amp;publication_year=2004&amp;pages=5763-5775&amp;author=E.+Hatcherauthor=A.+V.+Soudackovauthor=S.+Hammes-Schiffer&amp;title=Proton-Coupled+Electron+Transfer+in+Soybean+Lipoxygenase&amp;doi=10.1021%2Fja039606o" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Proton-Coupled Electron Transfer in Soybean Lipoxygenase</span></div><div class="casAuthors">Hatcher, Elizabeth; Soudackov, Alexander V.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5763-5775</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The proton-coupled electron transfer reaction catalyzed by soybean lipoxygenase-1 is studied with a multistate continuum theory that represents the transferring hydrogen nucleus as a quantum mech. wave function.  The inner-sphere reorganization energy of the iron cofactor is calcd. with d. functional theory, and the outer-sphere reorganization energy of the protein is calcd. with the frequency-resolved cavity model for conformations obtained with docking simulations.  Both classical and quantum mech. treatments of the proton donor-acceptor vibrational motion are presented.  The temp. dependence of the calcd. rates and kinetic isotope effects is in agreement with the exptl. data.  The weak temp. dependence of the rates is due to the relatively small free energy barrier arising from a balance between the reorganization energy and the reaction free energy.  The unusually high deuterium kinetic isotope effect of 81 is due to the small overlap of the reactant and product proton vibrational wave functions and the dominance of the lowest energy reactant and product vibronic states in the tunneling process.  The temp. dependence of the kinetic isotope effect is strongly influenced by the proton donor-acceptor distance with the dominant contribution to the overall rate.  This dominant proton donor-acceptor distance is significantly smaller than the equil. donor-acceptor distance and is detd. by a balance between the larger coupling and the smaller Boltzmann probability as the distance decreases.  Thus, the proton donor-acceptor vibrational motion plays a vital role in decreasing the dominant donor-acceptor distance relative to its equil. value to facilitate the proton-coupled electron transfer reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsrMbjRwFRkrVg90H21EOLACvtfcHk0lhD398p645ixA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2cXjt1Ghur8%253D&amp;md5=c3de26943ff2aa6ef66769aba01d26ca</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fja039606o&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja039606o%26sid%3Dliteratum%253Aachs%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DProton-Coupled%2520Electron%2520Transfer%2520in%2520Soybean%2520Lipoxygenase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D5763%26epage%3D5775%26doi%3D10.1021%2Fja039606o" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glickman, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Nature of Rate-Limiting Steps in the Soybean Lipoxygenase-1 Reaction</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">14077</span>– <span class="NLM_lpage">14092</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00043a013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00043a013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK2MXos1Wms7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=1995&amp;pages=14077-14092&amp;author=M.+H.+Glickmanauthor=J.+P.+Klinman&amp;title=Nature+of+Rate-Limiting+Steps+in+the+Soybean+Lipoxygenase-1+Reaction&amp;doi=10.1021%2Fbi00043a013" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Nature of Rate-Limiting Steps in the Soybean Lipoxygenase-1 Reaction</span></div><div class="casAuthors">Glickman, Michael H.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">14077-92</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of kinetic isotope effect expts. were performed with the goal of understanding the nature of rate-limiting steps in the soybean lipoxygenase-1 (SBL-1) reaction.  SBL-1 was reacted with linoleic acid (LA) and deuterated linoleic acid (D-LA) under a variety of exptl. conditions involving changes in temp., pH, viscosity, and replacement of H2O with D2O.  The extrapolated intrinsic primary H/D isotope effect can be estd. to be possibly as large as 80.  This value is probably the largest effect published for an enzymic reaction, and much larger than that predicted from semiclassical models.  Due to this large primary isotope effect, the C-D bond cleavage fully limits the rate of reaction under all conditions tested.  In the case of protonated linoleic acid, a no. of steps are partially rate-limiting at room temp.; three distinct mechanistic steps which include substrate binding, an H2O/D2O sensitive step, and C-H bond cleavage have been characterized.  Use of glucose as a solvent viscosogen demonstrates that substrate binding is approx. 48% rate-limiting for LA at 20°.  SBL-1 is one of the few enzymes that fit the definition of a "perfect enzyme", in the sense that further optimization of any single step at room temp. will not significantly increase the overall rate.  At lower temps., the step sensitive to solvent deuteration begins to dominate the reaction, whereas at higher temps., the hydrogen abstraction step is rate-limiting.  The pH dependence of kcat/Km for SBL-1 can be explained as arising from two pKa's, one controlling substrate binding and the other substrate release.  Below pH 7.8, the rate of substrate release increases, thus decreasing the commitment to catalysis and unmasking the large intrinsic isotope effect on the subsequent hydrogen abstraction.  An abnormally high pKa, in the range of 7-8, has been detd. for LA in the concn. range employed in these studies.  The authors propose that the neg. charged form of LA, predominating above pH 7.8, is the preferred substrate with larger commitments to catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH4-SiOjfKPbVg90H21EOLACvtfcHk0lhD398p645ixA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXos1Wms7c%253D&amp;md5=1bce6fa8366f7a36f5196031d68f0e1a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fbi00043a013&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00043a013%26sid%3Dliteratum%253Aachs%26aulast%3DGlickman%26aufirst%3DM.%2BH.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DNature%2520of%2520Rate-Limiting%2520Steps%2520in%2520the%2520Soybean%2520Lipoxygenase-1%2520Reaction%26jtitle%3DBiochemistry%26date%3D1995%26volume%3D34%26spage%3D14077%26epage%3D14092%26doi%3D10.1021%2Fbi00043a013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glickman, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Extremely Large Isotope Effects in the Soybean Lipoxygenase-Linoleic Acid Reaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">793</span>– <span class="NLM_lpage">794</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja00081a060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00081a060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK2cXnt1aksg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=116&amp;publication_year=1994&amp;pages=793-794&amp;author=M.+H.+Glickmanauthor=J.+S.+Wisemanauthor=J.+P.+Klinman&amp;title=Extremely+Large+Isotope+Effects+in+the+Soybean+Lipoxygenase-Linoleic+Acid+Reaction&amp;doi=10.1021%2Fja00081a060" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Extremely Large Isotope Effects in the Soybean Lipoxygenase-Linoleic Acid Reaction</span></div><div class="casAuthors">Glickman, Michael H.; Wiseman, Jeffrey S.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">793-4</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Very large primary deuterium isotope effects are obsd. for the oxidn. of linoleic acid by lipoxygenase.  Primary kinetic (kH/kD) isotope effects were measured both competitively and noncompetitively, with substrates synthesized by different methods and under different conditions.  At room temp. and pH 9, comparing linoleic acid to [11,11-2H2]linoleic acid shows a D(V) = 43.4 (±1.9) and D(V/K) = 27.4 (±3.8).  In all cases an upper limit of ca. 60 was obtained for the isotope effects by changing reaction conditions.  These seem to be the largest published primary (kH/kD) isotope effects in an enzymic system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJzH6GqLdVX7Vg90H21EOLACvtfcHk0lgBUZhBeiYA7w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2cXnt1aksg%253D%253D&amp;md5=b59c096f56bfa2d1102bebb63c5c8258</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fja00081a060&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00081a060%26sid%3Dliteratum%253Aachs%26aulast%3DGlickman%26aufirst%3DM.%2BH.%26aulast%3DWiseman%26aufirst%3DJ.%2BS.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DExtremely%2520Large%2520Isotope%2520Effects%2520in%2520the%2520Soybean%2520Lipoxygenase-Linoleic%2520Acid%2520Reaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1994%26volume%3D116%26spage%3D793%26epage%3D794%26doi%3D10.1021%2Fja00081a060" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jonsson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glickman, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Experimental Evidence for Extensive Tunneling of Hydrogen in the Lipoxygenase Reaction: Implications for Enzyme Catalysis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">10319</span>– <span class="NLM_lpage">10320</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja961827p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja961827p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK28XmtFKks7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=118&amp;publication_year=1996&amp;pages=10319-10320&amp;author=T.+Jonssonauthor=M.+H.+Glickmanauthor=S.+J.+Sunauthor=J.+P.+Klinman&amp;title=Experimental+Evidence+for+Extensive+Tunneling+of+Hydrogen+in+the+Lipoxygenase+Reaction%3A+Implications+for+Enzyme+Catalysis&amp;doi=10.1021%2Fja961827p" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental Evidence for Extensive Tunneling of Hydrogen in the Lipoxygenase Reaction: Implications for Enzyme Catalysis</span></div><div class="casAuthors">Jonsson, Thorlakur; Glickman, Michael H.; Sun, Shujun; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">10319-10320</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogen tunneling has recently has been shown to take place in the enzyme-catalyzed hydrogen transfer reactions of yeast alc. dehydrogenase, horse liver alc. dehydrogenase, bovine serum amine oxidase, and monoamine oxidase B.  In all cases, the obsd. behavior is consistent with the view of a tunneling correction to the reaction rate, such that the lighter isotopes of hydrogen undergo tunneling to a greater extent than the heavier ones.  Recent investigations of steady state kinetic isotope effects in the soybean lipoxygenase reaction have indicated the largest deuterium isotope effect seen for a biol. catalyst (kH/kD = 48 at 25°).  Extension of these measurements to single turnover conditions supports the view that the enormous rate discrimination between hydrogen and deuterium occurs in the C-H bond activation step.  Measurement of the temp. dependence for C-H(D) cleavage under both pre-steady state and steady state conditions reveals a nearly activationless process.  The combined observation of very small enthalpies of activation and very large, nearly temp.-independent isotope effects leads us to conclude that soybean lipoxygenase facilitates the loss of hydrogen from substrate near room temp. by invoking extensive tunneling.  This type of behavior has been seen with small mols., but generally only at very low temps.  A diagram is presented to encompass the observe range of enzymic behavior, in which tunneling varies from moderate to extensive.  As predicted, a correlation can be obsd. between enzymic enthalpies of activation and the magnitude of isotope effects on Arrhenius prefactors.  These results indicate that optimal catalytic efficiency in enzyme reactions requires a modulation of reaction barrier width as well as height.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru7wKNDte5FbVg90H21EOLACvtfcHk0lgBUZhBeiYA7w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK28XmtFKks7w%253D&amp;md5=2d3e621a9c0194942cab4f8ec21c67f3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fja961827p&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja961827p%26sid%3Dliteratum%253Aachs%26aulast%3DJonsson%26aufirst%3DT.%26aulast%3DGlickman%26aufirst%3DM.%2BH.%26aulast%3DSun%26aufirst%3DS.%2BJ.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DExperimental%2520Evidence%2520for%2520Extensive%2520Tunneling%2520of%2520Hydrogen%2520in%2520the%2520Lipoxygenase%2520Reaction%253A%2520Implications%2520for%2520Enzyme%2520Catalysis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D10319%26epage%3D10320%26doi%3D10.1021%2Fja961827p" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Temperature-Dependent Isotope Effects in Soybean Lipoxygenase-1: Correlating Hydrogen Tunneling with Protein Dynamics</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">3865</span>– <span class="NLM_lpage">3874</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja012205t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja012205t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD38XitVKqurg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=124&amp;publication_year=2002&amp;pages=3865-3874&amp;author=M.+J.+Knappauthor=K.+Rickertauthor=J.+P.+Klinman&amp;title=Temperature-Dependent+Isotope+Effects+in+Soybean+Lipoxygenase-1%3A+Correlating+Hydrogen+Tunneling+with+Protein+Dynamics&amp;doi=10.1021%2Fja012205t" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Temperature-Dependent Isotope Effects in Soybean Lipoxygenase-1: Correlating Hydrogen Tunneling with Protein Dynamics</span></div><div class="casAuthors">Knapp, Michael J.; Rickert, Keith; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3865-3874</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The hydrogen-atom transfer in soybean lipoxygenase-1 (SLO) exhibits a large kinetic isotope effect on kcat (KIE = 81) near room temp. and a very weak temp. dependence (Eact = 2.1 kcal/mol).  These properties are consistent with H· transfer that occurs entirely by a tunneling event.  Mutants of SLO were prepd., and the temp. dependence of the KIE was measured, to test for alterations in the tunneling behavior.  All mutants studied exhibit KIEs of similar, large magnitude at 30°, despite an up to 3 orders of magnitude change in kcat.  Eact for two of the mutants (Leu754 → Ala, Leu546 → Ala) is larger than for wild-type (WT), and the KIE becomes slightly more temp. dependent.  In contrast, Ile553 → Ala exhibits kcat and Eact parameters similar to wild-type soybean lipoxygenase-1 (WT-SLO) for protiated substrate; however, the KIE is markedly temp. dependent.  The behavior of the former two mutants could reflect increased reorganization energies (λ), but the behavior of the latter mutant is inconsistent with this description.  We have invoked a full H· tunneling model (Kuznetsov, A. M.; Ulstrup, J. Can. J. Chem. 1999, 77, 1085-1096) to explain the temp. dependence of the KIE, which is indicative of the extent to which distance sampling (gating) modulates hydrogen transfer.  WT-SLO exhibits a very small Eact and a nearly temp.-independent KIE, which was modeled as arising from a compressed hydrogen transfer distance with little modulation of the hydrogen transfer distance.  The observations on the Leu754 → Ala and Leu546 → Ala mutants were modeled as arising from a slightly less compressed active site with greater modulation of the hydrogen transfer distance by environmental dynamics.  Finally, the obsd. behavior of the Ile553 → Ala mutant indicates a relaxed active site with extensive involvement of gating to facilitate hydrogen transfer.  We conclude that WT-SLO has an active site structure that is well organized to support hydrogen tunneling and that mutations perturb structural elements that support hydrogen tunneling.  Modest alterations in active site residues increase λ and/or increase the hydrogen transfer distance, thereby affecting the probability that tunneling can occur.  These studies allow the detection and characterization of a protein-gating mode in catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVlq9ISuo2KrVg90H21EOLACvtfcHk0lgBUZhBeiYA7w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD38XitVKqurg%253D&amp;md5=e63570c9ed25cd5d141ecee52583487b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fja012205t&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja012205t%26sid%3Dliteratum%253Aachs%26aulast%3DKnapp%26aufirst%3DM.%2BJ.%26aulast%3DRickert%26aufirst%3DK.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DTemperature-Dependent%2520Isotope%2520Effects%2520in%2520Soybean%2520Lipoxygenase-1%253A%2520Correlating%2520Hydrogen%2520Tunneling%2520with%2520Protein%2520Dynamics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2002%26volume%3D124%26spage%3D3865%26epage%3D3874%26doi%3D10.1021%2Fja012205t" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Proton-Coupled Electron Transfer Reactions: Analytical Rate Constants and Case Study of Kinetic Isotope Effects in Lipoxygenase</span>. <i>Faraday Discuss.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>195</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">189</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C6FD00122J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1039%2FC6FD00122J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=27735009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsVOlsb0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=195&amp;publication_year=2016&amp;pages=171-189&amp;author=A.+V.+Soudackovauthor=S.+Hammes-Schiffer&amp;title=Proton-Coupled+Electron+Transfer+Reactions%3A+Analytical+Rate+Constants+and+Case+Study+of+Kinetic+Isotope+Effects+in+Lipoxygenase&amp;doi=10.1039%2FC6FD00122J" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Proton-coupled electron transfer reactions: analytical rate constants and case study of kinetic isotope effects in lipoxygenase</span></div><div class="casAuthors">Soudackov, Alexander V.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Faraday Discussions</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue">Reaction Rate Theory</span>),
    <span class="NLM_cas:pages">171-189</span>CODEN:
                <span class="NLM_cas:coden">FDISE6</span>;
        ISSN:<span class="NLM_cas:issn">1359-6640</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A general theory has been developed for proton-coupled electron transfer (PCET), which is vital to a wide range of chem. and biol. processes.  This theory describes PCET reactions in terms of nonadiabatic transitions between reactant and product electron-proton vibronic states and includes the effects of thermal fluctuations of the solvent or protein environment, as well as the proton donor-acceptor motion.  Within the framework of this general PCET theory, a series of anal. rate const. expressions has been derived for PCET reactions in well-defined regimes.  Herein, the application of this theory to PCET in the enzyme soybean lipoxygenase illustrates the regimes of validity for the various rate const. expressions and elucidates the fundamental phys. principles dictating PCET reactions.  Such theor. studies provide significant phys. insights that guide the interpretation of exptl. data and lead to exptl. testable predictions.  A combination of theor. treatments with at.-level simulations is essential to understanding PCET.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKY2SDFI5vRLVg90H21EOLACvtfcHk0lgBUZhBeiYA7w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XpsVOlsb0%253D&amp;md5=103341468cac8348e786883eee5b1a72</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1039%2FC6FD00122J&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6FD00122J%26sid%3Dliteratum%253Aachs%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DProton-Coupled%2520Electron%2520Transfer%2520Reactions%253A%2520Analytical%2520Rate%2520Constants%2520and%2520Case%2520Study%2520of%2520Kinetic%2520Isotope%2520Effects%2520in%2520Lipoxygenase%26jtitle%3DFaraday%2520Discuss.%26date%3D2016%26volume%3D195%26spage%3D171%26epage%3D189%26doi%3D10.1039%2FC6FD00122J" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Proton-Coupled Electron Transfer in Soybean Lipoxygenase: Dynamical Behavior and Temperature Dependence of Kinetic Isotope Effects</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">196</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja0667211</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0667211" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlaju73N" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=129&amp;publication_year=2007&amp;pages=187-196&amp;author=E.+Hatcherauthor=A.+V.+Soudackovauthor=S.+Hammes-Schiffer&amp;title=Proton-Coupled+Electron+Transfer+in+Soybean+Lipoxygenase%3A+Dynamical+Behavior+and+Temperature+Dependence+of+Kinetic+Isotope+Effects&amp;doi=10.1021%2Fja0667211" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Proton-Coupled Electron Transfer in Soybean Lipoxygenase: Dynamical Behavior and Temperature Dependence of Kinetic Isotope Effects</span></div><div class="casAuthors">Hatcher, Elizabeth; Soudackov, Alexander V.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">187-196</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dynamical behavior and the temp. dependence of the kinetic isotope effects (KIEs) are examd. for the proton-coupled electron transfer reaction catalyzed by the enzyme soybean lipoxygenase.  The calcns. are based on a vibronically nonadiabatic formulation that includes the quantum mech. effects of the active electrons and the transferring proton, as well as the motions of all atoms in the complete solvated enzyme system.  The rate const. is represented by the time integral of a probability flux correlation function that depends on the vibronic coupling and on time correlation functions of the energy gap and the proton donor-acceptor mode, which can be calcd. from classical mol. dynamics simulations of the entire system.  The dynamical behavior of the probability flux correlation function is dominated by the equil. protein and solvent motions and is not significantly influenced by the proton donor-acceptor motion.  The magnitude of the overall rate is strongly influenced by the proton donor-acceptor frequency, the vibronic coupling, and the protein/solvent reorganization energy.  The calcns. reproduce the exptl. obsd. magnitude and temp. dependence of the KIE for the soybean lipoxygenase reaction without fitting any parameters directly to the exptl. kinetic data.  The temp. dependence of the KIE is detd. predominantly by the proton donor-acceptor frequency and the distance dependence of the vibronic couplings for hydrogen and deuterium.  The ratio of the overlaps of the hydrogen and deuterium vibrational wavefunctions strongly impacts the magnitude of the KIE but does not significantly influence its temp. dependence.  For this enzyme reaction, the large magnitude of the KIE arises mainly from the dominance of tunneling between the ground vibronic states and the relatively large ratio of the overlaps between the corresponding hydrogen and deuterium vibrational wavefunctions.  The weak temp. dependence of the KIE is due in part to the dominance of the local component of the proton donor-acceptor motion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNgQiCJB5OY7Vg90H21EOLACvtfcHk0ljt4Mp4XBqtbQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28Xhtlaju73N&amp;md5=63db97f6cd817adf179e0a9cc71fc3c3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fja0667211&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0667211%26sid%3Dliteratum%253Aachs%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DProton-Coupled%2520Electron%2520Transfer%2520in%2520Soybean%2520Lipoxygenase%253A%2520Dynamical%2520Behavior%2520and%2520Temperature%2520Dependence%2520of%2520Kinetic%2520Isotope%2520Effects%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D187%26epage%3D196%26doi%3D10.1021%2Fja0667211" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Impact of Distal Mutation on Hydrogen Transfer Interface and Substrate Conformation in Soybean Lipoxygenase</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">6653</span>– <span class="NLM_lpage">6660</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jp100133p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp100133p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1WrtLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=114&amp;publication_year=2010&amp;pages=6653-6660&amp;author=S.+J.+Edwardsauthor=A.+V.+Soudackovauthor=S.+Hammes-Schiffer&amp;title=Impact+of+Distal+Mutation+on+Hydrogen+Transfer+Interface+and+Substrate+Conformation+in+Soybean+Lipoxygenase&amp;doi=10.1021%2Fjp100133p" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Distal Mutation on Hydrogen Transfer Interface and Substrate Conformation in Soybean Lipoxygenase</span></div><div class="casAuthors">Edwards, Sarah J.; Soudackov, Alexander V.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6653-6660</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The impact of distal mutation on the hydrogen transfer interface properties and on the substrate mobility, conformation, and orientation in soybean lipoxygenase-1 (SLO) is examd.  SLO catalyzes a hydrogen abstraction reaction that occurs by a proton-coupled electron transfer mechanism.  Mutation of isoleucine 553 to less bulky residues has been found exptl. to increase the magnitude and temp. dependence of the kinetic isotope effect for this reaction.  This residue borders the linoleic acid substrate but is ∼15 Å from the active site iron.  In the present study, we model these exptl. data with a vibronically nonadiabatic theory and perform all-atom mol. dynamics simulations on the complete solvated wild-type and mutant enzymes.  Our calcns. indicate that the proton transfer equil. distance increases and the assocd. frequency decreases as residue 553 becomes less bulky.  The mol. dynamics simulations illustrate that this mutation impacts the mobility, geometrical conformation, and orientation of the linoleic acid within the active site.  In turn, these effects alter the proton donor-acceptor equil. distance and frequency, leading to the exptl. obsd. changes in the magnitude and temp. dependence of the kinetic isotope effect.  This study provides insight into how the effects of distal mutations may be transmitted in enzymes to ultimately impact the catalytic rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCwBTFxq69I7Vg90H21EOLACvtfcHk0ljt4Mp4XBqtbQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXlt1WrtLw%253D&amp;md5=72b2f9a34f8c590693d305d3ea488c64</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjp100133p&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp100133p%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DS.%2BJ.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DImpact%2520of%2520Distal%2520Mutation%2520on%2520Hydrogen%2520Transfer%2520Interface%2520and%2520Substrate%2520Conformation%2520in%2520Soybean%2520Lipoxygenase%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2010%26volume%3D114%26spage%3D6653%26epage%3D6660%26doi%3D10.1021%2Fjp100133p" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Experimental Evidence for Hydrogen Tunneling When the Isotopic Arrhenius Prefactor (a(H)/a(D)) Is Unity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">17632</span>– <span class="NLM_lpage">17633</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja806354w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja806354w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=130&amp;publication_year=2008&amp;pages=17632-17633&amp;author=S.+C.+Sharmaauthor=J.+P.+Klinman&amp;title=Experimental+Evidence+for+Hydrogen+Tunneling+When+the+Isotopic+Arrhenius+Prefactor+%28a%28H%29%2Fa%28D%29%29+Is+Unity&amp;doi=10.1021%2Fja806354w" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja806354w&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja806354w%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BC.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DExperimental%2520Evidence%2520for%2520Hydrogen%2520Tunneling%2520When%2520the%2520Isotopic%2520Arrhenius%2520Prefactor%2520%2528a%2528H%2529%252Fa%2528D%2529%2529%2520Is%2520Unity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D17632%26epage%3D17633%26doi%3D10.1021%2Fja806354w" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomchick, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Enzyme Structure and Dynamics Affect Hydrogen Tunneling: The Impact of a Remote Side Chain (1553) in Soybean Lipoxygenase-1</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">1146</span>– <span class="NLM_lpage">1151</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.0710643105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1073%2Fpnas.0710643105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=18216254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhs1KmtLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=105&amp;publication_year=2008&amp;pages=1146-1151&amp;author=M.+P.+Meyerauthor=D.+R.+Tomchickauthor=J.+P.+Klinman&amp;title=Enzyme+Structure+and+Dynamics+Affect+Hydrogen+Tunneling%3A+The+Impact+of+a+Remote+Side+Chain+%281553%29+in+Soybean+Lipoxygenase-1&amp;doi=10.1073%2Fpnas.0710643105" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme structure and dynamics affect hydrogen tunneling: The impact of a remote side chain (I553) in soybean lipoxygenase-1</span></div><div class="casAuthors">Meyer, Matthew P.; Tomchick, Diana R.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1146-1151, S1146/1-S1146/12</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">This study examines the impact of a series of mutations at position 553 on the kinetic and structural properties of soybean lipoxygenase-1 (SLO-1).  The previously uncharacterized mutants reported herein are I553L, I553V, and I553G.  High-resoln. x-ray studies of these mutants, together with the earlier studied I553A, show almost no structural change in relation to the WT-enzyme.  By contrast, a progression in kinetic behavior occurs in which the decrease in the size of the side chain at position 553 leads to an increased importance of donor-acceptor distance sampling in the course of the hydrogen transfer process.  These dynamical changes in behavior are interpreted in the context of two general classes of protein motions, preorganization and reorganization, with the latter including the distance sampling modes.  The aggregate data for SLO-1 show how judicious placement of hydrophobic side chains can influence enzyme catalysis via enhanced donor-acceptor hydro-genic wave function overlap.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBbF8U0rjNi7Vg90H21EOLACvtfcHk0ljt4Mp4XBqtbQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXhs1KmtLw%253D&amp;md5=9a5a789879adba5216bab4822f4ad9a4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0710643105&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0710643105%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DM.%2BP.%26aulast%3DTomchick%26aufirst%3DD.%2BR.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DEnzyme%2520Structure%2520and%2520Dynamics%2520Affect%2520Hydrogen%2520Tunneling%253A%2520The%2520Impact%2520of%2520a%2520Remote%2520Side%2520Chain%2520%25281553%2529%2520in%2520Soybean%2520Lipoxygenase-1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D1146%26epage%3D1151%26doi%3D10.1073%2Fpnas.0710643105" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Modeling Temperature Dependent Kinetic Isotope Effects for Hydrogen Transfer in a Series of Soybean Lipoxygenase Mutants: The Effect of Anharmonicity Upon Transfer Distance</span>. <i>Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>319</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">296</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.chemphys.2005.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1016%2Fj.chemphys.2005.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Ort7zL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=319&amp;publication_year=2005&amp;pages=283-296&amp;author=M.+P.+Meyerauthor=J.+P.+Klinman&amp;title=Modeling+Temperature+Dependent+Kinetic+Isotope+Effects+for+Hydrogen+Transfer+in+a+Series+of+Soybean+Lipoxygenase+Mutants%3A+The+Effect+of+Anharmonicity+Upon+Transfer+Distance&amp;doi=10.1016%2Fj.chemphys.2005.05.017" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling temperature dependent kinetic isotope effects for hydrogen transfer in a series of soybean lipoxygenase mutants: The effect of anharmonicity upon transfer distance</span></div><div class="casAuthors">Meyer, Matthew P.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Physics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">319</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">283-296</span>CODEN:
                <span class="NLM_cas:coden">CMPHC2</span>;
        ISSN:<span class="NLM_cas:issn">0301-0104</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Soybean lipoxygenase-1 (SLO) catalyzes the oxidn. of linoleic acid.  The rate-limiting step in this transformation is the net abstraction of the pro-S hydrogen atom from the center of the 1,5-pentadienyl moiety in linoleic acid.  The large deuterium kinetic isotope effect (KIE) for this step appears in the first order rate const. (Dkcat = 81±5 at T = 25°).  Furthermore, the KIE and the rate for protium abstraction are weakly temp. dependent (EA,D - EA,H = 0.9±0.2 kcal/mol and EA,H = 2.1±0.2 kcal/mol, resp.).  Mutations at a hydrophobic site about 13 Å from the active site Fe(III), Ile553, induce a marked temp. dependence that varies roughly in accordance with the degree to which the residue is changed in bulk from the wild type Ile.  While the temp. dependence for these mutants varies from the wild type enzyme, the magnitude of the KIE at 25° is on the same order of magnitude.  A hydrogen tunneling model [Kuznetsov, A. M., Ulstrup, J. Can. J. Chem. 77 (1999) 1085-1096] is utilized to model the KIE temp. profiles for the wild type SLO and each Ile553 mutant.  Hydrogenic wavefunctions are modeled using harmonic oscillators and Morse oscillators in order to explore the effects of anharmonicity upon computed kinetic observables used to characterize this hydrogen transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0punSHBQzTLVg90H21EOLACvtfcHk0lhDJ-SGtrDjBg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXht1Ort7zL&amp;md5=fc46ef4a3f7b45c2c5ec1dd44ac3c2ca</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.chemphys.2005.05.017&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chemphys.2005.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DM.%2BP.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DModeling%2520Temperature%2520Dependent%2520Kinetic%2520Isotope%2520Effects%2520for%2520Hydrogen%2520Transfer%2520in%2520a%2520Series%2520of%2520Soybean%2520Lipoxygenase%2520Mutants%253A%2520The%2520Effect%2520of%2520Anharmonicity%2520Upon%2520Transfer%2520Distance%26jtitle%3DChem.%2520Phys.%26date%3D2005%26volume%3D319%26spage%3D283%26epage%3D296%26doi%3D10.1016%2Fj.chemphys.2005.05.017" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Enhanced Rigidification within a Double Mutant of Soybean Lipoxygenase Provides Experimental Support for Vibronically Nonadiabatic Proton-Coupled Electron Transfer Models</span>. <i>ACS Catal.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3569</span>– <span class="NLM_lpage">3574</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscatal.7b00688</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscatal.7b00688" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmt1Kqtrg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2017&amp;pages=3569-3574&amp;author=S.+Huauthor=A.+V.+Soudackovauthor=S.+Hammes-Schifferauthor=J.+P.+Klinman&amp;title=Enhanced+Rigidification+within+a+Double+Mutant+of+Soybean+Lipoxygenase+Provides+Experimental+Support+for+Vibronically+Nonadiabatic+Proton-Coupled+Electron+Transfer+Models&amp;doi=10.1021%2Facscatal.7b00688" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced rigidification within a double mutant of soybean lipoxygenase provides experimental support for vibronically nonadiabatic proton-coupled electron transfer models</span></div><div class="casAuthors">Hu, Shenshen; Soudackov, Alexander V.; Hammes-Schiffer, Sharon; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Catalysis</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3569-3574</span>CODEN:
                <span class="NLM_cas:coden">ACCACS</span>;
        ISSN:<span class="NLM_cas:issn">2155-5435</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Soybean lipoxygenase (SLO) is a prototype for nonadiabatic hydrogen tunneling reactions and, as such, has served as the subject of numerous theor. studies.  Here, we report a nearly temp.-independent kinetic isotope effect (KIE) with an av. KIE value of 661 ± 27 for a double mutant (DM) of SLO at 6 temps.  The data were well-reproduced within a vibronically nonadiabatic proton-coupled electron transfer model in which the active site became rigidified compared to the wild-type enzyme and single-site mutants.  A combined temp.-pressure perturbation further showed that temp.-dependent global motions within DM-SLO were more resistant to perturbation by elevated pressure.  These findings provided strong exptl. support for the model of hydrogen tunneling in SLO, where optimization of both local protein and ligand motions and distal conformational rearrangements is a prerequisite for effective proton vibrational wave function overlap between the substrate and the active site Fe cofactor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBJL8MFFez27Vg90H21EOLACvtfcHk0lhDJ-SGtrDjBg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXmt1Kqtrg%253D&amp;md5=8595deec42ff84a7ab9122606914d275</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facscatal.7b00688&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscatal.7b00688%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DS.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DEnhanced%2520Rigidification%2520within%2520a%2520Double%2520Mutant%2520of%2520Soybean%2520Lipoxygenase%2520Provides%2520Experimental%2520Support%2520for%2520Vibronically%2520Nonadiabatic%2520Proton-Coupled%2520Electron%2520Transfer%2520Models%26jtitle%3DACS%2520Catal.%26date%3D2017%26volume%3D7%26spage%3D3569%26epage%3D3574%26doi%3D10.1021%2Facscatal.7b00688" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scouras, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, C. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alber, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Extremely Elevated Room-Temperature Kinetic Isotope Effects Quantify the Critical Role of Barrier Width in Enzymatic C-H Activation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">8157</span>– <span class="NLM_lpage">8160</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja502726s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja502726s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptVaht7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=136&amp;publication_year=2014&amp;pages=8157-8160&amp;author=S.+Huauthor=S.+C.+Sharmaauthor=A.+D.+Scourasauthor=A.+V.+Soudackovauthor=C.+A.+M.+Carrauthor=S.+Hammes-Schifferauthor=T.+Alberauthor=J.+P.+Klinman&amp;title=Extremely+Elevated+Room-Temperature+Kinetic+Isotope+Effects+Quantify+the+Critical+Role+of+Barrier+Width+in+Enzymatic+C-H+Activation&amp;doi=10.1021%2Fja502726s" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Extremely elevated room-temperature kinetic isotope effects quantify the critical role of barrier width in enzymatic C-H activation</span></div><div class="casAuthors">Hu, Shenshen; Sharma, Sudhir C.; Scouras, Alexander D.; Soudackov, Alexander V.; Carr, Cody A. Marcus; Hammes-Schiffer, Sharon; Alber, Tom; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8157-8160</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Soybean lipoxygenase (I) has served as a prototype for H-tunneling reactions, as a result of its unusual kinetic isotope effects (KIEs) and their temp. dependencies.  Here, using a synergy of kinetic, structural, and theor. studies, the authors show how the interplay between donor-acceptor distance and active site flexibility leads to catalytic behavior previously predicted by quantum tunneling theory.  Modification of the size of 2 hydrophobic residues by site-specific mutagenesis in I reduced the reaction rate 104-fold and was accompanied by an enormous and unprecedented room-temp. KIE.  Fitting of the kinetic data to a non-adiabatic model implicated an expansion of the active site that could not be compensated by donor-acceptor distance sampling.  A 1.7 Å resoln. x-ray structure of the double mutant further indicated an unaltered backbone conformation, almost identical side-chain conformations, and a significantly enlarged active site cavity.  These findings show the compelling property of room-temp. H-tunneling within a biol. context and demonstrate the very high sensitivity of such tunneling to barrier width.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKqfdvn6JAErVg90H21EOLACvtfcHk0lhDJ-SGtrDjBg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXptVaht7Y%253D&amp;md5=8f826611721093d5b32fc8f5b847aeef</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fja502726s&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja502726s%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DS.%26aulast%3DSharma%26aufirst%3DS.%2BC.%26aulast%3DScouras%26aufirst%3DA.%2BD.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DCarr%26aufirst%3DC.%2BA.%2BM.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26aulast%3DAlber%26aufirst%3DT.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DExtremely%2520Elevated%2520Room-Temperature%2520Kinetic%2520Isotope%2520Effects%2520Quantify%2520the%2520Critical%2520Role%2520of%2520Barrier%2520Width%2520in%2520Enzymatic%2520C-H%2520Activation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D8157%26epage%3D8160%26doi%3D10.1021%2Fja502726s" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offenbacher, A. R.</span></span> <span> </span><span class="NLM_article-title">Understanding Biological Hydrogen Transfer through the Lens of Temperature Dependent Kinetic Isotope Effects</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1974</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.accounts.8b00226</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.8b00226" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsF2jtbnP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=51&amp;publication_year=2018&amp;pages=1966-1974&amp;author=J.+K.+Klinmanauthor=A.+R.+Offenbacher&amp;title=Understanding+Biological+Hydrogen+Transfer+through+the+Lens+of+Temperature+Dependent+Kinetic+Isotope+Effects&amp;doi=10.1021%2Facs.accounts.8b00226" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding Biological Hydrogen Transfer Through the Lens of Temperature Dependent Kinetic Isotope Effects</span></div><div class="casAuthors">Klinman, Judith P.; Offenbacher, Adam R.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1966-1974</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogen atom transfer (HAT) is a salient feature of many enzymic C-H cleavage mechanisms.  In systems where kinetic isolation of HAT is achieved, selective labeling of substrate with hydrogen isotopes, such as deuterium, enables the detn. of intrinsic kinetic isotope effects (KIEs).  While the magnitude of the KIE is itself informative, ultimately the size of the temp. dependence of the KIE, ΔEa = Ea(D)-Ea(H), serves as a crit., and often misinterpreted (or even ignored) descriptor of the reaction coordinate.  As will be highlighted in this Accounts, ΔEa is one of the most robust parameters to emerge from studies of enzyme catalyzed hydrogen transfer.  Kinetic parameters for C-H reactions via HAT can appear consistent with either classical "over-the-barrier" or "Bell-like tunneling correction" models.  However, neither of these models is able to explain the observation of near-zero ΔEa values with many native enzymes that increase upon extrinsic or intrinsic perturbations to function.  Instead, a full tunneling model has been developed that can account for the aggregate trends in the temp. dependence of the KIE.  This model is reminiscent of Marcus-like theory for electron tunneling, with the addnl. incorporation of an H atom donor-acceptor distance (DAD) sampling term for effective wave function overlap; the role of the latter term is manifested in the exptl. detd. ΔEa.  Three enzyme systems from this lab. that illustrate different aspects of HAT are presented: taurine dioxygenase, the dual copper β-monooxygenases, and soybean lipoxygenase (SLO).  The latter provides a particularly compelling system for understanding the properties of hydrogen tunneling, showing systematic increases in ΔEa upon redn. in the size of hydrophobic residues both proximal and distal from the active site iron cofactor.  Of note, recent ENDOR-based studies of enzyme-substrate complexes with SLO indicate an increase in DAD for mutants with increased ΔEa, observations that are inconsistent with "Bell-like correction" models.  Overall, the surmounting kinetic and biophys. evidence corroborates a multidimensional approach for understanding HAT, offering a robust mechanistic explanation for the magnitude and trends of the KIE and ΔEa.  Recent DFT and QM/MM computations on SLO are compared to the developed nonadiabatic anal. constructs, providing considerable insight into ground state structures and reactivity.  However, QM/MM is unable to readily reproduce the small ΔEa values characteristic of native enzymes.  Future theor. developments to capture these exptl. observations may necessitate a parsing of protein motions for local, substrate deuteration-sensitive modes from the larger conformational landscape, in the process providing deeper understanding of how native enzymes have evolved to transiently optimize their active site configurations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP7XMNQ6YPIbVg90H21EOLACvtfcHk0li68eYOO7uKwQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhsF2jtbnP&amp;md5=fa35a3eabc0c60f7131f911866f35e9d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.8b00226&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.8b00226%26sid%3Dliteratum%253Aachs%26aulast%3DKlinman%26aufirst%3DJ.%2BK.%26aulast%3DOffenbacher%26aufirst%3DA.%2BR.%26atitle%3DUnderstanding%2520Biological%2520Hydrogen%2520Transfer%2520through%2520the%2520Lens%2520of%2520Temperature%2520Dependent%2520Kinetic%2520Isotope%2520Effects%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2018%26volume%3D51%26spage%3D1966%26epage%3D1974%26doi%3D10.1021%2Facs.accounts.8b00226" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Kinetic Detection of Orthogonal Protein and Chemical Coordinates in Enzyme Catalysis: Double Mutants of Soybean Lipoxygenase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">5447</span>– <span class="NLM_lpage">5456</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.biochem.5b00374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.5b00374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFejtbzE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=54&amp;publication_year=2015&amp;pages=5447-5456&amp;author=S.+C.+Sharmaauthor=J.+P.+Klinman&amp;title=Kinetic+Detection+of+Orthogonal+Protein+and+Chemical+Coordinates+in+Enzyme+Catalysis%3A+Double+Mutants+of+Soybean+Lipoxygenase&amp;doi=10.1021%2Facs.biochem.5b00374" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Detection of Orthogonal Protein and Chemical Coordinates in Enzyme Catalysis: Double Mutants of Soybean Lipoxygenase</span></div><div class="casAuthors">Sharma, Sudhir C.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">5447-5456</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Soybean lipoxygenase-1 (SLO-1) is a paradigmatic enzyme system for studying the contribution of hydrogen tunneling to enzymic proton-coupled electron transfer processes.  In this study, the impact of pairs of double mutants on the properties of SLO-1 is presented.  Steady-state rates and their deuterium kinetic isotope effects (KIEs) have been measured for the bimol. reaction of enzyme with free substrate (kcat/Km) and compared to the unimol. rate const., kcat.  A key kinetic finding is that the competitive KIEs on the second-order rate const. (kcat/Km) are all reduced from Dkcat and, despite large changes in rate and activation parameters, remain essentially unaltered under a variety of conditions.  These data implicate a protein reaction coordinate that is orthogonal to the chem. reaction coordinate and controls the concn. of the active enzyme.  This study introduces a new means to interrogate the alteration of conformational landscapes that can occur following site-specific mutagenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwwbzEqN7MtLVg90H21EOLACvtfcHk0li68eYOO7uKwQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXhtFejtbzE&amp;md5=c6188bedfa5918ce1313d7e3df13e987</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.5b00374&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.5b00374%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BC.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DKinetic%2520Detection%2520of%2520Orthogonal%2520Protein%2520and%2520Chemical%2520Coordinates%2520in%2520Enzyme%2520Catalysis%253A%2520Double%2520Mutants%2520of%2520Soybean%2520Lipoxygenase%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26spage%3D5447%26epage%3D5456%26doi%3D10.1021%2Facs.biochem.5b00374" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cattin-Ortola, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munos, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Hydrostatic Pressure Studies Distinguish Global from Local Protein Motions in C-H Activation by Soybean Lipoxygenase-1</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9361</span>– <span class="NLM_lpage">9364</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.201603592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1002%2Fanie.201603592" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=55&amp;publication_year=2016&amp;pages=9361-9364&amp;author=S.+Huauthor=J.+Cattin-Ortolaauthor=J.+W.+Munosauthor=J.+P.+Klinman&amp;title=Hydrostatic+Pressure+Studies+Distinguish+Global+from+Local+Protein+Motions+in+C-H+Activation+by+Soybean+Lipoxygenase-1&amp;doi=10.1002%2Fanie.201603592" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201603592&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201603592%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DS.%26aulast%3DCattin-Ortola%26aufirst%3DJ.%26aulast%3DMunos%26aufirst%3DJ.%2BW.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DHydrostatic%2520Pressure%2520Studies%2520Distinguish%2520Global%2520from%2520Local%2520Protein%2520Motions%2520in%2520C-H%2520Activation%2520by%2520Soybean%2520Lipoxygenase-1%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D9361%26epage%3D9364%26doi%3D10.1002%2Fanie.201603592" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Offenbacher, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poss, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, C. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scouras, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prigozhin, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iavarone, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alber, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Hydrogen–Deuterium Exchange of Lipoxygenase Uncovers a Relationship between Distal, Solvent Exposed Protein Motions and the Thermal Activation Barrier for Catalytic Proton-Coupled Electron Tunneling</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">570</span>– <span class="NLM_lpage">579</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscentsci.7b00142</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.7b00142" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpsFShu70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2017&amp;pages=570-579&amp;author=A.+R.+Offenbacherauthor=S.+Huauthor=E.+M.+Possauthor=C.+A.+M.+Carrauthor=A.+D.+Scourasauthor=D.+M.+Prigozhinauthor=A.+T.+Iavaroneauthor=A.+Pallaauthor=T.+Alberauthor=J.+S.+Fraserauthor=J.+P.+Klinman&amp;title=Hydrogen%E2%80%93Deuterium+Exchange+of+Lipoxygenase+Uncovers+a+Relationship+between+Distal%2C+Solvent+Exposed+Protein+Motions+and+the+Thermal+Activation+Barrier+for+Catalytic+Proton-Coupled+Electron+Tunneling&amp;doi=10.1021%2Facscentsci.7b00142" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen-Deuterium Exchange of Lipoxygenase Uncovers a Relationship between Distal, Solvent Exposed Protein Motions and the Thermal Activation Barrier for Catalytic Proton-Coupled Electron Tunneling</span></div><div class="casAuthors">Offenbacher, Adam R.; Hu, Shenshen; Poss, Erin M.; Carr, Cody A. M.; Scouras, Alexander D.; Prigozhin, Daniil M.; Iavarone, Anthony T.; Palla, Ali; Alber, Tom; Fraser, James S.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">570-579</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Defining specific pathways for efficient heat transfer from protein-solvent interfaces to their active sites represents one of the compelling and timely challenges in the quest for a phys. description of the origins of enzyme catalysis.  Enzymic hydrogen tunneling reactions constitute excellent systems in which to validate exptl. approaches to this important question, given the inherent temp. independence of quantum mech. wave function overlap.  Here, we present the application of H-D-exchange coupled to mass spectrometry toward the spatial resoln. of protein motions that can be related to enzymic catalytic parameters.  Employing the proton-coupled electron transfer reaction of soybean lipoxygenase as proof of principle, we 1st corroborated the impact of active site mutation on increased local flexibility and 2nd, uncovered a solvent-exposed loop, 15-34 Å from the reactive ferric center whose temp.-dependent motions were demonstrated to mirror the enthalpic barrier for catalytic C-H bond cleavage.  A network that connects this surface loop to the active site was structurally identified and supported by changes in kinetic parameters that resulted from site-specific mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNZ9uHrEh-gbVg90H21EOLACvtfcHk0li68eYOO7uKwQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXpsFShu70%253D&amp;md5=8881a7c990b8ce62dc0525656f09e563</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.7b00142&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.7b00142%26sid%3Dliteratum%253Aachs%26aulast%3DOffenbacher%26aufirst%3DA.%2BR.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DPoss%26aufirst%3DE.%2BM.%26aulast%3DCarr%26aufirst%3DC.%2BA.%2BM.%26aulast%3DScouras%26aufirst%3DA.%2BD.%26aulast%3DPrigozhin%26aufirst%3DD.%2BM.%26aulast%3DIavarone%26aufirst%3DA.%2BT.%26aulast%3DPalla%26aufirst%3DA.%26aulast%3DAlber%26aufirst%3DT.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DHydrogen%25E2%2580%2593Deuterium%2520Exchange%2520of%2520Lipoxygenase%2520Uncovers%2520a%2520Relationship%2520between%2520Distal%252C%2520Solvent%2520Exposed%2520Protein%2520Motions%2520and%2520the%2520Thermal%2520Activation%2520Barrier%2520for%2520Catalytic%2520Proton-Coupled%2520Electron%2520Tunneling%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2017%26volume%3D3%26spage%3D570%26epage%3D579%26doi%3D10.1021%2Facscentsci.7b00142" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horitani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offenbacher, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, C. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeke, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutsail, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, B. M.</span></span> <span> </span><span class="NLM_article-title">C-13 Endor Spectroscopy of Lipoxygenase-Substrate Complexes Reveals the Structural Basis for C-H Activation by Tunneling</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">1984</span>– <span class="NLM_lpage">1997</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.6b11856</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b11856" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVCisrg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=139&amp;publication_year=2017&amp;pages=1984-1997&amp;author=M.+Horitaniauthor=A.+R.+Offenbacherauthor=C.+A.+M.+Carrauthor=T.+Yuauthor=V.+Hoekeauthor=G.+E.+Cutsailauthor=S.+Hammes-Schifferauthor=J.+P.+Klinmanauthor=B.+M.+Hoffman&amp;title=C-13+Endor+Spectroscopy+of+Lipoxygenase-Substrate+Complexes+Reveals+the+Structural+Basis+for+C-H+Activation+by+Tunneling&amp;doi=10.1021%2Fjacs.6b11856" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">13C ENDOR Spectroscopy of Lipoxygenase-Substrate Complexes Reveals the Structural Basis for C-H Activation by Tunneling</span></div><div class="casAuthors">Horitani, Masaki; Offenbacher, Adam R.; Carr, Cody A. Marcus; Yu, Tao; Hoeke, Veronika; Cutsail, George E., III; Hammes-Schiffer, Sharon; Klinman, Judith P.; Hoffman, Brian M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1984-1997</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In enzymic C-H activation by H-tunneling, reduced barrier width is important for efficient H-wave-function overlap during catalysis.  For native enzymes displaying non-adiabatic tunneling, the dominant reactive H donor-acceptor distance (DAD) is typically ∼2.7 Å, considerably shorter than normal van der Waals distances.  Without a ground state substrate-bound structure for the prototypical nonadiabatic tunneling system, soybean lipoxygenase (SLO), it has remained unclear whether the requisite close tunneling distance occurs through an unusual ground-state active site arrangement or by thermally sampling conformational substates.  Here, the authors introduced Mn2+ as a spin-probe surrogate for the SLO Fe-ion; x-ray diffraction showed that Mn-SLO was structurally faithful to native enzyme.  13C-ENDOR then revealed the locations of 13C10 and reactive 13C11 of linoleic acid relative to the metal; 1H-ENDOR and mol. dynamics simulations of the fully solvated SLO model using ENDOR-derived restraints gave addnl. metrical information.  The resulting 3-dimensional representation of the SLO active site ground state contained a reactive conformer with a H DAD of ∼3.1 Å, approx. van der Waals contact, plus an inactive conformer with an even longer DAD, establishing that stochastic conformational sampling was required to achieve reactive tunneling geometries.  Tunneling-impaired SLO variants showed increased DADs and variations in substrate positioning and rigidity, confirming previous kinetic and theor. predictions of such behavior.  Overall, this investigation highlighted the (1) predictive power of non-adiabatic quantum treatments of proton-coupled electron transfer in SLO and (2) sensitivity of ENDOR probes to test, detect, and corroborate kinetically predicted trends in active site reactivity, and to reveal unexpected features of active site architecture.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl7b4GGYIScbVg90H21EOLACvtfcHk0lipiFyCRCuwkQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhsVCisrg%253D&amp;md5=899c3e1bdbd63671367a417ad6f28c19</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b11856&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b11856%26sid%3Dliteratum%253Aachs%26aulast%3DHoritani%26aufirst%3DM.%26aulast%3DOffenbacher%26aufirst%3DA.%2BR.%26aulast%3DCarr%26aufirst%3DC.%2BA.%2BM.%26aulast%3DYu%26aufirst%3DT.%26aulast%3DHoeke%26aufirst%3DV.%26aulast%3DCutsail%26aufirst%3DG.%2BE.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26aulast%3DHoffman%26aufirst%3DB.%2BM.%26atitle%3DC-13%2520Endor%2520Spectroscopy%2520of%2520Lipoxygenase-Substrate%2520Complexes%2520Reveals%2520the%2520Structural%2520Basis%2520for%2520C-H%2520Activation%2520by%2520Tunneling%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D1984%26epage%3D1997%26doi%3D10.1021%2Fjacs.6b11856" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohen, A.</span></span> <span> </span><span class="NLM_article-title">Hydrogen Tunneling Links Protein Dynamics to Enzyme Catalysis</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">496</span>, <span class="refDoi">&nbsp;DOI: 10.1146/annurev-biochem-051710-133623</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1146%2Fannurev-biochem-051710-133623" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=23746260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOqsr3I" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=82&amp;publication_year=2013&amp;pages=471-496&amp;author=J.+P.+Klinmanauthor=A.+Kohen&amp;title=Hydrogen+Tunneling+Links+Protein+Dynamics+to+Enzyme+Catalysis&amp;doi=10.1146%2Fannurev-biochem-051710-133623" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen tunneling links protein dynamics to enzyme catalysis</span></div><div class="casAuthors">Klinman, Judith P.; Kohen, Amnon</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">471-496</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The relationship between protein dynamics and function is a subject of considerable contemporary interest.  Although protein motions are frequently obsd. during ligand binding and release steps, the contribution of protein motions to the catalysis of bond making/breaking processes is more difficult to probe and verify.  Here, we show how the quantum mech. hydrogen tunneling assocd. with enzymic C-H bond cleavage provides a unique window into the necessity of protein dynamics for achieving optimal catalysis.  Exptl. findings support a hierarchy of thermodynamically equilibrated motions that control the H-donor and -acceptor distance and active-site electrostatics, creating an ensemble of conformations suitable for H-tunneling.  A possible extension of this view to Me transfer and other catalyzed reactions is also presented.  The impact of understanding these dynamics on the conceptual framework for enzyme activity, inhibitor/drug design, and biomimetic catalyst design is likely to be substantial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJuUE_QVx_NrVg90H21EOLACvtfcHk0lipiFyCRCuwkQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOqsr3I&amp;md5=6e24730783d3737dc346d51c2a27d473</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-051710-133623&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-051710-133623%26sid%3Dliteratum%253Aachs%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26aulast%3DKohen%26aufirst%3DA.%26atitle%3DHydrogen%2520Tunneling%2520Links%2520Protein%2520Dynamics%2520to%2520Enzyme%2520Catalysis%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2013%26volume%3D82%26spage%3D471%26epage%3D496%26doi%3D10.1146%2Fannurev-biochem-051710-133623" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kohen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartolucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Enzyme Dynamics and Hydrogen Tunnelling in a Thermophilic Alcohol Dehydrogenase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>399</i></span>,  <span class="NLM_fpage">496</span>– <span class="NLM_lpage">499</span>, <span class="refDoi">&nbsp;DOI: 10.1038/20981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1038%2F20981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10365965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=399&amp;publication_year=1999&amp;pages=496-499&amp;author=A.+Kohenauthor=R.+Cannioauthor=S.+Bartolucciauthor=J.+P.+Klinman&amp;title=Enzyme+Dynamics+and+Hydrogen+Tunnelling+in+a+Thermophilic+Alcohol+Dehydrogenase&amp;doi=10.1038%2F20981" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme dynamics and hydrogen tunnelling in a thermophilic alcohol dehydrogenase</span></div><div class="casAuthors">Kohen, Amnon; Cannio, Raffaele; Bartolucci, Simonetta; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">496-499</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Biol. catalysts (enzymes) speed up reactions by many orders of magnitude using fundamental phys. processes to increase chem. reactivity.  Hydrogen tunnelling has increasingly been found to contribute to enzyme reactions at room temp.  Tunnelling is the phenomenon by which a particle transfers through a reaction barrier as a result of its wave-like property.  In reactions involving small mols., the relative importance of tunnelling increases as the temp. is reduced.  We have now investigated whether hydrogen tunnelling occurs at elevated temps. in a biol. system that functions physiol. under such conditions.  Using a thermophilic alc. dehydrogenase (ADH), we find that hydrogen tunnelling makes a significant contribution at 65°C; this is analogous to previous findings with mesophilic ADH at 25°C.  Contrary to predictions for tunnelling through a rigid barrier, the tunnelling with the thermophilic ADH decreases at and below room temp.  These findings provide exptl. evidence for a role of thermally excited enzyme fluctuations in modulating enzyme-catalyzed bond cleavage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0tQQNwnXApLVg90H21EOLACvtfcHk0lipiFyCRCuwkQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLg%253D&amp;md5=8515907854e2ac0724ce23b545beb9a3</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2F20981&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20981%26sid%3Dliteratum%253Aachs%26aulast%3DKohen%26aufirst%3DA.%26aulast%3DCannio%26aufirst%3DR.%26aulast%3DBartolucci%26aufirst%3DS.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DEnzyme%2520Dynamics%2520and%2520Hydrogen%2520Tunnelling%2520in%2520a%2520Thermophilic%2520Alcohol%2520Dehydrogenase%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D496%26epage%3D499%26doi%3D10.1038%2F20981" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, Z. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahnson, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Impaired Protein Conformational Landscapes as Revealed in Anomalous Arrhenius Prefactors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">10520</span>– <span class="NLM_lpage">10525</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1104989108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1073%2Fpnas.1104989108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=21670258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovFentLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=108&amp;publication_year=2011&amp;pages=10520-10525&amp;author=Z.+D.+Nagelauthor=M.+Dongauthor=B.+J.+Bahnsonauthor=J.+P.+Klinman&amp;title=Impaired+Protein+Conformational+Landscapes+as+Revealed+in+Anomalous+Arrhenius+Prefactors&amp;doi=10.1073%2Fpnas.1104989108" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Impaired protein conformational landscapes as revealed in anomalous Arrhenius prefactors</span></div><div class="casAuthors">Nagel, Zachary D.; Dong, Ming; Bahnson, Brian J.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10520-10525, S10520/1-S10520/3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A growing body of data supports a role for protein motion in enzyme catalysis.  In particular, the ability of enzymes to sample catalytically relevant conformational substates has been invoked to model kinetic and spectroscopic data.  However, direct exptl. links between rapidly interconverting conformations and the chem. steps of catalysis remain rare.  We report here on the kinetic anal. and characterization of the hydride transfer step catalyzed by a series of mutant thermophilic alc. dehydrogenases (ht-ADH), presenting evidence for Arrhenius prefactor values that become enormously elevated above an expected value of approx. 1013 s-1 when the enzyme operates below its optimal temp. range.  Restoration of normal Arrhenius behavior in the ht-ADH reaction occurs at elevated temps.  A simple model, in which reduced temp. alters the ability of the ht-ADH variants to sample the catalytically relevant region of conformational space, can reproduce the available data.  These findings indicate an impaired landscape that has been generated by the combined condition of reduced temp. and mutation at a single, active-site hydrophobic side chain.  The broader implication is that optimal enzyme function requires the maintenance of a relatively smooth landscape that minimizes low energy traps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVxw5hY0gxsbVg90H21EOLACvtfcHk0lhP6RGAUz7cdQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXovFentLg%253D&amp;md5=74b719a1094871a0de2ad383a0a9dcf3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1104989108&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1104989108%26sid%3Dliteratum%253Aachs%26aulast%3DNagel%26aufirst%3DZ.%2BD.%26aulast%3DDong%26aufirst%3DM.%26aulast%3DBahnson%26aufirst%3DB.%2BJ.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DImpaired%2520Protein%2520Conformational%2520Landscapes%2520as%2520Revealed%2520in%2520Anomalous%2520Arrhenius%2520Prefactors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D10520%26epage%3D10525%26doi%3D10.1073%2Fpnas.1104989108" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohen, A.</span></span> <span> </span><span class="NLM_article-title">Extension and Limits of the Network of Coupled Motions Correlated to Hydride Transfer in Dihydrofolate Reductase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">2575</span>– <span class="NLM_lpage">2582</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja411998h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja411998h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1eitbk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=136&amp;publication_year=2014&amp;pages=2575-2582&amp;author=P.+Singhauthor=A.+Senauthor=K.+Francisauthor=A.+Kohen&amp;title=Extension+and+Limits+of+the+Network+of+Coupled+Motions+Correlated+to+Hydride+Transfer+in+Dihydrofolate+Reductase&amp;doi=10.1021%2Fja411998h" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Extension and limits of the network of coupled motions correlated to hydride transfer in dihydrofolate reductase</span></div><div class="casAuthors">Singh, Priyanka; Sen, Arundhuti; Francis, Kevin; Kohen, Amnon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2575-2582</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzyme catalysis has been studied extensively, but the role of enzyme dynamics in the catalyzed chem. conversion is still an enigma.  Dihydrofolate reductase (DHFR) is often used as a model system to assess a network of coupled motions across the protein that may affect the catalyzed chem. transformation.  Mol. dynamics simulations, quantum mech./mol. mech. studies, and bioinformatics studies have suggested the presence of a "global dynamic network" of residues in DHFR.  Earlier studies of 2 DHFR distal mutants, G121V and M42W, indicated that these residues affect the chem. step synergistically.  While this finding was in accordance with the concept of a network of functional motions across the protein, 2 residues do not constitute a network.  To better define the extent and limits of the proposed network, the current work studied 2 remote residues predicted to be part of the same network: Trp-133 and Phe-125.  The effect of mutations in these residues on the nature of the chem. step was examd. via measurements of the temp.-dependence of the intrinsic kinetic isotope effects (KIEs) and other kinetic parameters, and double mutants were used to tie the findings to Gly-121 and Met-42.  The findings indicated that residue Phe-125, which was implicated by both calcns. and bioinformatic methods, is a part of the same global dynamic network as Gly-121 and Met-42, while Trp-133, implicated only by bioinformatics, is not.  These findings extend the understanding of the proposed network and the relations between functional and genomic couplings.  Delineating that network illuminates the need to consider remote residues and protein structural dynamics in the rational design of drugs and of biomimetic catalysts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopv3iI2c9-4bVg90H21EOLACvtfcHk0lhP6RGAUz7cdQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXht1eitbk%253D&amp;md5=818cc6c8b59ac5f3804922bae67cdf4e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja411998h&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja411998h%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DP.%26aulast%3DSen%26aufirst%3DA.%26aulast%3DFrancis%26aufirst%3DK.%26aulast%3DKohen%26aufirst%3DA.%26atitle%3DExtension%2520and%2520Limits%2520of%2520the%2520Network%2520of%2520Coupled%2520Motions%2520Correlated%2520to%2520Hydride%2520Transfer%2520in%2520Dihydrofolate%2520Reductase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D2575%26epage%3D2582%26doi%3D10.1021%2Fja411998h" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Francis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohen, A.</span></span> <span> </span><span class="NLM_article-title">Protein Motions and the Activation of the Ch Bond Catalyzed by Dihydrofolate Reductase</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">24</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cbpa.2014.03.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1016%2Fj.cbpa.2014.03.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=24742825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1KqtrzP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2014&amp;pages=19-24&amp;author=K.+Francisauthor=A.+Kohen&amp;title=Protein+Motions+and+the+Activation+of+the+Ch+Bond+Catalyzed+by+Dihydrofolate+Reductase&amp;doi=10.1016%2Fj.cbpa.2014.03.009" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Protein motions and the activation of the C-H bond catalyzed by dihydrofolate reductase</span></div><div class="casAuthors">Francis, Kevin; Kohen, Amnon</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">19-24</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The role of protein motions in enzymic C-H → C transfer is an area of great contemporary debate.  An effective tool in probing such a role is the temp. dependence of the intrinsic kinetic isotope effects for the enzyme-catalyzed reaction.  The outcome of those expts. is interpreted within the context of phenomenol. Marcus-like models of hydrogen tunneling.  Here, the authors focus on recent studies of dihydrofolate reductase (DHFR) and how the role of protein motions in the catalyzed reaction has been demonstrated.  The motions in DHFR are controlled by local effects of active site residues, global effects involving remote residues across the enzyme, and appear to be preserved during the evolution of the enzyme from bacteria to humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtBmNL987ZDLVg90H21EOLACvtfcHk0lhP6RGAUz7cdQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXht1KqtrzP&amp;md5=204e9deea02d9ebdcab2d91ff37eb219</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2014.03.009&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2014.03.009%26sid%3Dliteratum%253Aachs%26aulast%3DFrancis%26aufirst%3DK.%26aulast%3DKohen%26aufirst%3DA.%26atitle%3DProtein%2520Motions%2520and%2520the%2520Activation%2520of%2520the%2520Ch%2520Bond%2520Catalyzed%2520by%2520Dihydrofolate%2520Reductase%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D21%26spage%3D19%26epage%3D24%26doi%3D10.1016%2Fj.cbpa.2014.03.009" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loveridge, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tey, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behiry, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunther, U. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crump, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allemann, R. K.</span></span> <span> </span><span class="NLM_article-title">The Role of Large-Scale Motions in Catalysis by Dihydrofolate Reductase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">20561</span>– <span class="NLM_lpage">20570</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja208844j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja208844j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVaisLnF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=133&amp;publication_year=2011&amp;pages=20561-20570&amp;author=E.+J.+Loveridgeauthor=L.+H.+Teyauthor=E.+M.+Behiryauthor=W.+M.+Dawsonauthor=R.+M.+Evansauthor=S.+B.+Whittakerauthor=U.+L.+Guntherauthor=C.+Williamsauthor=M.+P.+Crumpauthor=R.+K.+Allemann&amp;title=The+Role+of+Large-Scale+Motions+in+Catalysis+by+Dihydrofolate+Reductase&amp;doi=10.1021%2Fja208844j" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Large-Scale Motions in Catalysis by Dihydrofolate Reductase</span></div><div class="casAuthors">Loveridge, E. Joel; Tey, Lai-Hock; Behiry, Enas M.; Dawson, William M.; Evans, Rhiannon M.; Whittaker, Sara B.-M.; Gunther, Ulrich L.; Williams, Christopher; Crump, Matthew P.; Allemann, Rudolf K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">20561-20570</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dihydrofolate reductase has long been used as a model system to study the coupling of protein motions to enzymic hydride transfer.  By studying environmental effects on hydride transfer in dihydrofolate reductase (DHFR) from the cold-adapted bacterium Moritella profunda (MpDHFR) and comparing the flexibility of this enzyme to that of DHFR from Escherichia coli (EcDHFR), we demonstrate that factors that affect large-scale (i.e., long-range, but not necessarily large amplitude) protein motions have no effect on the kinetic isotope effect on hydride transfer or its temp. dependence, although the rates of the catalyzed reaction are affected.  Hydrogen/deuterium exchange studies by NMR-spectroscopy show that MpDHFR is a more flexible enzyme than EcDHFR.  NMR expts. with EcDHFR in the presence of cosolvents suggest differences in the conformational ensemble of the enzyme.  The fact that enzymes from different environmental niches and with different flexibilities display the same behavior of the kinetic isotope effect on hydride transfer strongly suggests that, while protein motions are important to generate the reaction ready conformation, an optimal conformation with the correct electrostatics and geometry for the reaction to occur, they do not influence the nature of the chem. step itself; large-scale motions do not couple directly to hydride transfer proper in DHFR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5qS2SlKFZ2rVg90H21EOLACvtfcHk0ljuNjsG496Dng"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXhsVaisLnF&amp;md5=d97e59da403072e939b6dc97ef5955c4</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fja208844j&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja208844j%26sid%3Dliteratum%253Aachs%26aulast%3DLoveridge%26aufirst%3DE.%2BJ.%26aulast%3DTey%26aufirst%3DL.%2BH.%26aulast%3DBehiry%26aufirst%3DE.%2BM.%26aulast%3DDawson%26aufirst%3DW.%2BM.%26aulast%3DEvans%26aufirst%3DR.%2BM.%26aulast%3DWhittaker%26aufirst%3DS.%2BB.%26aulast%3DGunther%26aufirst%3DU.%2BL.%26aulast%3DWilliams%26aufirst%3DC.%26aulast%3DCrump%26aufirst%3DM.%2BP.%26aulast%3DAllemann%26aufirst%3DR.%2BK.%26atitle%3DThe%2520Role%2520of%2520Large-Scale%2520Motions%2520in%2520Catalysis%2520by%2520Dihydrofolate%2520Reductase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D20561%26epage%3D20570%26doi%3D10.1021%2Fja208844j" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Bedem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samelson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alber, T.</span></span> <span> </span><span class="NLM_article-title">Accessing Protein Conformational Ensembles Using Room-Temperature X-Ray Crystallography</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">16247</span>– <span class="NLM_lpage">16252</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1111325108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1073%2Fpnas.1111325108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=21918110" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Kmtb%252FO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=108&amp;publication_year=2011&amp;pages=16247-16252&amp;author=J.+S.+Fraserauthor=H.+van+den+Bedemauthor=A.+J.+Samelsonauthor=P.+T.+Langauthor=J.+M.+Holtonauthor=N.+Echolsauthor=T.+Alber&amp;title=Accessing+Protein+Conformational+Ensembles+Using+Room-Temperature+X-Ray+Crystallography&amp;doi=10.1073%2Fpnas.1111325108" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Accessing protein conformational ensembles using room-temperature X-ray crystallography</span></div><div class="casAuthors">Fraser, James S.; van den Bedemb, H. Enry; Samelson, Avi J.; Lang, P. Therese; Holton, James M.; Echols, Nathaniel; Alber, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">16247-16252, S16247/1-S16247/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Modern protein crystal structures are based nearly exclusively on X-ray data collected at cryogenic temps. (generally 100 K).  The cooling process is thought to introduce little bias in the functional interpretation of structural results, because cryogenic temps. minimally perturb the overall protein backbone fold.  In contrast, here we show that flash cooling biases previously hidden structural ensembles in protein crystals.  By analyzing available data for 30 different proteins using new computational tools for electron-d. sampling, model refinement, and mol. packing anal., we found that crystal cryocooling remodels the conformational distributions of more than 35% of side chains and eliminates packing defects necessary for functional motions.  In the signaling switch protein, H-Ras, an allosteric network consistent with fluctuations detected in soln. by NMR was uncovered in the room-temp., but not the cryogenic, electron-d. maps.  These results expose a bias in structural databases toward smaller, overpacked, and unrealistically unique models.  Monitoring room-temp. conformational ensembles by X-ray crystallog. can reveal motions crucial for catalysis, ligand binding, and allosteric regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCOxoYI1n2J7Vg90H21EOLACvtfcHk0ljuNjsG496Dng"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXht1Kmtb%252FO&amp;md5=3208af982595fe975437459890bcb436</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1111325108&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1111325108%26sid%3Dliteratum%253Aachs%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3Dvan%2Bden%2BBedem%26aufirst%3DH.%26aulast%3DSamelson%26aufirst%3DA.%2BJ.%26aulast%3DLang%26aufirst%3DP.%2BT.%26aulast%3DHolton%26aufirst%3DJ.%2BM.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DAlber%26aufirst%3DT.%26atitle%3DAccessing%2520Protein%2520Conformational%2520Ensembles%2520Using%2520Room-Temperature%2520X-Ray%2520Crystallography%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D16247%26epage%3D16252%26doi%3D10.1073%2Fpnas.1111325108" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span>; <span class="NLM_string-name">Kenner, L. R.</span>; <span class="NLM_string-name">Warkentin, M.</span>; <span class="NLM_string-name">Woldeyes, R. A.</span>; <span class="NLM_string-name">Hopkins, J. B.</span>; <span class="NLM_string-name">Thompson, M. C.</span>; <span class="NLM_string-name">Brewster, A. S.</span>; <span class="NLM_string-name">Van Benschoten, A. H.</span>; <span class="NLM_string-name">Baxter, E. L.</span>; <span class="NLM_string-name">Uervirojnangkoorn, M.</span>; <span class="NLM_string-name">McPhillips, S. E.</span>; <span class="NLM_string-name">Song, J.</span>; <span class="NLM_string-name">Alonso-Mori, R.</span>; <span class="NLM_string-name">Holton, J. M.</span>; <span class="NLM_string-name">Weis, W. I.</span>; <span class="NLM_string-name">Brunger, A. T.</span>; <span class="NLM_string-name">Soltis, S. M.</span>; <span class="NLM_string-name">Lemke, H.</span>; <span class="NLM_string-name">Gonzalez, A.</span>; <span class="NLM_string-name">Sauter, N. K.</span>; <span class="NLM_string-name">Cohen, A. E.</span>; <span class="NLM_string-name">van den Bedem, H.</span>; <span class="NLM_string-name">Thorne, R. E.</span>; <span class="NLM_string-name">Fraser, J. S.</span></span> <span> </span><span class="NLM_article-title">Mapping the Conformational Landscape of a Dynamic Enzyme by Multitemperature and XFEL Crystallography</span>.  <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume">4</span>,<span class="refDoi">&nbsp;DOI: 10.7554/eLife.07574</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.7554%2FeLife.07574" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2015&amp;author=D.+A.+Keedy&amp;author=L.+R.+Kenner&amp;author=M.+Warkentin&amp;author=R.+A.+Woldeyes&amp;author=J.+B.+Hopkins&amp;author=M.+C.+Thompson&amp;author=A.+S.+Brewster&amp;author=A.+H.+Van+Benschoten&amp;author=E.+L.+Baxter&amp;author=M.+Uervirojnangkoorn&amp;author=S.+E.+McPhillips&amp;author=J.+Song&amp;author=R.+Alonso-Mori&amp;author=J.+M.+Holton&amp;author=W.+I.+Weis&amp;author=A.+T.+Brunger&amp;author=S.+M.+Soltis&amp;author=H.+Lemke&amp;author=A.+Gonzalez&amp;author=N.+K.+Sauter&amp;author=A.+E.+Cohen&amp;author=H.+van+den+Bedem&amp;author=R.+E.+Thorne&amp;author=J.+S.+Fraser&amp;title=Mapping+the+Conformational+Landscape+of+a+Dynamic+Enzyme+by+Multitemperature+and+XFEL+Crystallography&amp;doi=10.7554%2FeLife.07574" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.7554%2FeLife.07574&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.07574%26sid%3Dliteratum%253Aachs%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26atitle%3DMapping%2520the%2520Conformational%2520Landscape%2520of%2520a%2520Dynamic%2520Enzyme%2520by%2520Multitemperature%2520and%2520XFEL%2520Crystallography%26jtitle%3DeLife%26date%3D2015%26volume%3D4%26doi%3D10.7554%2FeLife.07574" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Bedem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivak, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petsko, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. S.</span></span> <span> </span><span class="NLM_article-title">Crystal Cryocooling Distorts Conformational Heterogeneity in a Model Michaelis Complex of Dhfr</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">899</span>– <span class="NLM_lpage">910</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.str.2014.04.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1016%2Fj.str.2014.04.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=24882744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslCgsb4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=2014&amp;pages=899-910&amp;author=D.+A.+Keedyauthor=H.+van+den+Bedemauthor=D.+A.+Sivakauthor=G.+A.+Petskoauthor=D.+Ringeauthor=M.+A.+Wilsonauthor=J.+S.+Fraser&amp;title=Crystal+Cryocooling+Distorts+Conformational+Heterogeneity+in+a+Model+Michaelis+Complex+of+Dhfr&amp;doi=10.1016%2Fj.str.2014.04.016" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal cryocooling distorts conformational heterogeneity in a model Michaelis complex of DHFR</span></div><div class="casAuthors">Keedy, Daniel A.; van den Bedem, Henry; Sivak, David A.; Petsko, Gregory A.; Ringe, Dagmar; Wilson, Mark A.; Fraser, James S.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">899-910</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Most macromol. X-ray structures are detd. from cryocooled crystals, but it is unclear whether cryocooling distorts functionally relevant flexibility.  Here we compare independently acquired pairs of high-resoln. data sets of a model Michaelis complex of dihydrofolate reductase (DHFR), collected by sep. groups at both room and cryogenic temps.  These data sets allow us to isolate the differences between exptl. procedures and between temps.  Our analyses of multiconformer models and time-averaged ensembles suggest that cryocooling suppresses and otherwise modifies side-chain and main-chain conformational heterogeneity, quenching dynamic contact networks.  Despite some idiosyncratic differences, most changes from room temp. to cryogenic temp. are conserved and likely reflect temp.-dependent solvent remodeling.  Both cryogenic data sets point to addnl. conformations not evident in the corresponding room temp. data sets, suggesting that cryocooling does not merely trap preexisting conformational heterogeneity.  Our results demonstrate that crystal cryocooling consistently distorts the energy landscape of DHFR, a paragon for understanding functional protein dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU-JIrWe57irVg90H21EOLACvtfcHk0ljuNjsG496Dng"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXpslCgsb4%253D&amp;md5=33df728ed5ba360d47dd06e46332b132</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.04.016&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.04.016%26sid%3Dliteratum%253Aachs%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3Dvan%2Bden%2BBedem%26aufirst%3DH.%26aulast%3DSivak%26aufirst%3DD.%2BA.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DRinge%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DM.%2BA.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26atitle%3DCrystal%2520Cryocooling%2520Distorts%2520Conformational%2520Heterogeneity%2520in%2520a%2520Model%2520Michaelis%2520Complex%2520of%2520Dhfr%26jtitle%3DStructure%26date%3D2014%26volume%3D22%26spage%3D899%26epage%3D910%26doi%3D10.1016%2Fj.str.2014.04.016" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slappendel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veldink, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vliegenthart, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aasa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malmstrom, B. G.</span></span> <span> </span><span class="NLM_article-title">Epr Spectroscopy of Soybean Lipoxygenase-1. Description and Quantification of the High-Spin Fe(Iii) Signals</span>. <i>Biochim. Biophys. Acta, Protein Struct.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>667</i></span>,  <span class="NLM_fpage">77</span>– <span class="NLM_lpage">86</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0005-2795(81)90068-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1016%2F0005-2795%2881%2990068-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=6260216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaL3MXnvFehtg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=667&amp;publication_year=1981&amp;pages=77-86&amp;author=S.+Slappendelauthor=G.+A.+Veldinkauthor=J.+F.+Vliegenthartauthor=R.+Aasaauthor=B.+G.+Malmstrom&amp;title=Epr+Spectroscopy+of+Soybean+Lipoxygenase-1.+Description+and+Quantification+of+the+High-Spin+Fe%28Iii%29+Signals&amp;doi=10.1016%2F0005-2795%2881%2990068-4" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">EPR spectroscopy of soybean lipoxygenase-1.  Description and quantification of the high-spin iron(III) signals</span></div><div class="casAuthors">Slappendel, Steven; Veldink, Gerrit A.; Vliegenthart, Johannes F. G.; Aasa, Roland; Malmstroem, Bo G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Protein Structure</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">667</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">77-86</span>CODEN:
                <span class="NLM_cas:coden">BBPTBH</span>;
        ISSN:<span class="NLM_cas:issn">0005-2795</span>.
    </div><div class="casAbstract">The interaction of soybean lipoxygenase-1 with 13-LS-hydroperoxy-9-cis,11-trans-octadecadienoic acid, the product of the enzymic dioxygenation of linoleic acid, results in the formation of either a yellow or purple enzyme, depending on the amt. of product used.  The compn. of the high-spin Fe(III) signals in the ESR spectra of both enzyme forms was studied and the amt. of EPR-visible Fe detd. by integration and simulation.  Sets of g values of the species building up the high-spin Fe(III) signal around g 6 are derived from both 3rd-order perturbation calcn. and exact numerical diagonalization of the spin Hamiltonian describing the system.  The Fe in the native, colorless enzyme is almost ESR-nondetectable.  The yellow form of the enzyme shows a complex ESR signal around g 6 which consists of contributions of ≥3 species with different ligand symmetry.  The signal corresponds to ∼75% of the total Fe content.  The g 6 signal of the purple Fe(III) enzyme corresponds roughly to the same amt. of Fe but the ratio between the different species is not the same as in the yellow enzyme.  This enzyme form also shows an addnl. g 4.3 signal with a large amplitude but a relatively low integrated intensity (∼10% of the total Fe content).  The results are consistent with the suggested mechanism of the catalytic function of Fe in lipoxygenase based on qual. ESR results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-yeBSG7_tULVg90H21EOLACvtfcHk0lhkEoqcuyCtGQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL3MXnvFehtg%253D%253D&amp;md5=2cb37f135de8328515e280ea89bb809a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2F0005-2795%2881%2990068-4&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2795%252881%252990068-4%26sid%3Dliteratum%253Aachs%26aulast%3DSlappendel%26aufirst%3DS.%26aulast%3DVeldink%26aufirst%3DG.%2BA.%26aulast%3DVliegenthart%26aufirst%3DJ.%2BF.%26aulast%3DAasa%26aufirst%3DR.%26aulast%3DMalmstrom%26aufirst%3DB.%2BG.%26atitle%3DEpr%2520Spectroscopy%2520of%2520Soybean%2520Lipoxygenase-1.%2520Description%2520and%2520Quantification%2520of%2520the%2520High-Spin%2520Fe%2528Iii%2529%2520Signals%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Protein%2520Struct.%26date%3D1981%26volume%3D667%26spage%3D77%26epage%3D86%26doi%3D10.1016%2F0005-2795%2881%2990068-4" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chasteen, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grady, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skorey, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neden, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riendeau, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Percival, M. D.</span></span> <span> </span><span class="NLM_article-title">Characterization of the Non-Heme Iron Center of Human 5-Lipoxygenase by Electron Paramagnetic Resonance, Fluorescence, and Ultraviolet-Visible Spectroscopy: Redox Cycling between Ferrous and Ferric States</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">9763</span>– <span class="NLM_lpage">9771</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00088a031</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00088a031" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK3sXlvFyntb4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=32&amp;publication_year=1993&amp;pages=9763-9771&amp;author=N.+D.+Chasteenauthor=J.+K.+Gradyauthor=K.+I.+Skoreyauthor=K.+J.+Nedenauthor=D.+Riendeauauthor=M.+D.+Percival&amp;title=Characterization+of+the+Non-Heme+Iron+Center+of+Human+5-Lipoxygenase+by+Electron+Paramagnetic+Resonance%2C+Fluorescence%2C+and+Ultraviolet-Visible+Spectroscopy%3A+Redox+Cycling+between+Ferrous+and+Ferric+States&amp;doi=10.1021%2Fbi00088a031" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the non-heme iron center of human 5-lipoxygenase by electron paramagnetic resonance, fluorescence, and ultraviolet-visible spectroscopy: Redox cycling between ferrous and ferric states</span></div><div class="casAuthors">Chasteen, N. Dennis; Grady, John K.; Skorey, Kathryn I.; Neden, Kevin J.; Riendeau, Denis; Percival, M. David</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">9763-71</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Purified human 5-lipoxygenase, a non-heme iron contg. enzyme, has been characterized by ESR, UV-visible and fluorescence spectroscopy.  As isolated, the enzyme is largely in the ferrous state and shows a weak X-band EPR signal extending from 0 to 700 G at 15 K, tentatively ascribed to integer spin Fe(II).  Titrn. of the protein with 13-HPOD (13-hydroperoxyoctadecadienoic acid) generates a strong multicomponent EPR signal in the g' ≈ 6 region, a yellow color assocd. with an increased absorption between 310 and 450 nm (ε330nm = 2400 M-1 cm-), and a 17% decrease in the intrinsic protein fluorescence.  The multiple component nature of the g' ≈ 6 signal indicates that the metal center in its oxidized state exists in more than one but related forms.  The g' ≈ 6 EPR signal and the yellow color reach a max. when approx. 1 mol of 13-HPOD is added/mol of iron; the resultant EPR spectrum accounts quant. for all of the iron in the protein with a signal at g' = 4.3 representing less than 3% of the total iron in the majority of samples.  Addn. of a hydroxyurea reducing agent abolished the g' ≈ 6 signal and yellow color of the protein and also reversed the decrease in fluorescence caused by the oxidant 13-HPOD.  The results indicate that the g' ≈ 6 EPR signal, the yellow color, and the decreased fluorescence are assocd. with the formation of the Fe(III) form of the enzyme.  The spectroscopic data demonstrate that the Fe(III) center of 5-lipoxygenase can be redox-cycled, consistent with its postulated role in the mechanism of oxidn. of unsatd. fatty acids to hydroperoxides.  Studies using the irreversible inhibitor 4-nitrocatechol, which binds to the ferric form of the enzyme, are also consistent with the redox cycling of the iron and demonstrate that the oxidn. state of the enzyme influences both its functional and phys. properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXVHQPqeif2rVg90H21EOLACvtfcHk0lhkEoqcuyCtGQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3sXlvFyntb4%253D&amp;md5=f1dab32ec0534608700a512fda59b3d5</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fbi00088a031&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00088a031%26sid%3Dliteratum%253Aachs%26aulast%3DChasteen%26aufirst%3DN.%2BD.%26aulast%3DGrady%26aufirst%3DJ.%2BK.%26aulast%3DSkorey%26aufirst%3DK.%2BI.%26aulast%3DNeden%26aufirst%3DK.%2BJ.%26aulast%3DRiendeau%26aufirst%3DD.%26aulast%3DPercival%26aufirst%3DM.%2BD.%26atitle%3DCharacterization%2520of%2520the%2520Non-Heme%2520Iron%2520Center%2520of%2520Human%25205-Lipoxygenase%2520by%2520Electron%2520Paramagnetic%2520Resonance%252C%2520Fluorescence%252C%2520and%2520Ultraviolet-Visible%2520Spectroscopy%253A%2520Redox%2520Cycling%2520between%2520Ferrous%2520and%2520Ferric%2520States%26jtitle%3DBiochemistry%26date%3D1993%26volume%3D32%26spage%3D9763%26epage%3D9771%26doi%3D10.1021%2Fbi00088a031" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mavrophilipos, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, K. S.</span></span> <span> </span><span class="NLM_article-title">Access of Ligands to the Ferric Center in Lipoxygenase-1</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">773</span>– <span class="NLM_lpage">783</span>, <span class="refDoi">&nbsp;DOI: 10.1016/S0006-3495(93)81438-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1016%2FS0006-3495%2893%2981438-3" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=64&amp;publication_year=1993&amp;pages=773-783&amp;author=B.+J.+Gaffneyauthor=D.+V.+Mavrophiliposauthor=K.+S.+Doctor&amp;title=Access+of+Ligands+to+the+Ferric+Center+in+Lipoxygenase-1&amp;doi=10.1016%2FS0006-3495%2893%2981438-3" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2FS0006-3495%2893%2981438-3&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-3495%252893%252981438-3%26sid%3Dliteratum%253Aachs%26aulast%3DGaffney%26aufirst%3DB.%2BJ.%26aulast%3DMavrophilipos%26aufirst%3DD.%2BV.%26aulast%3DDoctor%26aufirst%3DK.%2BS.%26atitle%3DAccess%2520of%2520Ligands%2520to%2520the%2520Ferric%2520Center%2520in%2520Lipoxygenase-1%26jtitle%3DBiophys.%2520J.%26date%3D1993%26volume%3D64%26spage%3D773%26epage%3D783%26doi%3D10.1016%2FS0006-3495%2893%2981438-3" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, Q.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavel, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigal, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, E. I.</span></span> <span> </span><span class="NLM_article-title">Epr Definition of the Non-Heme Ferric Active Sites of Mammalian 15-Lipoxygenases: Major Spectral Differences Relative to Human 5-Lipoxygenases and Plant Lipoxygenases and Their Ligand Field Origin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">7422</span>– <span class="NLM_lpage">7427</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja00133a015</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00133a015" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=117&amp;publication_year=1995&amp;pages=7422-7427&amp;author=Y.+Zhangauthor=Q.-F.+Ganauthor=E.+G.+Pavelauthor=E.+Sigalauthor=E.+I.+Solomon&amp;title=Epr+Definition+of+the+Non-Heme+Ferric+Active+Sites+of+Mammalian+15-Lipoxygenases%3A+Major+Spectral+Differences+Relative+to+Human+5-Lipoxygenases+and+Plant+Lipoxygenases+and+Their+Ligand+Field+Origin&amp;doi=10.1021%2Fja00133a015" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fja00133a015&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00133a015%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DGan%26aufirst%3DQ.-F.%26aulast%3DPavel%26aufirst%3DE.%2BG.%26aulast%3DSigal%26aufirst%3DE.%26aulast%3DSolomon%26aufirst%3DE.%2BI.%26atitle%3DEpr%2520Definition%2520of%2520the%2520Non-Heme%2520Ferric%2520Active%2520Sites%2520of%2520Mammalian%252015-Lipoxygenases%253A%2520Major%2520Spectral%2520Differences%2520Relative%2520to%2520Human%25205-Lipoxygenases%2520and%2520Plant%2520Lipoxygenases%2520and%2520Their%2520Ligand%2520Field%2520Origin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1995%26volume%3D117%26spage%3D7422%26epage%3D7427%26doi%3D10.1021%2Fja00133a015" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowling, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grove, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarrow, R. C.</span></span> <span> </span><span class="NLM_article-title">Structure and Kinetics of Formation of Catechol Complexes of Ferric Soybean Lipoxygenase-1</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">15219</span>– <span class="NLM_lpage">15229</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00046a030</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00046a030" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK2MXovFOjtbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=1995&amp;pages=15219-15229&amp;author=M.+J.+Nelsonauthor=B.+A.+Brennanauthor=D.+B.+Chaseauthor=R.+A.+Cowlingauthor=G.+N.+Groveauthor=R.+C.+Scarrow&amp;title=Structure+and+Kinetics+of+Formation+of+Catechol+Complexes+of+Ferric+Soybean+Lipoxygenase-1&amp;doi=10.1021%2Fbi00046a030" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Kinetics of Formation of Catechol Complexes of Ferric Soybean Lipoxygenase-1</span></div><div class="casAuthors">Nelson, Mark J.; Brennan, Bridget A.; Chase, D. Bruce; Cowling, Rebecca A.; Grove, Geoffrey N.; Scarrow, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">15219-29</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ferric soybean lipoxygenase forms stable complexes with 4-substituted catechols.  The structure of the complex between the enzyme and 3,4-dihydroxybenzonitrile has been studied by resonance Raman, ESR, visible, and x-ray spectroscopies.  It is a bidentate iron-catecholate complex with at least one water ligand.  The kinetics of formation of complexes between lipoxygenase and 3,4-dihydroxybenzonitrile and 3,4-dihydroxyacetophenone have been studied by stopped-flow spectroscopy.  The data are consistent with two kinetically distinct, reversible steps.  The pH dependence of the first step suggests that the substrate for the reaction is the catechol monoanion.  When these results are combined, plausible mechanisms for the complexation reaction are suggested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF6XlDfC4WJrVg90H21EOLACvtfcHk0liibmV2XBxuIQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXovFOjtbc%253D&amp;md5=87c812d60f08b04d97e80f7833b9acb9</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fbi00046a030&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00046a030%26sid%3Dliteratum%253Aachs%26aulast%3DNelson%26aufirst%3DM.%2BJ.%26aulast%3DBrennan%26aufirst%3DB.%2BA.%26aulast%3DChase%26aufirst%3DD.%2BB.%26aulast%3DCowling%26aufirst%3DR.%2BA.%26aulast%3DGrove%26aufirst%3DG.%2BN.%26aulast%3DScarrow%26aufirst%3DR.%2BC.%26atitle%3DStructure%2520and%2520Kinetics%2520of%2520Formation%2520of%2520Catechol%2520Complexes%2520of%2520Ferric%2520Soybean%2520Lipoxygenase-1%26jtitle%3DBiochemistry%26date%3D1995%26volume%3D34%26spage%3D15219%26epage%3D15229%26doi%3D10.1021%2Fbi00046a030" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pham, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jankun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skrzypczak-Jankun, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flowers, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, M. O.</span></span> <span> </span><span class="NLM_article-title">Structural and Thermochemical Characterization of Lipoxygenase-Catechol Complexes</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">17952</span>– <span class="NLM_lpage">17957</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi981989t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi981989t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK1cXns1Ckt7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=37&amp;publication_year=1998&amp;pages=17952-17957&amp;author=C.+Phamauthor=J.+Jankunauthor=E.+Skrzypczak-Jankunauthor=R.+A.+Flowersauthor=M.+O.+Funk&amp;title=Structural+and+Thermochemical+Characterization+of+Lipoxygenase-Catechol+Complexes&amp;doi=10.1021%2Fbi981989t" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Thermochemical Characterization of Lipoxygenase-Catechol Complexes</span></div><div class="casAuthors">Pham, Chau; Jankun, Jerzy; Skrzypczak-Jankun, Ewa; Flowers, Robert A., II; Funk, Max O. Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">17952-17957</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A complex between native, iron(II) soybean lipoxygenase 3 and 4-nitrocatechol, a known inhibitor of the enzyme, has been detected by isothermal titrn. calorimetry and characterized by X-ray crystallog.  The compd. moors in the central cavity of the protein close to the essential iron atom, but not in a bonding arrangement with it.  The iron ligands experience a significant rearrangement upon formation of the complex relative to their positions in the native enzyme; a water mol. becomes bound to iron in the complex, and one histidine ligand moves away from the iron to become involved in a hydrogen bonding interaction with the catechol.  These changes in position result in a trigonal pyramid coordination geometry for iron in the complex.  Mol. modeling and force field calcns. predict more than one stable complex between 4-nitrocatechol and the central cavity of lipoxygenase 3, but the interaction having the small mol. in the same orientation as the one found in the crystal structure was the most favorable.  These observations reveal specific details of the interaction between lipoxygenase and a small mol. and raise the possibility that changes in the ligand environment of the iron atom could be a feature of the product activation reaction or the catalytic mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHfShk3q3eKbVg90H21EOLACvtfcHk0liibmV2XBxuIQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1cXns1Ckt7w%253D&amp;md5=56a972ac677763307d6cc4cb4dc4bf0b</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fbi981989t&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi981989t%26sid%3Dliteratum%253Aachs%26aulast%3DPham%26aufirst%3DC.%26aulast%3DJankun%26aufirst%3DJ.%26aulast%3DSkrzypczak-Jankun%26aufirst%3DE.%26aulast%3DFlowers%26aufirst%3DR.%2BA.%26aulast%3DFunk%26aufirst%3DM.%2BO.%26atitle%3DStructural%2520and%2520Thermochemical%2520Characterization%2520of%2520Lipoxygenase-Catechol%2520Complexes%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D17952%26epage%3D17957%26doi%3D10.1021%2Fbi981989t" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, M. J.</span></span> <span> </span><span class="NLM_article-title">Catecholate Complexes of Ferric Soybean Lipoxygenase-1</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">4273</span>– <span class="NLM_lpage">4278</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00412a011</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00412a011" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaL1cXktF2gs7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=27&amp;publication_year=1988&amp;pages=4273-4278&amp;author=M.+J.+Nelson&amp;title=Catecholate+Complexes+of+Ferric+Soybean+Lipoxygenase-1&amp;doi=10.1021%2Fbi00412a011" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Catecholate complexes of ferric soybean lipoxygenase 1</span></div><div class="casAuthors">Nelson, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4273-8</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Lipoxygenases are non-heme Fe-contg. enzymes that catalyze the peroxidn. of unsatd. fatty acids contg. 1,4-dienes to yield 1,3-diene-5-hydroperoxides.  The mechanism is thought to include a ferrous-bis(allyl) radical complex as an intermediate.  The complexes formed between ferric soybean lipoxygenase 1 and a series of 4-substituted catechols were used to study the Fe environment.  At pH 7, substituted catechols with sufficiently electron-withdrawing substituents (e.g., 3,4-dihydroxybenzonitrile, 3,4-dihydroxybenzaldehyde) formed stable complexes that were best viewed as Fe3+-catechol2- on the basis of the positions and no. of charge-transfer bands in their visible spectra.  Catecholes with less electron-withdrawing substituents (e.g., catechol, 2,3-dihydroxynaphthalene) efficiently reduced the Fe.  More of the substituted catechols were competent to reduce the Fe at pH 8, and at pH 9 all of the catechols tested reduced ferric lipoxygenase.  Ests. of the pH-dependent oxidn. potentials of the catechols suggested that the redn. potential of Fe(III) in lipoxygenase was approx. +0.6 V (relative to the normal H electrode) at pH 7-9.  These catechols were slow-binding inhibitors of lipoxygenase at pH 7, with final Ki values of 6-100 μM.  The magnitude of Ki did not correlate with the electron-withdrawing character of the substituents; presumably, other effects, such as charge and polarity, are more important.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdc5KXci4es7Vg90H21EOLACvtfcHk0liibmV2XBxuIQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL1cXktF2gs7g%253D&amp;md5=e5105f78692146d428acb7eca9a4c310</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fbi00412a011&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00412a011%26sid%3Dliteratum%253Aachs%26aulast%3DNelson%26aufirst%3DM.%2BJ.%26atitle%3DCatecholate%2520Complexes%2520of%2520Ferric%2520Soybean%2520Lipoxygenase-1%26jtitle%3DBiochemistry%26date%3D1988%26volume%3D27%26spage%3D4273%26epage%3D4278%26doi%3D10.1021%2Fbi00412a011" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kemal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis-Flamberg, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krupinski-Olsen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorter, A. L.</span></span> <span> </span><span class="NLM_article-title">Reductive Inactivation of Soybean Lipoxygenase 1 by Catechols: A Possible Mechanism for Regulation of Lipoxygenase Activity</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">7064</span>– <span class="NLM_lpage">7072</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00396a031</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00396a031" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaL2sXlvFeqsLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=1987&amp;pages=7064-7072&amp;author=C.+Kemalauthor=P.+Louis-Flambergauthor=R.+Krupinski-Olsenauthor=A.+L.+Shorter&amp;title=Reductive+Inactivation+of+Soybean+Lipoxygenase+1+by+Catechols%3A+A+Possible+Mechanism+for+Regulation+of+Lipoxygenase+Activity&amp;doi=10.1021%2Fbi00396a031" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Reductive inactivation of soybean lipoxygenase 1 by catechols:  a possible mechanism for regulation of lipoxygenase activity</span></div><div class="casAuthors">Kemal, Cemal; Louis-Flamberg, Pearl; Krupinski-Olsen, Rosita; Shorter, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7064-72</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Nordihydroguaiaretic acid (NDGA), one of the most efficient inhibitors of lipoxygenases, is shown, by EPR, CD, and fluorescence studies, to reduce the catalytically active ferric soybean lipoxygenase 1 (Eox) to the inactive ferrous form (Ered).  In decreasing order of reactivity, the following also reduced Eox: catechol &gt; hydroquinone &gt; esculetin ≈ norepinephrine &gt; 2,6-di-tert-butyl-4-methylphenol &gt; caffeic acid &gt; α-tocopherol &gt; dithiothreitol.  The redn. of Eox by NDGA (k = 8.1 × 106 M-1 s-1, pH 9.0, 25°) was almost as fast as the Eox-catalyzed conversion of linoleate (LH) to 13(S)-hydroperoxy-9(Z),11(E)-octadecadienoate (LOOH) and the oxidn. of Ered by LOOH to give Eox.  Thus, NDGA can efficiently inhibit the Eox-catalyzed conversion of LH to LOOH by reducing Eox to the inactive Ered, thereby diminishing the turnover rate.  Lipoxygenase catalyzed the oxidn. of NDGA by LOOH at a rate that was consistent with the independently detd. rate const. for the redn. of Eox by NDGA.  All 4 reducing equiv from the 2 catechol groups in NDGA could be utilized in the redn. of Eox, leading to the consumption of 4 mol LOOH/mol NDGA initially present.  Because the catalytically inactive Ered was oxidized by fatty acid hydroperoxides (e.g., LOOH) to give the active Eox, reducing agents such as NDGA were most effective as lipoxygenase inhibitors at low hydroperoxide concns.  The results suggested that in vivo, where lipid hydroperoxides are maintained at low steady-state levels, redn. of lipoxygenase from the ferric to ferrous state may be important in the regulation of lipoxygenase activity and hence leukotriene biosynthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9wPoiujbK87Vg90H21EOLACvtfcHk0liibmV2XBxuIQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL2sXlvFeqsLw%253D&amp;md5=fdf018c23827865699dfc617044d7e54</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fbi00396a031&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00396a031%26sid%3Dliteratum%253Aachs%26aulast%3DKemal%26aufirst%3DC.%26aulast%3DLouis-Flamberg%26aufirst%3DP.%26aulast%3DKrupinski-Olsen%26aufirst%3DR.%26aulast%3DShorter%26aufirst%3DA.%2BL.%26atitle%3DReductive%2520Inactivation%2520of%2520Soybean%2520Lipoxygenase%25201%2520by%2520Catechols%253A%2520A%2520Possible%2520Mechanism%2520for%2520Regulation%2520of%2520Lipoxygenase%2520Activity%26jtitle%3DBiochemistry%26date%3D1987%26volume%3D26%26spage%3D7064%26epage%3D7072%26doi%3D10.1021%2Fbi00396a031" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holman, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, E. I.</span></span> <span> </span><span class="NLM_article-title">Spectroscopic and Functional Characterization of a Ligand Coordination Mutant of Soybean Lipoxygenase-1: First Coordination Sphere Analogue of Human 15-Lipoxygenase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">12564</span>– <span class="NLM_lpage">12572</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja982844c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja982844c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=120&amp;publication_year=1998&amp;pages=12564-12572&amp;author=T.+R.+Holmanauthor=J.+Zhouauthor=E.+I.+Solomon&amp;title=Spectroscopic+and+Functional+Characterization+of+a+Ligand+Coordination+Mutant+of+Soybean+Lipoxygenase-1%3A+First+Coordination+Sphere+Analogue+of+Human+15-Lipoxygenase&amp;doi=10.1021%2Fja982844c" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fja982844c&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja982844c%26sid%3Dliteratum%253Aachs%26aulast%3DHolman%26aufirst%3DT.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DSolomon%26aufirst%3DE.%2BI.%26atitle%3DSpectroscopic%2520and%2520Functional%2520Characterization%2520of%2520a%2520Ligand%2520Coordination%2520Mutant%2520of%2520Soybean%2520Lipoxygenase-1%253A%2520First%2520Coordination%2520Sphere%2520Analogue%2520of%2520Human%252015-Lipoxygenase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D12564%26epage%3D12572%26doi%3D10.1021%2Fja982844c" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bellissent-Funel, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassanali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Havenith, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henchman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterpone, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A. E.</span></span> <span> </span><span class="NLM_article-title">Water Determines the Structure and Dynamics of Proteins</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">7673</span>– <span class="NLM_lpage">97</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.chemrev.5b00664</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00664" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFeqsrw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=116&amp;publication_year=2016&amp;pages=7673-97&amp;author=M.+C.+Bellissent-Funelauthor=A.+Hassanaliauthor=M.+Havenithauthor=R.+Henchmanauthor=P.+Pohlauthor=F.+Sterponeauthor=D.+van+der+Spoelauthor=Y.+Xuauthor=A.+E.+Garcia&amp;title=Water+Determines+the+Structure+and+Dynamics+of+Proteins&amp;doi=10.1021%2Facs.chemrev.5b00664" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Water Determines the Structure and Dynamics of Proteins</span></div><div class="casAuthors">Bellissent-Funel, Marie-Claire; Hassanali, Ali; Havenith, Martina; Henchman, Richard; Pohl, Peter; Sterpone, Fabio; van der Spoel, David; Xu, Yao; Garcia, Angel E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">7673-7697</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water is an essential participant in the stability, structure, dynamics, and function of proteins and other biomols.  Thermodynamically, changes in the aq. environment affect the stability of biomols.  Structurally, water participates chem. in the catalytic function of proteins and nucleic acids and phys. in the collapse of the protein chain during folding through hydrophobic collapse and mediates binding through the hydrogen bond in complex formation.  Water is a partner that slaves the dynamics of proteins, and water interaction with proteins affect their dynamics.  Here we provide a review of the exptl. and computational advances over the past decade in understanding the role of water in the dynamics, structure, and function of proteins.  We focus on the combination of X-ray and neutron crystallog., NMR, terahertz spectroscopy, mass spectroscopy, thermodn., and computer simulations to reveal how water assist proteins in their function.  The recent advances in computer simulations and the enhanced sensitivity of exptl. tools promise major advances in the understanding of protein dynamics, and water surely will be a protagonist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5TmeeJWenT7Vg90H21EOLACvtfcHk0lhZi0_7XWiZoA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XnvFeqsrw%253D&amp;md5=365764ba66b0f0b2f94d1d401ee4d691</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00664&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00664%26sid%3Dliteratum%253Aachs%26aulast%3DBellissent-Funel%26aufirst%3DM.%2BC.%26aulast%3DHassanali%26aufirst%3DA.%26aulast%3DHavenith%26aufirst%3DM.%26aulast%3DHenchman%26aufirst%3DR.%26aulast%3DPohl%26aufirst%3DP.%26aulast%3DSterpone%26aufirst%3DF.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DGarcia%26aufirst%3DA.%2BE.%26atitle%3DWater%2520Determines%2520the%2520Structure%2520and%2520Dynamics%2520of%2520Proteins%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D7673%26epage%3D97%26doi%3D10.1021%2Facs.chemrev.5b00664" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abad, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camps, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabrias, G.</span></span> <span> </span><span class="NLM_article-title">Is Hydrogen Tunneling Involved in Acylcoa Desaturase Reactions? The Case of a Delta(9) Desaturase That Transforms (E)-11-Tetradecenoic Acid into (Z,E)-9,11-Tetradecadienoic Acid</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">3279</span>– <span class="NLM_lpage">3281</span>, <span class="refDoi">&nbsp;DOI: 10.1002/1521-3773(20000915)39:18&lt;3279::AID-ANIE3279&gt;3.0.CO;2-G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1002%2F1521-3773%2820000915%2939%3A18%3C3279%3A%3AAID-ANIE3279%3E3.0.CO%3B2-G" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=2000&amp;pages=3279-3281&amp;author=J.+L.+Abadauthor=F.+Campsauthor=G.+Fabrias&amp;title=Is+Hydrogen+Tunneling+Involved+in+Acylcoa+Desaturase+Reactions%3F+The+Case+of+a+Delta%289%29+Desaturase+That+Transforms+%28E%29-11-Tetradecenoic+Acid+into+%28Z%2CE%29-9%2C11-Tetradecadienoic+Acid&amp;doi=10.1002%2F1521-3773%2820000915%2939%3A18%3C3279%3A%3AAID-ANIE3279%3E3.0.CO%3B2-G" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820000915%2939%3A18%3C3279%3A%3AAID-ANIE3279%3E3.0.CO%3B2-G&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820000915%252939%253A18%253C3279%253A%253AAID-ANIE3279%253E3.0.CO%253B2-G%26sid%3Dliteratum%253Aachs%26aulast%3DAbad%26aufirst%3DJ.%2BL.%26aulast%3DCamps%26aufirst%3DF.%26aulast%3DFabrias%26aufirst%3DG.%26atitle%3DIs%2520Hydrogen%2520Tunneling%2520Involved%2520in%2520Acylcoa%2520Desaturase%2520Reactions%253F%2520The%2520Case%2520of%2520a%2520Delta%25289%2529%2520Desaturase%2520That%2520Transforms%2520%2528E%2529-11-Tetradecenoic%2520Acid%2520into%2520%2528Z%252CE%2529-9%252C11-Tetradecadienoic%2520Acid%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2000%26volume%3D39%26spage%3D3279%26epage%3D3281%26doi%3D10.1002%2F1521-3773%2820000915%2939%3A18%3C3279%3A%3AAID-ANIE3279%3E3.0.CO%3B2-G" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biela, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasief, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Dissecting the Hydrophobic Effect on the Molecular Level: The Role of Water, Enthalpy, and Entropy in Ligand Binding to Thermolysin</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1822</span>– <span class="NLM_lpage">1828</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.201208561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1002%2Fanie.201208561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFSitw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=2013&amp;pages=1822-1828&amp;author=A.+Bielaauthor=N.+N.+Nasiefauthor=M.+Betzauthor=A.+Heineauthor=D.+Hangauerauthor=G.+Klebe&amp;title=Dissecting+the+Hydrophobic+Effect+on+the+Molecular+Level%3A+The+Role+of+Water%2C+Enthalpy%2C+and+Entropy+in+Ligand+Binding+to+Thermolysin&amp;doi=10.1002%2Fanie.201208561" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Dissecting the Hydrophobic Effect on the Molecular Level: The Role of Water, Enthalpy, and Entropy in Ligand Binding to Thermolysin</span></div><div class="casAuthors">Biela, Adam; Nasief, Nader N.; Betz, Michael; Heine, Andreas; Hangauer, David; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1822-1828</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH &amp; Co. KGaA</span>)
        </div><div class="casAbstract">The hydrophobic effect is assocd. with the successive replacement of water mols. in the binding site of a protein by hydrophobic groups of the ligand.  Although the hydrophobic effect is assumed to be entropy-driven, large changes in enthalpy and entropy are obsd. with the model system thermolysin.  Structural changes in the binding features of the water mols. ultimately det. the thermodn. of the hydrophobic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYVn8gd7ufubVg90H21EOLACvtfcHk0lhZi0_7XWiZoA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXjtFSitw%253D%253D&amp;md5=d9fa6a9a10b9f4ed78e68436bfc50c45</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fanie.201208561&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201208561%26sid%3Dliteratum%253Aachs%26aulast%3DBiela%26aufirst%3DA.%26aulast%3DNasief%26aufirst%3DN.%2BN.%26aulast%3DBetz%26aufirst%3DM.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DHangauer%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DDissecting%2520the%2520Hydrophobic%2520Effect%2520on%2520the%2520Molecular%2520Level%253A%2520The%2520Role%2520of%2520Water%252C%2520Enthalpy%252C%2520and%2520Entropy%2520in%2520Ligand%2520Binding%2520to%2520Thermolysin%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D1822%26epage%3D1828%26doi%3D10.1002%2Fanie.201208561" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mecinovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moustakas, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, S. W.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mack, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockett, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heroux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitesides, G. M.</span></span> <span> </span><span class="NLM_article-title">Mechanism of the Hydrophobic Effect in the Biomolecular Recognition of Arylsulfonamides by Carbonic Anhydrase</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">17889</span>– <span class="NLM_lpage">17894</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1114107108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1073%2Fpnas.1114107108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=22011572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOktLvK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=108&amp;publication_year=2011&amp;pages=17889-17894&amp;author=P.+W.+Snyderauthor=J.+Mecinovicauthor=D.+T.+Moustakasauthor=S.+W.+Thomasauthor=M.+Harderauthor=E.+T.+Mackauthor=M.+R.+Lockettauthor=A.+Herouxauthor=W.+Shermanauthor=G.+M.+Whitesides&amp;title=Mechanism+of+the+Hydrophobic+Effect+in+the+Biomolecular+Recognition+of+Arylsulfonamides+by+Carbonic+Anhydrase&amp;doi=10.1073%2Fpnas.1114107108" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of the hydrophobic effect in the biomolecular recognition of arylsulfonamides by carbonic anhydrase</span></div><div class="casAuthors">Snyder, Phillip W.; Mecinoviv, Jasmin; Moustakas, Demetri T.; Thomas, Samuel W., III; Harder, Michael; Mack, Eric T.; Lockett, Matthew R.; Heroux, Annie; Sherman, Woody; Whitesides, George M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17889-17894, S17889/1-S17889/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The hydrophobic effect - a rationalization of the insoly. of nonpolar mols. in water-is centrally important to biomol. recognition.  Despite extensive research devoted to the hydrophobic effect, its mol. mechanisms remain controversial, and there are still no reliably predictive models for its role in protein-ligand binding.  Here we describe a particularly well-defined system of protein and ligands - carbonic anhydrase and a series of structurally homologous heterocyclic arom. sulfonamides - that we use to characterize hydrophobic interactions thermodynamically and structurally.  In binding to this structurally rigid protein, a set of ligands (also defined to be structurally rigid) shows the expected gain in binding free energy as hydrophobic surface area is added.  Isothermal titrn. calorimetry demonstrates that enthalpy dets. these increases in binding affinity, and that changes in the heat capacity of binding are. neg.  X-ray crystallog. and mol. dynamics simulations are compatible with the proposal that the differences in binding between the homologous ligands stem from changes in the no. and organization of water mols. localized in the active site in the bound complexes, rather than (or perhaps in addn. to) release of structured water from the apposed hydrophobic surfaces.  These results support the hypothesis that structured water mols. - including both the mols. of water displaced by the ligands and those reorganized upon ligand binding - det. the thermodn. of binding of these ligands at the active site of the protein.  Hydrophobic effects in various contexts have different structural and thermodn. origins, although all may be manifestations of the differences in characteristics of bulk water and water close to hydrophobic surfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdykl8V0IikrVg90H21EOLACvtfcHk0lhbeL6zGLxLUA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOktLvK&amp;md5=ba721fce93bd15d5028c8426c017897d</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1114107108&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1114107108%26sid%3Dliteratum%253Aachs%26aulast%3DSnyder%26aufirst%3DP.%2BW.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DMoustakas%26aufirst%3DD.%2BT.%26aulast%3DThomas%26aufirst%3DS.%2BW.%26aulast%3DHarder%26aufirst%3DM.%26aulast%3DMack%26aufirst%3DE.%2BT.%26aulast%3DLockett%26aufirst%3DM.%2BR.%26aulast%3DHeroux%26aufirst%3DA.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DWhitesides%26aufirst%3DG.%2BM.%26atitle%3DMechanism%2520of%2520the%2520Hydrophobic%2520Effect%2520in%2520the%2520Biomolecular%2520Recognition%2520of%2520Arylsulfonamides%2520by%2520Carbonic%2520Anhydrase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D17889%26epage%3D17894%26doi%3D10.1073%2Fpnas.1114107108" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chervenak, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toone, E. J.</span></span> <span> </span><span class="NLM_article-title">A Direct Measure of the Contribution of Solvent Reorganization to the Enthalpy of Binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">10533</span>– <span class="NLM_lpage">10539</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja00102a021</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00102a021" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK2MXhslyisrs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=116&amp;publication_year=1994&amp;pages=10533-10539&amp;author=M.+C.+Chervenakauthor=E.+J.+Toone&amp;title=A+Direct+Measure+of+the+Contribution+of+Solvent+Reorganization+to+the+Enthalpy+of+Binding&amp;doi=10.1021%2Fja00102a021" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A Direct Measure of the Contribution of Solvent Reorganization to the Enthalpy of Binding</span></div><div class="casAuthors">Chervenak, Mary C.; Toone, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10533-9</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">The thermodn. of assocn. of several binding systems, including protein-carbohydrate, small mol.-small mol., protein-peptide, and protein-nucleic acid, were evaluated calorimetrically in light and heavy water.  In every case, the enthalpy of binding in D2O was decreased relative to that in H2O: the differences range from 400 to 1800 cal mol-∞1.  A compensating change in ΔS left the free energy of binding virtually unchanged in each case.  A strong correlation between the differential enthalpy of binding and ΔCp for binding was obsd., with a slope of 5 K.  An anal. of the obsd. effect utilizing a Born-Haber thermodn. cycle shows that the measured decrease in enthalpy represents approx. 10% of the binding enthalpy arising from solvent reorganization.  For the range of systems investigated, solvent reorganization provides 25-100% of the obsd. enthalpy of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxen1jg8NXYLVg90H21EOLACvtfcHk0lhbeL6zGLxLUA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXhslyisrs%253D&amp;md5=fc6a558ad9e76aa7b7c28ac930f58510</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fja00102a021&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00102a021%26sid%3Dliteratum%253Aachs%26aulast%3DChervenak%26aufirst%3DM.%2BC.%26aulast%3DToone%26aufirst%3DE.%2BJ.%26atitle%3DA%2520Direct%2520Measure%2520of%2520the%2520Contribution%2520of%2520Solvent%2520Reorganization%2520to%2520the%2520Enthalpy%2520of%2520Binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1994%26volume%3D116%26spage%3D10533%26epage%3D10539%26doi%3D10.1021%2Fja00102a021" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barillari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essex, J. W.</span></span> <span> </span><span class="NLM_article-title">Classification of Water Molecules in Protein Binding Sites</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">2577</span>– <span class="NLM_lpage">2587</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja066980q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja066980q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsFGjt7o%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=129&amp;publication_year=2007&amp;pages=2577-2587&amp;author=C.+Barillariauthor=J.+Taylorauthor=R.+Vinerauthor=J.+W.+Essex&amp;title=Classification+of+Water+Molecules+in+Protein+Binding+Sites&amp;doi=10.1021%2Fja066980q" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of Water Molecules in Protein Binding Sites</span></div><div class="casAuthors">Barillari, Caterina; Taylor, Justine; Viner, Russell; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2577-2587</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water mols. play a crucial role in mediating the interaction between a ligand and a macromol. receptor.  An understanding of the nature and role of each water mol. in the active site of a protein could greatly increase the efficiency of rational drug design approaches: if the propensity of a water mol. for displacement can be detd., then synthetic effort may be most profitably applied to the design of specific ligands with the displacement of this water mol. in mind.  In this paper, a thermodn. anal. of water mols. in the binding sites of six proteins, each complexed with a no. of inhibitors, is presented.  Two classes of water mols. were identified: those conserved and not displaced by any of the ligands, and those that are displaced by some ligands.  The abs. binding free energies of 54 water mols. were calcd. using the double decoupling method, with replica exchange thermodn. integration in Monte Carlo simulations.  It was found that conserved water mols. are on av. more tightly bound than displaced water mols.  In addn., Bayesian statistics is used to calc. the probability that a particular water mol. may be displaced by an appropriately designed ligand, given the calcd. binding free energy of the water mol.  This approach therefore allows the numerical assessment of whether or not a given water mol. should be targeted for displacement as part of a rational drug design strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr38ETxUkD-NrVg90H21EOLACvtfcHk0lhbeL6zGLxLUA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXhsFGjt7o%253D&amp;md5=e74664f19993a9e9c087dbb9c1d54c3e</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fja066980q&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja066980q%26sid%3Dliteratum%253Aachs%26aulast%3DBarillari%26aufirst%3DC.%26aulast%3DTaylor%26aufirst%3DJ.%26aulast%3DViner%26aufirst%3DR.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DClassification%2520of%2520Water%2520Molecules%2520in%2520Protein%2520Binding%2520Sites%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D2577%26epage%3D2587%26doi%3D10.1021%2Fja066980q" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2817</span>– <span class="NLM_lpage">2831</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=130&amp;publication_year=2008&amp;pages=2817-2831&amp;author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&amp;title=Role+of+the+Active-Site+Solvent+in+the+Thermodynamics+of+Factor+Xa+Ligand+Binding&amp;doi=10.1021%2Fja0771033" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span></div><div class="casAuthors">Abel, Robert; Young, Tom; Farid, Ramy; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2817-2831</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the underlying physics of the binding of small-mol. ligands to protein active sites is a key objective of computational chem. and biol.  It is widely believed that displacement of water mols. from the active site by the ligand is a principal (if not the dominant) source of binding free energy.  Although continuum theories of hydration are routinely used to describe the contributions of the solvent to the binding affinity of the complex, it is still an unsettled question as to whether or not these continuum solvation theories describe the underlying mol. physics with sufficient accuracy to reliably rank the binding affinities of a set of ligands for a given protein.  Here we develop a novel, computationally efficient descriptor of the contribution of the solvent to the binding free energy of a small mol. and its assocd. receptor that captures the effects of the ligand displacing the solvent from the protein active site with at. detail.  This descriptor quant. predicts (R2 = 0.81) the binding free energy differences between congeneric ligand pairs for the test system factor Xa, elucidates phys. properties of the active site solvent that appear to be missing in most continuum theories of hydration, and identifies several features of the hydration of the factor Xa active site relevant to the structure-activity relationship of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS42r59nNXH7Vg90H21EOLACvtfcHk0li8W30jlJgcWQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D&amp;md5=8eefe76547a82bccaea38f2c84e6cd5f</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DRole%2520of%2520the%2520Active-Site%2520Solvent%2520in%2520the%2520Thermodynamics%2520of%2520Factor%2520Xa%2520Ligand%2520Binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Basran, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutcliffe, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scrutton, N. S.</span></span> <span> </span><span class="NLM_article-title">Enzymatic H-Transfer Requires Vibration-Driven Extreme Tunneling</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">3218</span>– <span class="NLM_lpage">3222</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi982719d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi982719d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaK1MXhtFClurc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=38&amp;publication_year=1999&amp;pages=3218-3222&amp;author=J.+Basranauthor=M.+J.+Sutcliffeauthor=N.+S.+Scrutton&amp;title=Enzymatic+H-Transfer+Requires+Vibration-Driven+Extreme+Tunneling&amp;doi=10.1021%2Fbi982719d" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Enzymic H-Transfer Requires Vibration-Driven Extreme Tunneling</span></div><div class="casAuthors">Basran, Jaswir; Sutcliffe, Michael J.; Scrutton, Nigel S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3218-3222</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymic breakage of the substrate C-H bond by Methylophilus methyltrophus (sp. W3A1) methylamine dehydrogenase (MADH) has been studied by stopped-flow spectroscopy.  The rate of redn. of the tryptophan tryptophylquinone (TTQ) cofactor has a large kinetic isotope effect (KIE = 16.8 ± 0.5), and the KIE is independent of temp.  Anal. of the temp. dependence of C-H bond breakage revealed that extreme (ground state) quantum tunneling is responsible for the transfer of the hydrogen nucleus.  Reaction rates are strongly dependent on temp., indicating thermally induced, vibrational motion drives the H-transfer reaction.  The data provide direct exptl. evidence for enzymic bond breakage by extreme tunneling driven by vibrational motion of the protein scaffold.  The results demonstrate that classical transition state theory and its tunneling derivs. do not adequately describe this enzymic reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry9BaMERdHkrVg90H21EOLACvtfcHk0li8W30jlJgcWQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1MXhtFClurc%253D&amp;md5=0eafc6b49552f43d1af2ef6968cea7b9</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fbi982719d&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi982719d%26sid%3Dliteratum%253Aachs%26aulast%3DBasran%26aufirst%3DJ.%26aulast%3DSutcliffe%26aufirst%3DM.%2BJ.%26aulast%3DScrutton%26aufirst%3DN.%2BS.%26atitle%3DEnzymatic%2520H-Transfer%2520Requires%2520Vibration-Driven%2520Extreme%2520Tunneling%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D3218%26epage%3D3222%26doi%3D10.1021%2Fbi982719d" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Francisco, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinman, J. P.</span></span> <span> </span><span class="NLM_article-title">Hydrogen Tunneling in Peptidylglycine Alpha-Hydroxylating Monooxygenase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">8194</span>– <span class="NLM_lpage">8195</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja025758s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja025758s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD38XksFOqsr8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=124&amp;publication_year=2002&amp;pages=8194-8195&amp;author=W.+A.+Franciscoauthor=M.+J.+Knappauthor=N.+J.+Blackburnauthor=J.+P.+Klinman&amp;title=Hydrogen+Tunneling+in+Peptidylglycine+Alpha-Hydroxylating+Monooxygenase&amp;doi=10.1021%2Fja025758s" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen Tunneling in Peptidylglycine α-Hydroxylating Monooxygenase</span></div><div class="casAuthors">Francisco, Wilson A.; Knapp, Michael J.; Blackburn, Ninian J.; Klinman, Judith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">8194-8195</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The temp. dependence of the primary and secondary intrinsic isotope effects for the C-H bond cleavage catalyzed by peptidylglycine α-hydroxylating monooxygenase has been detd.  Anal. of the magnitude and Arrhenius behavior of the intrinsic isotope effects provides strong evidence for the use of tunneling as a primary catalytic strategy for this enzyme.  Modeling of the isotope effect data allows for a comparison to the hydrogen transfer catalyzed by soybean lipoxygenase in terms of environmental reorganization energy and frequency of the protein vibration that controls the hydrogen transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCf6jrSC5J57Vg90H21EOLACvtfcHk0li8W30jlJgcWQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD38XksFOqsr8%253D&amp;md5=33cc4af7f3fb7ce66abea707ecb28326</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fja025758s&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja025758s%26sid%3Dliteratum%253Aachs%26aulast%3DFrancisco%26aufirst%3DW.%2BA.%26aulast%3DKnapp%26aufirst%3DM.%2BJ.%26aulast%3DBlackburn%26aufirst%3DN.%2BJ.%26aulast%3DKlinman%26aufirst%3DJ.%2BP.%26atitle%3DHydrogen%2520Tunneling%2520in%2520Peptidylglycine%2520Alpha-Hydroxylating%2520Monooxygenase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2002%26volume%3D124%26spage%3D8194%26epage%3D8195%26doi%3D10.1021%2Fja025758s" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sikorski, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markham, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopalan, P. T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benkovic, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohen, A.</span></span> <span> </span><span class="NLM_article-title">Tunneling and Coupled Motion in the Escherichia Coli Dihydrofolate Reductase Catalysis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">4778</span>– <span class="NLM_lpage">4779</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja031683w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja031683w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisVygsLY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=126&amp;publication_year=2004&amp;pages=4778-4779&amp;author=R.+S.+Sikorskiauthor=L.+Wangauthor=K.+A.+Markhamauthor=P.+T.+R.+Rajagopalanauthor=S.+J.+Benkovicauthor=A.+Kohen&amp;title=Tunneling+and+Coupled+Motion+in+the+Escherichia+Coli+Dihydrofolate+Reductase+Catalysis&amp;doi=10.1021%2Fja031683w" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Tunneling and Coupled Motion in the Escherichia coli Dihydrofolate Reductase Catalysis</span></div><div class="casAuthors">Sikorski, R. Steven; Wang, Lin; Markham, Kelli A.; Rajagopalan, P. T. Ravi; Benkovic, Stephen J.; Kohen, Amnon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4778-4779</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">H-transfer was studied in the complex kinetic cascade of dihydrofolate reductase.  Intrinsic kinetic isotope effects, their temp. dependence, and other temp.-dependent parameters indicated H-tunneling, but no 1° to 2° coupled motion.  The data also suggested environmentally coupled tunneling and commitment to catalysis on pre-steady-state isotope effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfUWlPsgr2t7Vg90H21EOLACvtfcHk0li8W30jlJgcWQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2cXisVygsLY%253D&amp;md5=99cf286374e540ecbc8cfc0e2f195a4b</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fja031683w&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja031683w%26sid%3Dliteratum%253Aachs%26aulast%3DSikorski%26aufirst%3DR.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMarkham%26aufirst%3DK.%2BA.%26aulast%3DRajagopalan%26aufirst%3DP.%2BT.%2BR.%26aulast%3DBenkovic%26aufirst%3DS.%2BJ.%26aulast%3DKohen%26aufirst%3DA.%26atitle%3DTunneling%2520and%2520Coupled%2520Motion%2520in%2520the%2520Escherichia%2520Coli%2520Dihydrofolate%2520Reductase%2520Catalysis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D4778%26epage%3D4779%26doi%3D10.1021%2Fja031683w" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Hydrogen Tunneling and Protein Motion in Enzyme Reactions</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">100</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar040199a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar040199a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVCrsLY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=2006&amp;pages=93-100&amp;author=S.+Hammes-Schiffer&amp;title=Hydrogen+Tunneling+and+Protein+Motion+in+Enzyme+Reactions&amp;doi=10.1021%2Far040199a" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen tunneling and protein motion in enzyme reactions</span></div><div class="casAuthors">Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">93-100</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Theor. perspectives on H-transfer reactions in enzymes are presented.  The proton-coupled electron transfer reaction catalyzed by soybean lipoxygenase and the hydride transfer reaction catalyzed by dihydrofolate reductase are discussed.  The 1st reaction is nonadiabatic and involves 2 distinct electronic states, whereas the 2nd reaction is predominantly adiabatic and occurs on the electronic ground state.  Theor. studies indicate that H-tunneling and protein motion play significant roles in both reactions.  In both cases, the proton donor-acceptor distance decreases relative to its equil. value to enable efficient H-tunneling.  Equil. thermal motions of the protein lead to conformational changes that facilitate H-transfer, but the nonequil. dynamical aspects of these motions have negligible impact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGordjIcMLgKErVg90H21EOLACvtfcHk0lhGN9FCeNJstg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXmvVCrsLY%253D&amp;md5=4eb45fbf92dd7b07f84bd9e34f9de972</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Far040199a&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far040199a%26sid%3Dliteratum%253Aachs%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DHydrogen%2520Tunneling%2520and%2520Protein%2520Motion%2520in%2520Enzyme%2520Reactions%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2006%26volume%3D39%26spage%3D93%26epage%3D100%26doi%3D10.1021%2Far040199a" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Fundamental Insights into Proton-Coupled Electron Transfer in Soybean Lipoxygenase from Quantum Mechanical/Molecular Mechanical Free Energy Simulations</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">3068</span>– <span class="NLM_lpage">3076</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.7b13642</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b13642" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Wisbo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=140&amp;publication_year=2018&amp;pages=3068-3076&amp;author=P.+Liauthor=A.+V.+Soudackovauthor=S.+Hammes-Schiffer&amp;title=Fundamental+Insights+into+Proton-Coupled+Electron+Transfer+in+Soybean+Lipoxygenase+from+Quantum+Mechanical%2FMolecular+Mechanical+Free+Energy+Simulations&amp;doi=10.1021%2Fjacs.7b13642" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Fundamental Insights into Proton-Coupled Electron Transfer in Soybean Lipoxygenase from Quantum Mechanical/Molecular Mechanical Free Energy Simulations</span></div><div class="casAuthors">Li, Pengfei; Soudackov, Alexander V.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3068-3076</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The proton-coupled electron transfer (PCET) reaction catalyzed by soybean lipoxygenase has served as a prototype for understanding hydrogen tunneling in enzymes.  Herein this PCET reaction is studied with mixed quantum mech./mol. mech. (QM/MM) free energy simulations.  The free energy surfaces are computed as functions of the proton donor-acceptor (C-O) distance and the proton coordinate, and the potential of mean force is computed as a function of the C-O distance, inherently including anharmonicity.  The simulation results are used to calc. the kinetic isotope effects for the wild-type enzyme (WT) and the L546A/L754A double mutant (DM), which have been measured exptl. to be ∼80 and ∼700, resp.  The PCET reaction is found to be exoergic for WT and slightly endoergic for the DM, and the equil. C-O distance for the reactant is found to be ∼0.2 Å greater for the DM than for WT.  The larger equil. distance for the DM, which is due mainly to less optimal substrate binding in the expanded binding cavity, is primarily responsible for its higher kinetic isotope effect.  The calcd. potentials of mean force are anharmonic and relatively soft at shorter C-O distances, allowing efficient thermal sampling of the shorter distances required for effective hydrogen tunneling.  The primarily local electrostatic field at the transferring hydrogen is ∼100 MV/cm in the direction to facilitate proton transfer and increases dramatically as the C-O distance decreases.  These simulations suggest that the overall protein environment is important for conformational sampling of active substrate configurations aligned for proton transfer, but the PCET reaction is influenced primarily by local electrostatic effects that facilitate conformational sampling of shorter proton donor-acceptor distances required for effective hydrogen tunneling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_8mb6abhxLbVg90H21EOLACvtfcHk0lhGN9FCeNJstg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXit1Wisbo%253D&amp;md5=7004dbdc16b392d9421fc9af6b2e6a95</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b13642&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b13642%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DP.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DFundamental%2520Insights%2520into%2520Proton-Coupled%2520Electron%2520Transfer%2520in%2520Soybean%2520Lipoxygenase%2520from%2520Quantum%2520Mechanical%252FMolecular%2520Mechanical%2520Free%2520Energy%2520Simulations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D3068%26epage%3D3076%26doi%3D10.1021%2Fjacs.7b13642" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soudackov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammes-Schiffer, S.</span></span> <span> </span><span class="NLM_article-title">Fundamental Insights into Proton-Coupled Electron Transfer in Soybean Lipoxygenase from Quantum Mechanical/Molecular Mechanical Free Energy Simulations</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">3068</span>– <span class="NLM_lpage">3076</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.7b13642</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b13642" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Wisbo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=140&amp;publication_year=2018&amp;pages=3068-3076&amp;author=P.+Liauthor=A.+V.+Soudackovauthor=S.+Hammes-Schiffer&amp;title=Fundamental+Insights+into+Proton-Coupled+Electron+Transfer+in+Soybean+Lipoxygenase+from+Quantum+Mechanical%2FMolecular+Mechanical+Free+Energy+Simulations&amp;doi=10.1021%2Fjacs.7b13642" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Fundamental Insights into Proton-Coupled Electron Transfer in Soybean Lipoxygenase from Quantum Mechanical/Molecular Mechanical Free Energy Simulations</span></div><div class="casAuthors">Li, Pengfei; Soudackov, Alexander V.; Hammes-Schiffer, Sharon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3068-3076</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The proton-coupled electron transfer (PCET) reaction catalyzed by soybean lipoxygenase has served as a prototype for understanding hydrogen tunneling in enzymes.  Herein this PCET reaction is studied with mixed quantum mech./mol. mech. (QM/MM) free energy simulations.  The free energy surfaces are computed as functions of the proton donor-acceptor (C-O) distance and the proton coordinate, and the potential of mean force is computed as a function of the C-O distance, inherently including anharmonicity.  The simulation results are used to calc. the kinetic isotope effects for the wild-type enzyme (WT) and the L546A/L754A double mutant (DM), which have been measured exptl. to be ∼80 and ∼700, resp.  The PCET reaction is found to be exoergic for WT and slightly endoergic for the DM, and the equil. C-O distance for the reactant is found to be ∼0.2 Å greater for the DM than for WT.  The larger equil. distance for the DM, which is due mainly to less optimal substrate binding in the expanded binding cavity, is primarily responsible for its higher kinetic isotope effect.  The calcd. potentials of mean force are anharmonic and relatively soft at shorter C-O distances, allowing efficient thermal sampling of the shorter distances required for effective hydrogen tunneling.  The primarily local electrostatic field at the transferring hydrogen is ∼100 MV/cm in the direction to facilitate proton transfer and increases dramatically as the C-O distance decreases.  These simulations suggest that the overall protein environment is important for conformational sampling of active substrate configurations aligned for proton transfer, but the PCET reaction is influenced primarily by local electrostatic effects that facilitate conformational sampling of shorter proton donor-acceptor distances required for effective hydrogen tunneling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_8mb6abhxLbVg90H21EOLACvtfcHk0lhGN9FCeNJstg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXit1Wisbo%253D&amp;md5=7004dbdc16b392d9421fc9af6b2e6a95</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b13642&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b13642%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DP.%26aulast%3DSoudackov%26aufirst%3DA.%2BV.%26aulast%3DHammes-Schiffer%26aufirst%3DS.%26atitle%3DFundamental%2520Insights%2520into%2520Proton-Coupled%2520Electron%2520Transfer%2520in%2520Soybean%2520Lipoxygenase%2520from%2520Quantum%2520Mechanical%252FMolecular%2520Mechanical%2520Free%2520Energy%2520Simulations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D3068%26epage%3D3076%26doi%3D10.1021%2Fjacs.7b13642" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newcomer, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brash, A. R.</span></span> <span> </span><span class="NLM_article-title">The Structural Basis for Specificity in Lipoxygenase Catalysis</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">309</span>, <span class="refDoi">&nbsp;DOI: 10.1002/pro.2626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1002%2Fpro.2626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=25524168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtFaksbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=24&amp;publication_year=2015&amp;pages=298-309&amp;author=M.+E.+Newcomerauthor=A.+R.+Brash&amp;title=The+Structural+Basis+for+Specificity+in+Lipoxygenase+Catalysis&amp;doi=10.1002%2Fpro.2626" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis for specificity in lipoxygenase catalysis</span></div><div class="casAuthors">Newcomer, Marcia E.; Brash, Alan R.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">298-309</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Many intriguing facets of lipoxygenase (LOX) catalysis are open to a detailed structural anal.  Polyunsatd. fatty acids with two to six double bonds are oxygenated precisely on a particular carbon, typically forming a single chiral fatty acid hydroperoxide product.  Mol. oxygen is not bound or liganded during catalysis, yet it is directed precisely to one position and one stereo configuration on the reacting fatty acid.  The transformations proceed upon exposure of substrate to enzyme in the presence of O2 (RH + O2 → ROOH), so it has proved challenging to capture the precise mode of substrate binding in the LOX active site.  Beginning with crystal structures with bound inhibitors or surrogate substrates, and most recently arachidonic acid bound under anaerobic conditions, a picture is consolidating of catalysis in a U-shaped fatty acid binding channel in which individual LOX enzymes use distinct amino acids to control the head-to-tail orientation of the fatty acid and register of the selected pentadiene opposite the non-heme iron, suitably positioned for the initial stereoselective hydrogen abstraction and subsequent reaction with O2.  Drawing on the crystal structures available currently, this review features the roles of the N-terminal β-barrel (C2-like, or PLAT domain) in substrate acquisition and sensitivity to cellular calcium, and the α-helical catalytic domain in fatty acid binding and reactions with O2 that produce hydroperoxide products with regio and stereospecificity.  LOX structures combine to explain how similar enzymes with conserved catalytic machinery differ in product, but not substrate, specificities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQAtaUuCTtmrVg90H21EOLACvtfcHk0ljwDVlW7kGOEg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXjtFaksbw%253D&amp;md5=201690b22d2eebd4a08d1cf92302147c</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2Fpro.2626&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2626%26sid%3Dliteratum%253Aachs%26aulast%3DNewcomer%26aufirst%3DM.%2BE.%26aulast%3DBrash%26aufirst%3DA.%2BR.%26atitle%3DThe%2520Structural%2520Basis%2520for%2520Specificity%2520in%2520Lipoxygenase%2520Catalysis%26jtitle%3DProtein%2520Sci.%26date%3D2015%26volume%3D24%26spage%3D298%26epage%3D309%26doi%3D10.1002%2Fpro.2626" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carr, C. A. M.</span></span> <span> </span><span class="NLM_article-title">On the Role of Enzyme Structure and Structural Dynamics in Lipoxygenase Catalysis</span>.  <i>Doctoral Dissertation</i>, <span class="NLM_publisher-name">University of California</span>, <span class="NLM_publisher-loc">Berkeley</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2013&amp;author=C.+A.+M.+Carr&amp;title=On+the+Role+of+Enzyme+Structure+and+Structural+Dynamics+in+Lipoxygenase+Catalysis" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DC.%2BA.%2BM.%26atitle%3DOn%2520the%2520Role%2520of%2520Enzyme%2520Structure%2520and%2520Structural%2520Dynamics%2520in%2520Lipoxygenase%2520Catalysis%26jtitle%3DDoctoral%2520Dissertation%26pub%3DUniversity%2520of%2520California%26date%3D2013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alber, T.</span></span> <span> </span><span class="NLM_article-title">Automated Protein Crystal Structure Determination Using ELVES</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">1537</span>– <span class="NLM_lpage">1542</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.0306241101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1073%2Fpnas.0306241101" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=101&amp;publication_year=2004&amp;pages=1537-1542&amp;author=J.+Holtonauthor=T.+Alber&amp;title=Automated+Protein+Crystal+Structure+Determination+Using+ELVES&amp;doi=10.1073%2Fpnas.0306241101" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0306241101&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0306241101%26sid%3Dliteratum%253Aachs%26aulast%3DHolton%26aufirst%3DJ.%26aulast%3DAlber%26aufirst%3DT.%26atitle%3DAutomated%2520Protein%2520Crystal%2520Structure%2520Determination%2520Using%2520ELVES%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D1537%26epage%3D1542%26doi%3D10.1073%2Fpnas.0306241101" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span>; <span class="NLM_string-name">Powell, H. R.</span></span> <span> </span><span class="NLM_article-title">Processing Diffraction Data with Mosflm</span>. In  <i>Evolving Methods for Macromolecular Crystallography</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span>, <span class="NLM_string-name">Sussman, J. L.</span></span>, Eds.; NATO Science Series; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Dordrecht</span>, <span class="NLM_year">2007</span>; Vol.  <span class="NLM_volume">245</span>, pp  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">51</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1007%2F978-1-4020-6316-9_4" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2007&amp;pages=41-51&amp;author=A.+G.+W.+Leslie&amp;author=H.+R.+Powellauthor=R.+J.+Read&amp;author=J.+L.+Sussman&amp;title=Evolving+Methods+for+Macromolecular+Crystallography" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1007%2F978-1-4020-6316-9_4&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4020-6316-9_4%26sid%3Dliteratum%253Aachs%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DProcessing%2520Diffraction%2520Data%2520with%2520Mosflm%26btitle%3DEvolving%2520Methods%2520for%2520Macromolecular%2520Crystallography%26aulast%3DRead%26aufirst%3DR.%2BJ.%26pub%3DSpringer%26date%3D2007%26volume%3D245%26spage%3D41%26epage%3D51" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Single-Crystal X-Ray Diffraction Data from a Position-Sensitive Detector</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">916</span>– <span class="NLM_lpage">924</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0021889888007903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0021889888007903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADyaL1MXhtVGmu7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=1988&amp;pages=916-924&amp;author=W.+Kabsch&amp;title=Evaluation+of+Single-Crystal+X-Ray+Diffraction+Data+from+a+Position-Sensitive+Detector&amp;doi=10.1107%2FS0021889888007903" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of single-crystal x-ray diffraction data from a position-sensitive detector</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">916-24</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">A Fortran program was developed for the redn. of single-crystal diffraction data from a sequence of adjacent rotation pictures recorded at a fixed x-ray wavelength by an electronic area detector.  The electronic pictures (data frames) covering the first 5° of crystal rotation are used to locate strong diffraction spots and to est. the background.  The orientation of the crystal is derived automatically from the list of obsd. spots and all parameters describing the diffraction pattern are refined.  The only input required from the user is the specification of the space group, approx. cell dimensions, and detector setting.  When the initialization step is finished the program goes back to the first picture and evaluates all data frames in the order they arrive from the measurement.  Each element of an electronic picture (pixel) is labeled by the indexes of the nearest reflection using the current refined parameters describing the diffraction geometry.  If the pixel is close to its nearest reflection the counts contribute to the 3-dimensional profile; otherwise the counts are used to update the background.  Each profile is represented as if the reflection had followed the shortest path through the Ewald sphere and had been recorded on the surface of the sphere.  Reflections close to the Ewald sphere are kept in a hash table to allow rapid access for updating the profiles.  Reflections which have completely passed through the Ewald sphere are removed from the table and saved for further processing.  Intensities are estd. by fitting their profiles to an av. shape learned from strong neighboring reflections.  Smoothly varying scaling factors are applied to the Lp-cor. intensities which minimize discrepancies between symmetry-related reflections and fit to a ref. data set if available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtr9BhTPWbOLVg90H21EOLACvtfcHk0ljwDVlW7kGOEg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL1MXhtVGmu7c%253D&amp;md5=f8d8c37891ec0d60758000879e7be9d6</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1107%2FS0021889888007903&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889888007903%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DEvaluation%2520of%2520Single-Crystal%2520X-Ray%2520Diffraction%2520Data%2520from%2520a%2520Position-Sensitive%2520Detector%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1988%26volume%3D21%26spage%3D916%26epage%3D924%26doi%3D10.1107%2FS0021889888007903" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Scaling and Assessment of Data Quality</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">82</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=62&amp;publication_year=2006&amp;pages=72-82&amp;author=P.+Evans&amp;title=Scaling+and+Assessment+of+Data+Quality&amp;doi=10.1107%2FS0907444905036693" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0ljwDVlW7kGOEg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&amp;md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1107%2FS0907444905036693&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520Assessment%2520of%2520Data%2520Quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2FS0907444905036693" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2010&amp;pages=125-132&amp;author=W.+Kabsch&amp;title=Xds&amp;doi=10.1107%2FS0907444909047337" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lirRtTCJGUJKQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&amp;md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How Good Are My Data and What Is the Resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=69&amp;publication_year=2013&amp;pages=1204-1214&amp;author=P.+R.+Evansauthor=G.+N.+Murshudov&amp;title=How+Good+Are+My+Data+and+What+Is+the+Resolution%3F&amp;doi=10.1107%2FS0907444913000061" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lirRtTCJGUJKQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&amp;md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520Good%2520Are%2520My%2520Data%2520and%2520What%2520Is%2520the%2520Resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the Ccp4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=67&amp;publication_year=2011&amp;pages=235-242&amp;author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&amp;title=Overview+of+the+Ccp4+Suite+and+Current+Developments&amp;doi=10.1107%2FS0907444910045749" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lirRtTCJGUJKQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&amp;md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520Ccp4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenbraun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Key, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timony, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottaviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliz, P.</span></span> <span> </span><span class="NLM_article-title">Cutting Edge: Collaboration Gets the Most out of Software</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e01456</span> <span class="refDoi">&nbsp;DOI: 10.7554/eLife.01456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.7554%2FeLife.01456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=24040512" title="">PubMed</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2&amp;publication_year=2013&amp;author=A.+Morinauthor=B.+Eisenbraunauthor=J.+Keyauthor=P.+C.+Sanschagrinauthor=M.+A.+Timonyauthor=M.+Ottavianoauthor=P.+Sliz&amp;title=Cutting+Edge%3A+Collaboration+Gets+the+Most+out+of+Software&amp;doi=10.7554%2FeLife.01456" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.7554%2FeLife.01456&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.01456%26sid%3Dliteratum%253Aachs%26aulast%3DMorin%26aufirst%3DA.%26aulast%3DEisenbraun%26aufirst%3DB.%26aulast%3DKey%26aufirst%3DJ.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DTimony%26aufirst%3DM.%2BA.%26aulast%3DOttaviano%26aufirst%3DM.%26aulast%3DSliz%26aufirst%3DP.%26atitle%3DCutting%2520Edge%253A%2520Collaboration%2520Gets%2520the%2520Most%2520out%2520of%2520Software%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26doi%3D10.7554%2FeLife.01456" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=40&amp;publication_year=2007&amp;pages=658-674&amp;author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&amp;title=Phaser+Crystallographic+Software&amp;doi=10.1107%2FS0021889807021206" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lj1mwwO7EtxLw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&amp;md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2010&amp;pages=486-501&amp;author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&amp;title=Features+and+Development+of+Coot&amp;doi=10.1107%2FS0907444910007493" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lj1mwwO7EtxLw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&amp;md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">Phenix: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2010&amp;pages=213-221&amp;author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&amp;title=Phenix%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&amp;doi=10.1107%2FS0907444909052925" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lj1mwwO7EtxLw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&amp;md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPhenix%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
den Bedem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latombe, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deacon, A. M.</span></span> <span> </span><span class="NLM_article-title">Modeling Discrete Heterogeneity in X-Ray Diffraction Data by Fitting Multi-Conformers</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1107</span>– <span class="NLM_lpage">1117</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444909030613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1107%2FS0907444909030613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=19770508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=65&amp;publication_year=2009&amp;pages=1107-1117&amp;author=H.+van%0Aden+Bedemauthor=A.+Dhanikauthor=J.-C.+Latombeauthor=A.+M.+Deacon&amp;title=Modeling+Discrete+Heterogeneity+in+X-Ray+Diffraction+Data+by+Fitting+Multi-Conformers&amp;doi=10.1107%2FS0907444909030613" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling discrete heterogeneity in X-ray diffraction data by fitting multi-conformers</span></div><div class="casAuthors">van den Bedem, Henry; Dhanik, Ankur; Latombe, Jean Claude; Deacon, Ashley M.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1107-1117</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The native state of a protein is regarded to be an ensemble of conformers, which allows assocn. with binding partners.  While some of this structural heterogeneity is retained upon crystn., reliably extg. heterogeneous features from diffraction data has remained a challenge.  In this study, a new algorithm for the automatic modeling of discrete heterogeneity is presented.  At high resoln., the authors' single multi-conformer model, with correlated structural features to represent heterogeneity, shows improved agreement with the diffraction data compared with a single-conformer model.  The model appears to be representative of the set of structures present in the crystal.  In contrast, below 2 Å resoln. representing ambiguous electron d. by correlated multi-conformers in a single model does not yield better agreement with the exptl. data.  Consistent with previous studies, this suggests that variability in multi-conformer models at lower resoln. levels reflects uncertainty more than coordinated motion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcZSvF9cJhcLVg90H21EOLACvtfcHk0lhD5LqblriENA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnJ&amp;md5=bd39757c428ec3105d1e6affe77ae830</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1107%2FS0907444909030613&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909030613%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bden%2BBedem%26aufirst%3DH.%26aulast%3DDhanik%26aufirst%3DA.%26aulast%3DLatombe%26aufirst%3DJ.-C.%26aulast%3DDeacon%26aufirst%3DA.%2BM.%26atitle%3DModeling%2520Discrete%2520Heterogeneity%2520in%2520X-Ray%2520Diffraction%2520Data%2520by%2520Fitting%2520Multi-Conformers%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1107%26epage%3D1117%26doi%3D10.1107%2FS0907444909030613" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den
Bedem, H.</span></span> <span> </span><span class="NLM_article-title">Exposing Hidden Alternative Backbone Conformations in X-Ray Crystallography Using Qfit</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e1004507</span> <span class="refDoi">&nbsp;DOI: 10.1371/journal.pcbi.1004507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1371%2Fjournal.pcbi.1004507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=26506617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvVKhtL4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2015&amp;author=D.+A.+Keedyauthor=J.+S.+Fraserauthor=H.+van+den%0ABedem&amp;title=Exposing+Hidden+Alternative+Backbone+Conformations+in+X-Ray+Crystallography+Using+Qfit&amp;doi=10.1371%2Fjournal.pcbi.1004507" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Exposing hidden alternative backbone conformations in X-ray crystallography using qFit</span></div><div class="casAuthors">Keedy, Daniel A.; Fraser, James S.; van den Bedem, Henry</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1004507/1-e1004507/22</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Proteins must move between different conformations of their native ensemble to perform their functions.  Crystal structures obtained from high-resoln. X-ray diffraction data reflect this heterogeneity as a spatial and temporal conformational av.  Although movement between natively populated alternative conformations can be crit. for characterizing mol. mechanisms, it is challenging to identify these conformations within electron d. maps.  Alternative side chain conformations are generally well sepd. into distinct rotameric conformations, but alternative backbone conformations can overlap at several at. positions.  Our model building program qFit uses mixed integer quadratic programming (MIQP) to evaluate an extremely large no. of combinations of sidechain conformers and backbone fragments to locally explain the electron d.  Here, we describe two major modeling enhancements to qFit: peptide flips and alternative glycine conformations.  We find that peptide flips fall into four stereotypical clusters and are enriched in glycine residues at the n + 1 position.  The potential for insights uncovered by new peptide flips and glycine conformations is exemplified by HIV protease, where different inhibitors are assocd. with peptide flips in the "flap" regions adjacent to the inhibitor binding site.  Our results paint a picture of peptide flips as conformational switches, often enabled by glycine flexibility, that result in dramatic local rearrangements.  Our results furthermore demonstrate the power of large-scale computational anal. to provide new insights into conformational heterogeneity.  Overall, improved modeling of backbone heterogeneity with high-resoln. X-ray data will connect dynamics to the structure-function relationship and help drive new design strategies for inhibitors of biomedically important systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1GkxACgHTjbVg90H21EOLACvtfcHk0lhD5LqblriENA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XkvVKhtL4%253D&amp;md5=dfb3faeb27a16df94cc9df4e9d26229d</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1004507&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1004507%26sid%3Dliteratum%253Aachs%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3Dvan%2Bden%2BBedem%26aufirst%3DH.%26atitle%3DExposing%2520Hidden%2520Alternative%2520Backbone%2520Conformations%2520in%2520X-Ray%2520Crystallography%2520Using%2520Qfit%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2015%26volume%3D11%26doi%3D10.1371%2Fjournal.pcbi.1004507" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dundas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binkowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turpaz, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span> <span> </span><span class="NLM_article-title">Castp: Computed Atlas of Surface Topography of Proteins with Structural and Topographical Mapping of Functionally Annotated Residues</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">W116</span>– <span class="NLM_lpage">W118</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gkl282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Fjacs.8b10992&amp;key=10.1093%2Fnar%2Fgkl282" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Fjacs.8b10992&amp;key=16844972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Fjacs.8b10992&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1yhtrw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=2006&amp;pages=W116-W118&amp;author=J.+Dundasauthor=Z.+Ouyangauthor=J.+Tsengauthor=A.+Binkowskiauthor=Y.+Turpazauthor=J.+Liang&amp;title=Castp%3A+Computed+Atlas+of+Surface+Topography+of+Proteins+with+Structural+and+Topographical+Mapping+of+Functionally+Annotated+Residues&amp;doi=10.1093%2Fnar%2Fgkl282" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">CASTp: computed atlas of surface topography of proteins with structural and topographical mapping of functionally annotated residues</span></div><div class="casAuthors">Dundas, Joe; Ouyang, Zheng; Tseng, Jeffery; Binkowski, Andrew; Turpaz, Yaron; Liang, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W116-W118</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Cavities on a proteins surface as well as specific amino acid positioning within it create the physicochem. properties needed for a protein to perform its function.  CASTp (http://cast.engr.uic.edu) is an online tool that locates and measures pockets and voids on 3D protein structures.  This new version of CASTp includes annotated functional information of specific residues on the protein structure.  The annotations are derived from the Protein Data Bank (PDB), Swiss-Prot, as well as Online Mendelian Inheritance in Man (OMIM), the latter contains information on the variant single nucleotide polymorphisms (SNPs) that are known to cause disease.  These annotated residues are mapped to surface pockets, interior voids or other regions of the PDB structures.  The authors use a semi-global pair-wise sequence alignment method to obtain sequence mapping between entries in Swiss-Prot, OMIM and entries in PDB.  The updated CASTp web server can be used to study surface features, functional regions and specific roles of key residues of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocC-dI1GB6RrVg90H21EOLACvtfcHk0lhD5LqblriENA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28Xps1yhtrw%253D&amp;md5=a5766c6a710f0b80633f68599134227f</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl282&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl282%26sid%3Dliteratum%253Aachs%26aulast%3DDundas%26aufirst%3DJ.%26aulast%3DOuyang%26aufirst%3DZ.%26aulast%3DTseng%26aufirst%3DJ.%26aulast%3DBinkowski%26aufirst%3DA.%26aulast%3DTurpaz%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DJ.%26atitle%3DCastp%253A%2520Computed%2520Atlas%2520of%2520Surface%2520Topography%2520of%2520Proteins%2520with%2520Structural%2520and%2520Topographical%2520Mapping%2520of%2520Functionally%2520Annotated%2520Residues%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3DW116%26epage%3DW118%26doi%3D10.1093%2Fnar%2Fgkl282" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://doi.org/10.2210/pdb5T5V/pdb" onclick="trackOutboundLink('http://doi.org/10.2210/pdb5T5V/pdb','PDB','5T5V'); return false;">PDB: 5T5V</a></li><li><a href="http://doi.org/10.2210/pdb5TQN/pdb" onclick="trackOutboundLink('http://doi.org/10.2210/pdb5TQN/pdb','PDB','5TQN'); return false;">PDB: 5TQN</a></li><li><a href="http://doi.org/10.2210/pdb5TR0/pdb" onclick="trackOutboundLink('http://doi.org/10.2210/pdb5TR0/pdb','PDB','5TR0'); return false;">PDB: 5TR0</a></li><li><a href="http://doi.org/10.2210/pdb5TQO/pdb" onclick="trackOutboundLink('http://doi.org/10.2210/pdb5TQO/pdb','PDB','5TQO'); return false;">PDB: 5TQO</a></li><li><a href="http://doi.org/10.2210/pdb3PZW/pdb" onclick="trackOutboundLink('http://doi.org/10.2210/pdb3PZW/pdb','PDB','3PZW'); return false;">PDB: 3PZW</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i24">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_50540"></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/jacs.8b10992">10.1021/jacs.8b10992</a>.</p><ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">(<a class="ext-link" href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf">PDF</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jacs.8b10992/suppl_file/ja8b10992_si_001.pdf">ja8b10992_si_001.pdf (4.98 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms &amp; Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7" style="width: 457.7px;"><h2 tabindex="0" class="recommended-header">Recommended Articles <i class="icon-angle-down recommendedIcon"></i></h2><div data-exp-type="" data-query-id="" class="show-recommended"><ul><li><a href="/doi/full/10.1021/acs.biochem.9b00861"><span class="show-recommended_slider-title">The Soybean Lipoxygenase–Substrate Complex: Correlation between the Properties of Tunneling-Ready States and ENDOR-Detected Structures of Ground States</span></a><a href="/doi/full/10.1021/acs.biochem.9b00861" class="journ_title">Biochemistry</a></li><li><a href="/doi/full/10.1021/acscentsci.7b00142"><span class="show-recommended_slider-title">Hydrogen–Deuterium Exchange of Lipoxygenase Uncovers a Relationship between Distal, Solvent Exposed Protein Motions and the Thermal Activation Barrier for Catalytic Proton-Coupled Electron Tunneling</span></a><a href="/doi/full/10.1021/acscentsci.7b00142" class="journ_title">ACS Central Science</a></li><li><a href="/doi/full/10.1021/acs.jpcb.9b07228"><span class="show-recommended_slider-title">Detecting and Characterizing the Kinetic Activation of Thermal Networks in Proteins: Thermal Transfer from a Distal, Solvent-Exposed Loop to the Active Site in Soybean Lipoxygenase</span></a><a href="/doi/full/10.1021/acs.jpcb.9b07228" class="journ_title">The Journal of Physical Chemistry B</a></li><li><a href="/doi/full/10.1021/jacs.7b08418"><span class="show-recommended_slider-title">Origins of Enzyme Catalysis: Experimental Findings for C–H Activation, New Models, and Their Relevance to Prevailing Theoretical Constructs</span></a><a href="/doi/full/10.1021/jacs.7b08418" class="journ_title">Journal of the American Chemical Society</a></li><li><a href="/doi/full/10.1021/jacs.0c07866"><span class="show-recommended_slider-title">Hydrogen–Deuterium Exchange within Adenosine Deaminase, a TIM Barrel Hydrolase, Identifies Networks for Thermal Activation of Catalysis</span></a><a href="/doi/full/10.1021/jacs.0c07866" class="journ_title">Journal of the American Chemical Society</a></li><div class="sponsoredContent">



        
        <div class="advertisement">
  <div class="advertisement-link">
    <a href="http://acsmediakit.org">ADVERTISEMENT</a>
  </div>
  <div id="dfp-journal-article-recommend">
    <script type="text/javascript">
      $(document).ready(function() {
        setTimeout(function() {
          googletag.cmd.push(function() { googletag.display("dfp-journal-article-recommend"); });
        }, 1000);
      });
    </script>
  </div>
</div>


</div></ul></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self" style="display: none;"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jacs.8b10992" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article><div class="w-slide"><div class="w-slide_head"><a href="#" class="w-slide__back"><i class=" icon-arrow_l" aria-hidden="true"></i>back</a><span class="w-slide__title"></span></div><div class="w-slide__content"></div></div></main><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2022<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS &amp; Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support &amp; Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"><input type="hidden" name="appID" value="literatum" class="appID"><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onclick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        <!-- <div id="gdpr-message-container">
			<a id="closebtn" href="#"><i class="icon-times-circle-o"></i></a> -->
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        <!-- </div> -->
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        <!-- AdRoll Pixel -->
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>

<!-- Twitter universal website tag code -->
<!-- Commented out for ACHS-6152
<script>
    !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe.apply(s,arguments):s.queue.push(arguments);
    },s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',
    a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');
    // Insert Twitter Pixel ID and Standard Event data below
    twq('init','o3njp');
    twq('track','PageView');
</script>
-->
<!-- Facebook Pixel Code -->
<!-- Commented out for ACHS-6152
<script>
    !function(f,b,e,v,n,t,s)
    {if(f.fbq)return;n=f.fbq=function(){n.callMethod?
    n.callMethod.apply(n,arguments):n.queue.push(arguments)};
    if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
    n.queue=[];t=b.createElement(e);t.async=!0;
    t.src=v;s=b.getElementsByTagName(e)[0];
    s.parentNode.insertBefore(t,s)}(window, document,'script',
    'https://connect.facebook.net/en_US/fbevents.js');
    fbq('init', '529840281033320');
    fbq('track', 'PageView');
</script>
<noscript><img height="1" width="1" style="display:none" src="https://www.facebook.com/tr?id=529840281033320&amp;ev=PageView&amp;noscript=1"
/></noscript>
-->





        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div><div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/jacs.8b10990">Solid-State NMR and MD Study of the Structure of the Statherin Mutant SNa15 on Mineral Surfaces</a></li><li><a href="/doi/full/10.1021/jacs.8b10932">Molecular Upconversion in Water in Heteropolynuclear Supramolecular Tb/Yb Assemblies</a></li><li><a href="/doi/full/10.1021/jacs.8b12124">Acid Responsive Hydrogen-Bonded Organic Frameworks</a></li><li><a href="/doi/full/10.1021/jacs.8b11259">Germanium Fluoride Nanocages as Optically Transparent n-Type Materials and Their Endohedral Metallofullerene Derivatives</a></li><li><a href="/doi/full/10.1021/jacs.8b10083">Flexible Polymer-Assisted Mesoscale Self-Assembly of Colloidal CsPbBr3 Perovskite Nanocrystals into Higher Order Superstructures with Strong Inter-Nanocrystal Electronic Coupling</a></li></ul></div></div></div>




        
        <script>
var currentJournal = "Journal of the American Chemical Society";
$(".citationLinks > a").click(function(e){
    if($(this).text() === "ACS Full Text ") {
        ga('send', 'event', 'Reference _Click', 'ACS Full Text', currentJournal);
    }
    else if($(this).text() === "Crossref") {
        ga('send', 'event', 'Reference _Click', 'Crossref', currentJournal);
    }
    else if($(this).text() === "CAS") {
        ga('send', 'event', 'Reference _Click', 'CAS', currentJournal);
    }
    else if($(this).text() === "PubMed") {
        ga('send', 'event', 'Reference _Click', 'PubMed', currentJournal);
    }
    else if($(this).text() === "Google Scholar") {
        ga('send', 'event', 'Reference _Click', 'Google Scholar', currentJournal);
    }
    else if($(this).hasClass("issue-item_open-url") === true) {
        ga('send', 'event', 'Reference _Click', 'OpenURL', currentJournal);
    }
});
</script>






        
        



        
    





        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        <div tabindex="0" class="showMenuAlertMsg-popup"><div class="pairing-affiliations-popup-content"><div class="pairing-affiliations-header"><button type="button" class="close paringAffClose"><i aria-hidden="true" class="icon-close_thin"></i></button></div><div class="responsrMsg"></div></div></div>




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	dartInstitutionType = 'education';
</script>
        </div>
    





        
        



        
    





        
        <script>
// LIT-455776 workaround
// Moving the popup before the header content for tab accessibility
document.addEventListener('DOMContentLoaded', () => {
    const headerNode = $('.header_content');
    const popupNode = $('.pairing-affiliations-popup');

    if (!headerNode.length || !popupNode.length)
        return;

    popupNode.insertBefore(headerNode);  
});
</script> 
        </div>
    </div>





	
    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-d93b9088814c7fcd2f1c.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-9dee82777d6af4ace557.js"></script>
<script type="text/javascript" src="/wro/l6j1~product.js"></script>



















<div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh770" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.10691291551239945&amp;iit=1651101087289&amp;tmr=load%3D1651101086899%26core%3D1651101086947%26main%3D1651101087281%26ifr%3D1651101087297&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=pubs.acs.org&amp;dr=&amp;du=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Fjacs.8b10992&amp;href=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Fjacs.8b10992&amp;dt=Biophysical%20Characterization%20of%20a%20Disabled%20Double%20Mutant%20of%20Soybean%20Lipoxygenase%3A%20The%20%E2%80%9CUndoing%E2%80%9D%20of%20Precise%20Substrate%20Positioning%20Relative%20to%20Metal%20Cofactor%20and%20an%20Identified%20Dynamical%20Network&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Ctype%3DArticle%2Cdescription%2Cimage%2Ctitle%2Curl&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=6269cd9e5e2b76f5&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=url%3Dhttps%253A%252F%252Fpubs.acs.org%252Fdoi%252Ffull%252F10.1021%252Fjacs.8b10992%26title%3DBiophysical%2520Characterization%2520of%2520a%2520Disabled%2520Double%2520Mutant%2520of%2520Soybean%2520Lipoxygenase%253A%2520The%2520%25E2%2580%259CUndoing%25E2%2580%259D%2520of%2520Precise%2520Substrate%2520Positioning%2520Relative%2520to%2520Metal%2520Cofactor%2520and%2520an%2520Identified%2520Dynamical%2520Network%26image%3Dhttps%253A%252F%252Fpubs.acs.org%252Fcms%252F10.1021%252Fjacs.8b10992%252Fasset%252Fimages%252Fjacs.8b10992.social.jpeg_v03%26description%3DSoybean%2520lipoxygenase%2520(SLO)%2520has%2520served%2520as%2520a%2520prototype%2520for%2520understanding%2520the%2520molecular%2520origin%2520of%2520enzymatic%2520rate%2520accelerations.%2520The%2520double%2520mutant%2520(DM)%2520L546A%252FL754A%2520is%2520considered%2520a%2520dramatic%2520outlier%252C%2520due%2520to%2520the%2520unprecedented%2520size%2520and%2520near%2520temperature-independence%2520of%2520its%2520primary%2520kinetic%2520isotope%2520effect%252C%2520low%2520catalytic%2520efficiency%252C%2520and%2520elevated%2520enthalpy%2520of%2520activation.%2520To%2520uncover%2520the%2520physical%2520basis%2520of%2520these%2520features%252C%2520we%2520herein%2520apply%2520three%2520structural%2520probes%253A%2520hydrogen%25E2%2580%2593deuterium%2520exchange%2520mass%2520spectrometry%252C%2520room-temperature%2520X-ray%2520crystallography%2520and%2520EPR%2520spectroscopy%2520on%2520four%2520SLO%2520variants%2520(wild-type%2520(WT)%2520enzyme%252C%2520DM%252C%2520and%2520the%2520two%2520parental%2520single%2520mutants%252C%2520L546A%2520and%2520L754A).%2520DM%2520is%2520found%2520to%2520incorporate%2520features%2520of%2520each%2520parent%252C%2520with%2520the%2520perturbation%2520at%2520position%2520546%2520predominantly%2520influencing%2520thermally%2520activated%2520motions%2520that%2520connect%2520the%2520active%2520site%2520to%2520a%2520protein%25E2%2580%2593solvent%2520interface%252C%2520while%2520mutation%2520at%2520position%2520754%2520disrupts%2520the%2520ligand%2520field%2520and%2520solvation%2520near%2520the%2520cofactor%2520iron.%2520However%252C%2520the%2520expanded%2520active%2520site%2520in%2520DM%2520leads%2520to%2520more%2520active%2520site%2520water%2520molecules%2520and%2520their%2520associated%2520hydrogen%2520bond%2520network%252C%2520and%2520the%2520individual%2520features%2520from%2520L546A%2520and%2520L754A%2520alone%2520cannot%2520explain%2520the%2520aggregate%2520kinetic%2520properties%2520for%2520DM.%2520Using%2520recently%2520published%2520QM%252FMM-derived%2520ground-state%2520SLO-substrate%2520complexes%2520for%2520WT%2520and%2520DM%252C%2520together%2520with%2520the%2520thorough%2520structural%2520analyses%2520presented%2520herein%252C%2520we%2520propose%2520that%2520the%2520impairment%2520of%2520DM%2520is%2520the%2520combined%2520result%2520of%2520a%2520repositioning%2520of%2520the%2520reactive%2520carbon%2520of%2520linoleic%2520acid%2520substrate%2520with%2520regard%2520to%2520both%2520the%2520iron%2520cofactor%2520and%2520a%2520catalytically%2520linked%2520dynamic%2520region%2520of%2520protein.%26type%3DArticle%26site_name%3DACS%2520Publications&amp;csi=undefined&amp;rev=v8.28.8-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style></body></html>