<html class="pb-page" data-request-id="3443dd99-49de-43b0-b7f2-83bac598fa3c" lang="en"><head data-pb-dropzone="head"><script type="text/javascript" async="" src="https://www.google-analytics.com/analytics.js"></script><script type="text/javascript" async="" src="https://www.googletagmanager.com/gtag/js?id=G-ESS0NB66T4&amp;l=dataLayer&amp;cx=c"></script><script type="text/javascript" src="https://m.addthis.com/live/red_lojson/300lo.json?si=6339f30c508b3fa9&amp;bkl=0&amp;bl=1&amp;pdt=623&amp;sid=6339f30c508b3fa9&amp;pub=xa-4faab26f2cff13a7&amp;rev=v8.28.8-wp&amp;ln=en&amp;pc=men&amp;cb=0&amp;ab=-&amp;dp=pubs.acs.org&amp;fp=doi%2F10.1021%2Facscentsci.1c00663&amp;fr=&amp;of=0&amp;pd=0&amp;irt=1&amp;vcl=1&amp;md=0&amp;ct=1&amp;tct=0&amp;abt=0&amp;cdn=0&amp;pi=1&amp;rb=0&amp;gen=100&amp;chr=UTF-8&amp;colc=1664742156582&amp;jsl=1&amp;uvs=6339f305f97112a6002&amp;skipb=1&amp;callback=addthis.cbs.jsonp__419715726025451930"></script><script type="text/javascript" src="https://v1.addthisedge.com/live/boost/xa-4faab26f2cff13a7/_ate.track.config_resp"></script><script type="text/javascript" src="https://z.moatads.com/addthismoatframe568911941483/moatframe.js"></script><meta name="pbContext" content=";article:article:10.1021/acscentsci.1c00663;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;requestedJournal:journal:acscii;website:website:acspubs;issue:issue:10.1021/acscii.2021.7.issue-8;journal:journal:acscii;pageGroup:string:Publication Pages;subPage:string:Full Text">




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="ACS Central Science"><meta name="dc.Title" content="Parallel Automated Flow Synthesis of Covalent Protein Complexes That Can Inhibit MYC-Driven Transcription"><meta name="dc.Creator" content="Sebastian  Pomplun"><meta name="dc.Creator" content="Muhammad  Jbara"><meta name="dc.Creator" content="Carly K.  Schissel"><meta name="dc.Creator" content="Susana  Wilson Hawken"><meta name="dc.Creator" content="Ann  Boija"><meta name="dc.Creator" content="Charles  Li"><meta name="dc.Creator" content="Isaac  Klein"><meta name="dc.Creator" content="Bradley L.  Pentelute"><meta name="dc.Description" content="Dysregulation of the transcription factor MYC is involved in many human cancers. The dimeric transcription factor complexes of MYC/MAX and MAX/MAX activate or inhibit, respectively, gene transcript..."><meta name="Description" content="Dysregulation of the transcription factor MYC is involved in many human cancers. The dimeric transcription factor complexes of MYC/MAX and MAX/MAX activate or inhibit, respectively, gene transcript..."><meta name="dc.Description" content="Via parallel automated flow synthesis, we prepared covalent dimeric protein analogues of the transcription factor MYC. The dimers can enter cells and inhibit MYC-driven cancer cell proliferation."><meta name="Description" content="Via parallel automated flow synthesis, we prepared covalent dimeric protein analogues of the transcription factor MYC. The dimers can enter cells and inhibit MYC-driven cancer cell proliferation."><meta name="dc.Publisher" content="American Chemical Society"><meta name="dc.Date" scheme="WTN8601" content="August 4, 2021"><meta name="dc.Type" content="research-article"><meta name="dc.Format" content="text/HTML"><meta name="dc.Identifier" scheme="doi" content="10.1021/acscentsci.1c00663"><meta name="dc.Language" content="en"><meta name="dc.Coverage" content="world"><meta name="dc.Rights" content="© 2021 The Authors. Published by American Chemical Society"><meta name="citation_fulltext_world_readable" content="">
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facscentsci.1c00663">
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facscentsci.1c00663">
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facscentsci.1c00663">
        
    
    

<title>Parallel Automated Flow Synthesis of Covalent Protein Complexes That Can Inhibit MYC-Driven Transcription | ACS Central Science</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive">





    






        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acscentsci.1c00663"><meta property="og:title" content="Parallel Automated Flow Synthesis of Covalent Protein Complexes That Can Inhibit MYC-Driven Transcription"><meta property="og:image" content="https://pubs.acs.org/cms/10.1021/acscentsci.1c00663/asset/images/acscentsci.1c00663.social.jpeg_v03"><meta property="og:description" content="Dysregulation of the transcription factor MYC is involved in many human cancers. The dimeric transcription factor complexes of MYC/MAX and MAX/MAX activate or inhibit, respectively, gene transcription upon binding to the same enhancer box DNA. Targeting these complexes in cancer is a long-standing challenge. Inspired by the inhibitory activity of the MAX/MAX dimer, we engineered covalently linked, synthetic homo- and heterodimeric protein complexes to attenuate oncogenic MYC-driven transcription. We prepared the covalent protein complexes (∼20 kDa, 167–231 residues) in a single shot via parallel automated flow synthesis in hours. The stabilized covalent dimers display DNA binding activity, are intrinsically cell-penetrant, and inhibit cancer cell proliferation in different cell lines. RNA sequencing and gene set enrichment analysis in A549 cancer cells confirmed that the synthetic dimers interfere with MYC-driven transcription. Our results demonstrate the potential of automated flow technology to rapidly deliver engineered synthetic protein complex mimetics that can serve as a starting point in developing inhibitors of MYC-driven cancer cell growth."><meta property="og:type" content="Article"><meta property="og:site_name" content="ACS Publications"><meta name="twitter:card" content="summary_large_image"><meta name="twitter:site" content="@ACSPublications"><meta name="twitter:image" content="https://pubs.acs.org/cms/10.1021/acscentsci.1c00663/asset/images/acscentsci.1c00663.social.jpeg_v03">
    





    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0">















    

    <meta name="dc.identifier" content="10.1021/acscentsci.1c00663">



<meta name="format-detection" content="telephone=no">




<link rel="stylesheet" type="text/css" href="/wro/lmwf~product.css">

<link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-47fdc05884749092d0b2.css"><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css">




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1663153435260.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1662482419870.css">
<script type="text/javascript" async="" src="//img.en25.com/i/elqCfg.min.js"></script><script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script><script type="text/javascript" async="" src="https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js"></script><script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer="" src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon="{&quot;rayId&quot;:&quot;5200035e6e4d5180&quot;,&quot;startTime&quot;:1570117212010}"></script>
<script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/v652eace1692a40cfa3763df669d7439c1639079717194" integrity="sha512-Gi7xpJR8tSkrpF7aordPZQlW2DLtzUlZcumS8dMQjwDHEnw9I7ZLyiOj/6tZStRBGtGgN6ceN6cMH8z7etPGlw==" data-cf-beacon="{&quot;rayId&quot;:&quot;7224ce26191e6f50&quot;,&quot;token&quot;:&quot;0562ec7bcba14d1cae3543aa43d6e749&quot;,&quot;version&quot;:&quot;2022.6.0&quot;,&quot;si&quot;:100}" crossorigin="anonymous"></script>

<!-- ACHS-7903: Global site tag (gtag.js) - Google Analytics -->
<script async="" src="https://www.googletagmanager.com/gtag/js?id=G-Q72ZQB7GTR"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-Q72ZQB7GTR');
</script>
            <script async="" src="https://www.googletagmanager.com/gtag/js?id=UA-7663985-4"></script>
            <script type="text/javascript">
                window.dataLayer = window.dataLayer || [];
                function gtag(){dataLayer.push(arguments);}
                gtag('js', new Date());

                gtag('config', 'UA-7663985-4');
            </script>
            











    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acscentsci.1c00663">
            
        







    <!-- loading Javascript for using the persistence Service -->
    <script src="/templates/jsp/pb2/js/plugin/thiss-ds.js"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            alt_context: window.location.origin,
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if (ms == undefined || ms === '') {
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            if (results.length == 0) {
                                persistService.entities(self.alt_context).then((res) => {
                                    return resolve(res.data);
                                }, function (err) {
                                    // failed
                                    reject('failed to retrieve entity from local storage (publisher domain): ' + err);
                                });
                            } else {
                                return resolve(results);
                            }
                        }, function (err) {
                            // failed
                            reject('failed to retrieve entity from local storage: ' + err);
                        });
                    // Set up the timeout
                    setTimeout(function () {
                        reject('Promise timed out ' + ms + ' ms');
                    }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                    persistService.update(self.alt_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage (publisher domain) duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            var results = res.data;
                            if (!results) {
                                persistService.remove(self.alt_context, entityId).then((res) => {
                                    return resolve(res);
                                }, function (err) {
                                    // failed
                                    console.log('failed to remove from local storage (publisher domain) duo to: ' + err);
                                });
                            }
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId, entityTitle) {
                return new Promise(function (resolve, reject) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {
                            doi: doi,
                            entityid: entityId,
                            entityTitle: entityTitle,
                            eventGroupKey: 'c1936332-731f-4bb8-9b16-ce8bea3ae198'
                        }

                    }).done(function (status) {
                        return resolve(status);
                    }).fail(function (status) {
                        return reject(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID, ssoInstitution.entity.title)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + ssoInstitution.entity.entityID,err);
                                    });
                            }));
                        }
                        if(allSsoInstitutions.length === 0 && true){
                            self.getAccessibleEntityStatus(doi, null, null)
                                .then(function () {
                                    console.log('No Entity found');
                                }, function (err) {
                                    console.log('failed to get the none entity status' ,err);
                                });
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            }).catch(function (err) {
                                reject(err);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();

                    if ($('.institution-preference-userconsent-checkbox').length > 0) {
                        let institutionPreferenceUserConsentCheckbox = $('.institution-preference-userconsent-checkbox').get(0);
                        if (!$(institutionPreferenceUserConsentCheckbox).is(":checked")) {
                            return;
                        }
                    }

                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err) {
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>







    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY">


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html">

<link rel="shortcut icon" href="/pb-assets/ux3/favicon-1635812676023.ico"><meta http-equiv="origin-trial" content="A+cA2PUOfIOKAdSDJOW5CP9ZlxONy1yu+hqAq72zUtKw4rLdihqRp6Nui/jUyCyegr+BUtH+C+Elv0ufn05yBQEAAACFeyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjY5NzY2Mzk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A+zsdH3aNZT/bkjT8U/o5ACzyaeNYzTvtoVmwf/KOilfv39pxY2AIsOwhQJv+YnXp98i3TqrQibIVtMWs5UHjgoAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjY5NzY2Mzk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="AxceVEhIegcDEHqLXFQ2+vPKqzCppoJYsRCZ/BdfVnbM/sUUF2BXV8lwNosyYjvoxnTh2FC8cOlAnA5uULr/zAUAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjY5NzY2Mzk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_2022092701.js" async=""></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><script src="https://widgets.figshare.com/static/figshare.js?_=1664742155922"></script><script id="yui__dyn_0" type="text/javascript" src="https://www.snapengage.com/chatjs/ServiceGetConfig?w=921a4ec7-cecf-43a6-afa5-6cd1346520b1"></script></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'central_science';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/central_science/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/central_science/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
					console.log("ad display: " + event.creativeId + " " + event.size[0] + "x" + event.size[1]);
				}
				else {
				    console.log("no ad found");
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>

<style>
.articleHeaderDropzone5 .advertisement { display: none; }
.article_header .advertisement {
  margin: -25px auto 25px auto;
}
</style>




        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&amp;EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Indiana University Libraries - Bloomington</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"></a>




        
        <div class="pj-badge"></div><a href="/journal1/acscii" title="ACS Central Science" class="header_article-logo"><img src="/cms/10.1021/acscii/asset/title.png" alt="ACS Central Science"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="acscii" name="SeriesKey" checked="checked" class="all-content"><span>ACS Cent. Sci.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"><select name="quickLinkJournal" class="quick-search_journals-select"><option value="acscii">ACS Central Science</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science &amp; Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aaemdr">ACS Applied Engineering Materials</option><option value="aamick">ACS Applied Materials &amp; Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aaoma6">ACS Applied Optical Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio &amp; Med Chem Au</option><option value="abseba">ACS Biomaterials Science &amp; Engineering</option><option value="accacs">ACS Catalysis</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health &amp; Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&amp;T Engineering</option><option value="aewcaa">ACS ES&amp;T Water</option><option value="afsthl">ACS Food Science &amp; Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic &amp; Inorganic Au</option><option value="aptsfn">ACS Pharmacology &amp; Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry &amp; Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&amp;EN Global Enterprise</option><option value="cbihbp">Chemical &amp; Biomedical Imaging</option><option value="cenear">Chemical &amp; Engineering News Archive</option><option value="chlseg">Chemical Health &amp; Safety</option><option value="chlseg0">Chemical Health &amp; Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth &amp; Design</option><option value="enfuem">Energy &amp; Fuels</option><option value="esthag">Environmental Science &amp; Technology</option><option value="estlcu">Environmental Science &amp; Technology Letters</option><option value="iepra6.2">I&amp;EC Product Research and Development</option><option value="iechad">Industrial &amp; Engineering Chemistry</option><option value="iecac0">Industrial &amp; Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series</option><option value="iecfa7">Industrial &amp; Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial &amp; Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial &amp; Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial &amp; Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical &amp; Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc20">Journal of Chemical Health &amp; Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial &amp; Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research &amp; Development</option><option value="orgnd7">Organometallics</option><option value="pcrhej">Precision Chemistry</option><option value="iepra6.1">Product R&amp;D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autocomplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autocomplete="false" placeholder="Page" required="required" class="quick-search_page-input"><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden="”true”"></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/acscentsci.1c00681">Chemoadaptive Polymeric Assemblies by Integrated Chemical Feedback in Self-Assembled Synthetic Protocells</a></li><li><a href="/doi/full/10.1021/acscentsci.1c00743">Large-Scale Production of Hierarchically Porous Metal–Organic Frameworks by a Reflux-Assisted Post-Synthetic Ligand Substitution Strategy</a></li></ul></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="”true”"></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="”true”"></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden="”true”"></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'Access Options'});">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'Authors &amp; Reviewers'});">Authors &amp;
                        Reviewers</a></li>
                <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'ACS Members'});">ACS Members</a></li>
				<li><a href="https://pubs.acs.org/page/virtual-collections.html" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'Virtual Issues'});">Virtual Issues</a></li>
                <li><a href="https://preferences.acs.org/journals" onclick="gtag('event', 'Click', { 'event_category': 'Home_MainMenu', 'event_label':'eAlert'});">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'RSS &amp; Mobile'});">RSS &amp; Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'Products &amp; Services'});">Products &amp; Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'Get Access'});">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'Manage My Account'});">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos &amp; Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'Support FAQs'});">Support FAQs </a></li>
                <li><a href="#" onclick="SnapEngage.startLink();return false; gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'Live Chat with Agent'});" class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'For Advertisers'});">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/who-we-serve/librarians/" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'For Librarians &amp; Account Managers'});">For
                        Librarians &amp; Account Managers</a></li>
            </ul>
        </li>
        <li class="menu-pair-section hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a></li>
                
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'Overview'});
">Overview</a></li>
                <li><a href="http://acsopenscience.org" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'ACS &amp; Open Access'});">ACS &amp; Open Access</a>
                </li>
                <li><a href="/#partners" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'Partners'});
">Partners</a></li>
                <li><a href="/events" onclick="gtag('event', 'Menu', { 'event_category': 'Top Nav', 'event_label':'Events'});">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!-- Types--><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><!-- Subjects--><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!--div.pub-modal_right-cover-gallery.hidden-lg.hidden-md//a(href="/coverGallery") Browse Our Cover Art Gallery
--><!-- Publications list--><!-- 'pub-modal_right--inner' adds styling required for IE--><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><!-- The widget data--><input type="hidden" id="ajaxUrl" name="ajaxUrl" value="/pb/widgets/achs-publication-list?widgetId=cabcb15c-6f7a-494b-a19c-eba4fafaee5c&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facscentsci.1c00663%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3BrequestedJournal%3Ajournal%3Aacscii%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Facscii.2021.7.issue-8%3Bjournal%3Ajournal%3Aacscii%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;key=5fcd18d94065d4df9cdfc009e01c6f2c3&amp;ajaxRequest=true"><input type="hidden" name="pubModalData" value=""></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facscentsci.1c00663&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facscentsci.1c00663&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acscentsci.1c00663&amp;href=/doi/10.1021/acscentsci.1c00663" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>ACS Cent. Sci.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 7</span><span class="cit-fg-issue">, 8</span><span class="cit-fg-pageRange">, 1408-1418</span></span></span></div></div></div></div></div><input type="hidden" name="acscii" id="copyRights" value="/pb-assets/json/journal-info-1660234310487.json"><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/acscii/7/8" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acscentsci.1c00645" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Research Article</span><a href="/doi/10.1021/acscentsci.1c00670" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/acscii" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/cms/10.1021/acscii/asset/title.png" alt="Journal Logo"></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Parallel Automated Flow Synthesis of Covalent Protein Complexes That Can Inhibit MYC-Driven Transcription</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Sebastian Pomplun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sebastian Pomplun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Massachusetts Institute of Technology, 77 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sebastian++Pomplun">Sebastian Pomplun</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Muhammad Jbara</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Muhammad Jbara</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Massachusetts Institute of Technology, 77 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Muhammad++Jbara">Muhammad Jbara</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="https://orcid.org/0000-0002-4206-5908" title="Orcid link">https://orcid.org/0000-0002-4206-5908</a></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Carly K. Schissel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carly K. Schissel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Massachusetts Institute of Technology, 77 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carly+K.++Schissel">Carly K. Schissel</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="https://orcid.org/0000-0003-0773-5168" title="Orcid link">https://orcid.org/0000-0003-0773-5168</a></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Susana Wilson Hawken</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Susana Wilson Hawken</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Whitehead Institute for Biomedical Research, Cambridge, Massachusetts 02142, United States</div><div class="loa-info-affiliations-info">Department of Biology, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Susana++Wilson+Hawken">Susana Wilson Hawken</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Ann Boija</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ann Boija</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Whitehead Institute for Biomedical Research, Cambridge, Massachusetts 02142, United States</div><div class="loa-info-affiliations-info">Department of Biology, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ann++Boija">Ann Boija</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Charles Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Charles Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Whitehead Institute for Biomedical Research, Cambridge, Massachusetts 02142, United States</div><div class="loa-info-affiliations-info">Department of Biology, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Charles++Li">Charles Li</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Isaac Klein</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Isaac Klein</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Whitehead Institute for Biomedical Research, Cambridge, Massachusetts 02142, United States</div><div class="loa-info-affiliations-info">Department of Biology, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Isaac++Klein">Isaac Klein</a></span></div></div></span></li><span class="comma-separator">,&nbsp;and&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Bradley L. Pentelute</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bradley L. Pentelute</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Massachusetts Institute of Technology, 77 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</div><div class="loa-info-affiliations-info">The Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, 500 Main Street, Cambridge, Massachusetts 02142, United States</div><div class="loa-info-affiliations-info">Center for Environmental Health Sciences, Massachusetts Institute of Technology, 77 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</div><div class="loa-info-affiliations-info">Broad Institute of MIT and Harvard, 415 Main Street, Cambridge, Massachusetts 02142, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="mailto:blp@mit.edu">blp@mit.edu</a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bradley+L.++Pentelute">Bradley L. Pentelute</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="https://orcid.org/0000-0002-7242-801X" title="Orcid link">https://orcid.org/0000-0002-7242-801X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="acscii"><input type="hidden" name="journalNameForjhpLink" value="ACS Central Science"><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facscentsci.1c00663&amp;href=/doi/10.1021%2Facscentsci.1c00663" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>ACS Cent. Sci.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 7</span><span class="cit-issue">, 8</span><span class="cit-pageRange">, 1408–1418</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 4, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>1 June 2021</li><li><span class="item_label"><b>Published</b> online</span>4 August 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 August 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acscentsci.1c00663" title="DOI URL">https://doi.org/10.1021/acscentsci.1c00663</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS &amp; PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="https://acsopenscience.org/open-access/licensing-options/"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice"><span>ACS AuthorChoice</span></a><span class="article_header-cc-by-nc"><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_cc-1612971093887.svg" alt="CC: Creative Commons"><span><strong>CC: </strong>Creative Commons</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_by-1612971093257.svg" alt="BY: Credit must be given to the creator"><span><strong>BY: </strong>Credit must be given to the creator</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nc-1612971093307.svg" alt="NC: Only noncommercial uses of the work are permitted"><span><strong>NC: </strong>Only noncommercial uses of the work are permitted</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nd-1612971093637.svg" alt="ND: No derivatives or adaptations of the work are permitted"><span><strong>ND: </strong>No derivatives or adaptations of the work are permitted</span></a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acscentsci.1c00663"><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3302</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" class="doi" value="10.1021/acscentsci.1c00663"><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Parallel Automated Flow Synthesis of Covalent Protein Complexes That Can Inhibit MYC-Driven Transcription&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sebastian&quot;,&quot;last_name&quot;:&quot;Pomplun&quot;},{&quot;first_name&quot;:&quot;Muhammad&quot;,&quot;last_name&quot;:&quot;Jbara&quot;},{&quot;first_name&quot;:&quot;Carly&quot;,&quot;last_name&quot;:&quot;K. Schissel&quot;},{&quot;first_name&quot;:&quot;Susana&quot;,&quot;last_name&quot;:&quot;Wilson Hawken&quot;},{&quot;first_name&quot;:&quot;Ann&quot;,&quot;last_name&quot;:&quot;Boija&quot;},{&quot;first_name&quot;:&quot;Charles&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Isaac&quot;,&quot;last_name&quot;:&quot;Klein&quot;},{&quot;first_name&quot;:&quot;Bradley&quot;,&quot;last_name&quot;:&quot;L. Pentelute&quot;}],&quot;source&quot;:&quot;ACS Central Science&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;volume&quot;:&quot;7&quot;,&quot;pages&quot;:&quot;1408-1418&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acscentsci.1c00663&quot;},&quot;abstract&quot;:&quot;Dysregulation of the transcription factor MYC is involved in many human cancers. The dimeric transcription factor complexes of MYC/MAX and MAX/MAX activate or inhibit, respectively, gene transcription upon binding to the same enhancer box DNA. Targeting these complexes in cancer is a long-standing challenge. Inspired by the inhibitory activity of the MAX/MAX dimer, we engineered covalently linked, synthetic homo- and heterodimeric protein complexes to attenuate oncogenic MYC-driven transcription. We prepared the covalent protein complexes (∼20 kDa, 167–231 residues) in a single shot via parallel automated flow synthesis in hours. The stabilized covalent dimers display DNA binding activity, are intrinsically cell-penetrant, and inhibit cancer cell proliferation in different cell lines. RNA sequencing and gene set enrichment analysis in A549 cancer cells confirmed that the synthetic dimers interfere with MYC-driven transcription. Our results demonstrate the potential of automated flow technology to rapidly deli&quot;}" class="meta-data"><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"><input type="hidden" name="pub-access" value="free" class="pub-access"><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"><input type="hidden" name="selected-share-option" class="selected-share-option"></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facscentsci.1c00663&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.1c00663" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facscentsci.1c00663&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.1c00663" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facscentsci.1c00663&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.1c00663" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facscentsci.1c00663&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.1c00663&amp;href=/doi/10.1021/acscentsci.1c00663" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acscentsci.1c00663"><a href="/doi/pdf/10.1021/acscentsci.1c00663" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acscentsci.1c00663&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fid%3Ddoi%3A10.1021%252Facscentsci.1c00663%26sid%3Dliteratum%253Aachs%26genre%3Darticle%26aulast%3DPomplun%26date%3D2021%26atitle%3DParallel%2BAutomated%2BFlow%2BSynthesis%2Bof%2BCovalent%2BProtein%2BComplexes%2BThat%2BCan%2BInhibit%2BMYC-Driven%2BTranscription%26jtitle%3DACS%2BCentral%2BScience%26title%3DACS%2BCentral%2BScience%26volume%3D7%26issue%3D8%26spage%3D1408%26epage%3D1418%26issn%3D2374-7943" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="acscii" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/acscii?ifr=1"></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollduration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292425" title="Oligomers">Oligomers</a>,</li><li><a href="/action/doSearch?ConceptID=291503" title="Organic polymers">Organic polymers</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/acscii/7/8" title="Go to ACS Central Science "><img src="/cms/10.1021/acscii.2021.7.issue-8/asset/acscii.2021.7.issue-8.largecover.jpg" alt="Go to ACS Central Science "></a></div></div>




        
        <!--Article Page E-Alerts-->
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      gtag('event', 'Article Page Click', { 'event_category': 'Registered e-Alert', 'event_label':'acscii'});

	  }else{
	     //send data to analytics for non-registered users
	      gtag('event', 'Article Page Click', { 'event_category': 'Get e-Alert', 'event_label':'acscii'});
	  }
  });
</script>
<!--End Article Script-->




        
        <div class="pj-badge"><input type="hidden" id="partnerJournalInfo" value="/pb-assets/json/journal-info-1660234310487.json"></div>




        
        <div class="aJhp_link"></div>
<input type="hidden" id="aJhpJournalInfo" value="/pb-assets/json/journal-info-1660234310487.json">
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=acscii" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/cms/10.1021/acscentsci.1c00663/asset/images/medium/oc1c00663_0006.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c00663&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Dysregulation of the transcription factor MYC is involved in many human cancers. The dimeric transcription factor complexes of MYC/MAX and MAX/MAX activate or inhibit, respectively, gene transcription upon binding to the same enhancer box DNA. Targeting these complexes in cancer is a long-standing challenge. Inspired by the inhibitory activity of the MAX/MAX dimer, we engineered covalently linked, synthetic homo- and heterodimeric protein complexes to attenuate oncogenic MYC-driven transcription. We prepared the covalent protein complexes (∼20 kDa, 167–231 residues) in a single shot via parallel automated flow synthesis in hours. The stabilized covalent dimers display DNA binding activity, are intrinsically cell-penetrant, and inhibit cancer cell proliferation in different cell lines. RNA sequencing and gene set enrichment analysis in A549 cancer cells confirmed that the synthetic dimers interfere with MYC-driven transcription. Our results demonstrate the potential of automated flow technology to rapidly deliver engineered synthetic protein complex mimetics that can serve as a starting point in developing inhibitors of MYC-driven cancer cell growth.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="hlFld-Abstract synopsis"><h2>Synopsis</h2><span><p class="articleBody_abstractText">Via parallel automated flow synthesis, we prepared covalent dimeric protein analogues of the transcription factor MYC. The dimers can enter cells and inhibit MYC-driven cancer cell proliferation.</p></span></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81467" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81467" class="dropBlock__holder"></div></div></div></div></div><hr><div class="NLM_p">Access to proteins is essential for academic and industrial research and for modern therapy development. Most proteins are obtained by biological expression. The chemical synthesis of proteins has emerged as a viable complementary approach and offers advantages, including the potential to incorporate unnatural amino acid or post-translational modifications. Chemical protein synthesis usually requires a combination of solid-phase and chemoselective ligation methods. <a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6">(1−6)</a></div><div class="NLM_p">The primary polypeptide sequences alone, however, are not sufficient for activity. Proteins fold into three-dimensionally defined secondary and tertiary structures to make possible diverse and complex tasks in living systems. Further, to reach their ultimate active architecture, many folded proteins form higher-order complexes containing multiple subunits. <a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> Controlling the correct assembly of the desired subunits can be challenging and depends on complex stability, protein concentration, and protein localization. The identity and composition of the subunits forming a protein complex are crucial and determine the complex’s activity.</div><div class="NLM_p">The transcription factor protein MYC forms a heterodimer with MAX in order to bind to the E-Box DNA sequence (CACGTG). <a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The MYC/MAX protein complex is part of the basic-helix–loop-helix/leucine-zipper (bHLH/Lz) transcription factor family and initiates several cellular processes, including cell proliferation and survival. <a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> MAX, alternatively, can homodimerize, compete for the E-Box DNA binding site, and inhibit MYC/MAX-driven transcription. <a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> MYC/MAX and MAX/MAX, thus, have opposite activities, and MYC overexpression is observed in &gt;50% of human cancers. <a onclick="showRef(event, 'ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15 ref16">(13−16)</a></div><div class="NLM_p">Promising strategies to inhibit the oncogenic MYC activity rely on stabilizing the natural MAX/MAX dimer or delivering protein analogues with a similar mechanism of action. The targeting of MYC with small molecules has largely remained elusive, mainly because the structure of MYC presents no binding pockets for small molecule ligands. <a onclick="showRef(event, 'ref17 ref18 ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19 ref20 ref21 ref22">(17−22)</a> To overcome the challenge of drugging MYC, Koehler et al. recently described a small molecule stabilizer of the MAX/MAX complex that inhibited the proliferation of several cancer cell lines and reduced the tumor burden in murine cancer models. <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> In a different approach, Soucek et al. developed the artificial miniprotein Omomyc: this dominant-negative form of MYC can compete for E-Box DNA binding and thus inhibit MYC/MAX-dependent transcription, ultimately resulting in tumor growth inhibition in various mouse models of cancer, causing only mild, well-tolerated, and reversible side effects. <a onclick="showRef(event, 'ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27">(24−27)</a></div><div class="NLM_p">Omomyc, in the same way as the biological MYC and MAX proteins, has to form dimeric complexes to be functional and bioactive. MYC, MAX, and Omomyc can interact with each other in different combinations. Upon delivery of a monomer to the cell, the dominating complex formed depends on the other proteins’ cellular concentrations and is hardly predictable. <a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The direct administration of defined and stable dimeric complexes would offer a superior degree of control over the concentration and composition of the bioactive dimer inhibitor, in addition to a potentially higher structural stability.</div><div class="NLM_p">We reasoned that dimeric analogues of MYC and MAX, covalently linked in their bioactive form, could result in promising oncosuppressive modalities (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). Preparing homogeneous, stable, well-defined protein–protein conjugates can be a challenge. <a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28−30)</a> Chemical synthesis approaches to generate covalently linked multimeric proteins have been mainly focused on preparing ubiquitinylated or sumoylated proteins. <a onclick="showRef(event, 'ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38">(31−38)</a> These strategies relied on chemical ligation or chemoenzymatic workflows, requiring the incorporation of unnatural amino acids or an engineered recognition sequence, respectively. In addition, ligation-based strategies to prepare covalently linked HIV protease heterodimers have been reported by Torbeev et al., with the aim to study asymmetric mutations of this enzyme dimer. <a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Previous dimerization strategies of MYC/MAX analogues relied on disulfide formation at the C-terminus of the leucine zipper region of the transcription factor analogues. <a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Furthermore, in a pioneering study, Kent et al. chemically prepared MYC/MAX and MAX/MAX dimers connected via oxime linkages and thioesters. <a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> While these defined dimers enabled DNA-binding studies, the reported strategies relied on dovetails with low chemical stability in a biological milieu, making them unsuitable for bioactivity studies in vivo.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acscentsci.1c00663/asset/images/medium/oc1c00663_0001.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0001.jpeg" id="gr1" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Covalent protein dimers prepared by automated flow protein synthesis (AFPS) are designed to compete with MYC/MAX for E-Box binding and inhibit oncogenic transcription. Inspired by the inhibitory activity of the MAX/MAX protein dimer assembly, we engineered covalently linked, synthetic homo- and heterodimeric analogues of MYC, MAX, and Omomyc, to inhibit MYC/MAX-dependent gene transcription.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c00663&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We have previously shown that automated fast-flow protein synthesis (AFPS) enables direct access to defined, bioactive, and stable protein domains, <a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a> obviating the need for postsynthesis or expression dimerization reactions, such as native chemical ligation. Solid-phase flow chemistry has been used previously to generate branched glycans. <a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Here, we use AFPS for the parallel single-shot assembly of covalently linked MAX–MAX and Omomyc–Omomyc homodimers and MYC–MAX and Omomyc–MAX heterodimers. The covalent dimers (167–231 residues) were synthesized within hours, and after a single HPLC purification, these protein complexes were directly used for biological studies. Despite their considerable size, the protein dimers are intrinsically cell-penetrating and specifically inhibit MYC-dependent gene transcription and proliferation of cancer cells.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26471" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26471" class="dropBlock__holder"></div></div></div></div></div><hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Single-Shot Flow Synthesis of Covalent Protein Dimers</h3><div class="NLM_p">We prepared covalent MAX–MAX <b>2</b> and Omomyc–Omomyc <b>3</b> homodimers by parallel single-shot fast-flow solid-phase synthesis. In MYC, MAX, and Omomyc dimers, the two C-termini of the DNA binding regions are in proximity to each other. SPPS proceeds from the peptide C-terminus to the N-terminus. Therefore, we envisioned obtaining the covalently linked protein dimers by assembling them starting from a bifunctional linker. We prepared linker <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>b) on a ChemMatrix Rink Amide resin (loading = 0.18 mmol/g). The linker contains an Alloc-protected lysine for late-stage modification, a β-alanine spacer, and a lysine with two free amine groups as a bifunctional starting point for the parallel synthesis. We loaded resin <b>1</b> into the fast-flow synthesizer reactor and prepared MAX–MAX <b>2</b> (164 residues, 3.4 h) and Omomyc–Omomyc <b>3</b> (184 residues, 3.7 h). As determined by in-line UV-Vis monitoring of the stepwise Fmoc deprotection, the synthesis of both dimers proceeded with high coupling efficiency at each coupling cycle (<a href="/doi/suppl/10.1021/acscentsci.1c00663/suppl_file/oc1c00663_si_001.pdf" class="ext-link">Figures S5 and S6</a>). It is noteworthy that each step involved the parallel coupling and subsequent deprotection of two amino acids simultaneously. After cleavage and side-chain deprotection, LC-MS analysis indicated the desired products as the major component of both crude reaction mixtures. Upon preparative HPLC purification, we obtained pure MAX–MAX <b>2</b> and Omomyc–Omomyc <b>3</b> in 6% and 8% yield, respectively (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>e,f). For advanced biological characterizations, we scaled up the Omomyc–Omomyc <b>3</b> preparation. For a single synthesis, we obtained 50 mg of pure material in 14% yield.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acscentsci.1c00663/asset/images/medium/oc1c00663_0002.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0002.jpeg" id="gr2" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Covalently linked dimeric transcription factor analogues were prepared via fast-flow solid-phase synthesis. (a) An automated flow peptide synthesizer built in the Pentelute laboratory. (b) Resin for protein synthesis with lysine-based linkers <b>1</b> or <b>4</b> for parallel dimeric protein synthesis. The linkers were manually coupled on the Rink Amide resin prior to AFPS. (c) Reagents and conditions for flow synthesis. (d) Sequences of MYC, MAX, and Omomyc. (e–h) Synthesis time, yields, and LC-MS characterization of purified homodimers <b>2</b> and <b>3</b> and heterodimers <b>5</b> and <b>6</b>. The panels show the total ion current chromatogram (TIC) as the base spectrum, the electrospray ionization (ESI) mass-to-charge spectra (left inset), and deconvoluted mass spectra (right inset). The peak at 922 in the <i>m</i>/<i>z</i> spectra is the ion of the internal reference compound of the mass spectrometer. The minor MS peak population at masses lower than the expected product mass (in parts e, g, and h) is a mixture of deletion and truncation products.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c00663&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We synthesized covalent MYC–MAX <b>5</b> and Omomyc–MAX <b>6</b> heterodimers by consecutive single-shot fast-flow solid-phase synthesis. To enable the consecutive assembly of the heterodimers, we prepared resin <b>4</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>b): the bifunctional linker, in this case, is based on lysine protected with Alloc on its side-chain (Nε) amine. With the fast-flow synthesizer, we first assembled MAX from the α-amine of the lysine linker. For the last amino acid, we added Boc-glycine and removed the Alloc protection from the Nε of the lysine linker. On this amine, we then assembled MYC or Omomyc, respectively (<a href="/doi/suppl/10.1021/acscentsci.1c00663/suppl_file/oc1c00663_si_001.pdf" class="ext-link">Schemes S1 and S2</a>). The synthesis time for each heterodimer amounted to ∼8 h (MYC–MAX <b>5</b>, 167 residues; Omomyc–MAX <b>6</b>, 175 residues). Both heterodimers were observed as the main component of the crude product mixture obtained from cleavage and side-chain deprotection. Upon preparative HPLC purification, we obtained pure MYC–MAX <b>5</b> and Omomyc–MAX <b>6</b> dimers in 4% and 5% yield, respectively (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>g,h).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Biophysical Characterization of the Synthetic Protein Dimers</h3><div class="NLM_p">Biophysical characterization confirmed the folding and DNA-binding activity of the four covalent protein dimers. We first analyzed dimers <b>2</b>, <b>3</b>, <b>5</b>, and <b>6</b> by sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE). All dimer constructs had bands at the expected height of ∼20 kDa, and the monomers <b>7</b>–<b>9</b> (synthesized by AFPS, see the <a href="/doi/suppl/10.1021/acscentsci.1c00663/suppl_file/oc1c00663_si_001.pdf" class="ext-link">SI</a>, sections 12.5–12.7) were observed at ∼10 kDa (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>a). The refolding of the protein dimers did not require special procedures. We dissolved the lyophilized compounds in folding buffer (MES 10 mM, KCl 150 mM, MgCl<sub>2</sub> 1 mM, TCEP 1 mM, glycerol 10%, pH = 6.5), and all four dimers displayed defined α-helical signatures, as determined by circular dichroism (CD, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>b). We next performed an electrophoretic mobility shift assay (EMSA) to determine the dimers’ DNA binding activity. At a DNA concentration of 1 μM and protein concentration of 2 μM, all dimers formed complexes with the E-Box DNA, as observed by the shift retardation on the gel (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>c). Monomeric MYC, tested as a negative control (4 μM), did not bind to E-Box DNA.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acscentsci.1c00663/asset/images/medium/oc1c00663_0003.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0003.jpeg" id="gr3" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. The synthetic protein dimers are homogeneous, fold, and associate to E-box DNA. The covalent linkage stabilizes the structure and DNA/protein complexes. (a) SDS-PAGE analysis of synthetic protein dimers and monomers (∼1 μg per protein loaded). The bands were visualized by Coomassie Blue staining. (b) Proteins were dissolved in folding buffer (MES 10 mM, KCl 150 mM, MgCl<sub>2</sub> 1 mM, TCEP 1 mM, glycerol 10%, pH = 6.5) at 0.1 mg/mL, and full wavelength CD spectra were recorded from 250 to 200 nm at 4 °C. The <i>y</i>-axis shows the mean residual ellipticity (MRE) and the <i>x</i>-axis the wavelength. (c) Each compound was incubated with E-Box DNA in folding buffer (final concentrations: 2 μM protein and 1 μM DNA) for 1 h at room temperature. Samples were run on 10% polyacrylamide gel in TBE buffer and visualized with ethidium bromide. (d) For melting temperature determination, CD spectra were recorded at 221 nm from +4 to +89 °C, in 5 °C steps. (e) Summary table of melting temperatures determined by CD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c00663&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Covalently linked dimers have stabilized structures in aqueous buffer compared to their noncovalent analogues. We recorded CD signals at 221 nm between 4 and 89 °C (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>d) to determine melting temperatures (<i>T</i><sub>m</sub>). With this method, we compared the noncovalent MAX/MAX (<b>8/8</b>) and Omomyc/Omomyc (<b>9/9</b>) dimers to our four synthetic covalent protein dimers (note: noncovalent dimers are indicated as protein/protein; covalent dimers are indicated as protein–protein). We also performed protein melting temperature measurements in the presence of equimolar E-Box DNA. Overall, the DNA stabilized the protein complexes’ structures, in agreement with the literature. <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The covalent linkage showed a significant stabilizing effect on the MAX dimers: we determined the <i>T</i><sub>m</sub> of the noncovalent MAX structure being 29 °C and the one of the covalent dimer 38 °C. Omomyc complexes, overall, displayed a higher structural stability than the other dimers tested. We did not observe a significant <i>T</i><sub>m</sub> difference for noncovalent Omomyc/Omomyc (<b>9/9</b>) compared to the covalent Omomyc–Omomyc (<b>3</b>). This observation might be explained by the greater stability of the Omomyc leucine zipper. The most stable complex of all structures tested was the covalent Omomyc–Omomyc dimer (<b>3</b>) in the presence of DNA, with a <i>T</i><sub>m</sub> of 67 °C. Finally, we tested the proteolytic stability of dimer <b>3</b>. After 1 h of incubation in human serum (5% in PBS) at 37 °C, we found 91% intact protein dimer remaining (<a href="/doi/suppl/10.1021/acscentsci.1c00663/suppl_file/oc1c00663_si_001.pdf" class="ext-link">Figure S1</a>).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Assessment of Cell-Penetration of the Protein Dimers</h3><div class="NLM_p">To assess cell penetration via microscopy and flow cytometry, we prepared variants of MAX–MAX and Omomyc–Omomyc labeled with 5-carboxytetramethylrhodamine (TAMRA). We performed the TAMRA labeling on the solid phase, upon Boc protection of the N-termini and removal of the Alloc protection from the C-terminal lysine, obtaining <b>10</b> and <b>11</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>a). We also synthesized an Omomyc–Omomyc derivative containing Mach3, a nuclear-targeting miniprotein that we recently designed. <a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> To obtain this 231 residue covalent protein complex, we removed Alloc from the C-terminal lysine and synthesized, by AFPS, the 41-residue Mach3 sequence from the resulting free amine and finally added a TAMRA label, obtaining compound <b>12</b>.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acscentsci.1c00663/asset/images/medium/oc1c00663_0004.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0004.jpeg" id="gr4" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Synthetic dimers are intrinsically cell-penetrating, and nuclear targeting is enhanced by non-natural sequence tags. (a) Reaction conditions. Alloc removal: Pd(PPh<sub>3</sub>)<sub>4</sub>, piperidine, CH<sub>2</sub>Cl<sub>2</sub>, r.t., 30 min. TAMRA labeling: 5-TAMRA, HATU, DIEA, DMF, r.t., 30 min. Mach3 synthesis: see AFPS conditions (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>c). Cleavage: TFA, EDT, thioanisole, cresol, H<sub>2</sub>O, r.t., 4 h. (b) TIC-LCMS chromatograms of MAX–MAX–TAMRA <b>10</b>, Omomyc–Omomyc–TAMRA <b>11</b>, and Omomyc–Omomyc–Mach3–TAMRA <b>12</b> with <i>m</i>/<i>z</i> and deconvoluted mass. (c) Flow cytometry histograms illustrating the dose-dependent increase in fluorescence of HeLa cells after 15 min of incubation with TAMRA-labeled dimers and Omomyc–TAMRA monomer at concentrations between 0.01 and 15 μM. (d) Micrographs from confocal microscopy; Hoechst (DAPI) labels the nuclei, and TAMRA-protein (Cy3) is observed throughout the cell after 15 min of incubation, followed by incubation in fresh media for 1 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c00663&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The synthetic dimers are intrinsically cell-penetrating while remaining nonlytic, and nuclear localization is enhanced by the addition of a targeting motif. Omomyc monomer (90 residues) has been shown to be an intrinsically cell-penetrating protein. <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The arginine-rich DNA binding region appears to be the primary driving force for cell penetration. We recently showed that the combination of multiple cell-penetrating sequences into a single molecule can enhance cell-penetrating activity with a synergistic effect. <a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Based on this rationale, we expected to observe cell-penetrating activity of our dimeric protein complexes containing two polycationic domains. In addition, we were curious if the miniprotein Mach3 could further enhance nuclear localization. To evaluate uptake, we first treated HeLa cells with fluorophore-labeled proteins and measured fluorescence via flow cytometry. All three analogues are taken up into cells in a dose-dependent manner after a brief (15 min) incubation (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>c and <a href="/doi/suppl/10.1021/acscentsci.1c00663/suppl_file/oc1c00663_si_001.pdf" class="ext-link">Figure S4</a>). The addition of (4′,6-diamidin-2-phenylindol) DAPI as a membrane-impermeable viability dye showed no staining of the gated population of TAMRA-fluorescent cells, suggesting that the constructs entered cells without compromising the membrane (<a href="/doi/suppl/10.1021/acscentsci.1c00663/suppl_file/oc1c00663_si_001.pdf" class="ext-link">Figures S2 and S3</a>). These findings were confirmed by fluorescence microscopy. The treatment of HeLa cells for 15 min with MAX–MAX–TAMRA <b>10</b> and Omomyc–Omomyc–TAMRA <b>11</b> followed by 1 h of incubation in fresh media and imaging via confocal microscopy revealed intense, punctate fluorescence, in agreement with previous observations and our observations on Omomyc (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>d). <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> However, treatment with Omomyc–Omomyc–Mach3 <b>12</b> resulted in punctate fluorescence as well as diffuse fluorescence in the nucleus, indicating endosomal escape and nuclear localization (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>d). These experiments show that the dimeric transcription factors are rapidly taken up into cells, and their nuclear localization can be improved with the addition of a non-natural targeting sequence. <a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Inhibition of Cancer Cell Proliferation and MYC-Driven Transcription upon Omomyc–Omomyc Treatment</h3><div class="NLM_p">The synthetic transcription factor analogue dimers inhibit the proliferation of cancer cells. MYC is known to drive cell proliferation in the majority of human cancers. <a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> We reasoned that our synthetic dimers’ combined ability to bind E-box DNA and enter cells should result in the inhibition of MYC-dependent gene transcription and cell proliferation. We tested the bioactivity of all compounds in three cell lines with a range of MYC expression levels; HeLa contains high MYC levels, A549 displays mid-level MYC expression, and H441 has low MYC expression. <a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> We treated the cells for 72 h with protein dimers and measured the proliferation with a CellTiter-Glo (CTG) assay (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>a). Cell proliferation inhibition followed the expected trend according to MYC expression levels; we observed the most substantial inhibition in HeLa cells and the weakest in H441 cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>b). All synthetic dimers demonstrated inhibitory activity, with the Omomyc–Omomyc dimer having the highest activity of the native dimers with an EC<sub>50</sub> of 4 μM. This observation is in line with our structural stability data. Moreover, Omomyc–Omomyc–Mach3 further decreased the EC<sub>50</sub> in each cell line (2 μM in HeLa cells), indicating that the nuclear-targeting moiety assists the transcription factor in reaching its target and imparts enhanced activity.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/acscentsci.1c00663/asset/images/medium/oc1c00663_0005.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0005.jpeg" id="gr5" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Synthetic transcription factors inhibit cancer cell proliferation and downregulate MYC target genes. (a) Cell proliferation assay of HeLa, A549, and H441 cells following treatment with synthetic protein dimers for 72 h, quantified via CellTiter-Glo (CTG). Experiments were performed in triplicate. (b) Summary table of proliferation inhibition EC<sub>50</sub> values. (c) A549 cells were treated with Omomyc–Omomyc <b>3</b> (12.5 μM) for 72 h. RNA was extracted and sequenced. Depicted is an MA plot showing differentially expressed genes (DEGs) in A549 cells treated with Omomyc–Omomyc <b>3</b> compared to DMSO. Upregulated genes with adjusted <i>p</i>-value &lt;0.05 and |log 2FC| ≥ 1 shown in red, downregulated genes with <i>p</i>-value &lt;0.05 and |log 2FC| ≤ 1 shown in blue. The experiment was performed in duplicate. Downregulated genes involved in <i>KRas</i> pathways are labeled. (d) GSEA comparing gene expression of DMSO treated cells compared to Omomyc–Omomyc. The enrichment plot of the MYC target gene signature shows a negative enrichment in the Omomyc–Omomyc condition (<i>q</i>-value &lt;0.05).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c00663&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The synthetic transcription factor analogue dimers interfere with MYC-driven gene expression, as determined by RNA-sequencing (RNA-seq) and gene set enrichment analysis (GSEA). To evaluate whether the compounds’ bioactivity is related to the suppression of MYC-driven expression, we performed RNA-seq on A549 cells treated with Omomyc–Omomyc <b>3</b>. Compared to the control cells, we observed downregulation of 431 and upregulation of 297 genes, indicating that the synthetic dimers have an effect on gene expression (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>c). Among the downregulated genes, we found several genes involved in <i>KRas</i> signaling pathways, which are known to drive cancer development in A549 nonsmall lung cancer cells. <a onclick="showRef(event, 'ref46 ref49'); return false;" href="javascript:void(0);" class="ref ref46 ref49">(46,49)</a> This finding is in accordance with previous reports showing that MYC is a dominant effector of <i>KRas</i> mutation-positive lung cancer pathogenesis. <a onclick="showRef(event, 'ref48 ref50'); return false;" href="javascript:void(0);" class="ref ref48 ref50">(48,50)</a> GSEA of our RNA-seq data shows a negative enrichment of the MYC-target gene set in the Omomyc–Omomyc <b>3</b> treated condition, further corroborating that the synthetic dimers interfere with MYC-driven gene expression programs (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>d).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10401" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10401" class="dropBlock__holder"></div></div></div></div></div><hr><div class="NLM_p">We show that flow synthesis enables rapid access to covalently linked homo- and heterodimeric analogues of MYC, MAX, and Omomyc. The automated flow synthesizers developed in our laboratory have been previously shown to provide chemical access to single-domain functional proteins up to 164 residues. <a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Here, we used dimeric linkers and orthogonal protecting group strategies to synthesize homodimeric and heterodimeric proteins with up to 231 residues and synthetic modifications. In our approach, two polypeptide chains are assembled in close proximity to each other, and no significant aggregation events interfered with the efficient double assembly of the dimeric proteins. The use of an orthogonally protected C-terminal lysine residue further enabled the practical late-stage TAMRA labeling of the protein dimers directly on the solid phase.</div><div class="NLM_p">MYC/MAX is a dimeric transcription factor complex that has been the object of intense studies due to its predominant role in cancer pathogenesis. <a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> While the therapeutic targeting of MYC with classical small molecule drugs has been challenging, recent work has indicated that transcription factor analogues themselves, such as Omomyc or MAX, could be used to affect MYC activity. <a onclick="showRef(event, 'ref25 ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref25 ref51 ref52">(25,51,52)</a> Omomyc, which inhibits MYC-dependent proliferation in cancer cell lines and mouse models, recently entered human clinical trials. <a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></div><div class="NLM_p">Given the importance of MYC as a drug target and the challenges related to previous inhibition strategies, our new chemical modalities are of high interest. We showed our dimeric protein complexes being cell-penetrating and interfering with MYC-driven oncogenic transcription. Building on the encouraging data reported for Omomyc, the use of defined covalent protein complexes, as described here, might represent a promising modality for cancer treatment. Indeed, we found our engineered protein dimers having inhibitory activity on MYC-driven proliferation comparable to Omomyc (in the low micromolar range).</div><div class="NLM_p">Transcription factors might be privileged scaffolds for intracellular delivery. Reaching intracellular targets with biomolecular compounds, such as proteins and oligonucleotides, usually requires the conjugation to cell-penetrating peptides, <a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> or the use of delivery vectors. <a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> The DNA binding regions of several transcription factors often are polycationic and α-helical, two traits of cell-penetrating peptides. <a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Indeed, several transcription factor proteins were independently shown to be intrinsically cell-penetrating, <a onclick="showRef(event, 'ref57 ref58 ref59 ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref57 ref58 ref59 ref60 ref61">(57−61)</a> including Soucek’s discovery regarding Omomyc. <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Our group recently reported the synergistic effect of multiple cell-penetrating moieties combined in one sequence. <a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> The two cationic helices of the dimeric MYC/MAX mimetics might therefore be the driving force for the intrinsic cell-penetrating activity of these ∼20 kDa macromolecules.</div><div class="NLM_p">Our parallel protein dimer synthesis mimics nature’s strategy of forming protein complexes. Many proteins associate into macromolecular complexes to become functional. Complex formation requires that subunits find each other in the crowded cellular environment while avoiding unspecific interactions. Kramer, Bukau, et al. have recently shown that native protein complex formation is enabled by parallel ribosomal cotranslation, folding, and assembly. <a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> In close analogy to this biological strategy, we also synthesized our protein dimers in parallel and in close proximity to each other, priming them for correct complex formation.</div><div class="NLM_p last">Dimeric protein complexes, such as bHLH or bZIP (basic leucine zippers), make up hundreds of transcription factors, <a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> and our results could define a framework that enables access to several analogues of these families. These synthetic compounds are valuable tools for the study of gene transcription and possibly disease treatment. We envision that synthetic transcription factor analogues in the future will enable on-demand gene regulation, and our synthesis strategy will enable access to defined protein complexes in their biologically relevant multimeric forms.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71778" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71778" class="dropBlock__holder"></div></div></div></div></div><hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Manual Preparation of Peptidyl Resins <b>1</b> and <b>4</b></h3><div class="NLM_p last">ChemMatrix Rink Amide resin (loading 0.18 mmol/g, typical scale: 100 mg, 0.02 mmol) was loaded into a fritted syringe (6 mL), swollen in DMF (4 mL) for 5 min, and then drained. Each Nα-Fmoc protected amino acid (0.2 mmol, 10 equiv) was dissolved in DMF containing 0.39 M HATU (0.5 mL). Immediately before the coupling, DIEA (100 μL, 30 equiv) was added to the mixture to activate the amino acid. After 15 s of preactivation, the mixture was added to the resin and reacted for 10 min, with occasional stirring. After completion of the coupling step, the syringe was drained, and the resin was washed with DMF (3 × 5 mL). Fmoc deprotection was performed by the addition of piperidine (20% in DMF, 3 mL) to the resin (1 × 1 min + 1 × 5 min), followed by draining and washing the resin with DMF (5 × 5 mL). For peptidyl resin <b>1</b>, the coupling cycles were performed sequentially with Fmoc–Lys(Alloc)–OH, Fmoc−βAla–OH, and Fmoc–Lys(Fmoc)–OH; for peptidyl resin <b>4</b>, the coupling cycles were performed sequentially with Fmoc–Lys(Boc)–OH, Fmoc−βAla–OH, and Fmoc–Lys(Alloc)–OH.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Automated Flow Peptide Synthesis (AFPS) Setup</h3><div class="NLM_p last">All peptides were synthesized on two automated-flow systems custom built in the Pentelute lab, which are similar to the published AFPS system. <a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> The synthesis conditions used are according to Hartrampf et al.: <a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> flow-rate = 40 mL/min, temperature = 90 °C (loop 1), 70 °C (loop 2; used for histidine), and 85–90 °C (reactor). The 50 mL/min pump head pumps 400 μL of liquid per pump stroke; the 5 mL/min pump head pumps 40 μL of liquid per pump stroke. The standard synthetic cycle involves a first step of prewashing the resin at elevated temperatures for 60 s at 40 mL/min. During the coupling step, three HPLC pumps are used: a 50 mL/min pump head pumps the activating agent, a second 50 mL/min pump head pumps the amino acid, and a 5 mL/min pump head pumps DIEA. The first two pumps are activated for 8 pumping strokes in order to prime the coupling agent and amino acid before the DIEA pump is activated. The three pumps are then actuated together for a period of 7 pumping strokes, after which the activating agent pump and amino acid pump are switched using a rotary valve to select DMF. The three pumps are actuated together for a final 8 pumping strokes, after which the DIEA pump is shut off, and the other two pumps continue to wash the resin for another 40 pump strokes. During the deprotection step, two HPLC pumps are used. Using a rotary valve, one HPLC pump selects deprotection stock solution and DMF. The pumps are activated for 13 pump strokes. Both solutions are mixed in a 1:1 ratio. Next, the rotary valves select DMF for both HPLC pumps, and the resin is washed for an additional 40 pump strokes. The coupling–deprotection cycle is repeated for all additional monomers.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Manual Boc–Gly–OH coupling</h3><div class="NLM_p last">Prior to site-selective modification via the Alloc-protected lysine, we blocked the protein N-termini with Boc–Gly–OH: Peptidyl resin (∼10 μmol theoretical loading) was loaded into a fritted syringe (6 mL), swollen in DMF (4 mL) for 5 min, and then drained. Boc–Gly–OH (18 mg, 100 μmol) and HATU (54 mg, 90 μmol) were dissolved in DMF (250 μL), activated with DIEA (38 mg, 52 μL, 300 μmol), added to the peptidyl resin, and incubated for 15 min. After this time, the resin was drained, washed with DMF (3 × 5 mL), and used for the next step.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Alloc Deprotection</h3><div class="NLM_p last">The peptidyl resin (∼10 μmol theoretical loading) was washed with dichloromethane (3 × 5 mL) and then treated with Pd(PPh<sub>3</sub>)<sub>4</sub> (11.0 mg, 10 μmol, 1 equiv) in dichloromethane/piperidine (8:2, 1 mL) for 30 min at room temperature under the exclusion of light. The resin was then drained and washed with dichloromethane (3 × 5 mL).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> TAMRA Labeling</h3><div class="NLM_p last">Peptidyl resin (∼10 μmol theoretical loading) was loaded into a fritted syringe (6 mL), swollen in DMF (4 mL) for 5 min, and then drained. 5-Carboxytetramethylrhodamine (5-TAMRA, 22 mg, 50 μmol, 5 equiv) and HATU (17 mg, 45 μmol, 4.5 equiv) were dissolved in DMF (500 μL), activated with DIEA (19 mg, 26 μL, 150 μmol), added to the peptidyl resin, and incubated for 30 min under the exclusion of light. After this time, the resin was drained, washed with DMF (3 × 5 mL), and stored until cleavage.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Cleavage Protocol</h3><div class="NLM_p last">After synthesis, the peptidyl resin was washed with dichloromethane (3 × 5 mL) and dried. Approximately 8 mL of cleavage solution (82.5% TFA, 5% water, 5% phenol, 5% thioanisole, 2.5% EDT) was added to the peptidyl resin inside the fritted syringe. The cleavage was kept at room temperature for 4 h, with occasional shaking. After this time, the cleavage mixture was transferred to a falcon tube (through the syringe frit, keeping the resin in the syringe), and the resin was washed with an additional 2 mL of cleavage solution. Ice cold diethyl ether (45 mL) was added to the cleavage mixture, and the precipitate was collected by centrifugation and triturated twice more with cold diethyl ether (45 mL). The supernatant was discarded. Residual ether was allowed to evaporate, and the peptide was dissolved in 50% acetonitrile in water with 0.1% TFA (long peptides were dissolved 70% acetonitrile in water with 0.1% TFA). The peptide solution was filtered with a Nylon 0.22 μm syringe filter, frozen, and then lyophilized until dry.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Polyacrylamide Gel Electrophoresis (PAGE)</h3><div class="NLM_p last">The analysis was performed using Bolt 4–12% Bis-Tris Plus gels (10 wells) at 165 V for 36 min utilizing prestained Invitrogen SeeBlue Plus2 molecular weight standard. Bolt LDS sample buffer (4×) was added to each protein sample (1 μg) for loading on the gel. The bands were visualized by Coomassie blue staining.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Electrophoretic Mobility Shift Assay (EMSA)</h3><div class="NLM_p last">The E-Box DNA probe (2 μM in binding buffer) was heated to 95 °C for 5 min and then allowed to cool down to room temperature over 15 min for double-strand annealing. Protein dimer (4 μM in binding buffer: MES 10 mM, KCl 150 mM, MgCl<sub>2</sub> 1 mM, TCEP 1 mM, glycerol 10%, pH = 6.5) was added to the DNA (final concentrations: 2 μM protein and 1 μM DNA), and the mixture was incubated for 1 h at room temperature. During the incubation, a 10% polyacrylamide gel was prerun (1 h, 4 °C, 100 V) in 1× TBE buffer. After that time, DNA protein mixture (20 μL) was mixed with 6× DNA loading dye (4 μL)) and loaded on the gel, which was run at 75 V, for 90 min at 4 °C. The gel was washed with water for 20 s and then stained with 0.02% ethidium bromide in 1× TBE buffer for 15 min at room temperature. Bands were visualized on a Biorad Gel imager.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Circular Dichroism (CD)</h3><div class="NLM_p last">Lyophilized samples were dissolved in folding buffer (MES 10 mM, KCl 150 mM, MgCl<sub>2</sub> 1 mM, TCEP 1 mM, glycerol 10%, pH = 6.5) at a final protein concentration of 0.1 mg/mL. The circular dichroism (CD) spectra were obtained using an AVIV 420 circular dichroism spectrometer with a 1 mm path length quartz cuvette. 300 μL samples were used for each measurement. For full wavelength scans, the CD spectra were recorded from 250 to 200 nm at 4 °C with 3 s averaging times at each wavelength. <i>Y</i>-axis values are reported in molar ellipticity. For melting temperature determination, CD spectra were recorded at 221 nm from 4 to 89 °C, with +5 °C steps and equilibration times of 60 s at each temperature. For measurements of DNA/protein complexes, equimolar E-Box DNA was added to the proteins in folding buffer; the mixtures were heated to 95 °C for 5 min, allowed to cool down to room temperature over 15 min, and then analyzed.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Cell Culture</h3><div class="NLM_p last">HeLa (ATCC CCL-2), A549 (ATCC CCL-185), and H441 (ATCC HTB-174) cancer cell lines were maintained in MEM, FK-12, and RPMI-1640 media each containing 10% v/v fetal bovine serum (FBS) and 1% v/v penicillin–streptomycin, respectively, at 37 °C and 5% CO<sub>2</sub>. Cells were passaged at 80% confluency using 0.25% trypsin–EDTA.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Flow Cytometry</h3><div class="NLM_p last">HeLa cells were plated at 10 000 cells per well in a 96-well plate the night before the experiment. On the day of the experiment, cells were treated with the indicated concentrations of TAMRA–Omo, TAMRA–OmoOmo, or TAMRA–OmoOmo–Mach3 for 15 min in serum-containing culture medium, washed once with PBS, and treated with 0.25% trypsin–EDTA for 30 min to digest membrane-bound protein, at 37 °C and 5% CO<sub>2</sub>. Cells were then washed with PBS, incubated in PBS containing 1× DAPI for 3 min, and then resuspended in PBS containing 2% FBS. Cells were then immediately analyzed on a BD FACS LSR II instrument using DAPI and PE channels.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Cell Proliferation Inhibition Assay</h3><div class="NLM_p last">Cells were plated at 5000 cells/well in a 96-well plate the day before the experiment. Synthetic proteins were prepared at varying concentrations in complete media and transferred to the plate. Cells were incubated at 37 °C and 5% CO<sub>2</sub> for 72 h, and cell proliferation was measured using the CellTiter-Glo assay quantified by luminescence.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Microscopy</h3><div class="NLM_p last">HeLa cells were plated at 10 000 cells/well in a 96-well 30 mm glass-bottom plate the night before the experiment. On the day of the experiment, cells were treated with TAMRA–Omo, TAMRA–OmoOmo, or TAMRA–OmoOmo–Mach3 (5 μM) in complete medium for 15 min, washed twice with fresh medium, and incubated at 37 °C and 5% CO<sub>2</sub> for 1 h before imaging. Micrographs were obtained on an RPI spinning disk confocal microscope on the RFP setting (561 nm 100 mW OPSL excitation laser, 605/70 nm emission) and DAPI setting (405 nm 100 mW OPSL excitation laser, 450/50 nm emission).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> RNA-seq and GSEA</h3><div class="NLM_p last">In a 6-well plate, 125 000 A549 cells were plated into each well. The following day, the cells were treated with Omomyc–Omomyc dimer (12.5 μM) in F12K media supplemented with 10% FBS and 1% pen/strep and incubated for 72 h. RNA was isolated using the Qiagen RNeasy Plus mini kit (74136) followed by DNase treatment (AM1906). KAPAHyperRiboErase libraries were prepared and sequenced on a Hi-seq 2500 instrument. Reads from sequencing were aligned using the HISAT2 htseq-count function. Differential gene expression analysis between treated and control cells was performed using the DESEQ2 package in R on raw aligned read counts. The differentially expressed genes were ranked by their log 2FC and adjusted <i>p</i>-value. Preranked gene set enrichment analysis (GSEA) was performed using gene sets in the Molecular Signatures Database (MSigBD) to identify MYC-target gene sets.</div></div><div id="sec999" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Safety Statement</h3><div class="NLM_p last">No unexpected or unusually high safety hazards were encountered.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80631" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80631" class="dropBlock__holder"></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acscentsci.1c00663?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acscentsci.1c00663</a>.</p><ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthesis and experimental procedures as well as characterization of all compounds (<a href="/doi/suppl/10.1021/acscentsci.1c00663/suppl_file/oc1c00663_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.1c00663/suppl_file/oc1c00663_si_001.pdf">oc1c00663_si_001.pdf (3.91 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acscentsci.1c00663" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms &amp; Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22780" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22780" class="dropBlock__holder"></div></div></div></div></div><hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bradley L. Pentelute</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Massachusetts Institute of
Technology, 77 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span>;&nbsp;
    <span class="hlFld-Affiliation affiliation">The
Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, 500 Main Street, Cambridge, Massachusetts 02142, United States</span>;&nbsp;
    <span class="hlFld-Affiliation affiliation">Center
for Environmental Health Sciences, Massachusetts
Institute of Technology, 77 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span>;&nbsp;
    <span class="hlFld-Affiliation affiliation">Broad Institute
of MIT and Harvard, 415
Main Street, Cambridge, Massachusetts 02142, United States</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="https://orcid.org/0000-0002-7242-801X" title="Orcid link">https://orcid.org/0000-0002-7242-801X</a>;&nbsp;
    </span><span class="author-email">
                Email: <a href="mailto:blp@mit.edu">blp@mit.edu</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sebastian Pomplun</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Massachusetts Institute of
Technology, 77 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Muhammad Jbara</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Massachusetts Institute of
Technology, 77 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="https://orcid.org/0000-0002-4206-5908" title="Orcid link">https://orcid.org/0000-0002-4206-5908</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carly K. Schissel</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Massachusetts Institute of
Technology, 77 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="https://orcid.org/0000-0003-0773-5168" title="Orcid link">https://orcid.org/0000-0003-0773-5168</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Susana Wilson Hawken</span> - <span class="hlFld-Affiliation affiliation">Whitehead
Institute for Biomedical Research, Cambridge, Massachusetts 02142, United States</span>;&nbsp;
    <span class="hlFld-Affiliation affiliation">Department
of Biology, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ann Boija</span> - <span class="hlFld-Affiliation affiliation">Whitehead
Institute for Biomedical Research, Cambridge, Massachusetts 02142, United States</span>;&nbsp;
    <span class="hlFld-Affiliation affiliation">Department
of Biology, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Charles Li</span> - <span class="hlFld-Affiliation affiliation">Whitehead
Institute for Biomedical Research, Cambridge, Massachusetts 02142, United States</span>;&nbsp;
    <span class="hlFld-Affiliation affiliation">Department
of Biology, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Isaac Klein</span> - <span class="hlFld-Affiliation affiliation">Whitehead
Institute for Biomedical Research, Cambridge, Massachusetts 02142, United States</span>;&nbsp;
    <span class="hlFld-Affiliation affiliation">Department
of Biology, Massachusetts Institute of Technology, Cambridge, Massachusetts 02139, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p class="last">S.P. and M.J. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): B.L.P. is a cofounder of Amide Technologies and Resolute Bio. Both companies focus on the development of protein and peptide therapeutics. MIT is in the process of filing a provisional patent application regarding the compounds described in this study. A.B. is an advisor to Syros Pharmaceuticals.<br></div></li></ul></div><div class="ack" id="ACK-d7e1020-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27322" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27322" class="dropBlock__holder"></div></div></div></div></div><hr><p class="last">We gratefully acknowledge Prof. Richard A. Young for his generous support of this project. We thank Wendy C. Salmon at the W. M. Keck Microscopy Facility at the Whitehead Institute for help with imaging, and the Swanson Biotechnology Center Flow Cytometry Facility at the Koch Institute for the use of their flow cytometers. C.K.S. (4000057398) acknowledges financial support from the National Science Foundation Graduate Research Fellowship (NSF Grant 1122374). This research was supported by a Bristol-Myers Squibb Unrestricted Grant in Synthetic Organic Chemistry awarded to B.L.P. S.P. thanks the Deutsche Forschungsgemeinschaft for a postdoctoral fellowship (DFG, PO 2413/1-1). M.J. gratefully acknowledges postdoctoral fellowship support from the Rothschild Foundation, the Fulbright Program, and the Israel Council for Higher Education (VATAT). A.B. acknowledges the Swedish Research Council for a postdoctoral fellowship (VR 2017-00372). We thank Dr. Andrei Loas for help with the preparation of this manuscript.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80135" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80135" class="dropBlock__holder"></div></div></div></div></div><hr></div><p class="references-count">
            This article references 62 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muir, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark-Lewis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, S. B. H.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Proteins by Native Chemical Ligation</span>. <i>Science</i>  <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>266</i></span>,  <span class="NLM_fpage">776</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.7973629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1126%2Fscience.7973629" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=7973629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADyaK2MXitVGgtrw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=266&amp;publication_year=1994&amp;pages=776&amp;author=P.+E.+Dawsonauthor=T.+W.+Muirauthor=I.+Clark-Lewisauthor=S.+B.+H.+Kent&amp;title=Synthesis+of+Proteins+by+Native+Chemical+Ligation&amp;doi=10.1126%2Fscience.7973629" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of proteins by native chemical ligation</span></div><div class="casAuthors">Dawson, Philip E.; Muir, Tom W.; Clark-Lewis, Ian; Kent, Stephen B. H.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">266</span>
        (<span class="NLM_cas:issue">5186</span>),
    <span class="NLM_cas:pages">776-9</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A simple technique has been devised that allows the direct synthesis of native backbone proteins of moderate size.  Chemoselective reaction of two unprotected peptide segments gives an initial thioester-linked species.  Spontaneous rearrangement of this transient intermediate yields a full-length product with a native peptide bond at the ligation site.  The utility of native chem. ligation was demonstrated by the one-step prepn. of a cytokine contg. multiple disulfides.  The polypeptide ligation product was folded and oxidized to form the native disulfide-contg. protein mol.  Native chem. ligation is an important step toward the general application of chem. to proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuXFAQmIIOgrVg90H21EOLACvtfcHk0lgMDhui3JOQ2Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXitVGgtrw%253D&amp;md5=1d352d817ab48851500610dcd32d1136</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.7973629&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7973629%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DP.%2BE.%26aulast%3DMuir%26aufirst%3DT.%2BW.%26aulast%3DClark-Lewis%26aufirst%3DI.%26aulast%3DKent%26aufirst%3DS.%2BB.%2BH.%26atitle%3DSynthesis%2520of%2520Proteins%2520by%2520Native%2520Chemical%2520Ligation%26jtitle%3DScience%26date%3D1994%26volume%3D266%26spage%3D776%26doi%3D10.1126%2Fscience.7973629" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bode, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baucom, K. D.</span></span> <span> </span><span class="NLM_article-title">Chemoselective Amide Ligations by Decarboxylative Condensations of N-Alkylhydroxylamines and α-Ketoacids</span>. <i>Angew. Chem., Int. Ed.</i>  <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1248</span>– <span class="NLM_lpage">1252</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.200503991</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1002%2Fanie.200503991" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvFyjsrY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=2006&amp;pages=1248-1252&amp;issue=8&amp;author=J.+W.+Bodeauthor=R.+M.+Foxauthor=K.+D.+Baucom&amp;title=Chemoselective+Amide+Ligations+by+Decarboxylative+Condensations+of+N-Alkylhydroxylamines+and+%CE%B1-Ketoacids&amp;doi=10.1002%2Fanie.200503991" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoselective amide ligations by decarboxylative condensations of N-alkylhydroxylamines and α-keto acids</span></div><div class="casAuthors">Bode, Jeffrey W.; Fox, Ryan M.; Baucom, Kyle D.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1248-1252</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH &amp; Co. KGaA</span>)
        </div><div class="casAbstract">The chemoselective amide bond-forming ligation between N-alkylhydroxylamines and α-keto acids requires no reagents and the only byproducts are water and carbon dioxide.  This process proceeds on unprotected peptide substrates without epimerization, and as such, this process has the potential to serve as a novel chemoselective ligation for the synthesis of peptides and complex materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1JDHd7uK1r7Vg90H21EOLACvtfcHk0ljkAum8TFIHFw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28XhvFyjsrY%253D&amp;md5=ca37ba7faf34db1a8b86fd4dcc7f1b54</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fanie.200503991&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200503991%26sid%3Dliteratum%253Aachs%26aulast%3DBode%26aufirst%3DJ.%2BW.%26aulast%3DFox%26aufirst%3DR.%2BM.%26aulast%3DBaucom%26aufirst%3DK.%2BD.%26atitle%3DChemoselective%2520Amide%2520Ligations%2520by%2520Decarboxylative%2520Condensations%2520of%2520N-Alkylhydroxylamines%2520and%2520%25CE%25B1-Ketoacids%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2006%26volume%3D45%26issue%3D8%26spage%3D1248%26epage%3D1252%26doi%3D10.1002%2Fanie.200503991" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Premdjee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larance, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conde-Frieboes, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, R. J.</span></span> <span> </span><span class="NLM_article-title">Chemical Synthesis of Phosphorylated Insulin-like Growth Factor Binding Protein 2</span>. <i>J. Am. Chem. Soc.</i>  <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>143</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">5336</span>– <span class="NLM_lpage">5342</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.1c02280</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.1c02280" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnsl2ktLo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=143&amp;publication_year=2021&amp;pages=5336-5342&amp;issue=14&amp;author=B.+Premdjeeauthor=A.+S.+Andersenauthor=M.+Laranceauthor=K.+W.+Conde-Frieboesauthor=R.+J.+Payne&amp;title=Chemical+Synthesis+of+Phosphorylated+Insulin-like+Growth+Factor+Binding+Protein+2&amp;doi=10.1021%2Fjacs.1c02280" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical synthesis of phosphorylated insulin-like growth factor binding protein 2</span></div><div class="casAuthors">Premdjee, Bhavesh; Andersen, Asser S.; Larance, Mark; Conde-Frieboes, Kilian W.; Payne, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5336-5342</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. protein synthesis is a powerful avenue for accessing homogeneously modified proteins.  While a significant no. of small modified proteins bearing native post-translational modifications and non-natural modifications have been generated to date, access to larger targets has proved challenging.  Herein, we describe the use of two ligation manifolds, namely, diselenide-selenoester ligation and native chem. ligation, to assemble a 31.5 kDa phosphorylated insulin-like growth factor binding protein 2 (IGFBP-2) that comprises 290 amino acid residues, a phosphoserine post-translational modification, and nine disulfide bonds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBHep9DkCBp7Vg90H21EOLACvtfcHk0ljkAum8TFIHFw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXnsl2ktLo%253D&amp;md5=ce3979b3357a66444c5f967890da9566</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjacs.1c02280&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.1c02280%26sid%3Dliteratum%253Aachs%26aulast%3DPremdjee%26aufirst%3DB.%26aulast%3DAndersen%26aufirst%3DA.%2BS.%26aulast%3DLarance%26aufirst%3DM.%26aulast%3DConde-Frieboes%26aufirst%3DK.%2BW.%26aulast%3DPayne%26aufirst%3DR.%2BJ.%26atitle%3DChemical%2520Synthesis%2520of%2520Phosphorylated%2520Insulin-like%2520Growth%2520Factor%2520Binding%2520Protein%25202%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2021%26volume%3D143%26issue%3D14%26spage%3D5336%26epage%3D5342%26doi%3D10.1021%2Fjacs.1c02280" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agouridas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Mahdi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diemer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cargoët, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monbaliu, J. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, O.</span></span> <span> </span><span class="NLM_article-title">Native Chemical Ligation and Extended Methods: Mechanisms, Catalysis, Scope, and Limitations</span>. <i>Chem. Rev.</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">7328</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.chemrev.8b00712</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00712" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1MXoslamtr0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=119&amp;publication_year=2019&amp;pages=7328&amp;issue=12&amp;author=V.+Agouridasauthor=O.+El+Mahdiauthor=V.+Diemerauthor=M.+Cargo%C3%ABtauthor=J.+C.+M.+Monbaliuauthor=O.+Melnyk&amp;title=Native+Chemical+Ligation+and+Extended+Methods%3A+Mechanisms%2C+Catalysis%2C+Scope%2C+and+Limitations&amp;doi=10.1021%2Facs.chemrev.8b00712" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Native Chemical Ligation and Extended Methods: Mechanisms, Catalysis, Scope, and Limitations</span></div><div class="casAuthors">Agouridas, Vangelis; El Mahdi, Ouafaa; Diemer, Vincent; Cargoet, Marine; Monbaliu, Jean-Christophe M.; Melnyk, Oleg</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">7328-7443</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The native chem. ligation reaction (NCL) involves reacting a C-terminal peptide thioester with an N-terminal cysteinyl peptide to produce a native peptide bond between the two fragments.  This reaction has considerably extended the size of polypeptides and proteins that can be produced by total synthesis and has also numerous applications in bioconjugation, polymer synthesis, material science, and micro- and nanotechnol. research.  The aim of the present review is to provide a thorough mechanistic overview of NCL and extended methods.  The most relevant properties of peptide thioesters, Cys peptides, and common solvents, reagents, additives, and catalysts used for these ligations are presented.  Mechanisms, selectivity and reactivity are, whenever possible, discussed through the insights of computational and phys. chem. studies.  The inherent limitations of NCL are discussed with insights from the mechanistic standpoint.  This review also presents a palette of O,S-, N,S-, or N,Se-acyl shift systems as thioester or selenoester surrogates and discusses the special mol. features that govern reactivity in each case.  Finally, the various thiol-based auxiliaries and thiol or selenol amino acid surrogates that have been developed so far are discussed with a special focus on the mechanism of long-range N,S-acyl migrations and selective dechalcogenation reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtWR5R5VOQybVg90H21EOLACvtfcHk0ljkAum8TFIHFw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXoslamtr0%253D&amp;md5=d608ef5f6d39e2b646e36224c35008f5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00712&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00712%26sid%3Dliteratum%253Aachs%26aulast%3DAgouridas%26aufirst%3DV.%26aulast%3DEl%2BMahdi%26aufirst%3DO.%26aulast%3DDiemer%26aufirst%3DV.%26aulast%3DCargo%25C3%25ABt%26aufirst%3DM.%26aulast%3DMonbaliu%26aufirst%3DJ.%2BC.%2BM.%26aulast%3DMelnyk%26aufirst%3DO.%26atitle%3DNative%2520Chemical%2520Ligation%2520and%2520Extended%2520Methods%253A%2520Mechanisms%252C%2520Catalysis%252C%2520Scope%252C%2520and%2520Limitations%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26issue%3D12%26spage%3D7328%26doi%3D10.1021%2Facs.chemrev.8b00712" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conibear, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, C. F. W.</span></span> <span> </span><span class="NLM_article-title">Native Chemical Ligation in Protein Synthesis and Semi-Synthesis</span>. <i>Chem. Soc. Rev.</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">9046</span>– <span class="NLM_lpage">9068</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C8CS00573G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1039%2FC8CS00573G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30418441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKmtrvP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=2018&amp;pages=9046-9068&amp;issue=24&amp;author=A.+C.+Conibearauthor=E.+E.+Watsonauthor=R.+J.+Payneauthor=C.+F.+W.+Becker&amp;title=Native+Chemical+Ligation+in+Protein+Synthesis+and+Semi-Synthesis&amp;doi=10.1039%2FC8CS00573G" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Native chemical ligation in protein synthesis and semi-synthesis</span></div><div class="casAuthors">Conibear, Anne C.; Watson, Emma E.; Payne, Richard J.; Becker, Christian F. W.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9046-9068</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Native chem. ligation (NCL) provides a highly efficient and robust means to chemoselectively link unprotected peptide and protein segments to generate proteins.  The ability to incorporate non-proteinogenic amino acids (e.g.D-amino acids or fluorescent labels) and post-translational modifications into proteins by stitching together peptide fragments has driven extremely important developments in peptide and protein science over the past 20 years.  Extensions of the original NCL concept (including the development of thiol- and selenol-derived amino acids and desulfurisation and deselenisation methods), improved access to peptide thioesters, and the use of the methodol. in combination with recombinantly expressed polypeptide fragments (termed Expressed Protein Ligation, EPL) have helped to further expand the utility of the methodol.  Over the past five years, there has been a dramatic increase in the no. of proteins that have been accessed by total chem. synthesis and semi-synthesis, including a large range of modified proteins; new records have also been set with regards to the size of proteins that can now be accessed via ligation chem.  Together these efforts have not only contributed to a better understanding of protein structure and function, but have also driven innovations in protein science.  In this tutorial review, we aim to provide the reader with the latest developments in NCL- and EPL-based ligation technologies as well as illustrated examples of using these methods, together with synthetic logic, to access proteins and modified proteins for biol. study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0dO1SAMx10LVg90H21EOLACvtfcHk0lhveF7myYJv7A"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXitFKmtrvP&amp;md5=148c164b9840ed1b1782a657f5cf9504</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1039%2FC8CS00573G&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8CS00573G%26sid%3Dliteratum%253Aachs%26aulast%3DConibear%26aufirst%3DA.%2BC.%26aulast%3DWatson%26aufirst%3DE.%2BE.%26aulast%3DPayne%26aufirst%3DR.%2BJ.%26aulast%3DBecker%26aufirst%3DC.%2BF.%2BW.%26atitle%3DNative%2520Chemical%2520Ligation%2520in%2520Protein%2520Synthesis%2520and%2520Semi-Synthesis%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2018%26volume%3D47%26issue%3D24%26spage%3D9046%26epage%3D9068%26doi%3D10.1039%2FC8CS00573G" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondalapati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jbara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brik, A.</span></span> <span> </span><span class="NLM_article-title">Expanding the Chemical Toolbox for the Synthesis of Large and Uniquely Modified Proteins</span>. <i>Nat. Chem.</i>  <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">418</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.2476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fnchem.2476" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=27102674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC28XmslShsL8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2016&amp;pages=407-418&amp;issue=5&amp;author=S.+Bondalapatiauthor=M.+Jbaraauthor=A.+Brik&amp;title=Expanding+the+Chemical+Toolbox+for+the+Synthesis+of+Large+and+Uniquely+Modified+Proteins&amp;doi=10.1038%2Fnchem.2476" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding the chemical toolbox for the synthesis of large and uniquely modified proteins</span></div><div class="casAuthors">Bondalapati, Somasekhar; Jbara, Muhammad; Brik, Ashraf</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">407-418</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Methods to prep. proteins that include a specific modification at a desired position are essential for understanding their cellular functions and phys. properties in living systems.  Chem. protein synthesis, which relies on the chemoselective ligation of unprotected peptides, enables the prepn. of modified proteins that are not easily fabricated by other methods.  In contrast to recombinant approaches, chem. synthesis can be used to prep. protein analogs such as D-proteins, which are useful in protein structure detn. and the discovery of novel therapeutics.  Post-translationally modifying proteins is another example where chem. protein synthesis proved itself as a powerful approach for prepg. samples with high homogeneity and in workable quantities.  In this Review, we discuss the basic principles of the field, focusing on novel chemoselective peptide ligation approaches such as native chem. ligation and the recent advances based on this method with a proven record of success in the synthesis of highly important protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3R5-DwrpbHLVg90H21EOLACvtfcHk0lhveF7myYJv7A"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XmslShsL8%253D&amp;md5=983bef06802d74a198853f6f2d130626</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2476&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2476%26sid%3Dliteratum%253Aachs%26aulast%3DBondalapati%26aufirst%3DS.%26aulast%3DJbara%26aufirst%3DM.%26aulast%3DBrik%26aufirst%3DA.%26atitle%3DExpanding%2520the%2520Chemical%2520Toolbox%2520for%2520the%2520Synthesis%2520of%2520Large%2520and%2520Uniquely%2520Modified%2520Proteins%26jtitle%3DNat.%2520Chem.%26date%3D2016%26volume%3D8%26issue%3D5%26spage%3D407%26epage%3D418%26doi%3D10.1038%2Fnchem.2476" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bertolini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenzl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kats, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wruck, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auburger, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tans, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukau, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, G.</span></span> <span> </span><span class="NLM_article-title">Interactions between Nascent Proteins Translated by Adjacent Ribosomes Drive Homomer Assembly</span>. <i>Science</i>  <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>371</i></span> (<span class="NLM_issue">6524</span>),  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">64</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.abc7151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1126%2Fscience.abc7151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=33384371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtVChsbY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=371&amp;publication_year=2021&amp;pages=57-64&amp;issue=6524&amp;author=M.+Bertoliniauthor=K.+Fenzlauthor=I.+Katsauthor=F.+Wruckauthor=F.+Tippmannauthor=J.+Schmittauthor=J.+J.+Auburgerauthor=S.+Tansauthor=B.+Bukauauthor=G.+Kramer&amp;title=Interactions+between+Nascent+Proteins+Translated+by+Adjacent+Ribosomes+Drive+Homomer+Assembly&amp;doi=10.1126%2Fscience.abc7151" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Interactions between nascent proteins translated by adjacent ribosomes drive homomer assembly</span></div><div class="casAuthors">Bertolini, Matilde; Fenzl, Kai; Kats, Ilia; Wruck, Florian; Tippmann, Frank; Schmitt, Jaro; Auburger, Josef Johannes; Tans, Sander; Bukau, Bernd; Kramer, Guenter</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">6524</span>),
    <span class="NLM_cas:pages">57-64</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Accurate assembly of newly synthesized proteins into functional oligomers is crucial for cell activity.  In this study, we investigated whether direct interaction of two nascent proteins, emerging from nearby ribosomes (co-co assembly), constitutes a general mechanism for oligomer formation.  We used proteome-wide screening to detect nascent chain-connected ribosome pairs and identified hundreds of homomer subunits that co-co assemble in human cells.  Interactions are mediated by five major domain classes, among which N-terminal coiled coils are the most prevalent.  We were able to reconstitute co-co assembly of nuclear lamin in Escherichia coli, demonstrating that dimer formation is independent of dedicated assembly machineries.  Co-co assembly may thus represent an efficient way to limit protein aggregation risks posed by diffusion-driven assembly routes and ensure isoform-specific homomer formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGok3X9ogzJrHbVg90H21EOLACvtfcHk0lhveF7myYJv7A"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXhtVChsbY%253D&amp;md5=41388a1562a4968843a1e8e4cd93f728</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.abc7151&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abc7151%26sid%3Dliteratum%253Aachs%26aulast%3DBertolini%26aufirst%3DM.%26aulast%3DFenzl%26aufirst%3DK.%26aulast%3DKats%26aufirst%3DI.%26aulast%3DWruck%26aufirst%3DF.%26aulast%3DTippmann%26aufirst%3DF.%26aulast%3DSchmitt%26aufirst%3DJ.%26aulast%3DAuburger%26aufirst%3DJ.%2BJ.%26aulast%3DTans%26aufirst%3DS.%26aulast%3DBukau%26aufirst%3DB.%26aulast%3DKramer%26aufirst%3DG.%26atitle%3DInteractions%2520between%2520Nascent%2520Proteins%2520Translated%2520by%2520Adjacent%2520Ribosomes%2520Drive%2520Homomer%2520Assembly%26jtitle%3DScience%26date%3D2021%26volume%3D371%26issue%3D6524%26spage%3D57%26epage%3D64%26doi%3D10.1126%2Fscience.abc7151" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shiber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Döring, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedrich, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zedan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukau, B.</span></span> <span> </span><span class="NLM_article-title">Cotranslational Assembly of Protein Complexes in Eukaryotes Revealed by Ribosome Profiling</span>. <i>Nature</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>561</i></span> (<span class="NLM_issue">7722</span>),  <span class="NLM_fpage">268</span>– <span class="NLM_lpage">272</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41586-018-0462-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fs41586-018-0462-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30158700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Wmt7%252FE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=561&amp;publication_year=2018&amp;pages=268-272&amp;issue=7722&amp;author=A.+Shiberauthor=K.+D%C3%B6ringauthor=U.+Friedrichauthor=K.+Klannauthor=D.+Merkerauthor=M.+Zedanauthor=F.+Tippmannauthor=G.+Kramerauthor=B.+Bukau&amp;title=Cotranslational+Assembly+of+Protein+Complexes+in+Eukaryotes+Revealed+by+Ribosome+Profiling&amp;doi=10.1038%2Fs41586-018-0462-y" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Cotranslational assembly of protein complexes in eukaryotes revealed by ribosome profiling</span></div><div class="casAuthors">Shiber, Ayala; Doering, Kristina; Friedrich, Ulrike; Klann, Kevin; Merker, Dorina; Zedan, Mostafa; Tippmann, Frank; Kramer, Guenter; Bukau, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">561</span>
        (<span class="NLM_cas:issue">7722</span>),
    <span class="NLM_cas:pages">268-272</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The folding of newly synthesized proteins to the native state is a major challenge within the crowded cellular environment, as non-productive interactions can lead to misfolding, aggregation, and degrdn.  Cells cope with this challenge by coupling synthesis with polypeptide folding and by using mol. chaperones to safeguard folding cotranslationally.  However, although most of the cellular proteome forms oligomeric assemblies, little is known about the final step of folding: the assembly of polypeptides into complexes.  In prokaryotes, a proof-of-concept study showed that the assembly of heterodimeric luciferase is an organized cotranslational process that is facilitated by spatially confined translation of the subunits encoded on a polycistronic mRNA.  In eukaryotes, however, fundamental differences, such as the rarity of polycistronic mRNAs and different chaperone constellations, raise the question of whether assembly is also coordinated with translation.  Here, we provide a systematic and mechanistic anal. of the assembly of protein complexes in eukaryotes using ribosome profiling.  We detd. the in vivo interactions of the nascent subunits from 12 hetero-oligomeric protein complexes of Saccharomyces cerevisiae at near-residue resoln.  We found that 9 complexes assembled cotranslationally; the 3 complexes that did not show cotranslational interactions were regulated by dedicated assembly chaperones5-7.  Cotranslational assembly often occurs uni-directionally, with one fully synthesized subunit engaging its nascent partner subunit, thereby counteracting its propensity for aggregation.  The onset of cotranslational subunit assocn. coincided directly with the full exposure of the nascent interaction domain at the ribosomal tunnel exit.  The action of ribosome-assocd. Hsp70 chaperone Ssb was coordinated with assembly.  Ssb transiently engaged partially synthesized interaction domains and then dissocd. before the onset of partner subunit assocn., presumably to prevent premature assembly interactions.  This study showed that cotranslational subunit assocn. is a prevalent mechanism for the assembly of hetero-oligomers in yeast and indicated that translation, folding and the assembly of protein complexes are integrated processes in eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwrhPj7fRwLrVg90H21EOLACvtfcHk0lhOi6O04R_IjA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Wmt7%252FE&amp;md5=3002026264d124d8921feb93bcc420cc</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0462-y&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0462-y%26sid%3Dliteratum%253Aachs%26aulast%3DShiber%26aufirst%3DA.%26aulast%3DD%25C3%25B6ring%26aufirst%3DK.%26aulast%3DFriedrich%26aufirst%3DU.%26aulast%3DKlann%26aufirst%3DK.%26aulast%3DMerker%26aufirst%3DD.%26aulast%3DZedan%26aufirst%3DM.%26aulast%3DTippmann%26aufirst%3DF.%26aulast%3DKramer%26aufirst%3DG.%26aulast%3DBukau%26aufirst%3DB.%26atitle%3DCotranslational%2520Assembly%2520of%2520Protein%2520Complexes%2520in%2520Eukaryotes%2520Revealed%2520by%2520Ribosome%2520Profiling%26jtitle%3DNature%26date%3D2018%26volume%3D561%26issue%3D7722%26spage%3D268%26epage%3D272%26doi%3D10.1038%2Fs41586-018-0462-y" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penn, L. Z.</span></span> <span> </span><span class="NLM_article-title">Reflecting on 25 Years with MYC</span>. <i>Nat. Rev. Cancer</i>  <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">976</span>– <span class="NLM_lpage">990</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nrc2231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fnrc2231" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=19029958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWhu7rJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2008&amp;pages=976-990&amp;issue=12&amp;author=N.+Meyerauthor=L.+Z.+Penn&amp;title=Reflecting+on+25+Years+with+MYC&amp;doi=10.1038%2Fnrc2231" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Reflecting on 25 years with MYC</span></div><div class="casAuthors">Meyer, Natalie; Penn, Linda Z.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">976-990</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Just over 25 years ago, MYC, the human homolog of a retroviral oncogene, was identified.  Since that time, MYC research has been intense and the advances impressive.  On reflection, it is astonishing how each incremental insight into MYC regulation and function has also had an impact on numerous biol. disciplines, including our understanding of mol. oncogenesis in general.  Here we chronicle the major advances in our understanding of MYC biol., and peer into the future of MYC research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR-DcM7mIL4bVg90H21EOLACvtfcHk0lhOi6O04R_IjA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWhu7rJ&amp;md5=93089742c2c5e30c1075d62fdd416835</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrc2231&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2231%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DN.%26aulast%3DPenn%26aufirst%3DL.%2BZ.%26atitle%3DReflecting%2520on%252025%2520Years%2520with%2520MYC%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2008%26volume%3D8%26issue%3D12%26spage%3D976%26epage%3D990%26doi%3D10.1038%2Fnrc2231" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kretzner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weintraub, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwood, E.</span></span> <span> </span><span class="NLM_article-title">Sequence-Specific DNA Binding by the c-Myc Protein</span>. <i>Science</i>  <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>250</i></span> (<span class="NLM_issue">4984</span>),  <span class="NLM_fpage">1149</span>– <span class="NLM_lpage">1151</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.2251503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1126%2Fscience.2251503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=2251503" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADyaK3MXlslOhsQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=250&amp;publication_year=1990&amp;pages=1149-1151&amp;issue=4984&amp;author=T.+Blackwellauthor=L.+Kretznerauthor=R.+Eisenmanauthor=H.+Weintraubauthor=E.+Blackwood&amp;title=Sequence-Specific+DNA+Binding+by+the+c-Myc+Protein&amp;doi=10.1126%2Fscience.2251503" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence-specific DNA binding by the c-Myc protein</span></div><div class="casAuthors">Blackwell, T. Keith; Kretzner, Leo; Blackwood, Elizabeth M.; Eisenman, Robert N.; Weintraub, Harold</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">250</span>
        (<span class="NLM_cas:issue">4984</span>),
    <span class="NLM_cas:pages">1149-51</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">While it has been known for some time that the c-Myc protein binds to random DNA sequences, no sequence-specific binding activity has been detected.  At its carboxyl terminus, c-Myc contains a basic-helix-loop-helix (bHLH) motif, which is important for dimerization and specific DNA binding, as demonstrated for other bHLH protein family members.  Of those studied, most bHLH bind to sites that contain a CA--TG consensus.  In this study, the technique of selected and amplified binding-sequence imprinting was used to identify a DNA sequence that was recognized by c-Myc.  A purified C-terminal fragment of human c-Myc that contained the bHLH domain bound in vitro in a sequence-specific manner to the sequence, CACGTG.  These results suggest that some of the biol. functions of Myc family proteins are accomplished by sequence-specific DNA binding that is mediated by the C-terminal region of the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKpAA5X4GPyrVg90H21EOLACvtfcHk0likKPqBO_b0OA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3MXlslOhsQ%253D%253D&amp;md5=85b4b62b8d672fdfeb57cb70ad87aca7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.2251503&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2251503%26sid%3Dliteratum%253Aachs%26aulast%3DBlackwell%26aufirst%3DT.%26aulast%3DKretzner%26aufirst%3DL.%26aulast%3DEisenman%26aufirst%3DR.%26aulast%3DWeintraub%26aufirst%3DH.%26aulast%3DBlackwood%26aufirst%3DE.%26atitle%3DSequence-Specific%2520DNA%2520Binding%2520by%2520the%2520c-Myc%2520Protein%26jtitle%3DScience%26date%3D1990%26volume%3D250%26issue%3D4984%26spage%3D1149%26epage%3D1151%26doi%3D10.1126%2Fscience.2251503" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackwood, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenman, R. N.</span></span> <span> </span><span class="NLM_article-title">Max : A Helix-Loop-Helix Zipper Protein That Complex with Myc</span>. <i>Science</i>  <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>251</i></span>,  <span class="NLM_fpage">1211</span>– <span class="NLM_lpage">1217</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.2006410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1126%2Fscience.2006410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=2006410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADyaK3MXltlShs7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=251&amp;publication_year=1991&amp;pages=1211-1217&amp;author=E.+M.+Blackwoodauthor=R.+N.+Eisenman&amp;title=Max%E2%80%AF%3A+A+Helix-Loop-Helix+Zipper+Protein+That+Complex+with+Myc&amp;doi=10.1126%2Fscience.2006410" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Max:  a helix-loop-helix zipper protein that forms a sequence-specific DNA-binding complex with Myc</span></div><div class="casAuthors">Blackwood, Elizabeth M.; Eisenman, Robert N.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">251</span>
        (<span class="NLM_cas:issue">4998</span>),
    <span class="NLM_cas:pages">1211-17</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The myc protooncogene family has been implicated in cell proliferation, differentiation, and neoplasia, but its mechanism of function at the mol. level is unknown.  The carboxyl terminus of Myc family proteins contains a basic region helix-loop-helix leucine zipper motif (bHLH-Zip), which has DNA-binding activity and has been predicted to mediate protein-protein interactions.  The bHLH-Zip region of c-Myc was used to screen a cDNA expression library, and a bHLH-Zip protein, termed Max, was identified.  Max specifically assocd. with c-Myc, N-Myc, and L-Myc proteins, but not with a no. of other bHLH, bZip, or bHLH-Zip proteins.  The interaction between Max and c-Myc was dependent on the integrity of the c-Myc HLH-Zip domain, but not on the basic region or other sequences outside the domain.  Furthermore, the Myc-Max complex bound to DNA in a sequence-specific manner under conditions where neither Max nor Myc exhibited appreciable binding.  The DNA-binding activity of the complex was dependent on both the dimerization domain and the basic region of c-Myc.  These results suggests that Myc family proteins undergo a restricted set of interactions in the cell and may belong to the more general class of eukaryotic DNA-binding transcription factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqNOoJSe0N4rVg90H21EOLACvtfcHk0likKPqBO_b0OA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3MXltlShs7Y%253D&amp;md5=e8b4df3b8d310027c50591a8610e9eba</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1126%2Fscience.2006410&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2006410%26sid%3Dliteratum%253Aachs%26aulast%3DBlackwood%26aufirst%3DE.%2BM.%26aulast%3DEisenman%26aufirst%3DR.%2BN.%26atitle%3DMax%25E2%2580%25AF%253A%2520A%2520Helix-Loop-Helix%2520Zipper%2520Protein%2520That%2520Complex%2520with%2520Myc%26jtitle%3DScience%26date%3D1991%26volume%3D251%26spage%3D1211%26epage%3D1217%26doi%3D10.1126%2Fscience.2006410" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferré-D’Amaré, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prendergast, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziff, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span> <span> </span><span class="NLM_article-title">Recognition by Max of Its Cognate DNA through a Dimeric b/HLH/Z Domain</span>. <i>Nature</i>  <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>363</i></span> (<span class="NLM_issue">6424</span>),  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">45</span>, <span class="refDoi">&nbsp;DOI: 10.1038/363038a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2F363038a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=8479534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADyaK3sXkvVyhsrw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=363&amp;publication_year=1993&amp;pages=38-45&amp;issue=6424&amp;author=A.+R.+Ferr%C3%A9-D%E2%80%99Amar%C3%A9author=G.+C.+Prendergastauthor=E.+B.+Ziffauthor=S.+K.+Burley&amp;title=Recognition+by+Max+of+Its+Cognate+DNA+through+a+Dimeric+b%2FHLH%2FZ+Domain&amp;doi=10.1038%2F363038a0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition by Max of its cognate DNA through a dimeric b/HLH/Z domain</span></div><div class="casAuthors">Ferre-D'Amare, Adrian R.; Prendergast, George C.; Ziff, Edward B.; Burley, Stephen K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">363</span>
        (<span class="NLM_cas:issue">6424</span>),
    <span class="NLM_cas:pages">38-45</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The three-dimensional structure of the basic/helix-loop-helix/leucine zipper domain of the transcription factor Max complexed with DNA has been detd. by X-ray crystallog. at 2.9 Å resoln.  Max binds as a dimer to its recognition sequence CACGTG by direct contacts between the α-helical basic region and the major groove.  This sym. homodimer, a new protein fold, is a parallel, left-handed, four-helix bundle, with each monomer contg. two α-helical segments sepd. by a loop.  The two α-helical segments are composed of the basic region plus helix 1 and helix 2 plus the leucine repeat, resp.  As in GCN4, the leucine repeat forms a parallel coiled coil.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuHiV_lNNN5LVg90H21EOLACvtfcHk0likKPqBO_b0OA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3sXkvVyhsrw%253D&amp;md5=4ebca2f013cae17c9b3440756ffbcf45</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2F363038a0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F363038a0%26sid%3Dliteratum%253Aachs%26aulast%3DFerr%25C3%25A9-D%25E2%2580%2599Amar%25C3%25A9%26aufirst%3DA.%2BR.%26aulast%3DPrendergast%26aufirst%3DG.%2BC.%26aulast%3DZiff%26aufirst%3DE.%2BB.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26atitle%3DRecognition%2520by%2520Max%2520of%2520Its%2520Cognate%2520DNA%2520through%2520a%2520Dimeric%2520b%252FHLH%252FZ%2520Domain%26jtitle%3DNature%26date%3D1993%26volume%3D363%26issue%3D6424%26spage%3D38%26epage%3D45%26doi%3D10.1038%2F363038a0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qing, G.</span></span> <span> </span><span class="NLM_article-title">Targeting Oncogenic Myc as a Strategy for Cancer Treatment</span>. <i>Signal Transduct. Target. Ther.</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">7</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41392-018-0008-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fs41392-018-0008-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=29527327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A280%3ADC%252BC1MrpvFCltw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2018&amp;pages=1-7&amp;issue=1&amp;author=H.+Chenauthor=H.+Liuauthor=G.+Qing&amp;title=Targeting+Oncogenic+Myc+as+a+Strategy+for+Cancer+Treatment&amp;doi=10.1038%2Fs41392-018-0008-7" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">DNA methylation profile is associated with the osteogenic potential of three distinct human odontogenic stem cells</span></div><div class="casAuthors">Ai Tingting; Zhang Jieni; Wang Xuedong; Zheng Xiaowen; Qin Xueyan; Li Weiran; Hu Wei; Lin Jiuxiang; Ai Tingting; Zhang Qian; Chen Feng</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Among the various sources of human autologous stem cells, stem cells isolated from dental tissues exhibit excellent properties in tissue engineering and regenerative medicine.  However, the distinct potential of these odontogenic cell lines remains unclear.  In this study, we analyzed DNA methylation patterns to determine whether specific differences existed among three different odontogenic cell types.  Using the HumanMethylation450 Beadchip, the whole genomes of human dental pulp stem cells (DPSCs), periodontal ligament stem cells (PDLSCs), and dental follicle progenitor cells (DFPCs) were compared.  Then, the osteogenic potential of these cells was evaluated both in vitro and in vivo, and the methylation levels of certain genes related to bone formation differed among the three cell lines.  P values less than 0.05 were considered to indicate statistical significance.  The three cell types showed highly similar DNA methylation patterns, although specific differences were identified.  Gene ontology analysis revealed that one of the most significantly different gene categories was related to bone formation.  Thus, expression of cell surface epitopes and osteogenic-related transcription factors as well as the bone formation capacity were compared.  The results showed that compared with DFPCs and DPSCs, PDLSCs had higher transcription levels of osteogenic-related factors, a higher in vitro osteogenic potential, and an increased new bone formation capacity in vivo.  In conclusion, the results of this study suggested that the differential DNA methylation profiles could be related to the osteogenic potential of these human odontogenic cell populations.  Additionally, the increased osteogenic potential of PDLSCs might aid researchers or clinicians in making better choices regarding tissue regeneration and clinical therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSr2v8N4kAiY70AGQpJZWpJfW6udTcc2eZKmV_S2GlZjrntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC1MrpvFCltw%253D%253D&amp;md5=7820bce50f4c78b0166542a690152a88</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41392-018-0008-7&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-018-0008-7%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DQing%26aufirst%3DG.%26atitle%3DTargeting%2520Oncogenic%2520Myc%2520as%2520a%2520Strategy%2520for%2520Cancer%2520Treatment%26jtitle%3DSignal%2520Transduct.%2520Target.%2520Ther.%26date%3D2018%26volume%3D3%26issue%3D1%26spage%3D1%26epage%3D7%26doi%3D10.1038%2Fs41392-018-0008-7" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalkat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Melo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lourenco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resetca, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamachi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penn, L. Z.</span></span> <span> </span><span class="NLM_article-title">MYC Deregulation in Primary Human Cancers</span>. <i>Genes</i>  <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">151</span>, <span class="refDoi">&nbsp;DOI: 10.3390/genes8060151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.3390%2Fgenes8060151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpslCmu7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2017&amp;pages=151&amp;issue=6&amp;author=M.+Kalkatauthor=J.+De+Meloauthor=K.+A.+Hickmanauthor=C.+Lourencoauthor=C.+Redelauthor=D.+Resetcaauthor=A.+Tamachiauthor=W.+B.+Tuauthor=L.+Z.+Penn&amp;title=MYC+Deregulation+in+Primary+Human+Cancers&amp;doi=10.3390%2Fgenes8060151" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">MYC deregulation in primary human cancers</span></div><div class="casAuthors">Kalkat, Manpreet; De Melo, Jason; Hickman, Katherine Ashley; Lourenco, Corey; Redel, Cornelia; Resetca, Diana; Tamachi, Aaliya; Tu, William B.; Penn, Linda Z.</div><div class="citationInfo"><span class="NLM_cas:title">Genes</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">151/1-151/30</span>CODEN:
                <span class="NLM_cas:coden">GENEG9</span>;
        ISSN:<span class="NLM_cas:issn">2073-4425</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">MYC regulates a complex biol. program by transcriptionally activating and repressing its numerous target genes.  As such, MYC is a master regulator of many processes, including cell cycle entry, ribosome biogenesis, and metab.  In cancer, the activity of the MYC transcriptional network is frequently deregulated, contributing to the initiation and maintenance of disease.  Deregulation often leads to constitutive overexpression of MYC, which can be achieved through gross genetic abnormalities, including copy no. alterations, chromosomal translocations, increased enhancer activity, or through aberrant signal transduction leading to increased MYC transcription or increased MYC mRNA and protein stability.  Herein, we summarize the frequency and modes of MYC deregulation and describe both well-established and more recent findings in a variety of cancer types.  Notably, these studies have highlighted that with an increased appreciation for the basic mechanisms deregulating MYC in cancer, new therapeutic vulnerabilities can be discovered and potentially exploited for the inhibition of this potent oncogene in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFN8ZCa1PFjbVg90H21EOLACvtfcHk0liPy8eSF9LKnQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXpslCmu7g%253D&amp;md5=5060fc3d66f75e698f9c28bcc9947475</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.3390%2Fgenes8060151&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fgenes8060151%26sid%3Dliteratum%253Aachs%26aulast%3DKalkat%26aufirst%3DM.%26aulast%3DDe%2BMelo%26aufirst%3DJ.%26aulast%3DHickman%26aufirst%3DK.%2BA.%26aulast%3DLourenco%26aufirst%3DC.%26aulast%3DRedel%26aufirst%3DC.%26aulast%3DResetca%26aufirst%3DD.%26aulast%3DTamachi%26aufirst%3DA.%26aulast%3DTu%26aufirst%3DW.%2BB.%26aulast%3DPenn%26aufirst%3DL.%2BZ.%26atitle%3DMYC%2520Deregulation%2520in%2520Primary%2520Human%2520Cancers%26jtitle%3DGenes%26date%3D2017%26volume%3D8%26issue%3D6%26spage%3D151%26doi%3D10.3390%2Fgenes8060151" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burge, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Amplification in Tumor Cells with Elevated C-Myc</span>. <i>Cell</i>  <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">67</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2012.08.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.cell.2012.08.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=23021215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsValsLjO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=151&amp;publication_year=2012&amp;pages=56-67&amp;author=C.+Y.+Linauthor=J.+Lovenauthor=P.+B.+Rahlauthor=R.+M.+Paranalauthor=C.+B.+Burgeauthor=J.+E.+Bradnerauthor=T.+I.+Leeauthor=R.+A.+Young&amp;title=Transcriptional+Amplification+in+Tumor+Cells+with+Elevated+C-Myc&amp;doi=10.1016%2Fj.cell.2012.08.026" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional amplification in tumor cells with elevated c-Myc</span></div><div class="casAuthors">Lin, Charles Y.; Loven, Jakob; Rahl, Peter B.; Paranal, Ronald M.; Burge, Christopher B.; Bradner, James E.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-67</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Elevated expression of the c-Myc transcription factor occurs frequently in human cancers and is assocd. with tumor aggression and poor clin. outcome.  The effect of high levels of c-Myc on global gene regulation is poorly understood but is widely thought to involve newly activated or repressed "Myc target genes." We report here that in tumor cells expressing high levels of c-Myc the transcription factor accumulates in the promoter regions of active genes and causes transcriptional amplification, producing increased levels of transcripts within the cell's gene expression program.  Thus, rather than binding and regulating a new set of genes, c-Myc amplifies the output of the existing gene expression program.  These results provide an explanation for the diverse effects of oncogenic c-Myc on gene expression in different tumor cells and suggest that transcriptional amplification reduces rate-limiting constraints for tumor cell growth and proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxyaUKvzZJsLVg90H21EOLACvtfcHk0liPy8eSF9LKnQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhsValsLjO&amp;md5=9e8be9f929931b0840fca955b45595a1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.08.026&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.08.026%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLoven%26aufirst%3DJ.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DBurge%26aufirst%3DC.%2BB.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DTranscriptional%2520Amplification%2520in%2520Tumor%2520Cells%2520with%2520Elevated%2520C-Myc%26jtitle%3DCell%26date%3D2012%26volume%3D151%26spage%3D56%26epage%3D67%26doi%3D10.1016%2Fj.cell.2012.08.026" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Regulation and Its Misregulation in Disease</span>. <i>Cell</i>  <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>152</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1237</span>– <span class="NLM_lpage">1251</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2013.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.cell.2013.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=23498934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFWiuro%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=152&amp;publication_year=2013&amp;pages=1237-1251&amp;issue=6&amp;author=T.+I.+Leeauthor=R.+A.+Young&amp;title=Transcriptional+Regulation+and+Its+Misregulation+in+Disease&amp;doi=10.1016%2Fj.cell.2013.02.014" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional regulation and its misregulation in disease</span></div><div class="casAuthors">Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1237-1251</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The gene expression programs that establish and maintain specific cell states in humans are controlled by thousands of transcription factors, cofactors, and chromatin regulators.  Misregulation of these gene expression programs can cause a broad range of diseases.  Here, we review recent advances in our understanding of transcriptional regulation and discuss how these have provided new insights into transcriptional misregulation in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRZI_3gsWJi7Vg90H21EOLACvtfcHk0ljOktnYv_x_ew"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXktFWiuro%253D&amp;md5=2eabd761a3721c8c509f0fa1a00ceab6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.02.014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DTranscriptional%2520Regulation%2520and%2520Its%2520Misregulation%2520in%2520Disease%26jtitle%3DCell%26date%3D2013%26volume%3D152%26issue%3D6%26spage%3D1237%26epage%3D1251%26doi%3D10.1016%2Fj.cell.2013.02.014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prochownik, E. V.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Inhibitors of the Myc Oncoprotein</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i>  <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1849</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">543</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bbagrm.2014.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.bbagrm.2014.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFGnsL0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1849&amp;publication_year=2015&amp;pages=525-543&amp;issue=5&amp;author=S.+Fletcherauthor=E.+V.+Prochownik&amp;title=Small-Molecule+Inhibitors+of+the+Myc+Oncoprotein&amp;doi=10.1016%2Fj.bbagrm.2014.03.005" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of the Myc oncoprotein</span></div><div class="casAuthors">Fletcher, Steven; Prochownik, Edward V.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1849</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">525-543</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The c-Myc (Myc) oncoprotein is among the most attractive of cancer targets given that it is de-regulated in the majority of tumors and that its inhibition profoundly affects their growth and/or survival.  However, its role as a seldom-mutated transcription factor, its lack of enzymic activity for which suitable pharmaceutical inhibitors could be crafted and its expression by normal cells have largely been responsible for its being viewed as "undruggable".  Work over the past several years, however, has begun to reverse this idea by allowing us to view Myc within the larger context of global gene regulatory control.  Thus, Myc and its obligate heterodimeric partner, Max, are integral to the coordinated recruitment and post-translational modification of components of the core transcriptional machinery.  Moreover, Myc over-expression re-programs numerous crit. cellular functions and alters the cell's susceptibility to their inhibition.  This new knowledge has therefore served as a framework upon which to develop new pharmaceutical approaches.  These include the continuing development of small mols. which act directly to inhibit the crit. Myc-Max interaction, those which act indirectly to prevent Myc-directed post-translational modifications necessary to initiate productive transcription and those which inhibit vital pathways upon which the Myc-transformed cell is particularly reliant.  This article is part of a Special Issue entitled: Myc proteins in cell biol. and pathol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpwveGWUG8ZLVg90H21EOLACvtfcHk0ljOktnYv_x_ew"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXntFGnsL0%253D&amp;md5=5f6a99316dc48b9f55796e2f7514cef4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.03.005&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DFletcher%26aufirst%3DS.%26aulast%3DProchownik%26aufirst%3DE.%2BV.%26atitle%3DSmall-Molecule%2520Inhibitors%2520of%2520the%2520Myc%2520Oncoprotein%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2015%26volume%3D1849%26issue%3D5%26spage%3D525%26epage%3D543%26doi%3D10.1016%2Fj.bbagrm.2014.03.005" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boike, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cioffi, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majewski, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Co, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henning, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenna, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Discovery of a Functional Covalent Ligand Targeting an Intrinsically Disordered Cysteine within MYC</span>. <i>Cell Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">10</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.chembiol.2020.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.chembiol.2020.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=33482085" title="">PubMed</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=28&amp;publication_year=2021&amp;pages=1-10&amp;author=L.+Boikeauthor=A.+G.+Cioffiauthor=F.+C.+Majewskiauthor=J.+Coauthor=N.+J.+Henningauthor=M.+D.+Jonesauthor=G.+Liuauthor=J.+M.+McKennaauthor=J.+A.+Tallaricoauthor=M.+Schirle&amp;title=Discovery+of+a+Functional+Covalent+Ligand+Targeting+an+Intrinsically+Disordered+Cysteine+within+MYC&amp;doi=10.1016%2Fj.chembiol.2020.09.001" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.09.001&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DBoike%26aufirst%3DL.%26aulast%3DCioffi%26aufirst%3DA.%2BG.%26aulast%3DMajewski%26aufirst%3DF.%2BC.%26aulast%3DCo%26aufirst%3DJ.%26aulast%3DHenning%26aufirst%3DN.%2BJ.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DMcKenna%26aufirst%3DJ.%2BM.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DSchirle%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520Functional%2520Covalent%2520Ligand%2520Targeting%2520an%2520Intrinsically%2520Disordered%2520Cysteine%2520within%2520MYC%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2021%26volume%3D28%26spage%3D1%26epage%3D10%26doi%3D10.1016%2Fj.chembiol.2020.09.001" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truica, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izquierdo-Ferrer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anker, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lysy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalmers, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unno, K.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Small-Molecule MYC Inhibitors Suppress Tumor Growth and Enhance Immunotherapy</span>. <i>Cancer Cell</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">483</span>– <span class="NLM_lpage">497.e15</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.ccell.2019.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.ccell.2019.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=31679823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2qurrP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=2019&amp;pages=483-497.e15&amp;issue=5&amp;author=H.+Hanauthor=A.+D.+Jainauthor=M.+I.+Truicaauthor=J.+Izquierdo-Ferrerauthor=J.+F.+Ankerauthor=B.+Lysyauthor=V.+Sagarauthor=Y.+Luanauthor=Z.+R.+Chalmersauthor=K.+Unno&amp;title=Small-Molecule+MYC+Inhibitors+Suppress+Tumor+Growth+and+Enhance+Immunotherapy&amp;doi=10.1016%2Fj.ccell.2019.10.001" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule MYC Inhibitors Suppress Tumor Growth and Enhance Immunotherapy</span></div><div class="casAuthors">Han, Huiying; Jain, Atul D.; Truica, Mihai I.; Izquierdo-Ferrer, Javier; Anker, Jonathan F.; Lysy, Barbara; Sagar, Vinay; Luan, Yi; Chalmers, Zachary R.; Unno, Kenji; Mok, Hanlin; Vatapalli, Rajita; Yoo, Young A.; Rodriguez, Yara; Kandela, Irawati; Parker, J. Brandon; Chakravarti, Debabrata; Mishra, Rama K.; Schiltz, Gary E.; Abdulkadir, Sarki A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">483-497.e15</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Small mols. that directly target MYC and are also well tolerated in vivo will provide invaluable chem. probes and potential anti-cancer therapeutic agents.  We developed a series of small-mol. MYC inhibitors that engage MYC inside cells, disrupt MYC/MAX dimers, and impair MYC-driven gene expression.  The compds. enhance MYC phosphorylation on threonine-58, consequently increasing proteasome-mediated MYC degrdn.  The initial lead, MYC inhibitor 361 (MYCi361), suppressed in vivo tumor growth in mice, increased tumor immune cell infiltration, upregulated PD-L1 on tumors, and sensitized tumors to anti-PD1 immunotherapy.  However, 361 demonstrated a narrow therapeutic index.  An improved analog, MYCi975 showed better tolerability.  These findings suggest the potential of small-mol. MYC inhibitors as chem. probes and possible anti-cancer therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeLMWdCetAj7Vg90H21EOLACvtfcHk0ljl0bsXIJ_MTg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXitV2qurrP&amp;md5=e341ee827774371b92c0b004c8213af1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.10.001&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DH.%26aulast%3DJain%26aufirst%3DA.%2BD.%26aulast%3DTruica%26aufirst%3DM.%2BI.%26aulast%3DIzquierdo-Ferrer%26aufirst%3DJ.%26aulast%3DAnker%26aufirst%3DJ.%2BF.%26aulast%3DLysy%26aufirst%3DB.%26aulast%3DSagar%26aufirst%3DV.%26aulast%3DLuan%26aufirst%3DY.%26aulast%3DChalmers%26aufirst%3DZ.%2BR.%26aulast%3DUnno%26aufirst%3DK.%26atitle%3DSmall-Molecule%2520MYC%2520Inhibitors%2520Suppress%2520Tumor%2520Growth%2520and%2520Enhance%2520Immunotherapy%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26issue%3D5%26spage%3D483%26epage%3D497.e15%26doi%3D10.1016%2Fj.ccell.2019.10.001" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koehler, A. N.</span></span> <span> </span><span class="NLM_article-title">A Complex Task? Direct Modulation of Transcription Factors with Small Molecules</span>. <i>Curr. Opin. Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">340</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cbpa.2010.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.cbpa.2010.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=20395165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=2010&amp;pages=331-340&amp;issue=3&amp;author=A.+N.+Koehler&amp;title=A+Complex+Task%3F+Direct+Modulation+of+Transcription+Factors+with+Small+Molecules&amp;doi=10.1016%2Fj.cbpa.2010.03.022" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A complex task? Direct modulation of transcription factors with small molecules</span></div><div class="casAuthors">Koehler, Angela N.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">331-340</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Transcription factors with aberrant activity in disease are promising yet untested targets for therapeutic development, particularly in oncol.  Directly inhibiting or activating the function of a transcription factor requires specific disruption or recruitment of protein-protein or protein-DNA interactions.  The discovery or design of small mols. that specifically modulate these interactions has thus far proven to be a significant challenge and the protein class is often perceived to be undruggable.  ' This review will summarize recent progress in the development of small-mol. probes of transcription factors and provide evidence to challenge the notion that this important protein class is chem. intractable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr83_m66fDJsrVg90H21EOLACvtfcHk0ljl0bsXIJ_MTg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL0%253D&amp;md5=a3c2dfa1aa4edc7e387b5048c87bf17e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.03.022&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DKoehler%26aufirst%3DA.%2BN.%26atitle%3DA%2520Complex%2520Task%253F%2520Direct%2520Modulation%2520of%2520Transcription%2520Factors%2520with%2520Small%2520Molecules%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26issue%3D3%26spage%3D331%26epage%3D340%26doi%3D10.1016%2Fj.cbpa.2010.03.022" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ulasov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenkranz, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobolev, A. S.</span></span> <span> </span><span class="NLM_article-title">Transcription Factors: Time to Deliver</span>. <i>J. Controlled Release</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>269</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">35</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.jconrel.2017.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.jconrel.2017.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=29113792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslyqsb7O" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=269&amp;publication_year=2018&amp;pages=24-35&amp;author=A.+V.+Ulasovauthor=A.+A.+Rosenkranzauthor=A.+S.+Sobolev&amp;title=Transcription+Factors%3A+Time+to+Deliver&amp;doi=10.1016%2Fj.jconrel.2017.11.004" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factors: Time to deliver</span></div><div class="casAuthors">Ulasov, Alexey V.; Rosenkranz, Andrey A.; Sobolev, Alexander S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Controlled Release</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24-35</span>CODEN:
                <span class="NLM_cas:coden">JCREEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-3659</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Transcription factors (TFs) are at the center of the broad regulatory network orchestrating gene expression programs that elicit different biol. responses.  For a long time, TFs have been considered as potent drug targets due to their implications in the pathogenesis of a variety of diseases.  At the same time, TFs, located at convergence points of cellular regulatory pathways, are powerful tools providing opportunities both for cell type change and for managing the state of cells.  This task formulation requires the TF modulation problem to come to the fore.  We review several ways to manage TF activity (small mols., transfection, nanocarriers, protein-based approaches), analyzing their limitations and the possibilities to overcome them.  Delivery of TFs could revolutionize the biomedical field.  Whether this forecast comes true will depend on the ability to develop convenient technologies for targeted delivery of TFs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWVPaCtQvLB7Vg90H21EOLACvtfcHk0ljl0bsXIJ_MTg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhslyqsb7O&amp;md5=eab4552471662dc9cedac533143e6015</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.jconrel.2017.11.004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jconrel.2017.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DUlasov%26aufirst%3DA.%2BV.%26aulast%3DRosenkranz%26aufirst%3DA.%2BA.%26aulast%3DSobolev%26aufirst%3DA.%2BS.%26atitle%3DTranscription%2520Factors%253A%2520Time%2520to%2520Deliver%26jtitle%3DJ.%2520Controlled%2520Release%26date%3D2018%26volume%3D269%26spage%3D24%26epage%3D35%26doi%3D10.1016%2Fj.jconrel.2017.11.004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madden, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Araujo, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerhardt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. M.</span></span> <span> </span><span class="NLM_article-title">Taking the Myc out of Cancer: Toward Therapeutic Strategies to Directly Inhibit c-Myc</span>. <i>Mol. Cancer</i>  <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">18</span>, <span class="refDoi">&nbsp;DOI: 10.1186/s12943-020-01291-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1186%2Fs12943-020-01291-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=33386068" title="">PubMed</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2021&amp;pages=1-18&amp;issue=1&amp;author=S.+K.+Maddenauthor=A.+D.+de+Araujoauthor=M.+Gerhardtauthor=D.+P.+Fairlieauthor=J.+M.+Mason&amp;title=Taking+the+Myc+out+of+Cancer%3A+Toward+Therapeutic+Strategies+to+Directly+Inhibit+c-Myc&amp;doi=10.1186%2Fs12943-020-01291-6" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1186%2Fs12943-020-01291-6&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-020-01291-6%26sid%3Dliteratum%253Aachs%26aulast%3DMadden%26aufirst%3DS.%2BK.%26aulast%3Dde%2BAraujo%26aufirst%3DA.%2BD.%26aulast%3DGerhardt%26aufirst%3DM.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BM.%26atitle%3DTaking%2520the%2520Myc%2520out%2520of%2520Cancer%253A%2520Toward%2520Therapeutic%2520Strategies%2520to%2520Directly%2520Inhibit%2520c-Myc%26jtitle%3DMol.%2520Cancer%26date%3D2021%26volume%3D20%26issue%3D1%26spage%3D1%26epage%3D18%26doi%3D10.1186%2Fs12943-020-01291-6" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Struntz, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deutzmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caballero, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wildschut, M. H. E.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Stabilization of the Max Homodimer with a Small Molecule Attenuates Myc-Driven Transcription</span>. <i>Cell Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">711</span>– <span class="NLM_lpage">723.e14</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.chembiol.2019.02.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.chembiol.2019.02.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30880155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltVSntL4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2019&amp;pages=711-723.e14&amp;issue=5&amp;author=N.+B.+Struntzauthor=A.+Chenauthor=A.+Deutzmannauthor=R.+M.+Wilsonauthor=E.+Stefanauthor=H.+L.+Evansauthor=M.+A.+Ramirezauthor=T.+Liangauthor=F.+Caballeroauthor=M.+H.+E.+Wildschut&amp;title=Stabilization+of+the+Max+Homodimer+with+a+Small+Molecule+Attenuates+Myc-Driven+Transcription&amp;doi=10.1016%2Fj.chembiol.2019.02.009" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of the Max Homodimer with a Small Molecule Attenuates Myc-Driven Transcription</span></div><div class="casAuthors">Struntz, Nicholas B.; Chen, Andrew; Deutzmann, Anja; Wilson, Robert M.; Stefan, Eric; Evans, Helen L.; Ramirez, Maricela A.; Liang, Tong; Caballero, Francisco; Wildschut, Mattheus H. E.; Neel, Dylan V.; Freeman, David B.; Pop, Marius S.; McConkey, Marie; Muller, Sandrine; Curtin, Brice H.; Tseng, Hanna; Frombach, Kristen R.; Butty, Vincent L.; Levine, Stuart S.; Feau, Clementine; Elmiligy, Sarah; Hong, Jiyoung A.; Lewis, Timothy A.; Vetere, Amedeo; Clemons, Paul A.; Malstrom, Scott E.; Ebert, Benjamin L.; Lin, Charles Y.; Felsher, Dean W.; Koehler, Angela N.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">711-723.e14</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The transcription factor Max is a basic-helix-loop-helix leucine zipper (bHLHLZ) protein that forms homodimers or interacts with other bHLHLZ proteins, including Myc and Mxd proteins.  Among this dynamic network of interactions, the Myc/Max heterodimer has crucial roles in regulating normal cellular processes, but its transcriptional activity is deregulated in a majority of human cancers.  Despite this significance, the arsenal of high-quality chem. probes to interrogate these proteins remains limited.  We used small mol. microarrays to identify compds. that bind Max in a mechanistically unbiased manner.  We discovered the asym. polycyclic lactam, KI-MS2-008, which stabilizes the Max homodimer while reducing Myc protein and Myc-regulated transcript levels.  KI-MS2-008 also decreases viable cancer cell growth in a Myc-dependent manner and suppresses tumor growth in vivo.  This approach demonstrates the feasibility of modulating Max with small mols. and supports altering Max dimerization as an alternative approach to targeting Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBFt8AthBLZLVg90H21EOLACvtfcHk0ljtLAplQxT8Vw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXltVSntL4%253D&amp;md5=d02f075ef93868ad66e0ce1329285953</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.02.009&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.02.009%26sid%3Dliteratum%253Aachs%26aulast%3DStruntz%26aufirst%3DN.%2BB.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DDeutzmann%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DR.%2BM.%26aulast%3DStefan%26aufirst%3DE.%26aulast%3DEvans%26aufirst%3DH.%2BL.%26aulast%3DRamirez%26aufirst%3DM.%2BA.%26aulast%3DLiang%26aufirst%3DT.%26aulast%3DCaballero%26aufirst%3DF.%26aulast%3DWildschut%26aufirst%3DM.%2BH.%2BE.%26atitle%3DStabilization%2520of%2520the%2520Max%2520Homodimer%2520with%2520a%2520Small%2520Molecule%2520Attenuates%2520Myc-Driven%2520Transcription%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26issue%3D5%26spage%3D711%26epage%3D723.e14%26doi%3D10.1016%2Fj.chembiol.2019.02.009" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helmer-Citterich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jucker, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesareni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasi, S.</span></span> <span> </span><span class="NLM_article-title">Design and Properties of a Myc Derivative That Efficiently Homodimerizes</span>. <i>Oncogene</i>  <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">2463</span>– <span class="NLM_lpage">2472</span>, <span class="refDoi">&nbsp;DOI: 10.1038/sj.onc.1202199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fsj.onc.1202199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=9824157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADyaK1cXnvVehtbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=1998&amp;pages=2463-2472&amp;issue=19&amp;author=L.+Soucekauthor=M.+Helmer-Citterichauthor=A.+Saccoauthor=R.+Juckerauthor=G.+Cesareniauthor=S.+Nasi&amp;title=Design+and+Properties+of+a+Myc+Derivative+That+Efficiently+Homodimerizes&amp;doi=10.1038%2Fsj.onc.1202199" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Design and properties of a Myc derivative that efficiently homodimerizes</span></div><div class="casAuthors">Soucek, Laura; Helmer-Citterich, Manuela; Sacco, Alessandra; Jucker, Richard; Cesareni, Gianni; Nasi, Sergio</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2463-2472</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">BHLH and bHLHZip are highly conserved structural domains mediating DNA binding and specific protein-protein interactions.  They are present in a family of transcription factors, acting as dimers, and their selective dimerization is utilized to switch on and off cell proliferation, differentiation or apoptosis.  Myc is a bHLHZip protein involved in growth control and cancer, which operates in a network with the structurally related proteins Max, Mad and Mnt.  It does not form homodimers, working as a heterodimer with Max; Max, instead, forms homodimers and heterodimers with Mad and Mnt.  Myc/Max dimers activate gene transcription, while Mad/Max and Mnt/Max complexes are Myc/Max antagonists and act as repressors.  Modifying the mol. recognition of dimers may provide a tool for interfering with Myc function and, in general, for directing the mol. switches operated via bHLH(Zip) proteins.  By mol. modeling and mutagenesis, we analyzed the contribution of single amino acids to the mol. recognition of Myc, creating bHLHZip domains with altered dimerization specificity.  We report that Myc recognition specificity is encoded in a short region within the leucine zipper; mutation of four amino acids generates a protein, Omomyc, that homodimerizes efficiently and can still heterodimerize with wild type Myc and Max.  Omomyc sequestered Myc in complexes with low DNA binding efficiency, preventing binding to Max and inhibiting Myc transcriptional activator function.  Consistently with these results, Omomyc produced a proliferation arrest in NIH3T3 cells.  These data demonstrate the feasibility of interfering with fundamental biol. processes, such as proliferation, by modifying the dimerization selectivity of a bHLHZip protein; this may facilitate the design of peptides of potential pharmacol. interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2ERzL-6wvGLVg90H21EOLACvtfcHk0ljtLAplQxT8Vw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1cXnvVehtbs%253D&amp;md5=cfb371923296046ca4c737c52c56cb2d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1202199&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1202199%26sid%3Dliteratum%253Aachs%26aulast%3DSoucek%26aufirst%3DL.%26aulast%3DHelmer-Citterich%26aufirst%3DM.%26aulast%3DSacco%26aufirst%3DA.%26aulast%3DJucker%26aufirst%3DR.%26aulast%3DCesareni%26aufirst%3DG.%26aulast%3DNasi%26aufirst%3DS.%26atitle%3DDesign%2520and%2520Properties%2520of%2520a%2520Myc%2520Derivative%2520That%2520Efficiently%2520Homodimerizes%26jtitle%3DOncogene%26date%3D1998%26volume%3D17%26issue%3D19%26spage%3D2463%26epage%3D2472%26doi%3D10.1038%2Fsj.onc.1202199" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massó-Vallés, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Blocking Myc to Treat Cancer: Reflecting on Two Decades of Omomyc</span>. <i>Cells</i>  <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">883</span>, <span class="refDoi">&nbsp;DOI: 10.3390/cells9040883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.3390%2Fcells9040883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1ChsrnO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2020&amp;pages=883&amp;issue=4&amp;author=D.+Mass%C3%B3-Vall%C3%A9sauthor=L.+Soucek&amp;title=Blocking+Myc+to+Treat+Cancer%3A+Reflecting+on+Two+Decades+of+Omomyc&amp;doi=10.3390%2Fcells9040883" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Blocking Myc to treat cancer: reflecting on two decades of Omomyc</span></div><div class="casAuthors">Masso-Valles, Daniel; Soucek, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Cells</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">883</span>CODEN:
                <span class="NLM_cas:coden">CELLC6</span>;
        ISSN:<span class="NLM_cas:issn">2073-4409</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">First designed and published in 1998 as a lab. tool to study Myc perturbation, Omomyc has come a long way in the past 22 years.  This dominant neg. has contributed to our understanding of Myc biol. when expressed, first, in normal and cancer cells, and later in genetically-engineered mice, and has shown remarkable anti-cancer properties in a wide range of tumor types.  The recently described therapeutic effect of purified Omomyc mini-protein-following the surprising discovery of its cell-penetrating capacity-constitutes a paradigm shift.  Now, much more than a proof of concept, the most characterized Myc inhibitor to date is advancing in its drug development pipeline, pushing Myc inhibition into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxDW79cqSCgLVg90H21EOLACvtfcHk0ljtLAplQxT8Vw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3cXis1ChsrnO&amp;md5=57cd831e89fe90d06629729d11f63a79</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.3390%2Fcells9040883&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcells9040883%26sid%3Dliteratum%253Aachs%26aulast%3DMass%25C3%25B3-Vall%25C3%25A9s%26aufirst%3DD.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DBlocking%2520Myc%2520to%2520Treat%2520Cancer%253A%2520Reflecting%2520on%2520Two%2520Decades%2520of%2520Omomyc%26jtitle%3DCells%26date%3D2020%26volume%3D9%26issue%3D4%26spage%3D883%26doi%3D10.3390%2Fcells9040883" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beaulieu, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jauset, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massó-Vallés, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Martín, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maltais, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zacarias-Fluck, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casacuberta-Serra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Pozo, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiore, C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Intrinsic Cell-Penetrating Activity Propels Omomyc from Proof of Concept to Viable Anti-Myc Therapy</span>. <i>Sci. Transl. Med.</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">484</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">14</span>, <span class="refDoi">&nbsp;DOI: 10.1126/scitranslmed.aar5012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1126%2Fscitranslmed.aar5012" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2019&amp;pages=1-14&amp;issue=484&amp;author=M.+E.+Beaulieuauthor=T.+Jausetauthor=D.+Mass%C3%B3-Vall%C3%A9sauthor=S.+Mart%C3%ADnez-Mart%C3%ADnauthor=P.+Rahlauthor=L.+Maltaisauthor=M.+F.+Zacarias-Fluckauthor=S.+Casacuberta-Serraauthor=E.+S.+Del+Pozoauthor=C.+Fiore&amp;title=Intrinsic+Cell-Penetrating+Activity+Propels+Omomyc+from+Proof+of+Concept+to+Viable+Anti-Myc+Therapy&amp;doi=10.1126%2Fscitranslmed.aar5012" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aar5012&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aar5012%26sid%3Dliteratum%253Aachs%26aulast%3DBeaulieu%26aufirst%3DM.%2BE.%26aulast%3DJauset%26aufirst%3DT.%26aulast%3DMass%25C3%25B3-Vall%25C3%25A9s%26aufirst%3DD.%26aulast%3DMart%25C3%25ADnez-Mart%25C3%25ADn%26aufirst%3DS.%26aulast%3DRahl%26aufirst%3DP.%26aulast%3DMaltais%26aufirst%3DL.%26aulast%3DZacarias-Fluck%26aufirst%3DM.%2BF.%26aulast%3DCasacuberta-Serra%26aufirst%3DS.%26aulast%3DDel%2BPozo%26aufirst%3DE.%2BS.%26aulast%3DFiore%26aufirst%3DC.%26atitle%3DIntrinsic%2520Cell-Penetrating%2520Activity%2520Propels%2520Omomyc%2520from%2520Proof%2520of%2520Concept%2520to%2520Viable%2520Anti-Myc%2520Therapy%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2019%26volume%3D11%26issue%3D484%26spage%3D1%26epage%3D14%26doi%3D10.1126%2Fscitranslmed.aar5012" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demma, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruprecht, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javaid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiswell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muise, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zelina, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Omomyc Reveals New Mechanisms To Inhibit the MYC Oncogene</span>. <i>Mol. Cell. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">27</span>, <span class="refDoi">&nbsp;DOI: 10.1128/MCB.00248-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1128%2FMCB.00248-19" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=2019&amp;pages=1-27&amp;issue=22&amp;author=M.+J.+Demmaauthor=C.+Mapelliauthor=A.+Sunauthor=S.+Bodeaauthor=B.+Ruprechtauthor=S.+Javaidauthor=D.+Wiswellauthor=E.+Muiseauthor=S.+Chenauthor=J.+Zelina&amp;title=Omomyc+Reveals+New+Mechanisms+To+Inhibit+the+MYC+Oncogene&amp;doi=10.1128%2FMCB.00248-19" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1128%2FMCB.00248-19&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00248-19%26sid%3Dliteratum%253Aachs%26aulast%3DDemma%26aufirst%3DM.%2BJ.%26aulast%3DMapelli%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DA.%26aulast%3DBodea%26aufirst%3DS.%26aulast%3DRuprecht%26aufirst%3DB.%26aulast%3DJavaid%26aufirst%3DS.%26aulast%3DWiswell%26aufirst%3DD.%26aulast%3DMuise%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DZelina%26aufirst%3DJ.%26atitle%3DOmomyc%2520Reveals%2520New%2520Mechanisms%2520To%2520Inhibit%2520the%2520MYC%2520Oncogene%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2019%26volume%3D39%26issue%3D22%26spage%3D1%26epage%3D27%26doi%3D10.1128%2FMCB.00248-19" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lobba, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fellmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmelstein, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maza, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kissman, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staahl, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urnes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lew, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogilevsky, C. S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Site-Specific Bioconjugation through Enzyme-Catalyzed Tyrosine–Cysteine Bond Formation</span>. <i>ACS Cent. Sci.</i>  <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1564</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscentsci.0c00940</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.0c00940" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1eltL3P" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2020&amp;pages=1564&amp;author=M.+J.+Lobbaauthor=C.+Fellmannauthor=A.+M.+Marmelsteinauthor=J.+C.+Mazaauthor=E.+N.+Kissmanauthor=S.+A.+Robinsonauthor=B.+T.+Staahlauthor=C.+Urnesauthor=R.+J.+Lewauthor=C.+S.+Mogilevsky&amp;title=Site-Specific+Bioconjugation+through+Enzyme-Catalyzed+Tyrosine%E2%80%93Cysteine+Bond+Formation&amp;doi=10.1021%2Facscentsci.0c00940" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Site-Specific Bioconjugation through Enzyme-Catalyzed Tyrosine-Cysteine Bond Formation</span></div><div class="casAuthors">Lobba, Marco J.; Fellmann, Christof; Marmelstein, Alan M.; Maza, Johnathan C.; Kissman, Elijah N.; Robinson, Stephanie A.; Staahl, Brett T.; Urnes, Cole; Lew, Rachel J.; Mogilevsky, Casey S.; Doudna, Jennifer A.; Francis, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1564-1571</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of protein-protein and protein-peptide conjugates is an important capability for producing vaccines, immunotherapeutics, and targeted delivery agents.  Herein we show that the enzyme tyrosinase is capable of oxidizing exposed tyrosine residues into o-quinones that react rapidly with cysteine residues on target proteins.  This coupling reaction occurs under mild aerobic conditions and has the rare ability to join full-size proteins in under 2 h.  The utility of the approach is demonstrated for the attachment of cationic peptides to enhance the cellular delivery of CRISPR-Cas9 20-fold and for the coupling of reporter proteins to a cancer-targeting antibody fragment without loss of its cell-specific binding ability.  The broad applicability of this technique provides a new building block approach for the synthesis of protein chimeras.  Enzymic oxidn. of tyrosine residues followed by reaction with cysteine thiols allows the covalent coupling of proteins and peptides using only native amino acid side chains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTq03w1W3XmLVg90H21EOLACvtfcHk0lhuHP2sMfe_CA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3cXhs1eltL3P&amp;md5=a0fe6337ef9daa4caa637a7ba54f9d68</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.0c00940&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.0c00940%26sid%3Dliteratum%253Aachs%26aulast%3DLobba%26aufirst%3DM.%2BJ.%26aulast%3DFellmann%26aufirst%3DC.%26aulast%3DMarmelstein%26aufirst%3DA.%2BM.%26aulast%3DMaza%26aufirst%3DJ.%2BC.%26aulast%3DKissman%26aufirst%3DE.%2BN.%26aulast%3DRobinson%26aufirst%3DS.%2BA.%26aulast%3DStaahl%26aufirst%3DB.%2BT.%26aulast%3DUrnes%26aufirst%3DC.%26aulast%3DLew%26aufirst%3DR.%2BJ.%26aulast%3DMogilevsky%26aufirst%3DC.%2BS.%26atitle%3DSite-Specific%2520Bioconjugation%2520through%2520Enzyme-Catalyzed%2520Tyrosine%25E2%2580%2593Cysteine%2520Bond%2520Formation%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2020%26volume%3D6%26spage%3D1564%26doi%3D10.1021%2Facscentsci.0c00940" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhanjee, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saebi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buslov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loftis, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span> <span> </span><span class="NLM_article-title">Protein-Protein Cross-Coupling via Palladium-Protein Oxidative Addition Complexes from Cysteine Residues</span>. <i>J. Am. Chem. Soc.</i>  <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">9124</span>– <span class="NLM_lpage">9129</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.0c03143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.0c03143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosF2gtL4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=142&amp;publication_year=2020&amp;pages=9124-9129&amp;issue=20&amp;author=H.+H.+Dhanjeeauthor=A.+Saebiauthor=I.+Buslovauthor=A.+R.+Loftisauthor=S.+L.+Buchwaldauthor=B.+L.+Pentelute&amp;title=Protein-Protein+Cross-Coupling+via+Palladium-Protein+Oxidative+Addition+Complexes+from+Cysteine+Residues&amp;doi=10.1021%2Fjacs.0c03143" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-Protein Cross-Coupling via Palladium-Protein Oxidative Addition Complexes from Cysteine Residues</span></div><div class="casAuthors">Dhanjee, Heemal H.; Saebi, Azin; Buslov, Ivan; Loftis, Alexander R.; Buchwald, Stephen L.; Pentelute, Bradley L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9124-9129</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Few chem. methods exist for the covalent conjugation of two proteins.  The authors report the prepn. of site-specific protein-protein conjugates that arise from the sequential cross-coupling of cysteine residues on two different proteins.  The method involves the synthesis of stable palladium-protein oxidative addn. complexes (Pd-protein OACs), a process that converts nucleophilic cysteine residues into an electrophilic S-aryl-Pd-X unit by taking advantage of an intramol. oxidative addn. strategy.  This process is demonstrated on proteins up to 83 kDa in size and can be conveniently carried out in water and open to air.  The resulting Pd-protein OACs can cross-couple with other thiol-contg. proteins to arrive at homogeneous protein-protein bioconjugates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLCs1S4RjC0bVg90H21EOLACvtfcHk0lhuHP2sMfe_CA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3cXosF2gtL4%253D&amp;md5=5944224f692948db7ba7f1bf3a557172</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjacs.0c03143&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.0c03143%26sid%3Dliteratum%253Aachs%26aulast%3DDhanjee%26aufirst%3DH.%2BH.%26aulast%3DSaebi%26aufirst%3DA.%26aulast%3DBuslov%26aufirst%3DI.%26aulast%3DLoftis%26aufirst%3DA.%2BR.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26atitle%3DProtein-Protein%2520Cross-Coupling%2520via%2520Palladium-Protein%2520Oxidative%2520Addition%2520Complexes%2520from%2520Cysteine%2520Residues%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26issue%3D20%26spage%3D9124%26epage%3D9129%26doi%3D10.1021%2Fjacs.0c03143" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, K. S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spasser, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlich, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bavikar, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brik, A.</span></span> <span> </span><span class="NLM_article-title">Total Chemical Synthesis of Di-Ubiquitin Chains</span>. <i>Angew. Chem., Int. Ed.</i>  <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span> (<span class="NLM_issue">48</span>),  <span class="NLM_fpage">9126</span>– <span class="NLM_lpage">9131</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.201003763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1002%2Fanie.201003763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVCjsLfN" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=49&amp;publication_year=2010&amp;pages=9126-9131&amp;issue=48&amp;author=K.+S.+A.+Kumarauthor=L.+Spasserauthor=L.+A.+Erlichauthor=S.+N.+Bavikarauthor=A.+Brik&amp;title=Total+Chemical+Synthesis+of+Di-Ubiquitin+Chains&amp;doi=10.1002%2Fanie.201003763" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Total Chemical Synthesis of Di-ubiquitin Chains</span></div><div class="casAuthors">Kumar, K. S. Ajish; Spasser, Liat; Erlich, Lesly A.; Bavikar, Sudhir N.; Brik, Ashraf</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">9126-9131, S9126/1-S9126/20</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH &amp; Co. KGaA</span>)
        </div><div class="casAbstract">The authors report the first total chem. synthesis of all the di-ubiquitin (di-Ub) chains by using δ-mercaptolysine-contg. peptides, Ub peptide thioesters and native chem. ligation methodol.  δ-Mercaptolysine was introduced to mediate the transthioesterification step of the Ub thioester and subsequent S→N acyl transfer to form the isopeptide bond.  These analogs displayed similar folding to Ub monomer, and were tested for their activities with the human isopeptidase T (IsoT).  Peptides UbK48-Ub, UbK63-Ub, UbK33-Ub, UbK29-Ub, UbK11-Ub, and UbK46-Ub were found to be substrates of this enzyme.  Furthermore, UbK46-Ub was the preferred substrate of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-sj-jsyxmALVg90H21EOLACvtfcHk0lj9CMcAtOekXw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXhsVCjsLfN&amp;md5=a14b34f4be34ee4dac4d02b37780e167</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201003763&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201003763%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DK.%2BS.%2BA.%26aulast%3DSpasser%26aufirst%3DL.%26aulast%3DErlich%26aufirst%3DL.%2BA.%26aulast%3DBavikar%26aufirst%3DS.%2BN.%26aulast%3DBrik%26aufirst%3DA.%26atitle%3DTotal%2520Chemical%2520Synthesis%2520of%2520Di-Ubiquitin%2520Chains%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26issue%3D48%26spage%3D9126%26epage%3D9131%26doi%3D10.1002%2Fanie.201003763" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinty, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellois, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muir, T. W.</span></span> <span> </span><span class="NLM_article-title">Auxiliary-Mediated Site-Specific Peptide Ubiquitylation</span>. <i>Angew. Chem., Int. Ed.</i>  <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>119</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">2872</span>– <span class="NLM_lpage">2876</span>, <span class="refDoi">&nbsp;DOI: 10.1002/ange.200605155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1002%2Fange.200605155" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=119&amp;publication_year=2007&amp;pages=2872-2876&amp;issue=16&amp;author=C.+Chatterjeeauthor=R.+K.+McGintyauthor=J.-P.+Pelloisauthor=T.+W.+Muir&amp;title=Auxiliary-Mediated+Site-Specific+Peptide+Ubiquitylation&amp;doi=10.1002%2Fange.200605155" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fange.200605155&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fange.200605155%26sid%3Dliteratum%253Aachs%26aulast%3DChatterjee%26aufirst%3DC.%26aulast%3DMcGinty%26aufirst%3DR.%2BK.%26aulast%3DPellois%26aufirst%3DJ.-P.%26aulast%3DMuir%26aufirst%3DT.%2BW.%26atitle%3DAuxiliary-Mediated%2520Site-Specific%2520Peptide%2520Ubiquitylation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2007%26volume%3D119%26issue%3D16%26spage%3D2872%26epage%3D2876%26doi%3D10.1002%2Fange.200605155" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ajish
Kumar, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haj-Yahya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olschewski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lashuel, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brik, A.</span></span> <span> </span><span class="NLM_article-title">Highly Efficient and Chemoselective Peptide Ubiquitylation</span>. <i>Angew. Chem., Int. Ed.</i>  <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">43</span>),  <span class="NLM_fpage">8090</span>– <span class="NLM_lpage">8094</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.200902936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1002%2Fanie.200902936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Kju77N" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=48&amp;publication_year=2009&amp;pages=8090-8094&amp;issue=43&amp;author=K.+S.+Ajish%0AKumarauthor=M.+Haj-Yahyaauthor=D.+Olschewskiauthor=H.+A.+Lashuelauthor=A.+Brik&amp;title=Highly+Efficient+and+Chemoselective+Peptide+Ubiquitylation&amp;doi=10.1002%2Fanie.200902936" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Efficient and Chemoselective Peptide Ubiquitylation</span></div><div class="casAuthors">Ajish Kumar, K. S.; Haj-Yahya, Mahmood; Olschewski, Diana; Lashuel, Hilal A.; Brik, Ashraf</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">8090-8094, S8090/1-S8090/31</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH &amp; Co. KGaA</span>)
        </div><div class="casAbstract">The authors synthesized δ-mercaptolysine and incorporated into a model peptide [α-synuclein(1-17); MDVFMKGLSKAKEGVVA; where Met was replaced by Nle to avoid any oxidn.] to allow a thiol capture step with ubiquitin thioester.  Thus, during ubiquitination of the model peptide, the thiol functionality of δ-mercaptolysine serves as a temporary handle in assisting the isopeptide formation.  Following the native chem. ligation step, the thiol moiety of δ-mercaptolysine is removed by desulfurization reaction, thereby converting the said amino acid into lysine and resulting in the formation of an isopeptide, where ubiquitin is linked to the model peptide via ε-amino group of lysine.  In combination with native chem. ligation, this strategy should facilitate the synthesis of ubiquitylated proteins for biochem. and structural studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGidcttT7KBLVg90H21EOLACvtfcHk0lj9CMcAtOekXw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXht1Kju77N&amp;md5=fe982bd9bfe9944599cb4eac83663082</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.200902936&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200902936%26sid%3Dliteratum%253Aachs%26aulast%3DAjish%2BKumar%26aufirst%3DK.%2BS.%26aulast%3DHaj-Yahya%26aufirst%3DM.%26aulast%3DOlschewski%26aufirst%3DD.%26aulast%3DLashuel%26aufirst%3DH.%2BA.%26aulast%3DBrik%26aufirst%3DA.%26atitle%3DHighly%2520Efficient%2520and%2520Chemoselective%2520Peptide%2520Ubiquitylation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26issue%3D43%26spage%3D8090%26epage%3D8094%26doi%3D10.1002%2Fanie.200902936" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fottner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittl, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn-Ghetko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jussupow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaila, V. R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bremm, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, K.</span></span> <span> </span><span class="NLM_article-title">Site-Specific Ubiquitylation and SUMOylation Using Genetic-Code Expansion and Sortase</span>. <i>Nat. Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">284</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41589-019-0227-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fs41589-019-0227-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30770915" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsFSls7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=2019&amp;pages=276-284&amp;issue=3&amp;author=M.+Fottnerauthor=A.+D.+Brunnerauthor=V.+Bittlauthor=D.+Horn-Ghetkoauthor=A.+Jussupowauthor=V.+R.+I.+Kailaauthor=A.+Bremmauthor=K.+Lang&amp;title=Site-Specific+Ubiquitylation+and+SUMOylation+Using+Genetic-Code+Expansion+and+Sortase&amp;doi=10.1038%2Fs41589-019-0227-4" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Site-specific ubiquitylation and SUMOylation using genetic-code expansion and sortase</span></div><div class="casAuthors">Fottner, Maximilian; Brunner, Andreas-David; Bittl, Verena; Horn-Ghetko, Daniel; Jussupow, Alexander; Kaila, Ville R. I.; Bremm, Anja; Lang, Kathrin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">276-284</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Post-translational modification of proteins with ubiquitin and ubiquitin-like proteins (Ubls) is central to the regulation of eukaryotic cellular processes.  The authors' ability to study the effects of ubiquitylation, however, is limited by the difficulty to prep. homogenously modified proteins in vitro and by the impossibility to selectively trigger specific ubiquitylation events in living cells.  Here the authors combine genetic-code expansion, bioorthogonal Staudinger redn. and sortase-mediated transpeptidation to develop a general tool to ubiquitylate proteins in an inducible fashion.  The generated ubiquitin conjugates display a native isopeptide bond and bear two point mutations in the ubiquitin C terminus that confer resistance toward deubiquitinases.  Nevertheless, physiol. integrity of sortase-generated diubiquitins in decoding cellular functions via recognition by ubiquitin-binding domains is retained.  The authors' approach allows the site-specific attachment of Ubls to nonrefoldable, multidomain proteins and enables inducible and ubiquitin-ligase-independent ubiquitylation of proteins in mammalian cells, providing a powerful tool to dissect the biol. functions of ubiquitylation with temporal control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY1Qlwj1GxIrVg90H21EOLACvtfcHk0lgt0S3mqhizRA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXnsFSls7c%253D&amp;md5=93007ea246ce639f169661a4cb962228</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0227-4&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0227-4%26sid%3Dliteratum%253Aachs%26aulast%3DFottner%26aufirst%3DM.%26aulast%3DBrunner%26aufirst%3DA.%2BD.%26aulast%3DBittl%26aufirst%3DV.%26aulast%3DHorn-Ghetko%26aufirst%3DD.%26aulast%3DJussupow%26aufirst%3DA.%26aulast%3DKaila%26aufirst%3DV.%2BR.%2BI.%26aulast%3DBremm%26aufirst%3DA.%26aulast%3DLang%26aufirst%3DK.%26atitle%3DSite-Specific%2520Ubiquitylation%2520and%2520SUMOylation%2520Using%2520Genetic-Code%2520Expansion%2520and%2520Sortase%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26issue%3D3%26spage%3D276%26epage%3D284%26doi%3D10.1038%2Fs41589-019-0227-4" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span> <span> </span><span class="NLM_article-title">Development and Application of Ubiquitin-Based Chemical Probes</span>. <i>Chem. Sci.</i>  <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">47</span>),  <span class="NLM_fpage">12633</span>– <span class="NLM_lpage">12646</span>, <span class="refDoi">&nbsp;DOI: 10.1039/D0SC03295F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1039%2FD0SC03295F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=34123237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFWhsr7L" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2020&amp;pages=12633-12646&amp;issue=47&amp;author=X.+Suiauthor=Y.+Wangauthor=Y.+X.+Duauthor=L.+J.+Liangauthor=Q.+Zhengauthor=Y.+M.+Liauthor=L.+Liu&amp;title=Development+and+Application+of+Ubiquitin-Based+Chemical+Probes&amp;doi=10.1039%2FD0SC03295F" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Development and application of ubiquitin-based chemical probes</span></div><div class="casAuthors">Sui, Xin; Wang, Yu; Du, Yun-Xiang; Liang, Lu-Jun; Zheng, Qingyun; Li, Yi-Ming; Liu, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">12633-12646</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Protein ubiquitination regulates almost every process in eukaryotic cells.  The study of the many enzymes involved in the ubiquitination system and the development of ubiquitination-assocd. therapeutics are important areas of current research.  Synthetic tools such as ubiquitin-based chem. probes have been making an increasing contribution to deciphering various biochem. components involved in ubiquitin conjugation, recruitment, signaling, and deconjugation.  In the present minireview, we summarize the progress of ubiquitin-based chem. probes with an emphasis on their various structures and chem. synthesis.  We discuss the utility of the ubiquitin-based chem. probes for discovering and profiling ubiquitin-dependent signaling systems, as well as the monitoring and visualization of ubiquitin-related enzymic machinery.  We also show how the probes can serve to elucidate the mol. mechanism of recognition and catalysis.  Collectively, the development and application of ubiquitin-based chem. probes emphasizes the importance and utility of chem. protein synthesis in modern chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGjlv2fA9kXbVg90H21EOLACvtfcHk0lgt0S3mqhizRA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3cXhsFWhsr7L&amp;md5=435454dd2d00472462c53bc111dd5820</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1039%2FD0SC03295F&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FD0SC03295F%26sid%3Dliteratum%253Aachs%26aulast%3DSui%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DY.%2BX.%26aulast%3DLiang%26aufirst%3DL.%2BJ.%26aulast%3DZheng%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DY.%2BM.%26aulast%3DLiu%26aufirst%3DL.%26atitle%3DDevelopment%2520and%2520Application%2520of%2520Ubiquitin-Based%2520Chemical%2520Probes%26jtitle%3DChem.%2520Sci.%26date%3D2020%26volume%3D11%26issue%3D47%26spage%3D12633%26epage%3D12646%26doi%3D10.1039%2FD0SC03295F" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Geurink, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Oualid, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hameed, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovaa, H.</span></span> <span> </span><span class="NLM_article-title">A General Chemical Ligation Approach Towards Isopeptide-Linked Ubiquitin and Ubiquitin-Like Assay Reagents</span>. <i>ChemBioChem</i>  <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">297</span>, <span class="refDoi">&nbsp;DOI: 10.1002/cbic.201100706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1002%2Fcbic.201100706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=22213387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1OiurrJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=13&amp;publication_year=2012&amp;pages=293-297&amp;issue=2&amp;author=P.+P.+Geurinkauthor=F.+El+Oualidauthor=A.+Jonkerauthor=D.+S.+Hameedauthor=H.+Ovaa&amp;title=A+General+Chemical+Ligation+Approach+Towards+Isopeptide-Linked+Ubiquitin+and+Ubiquitin-Like+Assay+Reagents&amp;doi=10.1002%2Fcbic.201100706" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A General Chemical Ligation Approach Towards Isopeptide-Linked Ubiquitin and Ubiquitin-Like Assay Reagents</span></div><div class="casAuthors">Geurink, Paul P.; El Oualid, Farid; Jonker, Anika; Hameed, Dharjath S.; Ovaa, Huib</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">293-297</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH &amp; Co. KGaA</span>)
        </div><div class="casAbstract">The 5-thiolysine-mediated ligation can be used to generate a wide range of isopeptide-linked ubiquitin Ub(I)-based FP (fluorescence anisotropy/flourescence polarization) assay reagents.  These are powerful tools for measuring Ub(I) deconjugating activity.  The methodol. can be adapted to any ubiquitin-like proteins (Ubl) for which the E1 enzyme is available.  In contrast to any other reagent, it is possible to incorporate substrate-based peptide sequences.  This is a major advantage, as it offers the possibility of more physiol. relevant assay reagents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqou4rjkVgI1rVg90H21EOLACvtfcHk0lgt0S3mqhizRA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXhs1OiurrJ&amp;md5=4c7d4048b95b1218a340123dd71d9716</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201100706&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201100706%26sid%3Dliteratum%253Aachs%26aulast%3DGeurink%26aufirst%3DP.%2BP.%26aulast%3DEl%2BOualid%26aufirst%3DF.%26aulast%3DJonker%26aufirst%3DA.%26aulast%3DHameed%26aufirst%3DD.%2BS.%26aulast%3DOvaa%26aufirst%3DH.%26atitle%3DA%2520General%2520Chemical%2520Ligation%2520Approach%2520Towards%2520Isopeptide-Linked%2520Ubiquitin%2520and%2520Ubiquitin-Like%2520Assay%2520Reagents%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26issue%3D2%26spage%3D293%26epage%3D297%26doi%3D10.1002%2Fcbic.201100706" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Premdjee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, R. J.</span></span> <span> </span><span class="NLM_article-title">Rapid and Efficient Protein Synthesis through Expansion of the Native Chemical Ligation Concept</span>. <i>Nat. Rev. Chem.</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">17</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41570-018-0122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fs41570-018-0122" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2&amp;publication_year=2018&amp;pages=1-17&amp;issue=4&amp;author=S.+S.+Kulkarniauthor=J.+Sayersauthor=B.+Premdjeeauthor=R.+J.+Payne&amp;title=Rapid+and+Efficient+Protein+Synthesis+through+Expansion+of+the+Native+Chemical+Ligation+Concept&amp;doi=10.1038%2Fs41570-018-0122" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fs41570-018-0122&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41570-018-0122%26sid%3Dliteratum%253Aachs%26aulast%3DKulkarni%26aufirst%3DS.%2BS.%26aulast%3DSayers%26aufirst%3DJ.%26aulast%3DPremdjee%26aufirst%3DB.%26aulast%3DPayne%26aufirst%3DR.%2BJ.%26atitle%3DRapid%2520and%2520Efficient%2520Protein%2520Synthesis%2520through%2520Expansion%2520of%2520the%2520Native%2520Chemical%2520Ligation%2520Concept%26jtitle%3DNat.%2520Rev.%2520Chem.%26date%3D2018%26volume%3D2%26issue%3D4%26spage%3D1%26epage%3D17%26doi%3D10.1038%2Fs41570-018-0122" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Q.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Quasi-Racemic X-Ray Structures of K27-Linked Ubiquitin Chains Prepared by Total Chemical Synthesis</span>. <i>J. Am. Chem. Soc.</i>  <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">7429</span>– <span class="NLM_lpage">7435</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.6b04031</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b04031" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC28XptFWmsbg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=138&amp;publication_year=2016&amp;pages=7429-7435&amp;issue=23&amp;author=M.+Panauthor=S.+Gaoauthor=Y.+Zhengauthor=X.+Tanauthor=H.+Lanauthor=X.+Tanauthor=D.+Sunauthor=L.+Luauthor=T.+Wangauthor=Q.+Zheng&amp;title=Quasi-Racemic+X-Ray+Structures+of+K27-Linked+Ubiquitin+Chains+Prepared+by+Total+Chemical+Synthesis&amp;doi=10.1021%2Fjacs.6b04031" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Quasi-Racemic X-ray Structures of K27-Linked Ubiquitin Chains Prepared by Total Chemical Synthesis</span></div><div class="casAuthors">Pan, Man; Gao, Shuai; Zheng, Yong; Tan, Xiaodan; Lan, Huan; Tan, Xianglong; Sun, Demeng; Lu, Lining; Wang, Tian; Zheng, Qingyun; Huang, Yichao; Wang, Jiawei; Liu, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7429-7435</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Quasi-racemic crystallog. has been used to det. the X-ray structures of K27-linked ubiquitin (Ub) chains prepd. through total chem. synthesis.  Crystal structures of K27-linked di- and tri-ubiquitins reveal that the isopeptide linkages are confined in a unique buried conformation, which provides the mol. basis for the distinctive function of K27 linkage compared to the other seven Ub chains.  K27-linked di- and triUb were found to adopt different structural conformations in the crystals, one being sym. whereas the other triangular.  Furthermore, bioactivity expts. showed that the ovarian tumor family de-ubiquitinase 2 significantly favors K27-linked triUb than K27-linked diUb.  K27-linked triUb represents the so-far largest chem. synthesized protein (228 amino acids) that has been crystd. to afford a high-resoln. X-ray structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovteTSewd5SLVg90H21EOLACvtfcHk0lhHoJV0tb2-dQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XptFWmsbg%253D&amp;md5=43d254b471483350485743a347a635ac</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b04031&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b04031%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DM.%26aulast%3DGao%26aufirst%3DS.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DX.%26aulast%3DLan%26aufirst%3DH.%26aulast%3DTan%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DQ.%26atitle%3DQuasi-Racemic%2520X-Ray%2520Structures%2520of%2520K27-Linked%2520Ubiquitin%2520Chains%2520Prepared%2520by%2520Total%2520Chemical%2520Synthesis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26issue%3D23%26spage%3D7429%26epage%3D7435%26doi%3D10.1021%2Fjacs.6b04031" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jbara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laps, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamnesky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolberger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brik, A.</span></span> <span> </span><span class="NLM_article-title">Palladium Prompted On-Demand Cysteine Chemistry for the Synthesis of Challenging and Uniquely Modified Proteins</span>. <i>Nat. Commun.</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">3154</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41467-018-05628-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fs41467-018-05628-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30089783" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A280%3ADC%252BB3c7mtlSrsA%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2018&amp;pages=3154&amp;issue=1&amp;author=M.+Jbaraauthor=S.+Lapsauthor=M.+Morganauthor=G.+Kamneskyauthor=G.+Mannauthor=C.+Wolbergerauthor=A.+Brik&amp;title=Palladium+Prompted+On-Demand+Cysteine+Chemistry+for+the+Synthesis+of+Challenging+and+Uniquely+Modified+Proteins&amp;doi=10.1038%2Fs41467-018-05628-0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium prompted on-demand cysteine chemistry for the synthesis of challenging and uniquely modified proteins</span></div><div class="casAuthors">Jbara Muhammad; Laps Shay; Kamnesky Guy; Mann Guy; Brik Ashraf; Morgan Michael; Wolberger Cynthia</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3154</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Organic chemistry allows for the modification and chemical preparation of protein analogues for various studies.  The thiolate side chain of the Cys residue has been a key functionality in these ventures.  In order to generate complex molecular targets, there is a particular need to incorporate orthogonal protecting groups of the thiolated amino acids to control the directionality of synthesis and modification site.  Here, we demonstrate the tuning of palladium chemoselectivity in aqueous medium for on-demand deprotection of several Cys-protecting groups that are useful in protein synthesis and modification.  These tools allow the preparation of highly complex analogues as we demonstrate in the synthesis of the copper storage protein and selectively modified peptides with multiple Cys residues.  We also report the synthesis of an activity-based probe comprising ubiquitinated histone H2A and its incorporation into nucleosomes and demonstrate its reactivity with deubiquitinating enzyme to generate a covalent nucleosome-enzyme complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQgWwVK8QQFg6-9TkgmBqTqfW6udTcc2eaCK0fZdGUejbntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BB3c7mtlSrsA%253D%253D&amp;md5=8e298b89bb60f3f1a3c85f7880c36ae0</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-05628-0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-05628-0%26sid%3Dliteratum%253Aachs%26aulast%3DJbara%26aufirst%3DM.%26aulast%3DLaps%26aufirst%3DS.%26aulast%3DMorgan%26aufirst%3DM.%26aulast%3DKamnesky%26aufirst%3DG.%26aulast%3DMann%26aufirst%3DG.%26aulast%3DWolberger%26aufirst%3DC.%26aulast%3DBrik%26aufirst%3DA.%26atitle%3DPalladium%2520Prompted%2520On-Demand%2520Cysteine%2520Chemistry%2520for%2520the%2520Synthesis%2520of%2520Challenging%2520and%2520Uniquely%2520Modified%2520Proteins%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26issue%3D1%26spage%3D3154%26doi%3D10.1038%2Fs41467-018-05628-0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Torbeev, V. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raghuraman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamelberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westler, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perozo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, S. B. H.</span></span> <span> </span><span class="NLM_article-title">Protein Conformational Dynamics in the Mechanism of HIV-1 Protease Catalysis</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i>  <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span> (<span class="NLM_issue">52</span>),  <span class="NLM_fpage">20982</span>– <span class="NLM_lpage">20987</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1111202108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1073%2Fpnas.1111202108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=22158985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvVCqtw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=108&amp;publication_year=2011&amp;pages=20982-20987&amp;issue=52&amp;author=V.+Y.+Torbeevauthor=H.+Raghuramanauthor=D.+Hamelbergauthor=M.+Tonelliauthor=W.+M.+Westlerauthor=E.+Perozoauthor=S.+B.+H.+Kent&amp;title=Protein+Conformational+Dynamics+in+the+Mechanism+of+HIV-1+Protease+Catalysis&amp;doi=10.1073%2Fpnas.1111202108" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Protein conformational dynamics in the mechanism of HIV-1 protease catalysis</span></div><div class="casAuthors">Torbeev, Vladimir Yu.; Raghuraman, H.; Hamelberg, Donald; Tonelli, Marco; Westler, William M.; Perozo, Eduardo; Kent, Stephen B. H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">20982-20987, S20982/1-S20982/44</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We have used chem. protein synthesis and advanced phys. methods to probe dynamics-function correlations for the HIV-1 protease, an enzyme that has received considerable attention as a target for the treatment of AIDS.  Chem. synthesis was used to prep. a series of unique analogs of the HIV-1 protease in which the flexibility of the "flap" structures (residues 37-61 in each monomer of the homodimeric protein mol.) was systematically varied.  These analog enzymes were further studied by X-ray crystallog., NMR relaxation, and pulse-EPR methods, in conjunction with mol. dynamics simulations.  We show that conformational isomerization in the flaps is correlated with structural reorganization of residues in the active site, and that it is preorganization of the active site that is a rate-limiting factor in catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlidz4XYglO7Vg90H21EOLACvtfcHk0ljlEfqYYI06PA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XnvVCqtw%253D%253D&amp;md5=1e5b305474c1d248d0a087c9ffc15f66</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1111202108&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1111202108%26sid%3Dliteratum%253Aachs%26aulast%3DTorbeev%26aufirst%3DV.%2BY.%26aulast%3DRaghuraman%26aufirst%3DH.%26aulast%3DHamelberg%26aufirst%3DD.%26aulast%3DTonelli%26aufirst%3DM.%26aulast%3DWestler%26aufirst%3DW.%2BM.%26aulast%3DPerozo%26aufirst%3DE.%26aulast%3DKent%26aufirst%3DS.%2BB.%2BH.%26atitle%3DProtein%2520Conformational%2520Dynamics%2520in%2520the%2520Mechanism%2520of%2520HIV-1%2520Protease%2520Catalysis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26issue%3D52%26spage%3D20982%26epage%3D20987%26doi%3D10.1073%2Fpnas.1111202108" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nair, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span> <span> </span><span class="NLM_article-title">X-Ray Structures of Myc-Max and Mad-Max Recognizing DNA: Molecular Bases of Regulation by Proto-Oncogenic Transcription Factors</span>. <i>Cell</i>  <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>112</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">205</span>, <span class="refDoi">&nbsp;DOI: 10.1016/S0092-8674(02)01284-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2FS0092-8674%2802%2901284-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=12553908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvVemsQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=112&amp;publication_year=2003&amp;pages=193-205&amp;issue=2&amp;author=S.+K.+Nairauthor=S.+K.+Burley&amp;title=X-Ray+Structures+of+Myc-Max+and+Mad-Max+Recognizing+DNA%3A+Molecular+Bases+of+Regulation+by+Proto-Oncogenic+Transcription+Factors&amp;doi=10.1016%2FS0092-8674%2802%2901284-9" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">X-ray structures of Myc-Max and Mad-Max recognizing DNA: Molecular bases of regulation by proto-oncogenic transcription factors</span></div><div class="casAuthors">Nair, Satish K.; Burley, Stephen K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">193-205</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">X-ray structures of the basic/helix-loop-helix/leucine zipper (bHLHZ) domains of Myc-Max and Mad-Max heterodimers bound to their common DNA target (Enhancer or E box hexanucleotide, 5'-CACGTG-3') have been detd. at 1.9 Å and 2.0 Å resoln., resp.  E box recognition by these two structurally similar transcription factor pairs dets. whether a cell will divide and proliferate (Myc-Max) or differentiate and become quiescent (Mad-Max).  Deregulation of Myc has been implicated in the development of many human cancers, including Burkitt's lymphoma, neuroblastomas, and small cell lung cancers.  Both quasisym. heterodimers resemble the sym. Max homodimer, albeit with marked structural differences in the coiled-coil leucine zipper regions that explain preferential homo- and heteromeric dimerization of these three evolutionarily related DNA-binding proteins.  The Myc-Max heterodimer, but not its Mad-Max counterpart, dimerizes to form a bivalent heterotetramer, which explains how Myc can upregulate expression of genes with promoters bearing widely sepd. E boxes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgxSisNQ3O1LVg90H21EOLACvtfcHk0ljlEfqYYI06PA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3sXpvVemsQ%253D%253D&amp;md5=d049bd00310a705c951ece664e06cd22</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2901284-9&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252901284-9%26sid%3Dliteratum%253Aachs%26aulast%3DNair%26aufirst%3DS.%2BK.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26atitle%3DX-Ray%2520Structures%2520of%2520Myc-Max%2520and%2520Mad-Max%2520Recognizing%2520DNA%253A%2520Molecular%2520Bases%2520of%2520Regulation%2520by%2520Proto-Oncogenic%2520Transcription%2520Factors%26jtitle%3DCell%26date%3D2003%26volume%3D112%26issue%3D2%26spage%3D193%26epage%3D205%26doi%3D10.1016%2FS0092-8674%2802%2901284-9" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canne, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferré-D’Amaré, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, S. B. H.</span></span> <span> </span><span class="NLM_article-title">Total Chemical Synthesis of a Unique Transcription Factor-Related Protein: CMyc—Max</span>. <i>J. Am. Chem. Soc.</i>  <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>117</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2998</span>– <span class="NLM_lpage">3007</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja00116a005</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00116a005" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADyaK2MXktleltrk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=117&amp;publication_year=1995&amp;pages=2998-3007&amp;issue=11&amp;author=L.+E.+Canneauthor=A.+R.+Ferr%C3%A9-D%E2%80%99Amar%C3%A9author=S.+K.+Burleyauthor=S.+B.+H.+Kent&amp;title=Total+Chemical+Synthesis+of+a+Unique+Transcription+Factor-Related+Protein%3A+CMyc%E2%80%94Max&amp;doi=10.1021%2Fja00116a005" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Total Chemical Synthesis of a Unique Transcription Factor-Related Protein: cMyc-Max</span></div><div class="casAuthors">Canne, Lynne E.; Ferre- D'Amare, Adrian R.; Burley, Stephen K.; Kent, Stephen B. H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2998-3007</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CMyc and Max are proteins that regulate gene expression by assocg. to form noncovalent dimers that bind to specific regions of double-stranded DNA and activating or inhibiting mRNA transcription.  Each of these transcription factor proteins contains an ∼90 amino acid residue basic/helix-loop-helix/zipper (b/HLH/Z) domain responsible for formation of the protein dimers and for DNA binding.  Noncovalent heterodimers of the cMyc and Max b/HLH/Z domains are believed to be functionally important in vivo but have not been accessible by conventional means (recDNA expression).  Here the authors report the total chem. synthesis of novel covalently-linked dimers of the b/HLH/Z domains of these transcription factors by a modular strategy based on the convergent chem. ligation of several unprotected peptide segments.  The resulting ∼20 kDa synthetic proteins were purified to homogeneity and characterized by electrospray mass spectrometry.  The synthetic covalent cMyc-Max heterodimer and the covalent Max homodimer control were shown to bind DNA specifically at the region involved in the regulation of mRNA transcription.  Access to defined heterodimeric forms of the b/HLH/Z transcription factors will allow the structural and functional properties of these important protein regulators of gene expression to be studied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDeztgTOYiY7Vg90H21EOLACvtfcHk0lie6ot6yXVo-w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXktleltrk%253D&amp;md5=cfcfc89fd62eba4d197654a8d918a672</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja00116a005&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00116a005%26sid%3Dliteratum%253Aachs%26aulast%3DCanne%26aufirst%3DL.%2BE.%26aulast%3DFerr%25C3%25A9-D%25E2%2580%2599Amar%25C3%25A9%26aufirst%3DA.%2BR.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DKent%26aufirst%3DS.%2BB.%2BH.%26atitle%3DTotal%2520Chemical%2520Synthesis%2520of%2520a%2520Unique%2520Transcription%2520Factor-Related%2520Protein%253A%2520CMyc%25E2%2580%2594Max%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1995%26volume%3D117%26issue%3D11%26spage%3D2998%26epage%3D3007%26doi%3D10.1021%2Fja00116a005" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mijalis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumont, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span> <span> </span><span class="NLM_article-title">A Fully Automated Flow-Based Approach for Accelerated Peptide Synthesis</span>. <i>Nat. Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">464</span>– <span class="NLM_lpage">466</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.2318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fnchembio.2318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=28244989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsVShsrk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=13&amp;publication_year=2017&amp;pages=464-466&amp;issue=5&amp;author=A.+J.+Mijalisauthor=D.+A.+Thomasauthor=M.+D.+Simonauthor=A.+Adamoauthor=R.+Beaumontauthor=K.+F.+Jensenauthor=B.+L.+Pentelute&amp;title=A+Fully+Automated+Flow-Based+Approach+for+Accelerated+Peptide+Synthesis&amp;doi=10.1038%2Fnchembio.2318" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A fully automated flow-based approach for accelerated peptide synthesis</span></div><div class="casAuthors">Mijalis, Alexander J.; Thomas, Dale A., III; Simon, Mark D.; Adamo, Andrea; Beaumont, Ryan; Jensen, Klavs F.; Pentelute, Bradley L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">464-466</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we report a fully automated, flow-based approach to solid-phase polypeptide synthesis, with amide bond formation in 7 s and total synthesis times of 40 s per amino acid residue.  Crude peptide purities and isolated yields were comparable to those for std.-batch solid-phase peptide synthesis.  At full capacity, this approach can yield tens of thousands of individual 30-mer peptides per yr.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRTWEyNbJGibVg90H21EOLACvtfcHk0lie6ot6yXVo-w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXjsVShsrk%253D&amp;md5=c8aa99dfdac88175f414098b2d2baf10</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2318&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2318%26sid%3Dliteratum%253Aachs%26aulast%3DMijalis%26aufirst%3DA.%2BJ.%26aulast%3DThomas%26aufirst%3DD.%2BA.%26aulast%3DSimon%26aufirst%3DM.%2BD.%26aulast%3DAdamo%26aufirst%3DA.%26aulast%3DBeaumont%26aufirst%3DR.%26aulast%3DJensen%26aufirst%3DK.%2BF.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26atitle%3DA%2520Fully%2520Automated%2520Flow-Based%2520Approach%2520for%2520Accelerated%2520Peptide%2520Synthesis%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26issue%3D5%26spage%3D464%26epage%3D466%26doi%3D10.1038%2Fnchembio.2318" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartrampf, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saebi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poskus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gates, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowfer, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antilla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schissel, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quartararo, A. J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Synthesis of Proteins by Automated Flow Chemistry</span>. <i>Science</i>  <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span> (<span class="NLM_issue">6494</span>),  <span class="NLM_fpage">980</span>– <span class="NLM_lpage">987</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.abb2491</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1126%2Fscience.abb2491" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=32467387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVCnur%252FM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=368&amp;publication_year=2020&amp;pages=980-987&amp;issue=6494&amp;author=N.+Hartrampfauthor=A.+Saebiauthor=M.+Poskusauthor=Z.+P.+Gatesauthor=A.+J.+Callahanauthor=A.+E.+Cowferauthor=S.+Hannaauthor=S.+Antillaauthor=C.+K.+Schisselauthor=A.+J.+Quartararo&amp;title=Synthesis+of+Proteins+by+Automated+Flow+Chemistry&amp;doi=10.1126%2Fscience.abb2491" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of proteins by automated flow chemistry</span></div><div class="casAuthors">Hartrampf, N.; Saebi, A.; Poskus, M.; Gates, Z. P.; Callahan, A. J.; Cowfer, A. E.; Hanna, S.; Antilla, S.; Schissel, C. K.; Quartararo, A. J.; Ye, X.; Mijalis, A. J.; Simon, M. D.; Loas, A.; Liu, S.; Jessen, C.; Nielsen, T. E.; Pentelute, B. L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6494</span>),
    <span class="NLM_cas:pages">980-987</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Ribosomes can produce proteins in minutes and are largely constrained to proteinogenic amino acids.  Here, we report highly efficient chem. matched with an automated fast-flow instrument for the direct manufg. of peptide chains up to 164 amino acids long over 327 consecutive reactions.  The machine is rapid: Peptide chain elongation is complete in hours.  We demonstrate the utility of this approach by the chem. synthesis of nine different protein chains that represent enzymes, structural units, and regulatory factors.  After purifn. and folding, the synthetic materials display biophys. and enzymic properties comparable to the biol. expressed proteins.  High-fidelity automated flow chem. is an alternative for producing single-domain proteins without the ribosome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4GKtPvg69AbVg90H21EOLACvtfcHk0lie6ot6yXVo-w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3cXhtVCnur%252FM&amp;md5=c7674247769238f6429c0131cfac3578</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb2491&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb2491%26sid%3Dliteratum%253Aachs%26aulast%3DHartrampf%26aufirst%3DN.%26aulast%3DSaebi%26aufirst%3DA.%26aulast%3DPoskus%26aufirst%3DM.%26aulast%3DGates%26aufirst%3DZ.%2BP.%26aulast%3DCallahan%26aufirst%3DA.%2BJ.%26aulast%3DCowfer%26aufirst%3DA.%2BE.%26aulast%3DHanna%26aufirst%3DS.%26aulast%3DAntilla%26aufirst%3DS.%26aulast%3DSchissel%26aufirst%3DC.%2BK.%26aulast%3DQuartararo%26aufirst%3DA.%2BJ.%26atitle%3DSynthesis%2520of%2520Proteins%2520by%2520Automated%2520Flow%2520Chemistry%26jtitle%3DScience%26date%3D2020%26volume%3D368%26issue%3D6494%26spage%3D980%26epage%3D987%26doi%3D10.1126%2Fscience.abb2491" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palmacci, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plante, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeberger, P. H.</span></span> <span> </span><span class="NLM_article-title">Automated Synthesis of Oligosaccharides</span>. <i>Helv. Chim. Acta</i>  <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">3975</span>, <span class="refDoi">&nbsp;DOI: 10.1002/hlca.200390331</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1002%2Fhlca.200390331" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvFCnsA%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=86&amp;publication_year=2003&amp;pages=3975&amp;author=E.+R.+Palmacciauthor=O.+J.+Planteauthor=M.+C.+Hewittauthor=P.+H.+Seeberger&amp;title=Automated+Synthesis+of+Oligosaccharides&amp;doi=10.1002%2Fhlca.200390331" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Automated synthesis of oligosaccharides</span></div><div class="casAuthors">Palmacci, Emma R.; Plante, Obadiah J.; Hewitt, Michael C.; Seeberger, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Helvetica Chimica Acta</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3975-3990</span>CODEN:
                <span class="NLM_cas:coden">HCACAV</span>;
        ISSN:<span class="NLM_cas:issn">0018-019X</span>.
    
            (<span class="NLM_cas:orgname">Verlag Helvetica Chimica Acta</span>)
        </div><div class="casAbstract">The chem. synthesis of oligosaccharides with an automated solid-phase synthesizer is described.  An octenediol linker served to attach the growing oligosaccharide chain to the solid support, and the desired structures were cleaved from the support via olefin metathesis to afford a pentenyl glycoside.  The automated syntheses of several important carbohydrates, including a penta-rhamnoside, a proteoglycan linkage-region tetrasaccharide, a phytoalexin elicitor dodeca-saccharide, and a branched Leishmania lipo-phospho-glycan tetrasaccharide, were accomplished in higher overall yield and ca. 20 times faster than with soln.-phase methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXd2F5RR7BrLVg90H21EOLACvtfcHk0liaHF4yYQ6G1w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2cXmvFCnsA%253D%253D&amp;md5=8d5180c6e77fb3d5709f3825454b8c1f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fhlca.200390331&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhlca.200390331%26sid%3Dliteratum%253Aachs%26aulast%3DPalmacci%26aufirst%3DE.%2BR.%26aulast%3DPlante%26aufirst%3DO.%2BJ.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DSeeberger%26aufirst%3DP.%2BH.%26atitle%3DAutomated%2520Synthesis%2520of%2520Oligosaccharides%26jtitle%3DHelv.%2520Chim.%2520Acta%26date%3D2003%26volume%3D86%26spage%3D3975%26doi%3D10.1002%2Fhlca.200390331" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schissel, C.</span>; <span class="NLM_string-name">Mohapatra, S.</span>; <span class="NLM_string-name">Wolfe, J.</span>; <span class="NLM_string-name">Fadzen, C.</span>; <span class="NLM_string-name">Bellovoda, K.</span>; <span class="NLM_string-name">Wu, C.-L.</span>; <span class="NLM_string-name">Wood, J.</span>; <span class="NLM_string-name">Malmberg, A.</span>; <span class="NLM_string-name">Loas, A.</span>; <span class="NLM_string-name">Gómez-Bombarelli, R.</span></span>; <span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Interpretable Deep Learning for De Novo Design of Cell-Penetrating Abiotic Polymers</span>. bioRxiv, <span class="NLM_year">2020</span>. <a href="https://doi.org/10.1101/2020.04.10.036566" class="extLink">https://doi.org/10.1101/2020.04.10.036566</a>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1101%2F2020.04.10.036566" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2020&amp;author=C.+Schissel&amp;author=S.+Mohapatra&amp;author=J.+Wolfe&amp;author=C.+Fadzen&amp;author=K.+Bellovoda&amp;author=C.-L.+Wu&amp;author=J.+Wood&amp;author=A.+Malmberg&amp;author=A.+Loas&amp;author=R.+G%C3%B3mez-Bombarelli&amp;title=Interpretable+Deep+Learning+for+De+Novo+Design+of+Cell-Penetrating+Abiotic+Polymers" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1101%2F2020.04.10.036566&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dweblink%26id%3Ddoi%3A10.1101%252F2020.04.10.036566%26sid%3Dliteratum%253Aachs%26aulast%3DSchissel%26aufirst%3DC.%26atitle%3DInterpretable%2520Deep%2520Learning%2520for%2520De%2520Novo%2520Design%2520of%2520Cell-Penetrating%2520Abiotic%2520Polymers%26date%3D2020" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fadzen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holden, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schissel, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span> <span> </span><span class="NLM_article-title">Chimeras of Cell-Penetrating Peptides Demonstrate Synergistic Improvement in Antisense Efficacy</span>. <i>Biochemistry</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">3980</span>– <span class="NLM_lpage">3989</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.biochem.9b00413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.9b00413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1ensrbJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=58&amp;publication_year=2019&amp;pages=3980-3989&amp;issue=38&amp;author=C.+M.+Fadzenauthor=R.+L.+Holdenauthor=J.+M.+Wolfeauthor=Z.+N.+Chooauthor=C.+K.+Schisselauthor=M.+Yaoauthor=G.+J.+Hansonauthor=B.+L.+Pentelute&amp;title=Chimeras+of+Cell-Penetrating+Peptides+Demonstrate+Synergistic+Improvement+in+Antisense+Efficacy&amp;doi=10.1021%2Facs.biochem.9b00413" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Chimeras of Cell-Penetrating Peptides Demonstrate Synergistic Improvement in Antisense Efficacy</span></div><div class="casAuthors">Fadzen, Colin M.; Holden, Rebecca L.; Wolfe, Justin M.; Choo, Zi-Ning; Schissel, Carly K.; Yao, Monica; Hanson, Gunnar J.; Pentelute, Bradley L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">3980-3989</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphorodiamidate morpholino oligonucleotides (PMOs) make up a promising class of therapeutics for genetic disease.  PMOs designed for "exon skipping" must be internalized into cells, reach the nucleus, and act on pre-mRNA to mediate their effects.  One tactic for improving PMO delivery and exon skipping is to covalently conjugate PMOs to cell-penetrating peptides (CPPs).  Here, we report the synthesis of PMOs conjugated to CPP chimeras, constructed by combining multiple CPPs into one sequence.  The chimeric CPPs synergistically improve PMO activity up to 70-fold compared to that of the PMO alone and beyond the expected effects of each component peptide.  By investigating the design space of CPP chimeras, we demonstrate that all components must be covalently attached, that the order of the two sequences matters, and that peptide identity can tune activity.  We identified one chimera (pVEC-Bpep) to investigate in more detail and found that it engages mechanisms of endocytosis different from those of its parent peptides.  We also examd. the extent to which the beneficial effect comes from improved cellular uptake as opposed to the downstream steps required for exon skipping.  Given the complexity of intracellular delivery, we anticipate this work will lead researchers to consider combining mols. with different physicochem. properties to aid in the delivery of biol. cargoes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4ughF4vq80rVg90H21EOLACvtfcHk0liaHF4yYQ6G1w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXhs1ensrbJ&amp;md5=21857f6abc53ad93039e90de4e9db86e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.9b00413&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.9b00413%26sid%3Dliteratum%253Aachs%26aulast%3DFadzen%26aufirst%3DC.%2BM.%26aulast%3DHolden%26aufirst%3DR.%2BL.%26aulast%3DWolfe%26aufirst%3DJ.%2BM.%26aulast%3DChoo%26aufirst%3DZ.%2BN.%26aulast%3DSchissel%26aufirst%3DC.%2BK.%26aulast%3DYao%26aufirst%3DM.%26aulast%3DHanson%26aufirst%3DG.%2BJ.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26atitle%3DChimeras%2520of%2520Cell-Penetrating%2520Peptides%2520Demonstrate%2520Synergistic%2520Improvement%2520in%2520Antisense%2520Efficacy%26jtitle%3DBiochemistry%26date%3D2019%26volume%3D58%26issue%3D38%26spage%3D3980%26epage%3D3989%26doi%3D10.1021%2Facs.biochem.9b00413" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorolla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogdawa, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golden, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewhurst, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruickshank, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, K.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Tumor Penetrating Peptides Inhibiting MYC as a Potent Targeted Therapeutic Strategy for Triple-Negative Breast Cancers</span>. <i>Oncogene</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">140</span>– <span class="NLM_lpage">150</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41388-018-0421-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fs41388-018-0421-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30076412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVCns7%252FI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=38&amp;publication_year=2019&amp;pages=140-150&amp;issue=1&amp;author=E.+Wangauthor=A.+Sorollaauthor=P.+T.+Cunninghamauthor=H.+M.+Bogdawaauthor=S.+Beckauthor=E.+Goldenauthor=R.+E.+Dewhurstauthor=L.+Florezauthor=M.+N.+Cruickshankauthor=K.+Hoffmann&amp;title=Tumor+Penetrating+Peptides+Inhibiting+MYC+as+a+Potent+Targeted+Therapeutic+Strategy+for+Triple-Negative+Breast+Cancers&amp;doi=10.1038%2Fs41388-018-0421-y" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor penetrating peptides inhibiting MYC as a potent targeted therapeutic strategy for triple-negative breast cancers</span></div><div class="casAuthors">Wang, Edina; Sorolla, Anabel; Cunningham, Paula T.; Bogdawa, Heique M.; Beck, Samuel; Golden, Emily; Dewhurst, Robert E.; Florez, Laura; Cruickshank, Mark N.; Hoffmann, Katrin; Hopkins, Richard M.; Kim, Jonghwan; Woo, Andrew J.; Watt, Paul M.; Blancafort, Pilar</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">140-150</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Overexpression of MYC oncogene is highly prevalent in many malignancies such as aggressive triple-neg. breast cancers (TNBCs) and it is assocd. with very poor outcome.  Despite decades of research, attempts to effectively inhibit MYC, particularly with small mols., still remain challenging due to the featureless nature of its protein structure.  Herein, we describe the engineering of the dominant-neg. MYC peptide (OmoMYC) linked to a functional penetrating 'Phylomer' peptide (FPPa) as a therapeutic strategy to inhibit MYC in TNBC.  We found FPPa-OmoMYC to be a potent inducer of apoptosis (with IC50 from 1-2 μM) in TNBC cells with negligible effects in non-tumorigenic cells.  Transcriptome anal. of FPPa-OmoMYC-treated cells indicated that the fusion protein inhibited MYC-dependent networks, inducing dynamic changes in transcriptional, metabolic, and apoptotic processes.  We demonstrated the efficacy of FPPa-OmoMYC in inhibiting breast cancer growth when injected orthotopically in TNBC allografts.  Lastly, we identified strong pharmacol. synergisms between FPPa-OmoMYC and chemotherapeutic agents.  This study highlights a novel therapeutic approach to target highly aggressive and chemoresistant MYC-activated cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqex9cMW5_g8rVg90H21EOLACvtfcHk0liaHF4yYQ6G1w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhsVCns7%252FI&amp;md5=8c94bfea4f2600c86ff70ef37618483b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fs41388-018-0421-y&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41388-018-0421-y%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DE.%26aulast%3DSorolla%26aufirst%3DA.%26aulast%3DCunningham%26aufirst%3DP.%2BT.%26aulast%3DBogdawa%26aufirst%3DH.%2BM.%26aulast%3DBeck%26aufirst%3DS.%26aulast%3DGolden%26aufirst%3DE.%26aulast%3DDewhurst%26aufirst%3DR.%2BE.%26aulast%3DFlorez%26aufirst%3DL.%26aulast%3DCruickshank%26aufirst%3DM.%2BN.%26aulast%3DHoffmann%26aufirst%3DK.%26atitle%3DTumor%2520Penetrating%2520Peptides%2520Inhibiting%2520MYC%2520as%2520a%2520Potent%2520Targeted%2520Therapeutic%2520Strategy%2520for%2520Triple-Negative%2520Breast%2520Cancers%26jtitle%3DOncogene%26date%3D2019%26volume%3D38%26issue%3D1%26spage%3D140%26epage%3D150%26doi%3D10.1038%2Fs41388-018-0421-y" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukazawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuoka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamatsuji, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shigemitsu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morita, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faiola, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durbin, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naomoto, Y.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Myc Effectively Targets KRAS Mutation-Positive Lung Cancer Expressing High Levels of Myc</span>. <i>Anticancer Res.</i>  <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4193</span>– <span class="NLM_lpage">4200</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=21036740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFym" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=30&amp;publication_year=2010&amp;pages=4193-4200&amp;issue=10&amp;author=T.+Fukazawaauthor=Y.+Maedaauthor=J.+Matsuokaauthor=T.+Yamatsujiauthor=K.+Shigemitsuauthor=I.+Moritaauthor=F.+Faiolaauthor=M.+L.+Durbinauthor=L.+Soucekauthor=Y.+Naomoto&amp;title=Inhibition+of+Myc+Effectively+Targets+KRAS+Mutation-Positive+Lung+Cancer+Expressing+High+Levels+of+Myc" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Myc effectively targets KRAS mutation-positive lung cancer expressing high levels of Myc</span></div><div class="casAuthors">Fukazawa, Takuya; Maeda, Yutaka; Matsuoka, Junji; Yamatsuji, Tomoki; Shigemitsu, Kaoru; Morita, Ichiro; Faiola, Francesco; Durbin, Mary L.; Soucek, Laura; Naomoto, Yoshio</div><div class="citationInfo"><span class="NLM_cas:title">Anticancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4193-4200</span>CODEN:
                <span class="NLM_cas:coden">ANTRD4</span>;
        ISSN:<span class="NLM_cas:issn">0250-7005</span>.
    
            (<span class="NLM_cas:orgname">International Institute of Anticancer Research</span>)
        </div><div class="casAbstract">Myc is an oncogenic transcription factor that promotes tumorigenesis.  Recently, a dominant neg. form of Myc (Omomyc) was shown to cause regression of lung tumors in a mouse model of lung cancer caused by KRAS mutation, suggesting that Myc might be a potential therapeutic target to treat the KRAS lung cancer.  However, it is not yer known whether Omomyc can also inhibit the growth of human lung tumors that carry a similar KRAS mutation.  In the present study, we demonstrate that Omomyc induces cell death of KRAS-mutated human lung adenocarcinoma A549 cells in vitro and in vivo.  However, Omomyc does not induce cell death in human lung adenocarcinoma H441 cells that also carry the KRAS mutation.  Interestingly, A549 cells express high levels of Myc, while H441 cells do not.  Co-expression of exogenous Myc with Omomyc in H441 cells induces cell death, indicating that Omomyc requires high levels of Myc to induce cell death in KRAS mutation-pos. lung adenocarcinoma.  Here, we show for the first time that KRAS mutation-pos. lung cancer displaying high levels of Myc could be treated by inhibiting Myc transactivation function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6iGfK0KS3D7Vg90H21EOLACvtfcHk0lipZLLf-yTBUQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXktFym&amp;md5=97ae611107884ef74a4abc6e19573a18</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFukazawa%26aufirst%3DT.%26aulast%3DMaeda%26aufirst%3DY.%26aulast%3DMatsuoka%26aufirst%3DJ.%26aulast%3DYamatsuji%26aufirst%3DT.%26aulast%3DShigemitsu%26aufirst%3DK.%26aulast%3DMorita%26aufirst%3DI.%26aulast%3DFaiola%26aufirst%3DF.%26aulast%3DDurbin%26aufirst%3DM.%2BL.%26aulast%3DSoucek%26aufirst%3DL.%26aulast%3DNaomoto%26aufirst%3DY.%26atitle%3DInhibition%2520of%2520Myc%2520Effectively%2520Targets%2520KRAS%2520Mutation-Positive%2520Lung%2520Cancer%2520Expressing%2520High%2520Levels%2520of%2520Myc%26jtitle%3DAnticancer%2520Res.%26date%3D2010%26volume%3D30%26issue%3D10%26spage%3D4193%26epage%3D4200" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spiegel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Modulation of Ras Signaling</span>. <i>Nat. Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">613</span>– <span class="NLM_lpage">622</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.1560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fnchembio.1560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=24929527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslGisbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2014&amp;pages=613-622&amp;issue=8&amp;author=J.+Spiegelauthor=P.+M.+Crommauthor=G.+Zimmermannauthor=T.+N.+Grossmannauthor=H.+Waldmann&amp;title=Small-Molecule+Modulation+of+Ras+Signaling&amp;doi=10.1038%2Fnchembio.1560" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule modulation of Ras signaling</span></div><div class="casAuthors">Spiegel, Jochen; Cromm, Philipp M.; Zimmermann, Gunther; Grossmann, Tom N.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">613-622</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite intense efforts in pharmaceutical industry and academia, a therapeutic grip on oncogenic Ras proteins has remained elusive.  Mutated Ras is assocd. with ∼20-30% of all human cancers often not responsive to established therapies.  In particular, K-Ras, the most frequently mutated Ras isoform, is considered one of the most important but 'undruggable' targets in cancer research.  Recently, new cavities on Ras for small-mol. ligands were identified, and selective direct targeting of mutated K-RasG12C has become possible for what is to our knowledge the first time.  In addn., impairment of Ras spatial organization, in particular via targeting the prenyl-binding Ras chaperone PDEδ, has opened a fresh perspective in anticancer research.  These recent advances fuel hopes for the development of new drugs targeting Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTcp4dzeVKK7Vg90H21EOLACvtfcHk0lipZLLf-yTBUQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXpslGisbw%253D&amp;md5=f8d84f054e1b392cf6bfe55d5e235c28</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1560&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1560%26sid%3Dliteratum%253Aachs%26aulast%3DSpiegel%26aufirst%3DJ.%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DZimmermann%26aufirst%3DG.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DSmall-Molecule%2520Modulation%2520of%2520Ras%2520Signaling%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26issue%3D8%26spage%3D613%26epage%3D622%26doi%3D10.1038%2Fnchembio.1560" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esquela-Kerscher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelnar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovcharenko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shingara, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">The Let-7 MicroRNA Represses Cell Proliferation Pathways in Human Cells</span>. <i>Cancer Res.</i>  <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">7713</span>– <span class="NLM_lpage">7722</span>, <span class="refDoi">&nbsp;DOI: 10.1158/0008-5472.CAN-07-1083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1158%2F0008-5472.CAN-07-1083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=17699775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptVGlsbo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=67&amp;publication_year=2007&amp;pages=7713-7722&amp;issue=16&amp;author=C.+D.+Johnsonauthor=A.+Esquela-Kerscherauthor=G.+Stefaniauthor=M.+Byromauthor=K.+Kelnarauthor=D.+Ovcharenkoauthor=M.+Wilsonauthor=X.+Wangauthor=J.+Sheltonauthor=J.+Shingara&amp;title=The+Let-7+MicroRNA+Represses+Cell+Proliferation+Pathways+in+Human+Cells&amp;doi=10.1158%2F0008-5472.CAN-07-1083" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">The let-7 MicroRNA Represses Cell Proliferation Pathways in Human Cells</span></div><div class="casAuthors">Johnson, Charles D.; Esquela-Kerscher, Aurora; Stefani, Giovanni; Byrom, Mike; Kelnar, Kevin; Ovcharenko, Dmitriy; Wilson, Mike; Wang, Xiaowei; Shelton, Jeffrey; Shingara, Jaclyn; Chin, Lena; Brown, David; Slack, Frank J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7713-7722</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">MicroRNAs play important roles in animal development, cell differentiation, and metab. and have been implicated in human cancer.  The let-7 microRNA controls the timing of cell cycle exit and terminal differentiation in Caenorhabditis elegans and is poorly expressed or deleted in human lung tumors.  Here, we show that let-7 is highly expressed in normal lung tissue, and that inhibiting let-7 function leads to increased cell division in A549 lung cancer cells.  Overexpression of let-7 in cancer cell lines alters cell cycle progression and reduces cell division, providing evidence that let-7 functions as a tumor suppressor in lung cells. let-7 was previously shown to regulate the expression of the RAS lung cancer oncogenes, and our work now shows that multiple genes involved in cell cycle and cell division functions are also directly or indirectly repressed by let-7.  This work reveals the let-7 microRNA to be a master regulator of cell proliferation pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3-hl_60Ut_LVg90H21EOLACvtfcHk0lipZLLf-yTBUQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXptVGlsbo%253D&amp;md5=102525e3b46833e6592d03efabb9321d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-1083&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-1083%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%2BD.%26aulast%3DEsquela-Kerscher%26aufirst%3DA.%26aulast%3DStefani%26aufirst%3DG.%26aulast%3DByrom%26aufirst%3DM.%26aulast%3DKelnar%26aufirst%3DK.%26aulast%3DOvcharenko%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DShelton%26aufirst%3DJ.%26aulast%3DShingara%26aufirst%3DJ.%26atitle%3DThe%2520Let-7%2520MicroRNA%2520Represses%2520Cell%2520Proliferation%2520Pathways%2520in%2520Human%2520Cells%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26issue%3D16%26spage%3D7713%26epage%3D7722%26doi%3D10.1158%2F0008-5472.CAN-07-1083" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebhardt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koelmel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ade, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuper, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Eyss, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letschert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Artista, L.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">OmoMYC Blunts Promoter Invasion by Oncogenic MYC to Inhibit Gene Expression Characteristic of MYC-Dependent Tumors</span>. <i>Oncogene</i>  <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">1911</span>– <span class="NLM_lpage">1924</span>, <span class="refDoi">&nbsp;DOI: 10.1038/onc.2016.354</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fonc.2016.354" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=27748763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1ynsbfF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=2017&amp;pages=1911-1924&amp;issue=14&amp;author=L.+A.+Jungauthor=A.+Gebhardtauthor=W.+Koelmelauthor=C.+P.+Adeauthor=S.+Walzauthor=J.+Kuperauthor=B.+Von+Eyssauthor=S.+Letschertauthor=C.+Redelauthor=L.+D%E2%80%99Artista&amp;title=OmoMYC+Blunts+Promoter+Invasion+by+Oncogenic+MYC+to+Inhibit+Gene+Expression+Characteristic+of+MYC-Dependent+Tumors&amp;doi=10.1038%2Fonc.2016.354" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">OmoMYC blunts promoter invasion by oncogenic MYC to inhibit gene expression characteristic of MYC-dependent tumors</span></div><div class="casAuthors">Jung, L. A.; Gebhardt, A.; Koelmel, W.; Ade, C. P.; Walz, S.; Kuper, J.; von Eyss, B.; Letschert, S.; Redel, C.; d'Artista, L.; Biankin, A.; Zender, L.; Sauer, M.; Wolf, E.; Evan, G.; Kisker, C.; Eilers, M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1911-1924</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MYC genes have both essential roles during normal development and exert oncogenic functions during tumorigenesis.  Expression of a dominant-neg. allele of MYC, termed OmoMYC, can induce rapid tumor regression in mouse models with little toxicity for normal tissues.  How OmoMYC discriminates between physiol. and oncogenic functions of MYC is unclear.  We have solved the crystal structure of OmoMYC and show that it forms a stable homodimer and as such recognizes DNA in the same manner as the MYC/MAX heterodimer.  OmoMYC attenuates both MYC-dependent activation and repression by competing with MYC/MAX for binding to chromatin, effectively lowering MYC/MAX occupancy at its cognate binding sites.  OmoMYC causes the largest decreases in promoter occupancy and changes in expression on genes that are invaded by oncogenic MYC levels.  A signature of OmoMYC-regulated genes defines subgroups with high MYC levels in multiple tumor entities and identifies novel targets for the eradication of MYC-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpv_ma5jMA0LVg90H21EOLACvtfcHk0ljXa1mWYY2vDw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28Xhs1ynsbfF&amp;md5=bb7307180be76887bcc94c2c6ae8fa2b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.354&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.354%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DL.%2BA.%26aulast%3DGebhardt%26aufirst%3DA.%26aulast%3DKoelmel%26aufirst%3DW.%26aulast%3DAde%26aufirst%3DC.%2BP.%26aulast%3DWalz%26aufirst%3DS.%26aulast%3DKuper%26aufirst%3DJ.%26aulast%3DVon%2BEyss%26aufirst%3DB.%26aulast%3DLetschert%26aufirst%3DS.%26aulast%3DRedel%26aufirst%3DC.%26aulast%3DD%25E2%2580%2599Artista%26aufirst%3DL.%26atitle%3DOmoMYC%2520Blunts%2520Promoter%2520Invasion%2520by%2520Oncogenic%2520MYC%2520to%2520Inhibit%2520Gene%2520Expression%2520Characteristic%2520of%2520MYC-Dependent%2520Tumors%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26issue%3D14%26spage%3D1911%26epage%3D1924%26doi%3D10.1038%2Fonc.2016.354" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montagne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaudoin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavoie, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinck, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavigne, P.</span></span> <span> </span><span class="NLM_article-title">The Max B-HLH-LZ Can Transduce into Cells and Inhibit c-Myc Transcriptional Activities</span>. <i>PLoS One</i>  <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">10</span>, <span class="refDoi">&nbsp;DOI: 10.1371/journal.pone.0032172</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1371%2Fjournal.pone.0032172" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2012&amp;pages=2-10&amp;issue=2&amp;author=M.+Montagneauthor=N.+Beaudoinauthor=D.+Fortinauthor=C.+L.+Lavoieauthor=R.+Klinckauthor=P.+Lavigne&amp;title=The+Max+B-HLH-LZ+Can+Transduce+into+Cells+and+Inhibit+c-Myc+Transcriptional+Activities&amp;doi=10.1371%2Fjournal.pone.0032172" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0032172&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0032172%26sid%3Dliteratum%253Aachs%26aulast%3DMontagne%26aufirst%3DM.%26aulast%3DBeaudoin%26aufirst%3DN.%26aulast%3DFortin%26aufirst%3DD.%26aulast%3DLavoie%26aufirst%3DC.%2BL.%26aulast%3DKlinck%26aufirst%3DR.%26aulast%3DLavigne%26aufirst%3DP.%26atitle%3DThe%2520Max%2520B-HLH-LZ%2520Can%2520Transduce%2520into%2520Cells%2520and%2520Inhibit%2520c-Myc%2520Transcriptional%2520Activities%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26issue%3D2%26spage%3D2%26epage%3D10%26doi%3D10.1371%2Fjournal.pone.0032172" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span> <i>Peptomyc</i>. <a href="https://www.biospace.com/article/releases/peptomyc-s-omomyc-based-therapy-omo-103-has-obtained-approval-for-first-in-human-phase-i-ii-trial-to-assess-the-efficacy-and-safety-of-a-novel-myc-inhibitor/" class="extLink">https://www.biospace.com/article/releases/peptomyc-s-omomyc-based-therapy-omo-103-has-obtained-approval-for-first-in-human-phase-i-ii-trial-to-assess-the-efficacy-and-safety-of-a-novel-myc-inhibitor/</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=+Peptomyc.+https%3A%2F%2Fwww.biospace.com%2Farticle%2Freleases%2Fpeptomyc-s-omomyc-based-therapy-omo-103-has-obtained-approval-for-first-in-human-phase-i-ii-trial-to-assess-the-efficacy-and-safety-of-a-novel-myc-inhibitor%2F." target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DPeptomyc" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herce, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, A. F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasper, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonhardt, H.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Cell-Permeable Nanobodies for Targeted Immunolabelling and Antigen Manipulation in Living Cells</span>. <i>Nat. Chem.</i>  <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">762</span>– <span class="NLM_lpage">771</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.2811</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fnchem.2811" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=28754949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFOjsLrM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2017&amp;pages=762-771&amp;issue=8&amp;author=H.+D.+Herceauthor=D.+Schumacherauthor=A.+F.+L.+Schneiderauthor=A.+K.+Ludwigauthor=F.+A.+Mannauthor=M.+Filliesauthor=M.+A.+Kasperauthor=S.+Reinkeauthor=E.+Krauseauthor=H.+Leonhardt&amp;title=Cell-Permeable+Nanobodies+for+Targeted+Immunolabelling+and+Antigen+Manipulation+in+Living+Cells&amp;doi=10.1038%2Fnchem.2811" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-permeable nanobodies for targeted immunolabelling and antigen manipulation in living cells</span></div><div class="casAuthors">Herce, Henry D.; Schumacher, Dominik; Schneider, Anselm F. L.; Ludwig, Anne K.; Mann, Florian A.; Fillies, Marion; Kasper, Marc-Andre; Reinke, Stefan; Krause, Eberhard; Leonhardt, Heinrich; Cardoso, M. Cristina; Hackenberger, Christian P. R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">762-771</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Functional antibody delivery in living cells would enable the labeling and manipulation of intracellular antigens, which constitutes a long-thought goal in cell biol. and medicine.  Here we present a modular strategy to create functional cell-permeable nanobodies capable of targeted labeling and manipulation of intracellular antigens in living cells.  The cell-permeable nanobodies are formed by the site-specific attachment of intracellularly stable (or cleavable) cyclic arginine-rich cell-penetrating peptides to camelid-derived single-chain VHH antibody fragments.  We used this strategy for the non-endocytic delivery of two recombinant nanobodies into living cells, which enabled the relocalization of the polymerase clamp PCNA (proliferating cell nuclear antigen) and tumor suppressor p53 to the nucleolus, and thereby allowed the detection of protein-protein interactions that involve these two proteins in living cells.  Furthermore, cell-permeable nanobodies permitted the co-transport of therapeutically relevant proteins, such as Mecp2, into the cells.  This technol. constitutes a major step in the labeling, delivery and targeted manipulation of intracellular antigens.  Ultimately, this approach opens the door towards immunostaining in living cells and the expansion of immunotherapies to intracellular antigen targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLMQ2tz9B_kbVg90H21EOLACvtfcHk0ljXa1mWYY2vDw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhtFOjsLrM&amp;md5=681c6fcd98272ea6c93d0eaa586c1a1d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2811&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2811%26sid%3Dliteratum%253Aachs%26aulast%3DHerce%26aufirst%3DH.%2BD.%26aulast%3DSchumacher%26aufirst%3DD.%26aulast%3DSchneider%26aufirst%3DA.%2BF.%2BL.%26aulast%3DLudwig%26aufirst%3DA.%2BK.%26aulast%3DMann%26aufirst%3DF.%2BA.%26aulast%3DFillies%26aufirst%3DM.%26aulast%3DKasper%26aufirst%3DM.%2BA.%26aulast%3DReinke%26aufirst%3DS.%26aulast%3DKrause%26aufirst%3DE.%26aulast%3DLeonhardt%26aufirst%3DH.%26atitle%3DCell-Permeable%2520Nanobodies%2520for%2520Targeted%2520Immunolabelling%2520and%2520Antigen%2520Manipulation%2520in%2520Living%2520Cells%26jtitle%3DNat.%2520Chem.%26date%3D2017%26volume%3D9%26issue%3D8%26spage%3D762%26epage%3D771%26doi%3D10.1038%2Fnchem.2811" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gersbach, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbas, C. F.</span></span> <span> </span><span class="NLM_article-title">Synthetic Zinc Finger Proteins: The Advent of Targeted Gene Regulation and Genome Modification Technologies</span>. <i>Acc. Chem. Res.</i>  <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2309</span>– <span class="NLM_lpage">2318</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar500039w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar500039w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptVWrur8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=2014&amp;pages=2309-2318&amp;issue=8&amp;author=C.+A.+Gersbachauthor=T.+Gajauthor=C.+F.+Barbas&amp;title=Synthetic+Zinc+Finger+Proteins%3A+The+Advent+of+Targeted+Gene+Regulation+and+Genome+Modification+Technologies&amp;doi=10.1021%2Far500039w" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic zinc finger proteins: The advent of targeted gene regulation and genome modification technologies</span></div><div class="casAuthors">Gersbach, Charles A.; Gaj, Thomas; Barbas, Carlos F., III</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2309-2318</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The understanding of gene regulation and the structure and function of the human genome increased dramatically at the end of the 20th century.  Yet the technologies for manipulating the genome have been slower to develop.  For instance, the field of gene therapy has been focused on correcting genetic diseases and augmenting tissue repair for more than 40 years.  However, with the exception of a few very low efficiency approaches, conventional genetic engineering methods have only been able to add auxiliary genes to cells.  This has been a substantial obstacle to the clin. success of gene therapies and has also led to severe unintended consequences in several cases.  Therefore, technologies that facilitate the precise modification of cellular genomes have diverse and significant implications in many facets of research and are essential for translating the products of the Genomic Revolution into tangible benefits for medicine and biotechnol.  To address this need, in the 1990s, we embarked on a mission to develop technologies for engineering protein-DNA interactions with the aim of creating custom tools capable of targeting any DNA sequence.  Our goal has been to allow researchers to reach into genomes to specifically regulate, knock out, or replace any gene.  To realize these goals, we initially focused on understanding and manipulating zinc finger proteins.  In particular, we sought to create a simple and straightforward method that enables unspecialized labs. to engineer custom DNA-modifying proteins using only defined modular components, a web-based utility, and std. recombinant DNA technol.  Two significant challenges we faced were (i) the development of zinc finger domains that target sequences not recognized by naturally occurring zinc finger proteins and (ii) detg. how individual zinc finger domains could be tethered together as polydactyl proteins to recognize unique locations within complex genomes.  We and others have since used this modular assembly method to engineer artificial proteins and enzymes that activate, repress, or create defined changes to user-specified genes in human cells, plants, and other organisms.  We have also engineered novel methods for externally controlling protein activity and delivery, as well as developed new strategies for the directed evolution of protein and enzyme function.  This Account summarizes our work in these areas and highlights independent studies that have successfully used the modular assembly approach to create proteins with novel function.  We also discuss emerging alternative methods for genomic targeting, including transcription activator-like effectors (TALEs) and CRISPR/Cas systems, and how they complement the synthetic zinc finger protein technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohe_yNDoa16rVg90H21EOLACvtfcHk0ljbQb6zffX_3Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXptVWrur8%253D&amp;md5=41677a68edbd08d43d57d2a1a842832d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Far500039w&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far500039w%26sid%3Dliteratum%253Aachs%26aulast%3DGersbach%26aufirst%3DC.%2BA.%26aulast%3DGaj%26aufirst%3DT.%26aulast%3DBarbas%26aufirst%3DC.%2BF.%26atitle%3DSynthetic%2520Zinc%2520Finger%2520Proteins%253A%2520The%2520Advent%2520of%2520Targeted%2520Gene%2520Regulation%2520and%2520Genome%2520Modification%2520Technologies%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2014%26volume%3D47%26issue%3D8%26spage%3D2309%26epage%3D2318%26doi%3D10.1021%2Far500039w" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eiríksdóttir, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konate, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langel, Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Divita, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshayes, S.</span></span> <span> </span><span class="NLM_article-title">Secondary Structure of Cell-Penetrating Peptides Controls Membrane Interaction and Insertion</span>. <i>Biochim. Biophys. Acta, Biomembr.</i>  <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1798</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1128</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bbamem.2010.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.bbamem.2010.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=20214875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsFOjtL0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1798&amp;publication_year=2010&amp;pages=1119-1128&amp;issue=6&amp;author=E.+Eir%C3%ADksd%C3%B3ttirauthor=K.+Konateauthor=%C3%9C.+Langelauthor=G.+Divitaauthor=S.+Deshayes&amp;title=Secondary+Structure+of+Cell-Penetrating+Peptides+Controls+Membrane+Interaction+and+Insertion&amp;doi=10.1016%2Fj.bbamem.2010.03.005" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Secondary structure of cell-penetrating peptides controls membrane interaction and insertion</span></div><div class="casAuthors">Eiriksdottir, Emelia; Konate, Karidia; Langel, Uelo; Divita, Gilles; Deshayes, Sebastien</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1798</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1119-1128</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The clin. use of efficient therapeutic agents is often limited by the poor permeability of the biol. membranes.  To enhance their cell delivery, short amphipathic peptides called cell-penetrating peptides (CPPs) have been intensively developed for the last two decades.  CPPs are based either on protein transduction domains, model peptide or chimeric constructs and have been used to deliver cargoes into cells through either covalent or non-covalent strategies.  Although several parameters are simultaneously involved in their internalization mechanism, recent focuses on CPPs suggested that structural properties and interactions with membrane phospholipids could play a major role in the cellular uptake mechanism.  In the present work, we report a comparative anal. of the structural plasticity of 10 well-known CPPs as well as their ability to interact with phospholipid membranes.  We propose a new classification of CPPs based on their structural properties, affinity for phospholipids and internalization pathways already reported in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDSOrAfLyyVbVg90H21EOLACvtfcHk0ljbQb6zffX_3Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXlsFOjtL0%253D&amp;md5=aad3a782269b4881ea27decfb1182402</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2010.03.005&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2010.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DEir%25C3%25ADksd%25C3%25B3ttir%26aufirst%3DE.%26aulast%3DKonate%26aufirst%3DK.%26aulast%3DLangel%26aufirst%3D%25C3%259C.%26aulast%3DDivita%26aufirst%3DG.%26aulast%3DDeshayes%26aufirst%3DS.%26atitle%3DSecondary%2520Structure%2520of%2520Cell-Penetrating%2520Peptides%2520Controls%2520Membrane%2520Interaction%2520and%2520Insertion%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Biomembr.%26date%3D2010%26volume%3D1798%26issue%3D6%26spage%3D1119%26epage%3D1128%26doi%3D10.1016%2Fj.bbamem.2010.03.005" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirk, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbas, C. F.</span></span> <span> </span><span class="NLM_article-title">Targeted Gene Knockout by Direct Delivery of Zinc-Finger Nuclease Proteins</span>. <i>Nat. Methods</i>  <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">807</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nmeth.2030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fnmeth.2030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=22751204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsVWhtr4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2012&amp;pages=805-807&amp;issue=8&amp;author=T.+Gajauthor=J.+Guoauthor=Y.+Katoauthor=S.+J.+Sirkauthor=C.+F.+Barbas&amp;title=Targeted+Gene+Knockout+by+Direct+Delivery+of+Zinc-Finger+Nuclease+Proteins&amp;doi=10.1038%2Fnmeth.2030" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted gene knockout by direct delivery of zinc-finger nuclease proteins</span></div><div class="casAuthors">Gaj, Thomas; Guo, Jing; Kato, Yoshio; Sirk, Shannon J.; Barbas, Carlos F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">805-807</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Zinc-finger nucleases (ZFNs) are versatile reagents that have redefined genome engineering.  Realizing the full potential of this technol. requires the development of safe and effective methods for delivering ZFNs into cells.  We demonstrate the intrinsic cell-penetrating capabilities of the std. ZFN architecture and show that direct delivery of ZFNs as proteins leads to efficient endogenous gene disruption in various mammalian cell types with minimal off-target effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx85YChQzVbbVg90H21EOLACvtfcHk0ljbQb6zffX_3Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XpsVWhtr4%253D&amp;md5=8a0583c307ca9b5584eddfc7c16ec32c</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2030&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2030%26sid%3Dliteratum%253Aachs%26aulast%3DGaj%26aufirst%3DT.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DKato%26aufirst%3DY.%26aulast%3DSirk%26aufirst%3DS.%2BJ.%26aulast%3DBarbas%26aufirst%3DC.%2BF.%26atitle%3DTargeted%2520Gene%2520Knockout%2520by%2520Direct%2520Delivery%2520of%2520Zinc-Finger%2520Nuclease%2520Proteins%26jtitle%3DNat.%2520Methods%26date%3D2012%26volume%3D9%26issue%3D8%26spage%3D805%26epage%3D807%26doi%3D10.1038%2Fnmeth.2030" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirk, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbas, C. F.</span></span> <span> </span><span class="NLM_article-title">Protein Delivery Using Cys2-His2 Zinc-Finger Domains</span>. <i>ACS Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1662</span>– <span class="NLM_lpage">1667</span>, <span class="refDoi">&nbsp;DOI: 10.1021/cb500282g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500282g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvVWnu7s%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2014&amp;pages=1662-1667&amp;issue=8&amp;author=T.+Gajauthor=J.+Liuauthor=K.+E.+Andersonauthor=S.+J.+Sirkauthor=C.+F.+Barbas&amp;title=Protein+Delivery+Using+Cys2-His2+Zinc-Finger+Domains&amp;doi=10.1021%2Fcb500282g" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Delivery Using Cys2-His2 Zinc-Finger Domains</span></div><div class="casAuthors">Gaj, Thomas; Liu, Jia; Anderson, Kimberly E.; Sirk, Shannon J.; Barbas, Carlos F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1662-1667</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of new methods for delivering proteins into cells is a central challenge for advancing both basic research and therapeutic applications.  We previously reported that zinc-finger nuclease proteins are intrinsically cell-permeable due to the cell-penetrating activity of the Cys2-His2 zinc-finger domain.  Here, we demonstrate that genetically fused zinc-finger motifs can transport proteins and enzymes into a wide range of primary and transformed mammalian cell types.  We show that zinc-finger domains mediate protein uptake at efficiencies that exceed conventional protein transduction systems and do so without compromising enzyme activity.  In addn., we demonstrate that zinc-finger proteins enter cells primarily through macropinocytosis and facilitate high levels of cytosolic delivery.  These findings establish zinc-finger proteins as not only useful tools for targeted genome engineering but also effective reagents for protein delivery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMizY5RFUtI7Vg90H21EOLACvtfcHk0lgYDzw-YRGVsw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXpvVWnu7s%253D&amp;md5=64c4274c4ed4f1924e8cc0db0bf35393</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fcb500282g&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500282g%26sid%3Dliteratum%253Aachs%26aulast%3DGaj%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DAnderson%26aufirst%3DK.%2BE.%26aulast%3DSirk%26aufirst%3DS.%2BJ.%26aulast%3DBarbas%26aufirst%3DC.%2BF.%26atitle%3DProtein%2520Delivery%2520Using%2520Cys2-His2%2520Zinc-Finger%2520Domains%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26issue%3D8%26spage%3D1662%26epage%3D1667%26doi%3D10.1021%2Fcb500282g" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knox, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piszkiewicz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">Cytosolic Delivery of Argininosuccinate Synthetase Using a Cell- Permeant Miniature Protein</span>. <i>ACS Cent. Sci.</i>  <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">641</span>– <span class="NLM_lpage">649</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscentsci.0c01603</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.0c01603" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3MXms1yitbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2021&amp;pages=641-649&amp;author=S.+L.+Knoxauthor=R.+Wissnerauthor=S.+Piszkiewiczauthor=A.+Schepartz&amp;title=Cytosolic+Delivery+of+Argininosuccinate+Synthetase+Using+a+Cell-+Permeant+Miniature+Protein&amp;doi=10.1021%2Facscentsci.0c01603" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Cytosolic Delivery of Argininosuccinate Synthetase Using a Cell-Permeant Miniature Protein</span></div><div class="casAuthors">Knox, Susan L.; Wissner, Rebecca; Piszkiewicz, Samantha; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">641-649</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Citrullinemia type I (CTLN-I) results from the absence or deficiency of argininosuccinate synthetase (AS), a 46 kDa enzyme that acts in the cytosol of hepatocytes to convert aspartic acid and citrulline into argininosuccinic acid.  AS is an essential component of the urea cycle, and its absence or deficiency results in the harmful accumulation of ammonia in blood and cerebrospinal fluid.  No disease-modifying treatment of CTLN-I exists.  Here we report that the cell-permeant miniature protein (CPMP) ZF5.3 (ZF) can deliver AS to the cytosol of cells in culture and the livers of healthy mice.  The fusion protein ZF-AS is catalytically active in vitro, stabilized in plasma, and traffics successfully to the cytosol of cultured Saos-2 and SK-HEP-1 cells, achieving cytosolic concns. greater than 100 nM.  This value is 3-10-fold higher than the concn. of endogenous AS (11 ± 1 to 44 ± 5 nM).  When injected into healthy C57BL/6 mice, ZF-AS reaches the mouse liver to establish concns. almost 200 nM above baseline.  These studies demonstrate that ZF5.3 can deliver a complex enzyme to the cytosol at therapeutically relevant concns. and support its application as an improved delivery vehicle for therapeutic proteins that function in the cytosol, including enzyme replacement therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFoQSGSuArBrVg90H21EOLACvtfcHk0lgYDzw-YRGVsw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXms1yitbw%253D&amp;md5=d734983ad5a5b5a2ed9ba86ff07acaf8</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.0c01603&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.0c01603%26sid%3Dliteratum%253Aachs%26aulast%3DKnox%26aufirst%3DS.%2BL.%26aulast%3DWissner%26aufirst%3DR.%26aulast%3DPiszkiewicz%26aufirst%3DS.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DCytosolic%2520Delivery%2520of%2520Argininosuccinate%2520Synthetase%2520Using%2520a%2520Cell-%2520Permeant%2520Miniature%2520Protein%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2021%26volume%3D7%26spage%3D641%26epage%3D649%26doi%3D10.1021%2Facscentsci.0c01603" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Libetti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiaramonte, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minuzzo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gnesutta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messina, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolfini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantovani, R.</span></span> <span> </span><span class="NLM_article-title">NF-YA Enters Cells through Cell Penetrating Peptides</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1866</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">440</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bbamcr.2018.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.bbamcr.2018.10.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30296497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSjurjP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1866&amp;publication_year=2019&amp;pages=430-440&amp;issue=3&amp;author=D.+Libettiauthor=A.+Bernardiniauthor=M.+L.+Chiaramonteauthor=M.+Minuzzoauthor=N.+Gnesuttaauthor=G.+Messinaauthor=D.+Dolfiniauthor=R.+Mantovani&amp;title=NF-YA+Enters+Cells+through+Cell+Penetrating+Peptides&amp;doi=10.1016%2Fj.bbamcr.2018.10.004" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">NF-YA enters cells through cell penetrating peptides</span></div><div class="casAuthors">Libetti, Debora; Bernardini, Andrea; Chiaramonte, Maria Luisa; Minuzzo, Mario; Gnesutta, Nerina; Messina, Graziella; Dolfini, Diletta; Mantovani, Roberto</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1866</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">430-440</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cell Penetrating Peptides -CPPs- are short aminoacidic stretches present in proteins that have the ability to translocate the plasma membrane and facilitate delivery of various mols.  They are usually rich in basic residues, and organized as alpha helixes.  NF-Y is a transcription factor heterotrimer formed by two Histone Fold Domain -HFD- subunits and the sequence-specific NF-YA.  NF-YA possesses two α-helixes rich in basic residues.  We show that it efficiently enters cells at nanomolar concns. in the absence of carrier peptides.  Mutagenesis identified at least two sep. CPPs in the A1 and A2, which overlap with previously identified nuclear localization signals (NLS).  The half-life of the transduced protein is short in human cancer cells, longer in mouse C2C12 myoblasts.  The internalized NF-YA is capable of trimerization with the HFD subunits and binding to the target CCAAT box.  Functionality is further suggested by protein transfection in C2C12 cells, leading to inhibition of differentiation to myotubes.  In conclusion, NF-YA contains CPPs, hinting at novel -and unexpected- properties of this subunit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCcO49T7KN8rVg90H21EOLACvtfcHk0lgYDzw-YRGVsw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXitVSjurjP&amp;md5=4b2656757bd627b3885ed0df7506adeb</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2018.10.004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2018.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DLibetti%26aufirst%3DD.%26aulast%3DBernardini%26aufirst%3DA.%26aulast%3DChiaramonte%26aufirst%3DM.%2BL.%26aulast%3DMinuzzo%26aufirst%3DM.%26aulast%3DGnesutta%26aufirst%3DN.%26aulast%3DMessina%26aufirst%3DG.%26aulast%3DDolfini%26aufirst%3DD.%26aulast%3DMantovani%26aufirst%3DR.%26atitle%3DNF-YA%2520Enters%2520Cells%2520through%2520Cell%2520Penetrating%2520Peptides%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2019%26volume%3D1866%26issue%3D3%26spage%3D430%26epage%3D440%26doi%3D10.1016%2Fj.bbamcr.2018.10.004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Y.</span></span> <span> </span><span class="NLM_article-title">The Cell-Penetrating FOXM1 N-Terminus (M1–138) Demonstrates Potent Inhibitory Effects on Cancer Cells by Targeting FOXM1 and FOXM1-Interacting Factor SMAD3</span>. <i>Theranostics</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2882</span>– <span class="NLM_lpage">2896</span>, <span class="refDoi">&nbsp;DOI: 10.7150/thno.32693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.7150%2Fthno.32693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=31244930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvF2htLnL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2019&amp;pages=2882-2896&amp;issue=10&amp;author=Z.+Zhangauthor=H.+Buauthor=J.+Yuauthor=Y.+Chenauthor=C.+Peiauthor=L.+Yuauthor=X.+Huangauthor=G.+Tanauthor=Y.+Tan&amp;title=The+Cell-Penetrating+FOXM1+N-Terminus+%28M1%E2%80%93138%29+Demonstrates+Potent+Inhibitory+Effects+on+Cancer+Cells+by+Targeting+FOXM1+and+FOXM1-Interacting+Factor+SMAD3&amp;doi=10.7150%2Fthno.32693" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The cell-penetrating FOXM1 N-terminus (M1-138) demonstrates potent inhibitory effects on cancer cells by targeting FOXM1 and FOXM1-interacting factor SMAD3</span></div><div class="casAuthors">Zhang, Zhenwang; Bu, Huitong; Yu, Jingwei; Chen, Yan; Pei, Chaozhu; Yu, Li; Huang, Xiaoqin; Tan, Guixiang; Tan, Yongjun</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2882-2896</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">Transcription factor FOXM1 is involved in stimulating cell proliferation, enhancing DNA damage repair, promoting metastasis of cancer cells, and the inhibition of FOXM1 has been shown to prevent the initiation and progression of multiple cancers and FOXM1 is considered to be an effective target for tumor therapeutic drug development.  The N-terminus of FOXM1 has been found to prevent transcriptional activities of FOXM1 and to mediate the interaction between FOXM1 and SMAD3.  A recombinant FOXM1 N-terminal domain (1-138aa) fused with a nine arginine cell-penetrating peptide is produced with an E. coli expression system and named as M1-138.  The effects of M1-138 on the proliferation, migration, and tumorigenic ability of cancer cells are analyzed in vitro with cell counting, transwell assays, and colony formation assays.  Electrophoretic mobility shift assays (EMSAs) and Luciferase activity assays are used to test the DNA binding ability and transcriptional activity of transcription factors.  The levels of mRNAs and proteins are measured by quant.-PCR, Western blotting or Immunohistochem.  The interactions among proteins are analyzed with Pull-down and Co-immunopptn. (Co-IP) assays.  The nude mouse engrafted tumor models are used to test the inhibitory effects of M1-138 in vivo.  M1-138 diminishes the proliferation and migration abilities of cancer cells through binding to FOXM1 and FOXM1-interacting factor SMAD3, and consequently attenuating FOXM1 transcriptional activities from both direct and indirect FOXM1-promoter binding mechanisms and interfering with the interaction between FOXM1 and SMAD3.  Treatment of M1-138 prevents tumorigenicity of cancer cells and inhibits tumor growth in nude mouse xenograft models with no obvious signs of toxicity.  M1-138 is a promising drug candidate for the development of anti-cancer therapeutics targeting FOXM1 and SMAD3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtiXontFCdrVg90H21EOLACvtfcHk0liJxGpcV1z45w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXhvF2htLnL&amp;md5=bdb0f9b36854b622c6a0e41d12684499</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.7150%2Fthno.32693&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.32693%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DBu%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DPei%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DTan%26aufirst%3DG.%26aulast%3DTan%26aufirst%3DY.%26atitle%3DThe%2520Cell-Penetrating%2520FOXM1%2520N-Terminus%2520%2528M1%25E2%2580%2593138%2529%2520Demonstrates%2520Potent%2520Inhibitory%2520Effects%2520on%2520Cancer%2520Cells%2520by%2520Targeting%2520FOXM1%2520and%2520FOXM1-Interacting%2520Factor%2520SMAD3%26jtitle%3DTheranostics%26date%3D2019%26volume%3D9%26issue%3D10%26spage%3D2882%26epage%3D2896%26doi%3D10.7150%2Fthno.32693" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jolma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campitelli, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taipale, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weirauch, M. T.</span></span> <span> </span><span class="NLM_article-title">The Human Transcription Factors</span>. <i>Cell</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>172</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">650</span>– <span class="NLM_lpage">665</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2018.01.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.cell.2018.01.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=29425488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1amsbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=172&amp;publication_year=2018&amp;pages=650-665&amp;issue=4&amp;author=S.+A.+Lambertauthor=A.+Jolmaauthor=L.+F.+Campitelliauthor=P.+K.+Dasauthor=Y.+Yinauthor=M.+Albuauthor=X.+Chenauthor=J.+Taipaleauthor=T.+R.+Hughesauthor=M.+T.+Weirauch&amp;title=The+Human+Transcription+Factors&amp;doi=10.1016%2Fj.cell.2018.01.029" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The Human Transcription Factors</span></div><div class="casAuthors">Lambert, Samuel A.; Jolma, Arttu; Campitelli, Laura F.; Das, Pratyush K.; Yin, Yimeng; Albu, Mihai; Chen, Xiaoting; Taipale, Jussi; Hughes, Timothy R.; Weirauch, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">650-665</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Transcription factors (TFs) recognize specific DNA sequences to control chromatin and transcription, forming a complex system that guides expression of the genome.  Despite keen interest in understanding how TFs control gene expression, it remains challenging to det. how the precise genomic binding sites of TFs are specified and how TF binding ultimately relates to regulation of transcription.  This review considers how TFs are identified and functionally characterized, principally through the lens of a catalog of over 1600 likely human TFs and binding motifs for two-thirds of them.  Major classes of human TFs differ markedly in their evolutionary trajectories and expression patterns, underscoring distinct functions.  TFs likewise underlie many different aspects of human physiol., disease, and variation, highlighting the importance of continued effort to understand TF-mediated gene regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGN_jQqNzpvrVg90H21EOLACvtfcHk0liJxGpcV1z45w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXis1amsbs%253D&amp;md5=a53a2bf38c15419c7589893428efd738</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.01.029&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.01.029%26sid%3Dliteratum%253Aachs%26aulast%3DLambert%26aufirst%3DS.%2BA.%26aulast%3DJolma%26aufirst%3DA.%26aulast%3DCampitelli%26aufirst%3DL.%2BF.%26aulast%3DDas%26aufirst%3DP.%2BK.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DAlbu%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DTaipale%26aufirst%3DJ.%26aulast%3DHughes%26aufirst%3DT.%2BR.%26aulast%3DWeirauch%26aufirst%3DM.%2BT.%26atitle%3DThe%2520Human%2520Transcription%2520Factors%26jtitle%3DCell%26date%3D2018%26volume%3D172%26issue%3D4%26spage%3D650%26epage%3D665%26doi%3D10.1016%2Fj.cell.2018.01.029" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></div><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><div class="cited-by__content"><div class="sciteMetricsDropzone">



        
        <div class="scite-metrics"><div data-lazyload="true" data-ajax-url="/pb/widgets/sciteMetric?widgetId=0b2ac3fd-c0dc-46e2-8934-13e804afe077&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facscentsci.1c00663%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3BrequestedJournal%3Ajournal%3Aacscii%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Facscii.2021.7.issue-8%3Bjournal%3Ajournal%3Aacscii%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;doi=10.1021%2Facscentsci.1c00663" class="scite-metrics__lazyLoad"><div class="js--loading"></div></div></div>
</div><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Ross J. Taylor, Michael B. Geeson, Toby Journeaux, <span class="NLM_string-name hlFld-ContribAuthor">Gonçalo J. L. Bernardes</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical and Enzymatic Methods for Post-Translational Protein–Protein Conjugation. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2022,</strong> <em>144 </em>
                                    (32)
                                     , 14404-14419. <a href="https://doi.org/10.1021/jacs.2c00129" title="DOI URL">https://doi.org/10.1021/jacs.2c00129</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.2c00129&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.2c00129%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DChemical%252Band%252BEnzymatic%252BMethods%252Bfor%252BPost-Translational%252BProtein%2525E2%252580%252593Protein%252BConjugation%26aulast%3DTaylor%26aufirst%3DRoss%2BJ.%26date%3D2022%26date%3D2022%26date%3D2022%26date%3D05012022%26date%3D01082022%26volume%3D144%26issue%3D32%26spage%3D14404%26epage%3D14419" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hisashi Masui, <span class="NLM_string-name hlFld-ContribAuthor">Shinichiro Fuse</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in the Solid- and Solution-Phase Synthesis of Peptides and Proteins Using Microflow Technology. </span><span class="cited-content_cbyCitation_journal-name">Organic Process Research &amp; Development</span><span> <strong>2022,</strong> <em>26 </em>
                                    (6)
                                     , 1751-1765. <a href="https://doi.org/10.1021/acs.oprd.2c00074" title="DOI URL">https://doi.org/10.1021/acs.oprd.2c00074</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.oprd.2c00074&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.oprd.2c00074%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Process%2520Research%2520%2526%2520Development%26atitle%3DRecent%252BAdvances%252Bin%252Bthe%252BSolid-%252Band%252BSolution-Phase%252BSynthesis%252Bof%252BPeptides%252Band%252BProteins%252BUsing%252BMicroflow%252BTechnology%26aulast%3DMasui%26aufirst%3DHisashi%26date%3D2022%26date%3D2022%26date%3D2022%26date%3D10032022%26date%3D27052022%26volume%3D26%26issue%3D6%26spage%3D1751%26epage%3D1765" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/cms/10.1021/acscentsci.1c00663/asset/images/medium/oc1c00663_0006.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c00663&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acscentsci.1c00663/asset/images/medium/oc1c00663_0001.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Covalent protein dimers prepared by automated flow protein synthesis (AFPS) are designed to compete with MYC/MAX for E-Box binding and inhibit oncogenic transcription. Inspired by the inhibitory activity of the MAX/MAX protein dimer assembly, we engineered covalently linked, synthetic homo- and heterodimeric analogues of MYC, MAX, and Omomyc, to inhibit MYC/MAX-dependent gene transcription.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c00663&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acscentsci.1c00663/asset/images/medium/oc1c00663_0002.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Covalently linked dimeric transcription factor analogues were prepared via fast-flow solid-phase synthesis. (a) An automated flow peptide synthesizer built in the Pentelute laboratory. (b) Resin for protein synthesis with lysine-based linkers <b>1</b> or <b>4</b> for parallel dimeric protein synthesis. The linkers were manually coupled on the Rink Amide resin prior to AFPS. (c) Reagents and conditions for flow synthesis. (d) Sequences of MYC, MAX, and Omomyc. (e–h) Synthesis time, yields, and LC-MS characterization of purified homodimers <b>2</b> and <b>3</b> and heterodimers <b>5</b> and <b>6</b>. The panels show the total ion current chromatogram (TIC) as the base spectrum, the electrospray ionization (ESI) mass-to-charge spectra (left inset), and deconvoluted mass spectra (right inset). The peak at 922 in the <i>m</i>/<i>z</i> spectra is the ion of the internal reference compound of the mass spectrometer. The minor MS peak population at masses lower than the expected product mass (in parts e, g, and h) is a mixture of deletion and truncation products.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c00663&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acscentsci.1c00663/asset/images/medium/oc1c00663_0003.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. The synthetic protein dimers are homogeneous, fold, and associate to E-box DNA. The covalent linkage stabilizes the structure and DNA/protein complexes. (a) SDS-PAGE analysis of synthetic protein dimers and monomers (∼1 μg per protein loaded). The bands were visualized by Coomassie Blue staining. (b) Proteins were dissolved in folding buffer (MES 10 mM, KCl 150 mM, MgCl<sub>2</sub> 1 mM, TCEP 1 mM, glycerol 10%, pH = 6.5) at 0.1 mg/mL, and full wavelength CD spectra were recorded from 250 to 200 nm at 4 °C. The <i>y</i>-axis shows the mean residual ellipticity (MRE) and the <i>x</i>-axis the wavelength. (c) Each compound was incubated with E-Box DNA in folding buffer (final concentrations: 2 μM protein and 1 μM DNA) for 1 h at room temperature. Samples were run on 10% polyacrylamide gel in TBE buffer and visualized with ethidium bromide. (d) For melting temperature determination, CD spectra were recorded at 221 nm from +4 to +89 °C, in 5 °C steps. (e) Summary table of melting temperatures determined by CD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c00663&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acscentsci.1c00663/asset/images/medium/oc1c00663_0004.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Synthetic dimers are intrinsically cell-penetrating, and nuclear targeting is enhanced by non-natural sequence tags. (a) Reaction conditions. Alloc removal: Pd(PPh<sub>3</sub>)<sub>4</sub>, piperidine, CH<sub>2</sub>Cl<sub>2</sub>, r.t., 30 min. TAMRA labeling: 5-TAMRA, HATU, DIEA, DMF, r.t., 30 min. Mach3 synthesis: see AFPS conditions (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>c). Cleavage: TFA, EDT, thioanisole, cresol, H<sub>2</sub>O, r.t., 4 h. (b) TIC-LCMS chromatograms of MAX–MAX–TAMRA <b>10</b>, Omomyc–Omomyc–TAMRA <b>11</b>, and Omomyc–Omomyc–Mach3–TAMRA <b>12</b> with <i>m</i>/<i>z</i> and deconvoluted mass. (c) Flow cytometry histograms illustrating the dose-dependent increase in fluorescence of HeLa cells after 15 min of incubation with TAMRA-labeled dimers and Omomyc–TAMRA monomer at concentrations between 0.01 and 15 μM. (d) Micrographs from confocal microscopy; Hoechst (DAPI) labels the nuclei, and TAMRA-protein (Cy3) is observed throughout the cell after 15 min of incubation, followed by incubation in fresh media for 1 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c00663&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/acscentsci.1c00663/asset/images/medium/oc1c00663_0005.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Synthetic transcription factors inhibit cancer cell proliferation and downregulate MYC target genes. (a) Cell proliferation assay of HeLa, A549, and H441 cells following treatment with synthetic protein dimers for 72 h, quantified via CellTiter-Glo (CTG). Experiments were performed in triplicate. (b) Summary table of proliferation inhibition EC<sub>50</sub> values. (c) A549 cells were treated with Omomyc–Omomyc <b>3</b> (12.5 μM) for 72 h. RNA was extracted and sequenced. Depicted is an MA plot showing differentially expressed genes (DEGs) in A549 cells treated with Omomyc–Omomyc <b>3</b> compared to DMSO. Upregulated genes with adjusted <i>p</i>-value &lt;0.05 and |log 2FC| ≥ 1 shown in red, downregulated genes with <i>p</i>-value &lt;0.05 and |log 2FC| ≤ 1 shown in blue. The experiment was performed in duplicate. Downregulated genes involved in <i>KRas</i> pathways are labeled. (d) GSEA comparing gene expression of DMSO treated cells compared to Omomyc–Omomyc. The enrichment plot of the MYC target gene signature shows a negative enrichment in the Omomyc–Omomyc condition (<i>q</i>-value &lt;0.05).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.1c00663/asset/images/large/oc1c00663_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.1c00663&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45420" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45420" class="dropBlock__holder"></div></div></div></div></div><hr></div><p class="rightTab-references-count">
            This article references 62 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muir, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark-Lewis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, S. B. H.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Proteins by Native Chemical Ligation</span>. <i>Science</i>  <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>266</i></span>,  <span class="NLM_fpage">776</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.7973629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1126%2Fscience.7973629" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=7973629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADyaK2MXitVGgtrw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=266&amp;publication_year=1994&amp;pages=776&amp;author=P.+E.+Dawsonauthor=T.+W.+Muirauthor=I.+Clark-Lewisauthor=S.+B.+H.+Kent&amp;title=Synthesis+of+Proteins+by+Native+Chemical+Ligation&amp;doi=10.1126%2Fscience.7973629" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of proteins by native chemical ligation</span></div><div class="casAuthors">Dawson, Philip E.; Muir, Tom W.; Clark-Lewis, Ian; Kent, Stephen B. H.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">266</span>
        (<span class="NLM_cas:issue">5186</span>),
    <span class="NLM_cas:pages">776-9</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A simple technique has been devised that allows the direct synthesis of native backbone proteins of moderate size.  Chemoselective reaction of two unprotected peptide segments gives an initial thioester-linked species.  Spontaneous rearrangement of this transient intermediate yields a full-length product with a native peptide bond at the ligation site.  The utility of native chem. ligation was demonstrated by the one-step prepn. of a cytokine contg. multiple disulfides.  The polypeptide ligation product was folded and oxidized to form the native disulfide-contg. protein mol.  Native chem. ligation is an important step toward the general application of chem. to proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuXFAQmIIOgrVg90H21EOLACvtfcHk0lhMaFhugFRFFw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXitVGgtrw%253D&amp;md5=1d352d817ab48851500610dcd32d1136</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.7973629&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7973629%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DP.%2BE.%26aulast%3DMuir%26aufirst%3DT.%2BW.%26aulast%3DClark-Lewis%26aufirst%3DI.%26aulast%3DKent%26aufirst%3DS.%2BB.%2BH.%26atitle%3DSynthesis%2520of%2520Proteins%2520by%2520Native%2520Chemical%2520Ligation%26jtitle%3DScience%26date%3D1994%26volume%3D266%26spage%3D776%26doi%3D10.1126%2Fscience.7973629" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bode, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baucom, K. D.</span></span> <span> </span><span class="NLM_article-title">Chemoselective Amide Ligations by Decarboxylative Condensations of N-Alkylhydroxylamines and α-Ketoacids</span>. <i>Angew. Chem., Int. Ed.</i>  <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1248</span>– <span class="NLM_lpage">1252</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.200503991</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1002%2Fanie.200503991" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvFyjsrY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=2006&amp;pages=1248-1252&amp;issue=8&amp;author=J.+W.+Bodeauthor=R.+M.+Foxauthor=K.+D.+Baucom&amp;title=Chemoselective+Amide+Ligations+by+Decarboxylative+Condensations+of+N-Alkylhydroxylamines+and+%CE%B1-Ketoacids&amp;doi=10.1002%2Fanie.200503991" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoselective amide ligations by decarboxylative condensations of N-alkylhydroxylamines and α-keto acids</span></div><div class="casAuthors">Bode, Jeffrey W.; Fox, Ryan M.; Baucom, Kyle D.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1248-1252</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH &amp; Co. KGaA</span>)
        </div><div class="casAbstract">The chemoselective amide bond-forming ligation between N-alkylhydroxylamines and α-keto acids requires no reagents and the only byproducts are water and carbon dioxide.  This process proceeds on unprotected peptide substrates without epimerization, and as such, this process has the potential to serve as a novel chemoselective ligation for the synthesis of peptides and complex materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1JDHd7uK1r7Vg90H21EOLACvtfcHk0lgvGd8c0C1uRw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28XhvFyjsrY%253D&amp;md5=ca37ba7faf34db1a8b86fd4dcc7f1b54</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fanie.200503991&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200503991%26sid%3Dliteratum%253Aachs%26aulast%3DBode%26aufirst%3DJ.%2BW.%26aulast%3DFox%26aufirst%3DR.%2BM.%26aulast%3DBaucom%26aufirst%3DK.%2BD.%26atitle%3DChemoselective%2520Amide%2520Ligations%2520by%2520Decarboxylative%2520Condensations%2520of%2520N-Alkylhydroxylamines%2520and%2520%25CE%25B1-Ketoacids%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2006%26volume%3D45%26issue%3D8%26spage%3D1248%26epage%3D1252%26doi%3D10.1002%2Fanie.200503991" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Premdjee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larance, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conde-Frieboes, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, R. J.</span></span> <span> </span><span class="NLM_article-title">Chemical Synthesis of Phosphorylated Insulin-like Growth Factor Binding Protein 2</span>. <i>J. Am. Chem. Soc.</i>  <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>143</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">5336</span>– <span class="NLM_lpage">5342</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.1c02280</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.1c02280" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnsl2ktLo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=143&amp;publication_year=2021&amp;pages=5336-5342&amp;issue=14&amp;author=B.+Premdjeeauthor=A.+S.+Andersenauthor=M.+Laranceauthor=K.+W.+Conde-Frieboesauthor=R.+J.+Payne&amp;title=Chemical+Synthesis+of+Phosphorylated+Insulin-like+Growth+Factor+Binding+Protein+2&amp;doi=10.1021%2Fjacs.1c02280" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical synthesis of phosphorylated insulin-like growth factor binding protein 2</span></div><div class="casAuthors">Premdjee, Bhavesh; Andersen, Asser S.; Larance, Mark; Conde-Frieboes, Kilian W.; Payne, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5336-5342</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. protein synthesis is a powerful avenue for accessing homogeneously modified proteins.  While a significant no. of small modified proteins bearing native post-translational modifications and non-natural modifications have been generated to date, access to larger targets has proved challenging.  Herein, we describe the use of two ligation manifolds, namely, diselenide-selenoester ligation and native chem. ligation, to assemble a 31.5 kDa phosphorylated insulin-like growth factor binding protein 2 (IGFBP-2) that comprises 290 amino acid residues, a phosphoserine post-translational modification, and nine disulfide bonds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBHep9DkCBp7Vg90H21EOLACvtfcHk0lgvGd8c0C1uRw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXnsl2ktLo%253D&amp;md5=ce3979b3357a66444c5f967890da9566</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjacs.1c02280&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.1c02280%26sid%3Dliteratum%253Aachs%26aulast%3DPremdjee%26aufirst%3DB.%26aulast%3DAndersen%26aufirst%3DA.%2BS.%26aulast%3DLarance%26aufirst%3DM.%26aulast%3DConde-Frieboes%26aufirst%3DK.%2BW.%26aulast%3DPayne%26aufirst%3DR.%2BJ.%26atitle%3DChemical%2520Synthesis%2520of%2520Phosphorylated%2520Insulin-like%2520Growth%2520Factor%2520Binding%2520Protein%25202%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2021%26volume%3D143%26issue%3D14%26spage%3D5336%26epage%3D5342%26doi%3D10.1021%2Fjacs.1c02280" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agouridas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Mahdi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diemer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cargoët, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monbaliu, J. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, O.</span></span> <span> </span><span class="NLM_article-title">Native Chemical Ligation and Extended Methods: Mechanisms, Catalysis, Scope, and Limitations</span>. <i>Chem. Rev.</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">7328</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.chemrev.8b00712</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00712" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1MXoslamtr0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=119&amp;publication_year=2019&amp;pages=7328&amp;issue=12&amp;author=V.+Agouridasauthor=O.+El+Mahdiauthor=V.+Diemerauthor=M.+Cargo%C3%ABtauthor=J.+C.+M.+Monbaliuauthor=O.+Melnyk&amp;title=Native+Chemical+Ligation+and+Extended+Methods%3A+Mechanisms%2C+Catalysis%2C+Scope%2C+and+Limitations&amp;doi=10.1021%2Facs.chemrev.8b00712" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Native Chemical Ligation and Extended Methods: Mechanisms, Catalysis, Scope, and Limitations</span></div><div class="casAuthors">Agouridas, Vangelis; El Mahdi, Ouafaa; Diemer, Vincent; Cargoet, Marine; Monbaliu, Jean-Christophe M.; Melnyk, Oleg</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">7328-7443</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The native chem. ligation reaction (NCL) involves reacting a C-terminal peptide thioester with an N-terminal cysteinyl peptide to produce a native peptide bond between the two fragments.  This reaction has considerably extended the size of polypeptides and proteins that can be produced by total synthesis and has also numerous applications in bioconjugation, polymer synthesis, material science, and micro- and nanotechnol. research.  The aim of the present review is to provide a thorough mechanistic overview of NCL and extended methods.  The most relevant properties of peptide thioesters, Cys peptides, and common solvents, reagents, additives, and catalysts used for these ligations are presented.  Mechanisms, selectivity and reactivity are, whenever possible, discussed through the insights of computational and phys. chem. studies.  The inherent limitations of NCL are discussed with insights from the mechanistic standpoint.  This review also presents a palette of O,S-, N,S-, or N,Se-acyl shift systems as thioester or selenoester surrogates and discusses the special mol. features that govern reactivity in each case.  Finally, the various thiol-based auxiliaries and thiol or selenol amino acid surrogates that have been developed so far are discussed with a special focus on the mechanism of long-range N,S-acyl migrations and selective dechalcogenation reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtWR5R5VOQybVg90H21EOLACvtfcHk0lgGvJbKkiDdlA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXoslamtr0%253D&amp;md5=d608ef5f6d39e2b646e36224c35008f5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00712&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00712%26sid%3Dliteratum%253Aachs%26aulast%3DAgouridas%26aufirst%3DV.%26aulast%3DEl%2BMahdi%26aufirst%3DO.%26aulast%3DDiemer%26aufirst%3DV.%26aulast%3DCargo%25C3%25ABt%26aufirst%3DM.%26aulast%3DMonbaliu%26aufirst%3DJ.%2BC.%2BM.%26aulast%3DMelnyk%26aufirst%3DO.%26atitle%3DNative%2520Chemical%2520Ligation%2520and%2520Extended%2520Methods%253A%2520Mechanisms%252C%2520Catalysis%252C%2520Scope%252C%2520and%2520Limitations%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26issue%3D12%26spage%3D7328%26doi%3D10.1021%2Facs.chemrev.8b00712" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conibear, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, C. F. W.</span></span> <span> </span><span class="NLM_article-title">Native Chemical Ligation in Protein Synthesis and Semi-Synthesis</span>. <i>Chem. Soc. Rev.</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">9046</span>– <span class="NLM_lpage">9068</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C8CS00573G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1039%2FC8CS00573G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30418441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKmtrvP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=2018&amp;pages=9046-9068&amp;issue=24&amp;author=A.+C.+Conibearauthor=E.+E.+Watsonauthor=R.+J.+Payneauthor=C.+F.+W.+Becker&amp;title=Native+Chemical+Ligation+in+Protein+Synthesis+and+Semi-Synthesis&amp;doi=10.1039%2FC8CS00573G" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Native chemical ligation in protein synthesis and semi-synthesis</span></div><div class="casAuthors">Conibear, Anne C.; Watson, Emma E.; Payne, Richard J.; Becker, Christian F. W.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9046-9068</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Native chem. ligation (NCL) provides a highly efficient and robust means to chemoselectively link unprotected peptide and protein segments to generate proteins.  The ability to incorporate non-proteinogenic amino acids (e.g.D-amino acids or fluorescent labels) and post-translational modifications into proteins by stitching together peptide fragments has driven extremely important developments in peptide and protein science over the past 20 years.  Extensions of the original NCL concept (including the development of thiol- and selenol-derived amino acids and desulfurisation and deselenisation methods), improved access to peptide thioesters, and the use of the methodol. in combination with recombinantly expressed polypeptide fragments (termed Expressed Protein Ligation, EPL) have helped to further expand the utility of the methodol.  Over the past five years, there has been a dramatic increase in the no. of proteins that have been accessed by total chem. synthesis and semi-synthesis, including a large range of modified proteins; new records have also been set with regards to the size of proteins that can now be accessed via ligation chem.  Together these efforts have not only contributed to a better understanding of protein structure and function, but have also driven innovations in protein science.  In this tutorial review, we aim to provide the reader with the latest developments in NCL- and EPL-based ligation technologies as well as illustrated examples of using these methods, together with synthetic logic, to access proteins and modified proteins for biol. study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0dO1SAMx10LVg90H21EOLACvtfcHk0lgGvJbKkiDdlA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXitFKmtrvP&amp;md5=148c164b9840ed1b1782a657f5cf9504</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1039%2FC8CS00573G&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8CS00573G%26sid%3Dliteratum%253Aachs%26aulast%3DConibear%26aufirst%3DA.%2BC.%26aulast%3DWatson%26aufirst%3DE.%2BE.%26aulast%3DPayne%26aufirst%3DR.%2BJ.%26aulast%3DBecker%26aufirst%3DC.%2BF.%2BW.%26atitle%3DNative%2520Chemical%2520Ligation%2520in%2520Protein%2520Synthesis%2520and%2520Semi-Synthesis%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2018%26volume%3D47%26issue%3D24%26spage%3D9046%26epage%3D9068%26doi%3D10.1039%2FC8CS00573G" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondalapati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jbara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brik, A.</span></span> <span> </span><span class="NLM_article-title">Expanding the Chemical Toolbox for the Synthesis of Large and Uniquely Modified Proteins</span>. <i>Nat. Chem.</i>  <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">418</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.2476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fnchem.2476" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=27102674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC28XmslShsL8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2016&amp;pages=407-418&amp;issue=5&amp;author=S.+Bondalapatiauthor=M.+Jbaraauthor=A.+Brik&amp;title=Expanding+the+Chemical+Toolbox+for+the+Synthesis+of+Large+and+Uniquely+Modified+Proteins&amp;doi=10.1038%2Fnchem.2476" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding the chemical toolbox for the synthesis of large and uniquely modified proteins</span></div><div class="casAuthors">Bondalapati, Somasekhar; Jbara, Muhammad; Brik, Ashraf</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">407-418</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Methods to prep. proteins that include a specific modification at a desired position are essential for understanding their cellular functions and phys. properties in living systems.  Chem. protein synthesis, which relies on the chemoselective ligation of unprotected peptides, enables the prepn. of modified proteins that are not easily fabricated by other methods.  In contrast to recombinant approaches, chem. synthesis can be used to prep. protein analogs such as D-proteins, which are useful in protein structure detn. and the discovery of novel therapeutics.  Post-translationally modifying proteins is another example where chem. protein synthesis proved itself as a powerful approach for prepg. samples with high homogeneity and in workable quantities.  In this Review, we discuss the basic principles of the field, focusing on novel chemoselective peptide ligation approaches such as native chem. ligation and the recent advances based on this method with a proven record of success in the synthesis of highly important protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3R5-DwrpbHLVg90H21EOLACvtfcHk0lixO05uK4zPhg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XmslShsL8%253D&amp;md5=983bef06802d74a198853f6f2d130626</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2476&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2476%26sid%3Dliteratum%253Aachs%26aulast%3DBondalapati%26aufirst%3DS.%26aulast%3DJbara%26aufirst%3DM.%26aulast%3DBrik%26aufirst%3DA.%26atitle%3DExpanding%2520the%2520Chemical%2520Toolbox%2520for%2520the%2520Synthesis%2520of%2520Large%2520and%2520Uniquely%2520Modified%2520Proteins%26jtitle%3DNat.%2520Chem.%26date%3D2016%26volume%3D8%26issue%3D5%26spage%3D407%26epage%3D418%26doi%3D10.1038%2Fnchem.2476" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bertolini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenzl, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kats, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wruck, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auburger, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tans, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukau, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, G.</span></span> <span> </span><span class="NLM_article-title">Interactions between Nascent Proteins Translated by Adjacent Ribosomes Drive Homomer Assembly</span>. <i>Science</i>  <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>371</i></span> (<span class="NLM_issue">6524</span>),  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">64</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.abc7151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1126%2Fscience.abc7151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=33384371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtVChsbY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=371&amp;publication_year=2021&amp;pages=57-64&amp;issue=6524&amp;author=M.+Bertoliniauthor=K.+Fenzlauthor=I.+Katsauthor=F.+Wruckauthor=F.+Tippmannauthor=J.+Schmittauthor=J.+J.+Auburgerauthor=S.+Tansauthor=B.+Bukauauthor=G.+Kramer&amp;title=Interactions+between+Nascent+Proteins+Translated+by+Adjacent+Ribosomes+Drive+Homomer+Assembly&amp;doi=10.1126%2Fscience.abc7151" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Interactions between nascent proteins translated by adjacent ribosomes drive homomer assembly</span></div><div class="casAuthors">Bertolini, Matilde; Fenzl, Kai; Kats, Ilia; Wruck, Florian; Tippmann, Frank; Schmitt, Jaro; Auburger, Josef Johannes; Tans, Sander; Bukau, Bernd; Kramer, Guenter</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">6524</span>),
    <span class="NLM_cas:pages">57-64</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Accurate assembly of newly synthesized proteins into functional oligomers is crucial for cell activity.  In this study, we investigated whether direct interaction of two nascent proteins, emerging from nearby ribosomes (co-co assembly), constitutes a general mechanism for oligomer formation.  We used proteome-wide screening to detect nascent chain-connected ribosome pairs and identified hundreds of homomer subunits that co-co assemble in human cells.  Interactions are mediated by five major domain classes, among which N-terminal coiled coils are the most prevalent.  We were able to reconstitute co-co assembly of nuclear lamin in Escherichia coli, demonstrating that dimer formation is independent of dedicated assembly machineries.  Co-co assembly may thus represent an efficient way to limit protein aggregation risks posed by diffusion-driven assembly routes and ensure isoform-specific homomer formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGok3X9ogzJrHbVg90H21EOLACvtfcHk0lixO05uK4zPhg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXhtVChsbY%253D&amp;md5=41388a1562a4968843a1e8e4cd93f728</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.abc7151&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abc7151%26sid%3Dliteratum%253Aachs%26aulast%3DBertolini%26aufirst%3DM.%26aulast%3DFenzl%26aufirst%3DK.%26aulast%3DKats%26aufirst%3DI.%26aulast%3DWruck%26aufirst%3DF.%26aulast%3DTippmann%26aufirst%3DF.%26aulast%3DSchmitt%26aufirst%3DJ.%26aulast%3DAuburger%26aufirst%3DJ.%2BJ.%26aulast%3DTans%26aufirst%3DS.%26aulast%3DBukau%26aufirst%3DB.%26aulast%3DKramer%26aufirst%3DG.%26atitle%3DInteractions%2520between%2520Nascent%2520Proteins%2520Translated%2520by%2520Adjacent%2520Ribosomes%2520Drive%2520Homomer%2520Assembly%26jtitle%3DScience%26date%3D2021%26volume%3D371%26issue%3D6524%26spage%3D57%26epage%3D64%26doi%3D10.1126%2Fscience.abc7151" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shiber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Döring, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedrich, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zedan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukau, B.</span></span> <span> </span><span class="NLM_article-title">Cotranslational Assembly of Protein Complexes in Eukaryotes Revealed by Ribosome Profiling</span>. <i>Nature</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>561</i></span> (<span class="NLM_issue">7722</span>),  <span class="NLM_fpage">268</span>– <span class="NLM_lpage">272</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41586-018-0462-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fs41586-018-0462-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30158700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Wmt7%252FE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=561&amp;publication_year=2018&amp;pages=268-272&amp;issue=7722&amp;author=A.+Shiberauthor=K.+D%C3%B6ringauthor=U.+Friedrichauthor=K.+Klannauthor=D.+Merkerauthor=M.+Zedanauthor=F.+Tippmannauthor=G.+Kramerauthor=B.+Bukau&amp;title=Cotranslational+Assembly+of+Protein+Complexes+in+Eukaryotes+Revealed+by+Ribosome+Profiling&amp;doi=10.1038%2Fs41586-018-0462-y" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Cotranslational assembly of protein complexes in eukaryotes revealed by ribosome profiling</span></div><div class="casAuthors">Shiber, Ayala; Doering, Kristina; Friedrich, Ulrike; Klann, Kevin; Merker, Dorina; Zedan, Mostafa; Tippmann, Frank; Kramer, Guenter; Bukau, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">561</span>
        (<span class="NLM_cas:issue">7722</span>),
    <span class="NLM_cas:pages">268-272</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The folding of newly synthesized proteins to the native state is a major challenge within the crowded cellular environment, as non-productive interactions can lead to misfolding, aggregation, and degrdn.  Cells cope with this challenge by coupling synthesis with polypeptide folding and by using mol. chaperones to safeguard folding cotranslationally.  However, although most of the cellular proteome forms oligomeric assemblies, little is known about the final step of folding: the assembly of polypeptides into complexes.  In prokaryotes, a proof-of-concept study showed that the assembly of heterodimeric luciferase is an organized cotranslational process that is facilitated by spatially confined translation of the subunits encoded on a polycistronic mRNA.  In eukaryotes, however, fundamental differences, such as the rarity of polycistronic mRNAs and different chaperone constellations, raise the question of whether assembly is also coordinated with translation.  Here, we provide a systematic and mechanistic anal. of the assembly of protein complexes in eukaryotes using ribosome profiling.  We detd. the in vivo interactions of the nascent subunits from 12 hetero-oligomeric protein complexes of Saccharomyces cerevisiae at near-residue resoln.  We found that 9 complexes assembled cotranslationally; the 3 complexes that did not show cotranslational interactions were regulated by dedicated assembly chaperones5-7.  Cotranslational assembly often occurs uni-directionally, with one fully synthesized subunit engaging its nascent partner subunit, thereby counteracting its propensity for aggregation.  The onset of cotranslational subunit assocn. coincided directly with the full exposure of the nascent interaction domain at the ribosomal tunnel exit.  The action of ribosome-assocd. Hsp70 chaperone Ssb was coordinated with assembly.  Ssb transiently engaged partially synthesized interaction domains and then dissocd. before the onset of partner subunit assocn., presumably to prevent premature assembly interactions.  This study showed that cotranslational subunit assocn. is a prevalent mechanism for the assembly of hetero-oligomers in yeast and indicated that translation, folding and the assembly of protein complexes are integrated processes in eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwrhPj7fRwLrVg90H21EOLACvtfcHk0lj6sbRz7_LOQA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Wmt7%252FE&amp;md5=3002026264d124d8921feb93bcc420cc</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0462-y&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0462-y%26sid%3Dliteratum%253Aachs%26aulast%3DShiber%26aufirst%3DA.%26aulast%3DD%25C3%25B6ring%26aufirst%3DK.%26aulast%3DFriedrich%26aufirst%3DU.%26aulast%3DKlann%26aufirst%3DK.%26aulast%3DMerker%26aufirst%3DD.%26aulast%3DZedan%26aufirst%3DM.%26aulast%3DTippmann%26aufirst%3DF.%26aulast%3DKramer%26aufirst%3DG.%26aulast%3DBukau%26aufirst%3DB.%26atitle%3DCotranslational%2520Assembly%2520of%2520Protein%2520Complexes%2520in%2520Eukaryotes%2520Revealed%2520by%2520Ribosome%2520Profiling%26jtitle%3DNature%26date%3D2018%26volume%3D561%26issue%3D7722%26spage%3D268%26epage%3D272%26doi%3D10.1038%2Fs41586-018-0462-y" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penn, L. Z.</span></span> <span> </span><span class="NLM_article-title">Reflecting on 25 Years with MYC</span>. <i>Nat. Rev. Cancer</i>  <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">976</span>– <span class="NLM_lpage">990</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nrc2231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fnrc2231" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=19029958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWhu7rJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2008&amp;pages=976-990&amp;issue=12&amp;author=N.+Meyerauthor=L.+Z.+Penn&amp;title=Reflecting+on+25+Years+with+MYC&amp;doi=10.1038%2Fnrc2231" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Reflecting on 25 years with MYC</span></div><div class="casAuthors">Meyer, Natalie; Penn, Linda Z.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">976-990</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Just over 25 years ago, MYC, the human homolog of a retroviral oncogene, was identified.  Since that time, MYC research has been intense and the advances impressive.  On reflection, it is astonishing how each incremental insight into MYC regulation and function has also had an impact on numerous biol. disciplines, including our understanding of mol. oncogenesis in general.  Here we chronicle the major advances in our understanding of MYC biol., and peer into the future of MYC research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR-DcM7mIL4bVg90H21EOLACvtfcHk0lj6sbRz7_LOQA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWhu7rJ&amp;md5=93089742c2c5e30c1075d62fdd416835</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrc2231&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2231%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DN.%26aulast%3DPenn%26aufirst%3DL.%2BZ.%26atitle%3DReflecting%2520on%252025%2520Years%2520with%2520MYC%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2008%26volume%3D8%26issue%3D12%26spage%3D976%26epage%3D990%26doi%3D10.1038%2Fnrc2231" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kretzner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weintraub, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwood, E.</span></span> <span> </span><span class="NLM_article-title">Sequence-Specific DNA Binding by the c-Myc Protein</span>. <i>Science</i>  <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>250</i></span> (<span class="NLM_issue">4984</span>),  <span class="NLM_fpage">1149</span>– <span class="NLM_lpage">1151</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.2251503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1126%2Fscience.2251503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=2251503" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADyaK3MXlslOhsQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=250&amp;publication_year=1990&amp;pages=1149-1151&amp;issue=4984&amp;author=T.+Blackwellauthor=L.+Kretznerauthor=R.+Eisenmanauthor=H.+Weintraubauthor=E.+Blackwood&amp;title=Sequence-Specific+DNA+Binding+by+the+c-Myc+Protein&amp;doi=10.1126%2Fscience.2251503" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence-specific DNA binding by the c-Myc protein</span></div><div class="casAuthors">Blackwell, T. Keith; Kretzner, Leo; Blackwood, Elizabeth M.; Eisenman, Robert N.; Weintraub, Harold</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">250</span>
        (<span class="NLM_cas:issue">4984</span>),
    <span class="NLM_cas:pages">1149-51</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">While it has been known for some time that the c-Myc protein binds to random DNA sequences, no sequence-specific binding activity has been detected.  At its carboxyl terminus, c-Myc contains a basic-helix-loop-helix (bHLH) motif, which is important for dimerization and specific DNA binding, as demonstrated for other bHLH protein family members.  Of those studied, most bHLH bind to sites that contain a CA--TG consensus.  In this study, the technique of selected and amplified binding-sequence imprinting was used to identify a DNA sequence that was recognized by c-Myc.  A purified C-terminal fragment of human c-Myc that contained the bHLH domain bound in vitro in a sequence-specific manner to the sequence, CACGTG.  These results suggest that some of the biol. functions of Myc family proteins are accomplished by sequence-specific DNA binding that is mediated by the C-terminal region of the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKpAA5X4GPyrVg90H21EOLACvtfcHk0liCvlUhzTaHqw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3MXlslOhsQ%253D%253D&amp;md5=85b4b62b8d672fdfeb57cb70ad87aca7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.2251503&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2251503%26sid%3Dliteratum%253Aachs%26aulast%3DBlackwell%26aufirst%3DT.%26aulast%3DKretzner%26aufirst%3DL.%26aulast%3DEisenman%26aufirst%3DR.%26aulast%3DWeintraub%26aufirst%3DH.%26aulast%3DBlackwood%26aufirst%3DE.%26atitle%3DSequence-Specific%2520DNA%2520Binding%2520by%2520the%2520c-Myc%2520Protein%26jtitle%3DScience%26date%3D1990%26volume%3D250%26issue%3D4984%26spage%3D1149%26epage%3D1151%26doi%3D10.1126%2Fscience.2251503" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackwood, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenman, R. N.</span></span> <span> </span><span class="NLM_article-title">Max : A Helix-Loop-Helix Zipper Protein That Complex with Myc</span>. <i>Science</i>  <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>251</i></span>,  <span class="NLM_fpage">1211</span>– <span class="NLM_lpage">1217</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.2006410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1126%2Fscience.2006410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=2006410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADyaK3MXltlShs7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=251&amp;publication_year=1991&amp;pages=1211-1217&amp;author=E.+M.+Blackwoodauthor=R.+N.+Eisenman&amp;title=Max%E2%80%AF%3A+A+Helix-Loop-Helix+Zipper+Protein+That+Complex+with+Myc&amp;doi=10.1126%2Fscience.2006410" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Max:  a helix-loop-helix zipper protein that forms a sequence-specific DNA-binding complex with Myc</span></div><div class="casAuthors">Blackwood, Elizabeth M.; Eisenman, Robert N.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">251</span>
        (<span class="NLM_cas:issue">4998</span>),
    <span class="NLM_cas:pages">1211-17</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The myc protooncogene family has been implicated in cell proliferation, differentiation, and neoplasia, but its mechanism of function at the mol. level is unknown.  The carboxyl terminus of Myc family proteins contains a basic region helix-loop-helix leucine zipper motif (bHLH-Zip), which has DNA-binding activity and has been predicted to mediate protein-protein interactions.  The bHLH-Zip region of c-Myc was used to screen a cDNA expression library, and a bHLH-Zip protein, termed Max, was identified.  Max specifically assocd. with c-Myc, N-Myc, and L-Myc proteins, but not with a no. of other bHLH, bZip, or bHLH-Zip proteins.  The interaction between Max and c-Myc was dependent on the integrity of the c-Myc HLH-Zip domain, but not on the basic region or other sequences outside the domain.  Furthermore, the Myc-Max complex bound to DNA in a sequence-specific manner under conditions where neither Max nor Myc exhibited appreciable binding.  The DNA-binding activity of the complex was dependent on both the dimerization domain and the basic region of c-Myc.  These results suggests that Myc family proteins undergo a restricted set of interactions in the cell and may belong to the more general class of eukaryotic DNA-binding transcription factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqNOoJSe0N4rVg90H21EOLACvtfcHk0liCvlUhzTaHqw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3MXltlShs7Y%253D&amp;md5=e8b4df3b8d310027c50591a8610e9eba</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1126%2Fscience.2006410&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2006410%26sid%3Dliteratum%253Aachs%26aulast%3DBlackwood%26aufirst%3DE.%2BM.%26aulast%3DEisenman%26aufirst%3DR.%2BN.%26atitle%3DMax%25E2%2580%25AF%253A%2520A%2520Helix-Loop-Helix%2520Zipper%2520Protein%2520That%2520Complex%2520with%2520Myc%26jtitle%3DScience%26date%3D1991%26volume%3D251%26spage%3D1211%26epage%3D1217%26doi%3D10.1126%2Fscience.2006410" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferré-D’Amaré, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prendergast, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziff, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span> <span> </span><span class="NLM_article-title">Recognition by Max of Its Cognate DNA through a Dimeric b/HLH/Z Domain</span>. <i>Nature</i>  <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>363</i></span> (<span class="NLM_issue">6424</span>),  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">45</span>, <span class="refDoi">&nbsp;DOI: 10.1038/363038a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2F363038a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=8479534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADyaK3sXkvVyhsrw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=363&amp;publication_year=1993&amp;pages=38-45&amp;issue=6424&amp;author=A.+R.+Ferr%C3%A9-D%E2%80%99Amar%C3%A9author=G.+C.+Prendergastauthor=E.+B.+Ziffauthor=S.+K.+Burley&amp;title=Recognition+by+Max+of+Its+Cognate+DNA+through+a+Dimeric+b%2FHLH%2FZ+Domain&amp;doi=10.1038%2F363038a0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition by Max of its cognate DNA through a dimeric b/HLH/Z domain</span></div><div class="casAuthors">Ferre-D'Amare, Adrian R.; Prendergast, George C.; Ziff, Edward B.; Burley, Stephen K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">363</span>
        (<span class="NLM_cas:issue">6424</span>),
    <span class="NLM_cas:pages">38-45</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The three-dimensional structure of the basic/helix-loop-helix/leucine zipper domain of the transcription factor Max complexed with DNA has been detd. by X-ray crystallog. at 2.9 Å resoln.  Max binds as a dimer to its recognition sequence CACGTG by direct contacts between the α-helical basic region and the major groove.  This sym. homodimer, a new protein fold, is a parallel, left-handed, four-helix bundle, with each monomer contg. two α-helical segments sepd. by a loop.  The two α-helical segments are composed of the basic region plus helix 1 and helix 2 plus the leucine repeat, resp.  As in GCN4, the leucine repeat forms a parallel coiled coil.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuHiV_lNNN5LVg90H21EOLACvtfcHk0lgKZv1quAqHVA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3sXkvVyhsrw%253D&amp;md5=4ebca2f013cae17c9b3440756ffbcf45</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2F363038a0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F363038a0%26sid%3Dliteratum%253Aachs%26aulast%3DFerr%25C3%25A9-D%25E2%2580%2599Amar%25C3%25A9%26aufirst%3DA.%2BR.%26aulast%3DPrendergast%26aufirst%3DG.%2BC.%26aulast%3DZiff%26aufirst%3DE.%2BB.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26atitle%3DRecognition%2520by%2520Max%2520of%2520Its%2520Cognate%2520DNA%2520through%2520a%2520Dimeric%2520b%252FHLH%252FZ%2520Domain%26jtitle%3DNature%26date%3D1993%26volume%3D363%26issue%3D6424%26spage%3D38%26epage%3D45%26doi%3D10.1038%2F363038a0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qing, G.</span></span> <span> </span><span class="NLM_article-title">Targeting Oncogenic Myc as a Strategy for Cancer Treatment</span>. <i>Signal Transduct. Target. Ther.</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">7</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41392-018-0008-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fs41392-018-0008-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=29527327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A280%3ADC%252BC1MrpvFCltw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2018&amp;pages=1-7&amp;issue=1&amp;author=H.+Chenauthor=H.+Liuauthor=G.+Qing&amp;title=Targeting+Oncogenic+Myc+as+a+Strategy+for+Cancer+Treatment&amp;doi=10.1038%2Fs41392-018-0008-7" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">DNA methylation profile is associated with the osteogenic potential of three distinct human odontogenic stem cells</span></div><div class="casAuthors">Ai Tingting; Zhang Jieni; Wang Xuedong; Zheng Xiaowen; Qin Xueyan; Li Weiran; Hu Wei; Lin Jiuxiang; Ai Tingting; Zhang Qian; Chen Feng</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Among the various sources of human autologous stem cells, stem cells isolated from dental tissues exhibit excellent properties in tissue engineering and regenerative medicine.  However, the distinct potential of these odontogenic cell lines remains unclear.  In this study, we analyzed DNA methylation patterns to determine whether specific differences existed among three different odontogenic cell types.  Using the HumanMethylation450 Beadchip, the whole genomes of human dental pulp stem cells (DPSCs), periodontal ligament stem cells (PDLSCs), and dental follicle progenitor cells (DFPCs) were compared.  Then, the osteogenic potential of these cells was evaluated both in vitro and in vivo, and the methylation levels of certain genes related to bone formation differed among the three cell lines.  P values less than 0.05 were considered to indicate statistical significance.  The three cell types showed highly similar DNA methylation patterns, although specific differences were identified.  Gene ontology analysis revealed that one of the most significantly different gene categories was related to bone formation.  Thus, expression of cell surface epitopes and osteogenic-related transcription factors as well as the bone formation capacity were compared.  The results showed that compared with DFPCs and DPSCs, PDLSCs had higher transcription levels of osteogenic-related factors, a higher in vitro osteogenic potential, and an increased new bone formation capacity in vivo.  In conclusion, the results of this study suggested that the differential DNA methylation profiles could be related to the osteogenic potential of these human odontogenic cell populations.  Additionally, the increased osteogenic potential of PDLSCs might aid researchers or clinicians in making better choices regarding tissue regeneration and clinical therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSr2v8N4kAiY70AGQpJZWpJfW6udTcc2eYLclgcxYCN5Lntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC1MrpvFCltw%253D%253D&amp;md5=7820bce50f4c78b0166542a690152a88</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41392-018-0008-7&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-018-0008-7%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DQing%26aufirst%3DG.%26atitle%3DTargeting%2520Oncogenic%2520Myc%2520as%2520a%2520Strategy%2520for%2520Cancer%2520Treatment%26jtitle%3DSignal%2520Transduct.%2520Target.%2520Ther.%26date%3D2018%26volume%3D3%26issue%3D1%26spage%3D1%26epage%3D7%26doi%3D10.1038%2Fs41392-018-0008-7" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalkat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Melo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lourenco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resetca, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamachi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penn, L. Z.</span></span> <span> </span><span class="NLM_article-title">MYC Deregulation in Primary Human Cancers</span>. <i>Genes</i>  <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">151</span>, <span class="refDoi">&nbsp;DOI: 10.3390/genes8060151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.3390%2Fgenes8060151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpslCmu7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2017&amp;pages=151&amp;issue=6&amp;author=M.+Kalkatauthor=J.+De+Meloauthor=K.+A.+Hickmanauthor=C.+Lourencoauthor=C.+Redelauthor=D.+Resetcaauthor=A.+Tamachiauthor=W.+B.+Tuauthor=L.+Z.+Penn&amp;title=MYC+Deregulation+in+Primary+Human+Cancers&amp;doi=10.3390%2Fgenes8060151" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">MYC deregulation in primary human cancers</span></div><div class="casAuthors">Kalkat, Manpreet; De Melo, Jason; Hickman, Katherine Ashley; Lourenco, Corey; Redel, Cornelia; Resetca, Diana; Tamachi, Aaliya; Tu, William B.; Penn, Linda Z.</div><div class="citationInfo"><span class="NLM_cas:title">Genes</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">151/1-151/30</span>CODEN:
                <span class="NLM_cas:coden">GENEG9</span>;
        ISSN:<span class="NLM_cas:issn">2073-4425</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">MYC regulates a complex biol. program by transcriptionally activating and repressing its numerous target genes.  As such, MYC is a master regulator of many processes, including cell cycle entry, ribosome biogenesis, and metab.  In cancer, the activity of the MYC transcriptional network is frequently deregulated, contributing to the initiation and maintenance of disease.  Deregulation often leads to constitutive overexpression of MYC, which can be achieved through gross genetic abnormalities, including copy no. alterations, chromosomal translocations, increased enhancer activity, or through aberrant signal transduction leading to increased MYC transcription or increased MYC mRNA and protein stability.  Herein, we summarize the frequency and modes of MYC deregulation and describe both well-established and more recent findings in a variety of cancer types.  Notably, these studies have highlighted that with an increased appreciation for the basic mechanisms deregulating MYC in cancer, new therapeutic vulnerabilities can be discovered and potentially exploited for the inhibition of this potent oncogene in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFN8ZCa1PFjbVg90H21EOLACvtfcHk0ljTRhHfcHZLfg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXpslCmu7g%253D&amp;md5=5060fc3d66f75e698f9c28bcc9947475</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.3390%2Fgenes8060151&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fgenes8060151%26sid%3Dliteratum%253Aachs%26aulast%3DKalkat%26aufirst%3DM.%26aulast%3DDe%2BMelo%26aufirst%3DJ.%26aulast%3DHickman%26aufirst%3DK.%2BA.%26aulast%3DLourenco%26aufirst%3DC.%26aulast%3DRedel%26aufirst%3DC.%26aulast%3DResetca%26aufirst%3DD.%26aulast%3DTamachi%26aufirst%3DA.%26aulast%3DTu%26aufirst%3DW.%2BB.%26aulast%3DPenn%26aufirst%3DL.%2BZ.%26atitle%3DMYC%2520Deregulation%2520in%2520Primary%2520Human%2520Cancers%26jtitle%3DGenes%26date%3D2017%26volume%3D8%26issue%3D6%26spage%3D151%26doi%3D10.3390%2Fgenes8060151" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burge, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Amplification in Tumor Cells with Elevated C-Myc</span>. <i>Cell</i>  <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">67</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2012.08.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.cell.2012.08.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=23021215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsValsLjO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=151&amp;publication_year=2012&amp;pages=56-67&amp;author=C.+Y.+Linauthor=J.+Lovenauthor=P.+B.+Rahlauthor=R.+M.+Paranalauthor=C.+B.+Burgeauthor=J.+E.+Bradnerauthor=T.+I.+Leeauthor=R.+A.+Young&amp;title=Transcriptional+Amplification+in+Tumor+Cells+with+Elevated+C-Myc&amp;doi=10.1016%2Fj.cell.2012.08.026" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional amplification in tumor cells with elevated c-Myc</span></div><div class="casAuthors">Lin, Charles Y.; Loven, Jakob; Rahl, Peter B.; Paranal, Ronald M.; Burge, Christopher B.; Bradner, James E.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-67</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Elevated expression of the c-Myc transcription factor occurs frequently in human cancers and is assocd. with tumor aggression and poor clin. outcome.  The effect of high levels of c-Myc on global gene regulation is poorly understood but is widely thought to involve newly activated or repressed "Myc target genes." We report here that in tumor cells expressing high levels of c-Myc the transcription factor accumulates in the promoter regions of active genes and causes transcriptional amplification, producing increased levels of transcripts within the cell's gene expression program.  Thus, rather than binding and regulating a new set of genes, c-Myc amplifies the output of the existing gene expression program.  These results provide an explanation for the diverse effects of oncogenic c-Myc on gene expression in different tumor cells and suggest that transcriptional amplification reduces rate-limiting constraints for tumor cell growth and proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxyaUKvzZJsLVg90H21EOLACvtfcHk0ljTRhHfcHZLfg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhsValsLjO&amp;md5=9e8be9f929931b0840fca955b45595a1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.08.026&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.08.026%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLoven%26aufirst%3DJ.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DBurge%26aufirst%3DC.%2BB.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DTranscriptional%2520Amplification%2520in%2520Tumor%2520Cells%2520with%2520Elevated%2520C-Myc%26jtitle%3DCell%26date%3D2012%26volume%3D151%26spage%3D56%26epage%3D67%26doi%3D10.1016%2Fj.cell.2012.08.026" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Regulation and Its Misregulation in Disease</span>. <i>Cell</i>  <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>152</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1237</span>– <span class="NLM_lpage">1251</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2013.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.cell.2013.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=23498934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFWiuro%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=152&amp;publication_year=2013&amp;pages=1237-1251&amp;issue=6&amp;author=T.+I.+Leeauthor=R.+A.+Young&amp;title=Transcriptional+Regulation+and+Its+Misregulation+in+Disease&amp;doi=10.1016%2Fj.cell.2013.02.014" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional regulation and its misregulation in disease</span></div><div class="casAuthors">Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1237-1251</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The gene expression programs that establish and maintain specific cell states in humans are controlled by thousands of transcription factors, cofactors, and chromatin regulators.  Misregulation of these gene expression programs can cause a broad range of diseases.  Here, we review recent advances in our understanding of transcriptional regulation and discuss how these have provided new insights into transcriptional misregulation in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRZI_3gsWJi7Vg90H21EOLACvtfcHk0lguGsu8CvP_xw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXktFWiuro%253D&amp;md5=2eabd761a3721c8c509f0fa1a00ceab6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.02.014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DTranscriptional%2520Regulation%2520and%2520Its%2520Misregulation%2520in%2520Disease%26jtitle%3DCell%26date%3D2013%26volume%3D152%26issue%3D6%26spage%3D1237%26epage%3D1251%26doi%3D10.1016%2Fj.cell.2013.02.014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prochownik, E. V.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Inhibitors of the Myc Oncoprotein</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i>  <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1849</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">543</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bbagrm.2014.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.bbagrm.2014.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFGnsL0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1849&amp;publication_year=2015&amp;pages=525-543&amp;issue=5&amp;author=S.+Fletcherauthor=E.+V.+Prochownik&amp;title=Small-Molecule+Inhibitors+of+the+Myc+Oncoprotein&amp;doi=10.1016%2Fj.bbagrm.2014.03.005" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of the Myc oncoprotein</span></div><div class="casAuthors">Fletcher, Steven; Prochownik, Edward V.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1849</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">525-543</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The c-Myc (Myc) oncoprotein is among the most attractive of cancer targets given that it is de-regulated in the majority of tumors and that its inhibition profoundly affects their growth and/or survival.  However, its role as a seldom-mutated transcription factor, its lack of enzymic activity for which suitable pharmaceutical inhibitors could be crafted and its expression by normal cells have largely been responsible for its being viewed as "undruggable".  Work over the past several years, however, has begun to reverse this idea by allowing us to view Myc within the larger context of global gene regulatory control.  Thus, Myc and its obligate heterodimeric partner, Max, are integral to the coordinated recruitment and post-translational modification of components of the core transcriptional machinery.  Moreover, Myc over-expression re-programs numerous crit. cellular functions and alters the cell's susceptibility to their inhibition.  This new knowledge has therefore served as a framework upon which to develop new pharmaceutical approaches.  These include the continuing development of small mols. which act directly to inhibit the crit. Myc-Max interaction, those which act indirectly to prevent Myc-directed post-translational modifications necessary to initiate productive transcription and those which inhibit vital pathways upon which the Myc-transformed cell is particularly reliant.  This article is part of a Special Issue entitled: Myc proteins in cell biol. and pathol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpwveGWUG8ZLVg90H21EOLACvtfcHk0lguGsu8CvP_xw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXntFGnsL0%253D&amp;md5=5f6a99316dc48b9f55796e2f7514cef4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.03.005&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DFletcher%26aufirst%3DS.%26aulast%3DProchownik%26aufirst%3DE.%2BV.%26atitle%3DSmall-Molecule%2520Inhibitors%2520of%2520the%2520Myc%2520Oncoprotein%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2015%26volume%3D1849%26issue%3D5%26spage%3D525%26epage%3D543%26doi%3D10.1016%2Fj.bbagrm.2014.03.005" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boike, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cioffi, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majewski, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Co, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henning, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenna, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Discovery of a Functional Covalent Ligand Targeting an Intrinsically Disordered Cysteine within MYC</span>. <i>Cell Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">10</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.chembiol.2020.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.chembiol.2020.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=33482085" title="">PubMed</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=28&amp;publication_year=2021&amp;pages=1-10&amp;author=L.+Boikeauthor=A.+G.+Cioffiauthor=F.+C.+Majewskiauthor=J.+Coauthor=N.+J.+Henningauthor=M.+D.+Jonesauthor=G.+Liuauthor=J.+M.+McKennaauthor=J.+A.+Tallaricoauthor=M.+Schirle&amp;title=Discovery+of+a+Functional+Covalent+Ligand+Targeting+an+Intrinsically+Disordered+Cysteine+within+MYC&amp;doi=10.1016%2Fj.chembiol.2020.09.001" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.09.001&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DBoike%26aufirst%3DL.%26aulast%3DCioffi%26aufirst%3DA.%2BG.%26aulast%3DMajewski%26aufirst%3DF.%2BC.%26aulast%3DCo%26aufirst%3DJ.%26aulast%3DHenning%26aufirst%3DN.%2BJ.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DMcKenna%26aufirst%3DJ.%2BM.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DSchirle%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520Functional%2520Covalent%2520Ligand%2520Targeting%2520an%2520Intrinsically%2520Disordered%2520Cysteine%2520within%2520MYC%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2021%26volume%3D28%26spage%3D1%26epage%3D10%26doi%3D10.1016%2Fj.chembiol.2020.09.001" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truica, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izquierdo-Ferrer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anker, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lysy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalmers, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unno, K.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Small-Molecule MYC Inhibitors Suppress Tumor Growth and Enhance Immunotherapy</span>. <i>Cancer Cell</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">483</span>– <span class="NLM_lpage">497.e15</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.ccell.2019.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.ccell.2019.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=31679823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2qurrP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=2019&amp;pages=483-497.e15&amp;issue=5&amp;author=H.+Hanauthor=A.+D.+Jainauthor=M.+I.+Truicaauthor=J.+Izquierdo-Ferrerauthor=J.+F.+Ankerauthor=B.+Lysyauthor=V.+Sagarauthor=Y.+Luanauthor=Z.+R.+Chalmersauthor=K.+Unno&amp;title=Small-Molecule+MYC+Inhibitors+Suppress+Tumor+Growth+and+Enhance+Immunotherapy&amp;doi=10.1016%2Fj.ccell.2019.10.001" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule MYC Inhibitors Suppress Tumor Growth and Enhance Immunotherapy</span></div><div class="casAuthors">Han, Huiying; Jain, Atul D.; Truica, Mihai I.; Izquierdo-Ferrer, Javier; Anker, Jonathan F.; Lysy, Barbara; Sagar, Vinay; Luan, Yi; Chalmers, Zachary R.; Unno, Kenji; Mok, Hanlin; Vatapalli, Rajita; Yoo, Young A.; Rodriguez, Yara; Kandela, Irawati; Parker, J. Brandon; Chakravarti, Debabrata; Mishra, Rama K.; Schiltz, Gary E.; Abdulkadir, Sarki A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">483-497.e15</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Small mols. that directly target MYC and are also well tolerated in vivo will provide invaluable chem. probes and potential anti-cancer therapeutic agents.  We developed a series of small-mol. MYC inhibitors that engage MYC inside cells, disrupt MYC/MAX dimers, and impair MYC-driven gene expression.  The compds. enhance MYC phosphorylation on threonine-58, consequently increasing proteasome-mediated MYC degrdn.  The initial lead, MYC inhibitor 361 (MYCi361), suppressed in vivo tumor growth in mice, increased tumor immune cell infiltration, upregulated PD-L1 on tumors, and sensitized tumors to anti-PD1 immunotherapy.  However, 361 demonstrated a narrow therapeutic index.  An improved analog, MYCi975 showed better tolerability.  These findings suggest the potential of small-mol. MYC inhibitors as chem. probes and possible anti-cancer therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeLMWdCetAj7Vg90H21EOLACvtfcHk0liwSevY_koOXA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXitV2qurrP&amp;md5=e341ee827774371b92c0b004c8213af1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.10.001&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DH.%26aulast%3DJain%26aufirst%3DA.%2BD.%26aulast%3DTruica%26aufirst%3DM.%2BI.%26aulast%3DIzquierdo-Ferrer%26aufirst%3DJ.%26aulast%3DAnker%26aufirst%3DJ.%2BF.%26aulast%3DLysy%26aufirst%3DB.%26aulast%3DSagar%26aufirst%3DV.%26aulast%3DLuan%26aufirst%3DY.%26aulast%3DChalmers%26aufirst%3DZ.%2BR.%26aulast%3DUnno%26aufirst%3DK.%26atitle%3DSmall-Molecule%2520MYC%2520Inhibitors%2520Suppress%2520Tumor%2520Growth%2520and%2520Enhance%2520Immunotherapy%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26issue%3D5%26spage%3D483%26epage%3D497.e15%26doi%3D10.1016%2Fj.ccell.2019.10.001" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koehler, A. N.</span></span> <span> </span><span class="NLM_article-title">A Complex Task? Direct Modulation of Transcription Factors with Small Molecules</span>. <i>Curr. Opin. Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">340</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cbpa.2010.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.cbpa.2010.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=20395165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=2010&amp;pages=331-340&amp;issue=3&amp;author=A.+N.+Koehler&amp;title=A+Complex+Task%3F+Direct+Modulation+of+Transcription+Factors+with+Small+Molecules&amp;doi=10.1016%2Fj.cbpa.2010.03.022" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A complex task? Direct modulation of transcription factors with small molecules</span></div><div class="casAuthors">Koehler, Angela N.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">331-340</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Transcription factors with aberrant activity in disease are promising yet untested targets for therapeutic development, particularly in oncol.  Directly inhibiting or activating the function of a transcription factor requires specific disruption or recruitment of protein-protein or protein-DNA interactions.  The discovery or design of small mols. that specifically modulate these interactions has thus far proven to be a significant challenge and the protein class is often perceived to be undruggable.  ' This review will summarize recent progress in the development of small-mol. probes of transcription factors and provide evidence to challenge the notion that this important protein class is chem. intractable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr83_m66fDJsrVg90H21EOLACvtfcHk0liwSevY_koOXA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL0%253D&amp;md5=a3c2dfa1aa4edc7e387b5048c87bf17e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.03.022&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DKoehler%26aufirst%3DA.%2BN.%26atitle%3DA%2520Complex%2520Task%253F%2520Direct%2520Modulation%2520of%2520Transcription%2520Factors%2520with%2520Small%2520Molecules%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26issue%3D3%26spage%3D331%26epage%3D340%26doi%3D10.1016%2Fj.cbpa.2010.03.022" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ulasov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenkranz, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobolev, A. S.</span></span> <span> </span><span class="NLM_article-title">Transcription Factors: Time to Deliver</span>. <i>J. Controlled Release</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>269</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">35</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.jconrel.2017.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.jconrel.2017.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=29113792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslyqsb7O" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=269&amp;publication_year=2018&amp;pages=24-35&amp;author=A.+V.+Ulasovauthor=A.+A.+Rosenkranzauthor=A.+S.+Sobolev&amp;title=Transcription+Factors%3A+Time+to+Deliver&amp;doi=10.1016%2Fj.jconrel.2017.11.004" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factors: Time to deliver</span></div><div class="casAuthors">Ulasov, Alexey V.; Rosenkranz, Andrey A.; Sobolev, Alexander S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Controlled Release</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24-35</span>CODEN:
                <span class="NLM_cas:coden">JCREEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-3659</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Transcription factors (TFs) are at the center of the broad regulatory network orchestrating gene expression programs that elicit different biol. responses.  For a long time, TFs have been considered as potent drug targets due to their implications in the pathogenesis of a variety of diseases.  At the same time, TFs, located at convergence points of cellular regulatory pathways, are powerful tools providing opportunities both for cell type change and for managing the state of cells.  This task formulation requires the TF modulation problem to come to the fore.  We review several ways to manage TF activity (small mols., transfection, nanocarriers, protein-based approaches), analyzing their limitations and the possibilities to overcome them.  Delivery of TFs could revolutionize the biomedical field.  Whether this forecast comes true will depend on the ability to develop convenient technologies for targeted delivery of TFs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWVPaCtQvLB7Vg90H21EOLACvtfcHk0liwSevY_koOXA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhslyqsb7O&amp;md5=eab4552471662dc9cedac533143e6015</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.jconrel.2017.11.004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jconrel.2017.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DUlasov%26aufirst%3DA.%2BV.%26aulast%3DRosenkranz%26aufirst%3DA.%2BA.%26aulast%3DSobolev%26aufirst%3DA.%2BS.%26atitle%3DTranscription%2520Factors%253A%2520Time%2520to%2520Deliver%26jtitle%3DJ.%2520Controlled%2520Release%26date%3D2018%26volume%3D269%26spage%3D24%26epage%3D35%26doi%3D10.1016%2Fj.jconrel.2017.11.004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madden, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Araujo, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerhardt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. M.</span></span> <span> </span><span class="NLM_article-title">Taking the Myc out of Cancer: Toward Therapeutic Strategies to Directly Inhibit c-Myc</span>. <i>Mol. Cancer</i>  <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">18</span>, <span class="refDoi">&nbsp;DOI: 10.1186/s12943-020-01291-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1186%2Fs12943-020-01291-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=33386068" title="">PubMed</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2021&amp;pages=1-18&amp;issue=1&amp;author=S.+K.+Maddenauthor=A.+D.+de+Araujoauthor=M.+Gerhardtauthor=D.+P.+Fairlieauthor=J.+M.+Mason&amp;title=Taking+the+Myc+out+of+Cancer%3A+Toward+Therapeutic+Strategies+to+Directly+Inhibit+c-Myc&amp;doi=10.1186%2Fs12943-020-01291-6" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1186%2Fs12943-020-01291-6&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-020-01291-6%26sid%3Dliteratum%253Aachs%26aulast%3DMadden%26aufirst%3DS.%2BK.%26aulast%3Dde%2BAraujo%26aufirst%3DA.%2BD.%26aulast%3DGerhardt%26aufirst%3DM.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BM.%26atitle%3DTaking%2520the%2520Myc%2520out%2520of%2520Cancer%253A%2520Toward%2520Therapeutic%2520Strategies%2520to%2520Directly%2520Inhibit%2520c-Myc%26jtitle%3DMol.%2520Cancer%26date%3D2021%26volume%3D20%26issue%3D1%26spage%3D1%26epage%3D18%26doi%3D10.1186%2Fs12943-020-01291-6" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Struntz, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deutzmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caballero, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wildschut, M. H. E.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Stabilization of the Max Homodimer with a Small Molecule Attenuates Myc-Driven Transcription</span>. <i>Cell Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">711</span>– <span class="NLM_lpage">723.e14</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.chembiol.2019.02.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.chembiol.2019.02.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30880155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltVSntL4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2019&amp;pages=711-723.e14&amp;issue=5&amp;author=N.+B.+Struntzauthor=A.+Chenauthor=A.+Deutzmannauthor=R.+M.+Wilsonauthor=E.+Stefanauthor=H.+L.+Evansauthor=M.+A.+Ramirezauthor=T.+Liangauthor=F.+Caballeroauthor=M.+H.+E.+Wildschut&amp;title=Stabilization+of+the+Max+Homodimer+with+a+Small+Molecule+Attenuates+Myc-Driven+Transcription&amp;doi=10.1016%2Fj.chembiol.2019.02.009" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of the Max Homodimer with a Small Molecule Attenuates Myc-Driven Transcription</span></div><div class="casAuthors">Struntz, Nicholas B.; Chen, Andrew; Deutzmann, Anja; Wilson, Robert M.; Stefan, Eric; Evans, Helen L.; Ramirez, Maricela A.; Liang, Tong; Caballero, Francisco; Wildschut, Mattheus H. E.; Neel, Dylan V.; Freeman, David B.; Pop, Marius S.; McConkey, Marie; Muller, Sandrine; Curtin, Brice H.; Tseng, Hanna; Frombach, Kristen R.; Butty, Vincent L.; Levine, Stuart S.; Feau, Clementine; Elmiligy, Sarah; Hong, Jiyoung A.; Lewis, Timothy A.; Vetere, Amedeo; Clemons, Paul A.; Malstrom, Scott E.; Ebert, Benjamin L.; Lin, Charles Y.; Felsher, Dean W.; Koehler, Angela N.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">711-723.e14</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The transcription factor Max is a basic-helix-loop-helix leucine zipper (bHLHLZ) protein that forms homodimers or interacts with other bHLHLZ proteins, including Myc and Mxd proteins.  Among this dynamic network of interactions, the Myc/Max heterodimer has crucial roles in regulating normal cellular processes, but its transcriptional activity is deregulated in a majority of human cancers.  Despite this significance, the arsenal of high-quality chem. probes to interrogate these proteins remains limited.  We used small mol. microarrays to identify compds. that bind Max in a mechanistically unbiased manner.  We discovered the asym. polycyclic lactam, KI-MS2-008, which stabilizes the Max homodimer while reducing Myc protein and Myc-regulated transcript levels.  KI-MS2-008 also decreases viable cancer cell growth in a Myc-dependent manner and suppresses tumor growth in vivo.  This approach demonstrates the feasibility of modulating Max with small mols. and supports altering Max dimerization as an alternative approach to targeting Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBFt8AthBLZLVg90H21EOLACvtfcHk0liYAk527P3VNw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXltVSntL4%253D&amp;md5=d02f075ef93868ad66e0ce1329285953</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.02.009&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.02.009%26sid%3Dliteratum%253Aachs%26aulast%3DStruntz%26aufirst%3DN.%2BB.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DDeutzmann%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DR.%2BM.%26aulast%3DStefan%26aufirst%3DE.%26aulast%3DEvans%26aufirst%3DH.%2BL.%26aulast%3DRamirez%26aufirst%3DM.%2BA.%26aulast%3DLiang%26aufirst%3DT.%26aulast%3DCaballero%26aufirst%3DF.%26aulast%3DWildschut%26aufirst%3DM.%2BH.%2BE.%26atitle%3DStabilization%2520of%2520the%2520Max%2520Homodimer%2520with%2520a%2520Small%2520Molecule%2520Attenuates%2520Myc-Driven%2520Transcription%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26issue%3D5%26spage%3D711%26epage%3D723.e14%26doi%3D10.1016%2Fj.chembiol.2019.02.009" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helmer-Citterich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jucker, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesareni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasi, S.</span></span> <span> </span><span class="NLM_article-title">Design and Properties of a Myc Derivative That Efficiently Homodimerizes</span>. <i>Oncogene</i>  <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">2463</span>– <span class="NLM_lpage">2472</span>, <span class="refDoi">&nbsp;DOI: 10.1038/sj.onc.1202199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fsj.onc.1202199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=9824157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADyaK1cXnvVehtbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=1998&amp;pages=2463-2472&amp;issue=19&amp;author=L.+Soucekauthor=M.+Helmer-Citterichauthor=A.+Saccoauthor=R.+Juckerauthor=G.+Cesareniauthor=S.+Nasi&amp;title=Design+and+Properties+of+a+Myc+Derivative+That+Efficiently+Homodimerizes&amp;doi=10.1038%2Fsj.onc.1202199" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Design and properties of a Myc derivative that efficiently homodimerizes</span></div><div class="casAuthors">Soucek, Laura; Helmer-Citterich, Manuela; Sacco, Alessandra; Jucker, Richard; Cesareni, Gianni; Nasi, Sergio</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2463-2472</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">BHLH and bHLHZip are highly conserved structural domains mediating DNA binding and specific protein-protein interactions.  They are present in a family of transcription factors, acting as dimers, and their selective dimerization is utilized to switch on and off cell proliferation, differentiation or apoptosis.  Myc is a bHLHZip protein involved in growth control and cancer, which operates in a network with the structurally related proteins Max, Mad and Mnt.  It does not form homodimers, working as a heterodimer with Max; Max, instead, forms homodimers and heterodimers with Mad and Mnt.  Myc/Max dimers activate gene transcription, while Mad/Max and Mnt/Max complexes are Myc/Max antagonists and act as repressors.  Modifying the mol. recognition of dimers may provide a tool for interfering with Myc function and, in general, for directing the mol. switches operated via bHLH(Zip) proteins.  By mol. modeling and mutagenesis, we analyzed the contribution of single amino acids to the mol. recognition of Myc, creating bHLHZip domains with altered dimerization specificity.  We report that Myc recognition specificity is encoded in a short region within the leucine zipper; mutation of four amino acids generates a protein, Omomyc, that homodimerizes efficiently and can still heterodimerize with wild type Myc and Max.  Omomyc sequestered Myc in complexes with low DNA binding efficiency, preventing binding to Max and inhibiting Myc transcriptional activator function.  Consistently with these results, Omomyc produced a proliferation arrest in NIH3T3 cells.  These data demonstrate the feasibility of interfering with fundamental biol. processes, such as proliferation, by modifying the dimerization selectivity of a bHLHZip protein; this may facilitate the design of peptides of potential pharmacol. interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2ERzL-6wvGLVg90H21EOLACvtfcHk0liYAk527P3VNw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1cXnvVehtbs%253D&amp;md5=cfb371923296046ca4c737c52c56cb2d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1202199&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1202199%26sid%3Dliteratum%253Aachs%26aulast%3DSoucek%26aufirst%3DL.%26aulast%3DHelmer-Citterich%26aufirst%3DM.%26aulast%3DSacco%26aufirst%3DA.%26aulast%3DJucker%26aufirst%3DR.%26aulast%3DCesareni%26aufirst%3DG.%26aulast%3DNasi%26aufirst%3DS.%26atitle%3DDesign%2520and%2520Properties%2520of%2520a%2520Myc%2520Derivative%2520That%2520Efficiently%2520Homodimerizes%26jtitle%3DOncogene%26date%3D1998%26volume%3D17%26issue%3D19%26spage%3D2463%26epage%3D2472%26doi%3D10.1038%2Fsj.onc.1202199" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massó-Vallés, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Blocking Myc to Treat Cancer: Reflecting on Two Decades of Omomyc</span>. <i>Cells</i>  <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">883</span>, <span class="refDoi">&nbsp;DOI: 10.3390/cells9040883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.3390%2Fcells9040883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1ChsrnO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2020&amp;pages=883&amp;issue=4&amp;author=D.+Mass%C3%B3-Vall%C3%A9sauthor=L.+Soucek&amp;title=Blocking+Myc+to+Treat+Cancer%3A+Reflecting+on+Two+Decades+of+Omomyc&amp;doi=10.3390%2Fcells9040883" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Blocking Myc to treat cancer: reflecting on two decades of Omomyc</span></div><div class="casAuthors">Masso-Valles, Daniel; Soucek, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Cells</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">883</span>CODEN:
                <span class="NLM_cas:coden">CELLC6</span>;
        ISSN:<span class="NLM_cas:issn">2073-4409</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">First designed and published in 1998 as a lab. tool to study Myc perturbation, Omomyc has come a long way in the past 22 years.  This dominant neg. has contributed to our understanding of Myc biol. when expressed, first, in normal and cancer cells, and later in genetically-engineered mice, and has shown remarkable anti-cancer properties in a wide range of tumor types.  The recently described therapeutic effect of purified Omomyc mini-protein-following the surprising discovery of its cell-penetrating capacity-constitutes a paradigm shift.  Now, much more than a proof of concept, the most characterized Myc inhibitor to date is advancing in its drug development pipeline, pushing Myc inhibition into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxDW79cqSCgLVg90H21EOLACvtfcHk0ljEnfyzLqOJ-g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3cXis1ChsrnO&amp;md5=57cd831e89fe90d06629729d11f63a79</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.3390%2Fcells9040883&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcells9040883%26sid%3Dliteratum%253Aachs%26aulast%3DMass%25C3%25B3-Vall%25C3%25A9s%26aufirst%3DD.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DBlocking%2520Myc%2520to%2520Treat%2520Cancer%253A%2520Reflecting%2520on%2520Two%2520Decades%2520of%2520Omomyc%26jtitle%3DCells%26date%3D2020%26volume%3D9%26issue%3D4%26spage%3D883%26doi%3D10.3390%2Fcells9040883" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beaulieu, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jauset, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massó-Vallés, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Martín, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maltais, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zacarias-Fluck, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casacuberta-Serra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Pozo, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiore, C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Intrinsic Cell-Penetrating Activity Propels Omomyc from Proof of Concept to Viable Anti-Myc Therapy</span>. <i>Sci. Transl. Med.</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">484</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">14</span>, <span class="refDoi">&nbsp;DOI: 10.1126/scitranslmed.aar5012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1126%2Fscitranslmed.aar5012" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2019&amp;pages=1-14&amp;issue=484&amp;author=M.+E.+Beaulieuauthor=T.+Jausetauthor=D.+Mass%C3%B3-Vall%C3%A9sauthor=S.+Mart%C3%ADnez-Mart%C3%ADnauthor=P.+Rahlauthor=L.+Maltaisauthor=M.+F.+Zacarias-Fluckauthor=S.+Casacuberta-Serraauthor=E.+S.+Del+Pozoauthor=C.+Fiore&amp;title=Intrinsic+Cell-Penetrating+Activity+Propels+Omomyc+from+Proof+of+Concept+to+Viable+Anti-Myc+Therapy&amp;doi=10.1126%2Fscitranslmed.aar5012" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aar5012&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aar5012%26sid%3Dliteratum%253Aachs%26aulast%3DBeaulieu%26aufirst%3DM.%2BE.%26aulast%3DJauset%26aufirst%3DT.%26aulast%3DMass%25C3%25B3-Vall%25C3%25A9s%26aufirst%3DD.%26aulast%3DMart%25C3%25ADnez-Mart%25C3%25ADn%26aufirst%3DS.%26aulast%3DRahl%26aufirst%3DP.%26aulast%3DMaltais%26aufirst%3DL.%26aulast%3DZacarias-Fluck%26aufirst%3DM.%2BF.%26aulast%3DCasacuberta-Serra%26aufirst%3DS.%26aulast%3DDel%2BPozo%26aufirst%3DE.%2BS.%26aulast%3DFiore%26aufirst%3DC.%26atitle%3DIntrinsic%2520Cell-Penetrating%2520Activity%2520Propels%2520Omomyc%2520from%2520Proof%2520of%2520Concept%2520to%2520Viable%2520Anti-Myc%2520Therapy%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2019%26volume%3D11%26issue%3D484%26spage%3D1%26epage%3D14%26doi%3D10.1126%2Fscitranslmed.aar5012" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demma, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruprecht, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javaid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiswell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muise, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zelina, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Omomyc Reveals New Mechanisms To Inhibit the MYC Oncogene</span>. <i>Mol. Cell. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">27</span>, <span class="refDoi">&nbsp;DOI: 10.1128/MCB.00248-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1128%2FMCB.00248-19" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=2019&amp;pages=1-27&amp;issue=22&amp;author=M.+J.+Demmaauthor=C.+Mapelliauthor=A.+Sunauthor=S.+Bodeaauthor=B.+Ruprechtauthor=S.+Javaidauthor=D.+Wiswellauthor=E.+Muiseauthor=S.+Chenauthor=J.+Zelina&amp;title=Omomyc+Reveals+New+Mechanisms+To+Inhibit+the+MYC+Oncogene&amp;doi=10.1128%2FMCB.00248-19" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1128%2FMCB.00248-19&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00248-19%26sid%3Dliteratum%253Aachs%26aulast%3DDemma%26aufirst%3DM.%2BJ.%26aulast%3DMapelli%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DA.%26aulast%3DBodea%26aufirst%3DS.%26aulast%3DRuprecht%26aufirst%3DB.%26aulast%3DJavaid%26aufirst%3DS.%26aulast%3DWiswell%26aufirst%3DD.%26aulast%3DMuise%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DZelina%26aufirst%3DJ.%26atitle%3DOmomyc%2520Reveals%2520New%2520Mechanisms%2520To%2520Inhibit%2520the%2520MYC%2520Oncogene%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2019%26volume%3D39%26issue%3D22%26spage%3D1%26epage%3D27%26doi%3D10.1128%2FMCB.00248-19" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lobba, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fellmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmelstein, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maza, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kissman, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staahl, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urnes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lew, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogilevsky, C. S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Site-Specific Bioconjugation through Enzyme-Catalyzed Tyrosine–Cysteine Bond Formation</span>. <i>ACS Cent. Sci.</i>  <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1564</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscentsci.0c00940</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.0c00940" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1eltL3P" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2020&amp;pages=1564&amp;author=M.+J.+Lobbaauthor=C.+Fellmannauthor=A.+M.+Marmelsteinauthor=J.+C.+Mazaauthor=E.+N.+Kissmanauthor=S.+A.+Robinsonauthor=B.+T.+Staahlauthor=C.+Urnesauthor=R.+J.+Lewauthor=C.+S.+Mogilevsky&amp;title=Site-Specific+Bioconjugation+through+Enzyme-Catalyzed+Tyrosine%E2%80%93Cysteine+Bond+Formation&amp;doi=10.1021%2Facscentsci.0c00940" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Site-Specific Bioconjugation through Enzyme-Catalyzed Tyrosine-Cysteine Bond Formation</span></div><div class="casAuthors">Lobba, Marco J.; Fellmann, Christof; Marmelstein, Alan M.; Maza, Johnathan C.; Kissman, Elijah N.; Robinson, Stephanie A.; Staahl, Brett T.; Urnes, Cole; Lew, Rachel J.; Mogilevsky, Casey S.; Doudna, Jennifer A.; Francis, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1564-1571</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of protein-protein and protein-peptide conjugates is an important capability for producing vaccines, immunotherapeutics, and targeted delivery agents.  Herein we show that the enzyme tyrosinase is capable of oxidizing exposed tyrosine residues into o-quinones that react rapidly with cysteine residues on target proteins.  This coupling reaction occurs under mild aerobic conditions and has the rare ability to join full-size proteins in under 2 h.  The utility of the approach is demonstrated for the attachment of cationic peptides to enhance the cellular delivery of CRISPR-Cas9 20-fold and for the coupling of reporter proteins to a cancer-targeting antibody fragment without loss of its cell-specific binding ability.  The broad applicability of this technique provides a new building block approach for the synthesis of protein chimeras.  Enzymic oxidn. of tyrosine residues followed by reaction with cysteine thiols allows the covalent coupling of proteins and peptides using only native amino acid side chains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTq03w1W3XmLVg90H21EOLACvtfcHk0lh8p0cl-SErnw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3cXhs1eltL3P&amp;md5=a0fe6337ef9daa4caa637a7ba54f9d68</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.0c00940&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.0c00940%26sid%3Dliteratum%253Aachs%26aulast%3DLobba%26aufirst%3DM.%2BJ.%26aulast%3DFellmann%26aufirst%3DC.%26aulast%3DMarmelstein%26aufirst%3DA.%2BM.%26aulast%3DMaza%26aufirst%3DJ.%2BC.%26aulast%3DKissman%26aufirst%3DE.%2BN.%26aulast%3DRobinson%26aufirst%3DS.%2BA.%26aulast%3DStaahl%26aufirst%3DB.%2BT.%26aulast%3DUrnes%26aufirst%3DC.%26aulast%3DLew%26aufirst%3DR.%2BJ.%26aulast%3DMogilevsky%26aufirst%3DC.%2BS.%26atitle%3DSite-Specific%2520Bioconjugation%2520through%2520Enzyme-Catalyzed%2520Tyrosine%25E2%2580%2593Cysteine%2520Bond%2520Formation%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2020%26volume%3D6%26spage%3D1564%26doi%3D10.1021%2Facscentsci.0c00940" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhanjee, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saebi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buslov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loftis, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span> <span> </span><span class="NLM_article-title">Protein-Protein Cross-Coupling via Palladium-Protein Oxidative Addition Complexes from Cysteine Residues</span>. <i>J. Am. Chem. Soc.</i>  <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">9124</span>– <span class="NLM_lpage">9129</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.0c03143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.0c03143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosF2gtL4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=142&amp;publication_year=2020&amp;pages=9124-9129&amp;issue=20&amp;author=H.+H.+Dhanjeeauthor=A.+Saebiauthor=I.+Buslovauthor=A.+R.+Loftisauthor=S.+L.+Buchwaldauthor=B.+L.+Pentelute&amp;title=Protein-Protein+Cross-Coupling+via+Palladium-Protein+Oxidative+Addition+Complexes+from+Cysteine+Residues&amp;doi=10.1021%2Fjacs.0c03143" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-Protein Cross-Coupling via Palladium-Protein Oxidative Addition Complexes from Cysteine Residues</span></div><div class="casAuthors">Dhanjee, Heemal H.; Saebi, Azin; Buslov, Ivan; Loftis, Alexander R.; Buchwald, Stephen L.; Pentelute, Bradley L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9124-9129</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Few chem. methods exist for the covalent conjugation of two proteins.  The authors report the prepn. of site-specific protein-protein conjugates that arise from the sequential cross-coupling of cysteine residues on two different proteins.  The method involves the synthesis of stable palladium-protein oxidative addn. complexes (Pd-protein OACs), a process that converts nucleophilic cysteine residues into an electrophilic S-aryl-Pd-X unit by taking advantage of an intramol. oxidative addn. strategy.  This process is demonstrated on proteins up to 83 kDa in size and can be conveniently carried out in water and open to air.  The resulting Pd-protein OACs can cross-couple with other thiol-contg. proteins to arrive at homogeneous protein-protein bioconjugates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLCs1S4RjC0bVg90H21EOLACvtfcHk0lh8p0cl-SErnw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3cXosF2gtL4%253D&amp;md5=5944224f692948db7ba7f1bf3a557172</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjacs.0c03143&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.0c03143%26sid%3Dliteratum%253Aachs%26aulast%3DDhanjee%26aufirst%3DH.%2BH.%26aulast%3DSaebi%26aufirst%3DA.%26aulast%3DBuslov%26aufirst%3DI.%26aulast%3DLoftis%26aufirst%3DA.%2BR.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26atitle%3DProtein-Protein%2520Cross-Coupling%2520via%2520Palladium-Protein%2520Oxidative%2520Addition%2520Complexes%2520from%2520Cysteine%2520Residues%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26issue%3D20%26spage%3D9124%26epage%3D9129%26doi%3D10.1021%2Fjacs.0c03143" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, K. S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spasser, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlich, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bavikar, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brik, A.</span></span> <span> </span><span class="NLM_article-title">Total Chemical Synthesis of Di-Ubiquitin Chains</span>. <i>Angew. Chem., Int. Ed.</i>  <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span> (<span class="NLM_issue">48</span>),  <span class="NLM_fpage">9126</span>– <span class="NLM_lpage">9131</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.201003763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1002%2Fanie.201003763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVCjsLfN" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=49&amp;publication_year=2010&amp;pages=9126-9131&amp;issue=48&amp;author=K.+S.+A.+Kumarauthor=L.+Spasserauthor=L.+A.+Erlichauthor=S.+N.+Bavikarauthor=A.+Brik&amp;title=Total+Chemical+Synthesis+of+Di-Ubiquitin+Chains&amp;doi=10.1002%2Fanie.201003763" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Total Chemical Synthesis of Di-ubiquitin Chains</span></div><div class="casAuthors">Kumar, K. S. Ajish; Spasser, Liat; Erlich, Lesly A.; Bavikar, Sudhir N.; Brik, Ashraf</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">9126-9131, S9126/1-S9126/20</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH &amp; Co. KGaA</span>)
        </div><div class="casAbstract">The authors report the first total chem. synthesis of all the di-ubiquitin (di-Ub) chains by using δ-mercaptolysine-contg. peptides, Ub peptide thioesters and native chem. ligation methodol.  δ-Mercaptolysine was introduced to mediate the transthioesterification step of the Ub thioester and subsequent S→N acyl transfer to form the isopeptide bond.  These analogs displayed similar folding to Ub monomer, and were tested for their activities with the human isopeptidase T (IsoT).  Peptides UbK48-Ub, UbK63-Ub, UbK33-Ub, UbK29-Ub, UbK11-Ub, and UbK46-Ub were found to be substrates of this enzyme.  Furthermore, UbK46-Ub was the preferred substrate of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-sj-jsyxmALVg90H21EOLACvtfcHk0lj9VIbf8IzCbQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXhsVCjsLfN&amp;md5=a14b34f4be34ee4dac4d02b37780e167</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201003763&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201003763%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DK.%2BS.%2BA.%26aulast%3DSpasser%26aufirst%3DL.%26aulast%3DErlich%26aufirst%3DL.%2BA.%26aulast%3DBavikar%26aufirst%3DS.%2BN.%26aulast%3DBrik%26aufirst%3DA.%26atitle%3DTotal%2520Chemical%2520Synthesis%2520of%2520Di-Ubiquitin%2520Chains%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26issue%3D48%26spage%3D9126%26epage%3D9131%26doi%3D10.1002%2Fanie.201003763" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinty, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellois, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muir, T. W.</span></span> <span> </span><span class="NLM_article-title">Auxiliary-Mediated Site-Specific Peptide Ubiquitylation</span>. <i>Angew. Chem., Int. Ed.</i>  <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>119</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">2872</span>– <span class="NLM_lpage">2876</span>, <span class="refDoi">&nbsp;DOI: 10.1002/ange.200605155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1002%2Fange.200605155" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=119&amp;publication_year=2007&amp;pages=2872-2876&amp;issue=16&amp;author=C.+Chatterjeeauthor=R.+K.+McGintyauthor=J.-P.+Pelloisauthor=T.+W.+Muir&amp;title=Auxiliary-Mediated+Site-Specific+Peptide+Ubiquitylation&amp;doi=10.1002%2Fange.200605155" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fange.200605155&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fange.200605155%26sid%3Dliteratum%253Aachs%26aulast%3DChatterjee%26aufirst%3DC.%26aulast%3DMcGinty%26aufirst%3DR.%2BK.%26aulast%3DPellois%26aufirst%3DJ.-P.%26aulast%3DMuir%26aufirst%3DT.%2BW.%26atitle%3DAuxiliary-Mediated%2520Site-Specific%2520Peptide%2520Ubiquitylation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2007%26volume%3D119%26issue%3D16%26spage%3D2872%26epage%3D2876%26doi%3D10.1002%2Fange.200605155" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ajish
Kumar, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haj-Yahya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olschewski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lashuel, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brik, A.</span></span> <span> </span><span class="NLM_article-title">Highly Efficient and Chemoselective Peptide Ubiquitylation</span>. <i>Angew. Chem., Int. Ed.</i>  <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">43</span>),  <span class="NLM_fpage">8090</span>– <span class="NLM_lpage">8094</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.200902936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1002%2Fanie.200902936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Kju77N" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=48&amp;publication_year=2009&amp;pages=8090-8094&amp;issue=43&amp;author=K.+S.+Ajish%0AKumarauthor=M.+Haj-Yahyaauthor=D.+Olschewskiauthor=H.+A.+Lashuelauthor=A.+Brik&amp;title=Highly+Efficient+and+Chemoselective+Peptide+Ubiquitylation&amp;doi=10.1002%2Fanie.200902936" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Efficient and Chemoselective Peptide Ubiquitylation</span></div><div class="casAuthors">Ajish Kumar, K. S.; Haj-Yahya, Mahmood; Olschewski, Diana; Lashuel, Hilal A.; Brik, Ashraf</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">8090-8094, S8090/1-S8090/31</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH &amp; Co. KGaA</span>)
        </div><div class="casAbstract">The authors synthesized δ-mercaptolysine and incorporated into a model peptide [α-synuclein(1-17); MDVFMKGLSKAKEGVVA; where Met was replaced by Nle to avoid any oxidn.] to allow a thiol capture step with ubiquitin thioester.  Thus, during ubiquitination of the model peptide, the thiol functionality of δ-mercaptolysine serves as a temporary handle in assisting the isopeptide formation.  Following the native chem. ligation step, the thiol moiety of δ-mercaptolysine is removed by desulfurization reaction, thereby converting the said amino acid into lysine and resulting in the formation of an isopeptide, where ubiquitin is linked to the model peptide via ε-amino group of lysine.  In combination with native chem. ligation, this strategy should facilitate the synthesis of ubiquitylated proteins for biochem. and structural studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGidcttT7KBLVg90H21EOLACvtfcHk0lj9VIbf8IzCbQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXht1Kju77N&amp;md5=fe982bd9bfe9944599cb4eac83663082</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.200902936&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200902936%26sid%3Dliteratum%253Aachs%26aulast%3DAjish%2BKumar%26aufirst%3DK.%2BS.%26aulast%3DHaj-Yahya%26aufirst%3DM.%26aulast%3DOlschewski%26aufirst%3DD.%26aulast%3DLashuel%26aufirst%3DH.%2BA.%26aulast%3DBrik%26aufirst%3DA.%26atitle%3DHighly%2520Efficient%2520and%2520Chemoselective%2520Peptide%2520Ubiquitylation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26issue%3D43%26spage%3D8090%26epage%3D8094%26doi%3D10.1002%2Fanie.200902936" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fottner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittl, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn-Ghetko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jussupow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaila, V. R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bremm, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, K.</span></span> <span> </span><span class="NLM_article-title">Site-Specific Ubiquitylation and SUMOylation Using Genetic-Code Expansion and Sortase</span>. <i>Nat. Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">284</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41589-019-0227-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fs41589-019-0227-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30770915" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsFSls7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=2019&amp;pages=276-284&amp;issue=3&amp;author=M.+Fottnerauthor=A.+D.+Brunnerauthor=V.+Bittlauthor=D.+Horn-Ghetkoauthor=A.+Jussupowauthor=V.+R.+I.+Kailaauthor=A.+Bremmauthor=K.+Lang&amp;title=Site-Specific+Ubiquitylation+and+SUMOylation+Using+Genetic-Code+Expansion+and+Sortase&amp;doi=10.1038%2Fs41589-019-0227-4" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Site-specific ubiquitylation and SUMOylation using genetic-code expansion and sortase</span></div><div class="casAuthors">Fottner, Maximilian; Brunner, Andreas-David; Bittl, Verena; Horn-Ghetko, Daniel; Jussupow, Alexander; Kaila, Ville R. I.; Bremm, Anja; Lang, Kathrin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">276-284</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Post-translational modification of proteins with ubiquitin and ubiquitin-like proteins (Ubls) is central to the regulation of eukaryotic cellular processes.  The authors' ability to study the effects of ubiquitylation, however, is limited by the difficulty to prep. homogenously modified proteins in vitro and by the impossibility to selectively trigger specific ubiquitylation events in living cells.  Here the authors combine genetic-code expansion, bioorthogonal Staudinger redn. and sortase-mediated transpeptidation to develop a general tool to ubiquitylate proteins in an inducible fashion.  The generated ubiquitin conjugates display a native isopeptide bond and bear two point mutations in the ubiquitin C terminus that confer resistance toward deubiquitinases.  Nevertheless, physiol. integrity of sortase-generated diubiquitins in decoding cellular functions via recognition by ubiquitin-binding domains is retained.  The authors' approach allows the site-specific attachment of Ubls to nonrefoldable, multidomain proteins and enables inducible and ubiquitin-ligase-independent ubiquitylation of proteins in mammalian cells, providing a powerful tool to dissect the biol. functions of ubiquitylation with temporal control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY1Qlwj1GxIrVg90H21EOLACvtfcHk0lj9VIbf8IzCbQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXnsFSls7c%253D&amp;md5=93007ea246ce639f169661a4cb962228</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0227-4&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0227-4%26sid%3Dliteratum%253Aachs%26aulast%3DFottner%26aufirst%3DM.%26aulast%3DBrunner%26aufirst%3DA.%2BD.%26aulast%3DBittl%26aufirst%3DV.%26aulast%3DHorn-Ghetko%26aufirst%3DD.%26aulast%3DJussupow%26aufirst%3DA.%26aulast%3DKaila%26aufirst%3DV.%2BR.%2BI.%26aulast%3DBremm%26aufirst%3DA.%26aulast%3DLang%26aufirst%3DK.%26atitle%3DSite-Specific%2520Ubiquitylation%2520and%2520SUMOylation%2520Using%2520Genetic-Code%2520Expansion%2520and%2520Sortase%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26issue%3D3%26spage%3D276%26epage%3D284%26doi%3D10.1038%2Fs41589-019-0227-4" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span> <span> </span><span class="NLM_article-title">Development and Application of Ubiquitin-Based Chemical Probes</span>. <i>Chem. Sci.</i>  <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">47</span>),  <span class="NLM_fpage">12633</span>– <span class="NLM_lpage">12646</span>, <span class="refDoi">&nbsp;DOI: 10.1039/D0SC03295F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1039%2FD0SC03295F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=34123237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFWhsr7L" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2020&amp;pages=12633-12646&amp;issue=47&amp;author=X.+Suiauthor=Y.+Wangauthor=Y.+X.+Duauthor=L.+J.+Liangauthor=Q.+Zhengauthor=Y.+M.+Liauthor=L.+Liu&amp;title=Development+and+Application+of+Ubiquitin-Based+Chemical+Probes&amp;doi=10.1039%2FD0SC03295F" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Development and application of ubiquitin-based chemical probes</span></div><div class="casAuthors">Sui, Xin; Wang, Yu; Du, Yun-Xiang; Liang, Lu-Jun; Zheng, Qingyun; Li, Yi-Ming; Liu, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">12633-12646</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Protein ubiquitination regulates almost every process in eukaryotic cells.  The study of the many enzymes involved in the ubiquitination system and the development of ubiquitination-assocd. therapeutics are important areas of current research.  Synthetic tools such as ubiquitin-based chem. probes have been making an increasing contribution to deciphering various biochem. components involved in ubiquitin conjugation, recruitment, signaling, and deconjugation.  In the present minireview, we summarize the progress of ubiquitin-based chem. probes with an emphasis on their various structures and chem. synthesis.  We discuss the utility of the ubiquitin-based chem. probes for discovering and profiling ubiquitin-dependent signaling systems, as well as the monitoring and visualization of ubiquitin-related enzymic machinery.  We also show how the probes can serve to elucidate the mol. mechanism of recognition and catalysis.  Collectively, the development and application of ubiquitin-based chem. probes emphasizes the importance and utility of chem. protein synthesis in modern chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGjlv2fA9kXbVg90H21EOLACvtfcHk0lh_cXCT7nZIOw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3cXhsFWhsr7L&amp;md5=435454dd2d00472462c53bc111dd5820</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1039%2FD0SC03295F&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FD0SC03295F%26sid%3Dliteratum%253Aachs%26aulast%3DSui%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DY.%2BX.%26aulast%3DLiang%26aufirst%3DL.%2BJ.%26aulast%3DZheng%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DY.%2BM.%26aulast%3DLiu%26aufirst%3DL.%26atitle%3DDevelopment%2520and%2520Application%2520of%2520Ubiquitin-Based%2520Chemical%2520Probes%26jtitle%3DChem.%2520Sci.%26date%3D2020%26volume%3D11%26issue%3D47%26spage%3D12633%26epage%3D12646%26doi%3D10.1039%2FD0SC03295F" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Geurink, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Oualid, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hameed, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovaa, H.</span></span> <span> </span><span class="NLM_article-title">A General Chemical Ligation Approach Towards Isopeptide-Linked Ubiquitin and Ubiquitin-Like Assay Reagents</span>. <i>ChemBioChem</i>  <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">297</span>, <span class="refDoi">&nbsp;DOI: 10.1002/cbic.201100706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1002%2Fcbic.201100706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=22213387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1OiurrJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=13&amp;publication_year=2012&amp;pages=293-297&amp;issue=2&amp;author=P.+P.+Geurinkauthor=F.+El+Oualidauthor=A.+Jonkerauthor=D.+S.+Hameedauthor=H.+Ovaa&amp;title=A+General+Chemical+Ligation+Approach+Towards+Isopeptide-Linked+Ubiquitin+and+Ubiquitin-Like+Assay+Reagents&amp;doi=10.1002%2Fcbic.201100706" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A General Chemical Ligation Approach Towards Isopeptide-Linked Ubiquitin and Ubiquitin-Like Assay Reagents</span></div><div class="casAuthors">Geurink, Paul P.; El Oualid, Farid; Jonker, Anika; Hameed, Dharjath S.; Ovaa, Huib</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">293-297</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH &amp; Co. KGaA</span>)
        </div><div class="casAbstract">The 5-thiolysine-mediated ligation can be used to generate a wide range of isopeptide-linked ubiquitin Ub(I)-based FP (fluorescence anisotropy/flourescence polarization) assay reagents.  These are powerful tools for measuring Ub(I) deconjugating activity.  The methodol. can be adapted to any ubiquitin-like proteins (Ubl) for which the E1 enzyme is available.  In contrast to any other reagent, it is possible to incorporate substrate-based peptide sequences.  This is a major advantage, as it offers the possibility of more physiol. relevant assay reagents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqou4rjkVgI1rVg90H21EOLACvtfcHk0lh_cXCT7nZIOw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXhs1OiurrJ&amp;md5=4c7d4048b95b1218a340123dd71d9716</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201100706&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201100706%26sid%3Dliteratum%253Aachs%26aulast%3DGeurink%26aufirst%3DP.%2BP.%26aulast%3DEl%2BOualid%26aufirst%3DF.%26aulast%3DJonker%26aufirst%3DA.%26aulast%3DHameed%26aufirst%3DD.%2BS.%26aulast%3DOvaa%26aufirst%3DH.%26atitle%3DA%2520General%2520Chemical%2520Ligation%2520Approach%2520Towards%2520Isopeptide-Linked%2520Ubiquitin%2520and%2520Ubiquitin-Like%2520Assay%2520Reagents%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26issue%3D2%26spage%3D293%26epage%3D297%26doi%3D10.1002%2Fcbic.201100706" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Premdjee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, R. J.</span></span> <span> </span><span class="NLM_article-title">Rapid and Efficient Protein Synthesis through Expansion of the Native Chemical Ligation Concept</span>. <i>Nat. Rev. Chem.</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">17</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41570-018-0122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fs41570-018-0122" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2&amp;publication_year=2018&amp;pages=1-17&amp;issue=4&amp;author=S.+S.+Kulkarniauthor=J.+Sayersauthor=B.+Premdjeeauthor=R.+J.+Payne&amp;title=Rapid+and+Efficient+Protein+Synthesis+through+Expansion+of+the+Native+Chemical+Ligation+Concept&amp;doi=10.1038%2Fs41570-018-0122" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fs41570-018-0122&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41570-018-0122%26sid%3Dliteratum%253Aachs%26aulast%3DKulkarni%26aufirst%3DS.%2BS.%26aulast%3DSayers%26aufirst%3DJ.%26aulast%3DPremdjee%26aufirst%3DB.%26aulast%3DPayne%26aufirst%3DR.%2BJ.%26atitle%3DRapid%2520and%2520Efficient%2520Protein%2520Synthesis%2520through%2520Expansion%2520of%2520the%2520Native%2520Chemical%2520Ligation%2520Concept%26jtitle%3DNat.%2520Rev.%2520Chem.%26date%3D2018%26volume%3D2%26issue%3D4%26spage%3D1%26epage%3D17%26doi%3D10.1038%2Fs41570-018-0122" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Q.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Quasi-Racemic X-Ray Structures of K27-Linked Ubiquitin Chains Prepared by Total Chemical Synthesis</span>. <i>J. Am. Chem. Soc.</i>  <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">7429</span>– <span class="NLM_lpage">7435</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.6b04031</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b04031" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC28XptFWmsbg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=138&amp;publication_year=2016&amp;pages=7429-7435&amp;issue=23&amp;author=M.+Panauthor=S.+Gaoauthor=Y.+Zhengauthor=X.+Tanauthor=H.+Lanauthor=X.+Tanauthor=D.+Sunauthor=L.+Luauthor=T.+Wangauthor=Q.+Zheng&amp;title=Quasi-Racemic+X-Ray+Structures+of+K27-Linked+Ubiquitin+Chains+Prepared+by+Total+Chemical+Synthesis&amp;doi=10.1021%2Fjacs.6b04031" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Quasi-Racemic X-ray Structures of K27-Linked Ubiquitin Chains Prepared by Total Chemical Synthesis</span></div><div class="casAuthors">Pan, Man; Gao, Shuai; Zheng, Yong; Tan, Xiaodan; Lan, Huan; Tan, Xianglong; Sun, Demeng; Lu, Lining; Wang, Tian; Zheng, Qingyun; Huang, Yichao; Wang, Jiawei; Liu, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7429-7435</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Quasi-racemic crystallog. has been used to det. the X-ray structures of K27-linked ubiquitin (Ub) chains prepd. through total chem. synthesis.  Crystal structures of K27-linked di- and tri-ubiquitins reveal that the isopeptide linkages are confined in a unique buried conformation, which provides the mol. basis for the distinctive function of K27 linkage compared to the other seven Ub chains.  K27-linked di- and triUb were found to adopt different structural conformations in the crystals, one being sym. whereas the other triangular.  Furthermore, bioactivity expts. showed that the ovarian tumor family de-ubiquitinase 2 significantly favors K27-linked triUb than K27-linked diUb.  K27-linked triUb represents the so-far largest chem. synthesized protein (228 amino acids) that has been crystd. to afford a high-resoln. X-ray structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovteTSewd5SLVg90H21EOLACvtfcHk0lh_cXCT7nZIOw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XptFWmsbg%253D&amp;md5=43d254b471483350485743a347a635ac</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b04031&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b04031%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DM.%26aulast%3DGao%26aufirst%3DS.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DX.%26aulast%3DLan%26aufirst%3DH.%26aulast%3DTan%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DQ.%26atitle%3DQuasi-Racemic%2520X-Ray%2520Structures%2520of%2520K27-Linked%2520Ubiquitin%2520Chains%2520Prepared%2520by%2520Total%2520Chemical%2520Synthesis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26issue%3D23%26spage%3D7429%26epage%3D7435%26doi%3D10.1021%2Fjacs.6b04031" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jbara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laps, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamnesky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolberger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brik, A.</span></span> <span> </span><span class="NLM_article-title">Palladium Prompted On-Demand Cysteine Chemistry for the Synthesis of Challenging and Uniquely Modified Proteins</span>. <i>Nat. Commun.</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">3154</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41467-018-05628-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fs41467-018-05628-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30089783" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A280%3ADC%252BB3c7mtlSrsA%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2018&amp;pages=3154&amp;issue=1&amp;author=M.+Jbaraauthor=S.+Lapsauthor=M.+Morganauthor=G.+Kamneskyauthor=G.+Mannauthor=C.+Wolbergerauthor=A.+Brik&amp;title=Palladium+Prompted+On-Demand+Cysteine+Chemistry+for+the+Synthesis+of+Challenging+and+Uniquely+Modified+Proteins&amp;doi=10.1038%2Fs41467-018-05628-0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium prompted on-demand cysteine chemistry for the synthesis of challenging and uniquely modified proteins</span></div><div class="casAuthors">Jbara Muhammad; Laps Shay; Kamnesky Guy; Mann Guy; Brik Ashraf; Morgan Michael; Wolberger Cynthia</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3154</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Organic chemistry allows for the modification and chemical preparation of protein analogues for various studies.  The thiolate side chain of the Cys residue has been a key functionality in these ventures.  In order to generate complex molecular targets, there is a particular need to incorporate orthogonal protecting groups of the thiolated amino acids to control the directionality of synthesis and modification site.  Here, we demonstrate the tuning of palladium chemoselectivity in aqueous medium for on-demand deprotection of several Cys-protecting groups that are useful in protein synthesis and modification.  These tools allow the preparation of highly complex analogues as we demonstrate in the synthesis of the copper storage protein and selectively modified peptides with multiple Cys residues.  We also report the synthesis of an activity-based probe comprising ubiquitinated histone H2A and its incorporation into nucleosomes and demonstrate its reactivity with deubiquitinating enzyme to generate a covalent nucleosome-enzyme complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQgWwVK8QQFg6-9TkgmBqTqfW6udTcc2eY0k-wgmeraRrntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BB3c7mtlSrsA%253D%253D&amp;md5=8e298b89bb60f3f1a3c85f7880c36ae0</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-05628-0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-05628-0%26sid%3Dliteratum%253Aachs%26aulast%3DJbara%26aufirst%3DM.%26aulast%3DLaps%26aufirst%3DS.%26aulast%3DMorgan%26aufirst%3DM.%26aulast%3DKamnesky%26aufirst%3DG.%26aulast%3DMann%26aufirst%3DG.%26aulast%3DWolberger%26aufirst%3DC.%26aulast%3DBrik%26aufirst%3DA.%26atitle%3DPalladium%2520Prompted%2520On-Demand%2520Cysteine%2520Chemistry%2520for%2520the%2520Synthesis%2520of%2520Challenging%2520and%2520Uniquely%2520Modified%2520Proteins%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26issue%3D1%26spage%3D3154%26doi%3D10.1038%2Fs41467-018-05628-0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Torbeev, V. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raghuraman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamelberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westler, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perozo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, S. B. H.</span></span> <span> </span><span class="NLM_article-title">Protein Conformational Dynamics in the Mechanism of HIV-1 Protease Catalysis</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i>  <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span> (<span class="NLM_issue">52</span>),  <span class="NLM_fpage">20982</span>– <span class="NLM_lpage">20987</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1111202108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1073%2Fpnas.1111202108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=22158985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvVCqtw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=108&amp;publication_year=2011&amp;pages=20982-20987&amp;issue=52&amp;author=V.+Y.+Torbeevauthor=H.+Raghuramanauthor=D.+Hamelbergauthor=M.+Tonelliauthor=W.+M.+Westlerauthor=E.+Perozoauthor=S.+B.+H.+Kent&amp;title=Protein+Conformational+Dynamics+in+the+Mechanism+of+HIV-1+Protease+Catalysis&amp;doi=10.1073%2Fpnas.1111202108" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Protein conformational dynamics in the mechanism of HIV-1 protease catalysis</span></div><div class="casAuthors">Torbeev, Vladimir Yu.; Raghuraman, H.; Hamelberg, Donald; Tonelli, Marco; Westler, William M.; Perozo, Eduardo; Kent, Stephen B. H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">20982-20987, S20982/1-S20982/44</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We have used chem. protein synthesis and advanced phys. methods to probe dynamics-function correlations for the HIV-1 protease, an enzyme that has received considerable attention as a target for the treatment of AIDS.  Chem. synthesis was used to prep. a series of unique analogs of the HIV-1 protease in which the flexibility of the "flap" structures (residues 37-61 in each monomer of the homodimeric protein mol.) was systematically varied.  These analog enzymes were further studied by X-ray crystallog., NMR relaxation, and pulse-EPR methods, in conjunction with mol. dynamics simulations.  We show that conformational isomerization in the flaps is correlated with structural reorganization of residues in the active site, and that it is preorganization of the active site that is a rate-limiting factor in catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlidz4XYglO7Vg90H21EOLACvtfcHk0li8ojPeM4EMOQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XnvVCqtw%253D%253D&amp;md5=1e5b305474c1d248d0a087c9ffc15f66</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1111202108&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1111202108%26sid%3Dliteratum%253Aachs%26aulast%3DTorbeev%26aufirst%3DV.%2BY.%26aulast%3DRaghuraman%26aufirst%3DH.%26aulast%3DHamelberg%26aufirst%3DD.%26aulast%3DTonelli%26aufirst%3DM.%26aulast%3DWestler%26aufirst%3DW.%2BM.%26aulast%3DPerozo%26aufirst%3DE.%26aulast%3DKent%26aufirst%3DS.%2BB.%2BH.%26atitle%3DProtein%2520Conformational%2520Dynamics%2520in%2520the%2520Mechanism%2520of%2520HIV-1%2520Protease%2520Catalysis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26issue%3D52%26spage%3D20982%26epage%3D20987%26doi%3D10.1073%2Fpnas.1111202108" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nair, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span> <span> </span><span class="NLM_article-title">X-Ray Structures of Myc-Max and Mad-Max Recognizing DNA: Molecular Bases of Regulation by Proto-Oncogenic Transcription Factors</span>. <i>Cell</i>  <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>112</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">205</span>, <span class="refDoi">&nbsp;DOI: 10.1016/S0092-8674(02)01284-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2FS0092-8674%2802%2901284-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=12553908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvVemsQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=112&amp;publication_year=2003&amp;pages=193-205&amp;issue=2&amp;author=S.+K.+Nairauthor=S.+K.+Burley&amp;title=X-Ray+Structures+of+Myc-Max+and+Mad-Max+Recognizing+DNA%3A+Molecular+Bases+of+Regulation+by+Proto-Oncogenic+Transcription+Factors&amp;doi=10.1016%2FS0092-8674%2802%2901284-9" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">X-ray structures of Myc-Max and Mad-Max recognizing DNA: Molecular bases of regulation by proto-oncogenic transcription factors</span></div><div class="casAuthors">Nair, Satish K.; Burley, Stephen K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">193-205</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">X-ray structures of the basic/helix-loop-helix/leucine zipper (bHLHZ) domains of Myc-Max and Mad-Max heterodimers bound to their common DNA target (Enhancer or E box hexanucleotide, 5'-CACGTG-3') have been detd. at 1.9 Å and 2.0 Å resoln., resp.  E box recognition by these two structurally similar transcription factor pairs dets. whether a cell will divide and proliferate (Myc-Max) or differentiate and become quiescent (Mad-Max).  Deregulation of Myc has been implicated in the development of many human cancers, including Burkitt's lymphoma, neuroblastomas, and small cell lung cancers.  Both quasisym. heterodimers resemble the sym. Max homodimer, albeit with marked structural differences in the coiled-coil leucine zipper regions that explain preferential homo- and heteromeric dimerization of these three evolutionarily related DNA-binding proteins.  The Myc-Max heterodimer, but not its Mad-Max counterpart, dimerizes to form a bivalent heterotetramer, which explains how Myc can upregulate expression of genes with promoters bearing widely sepd. E boxes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgxSisNQ3O1LVg90H21EOLACvtfcHk0ljKpR61R-tIeA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3sXpvVemsQ%253D%253D&amp;md5=d049bd00310a705c951ece664e06cd22</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2901284-9&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252901284-9%26sid%3Dliteratum%253Aachs%26aulast%3DNair%26aufirst%3DS.%2BK.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26atitle%3DX-Ray%2520Structures%2520of%2520Myc-Max%2520and%2520Mad-Max%2520Recognizing%2520DNA%253A%2520Molecular%2520Bases%2520of%2520Regulation%2520by%2520Proto-Oncogenic%2520Transcription%2520Factors%26jtitle%3DCell%26date%3D2003%26volume%3D112%26issue%3D2%26spage%3D193%26epage%3D205%26doi%3D10.1016%2FS0092-8674%2802%2901284-9" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canne, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferré-D’Amaré, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, S. B. H.</span></span> <span> </span><span class="NLM_article-title">Total Chemical Synthesis of a Unique Transcription Factor-Related Protein: CMyc—Max</span>. <i>J. Am. Chem. Soc.</i>  <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>117</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2998</span>– <span class="NLM_lpage">3007</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja00116a005</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00116a005" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADyaK2MXktleltrk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=117&amp;publication_year=1995&amp;pages=2998-3007&amp;issue=11&amp;author=L.+E.+Canneauthor=A.+R.+Ferr%C3%A9-D%E2%80%99Amar%C3%A9author=S.+K.+Burleyauthor=S.+B.+H.+Kent&amp;title=Total+Chemical+Synthesis+of+a+Unique+Transcription+Factor-Related+Protein%3A+CMyc%E2%80%94Max&amp;doi=10.1021%2Fja00116a005" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Total Chemical Synthesis of a Unique Transcription Factor-Related Protein: cMyc-Max</span></div><div class="casAuthors">Canne, Lynne E.; Ferre- D'Amare, Adrian R.; Burley, Stephen K.; Kent, Stephen B. H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2998-3007</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CMyc and Max are proteins that regulate gene expression by assocg. to form noncovalent dimers that bind to specific regions of double-stranded DNA and activating or inhibiting mRNA transcription.  Each of these transcription factor proteins contains an ∼90 amino acid residue basic/helix-loop-helix/zipper (b/HLH/Z) domain responsible for formation of the protein dimers and for DNA binding.  Noncovalent heterodimers of the cMyc and Max b/HLH/Z domains are believed to be functionally important in vivo but have not been accessible by conventional means (recDNA expression).  Here the authors report the total chem. synthesis of novel covalently-linked dimers of the b/HLH/Z domains of these transcription factors by a modular strategy based on the convergent chem. ligation of several unprotected peptide segments.  The resulting ∼20 kDa synthetic proteins were purified to homogeneity and characterized by electrospray mass spectrometry.  The synthetic covalent cMyc-Max heterodimer and the covalent Max homodimer control were shown to bind DNA specifically at the region involved in the regulation of mRNA transcription.  Access to defined heterodimeric forms of the b/HLH/Z transcription factors will allow the structural and functional properties of these important protein regulators of gene expression to be studied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDeztgTOYiY7Vg90H21EOLACvtfcHk0ljKpR61R-tIeA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXktleltrk%253D&amp;md5=cfcfc89fd62eba4d197654a8d918a672</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja00116a005&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00116a005%26sid%3Dliteratum%253Aachs%26aulast%3DCanne%26aufirst%3DL.%2BE.%26aulast%3DFerr%25C3%25A9-D%25E2%2580%2599Amar%25C3%25A9%26aufirst%3DA.%2BR.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DKent%26aufirst%3DS.%2BB.%2BH.%26atitle%3DTotal%2520Chemical%2520Synthesis%2520of%2520a%2520Unique%2520Transcription%2520Factor-Related%2520Protein%253A%2520CMyc%25E2%2580%2594Max%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1995%26volume%3D117%26issue%3D11%26spage%3D2998%26epage%3D3007%26doi%3D10.1021%2Fja00116a005" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mijalis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumont, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span> <span> </span><span class="NLM_article-title">A Fully Automated Flow-Based Approach for Accelerated Peptide Synthesis</span>. <i>Nat. Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">464</span>– <span class="NLM_lpage">466</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.2318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fnchembio.2318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=28244989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsVShsrk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=13&amp;publication_year=2017&amp;pages=464-466&amp;issue=5&amp;author=A.+J.+Mijalisauthor=D.+A.+Thomasauthor=M.+D.+Simonauthor=A.+Adamoauthor=R.+Beaumontauthor=K.+F.+Jensenauthor=B.+L.+Pentelute&amp;title=A+Fully+Automated+Flow-Based+Approach+for+Accelerated+Peptide+Synthesis&amp;doi=10.1038%2Fnchembio.2318" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A fully automated flow-based approach for accelerated peptide synthesis</span></div><div class="casAuthors">Mijalis, Alexander J.; Thomas, Dale A., III; Simon, Mark D.; Adamo, Andrea; Beaumont, Ryan; Jensen, Klavs F.; Pentelute, Bradley L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">464-466</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we report a fully automated, flow-based approach to solid-phase polypeptide synthesis, with amide bond formation in 7 s and total synthesis times of 40 s per amino acid residue.  Crude peptide purities and isolated yields were comparable to those for std.-batch solid-phase peptide synthesis.  At full capacity, this approach can yield tens of thousands of individual 30-mer peptides per yr.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRTWEyNbJGibVg90H21EOLACvtfcHk0ljKpR61R-tIeA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXjsVShsrk%253D&amp;md5=c8aa99dfdac88175f414098b2d2baf10</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2318&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2318%26sid%3Dliteratum%253Aachs%26aulast%3DMijalis%26aufirst%3DA.%2BJ.%26aulast%3DThomas%26aufirst%3DD.%2BA.%26aulast%3DSimon%26aufirst%3DM.%2BD.%26aulast%3DAdamo%26aufirst%3DA.%26aulast%3DBeaumont%26aufirst%3DR.%26aulast%3DJensen%26aufirst%3DK.%2BF.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26atitle%3DA%2520Fully%2520Automated%2520Flow-Based%2520Approach%2520for%2520Accelerated%2520Peptide%2520Synthesis%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26issue%3D5%26spage%3D464%26epage%3D466%26doi%3D10.1038%2Fnchembio.2318" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartrampf, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saebi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poskus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gates, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowfer, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antilla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schissel, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quartararo, A. J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Synthesis of Proteins by Automated Flow Chemistry</span>. <i>Science</i>  <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span> (<span class="NLM_issue">6494</span>),  <span class="NLM_fpage">980</span>– <span class="NLM_lpage">987</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.abb2491</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1126%2Fscience.abb2491" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=32467387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVCnur%252FM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=368&amp;publication_year=2020&amp;pages=980-987&amp;issue=6494&amp;author=N.+Hartrampfauthor=A.+Saebiauthor=M.+Poskusauthor=Z.+P.+Gatesauthor=A.+J.+Callahanauthor=A.+E.+Cowferauthor=S.+Hannaauthor=S.+Antillaauthor=C.+K.+Schisselauthor=A.+J.+Quartararo&amp;title=Synthesis+of+Proteins+by+Automated+Flow+Chemistry&amp;doi=10.1126%2Fscience.abb2491" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of proteins by automated flow chemistry</span></div><div class="casAuthors">Hartrampf, N.; Saebi, A.; Poskus, M.; Gates, Z. P.; Callahan, A. J.; Cowfer, A. E.; Hanna, S.; Antilla, S.; Schissel, C. K.; Quartararo, A. J.; Ye, X.; Mijalis, A. J.; Simon, M. D.; Loas, A.; Liu, S.; Jessen, C.; Nielsen, T. E.; Pentelute, B. L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6494</span>),
    <span class="NLM_cas:pages">980-987</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Ribosomes can produce proteins in minutes and are largely constrained to proteinogenic amino acids.  Here, we report highly efficient chem. matched with an automated fast-flow instrument for the direct manufg. of peptide chains up to 164 amino acids long over 327 consecutive reactions.  The machine is rapid: Peptide chain elongation is complete in hours.  We demonstrate the utility of this approach by the chem. synthesis of nine different protein chains that represent enzymes, structural units, and regulatory factors.  After purifn. and folding, the synthetic materials display biophys. and enzymic properties comparable to the biol. expressed proteins.  High-fidelity automated flow chem. is an alternative for producing single-domain proteins without the ribosome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4GKtPvg69AbVg90H21EOLACvtfcHk0li9slBmLH29lA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3cXhtVCnur%252FM&amp;md5=c7674247769238f6429c0131cfac3578</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb2491&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb2491%26sid%3Dliteratum%253Aachs%26aulast%3DHartrampf%26aufirst%3DN.%26aulast%3DSaebi%26aufirst%3DA.%26aulast%3DPoskus%26aufirst%3DM.%26aulast%3DGates%26aufirst%3DZ.%2BP.%26aulast%3DCallahan%26aufirst%3DA.%2BJ.%26aulast%3DCowfer%26aufirst%3DA.%2BE.%26aulast%3DHanna%26aufirst%3DS.%26aulast%3DAntilla%26aufirst%3DS.%26aulast%3DSchissel%26aufirst%3DC.%2BK.%26aulast%3DQuartararo%26aufirst%3DA.%2BJ.%26atitle%3DSynthesis%2520of%2520Proteins%2520by%2520Automated%2520Flow%2520Chemistry%26jtitle%3DScience%26date%3D2020%26volume%3D368%26issue%3D6494%26spage%3D980%26epage%3D987%26doi%3D10.1126%2Fscience.abb2491" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palmacci, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plante, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeberger, P. H.</span></span> <span> </span><span class="NLM_article-title">Automated Synthesis of Oligosaccharides</span>. <i>Helv. Chim. Acta</i>  <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">3975</span>, <span class="refDoi">&nbsp;DOI: 10.1002/hlca.200390331</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1002%2Fhlca.200390331" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvFCnsA%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=86&amp;publication_year=2003&amp;pages=3975&amp;author=E.+R.+Palmacciauthor=O.+J.+Planteauthor=M.+C.+Hewittauthor=P.+H.+Seeberger&amp;title=Automated+Synthesis+of+Oligosaccharides&amp;doi=10.1002%2Fhlca.200390331" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Automated synthesis of oligosaccharides</span></div><div class="casAuthors">Palmacci, Emma R.; Plante, Obadiah J.; Hewitt, Michael C.; Seeberger, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Helvetica Chimica Acta</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3975-3990</span>CODEN:
                <span class="NLM_cas:coden">HCACAV</span>;
        ISSN:<span class="NLM_cas:issn">0018-019X</span>.
    
            (<span class="NLM_cas:orgname">Verlag Helvetica Chimica Acta</span>)
        </div><div class="casAbstract">The chem. synthesis of oligosaccharides with an automated solid-phase synthesizer is described.  An octenediol linker served to attach the growing oligosaccharide chain to the solid support, and the desired structures were cleaved from the support via olefin metathesis to afford a pentenyl glycoside.  The automated syntheses of several important carbohydrates, including a penta-rhamnoside, a proteoglycan linkage-region tetrasaccharide, a phytoalexin elicitor dodeca-saccharide, and a branched Leishmania lipo-phospho-glycan tetrasaccharide, were accomplished in higher overall yield and ca. 20 times faster than with soln.-phase methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXd2F5RR7BrLVg90H21EOLACvtfcHk0li9slBmLH29lA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2cXmvFCnsA%253D%253D&amp;md5=8d5180c6e77fb3d5709f3825454b8c1f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fhlca.200390331&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhlca.200390331%26sid%3Dliteratum%253Aachs%26aulast%3DPalmacci%26aufirst%3DE.%2BR.%26aulast%3DPlante%26aufirst%3DO.%2BJ.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DSeeberger%26aufirst%3DP.%2BH.%26atitle%3DAutomated%2520Synthesis%2520of%2520Oligosaccharides%26jtitle%3DHelv.%2520Chim.%2520Acta%26date%3D2003%26volume%3D86%26spage%3D3975%26doi%3D10.1002%2Fhlca.200390331" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schissel, C.</span>; <span class="NLM_string-name">Mohapatra, S.</span>; <span class="NLM_string-name">Wolfe, J.</span>; <span class="NLM_string-name">Fadzen, C.</span>; <span class="NLM_string-name">Bellovoda, K.</span>; <span class="NLM_string-name">Wu, C.-L.</span>; <span class="NLM_string-name">Wood, J.</span>; <span class="NLM_string-name">Malmberg, A.</span>; <span class="NLM_string-name">Loas, A.</span>; <span class="NLM_string-name">Gómez-Bombarelli, R.</span></span>; <span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Interpretable Deep Learning for De Novo Design of Cell-Penetrating Abiotic Polymers</span>. bioRxiv, <span class="NLM_year">2020</span>. <a href="https://doi.org/10.1101/2020.04.10.036566" class="extLink">https://doi.org/10.1101/2020.04.10.036566</a>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1101%2F2020.04.10.036566" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2020&amp;author=C.+Schissel&amp;author=S.+Mohapatra&amp;author=J.+Wolfe&amp;author=C.+Fadzen&amp;author=K.+Bellovoda&amp;author=C.-L.+Wu&amp;author=J.+Wood&amp;author=A.+Malmberg&amp;author=A.+Loas&amp;author=R.+G%C3%B3mez-Bombarelli&amp;title=Interpretable+Deep+Learning+for+De+Novo+Design+of+Cell-Penetrating+Abiotic+Polymers" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1101%2F2020.04.10.036566&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dweblink%26id%3Ddoi%3A10.1101%252F2020.04.10.036566%26sid%3Dliteratum%253Aachs%26aulast%3DSchissel%26aufirst%3DC.%26atitle%3DInterpretable%2520Deep%2520Learning%2520for%2520De%2520Novo%2520Design%2520of%2520Cell-Penetrating%2520Abiotic%2520Polymers%26date%3D2020" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fadzen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holden, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schissel, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span> <span> </span><span class="NLM_article-title">Chimeras of Cell-Penetrating Peptides Demonstrate Synergistic Improvement in Antisense Efficacy</span>. <i>Biochemistry</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">3980</span>– <span class="NLM_lpage">3989</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.biochem.9b00413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.9b00413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1ensrbJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=58&amp;publication_year=2019&amp;pages=3980-3989&amp;issue=38&amp;author=C.+M.+Fadzenauthor=R.+L.+Holdenauthor=J.+M.+Wolfeauthor=Z.+N.+Chooauthor=C.+K.+Schisselauthor=M.+Yaoauthor=G.+J.+Hansonauthor=B.+L.+Pentelute&amp;title=Chimeras+of+Cell-Penetrating+Peptides+Demonstrate+Synergistic+Improvement+in+Antisense+Efficacy&amp;doi=10.1021%2Facs.biochem.9b00413" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Chimeras of Cell-Penetrating Peptides Demonstrate Synergistic Improvement in Antisense Efficacy</span></div><div class="casAuthors">Fadzen, Colin M.; Holden, Rebecca L.; Wolfe, Justin M.; Choo, Zi-Ning; Schissel, Carly K.; Yao, Monica; Hanson, Gunnar J.; Pentelute, Bradley L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">3980-3989</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphorodiamidate morpholino oligonucleotides (PMOs) make up a promising class of therapeutics for genetic disease.  PMOs designed for "exon skipping" must be internalized into cells, reach the nucleus, and act on pre-mRNA to mediate their effects.  One tactic for improving PMO delivery and exon skipping is to covalently conjugate PMOs to cell-penetrating peptides (CPPs).  Here, we report the synthesis of PMOs conjugated to CPP chimeras, constructed by combining multiple CPPs into one sequence.  The chimeric CPPs synergistically improve PMO activity up to 70-fold compared to that of the PMO alone and beyond the expected effects of each component peptide.  By investigating the design space of CPP chimeras, we demonstrate that all components must be covalently attached, that the order of the two sequences matters, and that peptide identity can tune activity.  We identified one chimera (pVEC-Bpep) to investigate in more detail and found that it engages mechanisms of endocytosis different from those of its parent peptides.  We also examd. the extent to which the beneficial effect comes from improved cellular uptake as opposed to the downstream steps required for exon skipping.  Given the complexity of intracellular delivery, we anticipate this work will lead researchers to consider combining mols. with different physicochem. properties to aid in the delivery of biol. cargoes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4ughF4vq80rVg90H21EOLACvtfcHk0li9slBmLH29lA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXhs1ensrbJ&amp;md5=21857f6abc53ad93039e90de4e9db86e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.9b00413&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.9b00413%26sid%3Dliteratum%253Aachs%26aulast%3DFadzen%26aufirst%3DC.%2BM.%26aulast%3DHolden%26aufirst%3DR.%2BL.%26aulast%3DWolfe%26aufirst%3DJ.%2BM.%26aulast%3DChoo%26aufirst%3DZ.%2BN.%26aulast%3DSchissel%26aufirst%3DC.%2BK.%26aulast%3DYao%26aufirst%3DM.%26aulast%3DHanson%26aufirst%3DG.%2BJ.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26atitle%3DChimeras%2520of%2520Cell-Penetrating%2520Peptides%2520Demonstrate%2520Synergistic%2520Improvement%2520in%2520Antisense%2520Efficacy%26jtitle%3DBiochemistry%26date%3D2019%26volume%3D58%26issue%3D38%26spage%3D3980%26epage%3D3989%26doi%3D10.1021%2Facs.biochem.9b00413" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorolla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogdawa, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golden, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewhurst, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruickshank, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, K.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Tumor Penetrating Peptides Inhibiting MYC as a Potent Targeted Therapeutic Strategy for Triple-Negative Breast Cancers</span>. <i>Oncogene</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">140</span>– <span class="NLM_lpage">150</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41388-018-0421-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fs41388-018-0421-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30076412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVCns7%252FI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=38&amp;publication_year=2019&amp;pages=140-150&amp;issue=1&amp;author=E.+Wangauthor=A.+Sorollaauthor=P.+T.+Cunninghamauthor=H.+M.+Bogdawaauthor=S.+Beckauthor=E.+Goldenauthor=R.+E.+Dewhurstauthor=L.+Florezauthor=M.+N.+Cruickshankauthor=K.+Hoffmann&amp;title=Tumor+Penetrating+Peptides+Inhibiting+MYC+as+a+Potent+Targeted+Therapeutic+Strategy+for+Triple-Negative+Breast+Cancers&amp;doi=10.1038%2Fs41388-018-0421-y" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor penetrating peptides inhibiting MYC as a potent targeted therapeutic strategy for triple-negative breast cancers</span></div><div class="casAuthors">Wang, Edina; Sorolla, Anabel; Cunningham, Paula T.; Bogdawa, Heique M.; Beck, Samuel; Golden, Emily; Dewhurst, Robert E.; Florez, Laura; Cruickshank, Mark N.; Hoffmann, Katrin; Hopkins, Richard M.; Kim, Jonghwan; Woo, Andrew J.; Watt, Paul M.; Blancafort, Pilar</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">140-150</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Overexpression of MYC oncogene is highly prevalent in many malignancies such as aggressive triple-neg. breast cancers (TNBCs) and it is assocd. with very poor outcome.  Despite decades of research, attempts to effectively inhibit MYC, particularly with small mols., still remain challenging due to the featureless nature of its protein structure.  Herein, we describe the engineering of the dominant-neg. MYC peptide (OmoMYC) linked to a functional penetrating 'Phylomer' peptide (FPPa) as a therapeutic strategy to inhibit MYC in TNBC.  We found FPPa-OmoMYC to be a potent inducer of apoptosis (with IC50 from 1-2 μM) in TNBC cells with negligible effects in non-tumorigenic cells.  Transcriptome anal. of FPPa-OmoMYC-treated cells indicated that the fusion protein inhibited MYC-dependent networks, inducing dynamic changes in transcriptional, metabolic, and apoptotic processes.  We demonstrated the efficacy of FPPa-OmoMYC in inhibiting breast cancer growth when injected orthotopically in TNBC allografts.  Lastly, we identified strong pharmacol. synergisms between FPPa-OmoMYC and chemotherapeutic agents.  This study highlights a novel therapeutic approach to target highly aggressive and chemoresistant MYC-activated cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqex9cMW5_g8rVg90H21EOLACvtfcHk0liD8U2Y1EWAKQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhsVCns7%252FI&amp;md5=8c94bfea4f2600c86ff70ef37618483b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fs41388-018-0421-y&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41388-018-0421-y%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DE.%26aulast%3DSorolla%26aufirst%3DA.%26aulast%3DCunningham%26aufirst%3DP.%2BT.%26aulast%3DBogdawa%26aufirst%3DH.%2BM.%26aulast%3DBeck%26aufirst%3DS.%26aulast%3DGolden%26aufirst%3DE.%26aulast%3DDewhurst%26aufirst%3DR.%2BE.%26aulast%3DFlorez%26aufirst%3DL.%26aulast%3DCruickshank%26aufirst%3DM.%2BN.%26aulast%3DHoffmann%26aufirst%3DK.%26atitle%3DTumor%2520Penetrating%2520Peptides%2520Inhibiting%2520MYC%2520as%2520a%2520Potent%2520Targeted%2520Therapeutic%2520Strategy%2520for%2520Triple-Negative%2520Breast%2520Cancers%26jtitle%3DOncogene%26date%3D2019%26volume%3D38%26issue%3D1%26spage%3D140%26epage%3D150%26doi%3D10.1038%2Fs41388-018-0421-y" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukazawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuoka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamatsuji, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shigemitsu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morita, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faiola, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durbin, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naomoto, Y.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Myc Effectively Targets KRAS Mutation-Positive Lung Cancer Expressing High Levels of Myc</span>. <i>Anticancer Res.</i>  <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4193</span>– <span class="NLM_lpage">4200</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=21036740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFym" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=30&amp;publication_year=2010&amp;pages=4193-4200&amp;issue=10&amp;author=T.+Fukazawaauthor=Y.+Maedaauthor=J.+Matsuokaauthor=T.+Yamatsujiauthor=K.+Shigemitsuauthor=I.+Moritaauthor=F.+Faiolaauthor=M.+L.+Durbinauthor=L.+Soucekauthor=Y.+Naomoto&amp;title=Inhibition+of+Myc+Effectively+Targets+KRAS+Mutation-Positive+Lung+Cancer+Expressing+High+Levels+of+Myc" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Myc effectively targets KRAS mutation-positive lung cancer expressing high levels of Myc</span></div><div class="casAuthors">Fukazawa, Takuya; Maeda, Yutaka; Matsuoka, Junji; Yamatsuji, Tomoki; Shigemitsu, Kaoru; Morita, Ichiro; Faiola, Francesco; Durbin, Mary L.; Soucek, Laura; Naomoto, Yoshio</div><div class="citationInfo"><span class="NLM_cas:title">Anticancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4193-4200</span>CODEN:
                <span class="NLM_cas:coden">ANTRD4</span>;
        ISSN:<span class="NLM_cas:issn">0250-7005</span>.
    
            (<span class="NLM_cas:orgname">International Institute of Anticancer Research</span>)
        </div><div class="casAbstract">Myc is an oncogenic transcription factor that promotes tumorigenesis.  Recently, a dominant neg. form of Myc (Omomyc) was shown to cause regression of lung tumors in a mouse model of lung cancer caused by KRAS mutation, suggesting that Myc might be a potential therapeutic target to treat the KRAS lung cancer.  However, it is not yer known whether Omomyc can also inhibit the growth of human lung tumors that carry a similar KRAS mutation.  In the present study, we demonstrate that Omomyc induces cell death of KRAS-mutated human lung adenocarcinoma A549 cells in vitro and in vivo.  However, Omomyc does not induce cell death in human lung adenocarcinoma H441 cells that also carry the KRAS mutation.  Interestingly, A549 cells express high levels of Myc, while H441 cells do not.  Co-expression of exogenous Myc with Omomyc in H441 cells induces cell death, indicating that Omomyc requires high levels of Myc to induce cell death in KRAS mutation-pos. lung adenocarcinoma.  Here, we show for the first time that KRAS mutation-pos. lung cancer displaying high levels of Myc could be treated by inhibiting Myc transactivation function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6iGfK0KS3D7Vg90H21EOLACvtfcHk0liD8U2Y1EWAKQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXktFym&amp;md5=97ae611107884ef74a4abc6e19573a18</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFukazawa%26aufirst%3DT.%26aulast%3DMaeda%26aufirst%3DY.%26aulast%3DMatsuoka%26aufirst%3DJ.%26aulast%3DYamatsuji%26aufirst%3DT.%26aulast%3DShigemitsu%26aufirst%3DK.%26aulast%3DMorita%26aufirst%3DI.%26aulast%3DFaiola%26aufirst%3DF.%26aulast%3DDurbin%26aufirst%3DM.%2BL.%26aulast%3DSoucek%26aufirst%3DL.%26aulast%3DNaomoto%26aufirst%3DY.%26atitle%3DInhibition%2520of%2520Myc%2520Effectively%2520Targets%2520KRAS%2520Mutation-Positive%2520Lung%2520Cancer%2520Expressing%2520High%2520Levels%2520of%2520Myc%26jtitle%3DAnticancer%2520Res.%26date%3D2010%26volume%3D30%26issue%3D10%26spage%3D4193%26epage%3D4200" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spiegel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Modulation of Ras Signaling</span>. <i>Nat. Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">613</span>– <span class="NLM_lpage">622</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.1560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fnchembio.1560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=24929527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslGisbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2014&amp;pages=613-622&amp;issue=8&amp;author=J.+Spiegelauthor=P.+M.+Crommauthor=G.+Zimmermannauthor=T.+N.+Grossmannauthor=H.+Waldmann&amp;title=Small-Molecule+Modulation+of+Ras+Signaling&amp;doi=10.1038%2Fnchembio.1560" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule modulation of Ras signaling</span></div><div class="casAuthors">Spiegel, Jochen; Cromm, Philipp M.; Zimmermann, Gunther; Grossmann, Tom N.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">613-622</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite intense efforts in pharmaceutical industry and academia, a therapeutic grip on oncogenic Ras proteins has remained elusive.  Mutated Ras is assocd. with ∼20-30% of all human cancers often not responsive to established therapies.  In particular, K-Ras, the most frequently mutated Ras isoform, is considered one of the most important but 'undruggable' targets in cancer research.  Recently, new cavities on Ras for small-mol. ligands were identified, and selective direct targeting of mutated K-RasG12C has become possible for what is to our knowledge the first time.  In addn., impairment of Ras spatial organization, in particular via targeting the prenyl-binding Ras chaperone PDEδ, has opened a fresh perspective in anticancer research.  These recent advances fuel hopes for the development of new drugs targeting Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTcp4dzeVKK7Vg90H21EOLACvtfcHk0lgJFO1KlS85Ug"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXpslGisbw%253D&amp;md5=f8d84f054e1b392cf6bfe55d5e235c28</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1560&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1560%26sid%3Dliteratum%253Aachs%26aulast%3DSpiegel%26aufirst%3DJ.%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DZimmermann%26aufirst%3DG.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DSmall-Molecule%2520Modulation%2520of%2520Ras%2520Signaling%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26issue%3D8%26spage%3D613%26epage%3D622%26doi%3D10.1038%2Fnchembio.1560" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esquela-Kerscher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelnar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovcharenko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shingara, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">The Let-7 MicroRNA Represses Cell Proliferation Pathways in Human Cells</span>. <i>Cancer Res.</i>  <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">7713</span>– <span class="NLM_lpage">7722</span>, <span class="refDoi">&nbsp;DOI: 10.1158/0008-5472.CAN-07-1083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1158%2F0008-5472.CAN-07-1083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=17699775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptVGlsbo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=67&amp;publication_year=2007&amp;pages=7713-7722&amp;issue=16&amp;author=C.+D.+Johnsonauthor=A.+Esquela-Kerscherauthor=G.+Stefaniauthor=M.+Byromauthor=K.+Kelnarauthor=D.+Ovcharenkoauthor=M.+Wilsonauthor=X.+Wangauthor=J.+Sheltonauthor=J.+Shingara&amp;title=The+Let-7+MicroRNA+Represses+Cell+Proliferation+Pathways+in+Human+Cells&amp;doi=10.1158%2F0008-5472.CAN-07-1083" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">The let-7 MicroRNA Represses Cell Proliferation Pathways in Human Cells</span></div><div class="casAuthors">Johnson, Charles D.; Esquela-Kerscher, Aurora; Stefani, Giovanni; Byrom, Mike; Kelnar, Kevin; Ovcharenko, Dmitriy; Wilson, Mike; Wang, Xiaowei; Shelton, Jeffrey; Shingara, Jaclyn; Chin, Lena; Brown, David; Slack, Frank J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7713-7722</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">MicroRNAs play important roles in animal development, cell differentiation, and metab. and have been implicated in human cancer.  The let-7 microRNA controls the timing of cell cycle exit and terminal differentiation in Caenorhabditis elegans and is poorly expressed or deleted in human lung tumors.  Here, we show that let-7 is highly expressed in normal lung tissue, and that inhibiting let-7 function leads to increased cell division in A549 lung cancer cells.  Overexpression of let-7 in cancer cell lines alters cell cycle progression and reduces cell division, providing evidence that let-7 functions as a tumor suppressor in lung cells. let-7 was previously shown to regulate the expression of the RAS lung cancer oncogenes, and our work now shows that multiple genes involved in cell cycle and cell division functions are also directly or indirectly repressed by let-7.  This work reveals the let-7 microRNA to be a master regulator of cell proliferation pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3-hl_60Ut_LVg90H21EOLACvtfcHk0lgJFO1KlS85Ug"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXptVGlsbo%253D&amp;md5=102525e3b46833e6592d03efabb9321d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-1083&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-1083%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%2BD.%26aulast%3DEsquela-Kerscher%26aufirst%3DA.%26aulast%3DStefani%26aufirst%3DG.%26aulast%3DByrom%26aufirst%3DM.%26aulast%3DKelnar%26aufirst%3DK.%26aulast%3DOvcharenko%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DShelton%26aufirst%3DJ.%26aulast%3DShingara%26aufirst%3DJ.%26atitle%3DThe%2520Let-7%2520MicroRNA%2520Represses%2520Cell%2520Proliferation%2520Pathways%2520in%2520Human%2520Cells%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26issue%3D16%26spage%3D7713%26epage%3D7722%26doi%3D10.1158%2F0008-5472.CAN-07-1083" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebhardt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koelmel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ade, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuper, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Eyss, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letschert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Artista, L.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">OmoMYC Blunts Promoter Invasion by Oncogenic MYC to Inhibit Gene Expression Characteristic of MYC-Dependent Tumors</span>. <i>Oncogene</i>  <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">1911</span>– <span class="NLM_lpage">1924</span>, <span class="refDoi">&nbsp;DOI: 10.1038/onc.2016.354</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fonc.2016.354" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=27748763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1ynsbfF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=2017&amp;pages=1911-1924&amp;issue=14&amp;author=L.+A.+Jungauthor=A.+Gebhardtauthor=W.+Koelmelauthor=C.+P.+Adeauthor=S.+Walzauthor=J.+Kuperauthor=B.+Von+Eyssauthor=S.+Letschertauthor=C.+Redelauthor=L.+D%E2%80%99Artista&amp;title=OmoMYC+Blunts+Promoter+Invasion+by+Oncogenic+MYC+to+Inhibit+Gene+Expression+Characteristic+of+MYC-Dependent+Tumors&amp;doi=10.1038%2Fonc.2016.354" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">OmoMYC blunts promoter invasion by oncogenic MYC to inhibit gene expression characteristic of MYC-dependent tumors</span></div><div class="casAuthors">Jung, L. A.; Gebhardt, A.; Koelmel, W.; Ade, C. P.; Walz, S.; Kuper, J.; von Eyss, B.; Letschert, S.; Redel, C.; d'Artista, L.; Biankin, A.; Zender, L.; Sauer, M.; Wolf, E.; Evan, G.; Kisker, C.; Eilers, M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1911-1924</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MYC genes have both essential roles during normal development and exert oncogenic functions during tumorigenesis.  Expression of a dominant-neg. allele of MYC, termed OmoMYC, can induce rapid tumor regression in mouse models with little toxicity for normal tissues.  How OmoMYC discriminates between physiol. and oncogenic functions of MYC is unclear.  We have solved the crystal structure of OmoMYC and show that it forms a stable homodimer and as such recognizes DNA in the same manner as the MYC/MAX heterodimer.  OmoMYC attenuates both MYC-dependent activation and repression by competing with MYC/MAX for binding to chromatin, effectively lowering MYC/MAX occupancy at its cognate binding sites.  OmoMYC causes the largest decreases in promoter occupancy and changes in expression on genes that are invaded by oncogenic MYC levels.  A signature of OmoMYC-regulated genes defines subgroups with high MYC levels in multiple tumor entities and identifies novel targets for the eradication of MYC-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpv_ma5jMA0LVg90H21EOLACvtfcHk0lgJFO1KlS85Ug"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28Xhs1ynsbfF&amp;md5=bb7307180be76887bcc94c2c6ae8fa2b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.354&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.354%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DL.%2BA.%26aulast%3DGebhardt%26aufirst%3DA.%26aulast%3DKoelmel%26aufirst%3DW.%26aulast%3DAde%26aufirst%3DC.%2BP.%26aulast%3DWalz%26aufirst%3DS.%26aulast%3DKuper%26aufirst%3DJ.%26aulast%3DVon%2BEyss%26aufirst%3DB.%26aulast%3DLetschert%26aufirst%3DS.%26aulast%3DRedel%26aufirst%3DC.%26aulast%3DD%25E2%2580%2599Artista%26aufirst%3DL.%26atitle%3DOmoMYC%2520Blunts%2520Promoter%2520Invasion%2520by%2520Oncogenic%2520MYC%2520to%2520Inhibit%2520Gene%2520Expression%2520Characteristic%2520of%2520MYC-Dependent%2520Tumors%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26issue%3D14%26spage%3D1911%26epage%3D1924%26doi%3D10.1038%2Fonc.2016.354" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montagne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaudoin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavoie, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinck, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavigne, P.</span></span> <span> </span><span class="NLM_article-title">The Max B-HLH-LZ Can Transduce into Cells and Inhibit c-Myc Transcriptional Activities</span>. <i>PLoS One</i>  <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">10</span>, <span class="refDoi">&nbsp;DOI: 10.1371/journal.pone.0032172</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1371%2Fjournal.pone.0032172" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2012&amp;pages=2-10&amp;issue=2&amp;author=M.+Montagneauthor=N.+Beaudoinauthor=D.+Fortinauthor=C.+L.+Lavoieauthor=R.+Klinckauthor=P.+Lavigne&amp;title=The+Max+B-HLH-LZ+Can+Transduce+into+Cells+and+Inhibit+c-Myc+Transcriptional+Activities&amp;doi=10.1371%2Fjournal.pone.0032172" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0032172&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0032172%26sid%3Dliteratum%253Aachs%26aulast%3DMontagne%26aufirst%3DM.%26aulast%3DBeaudoin%26aufirst%3DN.%26aulast%3DFortin%26aufirst%3DD.%26aulast%3DLavoie%26aufirst%3DC.%2BL.%26aulast%3DKlinck%26aufirst%3DR.%26aulast%3DLavigne%26aufirst%3DP.%26atitle%3DThe%2520Max%2520B-HLH-LZ%2520Can%2520Transduce%2520into%2520Cells%2520and%2520Inhibit%2520c-Myc%2520Transcriptional%2520Activities%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26issue%3D2%26spage%3D2%26epage%3D10%26doi%3D10.1371%2Fjournal.pone.0032172" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span> <i>Peptomyc</i>. <a href="https://www.biospace.com/article/releases/peptomyc-s-omomyc-based-therapy-omo-103-has-obtained-approval-for-first-in-human-phase-i-ii-trial-to-assess-the-efficacy-and-safety-of-a-novel-myc-inhibitor/" class="extLink">https://www.biospace.com/article/releases/peptomyc-s-omomyc-based-therapy-omo-103-has-obtained-approval-for-first-in-human-phase-i-ii-trial-to-assess-the-efficacy-and-safety-of-a-novel-myc-inhibitor/</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=+Peptomyc.+https%3A%2F%2Fwww.biospace.com%2Farticle%2Freleases%2Fpeptomyc-s-omomyc-based-therapy-omo-103-has-obtained-approval-for-first-in-human-phase-i-ii-trial-to-assess-the-efficacy-and-safety-of-a-novel-myc-inhibitor%2F." target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DPeptomyc" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herce, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, A. F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasper, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonhardt, H.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Cell-Permeable Nanobodies for Targeted Immunolabelling and Antigen Manipulation in Living Cells</span>. <i>Nat. Chem.</i>  <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">762</span>– <span class="NLM_lpage">771</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.2811</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fnchem.2811" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=28754949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFOjsLrM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2017&amp;pages=762-771&amp;issue=8&amp;author=H.+D.+Herceauthor=D.+Schumacherauthor=A.+F.+L.+Schneiderauthor=A.+K.+Ludwigauthor=F.+A.+Mannauthor=M.+Filliesauthor=M.+A.+Kasperauthor=S.+Reinkeauthor=E.+Krauseauthor=H.+Leonhardt&amp;title=Cell-Permeable+Nanobodies+for+Targeted+Immunolabelling+and+Antigen+Manipulation+in+Living+Cells&amp;doi=10.1038%2Fnchem.2811" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-permeable nanobodies for targeted immunolabelling and antigen manipulation in living cells</span></div><div class="casAuthors">Herce, Henry D.; Schumacher, Dominik; Schneider, Anselm F. L.; Ludwig, Anne K.; Mann, Florian A.; Fillies, Marion; Kasper, Marc-Andre; Reinke, Stefan; Krause, Eberhard; Leonhardt, Heinrich; Cardoso, M. Cristina; Hackenberger, Christian P. R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">762-771</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Functional antibody delivery in living cells would enable the labeling and manipulation of intracellular antigens, which constitutes a long-thought goal in cell biol. and medicine.  Here we present a modular strategy to create functional cell-permeable nanobodies capable of targeted labeling and manipulation of intracellular antigens in living cells.  The cell-permeable nanobodies are formed by the site-specific attachment of intracellularly stable (or cleavable) cyclic arginine-rich cell-penetrating peptides to camelid-derived single-chain VHH antibody fragments.  We used this strategy for the non-endocytic delivery of two recombinant nanobodies into living cells, which enabled the relocalization of the polymerase clamp PCNA (proliferating cell nuclear antigen) and tumor suppressor p53 to the nucleolus, and thereby allowed the detection of protein-protein interactions that involve these two proteins in living cells.  Furthermore, cell-permeable nanobodies permitted the co-transport of therapeutically relevant proteins, such as Mecp2, into the cells.  This technol. constitutes a major step in the labeling, delivery and targeted manipulation of intracellular antigens.  Ultimately, this approach opens the door towards immunostaining in living cells and the expansion of immunotherapies to intracellular antigen targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLMQ2tz9B_kbVg90H21EOLACvtfcHk0lgVQ5KUz_1oxw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhtFOjsLrM&amp;md5=681c6fcd98272ea6c93d0eaa586c1a1d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2811&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2811%26sid%3Dliteratum%253Aachs%26aulast%3DHerce%26aufirst%3DH.%2BD.%26aulast%3DSchumacher%26aufirst%3DD.%26aulast%3DSchneider%26aufirst%3DA.%2BF.%2BL.%26aulast%3DLudwig%26aufirst%3DA.%2BK.%26aulast%3DMann%26aufirst%3DF.%2BA.%26aulast%3DFillies%26aufirst%3DM.%26aulast%3DKasper%26aufirst%3DM.%2BA.%26aulast%3DReinke%26aufirst%3DS.%26aulast%3DKrause%26aufirst%3DE.%26aulast%3DLeonhardt%26aufirst%3DH.%26atitle%3DCell-Permeable%2520Nanobodies%2520for%2520Targeted%2520Immunolabelling%2520and%2520Antigen%2520Manipulation%2520in%2520Living%2520Cells%26jtitle%3DNat.%2520Chem.%26date%3D2017%26volume%3D9%26issue%3D8%26spage%3D762%26epage%3D771%26doi%3D10.1038%2Fnchem.2811" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gersbach, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbas, C. F.</span></span> <span> </span><span class="NLM_article-title">Synthetic Zinc Finger Proteins: The Advent of Targeted Gene Regulation and Genome Modification Technologies</span>. <i>Acc. Chem. Res.</i>  <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2309</span>– <span class="NLM_lpage">2318</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar500039w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar500039w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptVWrur8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=2014&amp;pages=2309-2318&amp;issue=8&amp;author=C.+A.+Gersbachauthor=T.+Gajauthor=C.+F.+Barbas&amp;title=Synthetic+Zinc+Finger+Proteins%3A+The+Advent+of+Targeted+Gene+Regulation+and+Genome+Modification+Technologies&amp;doi=10.1021%2Far500039w" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic zinc finger proteins: The advent of targeted gene regulation and genome modification technologies</span></div><div class="casAuthors">Gersbach, Charles A.; Gaj, Thomas; Barbas, Carlos F., III</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2309-2318</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The understanding of gene regulation and the structure and function of the human genome increased dramatically at the end of the 20th century.  Yet the technologies for manipulating the genome have been slower to develop.  For instance, the field of gene therapy has been focused on correcting genetic diseases and augmenting tissue repair for more than 40 years.  However, with the exception of a few very low efficiency approaches, conventional genetic engineering methods have only been able to add auxiliary genes to cells.  This has been a substantial obstacle to the clin. success of gene therapies and has also led to severe unintended consequences in several cases.  Therefore, technologies that facilitate the precise modification of cellular genomes have diverse and significant implications in many facets of research and are essential for translating the products of the Genomic Revolution into tangible benefits for medicine and biotechnol.  To address this need, in the 1990s, we embarked on a mission to develop technologies for engineering protein-DNA interactions with the aim of creating custom tools capable of targeting any DNA sequence.  Our goal has been to allow researchers to reach into genomes to specifically regulate, knock out, or replace any gene.  To realize these goals, we initially focused on understanding and manipulating zinc finger proteins.  In particular, we sought to create a simple and straightforward method that enables unspecialized labs. to engineer custom DNA-modifying proteins using only defined modular components, a web-based utility, and std. recombinant DNA technol.  Two significant challenges we faced were (i) the development of zinc finger domains that target sequences not recognized by naturally occurring zinc finger proteins and (ii) detg. how individual zinc finger domains could be tethered together as polydactyl proteins to recognize unique locations within complex genomes.  We and others have since used this modular assembly method to engineer artificial proteins and enzymes that activate, repress, or create defined changes to user-specified genes in human cells, plants, and other organisms.  We have also engineered novel methods for externally controlling protein activity and delivery, as well as developed new strategies for the directed evolution of protein and enzyme function.  This Account summarizes our work in these areas and highlights independent studies that have successfully used the modular assembly approach to create proteins with novel function.  We also discuss emerging alternative methods for genomic targeting, including transcription activator-like effectors (TALEs) and CRISPR/Cas systems, and how they complement the synthetic zinc finger protein technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohe_yNDoa16rVg90H21EOLACvtfcHk0lgVQ5KUz_1oxw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXptVWrur8%253D&amp;md5=41677a68edbd08d43d57d2a1a842832d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Far500039w&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far500039w%26sid%3Dliteratum%253Aachs%26aulast%3DGersbach%26aufirst%3DC.%2BA.%26aulast%3DGaj%26aufirst%3DT.%26aulast%3DBarbas%26aufirst%3DC.%2BF.%26atitle%3DSynthetic%2520Zinc%2520Finger%2520Proteins%253A%2520The%2520Advent%2520of%2520Targeted%2520Gene%2520Regulation%2520and%2520Genome%2520Modification%2520Technologies%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2014%26volume%3D47%26issue%3D8%26spage%3D2309%26epage%3D2318%26doi%3D10.1021%2Far500039w" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eiríksdóttir, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konate, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langel, Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Divita, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshayes, S.</span></span> <span> </span><span class="NLM_article-title">Secondary Structure of Cell-Penetrating Peptides Controls Membrane Interaction and Insertion</span>. <i>Biochim. Biophys. Acta, Biomembr.</i>  <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1798</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1128</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bbamem.2010.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.bbamem.2010.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=20214875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsFOjtL0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1798&amp;publication_year=2010&amp;pages=1119-1128&amp;issue=6&amp;author=E.+Eir%C3%ADksd%C3%B3ttirauthor=K.+Konateauthor=%C3%9C.+Langelauthor=G.+Divitaauthor=S.+Deshayes&amp;title=Secondary+Structure+of+Cell-Penetrating+Peptides+Controls+Membrane+Interaction+and+Insertion&amp;doi=10.1016%2Fj.bbamem.2010.03.005" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Secondary structure of cell-penetrating peptides controls membrane interaction and insertion</span></div><div class="casAuthors">Eiriksdottir, Emelia; Konate, Karidia; Langel, Uelo; Divita, Gilles; Deshayes, Sebastien</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1798</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1119-1128</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The clin. use of efficient therapeutic agents is often limited by the poor permeability of the biol. membranes.  To enhance their cell delivery, short amphipathic peptides called cell-penetrating peptides (CPPs) have been intensively developed for the last two decades.  CPPs are based either on protein transduction domains, model peptide or chimeric constructs and have been used to deliver cargoes into cells through either covalent or non-covalent strategies.  Although several parameters are simultaneously involved in their internalization mechanism, recent focuses on CPPs suggested that structural properties and interactions with membrane phospholipids could play a major role in the cellular uptake mechanism.  In the present work, we report a comparative anal. of the structural plasticity of 10 well-known CPPs as well as their ability to interact with phospholipid membranes.  We propose a new classification of CPPs based on their structural properties, affinity for phospholipids and internalization pathways already reported in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDSOrAfLyyVbVg90H21EOLACvtfcHk0lih1aq2nINQtQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXlsFOjtL0%253D&amp;md5=aad3a782269b4881ea27decfb1182402</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2010.03.005&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2010.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DEir%25C3%25ADksd%25C3%25B3ttir%26aufirst%3DE.%26aulast%3DKonate%26aufirst%3DK.%26aulast%3DLangel%26aufirst%3D%25C3%259C.%26aulast%3DDivita%26aufirst%3DG.%26aulast%3DDeshayes%26aufirst%3DS.%26atitle%3DSecondary%2520Structure%2520of%2520Cell-Penetrating%2520Peptides%2520Controls%2520Membrane%2520Interaction%2520and%2520Insertion%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Biomembr.%26date%3D2010%26volume%3D1798%26issue%3D6%26spage%3D1119%26epage%3D1128%26doi%3D10.1016%2Fj.bbamem.2010.03.005" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirk, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbas, C. F.</span></span> <span> </span><span class="NLM_article-title">Targeted Gene Knockout by Direct Delivery of Zinc-Finger Nuclease Proteins</span>. <i>Nat. Methods</i>  <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">807</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nmeth.2030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1038%2Fnmeth.2030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=22751204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsVWhtr4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2012&amp;pages=805-807&amp;issue=8&amp;author=T.+Gajauthor=J.+Guoauthor=Y.+Katoauthor=S.+J.+Sirkauthor=C.+F.+Barbas&amp;title=Targeted+Gene+Knockout+by+Direct+Delivery+of+Zinc-Finger+Nuclease+Proteins&amp;doi=10.1038%2Fnmeth.2030" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted gene knockout by direct delivery of zinc-finger nuclease proteins</span></div><div class="casAuthors">Gaj, Thomas; Guo, Jing; Kato, Yoshio; Sirk, Shannon J.; Barbas, Carlos F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">805-807</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Zinc-finger nucleases (ZFNs) are versatile reagents that have redefined genome engineering.  Realizing the full potential of this technol. requires the development of safe and effective methods for delivering ZFNs into cells.  We demonstrate the intrinsic cell-penetrating capabilities of the std. ZFN architecture and show that direct delivery of ZFNs as proteins leads to efficient endogenous gene disruption in various mammalian cell types with minimal off-target effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx85YChQzVbbVg90H21EOLACvtfcHk0lih1aq2nINQtQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XpsVWhtr4%253D&amp;md5=8a0583c307ca9b5584eddfc7c16ec32c</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2030&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2030%26sid%3Dliteratum%253Aachs%26aulast%3DGaj%26aufirst%3DT.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DKato%26aufirst%3DY.%26aulast%3DSirk%26aufirst%3DS.%2BJ.%26aulast%3DBarbas%26aufirst%3DC.%2BF.%26atitle%3DTargeted%2520Gene%2520Knockout%2520by%2520Direct%2520Delivery%2520of%2520Zinc-Finger%2520Nuclease%2520Proteins%26jtitle%3DNat.%2520Methods%26date%3D2012%26volume%3D9%26issue%3D8%26spage%3D805%26epage%3D807%26doi%3D10.1038%2Fnmeth.2030" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirk, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbas, C. F.</span></span> <span> </span><span class="NLM_article-title">Protein Delivery Using Cys2-His2 Zinc-Finger Domains</span>. <i>ACS Chem. Biol.</i>  <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1662</span>– <span class="NLM_lpage">1667</span>, <span class="refDoi">&nbsp;DOI: 10.1021/cb500282g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500282g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvVWnu7s%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2014&amp;pages=1662-1667&amp;issue=8&amp;author=T.+Gajauthor=J.+Liuauthor=K.+E.+Andersonauthor=S.+J.+Sirkauthor=C.+F.+Barbas&amp;title=Protein+Delivery+Using+Cys2-His2+Zinc-Finger+Domains&amp;doi=10.1021%2Fcb500282g" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Delivery Using Cys2-His2 Zinc-Finger Domains</span></div><div class="casAuthors">Gaj, Thomas; Liu, Jia; Anderson, Kimberly E.; Sirk, Shannon J.; Barbas, Carlos F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1662-1667</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of new methods for delivering proteins into cells is a central challenge for advancing both basic research and therapeutic applications.  We previously reported that zinc-finger nuclease proteins are intrinsically cell-permeable due to the cell-penetrating activity of the Cys2-His2 zinc-finger domain.  Here, we demonstrate that genetically fused zinc-finger motifs can transport proteins and enzymes into a wide range of primary and transformed mammalian cell types.  We show that zinc-finger domains mediate protein uptake at efficiencies that exceed conventional protein transduction systems and do so without compromising enzyme activity.  In addn., we demonstrate that zinc-finger proteins enter cells primarily through macropinocytosis and facilitate high levels of cytosolic delivery.  These findings establish zinc-finger proteins as not only useful tools for targeted genome engineering but also effective reagents for protein delivery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMizY5RFUtI7Vg90H21EOLACvtfcHk0lih1aq2nINQtQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXpvVWnu7s%253D&amp;md5=64c4274c4ed4f1924e8cc0db0bf35393</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fcb500282g&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500282g%26sid%3Dliteratum%253Aachs%26aulast%3DGaj%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DAnderson%26aufirst%3DK.%2BE.%26aulast%3DSirk%26aufirst%3DS.%2BJ.%26aulast%3DBarbas%26aufirst%3DC.%2BF.%26atitle%3DProtein%2520Delivery%2520Using%2520Cys2-His2%2520Zinc-Finger%2520Domains%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26issue%3D8%26spage%3D1662%26epage%3D1667%26doi%3D10.1021%2Fcb500282g" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knox, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piszkiewicz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">Cytosolic Delivery of Argininosuccinate Synthetase Using a Cell- Permeant Miniature Protein</span>. <i>ACS Cent. Sci.</i>  <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">641</span>– <span class="NLM_lpage">649</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscentsci.0c01603</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.0c01603" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BB3MXms1yitbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2021&amp;pages=641-649&amp;author=S.+L.+Knoxauthor=R.+Wissnerauthor=S.+Piszkiewiczauthor=A.+Schepartz&amp;title=Cytosolic+Delivery+of+Argininosuccinate+Synthetase+Using+a+Cell-+Permeant+Miniature+Protein&amp;doi=10.1021%2Facscentsci.0c01603" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Cytosolic Delivery of Argininosuccinate Synthetase Using a Cell-Permeant Miniature Protein</span></div><div class="casAuthors">Knox, Susan L.; Wissner, Rebecca; Piszkiewicz, Samantha; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">641-649</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Citrullinemia type I (CTLN-I) results from the absence or deficiency of argininosuccinate synthetase (AS), a 46 kDa enzyme that acts in the cytosol of hepatocytes to convert aspartic acid and citrulline into argininosuccinic acid.  AS is an essential component of the urea cycle, and its absence or deficiency results in the harmful accumulation of ammonia in blood and cerebrospinal fluid.  No disease-modifying treatment of CTLN-I exists.  Here we report that the cell-permeant miniature protein (CPMP) ZF5.3 (ZF) can deliver AS to the cytosol of cells in culture and the livers of healthy mice.  The fusion protein ZF-AS is catalytically active in vitro, stabilized in plasma, and traffics successfully to the cytosol of cultured Saos-2 and SK-HEP-1 cells, achieving cytosolic concns. greater than 100 nM.  This value is 3-10-fold higher than the concn. of endogenous AS (11 ± 1 to 44 ± 5 nM).  When injected into healthy C57BL/6 mice, ZF-AS reaches the mouse liver to establish concns. almost 200 nM above baseline.  These studies demonstrate that ZF5.3 can deliver a complex enzyme to the cytosol at therapeutically relevant concns. and support its application as an improved delivery vehicle for therapeutic proteins that function in the cytosol, including enzyme replacement therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFoQSGSuArBrVg90H21EOLACvtfcHk0lj8Yh4I0y1hJw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3MXms1yitbw%253D&amp;md5=d734983ad5a5b5a2ed9ba86ff07acaf8</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.0c01603&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.0c01603%26sid%3Dliteratum%253Aachs%26aulast%3DKnox%26aufirst%3DS.%2BL.%26aulast%3DWissner%26aufirst%3DR.%26aulast%3DPiszkiewicz%26aufirst%3DS.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DCytosolic%2520Delivery%2520of%2520Argininosuccinate%2520Synthetase%2520Using%2520a%2520Cell-%2520Permeant%2520Miniature%2520Protein%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2021%26volume%3D7%26spage%3D641%26epage%3D649%26doi%3D10.1021%2Facscentsci.0c01603" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Libetti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiaramonte, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minuzzo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gnesutta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messina, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolfini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantovani, R.</span></span> <span> </span><span class="NLM_article-title">NF-YA Enters Cells through Cell Penetrating Peptides</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1866</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">440</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bbamcr.2018.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.bbamcr.2018.10.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=30296497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSjurjP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1866&amp;publication_year=2019&amp;pages=430-440&amp;issue=3&amp;author=D.+Libettiauthor=A.+Bernardiniauthor=M.+L.+Chiaramonteauthor=M.+Minuzzoauthor=N.+Gnesuttaauthor=G.+Messinaauthor=D.+Dolfiniauthor=R.+Mantovani&amp;title=NF-YA+Enters+Cells+through+Cell+Penetrating+Peptides&amp;doi=10.1016%2Fj.bbamcr.2018.10.004" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">NF-YA enters cells through cell penetrating peptides</span></div><div class="casAuthors">Libetti, Debora; Bernardini, Andrea; Chiaramonte, Maria Luisa; Minuzzo, Mario; Gnesutta, Nerina; Messina, Graziella; Dolfini, Diletta; Mantovani, Roberto</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1866</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">430-440</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cell Penetrating Peptides -CPPs- are short aminoacidic stretches present in proteins that have the ability to translocate the plasma membrane and facilitate delivery of various mols.  They are usually rich in basic residues, and organized as alpha helixes.  NF-Y is a transcription factor heterotrimer formed by two Histone Fold Domain -HFD- subunits and the sequence-specific NF-YA.  NF-YA possesses two α-helixes rich in basic residues.  We show that it efficiently enters cells at nanomolar concns. in the absence of carrier peptides.  Mutagenesis identified at least two sep. CPPs in the A1 and A2, which overlap with previously identified nuclear localization signals (NLS).  The half-life of the transduced protein is short in human cancer cells, longer in mouse C2C12 myoblasts.  The internalized NF-YA is capable of trimerization with the HFD subunits and binding to the target CCAAT box.  Functionality is further suggested by protein transfection in C2C12 cells, leading to inhibition of differentiation to myotubes.  In conclusion, NF-YA contains CPPs, hinting at novel -and unexpected- properties of this subunit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCcO49T7KN8rVg90H21EOLACvtfcHk0lj8Yh4I0y1hJw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXitVSjurjP&amp;md5=4b2656757bd627b3885ed0df7506adeb</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2018.10.004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2018.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DLibetti%26aufirst%3DD.%26aulast%3DBernardini%26aufirst%3DA.%26aulast%3DChiaramonte%26aufirst%3DM.%2BL.%26aulast%3DMinuzzo%26aufirst%3DM.%26aulast%3DGnesutta%26aufirst%3DN.%26aulast%3DMessina%26aufirst%3DG.%26aulast%3DDolfini%26aufirst%3DD.%26aulast%3DMantovani%26aufirst%3DR.%26atitle%3DNF-YA%2520Enters%2520Cells%2520through%2520Cell%2520Penetrating%2520Peptides%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2019%26volume%3D1866%26issue%3D3%26spage%3D430%26epage%3D440%26doi%3D10.1016%2Fj.bbamcr.2018.10.004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Y.</span></span> <span> </span><span class="NLM_article-title">The Cell-Penetrating FOXM1 N-Terminus (M1–138) Demonstrates Potent Inhibitory Effects on Cancer Cells by Targeting FOXM1 and FOXM1-Interacting Factor SMAD3</span>. <i>Theranostics</i>  <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2882</span>– <span class="NLM_lpage">2896</span>, <span class="refDoi">&nbsp;DOI: 10.7150/thno.32693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.7150%2Fthno.32693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=31244930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvF2htLnL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2019&amp;pages=2882-2896&amp;issue=10&amp;author=Z.+Zhangauthor=H.+Buauthor=J.+Yuauthor=Y.+Chenauthor=C.+Peiauthor=L.+Yuauthor=X.+Huangauthor=G.+Tanauthor=Y.+Tan&amp;title=The+Cell-Penetrating+FOXM1+N-Terminus+%28M1%E2%80%93138%29+Demonstrates+Potent+Inhibitory+Effects+on+Cancer+Cells+by+Targeting+FOXM1+and+FOXM1-Interacting+Factor+SMAD3&amp;doi=10.7150%2Fthno.32693" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The cell-penetrating FOXM1 N-terminus (M1-138) demonstrates potent inhibitory effects on cancer cells by targeting FOXM1 and FOXM1-interacting factor SMAD3</span></div><div class="casAuthors">Zhang, Zhenwang; Bu, Huitong; Yu, Jingwei; Chen, Yan; Pei, Chaozhu; Yu, Li; Huang, Xiaoqin; Tan, Guixiang; Tan, Yongjun</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2882-2896</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">Transcription factor FOXM1 is involved in stimulating cell proliferation, enhancing DNA damage repair, promoting metastasis of cancer cells, and the inhibition of FOXM1 has been shown to prevent the initiation and progression of multiple cancers and FOXM1 is considered to be an effective target for tumor therapeutic drug development.  The N-terminus of FOXM1 has been found to prevent transcriptional activities of FOXM1 and to mediate the interaction between FOXM1 and SMAD3.  A recombinant FOXM1 N-terminal domain (1-138aa) fused with a nine arginine cell-penetrating peptide is produced with an E. coli expression system and named as M1-138.  The effects of M1-138 on the proliferation, migration, and tumorigenic ability of cancer cells are analyzed in vitro with cell counting, transwell assays, and colony formation assays.  Electrophoretic mobility shift assays (EMSAs) and Luciferase activity assays are used to test the DNA binding ability and transcriptional activity of transcription factors.  The levels of mRNAs and proteins are measured by quant.-PCR, Western blotting or Immunohistochem.  The interactions among proteins are analyzed with Pull-down and Co-immunopptn. (Co-IP) assays.  The nude mouse engrafted tumor models are used to test the inhibitory effects of M1-138 in vivo.  M1-138 diminishes the proliferation and migration abilities of cancer cells through binding to FOXM1 and FOXM1-interacting factor SMAD3, and consequently attenuating FOXM1 transcriptional activities from both direct and indirect FOXM1-promoter binding mechanisms and interfering with the interaction between FOXM1 and SMAD3.  Treatment of M1-138 prevents tumorigenicity of cancer cells and inhibits tumor growth in nude mouse xenograft models with no obvious signs of toxicity.  M1-138 is a promising drug candidate for the development of anti-cancer therapeutics targeting FOXM1 and SMAD3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtiXontFCdrVg90H21EOLACvtfcHk0lj8Yh4I0y1hJw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXhvF2htLnL&amp;md5=bdb0f9b36854b622c6a0e41d12684499</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.7150%2Fthno.32693&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.32693%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DBu%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DPei%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DTan%26aufirst%3DG.%26aulast%3DTan%26aufirst%3DY.%26atitle%3DThe%2520Cell-Penetrating%2520FOXM1%2520N-Terminus%2520%2528M1%25E2%2580%2593138%2529%2520Demonstrates%2520Potent%2520Inhibitory%2520Effects%2520on%2520Cancer%2520Cells%2520by%2520Targeting%2520FOXM1%2520and%2520FOXM1-Interacting%2520Factor%2520SMAD3%26jtitle%3DTheranostics%26date%3D2019%26volume%3D9%26issue%3D10%26spage%3D2882%26epage%3D2896%26doi%3D10.7150%2Fthno.32693" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jolma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campitelli, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taipale, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weirauch, M. T.</span></span> <span> </span><span class="NLM_article-title">The Human Transcription Factors</span>. <i>Cell</i>  <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>172</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">650</span>– <span class="NLM_lpage">665</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2018.01.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=10.1016%2Fj.cell.2018.01.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=29425488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facscentsci.1c00663&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1amsbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=172&amp;publication_year=2018&amp;pages=650-665&amp;issue=4&amp;author=S.+A.+Lambertauthor=A.+Jolmaauthor=L.+F.+Campitelliauthor=P.+K.+Dasauthor=Y.+Yinauthor=M.+Albuauthor=X.+Chenauthor=J.+Taipaleauthor=T.+R.+Hughesauthor=M.+T.+Weirauch&amp;title=The+Human+Transcription+Factors&amp;doi=10.1016%2Fj.cell.2018.01.029" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The Human Transcription Factors</span></div><div class="casAuthors">Lambert, Samuel A.; Jolma, Arttu; Campitelli, Laura F.; Das, Pratyush K.; Yin, Yimeng; Albu, Mihai; Chen, Xiaoting; Taipale, Jussi; Hughes, Timothy R.; Weirauch, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">650-665</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Transcription factors (TFs) recognize specific DNA sequences to control chromatin and transcription, forming a complex system that guides expression of the genome.  Despite keen interest in understanding how TFs control gene expression, it remains challenging to det. how the precise genomic binding sites of TFs are specified and how TF binding ultimately relates to regulation of transcription.  This review considers how TFs are identified and functionally characterized, principally through the lens of a catalog of over 1600 likely human TFs and binding motifs for two-thirds of them.  Major classes of human TFs differ markedly in their evolutionary trajectories and expression patterns, underscoring distinct functions.  TFs likewise underlie many different aspects of human physiol., disease, and variation, highlighting the importance of continued effort to understand TF-mediated gene regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGN_jQqNzpvrVg90H21EOLACvtfcHk0ljKOL30XUjV4Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXis1amsbs%253D&amp;md5=a53a2bf38c15419c7589893428efd738</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.01.029&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.01.029%26sid%3Dliteratum%253Aachs%26aulast%3DLambert%26aufirst%3DS.%2BA.%26aulast%3DJolma%26aufirst%3DA.%26aulast%3DCampitelli%26aufirst%3DL.%2BF.%26aulast%3DDas%26aufirst%3DP.%2BK.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DAlbu%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DTaipale%26aufirst%3DJ.%26aulast%3DHughes%26aufirst%3DT.%2BR.%26aulast%3DWeirauch%26aufirst%3DM.%2BT.%26atitle%3DThe%2520Human%2520Transcription%2520Factors%26jtitle%3DCell%26date%3D2018%26volume%3D172%26issue%3D4%26spage%3D650%26epage%3D665%26doi%3D10.1016%2Fj.cell.2018.01.029" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i25">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_20646"></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acscentsci.1c00663?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acscentsci.1c00663</a>.</p><ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthesis and experimental procedures as well as characterization of all compounds (<a class="ext-link" href="/doi/suppl/10.1021/acscentsci.1c00663/suppl_file/oc1c00663_si_001.pdf">PDF</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.1c00663/suppl_file/oc1c00663_si_001.pdf">oc1c00663_si_001.pdf (3.91 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms &amp; Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facscentsci.1c00663&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facscentsci.1c00663%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3BrequestedJournal%3Ajournal%3Aacscii%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Facscii.2021.7.issue-8%3Bjournal%3Ajournal%3Aacscii%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acscentsci.1c00663" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2022<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS &amp; Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support &amp; Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    gtag('event', 'Export', { 'event_category': 'Citation', 'event_label':'Main'});

	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   
	   gtag('event', 'Export', { 'event_category': 'Citation', 'event_label':label});
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        gtag('event', 'Top', { 'event_category': 'Article', 'event_label':'Right Arrow'});
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     gtag('event', 'Top', { 'event_category': 'Article', 'event_label':'Left Arrow'});
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		gtag('event', 'Top', { 'event_category': 'Article', 'event_label':'Share'});
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		gtag('event', 'Sidebar', { 'event_category': 'Article', 'event_label':'Figures'});
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
	    gtag('event', 'Sidebar', { 'event_category': 'Article', 'event_label':'References'});
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		gtag('event', 'Sidebar', { 'event_category': 'Article', 'event_label':'Supporting Info'});
	});

	$('.tab__close').on('click', function(){
		gtag('event', 'Sidebar', { 'event_category': 'Article', 'event_label':'Close'});
	}); 



})();

</script>




        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"><input type="hidden" name="appID" value="literatum" class="appID"><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onclick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        <!-- <div id="gdpr-message-container">
			<a id="closebtn" href="#"><i class="icon-times-circle-o"></i></a> -->
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        <!-- </div> -->
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        gtag('event', 'GDPR Modal', { 'event_category': 'Pubs GDPR', 'event_label':'GDPR Dismissal'});
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            gtag('event', 'Button', { 'event_category': 'Top Nav', 'event_label':'Activity'});
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            gtag('event', 'Button', { 'event_category': 'Top Nav', 'event_label':'Publications'});

        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            gtag('event', 'Button', { 'event_category': 'Top Nav', 'event_label':'Hamburger'});
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        <!-- AdRoll Pixel -->
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>

<!-- Twitter universal website tag code -->
<!-- Commented out for ACHS-6152
<script>
    !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe.apply(s,arguments):s.queue.push(arguments);
    },s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',
    a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');
    // Insert Twitter Pixel ID and Standard Event data below
    twq('init','o3njp');
    twq('track','PageView');
</script>
-->
<!-- Facebook Pixel Code -->
<!-- Commented out for ACHS-6152
<script>
    !function(f,b,e,v,n,t,s)
    {if(f.fbq)return;n=f.fbq=function(){n.callMethod?
    n.callMethod.apply(n,arguments):n.queue.push(arguments)};
    if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
    n.queue=[];t=b.createElement(e);t.async=!0;
    t.src=v;s=b.getElementsByTagName(e)[0];
    s.parentNode.insertBefore(t,s)}(window, document,'script',
    'https://connect.facebook.net/en_US/fbevents.js');
    fbq('init', '529840281033320');
    fbq('track', 'PageView');
</script>
<noscript><img height="1" width="1" style="display:none" src="https://www.facebook.com/tr?id=529840281033320&amp;ev=PageView&amp;noscript=1"
/></noscript>
-->





        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
var currentJournal = "ACS Central Science";
$(".citationLinks > a").click(function(e){
    if($(this).text() === "ACS Full Text ") {
        gtag('event', 'ACS Full Text', { 'event_category': 'Reference _Click', 'event_label':currentJournal});
    }
    else if($(this).text() === "Crossref") {
        gtag('event', 'Crossref', { 'event_category': 'Reference _Click', 'event_label':currentJournal});
    }
    else if($(this).text() === "CAS") {
        gtag('event', 'CAS', { 'event_category': 'Reference _Click', 'event_label':currentJournal});
    }
    else if($(this).text() === "PubMed") {
        gtag('event', 'PubMed', { 'event_category': 'Reference _Click', 'event_label':currentJournal});
    }
    else if($(this).text() === "Google Scholar") {
        gtag('event', 'Google Scholar', { 'event_category': 'Reference _Click', 'event_label':currentJournal});
    }
    else if($(this).hasClass("issue-item_open-url") === true) {
        gtag('event', 'OpenURL', { 'event_category': 'Reference _Click', 'event_label':currentJournal});
    }
});
</script>




        
        



        
    





        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        <div class="showMenuAlertMsg-popup"><div tabindex="0" class="pairing-affiliations-popup-content"><div class="pairing-affiliations-header"><button type="button" class="close paringAffClose"><i aria-hidden="true" class="icon-close_thin"></i></button></div><div class="responsrMsg"></div></div></div>




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	dartInstitutionType = 'education';
</script>
        </div>
    





        
        



        
    





        
        <script>
// LIT-455776 workaround
// Moving the popup before the header content for tab accessibility
document.addEventListener('DOMContentLoaded', () => {
    const headerNode = $('.header_content');
    const popupNode = $('.pairing-affiliations-popup');

    if (!headerNode.length || !popupNode.length)
        return;

    popupNode.insertBefore(headerNode);  
});
</script> 




        
        <!-- begin SnapEngage code -->
<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>
<!-- end SnapEngage code -->
        </div>
    </div>





	
    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-4decb3f7aee589293b6b.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-05a28f2b6d38c65dce85.js"></script>
<script type="text/javascript" src="/wro/lmwf~product.js"></script>

















<script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/v652eace1692a40cfa3763df669d7439c1639079717194" integrity="sha512-Gi7xpJR8tSkrpF7aordPZQlW2DLtzUlZcumS8dMQjwDHEnw9I7ZLyiOj/6tZStRBGtGgN6ceN6cMH8z7etPGlw==" data-cf-beacon="{&quot;rayId&quot;:&quot;754026a1cb83a61f&quot;,&quot;token&quot;:&quot;0562ec7bcba14d1cae3543aa43d6e749&quot;,&quot;version&quot;:&quot;2022.8.1&quot;,&quot;si&quot;:100}" crossorigin="anonymous"></script>


<div id="designstudio-button" class="designstudio-button" style="position:fixed;z-index:999998;left:-9999px;font-size:14px;"><button id="transparent-button" class="transparent-button" type="button" style="background: transparent !important; border: none !important; cursor: pointer !important; position: absolute !important; left: 0 !important; top: 0 !important; bottom: 0 !important; right: 0 !important; width: 100% !important; height: 100% !important; display: block !important; text-indent: -99999px !important; text-align: left !important; padding: 0 !important; overflow: hidden !important; z-index: 1 !important; border-radius: 0 !important; margin: 0 !important; -webkit-appearance: none !important; box-shadow: none !important; line-height: 100% !important; ">Live Chat</button><iframe id="iframe-designstudio-button" translate="no" style="border:none;display:block;margin:0.7142857142857143em;width:4.285714285714286em;height:4.285714285714286em;" name="designstudio-button-frame" title="Live Chat Button" tabindex="-1" scrolling="no"></iframe></div><div id="designstudio" style="font-size:16px;-webkit-transform:translateZ(0);-moz-transform:translateZ(0);-ms-transform:translateZ(0);transform:translateZ(0);display:none;z-index:100003;position:fixed;height:32.75em;width:25.25em;right:0;float:right;bottom:0;"><div id="draggable" style="width:calc(100% - 5.9375em);left:0;top:0;position:absolute;height:3em;z-index:13000;margin-top:12px;margin-left:12px;cursor:move;"></div><iframe id="designstudio-iframe" translate="no" style="-webkit-transform:translateZ(0);-moz-transform:translateZ(0);-ms-transform:translateZ(0);transform:translateZ(0);display:block;position:absolute;border:none;width:100%;height:100%;" name="designstudio-frame" title="Live Chat Box"></iframe></div><div id="designstudio-minimize" style="position:fixed;z-index:999998;display:none;" tabindex="0"><iframe id="iframe-minimize" translate="no" style="border:none;display:block;font-size:14px;" name="designstudio-minimize-frame" title="Live Chat Button" tabindex="-1" scrolling="no"><div id="minimize-content"><div id="minimize-notify" class=" minimize-notify-right-avatar minimize-notify-bottom"><span id="minimize-notify-counter"></span></div><div id="minimize-avatar-container" class=" right bottom"><img id="minimize-avatar" alt="minimized chat"></div><div id="minimize-svg" class=" right bottom"><div id="svg-container"><svg viewBox="0 0 600 600"><defs><filter id="dropshadow" height="140%"><fegaussianblur in="SourceAlpha" stddeviation="15"></fegaussianblur><feoffset result="offsetblur" dx="10" dy="10"></feoffset><fecomponenttransfer><fefunca type="linear" slope="0.8"></fefunca></fecomponenttransfer><femerge><femergenode></femergenode><femergenode in="SourceGraphic"></femergenode></femerge></filter></defs><g id="comments-o" class="designstudio-container"><circle class="designstudio-circle" cx="50%" cy="50%" r="260" fill="#FF9900" linejoin="round" stroke="#FFFFFF" stroke-width="0" fill-opacity="1" position="absolute"></circle><path class="designstudio-icon" fill="#FFFFFF" d="M704 1152q-153 0 -286 -52t-211.5 -141t-78.5 -191q0 -82 53 -158t149 -132l97 -56l-35 -84q34 20 62 39l44 31l53 -10q78 -14 153 -14q153 0 286 52t211.5 141t78.5 191t-78.5 191t-211.5 141t-286 52zM704 1280q191 0 353.5 -68.5t256.5 -186.5t94 -257t-94 -257 t-256.5 -186.5t-353.5 -68.5q-86 0 -176 16q-124 -88 -278 -128q-36 -9 -86 -16h-3q-11 0 -20.5 8t-11.5 21q-1 3 -1 6.5t0.5 6.5t2 6l2.5 5t3.5 5.5t4 5t4.5 5t4 4.5q5 6 23 25t26 29.5t22.5 29t25 38.5t20.5 44q-124 72 -195 177t-71 224q0 139 94 257t256.5 186.5 t353.5 68.5zM1526 111q10 -24 20.5 -44t25 -38.5t22.5 -29t26 -29.5t23 -25q1 -1 4 -4.5t4.5 -5t4 -5t3.5 -5.5l2.5 -5t2 -6t0.5 -6.5t-1 -6.5q-3 -14 -13 -22t-22 -7q-50 7 -86 16q-154 40 -278 128q-90 -16 -176 -16q-271 0 -472 132q58 -4 88 -4q161 0 309 45t264 129 q125 92 192 212t67 254q0 77 -23 152q129 -71 204 -178t75 -230q0 -120 -71 -224.5t-195 -176.5z" transform="scale(.195, -.195) translate(600, -2000)"></path></g></svg></div></div><div id="minimize-messages" class=" message-left bottom"></div></div></iframe></div><style id="design-studio-animate">.designstudio-animated {-webkit-animation-duration: .5s;animation-duration: .5s;-webkit-animation-fill-mode: both;animation-fill-mode: both;}@keyframes designstudio-fadeOut {from {opacity: 1;}to {opacity: 0;}}.designstudio-fadeOut {animation-name: designstudio-fadeOut;}@-webkit-keyframes designstudio-fadeInUp {from {opacity: 0;-webkit-transform: translate3d(0, 100%, 0);transform: translate3d(0, 100%, 0);}to {opacity: 1;-webkit-transform: none;transform: none;}}@keyframes designstudio-fadeInUp {from {opacity: 0;-webkit-transform: translate3d(0, 100%, 0);transform: translate3d(0, 100%, 0);}to {opacity: 1;-webkit-transform: none;transform: none;}}.designstudio-fadeInUp {-webkit-animation-name: designstudio-fadeInUp;animation-name: designstudio-fadeInUp;}@-webkit-keyframes designstudio-fadeInDown {from {opacity: 0;-webkit-transform: translate3d(0, -100%, 0);transform: translate3d(0, -100%, 0);}to {opacity: 1;-webkit-transform: none;transform: none;}}@keyframes designstudio-fadeInDown {from {opacity: 0;-webkit-transform: translate3d(0, -100%, 0);transform: translate3d(0, -100%, 0);}to {opacity: 1;-webkit-transform: none;transform: none;}}.designstudio-fadeInDown {-webkit-animation-name: designstudio-fadeInDown;animation-name: designstudio-fadeInDown;}@-webkit-keyframes designstudio-fadeInLeft {from {opacity: 0;-webkit-transform: translate3d(-100%, 0, 0);transform: translate3d(-100%, 0, 0);}to {opacity: 1;-webkit-transform: none;transform: none;}}@keyframes designstudio-fadeInLeft {from {opacity: 0;-webkit-transform: translate3d(-100%, 0, 0);transform: translate3d(-100%, 0, 0);}to {opacity: 1;-webkit-transform: none;transform: none;}}.designstudio-fadeInLeft {-webkit-animation-name: designstudio-fadeInLeft;animation-name: designstudio-fadeInLeft;}@-webkit-keyframes designstudio-fadeInRight {from {opacity: 0;-webkit-transform: translate3d(100%, 0, 0);transform: translate3d(100%, 0, 0);}to {opacity: 1;-webkit-transform: none;transform: none;}}@keyframes designstudio-fadeInRight {from {opacity: 0;-webkit-transform: translate3d(100%, 0, 0);transform: translate3d(100%, 0, 0);}to {opacity: 1;-webkit-transform: none;transform: none;}}.designstudio-fadeInRight {-webkit-animation-name: designstudio-fadeInRight;animation-name: designstudio-fadeInRight;}@-webkit-keyframes designstudio-fadeOutUp {from {opacity: 1;}to {opacity: 0;-webkit-transform: translate3d(0, -100%, 0);transform: translate3d(0, -100%, 0);}}@keyframes designstudio-fadeOutUp {from {opacity: 1;}to {opacity: 0;-webkit-transform: translate3d(0, -100%, 0);transform: translate3d(0, -100%, 0);}}.designstudio-fadeOutUp {-webkit-animation-name: designstudio-fadeOutUp;animation-name: designstudio-fadeOutUp;}@-webkit-keyframes designstudio-fadeOutDown {from {opacity: 1;}to {opacity: 0;-webkit-transform: translate3d(0, 100%, 0);transform: translate3d(0, 100%, 0);}}@keyframes designstudio-fadeOutDown {from {opacity: 1; }to {opacity: 0;-webkit-transform: translate3d(0, 100%, 0);transform: translate3d(0, 100%, 0);}}.designstudio-fadeOutDown {-webkit-animation-name: designstudio-fadeOutDown;animation-name: designstudio-fadeOutDown;}@-webkit-keyframes designstudio-fadeOutLeft {from {opacity: 1;}to {opacity: 0;-webkit-transform: translate3d(-100%, 0, 0);transform: translate3d(-100%, 0, 0);}}@keyframes designstudio-fadeOutLeft {from {opacity: 1;}to {opacity: 0;-webkit-transform: translate3d(-100%, 0, 0);transform: translate3d(-100%, 0, 0);}}.designstudio-fadeOutLeft {-webkit-animation-name: designstudio-fadeOutLeft;animation-name: designstudio-fadeOutLeft;}@-webkit-keyframes designstudio-fadeOutRight {from {opacity: 1;}to {opacity: 0;-webkit-transform: translate3d(100%, 0, 0);transform: translate3d(100%, 0, 0);}}@keyframes designstudio-fadeOutRight {from {opacity: 1;}to {opacity: 0;-webkit-transform: translate3d(100%, 0, 0);transform: translate3d(100%, 0, 0);}}.designstudio-fadeOutRight {-webkit-animation-name: designstudio-fadeOutRight;animation-name: designstudio-fadeOutRight;}@-webkit-keyframes designstudio-slideInDown {from {-webkit-transform: translate3d(0, -100% , 0);transform: translate3d(0, -100% , 0);visibility: visible;}to {-webkit-transform: translate3d(0, 0, 0);transform: translate3d(0, 0, 0);}}@keyframes designstudio-slideInDown {from {-webkit-transform: translate3d(0, -100% , 0);transform: translate3d(0, -100% , 0);visibility: visible;}to {-webkit-transform: translate3d(0, 0, 0);transform: translate3d(0, 0, 0);}}.designstudio-slideInDown {-webkit-animation-name: designstudio-slideInDown;animation-name: designstudio-slideInDown;}@-webkit-keyframes designstudio-slideInLeft {from {-webkit-transform: translate3d(-100% , 0, 0);transform: translate3d(-100% , 0, 0);visibility: visible;}to {-webkit-transform: translate3d(0, 0, 0);transform: translate3d(0, 0, 0);}}@keyframes designstudio-slideInLeft {from {-webkit-transform: translate3d(-100% , 0, 0);transform: translate3d(-100% , 0, 0);visibility: visible;}to {-webkit-transform: translate3d(0, 0, 0);transform: translate3d(0, 0, 0);}}.designstudio-slideInLeft {-webkit-animation-name: designstudio-slideInLeft;animation-name: designstudio-slideInLeft;}@-webkit-keyframes designstudio-slideInRight {from {-webkit-transform: translate3d(100% , 0, 0);transform: translate3d(100% , 0, 0);visibility: visible;}to {-webkit-transform: translate3d(0, 0, 0);transform: translate3d(0, 0, 0);}}@keyframes designstudio-slideInRight {from {-webkit-transform: translate3d(100% , 0, 0);transform: translate3d(100% , 0, 0);visibility: visible;}to {-webkit-transform: translate3d(0, 0, 0);transform: translate3d(0, 0, 0);}}.designstudio-slideInRight {-webkit-animation-name: designstudio-slideInRight;animation-name: designstudio-slideInRight;}@-webkit-keyframes designstudio-slideInUp {from {-webkit-transform: translate3d(0, 100% , 0);transform: translate3d(0, 100% , 0);visibility: visible;}to {-webkit-transform: translate3d(0, 0, 0);transform: translate3d(0, 0, 0);}}@keyframes designstudio-slideInUp {from {-webkit-transform: translate3d(0, 100% , 0);transform: translate3d(0, 100% , 0);visibility: visible;}to {-webkit-transform: translate3d(0, 0, 0);transform: translate3d(0, 0, 0);}} .designstudio-slideInUp {-webkit-animation-name: designstudio-slideInUp;animation-name: designstudio-slideInUp;}</style><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh230" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.4851396838720402&amp;iit=1664742156568&amp;tmr=load%3D1664742156333%26core%3D1664742156374%26main%3D1664742156562%26ifr%3D1664742156577&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=pubs.acs.org&amp;dr=&amp;du=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facscentsci.1c00663&amp;href=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facscentsci.1c00663&amp;dt=Parallel%20Automated%20Flow%20Synthesis%20of%20Covalent%20Protein%20Complexes%20That%20Can%20Inhibit%20MYC-Driven%20Transcription&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Ctype%3DArticle%2Cdescription%2Cimage%2Ctitle%2Curl&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=6339f30c508b3fa9&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=url%3Dhttps%253A%252F%252Fpubs.acs.org%252Fdoi%252Ffull%252F10.1021%252Facscentsci.1c00663%26title%3DParallel%2520Automated%2520Flow%2520Synthesis%2520of%2520Covalent%2520Protein%2520Complexes%2520That%2520Can%2520Inhibit%2520MYC-Driven%2520Transcription%26image%3Dhttps%253A%252F%252Fpubs.acs.org%252Fcms%252F10.1021%252Facscentsci.1c00663%252Fasset%252Fimages%252Facscentsci.1c00663.social.jpeg_v03%26description%3DDysregulation%2520of%2520the%2520transcription%2520factor%2520MYC%2520is%2520involved%2520in%2520many%2520human%2520cancers.%2520The%2520dimeric%2520transcription%2520factor%2520complexes%2520of%2520MYC%252FMAX%2520and%2520MAX%252FMAX%2520activate%2520or%2520inhibit%252C%2520respectively%252C%2520gene%2520transcription%2520upon%2520binding%2520to%2520the%2520same%2520enhancer%2520box%2520DNA.%2520Targeting%2520these%2520complexes%2520in%2520cancer%2520is%2520a%2520long-standing%2520challenge.%2520Inspired%2520by%2520the%2520inhibitory%2520activity%2520of%2520the%2520MAX%252FMAX%2520dimer%252C%2520we%2520engineered%2520covalently%2520linked%252C%2520synthetic%2520homo-%2520and%2520heterodimeric%2520protein%2520complexes%2520to%2520attenuate%2520oncogenic%2520MYC-driven%2520transcription.%2520We%2520prepared%2520the%2520covalent%2520protein%2520complexes%2520(%25E2%2588%25BC20%2520kDa%252C%2520167%25E2%2580%2593231%2520residues)%2520in%2520a%2520single%2520shot%2520via%2520parallel%2520automated%2520flow%2520synthesis%2520in%2520hours.%2520The%2520stabilized%2520covalent%2520dimers%2520display%2520DNA%2520binding%2520activity%252C%2520are%2520intrinsically%2520cell-penetrant%252C%2520and%2520inhibit%2520cancer%2520cell%2520proliferation%2520in%2520different%2520cell%2520lines.%2520RNA%2520sequencing%2520and%2520gene%2520set%2520enrichment%2520analysis%2520in%2520A549%2520cancer%2520cells%2520confirmed%2520that%2520the%2520synthetic%2520dimers%2520interfere%2520with%2520MYC-driven%2520transcription.%2520Our%2520results%2520demonstrate%2520the%2520potential%2520of%2520automated%2520flow%2520technology%2520to%2520rapidly%2520deliver%2520engineered%2520synthetic%2520protein%2520complex%2520mimetics%2520that%2520can%2520serve%2520as%2520a%2520starting%2520point%2520in%2520developing%2520inhibitors%2520of%2520MYC-driven%2520cancer%2520cell%2520growth.%26type%3DArticle%26site_name%3DACS%2520Publications&amp;csi=undefined&amp;rev=v8.28.8-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style></body></html>