<html class="pb-page" data-request-id="63b054b8-22ec-49cb-a701-1b0548f9fced" lang="en"><head data-pb-dropzone="head"><script src="https://sso.acs.org/idp/status?padding=jsonPaddingFunction&amp;appid=atypon-lives&amp;callback=jsonPaddingFunction&amp;_=1651100219196"></script><script type="text/javascript" src="https://m.addthis.com/live/red_lojson/300lo.json?si=6269ca3bc7d77d46&amp;bkl=0&amp;bl=1&amp;pdt=567&amp;sid=6269ca3bc7d77d46&amp;pub=xa-4faab26f2cff13a7&amp;rev=v8.28.8-wp&amp;ln=en&amp;pc=men&amp;cb=0&amp;ab=-&amp;dp=pubs.acs.org&amp;fp=doi%2F10.1021%2Facscentsci.8b00566&amp;fr=&amp;of=2&amp;pd=0&amp;irt=1&amp;vcl=1&amp;md=0&amp;ct=1&amp;tct=0&amp;abt=0&amp;cdn=0&amp;pi=1&amp;rb=0&amp;gen=100&amp;chr=UTF-8&amp;colc=1651100219535&amp;jsl=1&amp;uvs=6269ca35afc5ed71003&amp;skipb=1&amp;callback=addthis.cbs.jsonp__359233639438471530"></script><script type="text/javascript" src="https://v1.addthisedge.com/live/boost/xa-4faab26f2cff13a7/_ate.track.config_resp"></script><script src="https://widgets.figshare.com/static/figshare.js?_=1651100219195"></script><script type="text/javascript" src="https://z.moatads.com/addthismoatframe568911941483/moatframe.js"></script><meta name="pbContext" content=";article:article:10.1021/acscentsci.8b00566;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;requestedJournal:journal:acscii;website:website:acspubs;journal:journal:acscii;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/acscii.2019.5.issue-1">




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="ACS Central Science"><meta name="dc.Title" content="Effective Distance for DNA-Mediated Charge Transport between Repair Proteins"><meta name="dc.Creator" content="Edmund
C. M.  Tse"><meta name="dc.Creator" content="Theodore J.  Zwang"><meta name="dc.Creator" content="Sebastian  Bedoya"><meta name="dc.Creator" content="Jacqueline K.  Barton"><meta name="dc.Description" content="The stacked aromatic base pairs within the DNA double helix facilitate charge transport down its length in the absence of lesions, mismatches, and other stacking perturbations. DNA repair proteins ..."><meta name="Description" content="The stacked aromatic base pairs within the DNA double helix facilitate charge transport down its length in the absence of lesions, mismatches, and other stacking perturbations. DNA repair proteins ..."><meta name="dc.Description" content="The length over which DNA-mediated charge transport between [4Fe4S] repair proteins is effective is found to be ∼3500 base pairs, providing a route for a rapid genome search for DNA lesions."><meta name="Description" content="The length over which DNA-mediated charge transport between [4Fe4S] repair proteins is effective is found to be ∼3500 base pairs, providing a route for a rapid genome search for DNA lesions."><meta name="dc.Publisher" content="American Chemical Society"><meta name="dc.Date" scheme="WTN8601" content="January 11, 2019"><meta name="dc.Type" content="research-article"><meta name="dc.Format" content="text/HTML"><meta name="dc.Identifier" scheme="doi" content="10.1021/acscentsci.8b00566"><meta name="dc.Language" content="en"><meta name="dc.Coverage" content="world"><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society"><meta name="citation_fulltext_world_readable" content="">
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facscentsci.8b00566">
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facscentsci.8b00566">
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facscentsci.8b00566">
        
    
    

<title>Effective Distance for DNA-Mediated Charge Transport between Repair Proteins | ACS Central Science</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive">





    






        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acscentsci.8b00566"><meta property="og:title" content="Effective Distance for DNA-Mediated Charge Transport between Repair Proteins"><meta property="og:image" content="https://pubs.acs.org/cms/10.1021/acscentsci.8b00566/asset/images/acscentsci.8b00566.social.jpeg_v03"><meta property="og:description" content="The stacked aromatic base pairs within the DNA double helix facilitate charge transport down its length in the absence of lesions, mismatches, and other stacking perturbations. DNA repair proteins containing [4Fe4S] clusters can take advantage of DNA charge transport (CT) chemistry to scan the genome for mistakes more efficiently. Here we examine the effective length over which charge can be transported along DNA between these repair proteins. We define the effective CT distance as the length of DNA within which two proteins are able to influence their ensemble affinity to the DNA duplex via CT. Endonuclease III, a DNA repair glycosylase containing a [4Fe4S] cluster, was incubated with DNA duplexes of different lengths (1.5–9 kb), and atomic force microscopy was used to quantify the binding of proteins to these duplexes to determine how the relative protein affinity changes with increasing DNA length. A sharp change in binding slope is observed at 3509 base pairs, or about 1.2 μm, that supports the existence of two regimes for protein binding, one within the range for DNA CT, one outside of the range for CT; DNA CT between the redox proteins bound to DNA effectively decreases the ensemble binding affinity of oxidized and reduced proteins to DNA. Utilizing an Endonuclease III mutant Y82A, which is defective in carrying out DNA CT, shows only one regime for protein binding. Decreasing the temperature to 4 °C or including metallointercalators on the duplex, both of which should enhance base stacking and decrease DNA floppiness, leads to extending the effective length for DNA charge transport to ∼5300 bp or 1.8 μm. These results thus support DNA charge transport between repair proteins over kilobase distances. The results furthermore highlight the ability of DNA repair proteins to search the genome quickly and efficiently&nbsp;using DNA charge transport chemistry."><meta property="og:type" content="Article"><meta property="og:site_name" content="ACS Publications"><meta name="twitter:card" content="summary_large_image"><meta name="twitter:site" content="@ACSPublications"><meta name="twitter:image" content="https://pubs.acs.org/cms/10.1021/acscentsci.8b00566/asset/images/acscentsci.8b00566.social.jpeg_v03">
    





    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0">















    

    <meta name="dc.identifier" content="10.1021/acscentsci.8b00566">



<meta name="format-detection" content="telephone=no">




<link rel="stylesheet" type="text/css" href="/wro/l6j1~product.css">

<link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-c7d2282eca032afdc856.css"><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css">




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1640088552890.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1648583148410.css">
<script type="text/javascript" async="" src="//img.en25.com/i/elqCfg.min.js"></script><script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script><script async="" src="//www.google-analytics.com/analytics.js"></script><script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer="" src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon="{&quot;rayId&quot;:&quot;5200035e6e4d5180&quot;,&quot;startTime&quot;:1570117212010}"></script>

            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    , 'auto'
                    );
                
                ga('send', 'pageview');
            </script>
            











    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acscentsci.8b00566">
            
        







    <!-- loading Javascript for using the persistence Service -->
    <script src="/templates/jsp/pb2/js/plugin/thiss-ds.js"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            alt_context: window.location.origin,
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if (ms == undefined || ms === '') {
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            if (results.length == 0) {
                                persistService.entities(self.alt_context).then((res) => {
                                    return resolve(res.data);
                                }, function (err) {
                                    // failed
                                    reject('failed to retrieve entity from local storage (publisher domain): ' + err);
                                });
                            } else {
                                return resolve(results);
                            }
                        }, function (err) {
                            // failed
                            reject('failed to retrieve entity from local storage: ' + err);
                        });
                    // Set up the timeout
                    setTimeout(function () {
                        reject('Promise timed out ' + ms + ' ms');
                    }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                    persistService.update(self.alt_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage (publisher domain) duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            var results = res.data;
                            if (!results) {
                                persistService.remove(self.alt_context, entityId).then((res) => {
                                    return resolve(res);
                                }, function (err) {
                                    // failed
                                    console.log('failed to remove from local storage (publisher domain) duo to: ' + err);
                                });
                            }
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId, entityTitle) {
                return new Promise(function (resolve, reject) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {
                            doi: doi,
                            entityid: entityId,
                            entityTitle: entityTitle,
                            eventGroupKey: '6c2508a6-f4ac-4cdf-a65d-b45ce4ca35a8'
                        }

                    }).done(function (status) {
                        return resolve(status);
                    }).fail(function (status) {
                        return reject(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID, ssoInstitution.entity.title)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + ssoInstitution.entity.entityID,err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            }).catch(function (err) {
                                reject(err);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();

                    if ($('.institution-preference-userconsent-checkbox').length > 0) {
                        let institutionPreferenceUserConsentCheckbox = $('.institution-preference-userconsent-checkbox').get(0);
                        if (!$(institutionPreferenceUserConsentCheckbox).is(":checked")) {
                            return;
                        }
                    }

                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err) {
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>







    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY">


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html">

<link rel="shortcut icon" href="/pb-assets/ux3/favicon-1635812676023.ico"><link type="text/css" rel="stylesheet" href="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/CcdcWidget.min.css"><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><meta http-equiv="origin-trial" content="AxujKG9INjsZ8/gUq8+dTruNvk7RjZQ1oFhhgQbcTJKDnZfbzSTE81wvC2Hzaf3TW4avA76LTZEMdiedF1vIbA4AAABueyJvcmlnaW4iOiJodHRwczovL2ltYXNkay5nb29nbGVhcGlzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzVGhpcmRQYXJ0eSI6dHJ1ZX0="><meta http-equiv="origin-trial" content="Azuce85ORtSnWe1MZDTv68qpaW3iHyfL9YbLRy0cwcCZwVnePnOmkUJlG8HGikmOwhZU22dElCcfrfX2HhrBPAkAAAB7eyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A16nvcdeoOAqrJcmjLRpl1I6f3McDD8EfofAYTt/P/H4/AWwB99nxiPp6kA0fXoiZav908Z8etuL16laFPUdfQsAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="AxBHdr0J44vFBQtZUqX9sjiqf5yWZ/OcHRcRMN3H9TH+t90V/j3ENW6C8+igBZFXMJ7G3Pr8Dd13632aLng42wgAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A88BWHFjcawUfKU3lIejLoryXoyjooBXLgWmGh+hNcqMK44cugvsI5YZbNarYvi3roc1fYbHA1AVbhAtuHZflgEAAAB2eyJvcmlnaW4iOiJodHRwczovL2dvb2dsZS5jb206NDQzIiwiZmVhdHVyZSI6IlRydXN0VG9rZW5zIiwiZXhwaXJ5IjoxNjUyNzc0NDAwLCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="AzoawhTRDevLR66Y6MROu167EDncFPBvcKOaQispTo9ouEt5LvcBjnRFqiAByRT+2cDHG1Yj4dXwpLeIhc98/gIAAACFeyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A6+nc62kbJgC46ypOwRsNW6RkDn2x7tgRh0wp7jb3DtFF7oEhu1hhm4rdZHZ6zXvnKZLlYcBlQUImC4d3kKihAcAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A/9La288e7MDEU2ifusFnMg1C2Ij6uoa/Z/ylwJIXSsWfK37oESIPbxbt4IU86OGqDEPnNVruUiMjfKo65H/CQwAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_2022042101.js" async=""></script><style></style></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'central_science';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/central_science/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/central_science/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
					console.log("ad display: " + event.creativeId + " " + event.size[0] + "x" + event.size[1]);
				}
				else {
				    console.log("no ad found");
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>

<style>
.articleHeaderDropzone5 .advertisement { display: none; }
.article_header .advertisement {
  margin: -25px auto 25px auto;
}
</style>




        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&amp;EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Indiana University Libraries - Bloomington</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"></a>




        
        <div class="pj-badge"></div><a href="/journal/acscii" title="ACS Central Science" class="header_article-logo"><img src="/cms/10.1021/acscii/asset/title.png" alt="ACS Central Science"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="/doi/10.1021/acscentsci.8b00551" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span><span class="hlFld-Title">Effective Distance for DNA-Mediated Charge Transport between Repair Proteins</span></span>
        </div>
         <a href="/doi/10.1021/acscentsci.8b00568" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete ui-autocomplete-input"><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="acscii" name="SeriesKey" checked="checked" class="all-content"><span>ACS Cent. Sci.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"><span>All Publications/Website</span></label></div><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" style="display: none;"></ul></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"><select name="quickLinkJournal" class="quick-search_journals-select"><option value="acscii">ACS Central Science</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science &amp; Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials &amp; Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio &amp; Med Chem Au</option><option value="abseba">ACS Biomaterials Science &amp; Engineering</option><option value="accacs">ACS Catalysis</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health &amp; Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&amp;T Engineering</option><option value="aewcaa">ACS ES&amp;T Water</option><option value="afsthl">ACS Food Science &amp; Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic &amp; Inorganic Au</option><option value="aptsfn">ACS Pharmacology &amp; Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry &amp; Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&amp;EN Global Enterprise</option><option value="cenear">Chemical &amp; Engineering News Archive</option><option value="chlseg">Chemical Health &amp; Safety</option><option value="chlseg0">Chemical Health &amp; Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth &amp; Design</option><option value="enfuem">Energy &amp; Fuels</option><option value="esthag">Environmental Science &amp; Technology</option><option value="estlcu">Environmental Science &amp; Technology Letters</option><option value="iepra6.2">I&amp;EC Product Research and Development</option><option value="iechad">Industrial &amp; Engineering Chemistry</option><option value="iecac0">Industrial &amp; Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series</option><option value="iecfa7">Industrial &amp; Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial &amp; Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial &amp; Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial &amp; Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical &amp; Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health &amp; Safety</option><option value="jchsc20">Journal of Chemical Health &amp; Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial &amp; Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research &amp; Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&amp;D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autocomplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autocomplete="false" placeholder="Page" required="required" class="quick-search_page-input"><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden="”true”"></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/acscentsci.8b00773">A Selective and Cell-Permeable Mitochondrial Calcium Uniporter (MCU) Inhibitor Preserves Mitochondrial Bioenergetics after Hypoxia/Reoxygenation Injury</a></li><li><a href="/doi/full/10.1021/acscentsci.8b00822">An ATP-Regulated Ion Transport Nanosystem for Homeostatic Perturbation Therapy and Sensitizing Photodynamic Therapy by Autophagy Inhibition of Tumors</a></li><li><a href="/doi/full/10.1021/acscentsci.8b00844">Red Room-Temperature Phosphorescence of CDs@Zeolite Composites Triggered by Heteroatoms in Zeolite Frameworks</a></li></ul></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="”true”"></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="”true”"></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden="”true”"></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors &amp; Reviewers');">Authors &amp;
                        Reviewers</a></li>
                <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
				<li><a href="https://pubs.acs.org/page/virtual-collections.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Virtual Issues');">Virtual Issues</a></li>
                <li><a href="https://preferences.acs.org/journals" onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS &amp; Mobile');">RSS &amp; Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products &amp; Services');">Products &amp; Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos &amp; Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#" onclick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');" class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/who-we-serve/librarians/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians &amp; Account Managers');">For
                        Librarians &amp; Account Managers</a></li>
            </ul>
        </li>
        <li class="menu-pair-section hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a></li>
                
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage" style="display: block;">Login</a>
            <a class="header_burger-menu_logout-link" href="#" style="display: none;">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS &amp; Open Access');">ACS &amp; Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="/events" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i><div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/acscentsci.8b00773">A Selective and Cell-Permeable Mitochondrial Calcium Uniporter (MCU) Inhibitor Preserves Mitochondrial Bioenergetics after Hypoxia/Reoxygenation Injury</a></li><li><a href="/doi/full/10.1021/acscentsci.8b00822">An ATP-Regulated Ion Transport Nanosystem for Homeostatic Perturbation Therapy and Sensitizing Photodynamic Therapy by Autophagy Inhibition of Tumors</a></li><li><a href="/doi/full/10.1021/acscentsci.8b00844">Red Room-Temperature Phosphorescence of CDs@Zeolite Composites Triggered by Heteroatoms in Zeolite Frameworks</a></li></ul></div></div>


</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!-- Types--><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><!-- Subjects--><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!--div.pub-modal_right-cover-gallery.hidden-lg.hidden-md//a(href="/coverGallery") Browse Our Cover Art Gallery
--><!-- Publications list--><!-- 'pub-modal_right--inner' adds styling required for IE--><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><!-- The widget data--><input type="hidden" id="ajaxUrl" name="ajaxUrl" value="/pb/widgets/achs-publication-list?widgetId=cabcb15c-6f7a-494b-a19c-eba4fafaee5c&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facscentsci.8b00566%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3BrequestedJournal%3Ajournal%3Aacscii%3Bwebsite%3Awebsite%3Aacspubs%3Bjournal%3Ajournal%3Aacscii%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Facscii.2019.5.issue-1&amp;key=c7ca46a97ce077568c7371697bcd489c0&amp;ajaxRequest=true"><input type="hidden" name="pubModalData" value=""></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article" style="min-height: 754px;"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facscentsci.8b00566&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facscentsci.8b00566&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right" style="display: inline;"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left" style="display: inline;"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acscentsci.8b00566&amp;href=/doi/10.1021/acscentsci.8b00566" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>ACS Cent. Sci.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 5</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 65-72</span></span></span></div></div></div></div></div><input type="hidden" name="acscii" id="copyRights" value="/pb-assets/json/journal-info-1641590078917.json"><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/acscii/5/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acscentsci.8b00551" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Research Article</span><a href="/doi/10.1021/acscentsci.8b00568" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/acscii" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/cms/10.1021/acscii/asset/title.png" alt="Journal Logo"></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="acscii" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info" href="#"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/acscii?ifr=1"></div></div></div><h1 class="article_header-title"><span class="hlFld-Title">Effective Distance for DNA-Mediated Charge Transport between Repair Proteins</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Edmund C. M. Tse</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Edmund C. M. Tse</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Chemistry and Chemical Engineering, California Institute of Technology, Pasadena, California 91125, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Edmund%0AC.+M.++Tse">Edmund C. M. Tse</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0002-9313-1290" title="Orcid link">http://orcid.org/0000-0002-9313-1290</a></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Theodore J. Zwang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Theodore J. Zwang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Chemistry and Chemical Engineering, California Institute of Technology, Pasadena, California 91125, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Theodore+J.++Zwang">Theodore J. Zwang</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Sebastian Bedoya</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sebastian Bedoya</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Chemistry and Chemical Engineering, California Institute of Technology, Pasadena, California 91125, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sebastian++Bedoya">Sebastian Bedoya</a></span></div></div></span></li><span class="comma-separator">,&nbsp;and&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Jacqueline K. Barton</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jacqueline K. Barton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Chemistry and Chemical Engineering, California Institute of Technology, Pasadena, California 91125, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="mailto:jkbarton@caltech.edu">jkbarton@caltech.edu</a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jacqueline+K.++Barton">Jacqueline K. Barton</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-9883-1600" title="Orcid link">http://orcid.org/0000-0001-9883-1600</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="acscii"><input type="hidden" name="journalNameForjhpLink" value="ACS Central Science"><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facscentsci.8b00566&amp;href=/doi/10.1021%2Facscentsci.8b00566" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>ACS Cent. Sci.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 5</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 65–72</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 11, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>14 August 2018</li><li><span class="item_label"><b>Published</b> online</span>11 January 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 23 January 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acscentsci.8b00566" title="DOI URL">https://doi.org/10.1021/acscentsci.8b00566</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS &amp; PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice"><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acscentsci.8b00566"><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2753</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">19</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" class="doi" value="10.1021/acscentsci.8b00566"><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="add-full-document achs-mendeley_button"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Effective Distance for DNA-Mediated Charge Transport between Repair Proteins&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Edmund\nC.&quot;,&quot;last_name&quot;:&quot;M. Tse&quot;},{&quot;first_name&quot;:&quot;Theodore&quot;,&quot;last_name&quot;:&quot;J. Zwang&quot;},{&quot;first_name&quot;:&quot;Sebastian&quot;,&quot;last_name&quot;:&quot;Bedoya&quot;},{&quot;first_name&quot;:&quot;Jacqueline&quot;,&quot;last_name&quot;:&quot;K. Barton&quot;}],&quot;source&quot;:&quot;ACS Central Science&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;11&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;5&quot;,&quot;pages&quot;:&quot;65-72&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acscentsci.8b00566&quot;},&quot;abstract&quot;:&quot;The stacked aromatic base pairs within the DNA double helix facilitate charge transport down its length in the absence of lesions, mismatches, and other stacking perturbations. DNA repair proteins containing [4Fe4S] clusters can take advantage of DNA charge transport (CT) chemistry to scan the genome for mistakes more efficiently. Here we examine the effective length over which charge can be transported along DNA between these repair proteins. We define the effective CT distance as the length of DNA within which two proteins are able to influence their ensemble affinity to the DNA duplex via CT. Endonuclease III, a DNA repair glycosylase containing a [4Fe4S] cluster, was incubated with DNA duplexes of different lengths (1.5–9 kb), and atomic force microscopy was used to quantify the binding of proteins to these duplexes to determine how the relative protein affinity changes with increasing DNA length. A sharp change in binding slope is observed at 3509 base pairs, or about 1.2 μm, that supports the existence &quot;}" class="meta-data"><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"><input type="hidden" name="pub-access" value="free" class="pub-access"><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"><input type="hidden" name="selected-share-option" class="selected-share-option"></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facscentsci.8b00566&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.8b00566" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facscentsci.8b00566&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.8b00566" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facscentsci.8b00566&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.8b00566" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facscentsci.8b00566&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.8b00566&amp;href=/doi/10.1021/acscentsci.8b00566" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acscentsci.8b00566"><a href="/doi/pdf/10.1021/acscentsci.8b00566" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acscentsci.8b00566&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fid%3Ddoi%3A10.1021%252Facscentsci.8b00566%26sid%3Dliteratum%253Aachs%26pmid%3D30693326%26genre%3Darticle%26aulast%3DTse%26date%3D2019%26atitle%3DEffective%2BDistance%2Bfor%2BDNA-Mediated%2BCharge%2BTransport%2Bbetween%2BRepair%2BProteins%26jtitle%3DACS%2BCentral%2BScience%26title%3DACS%2BCentral%2BScience%26volume%3D5%26issue%3D1%26spage%3D65%26epage%3D72%26issn%3D2374-7943" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="acscii" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info" href="#"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/acscii?ifr=1"></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollduration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy" style="padding-left: 523.167px;"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291967" title="Cluster chemistry">Cluster chemistry</a>,</li><li><a href="/action/doSearch?ConceptID=291719" title="Redox reactions">Redox reactions</a>,</li><li><a href="/action/doSearch?ConceptID=291568" title="Genomics">Genomics</a>,</li><li><a href="/action/doSearch?ConceptID=290806" title="Charge transport">Charge transport</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="external-alerts-management" style="margin-left: -59px;"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="acscii" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info" href="#"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/acscii?ifr=1"></div><div class="cover-image"><div class="cover-image__image"><a href="/toc/acscii/5/1" title="Go to ACS Central Science "><img src="/cms/10.1021/acscii.2019.5.issue-1/asset/acscii.2019.5.issue-1.largecover.jpg" alt="Go to ACS Central Science "></a></div></div>




        
        <!--Article Page E-Alerts-->
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','acscii');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','acscii');
	  }
  });
</script>
<!--End Article Script-->




        
        <div class="pj-badge"><input type="hidden" id="partnerJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json"></div>




        
        <div class="aJhp_link"></div>
<input type="hidden" id="aJhpJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json">
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=acscii" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0007.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The stacked aromatic base pairs within the DNA double helix facilitate charge transport down its length in the absence of lesions, mismatches, and other stacking perturbations. DNA repair proteins containing [4Fe4S] clusters can take advantage of DNA charge transport (CT) chemistry to scan the genome for mistakes more efficiently. Here we examine the effective length over which charge can be transported along DNA between these repair proteins. We define the effective CT distance as the length of DNA within which two proteins are able to influence their ensemble affinity to the DNA duplex via CT. Endonuclease III, a DNA repair glycosylase containing a [4Fe4S] cluster, was incubated with DNA duplexes of different lengths (1.5–9 kb), and atomic force microscopy was used to quantify the binding of proteins to these duplexes to determine how the relative protein affinity changes with increasing DNA length. A sharp change in binding slope is observed at 3509 base pairs, or about 1.2 μm, that supports the existence of two regimes for protein binding, one within the range for DNA CT, one outside of the range for CT; DNA CT between the redox proteins bound to DNA effectively decreases the ensemble binding affinity of oxidized and reduced proteins to DNA. Utilizing an Endonuclease III mutant Y82A, which is defective in carrying out DNA CT, shows only one regime for protein binding. Decreasing the temperature to 4 °C or including metallointercalators on the duplex, both of which should enhance base stacking and decrease DNA floppiness, leads to extending the effective length for DNA charge transport to ∼5300 bp or 1.8 μm. These results thus support DNA charge transport between repair proteins over kilobase distances. The results furthermore highlight the ability of DNA repair proteins to search the genome quickly and efficiently&nbsp;using DNA charge transport chemistry.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left ui-resizable" style="width: 730px;"><div class="hlFld-Abstract synopsis"><h2>Synopsis</h2><span><p class="articleBody_abstractText">The length over which DNA-mediated charge transport between [4Fe4S] repair proteins is effective is found to be ∼3500 base pairs, providing a route for a rapid genome search for DNA lesions.</p></span></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08557" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08557" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i13" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#_i14" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i19" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i22" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i23" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div class="NLM_p">Cellular oxidative stress, external UV irradiation, and environmental mutagens cause genomic lesions and base pair modifications in living organisms on the order of tens per cell per second. <a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Various repair mechanisms are present in biological systems to search and correct DNA lesions and mismatches, thereby upholding genomic integrity. <a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> It is important that this repair be carried out quickly, before the damage can influence downstream processes or lead to mutations. <a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> In humans, accumulation of mutations caused by defective DNA repair can lead to the proliferation of tumor cells and development of cancer. <a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a></div><div class="NLM_p">Recently, many DNA repair proteins have been found to possess redox-active [4Fe4S] clusters that may expedite their DNA damage search. <a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> Experiments using DNA-modified electrodes show that these proteins all share a broad DNA-bound potential of ∼80 mV versus NHE, well below the potential at which DNA bases are oxidized. <a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> Although these [4Fe4S] cluster repair proteins are typically present in cells in relatively low copy numbers, <a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> they must find and repair thousands of mutagenic lesions within the 20 min doubling time of <i>Escherichia coli</i>. <a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> Translocation is insufficient to explain the fast action of these proteins, especially when accounting for limitations of diffusion in the congested cellular environment. <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><div class="NLM_p">DNA-processing proteins containing [4Fe4S] clusters are found in archaea, bacteria, and eukaryotes. <a onclick="showRef(event, 'ref18 ref19 ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21 ref22 ref23">(18−23)</a> Primases and polymerases containing [4Fe4S] clusters function as redox switches that control DNA replication processes. <a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> Base excision repair proteins such as EndoIII, MutY, and&nbsp;uracil DNA glycosylase (UDG), as well as SF2 helicases&nbsp;XPD, and DinG  contain redox-active [4Fe4S] clusters. <a onclick="showRef(event, 'ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33">(26−33)</a> These [4Fe4S] clusters may play a vital functional role in the DNA damage search and repair scheme, though there is still much to be understood about how they function in biological systems. <a onclick="showRef(event, 'ref34 ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref34 ref35 ref36 ref37">(34−37)</a></div><div class="NLM_p">Recent experiments with DNA-modified Au electrodes and microscale thermophoresis (MST) suggest that the [4Fe4S] cluster acts as a redox-modulated affinity switch for DNA. <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Using MST to quantify the binding affinity of DNA to EndoIII, EndoIII with an oxidized [4Fe4S]<sup>3+</sup> cluster was found to bind DNA 550-fold stronger than that with a reduced [4Fe4S]<sup>2+</sup> cluster. <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Electrochemical experiments also demonstrated that binding to the anionic phosphate backbone of DNA shifts the redox potential of [4Fe4S] clusters in DNA repair proteins negative by about 200 mV. <a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> This shift in redox potential corresponds to a change in the DNA-binding affinity that is similar to the values determined using MST. <a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The effect of binding to the polyanionic phosphate backbone of DNA contributes substantially to the shift in redox potential of the [4Fe4S] cluster in DNA repair proteins. <a onclick="showRef(event, 'ref38 ref40'); return false;" href="javascript:void(0);" class="ref ref38 ref40">(38,40)</a> Previous experiments have demonstrated that changing buffer salts result in only small differences in redox potential. <a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Atomic force microscopy (AFM) experiments show that these proteins reduce and oxidize one another through a DNA-mediated redox signaling process, effectively allowing for [4Fe4S] cluster proteins to change one another’s affinity to DNA. <a onclick="showRef(event, 'ref11 ref38'); return false;" href="javascript:void(0);" class="ref ref11 ref38">(11,38)</a></div><div class="NLM_p">We have proposed that these proteins containing [4Fe4S] clusters utilize DNA charge transport (CT) as a means more rapidly and efficiently to search the genome for lesions. Using ultrafast spectroscopy and electrochemical experiments, DNA CT occurs rapidly through well stacked DNA in both the excited and ground states but is interrupted by lesions, mismatches, and other perturbations to the duplex base pair stacking. <a onclick="showRef(event, 'ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref42 ref43 ref44">(42−44)</a> Redox-active [4Fe4S] cluster repair proteins can utilize the disruption of DNA CT by lesions to screen the genome for DNA damage. <a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> Inside cells, oxidative stress may oxidize the [4Fe4S] cluster of the proteins, or oxidize DNA bases that can then undergo CT with the protein. <a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> When the proteins bind to the DNA polyanion, they are activated toward oxidation. Hence, they are more prone to bind to DNA and lose an electron in the process. <a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> If there is no DNA damage, the electron can travel down the length of DNA and reduce the oxidized [4Fe4S]<sup>3+</sup> cluster of another repair protein, thereby decreasing its affinity and releasing it from DNA. <a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> This released protein can then continue its search elsewhere. <a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> This method, as a first step in the search process, allows for proteins to find DNA damage much faster than by translocation alone. <a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Incorporating DNA-mediated charge transport into the search, and assuming an effective DNA CT length of 200–500 bp between proteins, the time needed for [4Fe4S] cluster repair proteins to search for DNA damage sites shortens by at least an order of magnitude. By utilizing DNA CT in the DNA damage search process, the time needed for [4Fe4S] repair proteins to scan the entire genome is well within the time constraints of other biological processes. <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> One key advantage of a DNA CT supported search scheme is that electrons can travel through DNA across a heavily congested cellular environment in a relatively unhindered fashion on a nanosecond time scale, <a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> as opposed to a search scheme utilizing only translocation by proteins that takes much longer to cover the same distance, and can be made even slower than 0.1–10 μm<sup>2</sup>/s in <i>E. coli</i> because the crowded cellular environment impedes movement. <a onclick="showRef(event, 'ref16 ref51 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref16 ref51 ref52 ref53">(16,51−53)</a></div><div class="NLM_p">However, what is the distance over which DNA CT can proceed? Previously we have utilized a 100 bp dsDNA-modified electrode to demonstrate that DNA CT can occur over at least a distance of 34 nm. <a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Practical synthetic limitations have, however, precluded the examination of longer distances on electrodes, including those larger than the DNA duplex persistence length of 50 nm (150 bp). <a onclick="showRef(event, 'ref55 ref56 ref57 ref58'); return false;" href="javascript:void(0);" class="ref ref55 ref56 ref57 ref58">(55−58)</a> In addition, extrapolation from experiments with shorter DNA on DNA-modified electrodes has been inadequate for predicting a maximum length because the rate of electron transfer is generally limited by the attachment to the electrode. <a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Moreover, photophysical experiments varying the length of DNA have shown an extremely shallow distance dependence for DNA CT, while the same synthetic limitations apply. <a onclick="showRef(event, 'ref54 ref59'); return false;" href="javascript:void(0);" class="ref ref54 ref59">(54,59)</a></div><div class="NLM_p">AFM experiments allow for the interrogation under equilibrium conditions of DNA CT distance over 100 times longer than that used on DNA-modified electrodes, but cannot elucidate the kinetics of this process. <a onclick="showRef(event, 'ref11 ref38'); return false;" href="javascript:void(0);" class="ref ref11 ref38">(11,38)</a> After proteins and DNA were allowed to interact, AFM is used to count the number of proteins bound on DNA. <a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The AFM experimental procedure does not change the population of proteins bound on DNA. <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Previous AFM experiments demonstrate that DNA-mediated CT can occur between different proteins containing [4Fe4S] clusters. <a onclick="showRef(event, 'ref16 ref47 ref48 ref50 ref60'); return false;" href="javascript:void(0);" class="ref ref16 ref47 ref48 ref50 ref60">(16,47,48,50,60)</a> Using our AFM assay to probe long-range signaling between proteins has, moreover, suggested that DNA CT might proceed over kilobase lengths. <a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Experimental data from these experiments suggest that DNA CT lowers the effective binding affinity of any pair of reduced and oxidized [4Fe4S] proteins relative to the affinity of a protein with oxidized [4Fe4S]<sup>3+</sup> that is not involved in DNA CT. <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Thus, if new proteins bind to DNA with reduced [4Fe4S]<sup>2+</sup> clusters and are capable of DNA CT with a bound protein containing an oxidized [4Fe4S]<sup>3+</sup>, the effective affinity of these proteins will be decreased relative to the oxidized [4Fe4S]<sup>3+</sup> protein alone.</div><div class="NLM_p">If a maximum DNA CT length exists, we expect to detect two regimes in AFM experiments with different observable affinities for DNA–protein interactions: shorter DNA where all the proteins that bind are capable of DNA CT with another protein, and longer DNA where proteins can bind outside of maximum DNA CT distance of other proteins (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). In the shorter regime all of the proteins have decreased affinities relative to a lone oxidized [4Fe4S]<sup>3+</sup> protein, but in the longer regime many of the oxidized proteins will not have their affinity decreased. As the DNA length increases beyond the maximum DNA CT length, the probability that two or more [4Fe4S] cluster proteins bind to DNA far enough away from one another to be outside of the range for DNA CT should increase, and the proportion of unperturbed [4Fe4S]<sup>3+</sup> binding should increase with it. With this in mind, we conducted experiments using DNA of different lengths and measured the binding of [4Fe4S] proteins to see if we could observe differences in the number of bound proteins per base pair and thereby determine the maximum effective length of DNA CT between two proteins. Within the context of our proposed model, the effective distance is defined as the length of DNA within which two proteins are able to influence their ensemble affinity to the DNA duplex via CT. Notably, the effective CT length is not determined by directly measuring the length of DNA in between the proteins given our inability to distinguish proteins in different redox states. We instead monitor the ensemble distribution of proteins on DNA after CT has occurred.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0001.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0001.jpeg" id="gr1" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Quantifying the DNA charge transport length. (a) Model showing the implications of DNA CT length on the redox signaling capability between [4Fe4S] repair proteins and subsequently the lesion search process. (b) AFM images of two linear dsDNA. The one on the top is in the regime where the DNA length is shorter than the CT length, while the one on the bottom is in the regime where the DNA length is longer than the CT length.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64362" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64362" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i13" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#_i14" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i19" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i22" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i23" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> DNA CT Length of WT EndoIII at Room Temperature (RT)</h3><div class="NLM_p">The number of proteins bound to DNA was counted using images collected using AFM techniques. The number of proteins bound to different lengths of DNA was used to determine the affinity of those proteins to DNA. AFM images were collected using double-stranded DNA of 1625, 1999, 2686, 3895, 5153, 5967, 7037, 7996, and 8922 bp in length prepared using PCR and plasmid excision methods (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The AFM results are summarized in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a> and <a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">Figure S1</a>. Two regimes are observed, designated by black and blue lines. The blue line, with a shallow slope, represents the regime where the DNA length is shorter than the CT length. Here the [4Fe4S] cluster proteins are within CT distance for all binding configurations. Increases in DNA length in this regime effectively add a lower affinity binding site, giving it a relatively shallow slope. The steeper black line represents the regime where DNA length is longer than the CT length. Here some [4Fe4S] cluster proteins are not within CT distance to communicate with each other. As the DNA CT length increases, the probability of [4Fe4S] cluster proteins not within the CT distance with each other increases, giving them a higher effective affinity and resulting in a steeper slope. Although the average number of bound proteins is less than one, the presence of DNA duplexes bound with 2 and 3 proteins shows that the binding of proteins to DNA of these lengths is influenced by DNA CT, and our data indicate that the prevalence of DNA with 2 and 3 proteins is suppressed by DNA CT. These data suggest that proteins in the 2+ and 3+ oxidation states within CT distance have a lower ensemble binding affinity than proteins that are isolated from each other. The point where the two regimes converge is the DNA length where it starts to become possible that proteins bind without experiencing DNA CT; we consider this the value for the maximum DNA CT length between proteins. At ambient temperatures, WT EndoIII exhibits a DNA CT length of 3509 ± 509 bp, or 1.192 ± 0.174 μm (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a> and <a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">Figure S1</a>), significantly longer than shown by previous measurements.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0006.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0006.jpeg" id="gr2" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Constructing Linear Double-Stranded DNA of Various Lengths by (a) Plasmid Excision Using SacI (Magenta) and (b) PCR Amplification Methods<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"></p><p class="last"><span class="fn-label"><sup>a</sup></span>The detailed protocols to prepare DNA duplexes of different lengths are presented in the <a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">SI</a>, Methods section. See <a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">Table S1</a> for the primer sequences and plasmid templates used in the PCR amplification steps.</p><p></p></figure><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0002.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0002.jpeg" id="gr3" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Measuring DNA-binding density of wild-type EndoIII at ambient temperature using atomic force microscopy. A sharp change in binding slope is observed at 3509 base pairs, or about 1.2 μm, that suggests the presence of two regimes for protein binding, one within the range for DNA CT (regime 1, blue), one outside of the range for CT (regime 2, black). 1310 DNA duplexes were analyzed from four independent preparations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> DNA CT Length of Y82A EndoIII Mutant at RT</h3><div class="NLM_p">To determine whether the observed slope change is in fact related to DNA CT, we performed control experiments with the Y82A EndoIII mutant that was previously characterized to be DNA CT-deficient with its enzymatic activity unperturbed. <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The AFM results are summarized in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> with additional information in <a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">Figure S2</a>. Only one regime is observed with no apparent change of slope as a function of DNA lengths. Since Y82A does not exhibit DNA CT but remains otherwise the same as WT EndoIII, <a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> the lack of the slope change in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> is consistent with the slope change observed for WT EndoIII in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a> being a result of DNA CT. As Y82A does not undergo redox signaling with each other via DNA CT, the DNA-binding affinity of Y82A is independent of each other. Y82A does not exhibit DNA CT but remains otherwise the same as WT EndoIII. <a onclick="showRef(event, 'ref12 ref61'); return false;" href="javascript:void(0);" class="ref ref12 ref61">(12,61)</a> Thus, these data in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> show that DNA CT deficiency is sufficient to eliminate the two-sloped feature observed in data collected with the wild-type protein in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>, which suggests that the change in slope in the wild-type protein is a result of DNA CT.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0003.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0003.jpeg" id="gr4" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Measuring DNA-binding density of Y82A EndoIII mutant at ambient temperature using atomic force microscopy. No change in binding slope as a function of DNA lengths is observed, indicating the absence of DNA CT. As Y82A does not undergo redox signaling with each other via DNA CT, the DNA-binding affinity of Y82A is independent of each other. 272 DNA duplexes were analyzed from four independent preparations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> DNA CT Length of WT EndoIII at 4 °C</h3><div class="NLM_p">To examine if the DNA CT length is temperature dependent, we conducted AFM experiments with samples prepared at colder temperatures. The AFM results are presented in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>. At 4 °C, WT EndoIII exhibits a much longer DNA CT length (5383 ± 174 bp, or 1.830 ± 0.059 μm) than that measured at RT. This increase in DNA CT length is expected, as a decrease in temperature limits molecular motion; i.e., DNA will be less floppy. <a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a></div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0004.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0004.jpeg" id="gr5" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Measuring DNA-binding density of wild-type EndoIII at 4 °C using atomic force microscopy. A sharp change in binding slope occurs at 5383 base pairs, or about 1.8 μm, indicating that the DNA CT length observed at 4 °C is significantly longer than that at RT. 151 DNA duplexes were analyzed from four independent preparations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> DNA CT Length of WT EndoIII at RT in the Presence of a Metallointercalator</h3><div class="NLM_p">To determine if the DNA CT length can be altered using molecular means, we conducted AFM experiments in the presence of [Ru(phen)<sub>2</sub>dppz]Cl<sub>2</sub> (50 μM), which is a known metallointercalator. <a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> The redox potential of the [4Fe4S] clusters in EndoIII is outside of the potential range at which the Ru intercalators are reduced or oxidized. <a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> The AFM results are displayed in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> with additional information in <a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">Figure S3</a>. The AFM experiments demonstrate that in the presence of 50 μM [Ru(phen)<sub>2</sub>dppz]Cl<sub>2</sub> WT EndoIII exhibits a DNA CT length of 5399 ± 1732 bp, or 1.836 ± 0.589 μm (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> and <a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">Figure S3</a>). This increase in DNA CT length is consistent with the increase in DNA stability upon intercalation of metal complexes. <a onclick="showRef(event, 'ref63 ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref63 ref65 ref66">(63,65,66)</a> Additionally, as one would expect, the results support the presence of π-stacked metallointercalators not impeding DNA CT.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0005.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0005.jpeg" id="gr6" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Measuring DNA-binding density of wild-type EndoIII in the presence of [Ru(phen)<sub>2</sub>dppz]Cl<sub>2</sub> (50 μM) at ambient temperature using atomic force microscopy. A sharp change in binding slope occurs at 5399 base pairs, or about 1.8 μm, suggesting that metallointercalators lengthen the DNA CT length observed. 449 DNA duplexes were analyzed from four independent preparations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83600" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83600" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i13" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#_i14" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i19" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i22" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i23" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Determining DNA CT Length Using AFM Experiments</h3><div class="NLM_p last">This study aims to measure the effective maximum distance for DNA-mediated charge transport between proteins by monitoring changes in the affinity of [4Fe4S] proteins to DNA with increasing duplex length. An obvious change in slope occurs for EndoIII that is capable of DNA CT, but not for its CT-deficient mutant Y82A. Although the average number of proteins bound is less than one in some cases, these data show binding of the second and higher number of proteins is significantly decreased beyond what would be expected if the second (and higher) protein bound independently of the first, which shows that this aggregate value is taking into account the influence of DNA CT on binding affinity. We can therefore consider the change in slope a feature of proteins capable of DNA CT, which indicates that an increase in affinity with DNA length is shallow at shorter DNA distances and steep at longer DNA distances. Additionally, it is possible to modify the point at which this slope change occurs by changing the temperature and including DNA intercalators, both of which can change the stability of the duplex and the efficiency of DNA CT.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Understanding DNA CT Length Using an Equilibrium Binding Model</h3><div class="NLM_p">The data collected using AFM represent a snapshot of the proteins bound to DNA at equilibrium. Fitting a plot of the number of proteins per duplex from AFM data shows an intersection of two different regimes, whereby increasing the length of DNA differently affects the increase in affinity of proteins for the additional binding sites. We interpret this change in slope as indicative of a maximum CT length at ambient temperature of 3509 ± 509 bp, or 1.192 ± 0.174 μm, which is not observed in CT-deficient proteins.</div><div class="NLM_p">To further understand these data, an equilibrium model was developed to help describe the DNA-binding behavior of proteins that are capable of DNA CT (see the <a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">SI</a>, Methods). The first protein that binds to DNA is insensitive to DNA CT, and the affinity we observe is dependent on the oxidation state of the protein [4Fe4S] cluster. <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> The dissociation constant for EndoIII with a reduced [4Fe4S]<sup>2+</sup> cluster was previously determined by MST to be 1.3 × 10<sup>–4</sup> per bp, and EndoIII with an oxidized [4Fe4S]<sup>3+</sup> cluster has an affinity of 2.3 × 10<sup>–7</sup> per bp. <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> These values were used as inputs of our model to estimate the percent of oxidized [4Fe4S]<sup>3+</sup> clusters in our sample, 5.6 ± 1.7% (see the <a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">SI</a>, Methods). Modeling the binding of additional proteins to incorporate contributions of DNA CT allows us then to determine the effective dissociation constant for proteins undergoing CT, which is 4.3 × 10<sup>–5</sup> ± 1 × 10<sup>–5</sup> per bp, a decrease in affinity of over 2 orders of magnitude compared to the oxidized protein that results from electron transfer with a nearby reduced protein.</div><div class="NLM_p">Importantly, using these values for the percent of oxidized [4Fe4S] clusters and the effective affinity of proteins undergoing CT, we may simulate the data collected by AFM (<a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">Figure S4</a>). Results from our theoretical calculations show the same trends as the collected data, including the two regimes above and below the effective CT distance with different slopes. The intersection of these two regimes in the simulated data provides a maximum CT length of 2990 bp, which agrees well with the experimental data and further strengthens the claim that the presence of two regimes in our experimental data is due to the effect of DNA CT on the DNA-binding affinity of redox-active proteins.</div><div class="NLM_p last">Other experimental conditions yield similar values for the percent of oxidized protein and decrease in affinity due to charge transport but show dramatic differences in the length of CT (<a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">Table S3</a>). Fitting similar data collected from a CT-deficient Y82A EndoIII mutant does not show any measurable CT distance, which is also supported by simulated data using the same model (<a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">Figure S4</a>). Decreasing the temperature at which proteins are incubated with DNA to 4 °C increases the measured CT length to 5383 ± 174 bp. As temperature decreases, the persistence length of the DNA backbone increases. <a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> The incorporation of the metallointercalator [Ru(phen)<sub>2</sub>dppz]Cl<sub>2</sub>, which is known to stabilize DNA duplexes, <a onclick="showRef(event, 'ref63 ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref63 ref65 ref66">(63,65,66)</a> increases the CT length to 5399 ± 1732 bp. These results suggest that as the rigidity of DNA increases, the CT length also increases, likely due to stabilizing the DNA duplex in a more CT-favorable state.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Biological Implications of DNA CT Length</h3><div class="NLM_p">Knowing the length that DNA CT can occur between proteins allows for predictions to be made about the effectiveness of DNA CT in increasing the efficiency of DNA damage repair. In our initial prediction, by assuming a DNA CT length of 200–500 bp, the time needed to search the <i>E. coli</i> genome decreases by an order of magnitude over calculations excluding DNA CT. <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The current AFM study indicates a DNA CT length of 3500 bp, so that the efficiency of the genome lesion search can significantly increase further or require fewer involved proteins. Incorporating the data from this AFM study will be useful for better modeling of the DNA damage search and repair process and the role that such long-distance CT may play.</div><div class="NLM_p last">Now we consider how DNA CT plays a role in biological systems using <i>E. coli</i> strain as an example. <i>E. coli</i> K-12 strain has a genome size of 4 639 221 bp. <a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> Dividing the number of total bp by the measured DNA CT length of 3509 bp per protein gives a rough calculation that 1322 [4Fe4S] repair proteins are needed to place the whole <i>E. coli</i> genome within DNA CT range of these [4Fe4S] repair proteins. Although individually these [4Fe4S] repair proteins may be present in cells in low copy numbers, previous studies have demonstrated that [4Fe4S] cluster proteins from different repair pathways can signal with each other <i>in vivo</i>. <a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Summing up the copy numbers of EndoIII, MutY, and DinG gives an estimate of the total number of known [4Fe4S]-containing DNA repair proteins in <i>E. coli</i> of at least about 1100. <a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> We have measured the redox potential of multiple proteins, and find that they are all close to one another, which suggests that they can all influence one another’s binding affinity through DNA CT. However, it is not clear how other differences within the cell may affect the search and repair process. With more [4Fe4S] proteins yet to be described, together these relevant [4Fe4S] repair proteins may participate in and enable a genome-wide search for DNA lesions via DNA CT.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53576" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53576" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i13" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#_i14" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i19" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i22" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i23" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div class="NLM_p last">In this report, we utilized atomic force microscopy and an equilibrium binding model to study the effective&nbsp;DNA charge transport length between oxidized and reduced&nbsp;Endonuclease III, a DNA repair protein carrying a redox-active [4Fe4S] cluster. AFM data reveal two regimes of DNA-binding behavior. In one regime, where the DNA length is shorter than the CT length, a copy of EndoIII in the reduced [4Fe4S]<sup>2+</sup> state bound on DNA modulates the DNA-binding affinity of a second copy of EndoIII with an oxidized [4Fe4S]<sup>3+</sup> cluster, no matter where the second copy is bound on DNA. Because the two EndoIII proteins are always within CT distance, their effective affinity for DNA is averaged. In another regime where the DNA length is longer than the CT length, there now exist binding sites where proteins with oxidized [4Fe4S]<sup>3+</sup> clusters can bind outside of CT range with a higher effective affinity. As the DNA length increases, the probability of additional EndoIII binding outside of CT distance with higher effective affinity increases. Modeling allowed us to extract the maximum effective DNA CT length between proteins, which we found to be 3509 base pairs, or about 1.2 μm at ambient temperature. A mechanistic understanding of this long-range DNA CT now requires exploration by theorists. <a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> The DNA CT length can be changed with temperature and by stabilizing DNA with metallointercalators. These data suggest that the effective DNA CT length can be controlled by its environment, which has interesting implications for both biological systems and technology that use DNA CT. An effective&nbsp;DNA CT length&nbsp;this&nbsp;long enables a fast genome lesion search scheme via long-range redox signaling.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81421" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81421" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i13" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#_i14" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i19" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i22" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i23" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> General Procedures</h3><div class="NLM_p last">Chemicals were obtained from commercial sources (Sigma-Aldrich, Fisher Scientific, VWR, and New England Biolabs) and used without further purification unless otherwise specified. DNA primer sequences were purchased from Integrated DNA Technologies, purified by high performance liquid chromatography (HPLC, HP 1100, Agilent), characterized using matrix-assisted laser desorption ionization (MALDI) mass spectrometry using an Autoflex MALDI TOF/TOF instrument (Bruker), and quantified using a 100 Bio UV–vis spectrophotometer (Cary, Agilent) as described previously. <a onclick="showRef(event, 'ref69 ref70 ref71 ref72 ref73'); return false;" href="javascript:void(0);" class="ref ref69 ref70 ref71 ref72 ref73">(69−73)</a> Phosphate buffer (pH 7.0, 5 mM NaH<sub>2</sub>PO<sub>4</sub>, 50 mM NaCl) was prepared using Milli-Q water (&gt;18 MΩ cm). Experiments performed were replicated at least three times using different samples, and data presented are from representative trials. The <a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">SI</a>, Methods section, contains protocols for overexpression and purification of wild-type EndoIII and Y82A mutant and synthesis of DNA duplexes of various lengths (<a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">Table S1</a>).</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> AFM Experiments</h3><div class="NLM_p last">AFM was conducted following protocols reported previously. <a onclick="showRef(event, 'ref11 ref38 ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref11 ref38 ref60 ref61">(11,38,60,61)</a> Briefly, mica surfaces were freshly cleaved with tape. Protein stock solution (100 nM) contained either WT EndoIII or Y82A mutant in phosphate buffer (pH 7.0, 5 mM NaH<sub>2</sub>PO<sub>4</sub>, 50 mM NaCl). Stock DNA solution (9 μM bp) contained the mixture of DNA duplexes of various lengths in Tris elution buffer (EB, 10 mM Tris-HCl, pH 8.5, Qiagen). A solution (23.5 μL) with a final protein concentration (12 nM) and a final total DNA concentration (5 μM bp) was prepared and incubated at ambient temperature for 1 h or 4 °C for 2 h to allow for the loading of protein onto DNA to reach equilibrium while minimizing cluster degradation. For experiments with metallointercalators, a final concentration of 50 μM [Ru(phen)<sub>2</sub>dppz]Cl<sub>2</sub> was added. MgCl<sub>2</sub> (200 mM, 1.5 μL per 25 μL total volume) was added to promote DNA adsorption on mica for AFM experiments. After pipetting 12.5 μL of DNA/protein/MgCl<sub>2</sub> solution onto a freshly cleaved mica surface and incubating for 2 min, a continuous stream of Milli-Q water (2 mL) was slowly poured over the top portion of the modified mica surface while holding the piece of mica in a vertical position to linearize the DNA. A piece of lint-free wipe was used to dab dry the bottom edge of the mica surface. The surface was dried using a stream of N<sub>2</sub> flowing in the same direction as the water rinse for 2 min. No significant safety hazards were encountered.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> AFM Instrumentation</h3><div class="NLM_p last">FESPA-V2 AFM tapping mode probes (Bruker Nano, Inc.) with a mean force constant of 2.8 N/m and a mean resonance frequency of 75 kHz were used in a MFP-3D AFM instrument (Asylum Research). Images were captured in air with scan areas of 5 × 5 μm<sup>2</sup> in tapping mode at a scan rate of 1 Hz to obtain images of quality high enough for AFM assay analysis (512 pixels/line, 512 lines/image). Over 1000 AFM images were collected, and over 2000 DNA strands were analyzed blind as described previously. <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> AFM Image Analysis</h3><div class="NLM_p last">WSxM software (Igor Pro) was used to analyze DNA contour lengths and height profiles of the proteins as described previously. <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> DNA and proteins were identified using the relative differential height profiles between protein and DNA. For each data set, images from at least three independent samples were analyzed. Distinguishable DNA and proteins were counted by hand. Duplexes that were overlapped, indistinguishable, or cut off by the edge of the image were excluded from the counting procedure. Binding density is defined as the number of the proteins bound on a DNA duplex. Data presented are from representative trials, and error bars represent standard error of all trials based on the total number of proteins observed (<a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">Table S2</a>, <i>n</i> &gt; 200 for all experiments).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51751" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51751" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i13" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#_i14" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i19" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i22" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i23" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acscentsci.8b00566" class="ext-link">10.1021/acscentsci.8b00566</a>.</p><ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supplemental methods, data, and figures including schematics, protein counts, box plots, UV–vis spectra, and AFM image (<a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf">oc8b00566_si_001.pdf (1.02 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acscentsci.8b00566" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget" style="width: 876.3px; display: block;"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms &amp; Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10121" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10121" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i13" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#_i14" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i19" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i22" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i23" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacqueline K. Barton</span> - <span class="hlFld-Affiliation affiliation">Division of Chemistry and
Chemical Engineering, California Institute
of Technology, Pasadena, California 91125, United States</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-9883-1600" title="Orcid link">http://orcid.org/0000-0001-9883-1600</a>;&nbsp;
    </span><span class="author-email">
                Email: <a href="mailto:jkbarton@caltech.edu">jkbarton@caltech.edu</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Edmund
C. M. Tse</span> - <span class="hlFld-Affiliation affiliation">Division of Chemistry and
Chemical Engineering, California Institute
of Technology, Pasadena, California 91125, United States</span>;&nbsp;
    <span>Present Address:
                        E.C.M.T.: Department of Chemistry, The University of Hong Kong, Pokfulam Road, Hong Kong SAR</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0002-9313-1290" title="Orcid link">http://orcid.org/0000-0002-9313-1290</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Theodore J. Zwang</span> - <span class="hlFld-Affiliation affiliation">Division of Chemistry and
Chemical Engineering, California Institute
of Technology, Pasadena, California 91125, United States</span>;&nbsp;
    <span>Present Address:
                        T.J.Z.: Department of Chemistry and Chemical Biology, Harvard University, Cambridge, MA 02138, USA</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sebastian Bedoya</span> - <span class="hlFld-Affiliation affiliation">Division of Chemistry and
Chemical Engineering, California Institute
of Technology, Pasadena, California 91125, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13808" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13808" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i13" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#_i14" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i19" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i22" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i23" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><p class="last">We gratefully recognize the NIH (GM126904) for financial support. E.C.M.T. appreciates the Croucher Foundation for a postdoctoral fellowship. T.J.Z. is an NSF fellow (DGE-1144469). S.B. acknowledges Joseph L. Koo and Helen C. Koo for a student undergraduate research fellowship. We thank Dr. Adam N. Boynton and Kelsey M. Boyle for providing metallointercalators for this study. We are also grateful to the Caltech Center for the Chemistry of Cellular Signaling for instrumentation. This research was enabled from the use of the Autoflex MALDI TOF in the Caltech CCE Multiuser Mass Spectrometry Laboratory, acquired with funds from the DOW corporation (CCEC.DOWINSTR). AFM experiments were carried out at the Molecular Materials Research Center of the Beckman Institute of the California Institute of Technology.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60673" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60673" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results"><span>Results</span></a></li><li><a href="#_i9" class="internalNav" title="Discussion"><span>Discussion</span></a></li><li><a href="#_i13" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#_i14" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i19" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i22" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i23" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr></div><p class="references-count">
            This article references 73 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartek, J.</span></span> <span> </span><span class="NLM_article-title">The DNA-damage Response in Human Biology and Disease</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>461</i></span> (<span class="NLM_issue">7267</span>),  <span class="NLM_fpage">1071</span>– <span class="NLM_lpage">1078</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature08467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnature08467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=19847258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGktLrK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=461&amp;publication_year=2009&amp;pages=1071-1078&amp;issue=7267&amp;author=S.+P.+Jacksonauthor=J.+Bartek&amp;title=The+DNA-damage+Response+in+Human+Biology+and+Disease&amp;doi=10.1038%2Fnature08467" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The DNA-damage response in human biology and disease</span></div><div class="casAuthors">Jackson, Stephen P.; Bartek, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7267</span>),
    <span class="NLM_cas:pages">1071-1078</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The prime objective for every life form is to deliver its genetic material, intact and unchanged, to the next generation.  This must be achieved despite const. assaults by endogenous and environmental agents on the DNA.  To counter this threat, life has evolved several systems to detect DNA damage, signal its presence and mediate its repair.  Such responses, which have an impact on a wide range of cellular events, are biol. significant because they prevent diverse human diseases.  Our improving understanding of DNA-damage responses is providing new avenues for disease management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ0Oyp6nX9V7Vg90H21EOLACvtfcHk0lhUb8VPrJm2Iw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGktLrK&amp;md5=cca7cfec66852124695ebeb515f255a5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature08467&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08467%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DS.%2BP.%26aulast%3DBartek%26aufirst%3DJ.%26atitle%3DThe%2520DNA-damage%2520Response%2520in%2520Human%2520Biology%2520and%2520Disease%26jtitle%3DNature%26date%3D2009%26volume%3D461%26issue%3D7267%26spage%3D1071%26epage%3D1078%26doi%3D10.1038%2Fnature08467" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, D. E.</span></span> <span> </span><span class="NLM_article-title">Repair of Endogenous DNA Damage</span>. <i>Cold Spring Harbor Symp. Quant. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">134</span>, <span class="refDoi">&nbsp;DOI: 10.1101/sqb.2000.65.127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1101%2Fsqb.2000.65.127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=12760027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlt1ejtr4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=65&amp;publication_year=2000&amp;pages=127-134&amp;author=T.+Lindahlauthor=D.+E.+Barnes&amp;title=Repair+of+Endogenous+DNA+Damage&amp;doi=10.1101%2Fsqb.2000.65.127" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Repair of endogenous DNA damage</span></div><div class="casAuthors">Lindahl, T.; Barnes, D. E.</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Symposia on Quantitative Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">127-133</span>CODEN:
                <span class="NLM_cas:coden">CSHSAZ</span>;
        ISSN:<span class="NLM_cas:issn">0091-7451</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review with refs. on how to correct an endogenous DNA damage in mammalian cells by the base excision repair (BER) pathways.  The different DNA glycosylases that are known to act on various forms of endogenously damaged DNA in mammalian cells are described.  The sequential transient protein-protein interactions occurring on the damaged DNA substrate during BER may promote the accuracy, speed, and specificity of the endogenous repair process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_nhlrmTM-i7Vg90H21EOLACvtfcHk0lhUb8VPrJm2Iw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3MXlt1ejtr4%253D&amp;md5=064b78d90329afaf07394c18ea589200</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1101%2Fsqb.2000.65.127&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fsqb.2000.65.127%26sid%3Dliteratum%253Aachs%26aulast%3DLindahl%26aufirst%3DT.%26aulast%3DBarnes%26aufirst%3DD.%2BE.%26atitle%3DRepair%2520of%2520Endogenous%2520DNA%2520Damage%26jtitle%3DCold%2520Spring%2520Harbor%2520Symp.%2520Quant.%2520Biol.%26date%3D2000%26volume%3D65%26spage%3D127%26epage%3D134%26doi%3D10.1101%2Fsqb.2000.65.127" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoeijmakers, J. H. J.</span></span> <span> </span><span class="NLM_article-title">Genome Maintenance Mechanisms for Preventing Cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>411</i></span> (<span class="NLM_issue">6835</span>),  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">374</span>, <span class="refDoi">&nbsp;DOI: 10.1038/35077232</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2F35077232" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=411&amp;publication_year=2001&amp;pages=366-374&amp;issue=6835&amp;author=J.+H.+J.+Hoeijmakers&amp;title=Genome+Maintenance+Mechanisms+for+Preventing+Cancer&amp;doi=10.1038%2F35077232" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2F35077232&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35077232%26sid%3Dliteratum%253Aachs%26aulast%3DHoeijmakers%26aufirst%3DJ.%2BH.%2BJ.%26atitle%3DGenome%2520Maintenance%2520Mechanisms%2520for%2520Preventing%2520Cancer%26jtitle%3DNature%26date%3D2001%26volume%3D411%26issue%3D6835%26spage%3D366%26epage%3D374%26doi%3D10.1038%2F35077232" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sancar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsey-Boltz, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unsal-Kacmaz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linn, S.</span></span> <span> </span><span class="NLM_article-title">Molecular Mechanisms of Mammalian DNA Repair and the DNA Damage Checkpoints</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">85</span>, <span class="refDoi">&nbsp;DOI: 10.1146/annurev.biochem.73.011303.073723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1146%2Fannurev.biochem.73.011303.073723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=15189136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmslagsrk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=73&amp;publication_year=2004&amp;pages=39-85&amp;author=A.+Sancarauthor=L.+A.+Lindsey-Boltzauthor=K.+Unsal-Kacmazauthor=S.+Linn&amp;title=Molecular+Mechanisms+of+Mammalian+DNA+Repair+and+the+DNA+Damage+Checkpoints&amp;doi=10.1146%2Fannurev.biochem.73.011303.073723" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms of mammalian DNA repair and the DNA damage checkpoints</span></div><div class="casAuthors">Sancar, Aziz; Lindsey-Boltz, Laura A.; Uensal-Kacmaz, Keziban; Linn, Stuart</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39-85</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  DNA damage is a relatively common event in the life of a cell and may lead to mutation, cancer, and cellular or organismic death.  Damage to DNA induces several cellular responses that enable the cell either to eliminate or cope with the damage or to activate a programmed cell death process, presumably to eliminate cells with potentially catastrophic mutations.  These DNA damage response reactions include: (a) removal of DNA damage and restoration of the continuity of the DNA duplex; (b) activation of a DNA damage checkpoint, which arrests cell cycle progression so as to allow for repair and prevention of the transmission of damaged or incompletely replicated chromosomes: (c) transcriptional response, which causes changes in the transcription profile that may be beneficial to the cell; and (d) apoptosis, which eliminates heavily damaged or seriously deregulated cells.  DNA repair mechanisms include direct repair, base excision repair, nucleotide excision repair, double-strand break repair, and cross-link repair.  The DNA damage checkpoints employ damage sensor proteins, such as ATM, ATR, the Rad17-RFC complex, and the 9-1-1 complex, to detect DNA damage and to initiate signal transduction cascades that employ Chk1 and Chk2 Ser/Thr kinases and Cdc25 phosphatases.  The signal transducers activate p53 and inactivate cyclin-dependent kinases to inhibit cell cycle progression from G1 to S (the G1/S checkpoint), DNA replication (the intra-S checkpoint), or G2 to mitosis (the G2/M checkpoint).  In this review the mol. mechanisms of DNA repair and the DNA damage checkpoints in mammalian cells are analyzed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_p5BqBCbTf7Vg90H21EOLACvtfcHk0lh0ZPEAEefPKw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2cXmslagsrk%253D&amp;md5=63f88da9f19d721af81defffba832615</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.73.011303.073723&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.73.011303.073723%26sid%3Dliteratum%253Aachs%26aulast%3DSancar%26aufirst%3DA.%26aulast%3DLindsey-Boltz%26aufirst%3DL.%2BA.%26aulast%3DUnsal-Kacmaz%26aufirst%3DK.%26aulast%3DLinn%26aufirst%3DS.%26atitle%3DMolecular%2520Mechanisms%2520of%2520Mammalian%2520DNA%2520Repair%2520and%2520the%2520DNA%2520Damage%2520Checkpoints%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2004%26volume%3D73%26spage%3D39%26epage%3D85%26doi%3D10.1146%2Fannurev.biochem.73.011303.073723" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, S.</span></span> <span> </span><span class="NLM_article-title">Base-excision Repair of Oxidative DNA Damage</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>447</i></span> (<span class="NLM_issue">7147</span>),  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">950</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature05978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnature05978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=17581577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD2sXms12js74%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=447&amp;publication_year=2007&amp;pages=941-950&amp;issue=7147&amp;author=S.+S.+Davidauthor=V.+L.+O%E2%80%99Sheaauthor=S.+Kundu&amp;title=Base-excision+Repair+of+Oxidative+DNA+Damage&amp;doi=10.1038%2Fnature05978" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Base-excision repair of oxidative DNA damage</span></div><div class="casAuthors">David, Sheila S.; O'Shea, Valerie L.; Kundu, Sucharita</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">447</span>
        (<span class="NLM_cas:issue">7147</span>),
    <span class="NLM_cas:pages">941-950</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Maintaining the chem. integrity of DNA in the face of assault by oxidizing agents is a const. challenge for living organisms.  Base-excision repair has an important role in preventing mutations assocd. with a common product of oxidative damage to DNA, 8-oxoguanine.  Recent structural studies have shown that 8-oxoguanine DNA glycosylases use an intricate series of steps to locate and excise 8-oxoguanine lesions efficiently against a high background of undamaged bases.  The importance of preventing mutations assocd. with 8-oxoguanine is shown by a direct assocn. between defects in the DNA glycosylase MUTYH and colorectal cancer.  The properties of other guanine oxidn. products and the assocd. DNA glycosylases that remove them are now also being revealed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot6-SWDCNHJLVg90H21EOLACvtfcHk0lh0ZPEAEefPKw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXms12js74%253D&amp;md5=8e58553790fbb8888f3ab9906c567026</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature05978&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05978%26sid%3Dliteratum%253Aachs%26aulast%3DDavid%26aufirst%3DS.%2BS.%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DV.%2BL.%26aulast%3DKundu%26aufirst%3DS.%26atitle%3DBase-excision%2520Repair%2520of%2520Oxidative%2520DNA%2520Damage%26jtitle%3DNature%26date%3D2007%26volume%3D447%26issue%3D7147%26spage%3D941%26epage%3D950%26doi%3D10.1038%2Fnature05978" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marteijn, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lans, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeijmakers, J. H. J.</span></span> <span> </span><span class="NLM_article-title">Understanding Nucleotide Excision Repair and its Roles in Cancer and Ageing</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">481</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nrm3822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnrm3822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=24954209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVaisL7I" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=2014&amp;pages=465-481&amp;issue=7&amp;author=J.+A.+Marteijnauthor=H.+Lansauthor=W.+Vermeulenauthor=J.+H.+J.+Hoeijmakers&amp;title=Understanding+Nucleotide+Excision+Repair+and+its+Roles+in+Cancer+and+Ageing&amp;doi=10.1038%2Fnrm3822" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding nucleotide excision repair and its roles in cancer and ageing</span></div><div class="casAuthors">Marteijn, Jurgen A.; Lans, Hannes; Vermeulen, Wim; Hoeijmakers, Jan H. J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-481</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Nucleotide excision repair (NER) eliminates various structurally unrelated DNA lesions by a multiwise 'cut and patch'-type reaction.  The global genome NER (GG-NER) subpathway prevents mutagenesis by probing the genome for helix-distorting lesions, whereas transcription-coupled NER (TC-NER) removes transcription-blocking lesions to permit unperturbed gene expression, thereby preventing cell death.  Consequently, defects in GG-NER result in cancer predisposition, whereas defects in TC-NER cause a variety of diseases ranging from UV radiation-sensitive syndrome to severe premature ageing conditions such as Cockayne syndrome.  Recent studies have uncovered new aspects of DNA-damage detection by NER, how NER is regulated by extensive post-translational modifications, and the dynamic chromatin interactions that control its efficiency.  Based on these findings, a mechanistic model is proposed that explains the complex genotype-phenotype correlations of transcription-coupled repair disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD-ZP89iPqubVg90H21EOLACvtfcHk0lh0ZPEAEefPKw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXhtVaisL7I&amp;md5=c48a0203cf22aeaf7d885737988598d9</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrm3822&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3822%26sid%3Dliteratum%253Aachs%26aulast%3DMarteijn%26aufirst%3DJ.%2BA.%26aulast%3DLans%26aufirst%3DH.%26aulast%3DVermeulen%26aufirst%3DW.%26aulast%3DHoeijmakers%26aufirst%3DJ.%2BH.%2BJ.%26atitle%3DUnderstanding%2520Nucleotide%2520Excision%2520Repair%2520and%2520its%2520Roles%2520in%2520Cancer%2520and%2520Ageing%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2014%26volume%3D15%26issue%3D7%26spage%3D465%26epage%3D481%26doi%3D10.1038%2Fnrm3822" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tubbs, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussenzweig, A.</span></span> <span> </span><span class="NLM_article-title">Endogenous DNA Damage as a Source of Genomic Instability in Cancer</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>168</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">644</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2017.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.cell.2017.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=28187286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1ygt7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=168&amp;publication_year=2017&amp;pages=644&amp;issue=4&amp;author=A.+Tubbsauthor=A.+Nussenzweig&amp;title=Endogenous+DNA+Damage+as+a+Source+of+Genomic+Instability+in+Cancer&amp;doi=10.1016%2Fj.cell.2017.01.002" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Endogenous DNA damage as a source of genomic instability in cancer</span></div><div class="casAuthors">Tubbs, Anthony; Nussenzweig, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">644-656</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Genome instability, defined as higher than normal rates of mutation, is a double-edged sword.  As a source of genetic diversity and natural selection, mutations are beneficial for evolution.  On the other hand, genomic instability can have catastrophic consequences for age-related diseases such as cancer.  Mutations arise either from inactivation of DNA repair pathways or in a repair-competent background due to genotoxic stress from cellular processes such as transcription and replication that overwhelm high-fidelity DNA repair.  Here, we review recent studies that shed light on endogenous sources of mutation and epigenomic features that promote genomic instability during cancer evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfnOzjrfwD8rVg90H21EOLACvtfcHk0liZRKdlkIx9oA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXis1ygt7Y%253D&amp;md5=88a3d335ab90c98819538b0ab849b602</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.01.002&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DTubbs%26aufirst%3DA.%26aulast%3DNussenzweig%26aufirst%3DA.%26atitle%3DEndogenous%2520DNA%2520Damage%2520as%2520a%2520Source%2520of%2520Genomic%2520Instability%2520in%2520Cancer%26jtitle%3DCell%26date%3D2017%26volume%3D168%26issue%3D4%26spage%3D644%26doi%3D10.1016%2Fj.cell.2017.01.002" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knijnenburg, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambwe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherniack, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Way, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, C. S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Genomic and Molecular Landscape of DNA Damage Repair Deficiency across The Cancer Genome Atlas</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">239</span>– <span class="NLM_lpage">254</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.celrep.2018.03.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.celrep.2018.03.076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=29617664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntVKqt74%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2018&amp;pages=239-254&amp;issue=1&amp;author=T.+A.+Knijnenburgauthor=L.+Wangauthor=M.+T.+Zimmermannauthor=N.+Chambweauthor=G.+F.+Gaoauthor=A.+D.+Cherniackauthor=H.+Fanauthor=H.+Shenauthor=G.+P.+Wayauthor=C.+S.+Greene&amp;title=Genomic+and+Molecular+Landscape+of+DNA+Damage+Repair+Deficiency+across+The+Cancer+Genome+Atlas&amp;doi=10.1016%2Fj.celrep.2018.03.076" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic and Molecular Landscape of DNA Damage Repair Deficiency across The Cancer Genome Atlas</span></div><div class="casAuthors">Knijnenburg, Theo A.; Wang, Linghua; Zimmermann, Michael T.; Chambwe, Nyasha; Gao, Galen F.; Cherniack, Andrew D.; Fan, Huihui; Shen, Hui; Way, Gregory P.; Greene, Casey S.; Liu, Yuexin; Akbani, Rehan; Feng, Bin; Donehower, Lawrence A.; Miller, Chase; Shen, Yang; Karimi, Mostafa; Chen, Haoran; Kim, Pora; Jia, Peilin; Shinbrot, Eve; Zhang, Shaojun; Liu, Jianfang; Hu, Hai; Bailey, Matthew H.; Yau, Christina; Wolf, Denise; Zhao, Zhongming; Weinstein, John N.; Li, Lei; Ding, Li; Mills, Gordon B.; Laird, Peter W.; Wheeler, David A.; Shmulevich, Ilya; Monnat, Raymond J., Jr.; Xiao, Yonghong; Wang, Chen</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">239-254.e6</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">DNA damage repair (DDR) pathways modulate cancer risk, progression, and therapeutic response.  We systematically analyzed somatic alterations to provide a comprehensive view of DDR deficiency across 33 cancer types.  Mutations with accompanying loss of heterozygosity were obsd. in over 1/3 of DDR genes, including TP53 and BRCA1/2.  Other prevalent alterations included epigenetic silencing of the direct repair genes EXO5, MGMT, and ALKBH3 in ∼20% of samples.  Homologous recombination deficiency (HRD) was present at varying frequency in many cancer types, most notably ovarian cancer.  However, in contrast to ovarian cancer, HRD was assocd. with worse outcomes in several other cancers.  Protein structure-based analyses allowed us to predict functional consequences of rare, recurrent DDR mutations.  A new machine-learning-based classifier developed from gene expression data allowed us to identify alterations that phenocopy deleterious TP53 mutations.  These frequent DDR gene alterations in many human cancers have functional consequences that may det. cancer progression and guide therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3qjeMN9_Q-bVg90H21EOLACvtfcHk0liZRKdlkIx9oA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXntVKqt74%253D&amp;md5=8eac929850bdbd897b51fa7026e931eb</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2018.03.076&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2018.03.076%26sid%3Dliteratum%253Aachs%26aulast%3DKnijnenburg%26aufirst%3DT.%2BA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZimmermann%26aufirst%3DM.%2BT.%26aulast%3DChambwe%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DG.%2BF.%26aulast%3DCherniack%26aufirst%3DA.%2BD.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DWay%26aufirst%3DG.%2BP.%26aulast%3DGreene%26aufirst%3DC.%2BS.%26atitle%3DGenomic%2520and%2520Molecular%2520Landscape%2520of%2520DNA%2520Damage%2520Repair%2520Deficiency%2520across%2520The%2520Cancer%2520Genome%2520Atlas%26jtitle%3DCell%2520Rep.%26date%3D2018%26volume%3D23%26issue%3D1%26spage%3D239%26epage%3D254%26doi%3D10.1016%2Fj.celrep.2018.03.076" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuss, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishida, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span> <span> </span><span class="NLM_article-title">Emerging critical roles of Fe–S clusters in DNA replication and repair</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1853</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1253</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bbamcr.2015.01.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.bbamcr.2015.01.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=25655665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitl2ks74%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1853&amp;publication_year=2015&amp;pages=1253&amp;issue=6&amp;author=J.+O.+Fussauthor=C.-L.+Tsaiauthor=J.+P.+Ishidaauthor=J.+A.+Tainer&amp;title=Emerging+critical+roles+of+Fe%E2%80%93S+clusters+in+DNA+replication+and+repair&amp;doi=10.1016%2Fj.bbamcr.2015.01.018" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging critical roles of Fe-S clusters in DNA replication and repair</span></div><div class="casAuthors">Fuss, Jill O.; Tsai, Chi-Lin; Ishida, Justin P.; Tainer, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1853</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1253-1271</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Fe-S clusters are partners in the origin of life that predate cells, acetyl-CoA metab., DNA, and the RNA world.  The double helix solved the mystery of DNA replication by base pairing for accurate copying.  Yet, for genome stability necessary to life, the double helix has equally important implications for damage repair.  Here we examine striking advances that uncover Fe-S cluster roles both in copying the genetic sequence by DNA polymerases and in crucial repair processes for genome maintenance, as mutational defects cause cancer and degenerative disease.  Moreover, we examine an exciting, controversial role for Fe-S clusters in a third element required for life - the long-range coordination and regulation of replication and repair events.  By their ability to delocalize electrons over both Fe and S centers, Fe-S clusters have unbeatable features for protein conformational control and charge transfer via double-stranded DNA that may fundamentally transform our understanding of life, replication, and repair.  This article is part of a Special Issue entitled: Fe/S proteins: Anal., structure, function, biogenesis and diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg8eONB8437bVg90H21EOLACvtfcHk0lhx9aba32zCpQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXitl2ks74%253D&amp;md5=500b7f2678b074fcd396989977a517b7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2015.01.018&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2015.01.018%26sid%3Dliteratum%253Aachs%26aulast%3DFuss%26aufirst%3DJ.%2BO.%26aulast%3DTsai%26aufirst%3DC.-L.%26aulast%3DIshida%26aufirst%3DJ.%2BP.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26atitle%3DEmerging%2520critical%2520roles%2520of%2520Fe%25E2%2580%2593S%2520clusters%2520in%2520DNA%2520replication%2520and%2520repair%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2015%26volume%3D1853%26issue%3D6%26spage%3D1253%26doi%3D10.1016%2Fj.bbamcr.2015.01.018" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouault, T. A.</span></span> <span> </span><span class="NLM_article-title">Iron-sulfur Proteins Hiding in Plain Sight</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">442</span>– <span class="NLM_lpage">445</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.1843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnchembio.1843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=26083061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1ygsbrE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2015&amp;pages=442-445&amp;issue=7&amp;author=T.+A.+Rouault&amp;title=Iron-sulfur+Proteins+Hiding+in+Plain+Sight&amp;doi=10.1038%2Fnchembio.1843" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Iron-sulfur proteins hiding in plain sight</span></div><div class="casAuthors">Rouault, Tracey A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">442-445</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent studies suggest that iron-sulfur (Fe-S) proteins may be unexpectedly abundant and functionally diverse in mammalian cells, but their identification still remains difficult.  The use of informatics along with traditional spectroscopic analyses could be key to discovering new Fe-S proteins and validating their functional roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpordnJZVNsibVg90H21EOLACvtfcHk0lhx9aba32zCpQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXht1ygsbrE&amp;md5=ce0bdf2205b3b3d285ef365efa5433d7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1843&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1843%26sid%3Dliteratum%253Aachs%26aulast%3DRouault%26aufirst%3DT.%2BA.%26atitle%3DIron-sulfur%2520Proteins%2520Hiding%2520in%2520Plain%2520Sight%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D7%26spage%3D442%26epage%3D445%26doi%3D10.1038%2Fnchembio.1843" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grodick, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segal, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA-Mediated Signaling by Proteins with 4Fe-4S Clusters is Necessary for Genomic Integrity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">6470</span>– <span class="NLM_lpage">6478</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja501973c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja501973c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlygtbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=136&amp;publication_year=2014&amp;pages=6470-6478&amp;issue=17&amp;author=M.+A.+Grodickauthor=H.+M.+Segalauthor=T.+J.+Zwangauthor=J.+K.+Barton&amp;title=DNA-Mediated+Signaling+by+Proteins+with+4Fe-4S+Clusters+is+Necessary+for+Genomic+Integrity&amp;doi=10.1021%2Fja501973c" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Mediated Signaling by Proteins with 4Fe-4S Clusters Is Necessary for Genomic Integrity</span></div><div class="casAuthors">Grodick, Michael A.; Segal, Helen M.; Zwang, Theodore J.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6470-6478</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Iron-sulfur clusters have increasingly been found to be assocd. with enzymes involved in DNA processing.  Here we describe a role for these redox clusters in DNA-mediated charge transport (CT) signaling in E. coli between DNA repair proteins from distinct pathways.  DNA-modified electrochem. shows that the 4Fe-4S cluster of DNA-bound DinG, an ATP-dependent helicase that repairs R-loops, is redox-active at cellular potentials and ATP hydrolysis increases DNA-mediated redox signaling.  Atomic force microscopy expts. demonstrate that DinG and Endonuclease III (EndoIII), a base excision repair enzyme, cooperate at long-range using DNA charge transport to redistribute to regions of DNA damage.  Genetics expts., moreover, reveal that this DNA-mediated signaling among proteins also occurs within the cell, and remarkably, is required for cellular viability under conditions of stress.  Silencing the gene encoding EndoIII in a strain of E. coli where repair by DinG is essential results in a significant growth defect that is rescued by complementation with EndoIII but not with an EndoIII mutant that is enzymically active but unable to carry out DNA charge transport.  This work thus elucidates a fundamental mechanism to coordinate the activities of DNA repair enzymes across the genome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfuCvre35b7bVg90H21EOLACvtfcHk0lhx9aba32zCpQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXmtlygtbw%253D&amp;md5=e72498bbc3efbcd815ef73f8065bde5e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fja501973c&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja501973c%26sid%3Dliteratum%253Aachs%26aulast%3DGrodick%26aufirst%3DM.%2BA.%26aulast%3DSegal%26aufirst%3DH.%2BM.%26aulast%3DZwang%26aufirst%3DT.%2BJ.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA-Mediated%2520Signaling%2520by%2520Proteins%2520with%25204Fe-4S%2520Clusters%2520is%2520Necessary%2520for%2520Genomic%2520Integrity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26issue%3D17%26spage%3D6470%26epage%3D6478%26doi%3D10.1021%2Fja501973c" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yavin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukianova, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA-Bound Redox Activity of DNA Repair Glycosylases Containing [4Fe-4S] Clusters</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">8397</span>– <span class="NLM_lpage">8407</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi047494n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi047494n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktF2ktL8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=44&amp;publication_year=2005&amp;pages=8397-8407&amp;issue=23&amp;author=A.+K.+Boalauthor=E.+Yavinauthor=O.+A.+Lukianovaauthor=V.+L.+O%E2%80%99Sheaauthor=S.+S.+Davidauthor=J.+K.+Barton&amp;title=DNA-Bound+Redox+Activity+of+DNA+Repair+Glycosylases+Containing+%5B4Fe-4S%5D+Clusters&amp;doi=10.1021%2Fbi047494n" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Bound Redox Activity of DNA Repair Glycosylases Containing [4Fe-4S] Clusters</span></div><div class="casAuthors">Boal, Amie K.; Yavin, Eylon; Lukianova, Olga A.; O'Shea, Valerie L.; David, Sheila S.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8397-8407</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MutY and endonuclease III (Endo III), two DNA glycosylases from Escherichia coli, and AfUDG, a uracil DNA glycosylase from Archeoglobus fulgidus, are all base excision repair enzymes that contain the [4Fe-4S]2+ cofactor.  Here we demonstrate that, when bound to DNA, these repair enzymes become redox-active; binding to DNA shifts the redox potential of the [4Fe-4S]3+/2+ couple to the range characteristic of high-potential iron proteins and activates the proteins toward oxidn.  Electrochem. on DNA-modified electrodes reveals potentials for Endo III and AfUDG of 58 and 95 mV vs. NHE, resp., comparable to 90 mV for MutY bound to DNA.  In the absence of DNA modification of the electrode, no redox activity can be detected, and on electrodes modified with DNA contg. an abasic site, the redox signals are dramatically attenuated; these observations show that the DNA base pair stack mediates electron transfer to the protein, and the potentials detd. are for the DNA-bound protein.  In EPR expts. at 10 K, redox activation upon DNA binding is also evident to yield the oxidized [4Fe-4S]3+ cluster and the partially degraded [3Fe-4S]1+ cluster.  EPR signals at g = 2.02 and 1.99 for MutY and g = 2.03 and 2.01 for Endo III are seen upon oxidn. of these proteins by Co(phen)33+ in the presence of DNA and are characteristic of [3Fe-4S]1+ clusters, while oxidn. of AfUDG bound to DNA yields EPR signals at g = 2.13, 2.04, and 2.02, indicative of both [4Fe-4S]3+ and [3Fe-4S]1+ clusters.  On the basis of this DNA-dependent redox activity, we propose a model for the rapid detection of DNA lesions using DNA-mediated electron transfer among these repair enzymes; redox activation upon DNA binding and charge transfer through well-matched DNA to an alternate bound repair protein can lead to the rapid redistribution of proteins onto genome sites in the vicinity of DNA lesions.  This redox activation furthermore establishes a functional role for the ubiquitous [4Fe-4S] clusters in DNA repair enzymes that involves redox chem. and provides a means to consider DNA-mediated signaling within the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTEk7wgWbSbLVg90H21EOLACvtfcHk0liNTwVpngh3rw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXktF2ktL8%253D&amp;md5=57dd0af2e4693ccf7f85964c0924ebc5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fbi047494n&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi047494n%26sid%3Dliteratum%253Aachs%26aulast%3DBoal%26aufirst%3DA.%2BK.%26aulast%3DYavin%26aufirst%3DE.%26aulast%3DLukianova%26aufirst%3DO.%2BA.%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DV.%2BL.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA-Bound%2520Redox%2520Activity%2520of%2520DNA%2520Repair%2520Glycosylases%2520Containing%2520%255B4Fe-4S%255D%2520Clusters%26jtitle%3DBiochemistry%26date%3D2005%26volume%3D44%26issue%3D23%26spage%3D8397%26epage%3D8407%26doi%3D10.1021%2Fbi047494n" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, S.
O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Electron Transfer Between Bases in Double Helical DNA</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>283</i></span> (<span class="NLM_issue">5400</span>),  <span class="NLM_fpage">375</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.283.5400.375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.283.5400.375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=9888851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK1MXns1eltQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=283&amp;publication_year=1999&amp;pages=375&amp;issue=5400&amp;author=S.%0AO.+Kelleyauthor=J.+K.+Barton&amp;title=Electron+Transfer+Between+Bases+in+Double+Helical+DNA&amp;doi=10.1126%2Fscience.283.5400.375" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Electron transfer between bases in double helical DNA</span></div><div class="casAuthors">Kelley, Shana O.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">5400</span>),
    <span class="NLM_cas:pages">375-381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Fluorescent analogs of adenine that selectively oxidize guanine were used to investigate photoinduced electron transfer through the DNA π-stack as a function of reactant stacking and energetics.  Small variations in these factors led to profound changes in the kinetics and distance dependences of DNA-mediated electron-transfer reactions.  Values of 3, a parameter reflecting the dependence of electron transfer on distance, ranged from 0.1 to 1.0 per angstrom.  Strong stacking interactions result in the fastest electron-transfer kinetics.  Electrons are thus transported preferentially through an intrastrand rather than interstrand pathway.  Reactant energetics also modulate the distance dependence of DNA-mediated charge transport.  These studies may resolve the range of disparate results previously reported, and paradigms must now be developed to describe these properties of the DNA π-stack, which can range from insulator- to "wire"-like.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRqzMiIoL8bVg90H21EOLACvtfcHk0liNTwVpngh3rw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1MXns1eltQ%253D%253D&amp;md5=aaf2498a7ea56d699a8440b38f39b8d8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1126%2Fscience.283.5400.375&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.283.5400.375%26sid%3Dliteratum%253Aachs%26aulast%3DKelley%26aufirst%3DS.%2BO.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DElectron%2520Transfer%2520Between%2520Bases%2520in%2520Double%2520Helical%2520DNA%26jtitle%3DScience%26date%3D1999%26volume%3D283%26issue%3D5400%26spage%3D375%26doi%3D10.1126%2Fscience.283.5400.375" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demple, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L.</span></span> <span> </span><span class="NLM_article-title">Repair of Oxidative Damage to DNA: Enzymology and Biology</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">915</span>– <span class="NLM_lpage">948</span>, <span class="refDoi">&nbsp;DOI: 10.1146/annurev.bi.63.070194.004411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1146%2Fannurev.bi.63.070194.004411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=7979257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK2MXitlCgtA%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=63&amp;publication_year=1994&amp;pages=915-948&amp;author=B.+Dempleauthor=L.+Harrison&amp;title=Repair+of+Oxidative+Damage+to+DNA%3A+Enzymology+and+Biology&amp;doi=10.1146%2Fannurev.bi.63.070194.004411" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Repair of oxidative damage to DNA: Enzymology and biology</span></div><div class="casAuthors">Demple, Bruce; Harrison, Lynn</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">915-48</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    </div><div class="casAbstract">A review, with 226 refs., on DNA glycosylases in the repair of oxidative damage, AP endonucleases and 3'-trimming in the repair of deoxyribose damage, and nucleotide excision and recombination in general repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPUyFucFNBs7Vg90H21EOLACvtfcHk0liNTwVpngh3rw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXitlCgtA%253D%253D&amp;md5=857c720aa8898b692f52b2c16e6d6480</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1146%2Fannurev.bi.63.070194.004411&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.bi.63.070194.004411%26sid%3Dliteratum%253Aachs%26aulast%3DDemple%26aufirst%3DB.%26aulast%3DHarrison%26aufirst%3DL.%26atitle%3DRepair%2520of%2520Oxidative%2520Damage%2520to%2520DNA%253A%2520Enzymology%2520and%2520Biology%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D1994%26volume%3D63%26spage%3D915%26epage%3D948%26doi%3D10.1146%2Fannurev.bi.63.070194.004411" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkhardt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span> <span> </span><span class="NLM_article-title">Quantifying Absolute Protein Synthesis Rates Reveals Principles Underlying Allocation of Cellular Resources</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>157</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">624</span>– <span class="NLM_lpage">635</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2014.02.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.cell.2014.02.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=24766808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFGitL8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=157&amp;publication_year=2014&amp;pages=624-635&amp;issue=3&amp;author=G.-W.+Liauthor=D.+Burkhardtauthor=C.+Grossauthor=J.+S.+Weissman&amp;title=Quantifying+Absolute+Protein+Synthesis+Rates+Reveals+Principles+Underlying+Allocation+of+Cellular+Resources&amp;doi=10.1016%2Fj.cell.2014.02.033" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying Absolute Protein Synthesis Rates Reveals Principles Underlying Allocation of Cellular Resources</span></div><div class="casAuthors">Li, Gene-Wei; Burkhardt, David; Gross, Carol; Weissman, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">624-635</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Quant. views of cellular functions require precise measures of rates of biomol. prodn., esp. proteins - the direct effectors of biol. processes.  Here, we present a genome-wide approach, based on ribosome profiling, for measuring abs. protein synthesis rates.  The resultant E. coli data set transforms our understanding of the extent to which protein synthesis is precisely controlled to optimize function and efficiency.  Members of multiprotein complexes are made in precise proportion to their stoichiometry, whereas components of functional modules are produced differentially according to their hierarchical role.  Ests. of abs. protein abundance also reveal principles for optimizing design.  These include how the level of different types of transcription factors is optimized for rapid response and how a metabolic pathway (methionine biosynthesis) balances prodn. cost with activity requirements.  Our studies reveal how general principles, important both for understanding natural systems and for synthesizing new ones, emerge from quant. analyses of protein synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLrDULHF1S0LVg90H21EOLACvtfcHk0lgJo_a2yVww5A"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXntFGitL8%253D&amp;md5=cc5a8290bb675fc516cf8fa446c4ed5b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.02.033&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.02.033%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.-W.%26aulast%3DBurkhardt%26aufirst%3DD.%26aulast%3DGross%26aufirst%3DC.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26atitle%3DQuantifying%2520Absolute%2520Protein%2520Synthesis%2520Rates%2520Reveals%2520Principles%2520Underlying%2520Allocation%2520of%2520Cellular%2520Resources%26jtitle%3DCell%26date%3D2014%26volume%3D157%26issue%3D3%26spage%3D624%26epage%3D635%26doi%3D10.1016%2Fj.cell.2014.02.033" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genereux, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sontz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gralnick, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Redox Signaling between DNA Repair Proteins for Efficient Lesion Detection</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">15237</span>– <span class="NLM_lpage">15242</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.0908059106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1073%2Fpnas.0908059106" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=106&amp;publication_year=2009&amp;pages=15237-15242&amp;issue=36&amp;author=A.+K.+Boalauthor=J.+C.+Genereuxauthor=P.+A.+Sontzauthor=J.+A.+Gralnickauthor=D.+K.+Newmanauthor=J.+K.+Barton&amp;title=Redox+Signaling+between+DNA+Repair+Proteins+for+Efficient+Lesion+Detection&amp;doi=10.1073%2Fpnas.0908059106" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0908059106&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0908059106%26sid%3Dliteratum%253Aachs%26aulast%3DBoal%26aufirst%3DA.%2BK.%26aulast%3DGenereux%26aufirst%3DJ.%2BC.%26aulast%3DSontz%26aufirst%3DP.%2BA.%26aulast%3DGralnick%26aufirst%3DJ.%2BA.%26aulast%3DNewman%26aufirst%3DD.%2BK.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DRedox%2520Signaling%2520between%2520DNA%2520Repair%2520Proteins%2520for%2520Efficient%2520Lesion%2520Detection%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26issue%3D36%26spage%3D15237%26epage%3D15242%26doi%3D10.1073%2Fpnas.0908059106" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popodi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, P. L.</span></span> <span> </span><span class="NLM_article-title">Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span> (<span class="NLM_issue">41</span>),  <span class="NLM_fpage">E2774</span>– <span class="NLM_lpage">E2783</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1210309109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1073%2Fpnas.1210309109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=22991466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKntbjM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=109&amp;publication_year=2012&amp;pages=E2774-E2783&amp;issue=41&amp;author=H.+Leeauthor=E.+Popodiauthor=H.+Tangauthor=P.+L.+Foster&amp;title=Rate+and+molecular+spectrum+of+spontaneous+mutations+in+the+bacterium+Escherichia+coli+as+determined+by+whole-genome+sequencing&amp;doi=10.1073%2Fpnas.1210309109" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing</span></div><div class="casAuthors">Lee, Heewook; Popodi, Ellen; Tang, Haixu; Foster, Patricia L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">E2774-E2783, SE2774/1-SE2774/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Knowledge of the rate and nature of spontaneous mutation is fundamental to understanding evolutionary and mol. processes.  In this report, we analyze spontaneous mutations accumulated over thousands of generations by wild-type Escherichia coli and a deriv. defective in mismatch repair (MMR), the primary pathway for correcting replication errors.  The major conclusions are (i) the mutation rate of a wild-type E. col strain is ∼1 × 10-3 per genome per generation; (ii) mutations in the wild-type strain have the expected mutational bias for G:C &gt; A:T mutations, but the bias changes to A:T &gt; G:C mutations in the absence of MMR; (iii) during replication, A:T &gt; G:C transitions preferentially occur with A templating the lagging strand and T templating the leading strand, whereas G:C &gt; A:T transitions preferentially occur with C templating the lagging strand and G templating the leading strand; (iv) there is a strong bias for transition mutations to occur at 5'ApC3'/3'TpG5' sites (where bases 5'A and 3'T are mutated) and, to a lesser extent, at 5'GpC3'/3'CpG5' sites (where bases 5'G and 3'C are mutated); (v) although the rate of small (≤4 nt) insertions and deletions is high at repeat sequences, these events occur at only 1/10th the genomic rate of base-pair substitutions.  MMR activity is genetically regulated, and bacteria isolated from nature often lack MMR capacity, suggesting that modulation of MMR can be adaptive.  Thus, comparing results from the wild-type and MMR-defective strains may lead to a deeper understanding of factors that det. mutation rates and spectra, how these factors may differ among organisms, and how they may be shaped by environmental conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtRFcPmZX-M7Vg90H21EOLACvtfcHk0lgJo_a2yVww5A"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhsFKntbjM&amp;md5=e44fb70553c7eadc11516ee3346dcd4d</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1210309109&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1210309109%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPopodi%26aufirst%3DE.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DFoster%26aufirst%3DP.%2BL.%26atitle%3DRate%2520and%2520molecular%2520spectrum%2520of%2520spontaneous%2520mutations%2520in%2520the%2520bacterium%2520Escherichia%2520coli%2520as%2520determined%2520by%2520whole-genome%2520sequencing%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26issue%3D41%26spage%3DE2774%26epage%3DE2783%26doi%3D10.1073%2Fpnas.1210309109" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agarwalla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroud, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, B. J.</span></span> <span> </span><span class="NLM_article-title">Redox Reactions of the Iron-Sulfur Cluster in a Ribosomal RNA Methyltransferase, RumA: OPTICAL AND EPR STUDIES</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span> (<span class="NLM_issue">33</span>),  <span class="NLM_fpage">34123</span>– <span class="NLM_lpage">34129</span>, <span class="refDoi">&nbsp;DOI: 10.1074/jbc.M405702200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1074%2Fjbc.M405702200" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=279&amp;publication_year=2004&amp;pages=34123-34129&amp;issue=33&amp;author=S.+Agarwallaauthor=R.+M.+Stroudauthor=B.+J.+Gaffney&amp;title=Redox+Reactions+of+the+Iron-Sulfur+Cluster+in+a+Ribosomal+RNA+Methyltransferase%2C+RumA%3A+OPTICAL+AND+EPR+STUDIES&amp;doi=10.1074%2Fjbc.M405702200" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M405702200&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M405702200%26sid%3Dliteratum%253Aachs%26aulast%3DAgarwalla%26aufirst%3DS.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DGaffney%26aufirst%3DB.%2BJ.%26atitle%3DRedox%2520Reactions%2520of%2520the%2520Iron-Sulfur%2520Cluster%2520in%2520a%2520Ribosomal%2520RNA%2520Methyltransferase%252C%2520RumA%253A%2520OPTICAL%2520AND%2520EPR%2520STUDIES%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26issue%3D33%26spage%3D34123%26epage%3D34129%26doi%3D10.1074%2Fjbc.M405702200" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Netz, D. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stith, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stümpfig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köpf, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genau, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stodola, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lill, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgers, P. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierik, A. J.</span></span> <span> </span><span class="NLM_article-title">Eukaryotic DNA polymerases require an iron-sulfur cluster for the formation of active complexes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">125</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnchembio.721" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFeitrbK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2012&amp;pages=125&amp;issue=1&amp;author=D.+J.+A.+Netzauthor=C.+M.+Stithauthor=M.+St%C3%BCmpfigauthor=G.+K%C3%B6pfauthor=D.+Vogelauthor=H.+M.+Genauauthor=J.+L.+Stodolaauthor=R.+Lillauthor=P.+M.+J.+Burgersauthor=A.+J.+Pierik&amp;title=Eukaryotic+DNA+polymerases+require+an+iron-sulfur+cluster+for+the+formation+of+active+complexes&amp;doi=10.1038%2Fnchembio.721" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Eukaryotic DNA polymerases require an iron-sulfur cluster for the formation of active complexes</span></div><div class="casAuthors">Netz, Daili J. A.; Stith, Carrie M.; Stuempfig, Martin; Koepf, Gabriele; Vogel, Daniel; Genau, Heide M.; Stodola, Joseph L.; Lill, Roland; Burgers, Peter M. J.; Pierik, Antonio J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The eukaryotic replicative DNA polymerases (Pol α, δ and ε) and the major DNA mutagenesis enzyme Pol ζ contain two conserved cysteine-rich metal-binding motifs (CysA and CysB) in the C-terminal domain (CTD) of their catalytic subunits.  Here we demonstrate by in vivo and in vitro approaches the presence of an essential [4Fe-4S] cluster in the CysB motif of all four yeast B-family DNA polymerases.  Loss of the [4Fe-4S] cofactor by cysteine ligand mutagenesis in Pol3 destabilized the CTD and abrogated interaction with the Pol31 and Pol32 subunits.  Reciprocally, overexpression of accessory subunits increased the amt. of the CTD-bound Fe-S cluster.  This implies an important physiol. role of the Fe-S cluster in polymerase complex stabilization.  Further, we demonstrate that the Zn-binding CysA motif is required for PCNA-mediated Pol δ processivity.  Together, our findings show that the function of eukaryotic replicative DNA polymerases crucially depends on different metallocenters for accessory subunit recruitment and replisome stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHCvlCJ7K7ALVg90H21EOLACvtfcHk0limeDl8DhSVMw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXhsFeitrbK&amp;md5=c0860ad52cbecb174470f8b9dda38696</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.721&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.721%26sid%3Dliteratum%253Aachs%26aulast%3DNetz%26aufirst%3DD.%2BJ.%2BA.%26aulast%3DStith%26aufirst%3DC.%2BM.%26aulast%3DSt%25C3%25BCmpfig%26aufirst%3DM.%26aulast%3DK%25C3%25B6pf%26aufirst%3DG.%26aulast%3DVogel%26aufirst%3DD.%26aulast%3DGenau%26aufirst%3DH.%2BM.%26aulast%3DStodola%26aufirst%3DJ.%2BL.%26aulast%3DLill%26aufirst%3DR.%26aulast%3DBurgers%26aufirst%3DP.%2BM.%2BJ.%26aulast%3DPierik%26aufirst%3DA.%2BJ.%26atitle%3DEukaryotic%2520DNA%2520polymerases%2520require%2520an%2520iron-sulfur%2520cluster%2520for%2520the%2520formation%2520of%2520active%2520complexes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26issue%3D1%26spage%3D125%26doi%3D10.1038%2Fnchembio.721" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asahara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bank, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholes, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salerno, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surerus, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munck, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCracken, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peisach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emptage, M. H.</span></span> <span> </span><span class="NLM_article-title">Endonuclease III is an Iron-sulfur Protein</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4450</span>– <span class="NLM_lpage">4455</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00436a049</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00436a049" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaL1MXitVyntr4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=28&amp;publication_year=1989&amp;pages=4450-4455&amp;issue=10&amp;author=R.+P.+Cunninghamauthor=H.+Asaharaauthor=J.+F.+Bankauthor=C.+P.+Scholesauthor=J.+C.+Salernoauthor=K.+Surerusauthor=E.+Munckauthor=J.+McCrackenauthor=J.+Peisachauthor=M.+H.+Emptage&amp;title=Endonuclease+III+is+an+Iron-sulfur+Protein&amp;doi=10.1021%2Fbi00436a049" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Endonuclease III is an iron-sulfur protein</span></div><div class="casAuthors">Cunningham, Richard P.; Asahara, Hitomi; Bank, Janet F.; Scholes, Charles P.; Salerno, John C.; Surerus, Kristene; Munck, Eckard; McCracken, John; Peisach, Jack; Emptage, Mark H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4450-5</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Elemental analyses, Moessbauer, and EPR data are reported to show that endonuclease III of Escherichia coli is an Fe-S protein.  Moessbauer spectra of protein freshly prepd. from E. coli grown on 57Fe-enriched medium demonstrate that the native enzyme contains a single 4Fe-4S cluster in the 2+ oxidn. state, with a net spin of zero.  Upon treatment with ferricyanide, a fraction (&lt;25%) of the clusters is oxidized into a state which yields an EPR spectrum near g = 2.01 typical of a 3Fe-4S cluster.  The magnetic field dependence of the linear elec. field effect verifies this assignment.  Electron spin echo modulation on the g = 2.01 form of the protein in deuterated solvent indicates the presence of exchangeable protons in the vicinity of the 3Fe-4S cluster.  The data obtained show that the [4Fe-4S]2+ cluster of the native enzyme is resistant to either oxidn. or redn., although photoredn. elicited a g = 1.94 type EPR signal characteristic of a [4Fe-4S]1+ cluster.  Evidently, endonuclease III is unique in being both a DNA repair enzyme and an Fe-S protein.  The function of the 4Fe-4S cluster remains to be established.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQln8BsjCEvbVg90H21EOLACvtfcHk0limeDl8DhSVMw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL1MXitVyntr4%253D&amp;md5=ede2875a8914ab295fe9f3683e582b86</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fbi00436a049&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00436a049%26sid%3Dliteratum%253Aachs%26aulast%3DCunningham%26aufirst%3DR.%2BP.%26aulast%3DAsahara%26aufirst%3DH.%26aulast%3DBank%26aufirst%3DJ.%2BF.%26aulast%3DScholes%26aufirst%3DC.%2BP.%26aulast%3DSalerno%26aufirst%3DJ.%2BC.%26aulast%3DSurerus%26aufirst%3DK.%26aulast%3DMunck%26aufirst%3DE.%26aulast%3DMcCracken%26aufirst%3DJ.%26aulast%3DPeisach%26aufirst%3DJ.%26aulast%3DEmptage%26aufirst%3DM.%2BH.%26atitle%3DEndonuclease%2520III%2520is%2520an%2520Iron-sulfur%2520Protein%26jtitle%3DBiochemistry%26date%3D1989%26volume%3D28%26issue%3D10%26spage%3D4450%26epage%3D4455%26doi%3D10.1021%2Fbi00436a049" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thayer, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahern, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span> <span> </span><span class="NLM_article-title">Novel DNA Binding Motifs in the DNA Repair Enzyme Endonuclease III Crystal Structure</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">4108</span>– <span class="NLM_lpage">4120</span>, <span class="refDoi">&nbsp;DOI: 10.1002/j.1460-2075.1995.tb00083.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1002%2Fj.1460-2075.1995.tb00083.x" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=1995&amp;pages=4108-4120&amp;issue=16&amp;author=M.+M.+Thayerauthor=H.+Ahernauthor=D.+Xingauthor=R.+P.+Cunninghamauthor=J.+A.+Tainer&amp;title=Novel+DNA+Binding+Motifs+in+the+DNA+Repair+Enzyme+Endonuclease+III+Crystal+Structure&amp;doi=10.1002%2Fj.1460-2075.1995.tb00083.x" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fj.1460-2075.1995.tb00083.x&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fj.1460-2075.1995.tb00083.x%26sid%3Dliteratum%253Aachs%26aulast%3DThayer%26aufirst%3DM.%2BM.%26aulast%3DAhern%26aufirst%3DH.%26aulast%3DXing%26aufirst%3DD.%26aulast%3DCunningham%26aufirst%3DR.%2BP.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26atitle%3DNovel%2520DNA%2520Binding%2520Motifs%2520in%2520the%2520DNA%2520Repair%2520Enzyme%2520Endonuclease%2520III%2520Crystal%2520Structure%26jtitle%3DEMBO%2520J.%26date%3D1995%26volume%3D14%26issue%3D16%26spage%3D4108%26epage%3D4120%26doi%3D10.1002%2Fj.1460-2075.1995.tb00083.x" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chepanoske, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golinelli, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span> <span> </span><span class="NLM_article-title">Positively Charged Residues within the Iron–Sulfur Cluster Loop of E. coli MutY Participate in Damage Recognition and Removal</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>380</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">19</span>, <span class="refDoi">&nbsp;DOI: 10.1006/abbi.2000.1890</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1006%2Fabbi.2000.1890" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=380&amp;publication_year=2000&amp;pages=11-19&amp;issue=1&amp;author=C.+L.+Chepanoskeauthor=M.-P.+Golinelliauthor=S.+D.+Williamsauthor=S.+S.+David&amp;title=Positively+Charged+Residues+within+the+Iron%E2%80%93Sulfur+Cluster+Loop+of+E.+coli+MutY+Participate+in+Damage+Recognition+and+Removal&amp;doi=10.1006%2Fabbi.2000.1890" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1006%2Fabbi.2000.1890&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabbi.2000.1890%26sid%3Dliteratum%253Aachs%26aulast%3DChepanoske%26aufirst%3DC.%2BL.%26aulast%3DGolinelli%26aufirst%3DM.-P.%26aulast%3DWilliams%26aufirst%3DS.%2BD.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26atitle%3DPositively%2520Charged%2520Residues%2520within%2520the%2520Iron%25E2%2580%2593Sulfur%2520Cluster%2520Loop%2520of%2520E.%2520coli%2520MutY%2520Participate%2520in%2520Damage%2520Recognition%2520and%2520Removal%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2000%26volume%3D380%26issue%3D1%26spage%3D11%26epage%3D19%26doi%3D10.1006%2Fabbi.2000.1890" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brinkmeyer, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span> <span> </span><span class="NLM_article-title">Distinct Functional Consequences of MUTYH Variants Associated with Colorectal Cancer: Damaged DNA Affinity, Glycosylase Activity and Interaction with PCNA and Hus1</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">51</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.dnarep.2015.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.dnarep.2015.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=26377631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtleks7vL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=2015&amp;pages=39-51&amp;author=M.+K.+Brinkmeyerauthor=S.+S.+David&amp;title=Distinct+Functional+Consequences+of+MUTYH+Variants+Associated+with+Colorectal+Cancer%3A+Damaged+DNA+Affinity%2C+Glycosylase+Activity+and+Interaction+with+PCNA+and+Hus1&amp;doi=10.1016%2Fj.dnarep.2015.08.001" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct functional consequences of MUTYH variants associated with colorectal cancer: Damaged DNA affinity, glycosylase activity and interaction with PCNA and Hus1</span></div><div class="casAuthors">Brinkmeyer, Megan K.; David, Sheila S.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39-51</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">MUTYH is a base excision repair (BER) enzyme that prevents mutations in DNA assocd. with 8-oxoguanine (OG) by catalyzing the removal of adenine from inappropriately formed OG:A base-pairs.  Germline mutations in the MUTYH gene are linked to colorectal polyposis and a high risk of colorectal cancer, a syndrome referred to as MUTYH-assocd. polyposis (MAP).  There are over 300 different MUTYH mutations assocd. with MAP and a large fraction of these gene changes code for missense MUTYH variants.  Herein, the adenine glycosylase activity, mismatch recognition properties, and interaction with relevant protein partners of human MUTYH and five MAP variants (R295C, P281L, Q324H, P502L, and R520Q) were examd.  P281L MUTYH was found to be severely compromised both in DNA binding and base excision activity, consistent with the location of this variation in the iron-sulfur cluster (FCL) DNA binding motif of MUTYH.  Both R295C and R520Q MUTYH were found to have low fractions of active enzyme, compromised affinity for damaged DNA, and reduced rates for adenine excision.  In contrast, both Q324H and P502L MUTYH function relatively similarly to WT MUTYH in both binding and glycosylase assays.  However, P502L and R520Q exhibited reduced affinity for PCNA (proliferation cell nuclear antigen), consistent with their location in the PCNA-binding motif of MUTYH.  Whereas, only Q324H, and not R295C, was found to have reduced affinity for Hus1 of the Rad9-Hus1-Rad1 complex, despite both being localized to the same region implicated for interaction with Hus1.  These results underscore the diversity of functional consequences due to MUTYH variants that may impact the progression of MAP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjxSwDOeuLK7Vg90H21EOLACvtfcHk0ljGsVkI7RfJcQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXhtleks7vL&amp;md5=aeeafa68215a91bd75b783c55d3c245d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2015.08.001&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2015.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DBrinkmeyer%26aufirst%3DM.%2BK.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26atitle%3DDistinct%2520Functional%2520Consequences%2520of%2520MUTYH%2520Variants%2520Associated%2520with%2520Colorectal%2520Cancer%253A%2520Damaged%2520DNA%2520Affinity%252C%2520Glycosylase%2520Activity%2520and%2520Interaction%2520with%2520PCNA%2520and%2520Hus1%26jtitle%3DDNA%2520Repair%26date%3D2015%26volume%3D34%26spage%3D39%26epage%3D51%26doi%3D10.1016%2Fj.dnarep.2015.08.001" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salay, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehlinger, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chazin, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">The [4Fe4S] Cluster of Human DNA Primase Functions as a Redox Switch using DNA Charge Transport</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span> (<span class="NLM_issue">6327</span>),  <span class="NLM_fpage">813</span>– <span class="NLM_lpage">822</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.aag1789</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.aag1789" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=355&amp;publication_year=2017&amp;pages=813-822&amp;issue=6327&amp;author=E.+O%E2%80%99Brienauthor=M.+E.+Holtauthor=M.+K.+Thompsonauthor=L.+E.+Salayauthor=A.+C.+Ehlingerauthor=W.+J.+Chazinauthor=J.+K.+Barton&amp;title=The+%5B4Fe4S%5D+Cluster+of+Human+DNA+Primase+Functions+as+a+Redox+Switch+using+DNA+Charge+Transport&amp;doi=10.1126%2Fscience.aag1789" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1126%2Fscience.aag1789&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aag1789%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DE.%26aulast%3DHolt%26aufirst%3DM.%2BE.%26aulast%3DThompson%26aufirst%3DM.%2BK.%26aulast%3DSalay%26aufirst%3DL.%2BE.%26aulast%3DEhlinger%26aufirst%3DA.%2BC.%26aulast%3DChazin%26aufirst%3DW.%2BJ.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DThe%2520%255B4Fe4S%255D%2520Cluster%2520of%2520Human%2520DNA%2520Primase%2520Functions%2520as%2520a%2520Redox%2520Switch%2520using%2520DNA%2520Charge%2520Transport%26jtitle%3DScience%26date%3D2017%26volume%3D355%26issue%3D6327%26spage%3D813%26epage%3D822%26doi%3D10.1126%2Fscience.aag1789" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartels, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stodola, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgers, P. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">A Redox Role for the [4Fe4S] Cluster of Yeast DNA Polymerase δ</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span> (<span class="NLM_issue">50</span>),  <span class="NLM_fpage">18339</span>– <span class="NLM_lpage">18348</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.7b10284</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b10284" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVKju7fJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=139&amp;publication_year=2017&amp;pages=18339-18348&amp;issue=50&amp;author=P.+L.+Bartelsauthor=J.+L.+Stodolaauthor=P.+M.+J.+Burgersauthor=J.+K.+Barton&amp;title=A+Redox+Role+for+the+%5B4Fe4S%5D+Cluster+of+Yeast+DNA+Polymerase+%CE%B4&amp;doi=10.1021%2Fjacs.7b10284" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Redox Role for the [4Fe4S] Cluster of Yeast DNA Polymerase δ</span></div><div class="casAuthors">Bartels, Phillip L.; Stodola, Joseph L.; Burgers, Peter M. J.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">18339-18348</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A [4Fe4S]2+ cluster in the C-terminal domain of the catalytic subunit of the eukaryotic B-family DNA polymerases is essential for the formation of active multi-subunit complexes.  Here we use a combination of electrochem. and biochem. methods to assess the redox activity of the [4Fe4S]2+ cluster in Saccharomyces cerevisiae polymerase (Pol) δ, the lagging strand DNA polymerase.  We find that Pol δ bound to DNA is indeed redox-active at physiol. potentials, generating a DNA-mediated signal electrochem. with a midpoint potential of 113 ± 5 mV vs. NHE.  Moreover, biochem. assays following electrochem. oxidn. of Pol δ reveal a significant slowing of DNA synthesis that can be fully reversed by redn. of the oxidized form.  A similar result is apparent with photooxidn. using a DNA-tethered anthraquinone.  These results demonstrate that the [4Fe4S] cluster in Pol δ can act as a redox switch for activity, and we propose that this switch can provide a rapid and reversible way to respond to replication stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT5FK9sBXvcLVg90H21EOLACvtfcHk0ljGsVkI7RfJcQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhvVKju7fJ&amp;md5=edbd20b63a557dd0f38092930f647963</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b10284&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b10284%26sid%3Dliteratum%253Aachs%26aulast%3DBartels%26aufirst%3DP.%2BL.%26aulast%3DStodola%26aufirst%3DJ.%2BL.%26aulast%3DBurgers%26aufirst%3DP.%2BM.%2BJ.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DA%2520Redox%2520Role%2520for%2520the%2520%255B4Fe4S%255D%2520Cluster%2520of%2520Yeast%2520DNA%2520Polymerase%2520%25CE%25B4%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26issue%3D50%26spage%3D18339%26epage%3D18348%26doi%3D10.1021%2Fjacs.7b10284" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engstrom, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinkmeyer, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ha, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgson, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span> <span> </span><span class="NLM_article-title">A Zinc Linchpin Motif in the MUTYH Glycosylase Interdomain Connector is Required for Efficient Repair of DNA Damage</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">7829</span>– <span class="NLM_lpage">7832</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja502942d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja502942d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=136&amp;publication_year=2014&amp;pages=7829-7832&amp;issue=22&amp;author=L.+M.+Engstromauthor=M.+K.+Brinkmeyerauthor=Y.+Haauthor=A.+G.+Raetzauthor=B.+Hedmanauthor=K.+O.+Hodgsonauthor=E.+I.+Solomonauthor=S.+S.+David&amp;title=A+Zinc+Linchpin+Motif+in+the+MUTYH+Glycosylase+Interdomain+Connector+is+Required+for+Efficient+Repair+of+DNA+Damage&amp;doi=10.1021%2Fja502942d" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fja502942d&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja502942d%26sid%3Dliteratum%253Aachs%26aulast%3DEngstrom%26aufirst%3DL.%2BM.%26aulast%3DBrinkmeyer%26aufirst%3DM.%2BK.%26aulast%3DHa%26aufirst%3DY.%26aulast%3DRaetz%26aufirst%3DA.%2BG.%26aulast%3DHedman%26aufirst%3DB.%26aulast%3DHodgson%26aufirst%3DK.%2BO.%26aulast%3DSolomon%26aufirst%3DE.%2BI.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26atitle%3DA%2520Zinc%2520Linchpin%2520Motif%2520in%2520the%2520MUTYH%2520Glycosylase%2520Interdomain%2520Connector%2520is%2520Required%2520for%2520Efficient%2520Repair%2520of%2520DNA%2520Damage%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26issue%3D22%26spage%3D7829%26epage%3D7832%26doi%3D10.1021%2Fja502942d" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porello, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span> <span> </span><span class="NLM_article-title">A Substrate Recognition Role for the [4Fe-4S]2+ Cluster of the DNA Repair Glycosylase MutY</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">6465</span>– <span class="NLM_lpage">6475</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi972433t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi972433t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=37&amp;publication_year=1998&amp;pages=6465-6475&amp;issue=18&amp;author=S.+L.+Porelloauthor=M.+J.+Cannonauthor=S.+S.+David&amp;title=A+Substrate+Recognition+Role+for+the+%5B4Fe-4S%5D2%2B+Cluster+of+the+DNA+Repair+Glycosylase+MutY&amp;doi=10.1021%2Fbi972433t" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fbi972433t&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi972433t%26sid%3Dliteratum%253Aachs%26aulast%3DPorello%26aufirst%3DS.%2BL.%26aulast%3DCannon%26aufirst%3DM.%2BJ.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26atitle%3DA%2520Substrate%2520Recognition%2520Role%2520for%2520the%2520%255B4Fe-4S%255D2%252B%2520Cluster%2520of%2520the%2520DNA%2520Repair%2520Glycosylase%2520MutY%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26issue%3D18%26spage%3D6465%26epage%3D6475%26doi%3D10.1021%2Fbi972433t" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pope, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span> <span> </span><span class="NLM_article-title">DNA Damage Recognition and Repair by the Murine MutY Homologue</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">102</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.dnarep.2004.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.dnarep.2004.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=15533841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptl2qtr0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2005&amp;pages=91-102&amp;issue=1&amp;author=M.+A.+Popeauthor=S.+S.+David&amp;title=DNA+Damage+Recognition+and+Repair+by+the+Murine+MutY+Homologue&amp;doi=10.1016%2Fj.dnarep.2004.08.004" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">DNA damage recognition and repair by the murine MutY homologue</span></div><div class="casAuthors">Pope, Mary Ann; David, Sheila S.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-102</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">E. coli MutY excises adenine from duplex DNA when it is mispaired with the mutagenic oxidative lesion 7,8-dihydro-8-oxo-2'-deoxyguanosine (OG).  While E. coli MutY has been extensively studied, a detailed kinetic anal. of a mammalian MutY homolog has been inhibited by poor overexpression in bacterial hosts.  This current work is the first detailed study of substrate recognition and repair of mismatched DNA by a mammalian adenine glycosylase, the murine MutY homolog (mMYH).  Similar to E. coli MutY, the processing of OG:A substrates by mMYH is biphasic, indicating that product release is rate-limiting.  Surprisingly, the intrinsic rates of adenine removal from both OG:A and G:A substrates by mMYH are diminished (∼10-fold) compared to E. coli MutY.  However, similar to E. coli MutY, the rate of adenine removal is approx. nine-fold faster with an OG:A- than a G:A-contg. substrate.  Interestingly, the rate of removal of 2-hydroxyadenine mispaired with OG or G in duplex DNA by mMYH was similar to the rate of adenine removal from the analogous context.  In contrast, 2-hydroxyadenine removal by E. coli MutY was significantly reduced compared to adenine removal opposite both OG and G.  Furthermore, dissocn. const. measurements with duplexes contg. noncleavable 2'-deoxyadenosine analogs indicate that mMYH is less sensitive to the structure of the base mispaired with OG or G than MutY.  Though in many respects the catalytic behavior of mMYH is similar to E. coli MutY, the subtle differences may translate into differences in their in vivo functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8iFzE3gzwn7Vg90H21EOLACvtfcHk0ljWpr2-qQWfuQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2cXptl2qtr0%253D&amp;md5=6b1609d40a0e3d24e9d3bbf429de1e8d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2004.08.004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2004.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DPope%26aufirst%3DM.%2BA.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26atitle%3DDNA%2520Damage%2520Recognition%2520and%2520Repair%2520by%2520the%2520Murine%2520MutY%2520Homologue%26jtitle%3DDNA%2520Repair%26date%3D2005%26volume%3D4%26issue%3D1%26spage%3D91%26epage%3D102%26doi%3D10.1016%2Fj.dnarep.2004.08.004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Structure of a Trapped Endonuclease III–DNA Covalent Intermediate</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">3461</span>– <span class="NLM_lpage">3471</span>, <span class="refDoi">&nbsp;DOI: 10.1093/emboj/cdg311</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1093%2Femboj%2Fcdg311" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=2003&amp;pages=3461-3471&amp;issue=13&amp;author=J.+C.+Frommeauthor=G.+L.+Verdine&amp;title=Structure+of+a+Trapped+Endonuclease+III%E2%80%93DNA+Covalent+Intermediate&amp;doi=10.1093%2Femboj%2Fcdg311" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Femboj%2Fcdg311&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252Fcdg311%26sid%3Dliteratum%253Aachs%26aulast%3DFromme%26aufirst%3DJ.%2BC.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DStructure%2520of%2520a%2520Trapped%2520Endonuclease%2520III%25E2%2580%2593DNA%2520Covalent%2520Intermediate%26jtitle%3DEMBO%2520J.%26date%3D2003%26volume%3D22%26issue%3D13%26spage%3D3461%26epage%3D3471%26doi%3D10.1093%2Femboj%2Fcdg311" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Handley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholes, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, R. P.</span></span> <span> </span><span class="NLM_article-title">Endonuclease III Interactions with DNA Substrates. 1. Binding and Footprinting Studies with Oligonucleotides Containing a Reduced Apyrimidinic Site</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2528</span>– <span class="NLM_lpage">2536</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00008a017</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00008a017" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=1995&amp;pages=2528-2536&amp;issue=8&amp;author=S.+O%E2%80%99Handleyauthor=C.+P.+Scholesauthor=R.+P.+Cunningham&amp;title=Endonuclease+III+Interactions+with+DNA+Substrates.+1.+Binding+and+Footprinting+Studies+with+Oligonucleotides+Containing+a+Reduced+Apyrimidinic+Site&amp;doi=10.1021%2Fbi00008a017" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fbi00008a017&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00008a017%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Handley%26aufirst%3DS.%26aulast%3DScholes%26aufirst%3DC.%2BP.%26aulast%3DCunningham%26aufirst%3DR.%2BP.%26atitle%3DEndonuclease%2520III%2520Interactions%2520with%2520DNA%2520Substrates.%25201.%2520Binding%2520and%2520Footprinting%2520Studies%2520with%2520Oligonucleotides%2520Containing%2520a%2520Reduced%2520Apyrimidinic%2520Site%26jtitle%3DBiochemistry%26date%3D1995%26volume%3D34%26issue%3D8%26spage%3D2528%26epage%3D2536%26doi%3D10.1021%2Fbi00008a017" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Handley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J.</span></span> <span> </span><span class="NLM_article-title">Atomic Structure of the DNA Repair [4Fe-4S] Enzyme Endonuclease III</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>258</i></span> (<span class="NLM_issue">5081</span>),  <span class="NLM_fpage">434</span>– <span class="NLM_lpage">440</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.1411536</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.1411536" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1411536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK3sXns1Smsw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=258&amp;publication_year=1992&amp;pages=434-440&amp;issue=5081&amp;author=C.+Kuoauthor=D.+McReeauthor=C.+Fisherauthor=S.+O%E2%80%99Handleyauthor=R.+Cunninghamauthor=J.+Tainer&amp;title=Atomic+Structure+of+the+DNA+Repair+%5B4Fe-4S%5D+Enzyme+Endonuclease+III&amp;doi=10.1126%2Fscience.1411536" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Atomic structure of the DNA repair [4Fe-4S] enzyme endonuclease III</span></div><div class="casAuthors">Kuo, Che Fu; McRee, Duncan E.; Fisher, Cindy L.; O'Handley, Suzanne F.; Cunningham, Richard P.; Tainer, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">258</span>
        (<span class="NLM_cas:issue">5081</span>),
    <span class="NLM_cas:pages">434-40</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The crystal structure of the DNA repair enzyme endonuclease III, which recognizes and cleaves DNA at damaged bases, has been solved to 2.0 angstrom resoln. with an R factor of 0.185.  This iron-sulfur [4Fe-4S] enzyme is elongated and bilobal with a deep cleft sepg. two similarly sized domains: a novel, sequence-continuous, six-helix domain (residues 22 to 132) and a Greek-key, four-helix domain formed by the amino-terminal and three carboxyl-terminal helices (residues 1 to 21 and 133 to 211) together with the [4Fe-4S] cluster.  The cluster is bound entirely within the carboxyl-terminal loop with a ligation pattern (Cys-X6-Cys-X2-Cys-X5-Cys) distinct from all other known [4Fe-4S] proteins.  Sequence conservation and the pos. electrostatic potential of conserved regions identify a surface suitable for binding duplex B-DNA across the long axis of the enzyme, matching a 46 angstrom length of protected DNA.  The primary role of the [4Fe-4S] cluster appears to involve positioning conserved basic residues for interaction with the DNA phosphate backbone.  The crystallog. identified inhibitor binding region, which recognizes the damaged base thymine glycol, is a seven-residue β-hairpin (residues 113 to 119).  Location and side chain orientation at the base of the inhibitor binding site implicate Glu112 in the N-glycosylase mechanism and Lys120 in the β-elimination mechanism.  Overall, the structure reveals an unusual fold and a new biol. function for [4Fe-4S] clusters and provides a structural basis for studying recognition of damaged DNA and the N-glycosylase and apurinic/apyrimidinic-lyase mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp46-I_W5BWR7Vg90H21EOLACvtfcHk0ljWpr2-qQWfuQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3sXns1Smsw%253D%253D&amp;md5=1333604d2d5452b8b7f8caa3aefabc91</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1126%2Fscience.1411536&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1411536%26sid%3Dliteratum%253Aachs%26aulast%3DKuo%26aufirst%3DC.%26aulast%3DMcRee%26aufirst%3DD.%26aulast%3DFisher%26aufirst%3DC.%26aulast%3DO%25E2%2580%2599Handley%26aufirst%3DS.%26aulast%3DCunningham%26aufirst%3DR.%26aulast%3DTainer%26aufirst%3DJ.%26atitle%3DAtomic%2520Structure%2520of%2520the%2520DNA%2520Repair%2520%255B4Fe-4S%255D%2520Enzyme%2520Endonuclease%2520III%26jtitle%3DScience%26date%3D1992%26volume%3D258%26issue%3D5081%26spage%3D434%26epage%3D440%26doi%3D10.1126%2Fscience.1411536" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yavin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Repair Glycosylases with a [4Fe-4S] Cluster: A Redox Cofactor for DNA-mediated Charge Transport?</span>. <i>J. Inorg. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>101</i></span> (<span class="NLM_issue">11–12</span>),  <span class="NLM_fpage">1913</span>– <span class="NLM_lpage">1921</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.jinorgbio.2007.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.jinorgbio.2007.05.001" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=101&amp;publication_year=2007&amp;pages=1913-1921&amp;issue=11%E2%80%9312&amp;author=A.+K.+Boalauthor=E.+Yavinauthor=J.+K.+Barton&amp;title=DNA+Repair+Glycosylases+with+a+%5B4Fe-4S%5D+Cluster%3A+A+Redox+Cofactor+for+DNA-mediated+Charge+Transport%3F&amp;doi=10.1016%2Fj.jinorgbio.2007.05.001" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.jinorgbio.2007.05.001&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jinorgbio.2007.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DBoal%26aufirst%3DA.%2BK.%26aulast%3DYavin%26aufirst%3DE.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Repair%2520Glycosylases%2520with%2520a%2520%255B4Fe-4S%255D%2520Cluster%253A%2520A%2520Redox%2520Cofactor%2520for%2520DNA-mediated%2520Charge%2520Transport%253F%26jtitle%3DJ.%2520Inorg.%2520Biochem.%26date%3D2007%26volume%3D101%26issue%3D11%25E2%2580%259312%26spage%3D1913%26epage%3D1921%26doi%3D10.1016%2Fj.jinorgbio.2007.05.001" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuss, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Q. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arvai, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span> <span> </span><span class="NLM_article-title">XPD Helicase Structures and Activities: Insights into the Cancer and Aging Phenotypes from XPD Mutations</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>133</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">789</span>– <span class="NLM_lpage">800</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2008.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.cell.2008.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=18510924" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD1cXntFaltrk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=133&amp;publication_year=2008&amp;pages=789-800&amp;issue=5&amp;author=L.+Fanauthor=J.+O.+Fussauthor=Q.+J.+Chengauthor=A.+S.+Arvaiauthor=M.+Hammelauthor=V.+A.+Robertsauthor=P.+K.+Cooperauthor=J.+A.+Tainer&amp;title=XPD+Helicase+Structures+and+Activities%3A+Insights+into+the+Cancer+and+Aging+Phenotypes+from+XPD+Mutations&amp;doi=10.1016%2Fj.cell.2008.04.030" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">XPD helicase structures and activities: insights into the cancer and aging phenotypes from XPD mutations</span></div><div class="casAuthors">Fan, Li; Fuss, Jill O.; Cheng, Quen J.; Arvai, Andrew S.; Hammel, Michael; Roberts, Victoria A.; Cooper, Priscilla K.; Tainer, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">789-800</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mutations in XPD helicase, required for nucleotide excision repair (NER) as part of the transcription/repair complex TFIIH, cause three distinct phenotypes: cancer-prone xeroderma pigmentosum (XP), or aging disorders Cockayne syndrome (CS), and trichothiodystrophy (TTD).  To clarify mol. differences underlying these diseases, we detd. crystal structures of the XPD catalytic core from Sulfolobus acidocaldarius and measured mutant enzyme activities.  Substrate-binding grooves sep. adjacent Rad51/RecA-like helicase domains (HD1, HD2) and an arch formed by 4FeS and Arch domains.  XP mutations map along the HD1 ATP-binding edge and HD2 DNA-binding channel and impair helicase activity essential for NER.  XP/CS mutations both impair helicase activity and likely affect HD2 functional movement.  TTD mutants lose or retain helicase activity but map to sites in all four domains expected to cause framework defects impacting TFIIH integrity.  These results provide a foundation for understanding disease consequences of mutations in XPD and related 4Fe-4S helicases including FancJ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4p4pSDz59pLVg90H21EOLACvtfcHk0ljUwr9G6kf5Xw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXntFaltrk%253D&amp;md5=997e8595afd3001a6a3134c143ff10b4</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2008.04.030&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2008.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DL.%26aulast%3DFuss%26aufirst%3DJ.%2BO.%26aulast%3DCheng%26aufirst%3DQ.%2BJ.%26aulast%3DArvai%26aufirst%3DA.%2BS.%26aulast%3DHammel%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DV.%2BA.%26aulast%3DCooper%26aufirst%3DP.%2BK.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26atitle%3DXPD%2520Helicase%2520Structures%2520and%2520Activities%253A%2520Insights%2520into%2520the%2520Cancer%2520and%2520Aging%2520Phenotypes%2520from%2520XPD%2520Mutations%26jtitle%3DCell%26date%3D2008%26volume%3D133%26issue%3D5%26spage%3D789%26epage%3D800%26doi%3D10.1016%2Fj.cell.2008.04.030" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lukianova, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span> <span> </span><span class="NLM_article-title">A Role for Iron–sulfur Clusters in DNA Repair</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">151</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cbpa.2005.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.cbpa.2005.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=15811798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivFSmu7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2005&amp;pages=145-151&amp;issue=2&amp;author=O.+A.+Lukianovaauthor=S.+S.+David&amp;title=A+Role+for+Iron%E2%80%93sulfur+Clusters+in+DNA+Repair&amp;doi=10.1016%2Fj.cbpa.2005.02.006" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A role for iron-sulfur clusters in DNA repair</span></div><div class="casAuthors">Lukianova, Olga A.; David, Sheila S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">145-151</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The presence of 4Fe-4S clusters in enzymes involved in DNA repair has posed the question of the role of these intricate cofactors in damaged DNA recognition and repair.  It is particularly intriguing that base excision repair glycosylases that remove a wide variety of damaged bases, and also have vastly different sequences and structures, have been found to contain this cofactor.  The accumulating biochem. and structural evidence indicates that the region supported by the cluster is intimately involved in DNA binding, and that such binding interactions impact catalysis of base removal.  Recent evidence has also established that binding of the glycosylases to DNA facilitates oxidn. of the [4Fe-4S]2+ cluster to the [4Fe-4S]3+ form.  Notably, the measured redox potentials for a variety of 4Fe-4S cluster-contg. glycosylases are remarkably similar.  Based on this DNA-mediated redox behavior, it has been suggested that this property may be used to enhance the activity of these enzymes by facilitating damaged DNA location.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-8OnHpqYlQbVg90H21EOLACvtfcHk0ljUwr9G6kf5Xw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXivFSmu7c%253D&amp;md5=62710de60d4903a9ba812dd7179e14a5</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2005.02.006&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2005.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DLukianova%26aufirst%3DO.%2BA.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26atitle%3DA%2520Role%2520for%2520Iron%25E2%2580%2593sulfur%2520Clusters%2520in%2520DNA%2520Repair%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D9%26issue%3D2%26spage%3D145%26epage%3D151%26doi%3D10.1016%2Fj.cbpa.2005.02.006" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouault, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, W.-H.</span></span> <span> </span><span class="NLM_article-title">Iron-sulphur Cluster Biogenesis and Mitochondrial Iron Homeostasis</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">345</span>– <span class="NLM_lpage">351</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nrm1620</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnrm1620" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=15803140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivVaktb4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2005&amp;pages=345-351&amp;issue=4&amp;author=T.+A.+Rouaultauthor=W.-H.+Tong&amp;title=Iron-sulphur+Cluster+Biogenesis+and+Mitochondrial+Iron+Homeostasis&amp;doi=10.1038%2Fnrm1620" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: Iron-sulphur cluster biogenesis and mitochondrial iron homeostasis</span></div><div class="casAuthors">Rouault, Tracey A.; Tong, Wing-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">345-351</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Iron-sulfur clusters are important cofactors for proteins that are involved in many cellular processes, including electron transport, enzymic catalysis and regulation.  The enzymes that catalyze the formation of iron-sulfur clusters are widely conserved from bacteria to humans.  Recent studies in model systems and humans reveal that iron-sulfur proteins have important roles in mitochondrial iron homeostasis and in the pathogenesis of the human disease Friedreich ataxia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD6B2xuSAXdLVg90H21EOLACvtfcHk0ljUwr9G6kf5Xw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXivVaktb4%253D&amp;md5=159664e92102e2d50b85f549a858ab98</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnrm1620&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1620%26sid%3Dliteratum%253Aachs%26aulast%3DRouault%26aufirst%3DT.%2BA.%26aulast%3DTong%26aufirst%3DW.-H.%26atitle%3DIron-sulphur%2520Cluster%2520Biogenesis%2520and%2520Mitochondrial%2520Iron%2520Homeostasis%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26issue%3D4%26spage%3D345%26epage%3D351%26doi%3D10.1038%2Fnrm1620" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouault, T.
A.</span></span> <span> </span><span class="NLM_article-title">Mammalian Iron-sulphur Proteins: Novel Insights into Biogenesis and Function</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">55</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nrm3909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnrm3909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=25425402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitValurzO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=2015&amp;pages=45-55&amp;issue=1&amp;author=T.%0AA.+Rouault&amp;title=Mammalian+Iron-sulphur+Proteins%3A+Novel+Insights+into+Biogenesis+and+Function&amp;doi=10.1038%2Fnrm3909" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian iron-sulfur proteins: novel insights into biogenesis and function</span></div><div class="casAuthors">Rouault, Tracey A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-55</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fe-S clusters are inorg. cofactors that are found in nearly all species and are composed of various combinations of Fe and S atoms.  Fe-S clusters can accept or donate single electrons to carry out oxidn. and redn. reactions and to facilitate electron transport.  Many details of how these complex modular structures are assembled and ligated to cellular proteins in the mitochondrial, nuclear, and cytosolic compartments of mammalian cells remain unclear.  Recent evidence indicates that a Leu-Tyr-Arg (LYR) tripeptide motif found in some Fe-S recipient proteins may facilitate the direct and shielded transfer of Fe-S clusters from a scaffold to client proteins.  Fe-S clusters are probably an unrecognized and elusive cofactor of many known proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqdvbYH-jqfrVg90H21EOLACvtfcHk0lj4PP2c3Ibirg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXitValurzO&amp;md5=a9250f89d32139aa70d8193cf72f9b95</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnrm3909&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3909%26sid%3Dliteratum%253Aachs%26aulast%3DRouault%26aufirst%3DT.%2BA.%26atitle%3DMammalian%2520Iron-sulphur%2520Proteins%253A%2520Novel%2520Insights%2520into%2520Biogenesis%2520and%2520Function%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D16%26issue%3D1%26spage%3D45%26epage%3D55%26doi%3D10.1038%2Fnrm3909" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veatch, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottschling, D. E.</span></span> <span> </span><span class="NLM_article-title">Mitochondrial Dysfunction Leads to Nuclear Genome Instability via an Iron-Sulfur Cluster Defect</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>137</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1258</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2009.04.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.cell.2009.04.014" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=137&amp;publication_year=2009&amp;pages=1247-1258&amp;issue=7&amp;author=J.+R.+Veatchauthor=M.+A.+McMurrayauthor=Z.+W.+Nelsonauthor=D.+E.+Gottschling&amp;title=Mitochondrial+Dysfunction+Leads+to+Nuclear+Genome+Instability+via+an+Iron-Sulfur+Cluster+Defect&amp;doi=10.1016%2Fj.cell.2009.04.014" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.04.014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.04.014%26sid%3Dliteratum%253Aachs%26aulast%3DVeatch%26aufirst%3DJ.%2BR.%26aulast%3DMcMurray%26aufirst%3DM.%2BA.%26aulast%3DNelson%26aufirst%3DZ.%2BW.%26aulast%3DGottschling%26aufirst%3DD.%2BE.%26atitle%3DMitochondrial%2520Dysfunction%2520Leads%2520to%2520Nuclear%2520Genome%2520Instability%2520via%2520an%2520Iron-Sulfur%2520Cluster%2520Defect%26jtitle%3DCell%26date%3D2009%26volume%3D137%26issue%3D7%26spage%3D1247%26epage%3D1258%26doi%3D10.1016%2Fj.cell.2009.04.014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tse, E. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">The Oxidation State of [4Fe4S] Clusters Modulates the DNA-Binding Affinity of DNA Repair Proteins</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">12784</span>– <span class="NLM_lpage">12792</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.7b07230</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b07230" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlCqs7nP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=139&amp;publication_year=2017&amp;pages=12784-12792&amp;issue=36&amp;author=E.+C.+M.+Tseauthor=T.+J.+Zwangauthor=J.+K.+Barton&amp;title=The+Oxidation+State+of+%5B4Fe4S%5D+Clusters+Modulates+the+DNA-Binding+Affinity+of+DNA+Repair+Proteins&amp;doi=10.1021%2Fjacs.7b07230" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The Oxidation State of [4Fe4S] Clusters Modulates the DNA-Binding Affinity of DNA Repair Proteins</span></div><div class="casAuthors">Tse, Edmund C. M.; Zwang, Theodore J.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">12784-12792</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abstr.: A central question important to understanding DNA repair is how certain proteins are able to search for, detect, and fix DNA damage on a biol. relevant timescale.  A feature of many base excision repair proteins is that they contain [4Fe4S] clusters that may aid their search for lesions.  In this report, we establish the importance of the oxidn. state of the redox-active [4Fe4S] cluster in the DNA damage detection process.  We utilize DNA-modified electrodes to generate repair proteins with [4Fe4S] clusters in the 2+ and 3+ states by bulk electrolysis under an O2-free atm.  Anaerobic microscale thermophoresis results indicate that proteins carrying [4Fe4S]3+ clusters bind to DNA 550 times more tightly than those with [4Fe4S]2+ clusters.  The measured increase in DNA-binding affinity matches the affinity change assocd. with the redox potential shift obsd. for [4Fe4S] cluster proteins upon binding to DNA.  We further devise an electrostatic model that shows this change in DNA-binding affinity of these proteins can be fully explained by the differences in electrostatic interactions between DNA and the [4Fe4S] cluster in the reduced vs. oxidized state.  We then utilize at. force microscopy (AFM) to demonstrate that the redox state of the [4Fe4S] clusters regulates the ability of two DNA repair proteins, Endonuclease III and DinG, to bind preferentially to DNA duplexes contg. a single site of DNA damage (here a base mismatch) which inhibits DNA charge transport.  Together, these results show that the redn. and oxidn. of [4Fe4S] clusters through DNA-mediated charge transport facilitates long-range signaling between [4Fe4S] repair proteins.  The redox-modulated change in DNA-binding affinity regulates the ability of [4Fe4S] repair proteins to collaborate in the lesion detection process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMG3dpTH_YD7Vg90H21EOLACvtfcHk0lj4PP2c3Ibirg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhtlCqs7nP&amp;md5=e9c6e823b9111ea182c82817bcad4ce9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b07230&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b07230%26sid%3Dliteratum%253Aachs%26aulast%3DTse%26aufirst%3DE.%2BC.%2BM.%26aulast%3DZwang%26aufirst%3DT.%2BJ.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DThe%2520Oxidation%2520State%2520of%2520%255B4Fe4S%255D%2520Clusters%2520Modulates%2520the%2520DNA-Binding%2520Affinity%2520of%2520DNA%2520Repair%2520Proteins%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26issue%3D36%26spage%3D12784%26epage%3D12792%26doi%3D10.1021%2Fjacs.7b07230" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gorodetsky, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Direct Electrochemistry of Endonuclease III in the Presence and Absence of DNA</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span> (<span class="NLM_issue">37</span>),  <span class="NLM_fpage">12082</span>– <span class="NLM_lpage">12083</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja064784d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja064784d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=128&amp;publication_year=2006&amp;pages=12082-12083&amp;issue=37&amp;author=A.+A.+Gorodetskyauthor=A.+K.+Boalauthor=J.+K.+Barton&amp;title=Direct+Electrochemistry+of+Endonuclease+III+in+the+Presence+and+Absence+of+DNA&amp;doi=10.1021%2Fja064784d" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja064784d&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja064784d%26sid%3Dliteratum%253Aachs%26aulast%3DGorodetsky%26aufirst%3DA.%2BA.%26aulast%3DBoal%26aufirst%3DA.%2BK.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDirect%2520Electrochemistry%2520of%2520Endonuclease%2520III%2520in%2520the%2520Presence%2520and%2520Absence%2520of%2520DNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26issue%3D37%26spage%3D12082%26epage%3D12083%26doi%3D10.1021%2Fja064784d" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartels, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuñez, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespilho, F. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Electrochemistry of the [4Fe4S] Cluster in Base Excision Repair Proteins: Tuning the Redox Potential with DNA</span>. <i>Langmuir</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2523</span>– <span class="NLM_lpage">2530</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.langmuir.6b04581</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.langmuir.6b04581" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVaiur0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=33&amp;publication_year=2017&amp;pages=2523-2530&amp;issue=10&amp;author=P.+L.+Bartelsauthor=A.+Zhouauthor=A.+R.+Arnoldauthor=N.+N.+Nu%C3%B1ezauthor=F.+N.+Crespilhoauthor=S.+S.+Davidauthor=J.+K.+Barton&amp;title=Electrochemistry+of+the+%5B4Fe4S%5D+Cluster+in+Base+Excision+Repair+Proteins%3A+Tuning+the+Redox+Potential+with+DNA&amp;doi=10.1021%2Facs.langmuir.6b04581" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Electrochemistry of the [4Fe4S] Cluster in Base Excision Repair Proteins: Tuning the Redox Potential with DNA</span></div><div class="casAuthors">Bartels, Phillip L.; Zhou, Andy; Arnold, Anna R.; Nunez, Nicole N.; Crespilho, Frank N.; David, Sheila S.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Langmuir</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2523-2530</span>CODEN:
                <span class="NLM_cas:coden">LANGD5</span>;
        ISSN:<span class="NLM_cas:issn">0743-7463</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Escherichia coli endonuclease III (EndoIII) and MutY are DNA glycosylases that contain [4Fe4S] clusters and that serve to maintain the integrity of the genome after oxidative stress.  Electrochem. studies on highly-oriented pyrolytic graphite (HOPG) revealed that DNA binding by EndoIII leads to a large neg. shift in midpoint potential of the cluster, consistent with stabilization of the oxidized [4Fe4S]3+ form.  However, the smooth, hydrophobic HOPG surface is non-ideal for working with proteins in the absence of DNA.  Here, we used thin film voltammetry on a pyrolytic graphite edge electrode to overcome these limitations.  Improved adsorption led to substantial signals for both EndoIII and MutY in the absence of DNA, and a large neg. potential shift was retained with DNA present.  In contrast, EndoIII mutants E200K, Y205H, and K208E, which provide electrostatic perturbations in the vicinity of the cluster, all showed DNA-free potentials within error of wild type; similarly, the presence of neg. charged poly-L-glutamate did not lead to a significant potential shift.  Overall, binding to the DNA polyanion was the dominant effect in tuning the redox potential of the [4Fe4S] cluster, helping to explain why all DNA-binding proteins with [4Fe4S] clusters studied to date have similar DNA-bound potentials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoebVWlyKmJZLVg90H21EOLACvtfcHk0li5TRHFhZoshg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXjtVaiur0%253D&amp;md5=a7984edceec68ce867d1e3a3eecdaa7d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.langmuir.6b04581&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.langmuir.6b04581%26sid%3Dliteratum%253Aachs%26aulast%3DBartels%26aufirst%3DP.%2BL.%26aulast%3DZhou%26aufirst%3DA.%26aulast%3DArnold%26aufirst%3DA.%2BR.%26aulast%3DNu%25C3%25B1ez%26aufirst%3DN.%2BN.%26aulast%3DCrespilho%26aufirst%3DF.%2BN.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DElectrochemistry%2520of%2520the%2520%255B4Fe4S%255D%2520Cluster%2520in%2520Base%2520Excision%2520Repair%2520Proteins%253A%2520Tuning%2520the%2520Redox%2520Potential%2520with%2520DNA%26jtitle%3DLangmuir%26date%3D2017%26volume%3D33%26issue%3D10%26spage%3D2523%26epage%3D2530%26doi%3D10.1021%2Facs.langmuir.6b04581" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chemler, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartels, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marvin, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raskin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, D. H.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">A human MUTYH variant linking colonic polyposis to redox degradation of the [4Fe4S]2+ cluster</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">873</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41557-018-0068-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fs41557-018-0068-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=29915346" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKgsL7O" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2018&amp;pages=873&amp;issue=8&amp;author=K.+J.+McDonnellauthor=J.+A.+Chemlerauthor=P.+L.+Bartelsauthor=E.+O%E2%80%99Brienauthor=M.+L.+Marvinauthor=J.+Ortegaauthor=R.+H.+Sternauthor=L.+Raskinauthor=G.-M.+Liauthor=D.+H.+Sherman&amp;title=A+human+MUTYH+variant+linking+colonic+polyposis+to+redox+degradation+of+the+%5B4Fe4S%5D2%2B+cluster&amp;doi=10.1038%2Fs41557-018-0068-x" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A human MUTYH variant linking colonic polyposis to redox degradation of the [4Fe4S]2+ cluster</span></div><div class="casAuthors">McDonnell, Kevin J.; Chemler, Joseph A.; Bartels, Phillip L.; O'Brien, Elizabeth; Marvin, Monica L.; Ortega, Janice; Stern, Ralph H.; Raskin, Leon; Li, Guo-Min; Sherman, David H.; Barton, Jacqueline K.; Gruber, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">873-880</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The human DNA repair enzyme MUTYH excises mispaired adenine residues in oxidized DNA.  Homozygous MUTYH mutations underlie the autosomal, recessive cancer syndrome MUTYH-assocd. polyposis.  We report a MUTYH variant, p.  C306W (c.918C&gt;G), with a tryptophan residue in place of native cysteine, that ligates the [4Fe4S] cluster in a patient with colonic polyposis and family history of early age colon cancer.  In bacterial MutY, the [4Fe4S] cluster is redox active, allowing rapid localization to target lesions by long-range, DNA-mediated signaling.  In the current study, using DNA electrochem., we det. that wild-type MUTYH is similarly redox-active, but MUTYH C306W undergoes rapid oxidative degrdn. of its cluster to [3Fe4S]+, with loss of redox signaling.  In MUTYH C306W, oxidative cluster degrdn. leads to decreased DNA binding and enzyme function.  This study confirms redox activity in eukaryotic DNA repair proteins and establishes MUTYH C306W as a pathogenic variant, highlighting the essential role of redox signaling by the [4Fe4S] cluster.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqd1vLIhEj1bVg90H21EOLACvtfcHk0li5TRHFhZoshg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKgsL7O&amp;md5=4ce582747197797232e56d1eec995161</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs41557-018-0068-x&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-018-0068-x%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonnell%26aufirst%3DK.%2BJ.%26aulast%3DChemler%26aufirst%3DJ.%2BA.%26aulast%3DBartels%26aufirst%3DP.%2BL.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DE.%26aulast%3DMarvin%26aufirst%3DM.%2BL.%26aulast%3DOrtega%26aufirst%3DJ.%26aulast%3DStern%26aufirst%3DR.%2BH.%26aulast%3DRaskin%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DG.-M.%26aulast%3DSherman%26aufirst%3DD.%2BH.%26atitle%3DA%2520human%2520MUTYH%2520variant%2520linking%2520colonic%2520polyposis%2520to%2520redox%2520degradation%2520of%2520the%2520%255B4Fe4S%255D2%252B%2520cluster%26jtitle%3DNat.%2520Chem.%26date%3D2018%26volume%3D10%26issue%3D8%26spage%3D873%26doi%3D10.1038%2Fs41557-018-0068-x" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghatlia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turro, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J.</span></span> <span> </span><span class="NLM_article-title">Long-range photoinduced electron transfer through a DNA helix</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>262</i></span> (<span class="NLM_issue">5136</span>),  <span class="NLM_fpage">1025</span>– <span class="NLM_lpage">1029</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.7802858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.7802858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=7802858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK2cXhtVSqtrg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=262&amp;publication_year=1993&amp;pages=1025-1029&amp;issue=5136&amp;author=C.+Murphyauthor=M.+Arkinauthor=Y.+Jenkinsauthor=N.+Ghatliaauthor=S.+Bossmannauthor=N.+Turroauthor=J.+Barton&amp;title=Long-range+photoinduced+electron+transfer+through+a+DNA+helix&amp;doi=10.1126%2Fscience.7802858" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Long-range photoinduced electron transfer through a DNA helix</span></div><div class="casAuthors">Murphy, C. J.; Arkin, M. R.; Jenkins, Y.; Ghatlia, N. D.; Bossmann, S. H.; Turro, N. J.; Barton, J. K.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">262</span>
        (<span class="NLM_cas:issue">5136</span>),
    <span class="NLM_cas:pages">1025-9</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">Rapid photoinduced electron transfer is demonstrated over a distance of &gt;40 angstroms between metallointercalators that are tethered to the 5' termini of a 15-base-pair DNA duplex.  An oligomeric assembly was synthesized in which the donor is Ru(phen)2dppz2+ (phen, phenanthroline, and dppz, dipyridophenazine) and the acceptor is Rh(phi)2phen3+ (phi, phenanthrenequinone diimine).  These metal complexes are intercalated either one or two base steps in from the helix termini.  Although the Ru-modified oligonucleotide hybridized to an unmodified complement luminesces intensely, the Ru-modified oligomer hybridized to the Rh-modified oligomer shows no detectable luminescence.  Time-resolved studies point to a lower limit of 109 per s for the quenching rate.  No quenching was obsd. upon metalation of 2 complementary octamers by Ru(phen)32+ and Rh(phen)33+ under conditions where the phen complexes do not intercalate.  The stacked arom. heterocycles of the DNA duplex therefore serve as an efficient medium for coupling electron donors and acceptors over very long distances.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5jM1emtEmcrVg90H21EOLACvtfcHk0li5TRHFhZoshg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2cXhtVSqtrg%253D&amp;md5=e15c6acf6047c76aa7bddb0c1edc86a2</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.7802858&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7802858%26sid%3Dliteratum%253Aachs%26aulast%3DMurphy%26aufirst%3DC.%26aulast%3DArkin%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DY.%26aulast%3DGhatlia%26aufirst%3DN.%26aulast%3DBossmann%26aufirst%3DS.%26aulast%3DTurro%26aufirst%3DN.%26aulast%3DBarton%26aufirst%3DJ.%26atitle%3DLong-range%2520photoinduced%2520electron%2520transfer%2520through%2520a%2520DNA%2520helix%26jtitle%3DScience%26date%3D1993%26volume%3D262%26issue%3D5136%26spage%3D1025%26epage%3D1029%26doi%3D10.1126%2Fscience.7802858" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiebig, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treadway, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zewail, A. H.</span></span> <span> </span><span class="NLM_article-title">Femtosecond dynamics of DNA-mediated electron transfer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">6014</span>– <span class="NLM_lpage">6019</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.96.11.6014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1073%2Fpnas.96.11.6014" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=96&amp;publication_year=1999&amp;pages=6014-6019&amp;issue=11&amp;author=C.+Wanauthor=T.+Fiebigauthor=S.+O.+Kelleyauthor=C.+R.+Treadwayauthor=J.+K.+Bartonauthor=A.+H.+Zewail&amp;title=Femtosecond+dynamics+of+DNA-mediated+electron+transfer&amp;doi=10.1073%2Fpnas.96.11.6014" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.11.6014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.11.6014%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DC.%26aulast%3DFiebig%26aufirst%3DT.%26aulast%3DKelley%26aufirst%3DS.%2BO.%26aulast%3DTreadway%26aufirst%3DC.%2BR.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26aulast%3DZewail%26aufirst%3DA.%2BH.%26atitle%3DFemtosecond%2520dynamics%2520of%2520DNA-mediated%2520electron%2520transfer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26issue%3D11%26spage%3D6014%26epage%3D6019%26doi%3D10.1073%2Fpnas.96.11.6014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stemp, E. D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmlin, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hörmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbara, P. F.</span></span> <span> </span><span class="NLM_article-title">Rates of DNA-Mediated Electron Transfer Between Metallointercalators</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>273</i></span> (<span class="NLM_issue">5274</span>),  <span class="NLM_fpage">475</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.273.5274.475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.273.5274.475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=8662532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK28Xks1Wmurk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=273&amp;publication_year=1996&amp;pages=475&amp;issue=5274&amp;author=M.+R.+Arkinauthor=E.+D.+A.+Stempauthor=R.+E.+Holmlinauthor=J.+K.+Bartonauthor=A.+H%C3%B6rmannauthor=E.+J.+C.+Olsonauthor=P.+F.+Barbara&amp;title=Rates+of+DNA-Mediated+Electron+Transfer+Between+Metallointercalators&amp;doi=10.1126%2Fscience.273.5274.475" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Rates of DNA-mediated electron transfer between metallointercalators</span></div><div class="casAuthors">Arkin, M. R.; Stemp, E. D. A.; Holmlin, R. E.; Barton, J. K.; Hormann, A.; Olson, E. J. C.; Barbara, P. F.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">5274</span>),
    <span class="NLM_cas:pages">475-480</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Ultrafast emission and absorption spectroscopies were used to measure the kinetics of DNA-mediated electron transfer reactions between metal complexes intercalated into DNA.  In the presence of rhodium(III) acceptor, a substantial fraction of photoexcited donor exhibits fast oxidative quenching (&gt;3×1010 per s).  Transient-absorption expts. indicate that, for a series of donors, the majority of back electron transfer is also very fast (∼1010 per s).  This rate is independent of the loading of acceptors on the helix, but is sensitive to sequence and π stacking.  The cooperative binding of donor and acceptor is considered unlikely on the basis of structural models and DNA photocleavage studies of binding.  The data show that the DNA double helix differs significantly from proteins as a bridge for electron transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryvkrg-vpLPrVg90H21EOLACvtfcHk0lif367kwsWnyg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK28Xks1Wmurk%253D&amp;md5=6906953202b7ae2208c7bd17a62b301d</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1126%2Fscience.273.5274.475&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.273.5274.475%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DStemp%26aufirst%3DE.%2BD.%2BA.%26aulast%3DHolmlin%26aufirst%3DR.%2BE.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26aulast%3DH%25C3%25B6rmann%26aufirst%3DA.%26aulast%3DOlson%26aufirst%3DE.%2BJ.%2BC.%26aulast%3DBarbara%26aufirst%3DP.%2BF.%26atitle%3DRates%2520of%2520DNA-Mediated%2520Electron%2520Transfer%2520Between%2520Metallointercalators%26jtitle%3DScience%26date%3D1996%26volume%3D273%26issue%3D5274%26spage%3D475%26doi%3D10.1126%2Fscience.273.5274.475" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sontz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muren, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Charge Transport for Sensing and Signaling</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1792</span>– <span class="NLM_lpage">1800</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar3001298</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar3001298" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=2012&amp;pages=1792-1800&amp;issue=10&amp;author=P.+A.+Sontzauthor=N.+B.+Murenauthor=J.+K.+Barton&amp;title=DNA+Charge+Transport+for+Sensing+and+Signaling&amp;doi=10.1021%2Far3001298" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Far3001298&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far3001298%26sid%3Dliteratum%253Aachs%26aulast%3DSontz%26aufirst%3DP.%2BA.%26aulast%3DMuren%26aufirst%3DN.%2BB.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Charge%2520Transport%2520for%2520Sensing%2520and%2520Signaling%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2012%26volume%3D45%26issue%3D10%26spage%3D1792%26epage%3D1800%26doi%3D10.1021%2Far3001298" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merino, E.
J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Biological Contexts for DNA Charge Transport Chemistry</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">229</span>– <span class="NLM_lpage">237</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cbpa.2008.01.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.cbpa.2008.01.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=18314014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslKlsbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2008&amp;pages=229-237&amp;issue=2&amp;author=E.%0AJ.+Merinoauthor=A.+K.+Boalauthor=J.+K.+Barton&amp;title=Biological+Contexts+for+DNA+Charge+Transport+Chemistry&amp;doi=10.1016%2Fj.cbpa.2008.01.046" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Biological contexts for DNA charge transport chemistry</span></div><div class="casAuthors">Merino, Edward J.; Boal, Amie K.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">229-237</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Many expts. have now shown that double helical DNA can serve as a conduit for efficient charge transport (CT) reactions over long distances in vitro.  These results prompt the consideration of biol. roles for DNA-mediated CT.  DNA CT has been demonstrated to occur in biol. relevant environments such as within the mitochondria and nuclei of HeLa cells as well as in isolated nucleosomes.  In mitochondria, DNA damage that results from CT is funneled to a crucial regulatory element.  Thus, DNA CT provides a strategy to funnel damage to particular sites in the genome.  DNA CT might also be important in long-range signaling to DNA-bound proteins.  Both DNA repair proteins, contg. Fe-S clusters, and the transcription factor, p53, which is regulated through thiol-disulfide switches, can be oxidized from a distance through DNA-mediated CT.  These observations highlight a means through which oxidative stress may be chem. signaled in the genome over long distances through CT from guanine radicals to DNA-bound proteins.  Moreover, DNA-mediated CT may also play a role in signaling among DNA-binding proteins, as has been proposed as a mechanism for how DNA repair glycosylases more efficiently detect lesions inside the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_JiLip4xOhLVg90H21EOLACvtfcHk0lgiF93lkId7aw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXlslKlsbc%253D&amp;md5=76215e9a6b925424dfd6ed7615a67739</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2008.01.046&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2008.01.046%26sid%3Dliteratum%253Aachs%26aulast%3DMerino%26aufirst%3DE.%2BJ.%26aulast%3DBoal%26aufirst%3DA.%2BK.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DBiological%2520Contexts%2520for%2520DNA%2520Charge%2520Transport%2520Chemistry%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D12%26issue%3D2%26spage%3D229%26epage%3D237%26doi%3D10.1016%2Fj.cbpa.2008.01.046" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grodick, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muren, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Charge Transport within the Cell</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">962</span>– <span class="NLM_lpage">973</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi501520w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi501520w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlyrs7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=54&amp;publication_year=2015&amp;pages=962-973&amp;issue=4&amp;author=M.+A.+Grodickauthor=N.+B.+Murenauthor=J.+K.+Barton&amp;title=DNA+Charge+Transport+within+the+Cell&amp;doi=10.1021%2Fbi501520w" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">DNA charge transport within the cell</span></div><div class="casAuthors">Grodick, Michael A.; Muren, Natalie B.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">962-973</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The unique characteristics of DNA charge transport (CT) have prompted an examn. of roles for this chem. within a biol. context.  Not only can DNA CT facilitate long-range oxidative damage of DNA, but redox-active proteins can couple to the DNA base stack and participate in long-range redox reactions using DNA CT.  DNA transcription factors with redox-active moieties such as SoxR and p53 can use DNA CT as a form of redox sensing.  DNA CT chem. also provides a means to monitor the integrity of the DNA, given the sensitivity of DNA CT to perturbations in base stacking as arise with mismatches and lesions.  Enzymes that utilize this chem. include an interesting and ever-growing class of DNA-processing enzymes involved in DNA repair, replication, and transcription that have been found to contain 4Fe-4S clusters.  DNA repair enzymes contg. 4Fe-4S clusters, that include endonuclease III (EndoIII), MutY, and DinG from bacteria, as well as XPD from archaea, have been shown to be redox-active when bound to DNA, share a DNA-bound redox potential, and can be reduced and oxidized at long-range via DNA CT.  Interactions between DNA and these proteins in soln., in addn. to genetics expts. within Escherichia coli, suggest that DNA-mediated CT can be used as a means of cooperative signaling among DNA repair proteins that contain 4Fe-4S clusters as a 1st step in finding DNA damage, even within cells.  On the basis of these data, one can consider also how DNA-mediated CT may be used as a means of signaling to coordinate DNA processing across the genome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofZd49KeRWhLVg90H21EOLACvtfcHk0lgiF93lkId7aw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXhtlyrs7g%253D&amp;md5=11008f93a6f8dd195abb82ea3c54e754</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fbi501520w&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi501520w%26sid%3Dliteratum%253Aachs%26aulast%3DGrodick%26aufirst%3DM.%2BA.%26aulast%3DMuren%26aufirst%3DN.%2BB.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Charge%2520Transport%2520within%2520the%2520Cell%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26issue%3D4%26spage%3D962%26epage%3D973%26doi%3D10.1021%2Fbi501520w" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grodick, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Charge Transport: from Chemical Principles to the Cell</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">197</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.chembiol.2015.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.chembiol.2015.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=26933744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlSqsb8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2016&amp;pages=183-197&amp;issue=1&amp;author=A.+R.+Arnoldauthor=M.+A.+Grodickauthor=J.+K.+Barton&amp;title=DNA+Charge+Transport%3A+from+Chemical+Principles+to+the+Cell&amp;doi=10.1016%2Fj.chembiol.2015.11.010" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">DNA Charge Transport: from Chemical Principles to the Cell</span></div><div class="casAuthors">Arnold, Anna R.; Grodick, Michael A.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">183-197</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The DNA double helix has captured the imagination of many, bringing it to the forefront of biol. research.  DNA has unique features that extend our interest into areas of chem., physics, material science, and engineering.  Our lab. has focused on studies of DNA charge transport (CT), wherein charges can efficiently travel long mol. distances through the DNA helix while maintaining an exquisite sensitivity to base pair π-stacking.  Because DNA CT chem. reports on the integrity of the DNA duplex, this property may be exploited to develop electrochem. devices to detect DNA lesions and DNA-binding proteins.  Furthermore, studies now indicate that DNA CT may also be used in the cell by, for example, DNA repair proteins, as a cellular diagnostic, in order to scan the genome to localize efficiently to damage sites.  In this review, we describe this evolution of DNA CT chem. from the discovery of fundamental chem. principles to applications in diagnostic strategies and possible roles in biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs9YMsEG1yKLVg90H21EOLACvtfcHk0lgiF93lkId7aw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XntlSqsb8%253D&amp;md5=556f33e3baa00e039092f8a6697214dc</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.11.010&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DArnold%26aufirst%3DA.%2BR.%26aulast%3DGrodick%26aufirst%3DM.%2BA.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Charge%2520Transport%253A%2520from%2520Chemical%2520Principles%2520to%2520the%2520Cell%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26issue%3D1%26spage%3D183%26epage%3D197%26doi%3D10.1016%2Fj.chembiol.2015.11.010" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Núñez, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmquist, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Evidence for DNA Charge Transport in the Nucleus</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">42</span>),  <span class="NLM_fpage">12465</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi011560t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi011560t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=40&amp;publication_year=2001&amp;pages=12465&amp;issue=42&amp;author=M.+E.+N%C3%BA%C3%B1ezauthor=G.+P.+Holmquistauthor=J.+K.+Barton&amp;title=Evidence+for+DNA+Charge+Transport+in+the+Nucleus&amp;doi=10.1021%2Fbi011560t" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fbi011560t&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi011560t%26sid%3Dliteratum%253Aachs%26aulast%3DN%25C3%25BA%25C3%25B1ez%26aufirst%3DM.%2BE.%26aulast%3DHolmquist%26aufirst%3DG.%2BP.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DEvidence%2520for%2520DNA%2520Charge%2520Transport%2520in%2520the%2520Nucleus%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26issue%3D42%26spage%3D12465%26doi%3D10.1021%2Fbi011560t" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Genereux, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA-Mediated Charge Transport in Redox Sensing and Signaling</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">891</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja907669c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja907669c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=132&amp;publication_year=2010&amp;pages=891&amp;issue=3&amp;author=J.+C.+Genereuxauthor=A.+K.+Boalauthor=J.+K.+Barton&amp;title=DNA-Mediated+Charge+Transport+in+Redox+Sensing+and+Signaling&amp;doi=10.1021%2Fja907669c" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fja907669c&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja907669c%26sid%3Dliteratum%253Aachs%26aulast%3DGenereux%26aufirst%3DJ.%2BC.%26aulast%3DBoal%26aufirst%3DA.%2BK.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA-Mediated%2520Charge%2520Transport%2520in%2520Redox%2520Sensing%2520and%2520Signaling%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26issue%3D3%26spage%3D891%26doi%3D10.1021%2Fja907669c" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, R.</span>; <span class="NLM_string-name">Kondev, J.</span>; <span class="NLM_string-name">Theriot, J.</span>; <span class="NLM_string-name">Orme, N.</span></span> <i>Physical Biology of the Cell</i>; <span class="NLM_publisher-name">Garland Science</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2013&amp;author=R.+Phillips&amp;author=J.+Kondev&amp;author=J.+Theriot&amp;author=N.+Orme&amp;title=Physical+Biology+of+the+Cell" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPhillips%26aufirst%3DR.%26btitle%3DPhysical%2520Biology%2520of%2520the%2520Cell%26pub%3DGarland%2520Science%26date%3D2013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nenninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mastroianni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullineaux, C. W.</span></span> <span> </span><span class="NLM_article-title">Size Dependence of Protein Diffusion in the Cytoplasm of Escherichia coli</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>192</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">4535</span>– <span class="NLM_lpage">4540</span>, <span class="refDoi">&nbsp;DOI: 10.1128/JB.00284-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1128%2FJB.00284-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=20581203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Wgt7bP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=192&amp;publication_year=2010&amp;pages=4535-4540&amp;issue=18&amp;author=A.+Nenningerauthor=G.+Mastroianniauthor=C.+W.+Mullineaux&amp;title=Size+Dependence+of+Protein+Diffusion+in+the+Cytoplasm+of+Escherichia+coli&amp;doi=10.1128%2FJB.00284-10" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Size dependence of protein diffusion in the cytoplasm of Escherichia coli</span></div><div class="casAuthors">Nenninger, Anja; Mastroianni, Giulia; Mullineaux, Conrad W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4535-4540</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Diffusion in the bacterial cytoplasm is regarded as the primary method of intracellular protein movement and must play a major role in controlling the rates of cell processes.  A no. of recent studies have used green fluorescent protein (GFP) tagging and fluorescence microscopy to probe the movement and distribution of proteins in the bacterial cytoplasm.  However, the dynamic behavior of indigenous proteins must be controlled by a complex mixt. of specific interactions, combined with the basic phys. constraints imposed by the viscosity and macromol. crowding of the cytoplasm.  These factors are difficult to unravel in studies with indigenous proteins.  To what extent the addn. of a GFP tag might affect the movement of a protein through the cytoplasm has also remained unknown.  To resolve these problems, the authors have carried out a systematic study of the size dependence of protein diffusion coeffs. in the Escherichia coli cytoplasm, using engineered GFP multimers (from 2 to 6 covalently linked GFP mols.).  Diffusion coeffs. were measured using confocal fluorescence recovery after photobleaching (FRAP).  At least up to 110 kDa (four linked GFP mols.), the diffusion coeff. varies with size roughly as would be predicted from the Einstein-Stokes equation for a classical (Newtonian) fluid.  Thus, protein diffusion coeffs. are predictable over this range.  GFP tagging of proteins has little impact on the diffusion coeff. over this size range and therefore need not significantly perturb protein movement.  Two indigenous E. coli proteins were used to show that their specific interactions within the cell are the main controllers of the diffusion rate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb7gYA9y_8BbVg90H21EOLACvtfcHk0lixPGYr5-ChCg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXht1Wgt7bP&amp;md5=30cb24212ae23023c36d4f2670e913c9</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1128%2FJB.00284-10&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.00284-10%26sid%3Dliteratum%253Aachs%26aulast%3DNenninger%26aufirst%3DA.%26aulast%3DMastroianni%26aufirst%3DG.%26aulast%3DMullineaux%26aufirst%3DC.%2BW.%26atitle%3DSize%2520Dependence%2520of%2520Protein%2520Diffusion%2520in%2520the%2520Cytoplasm%2520of%2520Escherichia%2520coli%26jtitle%3DJ.%2520Bacteriol.%26date%3D2010%26volume%3D192%26issue%3D18%26spage%3D4535%26epage%3D4540%26doi%3D10.1128%2FJB.00284-10" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mommer, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sourjik, V.</span></span> <span> </span><span class="NLM_article-title">Mobility of Cytoplasmic, Membrane, and DNA-Binding Proteins in Escherichia coli</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>98</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">559</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bpj.2009.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.bpj.2009.11.002" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=98&amp;publication_year=2010&amp;pages=552-559&amp;issue=4&amp;author=M.+Kumarauthor=M.+S.+Mommerauthor=V.+Sourjik&amp;title=Mobility+of+Cytoplasmic%2C+Membrane%2C+and+DNA-Binding+Proteins+in+Escherichia+coli&amp;doi=10.1016%2Fj.bpj.2009.11.002" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2009.11.002&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2009.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DM.%26aulast%3DMommer%26aufirst%3DM.%2BS.%26aulast%3DSourjik%26aufirst%3DV.%26atitle%3DMobility%2520of%2520Cytoplasmic%252C%2520Membrane%252C%2520and%2520DNA-Binding%2520Proteins%2520in%2520Escherichia%2520coli%26jtitle%3DBiophys.%2520J.%26date%3D2010%26volume%3D98%26issue%3D4%26spage%3D552%26epage%3D559%26doi%3D10.1016%2Fj.bpj.2009.11.002" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slinker, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muren, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">renfrew, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Charge Transport over 34 nm</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">228</span>– <span class="NLM_lpage">233</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.982</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnchem.982" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=21336329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1Wru7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2011&amp;pages=228-233&amp;author=J.+D.+Slinkerauthor=N.+B.+Murenauthor=S.+E.+renfrewauthor=J.+K.+Barton&amp;title=DNA+Charge+Transport+over+34+nm&amp;doi=10.1038%2Fnchem.982" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">DNA charge transport over 34 nm</span></div><div class="casAuthors">Slinker, Jason D.; Muren, Natalie B.; Renfrew, Sara E.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">228-233</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mol. wires show promise in nanoscale electronics, but the synthesis of uniform, long conductive mols. is a significant challenge.  DNA of precise length, by contrast, is synthesized easily, but its cond. over the distances required for nanoscale devices has not been explored.  Here we demonstrate DNA charge transport (CT) over 34 nm in 100-mer monolayers on gold.  Multiplexed gold electrodes modified with 100-mer DNA yield sizable electrochem. signals from a distal, covalent Nile Blue redox probe.  Significant signal attenuation upon incorporation of a single base-pair mismatch demonstrates that CT is DNA-mediated.  Efficient cleavage of these 100-mers by a restriction enzyme indicates that the DNA adopts a native conformation accessible to protein binding.  Similar electron-transfer rates measured through 100-mer and 17-mer monolayers are consistent with rate-limiting electron tunneling through the satd. carbon linker.  This DNA-mediated CT distance of 34 nm surpasses that of most reports of mol. wires.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGkMrXL2figbVg90H21EOLACvtfcHk0lixPGYr5-ChCg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXit1Wru7g%253D&amp;md5=35aa021f2af63744480f812085648101</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnchem.982&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.982%26sid%3Dliteratum%253Aachs%26aulast%3DSlinker%26aufirst%3DJ.%2BD.%26aulast%3DMuren%26aufirst%3DN.%2BB.%26aulast%3Drenfrew%26aufirst%3DS.%2BE.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Charge%2520Transport%2520over%252034%2520nm%26jtitle%3DNat.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D228%26epage%3D233%26doi%3D10.1038%2Fnchem.982" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagerman, P. J.</span></span> <span> </span><span class="NLM_article-title">Application of the method of phage T4 DNA ligase-catalyzed ring-closure to the study of DNA structure: II. NaCl-dependence of DNA flexibility and helical repeat</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>212</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">376</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0022-2836(90)90131-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2F0022-2836%2890%2990131-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=2319604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK3cXksleiu7k%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=212&amp;publication_year=1990&amp;pages=363-376&amp;issue=2&amp;author=W.+H.+Taylorauthor=P.+J.+Hagerman&amp;title=Application+of+the+method+of+phage+T4+DNA+ligase-catalyzed+ring-closure+to+the+study+of+DNA+structure%3A+II.+NaCl-dependence+of+DNA+flexibility+and+helical+repeat&amp;doi=10.1016%2F0022-2836%2890%2990131-5" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Application of the method of phage T4 DNA ligase-catalyzed ring-closure to the study of DNA structure.  II.  Sodium chloride-dependence of DNA flexibility and helical repeat</span></div><div class="casAuthors">Taylor, William H.; Hagerman, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">212</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">363-76</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">It is possible to det. the molar cyclization factor jM from single ligation reactions in which both circular and linear dimer DNA species are formed concurrently from linear monomers.  This approach represents a significant improvement over previous methods, in which jM is evaluated from the ratio of the rate consts. for 2 sep. processes; namely (1) the cyclization of linear DNA, and (2) the assocn. of 2 linear mols. to form linear dimers.  Detn. of jM for a 366 base-pair mol. yields 5.8 × 10-8M, in close agreement with the value of 5.6 × 10-8M detd. by D. Shore et al. (1981) for the same mol.  Using the current approach for the detn. of jM, the dependence on NaCl concn. (0-162 mM NaCl, 1 mM MgCl2) of both the lateral and torsional flexibilities of DNA was investigated.  The principal observation is that both quantities are essentially const. over the above range of NaCl concns., with the persistence length P ≈ 450 Å, and the torsional elastic const. C ≈ 2.0 × 10-19 erg cm.  These observations are in accord with the previous theor. prediction that P becomes essentially independent of NaCl concn. &gt;10-20 mM.  The dependence of the helical repeat of DNA on NaCl concn. over the above range was examd. and the value of 10.44 base-pairs per turn was found to be essentially const. over that range.  This last result suggests that earlier studies have overestimated the dependence of DNA helical twist on salt concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqB6EdmWu0H7Vg90H21EOLACvtfcHk0ljB89P09vRvAQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3cXksleiu7k%253D&amp;md5=26f07d95faeb72aa1ce2d7ff76a3c18f</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2890%2990131-5&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252890%252990131-5%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DW.%2BH.%26aulast%3DHagerman%26aufirst%3DP.%2BJ.%26atitle%3DApplication%2520of%2520the%2520method%2520of%2520phage%2520T4%2520DNA%2520ligase-catalyzed%2520ring-closure%2520to%2520the%2520study%2520of%2520DNA%2520structure%253A%2520II.%2520NaCl-dependence%2520of%2520DNA%2520flexibility%2520and%2520helical%2520repeat%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1990%26volume%3D212%26issue%3D2%26spage%3D363%26epage%3D376%26doi%3D10.1016%2F0022-2836%2890%2990131-5" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bustamante, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siggia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span> <span> </span><span class="NLM_article-title">Entropic elasticity of lambda-phage DNA</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>265</i></span> (<span class="NLM_issue">5178</span>),  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1600</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.8079175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.8079175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=8079175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A280%3ADyaK2czmtVamsw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=265&amp;publication_year=1994&amp;pages=1599-1600&amp;issue=5178&amp;author=C.+Bustamanteauthor=J.+Markoauthor=E.+Siggiaauthor=S.+Smith&amp;title=Entropic+elasticity+of+lambda-phage+DNA&amp;doi=10.1126%2Fscience.8079175" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Entropic elasticity of lambda-phage DNA</span></div><div class="casAuthors">Bustamante C; Marko J F; Siggia E D; Smith S</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">5178</span>),
    <span class="NLM_cas:pages">1599-600</span>
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQsGUJ2IL7MSRfKb6ELK8t0fW6udTcc2eZv_6TLseM8Vrntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADyaK2czmtVamsw%253D%253D&amp;md5=2b6782dc3f2f4ca980772b7516d6df00</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1126%2Fscience.8079175&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8079175%26sid%3Dliteratum%253Aachs%26aulast%3DBustamante%26aufirst%3DC.%26aulast%3DMarko%26aufirst%3DJ.%26aulast%3DSiggia%26aufirst%3DE.%26aulast%3DSmith%26aufirst%3DS.%26atitle%3DEntropic%2520elasticity%2520of%2520lambda-phage%2520DNA%26jtitle%3DScience%26date%3D1994%26volume%3D265%26issue%3D5178%26spage%3D1599%26epage%3D1600%26doi%3D10.1126%2Fscience.8079175" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagerman, P. J.</span></span> <span> </span><span class="NLM_article-title">Flexibility of DNA</span>. <i>Annu. Rev. Biophys. Biophys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">286</span>, <span class="refDoi">&nbsp;DOI: 10.1146/annurev.bb.17.060188.001405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1146%2Fannurev.bb.17.060188.001405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=3293588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaL1cXltVKmtbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=1988&amp;pages=265-286&amp;issue=1&amp;author=P.+J.+Hagerman&amp;title=Flexibility+of+DNA&amp;doi=10.1146%2Fannurev.bb.17.060188.001405" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Flexibility of DNA</span></div><div class="casAuthors">Hagerman, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Biophysical Chemistry</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">265-86</span>CODEN:
                <span class="NLM_cas:coden">ARBCEY</span>;
        ISSN:<span class="NLM_cas:issn">0883-9182</span>.
    </div><div class="casAbstract">A review with 113 refs. on the intrinsic flexibility of DNA.  The topics discussed include directional persistence as a function of chain stiffness, exptl. approaches to the detn. of persistence length, torsional stiffness of DNA, and factors contributing to the stiffness of DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Z53AVxScXLVg90H21EOLACvtfcHk0ljB89P09vRvAQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL1cXltVKmtbw%253D&amp;md5=05c802dfe48bb12a555e41ed948e075f</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1146%2Fannurev.bb.17.060188.001405&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.bb.17.060188.001405%26sid%3Dliteratum%253Aachs%26aulast%3DHagerman%26aufirst%3DP.%2BJ.%26atitle%3DFlexibility%2520of%2520DNA%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biophys.%2520Chem.%26date%3D1988%26volume%3D17%26issue%3D1%26spage%3D265%26epage%3D286%26doi%3D10.1146%2Fannurev.bb.17.060188.001405" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finzi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bustamante, C.</span></span> <span> </span><span class="NLM_article-title">Direct mechanical measurements of the elasticity of single DNA molecules by using magnetic beads</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>258</i></span> (<span class="NLM_issue">5085</span>),  <span class="NLM_fpage">1122</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.1439819</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.1439819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1439819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK3sXps1Cguw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=258&amp;publication_year=1992&amp;pages=1122-1126&amp;issue=5085&amp;author=S.+Smithauthor=L.+Finziauthor=C.+Bustamante&amp;title=Direct+mechanical+measurements+of+the+elasticity+of+single+DNA+molecules+by+using+magnetic+beads&amp;doi=10.1126%2Fscience.1439819" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Direct mechanical measurements of the elasticity of single DNA molecules by using magnetic beads</span></div><div class="casAuthors">Smith, Steven B.; Finzi, Laura; Bustamante, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">258</span>
        (<span class="NLM_cas:issue">5085</span>),
    <span class="NLM_cas:pages">1122-6</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">Single DNA mols. were chem. attached by one end to a glass surface and by their other end to a magnetic bead.  Equil. positions of the beads were obsd. in an optical microscope while the beads were acted on by known magnetic and hydrodynamic forces.  Extension vs. force curves were obtained for individual DNA mols. at three different salt concns. with forces between 10-14 and 10-11 newtons.  Deviations from the force curves predicted by the freely jointed chain model suggest that DNA has significant local curvature in soln.  Ethidium bromide and 4',6-diamidino-2-phenylindole had little effect on the elastic response of the mols., but their extent of intercalation was directly measured.  Conversely, the effect of bend-inducing cis-diamminedichloroplatinium(II) was large and supports the hypothesis of natural curvature in DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUQD0F589HNbVg90H21EOLACvtfcHk0lgMdFrp9c4VLg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3sXps1Cguw%253D%253D&amp;md5=4cbf5b9d4aba803be42ba52dcb9d05f7</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1126%2Fscience.1439819&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1439819%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DFinzi%26aufirst%3DL.%26aulast%3DBustamante%26aufirst%3DC.%26atitle%3DDirect%2520mechanical%2520measurements%2520of%2520the%2520elasticity%2520of%2520single%2520DNA%2520molecules%2520by%2520using%2520magnetic%2520beads%26jtitle%3DScience%26date%3D1992%26volume%3D258%26issue%3D5085%26spage%3D1122%26epage%3D1126%26doi%3D10.1126%2Fscience.1439819" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zwang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, E. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Sensing DNA through DNA Charge Transport</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1799</span>– <span class="NLM_lpage">1809</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acschembio.8b00347</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00347" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpvFGjs70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=13&amp;publication_year=2018&amp;pages=1799-1809&amp;issue=7&amp;author=T.+J.+Zwangauthor=E.+C.+M.+Tseauthor=J.+K.+Barton&amp;title=Sensing+DNA+through+DNA+Charge+Transport&amp;doi=10.1021%2Facschembio.8b00347" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Sensing DNA through DNA Charge Transport</span></div><div class="casAuthors">Zwang, Theodore J.; Tse, Edmund C. M.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1799-1809</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA charge transport chem. involves the migration of charge over long mol. distances through the arom. base pair stack within the DNA helix.  This migration depends upon the intimate coupling of bases stacked one with another, and hence any perturbation in that stacking, through base modifications or protein binding, can be sensed elec.  In this review, the authors describe the many ways DNA charge transport chem. has been utilized to sense changes in DNA, including the presence of lesions, mismatches, DNA-binding proteins, protein activity, and even reactions under weak magnetic fields.  Charge transport chem. is remarkable in its ability to sense the integrity of DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAq1l1RQptELVg90H21EOLACvtfcHk0lgMdFrp9c4VLg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXpvFGjs70%253D&amp;md5=c354086b2190b636205ac1746678ab54</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00347&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00347%26sid%3Dliteratum%253Aachs%26aulast%3DZwang%26aufirst%3DT.%2BJ.%26aulast%3DTse%26aufirst%3DE.%2BC.%2BM.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DSensing%2520DNA%2520through%2520DNA%2520Charge%2520Transport%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D7%26spage%3D1799%26epage%3D1809%26doi%3D10.1021%2Facschembio.8b00347" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sontz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mui, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuss, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Charge Transport as a First Step in Coordinating the Detection of Lesions by Repair Proteins</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1856</span>– <span class="NLM_lpage">1861</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1120063109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1073%2Fpnas.1120063109" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=109&amp;publication_year=2012&amp;pages=1856-1861&amp;issue=6&amp;author=P.+A.+Sontzauthor=T.+P.+Muiauthor=J.+O.+Fussauthor=J.+A.+Tainerauthor=J.+K.+Barton&amp;title=DNA+Charge+Transport+as+a+First+Step+in+Coordinating+the+Detection+of+Lesions+by+Repair+Proteins&amp;doi=10.1073%2Fpnas.1120063109" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1120063109&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1120063109%26sid%3Dliteratum%253Aachs%26aulast%3DSontz%26aufirst%3DP.%2BA.%26aulast%3DMui%26aufirst%3DT.%2BP.%26aulast%3DFuss%26aufirst%3DJ.%2BO.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Charge%2520Transport%2520as%2520a%2520First%2520Step%2520in%2520Coordinating%2520the%2520Detection%2520of%2520Lesions%2520by%2520Repair%2520Proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26issue%3D6%26spage%3D1856%26epage%3D1861%26doi%3D10.1073%2Fpnas.1120063109" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romano, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sontz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Mutants of the Base Excision Repair Glycosylase, Endonuclease III: DNA Charge Transport as a First Step in Lesion Detection</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">27</span>),  <span class="NLM_fpage">6133</span>– <span class="NLM_lpage">6145</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi2003179</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi2003179" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=50&amp;publication_year=2011&amp;pages=6133-6145&amp;issue=27&amp;author=C.+A.+Romanoauthor=P.+A.+Sontzauthor=J.+K.+Barton&amp;title=Mutants+of+the+Base+Excision+Repair+Glycosylase%2C+Endonuclease+III%3A+DNA+Charge+Transport+as+a+First+Step+in+Lesion+Detection&amp;doi=10.1021%2Fbi2003179" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fbi2003179&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi2003179%26sid%3Dliteratum%253Aachs%26aulast%3DRomano%26aufirst%3DC.%2BA.%26aulast%3DSontz%26aufirst%3DP.%2BA.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DMutants%2520of%2520the%2520Base%2520Excision%2520Repair%2520Glycosylase%252C%2520Endonuclease%2520III%253A%2520DNA%2520Charge%2520Transport%2520as%2520a%2520First%2520Step%2520in%2520Lesion%2520Detection%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26issue%3D27%26spage%3D6133%26epage%3D6145%26doi%3D10.1021%2Fbi2003179" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Geggier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotlyar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vologodskii, A.</span></span> <span> </span><span class="NLM_article-title">Temperature dependence of DNA persistence length</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1419</span>– <span class="NLM_lpage">1426</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gkq932</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1093%2Fnar%2Fgkq932" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=20952402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC3MXis1Chsbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=2011&amp;pages=1419-1426&amp;issue=4&amp;author=S.+Geggierauthor=A.+Kotlyarauthor=A.+Vologodskii&amp;title=Temperature+dependence+of+DNA+persistence+length&amp;doi=10.1093%2Fnar%2Fgkq932" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Temperature dependence of DNA persistence length</span></div><div class="casAuthors">Geggier, Stephanie; Kotlyar, Alexander; Vologodskii, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1419-1426</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We have detd. the temp. dependence of DNA persistence length, a, using two different methods.  The first approach was based on measuring the j-factors of short DNA fragments at various temps.  Fitting the measured j-factors by the theor. equation allowed us to obtain the values of a for temps. between 5°C and 42°C.  The second approach was based on measuring the equil. distribution of the linking no. between the strands of circular DNA at different temps.  The major contribution into the distribution variance comes from the fluctuations of DNA writhe in the nicked circular mols. which are specified by the value of a.  The computation-based anal. of the measured variances was used to obtain the values of a for temps. up to 60°C.  We found a good agreement between the results obtained by these two methods.  Our data show that DNA persistence length strongly depends on temp. and accounting for this dependence is important in quant. comparison between exptl. results obtained at different temps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM61PoCxbJPrVg90H21EOLACvtfcHk0lhiIiJQEjAizQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXis1Chsbs%253D&amp;md5=2e1a5b0503af764b4b78de73ccbe463c</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq932&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq932%26sid%3Dliteratum%253Aachs%26aulast%3DGeggier%26aufirst%3DS.%26aulast%3DKotlyar%26aufirst%3DA.%26aulast%3DVologodskii%26aufirst%3DA.%26atitle%3DTemperature%2520dependence%2520of%2520DNA%2520persistence%2520length%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26issue%3D4%26spage%3D1419%26epage%3D1426%26doi%3D10.1093%2Fnar%2Fgkq932" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of Δ-[Ru(bpy)2dppz]2+ bound to mismatched DNA reveals side-by-side metalloinsertion and intercalation</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">620</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.1375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnchem.1375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=22824892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC38XotlCks7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2012&amp;pages=615-620&amp;issue=8&amp;author=H.+Songauthor=J.+T.+Kaiserauthor=J.+K.+Barton&amp;title=Crystal+structure+of+%CE%94-%5BRu%28bpy%292dppz%5D2%2B+bound+to+mismatched+DNA+reveals+side-by-side+metalloinsertion+and+intercalation&amp;doi=10.1038%2Fnchem.1375" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of Δ-[Ru(bpy)2dppz]2+ bound to mismatched DNA reveals side-by-side metalloinsertion and intercalation</span></div><div class="casAuthors">Song, Hang; Kaiser, Jens T.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">615-620</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">DNA mismatches represent a novel target in the development of diagnostics and therapeutics for cancer, because deficiencies in DNA mismatch repair are implicated in cancers, and cells that are repair-deficient show a high frequency of mismatches.  Metal complexes with bulky intercalating ligands serve as probes for DNA mismatches.  Here, we report the high-resoln. (0.92 Å) crystal structure of the ruthenium light switch' complex Δ-[Ru(bpy)2dppz]2+ (bpy = 2,2'-bipyridine and dppz = dipyridophenazine), which is known to show luminescence on binding to duplex DNA, bound to both mismatched and well-matched sites in the oligonucleotide 5'-(dCGGAAATTACCG)2-3' (underline denotes AA mismatches).  Two crystallog. independent views reveal that the complex binds mismatches through metalloinsertion, ejecting both mispaired adenosines.  Addnl. ruthenium complexes are intercalated at well-matched sites, creating an array of complexes in the minor groove stabilized by stacking interactions between bpy ligands and extruded adenosines.  This structure attests to the generality of metalloinsertion and metallointercalation as DNA binding modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB4v1FFhXYFrVg90H21EOLACvtfcHk0lhiIiJQEjAizQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XotlCks7c%253D&amp;md5=72f10ab4bbbcc7b277bdc1e4b4bb95ef</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1375&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1375%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DH.%26aulast%3DKaiser%26aufirst%3DJ.%2BT.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DCrystal%2520structure%2520of%2520%25CE%2594-%255BRu%2528bpy%25292dppz%255D2%252B%2520bound%2520to%2520mismatched%2520DNA%2520reveals%2520side-by-side%2520metalloinsertion%2520and%2520intercalation%26jtitle%3DNat.%2520Chem.%26date%3D2012%26volume%3D4%26issue%3D8%26spage%3D615%26epage%3D620%26doi%3D10.1038%2Fnchem.1375" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gorodetsky, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Electrochemistry using self-assembled DNA monolayers on highly oriented pyrolytic graphite</span>. <i>Langmuir</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">7917</span>– <span class="NLM_lpage">7922</span>, <span class="refDoi">&nbsp;DOI: 10.1021/la0611054</span> </span><div class="citationLinks">[<a href="/doi/10.1021/la0611054" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnt1Ggur4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=2006&amp;pages=7917-7922&amp;author=A.+A.+Gorodetskyauthor=J.+K.+Barton&amp;title=Electrochemistry+using+self-assembled+DNA+monolayers+on+highly+oriented+pyrolytic+graphite&amp;doi=10.1021%2Fla0611054" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Electrochemistry Using Self-Assembled DNA Monolayers on Highly Oriented Pyrolytic Graphite</span></div><div class="casAuthors">Gorodetsky, Alon A.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Langmuir</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7917-7922</span>CODEN:
                <span class="NLM_cas:coden">LANGD5</span>;
        ISSN:<span class="NLM_cas:issn">0743-7463</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Duplex DNA functionalized with pyrene has been utilized to fabricate DNA-modified electrodes on highly oriented pyrolytic graphite (HOPG).  Films have been characterized using AFM and radioactive labeling as well as electrochem.  The data obtained are consistent with a close-packed structure in the film with helixes oriented in a nearly upright orientation, as seen earlier with the fabrication of thiol-tethered duplexes on gold.  Also as on gold, the authors observe the redn. of DNA-bound intercalators in a DNA-mediated reaction.  The redn. of the intercalator is attenuated in the presence of the single-base mismatches, CA and GT, independent of the sequence compn. of the oligonucleotide.  This sensitivity to single-base mismatches is enhanced when methylene blue redn. is coupled in an electrocatalytic cycle with ferricyanide.  The extended potential range afforded by the HOPG surface has allowed the authors also to investigate the electrochem. of previously inaccessible metallointercalators, Ru(bpy)2dppz2+ and Os(phen)2dppz2+, at the DNA-modified HOPG surface.  These results support the application of DNA-modified HOPG as a convenient and reproducible surface for electrochem. DNA sensors using DNA-mediated charge transport.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra9MldP0UdbLVg90H21EOLACvtfcHk0lhiIiJQEjAizQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28Xnt1Ggur4%253D&amp;md5=f530413f654439e1b031a0c4d624d008</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fla0611054&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fla0611054%26sid%3Dliteratum%253Aachs%26aulast%3DGorodetsky%26aufirst%3DA.%2BA.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DElectrochemistry%2520using%2520self-assembled%2520DNA%2520monolayers%2520on%2520highly%2520oriented%2520pyrolytic%2520graphite%26jtitle%3DLangmuir%26date%3D2006%26volume%3D22%26spage%3D7917%26epage%3D7922%26doi%3D10.1021%2Fla0611054" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jennette, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippard, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliades, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, W. R.</span></span> <span> </span><span class="NLM_article-title">Metallointercalation Reagents. 2-Hydroxyethanethiolato(2,2′,2″-terpyridine)-platinum(II) Monocation Binds Strongly to DNA By Intercalation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>71</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">3839</span>– <span class="NLM_lpage">3843</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.71.10.3839</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1073%2Fpnas.71.10.3839" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=71&amp;publication_year=1974&amp;pages=3839-3843&amp;issue=10&amp;author=K.+W.+Jennetteauthor=S.+J.+Lippardauthor=G.+A.+Vassiliadesauthor=W.+R.+Bauer&amp;title=Metallointercalation+Reagents.+2-Hydroxyethanethiolato%282%2C2%E2%80%B2%2C2%E2%80%B3-terpyridine%29-platinum%28II%29+Monocation+Binds+Strongly+to+DNA+By+Intercalation&amp;doi=10.1073%2Fpnas.71.10.3839" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1073%2Fpnas.71.10.3839&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.71.10.3839%26sid%3Dliteratum%253Aachs%26aulast%3DJennette%26aufirst%3DK.%2BW.%26aulast%3DLippard%26aufirst%3DS.%2BJ.%26aulast%3DVassiliades%26aufirst%3DG.%2BA.%26aulast%3DBauer%26aufirst%3DW.%2BR.%26atitle%3DMetallointercalation%2520Reagents.%25202-Hydroxyethanethiolato%25282%252C2%25E2%2580%25B2%252C2%25E2%2580%25B3-terpyridine%2529-platinum%2528II%2529%2520Monocation%2520Binds%2520Strongly%2520to%2520DNA%2520By%2520Intercalation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1974%26volume%3D71%26issue%3D10%26spage%3D3839%26epage%3D3843%26doi%3D10.1073%2Fpnas.71.10.3839" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kielkopf, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erkkila, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Structure of a photoactive rhodium complex intercalated into DNA</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">121</span>, <span class="refDoi">&nbsp;DOI: 10.1038/72385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2F72385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10655613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtlSkt7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2000&amp;pages=117-121&amp;author=C.+L.+Kielkopfauthor=K.+E.+Erkkilaauthor=B.+P.+Hudsonauthor=J.+K.+Bartonauthor=D.+C.+Rees&amp;title=Structure+of+a+photoactive+rhodium+complex+intercalated+into+DNA&amp;doi=10.1038%2F72385" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a photoactive rhodium complex intercalated into DNA</span></div><div class="casAuthors">Kielkopf, Clara L.; Erkkila, Kathryn E.; Hudson, Brian P.; Barton, Jacqueline K.; Rees, Douglas C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">117-121</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Intercalating complexes of rhodium(III) are strong photo-oxidants that promote DNA strand cleavage or electron transfer through the double helix.  The 1.2 Å resoln. crystal structure of a sequence-specific rhodium intercalators bound to a DNA helix provides a rationale for the sequence specificity of rhodium intercalators.  It also explains how intercalation in the center of an oligonucleotide modifies DNA conformation.  The rhodium complex intercalates via the major groove where specific contacts are formed with the edges of the bases at the target site.  The phi ligand is deeply inserted into the DNA base pair stack.  The primary conformational change of the DNA is a doubling of the rise per residue, with no change in sugar pucker from B-form DNA.  Based upon the five crystallog. independent views of an intercalated DNA helix obsd. in this structure, the intercalator may be considered as an addnl. base pair with specific functional groups positioned in the major groove.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-pj5VKd-CNrVg90H21EOLACvtfcHk0lizWJTkJ7ob9A"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3cXhtlSkt7w%253D&amp;md5=f2609dd605f0fc9a1b6774cdc792b921</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2F72385&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F72385%26sid%3Dliteratum%253Aachs%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26aulast%3DErkkila%26aufirst%3DK.%2BE.%26aulast%3DHudson%26aufirst%3DB.%2BP.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DStructure%2520of%2520a%2520photoactive%2520rhodium%2520complex%2520intercalated%2520into%2520DNA%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2000%26volume%3D7%26spage%3D117%26epage%3D121%26doi%3D10.1038%2F72385" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blattner, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plunkett, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloch, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perna, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burland, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collado-Vides, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glasner, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rode, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayhew, G. F.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">The Complete Genome Sequence of Escherichia coli K-12</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>277</i></span> (<span class="NLM_issue">5331</span>),  <span class="NLM_fpage">1453</span>– <span class="NLM_lpage">1462</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.277.5331.1453</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.277.5331.1453" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=9278503" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK2sXlvVGnu78%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=277&amp;publication_year=1997&amp;pages=1453-1462&amp;issue=5331&amp;author=F.+R.+Blattnerauthor=G.+Plunkettauthor=C.+A.+Blochauthor=N.+T.+Pernaauthor=V.+Burlandauthor=M.+Rileyauthor=J.+Collado-Videsauthor=J.+D.+Glasnerauthor=C.+K.+Rodeauthor=G.+F.+Mayhew&amp;title=The+Complete+Genome+Sequence+of+Escherichia+coli+K-12&amp;doi=10.1126%2Fscience.277.5331.1453" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Complete genome sequence of Escherichia coli K-12</span></div><div class="casAuthors">Blattner, Frederick R.; Plunkett, Guy, III; Bloch, Craig A.; Perna, Nicole T.; Burland, Valerie; Riley, Monica; Collado-Vides, Julio; Glasner, Jeremy D.; Rode, Christopher K.; Mayhew, George F.; Gregor, Jason; Davis, Nelson Wayne; Kirkpatrick, Heather A.; Goeden, Michael A.; Rose, Debra J.; Mau, Bob; Shao, Ying</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5331</span>),
    <span class="NLM_cas:pages">1453-1462</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The 4,639,221-base pair sequence of Escherichia coli K-12 is presented.  Of 4288 protein-coding genes annotated, 38 percent have no attributed function.  Comparison with five other sequenced microbes reveals ubiquitous as well as narrowly distributed gene families; many families of similar genes within E. coli are also evident.  The largest family of paralogous proteins contains 80 ABC transporters.  The genome as a whole is strikingly organized with respect to the local direction of replication; guanines, oligonucleotides possibly related to replication and recombination, and most genes are so oriented.  The genome also contains insertion sequence (IS) elements, phage remnants, and many other patches of unusual compn. indicating genome plasticity through horizontal transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOJTBu6TS_trVg90H21EOLACvtfcHk0lizWJTkJ7ob9A"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2sXlvVGnu78%253D&amp;md5=a3578b591b92dd4dacc5d9c4add9a283</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1126%2Fscience.277.5331.1453&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.277.5331.1453%26sid%3Dliteratum%253Aachs%26aulast%3DBlattner%26aufirst%3DF.%2BR.%26aulast%3DPlunkett%26aufirst%3DG.%26aulast%3DBloch%26aufirst%3DC.%2BA.%26aulast%3DPerna%26aufirst%3DN.%2BT.%26aulast%3DBurland%26aufirst%3DV.%26aulast%3DRiley%26aufirst%3DM.%26aulast%3DCollado-Vides%26aufirst%3DJ.%26aulast%3DGlasner%26aufirst%3DJ.%2BD.%26aulast%3DRode%26aufirst%3DC.%2BK.%26aulast%3DMayhew%26aufirst%3DG.%2BF.%26atitle%3DThe%2520Complete%2520Genome%2520Sequence%2520of%2520Escherichia%2520coli%2520K-12%26jtitle%3DScience%26date%3D1997%26volume%3D277%26issue%3D5331%26spage%3D1453%26epage%3D1462%26doi%3D10.1126%2Fscience.277.5331.1453" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beratan, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naaman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldeck, D. H.</span></span> <span> </span><span class="NLM_article-title">Charge and spin transport through nucleic acids</span>. <i>Curr. Opin. Electrochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">175</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.coelec.2017.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.coelec.2017.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVGgsLfO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2017&amp;pages=175&amp;issue=1&amp;author=D.+N.+Beratanauthor=R.+Naamanauthor=D.+H.+Waldeck&amp;title=Charge+and+spin+transport+through+nucleic+acids&amp;doi=10.1016%2Fj.coelec.2017.08.017" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Charge and spin transport through nucleic acids</span></div><div class="casAuthors">Beratan, David N.; Naaman, Ron; Waldeck, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Electrochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">175-181</span>CODEN:
                <span class="NLM_cas:coden">COEUCY</span>;
        ISSN:<span class="NLM_cas:issn">2451-9111</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Recent developments in our understanding of charge and spin transport through nucleic acid duplexes are discussed.  Particular emphasis is placed on recent findings that point to the importance of nucleobase fluctuations, transient but extended length-scale electronic coherence, and chiral-induced spin selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjgI4Sow_robVg90H21EOLACvtfcHk0lj8l1Lk4KWrNw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhtVGgsLfO&amp;md5=ef6db6edb7b87ee34cf64f369253ffa4</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.coelec.2017.08.017&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coelec.2017.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DBeratan%26aufirst%3DD.%2BN.%26aulast%3DNaaman%26aufirst%3DR.%26aulast%3DWaldeck%26aufirst%3DD.%2BH.%26atitle%3DCharge%2520and%2520spin%2520transport%2520through%2520nucleic%2520acids%26jtitle%3DCurr.%2520Opin.%2520Electrochem.%26date%3D2017%26volume%3D4%26issue%3D1%26spage%3D175%26doi%3D10.1016%2Fj.coelec.2017.08.017" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, M. G.</span></span> <span> </span><span class="NLM_article-title">Electrochemistry of Methylene Blue Bound to a DNA-Modified Electrode</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">37</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bc960070o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bc960070o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK2sXps1OltQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=1997&amp;pages=31-37&amp;issue=1&amp;author=S.+O.+Kelleyauthor=J.+K.+Bartonauthor=N.+M.+Jacksonauthor=M.+G.+Hill&amp;title=Electrochemistry+of+Methylene+Blue+Bound+to+a+DNA-Modified+Electrode&amp;doi=10.1021%2Fbc960070o" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Electrochemistry of Methylene Blue Bound to a DNA-Modified Electrode</span></div><div class="casAuthors">Kelley, Shana O.; Barton, Jacqueline K.; Jackson, Nicole M.; Hill, Michael G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-37</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Gold surfaces have been derivatized with 15-base-pair double-stranded DNA oligonucleotides contg. a pendant 5' hexanethiol linker.  The electrochem. of intercalated methylene blue has been investigated at these modified electrodes.  Chronocoulometry, cyclic voltammetry, ellipsometry, and quantitation via 32P labeling are all consistent with a surface coverage of ≥75% with the DNA helixes stacked at an angle from the electrode surface.  Cyclic voltammetry at low methylene blue/duplex stoichiometries yields well-behaved surface waves with E° = -0.25 V (vs SCE), a value 0.03 V neg. of that in aq. soln.  A binding isotherm for methylene blue at an electrode derivatized with the double-stranded sequence 5' SH-(CH2)6-p-AGTACAGTCATCGCG 3' was obtained from coulometric titrns. and gave an affinity const. equal to 3.8(5) × 106 M-1 with a satn. value of 1.4(2) methylene blue intercalators per DNA duplex.  Taken together, these expts. support a model for the surface morphol. in which DNA duplexes are densely packed; methylene blue therefore reversibly binds to sites in the DNA that are close to the bulk soln.  Electrochem. at DNA-derivatized electrodes provides a valuable methodol. to examine DNA-bound redox reactions and may offer new insight into DNA-mediated electron transfers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRGWG2RWENQLVg90H21EOLACvtfcHk0lj8l1Lk4KWrNw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2sXps1OltQ%253D%253D&amp;md5=a118fc772c3ec4d406393a1e2223c9e3</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fbc960070o&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbc960070o%26sid%3Dliteratum%253Aachs%26aulast%3DKelley%26aufirst%3DS.%2BO.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26aulast%3DJackson%26aufirst%3DN.%2BM.%26aulast%3DHill%26aufirst%3DM.%2BG.%26atitle%3DElectrochemistry%2520of%2520Methylene%2520Blue%2520Bound%2520to%2520a%2520DNA-Modified%2520Electrode%26jtitle%3DBioconjugate%2520Chem.%26date%3D1997%26volume%3D8%26issue%3D1%26spage%3D31%26epage%3D37%26doi%3D10.1021%2Fbc960070o" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Long-Range Electron Transfer through DNA Films</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">945</span>, <span class="refDoi">&nbsp;DOI: 10.1002/(SICI)1521-3773(19990401)38:7&lt;941::AID-ANIE941&gt;3.0.CO;2-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1002%2F%28SICI%291521-3773%2819990401%2938%3A7%3C941%3A%3AAID-ANIE941%3E3.0.CO%3B2-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK1MXisFWqsrw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=38&amp;publication_year=1999&amp;pages=941-945&amp;issue=7&amp;author=S.+O.+Kelleyauthor=N.+M.+Jacksonauthor=M.+G.+Hillauthor=J.+K.+Barton&amp;title=Long-Range+Electron+Transfer+through+DNA+Films&amp;doi=10.1002%2F%28SICI%291521-3773%2819990401%2938%3A7%3C941%3A%3AAID-ANIE941%3E3.0.CO%3B2-7" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Long-range electron transfer through DNA films</span></div><div class="casAuthors">Kelley, Shana O.; Jackson, Nicole M.; Hill, Michael G.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-945</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">In an effort to investigate DNA-mediated electron transfer involving ground-state reactants, the authors studied redox-active intercalators bound at discrete sites within the individual helixes of a DNA monolayer on gold.  To investigate charge transduction through DNA as a function of distance, the authors have site-specifically cross-linked a redox-active intercalator, daunomycin (DM), into the DNA characterized.  The site of intercalation was controlled in the duplexes by incorporating a single guanine-cytosine (GC) base step in otherwise adenine-thymine (AT) or inosine-cytosine (IC) sequences; as DM requires the N2 atom of guanine for covalent crosslinking, the intercalator is constrained to these positions.  Moving the GC step along the duplex therefore provided a systematic variation in the location of the DM-binding site relative to the thiol-terminated linker.  Crosslinking DM to the DNA in soln., and then depositing the labeled duplexes onto gold, afforded a series of films in which the intercalator was linked quant. at a known sepn. from the electrode surface.  Here, long-range charge transport through DNA-modified films has been demonstrated in a series of structurally characterized assemblies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8SZO4Dz9OKrVg90H21EOLACvtfcHk0lj8l1Lk4KWrNw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1MXisFWqsrw%253D&amp;md5=3012e102a1211872e74c53d889d33e79</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990401%2938%3A7%3C941%3A%3AAID-ANIE941%3E3.0.CO%3B2-7&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990401%252938%253A7%253C941%253A%253AAID-ANIE941%253E3.0.CO%253B2-7%26sid%3Dliteratum%253Aachs%26aulast%3DKelley%26aufirst%3DS.%2BO.%26aulast%3DJackson%26aufirst%3DN.%2BM.%26aulast%3DHill%26aufirst%3DM.%2BG.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DLong-Range%2520Electron%2520Transfer%2520through%2520DNA%2520Films%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26issue%3D7%26spage%3D941%26epage%3D945%26doi%3D10.1002%2F%28SICI%291521-3773%2819990401%2938%3A7%3C941%3A%3AAID-ANIE941%3E3.0.CO%3B2-7" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slinker, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muren, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorodetsky, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Multiplexed DNA-Modified Electrodes</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2769</span>– <span class="NLM_lpage">2774</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja909915m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja909915m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2rurs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=132&amp;publication_year=2010&amp;pages=2769-2774&amp;issue=8&amp;author=J.+D.+Slinkerauthor=N.+B.+Murenauthor=A.+A.+Gorodetskyauthor=J.+K.+Barton&amp;title=Multiplexed+DNA-Modified+Electrodes&amp;doi=10.1021%2Fja909915m" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Multiplexed DNA-Modified Electrodes</span></div><div class="casAuthors">Slinker, Jason D.; Muren, Natalie B.; Gorodetsky, Alon A.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2769-2774</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the use of silicon chips with 16 DNA-modified electrodes (DME chips) utilizing DNA-mediated charge transport for multiplexed detection of DNA and DNA-binding protein targets.  Four DNA sequences were simultaneously distinguished on a single DME chip with 4-fold redundancy, including one incorporating a single base mismatch.  These chips also enabled investigation of the sequence-specific activity of the restriction enzyme Alu1.  DME chips supported dense DNA monolayer formation with high reproducibility, as confirmed by statistical comparison to com. available rod electrodes.  The working electrode areas on the chips were reduced to 10 μm in diam., revealing microelectrode behavior that is beneficial for high sensitivity and rapid kinetic anal.  These results illustrate how DME chips facilitate sensitive and selective detection of DNA and DNA-binding protein targets in a robust and internally standardized multiplexed format.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgnAATmqwzP7Vg90H21EOLACvtfcHk0lhKtCHoGyjoHw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2rurs%253D&amp;md5=840ee36ecfb09e3b4958c88075fb68cd</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fja909915m&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja909915m%26sid%3Dliteratum%253Aachs%26aulast%3DSlinker%26aufirst%3DJ.%2BD.%26aulast%3DMuren%26aufirst%3DN.%2BB.%26aulast%3DGorodetsky%26aufirst%3DA.%2BA.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DMultiplexed%2520DNA-Modified%2520Electrodes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26issue%3D8%26spage%3D2769%26epage%3D2774%26doi%3D10.1021%2Fja909915m" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boon, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Electrochemistry as a Probe of Base Pair Stacking in A-, B-, and Z-Form DNA</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1140</span>– <span class="NLM_lpage">1147</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bc034139l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bc034139l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotlaru7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=2003&amp;pages=1140-1147&amp;issue=6&amp;author=E.+M.+Boonauthor=J.+K.+Barton&amp;title=DNA+Electrochemistry+as+a+Probe+of+Base+Pair+Stacking+in+A-%2C+B-%2C+and+Z-Form+DNA&amp;doi=10.1021%2Fbc034139l" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">DNA Electrochemistry as a Probe of Base Pair Stacking in A-, B-, and Z-Form DNA</span></div><div class="casAuthors">Boon, Elizabeth M.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1140-1147</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA-mediated charge transport (CT) chem. is sensitive to DNA structure and base pair stacking.  In an electrochem. assay based upon DNA CT, DNA-modified electrode surfaces are used to examine the electrochem. redn. of methylene blue (MB), a small mol. that binds to the DNA film by intercalation.  Here electrochem. the authors probe CT in the three primary conformations of double-stranded nucleic acids, A-, B-, and Z-form DNA.  The A-form is examd. in the context of a DNA/RNA hybrid duplex and Z-DNA, in duplexes contg. d(mCG)8 sequences at high Mg2+ concns.  Both A- and B-DNA support efficient DNA CT as measured by MB redn. in the DNA film; a lower level of redn. is evident with the Z-form film. Furthermore, mismatches incorporated into A-form duplexes, as in B-form duplexes, disrupt MB redn., thus providing a strategy for mutation detection through testing of RNA transcripts at DNA electrodes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcFHUxYm3uLLVg90H21EOLACvtfcHk0lhKtCHoGyjoHw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3sXotlaru7w%253D&amp;md5=20caa24d6728d0202d74762850229ab0</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fbc034139l&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbc034139l%26sid%3Dliteratum%253Aachs%26aulast%3DBoon%26aufirst%3DE.%2BM.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Electrochemistry%2520as%2520a%2520Probe%2520of%2520Base%2520Pair%2520Stacking%2520in%2520A-%252C%2520B-%252C%2520and%2520Z-Form%2520DNA%26jtitle%3DBioconjugate%2520Chem.%26date%3D2003%26volume%3D14%26issue%3D6%26spage%3D1140%26epage%3D1147%26doi%3D10.1021%2Fbc034139l" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zwang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, E. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">A Compass at Weak Magnetic Fields Using Thymine Dimer Repair</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">412</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscentsci.8b00008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.8b00008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjvVGnu7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2018&amp;pages=405-412&amp;issue=3&amp;author=T.+J.+Zwangauthor=E.+C.+M.+Tseauthor=D.+Zhongauthor=J.+K.+Barton&amp;title=A+Compass+at+Weak+Magnetic+Fields+Using+Thymine+Dimer+Repair&amp;doi=10.1021%2Facscentsci.8b00008" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">A Compass at Weak Magnetic Fields Using Thymine Dimer Repair</span></div><div class="casAuthors">Zwang, Theodore J.; Tse, Edmund C. M.; Zhong, Dongping; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-412</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">How birds sense the variations in Earth's magnetic field for navigation is poorly understood, although cryptochromes, proteins homologous to photolyases, have been proposed to participate in this magnetic sensing.  Here, in electrochem. studies with an applied magnetic field, we monitor the repair of cyclobutane pyrimidine dimer lesions in duplex DNA by photolyase, mutants of photolyase, and by a modified cryptochrome.  We found that the yield of dimer repair was dependent on the strength and angle of the applied magnetic field even when using magnetic fields weaker than 1 G.  This high sensitivity to weak magnetic fields depended upon a fast radical pair reaction on the thymines leading to repair.  These data illustrate chem. how cyclobutane pyrimidine dimer repair may be used in a biol. compass informed by variations in Earth's magnetic field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzGMHpXBfW7bVg90H21EOLACvtfcHk0lhKtCHoGyjoHw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXjvVGnu7w%253D&amp;md5=45f72e4d218ccda1f9a723c3978ce2aa</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.8b00008&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.8b00008%26sid%3Dliteratum%253Aachs%26aulast%3DZwang%26aufirst%3DT.%2BJ.%26aulast%3DTse%26aufirst%3DE.%2BC.%2BM.%26aulast%3DZhong%26aufirst%3DD.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DA%2520Compass%2520at%2520Weak%2520Magnetic%2520Fields%2520Using%2520Thymine%2520Dimer%2520Repair%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2018%26volume%3D4%26issue%3D3%26spage%3D405%26epage%3D412%26doi%3D10.1021%2Facscentsci.8b00008" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></div><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 19 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Aaron M. Fleming, Shereen A. Howpay Manage, <span class="NLM_string-name hlFld-ContribAuthor">Cynthia J. Burrows</span>. </span><span class="cited-content_cbyCitation_article-title">Binding of AP Endonuclease-1 to G-Quadruplex DNA Depends on the N-Terminal Domain, Mg2+, and Ionic Strength. </span><span class="cited-content_cbyCitation_journal-name">ACS Bio &amp; Med Chem Au</span><span> <strong>2021,</strong> <em>1 </em>
                                    (1)
                                     , 44-56. <a href="https://doi.org/10.1021/acsbiomedchemau.1c00031" title="DOI URL">https://doi.org/10.1021/acsbiomedchemau.1c00031</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsbiomedchemau.1c00031&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsbiomedchemau.1c00031%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Bio%2520%2526%2520Med%2520Chem%2520Au%26atitle%3DBinding%252Bof%252BAP%252BEndonuclease-1%252Bto%252BG-Quadruplex%252BDNA%252BDepends%252Bon%252Bthe%252BN-Terminal%252BDomain%25252C%252BMg2%25252B%25252C%252Band%252BIonic%252BStrength%26aulast%3DFleming%26aufirst%3DAaron%2BM.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D16082021%26date%3D14102021%26date%3D13102021%26date%3D29102021%26volume%3D1%26issue%3D1%26spage%3D44%26epage%3D56" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Miguel N. Pinto, Josy ter Beek, Levi A. Ekanger, Erik Johansson, <span class="NLM_string-name hlFld-ContribAuthor">Jacqueline K. Barton</span>. </span><span class="cited-content_cbyCitation_article-title">The [4Fe4S] Cluster of Yeast DNA Polymerase ε Is Redox Active and Can Undergo DNA-Mediated Signaling. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (39)
                                     , 16147-16153. <a href="https://doi.org/10.1021/jacs.1c07150" title="DOI URL">https://doi.org/10.1021/jacs.1c07150</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.1c07150&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.1c07150%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DThe%252B%25255B4Fe4S%25255D%252BCluster%252Bof%252BYeast%252BDNA%252BPolymerase%252B%2525CE%2525B5%252BIs%252BRedox%252BActive%252Band%252BCan%252BUndergo%252BDNA-Mediated%252BSignaling%26aulast%3DPinto%26aufirst%3DMiguel%2BN.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D09072021%26date%3D24092021%26volume%3D143%26issue%3D39%26spage%3D16147%26epage%3D16153" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hanie Yousefi, Alam Mahmud, Dingran Chang, Jagotamoy Das, Surath Gomis, Jenise B. Chen, Hansen Wang, Terek Been, Lily Yip, Eric Coomes, Zhijie Li, Samira Mubareka, Allison McGeer, Natasha Christie, Scott Gray-Owen, Alan Cochrane, James M. Rini, Edward H. Sargent, <span class="NLM_string-name hlFld-ContribAuthor">Shana O. Kelley</span>. </span><span class="cited-content_cbyCitation_article-title">Detection of SARS-CoV-2 Viral Particles Using Direct, Reagent-Free Electrochemical Sensing. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (4)
                                     , 1722-1727. <a href="https://doi.org/10.1021/jacs.0c10810" title="DOI URL">https://doi.org/10.1021/jacs.0c10810</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c10810&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c10810%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DDetection%252Bof%252BSARS-CoV-2%252BViral%252BParticles%252BUsing%252BDirect%25252C%252BReagent-Free%252BElectrochemical%252BSensing%26aulast%3DYousefi%26aufirst%3DHanie%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D12102020%26date%3D22012021%26volume%3D143%26issue%3D4%26spage%3D1722%26epage%3D1727" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Aaron M. Fleming, Judy Zhu, Manuel Jara-Espejo, <span class="NLM_string-name hlFld-ContribAuthor">Cynthia J. Burrows</span>. </span><span class="cited-content_cbyCitation_article-title">Cruciform DNA Sequences in Gene Promoters Can Impact Transcription upon Oxidative Modification of 2′-Deoxyguanosine. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2020,</strong> <em>59 </em>
                                    (28)
                                     , 2616-2626. <a href="https://doi.org/10.1021/acs.biochem.0c00387" title="DOI URL">https://doi.org/10.1021/acs.biochem.0c00387</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.0c00387&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.0c00387%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DCruciform%252BDNA%252BSequences%252Bin%252BGene%252BPromoters%252BCan%252BImpact%252BTranscription%252Bupon%252BOxidative%252BModification%252Bof%252B2%2525E2%252580%2525B2-Deoxyguanosine%26aulast%3DFleming%26aufirst%3DAaron%2BM.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D09052020%26date%3D20062020%26date%3D25062020%26date%3D22062020%26volume%3D59%26issue%3D28%26spage%3D2616%26epage%3D2626" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Aaron M. Fleming, <span class="NLM_string-name hlFld-ContribAuthor">Cynthia J. Burrows</span>. </span><span class="cited-content_cbyCitation_article-title">Interplay of Guanine Oxidation and G-Quadruplex Folding in Gene Promoters. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2020,</strong> <em>142 </em>
                                    (3)
                                     , 1115-1136. <a href="https://doi.org/10.1021/jacs.9b11050" title="DOI URL">https://doi.org/10.1021/jacs.9b11050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.9b11050&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.9b11050%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DInterplay%252Bof%252BGuanine%252BOxidation%252Band%252BG-Quadruplex%252BFolding%252Bin%252BGene%252BPromoters%26aulast%3DFleming%26aufirst%3DAaron%2BM.%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D14102019%26date%3D09012020%26date%3D27122019%26volume%3D142%26issue%3D3%26spage%3D1115%26epage%3D1136" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Aleem Syed, <span class="NLM_string-name hlFld-ContribAuthor">John A. Tainer</span>. </span><span class="cited-content_cbyCitation_article-title">Charge Transport Communication through DNA by Protein Fe–S Clusters: How Far Is Not Too Far?. </span><span class="cited-content_cbyCitation_journal-name">ACS Central Science</span><span> <strong>2019,</strong> <em>5 </em>
                                    (1)
                                     , 7-9. <a href="https://doi.org/10.1021/acscentsci.8b00909" title="DOI URL">https://doi.org/10.1021/acscentsci.8b00909</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscentsci.8b00909&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscentsci.8b00909%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Central%2520Science%26atitle%3DCharge%252BTransport%252BCommunication%252Bthrough%252BDNA%252Bby%252BProtein%252BFe%2525E2%252580%252593S%252BClusters%25253A%252BHow%252BFar%252BIs%252BNot%252BToo%252BFar%25253F%26aulast%3DSyed%26aufirst%3DAleem%26date%3D2019%26date%3D2019%26date%3D11012019%26volume%3D5%26issue%3D1%26spage%3D7%26epage%3D9" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aaron M.  Fleming</span>, <span class="hlFld-ContribAuthor ">Cynthia J.  Burrows</span>. </span><span class="cited-content_cbyCitation_article-title">Chemistry of ROS-mediated oxidation to the guanine base in DNA and its biological consequences. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Radiation Biology</span><span> <strong>2022,</strong> <em>98 </em>
                                    (3)
                                     , 452-460. <a href="https://doi.org/10.1080/09553002.2021.2003464" title="DOI URL">https://doi.org/10.1080/09553002.2021.2003464</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/09553002.2021.2003464&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F09553002.2021.2003464%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Radiation%2520Biology%26atitle%3DChemistry%252Bof%252BROS-mediated%252Boxidation%252Bto%252Bthe%252Bguanine%252Bbase%252Bin%252BDNA%252Band%252Bits%252Bbiological%252Bconsequences%26aulast%3DFleming%26aufirst%3DAaron%2BM.%26date%3D2022%26date%3D2021%26volume%3D98%26issue%3D3%26spage%3D452%26epage%3D460" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maryia  Drobysh</span>, <span class="hlFld-ContribAuthor ">Almira  Ramanaviciene</span>, <span class="hlFld-ContribAuthor ">Roman  Viter</span>, <span class="hlFld-ContribAuthor ">Chien-Fu  Chen</span>, <span class="hlFld-ContribAuthor ">Urte  Samukaite-Bubniene</span>, <span class="hlFld-ContribAuthor ">Vilma  Ratautaite</span>, <span class="hlFld-ContribAuthor ">Arunas  Ramanavicius</span>. </span><span class="cited-content_cbyCitation_article-title">Biosensors for the Determination of SARS-CoV-2 Virus and Diagnosis of COVID-19 Infection. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2022,</strong> <em>23 </em>
                                    (2)
                                     , 666. <a href="https://doi.org/10.3390/ijms23020666" title="DOI URL">https://doi.org/10.3390/ijms23020666</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms23020666&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms23020666%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DBiosensors%252Bfor%252Bthe%252BDetermination%252Bof%252BSARS-CoV-2%252BVirus%252Band%252BDiagnosis%252Bof%252BCOVID-19%252BInfection%26aulast%3DDrobysh%26aufirst%3DMaryia%26date%3D2022%26date%3D2022%26volume%3D23%26issue%3D2%26spage%3D666" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shreanshi  Agrahari</span>, <span class="hlFld-ContribAuthor ">Ravindra  Kumar Gautam</span>, <span class="hlFld-ContribAuthor ">Ankit  Kumar Singh</span>, <span class="hlFld-ContribAuthor ">Ida  Tiwari</span>. </span><span class="cited-content_cbyCitation_article-title">Nanoscale materials-based hybrid frameworks modified electrochemical biosensors for early cancer diagnostics: An overview of current trends and challenges. </span><span class="cited-content_cbyCitation_journal-name">Microchemical Journal</span><span> <strong>2021,</strong> <em>68 </em>, 106980. <a href="https://doi.org/10.1016/j.microc.2021.106980" title="DOI URL">https://doi.org/10.1016/j.microc.2021.106980</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.microc.2021.106980&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.microc.2021.106980%26sid%3Dliteratum%253Aachs%26jtitle%3DMicrochemical%2520Journal%26atitle%3DNanoscale%252Bmaterials-based%252Bhybrid%252Bframeworks%252Bmodified%252Belectrochemical%252Bbiosensors%252Bfor%252Bearly%252Bcancer%252Bdiagnostics%25253A%252BAn%252Boverview%252Bof%252Bcurrent%252Btrends%252Band%252Bchallenges%26aulast%3DAgrahari%26aufirst%3DShreanshi%26date%3D2021%26volume%3D68%26spage%3D106980" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aaron M  Fleming</span>, <span class="hlFld-ContribAuthor ">Cynthia J  Burrows</span>. </span><span class="cited-content_cbyCitation_article-title">Oxidative stress-mediated epigenetic regulation by G-quadruplexes. </span><span class="cited-content_cbyCitation_journal-name">NAR Cancer</span><span> <strong>2021,</strong> <em>3 </em>
                                    (3)
                                     <a href="https://doi.org/10.1093/narcan/zcab038" title="DOI URL">https://doi.org/10.1093/narcan/zcab038</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/narcan/zcab038&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnarcan%2Fzcab038%26sid%3Dliteratum%253Aachs%26jtitle%3DNAR%2520Cancer%26atitle%3DOxidative%252Bstress-mediated%252Bepigenetic%252Bregulation%252Bby%252BG-quadruplexes%26aulast%3DFleming%26aufirst%3DAaron%2BM%26date%3D2021%26date%3D2021%26volume%3D3%26issue%3D3" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fung  Man Ngo</span>, <span class="hlFld-ContribAuthor ">Edmund C. M.  Tse</span>. </span><span class="cited-content_cbyCitation_article-title">Bioinorganic Platforms for Sensing, Biomimicry, and Energy Catalysis. </span><span class="cited-content_cbyCitation_journal-name">Chemistry Letters</span><span> <strong>2021,</strong> <em>50 </em>
                                    (5)
                                     , 974-986. <a href="https://doi.org/10.1246/cl.200875" title="DOI URL">https://doi.org/10.1246/cl.200875</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1246/cl.200875&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1246%2Fcl.200875%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520Letters%26atitle%3DBioinorganic%252BPlatforms%252Bfor%252BSensing%25252C%252BBiomimicry%25252C%252Band%252BEnergy%252BCatalysis%26aulast%3DMan%2BNgo%26aufirst%3DFung%26date%3D2021%26volume%3D50%26issue%3D5%26spage%3D974%26epage%3D986" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Susan E.  Tsutakawa</span>, <span class="hlFld-ContribAuthor ">Chi-Lin  Tsai</span>, <span class="hlFld-ContribAuthor ">Chunli  Yan</span>, <span class="hlFld-ContribAuthor ">Amer  Bralić</span>, <span class="hlFld-ContribAuthor ">Walter J.  Chazin</span>, <span class="hlFld-ContribAuthor ">Samir M.  Hamdan</span>, <span class="hlFld-ContribAuthor ">Orlando D.  Schärer</span>, <span class="hlFld-ContribAuthor ">Ivaylo  Ivanov</span>, <span class="hlFld-ContribAuthor ">John A.  Tainer</span>. </span><span class="cited-content_cbyCitation_article-title">Envisioning how the prototypic molecular machine TFIIH functions in transcription initiation and DNA repair. </span><span class="cited-content_cbyCitation_journal-name">DNA Repair</span><span> <strong>2020,</strong> <em>96 </em>, 102972. <a href="https://doi.org/10.1016/j.dnarep.2020.102972" title="DOI URL">https://doi.org/10.1016/j.dnarep.2020.102972</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.dnarep.2020.102972&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.dnarep.2020.102972%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520Repair%26atitle%3DEnvisioning%252Bhow%252Bthe%252Bprototypic%252Bmolecular%252Bmachine%252BTFIIH%252Bfunctions%252Bin%252Btranscription%252Binitiation%252Band%252BDNA%252Brepair%26aulast%3DTsutakawa%26aufirst%3DSusan%2BE.%26date%3D2020%26volume%3D96%26spage%3D102972" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">James B.  Derr</span>, <span class="hlFld-ContribAuthor ">Jesse  Tamayo</span>, <span class="hlFld-ContribAuthor ">John A.  Clark</span>, <span class="hlFld-ContribAuthor ">Maryann  Morales</span>, <span class="hlFld-ContribAuthor ">Maximillian F.  Mayther</span>, <span class="hlFld-ContribAuthor ">Eli M.  Espinoza</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Rybicka-Jasińska</span>, <span class="hlFld-ContribAuthor ">Valentine I.  Vullev</span>. </span><span class="cited-content_cbyCitation_article-title">Multifaceted aspects of charge transfer. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2020,</strong> <em>22 </em>
                                    (38)
                                     , 21583-21629. <a href="https://doi.org/10.1039/D0CP01556C" title="DOI URL">https://doi.org/10.1039/D0CP01556C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CP01556C&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CP01556C%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DMultifaceted%252Baspects%252Bof%252Bcharge%252Btransfer%26aulast%3DDerr%26aufirst%3DJames%2BB.%26date%3D2020%26date%3D2020%26volume%3D22%26issue%3D38%26spage%3D21583%26epage%3D21629" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abhishek  Aggarwal</span>, <span class="hlFld-ContribAuthor ">Saientan  Bag</span>, <span class="hlFld-ContribAuthor ">Ravindra  Venkatramani</span>, <span class="hlFld-ContribAuthor ">Manish  Jain</span>, <span class="hlFld-ContribAuthor ">Prabal K.  Maiti</span>. </span><span class="cited-content_cbyCitation_article-title">Multiscale modelling reveals higher charge transport efficiencies of DNA relative to RNA independent of mechanism. </span><span class="cited-content_cbyCitation_journal-name">Nanoscale</span><span> <strong>2020,</strong> <em>12 </em>
                                    (36)
                                     , 18750-18760. <a href="https://doi.org/10.1039/D0NR02382E" title="DOI URL">https://doi.org/10.1039/D0NR02382E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0NR02382E&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0NR02382E%26sid%3Dliteratum%253Aachs%26jtitle%3DNanoscale%26atitle%3DMultiscale%252Bmodelling%252Breveals%252Bhigher%252Bcharge%252Btransport%252Befficiencies%252Bof%252BDNA%252Brelative%252Bto%252BRNA%252Bindependent%252Bof%252Bmechanism%26aulast%3DAggarwal%26aufirst%3DAbhishek%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D36%26spage%3D18750%26epage%3D18760" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aaron M.  Fleming</span>, <span class="hlFld-ContribAuthor ">Cynthia J.  Burrows</span>. </span><span class="cited-content_cbyCitation_article-title">On the irrelevancy of hydroxyl radical to DNA damage from oxidative stress and implications for epigenetics. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2020,</strong> <em>49 </em>
                                    (18)
                                     , 6524-6528. <a href="https://doi.org/10.1039/D0CS00579G" title="DOI URL">https://doi.org/10.1039/D0CS00579G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CS00579G&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CS00579G%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DOn%252Bthe%252Birrelevancy%252Bof%252Bhydroxyl%252Bradical%252Bto%252BDNA%252Bdamage%252Bfrom%252Boxidative%252Bstress%252Band%252Bimplications%252Bfor%252Bepigenetics%26aulast%3DFleming%26aufirst%3DAaron%2BM.%26date%3D2020%26date%3D2020%26volume%3D49%26issue%3D18%26spage%3D6524%26epage%3D6528" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Scott H.  Saunders</span>, <span class="hlFld-ContribAuthor ">Edmund C.M.  Tse</span>, <span class="hlFld-ContribAuthor ">Matthew D.  Yates</span>, <span class="hlFld-ContribAuthor ">Fernanda Jiménez  Otero</span>, <span class="hlFld-ContribAuthor ">Scott A.  Trammell</span>, <span class="hlFld-ContribAuthor ">Eric D.A.  Stemp</span>, <span class="hlFld-ContribAuthor ">Jacqueline K.  Barton</span>, <span class="hlFld-ContribAuthor ">Leonard M.  Tender</span>, <span class="hlFld-ContribAuthor ">Dianne K.  Newman</span>. </span><span class="cited-content_cbyCitation_article-title">Extracellular DNA Promotes Efficient Extracellular Electron Transfer by Pyocyanin in Pseudomonas aeruginosa Biofilms. </span><span class="cited-content_cbyCitation_journal-name">Cell</span><span> <strong>2020,</strong> <em>182 </em>
                                    (4)
                                     , 919-932.e19. <a href="https://doi.org/10.1016/j.cell.2020.07.006" title="DOI URL">https://doi.org/10.1016/j.cell.2020.07.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cell.2020.07.006&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cell.2020.07.006%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%26atitle%3DExtracellular%252BDNA%252BPromotes%252BEfficient%252BExtracellular%252BElectron%252BTransfer%252Bby%252BPyocyanin%252Bin%252BPseudomonas%252Baeruginosa%252BBiofilms%26aulast%3DSaunders%26aufirst%3DScott%2BH.%26date%3D2020%26volume%3D182%26issue%3D4%26spage%3D919%26epage%3D932.e19" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alex-Adrian  Farcaș</span>, <span class="hlFld-ContribAuthor ">Attila  Bende</span>. </span><span class="cited-content_cbyCitation_article-title">The influence of monovalent and divalent metal cations on the stability of the DNA-protein interaction in the nucleosome core particle. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 269-290. <a href="https://doi.org/10.1016/bs.aiq.2020.05.001" title="DOI URL">https://doi.org/10.1016/bs.aiq.2020.05.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.aiq.2020.05.001&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.aiq.2020.05.001%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252Binfluence%252Bof%252Bmonovalent%252Band%252Bdivalent%252Bmetal%252Bcations%252Bon%252Bthe%252Bstability%252Bof%252Bthe%252BDNA-protein%252Binteraction%252Bin%252Bthe%252Bnucleosome%252Bcore%252Bparticle%26aulast%3DFarca%25C8%2599%26aufirst%3DAlex-Adrian%26date%3D2020%26spage%3D269%26epage%3D290%26pub%3DElsevier%26atitle%3DChemical%252BPhysics%252Band%252BQuantum%252BChemistry%26date%3D2020%26volume%3D81" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tian  Zeng</span>, <span class="hlFld-ContribAuthor ">Heng-Liang  Wu</span>, <span class="hlFld-ContribAuthor ">Ying  Li</span>, <span class="hlFld-ContribAuthor ">Edmund C.M.  Tse</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Barile</span>. </span><span class="cited-content_cbyCitation_article-title">Physical and electrochemical characterization of a Cu-based oxygen reduction electrocatalyst inside and outside a lipid membrane with controlled proton transfer kinetics. </span><span class="cited-content_cbyCitation_journal-name">Electrochimica Acta</span><span> <strong>2019,</strong> <em>320 </em>, 134611. <a href="https://doi.org/10.1016/j.electacta.2019.134611" title="DOI URL">https://doi.org/10.1016/j.electacta.2019.134611</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.electacta.2019.134611&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.electacta.2019.134611%26sid%3Dliteratum%253Aachs%26jtitle%3DElectrochimica%2520Acta%26atitle%3DPhysical%252Band%252Belectrochemical%252Bcharacterization%252Bof%252Ba%252BCu-based%252Boxygen%252Breduction%252Belectrocatalyst%252Binside%252Band%252Boutside%252Ba%252Blipid%252Bmembrane%252Bwith%252Bcontrolled%252Bproton%252Btransfer%252Bkinetics%26aulast%3DZeng%26aufirst%3DTian%26date%3D2019%26volume%3D320%26spage%3D134611" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sergio  Kogikoski</span>, <span class="hlFld-ContribAuthor ">Waldemir J.  Paschoalino</span>, <span class="hlFld-ContribAuthor ">Lory  Cantelli</span>, <span class="hlFld-ContribAuthor ">Wilgner  Silva</span>, <span class="hlFld-ContribAuthor ">Lauro T.  Kubota</span>. </span><span class="cited-content_cbyCitation_article-title">Electrochemical sensing based on DNA nanotechnology. </span><span class="cited-content_cbyCitation_journal-name">TrAC Trends in Analytical Chemistry</span><span> <strong>2019,</strong> <em>118 </em>, 597-605. <a href="https://doi.org/10.1016/j.trac.2019.06.021" title="DOI URL">https://doi.org/10.1016/j.trac.2019.06.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.trac.2019.06.021&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.trac.2019.06.021%26sid%3Dliteratum%253Aachs%26jtitle%3DTrAC%2520Trends%2520in%2520Analytical%2520Chemistry%26atitle%3DElectrochemical%252Bsensing%252Bbased%252Bon%252BDNA%252Bnanotechnology%26aulast%3DKogikoski%26aufirst%3DSergio%26date%3D2019%26volume%3D118%26spage%3D597%26epage%3D605" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li></ol></div></div></div></div><div class="ui-resizable-handle ui-resizable-e" style="z-index: 90;"></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class=""><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0007.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0001.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Quantifying the DNA charge transport length. (a) Model showing the implications of DNA CT length on the redox signaling capability between [4Fe4S] repair proteins and subsequently the lesion search process. (b) AFM images of two linear dsDNA. The one on the top is in the regime where the DNA length is shorter than the CT length, while the one on the bottom is in the regime where the DNA length is longer than the CT length.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0006.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Constructing Linear Double-Stranded DNA of Various Lengths by (a) Plasmid Excision Using SacI (Magenta) and (b) PCR Amplification Methods<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"></p><p class="last"><span class="fn-label"><sup>a</sup></span>The detailed protocols to prepare DNA duplexes of different lengths are presented in the <a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">SI</a>, Methods section. See <a href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf" class="ext-link">Table S1</a> for the primer sequences and plasmid templates used in the PCR amplification steps.</p><p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0002.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Measuring DNA-binding density of wild-type EndoIII at ambient temperature using atomic force microscopy. A sharp change in binding slope is observed at 3509 base pairs, or about 1.2 μm, that suggests the presence of two regimes for protein binding, one within the range for DNA CT (regime 1, blue), one outside of the range for CT (regime 2, black). 1310 DNA duplexes were analyzed from four independent preparations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0003.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Measuring DNA-binding density of Y82A EndoIII mutant at ambient temperature using atomic force microscopy. No change in binding slope as a function of DNA lengths is observed, indicating the absence of DNA CT. As Y82A does not undergo redox signaling with each other via DNA CT, the DNA-binding affinity of Y82A is independent of each other. 272 DNA duplexes were analyzed from four independent preparations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0004.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Measuring DNA-binding density of wild-type EndoIII at 4 °C using atomic force microscopy. A sharp change in binding slope occurs at 5383 base pairs, or about 1.8 μm, indicating that the DNA CT length observed at 4 °C is significantly longer than that at RT. 151 DNA duplexes were analyzed from four independent preparations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/acscentsci.8b00566/asset/images/medium/oc-2018-005666_0005.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Measuring DNA-binding density of wild-type EndoIII in the presence of [Ru(phen)<sub>2</sub>dppz]Cl<sub>2</sub> (50 μM) at ambient temperature using atomic force microscopy. A sharp change in binding slope occurs at 5399 base pairs, or about 1.8 μm, suggesting that metallointercalators lengthen the DNA CT length observed. 449 DNA duplexes were analyzed from four independent preparations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.8b00566/asset/images/large/oc-2018-005666_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.8b00566&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83431" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83431" class="dropBlock__holder"></div></div></div></div></div><hr></div><p class="rightTab-references-count">
            This article references 73 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartek, J.</span></span> <span> </span><span class="NLM_article-title">The DNA-damage Response in Human Biology and Disease</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>461</i></span> (<span class="NLM_issue">7267</span>),  <span class="NLM_fpage">1071</span>– <span class="NLM_lpage">1078</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature08467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnature08467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=19847258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGktLrK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=461&amp;publication_year=2009&amp;pages=1071-1078&amp;issue=7267&amp;author=S.+P.+Jacksonauthor=J.+Bartek&amp;title=The+DNA-damage+Response+in+Human+Biology+and+Disease&amp;doi=10.1038%2Fnature08467" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The DNA-damage response in human biology and disease</span></div><div class="casAuthors">Jackson, Stephen P.; Bartek, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7267</span>),
    <span class="NLM_cas:pages">1071-1078</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The prime objective for every life form is to deliver its genetic material, intact and unchanged, to the next generation.  This must be achieved despite const. assaults by endogenous and environmental agents on the DNA.  To counter this threat, life has evolved several systems to detect DNA damage, signal its presence and mediate its repair.  Such responses, which have an impact on a wide range of cellular events, are biol. significant because they prevent diverse human diseases.  Our improving understanding of DNA-damage responses is providing new avenues for disease management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ0Oyp6nX9V7Vg90H21EOLACvtfcHk0lhnKqNODUNZQg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGktLrK&amp;md5=cca7cfec66852124695ebeb515f255a5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature08467&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08467%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DS.%2BP.%26aulast%3DBartek%26aufirst%3DJ.%26atitle%3DThe%2520DNA-damage%2520Response%2520in%2520Human%2520Biology%2520and%2520Disease%26jtitle%3DNature%26date%3D2009%26volume%3D461%26issue%3D7267%26spage%3D1071%26epage%3D1078%26doi%3D10.1038%2Fnature08467" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, D. E.</span></span> <span> </span><span class="NLM_article-title">Repair of Endogenous DNA Damage</span>. <i>Cold Spring Harbor Symp. Quant. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">134</span>, <span class="refDoi">&nbsp;DOI: 10.1101/sqb.2000.65.127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1101%2Fsqb.2000.65.127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=12760027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlt1ejtr4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=65&amp;publication_year=2000&amp;pages=127-134&amp;author=T.+Lindahlauthor=D.+E.+Barnes&amp;title=Repair+of+Endogenous+DNA+Damage&amp;doi=10.1101%2Fsqb.2000.65.127" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Repair of endogenous DNA damage</span></div><div class="casAuthors">Lindahl, T.; Barnes, D. E.</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Symposia on Quantitative Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">127-133</span>CODEN:
                <span class="NLM_cas:coden">CSHSAZ</span>;
        ISSN:<span class="NLM_cas:issn">0091-7451</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review with refs. on how to correct an endogenous DNA damage in mammalian cells by the base excision repair (BER) pathways.  The different DNA glycosylases that are known to act on various forms of endogenously damaged DNA in mammalian cells are described.  The sequential transient protein-protein interactions occurring on the damaged DNA substrate during BER may promote the accuracy, speed, and specificity of the endogenous repair process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_nhlrmTM-i7Vg90H21EOLACvtfcHk0lhnKqNODUNZQg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3MXlt1ejtr4%253D&amp;md5=064b78d90329afaf07394c18ea589200</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1101%2Fsqb.2000.65.127&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fsqb.2000.65.127%26sid%3Dliteratum%253Aachs%26aulast%3DLindahl%26aufirst%3DT.%26aulast%3DBarnes%26aufirst%3DD.%2BE.%26atitle%3DRepair%2520of%2520Endogenous%2520DNA%2520Damage%26jtitle%3DCold%2520Spring%2520Harbor%2520Symp.%2520Quant.%2520Biol.%26date%3D2000%26volume%3D65%26spage%3D127%26epage%3D134%26doi%3D10.1101%2Fsqb.2000.65.127" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoeijmakers, J. H. J.</span></span> <span> </span><span class="NLM_article-title">Genome Maintenance Mechanisms for Preventing Cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>411</i></span> (<span class="NLM_issue">6835</span>),  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">374</span>, <span class="refDoi">&nbsp;DOI: 10.1038/35077232</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2F35077232" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=411&amp;publication_year=2001&amp;pages=366-374&amp;issue=6835&amp;author=J.+H.+J.+Hoeijmakers&amp;title=Genome+Maintenance+Mechanisms+for+Preventing+Cancer&amp;doi=10.1038%2F35077232" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2F35077232&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35077232%26sid%3Dliteratum%253Aachs%26aulast%3DHoeijmakers%26aufirst%3DJ.%2BH.%2BJ.%26atitle%3DGenome%2520Maintenance%2520Mechanisms%2520for%2520Preventing%2520Cancer%26jtitle%3DNature%26date%3D2001%26volume%3D411%26issue%3D6835%26spage%3D366%26epage%3D374%26doi%3D10.1038%2F35077232" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sancar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsey-Boltz, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unsal-Kacmaz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linn, S.</span></span> <span> </span><span class="NLM_article-title">Molecular Mechanisms of Mammalian DNA Repair and the DNA Damage Checkpoints</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">85</span>, <span class="refDoi">&nbsp;DOI: 10.1146/annurev.biochem.73.011303.073723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1146%2Fannurev.biochem.73.011303.073723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=15189136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmslagsrk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=73&amp;publication_year=2004&amp;pages=39-85&amp;author=A.+Sancarauthor=L.+A.+Lindsey-Boltzauthor=K.+Unsal-Kacmazauthor=S.+Linn&amp;title=Molecular+Mechanisms+of+Mammalian+DNA+Repair+and+the+DNA+Damage+Checkpoints&amp;doi=10.1146%2Fannurev.biochem.73.011303.073723" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms of mammalian DNA repair and the DNA damage checkpoints</span></div><div class="casAuthors">Sancar, Aziz; Lindsey-Boltz, Laura A.; Uensal-Kacmaz, Keziban; Linn, Stuart</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39-85</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  DNA damage is a relatively common event in the life of a cell and may lead to mutation, cancer, and cellular or organismic death.  Damage to DNA induces several cellular responses that enable the cell either to eliminate or cope with the damage or to activate a programmed cell death process, presumably to eliminate cells with potentially catastrophic mutations.  These DNA damage response reactions include: (a) removal of DNA damage and restoration of the continuity of the DNA duplex; (b) activation of a DNA damage checkpoint, which arrests cell cycle progression so as to allow for repair and prevention of the transmission of damaged or incompletely replicated chromosomes: (c) transcriptional response, which causes changes in the transcription profile that may be beneficial to the cell; and (d) apoptosis, which eliminates heavily damaged or seriously deregulated cells.  DNA repair mechanisms include direct repair, base excision repair, nucleotide excision repair, double-strand break repair, and cross-link repair.  The DNA damage checkpoints employ damage sensor proteins, such as ATM, ATR, the Rad17-RFC complex, and the 9-1-1 complex, to detect DNA damage and to initiate signal transduction cascades that employ Chk1 and Chk2 Ser/Thr kinases and Cdc25 phosphatases.  The signal transducers activate p53 and inactivate cyclin-dependent kinases to inhibit cell cycle progression from G1 to S (the G1/S checkpoint), DNA replication (the intra-S checkpoint), or G2 to mitosis (the G2/M checkpoint).  In this review the mol. mechanisms of DNA repair and the DNA damage checkpoints in mammalian cells are analyzed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_p5BqBCbTf7Vg90H21EOLACvtfcHk0ljF2Rj3Bg70Mg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2cXmslagsrk%253D&amp;md5=63f88da9f19d721af81defffba832615</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.73.011303.073723&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.73.011303.073723%26sid%3Dliteratum%253Aachs%26aulast%3DSancar%26aufirst%3DA.%26aulast%3DLindsey-Boltz%26aufirst%3DL.%2BA.%26aulast%3DUnsal-Kacmaz%26aufirst%3DK.%26aulast%3DLinn%26aufirst%3DS.%26atitle%3DMolecular%2520Mechanisms%2520of%2520Mammalian%2520DNA%2520Repair%2520and%2520the%2520DNA%2520Damage%2520Checkpoints%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2004%26volume%3D73%26spage%3D39%26epage%3D85%26doi%3D10.1146%2Fannurev.biochem.73.011303.073723" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, S.</span></span> <span> </span><span class="NLM_article-title">Base-excision Repair of Oxidative DNA Damage</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>447</i></span> (<span class="NLM_issue">7147</span>),  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">950</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature05978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnature05978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=17581577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD2sXms12js74%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=447&amp;publication_year=2007&amp;pages=941-950&amp;issue=7147&amp;author=S.+S.+Davidauthor=V.+L.+O%E2%80%99Sheaauthor=S.+Kundu&amp;title=Base-excision+Repair+of+Oxidative+DNA+Damage&amp;doi=10.1038%2Fnature05978" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Base-excision repair of oxidative DNA damage</span></div><div class="casAuthors">David, Sheila S.; O'Shea, Valerie L.; Kundu, Sucharita</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">447</span>
        (<span class="NLM_cas:issue">7147</span>),
    <span class="NLM_cas:pages">941-950</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Maintaining the chem. integrity of DNA in the face of assault by oxidizing agents is a const. challenge for living organisms.  Base-excision repair has an important role in preventing mutations assocd. with a common product of oxidative damage to DNA, 8-oxoguanine.  Recent structural studies have shown that 8-oxoguanine DNA glycosylases use an intricate series of steps to locate and excise 8-oxoguanine lesions efficiently against a high background of undamaged bases.  The importance of preventing mutations assocd. with 8-oxoguanine is shown by a direct assocn. between defects in the DNA glycosylase MUTYH and colorectal cancer.  The properties of other guanine oxidn. products and the assocd. DNA glycosylases that remove them are now also being revealed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot6-SWDCNHJLVg90H21EOLACvtfcHk0lj3-FBlevYjgg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXms12js74%253D&amp;md5=8e58553790fbb8888f3ab9906c567026</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature05978&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05978%26sid%3Dliteratum%253Aachs%26aulast%3DDavid%26aufirst%3DS.%2BS.%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DV.%2BL.%26aulast%3DKundu%26aufirst%3DS.%26atitle%3DBase-excision%2520Repair%2520of%2520Oxidative%2520DNA%2520Damage%26jtitle%3DNature%26date%3D2007%26volume%3D447%26issue%3D7147%26spage%3D941%26epage%3D950%26doi%3D10.1038%2Fnature05978" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marteijn, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lans, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeijmakers, J. H. J.</span></span> <span> </span><span class="NLM_article-title">Understanding Nucleotide Excision Repair and its Roles in Cancer and Ageing</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">481</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nrm3822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnrm3822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=24954209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVaisL7I" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=2014&amp;pages=465-481&amp;issue=7&amp;author=J.+A.+Marteijnauthor=H.+Lansauthor=W.+Vermeulenauthor=J.+H.+J.+Hoeijmakers&amp;title=Understanding+Nucleotide+Excision+Repair+and+its+Roles+in+Cancer+and+Ageing&amp;doi=10.1038%2Fnrm3822" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding nucleotide excision repair and its roles in cancer and ageing</span></div><div class="casAuthors">Marteijn, Jurgen A.; Lans, Hannes; Vermeulen, Wim; Hoeijmakers, Jan H. J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-481</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Nucleotide excision repair (NER) eliminates various structurally unrelated DNA lesions by a multiwise 'cut and patch'-type reaction.  The global genome NER (GG-NER) subpathway prevents mutagenesis by probing the genome for helix-distorting lesions, whereas transcription-coupled NER (TC-NER) removes transcription-blocking lesions to permit unperturbed gene expression, thereby preventing cell death.  Consequently, defects in GG-NER result in cancer predisposition, whereas defects in TC-NER cause a variety of diseases ranging from UV radiation-sensitive syndrome to severe premature ageing conditions such as Cockayne syndrome.  Recent studies have uncovered new aspects of DNA-damage detection by NER, how NER is regulated by extensive post-translational modifications, and the dynamic chromatin interactions that control its efficiency.  Based on these findings, a mechanistic model is proposed that explains the complex genotype-phenotype correlations of transcription-coupled repair disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD-ZP89iPqubVg90H21EOLACvtfcHk0lj3-FBlevYjgg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXhtVaisL7I&amp;md5=c48a0203cf22aeaf7d885737988598d9</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrm3822&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3822%26sid%3Dliteratum%253Aachs%26aulast%3DMarteijn%26aufirst%3DJ.%2BA.%26aulast%3DLans%26aufirst%3DH.%26aulast%3DVermeulen%26aufirst%3DW.%26aulast%3DHoeijmakers%26aufirst%3DJ.%2BH.%2BJ.%26atitle%3DUnderstanding%2520Nucleotide%2520Excision%2520Repair%2520and%2520its%2520Roles%2520in%2520Cancer%2520and%2520Ageing%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2014%26volume%3D15%26issue%3D7%26spage%3D465%26epage%3D481%26doi%3D10.1038%2Fnrm3822" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tubbs, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussenzweig, A.</span></span> <span> </span><span class="NLM_article-title">Endogenous DNA Damage as a Source of Genomic Instability in Cancer</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>168</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">644</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2017.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.cell.2017.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=28187286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1ygt7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=168&amp;publication_year=2017&amp;pages=644&amp;issue=4&amp;author=A.+Tubbsauthor=A.+Nussenzweig&amp;title=Endogenous+DNA+Damage+as+a+Source+of+Genomic+Instability+in+Cancer&amp;doi=10.1016%2Fj.cell.2017.01.002" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Endogenous DNA damage as a source of genomic instability in cancer</span></div><div class="casAuthors">Tubbs, Anthony; Nussenzweig, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">644-656</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Genome instability, defined as higher than normal rates of mutation, is a double-edged sword.  As a source of genetic diversity and natural selection, mutations are beneficial for evolution.  On the other hand, genomic instability can have catastrophic consequences for age-related diseases such as cancer.  Mutations arise either from inactivation of DNA repair pathways or in a repair-competent background due to genotoxic stress from cellular processes such as transcription and replication that overwhelm high-fidelity DNA repair.  Here, we review recent studies that shed light on endogenous sources of mutation and epigenomic features that promote genomic instability during cancer evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfnOzjrfwD8rVg90H21EOLACvtfcHk0lj3-FBlevYjgg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXis1ygt7Y%253D&amp;md5=88a3d335ab90c98819538b0ab849b602</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.01.002&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DTubbs%26aufirst%3DA.%26aulast%3DNussenzweig%26aufirst%3DA.%26atitle%3DEndogenous%2520DNA%2520Damage%2520as%2520a%2520Source%2520of%2520Genomic%2520Instability%2520in%2520Cancer%26jtitle%3DCell%26date%3D2017%26volume%3D168%26issue%3D4%26spage%3D644%26doi%3D10.1016%2Fj.cell.2017.01.002" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knijnenburg, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambwe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherniack, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Way, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, C. S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Genomic and Molecular Landscape of DNA Damage Repair Deficiency across The Cancer Genome Atlas</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">239</span>– <span class="NLM_lpage">254</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.celrep.2018.03.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.celrep.2018.03.076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=29617664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntVKqt74%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2018&amp;pages=239-254&amp;issue=1&amp;author=T.+A.+Knijnenburgauthor=L.+Wangauthor=M.+T.+Zimmermannauthor=N.+Chambweauthor=G.+F.+Gaoauthor=A.+D.+Cherniackauthor=H.+Fanauthor=H.+Shenauthor=G.+P.+Wayauthor=C.+S.+Greene&amp;title=Genomic+and+Molecular+Landscape+of+DNA+Damage+Repair+Deficiency+across+The+Cancer+Genome+Atlas&amp;doi=10.1016%2Fj.celrep.2018.03.076" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic and Molecular Landscape of DNA Damage Repair Deficiency across The Cancer Genome Atlas</span></div><div class="casAuthors">Knijnenburg, Theo A.; Wang, Linghua; Zimmermann, Michael T.; Chambwe, Nyasha; Gao, Galen F.; Cherniack, Andrew D.; Fan, Huihui; Shen, Hui; Way, Gregory P.; Greene, Casey S.; Liu, Yuexin; Akbani, Rehan; Feng, Bin; Donehower, Lawrence A.; Miller, Chase; Shen, Yang; Karimi, Mostafa; Chen, Haoran; Kim, Pora; Jia, Peilin; Shinbrot, Eve; Zhang, Shaojun; Liu, Jianfang; Hu, Hai; Bailey, Matthew H.; Yau, Christina; Wolf, Denise; Zhao, Zhongming; Weinstein, John N.; Li, Lei; Ding, Li; Mills, Gordon B.; Laird, Peter W.; Wheeler, David A.; Shmulevich, Ilya; Monnat, Raymond J., Jr.; Xiao, Yonghong; Wang, Chen</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">239-254.e6</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">DNA damage repair (DDR) pathways modulate cancer risk, progression, and therapeutic response.  We systematically analyzed somatic alterations to provide a comprehensive view of DDR deficiency across 33 cancer types.  Mutations with accompanying loss of heterozygosity were obsd. in over 1/3 of DDR genes, including TP53 and BRCA1/2.  Other prevalent alterations included epigenetic silencing of the direct repair genes EXO5, MGMT, and ALKBH3 in ∼20% of samples.  Homologous recombination deficiency (HRD) was present at varying frequency in many cancer types, most notably ovarian cancer.  However, in contrast to ovarian cancer, HRD was assocd. with worse outcomes in several other cancers.  Protein structure-based analyses allowed us to predict functional consequences of rare, recurrent DDR mutations.  A new machine-learning-based classifier developed from gene expression data allowed us to identify alterations that phenocopy deleterious TP53 mutations.  These frequent DDR gene alterations in many human cancers have functional consequences that may det. cancer progression and guide therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3qjeMN9_Q-bVg90H21EOLACvtfcHk0lgqUv05y8b82g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXntVKqt74%253D&amp;md5=8eac929850bdbd897b51fa7026e931eb</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2018.03.076&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2018.03.076%26sid%3Dliteratum%253Aachs%26aulast%3DKnijnenburg%26aufirst%3DT.%2BA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZimmermann%26aufirst%3DM.%2BT.%26aulast%3DChambwe%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DG.%2BF.%26aulast%3DCherniack%26aufirst%3DA.%2BD.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DWay%26aufirst%3DG.%2BP.%26aulast%3DGreene%26aufirst%3DC.%2BS.%26atitle%3DGenomic%2520and%2520Molecular%2520Landscape%2520of%2520DNA%2520Damage%2520Repair%2520Deficiency%2520across%2520The%2520Cancer%2520Genome%2520Atlas%26jtitle%3DCell%2520Rep.%26date%3D2018%26volume%3D23%26issue%3D1%26spage%3D239%26epage%3D254%26doi%3D10.1016%2Fj.celrep.2018.03.076" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuss, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishida, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span> <span> </span><span class="NLM_article-title">Emerging critical roles of Fe–S clusters in DNA replication and repair</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1853</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1253</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bbamcr.2015.01.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.bbamcr.2015.01.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=25655665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitl2ks74%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1853&amp;publication_year=2015&amp;pages=1253&amp;issue=6&amp;author=J.+O.+Fussauthor=C.-L.+Tsaiauthor=J.+P.+Ishidaauthor=J.+A.+Tainer&amp;title=Emerging+critical+roles+of+Fe%E2%80%93S+clusters+in+DNA+replication+and+repair&amp;doi=10.1016%2Fj.bbamcr.2015.01.018" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging critical roles of Fe-S clusters in DNA replication and repair</span></div><div class="casAuthors">Fuss, Jill O.; Tsai, Chi-Lin; Ishida, Justin P.; Tainer, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1853</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1253-1271</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Fe-S clusters are partners in the origin of life that predate cells, acetyl-CoA metab., DNA, and the RNA world.  The double helix solved the mystery of DNA replication by base pairing for accurate copying.  Yet, for genome stability necessary to life, the double helix has equally important implications for damage repair.  Here we examine striking advances that uncover Fe-S cluster roles both in copying the genetic sequence by DNA polymerases and in crucial repair processes for genome maintenance, as mutational defects cause cancer and degenerative disease.  Moreover, we examine an exciting, controversial role for Fe-S clusters in a third element required for life - the long-range coordination and regulation of replication and repair events.  By their ability to delocalize electrons over both Fe and S centers, Fe-S clusters have unbeatable features for protein conformational control and charge transfer via double-stranded DNA that may fundamentally transform our understanding of life, replication, and repair.  This article is part of a Special Issue entitled: Fe/S proteins: Anal., structure, function, biogenesis and diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg8eONB8437bVg90H21EOLACvtfcHk0lgqUv05y8b82g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXitl2ks74%253D&amp;md5=500b7f2678b074fcd396989977a517b7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2015.01.018&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2015.01.018%26sid%3Dliteratum%253Aachs%26aulast%3DFuss%26aufirst%3DJ.%2BO.%26aulast%3DTsai%26aufirst%3DC.-L.%26aulast%3DIshida%26aufirst%3DJ.%2BP.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26atitle%3DEmerging%2520critical%2520roles%2520of%2520Fe%25E2%2580%2593S%2520clusters%2520in%2520DNA%2520replication%2520and%2520repair%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2015%26volume%3D1853%26issue%3D6%26spage%3D1253%26doi%3D10.1016%2Fj.bbamcr.2015.01.018" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouault, T. A.</span></span> <span> </span><span class="NLM_article-title">Iron-sulfur Proteins Hiding in Plain Sight</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">442</span>– <span class="NLM_lpage">445</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.1843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnchembio.1843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=26083061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1ygsbrE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2015&amp;pages=442-445&amp;issue=7&amp;author=T.+A.+Rouault&amp;title=Iron-sulfur+Proteins+Hiding+in+Plain+Sight&amp;doi=10.1038%2Fnchembio.1843" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Iron-sulfur proteins hiding in plain sight</span></div><div class="casAuthors">Rouault, Tracey A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">442-445</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent studies suggest that iron-sulfur (Fe-S) proteins may be unexpectedly abundant and functionally diverse in mammalian cells, but their identification still remains difficult.  The use of informatics along with traditional spectroscopic analyses could be key to discovering new Fe-S proteins and validating their functional roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpordnJZVNsibVg90H21EOLACvtfcHk0li2iI7b42zeVg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXht1ygsbrE&amp;md5=ce0bdf2205b3b3d285ef365efa5433d7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1843&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1843%26sid%3Dliteratum%253Aachs%26aulast%3DRouault%26aufirst%3DT.%2BA.%26atitle%3DIron-sulfur%2520Proteins%2520Hiding%2520in%2520Plain%2520Sight%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D7%26spage%3D442%26epage%3D445%26doi%3D10.1038%2Fnchembio.1843" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grodick, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segal, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA-Mediated Signaling by Proteins with 4Fe-4S Clusters is Necessary for Genomic Integrity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">6470</span>– <span class="NLM_lpage">6478</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja501973c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja501973c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlygtbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=136&amp;publication_year=2014&amp;pages=6470-6478&amp;issue=17&amp;author=M.+A.+Grodickauthor=H.+M.+Segalauthor=T.+J.+Zwangauthor=J.+K.+Barton&amp;title=DNA-Mediated+Signaling+by+Proteins+with+4Fe-4S+Clusters+is+Necessary+for+Genomic+Integrity&amp;doi=10.1021%2Fja501973c" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Mediated Signaling by Proteins with 4Fe-4S Clusters Is Necessary for Genomic Integrity</span></div><div class="casAuthors">Grodick, Michael A.; Segal, Helen M.; Zwang, Theodore J.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6470-6478</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Iron-sulfur clusters have increasingly been found to be assocd. with enzymes involved in DNA processing.  Here we describe a role for these redox clusters in DNA-mediated charge transport (CT) signaling in E. coli between DNA repair proteins from distinct pathways.  DNA-modified electrochem. shows that the 4Fe-4S cluster of DNA-bound DinG, an ATP-dependent helicase that repairs R-loops, is redox-active at cellular potentials and ATP hydrolysis increases DNA-mediated redox signaling.  Atomic force microscopy expts. demonstrate that DinG and Endonuclease III (EndoIII), a base excision repair enzyme, cooperate at long-range using DNA charge transport to redistribute to regions of DNA damage.  Genetics expts., moreover, reveal that this DNA-mediated signaling among proteins also occurs within the cell, and remarkably, is required for cellular viability under conditions of stress.  Silencing the gene encoding EndoIII in a strain of E. coli where repair by DinG is essential results in a significant growth defect that is rescued by complementation with EndoIII but not with an EndoIII mutant that is enzymically active but unable to carry out DNA charge transport.  This work thus elucidates a fundamental mechanism to coordinate the activities of DNA repair enzymes across the genome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfuCvre35b7bVg90H21EOLACvtfcHk0li2iI7b42zeVg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXmtlygtbw%253D&amp;md5=e72498bbc3efbcd815ef73f8065bde5e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fja501973c&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja501973c%26sid%3Dliteratum%253Aachs%26aulast%3DGrodick%26aufirst%3DM.%2BA.%26aulast%3DSegal%26aufirst%3DH.%2BM.%26aulast%3DZwang%26aufirst%3DT.%2BJ.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA-Mediated%2520Signaling%2520by%2520Proteins%2520with%25204Fe-4S%2520Clusters%2520is%2520Necessary%2520for%2520Genomic%2520Integrity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26issue%3D17%26spage%3D6470%26epage%3D6478%26doi%3D10.1021%2Fja501973c" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yavin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukianova, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA-Bound Redox Activity of DNA Repair Glycosylases Containing [4Fe-4S] Clusters</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">8397</span>– <span class="NLM_lpage">8407</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi047494n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi047494n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktF2ktL8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=44&amp;publication_year=2005&amp;pages=8397-8407&amp;issue=23&amp;author=A.+K.+Boalauthor=E.+Yavinauthor=O.+A.+Lukianovaauthor=V.+L.+O%E2%80%99Sheaauthor=S.+S.+Davidauthor=J.+K.+Barton&amp;title=DNA-Bound+Redox+Activity+of+DNA+Repair+Glycosylases+Containing+%5B4Fe-4S%5D+Clusters&amp;doi=10.1021%2Fbi047494n" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Bound Redox Activity of DNA Repair Glycosylases Containing [4Fe-4S] Clusters</span></div><div class="casAuthors">Boal, Amie K.; Yavin, Eylon; Lukianova, Olga A.; O'Shea, Valerie L.; David, Sheila S.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8397-8407</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MutY and endonuclease III (Endo III), two DNA glycosylases from Escherichia coli, and AfUDG, a uracil DNA glycosylase from Archeoglobus fulgidus, are all base excision repair enzymes that contain the [4Fe-4S]2+ cofactor.  Here we demonstrate that, when bound to DNA, these repair enzymes become redox-active; binding to DNA shifts the redox potential of the [4Fe-4S]3+/2+ couple to the range characteristic of high-potential iron proteins and activates the proteins toward oxidn.  Electrochem. on DNA-modified electrodes reveals potentials for Endo III and AfUDG of 58 and 95 mV vs. NHE, resp., comparable to 90 mV for MutY bound to DNA.  In the absence of DNA modification of the electrode, no redox activity can be detected, and on electrodes modified with DNA contg. an abasic site, the redox signals are dramatically attenuated; these observations show that the DNA base pair stack mediates electron transfer to the protein, and the potentials detd. are for the DNA-bound protein.  In EPR expts. at 10 K, redox activation upon DNA binding is also evident to yield the oxidized [4Fe-4S]3+ cluster and the partially degraded [3Fe-4S]1+ cluster.  EPR signals at g = 2.02 and 1.99 for MutY and g = 2.03 and 2.01 for Endo III are seen upon oxidn. of these proteins by Co(phen)33+ in the presence of DNA and are characteristic of [3Fe-4S]1+ clusters, while oxidn. of AfUDG bound to DNA yields EPR signals at g = 2.13, 2.04, and 2.02, indicative of both [4Fe-4S]3+ and [3Fe-4S]1+ clusters.  On the basis of this DNA-dependent redox activity, we propose a model for the rapid detection of DNA lesions using DNA-mediated electron transfer among these repair enzymes; redox activation upon DNA binding and charge transfer through well-matched DNA to an alternate bound repair protein can lead to the rapid redistribution of proteins onto genome sites in the vicinity of DNA lesions.  This redox activation furthermore establishes a functional role for the ubiquitous [4Fe-4S] clusters in DNA repair enzymes that involves redox chem. and provides a means to consider DNA-mediated signaling within the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTEk7wgWbSbLVg90H21EOLACvtfcHk0li2iI7b42zeVg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXktF2ktL8%253D&amp;md5=57dd0af2e4693ccf7f85964c0924ebc5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fbi047494n&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi047494n%26sid%3Dliteratum%253Aachs%26aulast%3DBoal%26aufirst%3DA.%2BK.%26aulast%3DYavin%26aufirst%3DE.%26aulast%3DLukianova%26aufirst%3DO.%2BA.%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DV.%2BL.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA-Bound%2520Redox%2520Activity%2520of%2520DNA%2520Repair%2520Glycosylases%2520Containing%2520%255B4Fe-4S%255D%2520Clusters%26jtitle%3DBiochemistry%26date%3D2005%26volume%3D44%26issue%3D23%26spage%3D8397%26epage%3D8407%26doi%3D10.1021%2Fbi047494n" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, S.
O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Electron Transfer Between Bases in Double Helical DNA</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>283</i></span> (<span class="NLM_issue">5400</span>),  <span class="NLM_fpage">375</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.283.5400.375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.283.5400.375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=9888851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK1MXns1eltQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=283&amp;publication_year=1999&amp;pages=375&amp;issue=5400&amp;author=S.%0AO.+Kelleyauthor=J.+K.+Barton&amp;title=Electron+Transfer+Between+Bases+in+Double+Helical+DNA&amp;doi=10.1126%2Fscience.283.5400.375" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Electron transfer between bases in double helical DNA</span></div><div class="casAuthors">Kelley, Shana O.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">5400</span>),
    <span class="NLM_cas:pages">375-381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Fluorescent analogs of adenine that selectively oxidize guanine were used to investigate photoinduced electron transfer through the DNA π-stack as a function of reactant stacking and energetics.  Small variations in these factors led to profound changes in the kinetics and distance dependences of DNA-mediated electron-transfer reactions.  Values of 3, a parameter reflecting the dependence of electron transfer on distance, ranged from 0.1 to 1.0 per angstrom.  Strong stacking interactions result in the fastest electron-transfer kinetics.  Electrons are thus transported preferentially through an intrastrand rather than interstrand pathway.  Reactant energetics also modulate the distance dependence of DNA-mediated charge transport.  These studies may resolve the range of disparate results previously reported, and paradigms must now be developed to describe these properties of the DNA π-stack, which can range from insulator- to "wire"-like.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRqzMiIoL8bVg90H21EOLACvtfcHk0lhpTDscNpC5oQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1MXns1eltQ%253D%253D&amp;md5=aaf2498a7ea56d699a8440b38f39b8d8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1126%2Fscience.283.5400.375&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.283.5400.375%26sid%3Dliteratum%253Aachs%26aulast%3DKelley%26aufirst%3DS.%2BO.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DElectron%2520Transfer%2520Between%2520Bases%2520in%2520Double%2520Helical%2520DNA%26jtitle%3DScience%26date%3D1999%26volume%3D283%26issue%3D5400%26spage%3D375%26doi%3D10.1126%2Fscience.283.5400.375" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demple, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L.</span></span> <span> </span><span class="NLM_article-title">Repair of Oxidative Damage to DNA: Enzymology and Biology</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">915</span>– <span class="NLM_lpage">948</span>, <span class="refDoi">&nbsp;DOI: 10.1146/annurev.bi.63.070194.004411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1146%2Fannurev.bi.63.070194.004411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=7979257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK2MXitlCgtA%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=63&amp;publication_year=1994&amp;pages=915-948&amp;author=B.+Dempleauthor=L.+Harrison&amp;title=Repair+of+Oxidative+Damage+to+DNA%3A+Enzymology+and+Biology&amp;doi=10.1146%2Fannurev.bi.63.070194.004411" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Repair of oxidative damage to DNA: Enzymology and biology</span></div><div class="casAuthors">Demple, Bruce; Harrison, Lynn</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">915-48</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    </div><div class="casAbstract">A review, with 226 refs., on DNA glycosylases in the repair of oxidative damage, AP endonucleases and 3'-trimming in the repair of deoxyribose damage, and nucleotide excision and recombination in general repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPUyFucFNBs7Vg90H21EOLACvtfcHk0lhpTDscNpC5oQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXitlCgtA%253D%253D&amp;md5=857c720aa8898b692f52b2c16e6d6480</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1146%2Fannurev.bi.63.070194.004411&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.bi.63.070194.004411%26sid%3Dliteratum%253Aachs%26aulast%3DDemple%26aufirst%3DB.%26aulast%3DHarrison%26aufirst%3DL.%26atitle%3DRepair%2520of%2520Oxidative%2520Damage%2520to%2520DNA%253A%2520Enzymology%2520and%2520Biology%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D1994%26volume%3D63%26spage%3D915%26epage%3D948%26doi%3D10.1146%2Fannurev.bi.63.070194.004411" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkhardt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span> <span> </span><span class="NLM_article-title">Quantifying Absolute Protein Synthesis Rates Reveals Principles Underlying Allocation of Cellular Resources</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>157</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">624</span>– <span class="NLM_lpage">635</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2014.02.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.cell.2014.02.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=24766808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFGitL8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=157&amp;publication_year=2014&amp;pages=624-635&amp;issue=3&amp;author=G.-W.+Liauthor=D.+Burkhardtauthor=C.+Grossauthor=J.+S.+Weissman&amp;title=Quantifying+Absolute+Protein+Synthesis+Rates+Reveals+Principles+Underlying+Allocation+of+Cellular+Resources&amp;doi=10.1016%2Fj.cell.2014.02.033" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying Absolute Protein Synthesis Rates Reveals Principles Underlying Allocation of Cellular Resources</span></div><div class="casAuthors">Li, Gene-Wei; Burkhardt, David; Gross, Carol; Weissman, Jonathan S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">624-635</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Quant. views of cellular functions require precise measures of rates of biomol. prodn., esp. proteins - the direct effectors of biol. processes.  Here, we present a genome-wide approach, based on ribosome profiling, for measuring abs. protein synthesis rates.  The resultant E. coli data set transforms our understanding of the extent to which protein synthesis is precisely controlled to optimize function and efficiency.  Members of multiprotein complexes are made in precise proportion to their stoichiometry, whereas components of functional modules are produced differentially according to their hierarchical role.  Ests. of abs. protein abundance also reveal principles for optimizing design.  These include how the level of different types of transcription factors is optimized for rapid response and how a metabolic pathway (methionine biosynthesis) balances prodn. cost with activity requirements.  Our studies reveal how general principles, important both for understanding natural systems and for synthesizing new ones, emerge from quant. analyses of protein synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLrDULHF1S0LVg90H21EOLACvtfcHk0lgwkyn0x6JRUg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXntFGitL8%253D&amp;md5=cc5a8290bb675fc516cf8fa446c4ed5b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.02.033&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.02.033%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.-W.%26aulast%3DBurkhardt%26aufirst%3DD.%26aulast%3DGross%26aufirst%3DC.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26atitle%3DQuantifying%2520Absolute%2520Protein%2520Synthesis%2520Rates%2520Reveals%2520Principles%2520Underlying%2520Allocation%2520of%2520Cellular%2520Resources%26jtitle%3DCell%26date%3D2014%26volume%3D157%26issue%3D3%26spage%3D624%26epage%3D635%26doi%3D10.1016%2Fj.cell.2014.02.033" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genereux, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sontz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gralnick, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Redox Signaling between DNA Repair Proteins for Efficient Lesion Detection</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">15237</span>– <span class="NLM_lpage">15242</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.0908059106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1073%2Fpnas.0908059106" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=106&amp;publication_year=2009&amp;pages=15237-15242&amp;issue=36&amp;author=A.+K.+Boalauthor=J.+C.+Genereuxauthor=P.+A.+Sontzauthor=J.+A.+Gralnickauthor=D.+K.+Newmanauthor=J.+K.+Barton&amp;title=Redox+Signaling+between+DNA+Repair+Proteins+for+Efficient+Lesion+Detection&amp;doi=10.1073%2Fpnas.0908059106" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0908059106&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0908059106%26sid%3Dliteratum%253Aachs%26aulast%3DBoal%26aufirst%3DA.%2BK.%26aulast%3DGenereux%26aufirst%3DJ.%2BC.%26aulast%3DSontz%26aufirst%3DP.%2BA.%26aulast%3DGralnick%26aufirst%3DJ.%2BA.%26aulast%3DNewman%26aufirst%3DD.%2BK.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DRedox%2520Signaling%2520between%2520DNA%2520Repair%2520Proteins%2520for%2520Efficient%2520Lesion%2520Detection%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26issue%3D36%26spage%3D15237%26epage%3D15242%26doi%3D10.1073%2Fpnas.0908059106" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popodi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, P. L.</span></span> <span> </span><span class="NLM_article-title">Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span> (<span class="NLM_issue">41</span>),  <span class="NLM_fpage">E2774</span>– <span class="NLM_lpage">E2783</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1210309109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1073%2Fpnas.1210309109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=22991466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKntbjM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=109&amp;publication_year=2012&amp;pages=E2774-E2783&amp;issue=41&amp;author=H.+Leeauthor=E.+Popodiauthor=H.+Tangauthor=P.+L.+Foster&amp;title=Rate+and+molecular+spectrum+of+spontaneous+mutations+in+the+bacterium+Escherichia+coli+as+determined+by+whole-genome+sequencing&amp;doi=10.1073%2Fpnas.1210309109" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing</span></div><div class="casAuthors">Lee, Heewook; Popodi, Ellen; Tang, Haixu; Foster, Patricia L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">E2774-E2783, SE2774/1-SE2774/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Knowledge of the rate and nature of spontaneous mutation is fundamental to understanding evolutionary and mol. processes.  In this report, we analyze spontaneous mutations accumulated over thousands of generations by wild-type Escherichia coli and a deriv. defective in mismatch repair (MMR), the primary pathway for correcting replication errors.  The major conclusions are (i) the mutation rate of a wild-type E. col strain is ∼1 × 10-3 per genome per generation; (ii) mutations in the wild-type strain have the expected mutational bias for G:C &gt; A:T mutations, but the bias changes to A:T &gt; G:C mutations in the absence of MMR; (iii) during replication, A:T &gt; G:C transitions preferentially occur with A templating the lagging strand and T templating the leading strand, whereas G:C &gt; A:T transitions preferentially occur with C templating the lagging strand and G templating the leading strand; (iv) there is a strong bias for transition mutations to occur at 5'ApC3'/3'TpG5' sites (where bases 5'A and 3'T are mutated) and, to a lesser extent, at 5'GpC3'/3'CpG5' sites (where bases 5'G and 3'C are mutated); (v) although the rate of small (≤4 nt) insertions and deletions is high at repeat sequences, these events occur at only 1/10th the genomic rate of base-pair substitutions.  MMR activity is genetically regulated, and bacteria isolated from nature often lack MMR capacity, suggesting that modulation of MMR can be adaptive.  Thus, comparing results from the wild-type and MMR-defective strains may lead to a deeper understanding of factors that det. mutation rates and spectra, how these factors may differ among organisms, and how they may be shaped by environmental conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtRFcPmZX-M7Vg90H21EOLACvtfcHk0lgwkyn0x6JRUg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhsFKntbjM&amp;md5=e44fb70553c7eadc11516ee3346dcd4d</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1210309109&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1210309109%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPopodi%26aufirst%3DE.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DFoster%26aufirst%3DP.%2BL.%26atitle%3DRate%2520and%2520molecular%2520spectrum%2520of%2520spontaneous%2520mutations%2520in%2520the%2520bacterium%2520Escherichia%2520coli%2520as%2520determined%2520by%2520whole-genome%2520sequencing%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26issue%3D41%26spage%3DE2774%26epage%3DE2783%26doi%3D10.1073%2Fpnas.1210309109" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agarwalla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroud, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, B. J.</span></span> <span> </span><span class="NLM_article-title">Redox Reactions of the Iron-Sulfur Cluster in a Ribosomal RNA Methyltransferase, RumA: OPTICAL AND EPR STUDIES</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span> (<span class="NLM_issue">33</span>),  <span class="NLM_fpage">34123</span>– <span class="NLM_lpage">34129</span>, <span class="refDoi">&nbsp;DOI: 10.1074/jbc.M405702200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1074%2Fjbc.M405702200" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=279&amp;publication_year=2004&amp;pages=34123-34129&amp;issue=33&amp;author=S.+Agarwallaauthor=R.+M.+Stroudauthor=B.+J.+Gaffney&amp;title=Redox+Reactions+of+the+Iron-Sulfur+Cluster+in+a+Ribosomal+RNA+Methyltransferase%2C+RumA%3A+OPTICAL+AND+EPR+STUDIES&amp;doi=10.1074%2Fjbc.M405702200" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M405702200&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M405702200%26sid%3Dliteratum%253Aachs%26aulast%3DAgarwalla%26aufirst%3DS.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DGaffney%26aufirst%3DB.%2BJ.%26atitle%3DRedox%2520Reactions%2520of%2520the%2520Iron-Sulfur%2520Cluster%2520in%2520a%2520Ribosomal%2520RNA%2520Methyltransferase%252C%2520RumA%253A%2520OPTICAL%2520AND%2520EPR%2520STUDIES%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26issue%3D33%26spage%3D34123%26epage%3D34129%26doi%3D10.1074%2Fjbc.M405702200" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Netz, D. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stith, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stümpfig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köpf, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genau, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stodola, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lill, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgers, P. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierik, A. J.</span></span> <span> </span><span class="NLM_article-title">Eukaryotic DNA polymerases require an iron-sulfur cluster for the formation of active complexes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">125</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnchembio.721" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFeitrbK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2012&amp;pages=125&amp;issue=1&amp;author=D.+J.+A.+Netzauthor=C.+M.+Stithauthor=M.+St%C3%BCmpfigauthor=G.+K%C3%B6pfauthor=D.+Vogelauthor=H.+M.+Genauauthor=J.+L.+Stodolaauthor=R.+Lillauthor=P.+M.+J.+Burgersauthor=A.+J.+Pierik&amp;title=Eukaryotic+DNA+polymerases+require+an+iron-sulfur+cluster+for+the+formation+of+active+complexes&amp;doi=10.1038%2Fnchembio.721" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Eukaryotic DNA polymerases require an iron-sulfur cluster for the formation of active complexes</span></div><div class="casAuthors">Netz, Daili J. A.; Stith, Carrie M.; Stuempfig, Martin; Koepf, Gabriele; Vogel, Daniel; Genau, Heide M.; Stodola, Joseph L.; Lill, Roland; Burgers, Peter M. J.; Pierik, Antonio J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The eukaryotic replicative DNA polymerases (Pol α, δ and ε) and the major DNA mutagenesis enzyme Pol ζ contain two conserved cysteine-rich metal-binding motifs (CysA and CysB) in the C-terminal domain (CTD) of their catalytic subunits.  Here we demonstrate by in vivo and in vitro approaches the presence of an essential [4Fe-4S] cluster in the CysB motif of all four yeast B-family DNA polymerases.  Loss of the [4Fe-4S] cofactor by cysteine ligand mutagenesis in Pol3 destabilized the CTD and abrogated interaction with the Pol31 and Pol32 subunits.  Reciprocally, overexpression of accessory subunits increased the amt. of the CTD-bound Fe-S cluster.  This implies an important physiol. role of the Fe-S cluster in polymerase complex stabilization.  Further, we demonstrate that the Zn-binding CysA motif is required for PCNA-mediated Pol δ processivity.  Together, our findings show that the function of eukaryotic replicative DNA polymerases crucially depends on different metallocenters for accessory subunit recruitment and replisome stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHCvlCJ7K7ALVg90H21EOLACvtfcHk0lgwkyn0x6JRUg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXhsFeitrbK&amp;md5=c0860ad52cbecb174470f8b9dda38696</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.721&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.721%26sid%3Dliteratum%253Aachs%26aulast%3DNetz%26aufirst%3DD.%2BJ.%2BA.%26aulast%3DStith%26aufirst%3DC.%2BM.%26aulast%3DSt%25C3%25BCmpfig%26aufirst%3DM.%26aulast%3DK%25C3%25B6pf%26aufirst%3DG.%26aulast%3DVogel%26aufirst%3DD.%26aulast%3DGenau%26aufirst%3DH.%2BM.%26aulast%3DStodola%26aufirst%3DJ.%2BL.%26aulast%3DLill%26aufirst%3DR.%26aulast%3DBurgers%26aufirst%3DP.%2BM.%2BJ.%26aulast%3DPierik%26aufirst%3DA.%2BJ.%26atitle%3DEukaryotic%2520DNA%2520polymerases%2520require%2520an%2520iron-sulfur%2520cluster%2520for%2520the%2520formation%2520of%2520active%2520complexes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26issue%3D1%26spage%3D125%26doi%3D10.1038%2Fnchembio.721" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asahara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bank, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholes, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salerno, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surerus, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munck, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCracken, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peisach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emptage, M. H.</span></span> <span> </span><span class="NLM_article-title">Endonuclease III is an Iron-sulfur Protein</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4450</span>– <span class="NLM_lpage">4455</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00436a049</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00436a049" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaL1MXitVyntr4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=28&amp;publication_year=1989&amp;pages=4450-4455&amp;issue=10&amp;author=R.+P.+Cunninghamauthor=H.+Asaharaauthor=J.+F.+Bankauthor=C.+P.+Scholesauthor=J.+C.+Salernoauthor=K.+Surerusauthor=E.+Munckauthor=J.+McCrackenauthor=J.+Peisachauthor=M.+H.+Emptage&amp;title=Endonuclease+III+is+an+Iron-sulfur+Protein&amp;doi=10.1021%2Fbi00436a049" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Endonuclease III is an iron-sulfur protein</span></div><div class="casAuthors">Cunningham, Richard P.; Asahara, Hitomi; Bank, Janet F.; Scholes, Charles P.; Salerno, John C.; Surerus, Kristene; Munck, Eckard; McCracken, John; Peisach, Jack; Emptage, Mark H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4450-5</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Elemental analyses, Moessbauer, and EPR data are reported to show that endonuclease III of Escherichia coli is an Fe-S protein.  Moessbauer spectra of protein freshly prepd. from E. coli grown on 57Fe-enriched medium demonstrate that the native enzyme contains a single 4Fe-4S cluster in the 2+ oxidn. state, with a net spin of zero.  Upon treatment with ferricyanide, a fraction (&lt;25%) of the clusters is oxidized into a state which yields an EPR spectrum near g = 2.01 typical of a 3Fe-4S cluster.  The magnetic field dependence of the linear elec. field effect verifies this assignment.  Electron spin echo modulation on the g = 2.01 form of the protein in deuterated solvent indicates the presence of exchangeable protons in the vicinity of the 3Fe-4S cluster.  The data obtained show that the [4Fe-4S]2+ cluster of the native enzyme is resistant to either oxidn. or redn., although photoredn. elicited a g = 1.94 type EPR signal characteristic of a [4Fe-4S]1+ cluster.  Evidently, endonuclease III is unique in being both a DNA repair enzyme and an Fe-S protein.  The function of the 4Fe-4S cluster remains to be established.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQln8BsjCEvbVg90H21EOLACvtfcHk0ljZUrqOZfMGow"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL1MXitVyntr4%253D&amp;md5=ede2875a8914ab295fe9f3683e582b86</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fbi00436a049&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00436a049%26sid%3Dliteratum%253Aachs%26aulast%3DCunningham%26aufirst%3DR.%2BP.%26aulast%3DAsahara%26aufirst%3DH.%26aulast%3DBank%26aufirst%3DJ.%2BF.%26aulast%3DScholes%26aufirst%3DC.%2BP.%26aulast%3DSalerno%26aufirst%3DJ.%2BC.%26aulast%3DSurerus%26aufirst%3DK.%26aulast%3DMunck%26aufirst%3DE.%26aulast%3DMcCracken%26aufirst%3DJ.%26aulast%3DPeisach%26aufirst%3DJ.%26aulast%3DEmptage%26aufirst%3DM.%2BH.%26atitle%3DEndonuclease%2520III%2520is%2520an%2520Iron-sulfur%2520Protein%26jtitle%3DBiochemistry%26date%3D1989%26volume%3D28%26issue%3D10%26spage%3D4450%26epage%3D4455%26doi%3D10.1021%2Fbi00436a049" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thayer, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahern, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span> <span> </span><span class="NLM_article-title">Novel DNA Binding Motifs in the DNA Repair Enzyme Endonuclease III Crystal Structure</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">4108</span>– <span class="NLM_lpage">4120</span>, <span class="refDoi">&nbsp;DOI: 10.1002/j.1460-2075.1995.tb00083.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1002%2Fj.1460-2075.1995.tb00083.x" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=1995&amp;pages=4108-4120&amp;issue=16&amp;author=M.+M.+Thayerauthor=H.+Ahernauthor=D.+Xingauthor=R.+P.+Cunninghamauthor=J.+A.+Tainer&amp;title=Novel+DNA+Binding+Motifs+in+the+DNA+Repair+Enzyme+Endonuclease+III+Crystal+Structure&amp;doi=10.1002%2Fj.1460-2075.1995.tb00083.x" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fj.1460-2075.1995.tb00083.x&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fj.1460-2075.1995.tb00083.x%26sid%3Dliteratum%253Aachs%26aulast%3DThayer%26aufirst%3DM.%2BM.%26aulast%3DAhern%26aufirst%3DH.%26aulast%3DXing%26aufirst%3DD.%26aulast%3DCunningham%26aufirst%3DR.%2BP.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26atitle%3DNovel%2520DNA%2520Binding%2520Motifs%2520in%2520the%2520DNA%2520Repair%2520Enzyme%2520Endonuclease%2520III%2520Crystal%2520Structure%26jtitle%3DEMBO%2520J.%26date%3D1995%26volume%3D14%26issue%3D16%26spage%3D4108%26epage%3D4120%26doi%3D10.1002%2Fj.1460-2075.1995.tb00083.x" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chepanoske, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golinelli, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span> <span> </span><span class="NLM_article-title">Positively Charged Residues within the Iron–Sulfur Cluster Loop of E. coli MutY Participate in Damage Recognition and Removal</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>380</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">19</span>, <span class="refDoi">&nbsp;DOI: 10.1006/abbi.2000.1890</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1006%2Fabbi.2000.1890" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=380&amp;publication_year=2000&amp;pages=11-19&amp;issue=1&amp;author=C.+L.+Chepanoskeauthor=M.-P.+Golinelliauthor=S.+D.+Williamsauthor=S.+S.+David&amp;title=Positively+Charged+Residues+within+the+Iron%E2%80%93Sulfur+Cluster+Loop+of+E.+coli+MutY+Participate+in+Damage+Recognition+and+Removal&amp;doi=10.1006%2Fabbi.2000.1890" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1006%2Fabbi.2000.1890&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabbi.2000.1890%26sid%3Dliteratum%253Aachs%26aulast%3DChepanoske%26aufirst%3DC.%2BL.%26aulast%3DGolinelli%26aufirst%3DM.-P.%26aulast%3DWilliams%26aufirst%3DS.%2BD.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26atitle%3DPositively%2520Charged%2520Residues%2520within%2520the%2520Iron%25E2%2580%2593Sulfur%2520Cluster%2520Loop%2520of%2520E.%2520coli%2520MutY%2520Participate%2520in%2520Damage%2520Recognition%2520and%2520Removal%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2000%26volume%3D380%26issue%3D1%26spage%3D11%26epage%3D19%26doi%3D10.1006%2Fabbi.2000.1890" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brinkmeyer, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span> <span> </span><span class="NLM_article-title">Distinct Functional Consequences of MUTYH Variants Associated with Colorectal Cancer: Damaged DNA Affinity, Glycosylase Activity and Interaction with PCNA and Hus1</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">51</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.dnarep.2015.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.dnarep.2015.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=26377631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtleks7vL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=2015&amp;pages=39-51&amp;author=M.+K.+Brinkmeyerauthor=S.+S.+David&amp;title=Distinct+Functional+Consequences+of+MUTYH+Variants+Associated+with+Colorectal+Cancer%3A+Damaged+DNA+Affinity%2C+Glycosylase+Activity+and+Interaction+with+PCNA+and+Hus1&amp;doi=10.1016%2Fj.dnarep.2015.08.001" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct functional consequences of MUTYH variants associated with colorectal cancer: Damaged DNA affinity, glycosylase activity and interaction with PCNA and Hus1</span></div><div class="casAuthors">Brinkmeyer, Megan K.; David, Sheila S.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39-51</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">MUTYH is a base excision repair (BER) enzyme that prevents mutations in DNA assocd. with 8-oxoguanine (OG) by catalyzing the removal of adenine from inappropriately formed OG:A base-pairs.  Germline mutations in the MUTYH gene are linked to colorectal polyposis and a high risk of colorectal cancer, a syndrome referred to as MUTYH-assocd. polyposis (MAP).  There are over 300 different MUTYH mutations assocd. with MAP and a large fraction of these gene changes code for missense MUTYH variants.  Herein, the adenine glycosylase activity, mismatch recognition properties, and interaction with relevant protein partners of human MUTYH and five MAP variants (R295C, P281L, Q324H, P502L, and R520Q) were examd.  P281L MUTYH was found to be severely compromised both in DNA binding and base excision activity, consistent with the location of this variation in the iron-sulfur cluster (FCL) DNA binding motif of MUTYH.  Both R295C and R520Q MUTYH were found to have low fractions of active enzyme, compromised affinity for damaged DNA, and reduced rates for adenine excision.  In contrast, both Q324H and P502L MUTYH function relatively similarly to WT MUTYH in both binding and glycosylase assays.  However, P502L and R520Q exhibited reduced affinity for PCNA (proliferation cell nuclear antigen), consistent with their location in the PCNA-binding motif of MUTYH.  Whereas, only Q324H, and not R295C, was found to have reduced affinity for Hus1 of the Rad9-Hus1-Rad1 complex, despite both being localized to the same region implicated for interaction with Hus1.  These results underscore the diversity of functional consequences due to MUTYH variants that may impact the progression of MAP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjxSwDOeuLK7Vg90H21EOLACvtfcHk0ljZUrqOZfMGow"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXhtleks7vL&amp;md5=aeeafa68215a91bd75b783c55d3c245d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2015.08.001&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2015.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DBrinkmeyer%26aufirst%3DM.%2BK.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26atitle%3DDistinct%2520Functional%2520Consequences%2520of%2520MUTYH%2520Variants%2520Associated%2520with%2520Colorectal%2520Cancer%253A%2520Damaged%2520DNA%2520Affinity%252C%2520Glycosylase%2520Activity%2520and%2520Interaction%2520with%2520PCNA%2520and%2520Hus1%26jtitle%3DDNA%2520Repair%26date%3D2015%26volume%3D34%26spage%3D39%26epage%3D51%26doi%3D10.1016%2Fj.dnarep.2015.08.001" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salay, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehlinger, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chazin, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">The [4Fe4S] Cluster of Human DNA Primase Functions as a Redox Switch using DNA Charge Transport</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span> (<span class="NLM_issue">6327</span>),  <span class="NLM_fpage">813</span>– <span class="NLM_lpage">822</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.aag1789</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.aag1789" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=355&amp;publication_year=2017&amp;pages=813-822&amp;issue=6327&amp;author=E.+O%E2%80%99Brienauthor=M.+E.+Holtauthor=M.+K.+Thompsonauthor=L.+E.+Salayauthor=A.+C.+Ehlingerauthor=W.+J.+Chazinauthor=J.+K.+Barton&amp;title=The+%5B4Fe4S%5D+Cluster+of+Human+DNA+Primase+Functions+as+a+Redox+Switch+using+DNA+Charge+Transport&amp;doi=10.1126%2Fscience.aag1789" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1126%2Fscience.aag1789&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aag1789%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DE.%26aulast%3DHolt%26aufirst%3DM.%2BE.%26aulast%3DThompson%26aufirst%3DM.%2BK.%26aulast%3DSalay%26aufirst%3DL.%2BE.%26aulast%3DEhlinger%26aufirst%3DA.%2BC.%26aulast%3DChazin%26aufirst%3DW.%2BJ.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DThe%2520%255B4Fe4S%255D%2520Cluster%2520of%2520Human%2520DNA%2520Primase%2520Functions%2520as%2520a%2520Redox%2520Switch%2520using%2520DNA%2520Charge%2520Transport%26jtitle%3DScience%26date%3D2017%26volume%3D355%26issue%3D6327%26spage%3D813%26epage%3D822%26doi%3D10.1126%2Fscience.aag1789" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartels, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stodola, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgers, P. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">A Redox Role for the [4Fe4S] Cluster of Yeast DNA Polymerase δ</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span> (<span class="NLM_issue">50</span>),  <span class="NLM_fpage">18339</span>– <span class="NLM_lpage">18348</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.7b10284</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b10284" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVKju7fJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=139&amp;publication_year=2017&amp;pages=18339-18348&amp;issue=50&amp;author=P.+L.+Bartelsauthor=J.+L.+Stodolaauthor=P.+M.+J.+Burgersauthor=J.+K.+Barton&amp;title=A+Redox+Role+for+the+%5B4Fe4S%5D+Cluster+of+Yeast+DNA+Polymerase+%CE%B4&amp;doi=10.1021%2Fjacs.7b10284" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Redox Role for the [4Fe4S] Cluster of Yeast DNA Polymerase δ</span></div><div class="casAuthors">Bartels, Phillip L.; Stodola, Joseph L.; Burgers, Peter M. J.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">18339-18348</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A [4Fe4S]2+ cluster in the C-terminal domain of the catalytic subunit of the eukaryotic B-family DNA polymerases is essential for the formation of active multi-subunit complexes.  Here we use a combination of electrochem. and biochem. methods to assess the redox activity of the [4Fe4S]2+ cluster in Saccharomyces cerevisiae polymerase (Pol) δ, the lagging strand DNA polymerase.  We find that Pol δ bound to DNA is indeed redox-active at physiol. potentials, generating a DNA-mediated signal electrochem. with a midpoint potential of 113 ± 5 mV vs. NHE.  Moreover, biochem. assays following electrochem. oxidn. of Pol δ reveal a significant slowing of DNA synthesis that can be fully reversed by redn. of the oxidized form.  A similar result is apparent with photooxidn. using a DNA-tethered anthraquinone.  These results demonstrate that the [4Fe4S] cluster in Pol δ can act as a redox switch for activity, and we propose that this switch can provide a rapid and reversible way to respond to replication stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT5FK9sBXvcLVg90H21EOLACvtfcHk0ljy46uxYL58Xg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhvVKju7fJ&amp;md5=edbd20b63a557dd0f38092930f647963</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b10284&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b10284%26sid%3Dliteratum%253Aachs%26aulast%3DBartels%26aufirst%3DP.%2BL.%26aulast%3DStodola%26aufirst%3DJ.%2BL.%26aulast%3DBurgers%26aufirst%3DP.%2BM.%2BJ.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DA%2520Redox%2520Role%2520for%2520the%2520%255B4Fe4S%255D%2520Cluster%2520of%2520Yeast%2520DNA%2520Polymerase%2520%25CE%25B4%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26issue%3D50%26spage%3D18339%26epage%3D18348%26doi%3D10.1021%2Fjacs.7b10284" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engstrom, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinkmeyer, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ha, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raetz, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgson, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span> <span> </span><span class="NLM_article-title">A Zinc Linchpin Motif in the MUTYH Glycosylase Interdomain Connector is Required for Efficient Repair of DNA Damage</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">7829</span>– <span class="NLM_lpage">7832</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja502942d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja502942d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=136&amp;publication_year=2014&amp;pages=7829-7832&amp;issue=22&amp;author=L.+M.+Engstromauthor=M.+K.+Brinkmeyerauthor=Y.+Haauthor=A.+G.+Raetzauthor=B.+Hedmanauthor=K.+O.+Hodgsonauthor=E.+I.+Solomonauthor=S.+S.+David&amp;title=A+Zinc+Linchpin+Motif+in+the+MUTYH+Glycosylase+Interdomain+Connector+is+Required+for+Efficient+Repair+of+DNA+Damage&amp;doi=10.1021%2Fja502942d" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fja502942d&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja502942d%26sid%3Dliteratum%253Aachs%26aulast%3DEngstrom%26aufirst%3DL.%2BM.%26aulast%3DBrinkmeyer%26aufirst%3DM.%2BK.%26aulast%3DHa%26aufirst%3DY.%26aulast%3DRaetz%26aufirst%3DA.%2BG.%26aulast%3DHedman%26aufirst%3DB.%26aulast%3DHodgson%26aufirst%3DK.%2BO.%26aulast%3DSolomon%26aufirst%3DE.%2BI.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26atitle%3DA%2520Zinc%2520Linchpin%2520Motif%2520in%2520the%2520MUTYH%2520Glycosylase%2520Interdomain%2520Connector%2520is%2520Required%2520for%2520Efficient%2520Repair%2520of%2520DNA%2520Damage%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26issue%3D22%26spage%3D7829%26epage%3D7832%26doi%3D10.1021%2Fja502942d" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porello, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span> <span> </span><span class="NLM_article-title">A Substrate Recognition Role for the [4Fe-4S]2+ Cluster of the DNA Repair Glycosylase MutY</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">6465</span>– <span class="NLM_lpage">6475</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi972433t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi972433t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=37&amp;publication_year=1998&amp;pages=6465-6475&amp;issue=18&amp;author=S.+L.+Porelloauthor=M.+J.+Cannonauthor=S.+S.+David&amp;title=A+Substrate+Recognition+Role+for+the+%5B4Fe-4S%5D2%2B+Cluster+of+the+DNA+Repair+Glycosylase+MutY&amp;doi=10.1021%2Fbi972433t" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fbi972433t&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi972433t%26sid%3Dliteratum%253Aachs%26aulast%3DPorello%26aufirst%3DS.%2BL.%26aulast%3DCannon%26aufirst%3DM.%2BJ.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26atitle%3DA%2520Substrate%2520Recognition%2520Role%2520for%2520the%2520%255B4Fe-4S%255D2%252B%2520Cluster%2520of%2520the%2520DNA%2520Repair%2520Glycosylase%2520MutY%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26issue%3D18%26spage%3D6465%26epage%3D6475%26doi%3D10.1021%2Fbi972433t" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pope, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span> <span> </span><span class="NLM_article-title">DNA Damage Recognition and Repair by the Murine MutY Homologue</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">102</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.dnarep.2004.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.dnarep.2004.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=15533841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptl2qtr0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2005&amp;pages=91-102&amp;issue=1&amp;author=M.+A.+Popeauthor=S.+S.+David&amp;title=DNA+Damage+Recognition+and+Repair+by+the+Murine+MutY+Homologue&amp;doi=10.1016%2Fj.dnarep.2004.08.004" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">DNA damage recognition and repair by the murine MutY homologue</span></div><div class="casAuthors">Pope, Mary Ann; David, Sheila S.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-102</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">E. coli MutY excises adenine from duplex DNA when it is mispaired with the mutagenic oxidative lesion 7,8-dihydro-8-oxo-2'-deoxyguanosine (OG).  While E. coli MutY has been extensively studied, a detailed kinetic anal. of a mammalian MutY homolog has been inhibited by poor overexpression in bacterial hosts.  This current work is the first detailed study of substrate recognition and repair of mismatched DNA by a mammalian adenine glycosylase, the murine MutY homolog (mMYH).  Similar to E. coli MutY, the processing of OG:A substrates by mMYH is biphasic, indicating that product release is rate-limiting.  Surprisingly, the intrinsic rates of adenine removal from both OG:A and G:A substrates by mMYH are diminished (∼10-fold) compared to E. coli MutY.  However, similar to E. coli MutY, the rate of adenine removal is approx. nine-fold faster with an OG:A- than a G:A-contg. substrate.  Interestingly, the rate of removal of 2-hydroxyadenine mispaired with OG or G in duplex DNA by mMYH was similar to the rate of adenine removal from the analogous context.  In contrast, 2-hydroxyadenine removal by E. coli MutY was significantly reduced compared to adenine removal opposite both OG and G.  Furthermore, dissocn. const. measurements with duplexes contg. noncleavable 2'-deoxyadenosine analogs indicate that mMYH is less sensitive to the structure of the base mispaired with OG or G than MutY.  Though in many respects the catalytic behavior of mMYH is similar to E. coli MutY, the subtle differences may translate into differences in their in vivo functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8iFzE3gzwn7Vg90H21EOLACvtfcHk0lhIxMepfXv5lg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2cXptl2qtr0%253D&amp;md5=6b1609d40a0e3d24e9d3bbf429de1e8d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2004.08.004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2004.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DPope%26aufirst%3DM.%2BA.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26atitle%3DDNA%2520Damage%2520Recognition%2520and%2520Repair%2520by%2520the%2520Murine%2520MutY%2520Homologue%26jtitle%3DDNA%2520Repair%26date%3D2005%26volume%3D4%26issue%3D1%26spage%3D91%26epage%3D102%26doi%3D10.1016%2Fj.dnarep.2004.08.004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Structure of a Trapped Endonuclease III–DNA Covalent Intermediate</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">3461</span>– <span class="NLM_lpage">3471</span>, <span class="refDoi">&nbsp;DOI: 10.1093/emboj/cdg311</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1093%2Femboj%2Fcdg311" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=2003&amp;pages=3461-3471&amp;issue=13&amp;author=J.+C.+Frommeauthor=G.+L.+Verdine&amp;title=Structure+of+a+Trapped+Endonuclease+III%E2%80%93DNA+Covalent+Intermediate&amp;doi=10.1093%2Femboj%2Fcdg311" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Femboj%2Fcdg311&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252Fcdg311%26sid%3Dliteratum%253Aachs%26aulast%3DFromme%26aufirst%3DJ.%2BC.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DStructure%2520of%2520a%2520Trapped%2520Endonuclease%2520III%25E2%2580%2593DNA%2520Covalent%2520Intermediate%26jtitle%3DEMBO%2520J.%26date%3D2003%26volume%3D22%26issue%3D13%26spage%3D3461%26epage%3D3471%26doi%3D10.1093%2Femboj%2Fcdg311" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Handley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholes, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, R. P.</span></span> <span> </span><span class="NLM_article-title">Endonuclease III Interactions with DNA Substrates. 1. Binding and Footprinting Studies with Oligonucleotides Containing a Reduced Apyrimidinic Site</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2528</span>– <span class="NLM_lpage">2536</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00008a017</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00008a017" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=1995&amp;pages=2528-2536&amp;issue=8&amp;author=S.+O%E2%80%99Handleyauthor=C.+P.+Scholesauthor=R.+P.+Cunningham&amp;title=Endonuclease+III+Interactions+with+DNA+Substrates.+1.+Binding+and+Footprinting+Studies+with+Oligonucleotides+Containing+a+Reduced+Apyrimidinic+Site&amp;doi=10.1021%2Fbi00008a017" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fbi00008a017&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00008a017%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Handley%26aufirst%3DS.%26aulast%3DScholes%26aufirst%3DC.%2BP.%26aulast%3DCunningham%26aufirst%3DR.%2BP.%26atitle%3DEndonuclease%2520III%2520Interactions%2520with%2520DNA%2520Substrates.%25201.%2520Binding%2520and%2520Footprinting%2520Studies%2520with%2520Oligonucleotides%2520Containing%2520a%2520Reduced%2520Apyrimidinic%2520Site%26jtitle%3DBiochemistry%26date%3D1995%26volume%3D34%26issue%3D8%26spage%3D2528%26epage%3D2536%26doi%3D10.1021%2Fbi00008a017" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Handley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J.</span></span> <span> </span><span class="NLM_article-title">Atomic Structure of the DNA Repair [4Fe-4S] Enzyme Endonuclease III</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>258</i></span> (<span class="NLM_issue">5081</span>),  <span class="NLM_fpage">434</span>– <span class="NLM_lpage">440</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.1411536</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.1411536" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1411536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK3sXns1Smsw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=258&amp;publication_year=1992&amp;pages=434-440&amp;issue=5081&amp;author=C.+Kuoauthor=D.+McReeauthor=C.+Fisherauthor=S.+O%E2%80%99Handleyauthor=R.+Cunninghamauthor=J.+Tainer&amp;title=Atomic+Structure+of+the+DNA+Repair+%5B4Fe-4S%5D+Enzyme+Endonuclease+III&amp;doi=10.1126%2Fscience.1411536" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Atomic structure of the DNA repair [4Fe-4S] enzyme endonuclease III</span></div><div class="casAuthors">Kuo, Che Fu; McRee, Duncan E.; Fisher, Cindy L.; O'Handley, Suzanne F.; Cunningham, Richard P.; Tainer, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">258</span>
        (<span class="NLM_cas:issue">5081</span>),
    <span class="NLM_cas:pages">434-40</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The crystal structure of the DNA repair enzyme endonuclease III, which recognizes and cleaves DNA at damaged bases, has been solved to 2.0 angstrom resoln. with an R factor of 0.185.  This iron-sulfur [4Fe-4S] enzyme is elongated and bilobal with a deep cleft sepg. two similarly sized domains: a novel, sequence-continuous, six-helix domain (residues 22 to 132) and a Greek-key, four-helix domain formed by the amino-terminal and three carboxyl-terminal helices (residues 1 to 21 and 133 to 211) together with the [4Fe-4S] cluster.  The cluster is bound entirely within the carboxyl-terminal loop with a ligation pattern (Cys-X6-Cys-X2-Cys-X5-Cys) distinct from all other known [4Fe-4S] proteins.  Sequence conservation and the pos. electrostatic potential of conserved regions identify a surface suitable for binding duplex B-DNA across the long axis of the enzyme, matching a 46 angstrom length of protected DNA.  The primary role of the [4Fe-4S] cluster appears to involve positioning conserved basic residues for interaction with the DNA phosphate backbone.  The crystallog. identified inhibitor binding region, which recognizes the damaged base thymine glycol, is a seven-residue β-hairpin (residues 113 to 119).  Location and side chain orientation at the base of the inhibitor binding site implicate Glu112 in the N-glycosylase mechanism and Lys120 in the β-elimination mechanism.  Overall, the structure reveals an unusual fold and a new biol. function for [4Fe-4S] clusters and provides a structural basis for studying recognition of damaged DNA and the N-glycosylase and apurinic/apyrimidinic-lyase mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp46-I_W5BWR7Vg90H21EOLACvtfcHk0lji7JeiBlGDXQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3sXns1Smsw%253D%253D&amp;md5=1333604d2d5452b8b7f8caa3aefabc91</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1126%2Fscience.1411536&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1411536%26sid%3Dliteratum%253Aachs%26aulast%3DKuo%26aufirst%3DC.%26aulast%3DMcRee%26aufirst%3DD.%26aulast%3DFisher%26aufirst%3DC.%26aulast%3DO%25E2%2580%2599Handley%26aufirst%3DS.%26aulast%3DCunningham%26aufirst%3DR.%26aulast%3DTainer%26aufirst%3DJ.%26atitle%3DAtomic%2520Structure%2520of%2520the%2520DNA%2520Repair%2520%255B4Fe-4S%255D%2520Enzyme%2520Endonuclease%2520III%26jtitle%3DScience%26date%3D1992%26volume%3D258%26issue%3D5081%26spage%3D434%26epage%3D440%26doi%3D10.1126%2Fscience.1411536" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yavin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Repair Glycosylases with a [4Fe-4S] Cluster: A Redox Cofactor for DNA-mediated Charge Transport?</span>. <i>J. Inorg. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>101</i></span> (<span class="NLM_issue">11–12</span>),  <span class="NLM_fpage">1913</span>– <span class="NLM_lpage">1921</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.jinorgbio.2007.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.jinorgbio.2007.05.001" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=101&amp;publication_year=2007&amp;pages=1913-1921&amp;issue=11%E2%80%9312&amp;author=A.+K.+Boalauthor=E.+Yavinauthor=J.+K.+Barton&amp;title=DNA+Repair+Glycosylases+with+a+%5B4Fe-4S%5D+Cluster%3A+A+Redox+Cofactor+for+DNA-mediated+Charge+Transport%3F&amp;doi=10.1016%2Fj.jinorgbio.2007.05.001" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.jinorgbio.2007.05.001&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jinorgbio.2007.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DBoal%26aufirst%3DA.%2BK.%26aulast%3DYavin%26aufirst%3DE.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Repair%2520Glycosylases%2520with%2520a%2520%255B4Fe-4S%255D%2520Cluster%253A%2520A%2520Redox%2520Cofactor%2520for%2520DNA-mediated%2520Charge%2520Transport%253F%26jtitle%3DJ.%2520Inorg.%2520Biochem.%26date%3D2007%26volume%3D101%26issue%3D11%25E2%2580%259312%26spage%3D1913%26epage%3D1921%26doi%3D10.1016%2Fj.jinorgbio.2007.05.001" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuss, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Q. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arvai, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span> <span> </span><span class="NLM_article-title">XPD Helicase Structures and Activities: Insights into the Cancer and Aging Phenotypes from XPD Mutations</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>133</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">789</span>– <span class="NLM_lpage">800</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2008.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.cell.2008.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=18510924" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD1cXntFaltrk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=133&amp;publication_year=2008&amp;pages=789-800&amp;issue=5&amp;author=L.+Fanauthor=J.+O.+Fussauthor=Q.+J.+Chengauthor=A.+S.+Arvaiauthor=M.+Hammelauthor=V.+A.+Robertsauthor=P.+K.+Cooperauthor=J.+A.+Tainer&amp;title=XPD+Helicase+Structures+and+Activities%3A+Insights+into+the+Cancer+and+Aging+Phenotypes+from+XPD+Mutations&amp;doi=10.1016%2Fj.cell.2008.04.030" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">XPD helicase structures and activities: insights into the cancer and aging phenotypes from XPD mutations</span></div><div class="casAuthors">Fan, Li; Fuss, Jill O.; Cheng, Quen J.; Arvai, Andrew S.; Hammel, Michael; Roberts, Victoria A.; Cooper, Priscilla K.; Tainer, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">789-800</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mutations in XPD helicase, required for nucleotide excision repair (NER) as part of the transcription/repair complex TFIIH, cause three distinct phenotypes: cancer-prone xeroderma pigmentosum (XP), or aging disorders Cockayne syndrome (CS), and trichothiodystrophy (TTD).  To clarify mol. differences underlying these diseases, we detd. crystal structures of the XPD catalytic core from Sulfolobus acidocaldarius and measured mutant enzyme activities.  Substrate-binding grooves sep. adjacent Rad51/RecA-like helicase domains (HD1, HD2) and an arch formed by 4FeS and Arch domains.  XP mutations map along the HD1 ATP-binding edge and HD2 DNA-binding channel and impair helicase activity essential for NER.  XP/CS mutations both impair helicase activity and likely affect HD2 functional movement.  TTD mutants lose or retain helicase activity but map to sites in all four domains expected to cause framework defects impacting TFIIH integrity.  These results provide a foundation for understanding disease consequences of mutations in XPD and related 4Fe-4S helicases including FancJ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4p4pSDz59pLVg90H21EOLACvtfcHk0lji7JeiBlGDXQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXntFaltrk%253D&amp;md5=997e8595afd3001a6a3134c143ff10b4</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2008.04.030&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2008.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DL.%26aulast%3DFuss%26aufirst%3DJ.%2BO.%26aulast%3DCheng%26aufirst%3DQ.%2BJ.%26aulast%3DArvai%26aufirst%3DA.%2BS.%26aulast%3DHammel%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DV.%2BA.%26aulast%3DCooper%26aufirst%3DP.%2BK.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26atitle%3DXPD%2520Helicase%2520Structures%2520and%2520Activities%253A%2520Insights%2520into%2520the%2520Cancer%2520and%2520Aging%2520Phenotypes%2520from%2520XPD%2520Mutations%26jtitle%3DCell%26date%3D2008%26volume%3D133%26issue%3D5%26spage%3D789%26epage%3D800%26doi%3D10.1016%2Fj.cell.2008.04.030" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lukianova, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span> <span> </span><span class="NLM_article-title">A Role for Iron–sulfur Clusters in DNA Repair</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">151</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cbpa.2005.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.cbpa.2005.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=15811798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivFSmu7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2005&amp;pages=145-151&amp;issue=2&amp;author=O.+A.+Lukianovaauthor=S.+S.+David&amp;title=A+Role+for+Iron%E2%80%93sulfur+Clusters+in+DNA+Repair&amp;doi=10.1016%2Fj.cbpa.2005.02.006" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A role for iron-sulfur clusters in DNA repair</span></div><div class="casAuthors">Lukianova, Olga A.; David, Sheila S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">145-151</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The presence of 4Fe-4S clusters in enzymes involved in DNA repair has posed the question of the role of these intricate cofactors in damaged DNA recognition and repair.  It is particularly intriguing that base excision repair glycosylases that remove a wide variety of damaged bases, and also have vastly different sequences and structures, have been found to contain this cofactor.  The accumulating biochem. and structural evidence indicates that the region supported by the cluster is intimately involved in DNA binding, and that such binding interactions impact catalysis of base removal.  Recent evidence has also established that binding of the glycosylases to DNA facilitates oxidn. of the [4Fe-4S]2+ cluster to the [4Fe-4S]3+ form.  Notably, the measured redox potentials for a variety of 4Fe-4S cluster-contg. glycosylases are remarkably similar.  Based on this DNA-mediated redox behavior, it has been suggested that this property may be used to enhance the activity of these enzymes by facilitating damaged DNA location.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-8OnHpqYlQbVg90H21EOLACvtfcHk0lhJXAMgrCGDbg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXivFSmu7c%253D&amp;md5=62710de60d4903a9ba812dd7179e14a5</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2005.02.006&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2005.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DLukianova%26aufirst%3DO.%2BA.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26atitle%3DA%2520Role%2520for%2520Iron%25E2%2580%2593sulfur%2520Clusters%2520in%2520DNA%2520Repair%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D9%26issue%3D2%26spage%3D145%26epage%3D151%26doi%3D10.1016%2Fj.cbpa.2005.02.006" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouault, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, W.-H.</span></span> <span> </span><span class="NLM_article-title">Iron-sulphur Cluster Biogenesis and Mitochondrial Iron Homeostasis</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">345</span>– <span class="NLM_lpage">351</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nrm1620</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnrm1620" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=15803140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivVaktb4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2005&amp;pages=345-351&amp;issue=4&amp;author=T.+A.+Rouaultauthor=W.-H.+Tong&amp;title=Iron-sulphur+Cluster+Biogenesis+and+Mitochondrial+Iron+Homeostasis&amp;doi=10.1038%2Fnrm1620" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: Iron-sulphur cluster biogenesis and mitochondrial iron homeostasis</span></div><div class="casAuthors">Rouault, Tracey A.; Tong, Wing-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">345-351</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Iron-sulfur clusters are important cofactors for proteins that are involved in many cellular processes, including electron transport, enzymic catalysis and regulation.  The enzymes that catalyze the formation of iron-sulfur clusters are widely conserved from bacteria to humans.  Recent studies in model systems and humans reveal that iron-sulfur proteins have important roles in mitochondrial iron homeostasis and in the pathogenesis of the human disease Friedreich ataxia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD6B2xuSAXdLVg90H21EOLACvtfcHk0lhJXAMgrCGDbg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXivVaktb4%253D&amp;md5=159664e92102e2d50b85f549a858ab98</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnrm1620&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1620%26sid%3Dliteratum%253Aachs%26aulast%3DRouault%26aufirst%3DT.%2BA.%26aulast%3DTong%26aufirst%3DW.-H.%26atitle%3DIron-sulphur%2520Cluster%2520Biogenesis%2520and%2520Mitochondrial%2520Iron%2520Homeostasis%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26issue%3D4%26spage%3D345%26epage%3D351%26doi%3D10.1038%2Fnrm1620" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouault, T.
A.</span></span> <span> </span><span class="NLM_article-title">Mammalian Iron-sulphur Proteins: Novel Insights into Biogenesis and Function</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">55</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nrm3909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnrm3909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=25425402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitValurzO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=2015&amp;pages=45-55&amp;issue=1&amp;author=T.%0AA.+Rouault&amp;title=Mammalian+Iron-sulphur+Proteins%3A+Novel+Insights+into+Biogenesis+and+Function&amp;doi=10.1038%2Fnrm3909" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian iron-sulfur proteins: novel insights into biogenesis and function</span></div><div class="casAuthors">Rouault, Tracey A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-55</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fe-S clusters are inorg. cofactors that are found in nearly all species and are composed of various combinations of Fe and S atoms.  Fe-S clusters can accept or donate single electrons to carry out oxidn. and redn. reactions and to facilitate electron transport.  Many details of how these complex modular structures are assembled and ligated to cellular proteins in the mitochondrial, nuclear, and cytosolic compartments of mammalian cells remain unclear.  Recent evidence indicates that a Leu-Tyr-Arg (LYR) tripeptide motif found in some Fe-S recipient proteins may facilitate the direct and shielded transfer of Fe-S clusters from a scaffold to client proteins.  Fe-S clusters are probably an unrecognized and elusive cofactor of many known proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqdvbYH-jqfrVg90H21EOLACvtfcHk0lhJXAMgrCGDbg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXitValurzO&amp;md5=a9250f89d32139aa70d8193cf72f9b95</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnrm3909&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3909%26sid%3Dliteratum%253Aachs%26aulast%3DRouault%26aufirst%3DT.%2BA.%26atitle%3DMammalian%2520Iron-sulphur%2520Proteins%253A%2520Novel%2520Insights%2520into%2520Biogenesis%2520and%2520Function%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D16%26issue%3D1%26spage%3D45%26epage%3D55%26doi%3D10.1038%2Fnrm3909" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veatch, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMurray, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottschling, D. E.</span></span> <span> </span><span class="NLM_article-title">Mitochondrial Dysfunction Leads to Nuclear Genome Instability via an Iron-Sulfur Cluster Defect</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>137</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1258</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cell.2009.04.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.cell.2009.04.014" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=137&amp;publication_year=2009&amp;pages=1247-1258&amp;issue=7&amp;author=J.+R.+Veatchauthor=M.+A.+McMurrayauthor=Z.+W.+Nelsonauthor=D.+E.+Gottschling&amp;title=Mitochondrial+Dysfunction+Leads+to+Nuclear+Genome+Instability+via+an+Iron-Sulfur+Cluster+Defect&amp;doi=10.1016%2Fj.cell.2009.04.014" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.04.014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.04.014%26sid%3Dliteratum%253Aachs%26aulast%3DVeatch%26aufirst%3DJ.%2BR.%26aulast%3DMcMurray%26aufirst%3DM.%2BA.%26aulast%3DNelson%26aufirst%3DZ.%2BW.%26aulast%3DGottschling%26aufirst%3DD.%2BE.%26atitle%3DMitochondrial%2520Dysfunction%2520Leads%2520to%2520Nuclear%2520Genome%2520Instability%2520via%2520an%2520Iron-Sulfur%2520Cluster%2520Defect%26jtitle%3DCell%26date%3D2009%26volume%3D137%26issue%3D7%26spage%3D1247%26epage%3D1258%26doi%3D10.1016%2Fj.cell.2009.04.014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tse, E. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">The Oxidation State of [4Fe4S] Clusters Modulates the DNA-Binding Affinity of DNA Repair Proteins</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">12784</span>– <span class="NLM_lpage">12792</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.7b07230</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b07230" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlCqs7nP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=139&amp;publication_year=2017&amp;pages=12784-12792&amp;issue=36&amp;author=E.+C.+M.+Tseauthor=T.+J.+Zwangauthor=J.+K.+Barton&amp;title=The+Oxidation+State+of+%5B4Fe4S%5D+Clusters+Modulates+the+DNA-Binding+Affinity+of+DNA+Repair+Proteins&amp;doi=10.1021%2Fjacs.7b07230" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The Oxidation State of [4Fe4S] Clusters Modulates the DNA-Binding Affinity of DNA Repair Proteins</span></div><div class="casAuthors">Tse, Edmund C. M.; Zwang, Theodore J.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">12784-12792</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abstr.: A central question important to understanding DNA repair is how certain proteins are able to search for, detect, and fix DNA damage on a biol. relevant timescale.  A feature of many base excision repair proteins is that they contain [4Fe4S] clusters that may aid their search for lesions.  In this report, we establish the importance of the oxidn. state of the redox-active [4Fe4S] cluster in the DNA damage detection process.  We utilize DNA-modified electrodes to generate repair proteins with [4Fe4S] clusters in the 2+ and 3+ states by bulk electrolysis under an O2-free atm.  Anaerobic microscale thermophoresis results indicate that proteins carrying [4Fe4S]3+ clusters bind to DNA 550 times more tightly than those with [4Fe4S]2+ clusters.  The measured increase in DNA-binding affinity matches the affinity change assocd. with the redox potential shift obsd. for [4Fe4S] cluster proteins upon binding to DNA.  We further devise an electrostatic model that shows this change in DNA-binding affinity of these proteins can be fully explained by the differences in electrostatic interactions between DNA and the [4Fe4S] cluster in the reduced vs. oxidized state.  We then utilize at. force microscopy (AFM) to demonstrate that the redox state of the [4Fe4S] clusters regulates the ability of two DNA repair proteins, Endonuclease III and DinG, to bind preferentially to DNA duplexes contg. a single site of DNA damage (here a base mismatch) which inhibits DNA charge transport.  Together, these results show that the redn. and oxidn. of [4Fe4S] clusters through DNA-mediated charge transport facilitates long-range signaling between [4Fe4S] repair proteins.  The redox-modulated change in DNA-binding affinity regulates the ability of [4Fe4S] repair proteins to collaborate in the lesion detection process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMG3dpTH_YD7Vg90H21EOLACvtfcHk0lgbxHmjmf7sEg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhtlCqs7nP&amp;md5=e9c6e823b9111ea182c82817bcad4ce9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b07230&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b07230%26sid%3Dliteratum%253Aachs%26aulast%3DTse%26aufirst%3DE.%2BC.%2BM.%26aulast%3DZwang%26aufirst%3DT.%2BJ.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DThe%2520Oxidation%2520State%2520of%2520%255B4Fe4S%255D%2520Clusters%2520Modulates%2520the%2520DNA-Binding%2520Affinity%2520of%2520DNA%2520Repair%2520Proteins%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26issue%3D36%26spage%3D12784%26epage%3D12792%26doi%3D10.1021%2Fjacs.7b07230" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gorodetsky, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Direct Electrochemistry of Endonuclease III in the Presence and Absence of DNA</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span> (<span class="NLM_issue">37</span>),  <span class="NLM_fpage">12082</span>– <span class="NLM_lpage">12083</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja064784d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja064784d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=128&amp;publication_year=2006&amp;pages=12082-12083&amp;issue=37&amp;author=A.+A.+Gorodetskyauthor=A.+K.+Boalauthor=J.+K.+Barton&amp;title=Direct+Electrochemistry+of+Endonuclease+III+in+the+Presence+and+Absence+of+DNA&amp;doi=10.1021%2Fja064784d" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja064784d&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja064784d%26sid%3Dliteratum%253Aachs%26aulast%3DGorodetsky%26aufirst%3DA.%2BA.%26aulast%3DBoal%26aufirst%3DA.%2BK.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDirect%2520Electrochemistry%2520of%2520Endonuclease%2520III%2520in%2520the%2520Presence%2520and%2520Absence%2520of%2520DNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26issue%3D37%26spage%3D12082%26epage%3D12083%26doi%3D10.1021%2Fja064784d" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartels, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuñez, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespilho, F. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Electrochemistry of the [4Fe4S] Cluster in Base Excision Repair Proteins: Tuning the Redox Potential with DNA</span>. <i>Langmuir</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2523</span>– <span class="NLM_lpage">2530</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.langmuir.6b04581</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.langmuir.6b04581" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVaiur0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=33&amp;publication_year=2017&amp;pages=2523-2530&amp;issue=10&amp;author=P.+L.+Bartelsauthor=A.+Zhouauthor=A.+R.+Arnoldauthor=N.+N.+Nu%C3%B1ezauthor=F.+N.+Crespilhoauthor=S.+S.+Davidauthor=J.+K.+Barton&amp;title=Electrochemistry+of+the+%5B4Fe4S%5D+Cluster+in+Base+Excision+Repair+Proteins%3A+Tuning+the+Redox+Potential+with+DNA&amp;doi=10.1021%2Facs.langmuir.6b04581" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Electrochemistry of the [4Fe4S] Cluster in Base Excision Repair Proteins: Tuning the Redox Potential with DNA</span></div><div class="casAuthors">Bartels, Phillip L.; Zhou, Andy; Arnold, Anna R.; Nunez, Nicole N.; Crespilho, Frank N.; David, Sheila S.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Langmuir</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2523-2530</span>CODEN:
                <span class="NLM_cas:coden">LANGD5</span>;
        ISSN:<span class="NLM_cas:issn">0743-7463</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Escherichia coli endonuclease III (EndoIII) and MutY are DNA glycosylases that contain [4Fe4S] clusters and that serve to maintain the integrity of the genome after oxidative stress.  Electrochem. studies on highly-oriented pyrolytic graphite (HOPG) revealed that DNA binding by EndoIII leads to a large neg. shift in midpoint potential of the cluster, consistent with stabilization of the oxidized [4Fe4S]3+ form.  However, the smooth, hydrophobic HOPG surface is non-ideal for working with proteins in the absence of DNA.  Here, we used thin film voltammetry on a pyrolytic graphite edge electrode to overcome these limitations.  Improved adsorption led to substantial signals for both EndoIII and MutY in the absence of DNA, and a large neg. potential shift was retained with DNA present.  In contrast, EndoIII mutants E200K, Y205H, and K208E, which provide electrostatic perturbations in the vicinity of the cluster, all showed DNA-free potentials within error of wild type; similarly, the presence of neg. charged poly-L-glutamate did not lead to a significant potential shift.  Overall, binding to the DNA polyanion was the dominant effect in tuning the redox potential of the [4Fe4S] cluster, helping to explain why all DNA-binding proteins with [4Fe4S] clusters studied to date have similar DNA-bound potentials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoebVWlyKmJZLVg90H21EOLACvtfcHk0lgbxHmjmf7sEg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXjtVaiur0%253D&amp;md5=a7984edceec68ce867d1e3a3eecdaa7d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.langmuir.6b04581&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.langmuir.6b04581%26sid%3Dliteratum%253Aachs%26aulast%3DBartels%26aufirst%3DP.%2BL.%26aulast%3DZhou%26aufirst%3DA.%26aulast%3DArnold%26aufirst%3DA.%2BR.%26aulast%3DNu%25C3%25B1ez%26aufirst%3DN.%2BN.%26aulast%3DCrespilho%26aufirst%3DF.%2BN.%26aulast%3DDavid%26aufirst%3DS.%2BS.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DElectrochemistry%2520of%2520the%2520%255B4Fe4S%255D%2520Cluster%2520in%2520Base%2520Excision%2520Repair%2520Proteins%253A%2520Tuning%2520the%2520Redox%2520Potential%2520with%2520DNA%26jtitle%3DLangmuir%26date%3D2017%26volume%3D33%26issue%3D10%26spage%3D2523%26epage%3D2530%26doi%3D10.1021%2Facs.langmuir.6b04581" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chemler, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartels, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marvin, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raskin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, D. H.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">A human MUTYH variant linking colonic polyposis to redox degradation of the [4Fe4S]2+ cluster</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">873</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41557-018-0068-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fs41557-018-0068-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=29915346" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKgsL7O" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2018&amp;pages=873&amp;issue=8&amp;author=K.+J.+McDonnellauthor=J.+A.+Chemlerauthor=P.+L.+Bartelsauthor=E.+O%E2%80%99Brienauthor=M.+L.+Marvinauthor=J.+Ortegaauthor=R.+H.+Sternauthor=L.+Raskinauthor=G.-M.+Liauthor=D.+H.+Sherman&amp;title=A+human+MUTYH+variant+linking+colonic+polyposis+to+redox+degradation+of+the+%5B4Fe4S%5D2%2B+cluster&amp;doi=10.1038%2Fs41557-018-0068-x" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A human MUTYH variant linking colonic polyposis to redox degradation of the [4Fe4S]2+ cluster</span></div><div class="casAuthors">McDonnell, Kevin J.; Chemler, Joseph A.; Bartels, Phillip L.; O'Brien, Elizabeth; Marvin, Monica L.; Ortega, Janice; Stern, Ralph H.; Raskin, Leon; Li, Guo-Min; Sherman, David H.; Barton, Jacqueline K.; Gruber, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">873-880</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The human DNA repair enzyme MUTYH excises mispaired adenine residues in oxidized DNA.  Homozygous MUTYH mutations underlie the autosomal, recessive cancer syndrome MUTYH-assocd. polyposis.  We report a MUTYH variant, p.  C306W (c.918C&gt;G), with a tryptophan residue in place of native cysteine, that ligates the [4Fe4S] cluster in a patient with colonic polyposis and family history of early age colon cancer.  In bacterial MutY, the [4Fe4S] cluster is redox active, allowing rapid localization to target lesions by long-range, DNA-mediated signaling.  In the current study, using DNA electrochem., we det. that wild-type MUTYH is similarly redox-active, but MUTYH C306W undergoes rapid oxidative degrdn. of its cluster to [3Fe4S]+, with loss of redox signaling.  In MUTYH C306W, oxidative cluster degrdn. leads to decreased DNA binding and enzyme function.  This study confirms redox activity in eukaryotic DNA repair proteins and establishes MUTYH C306W as a pathogenic variant, highlighting the essential role of redox signaling by the [4Fe4S] cluster.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqd1vLIhEj1bVg90H21EOLACvtfcHk0lhfsIRpUHF0oA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKgsL7O&amp;md5=4ce582747197797232e56d1eec995161</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs41557-018-0068-x&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-018-0068-x%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonnell%26aufirst%3DK.%2BJ.%26aulast%3DChemler%26aufirst%3DJ.%2BA.%26aulast%3DBartels%26aufirst%3DP.%2BL.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DE.%26aulast%3DMarvin%26aufirst%3DM.%2BL.%26aulast%3DOrtega%26aufirst%3DJ.%26aulast%3DStern%26aufirst%3DR.%2BH.%26aulast%3DRaskin%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DG.-M.%26aulast%3DSherman%26aufirst%3DD.%2BH.%26atitle%3DA%2520human%2520MUTYH%2520variant%2520linking%2520colonic%2520polyposis%2520to%2520redox%2520degradation%2520of%2520the%2520%255B4Fe4S%255D2%252B%2520cluster%26jtitle%3DNat.%2520Chem.%26date%3D2018%26volume%3D10%26issue%3D8%26spage%3D873%26doi%3D10.1038%2Fs41557-018-0068-x" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghatlia, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turro, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J.</span></span> <span> </span><span class="NLM_article-title">Long-range photoinduced electron transfer through a DNA helix</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>262</i></span> (<span class="NLM_issue">5136</span>),  <span class="NLM_fpage">1025</span>– <span class="NLM_lpage">1029</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.7802858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.7802858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=7802858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK2cXhtVSqtrg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=262&amp;publication_year=1993&amp;pages=1025-1029&amp;issue=5136&amp;author=C.+Murphyauthor=M.+Arkinauthor=Y.+Jenkinsauthor=N.+Ghatliaauthor=S.+Bossmannauthor=N.+Turroauthor=J.+Barton&amp;title=Long-range+photoinduced+electron+transfer+through+a+DNA+helix&amp;doi=10.1126%2Fscience.7802858" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Long-range photoinduced electron transfer through a DNA helix</span></div><div class="casAuthors">Murphy, C. J.; Arkin, M. R.; Jenkins, Y.; Ghatlia, N. D.; Bossmann, S. H.; Turro, N. J.; Barton, J. K.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">262</span>
        (<span class="NLM_cas:issue">5136</span>),
    <span class="NLM_cas:pages">1025-9</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">Rapid photoinduced electron transfer is demonstrated over a distance of &gt;40 angstroms between metallointercalators that are tethered to the 5' termini of a 15-base-pair DNA duplex.  An oligomeric assembly was synthesized in which the donor is Ru(phen)2dppz2+ (phen, phenanthroline, and dppz, dipyridophenazine) and the acceptor is Rh(phi)2phen3+ (phi, phenanthrenequinone diimine).  These metal complexes are intercalated either one or two base steps in from the helix termini.  Although the Ru-modified oligonucleotide hybridized to an unmodified complement luminesces intensely, the Ru-modified oligomer hybridized to the Rh-modified oligomer shows no detectable luminescence.  Time-resolved studies point to a lower limit of 109 per s for the quenching rate.  No quenching was obsd. upon metalation of 2 complementary octamers by Ru(phen)32+ and Rh(phen)33+ under conditions where the phen complexes do not intercalate.  The stacked arom. heterocycles of the DNA duplex therefore serve as an efficient medium for coupling electron donors and acceptors over very long distances.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5jM1emtEmcrVg90H21EOLACvtfcHk0lhfsIRpUHF0oA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2cXhtVSqtrg%253D&amp;md5=e15c6acf6047c76aa7bddb0c1edc86a2</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.7802858&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.7802858%26sid%3Dliteratum%253Aachs%26aulast%3DMurphy%26aufirst%3DC.%26aulast%3DArkin%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DY.%26aulast%3DGhatlia%26aufirst%3DN.%26aulast%3DBossmann%26aufirst%3DS.%26aulast%3DTurro%26aufirst%3DN.%26aulast%3DBarton%26aufirst%3DJ.%26atitle%3DLong-range%2520photoinduced%2520electron%2520transfer%2520through%2520a%2520DNA%2520helix%26jtitle%3DScience%26date%3D1993%26volume%3D262%26issue%3D5136%26spage%3D1025%26epage%3D1029%26doi%3D10.1126%2Fscience.7802858" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiebig, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treadway, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zewail, A. H.</span></span> <span> </span><span class="NLM_article-title">Femtosecond dynamics of DNA-mediated electron transfer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">6014</span>– <span class="NLM_lpage">6019</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.96.11.6014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1073%2Fpnas.96.11.6014" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=96&amp;publication_year=1999&amp;pages=6014-6019&amp;issue=11&amp;author=C.+Wanauthor=T.+Fiebigauthor=S.+O.+Kelleyauthor=C.+R.+Treadwayauthor=J.+K.+Bartonauthor=A.+H.+Zewail&amp;title=Femtosecond+dynamics+of+DNA-mediated+electron+transfer&amp;doi=10.1073%2Fpnas.96.11.6014" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.11.6014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.11.6014%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DC.%26aulast%3DFiebig%26aufirst%3DT.%26aulast%3DKelley%26aufirst%3DS.%2BO.%26aulast%3DTreadway%26aufirst%3DC.%2BR.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26aulast%3DZewail%26aufirst%3DA.%2BH.%26atitle%3DFemtosecond%2520dynamics%2520of%2520DNA-mediated%2520electron%2520transfer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26issue%3D11%26spage%3D6014%26epage%3D6019%26doi%3D10.1073%2Fpnas.96.11.6014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stemp, E. D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmlin, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hörmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbara, P. F.</span></span> <span> </span><span class="NLM_article-title">Rates of DNA-Mediated Electron Transfer Between Metallointercalators</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>273</i></span> (<span class="NLM_issue">5274</span>),  <span class="NLM_fpage">475</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.273.5274.475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.273.5274.475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=8662532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK28Xks1Wmurk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=273&amp;publication_year=1996&amp;pages=475&amp;issue=5274&amp;author=M.+R.+Arkinauthor=E.+D.+A.+Stempauthor=R.+E.+Holmlinauthor=J.+K.+Bartonauthor=A.+H%C3%B6rmannauthor=E.+J.+C.+Olsonauthor=P.+F.+Barbara&amp;title=Rates+of+DNA-Mediated+Electron+Transfer+Between+Metallointercalators&amp;doi=10.1126%2Fscience.273.5274.475" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Rates of DNA-mediated electron transfer between metallointercalators</span></div><div class="casAuthors">Arkin, M. R.; Stemp, E. D. A.; Holmlin, R. E.; Barton, J. K.; Hormann, A.; Olson, E. J. C.; Barbara, P. F.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">5274</span>),
    <span class="NLM_cas:pages">475-480</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Ultrafast emission and absorption spectroscopies were used to measure the kinetics of DNA-mediated electron transfer reactions between metal complexes intercalated into DNA.  In the presence of rhodium(III) acceptor, a substantial fraction of photoexcited donor exhibits fast oxidative quenching (&gt;3×1010 per s).  Transient-absorption expts. indicate that, for a series of donors, the majority of back electron transfer is also very fast (∼1010 per s).  This rate is independent of the loading of acceptors on the helix, but is sensitive to sequence and π stacking.  The cooperative binding of donor and acceptor is considered unlikely on the basis of structural models and DNA photocleavage studies of binding.  The data show that the DNA double helix differs significantly from proteins as a bridge for electron transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryvkrg-vpLPrVg90H21EOLACvtfcHk0lhfsIRpUHF0oA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK28Xks1Wmurk%253D&amp;md5=6906953202b7ae2208c7bd17a62b301d</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1126%2Fscience.273.5274.475&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.273.5274.475%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DStemp%26aufirst%3DE.%2BD.%2BA.%26aulast%3DHolmlin%26aufirst%3DR.%2BE.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26aulast%3DH%25C3%25B6rmann%26aufirst%3DA.%26aulast%3DOlson%26aufirst%3DE.%2BJ.%2BC.%26aulast%3DBarbara%26aufirst%3DP.%2BF.%26atitle%3DRates%2520of%2520DNA-Mediated%2520Electron%2520Transfer%2520Between%2520Metallointercalators%26jtitle%3DScience%26date%3D1996%26volume%3D273%26issue%3D5274%26spage%3D475%26doi%3D10.1126%2Fscience.273.5274.475" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sontz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muren, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Charge Transport for Sensing and Signaling</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1792</span>– <span class="NLM_lpage">1800</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar3001298</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar3001298" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=2012&amp;pages=1792-1800&amp;issue=10&amp;author=P.+A.+Sontzauthor=N.+B.+Murenauthor=J.+K.+Barton&amp;title=DNA+Charge+Transport+for+Sensing+and+Signaling&amp;doi=10.1021%2Far3001298" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Far3001298&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far3001298%26sid%3Dliteratum%253Aachs%26aulast%3DSontz%26aufirst%3DP.%2BA.%26aulast%3DMuren%26aufirst%3DN.%2BB.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Charge%2520Transport%2520for%2520Sensing%2520and%2520Signaling%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2012%26volume%3D45%26issue%3D10%26spage%3D1792%26epage%3D1800%26doi%3D10.1021%2Far3001298" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merino, E.
J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Biological Contexts for DNA Charge Transport Chemistry</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">229</span>– <span class="NLM_lpage">237</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cbpa.2008.01.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.cbpa.2008.01.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=18314014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslKlsbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2008&amp;pages=229-237&amp;issue=2&amp;author=E.%0AJ.+Merinoauthor=A.+K.+Boalauthor=J.+K.+Barton&amp;title=Biological+Contexts+for+DNA+Charge+Transport+Chemistry&amp;doi=10.1016%2Fj.cbpa.2008.01.046" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Biological contexts for DNA charge transport chemistry</span></div><div class="casAuthors">Merino, Edward J.; Boal, Amie K.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">229-237</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Many expts. have now shown that double helical DNA can serve as a conduit for efficient charge transport (CT) reactions over long distances in vitro.  These results prompt the consideration of biol. roles for DNA-mediated CT.  DNA CT has been demonstrated to occur in biol. relevant environments such as within the mitochondria and nuclei of HeLa cells as well as in isolated nucleosomes.  In mitochondria, DNA damage that results from CT is funneled to a crucial regulatory element.  Thus, DNA CT provides a strategy to funnel damage to particular sites in the genome.  DNA CT might also be important in long-range signaling to DNA-bound proteins.  Both DNA repair proteins, contg. Fe-S clusters, and the transcription factor, p53, which is regulated through thiol-disulfide switches, can be oxidized from a distance through DNA-mediated CT.  These observations highlight a means through which oxidative stress may be chem. signaled in the genome over long distances through CT from guanine radicals to DNA-bound proteins.  Moreover, DNA-mediated CT may also play a role in signaling among DNA-binding proteins, as has been proposed as a mechanism for how DNA repair glycosylases more efficiently detect lesions inside the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_JiLip4xOhLVg90H21EOLACvtfcHk0lhHzXTg_5CaCw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXlslKlsbc%253D&amp;md5=76215e9a6b925424dfd6ed7615a67739</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2008.01.046&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2008.01.046%26sid%3Dliteratum%253Aachs%26aulast%3DMerino%26aufirst%3DE.%2BJ.%26aulast%3DBoal%26aufirst%3DA.%2BK.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DBiological%2520Contexts%2520for%2520DNA%2520Charge%2520Transport%2520Chemistry%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D12%26issue%3D2%26spage%3D229%26epage%3D237%26doi%3D10.1016%2Fj.cbpa.2008.01.046" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grodick, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muren, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Charge Transport within the Cell</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">962</span>– <span class="NLM_lpage">973</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi501520w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi501520w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlyrs7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=54&amp;publication_year=2015&amp;pages=962-973&amp;issue=4&amp;author=M.+A.+Grodickauthor=N.+B.+Murenauthor=J.+K.+Barton&amp;title=DNA+Charge+Transport+within+the+Cell&amp;doi=10.1021%2Fbi501520w" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">DNA charge transport within the cell</span></div><div class="casAuthors">Grodick, Michael A.; Muren, Natalie B.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">962-973</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The unique characteristics of DNA charge transport (CT) have prompted an examn. of roles for this chem. within a biol. context.  Not only can DNA CT facilitate long-range oxidative damage of DNA, but redox-active proteins can couple to the DNA base stack and participate in long-range redox reactions using DNA CT.  DNA transcription factors with redox-active moieties such as SoxR and p53 can use DNA CT as a form of redox sensing.  DNA CT chem. also provides a means to monitor the integrity of the DNA, given the sensitivity of DNA CT to perturbations in base stacking as arise with mismatches and lesions.  Enzymes that utilize this chem. include an interesting and ever-growing class of DNA-processing enzymes involved in DNA repair, replication, and transcription that have been found to contain 4Fe-4S clusters.  DNA repair enzymes contg. 4Fe-4S clusters, that include endonuclease III (EndoIII), MutY, and DinG from bacteria, as well as XPD from archaea, have been shown to be redox-active when bound to DNA, share a DNA-bound redox potential, and can be reduced and oxidized at long-range via DNA CT.  Interactions between DNA and these proteins in soln., in addn. to genetics expts. within Escherichia coli, suggest that DNA-mediated CT can be used as a means of cooperative signaling among DNA repair proteins that contain 4Fe-4S clusters as a 1st step in finding DNA damage, even within cells.  On the basis of these data, one can consider also how DNA-mediated CT may be used as a means of signaling to coordinate DNA processing across the genome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofZd49KeRWhLVg90H21EOLACvtfcHk0lhHzXTg_5CaCw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXhtlyrs7g%253D&amp;md5=11008f93a6f8dd195abb82ea3c54e754</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fbi501520w&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi501520w%26sid%3Dliteratum%253Aachs%26aulast%3DGrodick%26aufirst%3DM.%2BA.%26aulast%3DMuren%26aufirst%3DN.%2BB.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Charge%2520Transport%2520within%2520the%2520Cell%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26issue%3D4%26spage%3D962%26epage%3D973%26doi%3D10.1021%2Fbi501520w" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grodick, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Charge Transport: from Chemical Principles to the Cell</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">197</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.chembiol.2015.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.chembiol.2015.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=26933744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC28XntlSqsb8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2016&amp;pages=183-197&amp;issue=1&amp;author=A.+R.+Arnoldauthor=M.+A.+Grodickauthor=J.+K.+Barton&amp;title=DNA+Charge+Transport%3A+from+Chemical+Principles+to+the+Cell&amp;doi=10.1016%2Fj.chembiol.2015.11.010" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">DNA Charge Transport: from Chemical Principles to the Cell</span></div><div class="casAuthors">Arnold, Anna R.; Grodick, Michael A.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">183-197</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The DNA double helix has captured the imagination of many, bringing it to the forefront of biol. research.  DNA has unique features that extend our interest into areas of chem., physics, material science, and engineering.  Our lab. has focused on studies of DNA charge transport (CT), wherein charges can efficiently travel long mol. distances through the DNA helix while maintaining an exquisite sensitivity to base pair π-stacking.  Because DNA CT chem. reports on the integrity of the DNA duplex, this property may be exploited to develop electrochem. devices to detect DNA lesions and DNA-binding proteins.  Furthermore, studies now indicate that DNA CT may also be used in the cell by, for example, DNA repair proteins, as a cellular diagnostic, in order to scan the genome to localize efficiently to damage sites.  In this review, we describe this evolution of DNA CT chem. from the discovery of fundamental chem. principles to applications in diagnostic strategies and possible roles in biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs9YMsEG1yKLVg90H21EOLACvtfcHk0ljlhnbUzfpqYg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XntlSqsb8%253D&amp;md5=556f33e3baa00e039092f8a6697214dc</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.11.010&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DArnold%26aufirst%3DA.%2BR.%26aulast%3DGrodick%26aufirst%3DM.%2BA.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Charge%2520Transport%253A%2520from%2520Chemical%2520Principles%2520to%2520the%2520Cell%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26issue%3D1%26spage%3D183%26epage%3D197%26doi%3D10.1016%2Fj.chembiol.2015.11.010" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Núñez, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmquist, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Evidence for DNA Charge Transport in the Nucleus</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">42</span>),  <span class="NLM_fpage">12465</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi011560t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi011560t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=40&amp;publication_year=2001&amp;pages=12465&amp;issue=42&amp;author=M.+E.+N%C3%BA%C3%B1ezauthor=G.+P.+Holmquistauthor=J.+K.+Barton&amp;title=Evidence+for+DNA+Charge+Transport+in+the+Nucleus&amp;doi=10.1021%2Fbi011560t" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fbi011560t&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi011560t%26sid%3Dliteratum%253Aachs%26aulast%3DN%25C3%25BA%25C3%25B1ez%26aufirst%3DM.%2BE.%26aulast%3DHolmquist%26aufirst%3DG.%2BP.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DEvidence%2520for%2520DNA%2520Charge%2520Transport%2520in%2520the%2520Nucleus%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26issue%3D42%26spage%3D12465%26doi%3D10.1021%2Fbi011560t" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Genereux, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA-Mediated Charge Transport in Redox Sensing and Signaling</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">891</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja907669c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja907669c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=132&amp;publication_year=2010&amp;pages=891&amp;issue=3&amp;author=J.+C.+Genereuxauthor=A.+K.+Boalauthor=J.+K.+Barton&amp;title=DNA-Mediated+Charge+Transport+in+Redox+Sensing+and+Signaling&amp;doi=10.1021%2Fja907669c" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fja907669c&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja907669c%26sid%3Dliteratum%253Aachs%26aulast%3DGenereux%26aufirst%3DJ.%2BC.%26aulast%3DBoal%26aufirst%3DA.%2BK.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA-Mediated%2520Charge%2520Transport%2520in%2520Redox%2520Sensing%2520and%2520Signaling%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26issue%3D3%26spage%3D891%26doi%3D10.1021%2Fja907669c" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, R.</span>; <span class="NLM_string-name">Kondev, J.</span>; <span class="NLM_string-name">Theriot, J.</span>; <span class="NLM_string-name">Orme, N.</span></span> <i>Physical Biology of the Cell</i>; <span class="NLM_publisher-name">Garland Science</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2013&amp;author=R.+Phillips&amp;author=J.+Kondev&amp;author=J.+Theriot&amp;author=N.+Orme&amp;title=Physical+Biology+of+the+Cell" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPhillips%26aufirst%3DR.%26btitle%3DPhysical%2520Biology%2520of%2520the%2520Cell%26pub%3DGarland%2520Science%26date%3D2013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nenninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mastroianni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullineaux, C. W.</span></span> <span> </span><span class="NLM_article-title">Size Dependence of Protein Diffusion in the Cytoplasm of Escherichia coli</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>192</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">4535</span>– <span class="NLM_lpage">4540</span>, <span class="refDoi">&nbsp;DOI: 10.1128/JB.00284-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1128%2FJB.00284-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=20581203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Wgt7bP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=192&amp;publication_year=2010&amp;pages=4535-4540&amp;issue=18&amp;author=A.+Nenningerauthor=G.+Mastroianniauthor=C.+W.+Mullineaux&amp;title=Size+Dependence+of+Protein+Diffusion+in+the+Cytoplasm+of+Escherichia+coli&amp;doi=10.1128%2FJB.00284-10" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Size dependence of protein diffusion in the cytoplasm of Escherichia coli</span></div><div class="casAuthors">Nenninger, Anja; Mastroianni, Giulia; Mullineaux, Conrad W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4535-4540</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Diffusion in the bacterial cytoplasm is regarded as the primary method of intracellular protein movement and must play a major role in controlling the rates of cell processes.  A no. of recent studies have used green fluorescent protein (GFP) tagging and fluorescence microscopy to probe the movement and distribution of proteins in the bacterial cytoplasm.  However, the dynamic behavior of indigenous proteins must be controlled by a complex mixt. of specific interactions, combined with the basic phys. constraints imposed by the viscosity and macromol. crowding of the cytoplasm.  These factors are difficult to unravel in studies with indigenous proteins.  To what extent the addn. of a GFP tag might affect the movement of a protein through the cytoplasm has also remained unknown.  To resolve these problems, the authors have carried out a systematic study of the size dependence of protein diffusion coeffs. in the Escherichia coli cytoplasm, using engineered GFP multimers (from 2 to 6 covalently linked GFP mols.).  Diffusion coeffs. were measured using confocal fluorescence recovery after photobleaching (FRAP).  At least up to 110 kDa (four linked GFP mols.), the diffusion coeff. varies with size roughly as would be predicted from the Einstein-Stokes equation for a classical (Newtonian) fluid.  Thus, protein diffusion coeffs. are predictable over this range.  GFP tagging of proteins has little impact on the diffusion coeff. over this size range and therefore need not significantly perturb protein movement.  Two indigenous E. coli proteins were used to show that their specific interactions within the cell are the main controllers of the diffusion rate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb7gYA9y_8BbVg90H21EOLACvtfcHk0ljlhnbUzfpqYg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXht1Wgt7bP&amp;md5=30cb24212ae23023c36d4f2670e913c9</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1128%2FJB.00284-10&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.00284-10%26sid%3Dliteratum%253Aachs%26aulast%3DNenninger%26aufirst%3DA.%26aulast%3DMastroianni%26aufirst%3DG.%26aulast%3DMullineaux%26aufirst%3DC.%2BW.%26atitle%3DSize%2520Dependence%2520of%2520Protein%2520Diffusion%2520in%2520the%2520Cytoplasm%2520of%2520Escherichia%2520coli%26jtitle%3DJ.%2520Bacteriol.%26date%3D2010%26volume%3D192%26issue%3D18%26spage%3D4535%26epage%3D4540%26doi%3D10.1128%2FJB.00284-10" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mommer, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sourjik, V.</span></span> <span> </span><span class="NLM_article-title">Mobility of Cytoplasmic, Membrane, and DNA-Binding Proteins in Escherichia coli</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>98</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">559</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bpj.2009.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.bpj.2009.11.002" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=98&amp;publication_year=2010&amp;pages=552-559&amp;issue=4&amp;author=M.+Kumarauthor=M.+S.+Mommerauthor=V.+Sourjik&amp;title=Mobility+of+Cytoplasmic%2C+Membrane%2C+and+DNA-Binding+Proteins+in+Escherichia+coli&amp;doi=10.1016%2Fj.bpj.2009.11.002" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2009.11.002&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2009.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DM.%26aulast%3DMommer%26aufirst%3DM.%2BS.%26aulast%3DSourjik%26aufirst%3DV.%26atitle%3DMobility%2520of%2520Cytoplasmic%252C%2520Membrane%252C%2520and%2520DNA-Binding%2520Proteins%2520in%2520Escherichia%2520coli%26jtitle%3DBiophys.%2520J.%26date%3D2010%26volume%3D98%26issue%3D4%26spage%3D552%26epage%3D559%26doi%3D10.1016%2Fj.bpj.2009.11.002" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slinker, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muren, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">renfrew, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Charge Transport over 34 nm</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">228</span>– <span class="NLM_lpage">233</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.982</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnchem.982" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=21336329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1Wru7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2011&amp;pages=228-233&amp;author=J.+D.+Slinkerauthor=N.+B.+Murenauthor=S.+E.+renfrewauthor=J.+K.+Barton&amp;title=DNA+Charge+Transport+over+34+nm&amp;doi=10.1038%2Fnchem.982" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">DNA charge transport over 34 nm</span></div><div class="casAuthors">Slinker, Jason D.; Muren, Natalie B.; Renfrew, Sara E.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">228-233</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mol. wires show promise in nanoscale electronics, but the synthesis of uniform, long conductive mols. is a significant challenge.  DNA of precise length, by contrast, is synthesized easily, but its cond. over the distances required for nanoscale devices has not been explored.  Here we demonstrate DNA charge transport (CT) over 34 nm in 100-mer monolayers on gold.  Multiplexed gold electrodes modified with 100-mer DNA yield sizable electrochem. signals from a distal, covalent Nile Blue redox probe.  Significant signal attenuation upon incorporation of a single base-pair mismatch demonstrates that CT is DNA-mediated.  Efficient cleavage of these 100-mers by a restriction enzyme indicates that the DNA adopts a native conformation accessible to protein binding.  Similar electron-transfer rates measured through 100-mer and 17-mer monolayers are consistent with rate-limiting electron tunneling through the satd. carbon linker.  This DNA-mediated CT distance of 34 nm surpasses that of most reports of mol. wires.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGkMrXL2figbVg90H21EOLACvtfcHk0lje5yy6bINaVA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXit1Wru7g%253D&amp;md5=35aa021f2af63744480f812085648101</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnchem.982&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.982%26sid%3Dliteratum%253Aachs%26aulast%3DSlinker%26aufirst%3DJ.%2BD.%26aulast%3DMuren%26aufirst%3DN.%2BB.%26aulast%3Drenfrew%26aufirst%3DS.%2BE.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Charge%2520Transport%2520over%252034%2520nm%26jtitle%3DNat.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D228%26epage%3D233%26doi%3D10.1038%2Fnchem.982" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagerman, P. J.</span></span> <span> </span><span class="NLM_article-title">Application of the method of phage T4 DNA ligase-catalyzed ring-closure to the study of DNA structure: II. NaCl-dependence of DNA flexibility and helical repeat</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>212</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">376</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0022-2836(90)90131-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2F0022-2836%2890%2990131-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=2319604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK3cXksleiu7k%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=212&amp;publication_year=1990&amp;pages=363-376&amp;issue=2&amp;author=W.+H.+Taylorauthor=P.+J.+Hagerman&amp;title=Application+of+the+method+of+phage+T4+DNA+ligase-catalyzed+ring-closure+to+the+study+of+DNA+structure%3A+II.+NaCl-dependence+of+DNA+flexibility+and+helical+repeat&amp;doi=10.1016%2F0022-2836%2890%2990131-5" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Application of the method of phage T4 DNA ligase-catalyzed ring-closure to the study of DNA structure.  II.  Sodium chloride-dependence of DNA flexibility and helical repeat</span></div><div class="casAuthors">Taylor, William H.; Hagerman, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">212</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">363-76</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">It is possible to det. the molar cyclization factor jM from single ligation reactions in which both circular and linear dimer DNA species are formed concurrently from linear monomers.  This approach represents a significant improvement over previous methods, in which jM is evaluated from the ratio of the rate consts. for 2 sep. processes; namely (1) the cyclization of linear DNA, and (2) the assocn. of 2 linear mols. to form linear dimers.  Detn. of jM for a 366 base-pair mol. yields 5.8 × 10-8M, in close agreement with the value of 5.6 × 10-8M detd. by D. Shore et al. (1981) for the same mol.  Using the current approach for the detn. of jM, the dependence on NaCl concn. (0-162 mM NaCl, 1 mM MgCl2) of both the lateral and torsional flexibilities of DNA was investigated.  The principal observation is that both quantities are essentially const. over the above range of NaCl concns., with the persistence length P ≈ 450 Å, and the torsional elastic const. C ≈ 2.0 × 10-19 erg cm.  These observations are in accord with the previous theor. prediction that P becomes essentially independent of NaCl concn. &gt;10-20 mM.  The dependence of the helical repeat of DNA on NaCl concn. over the above range was examd. and the value of 10.44 base-pairs per turn was found to be essentially const. over that range.  This last result suggests that earlier studies have overestimated the dependence of DNA helical twist on salt concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqB6EdmWu0H7Vg90H21EOLACvtfcHk0lje5yy6bINaVA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3cXksleiu7k%253D&amp;md5=26f07d95faeb72aa1ce2d7ff76a3c18f</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2890%2990131-5&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252890%252990131-5%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DW.%2BH.%26aulast%3DHagerman%26aufirst%3DP.%2BJ.%26atitle%3DApplication%2520of%2520the%2520method%2520of%2520phage%2520T4%2520DNA%2520ligase-catalyzed%2520ring-closure%2520to%2520the%2520study%2520of%2520DNA%2520structure%253A%2520II.%2520NaCl-dependence%2520of%2520DNA%2520flexibility%2520and%2520helical%2520repeat%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1990%26volume%3D212%26issue%3D2%26spage%3D363%26epage%3D376%26doi%3D10.1016%2F0022-2836%2890%2990131-5" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bustamante, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siggia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span> <span> </span><span class="NLM_article-title">Entropic elasticity of lambda-phage DNA</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>265</i></span> (<span class="NLM_issue">5178</span>),  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1600</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.8079175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.8079175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=8079175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A280%3ADyaK2czmtVamsw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=265&amp;publication_year=1994&amp;pages=1599-1600&amp;issue=5178&amp;author=C.+Bustamanteauthor=J.+Markoauthor=E.+Siggiaauthor=S.+Smith&amp;title=Entropic+elasticity+of+lambda-phage+DNA&amp;doi=10.1126%2Fscience.8079175" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Entropic elasticity of lambda-phage DNA</span></div><div class="casAuthors">Bustamante C; Marko J F; Siggia E D; Smith S</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">5178</span>),
    <span class="NLM_cas:pages">1599-600</span>
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQsGUJ2IL7MSRfKb6ELK8t0fW6udTcc2eaF8-9v-F_dUrntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADyaK2czmtVamsw%253D%253D&amp;md5=2b6782dc3f2f4ca980772b7516d6df00</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1126%2Fscience.8079175&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8079175%26sid%3Dliteratum%253Aachs%26aulast%3DBustamante%26aufirst%3DC.%26aulast%3DMarko%26aufirst%3DJ.%26aulast%3DSiggia%26aufirst%3DE.%26aulast%3DSmith%26aufirst%3DS.%26atitle%3DEntropic%2520elasticity%2520of%2520lambda-phage%2520DNA%26jtitle%3DScience%26date%3D1994%26volume%3D265%26issue%3D5178%26spage%3D1599%26epage%3D1600%26doi%3D10.1126%2Fscience.8079175" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagerman, P. J.</span></span> <span> </span><span class="NLM_article-title">Flexibility of DNA</span>. <i>Annu. Rev. Biophys. Biophys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">286</span>, <span class="refDoi">&nbsp;DOI: 10.1146/annurev.bb.17.060188.001405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1146%2Fannurev.bb.17.060188.001405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=3293588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaL1cXltVKmtbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=1988&amp;pages=265-286&amp;issue=1&amp;author=P.+J.+Hagerman&amp;title=Flexibility+of+DNA&amp;doi=10.1146%2Fannurev.bb.17.060188.001405" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Flexibility of DNA</span></div><div class="casAuthors">Hagerman, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Biophysical Chemistry</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">265-86</span>CODEN:
                <span class="NLM_cas:coden">ARBCEY</span>;
        ISSN:<span class="NLM_cas:issn">0883-9182</span>.
    </div><div class="casAbstract">A review with 113 refs. on the intrinsic flexibility of DNA.  The topics discussed include directional persistence as a function of chain stiffness, exptl. approaches to the detn. of persistence length, torsional stiffness of DNA, and factors contributing to the stiffness of DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-Z53AVxScXLVg90H21EOLACvtfcHk0lhWr2emHL-pxA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL1cXltVKmtbw%253D&amp;md5=05c802dfe48bb12a555e41ed948e075f</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1146%2Fannurev.bb.17.060188.001405&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.bb.17.060188.001405%26sid%3Dliteratum%253Aachs%26aulast%3DHagerman%26aufirst%3DP.%2BJ.%26atitle%3DFlexibility%2520of%2520DNA%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biophys.%2520Chem.%26date%3D1988%26volume%3D17%26issue%3D1%26spage%3D265%26epage%3D286%26doi%3D10.1146%2Fannurev.bb.17.060188.001405" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finzi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bustamante, C.</span></span> <span> </span><span class="NLM_article-title">Direct mechanical measurements of the elasticity of single DNA molecules by using magnetic beads</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>258</i></span> (<span class="NLM_issue">5085</span>),  <span class="NLM_fpage">1122</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.1439819</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.1439819" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1439819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK3sXps1Cguw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=258&amp;publication_year=1992&amp;pages=1122-1126&amp;issue=5085&amp;author=S.+Smithauthor=L.+Finziauthor=C.+Bustamante&amp;title=Direct+mechanical+measurements+of+the+elasticity+of+single+DNA+molecules+by+using+magnetic+beads&amp;doi=10.1126%2Fscience.1439819" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Direct mechanical measurements of the elasticity of single DNA molecules by using magnetic beads</span></div><div class="casAuthors">Smith, Steven B.; Finzi, Laura; Bustamante, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">258</span>
        (<span class="NLM_cas:issue">5085</span>),
    <span class="NLM_cas:pages">1122-6</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">Single DNA mols. were chem. attached by one end to a glass surface and by their other end to a magnetic bead.  Equil. positions of the beads were obsd. in an optical microscope while the beads were acted on by known magnetic and hydrodynamic forces.  Extension vs. force curves were obtained for individual DNA mols. at three different salt concns. with forces between 10-14 and 10-11 newtons.  Deviations from the force curves predicted by the freely jointed chain model suggest that DNA has significant local curvature in soln.  Ethidium bromide and 4',6-diamidino-2-phenylindole had little effect on the elastic response of the mols., but their extent of intercalation was directly measured.  Conversely, the effect of bend-inducing cis-diamminedichloroplatinium(II) was large and supports the hypothesis of natural curvature in DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUQD0F589HNbVg90H21EOLACvtfcHk0lhWr2emHL-pxA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3sXps1Cguw%253D%253D&amp;md5=4cbf5b9d4aba803be42ba52dcb9d05f7</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1126%2Fscience.1439819&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1439819%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DFinzi%26aufirst%3DL.%26aulast%3DBustamante%26aufirst%3DC.%26atitle%3DDirect%2520mechanical%2520measurements%2520of%2520the%2520elasticity%2520of%2520single%2520DNA%2520molecules%2520by%2520using%2520magnetic%2520beads%26jtitle%3DScience%26date%3D1992%26volume%3D258%26issue%3D5085%26spage%3D1122%26epage%3D1126%26doi%3D10.1126%2Fscience.1439819" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zwang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, E. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Sensing DNA through DNA Charge Transport</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1799</span>– <span class="NLM_lpage">1809</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acschembio.8b00347</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00347" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpvFGjs70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=13&amp;publication_year=2018&amp;pages=1799-1809&amp;issue=7&amp;author=T.+J.+Zwangauthor=E.+C.+M.+Tseauthor=J.+K.+Barton&amp;title=Sensing+DNA+through+DNA+Charge+Transport&amp;doi=10.1021%2Facschembio.8b00347" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Sensing DNA through DNA Charge Transport</span></div><div class="casAuthors">Zwang, Theodore J.; Tse, Edmund C. M.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1799-1809</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA charge transport chem. involves the migration of charge over long mol. distances through the arom. base pair stack within the DNA helix.  This migration depends upon the intimate coupling of bases stacked one with another, and hence any perturbation in that stacking, through base modifications or protein binding, can be sensed elec.  In this review, the authors describe the many ways DNA charge transport chem. has been utilized to sense changes in DNA, including the presence of lesions, mismatches, DNA-binding proteins, protein activity, and even reactions under weak magnetic fields.  Charge transport chem. is remarkable in its ability to sense the integrity of DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAq1l1RQptELVg90H21EOLACvtfcHk0lhWr2emHL-pxA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXpvFGjs70%253D&amp;md5=c354086b2190b636205ac1746678ab54</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00347&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00347%26sid%3Dliteratum%253Aachs%26aulast%3DZwang%26aufirst%3DT.%2BJ.%26aulast%3DTse%26aufirst%3DE.%2BC.%2BM.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DSensing%2520DNA%2520through%2520DNA%2520Charge%2520Transport%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D7%26spage%3D1799%26epage%3D1809%26doi%3D10.1021%2Facschembio.8b00347" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sontz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mui, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuss, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Charge Transport as a First Step in Coordinating the Detection of Lesions by Repair Proteins</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1856</span>– <span class="NLM_lpage">1861</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1120063109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1073%2Fpnas.1120063109" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=109&amp;publication_year=2012&amp;pages=1856-1861&amp;issue=6&amp;author=P.+A.+Sontzauthor=T.+P.+Muiauthor=J.+O.+Fussauthor=J.+A.+Tainerauthor=J.+K.+Barton&amp;title=DNA+Charge+Transport+as+a+First+Step+in+Coordinating+the+Detection+of+Lesions+by+Repair+Proteins&amp;doi=10.1073%2Fpnas.1120063109" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1120063109&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1120063109%26sid%3Dliteratum%253Aachs%26aulast%3DSontz%26aufirst%3DP.%2BA.%26aulast%3DMui%26aufirst%3DT.%2BP.%26aulast%3DFuss%26aufirst%3DJ.%2BO.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Charge%2520Transport%2520as%2520a%2520First%2520Step%2520in%2520Coordinating%2520the%2520Detection%2520of%2520Lesions%2520by%2520Repair%2520Proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26issue%3D6%26spage%3D1856%26epage%3D1861%26doi%3D10.1073%2Fpnas.1120063109" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romano, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sontz, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Mutants of the Base Excision Repair Glycosylase, Endonuclease III: DNA Charge Transport as a First Step in Lesion Detection</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">27</span>),  <span class="NLM_fpage">6133</span>– <span class="NLM_lpage">6145</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi2003179</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi2003179" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=50&amp;publication_year=2011&amp;pages=6133-6145&amp;issue=27&amp;author=C.+A.+Romanoauthor=P.+A.+Sontzauthor=J.+K.+Barton&amp;title=Mutants+of+the+Base+Excision+Repair+Glycosylase%2C+Endonuclease+III%3A+DNA+Charge+Transport+as+a+First+Step+in+Lesion+Detection&amp;doi=10.1021%2Fbi2003179" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fbi2003179&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi2003179%26sid%3Dliteratum%253Aachs%26aulast%3DRomano%26aufirst%3DC.%2BA.%26aulast%3DSontz%26aufirst%3DP.%2BA.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DMutants%2520of%2520the%2520Base%2520Excision%2520Repair%2520Glycosylase%252C%2520Endonuclease%2520III%253A%2520DNA%2520Charge%2520Transport%2520as%2520a%2520First%2520Step%2520in%2520Lesion%2520Detection%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26issue%3D27%26spage%3D6133%26epage%3D6145%26doi%3D10.1021%2Fbi2003179" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Geggier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotlyar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vologodskii, A.</span></span> <span> </span><span class="NLM_article-title">Temperature dependence of DNA persistence length</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1419</span>– <span class="NLM_lpage">1426</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gkq932</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1093%2Fnar%2Fgkq932" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=20952402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC3MXis1Chsbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=2011&amp;pages=1419-1426&amp;issue=4&amp;author=S.+Geggierauthor=A.+Kotlyarauthor=A.+Vologodskii&amp;title=Temperature+dependence+of+DNA+persistence+length&amp;doi=10.1093%2Fnar%2Fgkq932" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Temperature dependence of DNA persistence length</span></div><div class="casAuthors">Geggier, Stephanie; Kotlyar, Alexander; Vologodskii, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1419-1426</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We have detd. the temp. dependence of DNA persistence length, a, using two different methods.  The first approach was based on measuring the j-factors of short DNA fragments at various temps.  Fitting the measured j-factors by the theor. equation allowed us to obtain the values of a for temps. between 5°C and 42°C.  The second approach was based on measuring the equil. distribution of the linking no. between the strands of circular DNA at different temps.  The major contribution into the distribution variance comes from the fluctuations of DNA writhe in the nicked circular mols. which are specified by the value of a.  The computation-based anal. of the measured variances was used to obtain the values of a for temps. up to 60°C.  We found a good agreement between the results obtained by these two methods.  Our data show that DNA persistence length strongly depends on temp. and accounting for this dependence is important in quant. comparison between exptl. results obtained at different temps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM61PoCxbJPrVg90H21EOLACvtfcHk0lj_8C-rE-uisw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXis1Chsbs%253D&amp;md5=2e1a5b0503af764b4b78de73ccbe463c</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq932&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq932%26sid%3Dliteratum%253Aachs%26aulast%3DGeggier%26aufirst%3DS.%26aulast%3DKotlyar%26aufirst%3DA.%26aulast%3DVologodskii%26aufirst%3DA.%26atitle%3DTemperature%2520dependence%2520of%2520DNA%2520persistence%2520length%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26issue%3D4%26spage%3D1419%26epage%3D1426%26doi%3D10.1093%2Fnar%2Fgkq932" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of Δ-[Ru(bpy)2dppz]2+ bound to mismatched DNA reveals side-by-side metalloinsertion and intercalation</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">620</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchem.1375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2Fnchem.1375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=22824892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC38XotlCks7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2012&amp;pages=615-620&amp;issue=8&amp;author=H.+Songauthor=J.+T.+Kaiserauthor=J.+K.+Barton&amp;title=Crystal+structure+of+%CE%94-%5BRu%28bpy%292dppz%5D2%2B+bound+to+mismatched+DNA+reveals+side-by-side+metalloinsertion+and+intercalation&amp;doi=10.1038%2Fnchem.1375" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of Δ-[Ru(bpy)2dppz]2+ bound to mismatched DNA reveals side-by-side metalloinsertion and intercalation</span></div><div class="casAuthors">Song, Hang; Kaiser, Jens T.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">615-620</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">DNA mismatches represent a novel target in the development of diagnostics and therapeutics for cancer, because deficiencies in DNA mismatch repair are implicated in cancers, and cells that are repair-deficient show a high frequency of mismatches.  Metal complexes with bulky intercalating ligands serve as probes for DNA mismatches.  Here, we report the high-resoln. (0.92 Å) crystal structure of the ruthenium light switch' complex Δ-[Ru(bpy)2dppz]2+ (bpy = 2,2'-bipyridine and dppz = dipyridophenazine), which is known to show luminescence on binding to duplex DNA, bound to both mismatched and well-matched sites in the oligonucleotide 5'-(dCGGAAATTACCG)2-3' (underline denotes AA mismatches).  Two crystallog. independent views reveal that the complex binds mismatches through metalloinsertion, ejecting both mispaired adenosines.  Addnl. ruthenium complexes are intercalated at well-matched sites, creating an array of complexes in the minor groove stabilized by stacking interactions between bpy ligands and extruded adenosines.  This structure attests to the generality of metalloinsertion and metallointercalation as DNA binding modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB4v1FFhXYFrVg90H21EOLACvtfcHk0lj_8C-rE-uisw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XotlCks7c%253D&amp;md5=72f10ab4bbbcc7b277bdc1e4b4bb95ef</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1375&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1375%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DH.%26aulast%3DKaiser%26aufirst%3DJ.%2BT.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DCrystal%2520structure%2520of%2520%25CE%2594-%255BRu%2528bpy%25292dppz%255D2%252B%2520bound%2520to%2520mismatched%2520DNA%2520reveals%2520side-by-side%2520metalloinsertion%2520and%2520intercalation%26jtitle%3DNat.%2520Chem.%26date%3D2012%26volume%3D4%26issue%3D8%26spage%3D615%26epage%3D620%26doi%3D10.1038%2Fnchem.1375" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gorodetsky, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Electrochemistry using self-assembled DNA monolayers on highly oriented pyrolytic graphite</span>. <i>Langmuir</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">7917</span>– <span class="NLM_lpage">7922</span>, <span class="refDoi">&nbsp;DOI: 10.1021/la0611054</span> </span><div class="citationLinks">[<a href="/doi/10.1021/la0611054" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnt1Ggur4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=2006&amp;pages=7917-7922&amp;author=A.+A.+Gorodetskyauthor=J.+K.+Barton&amp;title=Electrochemistry+using+self-assembled+DNA+monolayers+on+highly+oriented+pyrolytic+graphite&amp;doi=10.1021%2Fla0611054" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Electrochemistry Using Self-Assembled DNA Monolayers on Highly Oriented Pyrolytic Graphite</span></div><div class="casAuthors">Gorodetsky, Alon A.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Langmuir</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7917-7922</span>CODEN:
                <span class="NLM_cas:coden">LANGD5</span>;
        ISSN:<span class="NLM_cas:issn">0743-7463</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Duplex DNA functionalized with pyrene has been utilized to fabricate DNA-modified electrodes on highly oriented pyrolytic graphite (HOPG).  Films have been characterized using AFM and radioactive labeling as well as electrochem.  The data obtained are consistent with a close-packed structure in the film with helixes oriented in a nearly upright orientation, as seen earlier with the fabrication of thiol-tethered duplexes on gold.  Also as on gold, the authors observe the redn. of DNA-bound intercalators in a DNA-mediated reaction.  The redn. of the intercalator is attenuated in the presence of the single-base mismatches, CA and GT, independent of the sequence compn. of the oligonucleotide.  This sensitivity to single-base mismatches is enhanced when methylene blue redn. is coupled in an electrocatalytic cycle with ferricyanide.  The extended potential range afforded by the HOPG surface has allowed the authors also to investigate the electrochem. of previously inaccessible metallointercalators, Ru(bpy)2dppz2+ and Os(phen)2dppz2+, at the DNA-modified HOPG surface.  These results support the application of DNA-modified HOPG as a convenient and reproducible surface for electrochem. DNA sensors using DNA-mediated charge transport.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra9MldP0UdbLVg90H21EOLACvtfcHk0ljVotaee87MsA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28Xnt1Ggur4%253D&amp;md5=f530413f654439e1b031a0c4d624d008</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fla0611054&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fla0611054%26sid%3Dliteratum%253Aachs%26aulast%3DGorodetsky%26aufirst%3DA.%2BA.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DElectrochemistry%2520using%2520self-assembled%2520DNA%2520monolayers%2520on%2520highly%2520oriented%2520pyrolytic%2520graphite%26jtitle%3DLangmuir%26date%3D2006%26volume%3D22%26spage%3D7917%26epage%3D7922%26doi%3D10.1021%2Fla0611054" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jennette, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippard, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliades, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, W. R.</span></span> <span> </span><span class="NLM_article-title">Metallointercalation Reagents. 2-Hydroxyethanethiolato(2,2′,2″-terpyridine)-platinum(II) Monocation Binds Strongly to DNA By Intercalation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>71</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">3839</span>– <span class="NLM_lpage">3843</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.71.10.3839</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1073%2Fpnas.71.10.3839" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=71&amp;publication_year=1974&amp;pages=3839-3843&amp;issue=10&amp;author=K.+W.+Jennetteauthor=S.+J.+Lippardauthor=G.+A.+Vassiliadesauthor=W.+R.+Bauer&amp;title=Metallointercalation+Reagents.+2-Hydroxyethanethiolato%282%2C2%E2%80%B2%2C2%E2%80%B3-terpyridine%29-platinum%28II%29+Monocation+Binds+Strongly+to+DNA+By+Intercalation&amp;doi=10.1073%2Fpnas.71.10.3839" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1073%2Fpnas.71.10.3839&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.71.10.3839%26sid%3Dliteratum%253Aachs%26aulast%3DJennette%26aufirst%3DK.%2BW.%26aulast%3DLippard%26aufirst%3DS.%2BJ.%26aulast%3DVassiliades%26aufirst%3DG.%2BA.%26aulast%3DBauer%26aufirst%3DW.%2BR.%26atitle%3DMetallointercalation%2520Reagents.%25202-Hydroxyethanethiolato%25282%252C2%25E2%2580%25B2%252C2%25E2%2580%25B3-terpyridine%2529-platinum%2528II%2529%2520Monocation%2520Binds%2520Strongly%2520to%2520DNA%2520By%2520Intercalation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1974%26volume%3D71%26issue%3D10%26spage%3D3839%26epage%3D3843%26doi%3D10.1073%2Fpnas.71.10.3839" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kielkopf, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erkkila, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Structure of a photoactive rhodium complex intercalated into DNA</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">121</span>, <span class="refDoi">&nbsp;DOI: 10.1038/72385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1038%2F72385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10655613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtlSkt7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2000&amp;pages=117-121&amp;author=C.+L.+Kielkopfauthor=K.+E.+Erkkilaauthor=B.+P.+Hudsonauthor=J.+K.+Bartonauthor=D.+C.+Rees&amp;title=Structure+of+a+photoactive+rhodium+complex+intercalated+into+DNA&amp;doi=10.1038%2F72385" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a photoactive rhodium complex intercalated into DNA</span></div><div class="casAuthors">Kielkopf, Clara L.; Erkkila, Kathryn E.; Hudson, Brian P.; Barton, Jacqueline K.; Rees, Douglas C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">117-121</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Intercalating complexes of rhodium(III) are strong photo-oxidants that promote DNA strand cleavage or electron transfer through the double helix.  The 1.2 Å resoln. crystal structure of a sequence-specific rhodium intercalators bound to a DNA helix provides a rationale for the sequence specificity of rhodium intercalators.  It also explains how intercalation in the center of an oligonucleotide modifies DNA conformation.  The rhodium complex intercalates via the major groove where specific contacts are formed with the edges of the bases at the target site.  The phi ligand is deeply inserted into the DNA base pair stack.  The primary conformational change of the DNA is a doubling of the rise per residue, with no change in sugar pucker from B-form DNA.  Based upon the five crystallog. independent views of an intercalated DNA helix obsd. in this structure, the intercalator may be considered as an addnl. base pair with specific functional groups positioned in the major groove.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-pj5VKd-CNrVg90H21EOLACvtfcHk0ljVotaee87MsA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3cXhtlSkt7w%253D&amp;md5=f2609dd605f0fc9a1b6774cdc792b921</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2F72385&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F72385%26sid%3Dliteratum%253Aachs%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26aulast%3DErkkila%26aufirst%3DK.%2BE.%26aulast%3DHudson%26aufirst%3DB.%2BP.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DStructure%2520of%2520a%2520photoactive%2520rhodium%2520complex%2520intercalated%2520into%2520DNA%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2000%26volume%3D7%26spage%3D117%26epage%3D121%26doi%3D10.1038%2F72385" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blattner, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plunkett, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloch, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perna, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burland, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collado-Vides, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glasner, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rode, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayhew, G. F.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">The Complete Genome Sequence of Escherichia coli K-12</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>277</i></span> (<span class="NLM_issue">5331</span>),  <span class="NLM_fpage">1453</span>– <span class="NLM_lpage">1462</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.277.5331.1453</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1126%2Fscience.277.5331.1453" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=9278503" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK2sXlvVGnu78%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=277&amp;publication_year=1997&amp;pages=1453-1462&amp;issue=5331&amp;author=F.+R.+Blattnerauthor=G.+Plunkettauthor=C.+A.+Blochauthor=N.+T.+Pernaauthor=V.+Burlandauthor=M.+Rileyauthor=J.+Collado-Videsauthor=J.+D.+Glasnerauthor=C.+K.+Rodeauthor=G.+F.+Mayhew&amp;title=The+Complete+Genome+Sequence+of+Escherichia+coli+K-12&amp;doi=10.1126%2Fscience.277.5331.1453" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Complete genome sequence of Escherichia coli K-12</span></div><div class="casAuthors">Blattner, Frederick R.; Plunkett, Guy, III; Bloch, Craig A.; Perna, Nicole T.; Burland, Valerie; Riley, Monica; Collado-Vides, Julio; Glasner, Jeremy D.; Rode, Christopher K.; Mayhew, George F.; Gregor, Jason; Davis, Nelson Wayne; Kirkpatrick, Heather A.; Goeden, Michael A.; Rose, Debra J.; Mau, Bob; Shao, Ying</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5331</span>),
    <span class="NLM_cas:pages">1453-1462</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The 4,639,221-base pair sequence of Escherichia coli K-12 is presented.  Of 4288 protein-coding genes annotated, 38 percent have no attributed function.  Comparison with five other sequenced microbes reveals ubiquitous as well as narrowly distributed gene families; many families of similar genes within E. coli are also evident.  The largest family of paralogous proteins contains 80 ABC transporters.  The genome as a whole is strikingly organized with respect to the local direction of replication; guanines, oligonucleotides possibly related to replication and recombination, and most genes are so oriented.  The genome also contains insertion sequence (IS) elements, phage remnants, and many other patches of unusual compn. indicating genome plasticity through horizontal transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOJTBu6TS_trVg90H21EOLACvtfcHk0ljVotaee87MsA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2sXlvVGnu78%253D&amp;md5=a3578b591b92dd4dacc5d9c4add9a283</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1126%2Fscience.277.5331.1453&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.277.5331.1453%26sid%3Dliteratum%253Aachs%26aulast%3DBlattner%26aufirst%3DF.%2BR.%26aulast%3DPlunkett%26aufirst%3DG.%26aulast%3DBloch%26aufirst%3DC.%2BA.%26aulast%3DPerna%26aufirst%3DN.%2BT.%26aulast%3DBurland%26aufirst%3DV.%26aulast%3DRiley%26aufirst%3DM.%26aulast%3DCollado-Vides%26aufirst%3DJ.%26aulast%3DGlasner%26aufirst%3DJ.%2BD.%26aulast%3DRode%26aufirst%3DC.%2BK.%26aulast%3DMayhew%26aufirst%3DG.%2BF.%26atitle%3DThe%2520Complete%2520Genome%2520Sequence%2520of%2520Escherichia%2520coli%2520K-12%26jtitle%3DScience%26date%3D1997%26volume%3D277%26issue%3D5331%26spage%3D1453%26epage%3D1462%26doi%3D10.1126%2Fscience.277.5331.1453" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beratan, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naaman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldeck, D. H.</span></span> <span> </span><span class="NLM_article-title">Charge and spin transport through nucleic acids</span>. <i>Curr. Opin. Electrochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">175</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.coelec.2017.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1016%2Fj.coelec.2017.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVGgsLfO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2017&amp;pages=175&amp;issue=1&amp;author=D.+N.+Beratanauthor=R.+Naamanauthor=D.+H.+Waldeck&amp;title=Charge+and+spin+transport+through+nucleic+acids&amp;doi=10.1016%2Fj.coelec.2017.08.017" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Charge and spin transport through nucleic acids</span></div><div class="casAuthors">Beratan, David N.; Naaman, Ron; Waldeck, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Electrochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">175-181</span>CODEN:
                <span class="NLM_cas:coden">COEUCY</span>;
        ISSN:<span class="NLM_cas:issn">2451-9111</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Recent developments in our understanding of charge and spin transport through nucleic acid duplexes are discussed.  Particular emphasis is placed on recent findings that point to the importance of nucleobase fluctuations, transient but extended length-scale electronic coherence, and chiral-induced spin selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjgI4Sow_robVg90H21EOLACvtfcHk0lgq-rSIKQKklg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhtVGgsLfO&amp;md5=ef6db6edb7b87ee34cf64f369253ffa4</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.coelec.2017.08.017&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coelec.2017.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DBeratan%26aufirst%3DD.%2BN.%26aulast%3DNaaman%26aufirst%3DR.%26aulast%3DWaldeck%26aufirst%3DD.%2BH.%26atitle%3DCharge%2520and%2520spin%2520transport%2520through%2520nucleic%2520acids%26jtitle%3DCurr.%2520Opin.%2520Electrochem.%26date%3D2017%26volume%3D4%26issue%3D1%26spage%3D175%26doi%3D10.1016%2Fj.coelec.2017.08.017" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, M. G.</span></span> <span> </span><span class="NLM_article-title">Electrochemistry of Methylene Blue Bound to a DNA-Modified Electrode</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">37</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bc960070o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bc960070o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK2sXps1OltQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=1997&amp;pages=31-37&amp;issue=1&amp;author=S.+O.+Kelleyauthor=J.+K.+Bartonauthor=N.+M.+Jacksonauthor=M.+G.+Hill&amp;title=Electrochemistry+of+Methylene+Blue+Bound+to+a+DNA-Modified+Electrode&amp;doi=10.1021%2Fbc960070o" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Electrochemistry of Methylene Blue Bound to a DNA-Modified Electrode</span></div><div class="casAuthors">Kelley, Shana O.; Barton, Jacqueline K.; Jackson, Nicole M.; Hill, Michael G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-37</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Gold surfaces have been derivatized with 15-base-pair double-stranded DNA oligonucleotides contg. a pendant 5' hexanethiol linker.  The electrochem. of intercalated methylene blue has been investigated at these modified electrodes.  Chronocoulometry, cyclic voltammetry, ellipsometry, and quantitation via 32P labeling are all consistent with a surface coverage of ≥75% with the DNA helixes stacked at an angle from the electrode surface.  Cyclic voltammetry at low methylene blue/duplex stoichiometries yields well-behaved surface waves with E° = -0.25 V (vs SCE), a value 0.03 V neg. of that in aq. soln.  A binding isotherm for methylene blue at an electrode derivatized with the double-stranded sequence 5' SH-(CH2)6-p-AGTACAGTCATCGCG 3' was obtained from coulometric titrns. and gave an affinity const. equal to 3.8(5) × 106 M-1 with a satn. value of 1.4(2) methylene blue intercalators per DNA duplex.  Taken together, these expts. support a model for the surface morphol. in which DNA duplexes are densely packed; methylene blue therefore reversibly binds to sites in the DNA that are close to the bulk soln.  Electrochem. at DNA-derivatized electrodes provides a valuable methodol. to examine DNA-bound redox reactions and may offer new insight into DNA-mediated electron transfers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRGWG2RWENQLVg90H21EOLACvtfcHk0lgq-rSIKQKklg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2sXps1OltQ%253D%253D&amp;md5=a118fc772c3ec4d406393a1e2223c9e3</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fbc960070o&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbc960070o%26sid%3Dliteratum%253Aachs%26aulast%3DKelley%26aufirst%3DS.%2BO.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26aulast%3DJackson%26aufirst%3DN.%2BM.%26aulast%3DHill%26aufirst%3DM.%2BG.%26atitle%3DElectrochemistry%2520of%2520Methylene%2520Blue%2520Bound%2520to%2520a%2520DNA-Modified%2520Electrode%26jtitle%3DBioconjugate%2520Chem.%26date%3D1997%26volume%3D8%26issue%3D1%26spage%3D31%26epage%3D37%26doi%3D10.1021%2Fbc960070o" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Long-Range Electron Transfer through DNA Films</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">945</span>, <span class="refDoi">&nbsp;DOI: 10.1002/(SICI)1521-3773(19990401)38:7&lt;941::AID-ANIE941&gt;3.0.CO;2-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=10.1002%2F%28SICI%291521-3773%2819990401%2938%3A7%3C941%3A%3AAID-ANIE941%3E3.0.CO%3B2-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADyaK1MXisFWqsrw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=38&amp;publication_year=1999&amp;pages=941-945&amp;issue=7&amp;author=S.+O.+Kelleyauthor=N.+M.+Jacksonauthor=M.+G.+Hillauthor=J.+K.+Barton&amp;title=Long-Range+Electron+Transfer+through+DNA+Films&amp;doi=10.1002%2F%28SICI%291521-3773%2819990401%2938%3A7%3C941%3A%3AAID-ANIE941%3E3.0.CO%3B2-7" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Long-range electron transfer through DNA films</span></div><div class="casAuthors">Kelley, Shana O.; Jackson, Nicole M.; Hill, Michael G.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-945</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">In an effort to investigate DNA-mediated electron transfer involving ground-state reactants, the authors studied redox-active intercalators bound at discrete sites within the individual helixes of a DNA monolayer on gold.  To investigate charge transduction through DNA as a function of distance, the authors have site-specifically cross-linked a redox-active intercalator, daunomycin (DM), into the DNA characterized.  The site of intercalation was controlled in the duplexes by incorporating a single guanine-cytosine (GC) base step in otherwise adenine-thymine (AT) or inosine-cytosine (IC) sequences; as DM requires the N2 atom of guanine for covalent crosslinking, the intercalator is constrained to these positions.  Moving the GC step along the duplex therefore provided a systematic variation in the location of the DM-binding site relative to the thiol-terminated linker.  Crosslinking DM to the DNA in soln., and then depositing the labeled duplexes onto gold, afforded a series of films in which the intercalator was linked quant. at a known sepn. from the electrode surface.  Here, long-range charge transport through DNA-modified films has been demonstrated in a series of structurally characterized assemblies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8SZO4Dz9OKrVg90H21EOLACvtfcHk0lgq-rSIKQKklg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1MXisFWqsrw%253D&amp;md5=3012e102a1211872e74c53d889d33e79</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990401%2938%3A7%3C941%3A%3AAID-ANIE941%3E3.0.CO%3B2-7&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990401%252938%253A7%253C941%253A%253AAID-ANIE941%253E3.0.CO%253B2-7%26sid%3Dliteratum%253Aachs%26aulast%3DKelley%26aufirst%3DS.%2BO.%26aulast%3DJackson%26aufirst%3DN.%2BM.%26aulast%3DHill%26aufirst%3DM.%2BG.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DLong-Range%2520Electron%2520Transfer%2520through%2520DNA%2520Films%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26issue%3D7%26spage%3D941%26epage%3D945%26doi%3D10.1002%2F%28SICI%291521-3773%2819990401%2938%3A7%3C941%3A%3AAID-ANIE941%3E3.0.CO%3B2-7" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slinker, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muren, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorodetsky, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">Multiplexed DNA-Modified Electrodes</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2769</span>– <span class="NLM_lpage">2774</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja909915m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja909915m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2rurs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=132&amp;publication_year=2010&amp;pages=2769-2774&amp;issue=8&amp;author=J.+D.+Slinkerauthor=N.+B.+Murenauthor=A.+A.+Gorodetskyauthor=J.+K.+Barton&amp;title=Multiplexed+DNA-Modified+Electrodes&amp;doi=10.1021%2Fja909915m" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Multiplexed DNA-Modified Electrodes</span></div><div class="casAuthors">Slinker, Jason D.; Muren, Natalie B.; Gorodetsky, Alon A.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2769-2774</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the use of silicon chips with 16 DNA-modified electrodes (DME chips) utilizing DNA-mediated charge transport for multiplexed detection of DNA and DNA-binding protein targets.  Four DNA sequences were simultaneously distinguished on a single DME chip with 4-fold redundancy, including one incorporating a single base mismatch.  These chips also enabled investigation of the sequence-specific activity of the restriction enzyme Alu1.  DME chips supported dense DNA monolayer formation with high reproducibility, as confirmed by statistical comparison to com. available rod electrodes.  The working electrode areas on the chips were reduced to 10 μm in diam., revealing microelectrode behavior that is beneficial for high sensitivity and rapid kinetic anal.  These results illustrate how DME chips facilitate sensitive and selective detection of DNA and DNA-binding protein targets in a robust and internally standardized multiplexed format.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgnAATmqwzP7Vg90H21EOLACvtfcHk0lg5Gj5_sCKQ0Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2rurs%253D&amp;md5=840ee36ecfb09e3b4958c88075fb68cd</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fja909915m&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja909915m%26sid%3Dliteratum%253Aachs%26aulast%3DSlinker%26aufirst%3DJ.%2BD.%26aulast%3DMuren%26aufirst%3DN.%2BB.%26aulast%3DGorodetsky%26aufirst%3DA.%2BA.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DMultiplexed%2520DNA-Modified%2520Electrodes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26issue%3D8%26spage%3D2769%26epage%3D2774%26doi%3D10.1021%2Fja909915m" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boon, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">DNA Electrochemistry as a Probe of Base Pair Stacking in A-, B-, and Z-Form DNA</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1140</span>– <span class="NLM_lpage">1147</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bc034139l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bc034139l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotlaru7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=2003&amp;pages=1140-1147&amp;issue=6&amp;author=E.+M.+Boonauthor=J.+K.+Barton&amp;title=DNA+Electrochemistry+as+a+Probe+of+Base+Pair+Stacking+in+A-%2C+B-%2C+and+Z-Form+DNA&amp;doi=10.1021%2Fbc034139l" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">DNA Electrochemistry as a Probe of Base Pair Stacking in A-, B-, and Z-Form DNA</span></div><div class="casAuthors">Boon, Elizabeth M.; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1140-1147</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA-mediated charge transport (CT) chem. is sensitive to DNA structure and base pair stacking.  In an electrochem. assay based upon DNA CT, DNA-modified electrode surfaces are used to examine the electrochem. redn. of methylene blue (MB), a small mol. that binds to the DNA film by intercalation.  Here electrochem. the authors probe CT in the three primary conformations of double-stranded nucleic acids, A-, B-, and Z-form DNA.  The A-form is examd. in the context of a DNA/RNA hybrid duplex and Z-DNA, in duplexes contg. d(mCG)8 sequences at high Mg2+ concns.  Both A- and B-DNA support efficient DNA CT as measured by MB redn. in the DNA film; a lower level of redn. is evident with the Z-form film. Furthermore, mismatches incorporated into A-form duplexes, as in B-form duplexes, disrupt MB redn., thus providing a strategy for mutation detection through testing of RNA transcripts at DNA electrodes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcFHUxYm3uLLVg90H21EOLACvtfcHk0lg5Gj5_sCKQ0Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3sXotlaru7w%253D&amp;md5=20caa24d6728d0202d74762850229ab0</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fbc034139l&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbc034139l%26sid%3Dliteratum%253Aachs%26aulast%3DBoon%26aufirst%3DE.%2BM.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DDNA%2520Electrochemistry%2520as%2520a%2520Probe%2520of%2520Base%2520Pair%2520Stacking%2520in%2520A-%252C%2520B-%252C%2520and%2520Z-Form%2520DNA%26jtitle%3DBioconjugate%2520Chem.%26date%3D2003%26volume%3D14%26issue%3D6%26spage%3D1140%26epage%3D1147%26doi%3D10.1021%2Fbc034139l" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zwang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, E. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, J. K.</span></span> <span> </span><span class="NLM_article-title">A Compass at Weak Magnetic Fields Using Thymine Dimer Repair</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">412</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acscentsci.8b00008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.8b00008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facscentsci.8b00566&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjvVGnu7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2018&amp;pages=405-412&amp;issue=3&amp;author=T.+J.+Zwangauthor=E.+C.+M.+Tseauthor=D.+Zhongauthor=J.+K.+Barton&amp;title=A+Compass+at+Weak+Magnetic+Fields+Using+Thymine+Dimer+Repair&amp;doi=10.1021%2Facscentsci.8b00008" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">A Compass at Weak Magnetic Fields Using Thymine Dimer Repair</span></div><div class="casAuthors">Zwang, Theodore J.; Tse, Edmund C. M.; Zhong, Dongping; Barton, Jacqueline K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-412</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">How birds sense the variations in Earth's magnetic field for navigation is poorly understood, although cryptochromes, proteins homologous to photolyases, have been proposed to participate in this magnetic sensing.  Here, in electrochem. studies with an applied magnetic field, we monitor the repair of cyclobutane pyrimidine dimer lesions in duplex DNA by photolyase, mutants of photolyase, and by a modified cryptochrome.  We found that the yield of dimer repair was dependent on the strength and angle of the applied magnetic field even when using magnetic fields weaker than 1 G.  This high sensitivity to weak magnetic fields depended upon a fast radical pair reaction on the thymines leading to repair.  These data illustrate chem. how cyclobutane pyrimidine dimer repair may be used in a biol. compass informed by variations in Earth's magnetic field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzGMHpXBfW7bVg90H21EOLACvtfcHk0ljjeFkksrf2WA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXjvVGnu7w%253D&amp;md5=45f72e4d218ccda1f9a723c3978ce2aa</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.8b00008&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.8b00008%26sid%3Dliteratum%253Aachs%26aulast%3DZwang%26aufirst%3DT.%2BJ.%26aulast%3DTse%26aufirst%3DE.%2BC.%2BM.%26aulast%3DZhong%26aufirst%3DD.%26aulast%3DBarton%26aufirst%3DJ.%2BK.%26atitle%3DA%2520Compass%2520at%2520Weak%2520Magnetic%2520Fields%2520Using%2520Thymine%2520Dimer%2520Repair%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2018%26volume%3D4%26issue%3D3%26spage%3D405%26epage%3D412%26doi%3D10.1021%2Facscentsci.8b00008" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i19">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_76867"></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acscentsci.8b00566">10.1021/acscentsci.8b00566</a>.</p><ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supplemental methods, data, and figures including schematics, protein counts, box plots, UV–vis spectra, and AFM image (<a class="ext-link" href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf">PDF</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.8b00566/suppl_file/oc8b00566_si_001.pdf">oc8b00566_si_001.pdf (1.02 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms &amp; Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7" style="width: 457.7px;"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facscentsci.8b00566&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facscentsci.8b00566%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3BrequestedJournal%3Ajournal%3Aacscii%3Bwebsite%3Awebsite%3Aacspubs%3Bjournal%3Ajournal%3Aacscii%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Facscii.2019.5.issue-1" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self" style="display: none;"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acscentsci.8b00566" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article><div class="w-slide"><div class="w-slide_head"><a href="#" class="w-slide__back"><i class=" icon-arrow_l" aria-hidden="true"></i>back</a><span class="w-slide__title"></span></div><div class="w-slide__content"></div></div></main><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2022<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS &amp; Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support &amp; Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"><input type="hidden" name="appID" value="literatum" class="appID"><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onclick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        <!-- <div id="gdpr-message-container">
			<a id="closebtn" href="#"><i class="icon-times-circle-o"></i></a> -->
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        <!-- </div> -->
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        <!-- AdRoll Pixel -->
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>

<!-- Twitter universal website tag code -->
<!-- Commented out for ACHS-6152
<script>
    !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe.apply(s,arguments):s.queue.push(arguments);
    },s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',
    a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');
    // Insert Twitter Pixel ID and Standard Event data below
    twq('init','o3njp');
    twq('track','PageView');
</script>
-->
<!-- Facebook Pixel Code -->
<!-- Commented out for ACHS-6152
<script>
    !function(f,b,e,v,n,t,s)
    {if(f.fbq)return;n=f.fbq=function(){n.callMethod?
    n.callMethod.apply(n,arguments):n.queue.push(arguments)};
    if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
    n.queue=[];t=b.createElement(e);t.async=!0;
    t.src=v;s=b.getElementsByTagName(e)[0];
    s.parentNode.insertBefore(t,s)}(window, document,'script',
    'https://connect.facebook.net/en_US/fbevents.js');
    fbq('init', '529840281033320');
    fbq('track', 'PageView');
</script>
<noscript><img height="1" width="1" style="display:none" src="https://www.facebook.com/tr?id=529840281033320&amp;ev=PageView&amp;noscript=1"
/></noscript>
-->





        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div><div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/acscentsci.8b00773">A Selective and Cell-Permeable Mitochondrial Calcium Uniporter (MCU) Inhibitor Preserves Mitochondrial Bioenergetics after Hypoxia/Reoxygenation Injury</a></li><li><a href="/doi/full/10.1021/acscentsci.8b00822">An ATP-Regulated Ion Transport Nanosystem for Homeostatic Perturbation Therapy and Sensitizing Photodynamic Therapy by Autophagy Inhibition of Tumors</a></li><li><a href="/doi/full/10.1021/acscentsci.8b00844">Red Room-Temperature Phosphorescence of CDs@Zeolite Composites Triggered by Heteroatoms in Zeolite Frameworks</a></li></ul></div></div></div>




        
        <script>
var currentJournal = "ACS Central Science";
$(".citationLinks > a").click(function(e){
    if($(this).text() === "ACS Full Text ") {
        ga('send', 'event', 'Reference _Click', 'ACS Full Text', currentJournal);
    }
    else if($(this).text() === "Crossref") {
        ga('send', 'event', 'Reference _Click', 'Crossref', currentJournal);
    }
    else if($(this).text() === "CAS") {
        ga('send', 'event', 'Reference _Click', 'CAS', currentJournal);
    }
    else if($(this).text() === "PubMed") {
        ga('send', 'event', 'Reference _Click', 'PubMed', currentJournal);
    }
    else if($(this).text() === "Google Scholar") {
        ga('send', 'event', 'Reference _Click', 'Google Scholar', currentJournal);
    }
    else if($(this).hasClass("issue-item_open-url") === true) {
        ga('send', 'event', 'Reference _Click', 'OpenURL', currentJournal);
    }
});
</script>






        
        



        
    





        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        <div tabindex="0" class="showMenuAlertMsg-popup"><div class="pairing-affiliations-popup-content"><div class="pairing-affiliations-header"><button type="button" class="close paringAffClose"><i aria-hidden="true" class="icon-close_thin"></i></button></div><div class="responsrMsg"></div></div></div>




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	dartInstitutionType = 'education';
</script>
        </div>
    





        
        



        
    





        
        <script>
// LIT-455776 workaround
// Moving the popup before the header content for tab accessibility
document.addEventListener('DOMContentLoaded', () => {
    const headerNode = $('.header_content');
    const popupNode = $('.pairing-affiliations-popup');

    if (!headerNode.length || !popupNode.length)
        return;

    popupNode.insertBefore(headerNode);  
});
</script> 
        </div>
    </div>





	
    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-d93b9088814c7fcd2f1c.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-9dee82777d6af4ace557.js"></script>
<script type="text/javascript" src="/wro/l6j1~product.js"></script>



















<div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh371" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.4256715146039237&amp;iit=1651100219525&amp;tmr=load%3D1651100219320%26core%3D1651100219352%26main%3D1651100219519%26ifr%3D1651100219530&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=pubs.acs.org&amp;dr=&amp;du=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facscentsci.8b00566&amp;href=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facscentsci.8b00566&amp;dt=Effective%20Distance%20for%20DNA-Mediated%20Charge%20Transport%20between%20Repair%20Proteins&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Ctype%3DArticle%2Cdescription%2Cimage%2Ctitle%2Curl&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=6269ca3bc7d77d46&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=url%3Dhttps%253A%252F%252Fpubs.acs.org%252Fdoi%252Ffull%252F10.1021%252Facscentsci.8b00566%26title%3DEffective%2520Distance%2520for%2520DNA-Mediated%2520Charge%2520Transport%2520between%2520Repair%2520Proteins%26image%3Dhttps%253A%252F%252Fpubs.acs.org%252Fcms%252F10.1021%252Facscentsci.8b00566%252Fasset%252Fimages%252Facscentsci.8b00566.social.jpeg_v03%26description%3DThe%2520stacked%2520aromatic%2520base%2520pairs%2520within%2520the%2520DNA%2520double%2520helix%2520facilitate%2520charge%2520transport%2520down%2520its%2520length%2520in%2520the%2520absence%2520of%2520lesions%252C%2520mismatches%252C%2520and%2520other%2520stacking%2520perturbations.%2520DNA%2520repair%2520proteins%2520containing%2520%255B4Fe4S%255D%2520clusters%2520can%2520take%2520advantage%2520of%2520DNA%2520charge%2520transport%2520(CT)%2520chemistry%2520to%2520scan%2520the%2520genome%2520for%2520mistakes%2520more%2520efficiently.%2520Here%2520we%2520examine%2520the%2520effective%2520length%2520over%2520which%2520charge%2520can%2520be%2520transported%2520along%2520DNA%2520between%2520these%2520repair%2520proteins.%2520We%2520define%2520the%2520effective%2520CT%2520distance%2520as%2520the%2520length%2520of%2520DNA%2520within%2520which%2520two%2520proteins%2520are%2520able%2520to%2520influence%2520their%2520ensemble%2520affinity%2520to%2520the%2520DNA%2520duplex%2520via%2520CT.%2520Endonuclease%2520III%252C%2520a%2520DNA%2520repair%2520glycosylase%2520containing%2520a%2520%255B4Fe4S%255D%2520cluster%252C%2520was%2520incubated%2520with%2520DNA%2520duplexes%2520of%2520different%2520lengths%2520(1.5%25E2%2580%25939%2520kb)%252C%2520and%2520atomic%2520force%2520microscopy%2520was%2520used%2520to%2520quantify%2520the%2520binding%2520of%2520proteins%2520to%2520these%2520duplexes%2520to%2520determine%2520how%2520the%2520relative%2520protein%2520affinity%2520changes%2520with%2520increasing%2520DNA%2520length.%2520A%2520sharp%2520change%2520in%2520binding%2520slope%2520is%2520observed%2520at%25203509%2520base%2520pairs%252C%2520or%2520about%25201.2%2520%25CE%25BCm%252C%2520that%2520supports%2520the%2520existence%2520of%2520two%2520regimes%2520for%2520protein%2520binding%252C%2520one%2520within%2520the%2520range%2520for%2520DNA%2520CT%252C%2520one%2520outside%2520of%2520the%2520range%2520for%2520CT%253B%2520DNA%2520CT%2520between%2520the%2520redox%2520proteins%2520bound%2520to%2520DNA%2520effectively%2520decreases%2520the%2520ensemble%2520binding%2520affinity%2520of%2520oxidized%2520and%2520reduced%2520proteins%2520to%2520DNA.%2520Utilizing%2520an%2520Endonuclease%2520III%2520mutant%2520Y82A%252C%2520which%2520is%2520defective%2520in%2520carrying%2520out%2520DNA%2520CT%252C%2520shows%2520only%2520one%2520regime%2520for%2520protein%2520binding.%2520Decreasing%2520the%2520temperature%2520to%25204%2520%25C2%25B0C%2520or%2520including%2520metallointercalators%2520on%2520the%2520duplex%252C%2520both%2520of%2520which%2520should%2520enhance%2520base%2520stacking%2520and%2520decrease%2520DNA%2520floppiness%252C%2520leads%2520to%2520extending%2520the%2520effective%2520length%2520for%2520DNA%2520charge%2520transport%2520to%2520%25E2%2588%25BC5300%2520bp%2520or%25201.8%2520%25CE%25BCm.%2520These%2520results%2520thus%2520support%2520DNA%2520charge%2520transport%2520between%2520repair%2520proteins%2520over%2520kilobase%2520distances.%2520The%2520results%2520furthermore%2520highlight%2520the%2520ability%2520of%2520DNA%2520repair%2520proteins%2520to%2520search%2520the%2520genome%2520quickly%2520and%2520efficiently%25C2%25A0using%2520DNA%2520charge%2520transport%2520chemistry.%26type%3DArticle%26site_name%3DACS%2520Publications&amp;csi=undefined&amp;rev=v8.28.8-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div></body></html>