<html class="pb-page" data-request-id="aad99f15-2b03-4e2e-8732-5863bf23927e" lang="en"><head data-pb-dropzone="head"><script type="text/javascript" src="https://m.addthis.com/live/red_lojson/300lo.json?si=627192dcd3816265&amp;bkl=0&amp;bl=1&amp;pdt=922&amp;sid=627192dcd3816265&amp;pub=xa-4faab26f2cff13a7&amp;rev=v8.28.8-wp&amp;ln=en&amp;pc=men&amp;cb=0&amp;ab=-&amp;dp=pubs.acs.org&amp;fp=doi%2F10.1021%2Facscentsci.9b01247&amp;fr=&amp;of=0&amp;pd=0&amp;irt=1&amp;vcl=1&amp;md=0&amp;ct=1&amp;tct=0&amp;abt=0&amp;cdn=0&amp;pi=1&amp;rb=0&amp;gen=100&amp;chr=UTF-8&amp;colc=1651610333270&amp;jsl=1&amp;uvs=627192d83f19d54a002&amp;skipb=1&amp;callback=addthis.cbs.jsonp__75672860341276880"></script><script type="text/javascript" src="https://v1.addthisedge.com/live/boost/xa-4faab26f2cff13a7/_ate.track.config_resp"></script><script type="text/javascript" src="https://z.moatads.com/addthismoatframe568911941483/moatframe.js"></script><meta name="pbContext" content=";page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;requestedJournal:journal:acscii;issue:issue:10.1021/acscii.2020.6.issue-2;website:website:acspubs;article:article:10.1021/acscentsci.9b01247;journal:journal:acscii;pageGroup:string:Publication Pages;subPage:string:Full Text">




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="ACS Central Science"><meta name="dc.Title" content="Enhancing the Signaling of GPCRs via Orthosteric Ions"><meta name="dc.Creator" content="H. C. Stephen  Chan"><meta name="dc.Creator" content="Yueming  Xu"><meta name="dc.Creator" content="Liang  Tan"><meta name="dc.Creator" content="Horst  Vogel"><meta name="dc.Creator" content="Jianjun  Cheng"><meta name="dc.Creator" content="Dong  Wu"><meta name="dc.Creator" content="Shuguang  Yuan"><meta name="dc.Description" content="G protein-coupled receptors play essential roles in cellular processes such as neuronal signaling, vision, olfaction, tasting, and metabolism. As GPCRs are the most important drug targets, understa..."><meta name="Description" content="G protein-coupled receptors play essential roles in cellular processes such as neuronal signaling, vision, olfaction, tasting, and metabolism. As GPCRs are the most important drug targets, understa..."><meta name="dc.Description" content="A unique strategy was developed to optimize the drug activity of GPCR, which opens a new mechanistic opportunity of GPCR signaling and helps design the next generation of drugs."><meta name="Description" content="A unique strategy was developed to optimize the drug activity of GPCR, which opens a new mechanistic opportunity of GPCR signaling and helps design the next generation of drugs."><meta name="dc.Publisher" content="American Chemical Society"><meta name="dc.Date" scheme="WTN8601" content="January 23, 2020"><meta name="dc.Type" content="research-article"><meta name="dc.Format" content="text/HTML"><meta name="dc.Identifier" scheme="doi" content="10.1021/acscentsci.9b01247"><meta name="dc.Language" content="en"><meta name="dc.Coverage" content="world"><meta name="dc.Rights" content="Copyright Â© 2020 American Chemical Society"><meta name="citation_fulltext_world_readable" content="">
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facscentsci.9b01247">
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facscentsci.9b01247">
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facscentsci.9b01247">
        
    
    

<title>Enhancing the Signaling of GPCRs via Orthosteric Ions | ACS Central Science</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive">





    






        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acscentsci.9b01247"><meta property="og:title" content="Enhancing the Signaling of GPCRs via Orthosteric Ions"><meta property="og:image" content="https://pubs.acs.org/cms/10.1021/acscentsci.9b01247/asset/images/acscentsci.9b01247.social.jpeg_v03"><meta property="og:description" content="G protein-coupled receptors play essential roles in cellular processes such as neuronal signaling, vision, olfaction, tasting, and metabolism. As GPCRs are the most important drug targets, understanding their interactions with ligands is of utmost importance for discovering related new medicines. In many GPCRs, an allosteric sodium ion next to the highly conserved residue D2.50 has been proposed to stabilize the inactive receptor state by mediating interactions between transmembrane helices. Here, we probed the existence of internal and functionally important sodium ions in the dopamine D2 receptor, using molecular dynamics simulations. Besides a new sodium ion at the allosteric ligand binding site, we discovered an additional sodium ion, located close to the orthosteric ligand binding site. Through cell-based activation assays, the signaling of D2 receptor with site-specific mutations was tested against a series of chemically modified agonists. We concluded an important structural role of this newly discovered orthosteric sodium ion in modulating the receptor signaling: It enables the coordination of a polar residue in the ligand binding site with an appropriately designed agonist molecule. An identical interaction was also observed in a recently released high-resolution crystal structure of mu-opioid receptor, which was reresolved in this work. Probably because of similar interactions, various metal ions have been found to increase the signaling of many other GPCRs. This unique principle and strategy could be used to optimize the drug activity of GPCR. Our findings open a new mechanistic opportunity of GPCR signaling and help design the next generation of drugs targeting GPCRs."><meta property="og:type" content="Article"><meta property="og:site_name" content="ACS Publications"><meta name="twitter:card" content="summary_large_image"><meta name="twitter:site" content="@ACSPublications"><meta name="twitter:image" content="https://pubs.acs.org/cms/10.1021/acscentsci.9b01247/asset/images/acscentsci.9b01247.social.jpeg_v03">
    





    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0">















    

    <meta name="dc.identifier" content="10.1021/acscentsci.9b01247">



<meta name="format-detection" content="telephone=no">




<link rel="stylesheet" type="text/css" href="/wro/l6j1~product.css">

<link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-c7d2282eca032afdc856.css"><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css">




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1640088552890.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1648583148410.css">
<script type="text/javascript" async="" src="//img.en25.com/i/elqCfg.min.js"></script><script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script><script async="" src="//www.google-analytics.com/analytics.js"></script><script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer="" src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon="{&quot;rayId&quot;:&quot;5200035e6e4d5180&quot;,&quot;startTime&quot;:1570117212010}"></script>

            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    , 'auto'
                    );
                
                ga('send', 'pageview');
            </script>
            











    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acscentsci.9b01247">
            
        







    <!-- loading Javascript for using the persistence Service -->
    <script src="/templates/jsp/pb2/js/plugin/thiss-ds.js"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            alt_context: window.location.origin,
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if (ms == undefined || ms === '') {
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            if (results.length == 0) {
                                persistService.entities(self.alt_context).then((res) => {
                                    return resolve(res.data);
                                }, function (err) {
                                    // failed
                                    reject('failed to retrieve entity from local storage (publisher domain): ' + err);
                                });
                            } else {
                                return resolve(results);
                            }
                        }, function (err) {
                            // failed
                            reject('failed to retrieve entity from local storage: ' + err);
                        });
                    // Set up the timeout
                    setTimeout(function () {
                        reject('Promise timed out ' + ms + ' ms');
                    }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                    persistService.update(self.alt_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage (publisher domain) duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            var results = res.data;
                            if (!results) {
                                persistService.remove(self.alt_context, entityId).then((res) => {
                                    return resolve(res);
                                }, function (err) {
                                    // failed
                                    console.log('failed to remove from local storage (publisher domain) duo to: ' + err);
                                });
                            }
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId, entityTitle) {
                return new Promise(function (resolve, reject) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {
                            doi: doi,
                            entityid: entityId,
                            entityTitle: entityTitle,
                            eventGroupKey: '4874e0aa-cc1d-4163-8626-3ad5e2132c30'
                        }

                    }).done(function (status) {
                        return resolve(status);
                    }).fail(function (status) {
                        return reject(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID, ssoInstitution.entity.title)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + ssoInstitution.entity.entityID,err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            }).catch(function (err) {
                                reject(err);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();

                    if ($('.institution-preference-userconsent-checkbox').length > 0) {
                        let institutionPreferenceUserConsentCheckbox = $('.institution-preference-userconsent-checkbox').get(0);
                        if (!$(institutionPreferenceUserConsentCheckbox).is(":checked")) {
                            return;
                        }
                    }

                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err) {
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>







    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY">


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html">

<link rel="shortcut icon" href="/pb-assets/ux3/favicon-1635812676023.ico"><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><meta http-equiv="origin-trial" content="AxujKG9INjsZ8/gUq8+dTruNvk7RjZQ1oFhhgQbcTJKDnZfbzSTE81wvC2Hzaf3TW4avA76LTZEMdiedF1vIbA4AAABueyJvcmlnaW4iOiJodHRwczovL2ltYXNkay5nb29nbGVhcGlzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzVGhpcmRQYXJ0eSI6dHJ1ZX0="><meta http-equiv="origin-trial" content="Azuce85ORtSnWe1MZDTv68qpaW3iHyfL9YbLRy0cwcCZwVnePnOmkUJlG8HGikmOwhZU22dElCcfrfX2HhrBPAkAAAB7eyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A16nvcdeoOAqrJcmjLRpl1I6f3McDD8EfofAYTt/P/H4/AWwB99nxiPp6kA0fXoiZav908Z8etuL16laFPUdfQsAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="AxBHdr0J44vFBQtZUqX9sjiqf5yWZ/OcHRcRMN3H9TH+t90V/j3ENW6C8+igBZFXMJ7G3Pr8Dd13632aLng42wgAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A88BWHFjcawUfKU3lIejLoryXoyjooBXLgWmGh+hNcqMK44cugvsI5YZbNarYvi3roc1fYbHA1AVbhAtuHZflgEAAAB2eyJvcmlnaW4iOiJodHRwczovL2dvb2dsZS5jb206NDQzIiwiZmVhdHVyZSI6IlRydXN0VG9rZW5zIiwiZXhwaXJ5IjoxNjUyNzc0NDAwLCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="AzoawhTRDevLR66Y6MROu167EDncFPBvcKOaQispTo9ouEt5LvcBjnRFqiAByRT+2cDHG1Yj4dXwpLeIhc98/gIAAACFeyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A6+nc62kbJgC46ypOwRsNW6RkDn2x7tgRh0wp7jb3DtFF7oEhu1hhm4rdZHZ6zXvnKZLlYcBlQUImC4d3kKihAcAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A/9La288e7MDEU2ifusFnMg1C2Ij6uoa/Z/ylwJIXSsWfK37oESIPbxbt4IU86OGqDEPnNVruUiMjfKo65H/CQwAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_2022042801.js" async=""></script><script src="https://widgets.figshare.com/static/figshare.js?_=1651610332502"></script><script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/CcdcJsmolUtilities.min.js?_=1651610332503"></script><script src="https://common.ccdc.cam.ac.uk/jsmol/14.2.7/JSmol.min.nojq.js?_=1651610332504"></script><link type="text/css" rel="stylesheet" href="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/CcdcWidget.min.css"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'central_science';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/central_science/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/central_science/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
					console.log("ad display: " + event.creativeId + " " + event.size[0] + "x" + event.size[1]);
				}
				else {
				    console.log("no ad found");
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>

<style>
.articleHeaderDropzone5 .advertisement { display: none; }
.article_header .advertisement {
  margin: -25px auto 25px auto;
}
</style>




        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&amp;EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Indiana University Libraries - Bloomington</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"></a>




        
        <div class="pj-badge"></div><a href="/journal1/acscii" title="ACS Central Science" class="header_article-logo"><img src="/cms/10.1021/acscii/asset/title.png" alt="ACS Central Science"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook" title="Facebook" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter" title="Twitter" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat" target="_blank" title="WeChat" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin" target="_blank" title="LinkedIn" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit" target="_blank" title="Reddit" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email" target="_blank" title="Email" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li><div class="atclear"></div></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="acscii" name="SeriesKey" checked="checked" class="all-content"><span>ACS Cent. Sci.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"><select name="quickLinkJournal" class="quick-search_journals-select"><option value="acscii">ACS Central Science</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science &amp; Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials &amp; Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio &amp; Med Chem Au</option><option value="abseba">ACS Biomaterials Science &amp; Engineering</option><option value="accacs">ACS Catalysis</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health &amp; Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&amp;T Engineering</option><option value="aewcaa">ACS ES&amp;T Water</option><option value="afsthl">ACS Food Science &amp; Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic &amp; Inorganic Au</option><option value="aptsfn">ACS Pharmacology &amp; Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry &amp; Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&amp;EN Global Enterprise</option><option value="cenear">Chemical &amp; Engineering News Archive</option><option value="chlseg">Chemical Health &amp; Safety</option><option value="chlseg0">Chemical Health &amp; Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth &amp; Design</option><option value="enfuem">Energy &amp; Fuels</option><option value="esthag">Environmental Science &amp; Technology</option><option value="estlcu">Environmental Science &amp; Technology Letters</option><option value="iepra6.2">I&amp;EC Product Research and Development</option><option value="iechad">Industrial &amp; Engineering Chemistry</option><option value="iecac0">Industrial &amp; Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series</option><option value="iecfa7">Industrial &amp; Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial &amp; Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial &amp; Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial &amp; Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical &amp; Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health &amp; Safety</option><option value="jchsc20">Journal of Chemical Health &amp; Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial &amp; Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research &amp; Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&amp;D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autocomplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autocomplete="false" placeholder="Page" required="required" class="quick-search_page-input"><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden="âtrueâ"></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/acscentsci.9b01216">Bottlebrush Bridge between Soft Gels and Firm Tissues</a></li><li><a href="/doi/full/10.1021/acscentsci.9b01129">Direct Quantification of Damaged Nucleotides in Oligonucleotides Using an Aerolysin Single Molecule Interface</a></li></ul></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="âtrueâ"></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="âtrueâ"></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden="âtrueâ"></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors &amp; Reviewers');">Authors &amp;
                        Reviewers</a></li>
                <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
				<li><a href="https://pubs.acs.org/page/virtual-collections.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Virtual Issues');">Virtual Issues</a></li>
                <li><a href="https://preferences.acs.org/journals" onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS &amp; Mobile');">RSS &amp; Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products &amp; Services');">Products &amp; Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos &amp; Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#" onclick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');" class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/who-we-serve/librarians/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians &amp; Account Managers');">For
                        Librarians &amp; Account Managers</a></li>
            </ul>
        </li>
        <li class="menu-pair-section hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a></li>
                
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS &amp; Open Access');">ACS &amp; Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="/events" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!-- Types--><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><!-- Subjects--><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!--div.pub-modal_right-cover-gallery.hidden-lg.hidden-md//a(href="/coverGallery") Browse Our Cover Art Gallery
--><!-- Publications list--><!-- 'pub-modal_right--inner' adds styling required for IE--><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><!-- The widget data--><input type="hidden" id="ajaxUrl" name="ajaxUrl" value="/pb/widgets/achs-publication-list?widgetId=cabcb15c-6f7a-494b-a19c-eba4fafaee5c&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3BrequestedJournal%3Ajournal%3Aacscii%3Bissue%3Aissue%3A10.1021%2Facscii.2020.6.issue-2%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facscentsci.9b01247%3Bjournal%3Ajournal%3Aacscii%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;key=c7ca46a97ce077568c7371697bcd489c0&amp;ajaxRequest=true"><input type="hidden" name="pubModalData" value=""></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facscentsci.9b01247&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facscentsci.9b01247&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acscentsci.9b01247&amp;href=/doi/10.1021/acscentsci.9b01247" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>ACS Cent. Sci.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 6</span><span class="cit-fg-issue">, 2</span><span class="cit-fg-pageRange">, 274-282</span></span></span></div></div></div></div></div><input type="hidden" name="acscii" id="copyRights" value="/pb-assets/json/journal-info-1641590078917.json"><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/acscii/6/2" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acscentsci.9b01237" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Research Article</span><a href="/doi/10.1021/acscentsci.9b01272" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/acscii" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/cms/10.1021/acscii/asset/title.png" alt="Journal Logo"></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Enhancing the Signaling of GPCRs via Orthosteric Ions</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">H. C. Stephen Chan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">H. C. Stephen Chan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen 518055, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=H.+C.+Stephen++Chan">H. C. Stephen Chan</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Yueming Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yueming Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">iHuman Institute, ShanghaiTech University, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yueming++Xu">Yueming Xu</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Liang Tan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Liang Tan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">iHuman Institute, ShanghaiTech University, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Liang++Tan">Liang Tan</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Horst Vogel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Horst Vogel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen 518055, China</div><div class="loa-info-affiliations-info">Institute of Chemical Sciences and Engineering, Ecole Polytechnique FÃ©dÃ©rale de Lausanne (EPFL), Lausanne 1015, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Horst++Vogel">Horst Vogel</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-9695-8894" title="Orcid link">http://orcid.org/0000-0001-9695-8894</a></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Jianjun Cheng</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianjun Cheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">iHuman Institute, ShanghaiTech University, Shanghai 201210, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="mailto:chengjj@shanghaitech.edu.cn">chengjj@shanghaitech.edu.cn</a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianjun++Cheng">Jianjun Cheng</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-6065-2682" title="Orcid link">http://orcid.org/0000-0001-6065-2682</a></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Dong Wu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dong Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">iHuman Institute, ShanghaiTech University, Shanghai 201210, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="mailto:wudong@shanghaitech.edu.cn">wudong@shanghaitech.edu.cn</a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dong++Wu">Dong Wu</a></span></div></div></span></li><span class="comma-separator">,&nbsp;and&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Shuguang Yuan</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shuguang Yuan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen 518055, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="mailto:shuguang.yuan@siat.ac.cn">shuguang.yuan@siat.ac.cn</a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shuguang++Yuan">Shuguang Yuan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-9858-4742" title="Orcid link">http://orcid.org/0000-0001-9858-4742</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="acscii"><input type="hidden" name="journalNameForjhpLink" value="ACS Central Science"><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facscentsci.9b01247&amp;href=/doi/10.1021%2Facscentsci.9b01247" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>ACS Cent. Sci.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 6</span><span class="cit-issue">, 2</span><span class="cit-pageRange">, 274â282</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 23, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>5 December 2019</li><li><span class="item_label"><b>Published</b> online</span>23 January 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 February 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acscentsci.9b01247" title="DOI URL">https://doi.org/10.1021/acscentsci.9b01247</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS &amp; PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice"><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acscentsci.9b01247"><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3834</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">10</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" class="doi" value="10.1021/acscentsci.9b01247"><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Enhancing the Signaling of GPCRs via Orthosteric Ions&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;H.&quot;,&quot;last_name&quot;:&quot;C. Stephen Chan&quot;},{&quot;first_name&quot;:&quot;Yueming&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Liang&quot;,&quot;last_name&quot;:&quot;Tan&quot;},{&quot;first_name&quot;:&quot;Horst&quot;,&quot;last_name&quot;:&quot;Vogel&quot;},{&quot;first_name&quot;:&quot;Jianjun&quot;,&quot;last_name&quot;:&quot;Cheng&quot;},{&quot;first_name&quot;:&quot;Dong&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Shuguang&quot;,&quot;last_name&quot;:&quot;Yuan&quot;}],&quot;source&quot;:&quot;ACS Central Science&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;2&quot;,&quot;volume&quot;:&quot;6&quot;,&quot;pages&quot;:&quot;274-282&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acscentsci.9b01247&quot;},&quot;abstract&quot;:&quot;G protein-coupled receptors play essential roles in cellular processes such as neuronal signaling, vision, olfaction, tasting, and metabolism. As GPCRs are the most important drug targets, understanding their interactions with ligands is of utmost importance for discovering related new medicines. In many GPCRs, an allosteric sodium ion next to the highly conserved residue D2.50 has been proposed to stabilize the inactive receptor state by mediating interactions between transmembrane helices. Here, we probed the existence of internal and functionally important sodium ions in the dopamine D2 receptor, using molecular dynamics simulations. Besides a new sodium ion at the allosteric ligand binding site, we discovered an additional sodium ion, located close to the orthosteric ligand binding site. Through cell-based activation assays, the signaling of D2 receptor with site-specific mutations was tested against a series of chemically modified agonists. We concluded an important structural role of this newly discover&quot;}" class="meta-data"><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"><input type="hidden" name="pub-access" value="free" class="pub-access"><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"><input type="hidden" name="selected-share-option" class="selected-share-option"></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facscentsci.9b01247&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.9b01247" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facscentsci.9b01247&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.9b01247" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facscentsci.9b01247&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.9b01247" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facscentsci.9b01247&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acscentsci.9b01247&amp;href=/doi/10.1021/acscentsci.9b01247" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook" title="Facebook" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter" title="Twitter" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat" target="_blank" title="WeChat" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin" target="_blank" title="LinkedIn" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit" target="_blank" title="Reddit" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><div class="atclear"></div></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acscentsci.9b01247"><a href="/doi/pdf/10.1021/acscentsci.9b01247" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acscentsci.9b01247&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fid%3Ddoi%3A10.1021%252Facscentsci.9b01247%26sid%3Dliteratum%253Aachs%26pmid%3D32123746%26genre%3Darticle%26aulast%3DChan%26date%3D2020%26atitle%3DEnhancing%2Bthe%2BSignaling%2Bof%2BGPCRs%2Bvia%2BOrthosteric%2BIons%26jtitle%3DACS%2BCentral%2BScience%26title%3DACS%2BCentral%2BScience%26volume%3D6%26issue%3D2%26spage%3D274%26epage%3D282%26issn%3D2374-7943" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="acscii" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/acscii?ifr=1"></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollduration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=290667" title="Ions">Ions</a>,</li><li><a href="/action/doSearch?ConceptID=291297" title="Cell signaling">Cell signaling</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/acscii/6/2" title="Go to ACS Central Science "><img src="/cms/10.1021/acscii.2020.6.issue-2/asset/acscii.2020.6.issue-2.largecover.jpg" alt="Go to ACS Central Science "></a></div></div>




        
        <!--Article Page E-Alerts-->
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','acscii');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','acscii');
	  }
  });
</script>
<!--End Article Script-->




        
        <div class="pj-badge"><input type="hidden" id="partnerJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json"></div>




        
        <div class="aJhp_link"></div>
<input type="hidden" id="aJhpJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json">
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=acscii" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0006.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.9b01247&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">G protein-coupled receptors play essential roles in cellular processes such as neuronal signaling, vision, olfaction, tasting, and metabolism. As GPCRs are the most important drug targets, understanding their interactions with ligands is of utmost importance for discovering related new medicines. In many GPCRs, an allosteric sodium ion next to the highly conserved residue D<sup>2.50</sup> has been proposed to stabilize the inactive receptor state by mediating interactions between transmembrane helices. Here, we probed the existence of internal and functionally important sodium ions in the dopamine D2 receptor, using molecular dynamics simulations. Besides a new sodium ion at the allosteric ligand binding site, we discovered an additional sodium ion, located close to the orthosteric ligand binding site. Through cell-based activation assays, the signaling of D2 receptor with site-specific mutations was tested against a series of chemically modified agonists. We concluded an important structural role of this newly discovered orthosteric sodium ion in modulating the receptor signaling: It enables the coordination of a polar residue in the ligand binding site with an appropriately designed agonist molecule. An identical interaction was also observed in a recently released high-resolution crystal structure of mu-opioid receptor, which was reresolved in this work. Probably because of similar interactions, various metal ions have been found to increase the signaling of many other GPCRs. This unique principle and strategy could be used to optimize the drug activity of GPCR. Our findings open a new mechanistic opportunity of GPCR signaling and help design the next generation of drugs targeting GPCRs.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span></span>&nbsp;Note</h4><p class="last">This paper was originally published ASAP on January 23, 2020. The title was corrected, and the paper reposted on January 24, 2020.</p></div><div class="hlFld-Abstract synopsis"><h2>Synopsis</h2><span><p class="articleBody_abstractText">A unique strategy was developed to optimize the drug activity of GPCR, which opens a new mechanistic opportunity of GPCR signaling and helps design the next generation of drugs.</p></span></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56675" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56675" class="dropBlock__holder"></div></div></div></div></div><hr><div class="NLM_p">Na<sup>+</sup> ions have been found to influence the functions of many members of class A G protein-coupled receptors (GPCRs), including adrenergic, dopaminergic, adenosine, opioid, and neurotensin receptors. <a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9">(1â9)</a> For example, the binding of certain ligands to these receptors are known to be sensitive to sodium ions. <a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> For the case of dopamine D2 receptor (D2R), the binding of an agonist leads to the coupling of the receptor to G<sub>i/o</sub> protein that in turn inhibits adenylyl cyclase. At physiological concentration, Na<sup>+</sup> ions decrease the affinities of agonists including the endogenous agonist dopamine but enhance the affinities for some antagonists. The mutation D80A<sup>2.50</sup> or D80E<sup>2.50</sup> abolishes Na<sup>+</sup> sensitivity. Several high-resolution crystal structures <a onclick="showRef(event, 'ref1 ref3 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref1 ref3 ref10 ref11">(1,3,10,11)</a> indicated that residues in the vicinity of D<sup>2.50</sup> form a distinct Na<sup>+</sup> mediated coordination network. As an example, in both 2.1 Ã high-resolution adenosine A<sub>2A</sub>R and adrenergic Î²<sub>1</sub>AR crystal structures, the side chain oxygen atoms of D<sup>2.50</sup> and S<sup>3.39</sup> as well as three additional water molecules coordinate with this conserved Na<sup>+</sup> ion. <a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Such interactions have been proposed to stabilize the inactive state of GPCRs. <a onclick="showRef(event, 'ref1 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref1 ref13 ref14">(1,13,14)</a> This phenomenon was also found by computational simulations. <a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a></div><div class="NLM_p last">Interestingly, Na<sup>+</sup>, as well as other metal ions, have been found elsewhere to modulate ligand binding to GPCRs at physiological conditions. <a onclick="showRef(event, 'ref13 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref13 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27">(13,17â27)</a> However, the molecular basis for this unique observation is still unknown. Here, we investigated the molecular mechanism through which the Na<sup>+</sup> ions influence the function of GPCRs. Starting from the recently resolved crystal structure of D2R (pdb: <a href="http://doi.org/10.2210/pdb6CM4/pdb">6CM4</a>), <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> we investigated the allosteric interactions between Na<sup>+</sup> and the agonist MLS1547 in the ligand binding pocket of D2R <a onclick="showRef(event, 'ref13 ref29'); return false;" href="javascript:void(0);" class="ref ref13 ref29">(13,29)</a> using all-atom molecular dynamics (MD) simulations. <a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Surprisingly, our MD simulations revealed an additional Na<sup>+</sup> ion in the ligand binding site next to extracellular loop 2 (ECL2). This second Na<sup>+</sup> coordinates the interactions between residue H393<sup>6.55</sup> and MLS1547 via two water molecules. Our findings were further confirmed by functional analysis of the interactions between the rationally designed synthetic analogues of MLS1547 and the mutants of D2R. A systematic analysis of high-resolution GPCR crystal structures also revealed the newly found coordination motif between Na<sup>+</sup> and ligand in other GPCRs. Our findings provide new opportunities for designing novel drugs targeting GPCRs.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07240" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07240" class="dropBlock__holder"></div></div></div></div></div><hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Allosteric Na<sup>+</sup> Ion Next to D80<sup>2.50</sup> in apo D2R</h3><div class="NLM_p">The allosteric Na<sup>+</sup> ion next to D<sup>2.50</sup> has been observed in several crystal structures of GPCRs, and is considered to play an essential role in GPCR signaling. <a onclick="showRef(event, 'ref1 ref31'); return false;" href="javascript:void(0);" class="ref ref1 ref31">(1,31)</a> Although no allosteric Na<sup>+</sup> ion is observed in the recently published 2.8 Ã resolution D2R crystal structure (pdb: <a href="http://doi.org/10.2210/pdb6CM4/pdb">6CM4</a>), <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> the signaling of D2R has been shown to depend critically on Na<sup>+</sup> ions. <a onclick="showRef(event, 'ref20 ref23 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref20 ref23 ref32 ref33">(20,23,32,33)</a> This raises a question whether an allosteric Na<sup>+</sup> ion next to D<sup>2.50</sup> may actually be present in the D2R also, but the resolution of the mentioned crystal structure might be too low to detect the electron density of a sodium ion. To evaluate whether an allosteric Na<sup>+</sup> ion could exist in D2R, we first performed 3 Ã 0.5 Î¼s all-atom MD simulations for apo D2R, based on its crystal structure (pdb: <a href="http://doi.org/10.2210/pdb6CM4/pdb">6CM4</a>) <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). The initial D2R structure did not contain any intrinsic sodium ion, and the sodium ions were present in the extramembrane bulk medium for crystal growth. Interestingly, an allosteric Na<sup>+</sup> ion was found in all three simulations (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A and <a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_002.mp4" class="ext-link">Movie 1</a>). During the initial 0.1 Î¼s simulation period (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B), Na<sup>+</sup> ions fluctuated in the extracellular bulk environment (zone I). During 0.1â0.2 Î¼s time scale, a Na<sup>+</sup> ion moved to D114<sup>3.32</sup> and formed a transient ionic interaction. After about 0.2 Î¼s, this Na<sup>+</sup> ion diffused deeper into the receptor in the vicinity of D80<sup>2.50</sup> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B) and was finally stabilized in this region until the end of the MD simulations. At this position the Na<sup>+</sup> ion formed a stable coordination complex with the side chain oxygen atoms of D80<sup>2.50</sup> and S121<sup>3.39</sup> of apo D2R together with three additional water molecules (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>C). This feature is in good agreement with allosteric Na<sup>+</sup> ions found in several high-resolution crystal structures of other GPCRs (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Figure S1</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0001.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0001.jpeg" id="gr1" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Allosteric Na<sup>+</sup> ion next to D80<sup>2.50</sup> in D2R. (A) Entrance pathway of a Na<sup>+</sup> ion from the extracellular receptor interface toward the allosteric site D80<sup>2.50</sup> of apo D2R. Zone I (gray): extracellular receptor interface at which the Na<sup>+</sup> ion is in the bulk environment. Zone II (red): transition zone in the vicinity of D114<sup>3.32</sup>. Zone III (blue): allosteric site in the vicinity of D80<sup>2.50</sup>. Yellow spheres: Na<sup>+</sup> ion positions during MD simulations. (B) Distance between allosteric Na<sup>+</sup> ion and D80<sup>2.50</sup>. (C) The final position of the allosteric Na<sup>+</sup> ion at the end of MD simulations. The oxygen atoms in D80<sup>2.50</sup> and S121<sup>3.39</sup> as well as those of three water molecules establish a dedicated coordination network with the allosteric Na<sup>+</sup> ion.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.9b01247&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> A New Orthosteric Na<sup>+</sup> Ion in the Extracellular Region of D2R</h3><div class="NLM_p">To study the role of Na<sup>+</sup> ions within the interaction of a potent agonist, for example, MLS1547, and D2R at the atomic level, we first computationally constructed the D2R-MLS1547 complex by proteinâligand docking. The binding model was identical to that of the dopamine D3 receptor (D3R) crystal structure (pdb: <a href="http://doi.org/10.2210/pdb3PBL/pdb">3PBL</a>, <a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Figure S2</a>). Then, we submitted the D2R-MLS1547 complex to 3 Ã 4.0 Î¼s all-atom MD simulations (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). MLS1547 was stable during MD simulations as indicated by its RMSD (root-mean-square deviation) (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Figure S3</a>). Because we observed also for this case the existence of the allosteric Na<sup>+</sup> ion next to D80<sup>2.50</sup>, we kept it in all subsequent MD simulations from the beginning, especially considering residue D<sup>2.50</sup> is deprotonated at the initial activation stage of GPCRs. <a onclick="showRef(event, 'ref1 ref34'); return false;" href="javascript:void(0);" class="ref ref1 ref34">(1,34)</a> Interestingly, we found in these simulations a second Na<sup>+</sup> ion captured in the orthosteric site next to the agonist molecule, forming coordination with both MLS1547 and D2R at 1.0â3.2 Î¼s time scale (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). Quantum mechanics calculation indicated that the âOH group is deprotonated in the quinolin-8-ol moiety of MLS1547, due to the presence of the strong electronegative -Cl atom in the <i>para</i> position. Here, the nitrogen atom and the negatively charged oxygen established a coordination with this orthosteric Na<sup>+</sup> ion. Moreover, H393<sup>6.55</sup> together with two water molecules from the bulk environment formed coordination bonds with the Na<sup>+</sup> ion (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0002.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0002.jpeg" id="gr2" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. A second Na<sup>+</sup> ion has been found in the orthosteric site of D2R using all-atom long-time MD simulations. (A) The entrance pathway of the orthosteric Na<sup>+</sup> toward agonist molecule MLS1547. Yellow spheres: Na<sup>+</sup> ion trajectories in the MD simulations. Black ball-and-stick: MLS1547. (B) The distance between orthosteric Na<sup>+</sup> ion and H393<sup>6.55</sup> (C) The final position of the orthosteric Na<sup>+</sup> ion at the end of MD simulations. MLS1547 and H393<sup>6.55</sup> as well as two water molecules establish a dedicated coordination network with the orthosteric Na<sup>+</sup> ion.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.9b01247&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0003.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0003.jpeg" id="gr3" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Interaction fingerprint between distinct residues in D2R and MLS1547. (A) The 2D interaction diagram of D2R in complex with MLS1547. Green ball-and-stick: MLS1547. (B) The interaction fingerprint between residues in D2R and MLS1547. It represents the interaction frequency during the MD simulations. (C) Arrestin recruitment for D2R variants showing mutations on key residues predicted by MD simulations. All mutated receptors show noticeably decreased signaling.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.9b01247&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We then performed a proteinâligand interaction fingerprint (IFP) analysis on the generated MD simulation trajectories. IFP is a very useful method to quantify the interaction frequency between a protein and a ligand of interest. <a onclick="showRef(event, 'ref35 ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37 ref38">(35â38)</a> The analysis identifies which residues are most important for a specific proteinâligand interaction network. Besides the coordinating interaction with residue H393<sup>6.55</sup>, IFP analysis on the MD simulation trajectories indicated a few additional interactions of MLS1547 within the receptor including (1) face-to-side ÏâÏ stacking interactions with the two aromatic residues F389<sup>6.51</sup> and F390<sup>6.52</sup>, (2) ionic interactions with D114<sup>3.32</sup>, and finally (3) hydrophobic interactions with the residues I183<sup>ECL2</sup> and I184<sup>ECL2</sup> in the ECL2 loop (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>B).</div><div class="NLM_p">To validate this newly discovered binding mode, we modified the D2R site specifically on the key residues indicated by the IFP analysis, and tested the activation of wildtype and mutant proteins by functional analyses. The EC<sub>50</sub> value for wildtype receptor activation by MLS1547 was 630 nM for G protein signaling and 250 nM for arrestin signaling. Interestingly, all mutations had a substantial influence on the agonist induced receptor activation (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>C and <a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">Table-</a><a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">1</a>). Specifically, mutation variants I83<sup>ECL2</sup>A, L94<sup>2.64</sup>A, D114<sup>3.32</sup> A, F389<sup>6.51</sup>A, and F390<sup>6.52</sup>A completely abolished the activations of both G protein and arrestin. Interestingly, the mutation of H393<sup>6.55</sup>A led to MLS1547 completely inactive toward arrestin signaling, whereas it was still active against G protein signaling (EC<sub>50</sub> = 350 nM). In contrast, mutant I84<sup>ECL2</sup>A was completely inactive toward G protein signaling but still had weak activity toward arrestin signaling (EC<sub>50</sub> = 2500 nM). These results echoed the unique binding mode of MLS1547 in the D2R complex.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Activities of Agonist MLS1547 against Different D2R variants (EC<sub>50</sub>, nM)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center"><col align="center"><col align="center"><col align="center"><col align="center"><col align="center"><col align="center"><col align="center"><col align="center"></colgroup><thead><tr class="colsep0" valign="top"><th class="colsep0 rowsep0" align="center">&nbsp;</th><th class="colsep0 rowsep0" align="center">WT</th><th class="colsep0 rowsep0" align="center">L94A</th><th class="colsep0 rowsep0" align="center">D114A</th><th class="colsep0 rowsep0" align="center">I83A</th><th class="colsep0 rowsep0" align="center">I84A</th><th class="colsep0 rowsep0" align="center">F389A</th><th class="colsep0 rowsep0" align="center">F390A</th><th class="colsep0 rowsep0" align="center">H393A</th></tr></thead><tbody><tr class="colsep0" valign="top"><td class="colsep0 rowsep0" align="center">G protein</td><td class="colsep0 rowsep0" align="center">290&nbsp;Â±&nbsp;90</td><td class="colsep0 rowsep0" align="center">inactive</td><td class="colsep0 rowsep0" align="center">inactive</td><td class="colsep0 rowsep0" align="center">inactive</td><td class="colsep0 rowsep0" align="center">9000&nbsp;Â±&nbsp;7600</td><td class="colsep0 rowsep0" align="center">inactive</td><td class="colsep0 rowsep0" align="center">inactive</td><td class="colsep0 rowsep0" align="center">inactive</td></tr><tr class="colsep0" valign="top"><td class="colsep0 rowsep0" align="center">Î²-arrestin</td><td class="colsep0 rowsep0" align="center">250&nbsp;Â±&nbsp;60</td><td class="colsep0 rowsep0" align="center">inactive</td><td class="colsep0 rowsep0" align="center">inactive</td><td class="colsep0 rowsep0" align="center">inactive</td><td class="colsep0 rowsep0" align="center">2500&nbsp;Â±&nbsp;600</td><td class="colsep0 rowsep0" align="center">inactive</td><td class="colsep0 rowsep0" align="center">inactive</td><td class="colsep0 rowsep0" align="center">inactive</td></tr></tbody></table></div></div><div class="NLM_p">To further validate our findings, we performed biological testing of dopamine against D2R (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Table-S1 and Figure S4</a>). For this case, the interaction between the orthosteric Na<sup>+</sup> and dopamine is missing, as indicated by docking (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Figure S5</a>). Previous studies found a similar binding mode for dopamine against D2R. <a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> These findings were confirmed by the observation that mutation H393<sup>6.55</sup>A does not show significant influences on dopamineâs activity: 970 nM (wildtype) vs 1002 nM (mutant).</div><div class="NLM_p">In contrast, MLS1547 abolished its activity completely. The orthosteric Na<sup>+</sup> ion coordinates agonist induced activation of D2R. As indicated in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A, a Na<sup>+</sup> ion is involved in the interaction network of the agonist MLS1547 with residues in the orthosteric binding pocket of D2R and modulates the activation of the receptor. For the stability of the ligandâreceptor complex, polar groups in the agonist molecule play a central role.</div><div class="NLM_p">To further investigate how the chemical structure of the agonist determines the unique binding mode, we synthesized several derivatives of MLS1547 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> and <a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Table-S2</a>). We first prepared 5-substituted quinolin-8-ol (see <a class="ref internalNav" href="#sec3" aria-label="Materials and Methods">Materials and Methods</a>). In the variant MLS-d1, a fluorine replaced the chlorine in MLS1547. The strong electronegativity of F decreased substantially the p<i>K</i><sub>a</sub> value of the hydroxyl group of MLS1547 from 6.3 to 4.2 (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Table-S2</a>). Consequently, the group is deprotonated facilitating the interaction with the orthosteric Na<sup>+</sup> ion (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A). This in turn improved the activation of the G protein with a decrease in EC<sub>50</sub> values by almost 34 times, to 18.6 nM (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Table-S2</a>). Interestingly, the removal of Cl in MLS-d2 increased the p<i>K</i><sub>a</sub> only slightly to 6.4 Â± 0.7, but resulted also in an improved activation of G protein as shown by the EC<sub>50</sub> values of 95 nM (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Table-S2</a>). We noticed that the activity of MLS-d2 is even slightly higher than MLS1547. Probably the presence of a -Cl might impose allosteric effects in the binding site as shown in <a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Figure S6</a>. Moreover, the relatively large -Cl is a little bit closer to the negatively charged D114<sup>3.32</sup>, inducing electrostatic repulsions. We also prepared MLS-d3, in which a bromine replaced the chlorine. In turn, the p<i>K</i><sub>a</sub> value increased to 7.4 and the compound was completely inactive (MLS-d3, <a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Table-S2</a>). Moreover, introducing a nitro group in the para position of -OH decreased the p<i>K</i><sub>a</sub> substantially to 3.4 (MLS-d4, <a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Table-S2</a>). However, the nitro group is too big to be accommodated in the binding pocket of D2R and therefore this compound did not show activation of G-protein and arrestin (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Figure S6</a>). Furthermore, one of the oxygen atoms in the nitro group was too close to the oxygen atom of D114<sup>3.32</sup>, which might lead to strong unfavorable electrostatic repulsion (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Figure S6</a>). In addition, we prepared compound MLS-d5, in which the N atom in quinolin-8-ol ring was replaced by a -CH group. The loss of coordination possibility led MLS-d5 to be completely inactive. Moreover, the estimated relative binding energies of MLS1547 variants correlate well with their EC<sub>50</sub> values (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Table-S2</a>). This implies that MLS1547 and certain variants enhance the signaling by increasing the ligand binding through unique ion-coordinating interactions (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0004.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0004.jpeg" id="gr4" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Synthesis and biological activity of MLS1547 derivatives. (A) Scheme for synthesis of compound MLS-d1. (B) Activation of D2R mediated signaling induced by dopamine, MLS1547- and MLS1547-derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.9b01247&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> An Allosteric Ion in the Crystal Structure of Î¼-Opioid Receptor and Potential Application in Other GPCRs</h3><div class="NLM_p">A recent high-resolution crystal structure of Î¼-opioid receptor (Î¼OR, PDB: <a href="http://doi.org/10.2210/pdb5C1M/pdb">5C1M</a>) showed that there is an âunidentified electron density mapâ between agonist BU72 and a His residue of Î¼OR. <a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> We tried filling this part of the electron density map with different ions, including Li, Na, Mg, Ni, and Zn. We found that only Mg fit perfectly to this region (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Figures S7 and S8</a>): the F<sub>O</sub>âF<sub>C</sub> electron density map was clean and the 2F<sub>O</sub>âF<sub>C</sub> map matched model very well, whereas all other ions always result in either negative FoâFc maps (ions are too large to fit in) or positive FoâFc maps (ions are too small to fit in). Specifically, the Mg coordinates with both the ligand BU72 and Î¼OR, which is identical to the interactions in this work for D2R. More importantly, Mg was confirmed to enhance the signaling of Î¼OR in other previous work. <a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> This further strongly supported that modulating the ligand binding via an orthosteric ion could be a universal strategy for GPCR ligand design.</div><div class="NLM_p last">To further validate whether similar interactions might happen in other GPCRs, a detailed inspection of family A GPCR structures shows that His is a frequent residue in TM6 of GPCRs (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">Figure S9</a>). Considering that the ligand binding pocket of many family A GPCRs is very large, it is possible that a His in TM6 in other receptors might establish similar interaction networks as reported in the present work. Indeed, the signaling of many other GPCRs have been found to be enhanced by various metal ions elsewhere, <a onclick="showRef(event, 'ref15 ref16 ref17 ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18 ref19 ref20 ref21">(15â21)</a> probably because of the similar coordinating interactions as well.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Conclusion and Perspectives</h3><div class="NLM_p">An allosteric sodium ion has been observed in several high-resolution GPCR structures including A<sub>2A</sub>R, D4R, Î²<sub>1</sub>AR, and others. <a onclick="showRef(event, 'ref1 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref3 ref4 ref5">(1,3â5)</a> The binding site of the allosteric sodium ion is located next to the highly conserved residue D<sup>2.50</sup> in the inner space of the TM region in family A GPCRs. The specific coordination of the allosteric sodium ion strengthen links between different TM helices to enhance the stability of the inactive state of the particular receptors (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>, left panel). <a onclick="showRef(event, 'ref1 ref2 ref14'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref14">(1,2,14)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0005.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0005.jpeg" id="gr5" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Two different roles of Na<sup>+</sup> in D2 receptor activation. Left panel: an allosteric Na<sup>+</sup> (yellow sphere) stabilizes the inactive state of receptor. Blue ball-and-sphere: dopamine molecule. Right panel: an orthosteric Na<sup>+</sup> enhances the activation of receptor via a coordinating interaction with both H393<sup>6.55</sup> and agonist molecule MLS1547 (black ball-and-stick).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.9b01247&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Using all-atom MD simulations of apo-D2R, we have identified the potential existence of an allosteric Na<sup>+</sup> ion next to D<sup>2.50</sup>, which resembles allosteric Na<sup>+</sup> ions found in crystal structures of other GPCRs. <a onclick="showRef(event, 'ref1 ref2 ref14 ref32'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref14 ref32">(1,2,14,32)</a> For the case of the D2R in complex with an agonist, the MD simulations revealed in an additional Na<sup>+</sup> ion located in the orthosteric ligand binding site in the extracellular region (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>, right panel). Different from previous findings, this newly discovered Na<sup>+</sup> enhances ligand binding by coordinating with both a His amino acid in TM6 of D2R and a negatively charged group in the ligand (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A), which in turn facilitates signaling of agonist molecules. Both site-specific mutagenesis experiments and structureâactivity relationship (SAR) studies are in excellent agreement with our proposed model. The complementary site-specific mutations in the ligand binding pocket of the D2R and the properties of the modified agonists are also in agreement with the proposed role of allosteric and orthosteric Na<sup>+</sup> ions for the activation of the D2R. An identical interaction was also observed in the high-resolution crystal structure of Î¼OR by reresolving the structure.</div><div class="NLM_p last">Our findings reveal details of the role of sodium ions as well as other metal ions in GPCR signaling. These findings could be used to design unique GPCR drug molecules or to optimize ligand properties.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43412" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43412" class="dropBlock__holder"></div></div></div></div></div><hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Tango Arrestin Recruitment Assay</h3><div class="NLM_p last">The D2R-arrestin recruitment assays were performed as previously described. <a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Briefly, HTLA cells expressing the TEV fused-Î²-arrestin2 were transfected with 1 Î¼g D2R wild-type or mutant DNA per 10 cm dish in 10% dialyzed FBS DMEM. The next day, cells were plated into white 384-well plates at a density of 15â¯000 cells per well in 20 Î¼L of 1% dialyzed FBS DMEM. After 6 h cells were stimulated with drugs ranging in concentration from 30 Î¼M to 1 pM diluted in drug buffer (20 mM HEPES, 1X HBSS, 0.1% BSA, 0.01% ascorbic acid, pH 7.4) at a 3-fold concentration. After overnight incubation at 37 Â°C and 5% CO<sub>2</sub>, media was decanted and 20 Î¼L of BriteGlo (Promega, after 1:20 dilution) was added per well. After 20 min, plates were read on an EnVision (PerkinElmer) at 1 ms per well. Luminescence counts per second (LCPS) were plotted as a function of drug concentration and analyzed using Graphpad Prism 8.0. Data were normalized to percent WT response, which was present in every experiment.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Split Luciferase Biosensor cAMP Assay for Measuring Activation of Gi Protein</h3><div class="NLM_p last">To determine G<sub>i</sub>-GPCR mediated cAMP production, we used Promegaâs split luciferase based GloSensor cAMP biosensor technology. On the first day, 1 Î¼g of target receptor DNA and 1 Î¼g of GloSensor cAMP DNA were cotransfected into ATCC HEK 293T by Lipofectamine 2000. After a minimum incubation of 6 h, cells are seeded into 384-well white clear bottom cell culture plates with F-12 supplemented with 10% FBS at a density of 10â15â¯000 cells in 40 Î¼L of medium per well. The assay was carried out after overnight incubation. On the day of the assay, supernatant was removed from the plates. Wells are loaded for 60 min at 37Â°C with 20 Î¼L of 2 mg/mL luciferin prepared in HBSS, pH 7.4. All the following steps were carried out at room temperature. To measure agonist activity at D2R, 10 Î¼L fo 4Ã test drug solution was added for 15 min before addition of 10 Î¼L of forskolin at a final concentration of 20 Î¼M, followed by counting of the plate for chemiluminescence after 15 min.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Organic Synthesis of Compounds</h3><div id="sec3_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Experimental Procedures for Organic Synthesis</h4><div class="NLM_p last">All commercial chemicals and solvents were used as obtained from the manufacturer without further purification. Flash chromatography were run on 200â300 mesh silica gel using a Teledyne CombiFlash instrument. <sup>1</sup>H NMR spectra were recorded on a Bruker AVANCE-III spectrometer at 800 MHz. <sup>13</sup>C NMR spectra were recorded on a Bruker AVANCE-III spectrometer at 151 MHz. NMR chemical shifts were reported in Î´ (ppm) using residual solvent peaks as standards (CDCl<sub>3</sub>â7.26 (H), 77.16 (C); CD<sub>3</sub>ODâ3.31 (H), 49.00 (C); DMSO-<i>d</i><sub>6</sub>â2.50 (H), 39.52 (C)). Mass spectra were measured using an LCMS-IT-TOF (Shimadzu) mass spectrometer in ESI mode. The purity of all final compounds (&gt;95%) were determined by analytical HPLC (Shim-pack GIST C18 column (250 Ã 4.6 mm, particle size 5 Î¼M); 0.05% TFA in H<sub>2</sub>O/0.05% TFA in MeOH gradient eluting system; flow rate = 1.0 mL/min). Preparative HPLC was conducted using Shimadzu HPLC system (Shim-pack GIST C18 column (250 Ã 20 mm, particle size 5 Î¼M); H<sub>2</sub>O/MeOH gradient eluting system; flow rate = 10.0 mL/min).<img class="imgWithoutFigViewer " src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0007.gif" alt="" id="fx1"></div></div><div id="sec3_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> 5-Fluoro-7-((4-(pyridin-2-yl)piperazin-1-yl)methyl)quinolin-8-ol (<b>MLS-d1</b>)</h4><div class="NLM_p last">To a mixture of compound <b>a1</b> (300 mg, 1.84 mmol) in anhydrous EtOH (20 mL) was added 1-(pyridin-2-yl)piperazine hydrochloride <b>b</b> (550 mg, 2.76 mmol) and formaldehyde (37% aqueous solution, 0.30 mL, 4.05 mmol). The mixture was stirred at 80 Â°C for 12 h under N<sub>2</sub>. Then the pH of the mixture was adjusted to 8.0 by saturated aqueous NaHCO<sub>3</sub>. The mixture was then extracted with ethyl acetate (30 mL Ã 2). The combined extracts were washed with brine (10 mL Ã 2), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuum to give a residue. The residue was purified by silica gel chromatography (petroleum ether/ethyl acetate = 1/0 to 1/1) to obtain a yellow solid, which was stirred in hexane (2 mL) at room temperature for 2 h. Compound <b>MLS-d1</b> (110 mg, 17.7% yield) was collected by filtration as a white solid. <sup>1</sup>H NMR (800 MHz, CD<sub>3</sub>OD) Î´ 8.86 (d, <i>J</i> = 4.1 Hz, 1H), 8.43 (d, <i>J</i> = 8.5 Hz, 1H), 8.08 (d, <i>J</i> = 5.1 Hz, 1H), 7.60â7.53 (m, 2H), 7.22 (d, <i>J</i> = 10.4 Hz, 1H), 6.83 (d, <i>J</i> = 8.6 Hz, 1H), 6.68 (t, <i>J</i> = 6.1 Hz, 1H), 3.92 (s, 2H), 3.58 (t, <i>J</i> = 5.1 Hz, 4H), 2.72 (t, <i>J</i> = 5.1 Hz, 4H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 159.0, 149.3, 149.2 (d, <i>J</i> = 243.3 Hz), 147.8 (d, <i>J</i> = 3.1 Hz), 147.6, 137.9 (d, <i>J</i> = 3.1 Hz), 137.5, 129.0, 121.9, 119.5 (d, <i>J</i> = 6.6 Hz), 117.5 (d, <i>J</i> = 18.2 Hz), 113.0, 111.9 (d, <i>J</i> = 20.3 Hz), 107.1, 55.9, 52.4 (2C), 44.7 (2C). HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calculated for C<sub>19</sub>H<sub>20</sub>FN<sub>4</sub>O<sup>+</sup> 339.1616, found 339.1623.</div></div><div id="sec3_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> 7-((4-(Pyridin-2-yl)piperazin-1-yl)methyl)quinolin-8-ol (<b>MLS-d2</b>)</h4><div class="NLM_p last">To a mixture of compound <b>a2</b> (250 mg, 1.72 mmol) in anhydrous EtOH (20 mL) was added 1-(pyridin-2-yl)piperazine hydrochloride <b>b</b> (513.4 mg, 2.58 mmol) and formaldehyde (37% aqueous solution, 0.28 mL, 3.79 mmol). The reaction mixture was stirred at 80 Â°C for 12 h under N<sub>2</sub>. Then the pH of the mixture was adjusted to 8.0 with saturated aqueous NaHCO<sub>3</sub>. The mixture was then extracted with ethyl acetate (200 mL Ã 2). The combined organic extracts were washed with brine (100 mL Ã 2), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuum to give a residue, which was purified by preparative HPLC to give compound <b>MLS-d2</b> (45.7 mg, 8.3% yield) as a gray solid. <sup>1</sup>H NMR (800 MHz, CD<sub>3</sub>OD) Î´ 8.79 (d, <i>J</i> = 3.1 Hz, 1H), 8.24 (dd, <i>J</i> = 8.3, 1.6 Hz, 1H), 8.08 (dd, <i>J</i> = 5.1, 1.9 Hz, 1H), 7.57â7.54 (m, 1H), 7.48 (dd, <i>J</i> = 8.3, 4.1 Hz, 1H), 7.40 (d, <i>J</i> = 8.3 Hz, 1H), 7.37 (d, <i>J</i> = 8.3 Hz, 1H), 6.82 (d, <i>J</i> = 8.6 Hz, 1H), 6.68 (dd, <i>J</i> = 7.1, 5.0 Hz, 1H), 3.95 (s, 2H), 3.58 (t, <i>J</i> = 5.2 Hz, 4H), 2.73 (t, <i>J</i> = 5.0 Hz, 4H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 159.0, 151.3, 148.2, 147.6, 138.2, 137.5, 136.0, 129.1, 127.7, 121.6, 119.7, 117.0, 113.0, 107.1, 56.3, 52.4 (2C), 44.7 (2C). HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calculated for C<sub>19</sub>H<sub>21</sub>N<sub>4</sub>O<sup>+</sup> 321.1710, found 321.1713.</div></div><div id="sec3_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> 5-Bromo-7-((4-(pyridin-2-yl)piperazin-1-yl)methyl)quinolin-8-ol (<b>MLS-d3</b>)</h4><div class="NLM_p last">To a mixture of <b>a3</b> (250 mg, 1.12 mmol) in anhydrous EtOH (20 mL) was added 1-(pyridin-2-yl)piperazine hydrochloride <b>b</b> (334 mg, 1.68 mmol) and formaldehyde (37% aqueous solution, 0.18 mL, 2.45 mmol). The mixture was stirred at 80 Â°C for 12 h under N<sub>2</sub>. Then the pH was adjusted to 8.0 by saturated aqueous NaHCO<sub>3</sub>. The mixture was then extracted with ethyl acetate (20 mL Ã 2). The combined extracts were washed with brine (10 mL Ã 2), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated in vacuum to give a residue, which was purified by preparative HPLC to give compound <b>MLS-d3</b> (21.0 mg, 5.8% yield) as a yellow solid. <sup>1</sup>H NMR (800 MHz, CD<sub>3</sub>OD) Î´ 8.85 (s, 1H), 8.51 (dd, <i>J</i> = 8.5, 1.5 Hz, 1H), 8.08 (dd, <i>J</i> = 5.1, 1.4 Hz, 1H), 7.77 (s, 1H), 7.63 (dd, <i>J</i> = 8.5, 4.2 Hz, 1H), 7.58â7.53 (m, 1H), 6.83 (d, <i>J</i> = 8.6 Hz, 1H), 6.68 (dd, <i>J</i> = 7.1, 5.0 Hz, 1H), 3.93 (s, 2H), 3.58 (t, <i>J</i> = 5.1 Hz, 4H), 2.73 (t, <i>J</i> = 5.1 Hz, 4H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 159.0, 151.6, 149.0, 147.6, 139.1, 137.5, 134.9, 132.2, 126.2, 123.1, 121.3, 113.0, 108.1, 107.1, 55.6, 52.4 (2C), 44.7 (2C). HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calculated for C<sub>19</sub>H<sub>20</sub>BrN<sub>4</sub>O<sup>+</sup> 399.0815, found 399.0827.</div></div><div id="sec3_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> 5-Nitro-7-((4-(pyridin-2-yl)piperazin-1-yl)methyl)quinolin-8-ol (<b>MLS-d4</b>)</h4><div class="NLM_p last">To a mixture of 1-(pyridin-2-yl)piperazine hydrochloride <b>b</b> (0.88 g, 4.41 mmol) in pyridine (3.0 mL) and H<sub>2</sub>O (3.0 mL) was added formaldehyde (37% aqueous solution, 0.36 mL, 4.41 mmol) at 50 Â°C. The reaction mixture was stirred at the same temperature for 5 min, and then added to a solution of <b>a4</b> (400 mg, 2.10 mmol) in pyridine (10.0 mL) at 50 Â°C under N<sub>2</sub>. The reaction mixture was stirred at 50 Â°C for 2 h, and then ethyl acetate (20 mL) was added at room temperature to give a yellow solid, which was collected by filtration and recrystallized from ethanol (20 mL) to give compound <b>MLS-d4</b> (100 mg, 13.0% yield) as a yellow solid. <sup>1</sup>H NMR (800 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.19 (d, <i>J</i> = 8.6 Hz, 1H), 8.75 (d, <i>J</i> = 3.2 Hz, 1H), 8.58 (s, 1H), 8.13 (dd, <i>J</i> = 4.7, 1.4 Hz, 1H), 7.67 (dd, <i>J</i> = 8.7, 4.0 Hz, 1H), 7.58â7.53 (m, 1H), 6.87 (d, <i>J</i> = 8.6 Hz, 1H), 6.68 (dd, <i>J</i> = 6.9, 5.0 Hz, 1H), 4.06 (s, 2H), 3.70 (t, <i>J</i> = 5.0 Hz, 4H), 3.02 (t, <i>J</i> = 5.0 Hz, 4H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 169.3, 158.5, 147.6, 147.0, 139.9, 137.8, 132.7, 132.2, 127.1, 124.5, 124.4, 115.6, 113.6, 107.5, 56.1, 51.7 (2C), 43.1 (2C). HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calculated for C<sub>19</sub>H<sub>20</sub>N<sub>5</sub>O<sub>3</sub><sup>+</sup> 366.1561, found 366.1556.</div></div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Synthesis of <b>a4</b></h3><img class="imgWithoutFigViewer " src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0008.gif" alt="" id="fx2"><div id="sec3_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 5-Nitrosoquinolin-8-ol (<b>a5</b>)</h4><div class="NLM_p last">Concentrated H<sub>2</sub>SO<sub>4</sub> (3 mL) was added slowly into H<sub>2</sub>O (65 mL), and the resulting solution was stirred vigorously at 15â18 Â°C. Compound <b>a1</b> (7.4 g, 50 mmol) was added. A solution of NaNO<sub>2</sub> (3.7 g, 54 mmol) in H<sub>2</sub>O (7 mL) was added dropwise over 30 min. The reaction mixture was stirred at 18â20 Â°C for 3 h, and then 20% NaOH was added slowly to adjust to pH 10â11, with the temperature kept below 25 Â°C. The mixture was filtered, and the filtrate was acidified to pH 5â6 with glacial acetic acid. The resulting solid was collected by filtration, washed with H<sub>2</sub>O (100 mL), and dried to give compound <b>a5</b> (8.4 g, 95% yield), which was used for the next step without further purification.</div></div><div id="sec3_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 5-Nitroquinolin-8-ol (<b>a4</b>)</h4><div class="NLM_p last">Fuming HNO<sub>3</sub> (0.10 mL) was added dropwise to the suspension of compound <b>a5</b> (7.5 g, 43 mmol) in acetic acid (23 mL), and the reaction mixture was stirred at 20 Â°C for 2 h. The mixture was basified at the temperature not exceeding 25 Â°C using 20% NaOH to pH 10â11, filtered, then the filtrate was acidified to pH 5â6 with glacial acetic acid. The formed precipitate was filtered, washed with H<sub>2</sub>O (100 mL), acetone (15 mL Ã 2), and then dried to give the compound <b>a4</b> (5.7 g, 70% yield).</div></div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Loop Modeling and Structural Preparations</h3><div id="sec3_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Loop Filling and Refinements</h4><div class="NLM_p last">The published crystal structures of D2R comprised the engineered receptors and inserted proteins in the intracellular loop ICL2 to facilitate crystallization. Before starting MD simulations, we removed the corresponding inserted proteins from the D2R crystal structures and used the loop refinement protocol in Modeler <a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> V9.10 to reconstruct and refine the ICL2 region. A total of 10â¯000 loops were generated for each receptor and the conformation with the lowest DOPE (Discrete Optimized Protein Energy) score was chosen for receptor construction. Repaired models were submitted to Rosetta V3.4 for loop refinement with kinematic loop modeling methods. <a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Kinematic closure (KIC) is an analytical calculation procedure inspired by robotics techniques for rapidly determining possible conformations of linked objects subject to constraints. In the Rosetta KIC implementation, 2N â 6 backbone torsions of an N-residue peptide segment (called nonpivot torsions) were set to values drawn randomly from the Ramachandran space of each residue type, and the remaining 6 phi/psi torsions (called pivot torsions) were solved analytically by KIC.</div></div><div id="sec3_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Protein Structure Preparations</h4><div class="NLM_p last">All protein models were prepared in Schrodinger suite software under the OPLS_2005 force field. <a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Hydrogen atoms were added to the repaired crystal structures at physiological pH (7.4) with the PROPKA <a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> tool to optimize the hydrogen bond network provided by the Protein Preparation tool in Schrodinger. The highly conserved D80<sup>2.50</sup> has been found to be deprotonated. Since we only study the initial binding process of D2R, D2.50 in GPCR is always in a deprotnated state as indicated by previous study. <a onclick="showRef(event, 'ref34 ref47'); return false;" href="javascript:void(0);" class="ref ref34 ref47">(34,47)</a> Thus, we kept the deprotonated state for D80<sup>2.50</sup> during MD simulations. Constrained energy minimizations were carried out on the full-atomic models, until the RMSD of heavy atoms coveraged to 0.4 Ã.</div></div><div id="sec3_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Ligand Structure Preparations</h4><div class="NLM_p last">All ligand structures were prepared in Schrodinger software. The LigPrep module in Schrodinger 2015 suite software was introduced for geometric optimization by using the OPLS_2005 force field. The ionization state of ligands was calculated with the Epik <a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> tool employing Hammett and Taft methods in combination with ionization and tautomerization tools. <a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div></div><div id="sec3_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Molecular Dynamics Simulations</h4><div class="NLM_p">Membrane systems were built using the membrane building tool g_membed <a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> in Gromacs with the receptor crystal structure prealigned in the OPM (Orientations of Proteins in Membranes) database. <a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Pre-equilibrated 120 POPC lipids coupled with 8000 TIP3P water molecules in a box â¼73 Ã 73 Ã 92 Ã<sup>3</sup> were used to build the protein/membrane/water system. We modeled the protein, lipids, water, and ions using the CHARMM36m force field. <a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Ligands were assigned with CHARMM CgenFF force field. <a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Ligand geometry was submitted to the GAUSSIAN 09 program <a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> for optimization at the HartreeâFock 6-31G* level when generating force field parameters. The system was gradually heated from 0 to 310 K followed by a 1 ns initial equilibration at constant volume with the temperature set at 310 K. The backbone of a particular protein and the heavy atoms of a particular ligand were restrained during the equilibration steps. All bond lengths to hydrogen atoms were constrained with M-SHAKE. Nonbonded interactions were treated using the force switch of 10â12 Ã. Long-range electrostatic interactions were computed by the Particle Mesh Ewald (PME) summation scheme. All MD simulations were done in Gromacs. <a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> The simulation parameter files were obtained from CHARMM-GUI Web site. <a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Figures were prepared in PyMOL and Inkscape. <a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></div><div class="NLM_p last">For the simulation of the apo-GPCR forms, all Na<sup>+</sup> ions were presented in the bulk environment. The allosteric Na<sup>+</sup> ions diffused into the inner space of D2R at a certain stage of the MD simulations. In the simulation of D2R in complex with MLS1547, we kept a Na<sup>+</sup> ion at the allosteric site next to D<sup>2.50</sup>, which is sampled in the simulation of apo GPCR forms.</div></div><div id="sec3_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Identifying a Mg<sup>2+</sup> in the Î¼OR crystal structure</h4><div class="NLM_p last">Both coordinates, electron density map and topology files for PDB: <a href="http://doi.org/10.2210/pdb5CM1/pdb">5CM1</a> were obtained from PDB database. We resolved the structure again with programs Phenix <a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> and Coot. <a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> The geometry of molecule BU72 was optimized using quantum mechanical (QM) method at level B3LYP/6-31G* by Jaguar <a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> in Schrodinger software <a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> prior to the refinement. The Phenix was used for metal ion identification. <a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01157" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01157" class="dropBlock__holder"></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acscentsci.9b01247?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acscentsci.9b01247</a>.</p><ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">MD simulations data (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Movie 1: An allosteric sodium ion next to D2.50 was captured by MD simulations. (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_002.mp4" class="ext-link">MP4</a>)</p></li><li><p class="inline">Movie 2: A new orthosteric sodium ion, which established coordinations with both receptor and ligand, was captured by MD simulations. (<a href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_003.mp4" class="ext-link">MP4</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_liveslides.mp4">oc9b01247_liveslides.mp4 (7.29 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf">oc9b01247_si_001.pdf (1.9 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_002.mp4">oc9b01247_si_002.mp4 (8.86 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_003.mp4">oc9b01247_si_003.mp4 (3.71 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acscentsci.9b01247" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms &amp; Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66702" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66702" class="dropBlock__holder"></div></div></div></div></div><hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianjun Cheng</span> - <span class="hlFld-Affiliation affiliation">iHuman
Institute, ShanghaiTech University, Shanghai 201210, China</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-6065-2682" title="Orcid link">http://orcid.org/0000-0001-6065-2682</a>;&nbsp;
    </span><span class="author-email">
                Email: <a href="mailto:chengjj@shanghaitech.edu.cn">chengjj@shanghaitech.edu.cn</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dong Wu</span> - <span class="hlFld-Affiliation affiliation">iHuman
Institute, ShanghaiTech University, Shanghai 201210, China</span>;&nbsp;
    <span class="author-email">
                Email: <a href="mailto:wudong@shanghaitech.edu.cn">wudong@shanghaitech.edu.cn</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shuguang Yuan</span> - <span class="hlFld-Affiliation affiliation">Shenzhen
Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen 518055, China</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-9858-4742" title="Orcid link">http://orcid.org/0000-0001-9858-4742</a>;&nbsp;
    </span><span class="author-email">
                Email: <a href="mailto:shuguang.yuan@siat.ac.cn">shuguang.yuan@siat.ac.cn</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">H. C. Stephen Chan</span> - <span class="hlFld-Affiliation affiliation">Shenzhen
Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen 518055, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yueming Xu</span> - <span class="hlFld-Affiliation affiliation">iHuman
Institute, ShanghaiTech University, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liang Tan</span> - <span class="hlFld-Affiliation affiliation">iHuman
Institute, ShanghaiTech University, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Horst Vogel</span> - <span class="hlFld-Affiliation affiliation">Shenzhen
Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen 518055, China</span>;&nbsp;
    <span class="hlFld-Affiliation affiliation">Institute
of Chemical Sciences and Engineering, Ecole
Polytechnique FÃ©dÃ©rale de Lausanne (EPFL), Lausanne 1015, Switzerland</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-9695-8894" title="Orcid link">http://orcid.org/0000-0001-9695-8894</a></span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): H.C.S.C., H.V., and S.Y. are the cofounders of AlphaMol Science Ltd.<br></div></li></ul></div><div class="ack" id="ACK-d7e1701-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44254" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44254" class="dropBlock__holder"></div></div></div></div></div><hr><p class="last">We would like to thank Fei Li from iHuman Institute for repeating some functional analyses. Our calculations were supported by the Interdisciplinary Centre for Mathematical and Computational Modelling in Warsaw (grant nos. GB70-3 &amp; GB71-3). This project was financially supported by the internal funding of Shenzhen Institutes of Advanced Technology, CAS.</p></div><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">Abbreviations</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23130" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23130" class="dropBlock__holder"></div></div></div></div></div><hr><table summary="" class="NLM_def-list" id="dl1"><tbody><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G protein-coupled receptor</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">ECL2</td><td class="NLM_def"><p class="first last">extracellular loop 2</p></td></tr></tbody></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68388" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68388" class="dropBlock__holder"></div></div></div></div></div><hr></div><p class="references-count">
            This article references 60 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenalti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abola, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Allosteric sodium in class A GPCR signaling</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">233</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.tibs.2014.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.tibs.2014.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=24767681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFCqs7k%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=2014&amp;pages=233&amp;issue=5&amp;author=V.+Katritchauthor=G.+Fenaltiauthor=E.+E.+Abolaauthor=B.+L.+Rothauthor=V.+Cherezovauthor=R.+C.+Stevens&amp;title=Allosteric+sodium+in+class+A+GPCR+signaling&amp;doi=10.1016%2Fj.tibs.2014.03.002" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric sodium in class A GPCR signaling</span></div><div class="casAuthors">Katritch, Vsevolod; Fenalti, Gustavo; Abola, Enrique E.; Roth, Bryan L.; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">233-244</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Despite their functional and structural diversity, G-protein-coupled receptors (GPCRs) share a common mechanism of signal transduction via conformational changes in the seven-transmembrane (7TM) helical domain.  New major insights into this mechanism come from the recent crystallog. discoveries of a partially hydrated sodium ion that is specifically bound in the middle of the 7TM bundle of multiple class A GPCRs.  This review discusses the remarkable structural conservation and distinct features of the Na+ pocket in this most populous GPCR class, as well as the conformational collapse of the pocket upon receptor activation.  New insights help to explain allosteric effects of sodium on GPCR agonist binding and activation, and sodium's role as a potential co-factor in class A GPCR function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_QJ04qVNc3LVg90H21EOLACvtfcHk0ljVSca71kx8dg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXmsFCqs7k%253D&amp;md5=c217e7eb69c93b1c76559cc587b318b5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2014.03.002&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2014.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DAbola%26aufirst%3DE.%2BE.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DAllosteric%2520sodium%2520in%2520class%2520A%2520GPCR%2520signaling%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2014%26volume%3D39%26issue%3D5%26spage%3D233%26doi%3D10.1016%2Fj.tibs.2014.03.002" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez-de-Teran, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massink, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ijzerman, A. P.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">The role of a sodium ion binding site in the allosteric modulation of the A(2A) adenosine G protein-coupled receptor</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2175</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.str.2013.09.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.str.2013.09.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=24210756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslCls7nP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2013&amp;pages=2175&amp;issue=12&amp;author=H.+Gutierrez-de-Teranauthor=A.+Massinkauthor=D.+Rodriguezauthor=W.+Liuauthor=G.+W.+Hanauthor=J.+S.+Josephauthor=I.+Katritchauthor=L.+H.+Heitmanauthor=L.+Xiaauthor=A.+P.+Ijzerman&amp;title=The+role+of+a+sodium+ion+binding+site+in+the+allosteric+modulation+of+the+A%282A%29+adenosine+G+protein-coupled+receptor&amp;doi=10.1016%2Fj.str.2013.09.020" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of a Sodium Ion Binding Site in the Allosteric Modulation of the A2A Adenosine G Protein-Coupled Receptor</span></div><div class="casAuthors">Gutierrez-de-Teran, Hugo; Massink, Arnault; Rodriguez, David; Liu, Wei; Han, Gye Won; Joseph, Jeremiah S.; Katritch, Ilia; Heitman, Laura H.; Xia, Lizi; IJzerman, Adriaan P.; Cherezov, Vadim; Katritch, Vsevolod; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2175-2185</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The function of G protein-coupled receptors (GPCRs) can be modulated by a no. of endogenous allosteric mols.  In this study, we used mol. dynamics, radioligand binding, and thermostability expts. to elucidate the role of the recently discovered sodium ion binding site in the allosteric modulation of the human A2A adenosine receptor, conserved among class A GPCRs.  While the binding of antagonists and sodium ions to the receptor was noncompetitive in nature, the binding of agonists and sodium ions appears to require mutually exclusive conformational states of the receptor.  Amiloride analogs can also bind to the sodium binding pocket, showing distinct patterns of agonist and antagonist modulation.  These findings suggest that physiol. concns. of sodium ions affect functionally relevant conformational states of GPCRs and can help to design novel synthetic allosteric modulators or bitopic ligands exploiting the sodium ion binding pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq52sWIHLurqLVg90H21EOLACvtfcHk0lhvILU5KtGPNg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhslCls7nP&amp;md5=9ad6e9fc35031b50f8aea83e65d6257f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.09.020&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.09.020%26sid%3Dliteratum%253Aachs%26aulast%3DGutierrez-de-Teran%26aufirst%3DH.%26aulast%3DMassink%26aufirst%3DA.%26aulast%3DRodriguez%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DJoseph%26aufirst%3DJ.%2BS.%26aulast%3DKatritch%26aufirst%3DI.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26atitle%3DThe%2520role%2520of%2520a%2520sodium%2520ion%2520binding%2520site%2520in%2520the%2520allosteric%2520modulation%2520of%2520the%2520A%25282A%2529%2520adenosine%2520G%2520protein-coupled%2520receptor%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26issue%3D12%26spage%3D2175%26doi%3D10.1016%2Fj.str.2013.09.020" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller-Gallacher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nehme, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warne, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schertler, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, C. G.</span></span> <span> </span><span class="NLM_article-title">The 2.1 A resolution structure of cyanopindolol-bound beta1-adrenoceptor identifies an intramembrane Na+ ion that stabilises the ligand-free receptor</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e92727</span>, <span class="refDoi">&nbsp;DOI: 10.1371/journal.pone.0092727</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1371%2Fjournal.pone.0092727" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=24663151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVWnsrvJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2014&amp;pages=e92727&amp;issue=3&amp;author=J.+L.+Miller-Gallacherauthor=R.+Nehmeauthor=T.+Warneauthor=P.+C.+Edwardsauthor=G.+F.+Schertlerauthor=A.+G.+Leslieauthor=C.+G.+Tate&amp;title=The+2.1+A+resolution+structure+of+cyanopindolol-bound+beta1-adrenoceptor+identifies+an+intramembrane+Na%2B+ion+that+stabilises+the+ligand-free+receptor&amp;doi=10.1371%2Fjournal.pone.0092727" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The 2.1 Ã resolution structure of cyanopindolol-bound Î²1-adrenoceptor identifies an intramembrane Na+ ion that stabilizes the ligand-free receptor</span></div><div class="casAuthors">Miller-Gallacher, Jennifer L.; Nehme, Rony; Warne, Tony; Edwards, Patricia C.; Schertler, Gebhard F. X.; Leslie, Andrew G. W.; Tate, Christopher G.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e92727/1-e92727/9, 9 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The Î²1-adrenoceptor (Î²1AR) is a G protein-coupled receptor (GPCR) that is activated by the endogenous agonists adrenaline and noradrenaline.  We have detd. the structure of an ultra-thermostable Î²1AR mutant bound to the weak partial agonist cyanopindolol to 2.1 Ã resoln.  High-quality crystals (100 Î¼m plates) were grown in lipidic cubic phase without the assistance of a T4 lysozyme or BRIL fusion in cytoplasmic loop 3, which is commonly employed for GPCR crystn.  An intramembrane Na+ ion was identified coordinated to Asp872.50, Ser1283.39 and 3 water mols., which is part of a more extensive network of water mols. in a cavity formed between transmembrane helixes 1, 2, 3, 6 and 7.  Remarkably, this water network and Na+ ion is highly conserved between Î²1AR and the adenosine A2A receptor (rmsd of 0.3 Ã), despite an overall rmsd of 2.4 Ã for all CÎ± atoms and only 23% amino acid identity in the transmembrane regions.  The affinity of agonist binding and nanobody Nb80 binding to Î²1AR is unaffected by Na+ ions, but the stability of the receptor is decreased by 7.5Â°C in the absence of Na+.  Mutation of amino acid side chains that are involved in the co-ordination of either Na+ or water mols. in the network decreases the stability of Î²1AR by 5-10Â°C.  The data suggest that the intramembrane Na+ and assocd. water network stabilize the ligand-free state of Î²1AR, but still permits the receptor to form the activated state which involves the collapse of the Na+ binding pocket on agonist binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohMXYcl2xn6LVg90H21EOLACvtfcHk0lhvILU5KtGPNg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXhsVWnsrvJ&amp;md5=f3f2d68c124755ac1c91e52efa5575b2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0092727&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0092727%26sid%3Dliteratum%253Aachs%26aulast%3DMiller-Gallacher%26aufirst%3DJ.%2BL.%26aulast%3DNehme%26aufirst%3DR.%26aulast%3DWarne%26aufirst%3DT.%26aulast%3DEdwards%26aufirst%3DP.%2BC.%26aulast%3DSchertler%26aufirst%3DG.%2BF.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DTate%26aufirst%3DC.%2BG.%26atitle%3DThe%25202.1%2520A%2520resolution%2520structure%2520of%2520cyanopindolol-bound%2520beta1-adrenoceptor%2520identifies%2520an%2520intramembrane%2520Na%252B%2520ion%2520that%2520stabilises%2520the%2520ligand-free%2520receptor%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26issue%3D3%26spage%3De92727%26doi%3D10.1371%2Fjournal.pone.0092727" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betz, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatakrishnan, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">D4 dopamine receptor high-resolution structures enable the discovery of selective agonists</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span> (<span class="NLM_issue">6361</span>),  <span class="NLM_fpage">381</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.aan5468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1126%2Fscience.aan5468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29051383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Kns73I" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=358&amp;publication_year=2017&amp;pages=381&amp;issue=6361&amp;author=S.+Wangauthor=D.+Wackerauthor=A.+Levitauthor=T.+Cheauthor=R.+M.+Betzauthor=J.+D.+McCorvyauthor=A.+J.+Venkatakrishnanauthor=X.+P.+Huangauthor=R.+O.+Drorauthor=B.+K.+Shoichet&amp;title=D4+dopamine+receptor+high-resolution+structures+enable+the+discovery+of+selective+agonists&amp;doi=10.1126%2Fscience.aan5468" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">D4 dopamine receptor high-resolution structures enable the discovery of selective agonists</span></div><div class="casAuthors">Wang, Sheng; Wacker, Daniel; Levit, Anat; Che, Tao; Betz, Robin M.; McCorvy, John D.; Venkatakrishnan, A. J.; Huang, Xi-Ping; Dror, Ron O.; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">6361</span>),
    <span class="NLM_cas:pages">381-386</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Dopamine receptors are implicated in the pathogenesis and treatment of nearly every neuropsychiatric disorder.  Although thousands of drugs interact with these receptors, our mol. understanding of dopaminergic drug selectivity and design remains clouded.  To illuminate dopamine receptor structure, function, and ligand recognition, we detd. crystal structures of the D4 dopamine receptor in its inactive state bound to the antipsychotic drug nemonapride, with resolns. up to 1.95 angstroms.  These structures suggest a mechanism for the control of constitutive signaling, and their unusually high resoln. enabled a structure-based campaign for new agonists of the D4 dopamine receptor.  The ability to efficiently exploit structure for specific probe discovery-rapidly moving from elucidating receptor structure to discovering previously unrecognized, selective agonists-testifies to the power of structure-based approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8NDKhkv_ILbVg90H21EOLACvtfcHk0lgfyB3QO_4ZIg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Kns73I&amp;md5=0bbb93e6bcb33577f9d9b7459273d896</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.aan5468&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aan5468%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DLevit%26aufirst%3DA.%26aulast%3DChe%26aufirst%3DT.%26aulast%3DBetz%26aufirst%3DR.%2BM.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DVenkatakrishnan%26aufirst%3DA.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DD4%2520dopamine%2520receptor%2520high-resolution%2520structures%2520enable%2520the%2520discovery%2520of%2520selective%2520agonists%26jtitle%3DScience%26date%3D2017%26volume%3D358%26issue%3D6361%26spage%3D381%26doi%3D10.1126%2Fscience.aan5468" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weinert, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olieric, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozerov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, K.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Serial millisecond crystallography for routine room-temperature structure determination at synchrotrons</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">542</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41467-017-00630-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fs41467-017-00630-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=28912485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A280%3ADC%252BC1cboslKltg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2017&amp;pages=542&amp;issue=1&amp;author=T.+Weinertauthor=N.+Oliericauthor=R.+Chengauthor=S.+Brunleauthor=D.+Jamesauthor=D.+Ozerovauthor=D.+Gashiauthor=L.+Veraauthor=M.+Marshauthor=K.+Jaeger&amp;title=Serial+millisecond+crystallography+for+routine+room-temperature+structure+determination+at+synchrotrons&amp;doi=10.1038%2Fs41467-017-00630-4" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Serial millisecond crystallography for routine room-temperature structure determination at synchrotrons</span></div><div class="casAuthors">Weinert Tobias; Olieric Natacha; James Daniel; Gashi Dardan; Jaeger Kathrin; Skopintsev Petr; Prota Andrea E; Panneels Valerie; Nogly Przemyslaw; Schertler Gebhard; Steinmetz Michel O; Standfuss Jorg; Cheng Robert; Hennig Michael; Brunle Steffen; Ermler Ulrich; Ozerov Dmitry; Gashi Dardan; Vera Laura; Marsh May; Dworkowski Florian; Panepucci Ezequiel; Basu Shibom; Wang Meitian; Dore Andrew S; Geng Tian; Cooke Robert M; Liang Mengning; Schertler Gebhard; Steinmetz Michel O</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">542</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Historically, room-temperature structure determination was succeeded by cryo-crystallography to mitigate radiation damage.  Here, we demonstrate that serial millisecond crystallography at a synchrotron beamline equipped with high-viscosity injector and high frame-rate detector allows typical crystallographic experiments to be performed at room-temperature.  Using a crystal scanning approach, we determine the high-resolution structure of the radiation sensitive molybdenum storage protein, demonstrate soaking of the drug colchicine into tubulin and native sulfur phasing of the human G protein-coupled adenosine receptor.  Serial crystallographic data for molecular replacement already converges in 1,000-10,000 diffraction patterns, which we collected in 3 to maximally 82 minutes.  Compared with serial data we collected at a free-electron laser, the synchrotron data are of slightly lower resolution, however fewer diffraction patterns are needed for de novo phasing.  Overall, the data we collected by room-temperature serial crystallography are of comparable quality to cryo-crystallographic data and can be routinely collected at synchrotrons.Serial crystallography was developed for protein crystal data collection with X-ray free-electron lasers.  Here the authors present several examples which show that serial crystallography using high-viscosity injectors can also be routinely employed for room-temperature data collection at synchrotrons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2XuDrQ2dPti6Ufuk_1xoTfW6udTcc2eaYUrala-RbJbntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC1cboslKltg%253D%253D&amp;md5=2fc0843781c5237cc59b3b64fffc4106</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00630-4&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00630-4%26sid%3Dliteratum%253Aachs%26aulast%3DWeinert%26aufirst%3DT.%26aulast%3DOlieric%26aufirst%3DN.%26aulast%3DCheng%26aufirst%3DR.%26aulast%3DBrunle%26aufirst%3DS.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DOzerov%26aufirst%3DD.%26aulast%3DGashi%26aufirst%3DD.%26aulast%3DVera%26aufirst%3DL.%26aulast%3DMarsh%26aufirst%3DM.%26aulast%3DJaeger%26aufirst%3DK.%26atitle%3DSerial%2520millisecond%2520crystallography%2520for%2520routine%2520room-temperature%2520structure%2520determination%2520at%2520synchrotrons%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26issue%3D1%26spage%3D542%26doi%3D10.1038%2Fs41467-017-00630-4" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuybeda, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Waal, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Eps, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutka, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartesaghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erramilli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morizumi, T.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Cryo-EM structure of human rhodopsin bound to an inhibitory G protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>558</i></span> (<span class="NLM_issue">7711</span>),  <span class="NLM_fpage">553</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41586-018-0215-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fs41586-018-0215-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29899450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFeqtbjF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=558&amp;publication_year=2018&amp;pages=553&amp;issue=7711&amp;author=Y.+Kangauthor=O.+Kuybedaauthor=P.+W.+de+Waalauthor=S.+Mukherjeeauthor=N.+Van+Epsauthor=P.+Dutkaauthor=X.+E.+Zhouauthor=A.+Bartesaghiauthor=S.+Erramilliauthor=T.+Morizumi&amp;title=Cryo-EM+structure+of+human+rhodopsin+bound+to+an+inhibitory+G+protein&amp;doi=10.1038%2Fs41586-018-0215-y" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM structure of human rhodopsin bound to an inhibitory G protein</span></div><div class="casAuthors">Kang, Yanyong; Kuybeda, Oleg; de Waal, Parker W.; Mukherjee, Somnath; Van Eps, Ned; Dutka, Przemyslaw; Zhou, X. Edward; Bartesaghi, Alberto; Erramilli, Satchal; Morizumi, Takefumi; Gu, Xin; Yin, Yanting; Liu, Ping; Jiang, Yi; Meng, Xing; Zhao, Gongpu; Melcher, Karsten; Ernst, Oliver P.; Kossiakoff, Anthony A.; Subramaniam, Sriram; Xu, H. Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">558</span>
        (<span class="NLM_cas:issue">7711</span>),
    <span class="NLM_cas:pages">553-558</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) comprise the largest family of mammalian transmembrane receptors.  They mediate numerous cellular pathways by coupling with downstream signalling transducers, including the heterotrimeric G proteins Gs (stimulatory) and Gi (inhibitory) and several arrestin proteins.  The structural mechanisms that define how G-protein-coupled receptors selectively couple to a specific type of G protein or arrestin remain unknown.  Here, using cryo-electron microscopy, we show that the major interactions between activated rhodopsin and Gi are mediated by the C-terminal helix of the Gi Î±-subunit, which is wedged into the cytoplasmic cavity of the transmembrane helix bundle and directly contacts the amino terminus of helix 8 of rhodopsin.  Structural comparisons of inactive, Gi-bound and arrestin-bound forms of rhodopsin with inactive and Gs-bound forms of the Î²2-adrenergic receptor provide a foundation to understand the unique structural signatures that are assocd. with the recognition of Gs, Gi and arrestin by activated G-protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo98ChG6dDQrVg90H21EOLACvtfcHk0lgOfSytVsMaWQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhtFeqtbjF&amp;md5=e6d8e79977e0420442fa16ec0c53c692</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0215-y&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0215-y%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DY.%26aulast%3DKuybeda%26aufirst%3DO.%26aulast%3Dde%2BWaal%26aufirst%3DP.%2BW.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DVan%2BEps%26aufirst%3DN.%26aulast%3DDutka%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DX.%2BE.%26aulast%3DBartesaghi%26aufirst%3DA.%26aulast%3DErramilli%26aufirst%3DS.%26aulast%3DMorizumi%26aufirst%3DT.%26atitle%3DCryo-EM%2520structure%2520of%2520human%2520rhodopsin%2520bound%2520to%2520an%2520inhibitory%2520G%2520protein%26jtitle%3DNature%26date%3D2018%26volume%3D558%26issue%3D7711%26spage%3D553%26doi%3D10.1038%2Fs41586-018-0215-y" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velgy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span> <span> </span><span class="NLM_article-title">GPCRs: What Can We Learn from Molecular Dynamics Simulations?</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1705</i></span>,  <span class="NLM_fpage">133</span>, <span class="refDoi">&nbsp;DOI: 10.1007/978-1-4939-7465-8_6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1007%2F978-1-4939-7465-8_6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29188561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFCnsrzL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1705&amp;publication_year=2018&amp;pages=133&amp;author=N.+Velgyauthor=G.+Hedgerauthor=P.+C.+Biggin&amp;title=GPCRs%3A+What+Can+We+Learn+from+Molecular+Dynamics+Simulations%3F&amp;doi=10.1007%2F978-1-4939-7465-8_6" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">GPCRs: what can we learn from molecular dynamics simulations?</span></div><div class="casAuthors">Velgy, Naushad; Hedger, George; Biggin, Philip C.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1705</span>
        (<span class="NLM_cas:issue">Computational Methods for GPCR Drug Discovery</span>),
    <span class="NLM_cas:pages">133-158</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Advances in the structural biol. of G-protein Coupled Receptors have resulted in a significant step forward in our understanding of how this important class of drug targets function at the mol. level.  However, it has also become apparent that they are very dynamic mols., and moreover, that the underlying dynamics is crucial in shaping the response to different ligands.  Mol. dynamics simulations can provide unique insight into the dynamic properties of GPCRs in a way that is complementary to many exptl. approaches.  In this chapter, we describe progress in three distinct areas that are particularly difficult to study with other techniques: at. level investigation of the conformational changes that occur when moving between the various states that GPCRs can exist in, the pathways that ligands adopt during binding/unbinding events and finally, the influence of lipids on the conformational dynamics of GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFBdYfV8QTCrVg90H21EOLACvtfcHk0lgOfSytVsMaWQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhvFCnsrzL&amp;md5=6271b8fd13228616e0f34eab451e9f4a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-7465-8_6&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-7465-8_6%26sid%3Dliteratum%253Aachs%26aulast%3DVelgy%26aufirst%3DN.%26aulast%3DHedger%26aufirst%3DG.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DGPCRs%253A%2520What%2520Can%2520We%2520Learn%2520from%2520Molecular%2520Dynamics%2520Simulations%253F%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2018%26volume%3D1705%26spage%3D133%26doi%3D10.1007%2F978-1-4939-7465-8_6" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarrasch, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun Kobilka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Cryo-EM structure of the activated GLP-1 receptor in complex with a G protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>546</i></span> (<span class="NLM_issue">7657</span>),  <span class="NLM_fpage">248</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature22394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fnature22394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=28538729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosVagtLo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=546&amp;publication_year=2017&amp;pages=248&amp;issue=7657&amp;author=Y.+Zhangauthor=B.+Sunauthor=D.+Fengauthor=H.+Huauthor=M.+Chuauthor=Q.+Quauthor=J.+T.+Tarraschauthor=S.+Liauthor=T.+Sun+Kobilkaauthor=B.+K.+Kobilka&amp;title=Cryo-EM+structure+of+the+activated+GLP-1+receptor+in+complex+with+a+G+protein&amp;doi=10.1038%2Fnature22394" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM structure of the activated GLP-1 receptor in complex with a G protein</span></div><div class="casAuthors">Zhang, Yan; Sun, Bingfa; Feng, Dan; Hu, Hongli; Chu, Matthew; Qu, Qianhui; Tarrasch, Jeffrey T.; Li, Shane; Sun Kobilka, Tong; Kobilka, Brian K.; Skiniotis, Georgios</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">546</span>
        (<span class="NLM_cas:issue">7657</span>),
    <span class="NLM_cas:pages">248-253</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Glucagon-like peptide 1 (GLP-1) is a hormone with essential roles in regulating insulin secretion, carbohydrate metab. and appetite.  GLP-1 effects are mediated through binding to the GLP-1 receptor (GLP-1R), a class B G-protein-coupled receptor (GPCR) that signals primarily through the stimulatory G protein Gs.  Class B GPCRs are important therapeutic targets; however, our understanding of their mechanism of action is limited by the lack of structural information on activated and full-length receptors.  Here we report the cryo-electron microscopy structure of the peptide-activated GLP-1R-Gs complex at near at. resoln.  The peptide is clasped between the N-terminal domain and the transmembrane core of the receptor, and further stabilized by extracellular loops.  Conformational changes in the transmembrane domain result in a sharp kink in the middle of transmembrane helix 6, which pivots its intracellular half outward to accommodate the Î±5-helix of the Ras-like domain of Gs.  These results provide a structural framework for understanding class B GPCR activation through hormone binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmDRFGRwMs1bVg90H21EOLACvtfcHk0lgOfSytVsMaWQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXosVagtLo%253D&amp;md5=5e7ebd94012ba47ecc299ad0502975de</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature22394&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22394%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DChu%26aufirst%3DM.%26aulast%3DQu%26aufirst%3DQ.%26aulast%3DTarrasch%26aufirst%3DJ.%2BT.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSun%2BKobilka%26aufirst%3DT.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DCryo-EM%2520structure%2520of%2520the%2520activated%2520GLP-1%2520receptor%2520in%2520complex%2520with%2520a%2520G%2520protein%26jtitle%3DNature%26date%3D2017%26volume%3D546%26issue%3D7657%26spage%3D248%26doi%3D10.1038%2Fnature22394" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lodowski, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elstner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegemann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kandori, H.</span></span> <span> </span><span class="NLM_article-title">Microbial and animal rhodopsins: structures, functions, and molecular mechanisms</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">126</span>, <span class="refDoi">&nbsp;DOI: 10.1021/cr4003769</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr4003769" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOnt7jN" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=114&amp;publication_year=2014&amp;pages=126&amp;issue=1&amp;author=O.+P.+Ernstauthor=D.+T.+Lodowskiauthor=M.+Elstnerauthor=P.+Hegemannauthor=L.+S.+Brownauthor=H.+Kandori&amp;title=Microbial+and+animal+rhodopsins%3A+structures%2C+functions%2C+and+molecular+mechanisms&amp;doi=10.1021%2Fcr4003769" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Microbial and animal rhodopsins: Structures, functions, and molecular mechanisms</span></div><div class="casAuthors">Ernst, Oliver P.; Lodowski, David T.; Elstner, Marcus; Hegemann, Peter; Brown, Leonid S.; Kandori, Hideki</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">126-163</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The authors provide mechanistic insights from biophys. and structural studies into the function of microbial and animal rhodopsins, with the latter as representatives of G protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5hz035KTkrrVg90H21EOLACvtfcHk0ljxu6DyaJFodA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOnt7jN&amp;md5=74e6baeb254a11b90c2a2a2aab6e942a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fcr4003769&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr4003769%26sid%3Dliteratum%253Aachs%26aulast%3DErnst%26aufirst%3DO.%2BP.%26aulast%3DLodowski%26aufirst%3DD.%2BT.%26aulast%3DElstner%26aufirst%3DM.%26aulast%3DHegemann%26aufirst%3DP.%26aulast%3DBrown%26aufirst%3DL.%2BS.%26aulast%3DKandori%26aufirst%3DH.%26atitle%3DMicrobial%2520and%2520animal%2520rhodopsins%253A%2520structures%252C%2520functions%252C%2520and%2520molecular%2520mechanisms%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26issue%3D1%26spage%3D126%26doi%3D10.1021%2Fcr4003769" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chubukov, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Structural Basis for Allosteric Regulation of GPCRs by Sodium Ions</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>337</i></span> (<span class="NLM_issue">6091</span>),  <span class="NLM_fpage">232</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.1219218</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1126%2Fscience.1219218" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=22798613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVKqtbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=337&amp;publication_year=2012&amp;pages=232&amp;issue=6091&amp;author=W.+Liuauthor=E.+Chunauthor=A.+A.+Thompsonauthor=P.+Chubukovauthor=F.+Xuauthor=V.+Katritchauthor=G.+W.+Hanauthor=C.+B.+Rothauthor=L.+H.+Heitmanauthor=A.+P.+IJzerman&amp;title=Structural+Basis+for+Allosteric+Regulation+of+GPCRs+by+Sodium+Ions&amp;doi=10.1126%2Fscience.1219218" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Allosteric Regulation of GPCRs by Sodium Ions</span></div><div class="casAuthors">Liu, Wei; Chun, Eugene; Thompson, Aaron A.; Chubukov, Pavel; Xu, Fei; Katritch, Vsevolod; Han, Gye Won; Roth, Christopher B.; Heitman, Laura H.; IJzerman, Adriaan P.; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">337</span>
        (<span class="NLM_cas:issue">6091</span>),
    <span class="NLM_cas:pages">232-236</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Pharmacol. responses of G protein-coupled receptors (GPCRs) can be fine-tuned by allosteric modulators.  Structural studies of such effects have been limited due to the medium resoln. of GPCR structures.  We reengineered the human A2A adenosine receptor by replacing its third intracellular loop with apocytochrome b562RIL and solved the structure at 1.8 angstrom resoln.  The high-resoln. structure allowed us to identify 57 ordered water mols. inside the receptor comprising three major clusters.  The central cluster harbors a putative sodium ion bound to the highly conserved aspartate residue Asp2.50.  Addnl., two cholesterols stabilize the conformation of helix VI, and one of 23 ordered lipids intercalates inside the ligand-binding pocket.  These high-resoln. details shed light on the potential role of structured water mols., sodium ions, and lipids/cholesterol in GPCR stabilization and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvcjQimGnIMLVg90H21EOLACvtfcHk0ljxu6DyaJFodA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XpvVKqtbs%253D&amp;md5=782b2235b76ef77b11b6456f8e4485e1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.1219218&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1219218%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DChun%26aufirst%3DE.%26aulast%3DThompson%26aufirst%3DA.%2BA.%26aulast%3DChubukov%26aufirst%3DP.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DRoth%26aufirst%3DC.%2BB.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26atitle%3DStructural%2520Basis%2520for%2520Allosteric%2520Regulation%2520of%2520GPCRs%2520by%2520Sodium%2520Ions%26jtitle%3DScience%26date%3D2012%26volume%3D337%26issue%3D6091%26spage%3D232%26doi%3D10.1126%2Fscience.1219218" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fenalti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giguere, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Molecular control of delta-opioid receptor signalling</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>506</i></span>,  <span class="NLM_fpage">191</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature12944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fnature12944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=24413399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVaktro%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=506&amp;publication_year=2014&amp;pages=191&amp;author=G.+Fenaltiauthor=P.+M.+Giguereauthor=V.+Katritchauthor=X.+P.+Huangauthor=A.+A.+Thompsonauthor=V.+Cherezovauthor=B.+L.+Rothauthor=R.+C.+Stevens&amp;title=Molecular+control+of+delta-opioid+receptor+signalling&amp;doi=10.1038%2Fnature12944" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular control of Î´-opioid receptor signalling</span></div><div class="casAuthors">Fenalti, Gustavo; Giguere, Patrick M.; Katritch, Vsevolod; Huang, Xi-Ping; Thompson, Aaron A.; Cherezov, Vadim; Roth, Bryan L.; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">506</span>
        (<span class="NLM_cas:issue">7487</span>),
    <span class="NLM_cas:pages">191-196</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Opioids represent widely prescribed and abused medications, although their signal transduction mechanisms are not well understood.  Here we present the 1.8 Ã high-resoln. crystal structure of the human Î´-opioid receptor (Î´-OR), revealing the presence and fundamental role of a sodium ion in mediating allosteric control of receptor functional selectivity and constitutive activity.  The distinctive Î´-OR sodium ion site architecture is centrally located in a polar interaction network in the seven-transmembrane bundle core, with the sodium ion stabilizing a reduced agonist affinity state, and thereby modulating signal transduction.  Site-directed mutagenesis and functional studies reveal that changing the allosteric sodium site residue Asn 131 to an alanine or a valine augments constitutive Î²-arrestin-mediated signaling.  Asp95Ala, Asn310Ala and Asn314Ala mutations transform classical Î´-opioid antagonists such as naltrindole into potent Î²-arrestin-biased agonists.  The data establish the mol. basis for allosteric sodium ion control in opioid signaling, revealing that sodium-coordinating residues act as 'efficacy switches' at a prototypic G-protein-coupled receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUFRulZU1cfrVg90H21EOLACvtfcHk0ljxu6DyaJFodA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXisVaktro%253D&amp;md5=88159956324bb2858286a4100a9cb8bf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature12944&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12944%26sid%3Dliteratum%253Aachs%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DGiguere%26aufirst%3DP.%2BM.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DThompson%26aufirst%3DA.%2BA.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DMolecular%2520control%2520of%2520delta-opioid%2520receptor%2520signalling%26jtitle%3DNature%26date%3D2014%26volume%3D506%26spage%3D191%26doi%3D10.1038%2Fnature12944" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H.</span></span> <span> </span><span class="NLM_article-title">W246<sup>6.48</sup> Opens a Gate for a Continuous Intrinsic Water Pathway during Activation of the Adenosine A Receptor</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">556</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.201409679</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1002%2Fanie.201409679" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=54&amp;publication_year=2014&amp;pages=556&amp;issue=2&amp;author=S.+Yuanauthor=Z.+Huauthor=S.+Filipekauthor=H.+Vogel&amp;title=W2466.48+Opens+a+Gate+for+a+Continuous+Intrinsic+Water+Pathway+during+Activation+of+the+Adenosine+A+Receptor&amp;doi=10.1002%2Fanie.201409679" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fanie.201409679&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201409679%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DFilipek%26aufirst%3DS.%26aulast%3DVogel%26aufirst%3DH.%26atitle%3DW2466.48%2520Opens%2520a%2520Gate%2520for%2520a%2520Continuous%2520Intrinsic%2520Water%2520Pathway%2520during%2520Activation%2520of%2520the%2520Adenosine%2520A%2520Receptor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D54%26issue%3D2%26spage%3D556%26doi%3D10.1002%2Fanie.201409679" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ericksen, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schetz, J. A.</span></span> <span> </span><span class="NLM_article-title">Ligand selectivity of D2 dopamine receptors is modulated by changes in local dynamics produced by sodium binding</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>328</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">40</span>, <span class="refDoi">&nbsp;DOI: 10.1124/jpet.108.141531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1124%2Fjpet.108.141531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=18849360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFymtw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=328&amp;publication_year=2009&amp;pages=40&amp;issue=1&amp;author=S.+S.+Ericksenauthor=D.+F.+Cummingsauthor=H.+Weinsteinauthor=J.+A.+Schetz&amp;title=Ligand+selectivity+of+D2+dopamine+receptors+is+modulated+by+changes+in+local+dynamics+produced+by+sodium+binding&amp;doi=10.1124%2Fjpet.108.141531" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand selectivity of D2 dopamine receptors is modulated by changes in local dynamics produced by sodium binding</span></div><div class="casAuthors">Ericksen, Spencer S.; Cummings, David F.; Weinstein, Harel; Schetz, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">328</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-54</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">We have uncovered a significant allosteric response of the D2 dopamine receptor to physiol. relevant concns. of sodium (140 mM), characterized by a sodium-enhanced binding affinity for a D4-selective class of agonists and antagonists.  This enhancement is significantly more pronounced in a D2-V2.61(91)F mutant and cannot be mimicked by an equiv. concn. of the sodium replacement cation N-methyl-D-glucamine.  This phenomenon was explored computationally at the mol. level by analyzing the effect of sodium binding on the dynamic properties of D2 receptor model constructs.  Normal mode anal. identified one mode (M19), which is involved in the open/closed motions of the binding cleft as being particularly sensitive to the sodium effect.  To examine the consequences for D2 receptor ligand recognition, one of the ligands, L-745,870 [3-{[4-(4-chlorophenyl) piperazin-1-yl]methyl}-1H-pyrrolo[2,3-b]pyridine or CPPMA, chlorophenylpiperazinyl methylazaindole], was docked into conformers along the M19 trajectory.  Structurally and pharmacol. well established ligand-receptor interactions, including the ionic interaction with D3.32(114) and interactions between the ligand aryl moieties and V2.61(91)F, were achieved only in "open" phase conformers.  The docking of (-)-raclopride [3,5-dichloro-N-(1-ethylpyrrolidin-2-ylmethyl)-2-hydroxy-6-methoxybenzamide] suggests that the same binding cleft changes in response to sodium-binding perturbation account as well for the enhancements in binding affinity for substituted benzamides in the wild-type D2 receptor.  Our findings demonstrate how key interactions can be modulated by occupancy at an allosteric site and are consistent with a mechanism in which sodium binding enhances the affinity of selected ligands through dynamic changes that increase accessibility of substituted benzamides and 1,4-DAP ligands to the orthosteric site and accessibility of 1,4-DAPs to V2.61(91)F.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9peW37CIIRLVg90H21EOLACvtfcHk0ljH8pXt68ZbOA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXisFymtw%253D%253D&amp;md5=24298d041694b3fc97bb93a057d53357</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1124%2Fjpet.108.141531&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.108.141531%26sid%3Dliteratum%253Aachs%26aulast%3DEricksen%26aufirst%3DS.%2BS.%26aulast%3DCummings%26aufirst%3DD.%2BF.%26aulast%3DWeinstein%26aufirst%3DH.%26aulast%3DSchetz%26aufirst%3DJ.%2BA.%26atitle%3DLigand%2520selectivity%2520of%2520D2%2520dopamine%2520receptors%2520is%2520modulated%2520by%2520changes%2520in%2520local%2520dynamics%2520produced%2520by%2520sodium%2520binding%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2009%26volume%3D328%26issue%3D1%26spage%3D40%26doi%3D10.1124%2Fjpet.108.141531" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipek, S.</span></span> <span> </span><span class="NLM_article-title">The Role of Water and Sodium Ions in the Activation of the mu-Opioid Receptor</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">10112</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.201302244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1002%2Fanie.201302244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Sku7fK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=2013&amp;pages=10112&amp;issue=38&amp;author=S.+Yuanauthor=H.+Vogelauthor=S.+Filipek&amp;title=The+Role+of+Water+and+Sodium+Ions+in+the+Activation+of+the+mu-Opioid+Receptor&amp;doi=10.1002%2Fanie.201302244" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Water and Sodium Ions in the Activation of the Î¼-Opioid Receptor</span></div><div class="casAuthors">Yuan, Shuguang; Vogel, Horst; Filipek, Slawomir</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">10112-10115</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH &amp; Co. KGaA</span>)
        </div><div class="casAbstract">The activation of G-protein-coupled receptors depends on the presence of water mols. inside the receptor and also on allosteric interactions.  The binding of sodium ions to the allosteric site of the Î¼ opioid receptor was studied by microsecond mol. dynamics simulations and their seemingly contradictory roles in preventing ligand binding and facilitating receptor activation were explained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFR_cJYR4RHbVg90H21EOLACvtfcHk0ljH8pXt68ZbOA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXht1Sku7fK&amp;md5=b5b29dbf482e8d309762e536c9fee2e7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fanie.201302244&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201302244%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DS.%26aulast%3DVogel%26aufirst%3DH.%26aulast%3DFilipek%26aufirst%3DS.%26atitle%3DThe%2520Role%2520of%2520Water%2520and%2520Sodium%2520Ions%2520in%2520the%2520Activation%2520of%2520the%2520mu-Opioid%2520Receptor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26issue%3D38%26spage%3D10112%26doi%3D10.1002%2Fanie.201302244" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golebiowski, J.</span></span> <span> </span><span class="NLM_article-title">Allosteric Na(+)-binding site modulates CXCR4 activation</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">24915</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C8CP04134B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1039%2FC8CP04134B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=30238101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslGns73E" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2018&amp;pages=24915&amp;issue=38&amp;author=X.+Congauthor=J.+Golebiowski&amp;title=Allosteric+Na%28%2B%29-binding+site+modulates+CXCR4+activation&amp;doi=10.1039%2FC8CP04134B" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Na+-binding site modulates CXCR4 activation</span></div><div class="casAuthors">Cong, Xiaojing; Golebiowski, Jerome</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">24915-24920</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) control most cellular communications with the environment and are the largest protein family of drug targets.  As strictly regulated mol. machines, profound comprehension of their activation mechanism is expected to significantly facilitate structure-based drug design.  This study provides atomistic-level description of the activation dynamics of the C-X-C chemokine receptor type 4 (CXCR4), a class A GPCR and important drug target.  Using mol. dynamics and enhanced sampling, we demonstrate how mutations and protonation of conserved residues trigger activation through microswitches at the receptor core, while sodium ion - a known allosteric modulator - inhibits it.  The findings point to a conserved mechanism of activation and the allosteric modulation by sodium in the chemokine receptor family.  From the tech. aspect, the enhanced sampling protocol effectively samples receptor conformational changes toward activation, and differentiates three variants of the receptor by their basal activity.  This work provides structural basis and a powerful in silico tool for CXCR4 agonist design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraoDas3gvNr7Vg90H21EOLACvtfcHk0ljH8pXt68ZbOA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhslGns73E&amp;md5=a91f5c365c8001a6abe3fb40e87687c5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1039%2FC8CP04134B&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8CP04134B%26sid%3Dliteratum%253Aachs%26aulast%3DCong%26aufirst%3DX.%26aulast%3DGolebiowski%26aufirst%3DJ.%26atitle%3DAllosteric%2520Na%2528%252B%2529-binding%2520site%2520modulates%2520CXCR4%2520activation%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2018%26volume%3D20%26issue%3D38%26spage%3D24915%26doi%3D10.1039%2FC8CP04134B" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caliman, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span> <span> </span><span class="NLM_article-title">Allosteric effects of sodium ion binding on activation of the m3 muscarinic g-protein-coupled receptor</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>108</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1796</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bpj.2015.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.bpj.2015.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=25863070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvVSktLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=108&amp;publication_year=2015&amp;pages=1796&amp;issue=7&amp;author=Y.+Miaoauthor=A.+D.+Calimanauthor=J.+A.+McCammon&amp;title=Allosteric+effects+of+sodium+ion+binding+on+activation+of+the+m3+muscarinic+g-protein-coupled+receptor&amp;doi=10.1016%2Fj.bpj.2015.03.003" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric effects of sodium ion binding on activation of the M3 muscarinic G-protein-coupled receptor</span></div><div class="casAuthors">Miao, Yinglong; Caliman, Alisha D.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1796-1806</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are important membrane proteins that mediate cellular signaling and represent primary targets for about 1/3 of currently marketed drugs.  Recent x-ray crystallog. studies identified distinct conformations of GPCRs in the active and inactive states.  An allosteric Na+ ion was previously found bound to a highly conserved D2.50 residue in inactive GPCRs, whereas the D2.50 allosteric pocket became collapsed in active GPCR structures.  However, the dynamic mechanisms underlying these observations remain elusive.  Here, the authors aimed to understand the mechanistic effects of Na+ binding on dynamic activation of the M3 muscarinic GPCR through long-timescale accelerated mol. dynamics (aMD) simulations.  The results showed that with the D2.50 residue deprotonated, the M3 receptor was bound by an allosteric Na+ ion and confined mostly in the inactive state with remarkably reduced flexibility.  In contrast, the D2.50-protonated receptor did not exhibit Na+ ion binding to the D2.50 allosteric site and sampled a significantly larger conformational space.  The receptor activation was captured and characterized by large-scale structural rearrangements of the transmembrane helixes via dynamic H-bond and salt bridge interactions.  The residue motions were highly correlated during receptor activation.  Further network anal. revealed that the allosteric signaling between residue D2.50 and key residues in the intracellular, extracellular, and orthosteric pockets was significantly weakened upon Na+ ion binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRzke7B8ju_bVg90H21EOLACvtfcHk0lgUVMcYDtcGmQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXkvVSktLg%253D&amp;md5=ab64d73ef6816c8ea7000616368bab92</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2015.03.003&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2015.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DMiao%26aufirst%3DY.%26aulast%3DCaliman%26aufirst%3DA.%2BD.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DAllosteric%2520effects%2520of%2520sodium%2520ion%2520binding%2520on%2520activation%2520of%2520the%2520m3%2520muscarinic%2520g-protein-coupled%2520receptor%26jtitle%3DBiophys.%2520J.%26date%3D2015%26volume%3D108%26issue%3D7%26spage%3D1796%26doi%3D10.1016%2Fj.bpj.2015.03.003" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Makman, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvorkin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, P. N.</span></span> <span> </span><span class="NLM_article-title">Sodium ion modulates D2 receptor characteristics of dopamine agonist and antagonist binding sites in striatum and retina</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>79</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">4212</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.79.13.4212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1073%2Fpnas.79.13.4212" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=6213964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADyaL38XlsFKqu7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=79&amp;publication_year=1982&amp;pages=4212&amp;issue=13&amp;author=M.+H.+Makmanauthor=B.+Dvorkinauthor=P.+N.+Klein&amp;title=Sodium+ion+modulates+D2+receptor+characteristics+of+dopamine+agonist+and+antagonist+binding+sites+in+striatum+and+retina&amp;doi=10.1073%2Fpnas.79.13.4212" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Sodium ion modulates D2 receptor characteristics of dopamine agonist and antagonist binding sites in striatum and retina</span></div><div class="casAuthors">Makman, Maynard H.; Dvorkin, B.; Klein, Patrice N.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4212-16</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Na+ influences binding of both dopamine  [51-61-6] agonists and antagonists to D2 receptors in striatum and retina.  Addnl., Na+ markedly potentiates the loss of high-affinity agonist binding due to the GTP analog p[NH]ppG  [34273-04-6].  3H-labeled 2-amino-6,7-dihydroxy-1,2,3,4-tetrahydronaphthalene (ADTN)  [53463-78-8] binds exclusively to an agonist conformation of D2 receptor in both striatum and retina, distinct from the antagonist conformation labeled by 3H-labeled spiroperidol  [749-02-0] or 3H-labeled domperidone  [57808-66-9] in striatum or by [3H]spiroperidol in retina.  Na+ is not required for interaction of [3H]ADTN or antagonist radioligand sites with the selective D2 agonist LY-141865  [80373-22-4], the D2 antagonist domperidone, or nonselective dopamine agonists or antagonists; however, Na+ is necessary for high affinity interaction of those radioligand sites with the D2 antagonists molindone  [7416-34-4] and metoclopramide  [364-62-5].  With Na+ present, striatal sites for [3H]ADTN, [3H]spiroperidol, and [3H]domperidone have similar affinities for antagonists but only [3H]ADTN sites have high affinity for agonists, Na+ further decreases the low afffinity of dopamine agonists for [3H]spiroperidol binding sites.  Also, Na+ enhances [3H]spiroperidol and decreases [3H]ADTN binding.  Na+ alone causes bound [3H]ADTN to dissoc. from at least 30% of striatal and 50% of retinal sites, and with Na+ present [3H]ADTN rapidly dissocs. from the remaining sites upon addn. of p[NH]ppG.  Evidently, D2 receptors in striatum and retina exist in distinct but interconvertible conformational states, with different properties depending on the presence or absence of Na+ and of guanine nucleotide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTisY8zF0jmLVg90H21EOLACvtfcHk0lgUVMcYDtcGmQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL38XlsFKqu7c%253D&amp;md5=375a51b02e4bc2eb5fa57883a6f87501</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.79.13.4212&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.79.13.4212%26sid%3Dliteratum%253Aachs%26aulast%3DMakman%26aufirst%3DM.%2BH.%26aulast%3DDvorkin%26aufirst%3DB.%26aulast%3DKlein%26aufirst%3DP.%2BN.%26atitle%3DSodium%2520ion%2520modulates%2520D2%2520receptor%2520characteristics%2520of%2520dopamine%2520agonist%2520and%2520antagonist%2520binding%2520sites%2520in%2520striatum%2520and%2520retina%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1982%26volume%3D79%26issue%3D13%26spage%3D4212%26doi%3D10.1073%2Fpnas.79.13.4212" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeman, P.</span></span> <span> </span><span class="NLM_article-title">Regulation of anterior pituitary D2 dopamine receptors by magnesium and sodium ions</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1842</span>, <span class="refDoi">&nbsp;DOI: 10.1111/j.1471-4159.1985.tb10542.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1111%2Fj.1471-4159.1985.tb10542.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=2932536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADyaL28XhtlWr" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=1985&amp;pages=1842&amp;issue=6&amp;author=M.+Watanabeauthor=S.+R.+Georgeauthor=P.+Seeman&amp;title=Regulation+of+anterior+pituitary+D2+dopamine+receptors+by+magnesium+and+sodium+ions&amp;doi=10.1111%2Fj.1471-4159.1985.tb10542.x" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of anterior pituitary D2 dopamine receptors by magnesium and sodium ions</span></div><div class="casAuthors">Watanabe, Masayuki; George, Susan R.; Seeman, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1842-9</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    </div><div class="casAbstract">At D2 dopamine receptors in porcine anterior pituitary tissue, Mg2+ shifted receptors to agonist high-affinity states, but decreased the affinity of the antagonist [3H]spiperone.  Conversely, Na+ shifted the receptors to agonist low-affinity states, but increased the affinity of [3H]spiperone.  Mg is proposed to stabilize the hormone-receptor-guanine nucleotide regulatory protein complex, whereas Na appears to destabilize this ternary complex.  Thus, Mg and Na appear to mediate their regulatory effects via a common component at the D2 dopamine-receptor ternary complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHpRHyaW1tFrVg90H21EOLACvtfcHk0liD4nZ1o1QH2g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL28XhtlWr&amp;md5=5f8b07cf6e45c1aa216acbba5c10b0c9</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fj.1471-4159.1985.tb10542.x&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1471-4159.1985.tb10542.x%26sid%3Dliteratum%253Aachs%26aulast%3DWatanabe%26aufirst%3DM.%26aulast%3DGeorge%26aufirst%3DS.%2BR.%26aulast%3DSeeman%26aufirst%3DP.%26atitle%3DRegulation%2520of%2520anterior%2520pituitary%2520D2%2520dopamine%2520receptors%2520by%2520magnesium%2520and%2520sodium%2520ions%26jtitle%3DJ.%2520Neurochem.%26date%3D1985%26volume%3D45%26issue%3D6%26spage%3D1842%26doi%3D10.1111%2Fj.1471-4159.1985.tb10542.x" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neale, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sljoka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyda, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichugin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larda, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, O. P.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Mechanistic insights into allosteric regulation of the A2A adenosine G protein-coupled receptor by physiological cations</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1372</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41467-018-03314-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fs41467-018-03314-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29636462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A280%3ADC%252BC1MjgvVWmsw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2018&amp;pages=1372&amp;issue=1&amp;author=L.+Yeauthor=C.+Nealeauthor=A.+Sljokaauthor=B.+Lydaauthor=D.+Pichuginauthor=N.+Tsuchimuraauthor=S.+T.+Lardaauthor=R.+Pomesauthor=A.+E.+Garciaauthor=O.+P.+Ernst&amp;title=Mechanistic+insights+into+allosteric+regulation+of+the+A2A+adenosine+G+protein-coupled+receptor+by+physiological+cations&amp;doi=10.1038%2Fs41467-018-03314-9" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic insights into allosteric regulation of the A2A adenosine G protein-coupled receptor by physiological cations</span></div><div class="casAuthors">Ye Libin; Pichugin Dmitry; Larda Sacha T; Prosser R Scott; Ye Libin; Pichugin Dmitry; Larda Sacha T; Pomes Regis; Ernst Oliver P; Prosser R Scott; Neale Chris; Garcia Angel E; Sljoka Adnan; Tsuchimura Nobuyuki; Lyda Brent; Sunahara Roger K; Pomes Regis; Ernst Oliver P</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1372</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cations play key roles in regulating G-protein-coupled receptors (GPCRs), although their mechanisms are poorly understood.  Here, (19)F NMR is used to delineate the effects of cations on functional states of the adenosine A2A GPCR.  While Na(+) reinforces an inactive ensemble and a partial-agonist stabilized state, Ca(2+) and Mg(2+) shift the equilibrium toward active states.  Positive allosteric effects of divalent cations are more pronounced with agonist and a G-protein-derived peptide.  In cell membranes, divalent cations enhance both the affinity and fraction of the high affinity agonist-bound state.  Molecular dynamics simulations suggest high concentrations of divalent cations bridge specific extracellular acidic residues, bringing TM5 and TM6 together at the extracellular surface and allosterically driving open the G-protein-binding cleft as shown by rigidity-transmission allostery theory.  An understanding of cation allostery should enable the design of allosteric agents and enhance our understanding of GPCR regulation in the cellular milieu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ3cGpsWtuylRe4jJGKT5N0fW6udTcc2eZvb4parMvGFLntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC1MjgvVWmsw%253D%253D&amp;md5=5e106149aea3e56b585bc5af9a2e1a77</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-03314-9&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-03314-9%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DL.%26aulast%3DNeale%26aufirst%3DC.%26aulast%3DSljoka%26aufirst%3DA.%26aulast%3DLyda%26aufirst%3DB.%26aulast%3DPichugin%26aufirst%3DD.%26aulast%3DTsuchimura%26aufirst%3DN.%26aulast%3DLarda%26aufirst%3DS.%2BT.%26aulast%3DPomes%26aufirst%3DR.%26aulast%3DGarcia%26aufirst%3DA.%2BE.%26aulast%3DErnst%26aufirst%3DO.%2BP.%26atitle%3DMechanistic%2520insights%2520into%2520allosteric%2520regulation%2520of%2520the%2520A2A%2520adenosine%2520G%2520protein-coupled%2520receptor%2520by%2520physiological%2520cations%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26issue%3D1%26spage%3D1372%26doi%3D10.1038%2Fs41467-018-03314-9" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Draper-Joyce, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shonberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopinathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein Herenbrink, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuano, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramyan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thal, D. M.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">The action of a negative allosteric modulator at the dopamine D2 receptor is dependent upon sodium ions</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1208</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41598-018-19642-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fs41598-018-19642-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29352161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A280%3ADC%252BC1MvisFerug%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2018&amp;pages=1208&amp;issue=1&amp;author=C.+J.+Draper-Joyceauthor=R.+K.+Vermaauthor=M.+Michinoauthor=J.+Shonbergauthor=A.+Kopinathanauthor=C.+Klein+Herenbrinkauthor=P.+J.+Scammellsauthor=B.+Capuanoauthor=A.+M.+Abramyanauthor=D.+M.+Thal&amp;title=The+action+of+a+negative+allosteric+modulator+at+the+dopamine+D2+receptor+is+dependent+upon+sodium+ions&amp;doi=10.1038%2Fs41598-018-19642-1" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The action of a negative allosteric modulator at the dopamine D2 receptor is dependent upon sodium ions</span></div><div class="casAuthors">Draper-Joyce Christopher J; Klein Herenbrink Carmen; Thal David M; Christopoulos Arthur; Lane J Robert; Verma Ravi Kumar; Michino Mayako; Abramyan Ara M; Shi Lei; Michino Mayako; Shonberg Jeremy; Kopinathan Anitha; Scammells Peter J; Capuano Ben; Javitch Jonathan A; Javitch Jonathan A</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1208</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Sodium ions (Na(+)) allosterically modulate the binding of orthosteric agonists and antagonists to many class A G protein-coupled receptors, including the dopamine D2 receptor (D2R).  Experimental and computational evidences have revealed that this effect is mediated by the binding of Na(+) to a conserved site located beneath the orthosteric binding site (OBS).  SB269652 acts as a negative allosteric modulator (NAM) of the D2R that adopts an extended bitopic pose, in which the tetrahydroisoquinoline moiety interacts with the OBS and the indole-2-carboxamide moiety occupies a secondary binding pocket (SBP).  In this study, we find that the presence of a Na(+) within the conserved Na(+)-binding pocket is required for the action of SB269652.  Using fragments of SB269652 and novel full-length analogues, we show that Na(+) is required for the high affinity binding of the tetrahydroisoquinoline moiety within the OBS, and that the interaction of the indole-2-carboxamide moiety with the SBP determines the degree of Na(+)-sensitivity.  Thus, we extend our understanding of the mode of action of this novel class of NAM by showing it acts synergistically with Na(+) to modulate the binding of orthosteric ligands at the D2R, providing opportunities for fine-tuning of modulatory effects in future allosteric drug design efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTvIHTTptFmHTToTC3aBaesfW6udTcc2eZvb4parMvGFLntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC1MvisFerug%253D%253D&amp;md5=7126d1cd2e69be311c508c596d81cc2f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-19642-1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-19642-1%26sid%3Dliteratum%253Aachs%26aulast%3DDraper-Joyce%26aufirst%3DC.%2BJ.%26aulast%3DVerma%26aufirst%3DR.%2BK.%26aulast%3DMichino%26aufirst%3DM.%26aulast%3DShonberg%26aufirst%3DJ.%26aulast%3DKopinathan%26aufirst%3DA.%26aulast%3DKlein%2BHerenbrink%26aufirst%3DC.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26aulast%3DCapuano%26aufirst%3DB.%26aulast%3DAbramyan%26aufirst%3DA.%2BM.%26aulast%3DThal%26aufirst%3DD.%2BM.%26atitle%3DThe%2520action%2520of%2520a%2520negative%2520allosteric%2520modulator%2520at%2520the%2520dopamine%2520D2%2520receptor%2520is%2520dependent%2520upon%2520sodium%2520ions%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26issue%3D1%26spage%3D1208%26doi%3D10.1038%2Fs41598-018-19642-1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chien, E. Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Won Han, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javitch, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Structure of the Human Dopamine D3 Receptor in Complex with a D2/D3 Selective Antagonist</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>330</i></span> (<span class="NLM_issue">6007</span>),  <span class="NLM_fpage">1091</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.1197410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1126%2Fscience.1197410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=21097933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVahurvM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=330&amp;publication_year=2010&amp;pages=1091&amp;issue=6007&amp;author=E.+Y.+T.+Chienauthor=W.+Liuauthor=Q.+Zhaoauthor=V.+Katritchauthor=G.+Won+Hanauthor=M.+A.+Hansonauthor=L.+Shiauthor=A.+H.+Newmanauthor=J.+A.+Javitchauthor=V.+Cherezov&amp;title=Structure+of+the+Human+Dopamine+D3+Receptor+in+Complex+with+a+D2%2FD3+Selective+Antagonist&amp;doi=10.1126%2Fscience.1197410" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Human Dopamine D3 Receptor in Complex with a D2/D3 Selective Antagonist</span></div><div class="casAuthors">Chien, Ellen Y. T.; Liu, Wei; Zhao, Qiang; Katritch, Vsevolod; Won Han, Gye; Hanson, Michael A.; Shi, Lei; Newman, Amy Hauck; Javitch, Jonathan A.; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">330</span>
        (<span class="NLM_cas:issue">6007</span>),
    <span class="NLM_cas:pages">1091-1095</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Dopamine modulates movement, cognition, and emotion through activation of dopamine G protein-coupled receptors in the brain.  The crystal structure of the human dopamine D3 receptor (D3R) in complex with the small mol. D2R/D3R-specific antagonist eticlopride reveals important features of the ligand binding pocket and extracellular loops.  On the intracellular side of the receptor, a locked conformation of the ionic lock and two distinctly different conformations of intracellular loop 2 are obsd.  Docking of R-22, a D3R-selective antagonist, reveals an extracellular extension of the eticlopride binding site that comprises a second binding pocket for the aryl amide of R-22, which differs between the highly homologous D2R and D3R.  This difference provides direction to the design of D3R-selective agents for treating drug abuse and other neuropsychiatric indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOEcfSdjMm-7Vg90H21EOLACvtfcHk0ljopk35acDkqA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXhsVahurvM&amp;md5=12c6f343daea9fb14ca605a092f28533</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1126%2Fscience.1197410&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1197410%26sid%3Dliteratum%253Aachs%26aulast%3DChien%26aufirst%3DE.%2BY.%2BT.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DWon%2BHan%26aufirst%3DG.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DJavitch%26aufirst%3DJ.%2BA.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DStructure%2520of%2520the%2520Human%2520Dopamine%2520D3%2520Receptor%2520in%2520Complex%2520with%2520a%2520D2%252FD3%2520Selective%2520Antagonist%26jtitle%3DScience%26date%3D2010%26volume%3D330%26issue%3D6007%26spage%3D1091%26doi%3D10.1126%2Fscience.1197410" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Selley, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childers, S. R.</span></span> <span> </span><span class="NLM_article-title">Effects of sodium on agonist efficacy for G-protein activation in Î¼-opioid receptor-transfected CHO cells and rat thalamus</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>130</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">987</span>, <span class="refDoi">&nbsp;DOI: 10.1038/sj.bjp.0703382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fsj.bjp.0703382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10882382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlt1Slurs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=130&amp;publication_year=2000&amp;pages=987&amp;issue=5&amp;author=D.+E.+Selleyauthor=C.-C.+Caoauthor=Q.+Liuauthor=S.+R.+Childers&amp;title=Effects+of+sodium+on+agonist+efficacy+for+G-protein+activation+in+%CE%BC-opioid+receptor-transfected+CHO+cells+and+rat+thalamus&amp;doi=10.1038%2Fsj.bjp.0703382" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of sodium on agonist efficacy for G-protein activation in Î¼-opioid receptor-transfected CHO cells and rat thalamus</span></div><div class="casAuthors">Selley, Dana E.; Cao, Cheng-Chun; Liu, Qixu; Childers, Steven R.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">987-996</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Sodium ions inhibit spontaneous Gi/Go-coupled receptor activity and promote agonist-induced responses in vitro.  The effects of sodium on the relative efficacy of opioid agonists for G-protein activation was measured by guanosine-5'-O-(Î³-35S)-triphosphate ([35S]-GTPÎ³S) binding in membranes from two Î¼-opioid receptor-contg. systems: CHO cells stably transfected with mouse Î¼ receptors (mMOR-CHO cells) and rat thalamus.  NaCl inhibited basal [35S]-GTPÎ³S binding in both systems, and this effect was partially mimicked by KCl.  In mMOR-CHO membranes, net [35S]-GTPÎ³S binding stimulated by partial but not full agonists was inhibited by NaCl with a potency that was inversely proportional to agonist efficacy.  Monovalent cations were required for agonist-stimulated [35S]-GTPÎ³S binding in this system, and increasing NaCl concns. magnified relative efficacy differences among agonists.  In thalamic membranes, which contain a lower receptor:G-protein ratio than mMOR-CHO cells, similar monovalent cation effects were obsd., with two exceptions: [35S]-GTPÎ³S binding stimulated by both full and partial agonists was inhibited by NaCl; and monovalent cations were not required to observe agonist-stimulated [35S]-GTPÎ³S binding.  Basal [35S]-GTPÎ³S binding stimulated by the absence of monovalent cations resembled that of agonist-stimulated binding and was blocked by pretreatment of mMOR-CHO cells with pertussis toxin.  These results indicate that sodium inhibits spontaneous and agonist-occupied Î¼ receptor-mediated G-protein activation in a manner inversely proportional to the efficacy of the agonist, and that spontaneous Î¼ receptor activity and the relative efficacy of partial agonists acting at these receptors are both increased by increases in the stoichiometric ratio of receptors:G-proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEfDtcsF_WjrVg90H21EOLACvtfcHk0ljopk35acDkqA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3cXlt1Slurs%253D&amp;md5=76ef93dd8224c4df5ff888f1fd22fa67</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0703382&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0703382%26sid%3Dliteratum%253Aachs%26aulast%3DSelley%26aufirst%3DD.%2BE.%26aulast%3DCao%26aufirst%3DC.-C.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DChilders%26aufirst%3DS.%2BR.%26atitle%3DEffects%2520of%2520sodium%2520on%2520agonist%2520efficacy%2520for%2520G-protein%2520activation%2520in%2520%25CE%25BC-opioid%2520receptor-transfected%2520CHO%2520cells%2520and%2520rat%2520thalamus%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2000%26volume%3D130%26issue%3D5%26spage%3D987%26doi%3D10.1038%2Fsj.bjp.0703382" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neve, K. A.</span></span> <span> </span><span class="NLM_article-title">Regulation of dopamine D2 receptors by sodium and pH</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">570</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=2017157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADyaK3MXit1WqsbY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=1991&amp;pages=570&amp;issue=4&amp;author=K.+A.+Neve&amp;title=Regulation+of+dopamine+D2+receptors+by+sodium+and+pH" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of dopamine D2 receptors by sodium and pH</span></div><div class="casAuthors">Neve, Kim A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">570-8</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">The role of Na+ and H+ in the regulation of D2 receptor affinity for ligands was studied to det. the mol. mechanisms of this phenomenon.  The potency of substituted benzamide derivs. and agonists at D2 receptors depended on the concn. of Na+ and H+, whereas the potency of other antagonists was relatively unaltered by changes in pH or Na+ concn.  The potency of agonists was generally decreased in the presence of NaCl or lowered pH.  For example, in the absence of Na the affinity of D2 receptors for dopamine was decreased 17-fold by lowering of the pH from 8.0 to pH 6.8.  Addn. of NaCl caused 2-4-fold decreases in affinity for most agonists.  The affinity of the receptors for 2 substituted benzamide derivs., on the other hand, was reduced 6-44-fold by elevated concns. of H+ but was enhanced 7-24-fold in the presence of Na+.  The regulation by H+ of the potency of dopamine was selective for D2 receptors, because binding of dopamine to neostriatal D1 receptors was unaffected by changes in pH.  Decreasing of the pH from 8.0 or 7.3 to 6.8 facilitated the dissocn. of the substituted benzamide ligand [125I]epidepride from D2 receptors but inhibited dissocn. of [3H]spiperone.  Furthermore, the presence of NaCl or lowered pH slowed inactivation of D2 receptors by N-ethylmaleimide.  Apparently, the conformation of D2 receptors is regulated by both Na+ and H+.  The affinity of D2 receptors for agonists and substituted benzamide antagonists varies according to the conformational state of the receptors, whereas other antagonists bind to both forms with approx. equal potency.  Amiloride is a compd. that interacts with many Na-binding macromols.  At equil., amiloride inhibited the binding of [3H]spiperone and [125I]epidepride in a manner suggesting a more complex interaction than simple competitive inhibition.  The rate of dissocn. of both radioligands was enhanced by amiloride, as would be expected for allosteric inhibition of binding.  The sensitivity of D2 receptors to pH, Na, and amiloride may be a reflection of the ability of D2 receptors to modulate Na+/H+ exchange.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBIR1cElUacLVg90H21EOLACvtfcHk0ljopk35acDkqA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3MXit1WqsbY%253D&amp;md5=1bd0b0473860a2fef260d01ae9630cfb</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNeve%26aufirst%3DK.%2BA.%26atitle%3DRegulation%2520of%2520dopamine%2520D2%2520receptors%2520by%2520sodium%2520and%2520pH%26jtitle%3DMol.%2520Pharmacol.%26date%3D1991%26volume%3D39%26issue%3D4%26spage%3D570" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ananthanarayanan, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerman, A.</span></span> <span> </span><span class="NLM_article-title">Role of metal ions in ligand-receptor interaction: insights from structural studies</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>246</i></span> (<span class="NLM_issue">1â2</span>),  <span class="NLM_fpage">53</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.mce.2005.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.mce.2005.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=16368180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsF2gtLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=246&amp;publication_year=2006&amp;pages=53&amp;issue=1%E2%80%932&amp;author=V.+S.+Ananthanarayananauthor=A.+Kerman&amp;title=Role+of+metal+ions+in+ligand-receptor+interaction%3A+insights+from+structural+studies&amp;doi=10.1016%2Fj.mce.2005.11.023" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Role of metal ions in ligand-receptor interaction: Insights from structural studies</span></div><div class="casAuthors">Ananthanarayanan, Vettai S.; Kerman, Aaron</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">246</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">53-59</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Exptl. data indicate that metal ions such as Na+, Ca2+ and Mg2+, which are present in millimolar concns. in the extracellular environment, modulate binding of ligands to plasma membrane receptors.  Here, we briefly review structural studies that demonstrate that various types of ligands, including peptide hormones and drugs, bind metal ions, in particular Ca2+, in the lipid milieu.  We propose that the metal ion-bound forms of ligands represent their bioactive conformations.  With a view to understanding the mechanism of modulation of ligand-receptor interactions by metal ions, we have computed a homol. model of the mu-opioid receptor, a G protein-coupled receptor (GPCR), and performed docking of specific agonist and antagonist ligands in the receptor.  This resulted in the formation of a ligand-metal ion-receptor (ternary) complex which accounted for the data on the structure-activity relationships of ligands and mutation data on the receptor.  Based on exptl. and modeling data, we have proposed a general mechanism of activation of GPCRs by their corresponding ligands wherein metal ions play a pivotal role.  Studies on overexpressed segments of mu-receptor are in progress to verify the above proposal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCaUGp7DmQG7Vg90H21EOLACvtfcHk0lgBxenJEkSdrg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28XhsF2gtLg%253D&amp;md5=461fb14c7023afc315f2688480e16493</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2005.11.023&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2005.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DAnanthanarayanan%26aufirst%3DV.%2BS.%26aulast%3DKerman%26aufirst%3DA.%26atitle%3DRole%2520of%2520metal%2520ions%2520in%2520ligand-receptor%2520interaction%253A%2520insights%2520from%2520structural%2520studies%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2006%26volume%3D246%26issue%3D1%25E2%2580%25932%26spage%3D53%26doi%3D10.1016%2Fj.mce.2005.11.023" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strasser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seifert, R.</span></span> <span> </span><span class="NLM_article-title">Modulation of GPCRs by monovalent cations and anions</span>. <i>Naunyn-Schmiedeberg's Arch. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>388</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">363</span>, <span class="refDoi">&nbsp;DOI: 10.1007/s00210-014-1073-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1007%2Fs00210-014-1073-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=25432095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVSitLzF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=388&amp;publication_year=2015&amp;pages=363&amp;issue=3&amp;author=A.+Strasserauthor=H.+J.+Wittmannauthor=E.+H.+Schneiderauthor=R.+Seifert&amp;title=Modulation+of+GPCRs+by+monovalent+cations+and+anions&amp;doi=10.1007%2Fs00210-014-1073-2" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of GPCRs by monovalent cations and anions</span></div><div class="casAuthors">Strasser, Andrea; Wittmann, Hans-Joachim; Schneider, Erich H.; Seifert, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Naunyn-Schmiedeberg's Archives of Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">388</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">363-380</span>CODEN:
                <span class="NLM_cas:coden">NSAPCC</span>;
        ISSN:<span class="NLM_cas:issn">0028-1298</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The recent resoln. of G-protein-coupled receptor (GPCR) structures in complex with Na+ bound to an allosteric modulatory site has renewed interest of the regulation of GPCRs by ions.  Here, we summarise key data on ion modulation of GPCRs, obtained in pharmacol., crystallog., mutagenesis and mol. modeling studies.  We show that ion modulation is a highly complex process, involving not only cations but also, rather neglected until now, anions.  Pharmacotherapeutic and toxicol. aspects are discussed.  We provide a math. framework for the anal. of ion effects.  Finally, we discuss open questions in the field and future research directions.  Most importantly, the in vivo relevance of the modulation of GPCR function by monovalent ions must be clarified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDmk3TO24ZO7Vg90H21EOLACvtfcHk0lgBxenJEkSdrg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXitVSitLzF&amp;md5=74281aba85e6e810d49d08f89980fb01</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs00210-014-1073-2&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00210-014-1073-2%26sid%3Dliteratum%253Aachs%26aulast%3DStrasser%26aufirst%3DA.%26aulast%3DWittmann%26aufirst%3DH.%2BJ.%26aulast%3DSchneider%26aufirst%3DE.%2BH.%26aulast%3DSeifert%26aufirst%3DR.%26atitle%3DModulation%2520of%2520GPCRs%2520by%2520monovalent%2520cations%2520and%2520anions%26jtitle%3DNaunyn-Schmiedeberg%2527s%2520Arch.%2520Pharmacol.%26date%3D2015%26volume%3D388%26issue%3D3%26spage%3D363%26doi%3D10.1007%2Fs00210-014-1073-2" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gulya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovaâ²cs, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaâ²sa, P.</span></span> <span> </span><span class="NLM_article-title">Regulation of endogenous calcium and magnesium levels by Î´ opioid receptors in the rat brain</span>. <i>Brain Res.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>547</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">32</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0006-8993(91)90570-L</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2F0006-8993%2891%2990570-L" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=547&amp;publication_year=1991&amp;pages=32&amp;issue=1&amp;author=K.+Gulyaauthor=G.+L.+Kova%E2%80%B2csauthor=P.+Ka%E2%80%B2sa&amp;title=Regulation+of+endogenous+calcium+and+magnesium+levels+by+%CE%B4+opioid+receptors+in+the+rat+brain&amp;doi=10.1016%2F0006-8993%2891%2990570-L" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2F0006-8993%2891%2990570-L&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-8993%252891%252990570-L%26sid%3Dliteratum%253Aachs%26aulast%3DGulya%26aufirst%3DK.%26aulast%3DKova%25E2%2580%25B2cs%26aufirst%3DG.%2BL.%26aulast%3DKa%25E2%2580%25B2sa%26aufirst%3DP.%26atitle%3DRegulation%2520of%2520endogenous%2520calcium%2520and%2520magnesium%2520levels%2520by%2520%25CE%25B4%2520opioid%2520receptors%2520in%2520the%2520rat%2520brain%26jtitle%3DBrain%2520Res.%26date%3D1991%26volume%3D547%26issue%3D1%26spage%3D32%26doi%3D10.1016%2F0006-8993%2891%2990570-L" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tejwani, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanissian, S. H.</span></span> <span> </span><span class="NLM_article-title">Modulation of mu, delta and kappa opioid receptors in rat brain by metal ions and histidine</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">445</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0028-3908(90)90166-O</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2F0028-3908%2890%2990166-O" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=2162496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADyaK3cXktlCqu70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=1990&amp;pages=445&amp;issue=5&amp;author=G.+A.+Tejwaniauthor=S.+H.+Hanissian&amp;title=Modulation+of+mu%2C+delta+and+kappa+opioid+receptors+in+rat+brain+by+metal+ions+and+histidine&amp;doi=10.1016%2F0028-3908%2890%2990166-O" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of mu, delta and kappa opioid receptors in rat brain by metal ions and histidine</span></div><div class="casAuthors">Tejwani, G. A.; Hanissian, S. H.</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">445-52</span>CODEN:
                <span class="NLM_cas:coden">NEPHBW</span>;
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    </div><div class="casAbstract">The effect of Zn and several other trace elements was studied on the binding of the opioid receptor agonists [3]DAGO, [3H]DSTLE ([Tyr-D-Ser-Gly-Phe-Leu-Thr]-enkephalin and [3H]EKC (ethylketocyclazocine), which are specific for the mu, delta, and kappa opioid receptors, resp., in the cerebral cortex of the rat.  Physiol. concns. of Zn were inhibitory to mu receptor binding, whereas the delta and kappa receptors were relatively insensitive to this inhibition.  Scatchard anal., using these opioid agonists, revealed curvilinear plots; concns. of Zn equal to or less than the IC50 (the concn. of cation which caused 50% inhibition of the binding of opioid ligand to its receptor) increased the KD of all 3 subtypes of receptor, with no effect on the Bmax and abolished the high affinity sites of the delta and kappa receptors.  Cu, Cd, and Hg also inhibited the binding of these ligands to their receptors.  Histidine was most effective in preventing the inhibitory effects of Zn and Cu, whereas it was less effective on Cd and without any effect on the inhibition caused by Hg.  Mg and Mn were stimulatory to opioid receptor binding, whereas Co and Ni had dual (stimulatory and inhibitory) effects.  Non-inhibitory concns. of Zn decreased the stimulatory effects of Mg and Mn on the mu and delta receptors, suggesting that part of the effect of Zn was through prevention of the actions of stimulatory cations.  The reducing reagent dithiothreitol partially protected against inhibition by Zn and the oxidizing reagent dithiobisnitrobenzic acid potentiated the inhibitory effects of Zn on the binding of [3H]DSTLE and [3H]DAGO.  These results suggest that physiol. concns. of free Zn can inhibit mu but not delta and kappa receptors.  The lower susceptibility of delta and kappa receptors to inhibition by Zn may be due to the presence of fewer sulfhydryl groups in these receptors, or due to their inaccessibility to binding by Zn.  Relative physiol. concns. of free Zn and histidine may govern the extent of binding of the mu ligands to their receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-rmCD3WDg4LVg90H21EOLACvtfcHk0lgBxenJEkSdrg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3cXktlCqu70%253D&amp;md5=608286de7e6e2a9f3902e6ade84b78ff</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2F0028-3908%2890%2990166-O&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0028-3908%252890%252990166-O%26sid%3Dliteratum%253Aachs%26aulast%3DTejwani%26aufirst%3DG.%2BA.%26aulast%3DHanissian%26aufirst%3DS.%2BH.%26atitle%3DModulation%2520of%2520mu%252C%2520delta%2520and%2520kappa%2520opioid%2520receptors%2520in%2520rat%2520brain%2520by%2520metal%2520ions%2520and%2520histidine%26jtitle%3DNeuropharmacology%26date%3D1990%26volume%3D29%26issue%3D5%26spage%3D445%26doi%3D10.1016%2F0028-3908%2890%2990166-O" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">Structure of the D2 dopamine receptor bound to the atypical antipsychotic drug risperidone</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>555</i></span> (<span class="NLM_issue">7695</span>),  <span class="NLM_fpage">269</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature25758</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fnature25758" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29466326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktFKqt7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=555&amp;publication_year=2018&amp;pages=269&amp;issue=7695&amp;author=S.+Wangauthor=T.+Cheauthor=A.+Levitauthor=B.+K.+Shoichetauthor=D.+Wackerauthor=B.+L.+Roth&amp;title=Structure+of+the+D2+dopamine+receptor+bound+to+the+atypical+antipsychotic+drug+risperidone&amp;doi=10.1038%2Fnature25758" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the D2 dopamine receptor bound to the atypical antipsychotic drug risperidone</span></div><div class="casAuthors">Wang, Sheng; Che, Tao; Levit, Anat; Shoichet, Brian K.; Wacker, Daniel; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">555</span>
        (<span class="NLM_cas:issue">7695</span>),
    <span class="NLM_cas:pages">269-273</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Dopamine is a neurotransmitter that has been implicated in processes as diverse as reward, addiction, control of coordinated movement, metab. and hormonal secretion.  Correspondingly, dysregulation of the dopaminergic system has been implicated in diseases such as schizophrenia, Parkinson's disease, depression, attention deficit hyperactivity disorder, and nausea and vomiting.  The actions of dopamine are mediated by a family of five G-protein-coupled receptors.  The D2 dopamine receptor (DRD2) is the primary target for both typical and atypical antipsychotic drugs, and for drugs used to treat Parkinson's disease.  Unfortunately, many drugs that target DRD2 cause serious and potentially life-threatening side effects due to promiscuous activities against related receptors.  Accordingly, a mol. understanding of the structure and function of DRD2 could provide a template for the design of safer and more effective medications.  Here we report the crystal structure of DRD2 in complex with the widely prescribed atypical antipsychotic drug risperidone.  The DRD2-risperidone structure reveals an unexpected mode of antipsychotic drug binding to dopamine receptors, and highlights structural determinants that are essential for the actions of risperidone and related drugs at DRD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeo4Ov49kft7Vg90H21EOLACvtfcHk0lhgHGk9iFzscg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXktFKqt7c%253D&amp;md5=2842adc1893293ea8dbf98ae3270a1b2</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnature25758&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature25758%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DChe%26aufirst%3DT.%26aulast%3DLevit%26aufirst%3DA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DStructure%2520of%2520the%2520D2%2520dopamine%2520receptor%2520bound%2520to%2520the%2520atypical%2520antipsychotic%2520drug%2520risperidone%26jtitle%3DNature%26date%3D2018%26volume%3D555%26issue%3D7695%26spage%3D269%26doi%3D10.1038%2Fnature25758" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Free, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moritz, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conroy, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meade, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Discovery and characterization of a G protein-biased agonist that inhibits beta-arrestin recruitment to the D2 dopamine receptor</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">96</span>, <span class="refDoi">&nbsp;DOI: 10.1124/mol.113.090563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1124%2Fmol.113.090563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=24755247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGksrvF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=86&amp;publication_year=2014&amp;pages=96&amp;issue=1&amp;author=R.+B.+Freeauthor=L.+S.+Chunauthor=A.+E.+Moritzauthor=B.+N.+Millerauthor=T.+B.+Doyleauthor=J.+L.+Conroyauthor=A.+Padronauthor=J.+A.+Meadeauthor=J.+Xiaoauthor=X.+Hu&amp;title=Discovery+and+characterization+of+a+G+protein-biased+agonist+that+inhibits+beta-arrestin+recruitment+to+the+D2+dopamine+receptor&amp;doi=10.1124%2Fmol.113.090563" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of a G protein-biased agonist that inhibits Î²-arrestin recruitment to the D2 dopamine receptor</span></div><div class="casAuthors">Free, R. Benjamin; Chun, Lani S.; Moritz, Amy E.; Miller, Brittney N.; Doyle, Trevor B.; Conroy, Jennie L.; Padron, Adrian; Meade, Julie A.; Xiao, Jingbo; Hu, Xin; Dulcey, Andres E.; Han, Yang; Duan, Lihua; Titus, Steve; Bryant-Genevier, Melanie; Barnaeva, Elena; Ferrer, Marc; Javitch, Jonathan A.; Beuming, Thijs; Shi, Lei; Southall, Noel T.; Marugan, Juan J.; Sibley, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">96-105, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A high-throughput screening campaign was conducted to interrogate a 380,000+ small-mol. library for novel D2 dopamine receptor modulators using a calcium mobilization assay.  Active agonist compds. from the primary screen were examd. for orthogonal D2 dopamine receptor signaling activities including cAMP modulation and Î²-arrestin recruitment.  Although the majority of the subsequently confirmed hits activated all signaling pathways tested, several compds. showed a diminished ability to stimulate Î²-arrestin recruitment.  One such compd. (MLS1547; 5-chloro-7-[(4-pyridin-2-ylpiperazin-1-yl)methyl]quinolin-8-ol) is a highly efficacious agonist at D2 receptor-mediated G protein-linked signaling, but does not recruit Î²-arrestin as demonstrated using two different assays.  This compd. does, however, antagonize dopamine-stimulated Î²-arrestin recruitment to the D2 receptor.  In an effort to investigate the chem. scaffold of MLS1547 further, we characterized a set of 24 analogs of MLS1547 with respect to their ability to inhibit cAMP accumulation or stimulate Î²-arrestin recruitment.  A no. of the analogs were similar to MLS1547 in that they displayed agonist activity for inhibiting cAMP accumulation, but did not stimulate Î²-arrestin recruitment (i.e., they were highly biased).  In contrast, other analogs displayed various degrees of G protein signaling bias.  These results provided the basis to use pharmacophore modeling and mol. docking analyses to build a preliminary structure-activity relationship of the functionally selective properties of this series of compds.  In summary, we have identified and characterized a novel G protein-biased agonist of the D2 dopamine receptor and identified structural features that may contribute to its biased signaling properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUnTQyiZvZxrVg90H21EOLACvtfcHk0lhgHGk9iFzscg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGksrvF&amp;md5=7a3412858a3d780d8f90ca4535d485fc</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1124%2Fmol.113.090563&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.113.090563%26sid%3Dliteratum%253Aachs%26aulast%3DFree%26aufirst%3DR.%2BB.%26aulast%3DChun%26aufirst%3DL.%2BS.%26aulast%3DMoritz%26aufirst%3DA.%2BE.%26aulast%3DMiller%26aufirst%3DB.%2BN.%26aulast%3DDoyle%26aufirst%3DT.%2BB.%26aulast%3DConroy%26aufirst%3DJ.%2BL.%26aulast%3DPadron%26aufirst%3DA.%26aulast%3DMeade%26aufirst%3DJ.%2BA.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DX.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520a%2520G%2520protein-biased%2520agonist%2520that%2520inhibits%2520beta-arrestin%2520recruitment%2520to%2520the%2520D2%2520dopamine%2520receptor%26jtitle%3DMol.%2520Pharmacol.%26date%3D2014%26volume%3D86%26issue%3D1%26spage%3D96%26doi%3D10.1124%2Fmol.113.090563" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonsson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beuming, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voth, G. A.</span></span> <span> </span><span class="NLM_article-title">Ligand-dependent activation and deactivation of the human adenosine A(2A) receptor</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">8749</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja404391q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja404391q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVCrt7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=135&amp;publication_year=2013&amp;pages=8749&amp;issue=23&amp;author=J.+Liauthor=A.+L.+Jonssonauthor=T.+Beumingauthor=J.+C.+Shelleyauthor=G.+A.+Voth&amp;title=Ligand-dependent+activation+and+deactivation+of+the+human+adenosine+A%282A%29+receptor&amp;doi=10.1021%2Fja404391q" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-Dependent Activation and Deactivation of the Human Adenosine A2A Receptor</span></div><div class="casAuthors">Li, Jianing; Jonsson, Amanda L.; Beuming, Thijs; Shelley, John C.; Voth, Gregory A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8749-8759</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are membrane proteins with crit. functions in cellular signal transduction, representing a primary class of drug targets.  Acting by direct binding, many drugs modulate GPCR activity and influence the signaling pathways assocd. with numerous diseases.  However, complete details of ligand-dependent GPCR activation/deactivation are difficult to obtain from expts.  Therefore, it remains unclear how ligands modulate a GPCR's activity.  To elucidate the ligand-dependent activation/deactivation mechanism of the human adenosine A2A receptor (AA2AR), a member of the class A GPCRs, we performed large-scale unbiased mol. dynamics and metadynamics simulations of the receptor embedded in a membrane.  At the at. level, we have obsd. distinct structural states that resemble the active and inactive states.  In particular, we noted key structural elements changing in a highly concerted fashion during the conformational transitions, including six conformational states of a tryptophan (Trp2466.48).  Our findings agree with a previously proposed view that, during activation, this tryptophan residue undergoes a rotameric transition that may be coupled to a series of coherent conformational changes, resulting in the opening of the G-protein binding site.  Further, metadynamics simulations provide quant. evidence for this mechanism, suggesting how ligand binding shifts the equil. between the active and inactive states.  Our anal. also proposes that a few specific residues are assocd. with agonism/antagonism, affinity, and selectivity, and suggests that the ligand-binding pocket can be thought of as having three distinct regions, providing dynamic features for structure-based design.  Addnl. simulations with AA2AR bound to a novel ligand are consistent with our proposed mechanism.  Generally, our study provides insights into the ligand-dependent AA2AR activation/deactivation in addn. to what has been found in crystal structures.  These results should aid in the discovery of more effective and selective GPCR ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_X5Tj8UviMLVg90H21EOLACvtfcHk0lhgHGk9iFzscg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXnvVCrt7g%253D&amp;md5=e20018f868df9a6031a00465d97ab0ca</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja404391q&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja404391q%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJonsson%26aufirst%3DA.%2BL.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DVoth%26aufirst%3DG.%2BA.%26atitle%3DLigand-dependent%2520activation%2520and%2520deactivation%2520of%2520the%2520human%2520adenosine%2520A%25282A%2529%2520receptor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26issue%3D23%26spage%3D8749%26doi%3D10.1021%2Fja404391q" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddy, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deary, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Structural Connection between Activation Microswitch and Allosteric Sodium Site in GPCR Signaling</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">259</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.str.2017.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.str.2017.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29395784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVSjs70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2018&amp;pages=259&amp;issue=2&amp;author=K.+L.+Whiteauthor=M.+T.+Eddyauthor=Z.-G.+Gaoauthor=G.+W.+Hanauthor=T.+Lianauthor=A.+Dearyauthor=N.+Patelauthor=K.+A.+Jacobsonauthor=V.+Katritchauthor=R.+C.+Stevens&amp;title=Structural+Connection+between+Activation+Microswitch+and+Allosteric+Sodium+Site+in+GPCR+Signaling&amp;doi=10.1016%2Fj.str.2017.12.013" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Connection between Activation Microswitch and Allosteric Sodium Site in GPCR Signaling</span></div><div class="casAuthors">White, Kate L.; Eddy, Matthew T.; Gao, Zhan-Guo; Han, Gye Won; Lian, Tiffany; Deary, Alexander; Patel, Nilkanth; Jacobson, Kenneth A.; Katritch, Vsevolod; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">259-269.e5</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Sodium ions are endogenous allosteric modulators of many G-protein-coupled receptors (GPCRs).  Mutation of key residues in the sodium binding motif causes a striking effect on G-protein signaling.  We report the crystal structures of agonist complexes for two variants in the first sodium coordination shell of the human A2A adenosine receptor, D522.50N and S913.39A.  Both structures present an overall active-like conformation; however, the variants show key changes in the activation motif NPxxY.  Changes in the hydrogen bonding network in this microswitch suggest a possible mechanism for modified G-protein signaling and enhanced thermal stability.  These structures, signaling data, and thermal stability anal. with a panel of pharmacol. ligands provide a basis for understanding the role of the sodium-coordinating residues on stability and G-protein signaling.  Utilizing the D2.50N variant is a promising method for stabilizing class A GPCRs to accelerate structural efforts and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo18IDIJKugCrVg90H21EOLACvtfcHk0liLolJUiDAKZQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXisVSjs70%253D&amp;md5=916e644534536f2a995288d207328c4d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2017.12.013&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2017.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DEddy%26aufirst%3DM.%2BT.%26aulast%3DGao%26aufirst%3DZ.-G.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DLian%26aufirst%3DT.%26aulast%3DDeary%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520Connection%2520between%2520Activation%2520Microswitch%2520and%2520Allosteric%2520Sodium%2520Site%2520in%2520GPCR%2520Signaling%26jtitle%3DStructure%26date%3D2018%26volume%3D26%26issue%3D2%26spage%3D259%26doi%3D10.1016%2Fj.str.2017.12.013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newton, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strange, P. G.</span></span> <span> </span><span class="NLM_article-title">Examining the Effects of Sodium Ions on the Binding of Antagonists to Dopamine D2 and D3 Receptors</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">e0158808</span>, <span class="refDoi">&nbsp;DOI: 10.1371/journal.pone.0158808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1371%2Fjournal.pone.0158808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=27379794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVaqtb3N" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2016&amp;pages=e0158808&amp;issue=7&amp;author=C.+L.+Newtonauthor=M.+D.+Woodauthor=P.+G.+Strange&amp;title=Examining+the+Effects+of+Sodium+Ions+on+the+Binding+of+Antagonists+to+Dopamine+D2+and+D3+Receptors&amp;doi=10.1371%2Fjournal.pone.0158808" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Examining the effects of sodium ions on the binding of antagonists to dopamine D2 and D3 receptors</span></div><div class="casAuthors">Newton, Claire L.; Wood, Martyn D.; Strange, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e0158808/1-e0158808/11</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Many G protein-coupled receptors have been shown to be sensitive to the presence of sodium ions (Na+).  Using radioligand competition binding assays, we have examd. and compared the effects of sodium ions on the binding affinities of a no. of structurally diverse ligands at human dopamine D2 and dopamine D3 receptor subtypes, which are important therapeutic targets for the treatment of psychotic disorders.  At both receptors, the binding affinities of the antagonists/inverse agonists SB-277011-A, L,741,626, GR 103691 and U 99194 were higher in the presence of sodium ions compared to those measured in the presence of the org. cation, N-methyl-d-glucamine, used to control for ionic strength.  Conversely, the affinities of spiperone and (+)-butaclamol were unaffected by the presence of sodium ions.  Interestingly, the binding of the antagonist/inverse agonist clozapine was affected by changes in ionic strength of the buffer used rather than the presence of specific cations.  Similar sensitivities to sodium ions were seen at both receptors, suggesting parallel effects of sodium ion interactions on receptor conformation.  However, no clear correlation between ligand characteristics, such as subtype selectivity, and sodium ion sensitivity were obsd.  Therefore, the properties which det. this sensitivity remain unclear.  However these findings do highlight the importance of careful consideration of assay buffer compn. for in vitro assays and when comparing data from different studies, and may indicate a further level of control for ligand binding in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv1PwNFNVf8bVg90H21EOLACvtfcHk0liLolJUiDAKZQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XhvVaqtb3N&amp;md5=39327177fdd523fe702a94a7e9427bb1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0158808&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0158808%26sid%3Dliteratum%253Aachs%26aulast%3DNewton%26aufirst%3DC.%2BL.%26aulast%3DWood%26aufirst%3DM.%2BD.%26aulast%3DStrange%26aufirst%3DP.%2BG.%26atitle%3DExamining%2520the%2520Effects%2520of%2520Sodium%2520Ions%2520on%2520the%2520Binding%2520of%2520Antagonists%2520to%2520Dopamine%2520D2%2520and%2520D3%2520Receptors%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26issue%3D7%26spage%3De0158808%26doi%3D10.1371%2Fjournal.pone.0158808" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Free, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sibley, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span> <span> </span><span class="NLM_article-title">Structural basis for Na(+)-sensitivity in dopamine D2 and D3 receptors</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">41</span>),  <span class="NLM_fpage">8618</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C5CC02204E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1039%2FC5CC02204E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=25896577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Wnuro%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=51&amp;publication_year=2015&amp;pages=8618&amp;issue=41&amp;author=M.+Michinoauthor=R.+B.+Freeauthor=T.+B.+Doyleauthor=D.+R.+Sibleyauthor=L.+Shi&amp;title=Structural+basis+for+Na%28%2B%29-sensitivity+in+dopamine+D2+and+D3+receptors&amp;doi=10.1039%2FC5CC02204E" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for Na+-sensitivity in dopamine D2 and D3 receptors</span></div><div class="casAuthors">Michino, Mayako; Free, R. Benjamin; Doyle, Trevor B.; Sibley, David R.; Shi, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">8618-8621</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">To understand the structural basis for the Na+-sensitivity of ligand binding to dopamine D2-like receptors, using computational anal. in combination with binding assays, the authors identified interactions crit. in propagating the impact of Na+ on receptor conformations and on the ligand-binding site.  The authors' findings expand the pharmacol.-relevant conformational spectrum of these receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonu0n6gq5E2bVg90H21EOLACvtfcHk0lgO0AXE39-TRA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Wnuro%253D&amp;md5=750d293936c4215a96a5deb95f6271d6</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1039%2FC5CC02204E&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC02204E%26sid%3Dliteratum%253Aachs%26aulast%3DMichino%26aufirst%3DM.%26aulast%3DFree%26aufirst%3DR.%2BB.%26aulast%3DDoyle%26aufirst%3DT.%2BB.%26aulast%3DSibley%26aufirst%3DD.%2BR.%26aulast%3DShi%26aufirst%3DL.%26atitle%3DStructural%2520basis%2520for%2520Na%2528%252B%2529-sensitivity%2520in%2520dopamine%2520D2%2520and%2520D3%2520receptors%26jtitle%3DChem.%2520Commun.%26date%3D2015%26volume%3D51%26issue%3D41%26spage%3D8618%26doi%3D10.1039%2FC5CC02204E" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vickery, O. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carvalheda, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaidi, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisliakov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zachariae, U.</span></span> <span> </span><span class="NLM_article-title">Intracellular Transfer of Na(+) in an Active-State G-Protein-Coupled Receptor</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">171</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.str.2017.11.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.str.2017.11.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29249607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvF2js7vF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2018&amp;pages=171&amp;issue=1&amp;author=O.+N.+Vickeryauthor=C.+A.+Carvalhedaauthor=S.+A.+Zaidiauthor=A.+V.+Pisliakovauthor=V.+Katritchauthor=U.+Zachariae&amp;title=Intracellular+Transfer+of+Na%28%2B%29+in+an+Active-State+G-Protein-Coupled+Receptor&amp;doi=10.1016%2Fj.str.2017.11.013" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular Transfer of Na+ in an Active-State G-Protein-Coupled Receptor</span></div><div class="casAuthors">Vickery, Owen N.; Carvalheda, Catarina A.; Zaidi, Saheem A.; Pisliakov, Andrei V.; Katritch, Vsevolod; Zachariae, Ulrich</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">171-180.e2</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Playing a central role in cell signaling, G-protein-coupled receptors (GPCRs) are the largest superfamily of membrane proteins and form the majority of drug targets in humans.  How extracellular agonist binding triggers the activation of GPCRs and assocd. intracellular effector proteins remains, however, poorly understood.  Structural studies have revealed that inactive class A GPCRs harbor a conserved binding site for Na+ ions in the center of their transmembrane domain, accessible from the extracellular space.  Here, we show that the opening of a conserved hydrated channel in the activated state receptors allows the Na+ ion to egress from its binding site into the cytosol.  Coupled with protonation changes, this ion movement occurs without significant energy barriers, and can be driven by physiol. transmembrane ion and voltage gradients.  We propose that Na+ ion exchange with the cytosol is a key step in GPCR activation.  Further, we hypothesize that this transition locks receptors in long-lived active-state conformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzBCbcEx6TibVg90H21EOLACvtfcHk0lgO0AXE39-TRA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhvF2js7vF&amp;md5=da6940bf420ad51c5b275e2fec7231ae</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2017.11.013&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2017.11.013%26sid%3Dliteratum%253Aachs%26aulast%3DVickery%26aufirst%3DO.%2BN.%26aulast%3DCarvalheda%26aufirst%3DC.%2BA.%26aulast%3DZaidi%26aufirst%3DS.%2BA.%26aulast%3DPisliakov%26aufirst%3DA.%2BV.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DZachariae%26aufirst%3DU.%26atitle%3DIntracellular%2520Transfer%2520of%2520Na%2528%252B%2529%2520in%2520an%2520Active-State%2520G-Protein-Coupled%2520Receptor%26jtitle%3DStructure%26date%3D2018%26volume%3D26%26issue%3D1%26spage%3D171%26doi%3D10.1016%2Fj.str.2017.11.013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vass, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kooistra, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritschel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leurs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span> <span> </span><span class="NLM_article-title">Molecular interaction fingerprint approaches for GPCR drug discovery</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">59</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.coph.2016.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.coph.2016.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=27479316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1eju7zJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=30&amp;publication_year=2016&amp;pages=59&amp;author=M.+Vassauthor=A.+J.+Kooistraauthor=T.+Ritschelauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&amp;title=Molecular+interaction+fingerprint+approaches+for+GPCR+drug+discovery&amp;doi=10.1016%2Fj.coph.2016.07.007" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular interaction fingerprint approaches for GPCR drug discovery</span></div><div class="casAuthors">Vass, Marton; Kooistra, Albert J.; Ritschel, Tina; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-68</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints (IFPs) are binary 1D representations of the 3D structure of protein-ligand complexes encoding the presence or absence of specific interactions between the binding pocket amino acids and the ligand.  Various implementations of IFPs have been developed and successfully applied for post-processing mol. docking results for G Protein-Coupled Receptor (GPCR) ligand binding mode prediction and virtual ligand screening.  Novel interaction fingerprint methods enable structural chemogenomics and polypharmacol. predictions by complementing the increasing amt. of GPCR structural data.  Machine learning methods are increasingly used to derive relationships between bioactivity data and fingerprint descriptors of chem. and structural information of binding sites, ligands, and protein-ligand interactions.  Factors that influence the application of IFPs include structure prepn., binding site definition, fingerprint similarity assessment, and data processing and these factors pose challenges as well possibilities to optimize interaction fingerprint methods for GPCR drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfcDOwflIZPrVg90H21EOLACvtfcHk0ljGiL-fieHdZA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28Xht1eju7zJ&amp;md5=567b591ab504d3d9ec6bec9556588118</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2016.07.007&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2016.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DVass%26aufirst%3DM.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DRitschel%26aufirst%3DT.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DMolecular%2520interaction%2520fingerprint%2520approaches%2520for%2520GPCR%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2016%26volume%3D30%26spage%3D59%26doi%3D10.1016%2Fj.coph.2016.07.007" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span> <span> </span><span class="NLM_article-title">Structural interaction fingerprints: a new approach to organizing, mining, analyzing, and designing protein-small molecule complexes</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">5</span>, <span class="refDoi">&nbsp;DOI: 10.1111/j.1747-0285.2005.00323.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1111%2Fj.1747-0285.2005.00323.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=16492144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVSjt7o%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=67&amp;publication_year=2006&amp;pages=5&amp;issue=1&amp;author=J.+Singhauthor=Z.+Dengauthor=G.+Naraleauthor=C.+Chuaqui&amp;title=Structural+interaction+fingerprints%3A+a+new+approach+to+organizing%2C+mining%2C+analyzing%2C+and+designing+protein-small+molecule+complexes&amp;doi=10.1111%2Fj.1747-0285.2005.00323.x" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structural interaction fingerprints: a new approach to organizing, mining, analyzing, and designing protein-small molecule complexes</span></div><div class="casAuthors">Singh, Juswinder; Deng, Zhan; Narale, Gaurav; Chuaqui, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology &amp; Drug Design</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-12</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  The combination of advances in structure-based drug design efforts in the pharmaceutical industry in parallel with structural genomics initiatives in the public domain has led to an explosion in the no. of structures of protein-small mol. complexes.  This information has crit. importance to both the understanding of the structural basis for mol. recognition in biol. systems and the design of better drugs.  A significant challenge exists in managing this vast amt. of data and fully leveraging it.  Here, we review our work to develop a simple, fast way to store, organize, mine, and analyze large nos. of protein-small mol. complexes.  We illustrate the utility of the approach to the management of inhibitor complexes from the protein kinase family.  Finally, we describe our recent efforts in applying this method to the design of target-focused chem. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyYr5KQpOFMLVg90H21EOLACvtfcHk0ljGiL-fieHdZA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28XhsVSjt7o%253D&amp;md5=2e291b90d4fafa433d544669d7678d57</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2005.00323.x&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2005.00323.x%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DNarale%26aufirst%3DG.%26aulast%3DChuaqui%26aufirst%3DC.%26atitle%3DStructural%2520interaction%2520fingerprints%253A%2520a%2520new%2520approach%2520to%2520organizing%252C%2520mining%252C%2520analyzing%252C%2520and%2520designing%2520protein-small%2520molecule%2520complexes%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2006%26volume%3D67%26issue%3D1%26spage%3D5%26doi%3D10.1111%2Fj.1747-0285.2005.00323.x" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span> <span> </span><span class="NLM_article-title">Structural interaction fingerprint (SIFt): a novel method for analyzing three-dimensional protein-ligand binding interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">337</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jm030331x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030331x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD3sXps1yrsrs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=2004&amp;pages=337&amp;issue=2&amp;author=Z.+Dengauthor=C.+Chuaquiauthor=J.+Singh&amp;title=Structural+interaction+fingerprint+%28SIFt%29%3A+a+novel+method+for+analyzing+three-dimensional+protein-ligand+binding+interactions&amp;doi=10.1021%2Fjm030331x" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Interaction Fingerprint (SIFt): A Novel Method for Analyzing Three-Dimensional Protein-Ligand Binding Interactions</span></div><div class="casAuthors">Deng, Zhan; Chuaqui, Claudio; Singh, Juswinder</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-344</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Representing and understanding the three-dimensional (3D) structural information of protein-ligand complexes is a crit. step in the rational drug discovery process.  Traditional anal. methods are proving inadequate and inefficient in dealing with the massive amt. of structural information being generated from x-ray crystallog., NMR, and in silico approaches such as structure-based docking expts.  Here, we present SIFt (structural interaction fingerprint), a novel method for representing and analyzing 3D protein-ligand binding interactions.  Key to this approach is the generation of an interaction fingerprint that translates 3D structural binding information from a protein-ligand complex into a one-dimensional binary string.  Each fingerprint represents the "structural interaction profile" of the complex that can be used to organize, analyze, and visualize the rich amt. of information encoded in ligand-receptor complexes and also to assist database mining.  We have applied SIFt to tackle three common tasks in structure-based drug design.  The first involved the anal. and organization of a typical set of results generated from a docking study.  Using SIFt, docking poses with similar binding modes were identified, clustered, and subsequently compared with conventional scoring function information.  A second application of SIFt was to analyze â¼90 known x-ray crystal structures of protein kinase-inhibitor complexes obtained from the Protein Databank.  Using SIFt, we were able to organize the structures and reveal striking similarities and diversity between their small mol. binding interactions.  Finally, we have shown how SIFt can be used as an effective mol. filter during the virtual chem. library screening process to select mols. with desirable binding mode(s) and/or desirable interaction patterns with the protein target.  In summary, SIFt shows promise to fully leverage the wealth of information being generated in rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkjji1ni0HHbVg90H21EOLACvtfcHk0liJbhbkDyi-UA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3sXps1yrsrs%253D&amp;md5=a652b70184155c107202cbe3bc74596a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm030331x&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030331x%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DStructural%2520interaction%2520fingerprint%2520%2528SIFt%2529%253A%2520a%2520novel%2520method%2520for%2520analyzing%2520three-dimensional%2520protein-ligand%2520binding%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D2%26spage%3D337%26doi%3D10.1021%2Fjm030331x" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palczewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S.</span></span> <span> </span><span class="NLM_article-title">Exploring a new ligand binding site of G protein-coupled receptors</span>. <i>Chemical Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">6480</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C8SC01680A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1039%2FC8SC01680A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=30310578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWhtL7I" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2018&amp;pages=6480&amp;author=H.+C.+S.+Chanauthor=J.+Wangauthor=K.+Palczewskiauthor=S.+Filipekauthor=H.+Vogelauthor=Z.-J.+Liuauthor=S.+Yuan&amp;title=Exploring+a+new+ligand+binding+site+of+G+protein-coupled+receptors&amp;doi=10.1039%2FC8SC01680A" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring a new ligand binding site of G protein-coupled receptors</span></div><div class="casAuthors">Chan, H. C. Stephen; Wang, Jingjing; Palczewski, Krzysztof; Filipek, Slawomir; Vogel, Horst; Liu, Zhi-Jie; Yuan, Shuguang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6480-6489</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Identifying a target ligand binding site is an important step for structure-based rational drug design as shown here for G protein-coupled receptors (GPCRs), which are among the most popular drug targets.  We applied long-time scale mol. dynamics simulations, coupled with mutagenesis studies, to two prototypical GPCRs, the M3 and M4 muscarinic acetylcholine receptors.  Our results indicate that unlike synthetic antagonists, which bind to the classic orthosteric site, the endogenous agonist acetylcholine is able to diffuse into a much deeper binding pocket.  We also discovered that the most recently resolved crystal structure of the LTB4 receptor comprised a bound inverse agonist, which extended its benzamidine moiety to the same binding pocket discovered in this work.  Anal. on all resolved GPCR crystal structures indicated that this new pocket could exist in most receptors.  Our findings provide new opportunities for GPCR drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3DxdJBSHfq7Vg90H21EOLACvtfcHk0liJbhbkDyi-UA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWhtL7I&amp;md5=8c52e5787c09118d3c62dd047b025cf8</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2FC8SC01680A&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8SC01680A%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DH.%2BC.%2BS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DPalczewski%26aufirst%3DK.%26aulast%3DFilipek%26aufirst%3DS.%26aulast%3DVogel%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DZ.-J.%26aulast%3DYuan%26aufirst%3DS.%26atitle%3DExploring%2520a%2520new%2520ligand%2520binding%2520site%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DChemical%2520Science%26date%3D2018%26volume%3D9%26spage%3D6480%26doi%3D10.1039%2FC8SC01680A" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kling, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span> <span> </span><span class="NLM_article-title">Comparative MD Simulations Indicate a Dual Role for Arg1323.50 in Dopamine-Dependent D2R Activation</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">e0146612</span>, <span class="refDoi">&nbsp;DOI: 10.1371/journal.pone.0146612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1371%2Fjournal.pone.0146612" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=26741139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC28XlslShtrc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2016&amp;pages=e0146612&amp;issue=1&amp;author=R.+C.+Klingauthor=T.+Clarkauthor=P.+Gmeiner&amp;title=Comparative+MD+Simulations+Indicate+a+Dual+Role+for+Arg1323.50+in+Dopamine-Dependent+D2R+Activation&amp;doi=10.1371%2Fjournal.pone.0146612" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative MD simulations indicate a dual role for arg1323.50 in dopamine-dependent D2R activation</span></div><div class="casAuthors">Kling, Ralf C.; Clark, Timothy; Gmeiner, Peter</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e0146612/1-e0146612/13</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Residue Arg3.50 belongs to the highly conserved DRY-motif of class A GPCRs, which is located at the bottom of TM3.  On the one hand, Arg3.50 has been reported to help stabilize the inactive state of GPCRs, but on the other hand has also been shown to be crucial for stabilizing active receptor conformations and mediating receptor-G protein coupling.  The combined results of these studies suggest that the exact function of Arg3.50 is likely to be receptor-dependent and must be characterized independently for every GPCR.  Consequently, we now present comparative mol.-dynamics simulations that use our recently described inactive-state and GÎ±-bound active-state homol. models of the dopamine D2 receptor (D2R), which are either bound to dopamine or ligand-free, performed to identify the function of Arg1323.50 in D2R.  Our results are consistent with a dynamic model of D2R activation in which Arg1323.50 adopts a dual role, both by stabilizing the inactive-state receptor conformation and enhancing dopamine-dependent D2R-G protein coupling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnbNGpP05zObVg90H21EOLACvtfcHk0lgjlrqXzJcN_Q"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XlslShtrc%253D&amp;md5=dddb24a2f1a80087c1019d9debaf9624</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0146612&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0146612%26sid%3Dliteratum%253Aachs%26aulast%3DKling%26aufirst%3DR.%2BC.%26aulast%3DClark%26aufirst%3DT.%26aulast%3DGmeiner%26aufirst%3DP.%26atitle%3DComparative%2520MD%2520Simulations%2520Indicate%2520a%2520Dual%2520Role%2520for%2520Arg1323.50%2520in%2520Dopamine-Dependent%2520D2R%2520Activation%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26issue%3D1%26spage%3De0146612%26doi%3D10.1371%2Fjournal.pone.0146612" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kling, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschammer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span> <span> </span><span class="NLM_article-title">Active-state model of a dopamine D2 receptor-Galphai complex stabilized by aripiprazole-type partial agonists</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">e100069</span>, <span class="refDoi">&nbsp;DOI: 10.1371/journal.pone.0100069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1371%2Fjournal.pone.0100069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=24932547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1egt7zM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2014&amp;pages=e100069&amp;issue=6&amp;author=R.+C.+Klingauthor=N.+Tschammerauthor=H.+Lanigauthor=T.+Clarkauthor=P.+Gmeiner&amp;title=Active-state+model+of+a+dopamine+D2+receptor-Galphai+complex+stabilized+by+aripiprazole-type+partial+agonists&amp;doi=10.1371%2Fjournal.pone.0100069" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Active-state model of a dopamine D2 receptor - GÎ±i complex stabilized by aripiprazole-type partial agonists</span></div><div class="casAuthors">Kling, Ralf C.; Tschammer, Nuska; Lanig, Harald; Clark, Timothy; Gmeiner, Peter</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e100069/1-e100069/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Partial agonists exhibit a submaximal capacity to enhance the coupling of one receptor to an intracellular binding partner.  Although a multitude of studies have reported different ligand-specific conformations for a given receptor, little is known about the mechanism by which different receptor conformations are connected to the capacity to activate the coupling to G-proteins.  We have now performed mol.-dynamics simulations employing our recently described active-state homol. model of the dopamine D2 receptor-GÎ±i protein-complex coupled to the partial agonists aripiprazole and FAUC350, in order to understand the structural determinants of partial agonism better.  We have compared our findings with our model of the D2R-GÎ±i-complex in the presence of the full agonist dopamine.  The two partial agonists are capable of inducing different conformations of important structural motifs, including the extracellular loop regions, the binding pocket and, in particular, intracellular G-protein-binding domains.  As G-protein-coupling to certain intracellular epitopes of the receptor is considered the key step of allosterically triggered nucleotide-exchange, it is tempting to assume that impaired coupling between the receptor and the G-protein caused by distinct ligand-specific conformations is a major determinant of partial agonist efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru44KV_-m_drVg90H21EOLACvtfcHk0lgjlrqXzJcN_Q"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXhs1egt7zM&amp;md5=d02e0973e621a4de6d43093aaacb931c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0100069&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0100069%26sid%3Dliteratum%253Aachs%26aulast%3DKling%26aufirst%3DR.%2BC.%26aulast%3DTschammer%26aufirst%3DN.%26aulast%3DLanig%26aufirst%3DH.%26aulast%3DClark%26aufirst%3DT.%26aulast%3DGmeiner%26aufirst%3DP.%26atitle%3DActive-state%2520model%2520of%2520a%2520dopamine%2520D2%2520receptor-Galphai%2520complex%2520stabilized%2520by%2520aripiprazole-type%2520partial%2520agonists%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26issue%3D6%26spage%3De100069%26doi%3D10.1371%2Fjournal.pone.0100069" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatakrishnan, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laeremans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feinberg, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanborn, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livingston, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorsen, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kling, R. C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Structural insights into micro-opioid receptor activation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>524</i></span> (<span class="NLM_issue">7565</span>),  <span class="NLM_fpage">315</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature14886</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fnature14886" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=26245379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht12jtbzM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=524&amp;publication_year=2015&amp;pages=315&amp;issue=7565&amp;author=W.+Huangauthor=A.+Manglikauthor=A.+J.+Venkatakrishnanauthor=T.+Laeremansauthor=E.+N.+Feinbergauthor=A.+L.+Sanbornauthor=H.+E.+Katoauthor=K.+E.+Livingstonauthor=T.+S.+Thorsenauthor=R.+C.+Kling&amp;title=Structural+insights+into+micro-opioid+receptor+activation&amp;doi=10.1038%2Fnature14886" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into Î¼-opioid receptor activation</span></div><div class="casAuthors">Huang, Weijiao; Manglik, Aashish; Venkatakrishnan, A. J.; Laeremans, Toon; Feinberg, Evan N.; Sanborn, Adrian L.; Kato, Hideaki E.; Livingston, Kathryn E.; Thorsen, Thor S.; Kling, Ralf C.; Granier, Sebastien; Gmeiner, Peter; Husbands, Stephen M.; Traynor, John R.; Weis, William I.; Steyaert, Jan; Dror, Ron O.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">524</span>
        (<span class="NLM_cas:issue">7565</span>),
    <span class="NLM_cas:pages">315-321</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activation of the Î¼-opioid receptor (Î¼OR) is responsible for the efficacy of the most effective analgesics.  To shed light on the structural basis for Î¼OR activation, here we report a 2.1 Ã X-ray crystal structure of the murine Î¼OR bound to the morphinan agonist BU72 and a G protein mimetic camelid antibody fragment Nb39 (nanobody 39).  The BU72-stabilized changes in the Î¼OR binding pocket are subtle and differ from those obsd. for agonist-bound structures of the Î²2-adrenergic receptor (Î²2AR) and the M2 muscarinic receptor (M2R).  Comparison with active Î²2AR reveals a common rearrangement in the packing of three conserved amino acids in the core of the Î¼OR, and mol. dynamics simulations illustrate how the ligand-binding pocket is conformationally linked to this conserved triad.  Addnl., an extensive polar network between the ligand-binding pocket and the cytoplasmic domains appears to play a similar role in signal propagation for all three G-protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkFUN1wheV07Vg90H21EOLACvtfcHk0ljuIVGgThvI8w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXht12jtbzM&amp;md5=b9af3e1351b04539379532c127c02651</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnature14886&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14886%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DVenkatakrishnan%26aufirst%3DA.%2BJ.%26aulast%3DLaeremans%26aufirst%3DT.%26aulast%3DFeinberg%26aufirst%3DE.%2BN.%26aulast%3DSanborn%26aufirst%3DA.%2BL.%26aulast%3DKato%26aufirst%3DH.%2BE.%26aulast%3DLivingston%26aufirst%3DK.%2BE.%26aulast%3DThorsen%26aufirst%3DT.%2BS.%26aulast%3DKling%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520insights%2520into%2520micro-opioid%2520receptor%2520activation%26jtitle%3DNature%26date%3D2015%26volume%3D524%26issue%3D7565%26spage%3D315%26doi%3D10.1038%2Fnature14886" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kroeze, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giguere, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciaky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">PRESTO-Tango as an open-source resource for interrogation of the druggable human GPCRome</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">362</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nsmb.3014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fnsmb.3014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=25895059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnslOgtLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=2015&amp;pages=362&amp;issue=5&amp;author=W.+K.+Kroezeauthor=M.+F.+Sassanoauthor=X.-P.+Huangauthor=K.+Lansuauthor=J.+D.+McCorvyauthor=P.+M.+Giguereauthor=N.+Sciakyauthor=B.+L.+Roth&amp;title=PRESTO-Tango+as+an+open-source+resource+for+interrogation+of+the+druggable+human+GPCRome&amp;doi=10.1038%2Fnsmb.3014" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">PRESTO-Tango as an open-source resource for interrogation of the druggable human GPCRome</span></div><div class="casAuthors">Kroeze, Wesley K.; Sassano, Maria F.; Huang, Xi-Ping; Lansu, Katherine; McCorvy, John D.; Giguere, Patrick M.; Sciaky, Noah; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural &amp; Molecular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">362-369</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are essential mediators of cellular signaling and are important targets of drug action.  Of the approx. 350 nonolfactory human GPCRs, more than 100 are still considered to be 'orphans' because their endogenous ligands remain unknown.  Here, we describe a unique open-source resource that allows interrogation of the druggable human GPCRome via a G protein-independent Î²-arrestin-recruitment assay.  We validate this unique platform at more than 120 nonorphan human GPCR targets, demonstrate its utility for discovering new ligands for orphan human GPCRs and describe a method (parallel receptorome expression and screening via transcriptional output, with transcriptional activation following arrestin translocation (PRESTO-Tango)) for the simultaneous and parallel interrogation of the entire human nonolfactory GPCRome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVbl7caV9gUrVg90H21EOLACvtfcHk0ljuIVGgThvI8w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXnslOgtLg%253D&amp;md5=2a78b74a5f1425fa9e8b79b730c16be6</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3014%26sid%3Dliteratum%253Aachs%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DLansu%26aufirst%3DK.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DGiguere%26aufirst%3DP.%2BM.%26aulast%3DSciaky%26aufirst%3DN.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DPRESTO-Tango%2520as%2520an%2520open-source%2520resource%2520for%2520interrogation%2520of%2520the%2520druggable%2520human%2520GPCRome%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2015%26volume%3D22%26issue%3D5%26spage%3D362%26doi%3D10.1038%2Fnsmb.3014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eswar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marti-Renom, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhusudhan, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eramian, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pieper, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Comparative protein structure modeling using MODELLER</span>. <i>Current protocols in protein science</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">2.9.1</span>, <span class="refDoi">&nbsp;DOI: 10.1002/0471140864.ps0209s50</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1002%2F0471140864.ps0209s50" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=50&amp;publication_year=2007&amp;pages=2.9.1&amp;author=N.+Eswarauthor=B.+Webbauthor=M.+A.+Marti-Renomauthor=M.+S.+Madhusudhanauthor=D.+Eramianauthor=M.+Y.+Shenauthor=U.+Pieperauthor=A.+Sali&amp;title=Comparative+protein+structure+modeling+using+MODELLER&amp;doi=10.1002%2F0471140864.ps0209s50" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2F0471140864.ps0209s50&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F0471140864.ps0209s50%26sid%3Dliteratum%253Aachs%26aulast%3DEswar%26aufirst%3DN.%26aulast%3DWebb%26aufirst%3DB.%26aulast%3DMarti-Renom%26aufirst%3DM.%2BA.%26aulast%3DMadhusudhan%26aufirst%3DM.%2BS.%26aulast%3DEramian%26aufirst%3DD.%26aulast%3DShen%26aufirst%3DM.%2BY.%26aulast%3DPieper%26aufirst%3DU.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DComparative%2520protein%2520structure%2520modeling%2520using%2520MODELLER%26jtitle%3DCurrent%2520protocols%2520in%2520protein%2520science%26date%3D2007%26volume%3D50%26spage%3D2.9.1%26doi%3D10.1002%2F0471140864.ps0209s50" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutsias, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kortemme, T.</span></span> <span> </span><span class="NLM_article-title">Sub-angstrom accuracy in protein loop reconstruction by robotics-inspired conformational sampling</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">551</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nmeth0809-551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fnmeth0809-551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=19644455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Whtbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2009&amp;pages=551&amp;issue=8&amp;author=D.+J.+Mandellauthor=E.+A.+Coutsiasauthor=T.+Kortemme&amp;title=Sub-angstrom+accuracy+in+protein+loop+reconstruction+by+robotics-inspired+conformational+sampling&amp;doi=10.1038%2Fnmeth0809-551" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Sub-angstrom accuracy in protein loop reconstruction by robotics-inspired conformational sampling</span></div><div class="casAuthors">Mandell, Daniel J.; Coutsias, Evangelos A.; Kortemme, Tanja</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">551-552</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQZjfGA_pks7Vg90H21EOLACvtfcHk0ljuIVGgThvI8w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Whtbw%253D&amp;md5=9d88311f894ae547071abc176e76ae3f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnmeth0809-551&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth0809-551%26sid%3Dliteratum%253Aachs%26aulast%3DMandell%26aufirst%3DD.%2BJ.%26aulast%3DCoutsias%26aufirst%3DE.%2BA.%26aulast%3DKortemme%26aufirst%3DT.%26atitle%3DSub-angstrom%2520accuracy%2520in%2520protein%2520loop%2520reconstruction%2520by%2520robotics-inspired%2520conformational%2520sampling%26jtitle%3DNat.%2520Methods%26date%3D2009%26volume%3D6%26issue%3D8%26spage%3D551%26doi%3D10.1038%2Fnmeth0809-551" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shivakumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Prediction of Absolute Solvation Free Energies using Molecular Dynamics Free Energy Perturbation and the OPLS Force Field</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1509</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct900587b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct900587b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkslKhu7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2010&amp;pages=1509&amp;issue=5&amp;author=D.+Shivakumarauthor=J.+Williamsauthor=Y.+J.+Wuauthor=W.+Dammauthor=J.+Shelleyauthor=W.+Sherman&amp;title=Prediction+of+Absolute+Solvation+Free+Energies+using+Molecular+Dynamics+Free+Energy+Perturbation+and+the+OPLS+Force+Field&amp;doi=10.1021%2Fct900587b" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Absolute Solvation Free Energies using Molecular Dynamics Free Energy Perturbation and the OPLS Force Field</span></div><div class="casAuthors">Shivakumar, Devleena; Williams, Joshua; Wu, Yujie; Damm, Wolfgang; Shelley, John; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1509-1519</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accurate prediction of protein-ligand binding free energies is a primary objective in computer-aided drug design.  The solvation free energy of a small mol. provides a surrogate to the desolvation of the ligand in the thermodn. process of protein-ligand binding.  Here, we use explicit solvent mol. dynamics free energy perturbation to predict the abs. solvation free energies of a set of 239 small mols., spanning diverse chem. functional groups commonly found in drugs and drug-like mols.  We also compare the performance of abs. solvation free energies obtained using the OPLS_2005 force field with two other commonly used small mol. force fields - general AMBER force field (GAFF) with AM1-BCC charges and CHARMm-MSI with CHelpG charges.  Using the OPLS_2005 force field, we obtain high correlation with exptl. solvation free energies (R2 = 0.94) and low av. unsigned errors for a majority of the functional groups compared to AM1-BCC/GAFF or CHelpG/CHARMm-MSI.  However, OPLS_2005 has errors of over 1.3 kcal/mol for certain classes of polar compds.  We show that predictions on these compd. classes can be improved by using a semiempirical charge assignment method with an implicit bond charge correction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon_fkbSHqg3bVg90H21EOLACvtfcHk0lirZ-X8IYRRDQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXkslKhu7g%253D&amp;md5=25d464fb9238e3235e881945c0b50a76</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fct900587b&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct900587b%26sid%3Dliteratum%253Aachs%26aulast%3DShivakumar%26aufirst%3DD.%26aulast%3DWilliams%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%2BJ.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DShelley%26aufirst%3DJ.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DPrediction%2520of%2520Absolute%2520Solvation%2520Free%2520Energies%2520using%2520Molecular%2520Dynamics%2520Free%2520Energy%2520Perturbation%2520and%2520the%2520OPLS%2520Force%2520Field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26issue%3D5%26spage%3D1509%26doi%3D10.1021%2Fct900587b" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sondergaard, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, M. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rostkowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span> <span> </span><span class="NLM_article-title">Improved Treatment of Ligands and Coupling Effects in Empirical Calculation and Rationalization of pK(a) Values</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2284</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct200133y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200133y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnt1Gnsrs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2011&amp;pages=2284&amp;issue=7&amp;author=C.+R.+Sondergaardauthor=M.+H.+M.+Olssonauthor=M.+Rostkowskiauthor=J.+H.+Jensen&amp;title=Improved+Treatment+of+Ligands+and+Coupling+Effects+in+Empirical+Calculation+and+Rationalization+of+pK%28a%29+Values&amp;doi=10.1021%2Fct200133y" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Treatment of Ligands and Coupling Effects in Empirical Calculation and Rationalization of pKa Values</span></div><div class="casAuthors">Sondergaard, Chresten R.; Olsson, Mats H. M.; Rostkowski, Michal; Jensen, Jan H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2284-2295</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The new empirical rules for protein pKa predictions implemented in the PROPKA3.0 software package have been extended to the prediction of pKa shifts of active site residues and ionizable ligand groups in protein-ligand complexes.  The authors present new algorithms that allow pKa shifts due to inductive (i.e., covalently coupled) intraligand interactions, as well as noncovalently coupled interligand interactions in multiligand complexes, to be included in the prediction.  The no. of different ligand chem. groups that are automatically recognized has been increased to 18, and the general implementation has been changed so that new functional groups can be added easily by the user, aided by a new and more general protonation scheme.  Except for a few cases, the new algorithms in PROPKA3.1 are found to yield results similar to or better than those obtained with PROPKA2.0.  Finally, the authors present a novel algorithm that identifies noncovalently coupled ionizable groups, where pKa prediction may be esp. difficult.  This is a general improvement to PROPKA and is applied to proteins with and without ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVEiJYZqkno7Vg90H21EOLACvtfcHk0lirZ-X8IYRRDQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXnt1Gnsrs%253D&amp;md5=cf4d4e20d6daa70de6ac623915e78160</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fct200133y&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200133y%26sid%3Dliteratum%253Aachs%26aulast%3DSondergaard%26aufirst%3DC.%2BR.%26aulast%3DOlsson%26aufirst%3DM.%2BH.%2BM.%26aulast%3DRostkowski%26aufirst%3DM.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26atitle%3DImproved%2520Treatment%2520of%2520Ligands%2520and%2520Coupling%2520Effects%2520in%2520Empirical%2520Calculation%2520and%2520Rationalization%2520of%2520pK%2528a%2529%2520Values%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26issue%3D7%26spage%3D2284%26doi%3D10.1021%2Fct200133y" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zarzycka, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaidi, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span> <span> </span><span class="NLM_article-title">Harnessing Ion-Binding Sites for GPCR Pharmacology</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>71</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">571</span>, <span class="refDoi">&nbsp;DOI: 10.1124/pr.119.017863</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1124%2Fpr.119.017863" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=31551350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFersrzE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=71&amp;publication_year=2019&amp;pages=571&amp;issue=4&amp;author=B.+Zarzyckaauthor=S.+A.+Zaidiauthor=B.+L.+Rothauthor=V.+Katritch&amp;title=Harnessing+Ion-Binding+Sites+for+GPCR+Pharmacology&amp;doi=10.1124%2Fpr.119.017863" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing ion-binding sites for GPCR pharmacology</span></div><div class="casAuthors">Zarzycka, Barbara; Zaidi, Saheem A.; Roth, Bryan L.; Katritch, Vsevolod</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">571-595</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Endogenous ions play important roles in the function and pharmacol. of G-protein coupled receptors (GPCRs).  Since then, numerous studies documenting the effects of mono- and divalent ions on GPCR function have been published.  While ions can act selectively and nonselectively at many sites in different receptors, the discovery of the conserved sodium ion site in class A GPCR structures in 2012 revealed the unique nature of Na1 site, which has emerged as a near-universal site for allosteric modulation of class A GPCR structure and function.  In this review, we synthesize and highlight recent advances in the functional, biophys., and structural characterization of ions bound to GPCRs.  Taken together, these findings provide a mol. understanding of the unique roles of Na1 and other ions as GPCR allosteric modulators.  Wewill also discuss how this knowledge can be applied to the redesign of receptors and ligand probes for desired functional and pharmacol. profiles.  Significance Statement--The function and pharmacol. of GPCRs strongly depend on the presence of mono and divalent ions in exptl. assays and in living organisms.  Recent insights into the mol. mechanism of this ion-dependent allosterism from structural, biophys., biochem., and computational studies provide quant. understandings of the pharmacol. effects of drugs in vitro and in vivo and open new avenues for the rational design of chem. probes and drug candidates with improved properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQejnWEn8A5LVg90H21EOLACvtfcHk0lirZ-X8IYRRDQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3cXhtFersrzE&amp;md5=0770795abba451d39a1f8a96f957e5f3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1124%2Fpr.119.017863&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.119.017863%26sid%3Dliteratum%253Aachs%26aulast%3DZarzycka%26aufirst%3DB.%26aulast%3DZaidi%26aufirst%3DS.%2BA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DKatritch%26aufirst%3DV.%26atitle%3DHarnessing%2520Ion-Binding%2520Sites%2520for%2520GPCR%2520Pharmacology%26jtitle%3DPharmacol.%2520Rev.%26date%3D2019%26volume%3D71%26issue%3D4%26spage%3D571%26doi%3D10.1124%2Fpr.119.017863" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span> <span> </span><span class="NLM_article-title">Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">6â7</span>),  <span class="NLM_fpage">591</span>, <span class="refDoi">&nbsp;DOI: 10.1007/s10822-010-9349-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1007%2Fs10822-010-9349-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=20354892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=24&amp;publication_year=2010&amp;pages=591&amp;issue=6%E2%80%937&amp;author=J.+R.+Greenwoodauthor=D.+Calkinsauthor=A.+P.+Sullivanauthor=J.+C.+Shelley&amp;title=Towards+the+comprehensive%2C+rapid%2C+and+accurate+prediction+of+the+favorable+tautomeric+states+of+drug-like+molecules+in+aqueous+solution&amp;doi=10.1007%2Fs10822-010-9349-1" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution</span></div><div class="casAuthors">Greenwood, Jeremy R.; Calkins, David; Sullivan, Arron P.; Shelley, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6-7</span>),
    <span class="NLM_cas:pages">591-604</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Generating the appropriate protonation states of drug-like mols. in soln. is important for success in both ligand- and structure-based virtual screening.  Screening collections of millions of compds. requires a method for detg. tautomers and their energies that is sufficiently rapid, accurate, and comprehensive.  To maximize enrichment, the lowest energy tautomers must be detd. from heterogeneous input, without over-enumerating unfavorable states.  While computationally expensive, the d. functional theory (DFT) method M06-2X/aug-cc-pVTZ(-f) [PB-SCRF] provides accurate energies for enumerated model tautomeric systems.  The empirical Hammett-Taft methodol. can very rapidly extrapolate substituent effects from model systems to drug-like mols. via the relationship between pKT and pKa.  Combining the 2 complementary approaches transforms the tautomer problem from a scientific challenge to one of engineering scale-up, and avoids issues that arise due to the very limited no. of measured pKT values, esp. for the complicated heterocycles often favored by medicinal chemists for their novelty and versatility.  Several hundreds of pre-calcd. tautomer energies and substituent pKa effects are tabulated in databases for use in structural adjustment by the program Epik, which treats tautomers as a subset of the larger problem of the protonation states in aq. ensembles and their energy penalties.  Accuracy and coverage is continually improved and expanded by parameterizing new systems of interest using DFT and exptl. data.  Recommendations are made for how to best incorporate tautomers in mol. design and virtual screening workflows.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpigkzHeo8JY7Vg90H21EOLACvtfcHk0lirZ-X8IYRRDQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbo%253D&amp;md5=1d7bc0f966ca793d6be80554868367b8</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9349-1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9349-1%26sid%3Dliteratum%253Aachs%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DCalkins%26aufirst%3DD.%26aulast%3DSullivan%26aufirst%3DA.%2BP.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26atitle%3DTowards%2520the%2520comprehensive%252C%2520rapid%252C%2520and%2520accurate%2520prediction%2520of%2520the%2520favorable%2520tautomeric%2520states%2520of%2520drug-like%2520molecules%2520in%2520aqueous%2520solution%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26issue%3D6%25E2%2580%25937%26spage%3D591%26doi%3D10.1007%2Fs10822-010-9349-1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoefling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aponte-Santamaria, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubmuller, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groenhof, G.</span></span> <span> </span><span class="NLM_article-title">g_membed: Efficient insertion of a membrane protein into an equilibrated lipid bilayer with minimal perturbation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2169</span>, <span class="refDoi">&nbsp;DOI: 10.1002/jcc.21507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1002%2Fjcc.21507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=20336801" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFWjs7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=31&amp;publication_year=2010&amp;pages=2169&amp;issue=11&amp;author=M.+G.+Wolfauthor=M.+Hoeflingauthor=C.+Aponte-Santamariaauthor=H.+Grubmullerauthor=G.+Groenhof&amp;title=g_membed%3A+Efficient+insertion+of+a+membrane+protein+into+an+equilibrated+lipid+bilayer+with+minimal+perturbation&amp;doi=10.1002%2Fjcc.21507" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">g_membed: Efficient insertion of a membrane protein into an equilibrated lipid bilayer with minimal perturbation</span></div><div class="casAuthors">Wolf, Maarten G.; Hoefling, Martin; Aponte-Santamaria, Camilo; Grubmueller, Helmut; Groenhof, Gerrit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2169-2174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley &amp; Sons, Inc.</span>)
        </div><div class="casAbstract">To efficiently insert a protein into an equilibrated and fully hydrated membrane with minimal membrane perturbation the authors present a computational tool, called g_membed, which is part of the Gromacs suite of programs.  The input consists of an equilibrated membrane system, either flat or curved, and a protein structure in the right position and orientation with respect to the lipid bilayer.  G_membed first decreases the width of the protein in the xy-plane and removes all mols. (generally lipids and waters) that overlap with the narrowed protein.  Then the protein is grown back to its full size in a short mol. dynamics simulation (typically 1000 steps), thereby pushing the lipids away to optimally accommodate the protein in the membrane.  After embedding the protein in the membrane, both the lipid properties and the hydration layer are still close to equil.  Thus, only a short equilibration run (less then 1 ns in the cases tested) is required to re-equilibrate the membrane.  Its simplicity makes g_membed very practical for use in scripting and high-throughput mol. dynamics simulations.  Â© 2010 Wiley Periodicals, Inc.  J Comput Chem, 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0efGbaCaVIbVg90H21EOLACvtfcHk0ljlX-YsLhy_jg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXnsFWjs7c%253D&amp;md5=1f0353d68fb867fb684283b61d52eadb</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21507&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21507%26sid%3Dliteratum%253Aachs%26aulast%3DWolf%26aufirst%3DM.%2BG.%26aulast%3DHoefling%26aufirst%3DM.%26aulast%3DAponte-Santamaria%26aufirst%3DC.%26aulast%3DGrubmuller%26aufirst%3DH.%26aulast%3DGroenhof%26aufirst%3DG.%26atitle%3Dg_membed%253A%2520Efficient%2520insertion%2520of%2520a%2520membrane%2520protein%2520into%2520an%2520equilibrated%2520lipid%2520bilayer%2520with%2520minimal%2520perturbation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26issue%3D11%26spage%3D2169%26doi%3D10.1002%2Fjcc.21507" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lomize, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogozheva, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosberg, H. I.</span></span> <span> </span><span class="NLM_article-title">Anisotropic solvent model of the lipid bilayer. 1. Parameterization of long-range electrostatics and first solvation shell effects</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">918</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ci2000192</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2000192" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvVSisbo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=51&amp;publication_year=2011&amp;pages=918&amp;issue=4&amp;author=A.+L.+Lomizeauthor=I.+D.+Pogozhevaauthor=H.+I.+Mosberg&amp;title=Anisotropic+solvent+model+of+the+lipid+bilayer.+1.+Parameterization+of+long-range+electrostatics+and+first+solvation+shell+effects&amp;doi=10.1021%2Fci2000192" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Anisotropic Solvent Model of the Lipid Bilayer. 1. Parameterization of Long-Range Electrostatics and First Solvation Shell Effects</span></div><div class="casAuthors">Lomize, Andrei L.; Pogozheva, Irina D.; Mosberg, Henry I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">918-929</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new implicit solvation model was developed for calcg. free energies of transfer of mols. from water to any solvent with defined bulk properties.  The transfer energy was calcd. as a sum of the first solvation shell energy and the long-range electrostatic contribution.  The first term was proportional to solvent accessible surface area and solvation parameters (Ïi) for different atom types.  The electrostatic term was computed as a product of group dipole moments and dipolar solvation parameter (Î·) for neutral mols. or using a modified Born equation for ions.  The regression coeffs. in linear dependencies of solvation parameters Ïi and Î· on dielec. const., solvatochromic polarizability parameter Ï*, and hydrogen-bonding donor and acceptor capacities of solvents were optimized using 1269 exptl. transfer energies from 19 org. solvents to water.  The root-mean-square errors for neutral compds. and ions were 0.82 and 1.61 kcal/mol, resp.  Quantification of energy components demonstrates the dominant roles of hydrophobic effect for nonpolar atoms and of hydrogen-bonding for polar atoms.  The estd. first solvation shell energy outweighs the long-range electrostatics for most compds. including ions.  The simplicity and computational efficiency of the model allows its application for modeling of macromols. in anisotropic environments, such as biol. membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonnF3m5c7LEbVg90H21EOLACvtfcHk0ljlX-YsLhy_jg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXjvVSisbo%253D&amp;md5=00ceb7dde2c69eb0353480b227bb93bc</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fci2000192&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2000192%26sid%3Dliteratum%253Aachs%26aulast%3DLomize%26aufirst%3DA.%2BL.%26aulast%3DPogozheva%26aufirst%3DI.%2BD.%26aulast%3DMosberg%26aufirst%3DH.%2BI.%26atitle%3DAnisotropic%2520solvent%2520model%2520of%2520the%2520lipid%2520bilayer.%25201.%2520Parameterization%2520of%2520long-range%2520electrostatics%2520and%2520first%2520solvation%2520shell%2520effects%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26issue%3D4%26spage%3D918%26doi%3D10.1021%2Fci2000192" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klauda, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venable, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freites, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâConnor, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobias, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondragon-Ramirez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vorobyov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, R. W.</span></span> <span> </span><span class="NLM_article-title">Update of the CHARMM All-Atom Additive Force Field for Lipids: Validation on Six Lipid Types</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">7830</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jp101759q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp101759q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsVGjtrk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=114&amp;publication_year=2010&amp;pages=7830&amp;issue=23&amp;author=J.+B.+Klaudaauthor=R.+M.+Venableauthor=J.+A.+Freitesauthor=J.+W.+O%E2%80%99Connorauthor=D.+J.+Tobiasauthor=C.+Mondragon-Ramirezauthor=I.+Vorobyovauthor=A.+D.+MacKerellauthor=R.+W.+Pastor&amp;title=Update+of+the+CHARMM+All-Atom+Additive+Force+Field+for+Lipids%3A+Validation+on+Six+Lipid+Types&amp;doi=10.1021%2Fjp101759q" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Update of the CHARMM All-Atom Additive Force Field for Lipids: Validation on Six Lipid Types</span></div><div class="casAuthors">Klauda, Jeffery B.; Venable, Richard M.; Freites, J. Alfredo; O'Connor, Joseph W.; Tobias, Douglas J.; Mondragon-Ramirez, Carlos; Vorobyov, Igor; MacKerell, Alexander D., Jr.; Pastor, Richard W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7830-7843</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A significant modification to the additive all-atom CHARMM lipid force field (FF) is developed and applied to phospholipid bilayers with both choline and ethanolamine contg. head groups and with both satd. and unsatd. aliph. chains.  Motivated by the current CHARMM lipid FF (C27 and C27r) systematically yielding values of the surface area per lipid that are smaller than exptl. ests. and gel-like structures of bilayers well above the gel transition temp., selected torsional, Lennard-Jones and partial at. charge parameters were modified by targeting both quantum mech. (QM) and exptl. data.  QM calcns. ranging from high-level ab initio calcns. on small mols. to semiempirical QM studies on a 1,2-dipalmitoyl-sn-phosphatidylcholine (DPPC) bilayer in combination with exptl. thermodn. data were used as target data for parameter optimization.  These changes were tested with simulations of pure bilayers at high hydration of the following six lipids: DPPC, 1,2-dimyristoyl-sn-phosphatidylcholine (DMPC), 1,2-dilauroyl-sn-phosphatidylcholine (DLPC), 1-palmitoyl-2-oleoyl-sn-phosphatidylcholine (POPC), 1,2-dioleoyl-sn-phosphatidylcholine (DOPC), and 1-palmitoyl-2-oleoyl-sn-phosphatidylethanolamine (POPE); simulations of a low hydration DOPC bilayer were also performed.  Agreement with exptl. surface area is on av. within 2%, and the d. profiles agree well with neutron and x-ray diffraction expts.  NMR deuterium order parameters (SCD) are well predicted with the new FF, including proper splitting of the SCD for the aliph. carbon adjacent to the carbonyl for DPPC, POPE, and POPC bilayers.  The area compressibility modulus and frequency dependence of 13C NMR relaxation rates of DPPC and the water distribution of low hydration DOPC bilayers also agree well with expt.  Accordingly, the presented lipid FF, referred to as C36, allows for mol. dynamics simulations to be run in the tensionless ensemble (NPT), and is anticipated to be of utility for simulations of pure lipid systems as well as heterogeneous systems including membrane proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhTVhlLR7mEbVg90H21EOLACvtfcHk0ljwEliuSdMKFA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXmsVGjtrk%253D&amp;md5=04c4d3aa1a59740190994d7df80cbb71</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjp101759q&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp101759q%26sid%3Dliteratum%253Aachs%26aulast%3DKlauda%26aufirst%3DJ.%2BB.%26aulast%3DVenable%26aufirst%3DR.%2BM.%26aulast%3DFreites%26aufirst%3DJ.%2BA.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DJ.%2BW.%26aulast%3DTobias%26aufirst%3DD.%2BJ.%26aulast%3DMondragon-Ramirez%26aufirst%3DC.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DPastor%26aufirst%3DR.%2BW.%26atitle%3DUpdate%2520of%2520the%2520CHARMM%2520All-Atom%2520Additive%2520Force%2520Field%2520for%2520Lipids%253A%2520Validation%2520on%2520Six%2520Lipid%2520Types%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2010%26volume%3D114%26issue%3D23%26spage%3D7830%26doi%3D10.1021%2Fjp101759q" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanommeslaeghe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raman, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Automation of the CHARMM General Force Field (CGenFF) II: assignment of bonded parameters and partial atomic charges</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3155</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ci3003649</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3003649" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=2012&amp;pages=3155&amp;issue=12&amp;author=K.+Vanommeslaegheauthor=E.+P.+Ramanauthor=A.+D.+MacKerell&amp;title=Automation+of+the+CHARMM+General+Force+Field+%28CGenFF%29+II%3A+assignment+of+bonded+parameters+and+partial+atomic+charges&amp;doi=10.1021%2Fci3003649" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) II: Assignment of Bonded Parameters and Partial Atomic Charges</span></div><div class="casAuthors">Vanommeslaeghe, K.; Raman, E. Prabhu; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3155-3168</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug candidates interacting with biol. systems.  In these simulations, the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and partial at. charges is required.  In the present article, algorithms for the assignment of parameters and charges for the CHARMM General Force Field (CGenFF) are presented.  These algorithms rely on the existing parameters and charges that were detd. as part of the parametrization of the force field.  Bonded parameters are assigned based on the similarity between the atom types that define said parameters, while charges are detd. using an extended bond-charge increment scheme.  Charge increments were optimized to reproduce the charges on model compds. that were part of the parametrization of the force field.  Case studies are presented to clarify the functioning of the algorithms and the significance of their output data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0zgm66CLdn7Vg90H21EOLACvtfcHk0ljwEliuSdMKFA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF&amp;md5=e676ad1f42cb1e98dd353d4d285e8d13</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fci3003649&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3003649%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520General%2520Force%2520Field%2520%2528CGenFF%2529%2520II%253A%2520assignment%2520of%2520bonded%2520parameters%2520and%2520partial%2520atomic%2520charges%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D12%26spage%3D3155%26doi%3D10.1021%2Fci3003649" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J.</span>; <span class="NLM_string-name">Trucks, G. W.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Petersson, G. A.</span>; <span class="NLM_string-name">Nakatsuji, H.</span>; <span class="NLM_string-name">Caricato, M.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Hratchian, H. P.</span>; <span class="NLM_string-name">Izmaylov, A. F.</span>; <span class="NLM_string-name">Bloino, J.</span>; <span class="NLM_string-name">Zheng, G.</span>; <span class="NLM_string-name">Sonnenberg, J. L.</span>; <span class="NLM_string-name">Hada, M.</span>; <span class="NLM_string-name">Ehara, M.</span>; <span class="NLM_string-name">Toyota, K.</span>; <span class="NLM_string-name">Fukuda, R.</span>; <span class="NLM_string-name">Hasegawa, J.</span>; <span class="NLM_string-name">Ishida, M.</span>; <span class="NLM_string-name">Nakajima, T.</span>; <span class="NLM_string-name">Honda, Y.</span>; <span class="NLM_string-name">Kitao, O.</span>; <span class="NLM_string-name">Nakai, H.</span>; <span class="NLM_string-name">Vreven, T.</span>; <span class="NLM_string-name">Montgomery,  Jr., J. A.</span>; <span class="NLM_string-name">Peralta, J. E.</span>; <span class="NLM_string-name">Ogliaro, F.</span>; <span class="NLM_string-name">Bearpark, M.</span>; <span class="NLM_string-name">Heyd, J. J.</span>; <span class="NLM_string-name">Brothers, E.</span>; <span class="NLM_string-name">Kudin, K. N.</span>; <span class="NLM_string-name">Staroverov, V. N.</span>; <span class="NLM_string-name">Kobayashi, R.</span>; <span class="NLM_string-name">Normand, J.</span>; <span class="NLM_string-name">Raghavachari, K.</span>; <span class="NLM_string-name">Rendell, A.</span>; <span class="NLM_string-name">Burant, J. C.</span>; <span class="NLM_string-name">Iyengar, S. S.</span>; <span class="NLM_string-name">Tomasi, J.</span>; <span class="NLM_string-name">Cossi, M.</span>; <span class="NLM_string-name">Rega, N.</span>; <span class="NLM_string-name">Millam, J. M.</span>; <span class="NLM_string-name">Klene, M.</span>; <span class="NLM_string-name">Knox, J. E.</span>; <span class="NLM_string-name">Cross, J. B.</span>; <span class="NLM_string-name">Bakken, V.</span>; <span class="NLM_string-name">Adamo, C.</span>; <span class="NLM_string-name">Jaramillo, J.</span>; <span class="NLM_string-name">Gomperts, R.</span>; <span class="NLM_string-name">Stratmann, R. E.</span>; <span class="NLM_string-name">Yazyev, O.</span>; <span class="NLM_string-name">Austin, A. J.</span>; <span class="NLM_string-name">Cammi, R.</span>; <span class="NLM_string-name">Pomelli, C.</span>; <span class="NLM_string-name">Ochterski, J. W.</span>; <span class="NLM_string-name">Martin, R. L.</span>; <span class="NLM_string-name">Morokuma, K.</span>; <span class="NLM_string-name">Zakrzewski, V. G.</span>; <span class="NLM_string-name">Voth, G. A.</span>; <span class="NLM_string-name">Salvador, P.</span>; <span class="NLM_string-name">Dannenberg, J. J.</span>; <span class="NLM_string-name">Dapprich, S.</span>; <span class="NLM_string-name">Daniels, A. D.</span>; <span class="NLM_string-name">Farkas, Ã.</span>; <span class="NLM_string-name">Foresman, J. B.</span>; <span class="NLM_string-name">Ortiz, J. V.</span>; <span class="NLM_string-name">Cioslowski, J.</span>; <span class="NLM_string-name">Fox, D. J.</span></span> <i>Gaussian 09</i>, Revision A.1; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford CT</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2009&amp;author=M.+J.+Frisch&amp;author=G.+W.+Trucks&amp;author=H.+B.+Schlegel&amp;author=G.+E.+Scuseria&amp;author=M.+A.+Robb&amp;author=J.+R.+Cheeseman&amp;author=G.+Scalmani&amp;author=V.+Barone&amp;author=B.+Mennucci&amp;author=G.+A.+Petersson&amp;author=H.+Nakatsuji&amp;author=M.+Caricato&amp;author=X.+Li&amp;author=H.+P.+Hratchian&amp;author=A.+F.+Izmaylov&amp;author=J.+Bloino&amp;author=G.+Zheng&amp;author=J.+L.+Sonnenberg&amp;author=M.+Hada&amp;author=M.+Ehara&amp;author=K.+Toyota&amp;author=R.+Fukuda&amp;author=J.+Hasegawa&amp;author=M.+Ishida&amp;author=T.+Nakajima&amp;author=Y.+Honda&amp;author=O.+Kitao&amp;author=H.+Nakai&amp;author=T.+Vreven&amp;author=J.+A.+Montgomery&amp;author=J.+E.+Peralta&amp;author=F.+Ogliaro&amp;author=M.+Bearpark&amp;author=J.+J.+Heyd&amp;author=E.+Brothers&amp;author=K.+N.+Kudin&amp;author=V.+N.+Staroverov&amp;author=R.+Kobayashi&amp;author=J.+Normand&amp;author=K.+Raghavachari&amp;author=A.+Rendell&amp;author=J.+C.+Burant&amp;author=S.+S.+Iyengar&amp;author=J.+Tomasi&amp;author=M.+Cossi&amp;author=N.+Rega&amp;author=J.+M.+Millam&amp;author=M.+Klene&amp;author=J.+E.+Knox&amp;author=J.+B.+Cross&amp;author=V.+Bakken&amp;author=C.+Adamo&amp;author=J.+Jaramillo&amp;author=R.+Gomperts&amp;author=R.+E.+Stratmann&amp;author=O.+Yazyev&amp;author=A.+J.+Austin&amp;author=R.+Cammi&amp;author=C.+Pomelli&amp;author=J.+W.+Ochterski&amp;author=R.+L.+Martin&amp;author=K.+Morokuma&amp;author=V.+G.+Zakrzewski&amp;author=G.+A.+Voth&amp;author=P.+Salvador&amp;author=J.+J.+Dannenberg&amp;author=S.+Dapprich&amp;author=A.+D.+Daniels&amp;author=%C3%96.+Farkas&amp;author=J.+B.+Foresman&amp;author=J.+V.+Ortiz&amp;author=J.+Cioslowski&amp;author=D.+J.+Fox&amp;title=Gaussian+09" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%26pub%3DGaussian%252C%2520Inc%26date%3D2009" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pronk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pall, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjelkmar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apostolov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirts, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasson, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Spoel, D.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">845</span>, <span class="refDoi">&nbsp;DOI: 10.1093/bioinformatics/btt055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1093%2Fbioinformatics%2Fbtt055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=23407358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFWmsrg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=2013&amp;pages=845&amp;issue=7&amp;author=S.+Pronkauthor=S.+Pallauthor=R.+Schulzauthor=P.+Larssonauthor=P.+Bjelkmarauthor=R.+Apostolovauthor=M.+R.+Shirtsauthor=J.+C.+Smithauthor=P.+M.+Kassonauthor=D.+van+der%0ASpoel&amp;title=GROMACS+4.5%3A+a+high-throughput+and+highly+parallel+open+source+molecular+simulation+toolkit&amp;doi=10.1093%2Fbioinformatics%2Fbtt055" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit</span></div><div class="casAuthors">Pronk, Sander; Pall, Szilard; Schulz, Roland; Larsson, Per; Bjelkmar, Paer; Apostolov, Rossen; Shirts, Michael R.; Smith, Jeremy C.; Kasson, Peter M.; van der Spoel, David; Hess, Berk; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">845-854</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Mol. simulation has historically been a low-throughput technique, but faster computers and increasing amts. of genomic and structural data are changing this by enabling large-scale automated simulation of, for instance, many conformers or mutants of biomols. with or without a range of ligands.  At the same time, advances in performance and scaling now make it possible to model complex biomol. interaction and function in a manner directly testable by expt.  These applications share a need for fast and efficient software that can be deployed on massive scale in clusters, web servers, distributed computing or cloud resources.  Results: Here, we present a range of new simulation algorithms and features developed during the past 4 years, leading up to the GROMACS 4.5 software package.  The software now automatically handles wide classes of biomols., such as proteins, nucleic acids and lipids, and comes with all commonly used force fields for these mols. built-in.  GROMACS supports several implicit solvent models, as well as new free-energy algorithms, and the software now uses multithreading for efficient parallelization even on low-end systems, including windows-based workstations.  Together with hand-tuned assembly kernels and state-of-the-art parallelization, this provides extremely high performance and cost efficiency for high-throughput as well as massively parallel simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr71lQgYMm017Vg90H21EOLACvtfcHk0liqty1DEJXhsw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXksFWmsrg%253D&amp;md5=4b25fd6ab4e33725ae56b5da63f4ad68</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt055&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt055%26sid%3Dliteratum%253Aachs%26aulast%3DPronk%26aufirst%3DS.%26aulast%3DPall%26aufirst%3DS.%26aulast%3DSchulz%26aufirst%3DR.%26aulast%3DLarsson%26aufirst%3DP.%26aulast%3DBjelkmar%26aufirst%3DP.%26aulast%3DApostolov%26aufirst%3DR.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26aulast%3DKasson%26aufirst%3DP.%2BM.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26atitle%3DGROMACS%25204.5%253A%2520a%2520high-throughput%2520and%2520highly%2520parallel%2520open%2520source%2520molecular%2520simulation%2520toolkit%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26issue%3D7%26spage%3D845%26doi%3D10.1093%2Fbioinformatics%2Fbtt055" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Im, W.</span></span> <span> </span><span class="NLM_article-title">CHARMM-GUI: a web-based graphical user interface for CHARMM</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1859</span>, <span class="refDoi">&nbsp;DOI: 10.1002/jcc.20945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1002%2Fjcc.20945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=18351591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD1cXosVKksbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=2008&amp;pages=1859&amp;issue=11&amp;author=S.+Joauthor=T.+Kimauthor=V.+G.+Iyerauthor=W.+Im&amp;title=CHARMM-GUI%3A+a+web-based+graphical+user+interface+for+CHARMM&amp;doi=10.1002%2Fjcc.20945" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM-GUI: a web-based graphical user interface for CHARMM</span></div><div class="casAuthors">Jo, Sunhwan; Kim, Taehoon; Iyer, Vidyashankara G.; Im, Wonpil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1859-1865</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley &amp; Sons, Inc.</span>)
        </div><div class="casAbstract">CHARMM is an academic research program used widely for macromol. mechanics and dynamics with versatile anal. and manipulation tools of at. coordinates and dynamics trajectories.  CHARMM-GUI, http://www.charmm-gui.org, has been developed to provide a web-based graphical user interface to generate various input files and mol. systems to facilitate and standardize the usage of common and advanced simulation techniques in CHARMM.  The web environment provides an ideal platform to build and validate a mol. model system in an interactive fashion such that, if a problem is found through visual inspection, one can go back to the previous setup and regenerate the whole system again.  In this article, we describe the currently available functional modules of CHARMM-GUI Input Generator that form a basis for the advanced simulation techniques.  Future directions of the CHARMM-GUI development project are also discussed briefly together with other features in the CHARMM-GUI website, such as Archive and Movie Gallery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7gpU-1N3WOLVg90H21EOLACvtfcHk0liqty1DEJXhsw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXosVKksbc%253D&amp;md5=112a3dd61d792b040f9f716b32220d7e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20945&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20945%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DIyer%26aufirst%3DV.%2BG.%26aulast%3DIm%26aufirst%3DW.%26atitle%3DCHARMM-GUI%253A%2520a%2520web-based%2520graphical%2520user%2520interface%2520for%2520CHARMM%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2008%26volume%3D29%26issue%3D11%26spage%3D1859%26doi%3D10.1002%2Fjcc.20945" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H.</span></span> <span> </span><span class="NLM_article-title">PyMOL and Inkscape Bridge the Data and the Data Visualization</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2041</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.str.2016.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.str.2016.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=27926832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVegtbjJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=24&amp;publication_year=2016&amp;pages=2041&amp;issue=12&amp;author=S.+Yuanauthor=H.+C.+Chanauthor=S.+Filipekauthor=H.+Vogel&amp;title=PyMOL+and+Inkscape+Bridge+the+Data+and+the+Data+Visualization&amp;doi=10.1016%2Fj.str.2016.11.012" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">PyMOL and Inkscape Bridge the Data and the Data Visualization</span></div><div class="casAuthors">Yuan, Shuguang; Chan, H. C. Stephen; Filipek, Slawomir; Vogel, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2041-2042</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  This article discussed about PyMOL, a dedicated tool for macromol. visualization and Inkscape vector software of professional quality for visualizing structural biol. information and bridge the macromol. data with data visualization protocols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJG3yim_6-FbVg90H21EOLACvtfcHk0liqty1DEJXhsw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XitVegtbjJ&amp;md5=b2e74dbe2aeb110039e22b3c84385b19</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2016.11.012&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2016.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DS.%26aulast%3DChan%26aufirst%3DH.%2BC.%26aulast%3DFilipek%26aufirst%3DS.%26aulast%3DVogel%26aufirst%3DH.%26atitle%3DPyMOL%2520and%2520Inkscape%2520Bridge%2520the%2520Data%2520and%2520the%2520Data%2520Visualization%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26issue%3D12%26spage%3D2041%26doi%3D10.1016%2Fj.str.2016.11.012" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">Pt 2</span>),  <span class="NLM_fpage">213</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2010&amp;pages=213&amp;issue=Pt+2&amp;author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleve&amp;title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&amp;doi=10.1107%2FS0907444909052925" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgzu9PXpQ2_ZQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&amp;md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3DPt%25202%26spage%3D213%26doi%3D10.1107%2FS0907444909052925" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">486</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2010&amp;pages=486&amp;issue=4&amp;author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&amp;title=Features+and+development+of+Coot&amp;doi=10.1107%2FS0907444910007493" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgzu9PXpQ2_ZQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&amp;md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D4%26spage%3D486%26doi%3D10.1107%2FS0907444910007493" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bochevarov, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braden, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philipp, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halls, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Jaguar: A high-performance quantum chemistry software program with strengths in life and materials sciences</span>. <i>Int. J. Quantum Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">2110</span>, <span class="refDoi">&nbsp;DOI: 10.1002/qua.24481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1002%2Fqua.24481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCqs7%252FP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=113&amp;publication_year=2013&amp;pages=2110&amp;issue=18&amp;author=A.+D.+Bochevarovauthor=E.+Harderauthor=T.+F.+Hughesauthor=J.+R.+Greenwoodauthor=D.+A.+Bradenauthor=D.+M.+Philippauthor=D.+Rinaldoauthor=M.+D.+Hallsauthor=J.+Zhangauthor=R.+A.+Friesner&amp;title=Jaguar%3A+A+high-performance+quantum+chemistry+software+program+with+strengths+in+life+and+materials+sciences&amp;doi=10.1002%2Fqua.24481" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Jaguar: A high-performance quantum chemistry software program with strengths in life and materials sciences</span></div><div class="casAuthors">Bochevarov, Art D.; Harder, Edward; Hughes, Thomas F.; Greenwood, Jeremy R.; Braden, Dale A.; Philipp, Dean M.; Rinaldo, David; Halls, Mathew D.; Zhang, Jing; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Quantum Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2110-2142</span>CODEN:
                <span class="NLM_cas:coden">IJQCB2</span>;
        ISSN:<span class="NLM_cas:issn">0020-7608</span>.
    
            (<span class="NLM_cas:orgname">John Wiley &amp; Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Jaguar is an ab initio quantum chem. program that specializes in fast electronic structure predictions for mol. systems of medium and large size.  Jaguar focuses on computational methods with reasonable computational scaling with the size of the system, such as d. functional theory (DFT) and local second-order Moller-Plesset perturbation theory.  The favorable scaling of the methods and the high efficiency of the program make it possible to conduct routine computations involving several thousand MOs.  This performance is achieved through a utilization of the pseudospectral approxn. and several levels of parallelization.  The speed advantages are beneficial for applying Jaguar in biomol. computational modeling.  Addnl., owing to its superior wave function guess for transition-metal-contg. systems, Jaguar finds applications in inorg. and bioinorg. chem.  The emphasis on larger systems and transition metal elements paves the way toward developing Jaguar for its use in materials science modeling.  The article describes the historical and new features of Jaguar, such as improved parallelization of many modules, innovations in ab initio pKa prediction, and new semiempirical corrections for nondynamic correlation errors in DFT.  Jaguar applications in drug discovery, materials science, force field parameterization, and other areas of computational research are reviewed.  Timing benchmarks and other results obtained from the most recent Jaguar code are provided.  The article concludes with a discussion of challenges and directions for future development of the program. Â© 2013 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt_0dnCf0Yq7Vg90H21EOLACvtfcHk0lgzu9PXpQ2_ZQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCqs7%252FP&amp;md5=2fc119bb9e0501e55d7292f7735c24ea</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fqua.24481&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fqua.24481%26sid%3Dliteratum%253Aachs%26aulast%3DBochevarov%26aufirst%3DA.%2BD.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DHughes%26aufirst%3DT.%2BF.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DBraden%26aufirst%3DD.%2BA.%26aulast%3DPhilipp%26aufirst%3DD.%2BM.%26aulast%3DRinaldo%26aufirst%3DD.%26aulast%3DHalls%26aufirst%3DM.%2BD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DJaguar%253A%2520A%2520high-performance%2520quantum%2520chemistry%2520software%2520program%2520with%2520strengths%2520in%2520life%2520and%2520materials%2520sciences%26jtitle%3DInt.%2520J.%2520Quantum%2520Chem.%26date%3D2013%26volume%3D113%26issue%3D18%26spage%3D2110%26doi%3D10.1002%2Fqua.24481" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span> <i>SchrÃ¶dinger</i>; <span class="NLM_publisher-name">SchrÃ¶dinger,
LLC</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=+Schr%C3%B6dinger%3B+Schr%C3%B6dinger%2C%0ALLC%2C+2018." target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2018" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></div><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 10 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Mark T. Agasid, Lars SÃ¸rensen, Leonhard H. Urner, Jun Yan, <span class="NLM_string-name hlFld-ContribAuthor">Carol V. Robinson</span>. </span><span class="cited-content_cbyCitation_article-title">The Effects of Sodium Ions on Ligand Binding and Conformational States of G Protein-Coupled ReceptorsâInsights from Mass Spectrometry. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (11)
                                     , 4085-4089. <a href="https://doi.org/10.1021/jacs.0c11837" title="DOI URL">https://doi.org/10.1021/jacs.0c11837</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c11837&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c11837%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DThe%252BEffects%252Bof%252BSodium%252BIons%252Bon%252BLigand%252BBinding%252Band%252BConformational%252BStates%252Bof%252BG%252BProtein-Coupled%252BReceptors%2525E2%252580%252594Insights%252Bfrom%252BMass%252BSpectrometry%26aulast%3DAgasid%26aufirst%3DMark%2BT.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D11112020%26date%3D12032021%26volume%3D143%26issue%3D11%26spage%3D4085%26epage%3D4089" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yibo Wang, Bo Zhang, Cong Lin, Ying Liu, Min Yang, Yinghua Peng, <span class="NLM_string-name hlFld-ContribAuthor">Xiaohui Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Dissecting Role of Charged Residue from Transmembrane Domain 5 of Latent Membrane Protein 1 via In Silico Simulations and Wet-Lab Experiments. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2021,</strong> <em>125 </em>
                                    (8)
                                     , 2124-2133. <a href="https://doi.org/10.1021/acs.jpcb.0c10708" title="DOI URL">https://doi.org/10.1021/acs.jpcb.0c10708</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.0c10708&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.0c10708%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DDissecting%252BRole%252Bof%252BCharged%252BResidue%252Bfrom%252BTransmembrane%252BDomain%252B5%252Bof%252BLatent%252BMembrane%252BProtein%252B1%252Bvia%252BIn%252BSilico%252BSimulations%252Band%252BWet-Lab%252BExperiments%26aulast%3DWang%26aufirst%3DYibo%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2021%26date%3D30112020%26date%3D07022021%26date%3D17022021%26volume%3D125%26issue%3D8%26spage%3D2124%26epage%3D2133" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ke Liu, Rongfeng Zou, Wenqiang Cui, Meiqing Li, Xueying Wang, Junlin Dong, Hongchun Li, Hongpei Li, Peihui Wang, Ximing Shao, Wu Su, H. C. Stephen Chan, Hongchang Li, <span class="NLM_string-name hlFld-ContribAuthor">Shuguang Yuan</span>. </span><span class="cited-content_cbyCitation_article-title">Clinical HDAC Inhibitors Are Effective Drugs to Prevent the Entry of SARS-CoV2. </span><span class="cited-content_cbyCitation_journal-name">ACS Pharmacology &amp; Translational Science</span><span> <strong>2020,</strong> <em>3 </em>
                                    (6)
                                     , 1361-1370. <a href="https://doi.org/10.1021/acsptsci.0c00163" title="DOI URL">https://doi.org/10.1021/acsptsci.0c00163</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsptsci.0c00163&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsptsci.0c00163%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Pharmacology%2520%2526%2520Translational%2520Science%26atitle%3DClinical%252BHDAC%252BInhibitors%252BAre%252BEffective%252BDrugs%252Bto%252BPrevent%252Bthe%252BEntry%252Bof%252BSARS-CoV2%26aulast%3DLiu%26aufirst%3DKe%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D05102020%26date%3D11112020%26volume%3D3%26issue%3D6%26spage%3D1361%26epage%3D1370" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rongfeng  Zou</span>, <span class="hlFld-ContribAuthor ">Xueying  Wang</span>, <span class="hlFld-ContribAuthor ">Shu  Li</span>, <span class="hlFld-ContribAuthor ">H. C. Stephen  Chan</span>, <span class="hlFld-ContribAuthor ">Horst  Vogel</span>, <span class="hlFld-ContribAuthor ">Shuguang  Yuan</span>. </span><span class="cited-content_cbyCitation_article-title">The role of metal ions in G proteinâcoupled receptor signalling and drug discovery. </span><span class="cited-content_cbyCitation_journal-name">WIREs Computational Molecular Science</span><span> <strong>2022,</strong> <em>12 </em>
                                    (2)
                                     <a href="https://doi.org/10.1002/wcms.1565" title="DOI URL">https://doi.org/10.1002/wcms.1565</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.1565&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.1565%26sid%3Dliteratum%253Aachs%26jtitle%3DWIREs%2520Computational%2520Molecular%2520Science%26atitle%3DThe%252Brole%252Bof%252Bmetal%252Bions%252Bin%252BG%252Bprotein%2525E2%252580%252590coupled%252Breceptor%252Bsignalling%252Band%252Bdrug%252Bdiscovery%26aulast%3DZou%26aufirst%3DRongfeng%26date%3D2022%26date%3D2021%26volume%3D12%26issue%3D2" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tetsuya  Hori</span>, <span class="hlFld-ContribAuthor ">Shigeyuki  Yokoyama</span>. </span><span class="cited-content_cbyCitation_article-title">Ions as GPCR allosteric modulators. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2022,</strong>,, 47-69. <a href="https://doi.org/10.1016/B978-0-12-819771-4.00006-3" title="DOI URL">https://doi.org/10.1016/B978-0-12-819771-4.00006-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-819771-4.00006-3&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-819771-4.00006-3%26sid%3Dliteratum%253Aachs%26atitle%3DIons%252Bas%252BGPCR%252Ballosteric%252Bmodulators%26aulast%3DHori%26aufirst%3DTetsuya%26date%3D2022%26spage%3D47%26epage%3D69%26pub%3DElsevier%26atitle%3DAllosteric%252BModulation%252Bof%252BG%252BProtein-Coupled%252BReceptors%26date%3D2022%26volume%3D16" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Siyu  Zhu</span>, <span class="hlFld-ContribAuthor ">Meixian  Wu</span>, <span class="hlFld-ContribAuthor ">Ziwei  Huang</span>, <span class="hlFld-ContribAuthor ">Jing  An</span>. </span><span class="cited-content_cbyCitation_article-title">Trends in application of advancing computational approaches in GPCR ligand discovery. </span><span class="cited-content_cbyCitation_journal-name">Experimental Biology and Medicine</span><span> <strong>2021,</strong> <em>246 </em>
                                    (9)
                                     , 1011-1024. <a href="https://doi.org/10.1177/1535370221993422" title="DOI URL">https://doi.org/10.1177/1535370221993422</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/1535370221993422&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F1535370221993422%26sid%3Dliteratum%253Aachs%26jtitle%3DExperimental%2520Biology%2520and%2520Medicine%26atitle%3DTrends%252Bin%252Bapplication%252Bof%252Badvancing%252Bcomputational%252Bapproaches%252Bin%252BGPCR%252Bligand%252Bdiscovery%26aulast%3DZhu%26aufirst%3DSiyu%26date%3D2021%26date%3D2021%26volume%3D246%26issue%3D9%26spage%3D1011%26epage%3D1024" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lauren M.  Slosky</span>, <span class="hlFld-ContribAuthor ">Marc G.  Caron</span>, <span class="hlFld-ContribAuthor ">Lawrence S.  Barak</span>. </span><span class="cited-content_cbyCitation_article-title">Biased Allosteric Modulators: New Frontiers in GPCR Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2021,</strong> <em>42 </em>
                                    (4)
                                     , 283-299. <a href="https://doi.org/10.1016/j.tips.2020.12.005" title="DOI URL">https://doi.org/10.1016/j.tips.2020.12.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2020.12.005&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2020.12.005%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DBiased%252BAllosteric%252BModulators%25253A%252BNew%252BFrontiers%252Bin%252BGPCR%252BDrug%252BDiscovery%26aulast%3DSlosky%26aufirst%3DLauren%2BM.%26date%3D2021%26volume%3D42%26issue%3D4%26spage%3D283%26epage%3D299" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">JesÃºs  Giraldo</span>, <span class="hlFld-ContribAuthor ">Bin  Zhou</span>, <span class="hlFld-ContribAuthor ">David  Roche</span>, <span class="hlFld-ContribAuthor ">Carles  Gil</span>, <span class="hlFld-ContribAuthor ">Jordi  Ortiz</span>, <span class="hlFld-ContribAuthor ">Isaias  Lans</span>, <span class="hlFld-ContribAuthor ">James  Dalton</span>, <span class="hlFld-ContribAuthor ">Pedro  Renault</span>. </span><span class="cited-content_cbyCitation_article-title">Analysis of the Function of Receptor Oligomers by Operational Models of Agonism. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/B978-0-12-820472-6.00012-8" title="DOI URL">https://doi.org/10.1016/B978-0-12-820472-6.00012-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-820472-6.00012-8&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-820472-6.00012-8%26sid%3Dliteratum%253Aachs%26atitle%3DAnalysis%252Bof%252Bthe%252BFunction%252Bof%252BReceptor%252BOligomers%252Bby%252BOperational%252BModels%252Bof%252BAgonism%26aulast%3DGiraldo%26aufirst%3DJes%25C3%25BAs%26date%3D2021%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BBiomedical%252BSciences%26date%3D2021%26volume%3D110" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mariona  Torrens-Fontanals</span>, <span class="hlFld-ContribAuthor ">Tomasz Maciej  Stepniewski</span>, <span class="hlFld-ContribAuthor ">David  Aranda-GarcÃ­a</span>, <span class="hlFld-ContribAuthor ">AdriÃ¡n  Morales-Pastor</span>, <span class="hlFld-ContribAuthor ">Brian  Medel-Lacruz</span>, <span class="hlFld-ContribAuthor ">Jana  Selent</span>. </span><span class="cited-content_cbyCitation_article-title">How Do Molecular Dynamics Data Complement Static Structural Data of GPCRs. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (16)
                                     , 5933. <a href="https://doi.org/10.3390/ijms21165933" title="DOI URL">https://doi.org/10.3390/ijms21165933</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21165933&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21165933%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DHow%252BDo%252BMolecular%252BDynamics%252BData%252BComplement%252BStatic%252BStructural%252BData%252Bof%252BGPCRs%26aulast%3DTorrens-Fontanals%26aufirst%3DMariona%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D16%26spage%3D5933" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenqiang  Cui</span>, <span class="hlFld-ContribAuthor ">Adnane  Aouidate</span>, <span class="hlFld-ContribAuthor ">Shouguo  Wang</span>, <span class="hlFld-ContribAuthor ">Qiuliyang  Yu</span>, <span class="hlFld-ContribAuthor ">Yanhua  Li</span>, <span class="hlFld-ContribAuthor ">Shuguang  Yuan</span>. </span><span class="cited-content_cbyCitation_article-title">Discovering Anti-Cancer Drugs via Computational Methods. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fphar.2020.00733" title="DOI URL">https://doi.org/10.3389/fphar.2020.00733</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2020.00733&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2020.00733%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DDiscovering%252BAnti-Cancer%252BDrugs%252Bvia%252BComputational%252BMethods%26aulast%3DCui%26aufirst%3DWenqiang%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0006.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.9b01247&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0001.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Allosteric Na<sup>+</sup> ion next to D80<sup>2.50</sup> in D2R. (A) Entrance pathway of a Na<sup>+</sup> ion from the extracellular receptor interface toward the allosteric site D80<sup>2.50</sup> of apo D2R. Zone I (gray): extracellular receptor interface at which the Na<sup>+</sup> ion is in the bulk environment. Zone II (red): transition zone in the vicinity of D114<sup>3.32</sup>. Zone III (blue): allosteric site in the vicinity of D80<sup>2.50</sup>. Yellow spheres: Na<sup>+</sup> ion positions during MD simulations. (B) Distance between allosteric Na<sup>+</sup> ion and D80<sup>2.50</sup>. (C) The final position of the allosteric Na<sup>+</sup> ion at the end of MD simulations. The oxygen atoms in D80<sup>2.50</sup> and S121<sup>3.39</sup> as well as those of three water molecules establish a dedicated coordination network with the allosteric Na<sup>+</sup> ion.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.9b01247&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0002.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. A second Na<sup>+</sup> ion has been found in the orthosteric site of D2R using all-atom long-time MD simulations. (A) The entrance pathway of the orthosteric Na<sup>+</sup> toward agonist molecule MLS1547. Yellow spheres: Na<sup>+</sup> ion trajectories in the MD simulations. Black ball-and-stick: MLS1547. (B) The distance between orthosteric Na<sup>+</sup> ion and H393<sup>6.55</sup> (C) The final position of the orthosteric Na<sup>+</sup> ion at the end of MD simulations. MLS1547 and H393<sup>6.55</sup> as well as two water molecules establish a dedicated coordination network with the orthosteric Na<sup>+</sup> ion.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.9b01247&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0003.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Interaction fingerprint between distinct residues in D2R and MLS1547. (A) The 2D interaction diagram of D2R in complex with MLS1547. Green ball-and-stick: MLS1547. (B) The interaction fingerprint between residues in D2R and MLS1547. It represents the interaction frequency during the MD simulations. (C) Arrestin recruitment for D2R variants showing mutations on key residues predicted by MD simulations. All mutated receptors show noticeably decreased signaling.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.9b01247&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0004.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Synthesis and biological activity of MLS1547 derivatives. (A) Scheme for synthesis of compound MLS-d1. (B) Activation of D2R mediated signaling induced by dopamine, MLS1547- and MLS1547-derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.9b01247&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/acscentsci.9b01247/asset/images/medium/oc9b01247_0005.gif" alt="" data-lg-src="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Two different roles of Na<sup>+</sup> in D2 receptor activation. Left panel: an allosteric Na<sup>+</sup> (yellow sphere) stabilizes the inactive state of receptor. Blue ball-and-sphere: dopamine molecule. Right panel: an orthosteric Na<sup>+</sup> enhances the activation of receptor via a coordinating interaction with both H393<sup>6.55</sup> and agonist molecule MLS1547 (black ball-and-stick).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acscentsci.9b01247/asset/images/large/oc9b01247_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acscentsci.9b01247&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i32">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10520" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10520" class="dropBlock__holder"></div></div></div></div></div><hr></div><p class="rightTab-references-count">
            This article references 60 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenalti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abola, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Allosteric sodium in class A GPCR signaling</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">233</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.tibs.2014.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.tibs.2014.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=24767681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFCqs7k%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=2014&amp;pages=233&amp;issue=5&amp;author=V.+Katritchauthor=G.+Fenaltiauthor=E.+E.+Abolaauthor=B.+L.+Rothauthor=V.+Cherezovauthor=R.+C.+Stevens&amp;title=Allosteric+sodium+in+class+A+GPCR+signaling&amp;doi=10.1016%2Fj.tibs.2014.03.002" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric sodium in class A GPCR signaling</span></div><div class="casAuthors">Katritch, Vsevolod; Fenalti, Gustavo; Abola, Enrique E.; Roth, Bryan L.; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">233-244</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Despite their functional and structural diversity, G-protein-coupled receptors (GPCRs) share a common mechanism of signal transduction via conformational changes in the seven-transmembrane (7TM) helical domain.  New major insights into this mechanism come from the recent crystallog. discoveries of a partially hydrated sodium ion that is specifically bound in the middle of the 7TM bundle of multiple class A GPCRs.  This review discusses the remarkable structural conservation and distinct features of the Na+ pocket in this most populous GPCR class, as well as the conformational collapse of the pocket upon receptor activation.  New insights help to explain allosteric effects of sodium on GPCR agonist binding and activation, and sodium's role as a potential co-factor in class A GPCR function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_QJ04qVNc3LVg90H21EOLACvtfcHk0lh8C5u6VfDuCA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXmsFCqs7k%253D&amp;md5=c217e7eb69c93b1c76559cc587b318b5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2014.03.002&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2014.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DAbola%26aufirst%3DE.%2BE.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DAllosteric%2520sodium%2520in%2520class%2520A%2520GPCR%2520signaling%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2014%26volume%3D39%26issue%3D5%26spage%3D233%26doi%3D10.1016%2Fj.tibs.2014.03.002" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez-de-Teran, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massink, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ijzerman, A. P.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">The role of a sodium ion binding site in the allosteric modulation of the A(2A) adenosine G protein-coupled receptor</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2175</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.str.2013.09.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.str.2013.09.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=24210756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslCls7nP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2013&amp;pages=2175&amp;issue=12&amp;author=H.+Gutierrez-de-Teranauthor=A.+Massinkauthor=D.+Rodriguezauthor=W.+Liuauthor=G.+W.+Hanauthor=J.+S.+Josephauthor=I.+Katritchauthor=L.+H.+Heitmanauthor=L.+Xiaauthor=A.+P.+Ijzerman&amp;title=The+role+of+a+sodium+ion+binding+site+in+the+allosteric+modulation+of+the+A%282A%29+adenosine+G+protein-coupled+receptor&amp;doi=10.1016%2Fj.str.2013.09.020" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of a Sodium Ion Binding Site in the Allosteric Modulation of the A2A Adenosine G Protein-Coupled Receptor</span></div><div class="casAuthors">Gutierrez-de-Teran, Hugo; Massink, Arnault; Rodriguez, David; Liu, Wei; Han, Gye Won; Joseph, Jeremiah S.; Katritch, Ilia; Heitman, Laura H.; Xia, Lizi; IJzerman, Adriaan P.; Cherezov, Vadim; Katritch, Vsevolod; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2175-2185</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The function of G protein-coupled receptors (GPCRs) can be modulated by a no. of endogenous allosteric mols.  In this study, we used mol. dynamics, radioligand binding, and thermostability expts. to elucidate the role of the recently discovered sodium ion binding site in the allosteric modulation of the human A2A adenosine receptor, conserved among class A GPCRs.  While the binding of antagonists and sodium ions to the receptor was noncompetitive in nature, the binding of agonists and sodium ions appears to require mutually exclusive conformational states of the receptor.  Amiloride analogs can also bind to the sodium binding pocket, showing distinct patterns of agonist and antagonist modulation.  These findings suggest that physiol. concns. of sodium ions affect functionally relevant conformational states of GPCRs and can help to design novel synthetic allosteric modulators or bitopic ligands exploiting the sodium ion binding pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq52sWIHLurqLVg90H21EOLACvtfcHk0lh8C5u6VfDuCA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhslCls7nP&amp;md5=9ad6e9fc35031b50f8aea83e65d6257f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.09.020&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.09.020%26sid%3Dliteratum%253Aachs%26aulast%3DGutierrez-de-Teran%26aufirst%3DH.%26aulast%3DMassink%26aufirst%3DA.%26aulast%3DRodriguez%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DJoseph%26aufirst%3DJ.%2BS.%26aulast%3DKatritch%26aufirst%3DI.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26atitle%3DThe%2520role%2520of%2520a%2520sodium%2520ion%2520binding%2520site%2520in%2520the%2520allosteric%2520modulation%2520of%2520the%2520A%25282A%2529%2520adenosine%2520G%2520protein-coupled%2520receptor%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26issue%3D12%26spage%3D2175%26doi%3D10.1016%2Fj.str.2013.09.020" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller-Gallacher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nehme, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warne, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schertler, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, C. G.</span></span> <span> </span><span class="NLM_article-title">The 2.1 A resolution structure of cyanopindolol-bound beta1-adrenoceptor identifies an intramembrane Na+ ion that stabilises the ligand-free receptor</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e92727</span>, <span class="refDoi">&nbsp;DOI: 10.1371/journal.pone.0092727</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1371%2Fjournal.pone.0092727" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=24663151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVWnsrvJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2014&amp;pages=e92727&amp;issue=3&amp;author=J.+L.+Miller-Gallacherauthor=R.+Nehmeauthor=T.+Warneauthor=P.+C.+Edwardsauthor=G.+F.+Schertlerauthor=A.+G.+Leslieauthor=C.+G.+Tate&amp;title=The+2.1+A+resolution+structure+of+cyanopindolol-bound+beta1-adrenoceptor+identifies+an+intramembrane+Na%2B+ion+that+stabilises+the+ligand-free+receptor&amp;doi=10.1371%2Fjournal.pone.0092727" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The 2.1 Ã resolution structure of cyanopindolol-bound Î²1-adrenoceptor identifies an intramembrane Na+ ion that stabilizes the ligand-free receptor</span></div><div class="casAuthors">Miller-Gallacher, Jennifer L.; Nehme, Rony; Warne, Tony; Edwards, Patricia C.; Schertler, Gebhard F. X.; Leslie, Andrew G. W.; Tate, Christopher G.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e92727/1-e92727/9, 9 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The Î²1-adrenoceptor (Î²1AR) is a G protein-coupled receptor (GPCR) that is activated by the endogenous agonists adrenaline and noradrenaline.  We have detd. the structure of an ultra-thermostable Î²1AR mutant bound to the weak partial agonist cyanopindolol to 2.1 Ã resoln.  High-quality crystals (100 Î¼m plates) were grown in lipidic cubic phase without the assistance of a T4 lysozyme or BRIL fusion in cytoplasmic loop 3, which is commonly employed for GPCR crystn.  An intramembrane Na+ ion was identified coordinated to Asp872.50, Ser1283.39 and 3 water mols., which is part of a more extensive network of water mols. in a cavity formed between transmembrane helixes 1, 2, 3, 6 and 7.  Remarkably, this water network and Na+ ion is highly conserved between Î²1AR and the adenosine A2A receptor (rmsd of 0.3 Ã), despite an overall rmsd of 2.4 Ã for all CÎ± atoms and only 23% amino acid identity in the transmembrane regions.  The affinity of agonist binding and nanobody Nb80 binding to Î²1AR is unaffected by Na+ ions, but the stability of the receptor is decreased by 7.5Â°C in the absence of Na+.  Mutation of amino acid side chains that are involved in the co-ordination of either Na+ or water mols. in the network decreases the stability of Î²1AR by 5-10Â°C.  The data suggest that the intramembrane Na+ and assocd. water network stabilize the ligand-free state of Î²1AR, but still permits the receptor to form the activated state which involves the collapse of the Na+ binding pocket on agonist binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohMXYcl2xn6LVg90H21EOLACvtfcHk0lgud-ZdYANm9g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXhsVWnsrvJ&amp;md5=f3f2d68c124755ac1c91e52efa5575b2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0092727&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0092727%26sid%3Dliteratum%253Aachs%26aulast%3DMiller-Gallacher%26aufirst%3DJ.%2BL.%26aulast%3DNehme%26aufirst%3DR.%26aulast%3DWarne%26aufirst%3DT.%26aulast%3DEdwards%26aufirst%3DP.%2BC.%26aulast%3DSchertler%26aufirst%3DG.%2BF.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DTate%26aufirst%3DC.%2BG.%26atitle%3DThe%25202.1%2520A%2520resolution%2520structure%2520of%2520cyanopindolol-bound%2520beta1-adrenoceptor%2520identifies%2520an%2520intramembrane%2520Na%252B%2520ion%2520that%2520stabilises%2520the%2520ligand-free%2520receptor%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26issue%3D3%26spage%3De92727%26doi%3D10.1371%2Fjournal.pone.0092727" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betz, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatakrishnan, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">D4 dopamine receptor high-resolution structures enable the discovery of selective agonists</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span> (<span class="NLM_issue">6361</span>),  <span class="NLM_fpage">381</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.aan5468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1126%2Fscience.aan5468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29051383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Kns73I" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=358&amp;publication_year=2017&amp;pages=381&amp;issue=6361&amp;author=S.+Wangauthor=D.+Wackerauthor=A.+Levitauthor=T.+Cheauthor=R.+M.+Betzauthor=J.+D.+McCorvyauthor=A.+J.+Venkatakrishnanauthor=X.+P.+Huangauthor=R.+O.+Drorauthor=B.+K.+Shoichet&amp;title=D4+dopamine+receptor+high-resolution+structures+enable+the+discovery+of+selective+agonists&amp;doi=10.1126%2Fscience.aan5468" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">D4 dopamine receptor high-resolution structures enable the discovery of selective agonists</span></div><div class="casAuthors">Wang, Sheng; Wacker, Daniel; Levit, Anat; Che, Tao; Betz, Robin M.; McCorvy, John D.; Venkatakrishnan, A. J.; Huang, Xi-Ping; Dror, Ron O.; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">6361</span>),
    <span class="NLM_cas:pages">381-386</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Dopamine receptors are implicated in the pathogenesis and treatment of nearly every neuropsychiatric disorder.  Although thousands of drugs interact with these receptors, our mol. understanding of dopaminergic drug selectivity and design remains clouded.  To illuminate dopamine receptor structure, function, and ligand recognition, we detd. crystal structures of the D4 dopamine receptor in its inactive state bound to the antipsychotic drug nemonapride, with resolns. up to 1.95 angstroms.  These structures suggest a mechanism for the control of constitutive signaling, and their unusually high resoln. enabled a structure-based campaign for new agonists of the D4 dopamine receptor.  The ability to efficiently exploit structure for specific probe discovery-rapidly moving from elucidating receptor structure to discovering previously unrecognized, selective agonists-testifies to the power of structure-based approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8NDKhkv_ILbVg90H21EOLACvtfcHk0lgud-ZdYANm9g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Kns73I&amp;md5=0bbb93e6bcb33577f9d9b7459273d896</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.aan5468&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aan5468%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DLevit%26aufirst%3DA.%26aulast%3DChe%26aufirst%3DT.%26aulast%3DBetz%26aufirst%3DR.%2BM.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DVenkatakrishnan%26aufirst%3DA.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DD4%2520dopamine%2520receptor%2520high-resolution%2520structures%2520enable%2520the%2520discovery%2520of%2520selective%2520agonists%26jtitle%3DScience%26date%3D2017%26volume%3D358%26issue%3D6361%26spage%3D381%26doi%3D10.1126%2Fscience.aan5468" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weinert, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olieric, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozerov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, K.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Serial millisecond crystallography for routine room-temperature structure determination at synchrotrons</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">542</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41467-017-00630-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fs41467-017-00630-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=28912485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A280%3ADC%252BC1cboslKltg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2017&amp;pages=542&amp;issue=1&amp;author=T.+Weinertauthor=N.+Oliericauthor=R.+Chengauthor=S.+Brunleauthor=D.+Jamesauthor=D.+Ozerovauthor=D.+Gashiauthor=L.+Veraauthor=M.+Marshauthor=K.+Jaeger&amp;title=Serial+millisecond+crystallography+for+routine+room-temperature+structure+determination+at+synchrotrons&amp;doi=10.1038%2Fs41467-017-00630-4" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Serial millisecond crystallography for routine room-temperature structure determination at synchrotrons</span></div><div class="casAuthors">Weinert Tobias; Olieric Natacha; James Daniel; Gashi Dardan; Jaeger Kathrin; Skopintsev Petr; Prota Andrea E; Panneels Valerie; Nogly Przemyslaw; Schertler Gebhard; Steinmetz Michel O; Standfuss Jorg; Cheng Robert; Hennig Michael; Brunle Steffen; Ermler Ulrich; Ozerov Dmitry; Gashi Dardan; Vera Laura; Marsh May; Dworkowski Florian; Panepucci Ezequiel; Basu Shibom; Wang Meitian; Dore Andrew S; Geng Tian; Cooke Robert M; Liang Mengning; Schertler Gebhard; Steinmetz Michel O</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">542</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Historically, room-temperature structure determination was succeeded by cryo-crystallography to mitigate radiation damage.  Here, we demonstrate that serial millisecond crystallography at a synchrotron beamline equipped with high-viscosity injector and high frame-rate detector allows typical crystallographic experiments to be performed at room-temperature.  Using a crystal scanning approach, we determine the high-resolution structure of the radiation sensitive molybdenum storage protein, demonstrate soaking of the drug colchicine into tubulin and native sulfur phasing of the human G protein-coupled adenosine receptor.  Serial crystallographic data for molecular replacement already converges in 1,000-10,000 diffraction patterns, which we collected in 3 to maximally 82 minutes.  Compared with serial data we collected at a free-electron laser, the synchrotron data are of slightly lower resolution, however fewer diffraction patterns are needed for de novo phasing.  Overall, the data we collected by room-temperature serial crystallography are of comparable quality to cryo-crystallographic data and can be routinely collected at synchrotrons.Serial crystallography was developed for protein crystal data collection with X-ray free-electron lasers.  Here the authors present several examples which show that serial crystallography using high-viscosity injectors can also be routinely employed for room-temperature data collection at synchrotrons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2XuDrQ2dPti6Ufuk_1xoTfW6udTcc2eYRz2uWSl99Trntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC1cboslKltg%253D%253D&amp;md5=2fc0843781c5237cc59b3b64fffc4106</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00630-4&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00630-4%26sid%3Dliteratum%253Aachs%26aulast%3DWeinert%26aufirst%3DT.%26aulast%3DOlieric%26aufirst%3DN.%26aulast%3DCheng%26aufirst%3DR.%26aulast%3DBrunle%26aufirst%3DS.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DOzerov%26aufirst%3DD.%26aulast%3DGashi%26aufirst%3DD.%26aulast%3DVera%26aufirst%3DL.%26aulast%3DMarsh%26aufirst%3DM.%26aulast%3DJaeger%26aufirst%3DK.%26atitle%3DSerial%2520millisecond%2520crystallography%2520for%2520routine%2520room-temperature%2520structure%2520determination%2520at%2520synchrotrons%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26issue%3D1%26spage%3D542%26doi%3D10.1038%2Fs41467-017-00630-4" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuybeda, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Waal, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Eps, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutka, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartesaghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erramilli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morizumi, T.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Cryo-EM structure of human rhodopsin bound to an inhibitory G protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>558</i></span> (<span class="NLM_issue">7711</span>),  <span class="NLM_fpage">553</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41586-018-0215-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fs41586-018-0215-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29899450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFeqtbjF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=558&amp;publication_year=2018&amp;pages=553&amp;issue=7711&amp;author=Y.+Kangauthor=O.+Kuybedaauthor=P.+W.+de+Waalauthor=S.+Mukherjeeauthor=N.+Van+Epsauthor=P.+Dutkaauthor=X.+E.+Zhouauthor=A.+Bartesaghiauthor=S.+Erramilliauthor=T.+Morizumi&amp;title=Cryo-EM+structure+of+human+rhodopsin+bound+to+an+inhibitory+G+protein&amp;doi=10.1038%2Fs41586-018-0215-y" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM structure of human rhodopsin bound to an inhibitory G protein</span></div><div class="casAuthors">Kang, Yanyong; Kuybeda, Oleg; de Waal, Parker W.; Mukherjee, Somnath; Van Eps, Ned; Dutka, Przemyslaw; Zhou, X. Edward; Bartesaghi, Alberto; Erramilli, Satchal; Morizumi, Takefumi; Gu, Xin; Yin, Yanting; Liu, Ping; Jiang, Yi; Meng, Xing; Zhao, Gongpu; Melcher, Karsten; Ernst, Oliver P.; Kossiakoff, Anthony A.; Subramaniam, Sriram; Xu, H. Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">558</span>
        (<span class="NLM_cas:issue">7711</span>),
    <span class="NLM_cas:pages">553-558</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) comprise the largest family of mammalian transmembrane receptors.  They mediate numerous cellular pathways by coupling with downstream signalling transducers, including the heterotrimeric G proteins Gs (stimulatory) and Gi (inhibitory) and several arrestin proteins.  The structural mechanisms that define how G-protein-coupled receptors selectively couple to a specific type of G protein or arrestin remain unknown.  Here, using cryo-electron microscopy, we show that the major interactions between activated rhodopsin and Gi are mediated by the C-terminal helix of the Gi Î±-subunit, which is wedged into the cytoplasmic cavity of the transmembrane helix bundle and directly contacts the amino terminus of helix 8 of rhodopsin.  Structural comparisons of inactive, Gi-bound and arrestin-bound forms of rhodopsin with inactive and Gs-bound forms of the Î²2-adrenergic receptor provide a foundation to understand the unique structural signatures that are assocd. with the recognition of Gs, Gi and arrestin by activated G-protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo98ChG6dDQrVg90H21EOLACvtfcHk0lh1Y7xfBnuxUQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhtFeqtbjF&amp;md5=e6d8e79977e0420442fa16ec0c53c692</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0215-y&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0215-y%26sid%3Dliteratum%253Aachs%26aulast%3DKang%26aufirst%3DY.%26aulast%3DKuybeda%26aufirst%3DO.%26aulast%3Dde%2BWaal%26aufirst%3DP.%2BW.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DVan%2BEps%26aufirst%3DN.%26aulast%3DDutka%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DX.%2BE.%26aulast%3DBartesaghi%26aufirst%3DA.%26aulast%3DErramilli%26aufirst%3DS.%26aulast%3DMorizumi%26aufirst%3DT.%26atitle%3DCryo-EM%2520structure%2520of%2520human%2520rhodopsin%2520bound%2520to%2520an%2520inhibitory%2520G%2520protein%26jtitle%3DNature%26date%3D2018%26volume%3D558%26issue%3D7711%26spage%3D553%26doi%3D10.1038%2Fs41586-018-0215-y" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velgy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggin, P. C.</span></span> <span> </span><span class="NLM_article-title">GPCRs: What Can We Learn from Molecular Dynamics Simulations?</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1705</i></span>,  <span class="NLM_fpage">133</span>, <span class="refDoi">&nbsp;DOI: 10.1007/978-1-4939-7465-8_6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1007%2F978-1-4939-7465-8_6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29188561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFCnsrzL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1705&amp;publication_year=2018&amp;pages=133&amp;author=N.+Velgyauthor=G.+Hedgerauthor=P.+C.+Biggin&amp;title=GPCRs%3A+What+Can+We+Learn+from+Molecular+Dynamics+Simulations%3F&amp;doi=10.1007%2F978-1-4939-7465-8_6" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">GPCRs: what can we learn from molecular dynamics simulations?</span></div><div class="casAuthors">Velgy, Naushad; Hedger, George; Biggin, Philip C.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1705</span>
        (<span class="NLM_cas:issue">Computational Methods for GPCR Drug Discovery</span>),
    <span class="NLM_cas:pages">133-158</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Advances in the structural biol. of G-protein Coupled Receptors have resulted in a significant step forward in our understanding of how this important class of drug targets function at the mol. level.  However, it has also become apparent that they are very dynamic mols., and moreover, that the underlying dynamics is crucial in shaping the response to different ligands.  Mol. dynamics simulations can provide unique insight into the dynamic properties of GPCRs in a way that is complementary to many exptl. approaches.  In this chapter, we describe progress in three distinct areas that are particularly difficult to study with other techniques: at. level investigation of the conformational changes that occur when moving between the various states that GPCRs can exist in, the pathways that ligands adopt during binding/unbinding events and finally, the influence of lipids on the conformational dynamics of GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFBdYfV8QTCrVg90H21EOLACvtfcHk0lgtXwtrcE3DDw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhvFCnsrzL&amp;md5=6271b8fd13228616e0f34eab451e9f4a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-7465-8_6&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-7465-8_6%26sid%3Dliteratum%253Aachs%26aulast%3DVelgy%26aufirst%3DN.%26aulast%3DHedger%26aufirst%3DG.%26aulast%3DBiggin%26aufirst%3DP.%2BC.%26atitle%3DGPCRs%253A%2520What%2520Can%2520We%2520Learn%2520from%2520Molecular%2520Dynamics%2520Simulations%253F%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2018%26volume%3D1705%26spage%3D133%26doi%3D10.1007%2F978-1-4939-7465-8_6" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarrasch, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun Kobilka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Cryo-EM structure of the activated GLP-1 receptor in complex with a G protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>546</i></span> (<span class="NLM_issue">7657</span>),  <span class="NLM_fpage">248</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature22394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fnature22394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=28538729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosVagtLo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=546&amp;publication_year=2017&amp;pages=248&amp;issue=7657&amp;author=Y.+Zhangauthor=B.+Sunauthor=D.+Fengauthor=H.+Huauthor=M.+Chuauthor=Q.+Quauthor=J.+T.+Tarraschauthor=S.+Liauthor=T.+Sun+Kobilkaauthor=B.+K.+Kobilka&amp;title=Cryo-EM+structure+of+the+activated+GLP-1+receptor+in+complex+with+a+G+protein&amp;doi=10.1038%2Fnature22394" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM structure of the activated GLP-1 receptor in complex with a G protein</span></div><div class="casAuthors">Zhang, Yan; Sun, Bingfa; Feng, Dan; Hu, Hongli; Chu, Matthew; Qu, Qianhui; Tarrasch, Jeffrey T.; Li, Shane; Sun Kobilka, Tong; Kobilka, Brian K.; Skiniotis, Georgios</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">546</span>
        (<span class="NLM_cas:issue">7657</span>),
    <span class="NLM_cas:pages">248-253</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Glucagon-like peptide 1 (GLP-1) is a hormone with essential roles in regulating insulin secretion, carbohydrate metab. and appetite.  GLP-1 effects are mediated through binding to the GLP-1 receptor (GLP-1R), a class B G-protein-coupled receptor (GPCR) that signals primarily through the stimulatory G protein Gs.  Class B GPCRs are important therapeutic targets; however, our understanding of their mechanism of action is limited by the lack of structural information on activated and full-length receptors.  Here we report the cryo-electron microscopy structure of the peptide-activated GLP-1R-Gs complex at near at. resoln.  The peptide is clasped between the N-terminal domain and the transmembrane core of the receptor, and further stabilized by extracellular loops.  Conformational changes in the transmembrane domain result in a sharp kink in the middle of transmembrane helix 6, which pivots its intracellular half outward to accommodate the Î±5-helix of the Ras-like domain of Gs.  These results provide a structural framework for understanding class B GPCR activation through hormone binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmDRFGRwMs1bVg90H21EOLACvtfcHk0lgtXwtrcE3DDw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXosVagtLo%253D&amp;md5=5e7ebd94012ba47ecc299ad0502975de</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature22394&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22394%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DChu%26aufirst%3DM.%26aulast%3DQu%26aufirst%3DQ.%26aulast%3DTarrasch%26aufirst%3DJ.%2BT.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSun%2BKobilka%26aufirst%3DT.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DCryo-EM%2520structure%2520of%2520the%2520activated%2520GLP-1%2520receptor%2520in%2520complex%2520with%2520a%2520G%2520protein%26jtitle%3DNature%26date%3D2017%26volume%3D546%26issue%3D7657%26spage%3D248%26doi%3D10.1038%2Fnature22394" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lodowski, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elstner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegemann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kandori, H.</span></span> <span> </span><span class="NLM_article-title">Microbial and animal rhodopsins: structures, functions, and molecular mechanisms</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">126</span>, <span class="refDoi">&nbsp;DOI: 10.1021/cr4003769</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr4003769" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOnt7jN" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=114&amp;publication_year=2014&amp;pages=126&amp;issue=1&amp;author=O.+P.+Ernstauthor=D.+T.+Lodowskiauthor=M.+Elstnerauthor=P.+Hegemannauthor=L.+S.+Brownauthor=H.+Kandori&amp;title=Microbial+and+animal+rhodopsins%3A+structures%2C+functions%2C+and+molecular+mechanisms&amp;doi=10.1021%2Fcr4003769" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Microbial and animal rhodopsins: Structures, functions, and molecular mechanisms</span></div><div class="casAuthors">Ernst, Oliver P.; Lodowski, David T.; Elstner, Marcus; Hegemann, Peter; Brown, Leonid S.; Kandori, Hideki</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">126-163</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The authors provide mechanistic insights from biophys. and structural studies into the function of microbial and animal rhodopsins, with the latter as representatives of G protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5hz035KTkrrVg90H21EOLACvtfcHk0lh-L_zMnDPFRQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOnt7jN&amp;md5=74e6baeb254a11b90c2a2a2aab6e942a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fcr4003769&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr4003769%26sid%3Dliteratum%253Aachs%26aulast%3DErnst%26aufirst%3DO.%2BP.%26aulast%3DLodowski%26aufirst%3DD.%2BT.%26aulast%3DElstner%26aufirst%3DM.%26aulast%3DHegemann%26aufirst%3DP.%26aulast%3DBrown%26aufirst%3DL.%2BS.%26aulast%3DKandori%26aufirst%3DH.%26atitle%3DMicrobial%2520and%2520animal%2520rhodopsins%253A%2520structures%252C%2520functions%252C%2520and%2520molecular%2520mechanisms%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26issue%3D1%26spage%3D126%26doi%3D10.1021%2Fcr4003769" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chubukov, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Structural Basis for Allosteric Regulation of GPCRs by Sodium Ions</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>337</i></span> (<span class="NLM_issue">6091</span>),  <span class="NLM_fpage">232</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.1219218</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1126%2Fscience.1219218" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=22798613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVKqtbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=337&amp;publication_year=2012&amp;pages=232&amp;issue=6091&amp;author=W.+Liuauthor=E.+Chunauthor=A.+A.+Thompsonauthor=P.+Chubukovauthor=F.+Xuauthor=V.+Katritchauthor=G.+W.+Hanauthor=C.+B.+Rothauthor=L.+H.+Heitmanauthor=A.+P.+IJzerman&amp;title=Structural+Basis+for+Allosteric+Regulation+of+GPCRs+by+Sodium+Ions&amp;doi=10.1126%2Fscience.1219218" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Allosteric Regulation of GPCRs by Sodium Ions</span></div><div class="casAuthors">Liu, Wei; Chun, Eugene; Thompson, Aaron A.; Chubukov, Pavel; Xu, Fei; Katritch, Vsevolod; Han, Gye Won; Roth, Christopher B.; Heitman, Laura H.; IJzerman, Adriaan P.; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">337</span>
        (<span class="NLM_cas:issue">6091</span>),
    <span class="NLM_cas:pages">232-236</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Pharmacol. responses of G protein-coupled receptors (GPCRs) can be fine-tuned by allosteric modulators.  Structural studies of such effects have been limited due to the medium resoln. of GPCR structures.  We reengineered the human A2A adenosine receptor by replacing its third intracellular loop with apocytochrome b562RIL and solved the structure at 1.8 angstrom resoln.  The high-resoln. structure allowed us to identify 57 ordered water mols. inside the receptor comprising three major clusters.  The central cluster harbors a putative sodium ion bound to the highly conserved aspartate residue Asp2.50.  Addnl., two cholesterols stabilize the conformation of helix VI, and one of 23 ordered lipids intercalates inside the ligand-binding pocket.  These high-resoln. details shed light on the potential role of structured water mols., sodium ions, and lipids/cholesterol in GPCR stabilization and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvcjQimGnIMLVg90H21EOLACvtfcHk0lh-L_zMnDPFRQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XpvVKqtbs%253D&amp;md5=782b2235b76ef77b11b6456f8e4485e1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.1219218&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1219218%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DChun%26aufirst%3DE.%26aulast%3DThompson%26aufirst%3DA.%2BA.%26aulast%3DChubukov%26aufirst%3DP.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DRoth%26aufirst%3DC.%2BB.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26atitle%3DStructural%2520Basis%2520for%2520Allosteric%2520Regulation%2520of%2520GPCRs%2520by%2520Sodium%2520Ions%26jtitle%3DScience%26date%3D2012%26volume%3D337%26issue%3D6091%26spage%3D232%26doi%3D10.1126%2Fscience.1219218" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fenalti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giguere, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Molecular control of delta-opioid receptor signalling</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>506</i></span>,  <span class="NLM_fpage">191</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature12944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fnature12944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=24413399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVaktro%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=506&amp;publication_year=2014&amp;pages=191&amp;author=G.+Fenaltiauthor=P.+M.+Giguereauthor=V.+Katritchauthor=X.+P.+Huangauthor=A.+A.+Thompsonauthor=V.+Cherezovauthor=B.+L.+Rothauthor=R.+C.+Stevens&amp;title=Molecular+control+of+delta-opioid+receptor+signalling&amp;doi=10.1038%2Fnature12944" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular control of Î´-opioid receptor signalling</span></div><div class="casAuthors">Fenalti, Gustavo; Giguere, Patrick M.; Katritch, Vsevolod; Huang, Xi-Ping; Thompson, Aaron A.; Cherezov, Vadim; Roth, Bryan L.; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">506</span>
        (<span class="NLM_cas:issue">7487</span>),
    <span class="NLM_cas:pages">191-196</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Opioids represent widely prescribed and abused medications, although their signal transduction mechanisms are not well understood.  Here we present the 1.8 Ã high-resoln. crystal structure of the human Î´-opioid receptor (Î´-OR), revealing the presence and fundamental role of a sodium ion in mediating allosteric control of receptor functional selectivity and constitutive activity.  The distinctive Î´-OR sodium ion site architecture is centrally located in a polar interaction network in the seven-transmembrane bundle core, with the sodium ion stabilizing a reduced agonist affinity state, and thereby modulating signal transduction.  Site-directed mutagenesis and functional studies reveal that changing the allosteric sodium site residue Asn 131 to an alanine or a valine augments constitutive Î²-arrestin-mediated signaling.  Asp95Ala, Asn310Ala and Asn314Ala mutations transform classical Î´-opioid antagonists such as naltrindole into potent Î²-arrestin-biased agonists.  The data establish the mol. basis for allosteric sodium ion control in opioid signaling, revealing that sodium-coordinating residues act as 'efficacy switches' at a prototypic G-protein-coupled receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUFRulZU1cfrVg90H21EOLACvtfcHk0lh-L_zMnDPFRQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXisVaktro%253D&amp;md5=88159956324bb2858286a4100a9cb8bf</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature12944&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12944%26sid%3Dliteratum%253Aachs%26aulast%3DFenalti%26aufirst%3DG.%26aulast%3DGiguere%26aufirst%3DP.%2BM.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DThompson%26aufirst%3DA.%2BA.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DMolecular%2520control%2520of%2520delta-opioid%2520receptor%2520signalling%26jtitle%3DNature%26date%3D2014%26volume%3D506%26spage%3D191%26doi%3D10.1038%2Fnature12944" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H.</span></span> <span> </span><span class="NLM_article-title">W246<sup>6.48</sup> Opens a Gate for a Continuous Intrinsic Water Pathway during Activation of the Adenosine A Receptor</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">556</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.201409679</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1002%2Fanie.201409679" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=54&amp;publication_year=2014&amp;pages=556&amp;issue=2&amp;author=S.+Yuanauthor=Z.+Huauthor=S.+Filipekauthor=H.+Vogel&amp;title=W2466.48+Opens+a+Gate+for+a+Continuous+Intrinsic+Water+Pathway+during+Activation+of+the+Adenosine+A+Receptor&amp;doi=10.1002%2Fanie.201409679" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fanie.201409679&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201409679%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DFilipek%26aufirst%3DS.%26aulast%3DVogel%26aufirst%3DH.%26atitle%3DW2466.48%2520Opens%2520a%2520Gate%2520for%2520a%2520Continuous%2520Intrinsic%2520Water%2520Pathway%2520during%2520Activation%2520of%2520the%2520Adenosine%2520A%2520Receptor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D54%26issue%3D2%26spage%3D556%26doi%3D10.1002%2Fanie.201409679" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ericksen, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schetz, J. A.</span></span> <span> </span><span class="NLM_article-title">Ligand selectivity of D2 dopamine receptors is modulated by changes in local dynamics produced by sodium binding</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>328</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">40</span>, <span class="refDoi">&nbsp;DOI: 10.1124/jpet.108.141531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1124%2Fjpet.108.141531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=18849360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFymtw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=328&amp;publication_year=2009&amp;pages=40&amp;issue=1&amp;author=S.+S.+Ericksenauthor=D.+F.+Cummingsauthor=H.+Weinsteinauthor=J.+A.+Schetz&amp;title=Ligand+selectivity+of+D2+dopamine+receptors+is+modulated+by+changes+in+local+dynamics+produced+by+sodium+binding&amp;doi=10.1124%2Fjpet.108.141531" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand selectivity of D2 dopamine receptors is modulated by changes in local dynamics produced by sodium binding</span></div><div class="casAuthors">Ericksen, Spencer S.; Cummings, David F.; Weinstein, Harel; Schetz, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">328</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-54</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">We have uncovered a significant allosteric response of the D2 dopamine receptor to physiol. relevant concns. of sodium (140 mM), characterized by a sodium-enhanced binding affinity for a D4-selective class of agonists and antagonists.  This enhancement is significantly more pronounced in a D2-V2.61(91)F mutant and cannot be mimicked by an equiv. concn. of the sodium replacement cation N-methyl-D-glucamine.  This phenomenon was explored computationally at the mol. level by analyzing the effect of sodium binding on the dynamic properties of D2 receptor model constructs.  Normal mode anal. identified one mode (M19), which is involved in the open/closed motions of the binding cleft as being particularly sensitive to the sodium effect.  To examine the consequences for D2 receptor ligand recognition, one of the ligands, L-745,870 [3-{[4-(4-chlorophenyl) piperazin-1-yl]methyl}-1H-pyrrolo[2,3-b]pyridine or CPPMA, chlorophenylpiperazinyl methylazaindole], was docked into conformers along the M19 trajectory.  Structurally and pharmacol. well established ligand-receptor interactions, including the ionic interaction with D3.32(114) and interactions between the ligand aryl moieties and V2.61(91)F, were achieved only in "open" phase conformers.  The docking of (-)-raclopride [3,5-dichloro-N-(1-ethylpyrrolidin-2-ylmethyl)-2-hydroxy-6-methoxybenzamide] suggests that the same binding cleft changes in response to sodium-binding perturbation account as well for the enhancements in binding affinity for substituted benzamides in the wild-type D2 receptor.  Our findings demonstrate how key interactions can be modulated by occupancy at an allosteric site and are consistent with a mechanism in which sodium binding enhances the affinity of selected ligands through dynamic changes that increase accessibility of substituted benzamides and 1,4-DAP ligands to the orthosteric site and accessibility of 1,4-DAPs to V2.61(91)F.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9peW37CIIRLVg90H21EOLACvtfcHk0ljoJouOFf0JKw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXisFymtw%253D%253D&amp;md5=24298d041694b3fc97bb93a057d53357</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1124%2Fjpet.108.141531&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.108.141531%26sid%3Dliteratum%253Aachs%26aulast%3DEricksen%26aufirst%3DS.%2BS.%26aulast%3DCummings%26aufirst%3DD.%2BF.%26aulast%3DWeinstein%26aufirst%3DH.%26aulast%3DSchetz%26aufirst%3DJ.%2BA.%26atitle%3DLigand%2520selectivity%2520of%2520D2%2520dopamine%2520receptors%2520is%2520modulated%2520by%2520changes%2520in%2520local%2520dynamics%2520produced%2520by%2520sodium%2520binding%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2009%26volume%3D328%26issue%3D1%26spage%3D40%26doi%3D10.1124%2Fjpet.108.141531" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipek, S.</span></span> <span> </span><span class="NLM_article-title">The Role of Water and Sodium Ions in the Activation of the mu-Opioid Receptor</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">10112</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.201302244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1002%2Fanie.201302244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Sku7fK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=2013&amp;pages=10112&amp;issue=38&amp;author=S.+Yuanauthor=H.+Vogelauthor=S.+Filipek&amp;title=The+Role+of+Water+and+Sodium+Ions+in+the+Activation+of+the+mu-Opioid+Receptor&amp;doi=10.1002%2Fanie.201302244" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Water and Sodium Ions in the Activation of the Î¼-Opioid Receptor</span></div><div class="casAuthors">Yuan, Shuguang; Vogel, Horst; Filipek, Slawomir</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">10112-10115</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH &amp; Co. KGaA</span>)
        </div><div class="casAbstract">The activation of G-protein-coupled receptors depends on the presence of water mols. inside the receptor and also on allosteric interactions.  The binding of sodium ions to the allosteric site of the Î¼ opioid receptor was studied by microsecond mol. dynamics simulations and their seemingly contradictory roles in preventing ligand binding and facilitating receptor activation were explained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFR_cJYR4RHbVg90H21EOLACvtfcHk0ljoJouOFf0JKw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXht1Sku7fK&amp;md5=b5b29dbf482e8d309762e536c9fee2e7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fanie.201302244&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201302244%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DS.%26aulast%3DVogel%26aufirst%3DH.%26aulast%3DFilipek%26aufirst%3DS.%26atitle%3DThe%2520Role%2520of%2520Water%2520and%2520Sodium%2520Ions%2520in%2520the%2520Activation%2520of%2520the%2520mu-Opioid%2520Receptor%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26issue%3D38%26spage%3D10112%26doi%3D10.1002%2Fanie.201302244" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golebiowski, J.</span></span> <span> </span><span class="NLM_article-title">Allosteric Na(+)-binding site modulates CXCR4 activation</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">24915</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C8CP04134B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1039%2FC8CP04134B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=30238101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslGns73E" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2018&amp;pages=24915&amp;issue=38&amp;author=X.+Congauthor=J.+Golebiowski&amp;title=Allosteric+Na%28%2B%29-binding+site+modulates+CXCR4+activation&amp;doi=10.1039%2FC8CP04134B" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Na+-binding site modulates CXCR4 activation</span></div><div class="casAuthors">Cong, Xiaojing; Golebiowski, Jerome</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">24915-24920</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) control most cellular communications with the environment and are the largest protein family of drug targets.  As strictly regulated mol. machines, profound comprehension of their activation mechanism is expected to significantly facilitate structure-based drug design.  This study provides atomistic-level description of the activation dynamics of the C-X-C chemokine receptor type 4 (CXCR4), a class A GPCR and important drug target.  Using mol. dynamics and enhanced sampling, we demonstrate how mutations and protonation of conserved residues trigger activation through microswitches at the receptor core, while sodium ion - a known allosteric modulator - inhibits it.  The findings point to a conserved mechanism of activation and the allosteric modulation by sodium in the chemokine receptor family.  From the tech. aspect, the enhanced sampling protocol effectively samples receptor conformational changes toward activation, and differentiates three variants of the receptor by their basal activity.  This work provides structural basis and a powerful in silico tool for CXCR4 agonist design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraoDas3gvNr7Vg90H21EOLACvtfcHk0ljoJouOFf0JKw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhslGns73E&amp;md5=a91f5c365c8001a6abe3fb40e87687c5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1039%2FC8CP04134B&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8CP04134B%26sid%3Dliteratum%253Aachs%26aulast%3DCong%26aufirst%3DX.%26aulast%3DGolebiowski%26aufirst%3DJ.%26atitle%3DAllosteric%2520Na%2528%252B%2529-binding%2520site%2520modulates%2520CXCR4%2520activation%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2018%26volume%3D20%26issue%3D38%26spage%3D24915%26doi%3D10.1039%2FC8CP04134B" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caliman, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCammon, J. A.</span></span> <span> </span><span class="NLM_article-title">Allosteric effects of sodium ion binding on activation of the m3 muscarinic g-protein-coupled receptor</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>108</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1796</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bpj.2015.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.bpj.2015.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=25863070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvVSktLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=108&amp;publication_year=2015&amp;pages=1796&amp;issue=7&amp;author=Y.+Miaoauthor=A.+D.+Calimanauthor=J.+A.+McCammon&amp;title=Allosteric+effects+of+sodium+ion+binding+on+activation+of+the+m3+muscarinic+g-protein-coupled+receptor&amp;doi=10.1016%2Fj.bpj.2015.03.003" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric effects of sodium ion binding on activation of the M3 muscarinic G-protein-coupled receptor</span></div><div class="casAuthors">Miao, Yinglong; Caliman, Alisha D.; McCammon, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1796-1806</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are important membrane proteins that mediate cellular signaling and represent primary targets for about 1/3 of currently marketed drugs.  Recent x-ray crystallog. studies identified distinct conformations of GPCRs in the active and inactive states.  An allosteric Na+ ion was previously found bound to a highly conserved D2.50 residue in inactive GPCRs, whereas the D2.50 allosteric pocket became collapsed in active GPCR structures.  However, the dynamic mechanisms underlying these observations remain elusive.  Here, the authors aimed to understand the mechanistic effects of Na+ binding on dynamic activation of the M3 muscarinic GPCR through long-timescale accelerated mol. dynamics (aMD) simulations.  The results showed that with the D2.50 residue deprotonated, the M3 receptor was bound by an allosteric Na+ ion and confined mostly in the inactive state with remarkably reduced flexibility.  In contrast, the D2.50-protonated receptor did not exhibit Na+ ion binding to the D2.50 allosteric site and sampled a significantly larger conformational space.  The receptor activation was captured and characterized by large-scale structural rearrangements of the transmembrane helixes via dynamic H-bond and salt bridge interactions.  The residue motions were highly correlated during receptor activation.  Further network anal. revealed that the allosteric signaling between residue D2.50 and key residues in the intracellular, extracellular, and orthosteric pockets was significantly weakened upon Na+ ion binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRzke7B8ju_bVg90H21EOLACvtfcHk0lga1rAd2iyI1g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXkvVSktLg%253D&amp;md5=ab64d73ef6816c8ea7000616368bab92</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2015.03.003&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2015.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DMiao%26aufirst%3DY.%26aulast%3DCaliman%26aufirst%3DA.%2BD.%26aulast%3DMcCammon%26aufirst%3DJ.%2BA.%26atitle%3DAllosteric%2520effects%2520of%2520sodium%2520ion%2520binding%2520on%2520activation%2520of%2520the%2520m3%2520muscarinic%2520g-protein-coupled%2520receptor%26jtitle%3DBiophys.%2520J.%26date%3D2015%26volume%3D108%26issue%3D7%26spage%3D1796%26doi%3D10.1016%2Fj.bpj.2015.03.003" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Makman, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvorkin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, P. N.</span></span> <span> </span><span class="NLM_article-title">Sodium ion modulates D2 receptor characteristics of dopamine agonist and antagonist binding sites in striatum and retina</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>79</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">4212</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.79.13.4212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1073%2Fpnas.79.13.4212" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=6213964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADyaL38XlsFKqu7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=79&amp;publication_year=1982&amp;pages=4212&amp;issue=13&amp;author=M.+H.+Makmanauthor=B.+Dvorkinauthor=P.+N.+Klein&amp;title=Sodium+ion+modulates+D2+receptor+characteristics+of+dopamine+agonist+and+antagonist+binding+sites+in+striatum+and+retina&amp;doi=10.1073%2Fpnas.79.13.4212" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Sodium ion modulates D2 receptor characteristics of dopamine agonist and antagonist binding sites in striatum and retina</span></div><div class="casAuthors">Makman, Maynard H.; Dvorkin, B.; Klein, Patrice N.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4212-16</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Na+ influences binding of both dopamine  [51-61-6] agonists and antagonists to D2 receptors in striatum and retina.  Addnl., Na+ markedly potentiates the loss of high-affinity agonist binding due to the GTP analog p[NH]ppG  [34273-04-6].  3H-labeled 2-amino-6,7-dihydroxy-1,2,3,4-tetrahydronaphthalene (ADTN)  [53463-78-8] binds exclusively to an agonist conformation of D2 receptor in both striatum and retina, distinct from the antagonist conformation labeled by 3H-labeled spiroperidol  [749-02-0] or 3H-labeled domperidone  [57808-66-9] in striatum or by [3H]spiroperidol in retina.  Na+ is not required for interaction of [3H]ADTN or antagonist radioligand sites with the selective D2 agonist LY-141865  [80373-22-4], the D2 antagonist domperidone, or nonselective dopamine agonists or antagonists; however, Na+ is necessary for high affinity interaction of those radioligand sites with the D2 antagonists molindone  [7416-34-4] and metoclopramide  [364-62-5].  With Na+ present, striatal sites for [3H]ADTN, [3H]spiroperidol, and [3H]domperidone have similar affinities for antagonists but only [3H]ADTN sites have high affinity for agonists, Na+ further decreases the low afffinity of dopamine agonists for [3H]spiroperidol binding sites.  Also, Na+ enhances [3H]spiroperidol and decreases [3H]ADTN binding.  Na+ alone causes bound [3H]ADTN to dissoc. from at least 30% of striatal and 50% of retinal sites, and with Na+ present [3H]ADTN rapidly dissocs. from the remaining sites upon addn. of p[NH]ppG.  Evidently, D2 receptors in striatum and retina exist in distinct but interconvertible conformational states, with different properties depending on the presence or absence of Na+ and of guanine nucleotide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTisY8zF0jmLVg90H21EOLACvtfcHk0lga1rAd2iyI1g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL38XlsFKqu7c%253D&amp;md5=375a51b02e4bc2eb5fa57883a6f87501</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.79.13.4212&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.79.13.4212%26sid%3Dliteratum%253Aachs%26aulast%3DMakman%26aufirst%3DM.%2BH.%26aulast%3DDvorkin%26aufirst%3DB.%26aulast%3DKlein%26aufirst%3DP.%2BN.%26atitle%3DSodium%2520ion%2520modulates%2520D2%2520receptor%2520characteristics%2520of%2520dopamine%2520agonist%2520and%2520antagonist%2520binding%2520sites%2520in%2520striatum%2520and%2520retina%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1982%26volume%3D79%26issue%3D13%26spage%3D4212%26doi%3D10.1073%2Fpnas.79.13.4212" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeman, P.</span></span> <span> </span><span class="NLM_article-title">Regulation of anterior pituitary D2 dopamine receptors by magnesium and sodium ions</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1842</span>, <span class="refDoi">&nbsp;DOI: 10.1111/j.1471-4159.1985.tb10542.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1111%2Fj.1471-4159.1985.tb10542.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=2932536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADyaL28XhtlWr" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=1985&amp;pages=1842&amp;issue=6&amp;author=M.+Watanabeauthor=S.+R.+Georgeauthor=P.+Seeman&amp;title=Regulation+of+anterior+pituitary+D2+dopamine+receptors+by+magnesium+and+sodium+ions&amp;doi=10.1111%2Fj.1471-4159.1985.tb10542.x" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of anterior pituitary D2 dopamine receptors by magnesium and sodium ions</span></div><div class="casAuthors">Watanabe, Masayuki; George, Susan R.; Seeman, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1842-9</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    </div><div class="casAbstract">At D2 dopamine receptors in porcine anterior pituitary tissue, Mg2+ shifted receptors to agonist high-affinity states, but decreased the affinity of the antagonist [3H]spiperone.  Conversely, Na+ shifted the receptors to agonist low-affinity states, but increased the affinity of [3H]spiperone.  Mg is proposed to stabilize the hormone-receptor-guanine nucleotide regulatory protein complex, whereas Na appears to destabilize this ternary complex.  Thus, Mg and Na appear to mediate their regulatory effects via a common component at the D2 dopamine-receptor ternary complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHpRHyaW1tFrVg90H21EOLACvtfcHk0ljaxGiPNuVUsg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL28XhtlWr&amp;md5=5f8b07cf6e45c1aa216acbba5c10b0c9</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fj.1471-4159.1985.tb10542.x&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1471-4159.1985.tb10542.x%26sid%3Dliteratum%253Aachs%26aulast%3DWatanabe%26aufirst%3DM.%26aulast%3DGeorge%26aufirst%3DS.%2BR.%26aulast%3DSeeman%26aufirst%3DP.%26atitle%3DRegulation%2520of%2520anterior%2520pituitary%2520D2%2520dopamine%2520receptors%2520by%2520magnesium%2520and%2520sodium%2520ions%26jtitle%3DJ.%2520Neurochem.%26date%3D1985%26volume%3D45%26issue%3D6%26spage%3D1842%26doi%3D10.1111%2Fj.1471-4159.1985.tb10542.x" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neale, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sljoka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyda, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichugin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larda, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, O. P.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Mechanistic insights into allosteric regulation of the A2A adenosine G protein-coupled receptor by physiological cations</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1372</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41467-018-03314-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fs41467-018-03314-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29636462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A280%3ADC%252BC1MjgvVWmsw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2018&amp;pages=1372&amp;issue=1&amp;author=L.+Yeauthor=C.+Nealeauthor=A.+Sljokaauthor=B.+Lydaauthor=D.+Pichuginauthor=N.+Tsuchimuraauthor=S.+T.+Lardaauthor=R.+Pomesauthor=A.+E.+Garciaauthor=O.+P.+Ernst&amp;title=Mechanistic+insights+into+allosteric+regulation+of+the+A2A+adenosine+G+protein-coupled+receptor+by+physiological+cations&amp;doi=10.1038%2Fs41467-018-03314-9" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic insights into allosteric regulation of the A2A adenosine G protein-coupled receptor by physiological cations</span></div><div class="casAuthors">Ye Libin; Pichugin Dmitry; Larda Sacha T; Prosser R Scott; Ye Libin; Pichugin Dmitry; Larda Sacha T; Pomes Regis; Ernst Oliver P; Prosser R Scott; Neale Chris; Garcia Angel E; Sljoka Adnan; Tsuchimura Nobuyuki; Lyda Brent; Sunahara Roger K; Pomes Regis; Ernst Oliver P</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1372</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cations play key roles in regulating G-protein-coupled receptors (GPCRs), although their mechanisms are poorly understood.  Here, (19)F NMR is used to delineate the effects of cations on functional states of the adenosine A2A GPCR.  While Na(+) reinforces an inactive ensemble and a partial-agonist stabilized state, Ca(2+) and Mg(2+) shift the equilibrium toward active states.  Positive allosteric effects of divalent cations are more pronounced with agonist and a G-protein-derived peptide.  In cell membranes, divalent cations enhance both the affinity and fraction of the high affinity agonist-bound state.  Molecular dynamics simulations suggest high concentrations of divalent cations bridge specific extracellular acidic residues, bringing TM5 and TM6 together at the extracellular surface and allosterically driving open the G-protein-binding cleft as shown by rigidity-transmission allostery theory.  An understanding of cation allostery should enable the design of allosteric agents and enhance our understanding of GPCR regulation in the cellular milieu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ3cGpsWtuylRe4jJGKT5N0fW6udTcc2ebfV9bFZ2x6A7ntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC1MjgvVWmsw%253D%253D&amp;md5=5e106149aea3e56b585bc5af9a2e1a77</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-03314-9&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-03314-9%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DL.%26aulast%3DNeale%26aufirst%3DC.%26aulast%3DSljoka%26aufirst%3DA.%26aulast%3DLyda%26aufirst%3DB.%26aulast%3DPichugin%26aufirst%3DD.%26aulast%3DTsuchimura%26aufirst%3DN.%26aulast%3DLarda%26aufirst%3DS.%2BT.%26aulast%3DPomes%26aufirst%3DR.%26aulast%3DGarcia%26aufirst%3DA.%2BE.%26aulast%3DErnst%26aufirst%3DO.%2BP.%26atitle%3DMechanistic%2520insights%2520into%2520allosteric%2520regulation%2520of%2520the%2520A2A%2520adenosine%2520G%2520protein-coupled%2520receptor%2520by%2520physiological%2520cations%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26issue%3D1%26spage%3D1372%26doi%3D10.1038%2Fs41467-018-03314-9" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Draper-Joyce, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shonberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopinathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein Herenbrink, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuano, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramyan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thal, D. M.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">The action of a negative allosteric modulator at the dopamine D2 receptor is dependent upon sodium ions</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">1208</span>, <span class="refDoi">&nbsp;DOI: 10.1038/s41598-018-19642-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fs41598-018-19642-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29352161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A280%3ADC%252BC1MvisFerug%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2018&amp;pages=1208&amp;issue=1&amp;author=C.+J.+Draper-Joyceauthor=R.+K.+Vermaauthor=M.+Michinoauthor=J.+Shonbergauthor=A.+Kopinathanauthor=C.+Klein+Herenbrinkauthor=P.+J.+Scammellsauthor=B.+Capuanoauthor=A.+M.+Abramyanauthor=D.+M.+Thal&amp;title=The+action+of+a+negative+allosteric+modulator+at+the+dopamine+D2+receptor+is+dependent+upon+sodium+ions&amp;doi=10.1038%2Fs41598-018-19642-1" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The action of a negative allosteric modulator at the dopamine D2 receptor is dependent upon sodium ions</span></div><div class="casAuthors">Draper-Joyce Christopher J; Klein Herenbrink Carmen; Thal David M; Christopoulos Arthur; Lane J Robert; Verma Ravi Kumar; Michino Mayako; Abramyan Ara M; Shi Lei; Michino Mayako; Shonberg Jeremy; Kopinathan Anitha; Scammells Peter J; Capuano Ben; Javitch Jonathan A; Javitch Jonathan A</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1208</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Sodium ions (Na(+)) allosterically modulate the binding of orthosteric agonists and antagonists to many class A G protein-coupled receptors, including the dopamine D2 receptor (D2R).  Experimental and computational evidences have revealed that this effect is mediated by the binding of Na(+) to a conserved site located beneath the orthosteric binding site (OBS).  SB269652 acts as a negative allosteric modulator (NAM) of the D2R that adopts an extended bitopic pose, in which the tetrahydroisoquinoline moiety interacts with the OBS and the indole-2-carboxamide moiety occupies a secondary binding pocket (SBP).  In this study, we find that the presence of a Na(+) within the conserved Na(+)-binding pocket is required for the action of SB269652.  Using fragments of SB269652 and novel full-length analogues, we show that Na(+) is required for the high affinity binding of the tetrahydroisoquinoline moiety within the OBS, and that the interaction of the indole-2-carboxamide moiety with the SBP determines the degree of Na(+)-sensitivity.  Thus, we extend our understanding of the mode of action of this novel class of NAM by showing it acts synergistically with Na(+) to modulate the binding of orthosteric ligands at the D2R, providing opportunities for fine-tuning of modulatory effects in future allosteric drug design efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTvIHTTptFmHTToTC3aBaesfW6udTcc2eYBypyzANMwerntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC1MvisFerug%253D%253D&amp;md5=7126d1cd2e69be311c508c596d81cc2f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-19642-1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-19642-1%26sid%3Dliteratum%253Aachs%26aulast%3DDraper-Joyce%26aufirst%3DC.%2BJ.%26aulast%3DVerma%26aufirst%3DR.%2BK.%26aulast%3DMichino%26aufirst%3DM.%26aulast%3DShonberg%26aufirst%3DJ.%26aulast%3DKopinathan%26aufirst%3DA.%26aulast%3DKlein%2BHerenbrink%26aufirst%3DC.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26aulast%3DCapuano%26aufirst%3DB.%26aulast%3DAbramyan%26aufirst%3DA.%2BM.%26aulast%3DThal%26aufirst%3DD.%2BM.%26atitle%3DThe%2520action%2520of%2520a%2520negative%2520allosteric%2520modulator%2520at%2520the%2520dopamine%2520D2%2520receptor%2520is%2520dependent%2520upon%2520sodium%2520ions%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26issue%3D1%26spage%3D1208%26doi%3D10.1038%2Fs41598-018-19642-1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chien, E. Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Won Han, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javitch, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Structure of the Human Dopamine D3 Receptor in Complex with a D2/D3 Selective Antagonist</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>330</i></span> (<span class="NLM_issue">6007</span>),  <span class="NLM_fpage">1091</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.1197410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1126%2Fscience.1197410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=21097933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVahurvM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=330&amp;publication_year=2010&amp;pages=1091&amp;issue=6007&amp;author=E.+Y.+T.+Chienauthor=W.+Liuauthor=Q.+Zhaoauthor=V.+Katritchauthor=G.+Won+Hanauthor=M.+A.+Hansonauthor=L.+Shiauthor=A.+H.+Newmanauthor=J.+A.+Javitchauthor=V.+Cherezov&amp;title=Structure+of+the+Human+Dopamine+D3+Receptor+in+Complex+with+a+D2%2FD3+Selective+Antagonist&amp;doi=10.1126%2Fscience.1197410" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Human Dopamine D3 Receptor in Complex with a D2/D3 Selective Antagonist</span></div><div class="casAuthors">Chien, Ellen Y. T.; Liu, Wei; Zhao, Qiang; Katritch, Vsevolod; Won Han, Gye; Hanson, Michael A.; Shi, Lei; Newman, Amy Hauck; Javitch, Jonathan A.; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">330</span>
        (<span class="NLM_cas:issue">6007</span>),
    <span class="NLM_cas:pages">1091-1095</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Dopamine modulates movement, cognition, and emotion through activation of dopamine G protein-coupled receptors in the brain.  The crystal structure of the human dopamine D3 receptor (D3R) in complex with the small mol. D2R/D3R-specific antagonist eticlopride reveals important features of the ligand binding pocket and extracellular loops.  On the intracellular side of the receptor, a locked conformation of the ionic lock and two distinctly different conformations of intracellular loop 2 are obsd.  Docking of R-22, a D3R-selective antagonist, reveals an extracellular extension of the eticlopride binding site that comprises a second binding pocket for the aryl amide of R-22, which differs between the highly homologous D2R and D3R.  This difference provides direction to the design of D3R-selective agents for treating drug abuse and other neuropsychiatric indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOEcfSdjMm-7Vg90H21EOLACvtfcHk0li42TCX9HBHoQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXhsVahurvM&amp;md5=12c6f343daea9fb14ca605a092f28533</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1126%2Fscience.1197410&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1197410%26sid%3Dliteratum%253Aachs%26aulast%3DChien%26aufirst%3DE.%2BY.%2BT.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DWon%2BHan%26aufirst%3DG.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DJavitch%26aufirst%3DJ.%2BA.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DStructure%2520of%2520the%2520Human%2520Dopamine%2520D3%2520Receptor%2520in%2520Complex%2520with%2520a%2520D2%252FD3%2520Selective%2520Antagonist%26jtitle%3DScience%26date%3D2010%26volume%3D330%26issue%3D6007%26spage%3D1091%26doi%3D10.1126%2Fscience.1197410" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Selley, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childers, S. R.</span></span> <span> </span><span class="NLM_article-title">Effects of sodium on agonist efficacy for G-protein activation in Î¼-opioid receptor-transfected CHO cells and rat thalamus</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>130</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">987</span>, <span class="refDoi">&nbsp;DOI: 10.1038/sj.bjp.0703382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fsj.bjp.0703382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10882382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlt1Slurs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=130&amp;publication_year=2000&amp;pages=987&amp;issue=5&amp;author=D.+E.+Selleyauthor=C.-C.+Caoauthor=Q.+Liuauthor=S.+R.+Childers&amp;title=Effects+of+sodium+on+agonist+efficacy+for+G-protein+activation+in+%CE%BC-opioid+receptor-transfected+CHO+cells+and+rat+thalamus&amp;doi=10.1038%2Fsj.bjp.0703382" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of sodium on agonist efficacy for G-protein activation in Î¼-opioid receptor-transfected CHO cells and rat thalamus</span></div><div class="casAuthors">Selley, Dana E.; Cao, Cheng-Chun; Liu, Qixu; Childers, Steven R.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">987-996</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Sodium ions inhibit spontaneous Gi/Go-coupled receptor activity and promote agonist-induced responses in vitro.  The effects of sodium on the relative efficacy of opioid agonists for G-protein activation was measured by guanosine-5'-O-(Î³-35S)-triphosphate ([35S]-GTPÎ³S) binding in membranes from two Î¼-opioid receptor-contg. systems: CHO cells stably transfected with mouse Î¼ receptors (mMOR-CHO cells) and rat thalamus.  NaCl inhibited basal [35S]-GTPÎ³S binding in both systems, and this effect was partially mimicked by KCl.  In mMOR-CHO membranes, net [35S]-GTPÎ³S binding stimulated by partial but not full agonists was inhibited by NaCl with a potency that was inversely proportional to agonist efficacy.  Monovalent cations were required for agonist-stimulated [35S]-GTPÎ³S binding in this system, and increasing NaCl concns. magnified relative efficacy differences among agonists.  In thalamic membranes, which contain a lower receptor:G-protein ratio than mMOR-CHO cells, similar monovalent cation effects were obsd., with two exceptions: [35S]-GTPÎ³S binding stimulated by both full and partial agonists was inhibited by NaCl; and monovalent cations were not required to observe agonist-stimulated [35S]-GTPÎ³S binding.  Basal [35S]-GTPÎ³S binding stimulated by the absence of monovalent cations resembled that of agonist-stimulated binding and was blocked by pretreatment of mMOR-CHO cells with pertussis toxin.  These results indicate that sodium inhibits spontaneous and agonist-occupied Î¼ receptor-mediated G-protein activation in a manner inversely proportional to the efficacy of the agonist, and that spontaneous Î¼ receptor activity and the relative efficacy of partial agonists acting at these receptors are both increased by increases in the stoichiometric ratio of receptors:G-proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEfDtcsF_WjrVg90H21EOLACvtfcHk0lgz9EGmB_IJRg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3cXlt1Slurs%253D&amp;md5=76ef93dd8224c4df5ff888f1fd22fa67</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0703382&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0703382%26sid%3Dliteratum%253Aachs%26aulast%3DSelley%26aufirst%3DD.%2BE.%26aulast%3DCao%26aufirst%3DC.-C.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DChilders%26aufirst%3DS.%2BR.%26atitle%3DEffects%2520of%2520sodium%2520on%2520agonist%2520efficacy%2520for%2520G-protein%2520activation%2520in%2520%25CE%25BC-opioid%2520receptor-transfected%2520CHO%2520cells%2520and%2520rat%2520thalamus%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2000%26volume%3D130%26issue%3D5%26spage%3D987%26doi%3D10.1038%2Fsj.bjp.0703382" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neve, K. A.</span></span> <span> </span><span class="NLM_article-title">Regulation of dopamine D2 receptors by sodium and pH</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">570</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=2017157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADyaK3MXit1WqsbY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=1991&amp;pages=570&amp;issue=4&amp;author=K.+A.+Neve&amp;title=Regulation+of+dopamine+D2+receptors+by+sodium+and+pH" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of dopamine D2 receptors by sodium and pH</span></div><div class="casAuthors">Neve, Kim A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">570-8</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">The role of Na+ and H+ in the regulation of D2 receptor affinity for ligands was studied to det. the mol. mechanisms of this phenomenon.  The potency of substituted benzamide derivs. and agonists at D2 receptors depended on the concn. of Na+ and H+, whereas the potency of other antagonists was relatively unaltered by changes in pH or Na+ concn.  The potency of agonists was generally decreased in the presence of NaCl or lowered pH.  For example, in the absence of Na the affinity of D2 receptors for dopamine was decreased 17-fold by lowering of the pH from 8.0 to pH 6.8.  Addn. of NaCl caused 2-4-fold decreases in affinity for most agonists.  The affinity of the receptors for 2 substituted benzamide derivs., on the other hand, was reduced 6-44-fold by elevated concns. of H+ but was enhanced 7-24-fold in the presence of Na+.  The regulation by H+ of the potency of dopamine was selective for D2 receptors, because binding of dopamine to neostriatal D1 receptors was unaffected by changes in pH.  Decreasing of the pH from 8.0 or 7.3 to 6.8 facilitated the dissocn. of the substituted benzamide ligand [125I]epidepride from D2 receptors but inhibited dissocn. of [3H]spiperone.  Furthermore, the presence of NaCl or lowered pH slowed inactivation of D2 receptors by N-ethylmaleimide.  Apparently, the conformation of D2 receptors is regulated by both Na+ and H+.  The affinity of D2 receptors for agonists and substituted benzamide antagonists varies according to the conformational state of the receptors, whereas other antagonists bind to both forms with approx. equal potency.  Amiloride is a compd. that interacts with many Na-binding macromols.  At equil., amiloride inhibited the binding of [3H]spiperone and [125I]epidepride in a manner suggesting a more complex interaction than simple competitive inhibition.  The rate of dissocn. of both radioligands was enhanced by amiloride, as would be expected for allosteric inhibition of binding.  The sensitivity of D2 receptors to pH, Na, and amiloride may be a reflection of the ability of D2 receptors to modulate Na+/H+ exchange.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBIR1cElUacLVg90H21EOLACvtfcHk0lgeyCB8MFlt2w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3MXit1WqsbY%253D&amp;md5=1bd0b0473860a2fef260d01ae9630cfb</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNeve%26aufirst%3DK.%2BA.%26atitle%3DRegulation%2520of%2520dopamine%2520D2%2520receptors%2520by%2520sodium%2520and%2520pH%26jtitle%3DMol.%2520Pharmacol.%26date%3D1991%26volume%3D39%26issue%3D4%26spage%3D570" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ananthanarayanan, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerman, A.</span></span> <span> </span><span class="NLM_article-title">Role of metal ions in ligand-receptor interaction: insights from structural studies</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>246</i></span> (<span class="NLM_issue">1â2</span>),  <span class="NLM_fpage">53</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.mce.2005.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.mce.2005.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=16368180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsF2gtLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=246&amp;publication_year=2006&amp;pages=53&amp;issue=1%E2%80%932&amp;author=V.+S.+Ananthanarayananauthor=A.+Kerman&amp;title=Role+of+metal+ions+in+ligand-receptor+interaction%3A+insights+from+structural+studies&amp;doi=10.1016%2Fj.mce.2005.11.023" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Role of metal ions in ligand-receptor interaction: Insights from structural studies</span></div><div class="casAuthors">Ananthanarayanan, Vettai S.; Kerman, Aaron</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">246</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">53-59</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Exptl. data indicate that metal ions such as Na+, Ca2+ and Mg2+, which are present in millimolar concns. in the extracellular environment, modulate binding of ligands to plasma membrane receptors.  Here, we briefly review structural studies that demonstrate that various types of ligands, including peptide hormones and drugs, bind metal ions, in particular Ca2+, in the lipid milieu.  We propose that the metal ion-bound forms of ligands represent their bioactive conformations.  With a view to understanding the mechanism of modulation of ligand-receptor interactions by metal ions, we have computed a homol. model of the mu-opioid receptor, a G protein-coupled receptor (GPCR), and performed docking of specific agonist and antagonist ligands in the receptor.  This resulted in the formation of a ligand-metal ion-receptor (ternary) complex which accounted for the data on the structure-activity relationships of ligands and mutation data on the receptor.  Based on exptl. and modeling data, we have proposed a general mechanism of activation of GPCRs by their corresponding ligands wherein metal ions play a pivotal role.  Studies on overexpressed segments of mu-receptor are in progress to verify the above proposal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCaUGp7DmQG7Vg90H21EOLACvtfcHk0lgeyCB8MFlt2w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28XhsF2gtLg%253D&amp;md5=461fb14c7023afc315f2688480e16493</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2005.11.023&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2005.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DAnanthanarayanan%26aufirst%3DV.%2BS.%26aulast%3DKerman%26aufirst%3DA.%26atitle%3DRole%2520of%2520metal%2520ions%2520in%2520ligand-receptor%2520interaction%253A%2520insights%2520from%2520structural%2520studies%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2006%26volume%3D246%26issue%3D1%25E2%2580%25932%26spage%3D53%26doi%3D10.1016%2Fj.mce.2005.11.023" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strasser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seifert, R.</span></span> <span> </span><span class="NLM_article-title">Modulation of GPCRs by monovalent cations and anions</span>. <i>Naunyn-Schmiedeberg's Arch. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>388</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">363</span>, <span class="refDoi">&nbsp;DOI: 10.1007/s00210-014-1073-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1007%2Fs00210-014-1073-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=25432095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVSitLzF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=388&amp;publication_year=2015&amp;pages=363&amp;issue=3&amp;author=A.+Strasserauthor=H.+J.+Wittmannauthor=E.+H.+Schneiderauthor=R.+Seifert&amp;title=Modulation+of+GPCRs+by+monovalent+cations+and+anions&amp;doi=10.1007%2Fs00210-014-1073-2" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of GPCRs by monovalent cations and anions</span></div><div class="casAuthors">Strasser, Andrea; Wittmann, Hans-Joachim; Schneider, Erich H.; Seifert, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Naunyn-Schmiedeberg's Archives of Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">388</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">363-380</span>CODEN:
                <span class="NLM_cas:coden">NSAPCC</span>;
        ISSN:<span class="NLM_cas:issn">0028-1298</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The recent resoln. of G-protein-coupled receptor (GPCR) structures in complex with Na+ bound to an allosteric modulatory site has renewed interest of the regulation of GPCRs by ions.  Here, we summarise key data on ion modulation of GPCRs, obtained in pharmacol., crystallog., mutagenesis and mol. modeling studies.  We show that ion modulation is a highly complex process, involving not only cations but also, rather neglected until now, anions.  Pharmacotherapeutic and toxicol. aspects are discussed.  We provide a math. framework for the anal. of ion effects.  Finally, we discuss open questions in the field and future research directions.  Most importantly, the in vivo relevance of the modulation of GPCR function by monovalent ions must be clarified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDmk3TO24ZO7Vg90H21EOLACvtfcHk0liOIx2KQr3f-A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXitVSitLzF&amp;md5=74281aba85e6e810d49d08f89980fb01</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs00210-014-1073-2&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00210-014-1073-2%26sid%3Dliteratum%253Aachs%26aulast%3DStrasser%26aufirst%3DA.%26aulast%3DWittmann%26aufirst%3DH.%2BJ.%26aulast%3DSchneider%26aufirst%3DE.%2BH.%26aulast%3DSeifert%26aufirst%3DR.%26atitle%3DModulation%2520of%2520GPCRs%2520by%2520monovalent%2520cations%2520and%2520anions%26jtitle%3DNaunyn-Schmiedeberg%2527s%2520Arch.%2520Pharmacol.%26date%3D2015%26volume%3D388%26issue%3D3%26spage%3D363%26doi%3D10.1007%2Fs00210-014-1073-2" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gulya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovaâ²cs, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaâ²sa, P.</span></span> <span> </span><span class="NLM_article-title">Regulation of endogenous calcium and magnesium levels by Î´ opioid receptors in the rat brain</span>. <i>Brain Res.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>547</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">32</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0006-8993(91)90570-L</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2F0006-8993%2891%2990570-L" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=547&amp;publication_year=1991&amp;pages=32&amp;issue=1&amp;author=K.+Gulyaauthor=G.+L.+Kova%E2%80%B2csauthor=P.+Ka%E2%80%B2sa&amp;title=Regulation+of+endogenous+calcium+and+magnesium+levels+by+%CE%B4+opioid+receptors+in+the+rat+brain&amp;doi=10.1016%2F0006-8993%2891%2990570-L" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2F0006-8993%2891%2990570-L&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-8993%252891%252990570-L%26sid%3Dliteratum%253Aachs%26aulast%3DGulya%26aufirst%3DK.%26aulast%3DKova%25E2%2580%25B2cs%26aufirst%3DG.%2BL.%26aulast%3DKa%25E2%2580%25B2sa%26aufirst%3DP.%26atitle%3DRegulation%2520of%2520endogenous%2520calcium%2520and%2520magnesium%2520levels%2520by%2520%25CE%25B4%2520opioid%2520receptors%2520in%2520the%2520rat%2520brain%26jtitle%3DBrain%2520Res.%26date%3D1991%26volume%3D547%26issue%3D1%26spage%3D32%26doi%3D10.1016%2F0006-8993%2891%2990570-L" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tejwani, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanissian, S. H.</span></span> <span> </span><span class="NLM_article-title">Modulation of mu, delta and kappa opioid receptors in rat brain by metal ions and histidine</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">445</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0028-3908(90)90166-O</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2F0028-3908%2890%2990166-O" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=2162496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADyaK3cXktlCqu70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=1990&amp;pages=445&amp;issue=5&amp;author=G.+A.+Tejwaniauthor=S.+H.+Hanissian&amp;title=Modulation+of+mu%2C+delta+and+kappa+opioid+receptors+in+rat+brain+by+metal+ions+and+histidine&amp;doi=10.1016%2F0028-3908%2890%2990166-O" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of mu, delta and kappa opioid receptors in rat brain by metal ions and histidine</span></div><div class="casAuthors">Tejwani, G. A.; Hanissian, S. H.</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">445-52</span>CODEN:
                <span class="NLM_cas:coden">NEPHBW</span>;
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    </div><div class="casAbstract">The effect of Zn and several other trace elements was studied on the binding of the opioid receptor agonists [3]DAGO, [3H]DSTLE ([Tyr-D-Ser-Gly-Phe-Leu-Thr]-enkephalin and [3H]EKC (ethylketocyclazocine), which are specific for the mu, delta, and kappa opioid receptors, resp., in the cerebral cortex of the rat.  Physiol. concns. of Zn were inhibitory to mu receptor binding, whereas the delta and kappa receptors were relatively insensitive to this inhibition.  Scatchard anal., using these opioid agonists, revealed curvilinear plots; concns. of Zn equal to or less than the IC50 (the concn. of cation which caused 50% inhibition of the binding of opioid ligand to its receptor) increased the KD of all 3 subtypes of receptor, with no effect on the Bmax and abolished the high affinity sites of the delta and kappa receptors.  Cu, Cd, and Hg also inhibited the binding of these ligands to their receptors.  Histidine was most effective in preventing the inhibitory effects of Zn and Cu, whereas it was less effective on Cd and without any effect on the inhibition caused by Hg.  Mg and Mn were stimulatory to opioid receptor binding, whereas Co and Ni had dual (stimulatory and inhibitory) effects.  Non-inhibitory concns. of Zn decreased the stimulatory effects of Mg and Mn on the mu and delta receptors, suggesting that part of the effect of Zn was through prevention of the actions of stimulatory cations.  The reducing reagent dithiothreitol partially protected against inhibition by Zn and the oxidizing reagent dithiobisnitrobenzic acid potentiated the inhibitory effects of Zn on the binding of [3H]DSTLE and [3H]DAGO.  These results suggest that physiol. concns. of free Zn can inhibit mu but not delta and kappa receptors.  The lower susceptibility of delta and kappa receptors to inhibition by Zn may be due to the presence of fewer sulfhydryl groups in these receptors, or due to their inaccessibility to binding by Zn.  Relative physiol. concns. of free Zn and histidine may govern the extent of binding of the mu ligands to their receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-rmCD3WDg4LVg90H21EOLACvtfcHk0liOIx2KQr3f-A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3cXktlCqu70%253D&amp;md5=608286de7e6e2a9f3902e6ade84b78ff</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2F0028-3908%2890%2990166-O&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0028-3908%252890%252990166-O%26sid%3Dliteratum%253Aachs%26aulast%3DTejwani%26aufirst%3DG.%2BA.%26aulast%3DHanissian%26aufirst%3DS.%2BH.%26atitle%3DModulation%2520of%2520mu%252C%2520delta%2520and%2520kappa%2520opioid%2520receptors%2520in%2520rat%2520brain%2520by%2520metal%2520ions%2520and%2520histidine%26jtitle%3DNeuropharmacology%26date%3D1990%26volume%3D29%26issue%3D5%26spage%3D445%26doi%3D10.1016%2F0028-3908%2890%2990166-O" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">Structure of the D2 dopamine receptor bound to the atypical antipsychotic drug risperidone</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>555</i></span> (<span class="NLM_issue">7695</span>),  <span class="NLM_fpage">269</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature25758</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fnature25758" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29466326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktFKqt7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=555&amp;publication_year=2018&amp;pages=269&amp;issue=7695&amp;author=S.+Wangauthor=T.+Cheauthor=A.+Levitauthor=B.+K.+Shoichetauthor=D.+Wackerauthor=B.+L.+Roth&amp;title=Structure+of+the+D2+dopamine+receptor+bound+to+the+atypical+antipsychotic+drug+risperidone&amp;doi=10.1038%2Fnature25758" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the D2 dopamine receptor bound to the atypical antipsychotic drug risperidone</span></div><div class="casAuthors">Wang, Sheng; Che, Tao; Levit, Anat; Shoichet, Brian K.; Wacker, Daniel; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">555</span>
        (<span class="NLM_cas:issue">7695</span>),
    <span class="NLM_cas:pages">269-273</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Dopamine is a neurotransmitter that has been implicated in processes as diverse as reward, addiction, control of coordinated movement, metab. and hormonal secretion.  Correspondingly, dysregulation of the dopaminergic system has been implicated in diseases such as schizophrenia, Parkinson's disease, depression, attention deficit hyperactivity disorder, and nausea and vomiting.  The actions of dopamine are mediated by a family of five G-protein-coupled receptors.  The D2 dopamine receptor (DRD2) is the primary target for both typical and atypical antipsychotic drugs, and for drugs used to treat Parkinson's disease.  Unfortunately, many drugs that target DRD2 cause serious and potentially life-threatening side effects due to promiscuous activities against related receptors.  Accordingly, a mol. understanding of the structure and function of DRD2 could provide a template for the design of safer and more effective medications.  Here we report the crystal structure of DRD2 in complex with the widely prescribed atypical antipsychotic drug risperidone.  The DRD2-risperidone structure reveals an unexpected mode of antipsychotic drug binding to dopamine receptors, and highlights structural determinants that are essential for the actions of risperidone and related drugs at DRD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeo4Ov49kft7Vg90H21EOLACvtfcHk0lg4LiqiQLQbhQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXktFKqt7c%253D&amp;md5=2842adc1893293ea8dbf98ae3270a1b2</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnature25758&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature25758%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DChe%26aufirst%3DT.%26aulast%3DLevit%26aufirst%3DA.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DStructure%2520of%2520the%2520D2%2520dopamine%2520receptor%2520bound%2520to%2520the%2520atypical%2520antipsychotic%2520drug%2520risperidone%26jtitle%3DNature%26date%3D2018%26volume%3D555%26issue%3D7695%26spage%3D269%26doi%3D10.1038%2Fnature25758" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Free, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moritz, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conroy, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meade, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Discovery and characterization of a G protein-biased agonist that inhibits beta-arrestin recruitment to the D2 dopamine receptor</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">96</span>, <span class="refDoi">&nbsp;DOI: 10.1124/mol.113.090563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1124%2Fmol.113.090563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=24755247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGksrvF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=86&amp;publication_year=2014&amp;pages=96&amp;issue=1&amp;author=R.+B.+Freeauthor=L.+S.+Chunauthor=A.+E.+Moritzauthor=B.+N.+Millerauthor=T.+B.+Doyleauthor=J.+L.+Conroyauthor=A.+Padronauthor=J.+A.+Meadeauthor=J.+Xiaoauthor=X.+Hu&amp;title=Discovery+and+characterization+of+a+G+protein-biased+agonist+that+inhibits+beta-arrestin+recruitment+to+the+D2+dopamine+receptor&amp;doi=10.1124%2Fmol.113.090563" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of a G protein-biased agonist that inhibits Î²-arrestin recruitment to the D2 dopamine receptor</span></div><div class="casAuthors">Free, R. Benjamin; Chun, Lani S.; Moritz, Amy E.; Miller, Brittney N.; Doyle, Trevor B.; Conroy, Jennie L.; Padron, Adrian; Meade, Julie A.; Xiao, Jingbo; Hu, Xin; Dulcey, Andres E.; Han, Yang; Duan, Lihua; Titus, Steve; Bryant-Genevier, Melanie; Barnaeva, Elena; Ferrer, Marc; Javitch, Jonathan A.; Beuming, Thijs; Shi, Lei; Southall, Noel T.; Marugan, Juan J.; Sibley, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">96-105, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A high-throughput screening campaign was conducted to interrogate a 380,000+ small-mol. library for novel D2 dopamine receptor modulators using a calcium mobilization assay.  Active agonist compds. from the primary screen were examd. for orthogonal D2 dopamine receptor signaling activities including cAMP modulation and Î²-arrestin recruitment.  Although the majority of the subsequently confirmed hits activated all signaling pathways tested, several compds. showed a diminished ability to stimulate Î²-arrestin recruitment.  One such compd. (MLS1547; 5-chloro-7-[(4-pyridin-2-ylpiperazin-1-yl)methyl]quinolin-8-ol) is a highly efficacious agonist at D2 receptor-mediated G protein-linked signaling, but does not recruit Î²-arrestin as demonstrated using two different assays.  This compd. does, however, antagonize dopamine-stimulated Î²-arrestin recruitment to the D2 receptor.  In an effort to investigate the chem. scaffold of MLS1547 further, we characterized a set of 24 analogs of MLS1547 with respect to their ability to inhibit cAMP accumulation or stimulate Î²-arrestin recruitment.  A no. of the analogs were similar to MLS1547 in that they displayed agonist activity for inhibiting cAMP accumulation, but did not stimulate Î²-arrestin recruitment (i.e., they were highly biased).  In contrast, other analogs displayed various degrees of G protein signaling bias.  These results provided the basis to use pharmacophore modeling and mol. docking analyses to build a preliminary structure-activity relationship of the functionally selective properties of this series of compds.  In summary, we have identified and characterized a novel G protein-biased agonist of the D2 dopamine receptor and identified structural features that may contribute to its biased signaling properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUnTQyiZvZxrVg90H21EOLACvtfcHk0lg4LiqiQLQbhQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGksrvF&amp;md5=7a3412858a3d780d8f90ca4535d485fc</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1124%2Fmol.113.090563&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.113.090563%26sid%3Dliteratum%253Aachs%26aulast%3DFree%26aufirst%3DR.%2BB.%26aulast%3DChun%26aufirst%3DL.%2BS.%26aulast%3DMoritz%26aufirst%3DA.%2BE.%26aulast%3DMiller%26aufirst%3DB.%2BN.%26aulast%3DDoyle%26aufirst%3DT.%2BB.%26aulast%3DConroy%26aufirst%3DJ.%2BL.%26aulast%3DPadron%26aufirst%3DA.%26aulast%3DMeade%26aufirst%3DJ.%2BA.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DX.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520a%2520G%2520protein-biased%2520agonist%2520that%2520inhibits%2520beta-arrestin%2520recruitment%2520to%2520the%2520D2%2520dopamine%2520receptor%26jtitle%3DMol.%2520Pharmacol.%26date%3D2014%26volume%3D86%26issue%3D1%26spage%3D96%26doi%3D10.1124%2Fmol.113.090563" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonsson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beuming, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voth, G. A.</span></span> <span> </span><span class="NLM_article-title">Ligand-dependent activation and deactivation of the human adenosine A(2A) receptor</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">8749</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja404391q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja404391q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvVCrt7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=135&amp;publication_year=2013&amp;pages=8749&amp;issue=23&amp;author=J.+Liauthor=A.+L.+Jonssonauthor=T.+Beumingauthor=J.+C.+Shelleyauthor=G.+A.+Voth&amp;title=Ligand-dependent+activation+and+deactivation+of+the+human+adenosine+A%282A%29+receptor&amp;doi=10.1021%2Fja404391q" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-Dependent Activation and Deactivation of the Human Adenosine A2A Receptor</span></div><div class="casAuthors">Li, Jianing; Jonsson, Amanda L.; Beuming, Thijs; Shelley, John C.; Voth, Gregory A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8749-8759</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are membrane proteins with crit. functions in cellular signal transduction, representing a primary class of drug targets.  Acting by direct binding, many drugs modulate GPCR activity and influence the signaling pathways assocd. with numerous diseases.  However, complete details of ligand-dependent GPCR activation/deactivation are difficult to obtain from expts.  Therefore, it remains unclear how ligands modulate a GPCR's activity.  To elucidate the ligand-dependent activation/deactivation mechanism of the human adenosine A2A receptor (AA2AR), a member of the class A GPCRs, we performed large-scale unbiased mol. dynamics and metadynamics simulations of the receptor embedded in a membrane.  At the at. level, we have obsd. distinct structural states that resemble the active and inactive states.  In particular, we noted key structural elements changing in a highly concerted fashion during the conformational transitions, including six conformational states of a tryptophan (Trp2466.48).  Our findings agree with a previously proposed view that, during activation, this tryptophan residue undergoes a rotameric transition that may be coupled to a series of coherent conformational changes, resulting in the opening of the G-protein binding site.  Further, metadynamics simulations provide quant. evidence for this mechanism, suggesting how ligand binding shifts the equil. between the active and inactive states.  Our anal. also proposes that a few specific residues are assocd. with agonism/antagonism, affinity, and selectivity, and suggests that the ligand-binding pocket can be thought of as having three distinct regions, providing dynamic features for structure-based design.  Addnl. simulations with AA2AR bound to a novel ligand are consistent with our proposed mechanism.  Generally, our study provides insights into the ligand-dependent AA2AR activation/deactivation in addn. to what has been found in crystal structures.  These results should aid in the discovery of more effective and selective GPCR ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_X5Tj8UviMLVg90H21EOLACvtfcHk0lg4LiqiQLQbhQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXnvVCrt7g%253D&amp;md5=e20018f868df9a6031a00465d97ab0ca</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja404391q&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja404391q%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJonsson%26aufirst%3DA.%2BL.%26aulast%3DBeuming%26aufirst%3DT.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DVoth%26aufirst%3DG.%2BA.%26atitle%3DLigand-dependent%2520activation%2520and%2520deactivation%2520of%2520the%2520human%2520adenosine%2520A%25282A%2529%2520receptor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26issue%3D23%26spage%3D8749%26doi%3D10.1021%2Fja404391q" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddy, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deary, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Structural Connection between Activation Microswitch and Allosteric Sodium Site in GPCR Signaling</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">259</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.str.2017.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.str.2017.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29395784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVSjs70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2018&amp;pages=259&amp;issue=2&amp;author=K.+L.+Whiteauthor=M.+T.+Eddyauthor=Z.-G.+Gaoauthor=G.+W.+Hanauthor=T.+Lianauthor=A.+Dearyauthor=N.+Patelauthor=K.+A.+Jacobsonauthor=V.+Katritchauthor=R.+C.+Stevens&amp;title=Structural+Connection+between+Activation+Microswitch+and+Allosteric+Sodium+Site+in+GPCR+Signaling&amp;doi=10.1016%2Fj.str.2017.12.013" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Connection between Activation Microswitch and Allosteric Sodium Site in GPCR Signaling</span></div><div class="casAuthors">White, Kate L.; Eddy, Matthew T.; Gao, Zhan-Guo; Han, Gye Won; Lian, Tiffany; Deary, Alexander; Patel, Nilkanth; Jacobson, Kenneth A.; Katritch, Vsevolod; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">259-269.e5</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Sodium ions are endogenous allosteric modulators of many G-protein-coupled receptors (GPCRs).  Mutation of key residues in the sodium binding motif causes a striking effect on G-protein signaling.  We report the crystal structures of agonist complexes for two variants in the first sodium coordination shell of the human A2A adenosine receptor, D522.50N and S913.39A.  Both structures present an overall active-like conformation; however, the variants show key changes in the activation motif NPxxY.  Changes in the hydrogen bonding network in this microswitch suggest a possible mechanism for modified G-protein signaling and enhanced thermal stability.  These structures, signaling data, and thermal stability anal. with a panel of pharmacol. ligands provide a basis for understanding the role of the sodium-coordinating residues on stability and G-protein signaling.  Utilizing the D2.50N variant is a promising method for stabilizing class A GPCRs to accelerate structural efforts and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo18IDIJKugCrVg90H21EOLACvtfcHk0liWEI62n-UuXw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXisVSjs70%253D&amp;md5=916e644534536f2a995288d207328c4d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2017.12.013&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2017.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DEddy%26aufirst%3DM.%2BT.%26aulast%3DGao%26aufirst%3DZ.-G.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DLian%26aufirst%3DT.%26aulast%3DDeary%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520Connection%2520between%2520Activation%2520Microswitch%2520and%2520Allosteric%2520Sodium%2520Site%2520in%2520GPCR%2520Signaling%26jtitle%3DStructure%26date%3D2018%26volume%3D26%26issue%3D2%26spage%3D259%26doi%3D10.1016%2Fj.str.2017.12.013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newton, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strange, P. G.</span></span> <span> </span><span class="NLM_article-title">Examining the Effects of Sodium Ions on the Binding of Antagonists to Dopamine D2 and D3 Receptors</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">e0158808</span>, <span class="refDoi">&nbsp;DOI: 10.1371/journal.pone.0158808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1371%2Fjournal.pone.0158808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=27379794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVaqtb3N" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2016&amp;pages=e0158808&amp;issue=7&amp;author=C.+L.+Newtonauthor=M.+D.+Woodauthor=P.+G.+Strange&amp;title=Examining+the+Effects+of+Sodium+Ions+on+the+Binding+of+Antagonists+to+Dopamine+D2+and+D3+Receptors&amp;doi=10.1371%2Fjournal.pone.0158808" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Examining the effects of sodium ions on the binding of antagonists to dopamine D2 and D3 receptors</span></div><div class="casAuthors">Newton, Claire L.; Wood, Martyn D.; Strange, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e0158808/1-e0158808/11</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Many G protein-coupled receptors have been shown to be sensitive to the presence of sodium ions (Na+).  Using radioligand competition binding assays, we have examd. and compared the effects of sodium ions on the binding affinities of a no. of structurally diverse ligands at human dopamine D2 and dopamine D3 receptor subtypes, which are important therapeutic targets for the treatment of psychotic disorders.  At both receptors, the binding affinities of the antagonists/inverse agonists SB-277011-A, L,741,626, GR 103691 and U 99194 were higher in the presence of sodium ions compared to those measured in the presence of the org. cation, N-methyl-d-glucamine, used to control for ionic strength.  Conversely, the affinities of spiperone and (+)-butaclamol were unaffected by the presence of sodium ions.  Interestingly, the binding of the antagonist/inverse agonist clozapine was affected by changes in ionic strength of the buffer used rather than the presence of specific cations.  Similar sensitivities to sodium ions were seen at both receptors, suggesting parallel effects of sodium ion interactions on receptor conformation.  However, no clear correlation between ligand characteristics, such as subtype selectivity, and sodium ion sensitivity were obsd.  Therefore, the properties which det. this sensitivity remain unclear.  However these findings do highlight the importance of careful consideration of assay buffer compn. for in vitro assays and when comparing data from different studies, and may indicate a further level of control for ligand binding in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv1PwNFNVf8bVg90H21EOLACvtfcHk0liWEI62n-UuXw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XhvVaqtb3N&amp;md5=39327177fdd523fe702a94a7e9427bb1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0158808&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0158808%26sid%3Dliteratum%253Aachs%26aulast%3DNewton%26aufirst%3DC.%2BL.%26aulast%3DWood%26aufirst%3DM.%2BD.%26aulast%3DStrange%26aufirst%3DP.%2BG.%26atitle%3DExamining%2520the%2520Effects%2520of%2520Sodium%2520Ions%2520on%2520the%2520Binding%2520of%2520Antagonists%2520to%2520Dopamine%2520D2%2520and%2520D3%2520Receptors%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26issue%3D7%26spage%3De0158808%26doi%3D10.1371%2Fjournal.pone.0158808" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Free, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sibley, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span> <span> </span><span class="NLM_article-title">Structural basis for Na(+)-sensitivity in dopamine D2 and D3 receptors</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">41</span>),  <span class="NLM_fpage">8618</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C5CC02204E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1039%2FC5CC02204E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=25896577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmt1Wnuro%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=51&amp;publication_year=2015&amp;pages=8618&amp;issue=41&amp;author=M.+Michinoauthor=R.+B.+Freeauthor=T.+B.+Doyleauthor=D.+R.+Sibleyauthor=L.+Shi&amp;title=Structural+basis+for+Na%28%2B%29-sensitivity+in+dopamine+D2+and+D3+receptors&amp;doi=10.1039%2FC5CC02204E" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for Na+-sensitivity in dopamine D2 and D3 receptors</span></div><div class="casAuthors">Michino, Mayako; Free, R. Benjamin; Doyle, Trevor B.; Sibley, David R.; Shi, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">8618-8621</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">To understand the structural basis for the Na+-sensitivity of ligand binding to dopamine D2-like receptors, using computational anal. in combination with binding assays, the authors identified interactions crit. in propagating the impact of Na+ on receptor conformations and on the ligand-binding site.  The authors' findings expand the pharmacol.-relevant conformational spectrum of these receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonu0n6gq5E2bVg90H21EOLACvtfcHk0liWEI62n-UuXw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Wnuro%253D&amp;md5=750d293936c4215a96a5deb95f6271d6</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1039%2FC5CC02204E&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC02204E%26sid%3Dliteratum%253Aachs%26aulast%3DMichino%26aufirst%3DM.%26aulast%3DFree%26aufirst%3DR.%2BB.%26aulast%3DDoyle%26aufirst%3DT.%2BB.%26aulast%3DSibley%26aufirst%3DD.%2BR.%26aulast%3DShi%26aufirst%3DL.%26atitle%3DStructural%2520basis%2520for%2520Na%2528%252B%2529-sensitivity%2520in%2520dopamine%2520D2%2520and%2520D3%2520receptors%26jtitle%3DChem.%2520Commun.%26date%3D2015%26volume%3D51%26issue%3D41%26spage%3D8618%26doi%3D10.1039%2FC5CC02204E" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vickery, O. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carvalheda, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaidi, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisliakov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zachariae, U.</span></span> <span> </span><span class="NLM_article-title">Intracellular Transfer of Na(+) in an Active-State G-Protein-Coupled Receptor</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">171</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.str.2017.11.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.str.2017.11.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=29249607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvF2js7vF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2018&amp;pages=171&amp;issue=1&amp;author=O.+N.+Vickeryauthor=C.+A.+Carvalhedaauthor=S.+A.+Zaidiauthor=A.+V.+Pisliakovauthor=V.+Katritchauthor=U.+Zachariae&amp;title=Intracellular+Transfer+of+Na%28%2B%29+in+an+Active-State+G-Protein-Coupled+Receptor&amp;doi=10.1016%2Fj.str.2017.11.013" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular Transfer of Na+ in an Active-State G-Protein-Coupled Receptor</span></div><div class="casAuthors">Vickery, Owen N.; Carvalheda, Catarina A.; Zaidi, Saheem A.; Pisliakov, Andrei V.; Katritch, Vsevolod; Zachariae, Ulrich</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">171-180.e2</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Playing a central role in cell signaling, G-protein-coupled receptors (GPCRs) are the largest superfamily of membrane proteins and form the majority of drug targets in humans.  How extracellular agonist binding triggers the activation of GPCRs and assocd. intracellular effector proteins remains, however, poorly understood.  Structural studies have revealed that inactive class A GPCRs harbor a conserved binding site for Na+ ions in the center of their transmembrane domain, accessible from the extracellular space.  Here, we show that the opening of a conserved hydrated channel in the activated state receptors allows the Na+ ion to egress from its binding site into the cytosol.  Coupled with protonation changes, this ion movement occurs without significant energy barriers, and can be driven by physiol. transmembrane ion and voltage gradients.  We propose that Na+ ion exchange with the cytosol is a key step in GPCR activation.  Further, we hypothesize that this transition locks receptors in long-lived active-state conformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzBCbcEx6TibVg90H21EOLACvtfcHk0li0ydr13LbHSg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhvF2js7vF&amp;md5=da6940bf420ad51c5b275e2fec7231ae</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2017.11.013&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2017.11.013%26sid%3Dliteratum%253Aachs%26aulast%3DVickery%26aufirst%3DO.%2BN.%26aulast%3DCarvalheda%26aufirst%3DC.%2BA.%26aulast%3DZaidi%26aufirst%3DS.%2BA.%26aulast%3DPisliakov%26aufirst%3DA.%2BV.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DZachariae%26aufirst%3DU.%26atitle%3DIntracellular%2520Transfer%2520of%2520Na%2528%252B%2529%2520in%2520an%2520Active-State%2520G-Protein-Coupled%2520Receptor%26jtitle%3DStructure%26date%3D2018%26volume%3D26%26issue%3D1%26spage%3D171%26doi%3D10.1016%2Fj.str.2017.11.013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vass, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kooistra, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritschel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leurs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span> <span> </span><span class="NLM_article-title">Molecular interaction fingerprint approaches for GPCR drug discovery</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">59</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.coph.2016.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.coph.2016.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=27479316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1eju7zJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=30&amp;publication_year=2016&amp;pages=59&amp;author=M.+Vassauthor=A.+J.+Kooistraauthor=T.+Ritschelauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&amp;title=Molecular+interaction+fingerprint+approaches+for+GPCR+drug+discovery&amp;doi=10.1016%2Fj.coph.2016.07.007" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular interaction fingerprint approaches for GPCR drug discovery</span></div><div class="casAuthors">Vass, Marton; Kooistra, Albert J.; Ritschel, Tina; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-68</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints (IFPs) are binary 1D representations of the 3D structure of protein-ligand complexes encoding the presence or absence of specific interactions between the binding pocket amino acids and the ligand.  Various implementations of IFPs have been developed and successfully applied for post-processing mol. docking results for G Protein-Coupled Receptor (GPCR) ligand binding mode prediction and virtual ligand screening.  Novel interaction fingerprint methods enable structural chemogenomics and polypharmacol. predictions by complementing the increasing amt. of GPCR structural data.  Machine learning methods are increasingly used to derive relationships between bioactivity data and fingerprint descriptors of chem. and structural information of binding sites, ligands, and protein-ligand interactions.  Factors that influence the application of IFPs include structure prepn., binding site definition, fingerprint similarity assessment, and data processing and these factors pose challenges as well possibilities to optimize interaction fingerprint methods for GPCR drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfcDOwflIZPrVg90H21EOLACvtfcHk0li0ydr13LbHSg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28Xht1eju7zJ&amp;md5=567b591ab504d3d9ec6bec9556588118</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2016.07.007&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2016.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DVass%26aufirst%3DM.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DRitschel%26aufirst%3DT.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DMolecular%2520interaction%2520fingerprint%2520approaches%2520for%2520GPCR%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2016%26volume%3D30%26spage%3D59%26doi%3D10.1016%2Fj.coph.2016.07.007" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span> <span> </span><span class="NLM_article-title">Structural interaction fingerprints: a new approach to organizing, mining, analyzing, and designing protein-small molecule complexes</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">5</span>, <span class="refDoi">&nbsp;DOI: 10.1111/j.1747-0285.2005.00323.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1111%2Fj.1747-0285.2005.00323.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=16492144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVSjt7o%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=67&amp;publication_year=2006&amp;pages=5&amp;issue=1&amp;author=J.+Singhauthor=Z.+Dengauthor=G.+Naraleauthor=C.+Chuaqui&amp;title=Structural+interaction+fingerprints%3A+a+new+approach+to+organizing%2C+mining%2C+analyzing%2C+and+designing+protein-small+molecule+complexes&amp;doi=10.1111%2Fj.1747-0285.2005.00323.x" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structural interaction fingerprints: a new approach to organizing, mining, analyzing, and designing protein-small molecule complexes</span></div><div class="casAuthors">Singh, Juswinder; Deng, Zhan; Narale, Gaurav; Chuaqui, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology &amp; Drug Design</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-12</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  The combination of advances in structure-based drug design efforts in the pharmaceutical industry in parallel with structural genomics initiatives in the public domain has led to an explosion in the no. of structures of protein-small mol. complexes.  This information has crit. importance to both the understanding of the structural basis for mol. recognition in biol. systems and the design of better drugs.  A significant challenge exists in managing this vast amt. of data and fully leveraging it.  Here, we review our work to develop a simple, fast way to store, organize, mine, and analyze large nos. of protein-small mol. complexes.  We illustrate the utility of the approach to the management of inhibitor complexes from the protein kinase family.  Finally, we describe our recent efforts in applying this method to the design of target-focused chem. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyYr5KQpOFMLVg90H21EOLACvtfcHk0lglD6XyVmzbJw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28XhsVSjt7o%253D&amp;md5=2e291b90d4fafa433d544669d7678d57</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2005.00323.x&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2005.00323.x%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DNarale%26aufirst%3DG.%26aulast%3DChuaqui%26aufirst%3DC.%26atitle%3DStructural%2520interaction%2520fingerprints%253A%2520a%2520new%2520approach%2520to%2520organizing%252C%2520mining%252C%2520analyzing%252C%2520and%2520designing%2520protein-small%2520molecule%2520complexes%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2006%26volume%3D67%26issue%3D1%26spage%3D5%26doi%3D10.1111%2Fj.1747-0285.2005.00323.x" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span> <span> </span><span class="NLM_article-title">Structural interaction fingerprint (SIFt): a novel method for analyzing three-dimensional protein-ligand binding interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">337</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jm030331x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030331x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD3sXps1yrsrs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=2004&amp;pages=337&amp;issue=2&amp;author=Z.+Dengauthor=C.+Chuaquiauthor=J.+Singh&amp;title=Structural+interaction+fingerprint+%28SIFt%29%3A+a+novel+method+for+analyzing+three-dimensional+protein-ligand+binding+interactions&amp;doi=10.1021%2Fjm030331x" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Interaction Fingerprint (SIFt): A Novel Method for Analyzing Three-Dimensional Protein-Ligand Binding Interactions</span></div><div class="casAuthors">Deng, Zhan; Chuaqui, Claudio; Singh, Juswinder</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-344</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Representing and understanding the three-dimensional (3D) structural information of protein-ligand complexes is a crit. step in the rational drug discovery process.  Traditional anal. methods are proving inadequate and inefficient in dealing with the massive amt. of structural information being generated from x-ray crystallog., NMR, and in silico approaches such as structure-based docking expts.  Here, we present SIFt (structural interaction fingerprint), a novel method for representing and analyzing 3D protein-ligand binding interactions.  Key to this approach is the generation of an interaction fingerprint that translates 3D structural binding information from a protein-ligand complex into a one-dimensional binary string.  Each fingerprint represents the "structural interaction profile" of the complex that can be used to organize, analyze, and visualize the rich amt. of information encoded in ligand-receptor complexes and also to assist database mining.  We have applied SIFt to tackle three common tasks in structure-based drug design.  The first involved the anal. and organization of a typical set of results generated from a docking study.  Using SIFt, docking poses with similar binding modes were identified, clustered, and subsequently compared with conventional scoring function information.  A second application of SIFt was to analyze â¼90 known x-ray crystal structures of protein kinase-inhibitor complexes obtained from the Protein Databank.  Using SIFt, we were able to organize the structures and reveal striking similarities and diversity between their small mol. binding interactions.  Finally, we have shown how SIFt can be used as an effective mol. filter during the virtual chem. library screening process to select mols. with desirable binding mode(s) and/or desirable interaction patterns with the protein target.  In summary, SIFt shows promise to fully leverage the wealth of information being generated in rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkjji1ni0HHbVg90H21EOLACvtfcHk0lglD6XyVmzbJw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3sXps1yrsrs%253D&amp;md5=a652b70184155c107202cbe3bc74596a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm030331x&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030331x%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DStructural%2520interaction%2520fingerprint%2520%2528SIFt%2529%253A%2520a%2520novel%2520method%2520for%2520analyzing%2520three-dimensional%2520protein-ligand%2520binding%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D2%26spage%3D337%26doi%3D10.1021%2Fjm030331x" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palczewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S.</span></span> <span> </span><span class="NLM_article-title">Exploring a new ligand binding site of G protein-coupled receptors</span>. <i>Chemical Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">6480</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C8SC01680A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1039%2FC8SC01680A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=30310578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWhtL7I" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2018&amp;pages=6480&amp;author=H.+C.+S.+Chanauthor=J.+Wangauthor=K.+Palczewskiauthor=S.+Filipekauthor=H.+Vogelauthor=Z.-J.+Liuauthor=S.+Yuan&amp;title=Exploring+a+new+ligand+binding+site+of+G+protein-coupled+receptors&amp;doi=10.1039%2FC8SC01680A" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring a new ligand binding site of G protein-coupled receptors</span></div><div class="casAuthors">Chan, H. C. Stephen; Wang, Jingjing; Palczewski, Krzysztof; Filipek, Slawomir; Vogel, Horst; Liu, Zhi-Jie; Yuan, Shuguang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6480-6489</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Identifying a target ligand binding site is an important step for structure-based rational drug design as shown here for G protein-coupled receptors (GPCRs), which are among the most popular drug targets.  We applied long-time scale mol. dynamics simulations, coupled with mutagenesis studies, to two prototypical GPCRs, the M3 and M4 muscarinic acetylcholine receptors.  Our results indicate that unlike synthetic antagonists, which bind to the classic orthosteric site, the endogenous agonist acetylcholine is able to diffuse into a much deeper binding pocket.  We also discovered that the most recently resolved crystal structure of the LTB4 receptor comprised a bound inverse agonist, which extended its benzamidine moiety to the same binding pocket discovered in this work.  Anal. on all resolved GPCR crystal structures indicated that this new pocket could exist in most receptors.  Our findings provide new opportunities for GPCR drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3DxdJBSHfq7Vg90H21EOLACvtfcHk0lglD6XyVmzbJw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWhtL7I&amp;md5=8c52e5787c09118d3c62dd047b025cf8</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2FC8SC01680A&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8SC01680A%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DH.%2BC.%2BS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DPalczewski%26aufirst%3DK.%26aulast%3DFilipek%26aufirst%3DS.%26aulast%3DVogel%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DZ.-J.%26aulast%3DYuan%26aufirst%3DS.%26atitle%3DExploring%2520a%2520new%2520ligand%2520binding%2520site%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DChemical%2520Science%26date%3D2018%26volume%3D9%26spage%3D6480%26doi%3D10.1039%2FC8SC01680A" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kling, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span> <span> </span><span class="NLM_article-title">Comparative MD Simulations Indicate a Dual Role for Arg1323.50 in Dopamine-Dependent D2R Activation</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">e0146612</span>, <span class="refDoi">&nbsp;DOI: 10.1371/journal.pone.0146612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1371%2Fjournal.pone.0146612" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=26741139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC28XlslShtrc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2016&amp;pages=e0146612&amp;issue=1&amp;author=R.+C.+Klingauthor=T.+Clarkauthor=P.+Gmeiner&amp;title=Comparative+MD+Simulations+Indicate+a+Dual+Role+for+Arg1323.50+in+Dopamine-Dependent+D2R+Activation&amp;doi=10.1371%2Fjournal.pone.0146612" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative MD simulations indicate a dual role for arg1323.50 in dopamine-dependent D2R activation</span></div><div class="casAuthors">Kling, Ralf C.; Clark, Timothy; Gmeiner, Peter</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e0146612/1-e0146612/13</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Residue Arg3.50 belongs to the highly conserved DRY-motif of class A GPCRs, which is located at the bottom of TM3.  On the one hand, Arg3.50 has been reported to help stabilize the inactive state of GPCRs, but on the other hand has also been shown to be crucial for stabilizing active receptor conformations and mediating receptor-G protein coupling.  The combined results of these studies suggest that the exact function of Arg3.50 is likely to be receptor-dependent and must be characterized independently for every GPCR.  Consequently, we now present comparative mol.-dynamics simulations that use our recently described inactive-state and GÎ±-bound active-state homol. models of the dopamine D2 receptor (D2R), which are either bound to dopamine or ligand-free, performed to identify the function of Arg1323.50 in D2R.  Our results are consistent with a dynamic model of D2R activation in which Arg1323.50 adopts a dual role, both by stabilizing the inactive-state receptor conformation and enhancing dopamine-dependent D2R-G protein coupling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnbNGpP05zObVg90H21EOLACvtfcHk0lgnRMdduKx_OQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XlslShtrc%253D&amp;md5=dddb24a2f1a80087c1019d9debaf9624</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0146612&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0146612%26sid%3Dliteratum%253Aachs%26aulast%3DKling%26aufirst%3DR.%2BC.%26aulast%3DClark%26aufirst%3DT.%26aulast%3DGmeiner%26aufirst%3DP.%26atitle%3DComparative%2520MD%2520Simulations%2520Indicate%2520a%2520Dual%2520Role%2520for%2520Arg1323.50%2520in%2520Dopamine-Dependent%2520D2R%2520Activation%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26issue%3D1%26spage%3De0146612%26doi%3D10.1371%2Fjournal.pone.0146612" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kling, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschammer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span> <span> </span><span class="NLM_article-title">Active-state model of a dopamine D2 receptor-Galphai complex stabilized by aripiprazole-type partial agonists</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">e100069</span>, <span class="refDoi">&nbsp;DOI: 10.1371/journal.pone.0100069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1371%2Fjournal.pone.0100069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=24932547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1egt7zM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2014&amp;pages=e100069&amp;issue=6&amp;author=R.+C.+Klingauthor=N.+Tschammerauthor=H.+Lanigauthor=T.+Clarkauthor=P.+Gmeiner&amp;title=Active-state+model+of+a+dopamine+D2+receptor-Galphai+complex+stabilized+by+aripiprazole-type+partial+agonists&amp;doi=10.1371%2Fjournal.pone.0100069" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Active-state model of a dopamine D2 receptor - GÎ±i complex stabilized by aripiprazole-type partial agonists</span></div><div class="casAuthors">Kling, Ralf C.; Tschammer, Nuska; Lanig, Harald; Clark, Timothy; Gmeiner, Peter</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e100069/1-e100069/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Partial agonists exhibit a submaximal capacity to enhance the coupling of one receptor to an intracellular binding partner.  Although a multitude of studies have reported different ligand-specific conformations for a given receptor, little is known about the mechanism by which different receptor conformations are connected to the capacity to activate the coupling to G-proteins.  We have now performed mol.-dynamics simulations employing our recently described active-state homol. model of the dopamine D2 receptor-GÎ±i protein-complex coupled to the partial agonists aripiprazole and FAUC350, in order to understand the structural determinants of partial agonism better.  We have compared our findings with our model of the D2R-GÎ±i-complex in the presence of the full agonist dopamine.  The two partial agonists are capable of inducing different conformations of important structural motifs, including the extracellular loop regions, the binding pocket and, in particular, intracellular G-protein-binding domains.  As G-protein-coupling to certain intracellular epitopes of the receptor is considered the key step of allosterically triggered nucleotide-exchange, it is tempting to assume that impaired coupling between the receptor and the G-protein caused by distinct ligand-specific conformations is a major determinant of partial agonist efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru44KV_-m_drVg90H21EOLACvtfcHk0lgnRMdduKx_OQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXhs1egt7zM&amp;md5=d02e0973e621a4de6d43093aaacb931c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0100069&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0100069%26sid%3Dliteratum%253Aachs%26aulast%3DKling%26aufirst%3DR.%2BC.%26aulast%3DTschammer%26aufirst%3DN.%26aulast%3DLanig%26aufirst%3DH.%26aulast%3DClark%26aufirst%3DT.%26aulast%3DGmeiner%26aufirst%3DP.%26atitle%3DActive-state%2520model%2520of%2520a%2520dopamine%2520D2%2520receptor-Galphai%2520complex%2520stabilized%2520by%2520aripiprazole-type%2520partial%2520agonists%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26issue%3D6%26spage%3De100069%26doi%3D10.1371%2Fjournal.pone.0100069" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatakrishnan, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laeremans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feinberg, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanborn, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livingston, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorsen, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kling, R. C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Structural insights into micro-opioid receptor activation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>524</i></span> (<span class="NLM_issue">7565</span>),  <span class="NLM_fpage">315</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature14886</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fnature14886" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=26245379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht12jtbzM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=524&amp;publication_year=2015&amp;pages=315&amp;issue=7565&amp;author=W.+Huangauthor=A.+Manglikauthor=A.+J.+Venkatakrishnanauthor=T.+Laeremansauthor=E.+N.+Feinbergauthor=A.+L.+Sanbornauthor=H.+E.+Katoauthor=K.+E.+Livingstonauthor=T.+S.+Thorsenauthor=R.+C.+Kling&amp;title=Structural+insights+into+micro-opioid+receptor+activation&amp;doi=10.1038%2Fnature14886" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into Î¼-opioid receptor activation</span></div><div class="casAuthors">Huang, Weijiao; Manglik, Aashish; Venkatakrishnan, A. J.; Laeremans, Toon; Feinberg, Evan N.; Sanborn, Adrian L.; Kato, Hideaki E.; Livingston, Kathryn E.; Thorsen, Thor S.; Kling, Ralf C.; Granier, Sebastien; Gmeiner, Peter; Husbands, Stephen M.; Traynor, John R.; Weis, William I.; Steyaert, Jan; Dror, Ron O.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">524</span>
        (<span class="NLM_cas:issue">7565</span>),
    <span class="NLM_cas:pages">315-321</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activation of the Î¼-opioid receptor (Î¼OR) is responsible for the efficacy of the most effective analgesics.  To shed light on the structural basis for Î¼OR activation, here we report a 2.1 Ã X-ray crystal structure of the murine Î¼OR bound to the morphinan agonist BU72 and a G protein mimetic camelid antibody fragment Nb39 (nanobody 39).  The BU72-stabilized changes in the Î¼OR binding pocket are subtle and differ from those obsd. for agonist-bound structures of the Î²2-adrenergic receptor (Î²2AR) and the M2 muscarinic receptor (M2R).  Comparison with active Î²2AR reveals a common rearrangement in the packing of three conserved amino acids in the core of the Î¼OR, and mol. dynamics simulations illustrate how the ligand-binding pocket is conformationally linked to this conserved triad.  Addnl., an extensive polar network between the ligand-binding pocket and the cytoplasmic domains appears to play a similar role in signal propagation for all three G-protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkFUN1wheV07Vg90H21EOLACvtfcHk0lgnRMdduKx_OQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXht12jtbzM&amp;md5=b9af3e1351b04539379532c127c02651</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnature14886&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14886%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DVenkatakrishnan%26aufirst%3DA.%2BJ.%26aulast%3DLaeremans%26aufirst%3DT.%26aulast%3DFeinberg%26aufirst%3DE.%2BN.%26aulast%3DSanborn%26aufirst%3DA.%2BL.%26aulast%3DKato%26aufirst%3DH.%2BE.%26aulast%3DLivingston%26aufirst%3DK.%2BE.%26aulast%3DThorsen%26aufirst%3DT.%2BS.%26aulast%3DKling%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520insights%2520into%2520micro-opioid%2520receptor%2520activation%26jtitle%3DNature%26date%3D2015%26volume%3D524%26issue%3D7565%26spage%3D315%26doi%3D10.1038%2Fnature14886" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kroeze, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lansu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giguere, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciaky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span> <span> </span><span class="NLM_article-title">PRESTO-Tango as an open-source resource for interrogation of the druggable human GPCRome</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">362</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nsmb.3014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fnsmb.3014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=25895059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnslOgtLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=2015&amp;pages=362&amp;issue=5&amp;author=W.+K.+Kroezeauthor=M.+F.+Sassanoauthor=X.-P.+Huangauthor=K.+Lansuauthor=J.+D.+McCorvyauthor=P.+M.+Giguereauthor=N.+Sciakyauthor=B.+L.+Roth&amp;title=PRESTO-Tango+as+an+open-source+resource+for+interrogation+of+the+druggable+human+GPCRome&amp;doi=10.1038%2Fnsmb.3014" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">PRESTO-Tango as an open-source resource for interrogation of the druggable human GPCRome</span></div><div class="casAuthors">Kroeze, Wesley K.; Sassano, Maria F.; Huang, Xi-Ping; Lansu, Katherine; McCorvy, John D.; Giguere, Patrick M.; Sciaky, Noah; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural &amp; Molecular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">362-369</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are essential mediators of cellular signaling and are important targets of drug action.  Of the approx. 350 nonolfactory human GPCRs, more than 100 are still considered to be 'orphans' because their endogenous ligands remain unknown.  Here, we describe a unique open-source resource that allows interrogation of the druggable human GPCRome via a G protein-independent Î²-arrestin-recruitment assay.  We validate this unique platform at more than 120 nonorphan human GPCR targets, demonstrate its utility for discovering new ligands for orphan human GPCRs and describe a method (parallel receptorome expression and screening via transcriptional output, with transcriptional activation following arrestin translocation (PRESTO-Tango)) for the simultaneous and parallel interrogation of the entire human nonolfactory GPCRome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVbl7caV9gUrVg90H21EOLACvtfcHk0ljAcV6NYGZegw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXnslOgtLg%253D&amp;md5=2a78b74a5f1425fa9e8b79b730c16be6</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3014%26sid%3Dliteratum%253Aachs%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DLansu%26aufirst%3DK.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DGiguere%26aufirst%3DP.%2BM.%26aulast%3DSciaky%26aufirst%3DN.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DPRESTO-Tango%2520as%2520an%2520open-source%2520resource%2520for%2520interrogation%2520of%2520the%2520druggable%2520human%2520GPCRome%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2015%26volume%3D22%26issue%3D5%26spage%3D362%26doi%3D10.1038%2Fnsmb.3014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eswar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marti-Renom, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madhusudhan, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eramian, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pieper, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Comparative protein structure modeling using MODELLER</span>. <i>Current protocols in protein science</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">2.9.1</span>, <span class="refDoi">&nbsp;DOI: 10.1002/0471140864.ps0209s50</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1002%2F0471140864.ps0209s50" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=50&amp;publication_year=2007&amp;pages=2.9.1&amp;author=N.+Eswarauthor=B.+Webbauthor=M.+A.+Marti-Renomauthor=M.+S.+Madhusudhanauthor=D.+Eramianauthor=M.+Y.+Shenauthor=U.+Pieperauthor=A.+Sali&amp;title=Comparative+protein+structure+modeling+using+MODELLER&amp;doi=10.1002%2F0471140864.ps0209s50" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2F0471140864.ps0209s50&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F0471140864.ps0209s50%26sid%3Dliteratum%253Aachs%26aulast%3DEswar%26aufirst%3DN.%26aulast%3DWebb%26aufirst%3DB.%26aulast%3DMarti-Renom%26aufirst%3DM.%2BA.%26aulast%3DMadhusudhan%26aufirst%3DM.%2BS.%26aulast%3DEramian%26aufirst%3DD.%26aulast%3DShen%26aufirst%3DM.%2BY.%26aulast%3DPieper%26aufirst%3DU.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DComparative%2520protein%2520structure%2520modeling%2520using%2520MODELLER%26jtitle%3DCurrent%2520protocols%2520in%2520protein%2520science%26date%3D2007%26volume%3D50%26spage%3D2.9.1%26doi%3D10.1002%2F0471140864.ps0209s50" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutsias, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kortemme, T.</span></span> <span> </span><span class="NLM_article-title">Sub-angstrom accuracy in protein loop reconstruction by robotics-inspired conformational sampling</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">551</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nmeth0809-551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1038%2Fnmeth0809-551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=19644455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Whtbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2009&amp;pages=551&amp;issue=8&amp;author=D.+J.+Mandellauthor=E.+A.+Coutsiasauthor=T.+Kortemme&amp;title=Sub-angstrom+accuracy+in+protein+loop+reconstruction+by+robotics-inspired+conformational+sampling&amp;doi=10.1038%2Fnmeth0809-551" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Sub-angstrom accuracy in protein loop reconstruction by robotics-inspired conformational sampling</span></div><div class="casAuthors">Mandell, Daniel J.; Coutsias, Evangelos A.; Kortemme, Tanja</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">551-552</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQZjfGA_pks7Vg90H21EOLACvtfcHk0ljAcV6NYGZegw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Whtbw%253D&amp;md5=9d88311f894ae547071abc176e76ae3f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnmeth0809-551&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth0809-551%26sid%3Dliteratum%253Aachs%26aulast%3DMandell%26aufirst%3DD.%2BJ.%26aulast%3DCoutsias%26aufirst%3DE.%2BA.%26aulast%3DKortemme%26aufirst%3DT.%26atitle%3DSub-angstrom%2520accuracy%2520in%2520protein%2520loop%2520reconstruction%2520by%2520robotics-inspired%2520conformational%2520sampling%26jtitle%3DNat.%2520Methods%26date%3D2009%26volume%3D6%26issue%3D8%26spage%3D551%26doi%3D10.1038%2Fnmeth0809-551" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shivakumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Prediction of Absolute Solvation Free Energies using Molecular Dynamics Free Energy Perturbation and the OPLS Force Field</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1509</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct900587b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct900587b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkslKhu7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2010&amp;pages=1509&amp;issue=5&amp;author=D.+Shivakumarauthor=J.+Williamsauthor=Y.+J.+Wuauthor=W.+Dammauthor=J.+Shelleyauthor=W.+Sherman&amp;title=Prediction+of+Absolute+Solvation+Free+Energies+using+Molecular+Dynamics+Free+Energy+Perturbation+and+the+OPLS+Force+Field&amp;doi=10.1021%2Fct900587b" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Absolute Solvation Free Energies using Molecular Dynamics Free Energy Perturbation and the OPLS Force Field</span></div><div class="casAuthors">Shivakumar, Devleena; Williams, Joshua; Wu, Yujie; Damm, Wolfgang; Shelley, John; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1509-1519</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accurate prediction of protein-ligand binding free energies is a primary objective in computer-aided drug design.  The solvation free energy of a small mol. provides a surrogate to the desolvation of the ligand in the thermodn. process of protein-ligand binding.  Here, we use explicit solvent mol. dynamics free energy perturbation to predict the abs. solvation free energies of a set of 239 small mols., spanning diverse chem. functional groups commonly found in drugs and drug-like mols.  We also compare the performance of abs. solvation free energies obtained using the OPLS_2005 force field with two other commonly used small mol. force fields - general AMBER force field (GAFF) with AM1-BCC charges and CHARMm-MSI with CHelpG charges.  Using the OPLS_2005 force field, we obtain high correlation with exptl. solvation free energies (R2 = 0.94) and low av. unsigned errors for a majority of the functional groups compared to AM1-BCC/GAFF or CHelpG/CHARMm-MSI.  However, OPLS_2005 has errors of over 1.3 kcal/mol for certain classes of polar compds.  We show that predictions on these compd. classes can be improved by using a semiempirical charge assignment method with an implicit bond charge correction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon_fkbSHqg3bVg90H21EOLACvtfcHk0ljAcV6NYGZegw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXkslKhu7g%253D&amp;md5=25d464fb9238e3235e881945c0b50a76</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fct900587b&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct900587b%26sid%3Dliteratum%253Aachs%26aulast%3DShivakumar%26aufirst%3DD.%26aulast%3DWilliams%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%2BJ.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DShelley%26aufirst%3DJ.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DPrediction%2520of%2520Absolute%2520Solvation%2520Free%2520Energies%2520using%2520Molecular%2520Dynamics%2520Free%2520Energy%2520Perturbation%2520and%2520the%2520OPLS%2520Force%2520Field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26issue%3D5%26spage%3D1509%26doi%3D10.1021%2Fct900587b" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sondergaard, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, M. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rostkowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span> <span> </span><span class="NLM_article-title">Improved Treatment of Ligands and Coupling Effects in Empirical Calculation and Rationalization of pK(a) Values</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2284</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct200133y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200133y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnt1Gnsrs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2011&amp;pages=2284&amp;issue=7&amp;author=C.+R.+Sondergaardauthor=M.+H.+M.+Olssonauthor=M.+Rostkowskiauthor=J.+H.+Jensen&amp;title=Improved+Treatment+of+Ligands+and+Coupling+Effects+in+Empirical+Calculation+and+Rationalization+of+pK%28a%29+Values&amp;doi=10.1021%2Fct200133y" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Treatment of Ligands and Coupling Effects in Empirical Calculation and Rationalization of pKa Values</span></div><div class="casAuthors">Sondergaard, Chresten R.; Olsson, Mats H. M.; Rostkowski, Michal; Jensen, Jan H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2284-2295</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The new empirical rules for protein pKa predictions implemented in the PROPKA3.0 software package have been extended to the prediction of pKa shifts of active site residues and ionizable ligand groups in protein-ligand complexes.  The authors present new algorithms that allow pKa shifts due to inductive (i.e., covalently coupled) intraligand interactions, as well as noncovalently coupled interligand interactions in multiligand complexes, to be included in the prediction.  The no. of different ligand chem. groups that are automatically recognized has been increased to 18, and the general implementation has been changed so that new functional groups can be added easily by the user, aided by a new and more general protonation scheme.  Except for a few cases, the new algorithms in PROPKA3.1 are found to yield results similar to or better than those obtained with PROPKA2.0.  Finally, the authors present a novel algorithm that identifies noncovalently coupled ionizable groups, where pKa prediction may be esp. difficult.  This is a general improvement to PROPKA and is applied to proteins with and without ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVEiJYZqkno7Vg90H21EOLACvtfcHk0lg-UmCg4puU9w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXnt1Gnsrs%253D&amp;md5=cf4d4e20d6daa70de6ac623915e78160</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fct200133y&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200133y%26sid%3Dliteratum%253Aachs%26aulast%3DSondergaard%26aufirst%3DC.%2BR.%26aulast%3DOlsson%26aufirst%3DM.%2BH.%2BM.%26aulast%3DRostkowski%26aufirst%3DM.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26atitle%3DImproved%2520Treatment%2520of%2520Ligands%2520and%2520Coupling%2520Effects%2520in%2520Empirical%2520Calculation%2520and%2520Rationalization%2520of%2520pK%2528a%2529%2520Values%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26issue%3D7%26spage%3D2284%26doi%3D10.1021%2Fct200133y" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zarzycka, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaidi, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span> <span> </span><span class="NLM_article-title">Harnessing Ion-Binding Sites for GPCR Pharmacology</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>71</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">571</span>, <span class="refDoi">&nbsp;DOI: 10.1124/pr.119.017863</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1124%2Fpr.119.017863" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=31551350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFersrzE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=71&amp;publication_year=2019&amp;pages=571&amp;issue=4&amp;author=B.+Zarzyckaauthor=S.+A.+Zaidiauthor=B.+L.+Rothauthor=V.+Katritch&amp;title=Harnessing+Ion-Binding+Sites+for+GPCR+Pharmacology&amp;doi=10.1124%2Fpr.119.017863" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing ion-binding sites for GPCR pharmacology</span></div><div class="casAuthors">Zarzycka, Barbara; Zaidi, Saheem A.; Roth, Bryan L.; Katritch, Vsevolod</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">571-595</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Endogenous ions play important roles in the function and pharmacol. of G-protein coupled receptors (GPCRs).  Since then, numerous studies documenting the effects of mono- and divalent ions on GPCR function have been published.  While ions can act selectively and nonselectively at many sites in different receptors, the discovery of the conserved sodium ion site in class A GPCR structures in 2012 revealed the unique nature of Na1 site, which has emerged as a near-universal site for allosteric modulation of class A GPCR structure and function.  In this review, we synthesize and highlight recent advances in the functional, biophys., and structural characterization of ions bound to GPCRs.  Taken together, these findings provide a mol. understanding of the unique roles of Na1 and other ions as GPCR allosteric modulators.  Wewill also discuss how this knowledge can be applied to the redesign of receptors and ligand probes for desired functional and pharmacol. profiles.  Significance Statement--The function and pharmacol. of GPCRs strongly depend on the presence of mono and divalent ions in exptl. assays and in living organisms.  Recent insights into the mol. mechanism of this ion-dependent allosterism from structural, biophys., biochem., and computational studies provide quant. understandings of the pharmacol. effects of drugs in vitro and in vivo and open new avenues for the rational design of chem. probes and drug candidates with improved properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQejnWEn8A5LVg90H21EOLACvtfcHk0lg-UmCg4puU9w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BB3cXhtFersrzE&amp;md5=0770795abba451d39a1f8a96f957e5f3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1124%2Fpr.119.017863&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.119.017863%26sid%3Dliteratum%253Aachs%26aulast%3DZarzycka%26aufirst%3DB.%26aulast%3DZaidi%26aufirst%3DS.%2BA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DKatritch%26aufirst%3DV.%26atitle%3DHarnessing%2520Ion-Binding%2520Sites%2520for%2520GPCR%2520Pharmacology%26jtitle%3DPharmacol.%2520Rev.%26date%3D2019%26volume%3D71%26issue%3D4%26spage%3D571%26doi%3D10.1124%2Fpr.119.017863" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span> <span> </span><span class="NLM_article-title">Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">6â7</span>),  <span class="NLM_fpage">591</span>, <span class="refDoi">&nbsp;DOI: 10.1007/s10822-010-9349-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1007%2Fs10822-010-9349-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=20354892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=24&amp;publication_year=2010&amp;pages=591&amp;issue=6%E2%80%937&amp;author=J.+R.+Greenwoodauthor=D.+Calkinsauthor=A.+P.+Sullivanauthor=J.+C.+Shelley&amp;title=Towards+the+comprehensive%2C+rapid%2C+and+accurate+prediction+of+the+favorable+tautomeric+states+of+drug-like+molecules+in+aqueous+solution&amp;doi=10.1007%2Fs10822-010-9349-1" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution</span></div><div class="casAuthors">Greenwood, Jeremy R.; Calkins, David; Sullivan, Arron P.; Shelley, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6-7</span>),
    <span class="NLM_cas:pages">591-604</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Generating the appropriate protonation states of drug-like mols. in soln. is important for success in both ligand- and structure-based virtual screening.  Screening collections of millions of compds. requires a method for detg. tautomers and their energies that is sufficiently rapid, accurate, and comprehensive.  To maximize enrichment, the lowest energy tautomers must be detd. from heterogeneous input, without over-enumerating unfavorable states.  While computationally expensive, the d. functional theory (DFT) method M06-2X/aug-cc-pVTZ(-f) [PB-SCRF] provides accurate energies for enumerated model tautomeric systems.  The empirical Hammett-Taft methodol. can very rapidly extrapolate substituent effects from model systems to drug-like mols. via the relationship between pKT and pKa.  Combining the 2 complementary approaches transforms the tautomer problem from a scientific challenge to one of engineering scale-up, and avoids issues that arise due to the very limited no. of measured pKT values, esp. for the complicated heterocycles often favored by medicinal chemists for their novelty and versatility.  Several hundreds of pre-calcd. tautomer energies and substituent pKa effects are tabulated in databases for use in structural adjustment by the program Epik, which treats tautomers as a subset of the larger problem of the protonation states in aq. ensembles and their energy penalties.  Accuracy and coverage is continually improved and expanded by parameterizing new systems of interest using DFT and exptl. data.  Recommendations are made for how to best incorporate tautomers in mol. design and virtual screening workflows.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpigkzHeo8JY7Vg90H21EOLACvtfcHk0lg-UmCg4puU9w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbo%253D&amp;md5=1d7bc0f966ca793d6be80554868367b8</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9349-1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9349-1%26sid%3Dliteratum%253Aachs%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DCalkins%26aufirst%3DD.%26aulast%3DSullivan%26aufirst%3DA.%2BP.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26atitle%3DTowards%2520the%2520comprehensive%252C%2520rapid%252C%2520and%2520accurate%2520prediction%2520of%2520the%2520favorable%2520tautomeric%2520states%2520of%2520drug-like%2520molecules%2520in%2520aqueous%2520solution%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26issue%3D6%25E2%2580%25937%26spage%3D591%26doi%3D10.1007%2Fs10822-010-9349-1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoefling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aponte-Santamaria, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubmuller, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groenhof, G.</span></span> <span> </span><span class="NLM_article-title">g_membed: Efficient insertion of a membrane protein into an equilibrated lipid bilayer with minimal perturbation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2169</span>, <span class="refDoi">&nbsp;DOI: 10.1002/jcc.21507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1002%2Fjcc.21507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=20336801" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFWjs7c%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=31&amp;publication_year=2010&amp;pages=2169&amp;issue=11&amp;author=M.+G.+Wolfauthor=M.+Hoeflingauthor=C.+Aponte-Santamariaauthor=H.+Grubmullerauthor=G.+Groenhof&amp;title=g_membed%3A+Efficient+insertion+of+a+membrane+protein+into+an+equilibrated+lipid+bilayer+with+minimal+perturbation&amp;doi=10.1002%2Fjcc.21507" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">g_membed: Efficient insertion of a membrane protein into an equilibrated lipid bilayer with minimal perturbation</span></div><div class="casAuthors">Wolf, Maarten G.; Hoefling, Martin; Aponte-Santamaria, Camilo; Grubmueller, Helmut; Groenhof, Gerrit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2169-2174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley &amp; Sons, Inc.</span>)
        </div><div class="casAbstract">To efficiently insert a protein into an equilibrated and fully hydrated membrane with minimal membrane perturbation the authors present a computational tool, called g_membed, which is part of the Gromacs suite of programs.  The input consists of an equilibrated membrane system, either flat or curved, and a protein structure in the right position and orientation with respect to the lipid bilayer.  G_membed first decreases the width of the protein in the xy-plane and removes all mols. (generally lipids and waters) that overlap with the narrowed protein.  Then the protein is grown back to its full size in a short mol. dynamics simulation (typically 1000 steps), thereby pushing the lipids away to optimally accommodate the protein in the membrane.  After embedding the protein in the membrane, both the lipid properties and the hydration layer are still close to equil.  Thus, only a short equilibration run (less then 1 ns in the cases tested) is required to re-equilibrate the membrane.  Its simplicity makes g_membed very practical for use in scripting and high-throughput mol. dynamics simulations.  Â© 2010 Wiley Periodicals, Inc.  J Comput Chem, 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0efGbaCaVIbVg90H21EOLACvtfcHk0litdj6FK_7P9g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXnsFWjs7c%253D&amp;md5=1f0353d68fb867fb684283b61d52eadb</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21507&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21507%26sid%3Dliteratum%253Aachs%26aulast%3DWolf%26aufirst%3DM.%2BG.%26aulast%3DHoefling%26aufirst%3DM.%26aulast%3DAponte-Santamaria%26aufirst%3DC.%26aulast%3DGrubmuller%26aufirst%3DH.%26aulast%3DGroenhof%26aufirst%3DG.%26atitle%3Dg_membed%253A%2520Efficient%2520insertion%2520of%2520a%2520membrane%2520protein%2520into%2520an%2520equilibrated%2520lipid%2520bilayer%2520with%2520minimal%2520perturbation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26issue%3D11%26spage%3D2169%26doi%3D10.1002%2Fjcc.21507" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lomize, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogozheva, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosberg, H. I.</span></span> <span> </span><span class="NLM_article-title">Anisotropic solvent model of the lipid bilayer. 1. Parameterization of long-range electrostatics and first solvation shell effects</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">918</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ci2000192</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci2000192" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvVSisbo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=51&amp;publication_year=2011&amp;pages=918&amp;issue=4&amp;author=A.+L.+Lomizeauthor=I.+D.+Pogozhevaauthor=H.+I.+Mosberg&amp;title=Anisotropic+solvent+model+of+the+lipid+bilayer.+1.+Parameterization+of+long-range+electrostatics+and+first+solvation+shell+effects&amp;doi=10.1021%2Fci2000192" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Anisotropic Solvent Model of the Lipid Bilayer. 1. Parameterization of Long-Range Electrostatics and First Solvation Shell Effects</span></div><div class="casAuthors">Lomize, Andrei L.; Pogozheva, Irina D.; Mosberg, Henry I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">918-929</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new implicit solvation model was developed for calcg. free energies of transfer of mols. from water to any solvent with defined bulk properties.  The transfer energy was calcd. as a sum of the first solvation shell energy and the long-range electrostatic contribution.  The first term was proportional to solvent accessible surface area and solvation parameters (Ïi) for different atom types.  The electrostatic term was computed as a product of group dipole moments and dipolar solvation parameter (Î·) for neutral mols. or using a modified Born equation for ions.  The regression coeffs. in linear dependencies of solvation parameters Ïi and Î· on dielec. const., solvatochromic polarizability parameter Ï*, and hydrogen-bonding donor and acceptor capacities of solvents were optimized using 1269 exptl. transfer energies from 19 org. solvents to water.  The root-mean-square errors for neutral compds. and ions were 0.82 and 1.61 kcal/mol, resp.  Quantification of energy components demonstrates the dominant roles of hydrophobic effect for nonpolar atoms and of hydrogen-bonding for polar atoms.  The estd. first solvation shell energy outweighs the long-range electrostatics for most compds. including ions.  The simplicity and computational efficiency of the model allows its application for modeling of macromols. in anisotropic environments, such as biol. membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonnF3m5c7LEbVg90H21EOLACvtfcHk0litdj6FK_7P9g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXjvVSisbo%253D&amp;md5=00ceb7dde2c69eb0353480b227bb93bc</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fci2000192&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2000192%26sid%3Dliteratum%253Aachs%26aulast%3DLomize%26aufirst%3DA.%2BL.%26aulast%3DPogozheva%26aufirst%3DI.%2BD.%26aulast%3DMosberg%26aufirst%3DH.%2BI.%26atitle%3DAnisotropic%2520solvent%2520model%2520of%2520the%2520lipid%2520bilayer.%25201.%2520Parameterization%2520of%2520long-range%2520electrostatics%2520and%2520first%2520solvation%2520shell%2520effects%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26issue%3D4%26spage%3D918%26doi%3D10.1021%2Fci2000192" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klauda, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venable, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freites, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâConnor, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobias, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondragon-Ramirez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vorobyov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, R. W.</span></span> <span> </span><span class="NLM_article-title">Update of the CHARMM All-Atom Additive Force Field for Lipids: Validation on Six Lipid Types</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">7830</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jp101759q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp101759q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsVGjtrk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=114&amp;publication_year=2010&amp;pages=7830&amp;issue=23&amp;author=J.+B.+Klaudaauthor=R.+M.+Venableauthor=J.+A.+Freitesauthor=J.+W.+O%E2%80%99Connorauthor=D.+J.+Tobiasauthor=C.+Mondragon-Ramirezauthor=I.+Vorobyovauthor=A.+D.+MacKerellauthor=R.+W.+Pastor&amp;title=Update+of+the+CHARMM+All-Atom+Additive+Force+Field+for+Lipids%3A+Validation+on+Six+Lipid+Types&amp;doi=10.1021%2Fjp101759q" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Update of the CHARMM All-Atom Additive Force Field for Lipids: Validation on Six Lipid Types</span></div><div class="casAuthors">Klauda, Jeffery B.; Venable, Richard M.; Freites, J. Alfredo; O'Connor, Joseph W.; Tobias, Douglas J.; Mondragon-Ramirez, Carlos; Vorobyov, Igor; MacKerell, Alexander D., Jr.; Pastor, Richard W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7830-7843</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-6106</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A significant modification to the additive all-atom CHARMM lipid force field (FF) is developed and applied to phospholipid bilayers with both choline and ethanolamine contg. head groups and with both satd. and unsatd. aliph. chains.  Motivated by the current CHARMM lipid FF (C27 and C27r) systematically yielding values of the surface area per lipid that are smaller than exptl. ests. and gel-like structures of bilayers well above the gel transition temp., selected torsional, Lennard-Jones and partial at. charge parameters were modified by targeting both quantum mech. (QM) and exptl. data.  QM calcns. ranging from high-level ab initio calcns. on small mols. to semiempirical QM studies on a 1,2-dipalmitoyl-sn-phosphatidylcholine (DPPC) bilayer in combination with exptl. thermodn. data were used as target data for parameter optimization.  These changes were tested with simulations of pure bilayers at high hydration of the following six lipids: DPPC, 1,2-dimyristoyl-sn-phosphatidylcholine (DMPC), 1,2-dilauroyl-sn-phosphatidylcholine (DLPC), 1-palmitoyl-2-oleoyl-sn-phosphatidylcholine (POPC), 1,2-dioleoyl-sn-phosphatidylcholine (DOPC), and 1-palmitoyl-2-oleoyl-sn-phosphatidylethanolamine (POPE); simulations of a low hydration DOPC bilayer were also performed.  Agreement with exptl. surface area is on av. within 2%, and the d. profiles agree well with neutron and x-ray diffraction expts.  NMR deuterium order parameters (SCD) are well predicted with the new FF, including proper splitting of the SCD for the aliph. carbon adjacent to the carbonyl for DPPC, POPE, and POPC bilayers.  The area compressibility modulus and frequency dependence of 13C NMR relaxation rates of DPPC and the water distribution of low hydration DOPC bilayers also agree well with expt.  Accordingly, the presented lipid FF, referred to as C36, allows for mol. dynamics simulations to be run in the tensionless ensemble (NPT), and is anticipated to be of utility for simulations of pure lipid systems as well as heterogeneous systems including membrane proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhTVhlLR7mEbVg90H21EOLACvtfcHk0litdj6FK_7P9g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXmsVGjtrk%253D&amp;md5=04c4d3aa1a59740190994d7df80cbb71</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjp101759q&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp101759q%26sid%3Dliteratum%253Aachs%26aulast%3DKlauda%26aufirst%3DJ.%2BB.%26aulast%3DVenable%26aufirst%3DR.%2BM.%26aulast%3DFreites%26aufirst%3DJ.%2BA.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DJ.%2BW.%26aulast%3DTobias%26aufirst%3DD.%2BJ.%26aulast%3DMondragon-Ramirez%26aufirst%3DC.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DPastor%26aufirst%3DR.%2BW.%26atitle%3DUpdate%2520of%2520the%2520CHARMM%2520All-Atom%2520Additive%2520Force%2520Field%2520for%2520Lipids%253A%2520Validation%2520on%2520Six%2520Lipid%2520Types%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2010%26volume%3D114%26issue%3D23%26spage%3D7830%26doi%3D10.1021%2Fjp101759q" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanommeslaeghe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raman, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Automation of the CHARMM General Force Field (CGenFF) II: assignment of bonded parameters and partial atomic charges</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3155</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ci3003649</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3003649" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=2012&amp;pages=3155&amp;issue=12&amp;author=K.+Vanommeslaegheauthor=E.+P.+Ramanauthor=A.+D.+MacKerell&amp;title=Automation+of+the+CHARMM+General+Force+Field+%28CGenFF%29+II%3A+assignment+of+bonded+parameters+and+partial+atomic+charges&amp;doi=10.1021%2Fci3003649" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) II: Assignment of Bonded Parameters and Partial Atomic Charges</span></div><div class="casAuthors">Vanommeslaeghe, K.; Raman, E. Prabhu; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3155-3168</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug candidates interacting with biol. systems.  In these simulations, the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and partial at. charges is required.  In the present article, algorithms for the assignment of parameters and charges for the CHARMM General Force Field (CGenFF) are presented.  These algorithms rely on the existing parameters and charges that were detd. as part of the parametrization of the force field.  Bonded parameters are assigned based on the similarity between the atom types that define said parameters, while charges are detd. using an extended bond-charge increment scheme.  Charge increments were optimized to reproduce the charges on model compds. that were part of the parametrization of the force field.  Case studies are presented to clarify the functioning of the algorithms and the significance of their output data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0zgm66CLdn7Vg90H21EOLACvtfcHk0lgXu5KW5KxKXg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF&amp;md5=e676ad1f42cb1e98dd353d4d285e8d13</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fci3003649&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3003649%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520General%2520Force%2520Field%2520%2528CGenFF%2529%2520II%253A%2520assignment%2520of%2520bonded%2520parameters%2520and%2520partial%2520atomic%2520charges%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D12%26spage%3D3155%26doi%3D10.1021%2Fci3003649" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J.</span>; <span class="NLM_string-name">Trucks, G. W.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Petersson, G. A.</span>; <span class="NLM_string-name">Nakatsuji, H.</span>; <span class="NLM_string-name">Caricato, M.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Hratchian, H. P.</span>; <span class="NLM_string-name">Izmaylov, A. F.</span>; <span class="NLM_string-name">Bloino, J.</span>; <span class="NLM_string-name">Zheng, G.</span>; <span class="NLM_string-name">Sonnenberg, J. L.</span>; <span class="NLM_string-name">Hada, M.</span>; <span class="NLM_string-name">Ehara, M.</span>; <span class="NLM_string-name">Toyota, K.</span>; <span class="NLM_string-name">Fukuda, R.</span>; <span class="NLM_string-name">Hasegawa, J.</span>; <span class="NLM_string-name">Ishida, M.</span>; <span class="NLM_string-name">Nakajima, T.</span>; <span class="NLM_string-name">Honda, Y.</span>; <span class="NLM_string-name">Kitao, O.</span>; <span class="NLM_string-name">Nakai, H.</span>; <span class="NLM_string-name">Vreven, T.</span>; <span class="NLM_string-name">Montgomery,  Jr., J. A.</span>; <span class="NLM_string-name">Peralta, J. E.</span>; <span class="NLM_string-name">Ogliaro, F.</span>; <span class="NLM_string-name">Bearpark, M.</span>; <span class="NLM_string-name">Heyd, J. J.</span>; <span class="NLM_string-name">Brothers, E.</span>; <span class="NLM_string-name">Kudin, K. N.</span>; <span class="NLM_string-name">Staroverov, V. N.</span>; <span class="NLM_string-name">Kobayashi, R.</span>; <span class="NLM_string-name">Normand, J.</span>; <span class="NLM_string-name">Raghavachari, K.</span>; <span class="NLM_string-name">Rendell, A.</span>; <span class="NLM_string-name">Burant, J. C.</span>; <span class="NLM_string-name">Iyengar, S. S.</span>; <span class="NLM_string-name">Tomasi, J.</span>; <span class="NLM_string-name">Cossi, M.</span>; <span class="NLM_string-name">Rega, N.</span>; <span class="NLM_string-name">Millam, J. M.</span>; <span class="NLM_string-name">Klene, M.</span>; <span class="NLM_string-name">Knox, J. E.</span>; <span class="NLM_string-name">Cross, J. B.</span>; <span class="NLM_string-name">Bakken, V.</span>; <span class="NLM_string-name">Adamo, C.</span>; <span class="NLM_string-name">Jaramillo, J.</span>; <span class="NLM_string-name">Gomperts, R.</span>; <span class="NLM_string-name">Stratmann, R. E.</span>; <span class="NLM_string-name">Yazyev, O.</span>; <span class="NLM_string-name">Austin, A. J.</span>; <span class="NLM_string-name">Cammi, R.</span>; <span class="NLM_string-name">Pomelli, C.</span>; <span class="NLM_string-name">Ochterski, J. W.</span>; <span class="NLM_string-name">Martin, R. L.</span>; <span class="NLM_string-name">Morokuma, K.</span>; <span class="NLM_string-name">Zakrzewski, V. G.</span>; <span class="NLM_string-name">Voth, G. A.</span>; <span class="NLM_string-name">Salvador, P.</span>; <span class="NLM_string-name">Dannenberg, J. J.</span>; <span class="NLM_string-name">Dapprich, S.</span>; <span class="NLM_string-name">Daniels, A. D.</span>; <span class="NLM_string-name">Farkas, Ã.</span>; <span class="NLM_string-name">Foresman, J. B.</span>; <span class="NLM_string-name">Ortiz, J. V.</span>; <span class="NLM_string-name">Cioslowski, J.</span>; <span class="NLM_string-name">Fox, D. J.</span></span> <i>Gaussian 09</i>, Revision A.1; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford CT</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2009&amp;author=M.+J.+Frisch&amp;author=G.+W.+Trucks&amp;author=H.+B.+Schlegel&amp;author=G.+E.+Scuseria&amp;author=M.+A.+Robb&amp;author=J.+R.+Cheeseman&amp;author=G.+Scalmani&amp;author=V.+Barone&amp;author=B.+Mennucci&amp;author=G.+A.+Petersson&amp;author=H.+Nakatsuji&amp;author=M.+Caricato&amp;author=X.+Li&amp;author=H.+P.+Hratchian&amp;author=A.+F.+Izmaylov&amp;author=J.+Bloino&amp;author=G.+Zheng&amp;author=J.+L.+Sonnenberg&amp;author=M.+Hada&amp;author=M.+Ehara&amp;author=K.+Toyota&amp;author=R.+Fukuda&amp;author=J.+Hasegawa&amp;author=M.+Ishida&amp;author=T.+Nakajima&amp;author=Y.+Honda&amp;author=O.+Kitao&amp;author=H.+Nakai&amp;author=T.+Vreven&amp;author=J.+A.+Montgomery&amp;author=J.+E.+Peralta&amp;author=F.+Ogliaro&amp;author=M.+Bearpark&amp;author=J.+J.+Heyd&amp;author=E.+Brothers&amp;author=K.+N.+Kudin&amp;author=V.+N.+Staroverov&amp;author=R.+Kobayashi&amp;author=J.+Normand&amp;author=K.+Raghavachari&amp;author=A.+Rendell&amp;author=J.+C.+Burant&amp;author=S.+S.+Iyengar&amp;author=J.+Tomasi&amp;author=M.+Cossi&amp;author=N.+Rega&amp;author=J.+M.+Millam&amp;author=M.+Klene&amp;author=J.+E.+Knox&amp;author=J.+B.+Cross&amp;author=V.+Bakken&amp;author=C.+Adamo&amp;author=J.+Jaramillo&amp;author=R.+Gomperts&amp;author=R.+E.+Stratmann&amp;author=O.+Yazyev&amp;author=A.+J.+Austin&amp;author=R.+Cammi&amp;author=C.+Pomelli&amp;author=J.+W.+Ochterski&amp;author=R.+L.+Martin&amp;author=K.+Morokuma&amp;author=V.+G.+Zakrzewski&amp;author=G.+A.+Voth&amp;author=P.+Salvador&amp;author=J.+J.+Dannenberg&amp;author=S.+Dapprich&amp;author=A.+D.+Daniels&amp;author=%C3%96.+Farkas&amp;author=J.+B.+Foresman&amp;author=J.+V.+Ortiz&amp;author=J.+Cioslowski&amp;author=D.+J.+Fox&amp;title=Gaussian+09" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%26pub%3DGaussian%252C%2520Inc%26date%3D2009" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pronk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pall, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjelkmar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apostolov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirts, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasson, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Spoel, D.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">845</span>, <span class="refDoi">&nbsp;DOI: 10.1093/bioinformatics/btt055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1093%2Fbioinformatics%2Fbtt055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=23407358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFWmsrg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=2013&amp;pages=845&amp;issue=7&amp;author=S.+Pronkauthor=S.+Pallauthor=R.+Schulzauthor=P.+Larssonauthor=P.+Bjelkmarauthor=R.+Apostolovauthor=M.+R.+Shirtsauthor=J.+C.+Smithauthor=P.+M.+Kassonauthor=D.+van+der%0ASpoel&amp;title=GROMACS+4.5%3A+a+high-throughput+and+highly+parallel+open+source+molecular+simulation+toolkit&amp;doi=10.1093%2Fbioinformatics%2Fbtt055" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit</span></div><div class="casAuthors">Pronk, Sander; Pall, Szilard; Schulz, Roland; Larsson, Per; Bjelkmar, Paer; Apostolov, Rossen; Shirts, Michael R.; Smith, Jeremy C.; Kasson, Peter M.; van der Spoel, David; Hess, Berk; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">845-854</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Mol. simulation has historically been a low-throughput technique, but faster computers and increasing amts. of genomic and structural data are changing this by enabling large-scale automated simulation of, for instance, many conformers or mutants of biomols. with or without a range of ligands.  At the same time, advances in performance and scaling now make it possible to model complex biomol. interaction and function in a manner directly testable by expt.  These applications share a need for fast and efficient software that can be deployed on massive scale in clusters, web servers, distributed computing or cloud resources.  Results: Here, we present a range of new simulation algorithms and features developed during the past 4 years, leading up to the GROMACS 4.5 software package.  The software now automatically handles wide classes of biomols., such as proteins, nucleic acids and lipids, and comes with all commonly used force fields for these mols. built-in.  GROMACS supports several implicit solvent models, as well as new free-energy algorithms, and the software now uses multithreading for efficient parallelization even on low-end systems, including windows-based workstations.  Together with hand-tuned assembly kernels and state-of-the-art parallelization, this provides extremely high performance and cost efficiency for high-throughput as well as massively parallel simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr71lQgYMm017Vg90H21EOLACvtfcHk0liqXaghk-UA-w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXksFWmsrg%253D&amp;md5=4b25fd6ab4e33725ae56b5da63f4ad68</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt055&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt055%26sid%3Dliteratum%253Aachs%26aulast%3DPronk%26aufirst%3DS.%26aulast%3DPall%26aufirst%3DS.%26aulast%3DSchulz%26aufirst%3DR.%26aulast%3DLarsson%26aufirst%3DP.%26aulast%3DBjelkmar%26aufirst%3DP.%26aulast%3DApostolov%26aufirst%3DR.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26aulast%3DKasson%26aufirst%3DP.%2BM.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26atitle%3DGROMACS%25204.5%253A%2520a%2520high-throughput%2520and%2520highly%2520parallel%2520open%2520source%2520molecular%2520simulation%2520toolkit%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26issue%3D7%26spage%3D845%26doi%3D10.1093%2Fbioinformatics%2Fbtt055" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Im, W.</span></span> <span> </span><span class="NLM_article-title">CHARMM-GUI: a web-based graphical user interface for CHARMM</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1859</span>, <span class="refDoi">&nbsp;DOI: 10.1002/jcc.20945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1002%2Fjcc.20945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=18351591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BD1cXosVKksbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=2008&amp;pages=1859&amp;issue=11&amp;author=S.+Joauthor=T.+Kimauthor=V.+G.+Iyerauthor=W.+Im&amp;title=CHARMM-GUI%3A+a+web-based+graphical+user+interface+for+CHARMM&amp;doi=10.1002%2Fjcc.20945" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM-GUI: a web-based graphical user interface for CHARMM</span></div><div class="casAuthors">Jo, Sunhwan; Kim, Taehoon; Iyer, Vidyashankara G.; Im, Wonpil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1859-1865</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley &amp; Sons, Inc.</span>)
        </div><div class="casAbstract">CHARMM is an academic research program used widely for macromol. mechanics and dynamics with versatile anal. and manipulation tools of at. coordinates and dynamics trajectories.  CHARMM-GUI, http://www.charmm-gui.org, has been developed to provide a web-based graphical user interface to generate various input files and mol. systems to facilitate and standardize the usage of common and advanced simulation techniques in CHARMM.  The web environment provides an ideal platform to build and validate a mol. model system in an interactive fashion such that, if a problem is found through visual inspection, one can go back to the previous setup and regenerate the whole system again.  In this article, we describe the currently available functional modules of CHARMM-GUI Input Generator that form a basis for the advanced simulation techniques.  Future directions of the CHARMM-GUI development project are also discussed briefly together with other features in the CHARMM-GUI website, such as Archive and Movie Gallery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7gpU-1N3WOLVg90H21EOLACvtfcHk0liqXaghk-UA-w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXosVKksbc%253D&amp;md5=112a3dd61d792b040f9f716b32220d7e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20945&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20945%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DIyer%26aufirst%3DV.%2BG.%26aulast%3DIm%26aufirst%3DW.%26atitle%3DCHARMM-GUI%253A%2520a%2520web-based%2520graphical%2520user%2520interface%2520for%2520CHARMM%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2008%26volume%3D29%26issue%3D11%26spage%3D1859%26doi%3D10.1002%2Fjcc.20945" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H.</span></span> <span> </span><span class="NLM_article-title">PyMOL and Inkscape Bridge the Data and the Data Visualization</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2041</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.str.2016.11.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1016%2Fj.str.2016.11.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=27926832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVegtbjJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=24&amp;publication_year=2016&amp;pages=2041&amp;issue=12&amp;author=S.+Yuanauthor=H.+C.+Chanauthor=S.+Filipekauthor=H.+Vogel&amp;title=PyMOL+and+Inkscape+Bridge+the+Data+and+the+Data+Visualization&amp;doi=10.1016%2Fj.str.2016.11.012" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">PyMOL and Inkscape Bridge the Data and the Data Visualization</span></div><div class="casAuthors">Yuan, Shuguang; Chan, H. C. Stephen; Filipek, Slawomir; Vogel, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2041-2042</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  This article discussed about PyMOL, a dedicated tool for macromol. visualization and Inkscape vector software of professional quality for visualizing structural biol. information and bridge the macromol. data with data visualization protocols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJG3yim_6-FbVg90H21EOLACvtfcHk0liqXaghk-UA-w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XitVegtbjJ&amp;md5=b2e74dbe2aeb110039e22b3c84385b19</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2016.11.012&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2016.11.012%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DS.%26aulast%3DChan%26aufirst%3DH.%2BC.%26aulast%3DFilipek%26aufirst%3DS.%26aulast%3DVogel%26aufirst%3DH.%26atitle%3DPyMOL%2520and%2520Inkscape%2520Bridge%2520the%2520Data%2520and%2520the%2520Data%2520Visualization%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26issue%3D12%26spage%3D2041%26doi%3D10.1016%2Fj.str.2016.11.012" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">Pt 2</span>),  <span class="NLM_fpage">213</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2010&amp;pages=213&amp;issue=Pt+2&amp;author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleve&amp;title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&amp;doi=10.1107%2FS0907444909052925" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgAH72g9dlhNQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&amp;md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3DPt%25202%26spage%3D213%26doi%3D10.1107%2FS0907444909052925" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">486</span>, <span class="refDoi">&nbsp;DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2010&amp;pages=486&amp;issue=4&amp;author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&amp;title=Features+and+development+of+Coot&amp;doi=10.1107%2FS0907444910007493" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgAH72g9dlhNQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&amp;md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26issue%3D4%26spage%3D486%26doi%3D10.1107%2FS0907444910007493" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bochevarov, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braden, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philipp, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halls, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Jaguar: A high-performance quantum chemistry software program with strengths in life and materials sciences</span>. <i>Int. J. Quantum Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">2110</span>, <span class="refDoi">&nbsp;DOI: 10.1002/qua.24481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=10.1002%2Fqua.24481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facscentsci.9b01247&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCqs7%252FP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=113&amp;publication_year=2013&amp;pages=2110&amp;issue=18&amp;author=A.+D.+Bochevarovauthor=E.+Harderauthor=T.+F.+Hughesauthor=J.+R.+Greenwoodauthor=D.+A.+Bradenauthor=D.+M.+Philippauthor=D.+Rinaldoauthor=M.+D.+Hallsauthor=J.+Zhangauthor=R.+A.+Friesner&amp;title=Jaguar%3A+A+high-performance+quantum+chemistry+software+program+with+strengths+in+life+and+materials+sciences&amp;doi=10.1002%2Fqua.24481" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Jaguar: A high-performance quantum chemistry software program with strengths in life and materials sciences</span></div><div class="casAuthors">Bochevarov, Art D.; Harder, Edward; Hughes, Thomas F.; Greenwood, Jeremy R.; Braden, Dale A.; Philipp, Dean M.; Rinaldo, David; Halls, Mathew D.; Zhang, Jing; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Quantum Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2110-2142</span>CODEN:
                <span class="NLM_cas:coden">IJQCB2</span>;
        ISSN:<span class="NLM_cas:issn">0020-7608</span>.
    
            (<span class="NLM_cas:orgname">John Wiley &amp; Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Jaguar is an ab initio quantum chem. program that specializes in fast electronic structure predictions for mol. systems of medium and large size.  Jaguar focuses on computational methods with reasonable computational scaling with the size of the system, such as d. functional theory (DFT) and local second-order Moller-Plesset perturbation theory.  The favorable scaling of the methods and the high efficiency of the program make it possible to conduct routine computations involving several thousand MOs.  This performance is achieved through a utilization of the pseudospectral approxn. and several levels of parallelization.  The speed advantages are beneficial for applying Jaguar in biomol. computational modeling.  Addnl., owing to its superior wave function guess for transition-metal-contg. systems, Jaguar finds applications in inorg. and bioinorg. chem.  The emphasis on larger systems and transition metal elements paves the way toward developing Jaguar for its use in materials science modeling.  The article describes the historical and new features of Jaguar, such as improved parallelization of many modules, innovations in ab initio pKa prediction, and new semiempirical corrections for nondynamic correlation errors in DFT.  Jaguar applications in drug discovery, materials science, force field parameterization, and other areas of computational research are reviewed.  Timing benchmarks and other results obtained from the most recent Jaguar code are provided.  The article concludes with a discussion of challenges and directions for future development of the program. Â© 2013 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt_0dnCf0Yq7Vg90H21EOLACvtfcHk0lgAH72g9dlhNQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCqs7%252FP&amp;md5=2fc119bb9e0501e55d7292f7735c24ea</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fqua.24481&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fqua.24481%26sid%3Dliteratum%253Aachs%26aulast%3DBochevarov%26aufirst%3DA.%2BD.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DHughes%26aufirst%3DT.%2BF.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DBraden%26aufirst%3DD.%2BA.%26aulast%3DPhilipp%26aufirst%3DD.%2BM.%26aulast%3DRinaldo%26aufirst%3DD.%26aulast%3DHalls%26aufirst%3DM.%2BD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DJaguar%253A%2520A%2520high-performance%2520quantum%2520chemistry%2520software%2520program%2520with%2520strengths%2520in%2520life%2520and%2520materials%2520sciences%26jtitle%3DInt.%2520J.%2520Quantum%2520Chem.%26date%3D2013%26volume%3D113%26issue%3D18%26spage%3D2110%26doi%3D10.1002%2Fqua.24481" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span> <i>SchrÃ¶dinger</i>; <span class="NLM_publisher-name">SchrÃ¶dinger,
LLC</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=+Schr%C3%B6dinger%3B+Schr%C3%B6dinger%2C%0ALLC%2C+2018." target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2018" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://doi.org/10.2210/pdb6CM4/pdb" onclick="trackOutboundLink('http://doi.org/10.2210/pdb6CM4/pdb','PDB','6CM4'); return false;">PDB: 6CM4</a></li><li><a href="http://doi.org/10.2210/pdb3PBL/pdb" onclick="trackOutboundLink('http://doi.org/10.2210/pdb3PBL/pdb','PDB','3PBL'); return false;">PDB: 3PBL</a></li><li><a href="http://doi.org/10.2210/pdb5C1M/pdb" onclick="trackOutboundLink('http://doi.org/10.2210/pdb5C1M/pdb','PDB','5C1M'); return false;">PDB: 5C1M</a></li><li><a href="http://doi.org/10.2210/pdb5CM1/pdb" onclick="trackOutboundLink('http://doi.org/10.2210/pdb5CM1/pdb','PDB','5CM1'); return false;">PDB: 5CM1</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i28">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_04564"></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acscentsci.9b01247?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acscentsci.9b01247</a>.</p><ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">MD simulations data (<a class="ext-link" href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Movie 1: An allosteric sodium ion next to D2.50 was captured by MD simulations. (<a class="ext-link" href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_002.mp4">MP4</a>)</p></li><li><p class="inline">Movie 2: A new orthosteric sodium ion, which established coordinations with both receptor and ligand, was captured by MD simulations. (<a class="ext-link" href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_003.mp4">MP4</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_liveslides.mp4">oc9b01247_liveslides.mp4 (7.29 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_001.pdf">oc9b01247_si_001.pdf (1.9 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_002.mp4">oc9b01247_si_002.mp4 (8.86 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acscentsci.9b01247/suppl_file/oc9b01247_si_003.mp4">oc9b01247_si_003.mp4 (3.71 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms &amp; Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facscentsci.9b01247&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3BrequestedJournal%3Ajournal%3Aacscii%3Bissue%3Aissue%3A10.1021%2Facscii.2020.6.issue-2%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facscentsci.9b01247%3Bjournal%3Ajournal%3Aacscii%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acscentsci.9b01247" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2022<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS &amp; Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support &amp; Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"><input type="hidden" name="appID" value="literatum" class="appID"><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onclick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        <!-- <div id="gdpr-message-container">
			<a id="closebtn" href="#"><i class="icon-times-circle-o"></i></a> -->
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        <!-- </div> -->
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        <!-- AdRoll Pixel -->
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>

<!-- Twitter universal website tag code -->
<!-- Commented out for ACHS-6152
<script>
    !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe.apply(s,arguments):s.queue.push(arguments);
    },s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',
    a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');
    // Insert Twitter Pixel ID and Standard Event data below
    twq('init','o3njp');
    twq('track','PageView');
</script>
-->
<!-- Facebook Pixel Code -->
<!-- Commented out for ACHS-6152
<script>
    !function(f,b,e,v,n,t,s)
    {if(f.fbq)return;n=f.fbq=function(){n.callMethod?
    n.callMethod.apply(n,arguments):n.queue.push(arguments)};
    if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
    n.queue=[];t=b.createElement(e);t.async=!0;
    t.src=v;s=b.getElementsByTagName(e)[0];
    s.parentNode.insertBefore(t,s)}(window, document,'script',
    'https://connect.facebook.net/en_US/fbevents.js');
    fbq('init', '529840281033320');
    fbq('track', 'PageView');
</script>
<noscript><img height="1" width="1" style="display:none" src="https://www.facebook.com/tr?id=529840281033320&amp;ev=PageView&amp;noscript=1"
/></noscript>
-->





        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
var currentJournal = "ACS Central Science";
$(".citationLinks > a").click(function(e){
    if($(this).text() === "ACS Full Text ") {
        ga('send', 'event', 'Reference _Click', 'ACS Full Text', currentJournal);
    }
    else if($(this).text() === "Crossref") {
        ga('send', 'event', 'Reference _Click', 'Crossref', currentJournal);
    }
    else if($(this).text() === "CAS") {
        ga('send', 'event', 'Reference _Click', 'CAS', currentJournal);
    }
    else if($(this).text() === "PubMed") {
        ga('send', 'event', 'Reference _Click', 'PubMed', currentJournal);
    }
    else if($(this).text() === "Google Scholar") {
        ga('send', 'event', 'Reference _Click', 'Google Scholar', currentJournal);
    }
    else if($(this).hasClass("issue-item_open-url") === true) {
        ga('send', 'event', 'Reference _Click', 'OpenURL', currentJournal);
    }
});
</script>






        
        



        
    





        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        <div tabindex="0" class="showMenuAlertMsg-popup"><div class="pairing-affiliations-popup-content"><div class="pairing-affiliations-header"><button type="button" class="close paringAffClose"><i aria-hidden="true" class="icon-close_thin"></i></button></div><div class="responsrMsg"></div></div></div>




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	dartInstitutionType = 'education';
</script>
        </div>
    





        
        



        
    





        
        <script>
// LIT-455776 workaround
// Moving the popup before the header content for tab accessibility
document.addEventListener('DOMContentLoaded', () => {
    const headerNode = $('.header_content');
    const popupNode = $('.pairing-affiliations-popup');

    if (!headerNode.length || !popupNode.length)
        return;

    popupNode.insertBefore(headerNode);  
});
</script> 
        </div>
    </div>





	
    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-d93b9088814c7fcd2f1c.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-9dee82777d6af4ace557.js"></script>
<script type="text/javascript" src="/wro/l6j1~product.js"></script>

















<script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/v652eace1692a40cfa3763df669d7439c1639079717194" integrity="sha512-Gi7xpJR8tSkrpF7aordPZQlW2DLtzUlZcumS8dMQjwDHEnw9I7ZLyiOj/6tZStRBGtGgN6ceN6cMH8z7etPGlw==" data-cf-beacon="{&quot;rayId&quot;:&quot;705bcd796936a62b&quot;,&quot;token&quot;:&quot;0562ec7bcba14d1cae3543aa43d6e749&quot;,&quot;version&quot;:&quot;2021.12.0&quot;,&quot;si&quot;:100}" crossorigin="anonymous"></script>


<div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh537" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.16748766416908134&amp;iit=1651610333259&amp;tmr=load%3D1651610332876%26core%3D1651610332964%26main%3D1651610333253%26ifr%3D1651610333265&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=pubs.acs.org&amp;dr=&amp;du=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facscentsci.9b01247&amp;href=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facscentsci.9b01247&amp;dt=Enhancing%20the%20Signaling%20of%20GPCRs%20via%20Orthosteric%20Ions&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Ctype%3DArticle%2Cdescription%2Cimage%2Ctitle%2Curl&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=627192dcd3816265&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=url%3Dhttps%253A%252F%252Fpubs.acs.org%252Fdoi%252Ffull%252F10.1021%252Facscentsci.9b01247%26title%3DEnhancing%2520the%2520Signaling%2520of%2520GPCRs%2520via%2520Orthosteric%2520Ions%26image%3Dhttps%253A%252F%252Fpubs.acs.org%252Fcms%252F10.1021%252Facscentsci.9b01247%252Fasset%252Fimages%252Facscentsci.9b01247.social.jpeg_v03%26description%3DG%2520protein-coupled%2520receptors%2520play%2520essential%2520roles%2520in%2520cellular%2520processes%2520such%2520as%2520neuronal%2520signaling%252C%2520vision%252C%2520olfaction%252C%2520tasting%252C%2520and%2520metabolism.%2520As%2520GPCRs%2520are%2520the%2520most%2520important%2520drug%2520targets%252C%2520understanding%2520their%2520interactions%2520with%2520ligands%2520is%2520of%2520utmost%2520importance%2520for%2520discovering%2520related%2520new%2520medicines.%2520In%2520many%2520GPCRs%252C%2520an%2520allosteric%2520sodium%2520ion%2520next%2520to%2520the%2520highly%2520conserved%2520residue%2520D2.50%2520has%2520been%2520proposed%2520to%2520stabilize%2520the%2520inactive%2520receptor%2520state%2520by%2520mediating%2520interactions%2520between%2520transmembrane%2520helices.%2520Here%252C%2520we%2520probed%2520the%2520existence%2520of%2520internal%2520and%2520functionally%2520important%2520sodium%2520ions%2520in%2520the%2520dopamine%2520D2%2520receptor%252C%2520using%2520molecular%2520dynamics%2520simulations.%2520Besides%2520a%2520new%2520sodium%2520ion%2520at%2520the%2520allosteric%2520ligand%2520binding%2520site%252C%2520we%2520discovered%2520an%2520additional%2520sodium%2520ion%252C%2520located%2520close%2520to%2520the%2520orthosteric%2520ligand%2520binding%2520site.%2520Through%2520cell-based%2520activation%2520assays%252C%2520the%2520signaling%2520of%2520D2%2520receptor%2520with%2520site-specific%2520mutations%2520was%2520tested%2520against%2520a%2520series%2520of%2520chemically%2520modified%2520agonists.%2520We%2520concluded%2520an%2520important%2520structural%2520role%2520of%2520this%2520newly%2520discovered%2520orthosteric%2520sodium%2520ion%2520in%2520modulating%2520the%2520receptor%2520signaling%253A%2520It%2520enables%2520the%2520coordination%2520of%2520a%2520polar%2520residue%2520in%2520the%2520ligand%2520binding%2520site%2520with%2520an%2520appropriately%2520designed%2520agonist%2520molecule.%2520An%2520identical%2520interaction%2520was%2520also%2520observed%2520in%2520a%2520recently%2520released%2520high-resolution%2520crystal%2520structure%2520of%2520mu-opioid%2520receptor%252C%2520which%2520was%2520reresolved%2520in%2520this%2520work.%2520Probably%2520because%2520of%2520similar%2520interactions%252C%2520various%2520metal%2520ions%2520have%2520been%2520found%2520to%2520increase%2520the%2520signaling%2520of%2520many%2520other%2520GPCRs.%2520This%2520unique%2520principle%2520and%2520strategy%2520could%2520be%2520used%2520to%2520optimize%2520the%2520drug%2520activity%2520of%2520GPCR.%2520Our%2520findings%2520open%2520a%2520new%2520mechanistic%2520opportunity%2520of%2520GPCR%2520signaling%2520and%2520help%2520design%2520the%2520next%2520generation%2520of%2520drugs%2520targeting%2520GPCRs.%26type%3DArticle%26site_name%3DACS%2520Publications&amp;csi=undefined&amp;rev=v8.28.8-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style></body></html>