<html class="pb-page" data-request-id="04122332-80e7-498d-84d4-9768bfdc8feb" lang="en"><head data-pb-dropzone="head"><script type="text/javascript" src="https://m.addthis.com/live/red_lojson/300lo.json?si=62603925394fb6f3&amp;bkl=0&amp;bl=1&amp;pdt=461&amp;sid=62603925394fb6f3&amp;pub=xa-4faab26f2cff13a7&amp;rev=v8.28.8-wp&amp;ln=en&amp;pc=men&amp;cb=0&amp;ab=-&amp;dp=pubs.acs.org&amp;fp=doi%2F10.1021%2Facs.joc.8b02082&amp;fr=&amp;of=2&amp;pd=0&amp;irt=1&amp;vcl=1&amp;md=0&amp;ct=1&amp;tct=0&amp;abt=0&amp;cdn=0&amp;pi=1&amp;rb=0&amp;gen=100&amp;chr=UTF-8&amp;colc=1650473253464&amp;jsl=1&amp;uvs=626038fcecfe5759016&amp;skipb=1&amp;callback=addthis.cbs.jsonp__94966735935710920"></script><script type="text/javascript" src="https://v1.addthisedge.com/live/boost/xa-4faab26f2cff13a7/_ate.track.config_resp"></script><script src="https://widgets.figshare.com/static/figshare.js?_=1650473252938"></script><script type="text/javascript" src="https://z.moatads.com/addthismoatframe568911941483/moatframe.js"></script><meta name="pbContext" content=";issue:issue:10.1021/joceah.2019.84.issue-4;page:string:Article/Chapter View;ctype:string:Journal Content;journal:journal:joceah;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.joc.8b02082;pageGroup:string:Publication Pages;subPage:string:Full Text;requestedJournal:journal:joceah">




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="The Journal of Organic Chemistry"><meta name="dc.Title" content="Synthesis of N2-Deoxyguanosine Modified DNAs and the Studies on Their Translesion Synthesis by the E. coli DNA Polymerase IV"><meta name="dc.Creator" content="Pratibha
P.  Ghodke"><meta name="dc.Creator" content="Praneeth  Bommisetti"><meta name="dc.Creator" content="Deepak T.  Nair"><meta name="dc.Creator" content="P. I.  Pradeepkumar"><meta name="dc.Description" content="We report the synthesis of N2-aryl (benzyl, naphthyl, anthracenyl, and pyrenyl)-deoxyguanosine (dG) modified phosphoramidite building blocks and the corresponding damaged DNAs. Primer extension stu..."><meta name="Description" content="We report the synthesis of N2-aryl (benzyl, naphthyl, anthracenyl, and pyrenyl)-deoxyguanosine (dG) modified phosphoramidite building blocks and the corresponding damaged DNAs. Primer extension stu..."><meta name="dc.Publisher" content="American Chemical Society"><meta name="dc.Date" scheme="WTN8601" content="January 10, 2019"><meta name="dc.Type" content="research-article"><meta name="dc.Format" content="text/HTML"><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.joc.8b02082"><meta name="dc.Language" content="en"><meta name="dc.Coverage" content="world"><meta name="dc.Rights" content="Copyright Â© 2019 American Chemical Society">
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.joc.8b02082">
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.joc.8b02082">
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.joc.8b02082">
        
    
    

<title>Synthesis of N2-Deoxyguanosine Modified DNAs and the Studies on Their Translesion Synthesis by the E. coli DNA Polymerase IV | The Journal of Organic Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive">





    






        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.joc.8b02082"><meta property="og:title" content="Synthesis of N2-Deoxyguanosine Modified DNAs and the Studies on Their Translesion Synthesis by the E. coli DNA Polymerase IV"><meta property="og:image" content="https://pubs.acs.org/cms/10.1021/acs.joc.8b02082/asset/images/acs.joc.8b02082.social.jpeg_v03"><meta property="og:description" content="We report the synthesis of N2-aryl (benzyl, naphthyl, anthracenyl, and pyrenyl)-deoxyguanosine (dG) modified phosphoramidite building blocks and the corresponding damaged DNAs. Primer extension studies using E. coli Pol IV, a translesion polymerase, demonstrate that translesion synthesis (TLS) across these N2-dG adducts is error free. However, the efficiency of TLS activity decreases with increase in the steric bulkiness of the adducts. Molecular dynamics simulations of damaged DNA-Pol IV complexes reveal the van der Waals interactions between key amino acid residues (Phe13, Ile31, Gly32, Gly33, Ser42, Pro73, Gly74, Phe76, and Tyr79) of the enzyme and adduct that help to accommodate the bulky damages in a hydrophobic pocket to facilitate TLS. Overall, the results presented here provide insights into the TLS across N2-aryl-dG damaged DNAs by Pol IV."><meta property="og:type" content="Article"><meta property="og:site_name" content="ACS Publications"><meta name="twitter:card" content="summary_large_image"><meta name="twitter:site" content="@ACSPublications"><meta name="twitter:image" content="https://pubs.acs.org/cms/10.1021/acs.joc.8b02082/asset/images/acs.joc.8b02082.social.jpeg_v03">
    





    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0">















    

    <meta name="dc.identifier" content="10.1021/acs.joc.8b02082">



<meta name="format-detection" content="telephone=no">




<link rel="stylesheet" type="text/css" href="/wro/l6j1~product.css">

<link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-c7d2282eca032afdc856.css"><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css">




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1640088552890.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1648583148410.css">
<script type="text/javascript" async="" src="//img.en25.com/i/elqCfg.min.js"></script><script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script><script async="" src="//www.google-analytics.com/analytics.js"></script><script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer="" src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon="{&quot;rayId&quot;:&quot;5200035e6e4d5180&quot;,&quot;startTime&quot;:1570117212010}"></script>

            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    , 'auto'
                    );
                
                ga('send', 'pageview');
            </script>
            











    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.joc.8b02082">
            
        







    <!-- loading Javascript for using the persistence Service -->
    <script src="/templates/jsp/pb2/js/plugin/thiss-ds.js"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            alt_context: window.location.origin,
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if (ms == undefined || ms === '') {
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            if (results.length == 0) {
                                persistService.entities(self.alt_context).then((res) => {
                                    return resolve(res.data);
                                }, function (err) {
                                    // failed
                                    reject('failed to retrieve entity from local storage (publisher domain): ' + err);
                                });
                            } else {
                                return resolve(results);
                            }
                        }, function (err) {
                            // failed
                            reject('failed to retrieve entity from local storage: ' + err);
                        });
                    // Set up the timeout
                    setTimeout(function () {
                        reject('Promise timed out ' + ms + ' ms');
                    }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                    persistService.update(self.alt_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage (publisher domain) duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            var results = res.data;
                            if (!results) {
                                persistService.remove(self.alt_context, entityId).then((res) => {
                                    return resolve(res);
                                }, function (err) {
                                    // failed
                                    console.log('failed to remove from local storage (publisher domain) duo to: ' + err);
                                });
                            }
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId, entityTitle) {
                return new Promise(function (resolve, reject) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {
                            doi: doi,
                            entityid: entityId,
                            entityTitle: entityTitle,
                            eventGroupKey: 'bdf36a11-ce97-458a-9cc2-1c8aab531602'
                        }

                    }).done(function (status) {
                        return resolve(status);
                    }).fail(function (status) {
                        return reject(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID, ssoInstitution.entity.title)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + ssoInstitution.entity.entityID,err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            }).catch(function (err) {
                                reject(err);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();

                    if ($('.institution-preference-userconsent-checkbox').length > 0) {
                        let institutionPreferenceUserConsentCheckbox = $('.institution-preference-userconsent-checkbox').get(0);
                        if (!$(institutionPreferenceUserConsentCheckbox).is(":checked")) {
                            return;
                        }
                    }

                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err) {
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>







    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY">


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html">

<link rel="shortcut icon" href="/pb-assets/ux3/favicon-1635812676023.ico"><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><meta http-equiv="origin-trial" content="AxujKG9INjsZ8/gUq8+dTruNvk7RjZQ1oFhhgQbcTJKDnZfbzSTE81wvC2Hzaf3TW4avA76LTZEMdiedF1vIbA4AAABueyJvcmlnaW4iOiJodHRwczovL2ltYXNkay5nb29nbGVhcGlzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzVGhpcmRQYXJ0eSI6dHJ1ZX0="><meta http-equiv="origin-trial" content="Azuce85ORtSnWe1MZDTv68qpaW3iHyfL9YbLRy0cwcCZwVnePnOmkUJlG8HGikmOwhZU22dElCcfrfX2HhrBPAkAAAB7eyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A16nvcdeoOAqrJcmjLRpl1I6f3McDD8EfofAYTt/P/H4/AWwB99nxiPp6kA0fXoiZav908Z8etuL16laFPUdfQsAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="AxBHdr0J44vFBQtZUqX9sjiqf5yWZ/OcHRcRMN3H9TH+t90V/j3ENW6C8+igBZFXMJ7G3Pr8Dd13632aLng42wgAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A88BWHFjcawUfKU3lIejLoryXoyjooBXLgWmGh+hNcqMK44cugvsI5YZbNarYvi3roc1fYbHA1AVbhAtuHZflgEAAAB2eyJvcmlnaW4iOiJodHRwczovL2dvb2dsZS5jb206NDQzIiwiZmVhdHVyZSI6IlRydXN0VG9rZW5zIiwiZXhwaXJ5IjoxNjUyNzc0NDAwLCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A8FHS1NmdCwGqD9DwOicnHHY+y27kdWfxKa0YHSGDfv0CSpDKRHTQdQmZVPDUdaFWUsxdgVxlwAd6o+dhJykPA0AAACWeyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiQ29udmVyc2lvbk1lYXN1cmVtZW50IiwiZXhwaXJ5IjoxNjQzMTU1MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlLCJ1c2FnZSI6InN1YnNldCJ9"><meta http-equiv="origin-trial" content="A8zdXi6dr1hwXEUjQrYiyYQGlU3557y5QWDnN0Lwgj9ePt66XMEvNkVWOEOWPd7TP9sBQ25X0Q15Lr1Nn4oGFQkAAACceyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiQ29udmVyc2lvbk1lYXN1cmVtZW50IiwiZXhwaXJ5IjoxNjQzMTU1MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlLCJ1c2FnZSI6InN1YnNldCJ9"><meta http-equiv="origin-trial" content="A4/Htern2udN9w3yJK9QgWQxQFruxOXsXL7cW60DyCl0EZFGCSme/J33Q/WzF7bBkVvhEWDlcBiUyZaim5CpFQwAAACceyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiQ29udmVyc2lvbk1lYXN1cmVtZW50IiwiZXhwaXJ5IjoxNjQzMTU1MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlLCJ1c2FnZSI6InN1YnNldCJ9"><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_2022041401.js" async=""></script><link type="text/css" rel="stylesheet" href="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/CcdcWidget.min.css"><style></style></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'organic_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/organic_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/organic_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
					console.log("ad display: " + event.creativeId + " " + event.size[0] + "x" + event.size[1]);
				}
				else {
				    console.log("no ad found");
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>

<style>
.articleHeaderDropzone5 .advertisement { display: none; }
.article_header .advertisement {
  margin: -25px auto 25px auto;
}
</style>




        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&amp;EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Indiana University Libraries - Bloomington</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"></a>




        
        <div class="pj-badge"></div><a href="/journal/joceah" title="The Journal of Organic Chemistry" class="header_article-logo"><img src="/cms/10.1021/joceah/asset/title.png" alt="The Journal of Organic Chemistry"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="/doi/10.1021/acs.joc.8b01685" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span><span class="hlFld-Title">Synthesis of <i>N</i><sup>2</sup>-Deoxyguanosine Modified DNAs and the Studies on Their Translesion Synthesis by the <i>E. coli</i> DNA Polymerase IV</span></span>
        </div>
         <a href="/doi/10.1021/acs.joc.8b02191" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete ui-autocomplete-input"><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="joceah" name="SeriesKey" checked="checked" class="all-content"><span>J. Org. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"><span>All Publications/Website</span></label></div><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" style="display: none;"></ul></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"><select name="quickLinkJournal" class="quick-search_journals-select"><option value="joceah">The Journal of Organic Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science &amp; Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials &amp; Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio &amp; Med Chem Au</option><option value="abseba">ACS Biomaterials Science &amp; Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health &amp; Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&amp;T Engineering</option><option value="aewcaa">ACS ES&amp;T Water</option><option value="afsthl">ACS Food Science &amp; Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic &amp; Inorganic Au</option><option value="aptsfn">ACS Pharmacology &amp; Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry &amp; Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&amp;EN Global Enterprise</option><option value="cenear">Chemical &amp; Engineering News Archive</option><option value="chlseg">Chemical Health &amp; Safety</option><option value="chlseg0">Chemical Health &amp; Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth &amp; Design</option><option value="enfuem">Energy &amp; Fuels</option><option value="esthag">Environmental Science &amp; Technology</option><option value="estlcu">Environmental Science &amp; Technology Letters</option><option value="iepra6.2">I&amp;EC Product Research and Development</option><option value="iechad">Industrial &amp; Engineering Chemistry</option><option value="iecac0">Industrial &amp; Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series</option><option value="iecfa7">Industrial &amp; Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial &amp; Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial &amp; Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial &amp; Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical &amp; Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health &amp; Safety</option><option value="jchsc20">Journal of Chemical Health &amp; Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial &amp; Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research &amp; Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&amp;D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autocomplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autocomplete="false" placeholder="Page" required="required" class="quick-search_page-input"><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden="âtrueâ"></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/acs.joc.8b02624">Solid-Phase Approaches for Labeling Targeting Peptides with Far-Red Emitting Coumarin Fluorophores</a></li><li><a href="/doi/full/10.1021/acs.joc.8b03144">[6Ï] Photocyclization to cis-Hexahydrocarbazol-4-ones: Substrate Modification, Mechanism, and Scope</a></li><li><a href="/doi/full/10.1021/acs.joc.8b02305">Mononuclear Rearrangement of the Z-Phenylhydrazones of Some 3-Acyl-1,2,4-oxadiazoles: Effect of Substituents on the Nucleophilic Character of the &gt;CâNâNHâC6 H5 Chain and on the Charge Density of N-2 of the 1,2,4-Oxadiazole Ring (Electrophilic Counterpart)</a></li><li><a href="/doi/full/10.1021/acs.joc.8b03145">Organotrifluoroborate Salts as Complexation Reagents for Synthesizing BODIPY Dyes Containing Both Fluoride and an Organo Substituent at the Boron Center</a></li><li><a href="/doi/full/10.1021/acs.joc.8b02978">Extended Triangulenium Ions: Syntheses and Characterization of Benzo-Bridged Dioxa- and Diazatriangulenium Dyes</a></li></ul></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="âtrueâ"></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="âtrueâ"></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden="âtrueâ"></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors &amp; Reviewers');">Authors &amp;
                        Reviewers</a></li>
                <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
				<li><a href="https://pubs.acs.org/page/virtual-collections.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Virtual Issues');">Virtual Issues</a></li>
                <li><a href="https://preferences.acs.org/journals" onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS &amp; Mobile');">RSS &amp; Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products &amp; Services');">Products &amp; Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos &amp; Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#" onclick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');" class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/who-we-serve/librarians/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians &amp; Account Managers');">For
                        Librarians &amp; Account Managers</a></li>
            </ul>
        </li>
        <li class="menu-pair-section hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a></li>
                
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage" style="display: block;">Login</a>
            <a class="header_burger-menu_logout-link" href="#" style="display: none;">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS &amp; Open Access');">ACS &amp; Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="/events" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i><div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/acs.joc.8b02624">Solid-Phase Approaches for Labeling Targeting Peptides with Far-Red Emitting Coumarin Fluorophores</a></li><li><a href="/doi/full/10.1021/acs.joc.8b03144">[6Ï] Photocyclization to cis-Hexahydrocarbazol-4-ones: Substrate Modification, Mechanism, and Scope</a></li><li><a href="/doi/full/10.1021/acs.joc.8b02305">Mononuclear Rearrangement of the Z-Phenylhydrazones of Some 3-Acyl-1,2,4-oxadiazoles: Effect of Substituents on the Nucleophilic Character of the &gt;CâNâNHâC6 H5 Chain and on the Charge Density of N-2 of the 1,2,4-Oxadiazole Ring (Electrophilic Counterpart)</a></li><li><a href="/doi/full/10.1021/acs.joc.8b03145">Organotrifluoroborate Salts as Complexation Reagents for Synthesizing BODIPY Dyes Containing Both Fluoride and an Organo Substituent at the Boron Center</a></li><li><a href="/doi/full/10.1021/acs.joc.8b02978">Extended Triangulenium Ions: Syntheses and Characterization of Benzo-Bridged Dioxa- and Diazatriangulenium Dyes</a></li></ul></div></div>


</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!-- Types--><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><!-- Subjects--><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!--div.pub-modal_right-cover-gallery.hidden-lg.hidden-md//a(href="/coverGallery") Browse Our Cover Art Gallery
--><!-- Publications list--><!-- 'pub-modal_right--inner' adds styling required for IE--><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><!-- The widget data--><input type="hidden" id="ajaxUrl" name="ajaxUrl" value="/pb/widgets/achs-publication-list?widgetId=cabcb15c-6f7a-494b-a19c-eba4fafaee5c&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjoceah.2019.84.issue-4%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Ajoceah%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.joc.8b02082%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3BrequestedJournal%3Ajournal%3Ajoceah&amp;key=c7ca46a97ce077568c7371697bcd489c0&amp;ajaxRequest=true"><input type="hidden" name="pubModalData" value=""></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article" style="min-height: 754px;"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.joc.8b02082&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.joc.8b02082&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right" style="display: inline;"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left" style="display: inline;"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.joc.8b02082&amp;href=/doi/10.1021/acs.joc.8b02082" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Org. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 84</span><span class="cit-fg-issue">, 4</span><span class="cit-fg-pageRange">, 1734-1747</span></span></span></div></div></div></div></div><input type="hidden" name="joceah" id="copyRights" value="/pb-assets/json/journal-info-1641590078917.json"><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/joceah/84/4" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.joc.8b01685" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.joc.8b02191" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/joceah" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/cms/10.1021/joceah/asset/title.png" alt="Journal Logo"></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="joceah" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info" href="#"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/joceah?ifr=1"></div></div></div><h1 class="article_header-title"><span class="hlFld-Title">Synthesis of <i>N</i><sup>2</sup>-Deoxyguanosine Modified DNAs and the Studies on Their Translesion Synthesis by the <i>E. coli</i> DNA Polymerase IV</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Pratibha P. Ghodke</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pratibha P. Ghodke</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Indian Institute of Technology Bombay, Mumbai 400076, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pratibha%0AP.++Ghodke">Pratibha P. Ghodke</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0002-8402-3933" title="Orcid link">http://orcid.org/0000-0002-8402-3933</a></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Praneeth Bommisetti</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Praneeth Bommisetti</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Indian Institute of Technology Bombay, Mumbai 400076, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Praneeth++Bommisetti">Praneeth Bommisetti</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Deepak T. Nair</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Deepak T. Nair</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Regional Centre for Biotechnology, NCR Biotech Science Cluster, third Milestone, Faridabad-Gurgaon Expressway, Faridabad 121001, India</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="mailto:deepak@rcb.res.in">deepak@rcb.res.in</a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Deepak+T.++Nair">Deepak T. Nair</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0002-0677-9444" title="Orcid link">http://orcid.org/0000-0002-0677-9444</a></div></div></span></li><span class="comma-separator">,&nbsp;and&nbsp;</span><li><span><span class="hlFld-ContribAuthor">P. I. Pradeepkumar</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">P. I. Pradeepkumar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Indian Institute of Technology Bombay, Mumbai 400076, India</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="mailto:pradeep@chem.iitb.ac.in">pradeep@chem.iitb.ac.in</a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=P.+I.++Pradeepkumar">P. I. Pradeepkumar</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-9104-3708" title="Orcid link">http://orcid.org/0000-0001-9104-3708</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="joceah"><input type="hidden" name="journalNameForjhpLink" value="The Journal of Organic Chemistry"><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.joc.8b02082&amp;href=/doi/10.1021%2Facs.joc.8b02082" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Org. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 84</span><span class="cit-issue">, 4</span><span class="cit-pageRange">, 1734â1747</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 10, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 August 2018</li><li><span class="item_label"><b>Published</b> online</span>10 January 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 15 February 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.joc.8b02082" title="DOI URL">https://doi.org/10.1021/acs.joc.8b02082</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1734%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPratibha%250AP.%2BGhodke%252C%2BPraneeth%2BBommisetti%252C%2BDeepak%2BT.%2BNair%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D84%26issueNum%3D4%26contentID%3Dacs.joc.8b02082%26title%3DSynthesis%2Bof%2BN2-Deoxyguanosine%2BModified%2BDNAs%2Band%2Bthe%2BStudies%2Bon%2BTheir%2BTranslesion%2BSynthesis%2Bby%2Bthe%2BE.%2Bcoli%2BDNA%2BPolymerase%2BIV%26numPages%3D14%26pa%3D%26issn%3D0022-3263%26publisherName%3Dacs%26publication%3Djoceah%26rpt%3Dn%26endPage%3D1747%26publicationDate%3DFebruary%2B2019">RIGHTS &amp; PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access"></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.joc.8b02082"><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">881</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">3</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" class="doi" value="10.1021/acs.joc.8b02082"><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="add-full-document achs-mendeley_button"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Synthesis of N2-Deoxyguanosine Modified DNAs and the Studies on Their Translesion Synthesis by the E. coli DNA Polymerase IV&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Pratibha\nP.&quot;,&quot;last_name&quot;:&quot;Ghodke&quot;},{&quot;first_name&quot;:&quot;Praneeth&quot;,&quot;last_name&quot;:&quot;Bommisetti&quot;},{&quot;first_name&quot;:&quot;Deepak&quot;,&quot;last_name&quot;:&quot;T. Nair&quot;},{&quot;first_name&quot;:&quot;P.&quot;,&quot;last_name&quot;:&quot;I. Pradeepkumar&quot;}],&quot;source&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;10&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;volume&quot;:&quot;84&quot;,&quot;pages&quot;:&quot;1734-1747&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.joc.8b02082&quot;},&quot;abstract&quot;:&quot;We report the synthesis of N2-aryl (benzyl, naphthyl, anthracenyl, and pyrenyl)-deoxyguanosine (dG) modified phosphoramidite building blocks and the corresponding damaged DNAs. Primer extension studies using E. coli Pol IV, a translesion polymerase, demonstrate that translesion synthesis (TLS) across these N2-dG adducts is error free. However, the efficiency of TLS activity decreases with increase in the steric bulkiness of the adducts. Molecular dynamics simulations of damaged DNA-Pol IV complexes reveal the van der Waals interactions between key amino acid residues (Phe13, Ile31, Gly32, Gly33, Ser42, Pro73, Gly74, Phe76, and Tyr79) of the enzyme and adduct that help to accommodate the bulky damages in a hydrophobic pocket to facilitate TLS. Overall, the results presented here provide insights into the TLS across N2-aryl-dG damaged DNAs by Pol IV.&quot;}" class="meta-data"><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"><input type="hidden" name="pub-access" value="full" class="pub-access"><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"><input type="hidden" name="selected-share-option" class="selected-share-option"></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.joc.8b02082&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.joc.8b02082" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.joc.8b02082&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.joc.8b02082" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.joc.8b02082&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.joc.8b02082" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.joc.8b02082&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.joc.8b02082&amp;href=/doi/10.1021/acs.joc.8b02082" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.joc.8b02082"><a href="/doi/pdf/10.1021/acs.joc.8b02082" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.joc.8b02082&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fid%3Ddoi%3A10.1021%252Facs.joc.8b02082%26sid%3Dliteratum%253Aachs%26pmid%3D30628447%26genre%3Darticle%26aulast%3DGhodke%26date%3D2019%26atitle%3DSynthesis%2Bof%2BN2-Deoxyguanosine%2BModified%2BDNAs%2Band%2Bthe%2BStudies%2Bon%2BTheir%2BTranslesion%2BSynthesis%2Bby%2Bthe%2BE.%2Bcoli%2BDNA%2BPolymerase%2BIV%26jtitle%3DThe%2BJournal%2Bof%2BOrganic%2BChemistry%26title%3DThe%2BJournal%2Bof%2BOrganic%2BChemistry%26volume%3D84%26issue%3D4%26spage%3D1734%26epage%3D1747%26issn%3D0022-3263" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="joceah" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info" href="#"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/joceah?ifr=1"></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollduration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy" style="padding-left: 523.167px;"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292547" title="Adducts">Adducts</a>,</li><li><a href="/action/doSearch?ConceptID=292161" title="Modification">Modification</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="external-alerts-management" style="margin-left: -59px;"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="joceah" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info" href="#"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/joceah?ifr=1"></div><div class="cover-image"><div class="cover-image__image"><a href="/toc/joceah/84/4" title="Go to The Journal of Organic Chemistry "><img src="/cms/10.1021/joceah.2019.84.issue-4/asset/joceah.2019.84.issue-4.largecover.jpg" alt="Go to The Journal of Organic Chemistry "></a></div></div>




        
        <!--Article Page E-Alerts-->
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','joceah');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','joceah');
	  }
  });
</script>
<!--End Article Script-->




        
        <div class="pj-badge"><input type="hidden" id="partnerJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json"></div>




        
        <div class="aJhp_link"><a href="https://pubs.acs.org/joc">The Journal of Organic Chemistry</a></div>
<input type="hidden" id="aJhpJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json">
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=joceah" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0011.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We report the synthesis of <i>N</i><sup>2</sup>-aryl (benzyl, naphthyl, anthracenyl, and pyrenyl)-deoxyguanosine (dG) modified phosphoramidite building blocks and the corresponding damaged DNAs. Primer extension studies using <i>E. coli</i> Pol IV, a translesion polymerase, demonstrate that translesion synthesis (TLS) across these <i>N</i><sup>2</sup>-dG adducts is error free. However, the efficiency of TLS activity decreases with increase in the steric bulkiness of the adducts. Molecular dynamics simulations of damaged DNA-Pol IV complexes reveal the van der Waals interactions between key amino acid residues (Phe13, Ile31, Gly32, Gly33, Ser42, Pro73, Gly74, Phe76, and Tyr79) of the enzyme and adduct that help to accommodate the bulky damages in a hydrophobic pocket to facilitate TLS. Overall, the results presented here provide insights into the TLS across <i>N</i><sup>2</sup>-aryl-dG damaged DNAs by Pol IV.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left ui-resizable" style="width: 730px;"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02463" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02463" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Summary and Conclusions"><span>Summary and Conclusions</span></a></li><li><a href="#_i16" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i36" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i37" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i39" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div class="NLM_p">Polycyclic aromatic hydrocarbons (PAHs) are fused benzenoid ring containing compounds that are well-known for their carcinogenic and mutagenic properties. <a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The exocyclic amino group of deoxyguanosine (dG) acts as a softer reaction center to form major carcinogenic <i>N</i><sup>2</sup>-dG DNA adducts. <a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> These adducts can lead to genomic alterations via stalling replicative polymerases. To avoid any detrimental consequence, cells use specialized low-fidelity DNA polymerases belonging to the Y-family to bypass the damaged sites. This process is called translesion synthesis (TLS), an evolutionarily conserved DNA damage tolerance pathway, which can be error-free or error-prone. <a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Among various TLS polymerases, Pol IV from <i>E. coli</i> is well-known for accurate and efficient bypass of <i>N</i><sup>2</sup>-dG adducts. <a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5â7)</a> Pol IV is an ortholog of human TLS polymerase, Pol Îº. <a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> A systematic study on the effects of the steric bulkiness of <i>N</i><sup>2</sup>-dG adducts on TLS by <i>E. coli</i> Pol IV has yet to be reported.</div><div class="NLM_p">Among the <i>N</i><sup>2</sup>-dG DNA adducts, <i>N</i><sup>2</sup>-benzyl-dG (<i>N</i><sup>2</sup>-Bn-dG, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) is potentially formed from three key carcinogens. <i>N</i>-Nitroso-<i>N</i>-benzylurea (BnNU) and <i>N</i>-nitrosobenzylmethylamine (NBnMA) could form <i>N</i><sup>2</sup>-Bn-dG adducts via benzyldiazonium ion <a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and methyl hydroxylation <a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> pathway, respectively. Whereas, benzyl halides could directly act as a benzylating agent. <a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> Among these carcinogens, NBnMA is believed to form <i>N</i><sup>7</sup>-benzyl-dG adduct in in vivo conditions, since the <i>N</i><sup>7</sup>-position is one of the major site of DNA alkylation. <a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The 9-(sulfoxymethyl)anthracene is an electrophilic metabolite of 9-hydroxymethyl-anthracene, which was alluded to be responsible for the formation of <i>N</i><sup>2</sup>-(9-anthracenylmethyl)-dG adduct (<i>N</i><sup>2</sup>-Anth-dG, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). <a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Halomethyl-anthracenes such as bromo and chloromethyl derivatives have as well been suggested to alkylate the <i>N</i><sup>2</sup>-position of dG. <a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> 1-Methylpyrene, a common environmental pollutant can enzymatically be activated to 1-(hydroxymethyl)pyrene, which in turn can be converted to 1-(sulfoxymethyl)pyrene (SMP). SMP on reaction with DNA leads to the formation of <i>N</i><sup>2</sup>-(1-pyrenylmethyl)-dG adduct (<i>N</i><sup>2</sup>-Pyre-dG, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). <a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> Though <i>N</i><sup>2</sup>-Bn, <i>N</i><sup>2</sup>-Naph, and <i>N</i><sup>2</sup>-Anth (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) adducts are yet to be detected in cellular DNA, the <i>N</i><sup>2</sup>-Pyre-dG adduct was detected in DNA from lung, liver, and kidney tissues of mice. <a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0001.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0001.jpeg" id="gr2" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of <i>N</i><sup>2</sup>-dG adducts which originate from polycyclic aromatic hydrocarbons.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To unravel the molecular mechanism of TLS across <i>N</i><sup>2</sup>-aryl-dG adducts formed by nondiol epoxide metabolic pathways by using various biochemical and biophysical methods, the corresponding modified DNA oligonucleotides are required. Herein we report a short and reliable synthesis of <i>N</i><sup>2</sup>-dG adducts of varying steric bulkiness and the corresponding modified DNAs. Using primer extension assays, molecular dynamics studies, we were able to gain insights on the TLS by <i>E. coli</i> Pol IV across <i>N</i><sup>2</sup>-(Bn, Naph, Anth, Pyre)-dG damaged templates.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58015" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58015" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Summary and Conclusions"><span>Summary and Conclusions</span></a></li><li><a href="#_i16" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i36" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i37" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i39" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div class="NLM_p">To access various <i>N</i><sup>2</sup>-dG damaged DNAs, either phosphoramidite or postoligomerization approaches were reported. <a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16â18)</a> Synthesis of the <i>N</i><sup>2</sup>-Bn-dG adduct was achieved by different methods such as substitution of C2-trimethanesulfonate group bearing dG by benzylamine, <a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> aromatic nucleophilic substitution of 2-fluoro-2â²-deoxyinosine (2-fluoro-dI), <a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> reaction of dG with benzyl bromide, <a onclick="showRef(event, 'ref10 ref21'); return false;" href="javascript:void(0);" class="ref ref10 ref21">(10,21)</a> reduction of <i>N</i><sup>2</sup>-benzoyl group using reducing agents, <a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> reductive amination of dG with benzaldehyde, <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and by Buchwald-Hartwig (BâH) coupling between TBDMS protected 2-bromo-2â²-deoxyinosine (2-bromo-dI) with benzylamine. <a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The <i>N</i><sup>2</sup>-Naph-dG adduct was synthesized using nucleophilic substitution of DMT protected 2-fluoro-dI with the corresponding amine. <a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Synthesis of <i>N</i><sup>2</sup>-Anth-dG adduct was achieved using BâH coupling <a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> or by the hydrolysis of 2â²-deoxy-4-desmethylwyosine. <a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The postoligomerization approach was also utilized for the synthesis of <i>N</i><sup>2</sup>-Anth-dG adduct employing 2-fluoro-dI bearing oligonucleotide. <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The <i>N</i><sup>2</sup>-Pyre-dG adduct was acessed by fluoro substitution method using 2-fluoro-dI convertible nucleoside and 1-(aminomethyl)pyrene. <a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><div class="NLM_p">Our objective was to develop a robust protocol to synthesize <i>N</i><sup>2</sup>-aryl-dG modified nucleoside and corresponding oligonucleotides (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) from a common precursor. To achieve this, first, we attempted various reported methods. However, in our hands, many of those methods were marred by lack of reproducibility, capricious yields, and poor solubility of various intermediates/amidites. <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> In the present work, we report a reliable and efficient method for the synthesis of various <i>N</i><sup>2</sup>-aryl-dG modified phosphoramidite building blocks employing BâH coupling as the key step. These modified building blocks were synthesized in the presence of a versatile <i>para</i>-nitrophenylethyl (NPE) protecting group at the <i>O</i><sup>6</sup>-position of dG. This route is more convenient since all of the bulky modifications can be synthesized from a common precursor with good yields.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Synthesis of <i>N</i><sup>2</sup>-Bn, <i>N</i><sup>2</sup>-Naph, <i>N</i><sup>2</sup>-Anth, and <i>N</i><sup>2</sup>-Pyre-dG Modified Phosphoramidites <b>5a</b>â<b>d</b></h3><div class="NLM_p">The synthetic <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> was utilized to access <i>N</i><sup>2</sup>-aryl-dG (<i>N</i><sup>2</sup>-Bn, <i>N</i><sup>2</sup>-Naph, <i>N</i><sup>2</sup>-Anth, and <i>N</i><sup>2</sup>-Pyre-dG) phosphoramidite building blocks. Using the protected 2-Br-dI, <b>1</b>, as a common precursor, <a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> the CâN bond formation was carried out with different polycyclic aromatic amines in the presence of Pd(OAc)<sub>2</sub>, (<i>R</i>)-BINAP, Cs<sub>2</sub>CO<sub>3</sub> in toluene. <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> For <i>N</i><sup>2</sup>-Bn and <i>N</i><sup>2</sup>-Naph-dG modifications, BâH coupling were carried out using commercially available amine, while for <i>N</i><sup>2</sup>-Anth-dG and <i>N</i><sup>2</sup>-Pyre-dG modifications, the respective amines were prepared by reported methods. <a onclick="showRef(event, 'ref25 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref25 ref29 ref30">(25,29,30)</a> Deprotection of acyl protecting group was accomplished using 33% MeNH<sub>2</sub> in EtOH (v/v) to obtain <i>N</i><sup>2</sup>-aryl-dG diols (<b>3a</b>â<b>d</b>). Final modified phosphoramidites <b>5a</b>â<b>d</b> were synthesized by the protection of the 5â²-OH group using DMT-Cl, followed by phosphitylation reaction using CEP-Cl, DIPEA in DCM.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0008.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0008.jpeg" id="gr1" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <i>N</i><sup>2</sup>-Bn, <i>N</i><sup>2</sup>-Naph, <i>N</i><sup>2</sup>-Anth, and <i>N</i><sup>2</sup>-Pyre-dG Modified Phosphoramidites <b>5a</b>â<b>d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"></p><p class="last"><span class="fn-label"><sup>a</sup></span>Reagent and conditions: (i) Pd(OAc)<sub>2</sub>, (<i>R</i>)-BINAP, respective amines, Cs<sub>2</sub>CO<sub>3</sub>, toluene, 85 Â°C; (ii) 33% MeNH<sub>2</sub> in EtOH, rt; (iii) DMT-Cl, pyridine, rt; (iv) CEP-Cl, DIPEA, DCM, rt.</p><p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Solid Phase Synthesis of <i>N</i><sup>2</sup>-dG Modified Oligonucleotides</h3><div class="NLM_p">The <i>N</i><sup>2</sup>-dG modified phosphoramidite building blocks (<b>5a</b>â<b>5d</b>) were used to synthesize the desired modified oligonucleotides. The presence of versatile NPE protecting group at <i>O</i><sup>6</sup>-position of dG provided enhanced solubility of respective phosphoramidites during solid phase DNA synthesis. <a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> We have chosen the NPE protecting group because of its chemical inertness and its stability in the presence of mild acids and bases. <a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The <i>N</i><sup>2</sup>-aryl-dG damaged DNA sequences (<a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">Table </a><a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">1</a>) were synthesized using automated DNA synthesizer employing appropriate controlled pore glass (CPG) solid supports. The 18-mer DNA sequences (<a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">Table </a><a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">1</a>) were utilized for crystallization with complementary 14-mer primer, incoming nucleotide and <i>E. coli</i> Pol IV. <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> These sequences were designed for easy assembling of the three functional complexes of Pol IV with three different DNA duplexes containing the <i>N</i><sup>2</sup>-dG adduct and incoming nucleotide. The 50-mer <i>N</i><sup>2</sup>-dG DNA template sequences were synthesized for the primer extension studies (<a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">Table </a><a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">1</a>). Deprotection of <i>N</i><sup>2</sup>-dG modified DNAs was carried out in four different steps. The first step involved the selective deprotection of the cyanoethyl group using 10% diethylamine in ACN. <a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The deprotection of the NPE group was performed using 1 M DBU in ACN. <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The cleavage of the CPG support and deprotection of the base protecting groups were carried out separately by treatment of aq. NH<sub>3</sub>. Modified DNAs were PAGE purified and the integrity of modified DNAs was confirmed by ESI-MS or MALDI-TOF (<a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">Table </a><a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">1</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. <i>N</i><sup>2</sup>-dG Damaged Oligonucleotides and Their Molecular Weights<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0009.gif" alt="" id="GRAPHIC-d7e788-autogenerated"><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Structures of modifications <u class="uu">G1</u> (<i>N</i><sup>2</sup>-Bn-dG), <u class="uu">G2</u> (<i>N</i><sup>2</sup>-Naph-dG), <u class="uu">G3</u> (<i>N</i><sup>2</sup>-Anth-dG), and <u class="uu">G4</u> (<i>N</i><sup>2</sup>-Pyre-dG) are shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>. The modified DNAs were characterized by MALDI-TOF in the positive reflectron/linear mode or negative ion electrospray ionization (ESI, only for D2) technique (calculated and found).</p></div></div><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Primer Extension Studies Using <i>E. coli</i> DNA PoI IV</h3><div class="NLM_p">Primer extension reactions with <i>E. coli</i> Pol IV were carried out using a primer-template system as shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A. Reactions were carried out with individual dNTP or with a mixture of dNTPs employing unmodified and modified DNA templates and were monitored at various time courses from 30 s to 2 h (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a> and Figures S1âS5 of the <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>). In the case of single nucleotide incorporation reactions, we observed a prominent extension product of 16 nucleotide length, which corresponds to correct base (dCTP) incorporation across all of the <i>N</i><sup>2</sup>-dG modified templates (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>BâF, lane C). With the use of 4 nM of Pol IV, â¼94% incorporation of the correct base was observed opposite to unmodified dG template at 1.5 h time period (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B, lane C). For the <i>N</i><sup>2</sup>-Bn-dG modification â¼80% (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C, lane C), for the <i>N</i><sup>2</sup>-Naph-dG modification â¼75% (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>D, lane C), for the <i>N</i><sup>2</sup>-Anth-dG modification â¼62% (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>E, lane C), and for the <i>N</i><sup>2</sup>-Pyre-dG modification â¼64% (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>F, lane C) incorporation products were observed after 1.5 h. These results reveal that the TLS employing Pol IV is error free irrespective of the steric bulkiness of the <i>N</i><sup>2</sup>-dG adducts. <a onclick="showRef(event, 'ref5 ref35'); return false;" href="javascript:void(0);" class="ref ref5 ref35">(5,35)</a> However, incorporation efficiency of Pol IV decreases with increasing steric bulkiness from <i>N</i><sup>2</sup>-Bn to <i>N</i><sup>2</sup>-Pyre-dG adducts. It should be noted that, in addition to the correct incorporation, a trace amount of dTTP misincorporation was also observed opposite to the <i>N</i><sup>2</sup>-Bn-dG (â¼10%, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C, lane T) and the <i>N</i><sup>2</sup>-Naph-dG modifications (â¼4%, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>D, lane T).</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0002.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0002.jpeg" id="gr3" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. PAGE (20%, 7 M urea) of primer extension reactions with single and with the mixture of dNTPs using unmodified (dG) and modified (<i>N</i><sup>2</sup>-dG adducts) templates employing <i>E. coli</i>. Pol IV. (A) complete sequence of the primer, P (15-mer) and the template, T (50-mer); (B) reactions with dG template (X = dG; lanes: P, primer; C, dCTP; A, dATP; G, dGTP; T, dTTP; N, mixture of dNTPs); (C) reactions with <i>N</i><sup>2</sup>-Bn-dG template; (D) reactions with <i>N</i><sup>2</sup>-Naph-dG template; (E) reactions with <i>N</i><sup>2</sup>-Anth-dG template; and (F) reactions with <i>N</i><sup>2</sup>-Pyre-dG template. All reactions were carried out at 37 Â°C for 1.5 h time course.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To verify the efficiency of <i>E. coli</i> Pol IV toward full length extension of the primer, running start experiments were carried out using 15-mer and 11-mer primers with the modified DNA templates. Running start experiments using 15-mer primer showed that the DNA Pol IV is unable to synthesize full length products irrespective of the chemical nature of the damage (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>BâF). Similar results were obtained in running start experiments with 11-mer primer (Figures S6 and S7 of the <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>). As expected, in the case of an unmodified template, we observed traces of the full-length product. These results are in line with the low processivity reported for <i>E. coli</i> Pol IV, which is a hallmark of this family of DNA polymerases. <a onclick="showRef(event, 'ref5 ref36'); return false;" href="javascript:void(0);" class="ref ref5 ref36">(5,36)</a> It should be noted that Pol IV belongs to the Y-family of DNA polymerases, and members of this family are known to bypass DNA lesions and rescue replication stalled at damaged nucleotides. The majority of Y-family DNA polymerases serve only to extend the replication fork past the DNA lesion after which they are replaced by replicative DNA polymerases. <a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0003.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0003.jpeg" id="gr4" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. PAGE (20%, 7 M urea) of full-length extension reactions employing Pol IV from <i>E. coli</i> with all dNTPs. (A) Complete sequence of template, T and primer, P (15-mer); (B) reactions with dG template (X = dG); (C) reactions with <i>N</i><sup>2</sup>-Bn-dG template; (D) reactions with <i>N</i><sup>2</sup>-Naph-dG template; (E) reactions with <i>N</i><sup>2</sup>-Anth-dG template; (F) reactions with <i>N</i><sup>2</sup>-Pyre-dG template. Lane P, primer; lanes 1 to 6: primer extension reactions with 250 Î¼M of mixture of dNTPs in different time course from 30 s, 30 min, 1, 2, 4, and 8 h; lane C, 50-mer size standard. All of the reactions were carried out at 37 Â°C.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Molecular Modeling and Dynamics Studies</h3><div class="NLM_p">Molecular dynamics (MD) simulations have been carried out to elucidate the factors that facilitate TLS across <i>N</i><sup>2</sup>-dG DNA adducts by <i>E. coli</i> Pol IV. The reported X-ray crystal structure of furfuryl modified <i>N</i><sup>2</sup>-dG DNA adduct in complex with the <i>E. coli</i> Pol IV (PDB ID: <a href="http://doi.org/10.2210/pdb4Q43/pdb">4Q43</a> <a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a>) was used to prepare the starting structures. The <i>N</i><sup>2</sup>-dG adducts, which were used in the experimental studies (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) were incorporated into the DNA (Figure S8, <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>). A total of four DNA-Pol IV ternary complexes (enzyme, DNA, and incoming nucleotide components) were prepared and were subjected to 100 ns of unrestrained MD simulations (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> and S9, <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>). Complexes were generated only for the insertion stage of dCTP. Trajectory analysis of each of these complexes illuminated the key interactions between the DNA and Pol IV at the active site.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0004.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0004.jpeg" id="gr5" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. DNA-Pol IV ternary complex used for MD simulations. The initial structure of <i>N</i><sup>2</sup>-Bn complex prepared from fdG containing crystal structure (PDB ID: <a href="http://doi.org/10.2210/pdb4Q43/pdb">4Q43</a>). The red colored rectangle is used to highlight the active site and zoomed view of the active site. Other complexes are shown in Figure S9, <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>. Cyan, magenta, green, and yellow are used for polymerase associated domain (PAD) and finger, palm, and thumb domains, respectively. DNA backbone is represented in orange-red color. The <i>N</i><sup>2</sup>-modified residue is highlighted in desire red color. Carbon atoms are represented in white (desire red for <i>N</i><sup>2</sup>-Bn-dG), nitrogen atoms in blue, oxygen atoms in red, and magnesium ions in orange color.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> Stability of the DNAâPol IV Ternary Complexes</h4><div class="NLM_p">To confirm the adequate sampling and the stability of DNA-Pol IV ternary complexes, root-mean-square deviations (RMSD) of backbone atoms of DNA and protein were calculated separately with respect to the first frame. Protein backbone showed marginal deviations with an average RMSD value &lt;1.8 Ã for all of the complexes (Figure S10, <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>). The DNA backbone showed considerable fluctuations with an average RMSD value &gt;2.0 Ã. In order to further investigate such large fluctuations in the DNA backbone, root-mean-square fluctuation (RMSF) values were calculated for DNA with respect to the first frame. Only a few nucleotides which do not interact with protein showed larger fluctuations, whereas the nucleotides around the active site showed fewer fluctuations (Figure S11, <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>). RMSF values of protein residues showed only nominal fluctuations in all of the complexes (Figure S12, <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>). Also, the distance between the Î± phosphate of dCTP and the O atom of 3â²-OH of the primer terminal dG is &lt;4.0 Ã in all of the complexes, indicative of a productive conformation for the nucleotide incorporation at the insertion stage. The incoming nucleotide dCTP forms WatsonâCrick (WâC) H-bonding with dG at the active site. These WâC H-bonds were monitored through the course of dynamics, and the percentage occupancy of each of H-bond was calculated (<a class="ref showTableEvent internalNav" data-id="tbl2" href="#tbl2">Table </a><a class="ref showTableEvent internalNav" data-id="tbl2" href="#tbl2">2</a>). The H-bond (a) formed by dG:<i>N</i><sup>2</sup>-H <b>Â·Â·Â·</b> O2:dCTP is affected to some extent in all the complexes. The interaction between the surrounding amino acids (AAs) with the <i>N</i><sup>2</sup>-modifications, made the orientation of the <i>N</i><sup>2</sup>-modified dG unfavorable for WâC H-bonding at certain time frames during the dynamics. The H-bond labeled (a) in <a class="ref showTableEvent internalNav" data-id="tbl2" href="#tbl2">Table </a><a class="ref showTableEvent internalNav" data-id="tbl2" href="#tbl2">2</a> is most affected in Anth complex in which the H-bond (b) was also disturbed (<a class="ref showTableEvent internalNav" data-id="tbl2" href="#tbl2">Table </a><a class="ref showTableEvent internalNav" data-id="tbl2" href="#tbl2">2</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Percentage Occupancies of Each H-Bond at the Active Site of <i>E. coli</i> Pol IV<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0010.gif" alt="" id="fx1"><div class="scrollable-table-wrap" style="width: 876.3px; display: block;"><table class="table "><colgroup><col align="left"><col align="center"><col align="center"><col align="center"></colgroup><thead><tr class="colsep0" valign="top"><th class="colsep0 rowsep0" align="center">complex with adduct</th><th class="colsep0 rowsep0" align="center">a</th><th class="colsep0 rowsep0" align="center">b</th><th class="colsep0 rowsep0" align="center">c</th></tr></thead><tbody><tr class="colsep0" valign="top"><td class="colsep0 rowsep0" align="left"><i>N</i><sup>2</sup>-Bn-dG</td><td class="colsep0 rowsep0" align="center">92.08%</td><td class="colsep0 rowsep0" align="center">95.26%</td><td class="colsep0 rowsep0" align="center">93.01%</td></tr><tr class="colsep0" valign="top"><td class="colsep0 rowsep0" align="left"><i>N</i><sup>2</sup>-Naph-dG</td><td class="colsep0 rowsep0" align="center">89.88%</td><td class="colsep0 rowsep0" align="center">95.59%</td><td class="colsep0 rowsep0" align="center">91.77%</td></tr><tr class="colsep0" valign="top"><td class="colsep0 rowsep0" align="left"><i>N</i><sup>2</sup>-Anth-dG</td><td class="colsep0 rowsep0" align="center">84.58%</td><td class="colsep0 rowsep0" align="center">88.16%</td><td class="colsep0 rowsep0" align="center">93.04%</td></tr><tr class="colsep0" valign="top"><td class="colsep0 rowsep0" align="left"><i>N</i><sup>2</sup>-Pyre-dG</td><td class="colsep0 rowsep0" align="center">88.18%</td><td class="colsep0 rowsep0" align="center">95.94%</td><td class="colsep0 rowsep0" align="center">92.27%</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">The dotted lines indicate the H-bond present between the dG and incoming nucleotide dCTP. Red arrow indicates the position of modification.</p></div></div></div></div><div id="sec2_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Clustering Analysis</h4><div class="NLM_p">MD snapshots (100 ns) of each of the complexes were clustered into 5 ensembles based on hierarchical agglomerative approach. <a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The percentage of each ensemble out of 100 ns of dynamics is tabulated in <a class="ref showTableEvent internalNav" data-id="tbl3" href="#tbl3">Table </a><a class="ref showTableEvent internalNav" data-id="tbl3" href="#tbl3">3</a>. To identify the differences in the major ensembles, representative structures of ensemble 1 and ensemble 2 were superimposed for all of the complexes. These representative structures differed primarily in the orientations of terminal nucleotides of DNA. The similarities in the orientations of the AAs surrounding the modifications are highlighted in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>. The residues surrounding <i>N</i><sup>2</sup>-Anth-dG adduct showed least differences compared to other adducts. Minor differences were observed in the orientations of Ser42 and Phe76 residues in the other complexes. <a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Percentage Occupancies of Each Ensemble out of 100 ns of Dynamics<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap" style="width: 876.3px; display: block;"><table class="table "><colgroup><col align="left"><col align="center"><col align="center"><col align="center"></colgroup><thead><tr class="colsep0" valign="top"><th class="colsep0 rowsep0" align="center">complex with adduct</th><th class="colsep0 rowsep0" align="center">ensemble&nbsp;1</th><th class="colsep0 rowsep0" align="center">ensemble&nbsp;2</th><th class="colsep0 rowsep0" align="center">ensemble&nbsp;3</th></tr></thead><tbody><tr class="colsep0" valign="top"><td class="colsep0 rowsep0" align="left"><i>N</i><sup>2</sup>-Bn-dG</td><td class="colsep0 rowsep0" align="center">43.3%</td><td class="colsep0 rowsep0" align="center">35.4%</td><td class="colsep0 rowsep0" align="center">14.1%</td></tr><tr class="colsep0" valign="top"><td class="colsep0 rowsep0" align="left"><i>N</i><sup>2</sup>-Naph-dG</td><td class="colsep0 rowsep0" align="center">59.9%</td><td class="colsep0 rowsep0" align="center">23.0%</td><td class="colsep0 rowsep0" align="center">7.2%</td></tr><tr class="colsep0" valign="top"><td class="colsep0 rowsep0" align="left"><i>N</i><sup>2</sup>-Anth-dG</td><td class="colsep0 rowsep0" align="center">56.4%</td><td class="colsep0 rowsep0" align="center">27.9%</td><td class="colsep0 rowsep0" align="center">8.2%</td></tr><tr class="colsep0" valign="top"><td class="colsep0 rowsep0" align="left"><i>N</i><sup>2</sup>-Pyre-dG</td><td class="colsep0 rowsep0" align="center">47.8%</td><td class="colsep0 rowsep0" align="center">31.8%</td><td class="colsep0 rowsep0" align="center">17.8%</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Every second frame was considered, and a total of 25â¯000 frames were considered for the calculation. Clustering was performed using hierarchical agglomerative approach, which was implemented by the CPPTRAJ <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> module of AMBER14 <a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></p></div></div></div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0005.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0005.jpeg" id="gr6" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Superposition of representative structures of ensembles 1 and 2 of <i>N</i><sup>2</sup>-dG-modified DNA-Pol IV ternary complexes. Only residues of interest are highlighted. (A) <i>N</i><sup>2</sup>-Bn complex; (B) <i>N</i><sup>2</sup>-Naph complex; (C) <i>N</i><sup>2</sup>-Anth complex;  and (D) <i>N</i><sup>2</sup>-Pyre complex. Lime green colored cartoon representation is used for the ensemble 1 and light pink colored cartoon representation is used for the ensemble 2. DNA backbone is represented in white for both the representative structures. Pale green colored carbon framework is used for <i>N</i><sup>2</sup>-adducts for ensemble 1 and pale pink colored carbon framework for ensemble 2. Nitrogen atoms are represented in blue and oxygen atoms in red. All of the hydrogens are removed for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To find out the similarities between the four ternary complexes at the active site, the representative structures of ensemble 1 of all of the complexes were superimposed onto each other (Figure S13, <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>). Orientations of the <i>N</i><sup>2</sup>-modified moieties were found to be strikingly similar to that of the furfuryl adduct in the active site of DNA-Pol IV complex as revealed by its crystal structure (PDBID: 4Q43). This implies that the Pol IV orients <i>N</i><sup>2</sup>-modified moieties into a particular conformation to facilitate TLS. To validate this, the the dihedral angle labeled Xi (Î¾) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>) was monitored through the course of dynamics. Bn, Naph, and Pyre complexes appeared to follow a pattern, but the Anth complex showed slight deviations. This difference might be a result of the unique position of linkage of the anthracene moiety to the nucleobase (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). The average dihedral angle values of 103.18Â°, 107.58Â°, 148.13Â°, and 96.31Â° were observed for Bn, Naph, Anth, and Pyre complexes, respectively. Based on the above results, few important AA residues were identified, and the trajectory was further analyzed to understand the interaction patterns between these adducts and AAs and also to probe the nature of the forces responsible for the accommodation of adducts in the active site.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0006.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0006.jpeg" id="gr7" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Plot of dihedral Xi (Î¾, shown above using red arrow) vs time for all the complexes. Black, red, blue, and pink colors were used to represent <i>N</i><sup>2</sup>-Bn, -Naph, -Anth, and -Pyre complexes, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Key Noncovalent Interactions and Interaction Energies</h4><div class="NLM_p">Average interaction energies of the <i>N</i><sup>2</sup>-dG adducts with the surrounding AA residues were calculated to identify the nature of interactions in the active site. Since our focus is the modified nucleobases, only residues in close proximity to these modifications (within 6 Ã) were chosen for the calculations. Interaction energies were calculated using the MM/GBSA method implemented in AmberTools package. The total average energy was calculated by summation of van der Waals, electrostatic, polar solvation, and nonpolar solvation values (internal = 0 for reference). The average energy values presented in Table S1, <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>, helped to qualitatively identify the chemical nature of these interactions. The results suggest that Ser42 interacts with the dG adducts primarily through van der Waals interactions. <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Upon visual inspection of trajectories, Ser42 was found to interact with Ï cloud of aromatic moiety through CâHÂ·Â·Â·Ï interactions. <a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> To verify this, the distance between C<sup>Î²</sup> of Ser42 and the centroid of the <i>N</i><sup>2</sup>-modified aromatic moiety was plotted for all of the complexes (Figure S14, <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>). The distances were in the appreciable range, i.e., &lt;4 Ã in all of the complexes, favorable for van der Waals interactions. In fact, there is a correlation between this distance and the dihedral Î¾ responsible for the particular orientation of the adducts (Figure S13, <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>). When the distance is between 3 and 4 Ã, the orientation of aromatic moiety favors strong Ï-cloud interactions with the Ser42 residue.</div><div class="NLM_p">Apart from Ser42, the other major interacting residues displayed similar noncovalent contacts. Phe76 played a crucial role in the complexes with high steric bulkiness, primarily by forming T-shaped CâHÂ·Â·Â·Ï interactions with the adducts (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>). The Phe13 also exhibited CâHÂ·Â·Â·Ï interactions. This residue exhibited significant interactions in Anth complex compared to moderate interactions found in Bn, Anth, and Pyre complexes (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>). Similarly, the CâH bond of Î± carbon (C<sup>Î±</sup>) from Gly32 showed CâHÂ·Â·Â·Ï interactions in Naph and Pyre (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>B,D) complexes. <a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The role of Tyr79 was found to be strategic as it interacts with Phe13 through parallel displaced stacking and with Phe76 through T-shaped interactions, which add strong hydrophobicity to the surroundings (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>). Ile 31 and Pro73 were also found to form moderate to weak interactions in addition to weak interactions from Gly33 and Gly74 in all of the complexes. The RMSF values for these residues in each of the complexes are reported in <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Table S2</a>. The RMSF values were mostly &lt;1 Ã, indicating the importance of these residues in accommodating the adducts.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0007.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0007.jpeg" id="gr8" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. van der Waals interactions between AAs of Pol IV and <i>N</i><sup>2</sup>-adducts found in DNA-Pol IV ternary complexes. (A) <i>N</i><sup>2</sup>-Bn complex; (B) <i>N</i><sup>2</sup>-Naph complex; (C) <i>N</i><sup>2</sup>-Anth complex;  and (D) <i>N</i><sup>2</sup>-Pyre complex. Phe76 interacts with the <i>N</i><sup>2</sup>-adduct through T-shaped CâH Â·Â·Â·Ï interactions in the Bn, Naph, Anth, and Pyre complexes. Carbon atom linked to OâH of Ser42 interacts with Ï cloud of Naph, Anth, and Pyre moieties. The backbone atom of Gly32 as well interacts with adducts through CâHÂ·Â·Â· Ï interactions in Naph and Pyre complexes. Phe13 as well forms T-shaped interactions with Anth moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The representative structures of other ensembles of each complex had similar interaction patterns as discussed above. Overall, the van der Waals interactions are the major forces responsible for the accommodation of <i>N</i><sup>2</sup>-modified moieties in the active site of Pol IV. The residues Phe13, Ile31, Gly32, Gly33, Ser42, Pro73, Gly74, Phe76, and Tyr79 were found to form a hydrophobic pocket <a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> into which the adducts are accommodated and our current MD results support the same. The role of Ser42 was elucidated before, <a onclick="showRef(event, 'ref6 ref41'); return false;" href="javascript:void(0);" class="ref ref6 ref41">(6,41)</a> and our results from the MD simulations underscore the importance of this residue in stabilizing nonpolar adducts through van der Waals interactions. General interaction patterns from Phe13, Phe76, Tyr79, and Gly32 were also identified. In summary, our MD studies provide insights on the molecular interactions that are responsible for the accommodation of <i>N</i><sup>2</sup>-dG adducts and their bypass by <i>E. coli</i> Pol IV, which are in line with the results emerged from the experiments.</div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Summary and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25882" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25882" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Summary and Conclusions"><span>Summary and Conclusions</span></a></li><li><a href="#_i16" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i36" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i37" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i39" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div class="NLM_p last">In conclusion, we have developed a robust and reliable protocol for the synthesis of <i>N</i><sup>2</sup>-aryl-dG modified DNAs. These modified DNAs were utilized to investigate the effect of steric bulkiness of the <i>N</i><sup>2</sup>-dG adducts on the TLS by Pol IV from <i>E. coli</i>. The Pol IV was found to be efficient in bypassing these adducts with high fidelity but with low processivity. Molecular modeling and dynamics studies were also carried out by incorporating these <i>N</i><sup>2</sup>-aryl-dG DNA adducts at the active site of <i>E. coli</i> Pol IV. The results revealed a common mechanism used by the enzyme that involves using its hydrophobic pocket at the active site to accommodate these modifications resulting in the efficient bypass of adducts. Further structural studies to validate these observations are currently being pursed.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21604" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21604" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Summary and Conclusions"><span>Summary and Conclusions</span></a></li><li><a href="#_i16" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i36" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i37" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i39" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div class="NLM_p">All chemicals and dry solvents were obtained from commercial sources and used without any further purification. Toluene, ACN, DCM, Et<sub>3</sub>N, DIPEA, dioxane, and pyridine were dried using calcium hydride. Thin layer chromatography (TLC) was performed on silica gel plates precoated with fluorescent indicator with visualization by UV light (at 260 nm) or by dipping into a solution of 5% conc. H<sub>2</sub>SO<sub>4</sub> in EtOH (v/v) and heating. Silica gel (100â200 mesh) was used for column chromatography. <sup>1</sup>H NMR (500 or 400 MHz), <sup>13</sup>C NMR (125 or 100 MHz), and <sup>31</sup>P NMR (202 or 162 MHz) were recorded on 500 or 400 MHz instruments. Chemical shifts in parts per million (Î´) are reported downfield from TMS (0 ppm) and referenced to the TMS signal or residual proton signal of the deuterated solvent as follows: CD<sub>3</sub>OD (3.31 ppm) for <sup>1</sup>H NMR spectra and CDCl<sub>3</sub> (77.2 ppm) or CD<sub>3</sub>OD (49.1 ppm) for <sup>13</sup>C NMR spectra. Multiplicities of <sup>1</sup>H NMR spin couplings are reported as s for singlet, br s for broad singlet, d for doublet, t for triplet, br t for broad triplet, dd for doublet of doublet, ddd for doublet of doublet of doublets, quint for quintet or m for multiplet and overlapping spin systems. Values for apparent coupling constants (<i>J</i>) are reported in Hz. High resolution mass spectra (HRMS) were obtained from Q-TOF analyzer in positive ion electrospray ionization (ESI) mode. All of the modified DNA sequences were synthesized using an automated solid phase synthesizer. Mass spectra of DNA oligonucleotides were obtained by MALDI (in positive linear or reflectron mode) or ESI (in negative mode). <i>E. coli</i> Pol IV was expressed and purified in house. <a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Procedure for the Buchwald-Hartwig Coupling and Deacetylation</h3><div class="NLM_p">In a screw capped tube, Pd(OAc)<sub>2</sub> (0.1 equiv) and (<i>R</i>)-BINAP (0.3 equiv) were taken and flushed with N<sub>2</sub> gas followed by addition of dry toluene (9.9 mL/mmol). Reaction mixture was stirred at room temperature for 5 min. Cs<sub>2</sub>CO<sub>3</sub> (1.4 equiv), respective amines (1.1 equiv), and bromo nucleoside <b>1</b> (1 equiv) were added in the sequential manner. Further, the tube was purged with N<sub>2</sub> gas, sealed with a Teflon-lined cap, and heated in an oil bath at 85 Â°C. After completion of reaction, the reaction mixture was passed through a Celite pad and washed with EtOAc (250 mL). The filtrate was concentrated to give a crude BâH coupled product. Further, the crude compound was subjected directly to deacetylation using 33% MeNH<sub>2</sub> in EtOH (v/v, 22 mL/mmol) and stirred at room temperature. After completion of the reaction, the mixture was concentrated under reduced pressure. The oily residue was purified by silica gel column chromatography to obtain deprotected nucleosides.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> <i>N</i><sup>2</sup>-Benzyl<i>-O</i><sup>6</sup>-(2-(4-nitrophenyl)ethyl)-2â²-deoxyguanosine (<b>3a</b>)</h4><div class="NLM_p last">Pd(OAc)<sub>2</sub> (5.94 mg, 0.02 mmol), (<i>R</i>)-BINAP (50 mg, 0.07 mmol), dry toluene (3 mL), Cs<sub>2</sub>CO<sub>3</sub> (131 mg, 0.37 mmol), benzylamine (0.03 mL, 0.29 mmol), and bromo nucleoside <b>1</b> (150 mg, 0.26 mmol) were used and heated for 18 h to give a brown crude compound <b>2a</b> (156 mg, 0.26 mmol), which was further dissolved in 33% MeNH<sub>2</sub> in EtOH (5 mL) and stirred at room temperature for 3 h. Silica gel column chromatography (4% MeOH in DCM) yielded slightly brownish solid <b>3a</b> (76 mg, 57% after two steps). <i>R</i><sub>f</sub> = 0.47 (10% MeOH in DCM); mp: 60â63 Â°C; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.12â8.07 (m, 2H), 7.61 (s, 1H), 7.33 (d, <i>J</i> = 8.6 Hz, 2H), 7.31â7.21 (m, 5H), 6.19 (dd, <i>J</i> = 9.4, 5.6 Hz, 1H), 5.58 (br t, 1H), 4.70 (d, <i>J</i> = 5 Hz, 1H), 4.64 (t, <i>J</i> = 6.9 Hz, 2H), 4.60 (dd, <i>J</i> = 5.9, 2.4 Hz, 2H), 4.15 (s, 1H), 3.92 (dd, <i>J</i> = 12.6, 1.8 Hz, 1H), 3.71 (d, <i>J</i> = 11.3 Hz, 1H), 3.16 (t, <i>J</i> = 6.7 Hz, 2H), 2.97 (ddd, <i>J</i> = 14.2, 9.1, 5.1 Hz, 1H), 2.23 (dd, <i>J</i> = 12.6, 5.6 Hz, 1H); <sup>13</sup>C{1H} NMR (100 MHz, CDCl<sub>3</sub>): Î´ 160.1, 158.5, 152.8, 146.9, 145.9, 139.3, 139.1, 130, 128.7, 127.3, 127.3, 123.8, 116.5, 89.1, 87.3, 73.3, 66.1, 63.5, 46.1, 40.2, 35.1; HRMS (ESI): Calcd for C<sub>25</sub>H<sub>27</sub>N<sub>6</sub>O<sub>6</sub>, [M + H]<sup>+</sup> 507.1992; found, [M + H]<sup>+</sup> 507.1980 (Î<i>m</i> â 0.0012, error â2.4 ppm).</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> <i>N</i><sup>2</sup>-(1-Naphthylmethyl)-<i>O</i><sup>6</sup>-(2-(4-nitrophenyl)ethyl)-2â²-deoxyguanosine (<b>3b</b>)</h4><div class="NLM_p last">The Pd(OAc)<sub>2</sub> (11 mg, 0.04 mmol), (<i>R</i>)-BINAP (88 mg, 0.14 mmol), dry toluene (7 mL), Cs<sub>2</sub>CO<sub>3</sub> (214 mg, 0.65 mmol), 1-naphthylmethylamine (0.07 mL, 0.51 mmol), and bromo nucleoside <b>1</b> (269 mg, 0.47 mmol) were used and heated for 21 h to give a dark brown crude compound <b>2b</b> (260 mg, 0.40 mmol), which was further dissolved in 33% MeNH<sub>2</sub> in EtOH (9 mL) and stirred at room temperature for 2.5 h. Silica gel column chromatography (4% MeOH in DCM) yielded yellow solid <b>3b</b> (171 mg, 64%, after two steps). <i>R</i><sub>f</sub> = 0.49 (10% MeOH in DCM); mp: 67â70 Â°C; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.09â8.02 (m, 1H), 7.97 (d, <i>J</i> = 8.3 Hz, 2H), 7.92â7.85 (m, 1H), 7.77 (d, <i>J</i> = 8.3 Hz, 1H), 7.60 (s, 1H), 7.57â7.36 (m, 6H), 7.16 (br s, 1H), 6.18 (dd, <i>J</i> = 9.2, 5.6 Hz, 1H), 5.51 (br s, 1H), 5.14â4.99 (m, 2H), 4.67 (d, <i>J</i> = 5.3 Hz, 1H), 4.60 (t, <i>J</i> = 7.0 Hz, 2H), 4.33 (s, 1H), 4.13 (s, 1H), 3.90 (d, <i>J</i> = 12 Hz, 1H), 3.68 (dd, <i>J</i> = 12.5, 1.8 Hz, 1H), 3.07 (t, <i>J</i> = 6.7 Hz, 2H), 3.00 (ddd, <i>J</i> = 14, 9.5, 5.5 Hz, 1H), 2.22 (dd, <i>J</i> = 13.4, 5.6 Hz, 1H); <sup>13</sup>C{1H} NMR (100 MHz, CDCl<sub>3</sub>): Î´ 161, 158.4, 153, 146.8, 145.6, 139.2, 134.2, 133.9, 131.3, 129.8, 129, 128.1, 126.4, 126, 125.5, 125.1, 123.7, 123.1, 116.4, 89, 87, 73, 66.1, 63.3, 44, 40.1, 35; HRMS (ESI): Calcd for C<sub>29</sub>H<sub>29</sub>N<sub>6</sub>O<sub>6</sub>, [M + H]<sup>+</sup> 557.2149; found, [M + H]<sup>+</sup> 557.2156 (Î<i>m</i> + 0.0007, error +1.3 ppm).</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> <i>N</i><sup>2</sup>-(9-Anthracenylmethyl)-<i>O</i><sup>6</sup>-(2-(4-nitrophenyl)ethyl)-2â²-deoxyguanosine (<b>3c</b>)</h4><div class="NLM_p last">Pd(OAc)<sub>2</sub> (3.9 mg, 0.01 mmol), (<i>R</i>)-BINAP (33 mg, 0.05 mmol), dry toluene (2 mL), Cs<sub>2</sub>CO<sub>3</sub> (87 mg, 0.24 mmol), 9-(aminomethyl)-anthracene <a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> (41 mg, 0.19 mmol), and bromo nucleoside <b>1</b> (100 mg, 0.17 mmol) were used and heated for 8 h to give a dark brown crude compound <b>2c</b> (127 mg, 0.18 mmol), which was further dissolved in 33% MeNH<sub>2</sub> in EtOH (3.7 mL) and stirred at room temperature for 3 h. Column chromatography (3% MeOH in DCM) yielded slightly greenish compound <b>3c</b> (59 mg, 54% after two steps). <i>R</i><sub>f</sub> = 0.50 (10% MeOH in DCM); mp: 83â86 Â°C; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.47 (s, 1H), 8.32â8.27 (m, 2H), 8.12 (d, <i>J</i> = 8.5 Hz, 2H), 8.03 (dd, <i>J</i> = 8.5, 1.0 Hz, 2H), 7.64 (s, 1H), 7.54â7.42 (m, 6H), 6.18 (br t, 1H), 5.57â5.47 (m, 2H), 5.18 (t, <i>J</i> = 4.4 Hz, 1H), 4.90â4.71 (m, 2H), 4.63 (br s, 1H), 4.07 (br s, 1H), 3.88â3.77 (m, 1H), 3.63â3.53 (m, 1H), 3.29 (br s, 2H), 3.12â2.92 (m, 1H), 2.25â2.13 (m, 1H); <sup>13</sup>C{1H} NMR (100 MHz, CDCl<sub>3</sub>): Î´ 161.1, 158.3, 153.1, 146.9, 145.9, 139.2, 131.6, 130.6, 130, 129.3, 128.7, 128.2, 126.6, 125.2, 124.1, 123.8, 116.5, 88.8, 87, 73, 66.3, 63.2, 40.1, 38.9, 35.3; HRMS (ESI): Calcd for C<sub>33</sub>H<sub>31</sub>N<sub>6</sub>O<sub>6</sub>, [M + H]<sup>+</sup> 607.2300; found, [M + H]<sup>+</sup> 607.2297 (Î<i>m</i> â 0.0003, error â0.5 ppm).</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> <i>N</i><sup>2</sup>-(1-Pyrenylmethyl)-<i>O</i><sup>6</sup>-(2-(4-nitrophenyl)ethyl)-2â²-deoxyguanosine (<b>3d</b>) <a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></h4><div class="NLM_p last">Pd(OAc)<sub>2</sub> (16 mg, 0.07 mmol), (<i>R</i>)-BINAP (132 mg, 0.21 mmol), dry toluene (7 mL), Cs<sub>2</sub>CO<sub>3</sub> (323 mg, 0.99 mmol), 1-(aminomethyl)pyrene <a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> (182 mg, 0.78 mmol), and bromo nucleoside <b>1</b> (403 mg, 0.71 mmol) were used and heated in an oil bath for 21 h at 85 Â°C to get crude compound <b>2d</b> (547 mg, 0.76 mmol), which was further dissolved in 33% MeNH<sub>2</sub> in EtOH (15 mL) and stirred at room temperature for 1 h 15 min. Column chromatography on a silica gel (2.5% MeOH in DCM) yielded yellow compound <b>3d</b> (202 mg, 45% after two steps). <i>R</i><sub>f</sub> = 0.49 (10% MeOH in DCM); mp: 75â78 Â°C; <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub><i>+</i> CDCl<sub>3</sub>): Î´ 8.24 (d, <i>J</i> = 9.1 Hz, 1H), 8.16â8.09 (m, 2H), 8.08â7.89 (m, 7H), 7.72 (s, 1H), 7.66 (br s, 2H), 6.86 (br s, 1H), 6.22 (dd, <i>J</i> = 8.9, 5.9 Hz, 1H), 5.23 (s, 2H), 4.55 (d, <i>J</i> = 5.4 Hz, 1H), 4.38 (t, <i>J</i> = 6.6 Hz, 2H), 4.09 (d, <i>J</i> = 1.3 Hz, 1H), 3.88 (d, <i>J</i> = 12.4 Hz, 1H), 3.73â3.66 (m, 2H), 3.58â3.49 (m, 1H), 2.88 (ddd, <i>J</i> = 14.5, 9.1, 5.6 Hz, 1H), 2.30â2.20 (m, 1H); <sup>13</sup>C{1H} NMR (100 MHz, MeOH-<i>d</i><sub>4</sub><i>+</i> CDCl<sub>3</sub>): Î´ 160.9, 158.5, 152.9, 146.4, 145.5, 139.1, 132.4, 131.3, 130.7, 129.4, 128.4, 127.9, 127.4, 127.2, 126.2, 125.4, 125.2, 124.9, 124.8, 123.4, 122.5, 115.9, 88.8, 87, 72.5, 65.9, 63.2, 44, 40.1, 34.7; HRMS (ESI): Calcd for C<sub>35</sub>H<sub>31</sub>N<sub>6</sub>O<sub>6</sub>, [M + H]<sup>+</sup> 631.2300; found, [M + H]<sup>+</sup> 631.2293 (Î<i>m</i> â 0.0007, error â1 ppm).</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> General Procedure for the DMT-Protection</h3><div class="NLM_p">The <i>N</i><sup>2</sup>-modified-dG nucleosides (1 equiv) was coevaporated with dry pyridine (5 mL) and dissolved in the same solvent (10 mL/mmol). To this DMT-Cl (1.22 equiv) was added and stirred at room temperature. Further, the reaction mixture was diluted with DCM (100 mL) and washed with saturated NaHCO<sub>3</sub> (40 mL) and water (2 Ã 50 mL). The DCM layer was dried over Na<sub>2</sub>SO<sub>4</sub> and evaporated. The crude DMT-protected compound was purified by silica gel column chromatography (using appropriate solvents along with 2% Et<sub>3</sub>N) to afford pure compound.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> <i>N</i><sup>2</sup>-Benzyl<i>-O</i><sup>6</sup>-(2-(4-nitrophenyl)ethyl)-5â²-(4,4â²-dimethoxytrityl)-2â²-deoxyguanosine (<b>4a</b>)</h4><div class="NLM_p last">Compound <b>3a</b> (251 mg, 0.49 mmol) in dry pyridine (5 mL) and DMT-Cl (332 mg, 0.99 mmol) was stirred for 24 h. Column chromatography (90% DCM in pet. ether + 2% Et<sub>3</sub>N) yielded compound <b>4a</b> as a pale yellow solid (237 mg, 60%). <i>R</i><sub>f</sub> = 0.55 (2% MeOH in DCM + 2% Et<sub>3</sub>N); mp: 71â74 Â°C; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.13â8.09 (m, 2H), 7.68 (s, 1H), 7.39 (d, <i>J</i> = 7.3 Hz, 4H), 7.31â7.26 (m, 8H), 7.25â7.14 (m, 4H), 6.78 (d, <i>J</i> = 8.0 Hz, 4H), 6.27 (t, <i>J</i> = 6.7 Hz, 1H), 5.17 (t, <i>J</i> = 5.5 Hz, 1H), 4.66 (t, <i>J</i> = 6.8 Hz, 2H), 4.63â4.58 (m, 1H), 4.53 (t, <i>J</i> = 5.8 Hz, 2H), 4.09â4.04 (m, 1H), 3.76 (s, 6H), 3.38 (dd, <i>J</i> = 9.8, 4.8 Hz, 1H), 3.34 (dd, <i>J</i> = 10.1, 5.4 Hz, 1H), 3.20 (t, <i>J</i> = 6.8 Hz, 2H), 2.87â2.81 (m, 1H), 2.38 (ddd, <i>J</i> = 13.2, 6.2, 4 Hz, 1H); <sup>13</sup>C{1H} NMR (100 MHz, CDCl<sub>3</sub>): Î´ 160.6, 158.8, 158.7, 153.8, 146.9, 146.1, 144.7, 139.6, 137.9, 135.8, 130.1, 130, 128.6, 128.2, 128, 127.4, 127.2, 127, 123.8, 115.5, 113.3, 86.6, 85.9, 84, 72.8, 66, 64.1, 55.3, 46.9, 39.5, 35.3. HRMS (ESI): Calcd for C<sub>46</sub>H<sub>45</sub>N<sub>6</sub>O<sub>8</sub>, [M + H]<sup>+</sup> 809.3299; found, [M + H]<sup>+</sup> 809.3329 (Î<i>m</i> + 0.003, error +3.7 ppm).</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>N</i><sup>2</sup>-(1-Naphthylmethyl)<i>-O</i><sup>6</sup>-(2-(4-nitrophenyl)ethyl)-5â²-(4,4â²-dimethoxytrityl)-2â²-deoxyguanosine (<b>4b</b>)</h4><div class="NLM_p last">Compound <b>3b</b> (171 mg, 0.30 mmol) in dry pyridine (3 mL), DMT-Cl (126 mg, 0.37 mmol) were stirred for 10 h. Column chromatography (1% MeOH in DCM + 2% Et<sub>3</sub>N) yielded compound <b>4b</b> as a yellow solid (154 mg, 60%). <i>R</i><sub>f</sub> = 0.50 (2% MeOH in DCM + 2% Et<sub>3</sub>N); mp: 85â88 Â°C; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.08â7.98 (m, 3H), 7.91â7.86 (m, 1H), 7.77 (d, <i>J</i> = 8.0 Hz, 1H), 7.70 (s, 1H), 7.55â7.46 (m, 2H), 7.44â7.35 (m, 4H), 7.30â7.26 (m, 5H), 7.26â7.13 (m, 4H), 6.80â6.72 (m, 4H), 6.29 (t, <i>J</i> = 6.6 Hz, 1H), 5.24â5.17 (m, 1H), 5.06â4.88 (m, 2H), 4.63 (t, <i>J</i> = 6.9 Hz, 2H), 4.56 (br s, 1H), 4.08â4.01 (m, 1H), 3.74 (d, <i>J</i> = 1.3 Hz, 6H), 3.68â3.63 (m, 1H), 3.39 (dd, <i>J</i> = 10, 5 Hz, 1H), 3.33 (dd, <i>J</i> = 9.8, 4.8 Hz, 1H), 3.14 (br s, 2H), 2.85â2.74 (m, 1H), 2.38 (ddd, <i>J</i> = 13.5, 6.5, 4.5 Hz, 1H); <sup>13</sup>C{1H} NMR (100 MHz, CDCl<sub>3</sub>): Î´ 160.7, 158.7, 158.6, 146.8, 146, 144.7, 137.9, 135.8, 134.9, 134.5, 133.9, 131.4, 130.1, 129.8, 129, 128.2, 128.1, 128, 127, 126.5, 126, 125.6, 123.8, 123.4, 115.7, 113.3, 86.6, 85.8, 84, 72.7, 66, 64.1, 55.3, 44.1, 39.6, 35.2; HRMS (ESI): Calcd for C<sub>50</sub>H<sub>47</sub>N<sub>6</sub>O<sub>8</sub>, [M + H]<sup>+</sup> 859.3455; found, [M + H]<sup>+</sup> 859.3478 (Î<i>m</i> + 0.0023, error +2.7 ppm).</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> <i>N</i><sup>2</sup>-(9-Anthracenylmethyl)<i>-O</i><sup>6</sup>-(2-(4-nitrophenyl)ethyl)-5â²-(4,4â²-dimethoxytrityl)-2â²-deoxyguanosine (<b>4c</b>)</h4><div class="NLM_p last">Compound <b>3c</b> (162 mg, 0.26 mmol) in dry pyridine (3 mL) and DMT-Cl (180 mg, 0.53 mmol) was stirred for 10 h. Column chromatography (1% MeOH in DCM + 2% Et<sub>3</sub>N) yielded compound <b>4c</b> as a slightly greenish solid (177 mg, 72%). <i>R</i><sub>f</sub> = 0.50 (2% MeOH in DCM + 2% Et<sub>3</sub>N); mp: 86â89 Â°C; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): Î´ 8.46 (s, 1H), 8.29â8.24 (m, 2H), 8.08 (d, <i>J</i> = 7.9 Hz, 2H), 8.05â8.00 (m, 2H), 7.75 (s, 1H), 7.50â7.36 (m, 8H), 7.28 (dd, <i>J</i> = 5.8, 2.7 Hz, 3H), 7.20â7.15 (m, 3H), 7.12â7.08 (m, 1H), 6.73 (d, <i>J</i> = 7.3 Hz, 4H), 6.46â6.28 (m, 1H), 5.51â5.36 (m, 2H), 5.14 (t, <i>J</i> = 4.0 Hz, 1H), 4.74â4.59 (m, 2H), 4.09 (br s, 1H), 3.70 (d, <i>J</i> = 1.5 Hz, 6H), 3.46â3.40 (m, 1H), 3.38â3.30 (m, 1H), 3.26â3.15 (br s, 2H), 2.91â2.85 (m, 1H), 2.52â2.44 (m, 1H); <sup>13</sup>C{1H} NMR (125 MHz, CDCl<sub>3</sub>): Î´ 160.7, 158.6, 153.9, 146.8, 146.1, 144.6, 137.7, 135.8, 135.7, 131.6, 130.5, 130.1, 130, 130, 129.4, 129.3, 129.1, 128.3, 128.2, 128, 127, 126.5, 125.2, 124.2, 123.8, 115.5, 113.2, 86.6, 85.9, 83.9, 72.8, 66, 64.1, 55.3, 45.9, 38.7, 35.3; HRMS (ESI): Calcd for C<sub>54</sub>H<sub>49</sub>N<sub>6</sub>O<sub>8</sub>, [M + H]<sup>+</sup> 909.3606; found, [M + H]<sup>+</sup> 909.3608 (Î<i>m</i> + 0.0002, error +0.1 ppm).</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i><sup>2</sup>-(1-Pyrenylmethyl)<i>-O</i><sup>6</sup>-(2-(4-nitrophenyl)ethyl)-5â²-(4,4â²-dimethoxytrityl)-2â²-deoxyguanosine (<b>4d</b>)</h4><div class="NLM_p last">Compound <b>3d</b> (151 mg, 0.24 mmol) in dry pyridine (3 mL) and DMT-Cl (102 mg, 0.30 mmol) was stirred for 12 h. Column chromatography (1% MeOH in DCM + 2% Et<sub>3</sub>N) yielded compound <b>4d</b> as a pale yellow solid (178 mg, 76%). <i>R</i><sub>f</sub> = 0.55 (2% MeOH in DCM + 2% Et<sub>3</sub>N); mp: 77â80 Â°C; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.25 (d, <i>J</i> = 9.3 Hz, 1H), 8.22â8.16 (m, 2H), 8.10â7.86 (m, 7H), 7.71 (s, 1H), 7.42â7.36 (m, 2H), 7.32â7.26 (m, 5H), 7.25â7.10 (m, 5H), 6.78â6.72 (m, 4H), 6.30 (t, <i>J</i> = 6.5 Hz, 1H), 5.37â5.32 (m, 1H), 5.27â5.13 (m, 2H), 4.64â4.50 (m, 3H), 4.09â4.03 (m, 1H), 3.70 (d, <i>J</i> = 2.5 Hz, 6H), 3.39 (dd, <i>J</i> = 9.9, 4.6 Hz, 1H), 3.35 (dd, <i>J</i> = 10, 5.2 Hz, 1H), 3.04 (br s, 2H), 2.87â2.77 (m, 1H), 2.37 (ddd, <i>J</i> = 13.4, 6.4, 4.0 Hz, 1H); <sup>13</sup>C{1H} NMR (100 MHz, CDCl<sub>3</sub>): Î´ 160.7, 158.7, 158.6, 153.9, 146.6, 145.9, 144.7, 137.9, 135.8, 132.4, 131.4, 130.8, 130.1, 129.7, 128.2, 128, 127.5, 127.4, 127, 126.2, 125.5, 125.3, 125, 124.9, 124.8, 123.6, 122.8, 115.7, 113.3, 86.6, 85.9, 84, 72.8, 65.9, 64.1, 55.3, 44.3, 39.6, 35.1; HRMS (ESI): Calcd for C<sub>56</sub>H<sub>48</sub>N<sub>6</sub>NaO<sub>8</sub>, [M + Na]<sup>+</sup> 955.3426; found, [M + Na]<sup>+</sup> 955.3431 (Î<i>m</i> + 0.0005, error +0.6 ppm).</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> General Procedure for the Phosphitylation</h3><div class="NLM_p">DMT protected nucleoside (1 equiv) was dissolved in DCM (10 mL/mmol) followed by addition of DIPEA (8 equiv) and CEP-Cl (2 equiv). The reaction mixture was stirred at room temperature under N<sub>2</sub> atmosphere. After completion, the reaction was quenched by addition of a trace amount of MeOH and stirred for 5 to 10 min. Further, the reaction mixture was diluted with DCM (100 mL) and washed with NaHCO<sub>3</sub> (3 Ã 20 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and evaporated under reduced pressure. Crude compound was purified by silica gel column chromatography (using appropriate solvents along with 2% Et<sub>3</sub>N) to obtain phosphitylated compound.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>N</i><sup>2</sup>-Benzyl<i>-O</i><sup>6</sup>-(2-(4-nitrophenyl)ethyl)5â²-<i>O</i>-(4,4â²-dimethoxytrityl)-3â²-(2-cyanoethyl diisopropylphosphoramidite)-2â²-deoxyguanosine (<b>5a</b>)</h4><div class="NLM_p last">Nucleoside <b>4a</b> (150 mg, 0.18 mmol), DCM (2 mL), DIPEA (0.25 mL, 1.48 mmol), and CEP-Cl (127 mg, 0.54 mmol) were used and stirred for 2 h. Further, MeOH (0.5 mL) was added and stirred for an additional 10 min. Column chromatography (35% DCM in pet. ether + 2% Et<sub>3</sub>N) yielded compound <b>5a</b> as a pale yellow solid (111 mg, 61%). <i>R</i><sub>f</sub> = 0.65 (90% DCM in pet. ether + 2% Et<sub>3</sub>N); mp: 58â61 Â°C; <sup>31</sup>P NMR (162 MHz, CDCl<sub>3</sub>): Î´ 148.81, 148.63; HRMS (ESI): Calcd for C<sub>55</sub>H<sub>62</sub>N<sub>8</sub>O<sub>9</sub>P, [M + H]<sup>+</sup> 1009.4377; found, [M + H]<sup>+</sup> 1009.4348 (Î<i>m</i> â 0.0029, error â3 ppm).</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i><sup>2</sup>-(1-Naphthylmethyl)<i>-O</i><sup>6</sup>-(2-(4-nitrophenyl)ethyl)5â²-<i>O</i>-(4,4â²-dimethoxytrityl)-3â²-(2-cyanoethyldiisopropylphosphoramidite)-2â²-deoxyguanosine (<b>5b</b>)</h4><div class="NLM_p last">The tritylated nucleoside <b>4b</b> (150 mg, 0.17 mmol), DCM (2 mL), DIPEA (0.25 mL, 1.36 mmol), and CEP-Cl (83 mg, 0.34 mmol) were used and stirred for 2 h 15 min. Further, MeOH (0.5 mL) was added and stirred for an additional 10 min. Column chromatography (35% DCM in pet. ether + 2% Et<sub>3</sub>N) yielded compound <b>5b</b> as a pale yellow solid (74 mg, 41%). <i>R</i><sub>f</sub> = 0.65 (75% DCM in pet. ether + 2% Et<sub>3</sub>N); mp: 70â73 Â°C; <sup>31</sup>P NMR (162 MHz, CDCl<sub>3</sub>): Î´ 148.72, 148.57; HRMS (ESI): Calcd for C<sub>59</sub>H<sub>64</sub>N<sub>8</sub>O<sub>9</sub>P, [M + H]<sup>+</sup> 1059.4528; found, [M + H]<sup>+</sup> 1059.4529 (Î<i>m</i> + 0.00009, error +0.1 ppm).</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>N</i><sup>2</sup>-(9-Anthracenylmethyl)<i>-O</i><sup>6</sup>-(2-(4-nitrophenyl)ethyl)5â²-<i>O</i>-(4,4â²-dimethoxytrityl)-3â²-(2-cyanoethyldiisopropylphosphoramidite)-2â²-deoxyguanosine (<b>5c</b>)</h4><div class="NLM_p last">Nucleoside <b>4c</b> (170 mg, 0.18 mmol), DCM (2 mL), DIPEA (0.26 mL, 1.49 mmol), and CEP-Cl (88 mg, 0.37 mmol) were used and stirred for 1 h. Further, MeOH (0.5 mL) was added and stirred for an additional 10 min. Column chromatography (40% DCM in pet. ether + 2% Et<sub>3</sub>N) yielded compound <b>5c</b> as a slightly greenish solid (75 mg, 37%). <i>R</i><sub>f</sub> = 0.60 (75% DCM in pet. ether + 2% Et<sub>3</sub>N); mp: 77â80 Â°C; <sup>31</sup>P NMR (202 MHz, CDCl<sub>3</sub>): Î´ 148.85, 148.72; HRMS (ESI): Calcd for C<sub>63</sub>H<sub>66</sub>N<sub>8</sub>O<sub>9</sub>P [M + H]<sup>+</sup>1109.4685; found, [M + H]<sup>+</sup> 1109.4686 (Î<i>m</i> + 0.00009, error +0.1 ppm).</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i><sup>2</sup>-(1-Pyrenylmethyl)<i>-O</i><sup>6</sup>-(2-(4-nitrophenyl)ethyl)5â²-<i>O</i>-(4,4â²-dimethoxytrityl)-3â²-(2-cyanoethyldiisopropylphosphoramidite)-2â²-deoxyguanosine (<b>5d</b>)</h4><div class="NLM_p last">Nucleoside <b>4d</b> (200 mg, 0.21 mmol), DCM (2.5 mL), DIPEA (0.3 mL, 1.68 mmol), and CEP-Cl (100 mg, 0.42 mmol) were used and stirred for 1 h 15 min. Further, MeOH (0.5 mL) was added and stirred for an additional 10 min. Column chromatography (45% DCM in pet. ether + 2% Et<sub>3</sub>N) yielded compound <b>5d</b> as a pale yellow solid (93 mg, 40%). <i>R</i><sub>f</sub> = 0.66 (75% DCM in pet. ether + 2% Et<sub>3</sub>N); mp: 80â83 Â°C; <sup>31</sup>P NMR (202 MHz, CDCl<sub>3</sub>): Î´ 148.72; HRMS (ESI): Calcd for C<sub>65</sub>H<sub>66</sub>N<sub>8</sub>O<sub>9</sub>P, [M + H]<sup>+</sup> 1133.4685; found, [M + H]<sup>+</sup> 1133.4690 (Î<i>m</i> + 0.0005, error +0.4 ppm).</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Solid Phase Synthesis, Deprotection, and Purification of <i>N</i><sup>2</sup>-dG Modified Oligonucleotides</h3><div class="NLM_p last">All DNA sequences (<a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">Table </a><a class="ref showTableEvent internalNav" data-id="tbl1" href="#tbl1">1</a>) were synthesized on 1 Î¼mol scale using appropriate CPG solid supports. Unmodified and modified phosphoramidite building blocks were dried under P<sub>2</sub>O<sub>5</sub> for 24 h and then dissolved in anhydrous ACN and stored at 4 Â°C under molecular sieves (4 Ã) for 10 h. Concentration used for the unmodified phosphoramidite was 0.067 M and for modified phosphoramidite was 0.1 M. Coupling time used for unmodified phosphoramidite was 2 min and for the <i>N</i><sup>2</sup>-dG modified phosphoramidites was 6 min. After solid phase synthesis, deprotection of <i>N</i><sup>2</sup>-Bn, <i>N</i><sup>2</sup>-Naph, <i>N</i><sup>2</sup>-Anth, and <i>N</i><sup>2</sup>-Pyre-dG modified oligos were carried out in four different steps. Initially, the deprotection of the cyanoethyl group of the modified DNA sequences were carried out using 10% diethylamine in ACN (800 Î¼L, v/v) for 5 min. Supernatant layer was collected, and solid support was washed with ACN (2 Ã 400 Î¼L). The CPG beads were air-dried (3 h) and treated with 1 M DBU in ACN (1 mL, v/v) for 1 h at room temperature to remove the NPE group. <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> After 1 h, the supernatant was removed and the CPG beads were washed with MeOH (2 Ã 1 mL) and ACN (3 Ã 1 mL). Furthermore, the modified DNAs intact with CPG was treated with 1 mL of 30% aq. NH<sub>3</sub> (v/v) for 3 h for the cleavage of solid support. To remove the base protecting groups, the supernatant solution was evaporated and again resuspended in 1 mL of 30% aq. NH<sub>3</sub> (v/v) and heated at 55 Â°C for 16 h. Further, the ammonia layer was evaporated to obtain the pellet of crude DNAs, which was purified by using 20% (7 M urea) denaturing PAGE (30 W, 3 h) with 1 X TBE running buffer (89 mM each Tris and boric acid and 2 mM EDTA, pH â¼8.3). The gel thickness was 1 mm, and the gel dimension was 20 Ã 30 cm. The desired oligo bands were visualized and marked under the UV lamp (260 nm). Further, the gel bands were cut and crushed into fine particles. The crushed gels were extracted three times with 15 mL of TEN buffer (10 mM Tris, 1 mM EDTA, 300 mM NaCl, pH â¼8.0) at 37 Â°C. Finally, desalting of oligos were carried out using Sep-Pak column and the concentrations of <i>N</i><sup>2</sup>-dG modified DNAs were measured at 260 nm in UVâvis spectrophotometer using the appropriate molar extinction coefficients (Îµ).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 5â²-Radiolabeling of Primers</h3><div class="NLM_p last">The radiolabeling reaction mixture contains the DNA (25 pmol), T4 polynucleotide kinase (PNK) enzyme (5 U), and [Î³-<sup>32</sup>P] ATP (20 Î¼Ci) in 1 X PNK buffer (50 mM Tris-HCl, 10 mM MgCl<sub>2</sub>, 5 mM DTT, 0.1 mM spermidine, 0.1 mM EDTA, pH 7.6). The total reaction volume was 10 Î¼L. The reaction mixture was incubated at 37 Â°C for 1 h followed by deactivation of PNK enzyme by heating at 70 Â°C for 3 min. The radiolabeled DNAs (primers and 50-mer DNA standard) were purified using nucleotide removal kit using the protocol provided by the manufacturer.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Primer Extension Assay</h3><div class="NLM_p last">The primer extension reactions were carried out using the mixture of 1:1.2 molar ratios of the primers with the unmodified and <i>N</i><sup>2</sup>-dG modified DNA templates. Initially, the primer-template mixture was annealed by heating at 95 Â°C for 3 min, followed by slow cooling to room temperature for 30 min. After annealing, the primer extension reactions were performed with primerâtemplate duplex in a total volume of 20 Î¼L, containing 40 nM primer, traces of 5â²-radiolabeled primer, 50 nM template, 50 Î¼g/mL of BSA, 0.1 mM of ammonium sulfate, 2.5 mM MgCl<sub>2</sub>, 4 Î¼L of 5X assay buffer (125 mM Tris-Cl, 5 mM dithiothreitol, pH 8.0), Pol IV (4 nM), and individual dNTP or mixture of dNTPs (10 Î¼M). The primer extension reactions were incubated at 37 Â°C for 5 min, and the reactions were initiated by the addition of the desired amount of dNTPs. The reactions were conducted for the indicated amount of time, and aliquots were withdrawn and quenched by the addition of stop solution (80% formamide, 0.025% each bromophenol blue, xylene cyanol and 50 mM EDTA). The extended products were run on 20% denaturing PAGE (7 M urea), autoradiogram was generated, and the bands were quantified by ImageQuantTL software. The running start experiments were carried out using 15-mer and 11-mer primers employing a mixture of dNTPs (250 Î¼M). The reaction conditions were the same as described previously. The total reaction volume contains 40 nM primer, traces of 5â²-radiolabeled primer, and 50 nM of each template. Finally, the reactions were terminated and analyzed as mentioned previously.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Molecular Modeling and Dynamics Studies</h3><div class="NLM_p">The modified nucleotides are constructed using GaussView and submitted to RED Server <i>Development</i> <a onclick="showRef(event, 'ref42 ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref42 ref43 ref44 ref45">(42â45)</a> for the calculation of restraint electrostatic potential (RESP) charges and parameters (bond, angle, and dihedral) required for the simulation (Figure S15âS18, <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>). The originally reported crystal structure (PDB ID: <a href="http://doi.org/10.2210/pdb4Q43/pdb">4Q43</a>) contains chemically modified dCTP at the active site which was required to prevent catalysis during crystallization. The modified dCTP is replaced with normal dCTP, and previously calculated bond, angle, and dihedral parameters were used (<a href="http://upjv.q4md-forcefieldtools.org/REDDB/projects/F-90/" class="extLink">http://upjv.q4md-forcefieldtools.org/REDDB/projects/F-90/</a>). The amber force fields <i>ff</i>99SB <a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and parmbsc0 <a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> were used for protein and DNA, respectively. The parameters for dCTP and modified nucleotides along with parameters for protein, DNA, and ions were fed to Leap program of AmberTools package to prepare input files. The Mg<sup>2+</sup> ions reported in the original crystal structure were retained. Additional potassium ions were added to neutralize the system. Each system was then immersed in TIP3P water box with water molecules extending up to 10 Ã. Then each complex was minimized in two stages to remove any atomic clashes.</div><div class="NLM_p">Energy minimization was carried out using SANDER module of Amber 14. In the first step, the <i>N</i><sup>2</sup>-dG adduct was left free and minimized for 10â¯000 steepest descent energy cycles, whereas in the second step the <i>N</i><sup>2</sup>-dG adduct was restricted and minimized for 10â¯000 steepest descent energy minimizations. Later the system was heated from 0.1 K to close to 300 K over 100 ps with constant volume. Then the complexes were heated to final temperature (300 K) at constant pressure. During the heating, the DNA was restricted with a restraint of 10 kcal/mol Ã<sup>2</sup>. This restriction was slowly removed over 8 stages of equilibration (with restraints of 10, 8, 5, 4, 3, 2, 1, and 0.5 kcal/mol Ã<sup>2</sup> 30 ps each). Each equilibration was carried under constant pressure and temperature (NPT) conditions. The pressure was maintained at 1 atm using Bendersen weak-coupling barostat with a pressure relaxation time of 2 ps. The temperature was maintained at 300 K using Langvein dynamics. The equilibration was followed by energy minimization (10â¯000 steepest descent cycles) and the minimized structure was used as starting structure for the subsequent equilibration. After 8 stages, 100 ps of equilibration was run under NPT conditions without any restrictions before proceeding to the production run. 100 ns of unrestrained dynamics (under NPT conditions) was performed on each complex using GPU accelerated version of Particle Mesh Ewald Molecular Dynamics (PMEMD) implemented in Amber 14. <a onclick="showRef(event, 'ref39 ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref39 ref48 ref49 ref50">(39,48â50)</a> A cutoff value of 10 Ã is set for calculations of nonbonded interactions by Particle Mesh Ewald (PME) method. The SHAKE algorithm was used to constraint all of the bonds involving hydrogen atoms with an integration time step of 2 fs. The output coordinate files were saved for every 2 ps and a total of 50â¯000 snapshots (100 ns) were recorded for each complex.</div><div class="NLM_p last">The trajectories were visualized using UCSF Chimera. <a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> CPPTRAJ module of Amber 14 was used for calculations of dihedrals, RMSD values, RMSF values, H-bond occupancies, and clustering of dynamics. Only heavy atoms were considered for the calculation of RMSD values. Distances between Ser and <i>N</i><sup>2</sup>-adducts were calculated using per frame analysis in UCSF Chimera. A distance of 3.0 Ã and angle of 150Â° were used as cutoff values for H-bond calculations. A cut-off distance of 4.0 Ã is used for van der Waals interactions. Clustering analysis was carried out employing hierarchical agglomerative approach using mass weighed RMSD. Interaction energies are calculated using MM/GB-SA <a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> method implemented as the program MMPBSA.py <a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> in AmberTools package. Images are rendered using PyMol (<a href="http://www.pymol.org" class="extLink">www.pymol.org</a>).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51781" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51781" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Summary and Conclusions"><span>Summary and Conclusions</span></a></li><li><a href="#_i16" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i36" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i37" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i39" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.joc.8b02082" class="ext-link">10.1021/acs.joc.8b02082</a>.</p><ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">PAGEs of nucleotide incorporation and full length extension assays; additional figures and tables from molecular modeling and dynamics studies; <sup>1</sup>H, <sup>13</sup>C, and <sup>31</sup>P NMR of all of the new compounds; and representative MALDI spectra of <i>N</i><sup>2</sup>-dG modified oligonucleotides (<a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf">jo8b02082_si_001.pdf (5.35 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.joc.8b02082" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget" style="width: 876.3px; display: block;"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms &amp; Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76706" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76706" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Summary and Conclusions"><span>Summary and Conclusions</span></a></li><li><a href="#_i16" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i36" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i37" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i39" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Deepak T. Nair</span> - <span class="hlFld-Affiliation affiliation">Regional
Centre for Biotechnology, NCR Biotech Science
Cluster, third Milestone,
Faridabad-Gurgaon Expressway, Faridabad 121001, India</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0002-0677-9444" title="Orcid link">http://orcid.org/0000-0002-0677-9444</a>;&nbsp;
    </span><span class="author-email">
                Email: <a href="mailto:deepak@rcb.res.in">deepak@rcb.res.in</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">P. I. Pradeepkumar</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Indian Institute of Technology
Bombay, Mumbai 400076, India</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-9104-3708" title="Orcid link">http://orcid.org/0000-0001-9104-3708</a>;&nbsp;
    </span><span class="author-email">
                Email: <a href="mailto:pradeep@chem.iitb.ac.in">pradeep@chem.iitb.ac.in</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pratibha
P. Ghodke</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Indian Institute of Technology
Bombay, Mumbai 400076, India</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0002-8402-3933" title="Orcid link">http://orcid.org/0000-0002-8402-3933</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Praneeth Bommisetti</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Indian Institute of Technology
Bombay, Mumbai 400076, India</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br></div></li></ul></div><div class="ack" id="ACK-d7e3016-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14666" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14666" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Summary and Conclusions"><span>Summary and Conclusions</span></a></li><li><a href="#_i16" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i36" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i37" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i39" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><p class="last">This work is financially supported by grants from Department of Biotechnology (DBT)-Government of India (Grant No. BT/PR8265/BRB/10/1228/2013). We are thankful to Dr. Claudia HÃ¶bartner (MPIbpc-GÃ¶ttingen), Prof. K. V. R. Chary, and Ms. Gitanjali A. Dhotre (TIFR-Mumbai) and the central facility supported by IRCC-IIT Bombay for providing ESI/MALDI spectra of the modified DNAs. We also thank Dr. S. Harikrishna for his suggestions on MD studies and Saurja Dasgupta for critically reading the manuscript. Computer Centre, IIT Bombay is gratefully acknowledged for providing high performance computing facilities. P.P.G. thanks Ms. Sushree P. Pany for acquiring the mass spectra of phosphoramidites. P.P.G. also thanks the Council of Scientific and Industrial Research (CSIR) for the Ph.D. fellowship, and P.B. thanks Department of Science and Technology (DST) for the INSPIRE Fellowship.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82372" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82372" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i15" class="internalNav" title="Summary and Conclusions"><span>Summary and Conclusions</span></a></li><li><a href="#_i16" class="internalNav" title="Experimental Section"><span>Experimental Section</span></a></li><li><a href="#_i36" class="internalNav" title="Supporting Information"><span>Supporting Information</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i37" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i39" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr></div><p class="references-count">
            This article references 53 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moorthy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlin, D. J.</span></span> <span> </span><span class="NLM_article-title">Polycyclic Aromatic Hydrocarbons: From Metabolism to Lung Cancer</span>. <i>Toxicol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>145</i></span>,  <span class="NLM_fpage">5</span>â <span class="NLM_lpage">15</span>, <span class="refDoi">&nbsp;DOI: 10.1093/toxsci/kfv040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1093%2Ftoxsci%2Fkfv040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=25911656" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFSrurzN" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=145&amp;publication_year=2015&amp;pages=5-15&amp;author=B.+Moorthyauthor=C.+Chuauthor=D.+J.+Carlin&amp;title=Polycyclic+Aromatic+Hydrocarbons%3A+From+Metabolism+to+Lung+Cancer&amp;doi=10.1093%2Ftoxsci%2Fkfv040" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Polycyclic aromatic hydrocarbons: from metabolism to lung cancer</span></div><div class="casAuthors">Moorthy, Bhagavatula; Chu, Chun; Carlin, Danielle J.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">145</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-15</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-0929</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Excessive exposure to polycyclic arom. hydrocarbons (PAHs) often results in lung cancer, a disease with the highest cancer mortality in the United States.  After entry into the lung, PAHs induce phase I metabolic enzymes such as cytochrome P 450 (CYP) monooxygenases, i.e.  CYP1A1/2 and 1B1, and phase II enzymes such as glutathione S-transferases, UDP glucuronyl transferases, NADPH quinone oxidoreductases (NQOs), aldo-keto reductases (AKRs), and epoxide hydrolases (EHs), via the aryl hydrocarbon receptor (AhR)-dependent and independent pathways.  Humans can also be exposed to PAHs through diet, via consumption of charcoal broiled foods.  Metab. of PAHs through the CYP1A1/1B1/EH pathway, CYP peroxidase pathway, and AKR pathway leads to the formation of the active carcinogens diol-epoxides, radical cations, and o-quinones.  These reactive metabolites produce DNA adducts, resulting in DNA mutations, alteration of gene expression profiles, and tumorigenesis.  Mutations in xenobiotic metabolic enzymes, as well as polymorphisms of tumor suppressor genes (e.g. p53) and/or genes involved in gene expression (e.g. X-ray repair cross-complementing proteins), are assocd. with lung cancer susceptibility in human populations from different ethnicities, gender, and age groups.  Although various metabolic activation/inactivation pathways, AhR signaling, and genetic susceptibilities contribute to lung cancer, the precise points at which PAHs induce tumor initiation remain unknown.  The goal of this review is to provide a current state-of-the-science of the mechanisms of human lung carcinogenesis mediated by PAHs, the exptl. approaches used to study this complex class of compds., and future directions for research of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoELfgYOijvfLVg90H21EOLACvtfcHk0lihc2whtRmk_A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXhsFSrurzN&amp;md5=b1b4dcbfa4b6c37239ddbd96224bb9fd</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfv040&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfv040%26sid%3Dliteratum%253Aachs%26aulast%3DMoorthy%26aufirst%3DB.%26aulast%3DChu%26aufirst%3DC.%26aulast%3DCarlin%26aufirst%3DD.%2BJ.%26atitle%3DPolycyclic%2520Aromatic%2520Hydrocarbons%253A%2520From%2520Metabolism%2520to%2520Lung%2520Cancer%26jtitle%3DToxicol.%2520Sci.%26date%3D2015%26volume%3D145%26spage%3D5%26epage%3D15%26doi%3D10.1093%2Ftoxsci%2Fkfv040" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dipple, A.</span></span> <span> </span><span class="NLM_article-title">DNA Adducts of Chemical Carcinogens</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">437</span>â <span class="NLM_lpage">441</span>, <span class="refDoi">&nbsp;DOI: 10.1093/carcin/16.3.437</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1093%2Fcarcin%2F16.3.437" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=7697795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADyaK2MXkvVyqur0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=1995&amp;pages=437-441&amp;author=A.+Dipple&amp;title=DNA+Adducts+of+Chemical+Carcinogens&amp;doi=10.1093%2Fcarcin%2F16.3.437" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">DNA adducts of chemical carcinogens</span></div><div class="casAuthors">Dipple, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">437-41</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review and discussion with 25 refs. on metabolic activation, types of metabolic reactions (oxidn., redn., conjugation), arylamine-DNA adducts, alkylation, arylamination, aralkylation, and carcinogens on guanine residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpja7lYAMpt97Vg90H21EOLACvtfcHk0lhXAepESaLaGA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXkvVyqur0%253D&amp;md5=087d09c1c810d0d27daab8884db1c919</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2F16.3.437&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252F16.3.437%26sid%3Dliteratum%253Aachs%26aulast%3DDipple%26aufirst%3DA.%26atitle%3DDNA%2520Adducts%2520of%2520Chemical%2520Carcinogens%26jtitle%3DCarcinogenesis%26date%3D1995%26volume%3D16%26spage%3D437%26epage%3D441%26doi%3D10.1093%2Fcarcin%2F16.3.437" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hemminki, K.</span></span> <span> </span><span class="NLM_article-title">Nucleic Acid Adducts of Chemical Carcinogens and Mutagens</span>. <i>Arch. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">249</span>â <span class="NLM_lpage">285</span>, <span class="refDoi">&nbsp;DOI: 10.1007/BF00316495</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1007%2FBF00316495" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=6347126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADyaL3sXitFSru7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=1983&amp;pages=249-285&amp;author=K.+Hemminki&amp;title=Nucleic+Acid+Adducts+of+Chemical+Carcinogens+and+Mutagens&amp;doi=10.1007%2FBF00316495" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleic acid adducts of chemical carcinogens and mutagens</span></div><div class="casAuthors">Hemminki, Kari</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Toxicology</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">249-85</span>CODEN:
                <span class="NLM_cas:coden">ARTODN</span>;
        ISSN:<span class="NLM_cas:issn">0340-5761</span>.
    </div><div class="casAbstract">A review with several refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaKEd-g6561LVg90H21EOLACvtfcHk0lhXAepESaLaGA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL3sXitFSru7Y%253D&amp;md5=b622b3e11fb8bc6977a5d683914393ed</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2FBF00316495&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00316495%26sid%3Dliteratum%253Aachs%26aulast%3DHemminki%26aufirst%3DK.%26atitle%3DNucleic%2520Acid%2520Adducts%2520of%2520Chemical%2520Carcinogens%2520and%2520Mutagens%26jtitle%3DArch.%2520Toxicol.%26date%3D1983%26volume%3D52%26spage%3D249%26epage%3D285%26doi%3D10.1007%2FBF00316495" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sale, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehmann, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodgate, R.</span></span> <span> </span><span class="NLM_article-title">Y-Family DNA Polymerases and their Role in Tolerance of Cellular DNA Damage</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">141</span>â <span class="NLM_lpage">152</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nrm3289</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1038%2Fnrm3289" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=22358330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC38Xis1ekurs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=13&amp;publication_year=2012&amp;pages=141-152&amp;author=J.+E.+Saleauthor=A.+R.+Lehmannauthor=R.+Woodgate&amp;title=Y-Family+DNA+Polymerases+and+their+Role+in+Tolerance+of+Cellular+DNA+Damage&amp;doi=10.1038%2Fnrm3289" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Y-family DNA polymerases and their role in tolerance of cellular DNA damage</span></div><div class="casAuthors">Sale, Julian E.; Lehmann, Alan R.; Woodgate, Roger</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">141-152</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The past 15 years have seen an explosion in our understanding of how cells replicate damaged DNA and how this can lead to mutagenesis.  The Y-family DNA polymerases lie at the heart of this process, which is commonly known as translesion synthesis.  This family of polymerases has unique features that enable them to synthesize DNA past damaged bases.  However, as they exhibit low fidelity when copying undamaged DNA, it is essential that they are only called into play when they are absolutely required.  Several layers of regulation ensure that this is achieved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJf4Ctv0l1jLVg90H21EOLACvtfcHk0liHaXLRrEe4pA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38Xis1ekurs%253D&amp;md5=0bfb44c9b222edae8a1779422cf32cd1</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrm3289&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3289%26sid%3Dliteratum%253Aachs%26aulast%3DSale%26aufirst%3DJ.%2BE.%26aulast%3DLehmann%26aufirst%3DA.%2BR.%26aulast%3DWoodgate%26aufirst%3DR.%26atitle%3DY-Family%2520DNA%2520Polymerases%2520and%2520their%2520Role%2520in%2520Tolerance%2520of%2520Cellular%2520DNA%2520Damage%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D141%26epage%3D152%26doi%3D10.1038%2Fnrm3289" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jarosz, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godoy, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essigmann, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. C.</span></span> <span> </span><span class="NLM_article-title">A Single Amino Acid Governs Enhanced Activity of DinB DNA Polymerases on Damaged Templates</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>439</i></span>,  <span class="NLM_fpage">225</span>â <span class="NLM_lpage">228</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature04318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1038%2Fnature04318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=16407906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A280%3ADC%252BD28%252FitVyrtw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=439&amp;publication_year=2006&amp;pages=225-228&amp;author=D.+F.+Jaroszauthor=V.+G.+Godoyauthor=J.+C.+Delaneyauthor=J.+M.+Essigmannauthor=G.+C.+Walker&amp;title=A+Single+Amino+Acid+Governs+Enhanced+Activity+of+DinB+DNA+Polymerases+on+Damaged+Templates&amp;doi=10.1038%2Fnature04318" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A single amino acid governs enhanced activity of DinB DNA polymerases on damaged templates</span></div><div class="casAuthors">Jarosz Daniel F; Godoy Veronica G; Delaney James C; Essigmann John M; Walker Graham C</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">439</span>
        (<span class="NLM_cas:issue">7073</span>),
    <span class="NLM_cas:pages">225-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Translesion synthesis (TLS) by Y-family DNA polymerases is a chief mechanism of DNA damage tolerance.  Such TLS can be accurate or error-prone, as it is for bypass of a cyclobutane pyrimidine dimer by DNA polymerase eta (XP-V or Rad30) or bypass of a (6-4) TT photoproduct by DNA polymerase V (UmuD'2C), respectively.  Although DinB is the only Y-family DNA polymerase conserved among all domains of life, the biological rationale for this striking conservation has remained enigmatic.  Here we report that the Escherichia coli dinB gene is required for resistance to some DNA-damaging agents that form adducts at the N2-position of deoxyguanosine (dG).  We show that DinB (DNA polymerase IV) catalyses accurate TLS over one such N2-dG adduct (N2-furfuryl-dG), and that DinB and its mammalian orthologue, DNA polymerase kappa, insert deoxycytidine (dC) opposite N2-furfuryl-dG with 10-15-fold greater catalytic proficiency than opposite undamaged dG.  We also show that mutating a single amino acid, the 'steric gate' residue of DinB (Phe13 --&gt; Val) and that of its archaeal homologue Dbh (Phe12 --&gt; Ala), separates the abilities of these enzymes to perform TLS over N2-dG adducts from their abilities to replicate an undamaged template.  We propose that DinB and its orthologues are specialized to catalyse relatively accurate TLS over some N2-dG adducts that are ubiquitous in nature, that lesion bypass occurs more efficiently than synthesis on undamaged DNA, and that this specificity may be achieved at least in part through a lesion-induced conformational change.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSygKbT0LHAudTiMd2aNQVUfW6udTcc2eY0aTAZgszLPrntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BD28%252FitVyrtw%253D%253D&amp;md5=d38a3179bb2c3e01de936f5b52d37ab1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature04318&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04318%26sid%3Dliteratum%253Aachs%26aulast%3DJarosz%26aufirst%3DD.%2BF.%26aulast%3DGodoy%26aufirst%3DV.%2BG.%26aulast%3DDelaney%26aufirst%3DJ.%2BC.%26aulast%3DEssigmann%26aufirst%3DJ.%2BM.%26aulast%3DWalker%26aufirst%3DG.%2BC.%26atitle%3DA%2520Single%2520Amino%2520Acid%2520Governs%2520Enhanced%2520Activity%2520of%2520DinB%2520DNA%2520Polymerases%2520on%2520Damaged%2520Templates%26jtitle%3DNature%26date%3D2006%26volume%3D439%26spage%3D225%26epage%3D228%26doi%3D10.1038%2Fnature04318" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kottur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gore, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeepkumar, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, D. T.</span></span> <span> </span><span class="NLM_article-title">Unique Structural Features in DNA Polymerase IV Enable Efficient Bypass of the N2 Adduct Induced by the Nitrofurazone Antibiotic</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">56</span>â <span class="NLM_lpage">67</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.str.2014.10.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1016%2Fj.str.2014.10.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=25497730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFCrtLjM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2015&amp;pages=56-67&amp;author=J.+Kotturauthor=A.+Sharmaauthor=K.+R.+Goreauthor=N.+Narayananauthor=B.+Samantaauthor=P.+I.+Pradeepkumarauthor=D.+T.+Nair&amp;title=Unique+Structural+Features+in+DNA+Polymerase+IV+Enable+Efficient+Bypass+of+the+N2+Adduct+Induced+by+the+Nitrofurazone+Antibiotic&amp;doi=10.1016%2Fj.str.2014.10.019" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Unique Structural Features in DNA Polymerase IV Enable Efficient Bypass of the N2 Adduct Induced by the Nitrofurazone Antibiotic</span></div><div class="casAuthors">Kottur, Jithesh; Sharma, Amit; Gore, Kiran R.; Narayanan, Naveen; Samanta, Biswajit; Pradeepkumar, Pushpangadan I.; Nair, Deepak T.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-67</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The redn. in the efficacy of therapeutic antibiotics represents a global problem of increasing intensity and concern.  Nitrofuran antibiotics act primarily through the formation of covalent adducts at the N2 atom of the deoxyguanosine nucleotide in genomic DNA.  These adducts inhibit replicative DNA polymerases (dPols), leading to the death of the prokaryote.  N2-furfuryl-deoxyguanosine (fdG) represents a stable structural analog of the nitrofuran-induced adducts.  Unlike other known dPols, DNA polymerase IV (PolIV) from E. coli can bypass the fdG adduct accurately with high catalytic efficiency.  This property of PolIV is central to its role in reducing the sensitivity of E. coli toward nitrofuran antibiotics such as nitrofurazone (NFZ).  We present the mechanism used by PolIV to bypass NFZ-induced adducts and thus improve viability of E. coli in the presence of NFZ.  Our results can be used to develop specific inhibitors of PolIV that may potentiate the activity of nitrofuran antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1UosfmyW8ZLVg90H21EOLACvtfcHk0liHaXLRrEe4pA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXitFCrtLjM&amp;md5=4cfe91793eb48666059432db8b735c38</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.10.019&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.10.019%26sid%3Dliteratum%253Aachs%26aulast%3DKottur%26aufirst%3DJ.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DGore%26aufirst%3DK.%2BR.%26aulast%3DNarayanan%26aufirst%3DN.%26aulast%3DSamanta%26aufirst%3DB.%26aulast%3DPradeepkumar%26aufirst%3DP.%2BI.%26aulast%3DNair%26aufirst%3DD.%2BT.%26atitle%3DUnique%2520Structural%2520Features%2520in%2520DNA%2520Polymerase%2520IV%2520Enable%2520Efficient%2520Bypass%2520of%2520the%2520N2%2520Adduct%2520Induced%2520by%2520the%2520Nitrofurazone%2520Antibiotic%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D56%26epage%3D67%26doi%3D10.1016%2Fj.str.2014.10.019" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furukohri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philippin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loechler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akiyama, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katayama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maki, H.</span></span> <span> </span><span class="NLM_article-title">DNA Polymerase IV Mediates Efficient and Quick Recovery of Replication Forks Stalled at N2-dG Adducts</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">8461</span>â <span class="NLM_lpage">8472</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gku547</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1093%2Fnar%2Fgku547" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=42&amp;publication_year=2014&amp;pages=8461-8472&amp;author=M.+Ikedaauthor=A.+Furukohriauthor=G.+Philippinauthor=E.+Loechlerauthor=M.+T.+Akiyamaauthor=T.+Katayamaauthor=R.+P.+Fuchsauthor=H.+Maki&amp;title=DNA+Polymerase+IV+Mediates+Efficient+and+Quick+Recovery+of+Replication+Forks+Stalled+at+N2-dG+Adducts&amp;doi=10.1093%2Fnar%2Fgku547" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku547&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku547%26sid%3Dliteratum%253Aachs%26aulast%3DIkeda%26aufirst%3DM.%26aulast%3DFurukohri%26aufirst%3DA.%26aulast%3DPhilippin%26aufirst%3DG.%26aulast%3DLoechler%26aufirst%3DE.%26aulast%3DAkiyama%26aufirst%3DM.%2BT.%26aulast%3DKatayama%26aufirst%3DT.%26aulast%3DFuchs%26aufirst%3DR.%2BP.%26aulast%3DMaki%26aufirst%3DH.%26atitle%3DDNA%2520Polymerase%2520IV%2520Mediates%2520Efficient%2520and%2520Quick%2520Recovery%2520of%2520Replication%2520Forks%2520Stalled%2520at%2520N2-dG%2520Adducts%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D8461%26epage%3D8472%26doi%3D10.1093%2Fnar%2Fgku547" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moschel, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudgins, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dipple, A.</span></span> <span> </span><span class="NLM_article-title">Aralkylation of Guanosine by the Carcinogen N-Nitroso-N-Benzylurea</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">533</span>â <span class="NLM_lpage">535</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jo01291a037</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo01291a037" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=1980&amp;pages=533-535&amp;author=R.+C.+Moschelauthor=W.+R.+Hudginsauthor=A.+Dipple&amp;title=Aralkylation+of+Guanosine+by+the+Carcinogen+N-Nitroso-N-Benzylurea&amp;doi=10.1021%2Fjo01291a037" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjo01291a037&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo01291a037%26sid%3Dliteratum%253Aachs%26aulast%3DMoschel%26aufirst%3DR.%2BC.%26aulast%3DHudgins%26aufirst%3DW.%2BR.%26aulast%3DDipple%26aufirst%3DA.%26atitle%3DAralkylation%2520of%2520Guanosine%2520by%2520the%2520Carcinogen%2520N-Nitroso-N-Benzylurea%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1980%26volume%3D45%26spage%3D533%26epage%3D535%26doi%3D10.1021%2Fjo01291a037" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, L. A.</span></span> <span> </span><span class="NLM_article-title">N-Nitrosobenzylmethylamine Is Activated to a DNA Benzylating Agent in Rats</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">19</span>â <span class="NLM_lpage">26</span>, <span class="refDoi">&nbsp;DOI: 10.1021/tx9601014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx9601014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=1997&amp;pages=19-26&amp;author=L.+A.+Peterson&amp;title=N-Nitrosobenzylmethylamine+Is+Activated+to+a+DNA+Benzylating+Agent+in+Rats&amp;doi=10.1021%2Ftx9601014" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Ftx9601014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx9601014%26sid%3Dliteratum%253Aachs%26aulast%3DPeterson%26aufirst%3DL.%2BA.%26atitle%3DN-Nitrosobenzylmethylamine%2520Is%2520Activated%2520to%2520a%2520DNA%2520Benzylating%2520Agent%2520in%2520Rats%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D1997%26volume%3D10%26spage%3D19%26epage%3D26%26doi%3D10.1021%2Ftx9601014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moon, K.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. S.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Characterization of Oligonucleotides Containing Site-Specific Bulky N2-Aralkylated Guanines and N6-Aralkylated Adenines</span>. <i>Arch. Pharmacal Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">139</span>â <span class="NLM_lpage">146</span>, <span class="refDoi">&nbsp;DOI: 10.1007/BF02975502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1007%2FBF02975502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10836739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjs12hsbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2000&amp;pages=139-146&amp;author=K.-Y.+Moonauthor=Y.+S.+Kim&amp;title=Synthesis+and+Characterization+of+Oligonucleotides+Containing+Site-Specific+Bulky+N2-Aralkylated+Guanines+and+N6-Aralkylated+Adenines&amp;doi=10.1007%2FBF02975502" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and characterization of oligonucleotides containing site-specific bulky N2-aralkylated guanines and N6-aralkylated adenines</span></div><div class="casAuthors">Moon, Ki-Young; Kim, Yeong Shik</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Pharmacal Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">139-146</span>CODEN:
                <span class="NLM_cas:coden">APHRDQ</span>;
        ISSN:<span class="NLM_cas:issn">0253-6269</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Korea</span>)
        </div><div class="casAbstract">7 - Bromomethylbenz[a]anthracene is a known mutagen and carcinogen.  The two major DNA adducts produced by this carcinogen, i.e., N2-(benz[a]anthracen-7-ylmethyl)-2'-deoxyguanosine (b[a]a2G) and N6-(benz[a]anthracen-7-ylmethyl)-2'-deoxyadenosine (b[a]a6A), as well as the simpler benzylated analogs, N2-benzyl-2'-deoxyguanosine (bn2G) and N6-benzyl-2'-deoxyadenosine (bn6A), were prepd. by direct aralkylation of 2'-deoxyguanosine and 2'-deoxyadenosine.  To det. the site-specific mutagenicity of these bulky exocyclic amino-substituted adducts, the suitably protected nucleosides were incorporated into 16-base oligodeoxyribonucleotides in place of a normal guanine or adenine residues which resp. are part of the ATG initiation codon for the lac Z' Î±-complementation gene by using an in situ activation approach and automated phosphite triester synthetic methods.  The base compn. and the incorporation of the bulky adducts into synthetic oligonucleotides were characterized after purifn. of the modified oligonucleotides by enzymic digestion and HPLC anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJFCEGfotRbLVg90H21EOLACvtfcHk0lj1NWS_gamoHw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3cXjs12hsbc%253D&amp;md5=a692e97c0abd6380d292e7cbad7cebf2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2FBF02975502&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02975502%26sid%3Dliteratum%253Aachs%26aulast%3DMoon%26aufirst%3DK.-Y.%26aulast%3DKim%26aufirst%3DY.%2BS.%26atitle%3DSynthesis%2520and%2520Characterization%2520of%2520Oligonucleotides%2520Containing%2520Site-Specific%2520Bulky%2520N2-Aralkylated%2520Guanines%2520and%2520N6-Aralkylated%2520Adenines%26jtitle%3DArch.%2520Pharmacal%2520Res.%26date%3D2000%26volume%3D23%26spage%3D139%26epage%3D146%26doi%3D10.1007%2FBF02975502" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koag, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span> <span> </span><span class="NLM_article-title">Structural and Kinetic Studies of the Effect of Guanine-N7 Alkylation and Metal Cofactors on DNA Replication</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">5105</span>â <span class="NLM_lpage">5116</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.biochem.8b00331</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00331" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ejsrjK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=57&amp;publication_year=2018&amp;pages=5105-5116&amp;author=Y.+Kouauthor=M.-C.+Koagauthor=S.+Lee&amp;title=Structural+and+Kinetic+Studies+of+the+Effect+of+Guanine-N7+Alkylation+and+Metal+Cofactors+on+DNA+Replication&amp;doi=10.1021%2Facs.biochem.8b00331" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Kinetic Studies of the Effect of Guanine N7 Alkylation and Metal Cofactors on DNA Replication</span></div><div class="casAuthors">Kou, Yi; Koag, Myong-Chul; Lee, Seongmin</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">5105-5116</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A wide variety of endogenous and exogenous alkylating agents attack DNA to preferentially generate N7-alkylguanine (N7-alkylG) adducts.  Studies on the effect of N7-alkylG lesions on biol. processes have been difficult due in part to complications arising from the chem. lability of the pos. charged N7-alkylG, which can readily produce secondary lesions.  To assess the effect of bulky N7-alkylG on DNA replication, we prepd. chem. stable N7-benzylguanine(N7bnG)-contg. DNA and evaluated nucleotide incorporation opposite the lesion by human DNA polymerase Î² (polÎ²), a model enzyme for high-fidelity DNA polymerases.  Kinetic studies showed that the N7-benzyl-G lesion greatly inhibited dCTP incorporation by polÎ².  The crystal structure of polÎ² incorporating dCTP opposite N7bnG showed a Watson-Crick N7bnG:dCTP.  The polÎ²-N7bnG:dCTP structure showed an open protein conformation, a relatively disordered dCTP, and lack of catalytic metal, which explained the inefficient nucleotide incorporation opposite N7bnG.  This indicates that polÎ² is sensitive to major groove adducts in the templating-base side and deters nucleotide incorporation opposite bulky N7-alkylG adducts by adopting a catalytically incompetent conformation.  Substituting Mg2+ for Mn2+ induced an open-to-closed conformational change due to the presence of catalytic metal and stably bound dCTP and increased the catalytic efficiency by â¼10-fold, highlighting the effect of binding of incoming nucleotide and catalytic metal on protein conformation and nucleotidyl transfer reaction.  Overall, these results suggest that, although bulky alkyl groups at guanine-N7 may not alter base-pairing properties of guanine, the major-groove-positioned lesions in the template could impede nucleotidyl transfer by some DNA polymerases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKx0GLtkJBIrVg90H21EOLACvtfcHk0lj1NWS_gamoHw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXht1ejsrjK&amp;md5=09f63033f165ccca4030d6b993b7d3c1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00331&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00331%26sid%3Dliteratum%253Aachs%26aulast%3DKou%26aufirst%3DY.%26aulast%3DKoag%26aufirst%3DM.-C.%26aulast%3DLee%26aufirst%3DS.%26atitle%3DStructural%2520and%2520Kinetic%2520Studies%2520of%2520the%2520Effect%2520of%2520Guanine-N7%2520Alkylation%2520and%2520Metal%2520Cofactors%2520on%2520DNA%2520Replication%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26spage%3D5105%26epage%3D5116%26doi%3D10.1021%2Facs.biochem.8b00331" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flesher, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehner, A. F.</span></span> <span> </span><span class="NLM_article-title">9-Sulfooxymethylanthracene is an ultimate electrophilic and carcinogenic form of 9-hydroxymethylanthracene</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>251</i></span>,  <span class="NLM_fpage">239</span>â <span class="NLM_lpage">243</span>, <span class="refDoi">&nbsp;DOI: 10.1006/bbrc.1998.9448</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1006%2Fbbrc.1998.9448" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=251&amp;publication_year=1998&amp;pages=239-243&amp;author=J.+W.+Flesherauthor=J.+Hornauthor=A.+F.+Lehner&amp;title=9-Sulfooxymethylanthracene+is+an+ultimate+electrophilic+and+carcinogenic+form+of+9-hydroxymethylanthracene&amp;doi=10.1006%2Fbbrc.1998.9448" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1998.9448&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1998.9448%26sid%3Dliteratum%253Aachs%26aulast%3DFlesher%26aufirst%3DJ.%2BW.%26aulast%3DHorn%26aufirst%3DJ.%26aulast%3DLehner%26aufirst%3DA.%2BF.%26atitle%3D9-Sulfooxymethylanthracene%2520is%2520an%2520ultimate%2520electrophilic%2520and%2520carcinogenic%2520form%2520of%25209-hydroxymethylanthracene%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1998%26volume%3D251%26spage%3D239%26epage%3D243%26doi%3D10.1006%2Fbbrc.1998.9448" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Casale, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, L. W.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Properties of an Oligodeoxynucleotide Containing a Polycyclic Aromatic Hydrocarbon Site Specifically Bound to the N2-Amino Group of a 2â²-Deoxyguanosine Residue</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">5264</span>â <span class="NLM_lpage">5271</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja00169a039</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00169a039" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=112&amp;publication_year=1990&amp;pages=5264-5271&amp;author=R.+Casaleauthor=L.+W.+McLaughlin&amp;title=Synthesis+and+Properties+of+an+Oligodeoxynucleotide+Containing+a+Polycyclic+Aromatic+Hydrocarbon+Site+Specifically+Bound+to+the+N2-Amino+Group+of+a+2%E2%80%B2-Deoxyguanosine+Residue&amp;doi=10.1021%2Fja00169a039" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fja00169a039&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00169a039%26sid%3Dliteratum%253Aachs%26aulast%3DCasale%26aufirst%3DR.%26aulast%3DMcLaughlin%26aufirst%3DL.%2BW.%26atitle%3DSynthesis%2520and%2520Properties%2520of%2520an%2520Oligodeoxynucleotide%2520Containing%2520a%2520Polycyclic%2520Aromatic%2520Hydrocarbon%2520Site%2520Specifically%2520Bound%2520to%2520the%2520N2-Amino%2520Group%2520of%2520a%25202%25E2%2580%25B2-Deoxyguanosine%2520Residue%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1990%26volume%3D112%26spage%3D5264%26epage%3D5271%26doi%3D10.1021%2Fja00169a039" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bendadani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monien, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobbernack, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glatt, H.</span></span> <span> </span><span class="NLM_article-title">The Carcinogen 1-Methylpyrene Forms Benzylic DNA Adducts in Mouse and Rat Tissues in Vivo via a Reactive Sulphuric Acid Ester</span>. <i>Arch. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">815</span>â <span class="NLM_lpage">821</span>, <span class="refDoi">&nbsp;DOI: 10.1007/s00204-013-1182-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1007%2Fs00204-013-1182-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=24337722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFajsb%252FE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=88&amp;publication_year=2014&amp;pages=815-821&amp;author=C.+Bendadaniauthor=W.+Meinlauthor=B.+H.+Monienauthor=G.+Dobbernackauthor=H.+Glatt&amp;title=The+Carcinogen+1-Methylpyrene+Forms+Benzylic+DNA+Adducts+in+Mouse+and+Rat+Tissues+in+Vivo+via+a+Reactive+Sulphuric+Acid+Ester&amp;doi=10.1007%2Fs00204-013-1182-6" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The carcinogen 1-methylpyrene forms benzylic DNA adducts in mouse and rat tissues in vivo via a reactive sulphuric acid ester</span></div><div class="casAuthors">Bendadani, Carolin; Meinl, Walter; Monien, Bernhard H.; Dobbernack, Gisela; Glatt, Hansruedi</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Toxicology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">815-821</span>CODEN:
                <span class="NLM_cas:coden">ARTODN</span>;
        ISSN:<span class="NLM_cas:issn">0340-5761</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The common polycyclic arom. hydrocarbon 1-methylpyrene is hepatocarcinogenic in the newborn mouse assay.  In vitro studies showed that it is metabolically activated via benzylic hydroxylation and sulphation to a reactive ester, which forms benzylic DNA adducts, N2-(1-methylpyrenyl)-2'-deoxyguanosine (MPdG) and N6-(1-methylpyrenyl)-2'-deoxyadenosine (MPdA).  Formation of these adducts was also obsd. in animals treated with the metabolites, 1-hydroxymethylpyrene and 1-sulfooxymethylpyrene (1-SMP), whereas corresponding data are missing for 1-methylpyrene.  In the present study, we treated mice with 1-methylpyrene and subsequently analyzed blood serum for the presence of the reactive metabolite 1-SMP and tissue DNA for the presence of MPdG and MPdA adducts.  We used wild-type mice and a mouse line transgenic for human sulfotransferases (SULT) 1A1 and 1A2, males and females.  All analyses were conducted using ultra-performance liq. chromatog. coupled with tandem mass spectrometry, for the adducts with isotope-labeled internal stds. 1-SMP was detected in all treated animals.  Its serum level was higher in transgenic mice than in the wild-type (p &lt; 0.001).  Likewise, both adducts were detected in liver, kidney and lung DNA of all exposed animals.  The transgene significantly enhanced the level of each adduct in each tissue of both sexes (p &lt; 0.01-0.001).  Adduct levels were highest in the liver, the target tissue of carcinogenesis, in each animal model used.  MPdG and MPdA adducts were also obsd. in rats treated with 1-methylpyrene.  Our findings corroborate the hypothesis that 1-SMP is indeed the ultimate carcinogen of 1-methylpyrene and that human SULT are able to mediate the terminal activation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-_bKPLkvUnbVg90H21EOLACvtfcHk0lhkPeDq7C2BYw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhvFajsb%252FE&amp;md5=fb85b6ac15fade532bb93abb32b51de1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2Fs00204-013-1182-6&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00204-013-1182-6%26sid%3Dliteratum%253Aachs%26aulast%3DBendadani%26aufirst%3DC.%26aulast%3DMeinl%26aufirst%3DW.%26aulast%3DMonien%26aufirst%3DB.%2BH.%26aulast%3DDobbernack%26aufirst%3DG.%26aulast%3DGlatt%26aufirst%3DH.%26atitle%3DThe%2520Carcinogen%25201-Methylpyrene%2520Forms%2520Benzylic%2520DNA%2520Adducts%2520in%2520Mouse%2520and%2520Rat%2520Tissues%2520in%2520Vivo%2520via%2520a%2520Reactive%2520Sulphuric%2520Acid%2520Ester%26jtitle%3DArch.%2520Toxicol.%26date%3D2014%26volume%3D88%26spage%3D815%26epage%3D821%26doi%3D10.1007%2Fs00204-013-1182-6" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bendadani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monien, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobbernack, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engst, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Himmelbauer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glatt, H.</span></span> <span> </span><span class="NLM_article-title">Determination of Sulfotransferase Forms Involved in the Metabolic Activation of the Genotoxicant 1-Hydroxymethylpyrene Using Bacterially Expressed Enzymes and Genetically Modified Mouse Models</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1060</span>â <span class="NLM_lpage">1069</span>, <span class="refDoi">&nbsp;DOI: 10.1021/tx500129g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx500129g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=27&amp;publication_year=2014&amp;pages=1060-1069&amp;author=C.+Bendadaniauthor=W.+Meinlauthor=B.+Monienauthor=G.+Dobbernackauthor=S.+Florianauthor=W.+Engstauthor=T.+Noldenauthor=H.+Himmelbauerauthor=H.+Glatt&amp;title=Determination+of+Sulfotransferase+Forms+Involved+in+the+Metabolic+Activation+of+the+Genotoxicant+1-Hydroxymethylpyrene+Using+Bacterially+Expressed+Enzymes+and+Genetically+Modified+Mouse+Models&amp;doi=10.1021%2Ftx500129g" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Ftx500129g&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx500129g%26sid%3Dliteratum%253Aachs%26aulast%3DBendadani%26aufirst%3DC.%26aulast%3DMeinl%26aufirst%3DW.%26aulast%3DMonien%26aufirst%3DB.%26aulast%3DDobbernack%26aufirst%3DG.%26aulast%3DFlorian%26aufirst%3DS.%26aulast%3DEngst%26aufirst%3DW.%26aulast%3DNolden%26aufirst%3DT.%26aulast%3DHimmelbauer%26aufirst%3DH.%26aulast%3DGlatt%26aufirst%3DH.%26atitle%3DDetermination%2520of%2520Sulfotransferase%2520Forms%2520Involved%2520in%2520the%2520Metabolic%2520Activation%2520of%2520the%2520Genotoxicant%25201-Hydroxymethylpyrene%2520Using%2520Bacterially%2520Expressed%2520Enzymes%2520and%2520Genetically%2520Modified%2520Mouse%2520Models%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2014%26volume%3D27%26spage%3D1060%26epage%3D1069%26doi%3D10.1021%2Ftx500129g" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guengerich, F. P.</span></span> <span> </span><span class="NLM_article-title">Interactions of Carcinogen-Bound DNA with Individual DNA Polymerases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">420</span>â <span class="NLM_lpage">452</span>, <span class="refDoi">&nbsp;DOI: 10.1021/cr0404693</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr0404693" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD28XitVyksw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=106&amp;publication_year=2006&amp;pages=420-452&amp;author=F.+P.+Guengerich&amp;title=Interactions+of+Carcinogen-Bound+DNA+with+Individual+DNA+Polymerases&amp;doi=10.1021%2Fcr0404693" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Interactions of Carcinogen-Bound DNA with Individual DNA Polymerases</span></div><div class="casAuthors">Guengerich, F. Peter</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">420-452</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Several types of studies with DNA polymerases and carcinogen-modified DNA will be presented and discussed, along with some of the author's own interpretations about relevance.  Although the interactions within cells are most important and many studies have been done, the focus of this review will be studies of individual, isolated DNA polymerases with oligonucleotides bearing chem. defined adducts at specific sites.  The most detailed understanding of systems is possible at this level, albeit reductionist, particularly using kinetic anal. and x-ray crystallog. to study structure and function.  Some discussion of site-specific mutagenesis and mutation spectra will be included.  The point can be made that reliance on only a single approach will never yield a particularly insightful understanding of mechanisms of chem. induced mutagenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps1kyQQ7A0LrVg90H21EOLACvtfcHk0lhkPeDq7C2BYw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28XitVyksw%253D%253D&amp;md5=9ba54dbba504a057056a59dffe3e23ca</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fcr0404693&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr0404693%26sid%3Dliteratum%253Aachs%26aulast%3DGuengerich%26aufirst%3DF.%2BP.%26atitle%3DInteractions%2520of%2520Carcinogen-Bound%2520DNA%2520with%2520Individual%2520DNA%2520Polymerases%26jtitle%3DChem.%2520Rev.%26date%3D2006%26volume%3D106%26spage%3D420%26epage%3D452%26doi%3D10.1021%2Fcr0404693" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essigmann, J. M.</span></span> <span> </span><span class="NLM_article-title">Biological Properties of Single Chemical-DNA Adducts: A Twenty Year Perspective</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">232</span>â <span class="NLM_lpage">252</span>, <span class="refDoi">&nbsp;DOI: 10.1021/tx700292a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx700292a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVWqs7bO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2008&amp;pages=232-252&amp;author=J.+C.+Delaneyauthor=J.+M.+Essigmann&amp;title=Biological+Properties+of+Single+Chemical-DNA+Adducts%3A+A+Twenty+Year+Perspective&amp;doi=10.1021%2Ftx700292a" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Biological Properties of Single Chemical-DNA Adducts: A Twenty Year Perspective</span></div><div class="casAuthors">Delaney, James C.; Essigmann, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">232-252</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The genome and its nucleotide precursor pool are under sustained attack by radiation, reactive oxygen and nitrogen species, chem. carcinogens, hydrolytic reactions, and certain drugs.  As a result, a large and heterogeneous population of damaged nucleotides forms in all cells.  Some of the lesions are repaired, but for those that remain, there can be serious biol. consequences.  For example, lesions that form in DNA can lead to altered gene expression, mutation, and death.  This perspective examines systems developed over the past 20 years to study the biol. properties of single DNA lesions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPu0MDivE9nLVg90H21EOLACvtfcHk0ljLNpuCKhOxFg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXhsVWqs7bO&amp;md5=42900335a8dd1208cf8716272bc021df</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Ftx700292a&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx700292a%26sid%3Dliteratum%253Aachs%26aulast%3DDelaney%26aufirst%3DJ.%2BC.%26aulast%3DEssigmann%26aufirst%3DJ.%2BM.%26atitle%3DBiological%2520Properties%2520of%2520Single%2520Chemical-DNA%2520Adducts%253A%2520A%2520Twenty%2520Year%2520Perspective%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26spage%3D232%26epage%3D252%26doi%3D10.1021%2Ftx700292a" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeCorte, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsarouhtsis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchimanchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, T. M.</span></span> <span> </span><span class="NLM_article-title">Improved Strategies for Postoligomerization Synthesis of Oligodeoxynucleotides Bearing Structurally Defined Adducts at the N2 Position of Deoxyguanosine</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">630</span>â <span class="NLM_lpage">637</span>, <span class="refDoi">&nbsp;DOI: 10.1021/tx9501795</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx9501795" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=1996&amp;pages=630-637&amp;author=B.+L.+DeCorteauthor=D.+Tsarouhtsisauthor=S.+Kuchimanchiauthor=M.+D.+Cooperauthor=P.+Hortonauthor=C.+M.+Harrisauthor=T.+M.+Harris&amp;title=Improved+Strategies+for+Postoligomerization+Synthesis+of+Oligodeoxynucleotides+Bearing+Structurally+Defined+Adducts+at+the+N2+Position+of+Deoxyguanosine&amp;doi=10.1021%2Ftx9501795" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Ftx9501795&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx9501795%26sid%3Dliteratum%253Aachs%26aulast%3DDeCorte%26aufirst%3DB.%2BL.%26aulast%3DTsarouhtsis%26aufirst%3DD.%26aulast%3DKuchimanchi%26aufirst%3DS.%26aulast%3DCooper%26aufirst%3DM.%2BD.%26aulast%3DHorton%26aufirst%3DP.%26aulast%3DHarris%26aufirst%3DC.%2BM.%26aulast%3DHarris%26aufirst%3DT.%2BM.%26atitle%3DImproved%2520Strategies%2520for%2520Postoligomerization%2520Synthesis%2520of%2520Oligodeoxynucleotides%2520Bearing%2520Structurally%2520Defined%2520Adducts%2520at%2520the%2520N2%2520Position%2520of%2520Deoxyguanosine%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D1996%26volume%3D9%26spage%3D630%26epage%3D637%26doi%3D10.1021%2Ftx9501795" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steinbrecher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wameling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oesch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidel, A.</span></span> <span> </span><span class="NLM_article-title">Activation of the C2 Position of Purine by the Trifluoromethanesulfonate Group: Synthesis of N2-Alkylated Deoxyguanosines</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">404</span>â <span class="NLM_lpage">406</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.199304041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1002%2Fanie.199304041" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=32&amp;publication_year=1993&amp;pages=404-406&amp;author=T.+Steinbrecherauthor=C.+Wamelingauthor=F.+Oeschauthor=A.+Seidel&amp;title=Activation+of+the+C2+Position+of+Purine+by+the+Trifluoromethanesulfonate+Group%3A+Synthesis+of+N2-Alkylated+Deoxyguanosines&amp;doi=10.1002%2Fanie.199304041" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fanie.199304041&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.199304041%26sid%3Dliteratum%253Aachs%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DWameling%26aufirst%3DC.%26aulast%3DOesch%26aufirst%3DF.%26aulast%3DSeidel%26aufirst%3DA.%26atitle%3DActivation%2520of%2520the%2520C2%2520Position%2520of%2520Purine%2520by%2520the%2520Trifluoromethanesulfonate%2520Group%253A%2520Synthesis%2520of%2520N2-Alkylated%2520Deoxyguanosines%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D1993%26volume%3D32%26spage%3D404%26epage%3D406%26doi%3D10.1002%2Fanie.199304041" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, G. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krawczyk, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terhorst, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Law, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">CoutrÃ©, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bischofberger, N.</span></span> <span> </span><span class="NLM_article-title">N2- and C8-Substituted Oligodeoxynucleotides with Enhanced Thrombin Inhibitory Activity in Vitro and in Vivo</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">2234</span>â <span class="NLM_lpage">2242</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jm970434d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970434d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=41&amp;publication_year=1998&amp;pages=2234-2242&amp;author=G.+X.+Heauthor=S.+H.+Krawczykauthor=S.+Swaminathanauthor=R.+G.+Sheaauthor=J.+P.+Doughertyauthor=T.+Terhorstauthor=V.+S.+Lawauthor=L.+C.+Griffinauthor=S.+Coutr%C3%A9author=N.+Bischofberger&amp;title=N2-+and+C8-Substituted+Oligodeoxynucleotides+with+Enhanced+Thrombin+Inhibitory+Activity+in+Vitro+and+in+Vivo&amp;doi=10.1021%2Fjm970434d" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm970434d&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970434d%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DG.%2BX.%26aulast%3DKrawczyk%26aufirst%3DS.%2BH.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DShea%26aufirst%3DR.%2BG.%26aulast%3DDougherty%26aufirst%3DJ.%2BP.%26aulast%3DTerhorst%26aufirst%3DT.%26aulast%3DLaw%26aufirst%3DV.%2BS.%26aulast%3DGriffin%26aufirst%3DL.%2BC.%26aulast%3DCoutr%25C3%25A9%26aufirst%3DS.%26aulast%3DBischofberger%26aufirst%3DN.%26atitle%3DN2-%2520and%2520C8-Substituted%2520Oligodeoxynucleotides%2520with%2520Enhanced%2520Thrombin%2520Inhibitory%2520Activity%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D2234%26epage%3D2242%26doi%3D10.1021%2Fjm970434d" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moon, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moschel, R. C.</span></span> <span> </span><span class="NLM_article-title">Effect of Ionic State of 2â²-Deoxyguanosine and Solvent on Its Aralkylation by Benzyl Bromide</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">696</span>â <span class="NLM_lpage">702</span>, <span class="refDoi">&nbsp;DOI: 10.1021/tx980012m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx980012m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=1998&amp;pages=696-702&amp;author=K.+Y.+Moonauthor=R.+C.+Moschel&amp;title=Effect+of+Ionic+State+of+2%E2%80%B2-Deoxyguanosine+and+Solvent+on+Its+Aralkylation+by+Benzyl+Bromide&amp;doi=10.1021%2Ftx980012m" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Ftx980012m&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx980012m%26sid%3Dliteratum%253Aachs%26aulast%3DMoon%26aufirst%3DK.%2BY.%26aulast%3DMoschel%26aufirst%3DR.%2BC.%26atitle%3DEffect%2520of%2520Ionic%2520State%2520of%25202%25E2%2580%25B2-Deoxyguanosine%2520and%2520Solvent%2520on%2520Its%2520Aralkylation%2520by%2520Benzyl%2520Bromide%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D1998%26volume%3D11%26spage%3D696%26epage%3D702%26doi%3D10.1021%2Ftx980012m" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sergueeva, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sergueev, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B. R.</span></span> <span> </span><span class="NLM_article-title">Rapid and Selective Reduction of Amide Group by Borane-Amine Complexes in Acyl Protected Nucleosides</span>. <i>Nucleos. Nucleot. Nucl.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">275</span>â <span class="NLM_lpage">282</span>, <span class="refDoi">&nbsp;DOI: 10.1080/15257770008033009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1080%2F15257770008033009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10772715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVagur4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2000&amp;pages=275-282&amp;author=Z.+A.+Sergueevaauthor=D.+S.+Sergueevauthor=B.+R.+Shaw&amp;title=Rapid+and+Selective+Reduction+of+Amide+Group+by+Borane-Amine+Complexes+in+Acyl+Protected+Nucleosides&amp;doi=10.1080%2F15257770008033009" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid and selective reduction of amide group by borane-amine complexes in acyl protected nucleosides</span></div><div class="casAuthors">Sergueeva, Zinaida A.; Sergueev, Dmitri S.; Shaw, Barbara Ramsay</div><div class="citationInfo"><span class="NLM_cas:title">Nucleosides, Nucleotides &amp; Nucleic Acids</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1 &amp; 2</span>),
    <span class="NLM_cas:pages">275-282</span>CODEN:
                <span class="NLM_cas:coden">NNNAFY</span>;
        ISSN:<span class="NLM_cas:issn">1525-7770</span>.
    
            (<span class="NLM_cas:orgname">Marcel Dekker, Inc.</span>)
        </div><div class="casAbstract">Borane-amine complexes provide an unusually fast and selective redn. of a deoxynucleoside N-acyl group to a corresponding N-alkyl group.  Three different nucleosides (dG, dA, and dC) each having one of three N-protecting groups (benzoyl, isobutyryl, or acetyl) were used to prep. N-alkylated nucleosides in good yields under mild conditions.  Deoxyribose O-acyl protecting groups remain intact at the conditions of N-acyl group redn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIlOUZJbRkbbVg90H21EOLACvtfcHk0li29KesvhWG-w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3cXhvVagur4%253D&amp;md5=31ba42c5e466f5421e4fcc86b76df13e</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1080%2F15257770008033009&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15257770008033009%26sid%3Dliteratum%253Aachs%26aulast%3DSergueeva%26aufirst%3DZ.%2BA.%26aulast%3DSergueev%26aufirst%3DD.%2BS.%26aulast%3DShaw%26aufirst%3DB.%2BR.%26atitle%3DRapid%2520and%2520Selective%2520Reduction%2520of%2520Amide%2520Group%2520by%2520Borane-Amine%2520Complexes%2520in%2520Acyl%2520Protected%2520Nucleosides%26jtitle%3DNucleos.%2520Nucleot.%2520Nucl.%26date%3D2000%26volume%3D19%26spage%3D275%26epage%3D282%26doi%3D10.1080%2F15257770008033009" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guengerich, F. P.</span></span> <span> </span><span class="NLM_article-title">Analysis of the Effect of Bulk at N2-Alkylguanine DNA Adducts on Catalytic Efficiency and Fidelity of the Processive DNA Polymerases Bacteriophage T7 Exonuclease- and HIV-1 Reverse Transcriptase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">19217</span>â <span class="NLM_lpage">19229</span>, <span class="refDoi">&nbsp;DOI: 10.1074/jbc.M313759200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1074%2Fjbc.M313759200" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=279&amp;publication_year=2004&amp;pages=19217-19229&amp;author=J.+Choiauthor=F.+P.+Guengerich&amp;title=Analysis+of+the+Effect+of+Bulk+at+N2-Alkylguanine+DNA+Adducts+on+Catalytic+Efficiency+and+Fidelity+of+the+Processive+DNA+Polymerases+Bacteriophage+T7+Exonuclease-+and+HIV-1+Reverse+Transcriptase&amp;doi=10.1074%2Fjbc.M313759200" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M313759200&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M313759200%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%26aulast%3DGuengerich%26aufirst%3DF.%2BP.%26atitle%3DAnalysis%2520of%2520the%2520Effect%2520of%2520Bulk%2520at%2520N2-Alkylguanine%2520DNA%2520Adducts%2520on%2520Catalytic%2520Efficiency%2520and%2520Fidelity%2520of%2520the%2520Processive%2520DNA%2520Polymerases%2520Bacteriophage%2520T7%2520Exonuclease-%2520and%2520HIV-1%2520Reverse%2520Transcriptase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D19217%26epage%3D19229%26doi%3D10.1074%2Fjbc.M313759200" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonala, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shishkina, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, F.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Biologically Active N2-Amine Adducts of 2â²-Deoxyguanosine</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">7281</span>â <span class="NLM_lpage">7284</span>, <span class="refDoi">&nbsp;DOI: 10.1016/S0040-4039(00)01196-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1016%2FS0040-4039%2800%2901196-5" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=41&amp;publication_year=2000&amp;pages=7281-7284&amp;author=R.+R.+Bonalaauthor=I.+G.+Shishkinaauthor=F.+Johnson&amp;title=Synthesis+of+Biologically+Active+N2-Amine+Adducts+of+2%E2%80%B2-Deoxyguanosine&amp;doi=10.1016%2FS0040-4039%2800%2901196-5" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2800%2901196-5&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252800%252901196-5%26sid%3Dliteratum%253Aachs%26aulast%3DBonala%26aufirst%3DR.%2BR.%26aulast%3DShishkina%26aufirst%3DI.%2BG.%26aulast%3DJohnson%26aufirst%3DF.%26atitle%3DSynthesis%2520of%2520Biologically%2520Active%2520N2-Amine%2520Adducts%2520of%25202%25E2%2580%25B2-Deoxyguanosine%26jtitle%3DTetrahedron%2520Lett.%26date%3D2000%26volume%3D41%26spage%3D7281%26epage%3D7284%26doi%3D10.1016%2FS0040-4039%2800%2901196-5" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luna, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stezowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiselyo, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, R. G.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Oligonucleotide Adducts of the Bay Region Diol Epoxide Metabolites of Carcinogenic Polycyclic Aromatic Hydrocarbons</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5604</span>â <span class="NLM_lpage">5613</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jo00122a048</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00122a048" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=60&amp;publication_year=1995&amp;pages=5604-5613&amp;author=H.+Leeauthor=E.+Lunaauthor=M.+Hinzauthor=J.+J.+Stezowskiauthor=A.+S.+Kiselyoauthor=R.+G.+Harvey&amp;title=Synthesis+of+Oligonucleotide+Adducts+of+the+Bay+Region+Diol+Epoxide+Metabolites+of+Carcinogenic+Polycyclic+Aromatic+Hydrocarbons&amp;doi=10.1021%2Fjo00122a048" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjo00122a048&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00122a048%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%26aulast%3DLuna%26aufirst%3DE.%26aulast%3DHinz%26aufirst%3DM.%26aulast%3DStezowski%26aufirst%3DJ.%2BJ.%26aulast%3DKiselyo%26aufirst%3DA.%2BS.%26aulast%3DHarvey%26aufirst%3DR.%2BG.%26atitle%3DSynthesis%2520of%2520Oligonucleotide%2520Adducts%2520of%2520the%2520Bay%2520Region%2520Diol%2520Epoxide%2520Metabolites%2520of%2520Carcinogenic%2520Polycyclic%2520Aromatic%2520Hydrocarbons%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1995%26volume%3D60%26spage%3D5604%26epage%3D5613%26doi%3D10.1021%2Fjo00122a048" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kath, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jergic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heltzel, J. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loparo, J. J.</span></span> <span> </span><span class="NLM_article-title">Polymerase Exchange on Single DNA Molecules Reveals Processivity Clamp Control of Translesion Synthesis</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">7647</span>â <span class="NLM_lpage">7652</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1321076111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1073%2Fpnas.1321076111" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=111&amp;publication_year=2014&amp;pages=7647-7652&amp;author=J.+E.+Kathauthor=S.+Jergicauthor=J.+M.+H.+Heltzelauthor=D.+T.+Jacobauthor=N.+E.+Dixonauthor=M.+D.+Suttonauthor=G.+C.+Walkerauthor=J.+J.+Loparo&amp;title=Polymerase+Exchange+on+Single+DNA+Molecules+Reveals+Processivity+Clamp+Control+of+Translesion+Synthesis&amp;doi=10.1073%2Fpnas.1321076111" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1321076111&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1321076111%26sid%3Dliteratum%253Aachs%26aulast%3DKath%26aufirst%3DJ.%2BE.%26aulast%3DJergic%26aufirst%3DS.%26aulast%3DHeltzel%26aufirst%3DJ.%2BM.%2BH.%26aulast%3DJacob%26aufirst%3DD.%2BT.%26aulast%3DDixon%26aufirst%3DN.%2BE.%26aulast%3DSutton%26aufirst%3DM.%2BD.%26aulast%3DWalker%26aufirst%3DG.%2BC.%26aulast%3DLoparo%26aufirst%3DJ.%2BJ.%26atitle%3DPolymerase%2520Exchange%2520on%2520Single%2520DNA%2520Molecules%2520Reveals%2520Processivity%2520Clamp%2520Control%2520of%2520Translesion%2520Synthesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D7647%26epage%3D7652%26doi%3D10.1073%2Fpnas.1321076111" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghodke, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harikrishna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeepkumar, P. I.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Polymerase-Mediated Bypass Studies of the N2-Deoxyguanosine DNA Damage Caused by a Lucidin Analogue</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">2128</span>â <span class="NLM_lpage">2138</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jo502627b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo502627b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=80&amp;publication_year=2015&amp;pages=2128-2138&amp;author=P.+P.+Ghodkeauthor=S.+Harikrishnaauthor=P.+I.+Pradeepkumar&amp;title=Synthesis+and+Polymerase-Mediated+Bypass+Studies+of+the+N2-Deoxyguanosine+DNA+Damage+Caused+by+a+Lucidin+Analogue&amp;doi=10.1021%2Fjo502627b" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjo502627b&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo502627b%26sid%3Dliteratum%253Aachs%26aulast%3DGhodke%26aufirst%3DP.%2BP.%26aulast%3DHarikrishna%26aufirst%3DS.%26aulast%3DPradeepkumar%26aufirst%3DP.%2BI.%26atitle%3DSynthesis%2520and%2520Polymerase-Mediated%2520Bypass%2520Studies%2520of%2520the%2520N2-Deoxyguanosine%2520DNA%2520Damage%2520Caused%2520by%2520a%2520Lucidin%2520Analogue%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2015%26volume%3D80%26spage%3D2128%26epage%3D2138%26doi%3D10.1021%2Fjo502627b" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Champeil, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradhan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakshman, M. K.</span></span> <span> </span><span class="NLM_article-title">Palladium-Catalyzed Synthesis of Nucleoside Adducts from Bay- and Fjord-Region Diol Epoxides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">5035</span>â <span class="NLM_lpage">5045</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jo070204z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo070204z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt12hu7o%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=72&amp;publication_year=2007&amp;pages=5035-5045&amp;author=E.+Champeilauthor=P.+Pradhanauthor=M.+K.+Lakshman&amp;title=Palladium-Catalyzed+Synthesis+of+Nucleoside+Adducts+from+Bay-+and+Fjord-Region+Diol+Epoxides&amp;doi=10.1021%2Fjo070204z" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-Catalyzed Synthesis of Nucleoside Adducts from Bay- and Fjord-Region Diol Epoxides</span></div><div class="casAuthors">Champeil, Elise; Pradhan, Padmanava; Lakshman, Mahesh K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5035-5045</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Palladium-catalyzed C-N bond formation has been utilized to synthesize covalent 2'-deoxyadenosine (dA) and 2'-deoxyguanosine (dG) adducts of benzo[a]pyrene (BaP) series 1 (syn) and benzo[c]phenanthrene (BcPh) series 2 (anti) diol epoxides.  For this, (Â±)-10Î±-amino-7Î²,8Î±,9Î²-tris-benzoyl-oxy-7,8,9,10-tetrahydro (BaP) and (Â±)-1Î²-amino-2Î±,3Î±,4Î²-tris-benzoyl-oxy-1,2,3,4-tetrahydro (BcPh) were coupled with 6-halo-9-[3,5-bis-O-(tert-butyldimethylsilyl)-Î²-D-erythro-pentofuranosyl]purine and O6-benzyl-3',5'-bis-O-(tert-butyldimethylsilyl)-2-bromo-2'-deoxyinosine, using a (Â±)-BINAP-Pd complex and Cs2CO3.  For the synthesis of the dA adducts, both the 6-chloro- as well as the 6-bromopurine nucleoside derivs. were analyzed for the C-N coupling reaction with the hydrocarbon amino tri-benzoates.  With the BaP amino tri-benzoate, the 6-chloro-nucleoside provided satisfactory results, whereas the 6-bromo analog proved to be superior with the BcPh amino tri-benzoate.  Overall, lower yields of the dA adducts were obtained with the more hindered fjord-region BcPh amino tri-benzoate as compared to the bay-region BaP amino tri-benzoate.  In contrast to reactions leading to the dA adducts, the C-N reactions of both BaP and BcPh amino tri-benzoates with the 2-bromo-2'-deoxyinosine deriv. proceeded in comparable yields.  This seems to indicate that such Pd-catalyzed adduct forming reactions at the C-6 position may be influenced by steric constraints of the amine component, whereas those at the C-2 position are less sensitive.  Diastereomeric adduct pairs were sepd. and characterized by spectral methods and by comparisons to adducts produced by direct displacement reactions as well as those formed from DNA alkylation by diol epoxides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz2CwZQEPKzLVg90H21EOLACvtfcHk0liC5_kzbLTb0A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXmt12hu7o%253D&amp;md5=99df3462308844fa54f7e921cf3bcf65</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjo070204z&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo070204z%26sid%3Dliteratum%253Aachs%26aulast%3DChampeil%26aufirst%3DE.%26aulast%3DPradhan%26aufirst%3DP.%26aulast%3DLakshman%26aufirst%3DM.%2BK.%26atitle%3DPalladium-Catalyzed%2520Synthesis%2520of%2520Nucleoside%2520Adducts%2520from%2520Bay-%2520and%2520Fjord-Region%2520Diol%2520Epoxides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2007%26volume%3D72%26spage%3D5035%26epage%3D5045%26doi%3D10.1021%2Fjo070204z" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Cu<sup>2+</sup> Fluorescent Sensor Based on Mesoporous Silica Nanosphere</span>. <i>Dyes Pigm.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">239</span>â <span class="NLM_lpage">246</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.dyepig.2012.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1016%2Fj.dyepig.2012.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsFGit70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=94&amp;publication_year=2012&amp;pages=239-246&amp;author=D.+Luauthor=J.+Leiauthor=Z.+Tianauthor=L.+Wangauthor=J.+Zhang&amp;title=Cu2%2B+Fluorescent+Sensor+Based+on+Mesoporous+Silica+Nanosphere&amp;doi=10.1016%2Fj.dyepig.2012.01.004" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Cu2+ fluorescent sensor based on mesoporous silica nanosphere</span></div><div class="casAuthors">Lu, Deli; Lei, Juying; Tian, Zhidan; Wang, Lingzhi; Zhang, Jinlong</div><div class="citationInfo"><span class="NLM_cas:title">Dyes and Pigments</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">239-246</span>CODEN:
                <span class="NLM_cas:coden">DYPIDX</span>;
        ISSN:<span class="NLM_cas:issn">0143-7208</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Monodisperse mesoporous silica nanosphere (MSN) modified with anthracene deriv., 2-(3-(triethoxysilyl) propylamino)-N-(9-anthrylMe) acetamide (SGAAn) was fabricated as a fluorescent sensor (SGAAn-MSN) for the detection of metal ions, which shows an exclusive selectivity to Cu2+.  Compared with SGAAn, SGAAn-MSN presents a higher sensitivity to Cu2+.  The loading amt. of SGAAn in MSN has a great influence on the detection sensitivity of SGAAn-MSN, which varies with the local concn. and accessibility of SGAAn in the pore channel of MSN.  Interference studies indicate that the addn. of other metal ions such as Ag+, Cd2+ and Pb2+ has a negligible effect on the selectivity of SGAAn-MSN to Cu2+.  The possible mechanism for the sensing behavior of SGAAn-MSN to Cu2+ is proposed based on the relation between fluorescence quench degree and Cu2+ concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopuymFCLOUWLVg90H21EOLACvtfcHk0liIK05b5pFJkg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XjsFGit70%253D&amp;md5=2a71800d6a25bd7f72c702c1940ee465</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.dyepig.2012.01.004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dyepig.2012.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DD.%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DCu2%252B%2520Fluorescent%2520Sensor%2520Based%2520on%2520Mesoporous%2520Silica%2520Nanosphere%26jtitle%3DDyes%2520Pigm.%26date%3D2012%26volume%3D94%26spage%3D239%26epage%3D246%26doi%3D10.1016%2Fj.dyepig.2012.01.004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abiraj, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gowda, D. C.</span></span> <span> </span><span class="NLM_article-title">Zinc/Ammonium Formate: A New Facile System for the Rapid and Selective Reduction of Oximes to Amines</span>. <i>J. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2003</i></span>,  <span class="NLM_fpage">332</span>â <span class="NLM_lpage">334</span>, <span class="refDoi">&nbsp;DOI: 10.3184/030823403103174281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.3184%2F030823403103174281" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2003&amp;publication_year=2003&amp;pages=332-334&amp;author=K.+Abirajauthor=D.+C.+Gowda&amp;title=Zinc%2FAmmonium+Formate%3A+A+New+Facile+System+for+the+Rapid+and+Selective+Reduction+of+Oximes+to+Amines&amp;doi=10.3184%2F030823403103174281" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.3184%2F030823403103174281&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.3184%252F030823403103174281%26sid%3Dliteratum%253Aachs%26aulast%3DAbiraj%26aufirst%3DK.%26aulast%3DGowda%26aufirst%3DD.%2BC.%26atitle%3DZinc%252FAmmonium%2520Formate%253A%2520A%2520New%2520Facile%2520System%2520for%2520the%2520Rapid%2520and%2520Selective%2520Reduction%2520of%2520Oximes%2520to%2520Amines%26jtitle%3DJ.%2520Chem.%2520Res.%26date%3D2003%26volume%3D2003%26spage%3D332%26epage%3D334%26doi%3D10.3184%2F030823403103174281" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, T.</span></span> <span> </span><span class="NLM_article-title">Selective Enzymatic Demethylation of <i>N</i><sup>2</sup>,<i>N</i><sup>2</sup>-Dimethylguanosine in RNA and Its Application in High-Throughput tRNA Sequencing</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5017</span>â <span class="NLM_lpage">5020</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.201700537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1002%2Fanie.201700537" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=56&amp;publication_year=2017&amp;pages=5017-5020&amp;author=Q.+Daiauthor=G.+Zhengauthor=M.+H.+Schwartzauthor=W.+C.+Clarkauthor=T.+Pan&amp;title=Selective+Enzymatic+Demethylation+of+N2%2CN2-Dimethylguanosine+in+RNA+and+Its+Application+in+High-Throughput+tRNA+Sequencing&amp;doi=10.1002%2Fanie.201700537" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.201700537&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201700537%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DG.%26aulast%3DSchwartz%26aufirst%3DM.%2BH.%26aulast%3DClark%26aufirst%3DW.%2BC.%26aulast%3DPan%26aufirst%3DT.%26atitle%3DSelective%2520Enzymatic%2520Demethylation%2520of%2520N2%252CN2-Dimethylguanosine%2520in%2520RNA%2520and%2520Its%2520Application%2520in%2520High-Throughput%2520tRNA%2520Sequencing%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5017%26epage%3D5020%26doi%3D10.1002%2Fanie.201700537" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Himmelsbach, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trichtinger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charubala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfleiderer, W.</span></span> <span> </span><span class="NLM_article-title">The p-nitrophenylethyl (NPE) group: A versatile new blocking group for phosphate and aglycone protection in nucleosides and nucleotides</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">59</span>â <span class="NLM_lpage">72</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0040-4020(84)85103-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1016%2F0040-4020%2884%2985103-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADyaL2cXltFWgsLo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=40&amp;publication_year=1984&amp;pages=59-72&amp;author=F.+Himmelsbachauthor=B.+S.+Schulzauthor=T.+Trichtingerauthor=R.+Charubalaauthor=W.+Pfleiderer&amp;title=The+p-nitrophenylethyl+%28NPE%29+group%3A+A+versatile+new+blocking+group+for+phosphate+and+aglycone+protection+in+nucleosides+and+nucleotides&amp;doi=10.1016%2F0040-4020%2884%2985103-0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The p-nitrophenylethyl (NPE) group.  A versatile new blocking group for phosphate and aglycone protection in nucleosides and nucleotides</span></div><div class="casAuthors">Himmelsbach, Frank; Schulz, Bernd S.; Trichtinger, Thomas; Charubala, Ramamurthy; Pfleiderer, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">59-72</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    </div><div class="casAbstract">The syntheses of new monomeric building blocks for oligonucleotide synthesis via the phosphotriester approach contg. the p-nitrophenylethyl group for phosphate and aglycon protection are described.  Blocking of the amide function in guanosines at O6 can be achieved by the Mitsunobu reaction forming the corresponding O6-p-nitrophenylethyl derivs.  Sugar-protected thymidine and uridine have been alkylated at O4 in an SN1-type reaction by p-nitrophenylethyl iodide-Ag2CO3 in C6H6 to form the O4-p-nitrophenylethyl derivs.  Protection of the NH2 group in 2'-deoxycytidine and cytidine can be performed directly by 1-(p-nitrophenylethoxycarbonyl)benzotriazole in DMF to obtain the corresponding carbamates as a new type of N4-acylated cytidine deriv.  p-Nitrophenylethoxycarbonylation of the NH2 group in 2'-deoxyadenosine and adenosine requires previous sugar protection by acyl or silyl groups and can then be achieved by p-nitrophenylethyl chloroformate or better by 1-methyl-3-p-nitrophenylethoxycarbonylimidazolium chloride to form N6-p-nitrophenylethoxycarbonyladenosines.  The various p-nitrophenylethyl blocking groups are stable under mild hydrolytic conditions (e.g., NH3 and Et3N) but can be cleaved selectively by DBU or DBN in aprotic solvents.  5'-O-Monomethoxytritylation as well as phosphorylation at the 3'-OH group can be effected to give the corresponding 3'-(2,5-dichlorophenyl p-nitrophenylethyl) phosphotriesters also in high yields.  Oximate cleavage of the latter compds. to the phosphodiesters and detritylation to the 5'-unblocked phosphotriesters do not affect the aglycon protecting groups, thereby demonstrating their general versatility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraD2QuKCEtrbVg90H21EOLACvtfcHk0liIK05b5pFJkg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL2cXltFWgsLo%253D&amp;md5=eab017017027631de697686f0c76bdaf</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2F0040-4020%2884%2985103-0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0040-4020%252884%252985103-0%26sid%3Dliteratum%253Aachs%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DSchulz%26aufirst%3DB.%2BS.%26aulast%3DTrichtinger%26aufirst%3DT.%26aulast%3DCharubala%26aufirst%3DR.%26aulast%3DPfleiderer%26aufirst%3DW.%26atitle%3DThe%2520p-nitrophenylethyl%2520%2528NPE%2529%2520group%253A%2520A%2520versatile%2520new%2520blocking%2520group%2520for%2520phosphate%2520and%2520aglycone%2520protection%2520in%2520nucleosides%2520and%2520nucleotides%26jtitle%3DTetrahedron%26date%3D1984%26volume%3D40%26spage%3D59%26epage%3D72%26doi%3D10.1016%2F0040-4020%2884%2985103-0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kottur, J.</span>; <span class="NLM_string-name">Ghodke, P. P.</span>; <span class="NLM_string-name">Nair, D. T.</span>; <span class="NLM_string-name">Pradeepkumar, P. I.</span></span> Manuscript in preparation.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=Kottur%2C+J.%3B+Ghodke%2C+P.+P.%3B+Nair%2C+D.+T.%3B+Pradeepkumar%2C+P.+I.+Manuscript+in+preparation." target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DKottur%26aufirst%3DJ." title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixit, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trejo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juliano, R. L.</span></span> <span> </span><span class="NLM_article-title">Intracellular Delivery of an Anionic Antisense Oligonucleotide via Receptor-Mediated Endocytosis</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2764</span>â <span class="NLM_lpage">2776</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gkn115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1093%2Fnar%2Fgkn115" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=2008&amp;pages=2764-2776&amp;author=R.+Alamauthor=V.+Dixitauthor=H.+Kangauthor=Z.+B.+Liauthor=X.+Chenauthor=J.+Trejoauthor=M.+Fisherauthor=R.+L.+Juliano&amp;title=Intracellular+Delivery+of+an+Anionic+Antisense+Oligonucleotide+via+Receptor-Mediated+Endocytosis&amp;doi=10.1093%2Fnar%2Fgkn115" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn115&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn115%26sid%3Dliteratum%253Aachs%26aulast%3DAlam%26aufirst%3DR.%26aulast%3DDixit%26aufirst%3DV.%26aulast%3DKang%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DZ.%2BB.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DTrejo%26aufirst%3DJ.%26aulast%3DFisher%26aufirst%3DM.%26aulast%3DJuliano%26aufirst%3DR.%2BL.%26atitle%3DIntracellular%2520Delivery%2520of%2520an%2520Anionic%2520Antisense%2520Oligonucleotide%2520via%2520Receptor-Mediated%2520Endocytosis%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D2764%26epage%3D2776%26doi%3D10.1093%2Fnar%2Fgkn115" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sholder, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creech, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loechler, E. L.</span></span> <span> </span><span class="NLM_article-title">How Y-Family DNA Polymerase IV Is More Accurate than Dpo4 at dCTP Insertion Opposite an <i>N</i><sup>2</sup>-dG Adduct of Benzo[<i>a</i>]Pyrene</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">144</span>â <span class="NLM_lpage">153</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.dnarep.2015.09.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1016%2Fj.dnarep.2015.09.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=26523515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKqsLfF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=35&amp;publication_year=2015&amp;pages=144-153&amp;author=G.+Sholderauthor=A.+Creechauthor=E.+L.+Loechler&amp;title=How+Y-Family+DNA+Polymerase+IV+Is+More+Accurate+than+Dpo4+at+dCTP+Insertion+Opposite+an+N2-dG+Adduct+of+Benzo%5Ba%5DPyrene&amp;doi=10.1016%2Fj.dnarep.2015.09.020" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">How Y-Family DNA polymerase IV is more accurate than Dpo4 at dCTP insertion opposite an N2-dG adduct of benzo[a]pyrene</span></div><div class="casAuthors">Sholder, Gabriel; Creech, Amanda; Loechler, Edward L.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">144-153</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">To bypass DNA damage, cells have Y-Family DNA polymerases (DNAPs).  One Y-Family-class includes DNAP Îº and DNAP IV, which accurately insert dCTP opposite N2-dG adducts, including from the carcinogen benzo[a]pyrene (BP).  Another class includes DNAP Î· and DNAP V, which insert accurately opposite UV-damage, but inaccurately opposite BP-N2-dG.  To investigate structural differences between Y-Family-classes, regions are swapped between DNAP IV (a Îº/IV-class-member) and Dpo4 (a Î·/V-class-member); the kinetic consequences are evaluated via primer-extension studies with a BP-N2-dG-contg. template.  Four key structural elements are revealed.  (1) Y-Family DNAPs have discreet non-covalent contacts between their little finger-domain (LF-Domain) and their catalytic core-domain (CC-Domain), which we call "non-covalent bridges" (NCBs).  Arg37 and Arg38 in DNAP IV's CC-Domain near the active site form a non-covalent bridge (AS-NCB) by interacting with Glu251 and Asp252, resp., in DNAP IV's LF-Domain.  Without these interactions dATP/dGTP/dTTP misinsertions increase.  DNAP IV's AS-NCB suppresses misinsertions better than Dpo4's equiv. AS-NCB.  (2) DNAP IV also suppresses dATP/dGTP/dTTP misinsertions via a second non-covalent bridge, which is â¼8 Ã from the active site (Distal-NCB).  Dpo4 has no Distal-NCB, rendering it inferior at dATP/dGTP/dTTP suppression.  (3) dCTP insertion is facilitated by the larger minor groove opening near the active site in DNAP IV vs. Dpo4, which is sensible given that Watson/Crick-like [dCTP:BP-N2-dG] pairing requires the BP-moiety to be in the minor groove.  (4) Compared to Dpo4, DNAP IV has a smaller major groove opening, which suppresses dGTP misinsertion, implying BP-N2-dG bulk in the major groove during Hoogsteen syn-adduct-dG:dGTP pairing.  In summary, DNAP IV has a large minor groove opening to enhance dCTP insertion, a plugged major groove opening to suppress dGTP misinsertion, and two non-covalent bridges (near and distal to the active site) to suppress dATP/dGTP/dTTP misinsertions; collectively these four structural features enhance DNAP IV's dNTP insertion fidelity opposite a BP-N2-dG adduct compared to Dpo4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK6l92GpV4m7Vg90H21EOLACvtfcHk0lj7u5fmY0Xgbw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKqsLfF&amp;md5=5a432c52b2c362b4b003d35c2b3d366c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2015.09.020&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2015.09.020%26sid%3Dliteratum%253Aachs%26aulast%3DSholder%26aufirst%3DG.%26aulast%3DCreech%26aufirst%3DA.%26aulast%3DLoechler%26aufirst%3DE.%2BL.%26atitle%3DHow%2520Y-Family%2520DNA%2520Polymerase%2520IV%2520Is%2520More%2520Accurate%2520than%2520Dpo4%2520at%2520dCTP%2520Insertion%2520Opposite%2520an%2520N2-dG%2520Adduct%2520of%2520Benzo%255Ba%255DPyrene%26jtitle%3DDNA%2520Repair%26date%3D2015%26volume%3D35%26spage%3D144%26epage%3D153%26doi%3D10.1016%2Fj.dnarep.2015.09.020" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghodke, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gore, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harikrishna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kottur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeepkumar, P. I.</span></span> <span> </span><span class="NLM_article-title">The N2-Furfuryl-Deoxyguanosine Adduct Does Not Alter the Structure of B-DNA</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">502</span>â <span class="NLM_lpage">511</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.joc.5b02341</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.joc.5b02341" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=81&amp;publication_year=2016&amp;pages=502-511&amp;author=P.+P.+Ghodkeauthor=K.+R.+Goreauthor=S.+Harikrishnaauthor=B.+Samantaauthor=J.+Kotturauthor=D.+T.+Nairauthor=P.+I.+Pradeepkumar&amp;title=The+N2-Furfuryl-Deoxyguanosine+Adduct+Does+Not+Alter+the+Structure+of+B-DNA&amp;doi=10.1021%2Facs.joc.5b02341" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.5b02341&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.5b02341%26sid%3Dliteratum%253Aachs%26aulast%3DGhodke%26aufirst%3DP.%2BP.%26aulast%3DGore%26aufirst%3DK.%2BR.%26aulast%3DHarikrishna%26aufirst%3DS.%26aulast%3DSamanta%26aufirst%3DB.%26aulast%3DKottur%26aufirst%3DJ.%26aulast%3DNair%26aufirst%3DD.%2BT.%26aulast%3DPradeepkumar%26aufirst%3DP.%2BI.%26atitle%3DThe%2520N2-Furfuryl-Deoxyguanosine%2520Adduct%2520Does%2520Not%2520Alter%2520the%2520Structure%2520of%2520B-DNA%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2016%26volume%3D81%26spage%3D502%26epage%3D511%26doi%3D10.1021%2Facs.joc.5b02341" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanner, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span> <span> </span><span class="NLM_article-title">Clustering Molecular Dynamics Trajectories: 1. Characterizing the Performance of Different Clustering Algorithms</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">2312</span>â <span class="NLM_lpage">2334</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct700119m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700119m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFaqsLzE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2007&amp;pages=2312-2334&amp;author=J.+Shaoauthor=S.+W.+Tannerauthor=N.+Thompsonauthor=T.+E.+Cheatham&amp;title=Clustering+Molecular+Dynamics+Trajectories%3A+1.+Characterizing+the+Performance+of+Different+Clustering+Algorithms&amp;doi=10.1021%2Fct700119m" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Clustering Molecular Dynamics Trajectories: 1. Characterizing the Performance of Different Clustering Algorithms</span></div><div class="casAuthors">Shao, Jianyin; Tanner, Stephen W.; Thompson, Nephi; Cheatham, Thomas E., III</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2334</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics simulation methods produce trajectories of at. positions (and optionally velocities and energies) as a function of time and provide a representation of the sampling of a given mol.'s energetically accessible conformational ensemble.  As simulations on the 10-100 ns time scale become routine, with sampled configurations stored on the picosecond time scale, such trajectories contain large amts. of data.  Data-mining techniques, like clustering, provide one means to group and make sense of the information in the trajectory.  In this work, several clustering algorithms were implemented, compared, and utilized to understand MD trajectory data.  The development of the algorithms into a freely available C code library, and their application to a simple test example of random (or systematically placed) points in a 2D plane (where the pairwise metric is the distance between points) provide a means to understand the relative performance.  Eleven different clustering algorithms were developed, ranging from top-down splitting (hierarchical) and bottom-up aggregating (including single-linkage edge joining, centroid-linkage, av.-linkage, complete-linkage, centripetal, and centripetal-complete) to various refinement (means, Bayesian, and self-organizing maps) and tree (COBWEB) algorithms.  Systematic testing in the context of MD simulation of various DNA systems (including DNA single strands and the interaction of a minor groove binding drug DB226 with a DNA hairpin) allows a more direct assessment of the relative merits of the distinct clustering algorithms.  Addnl., means to assess the relative performance and differences between the algorithms, to dynamically select the initial cluster count, and to achieve faster data mining by "sieved clustering" were evaluated.  Overall, it was found that there is no one perfect "one size fits all" algorithm for clustering MD trajectories and that the results strongly depend on the choice of atoms for the pairwise comparison.  Some algorithms tend to produce homogeneously sized clusters, whereas others have a tendency to produce singleton clusters.  Issues related to the choice of a pairwise metric, clustering metrics, which atom selection is used for the comparison, and about the relative performance are discussed.  Overall, the best performance was obsd. with the av.-linkage, means, and SOM algorithms.  If the cluster count is not known in advance, the hierarchical or av.-linkage clustering algorithms are recommended.  Although these algorithms perform well, it is important to be aware of the limitations or weaknesses of each algorithm, specifically the high sensitivity to outliers with hierarchical, the tendency to generate homogenously sized clusters with means, and the tendency to produce small or singleton clusters with av.-linkage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzr3sLW2dkQbVg90H21EOLACvtfcHk0lhytYliYp1E4g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXhtFaqsLzE&amp;md5=668cff11c29736325a4788379cc43def</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fct700119m&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700119m%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DTanner%26aufirst%3DS.%2BW.%26aulast%3DThompson%26aufirst%3DN.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26atitle%3DClustering%2520Molecular%2520Dynamics%2520Trajectories%253A%25201.%2520Characterizing%2520the%2520Performance%2520of%2520Different%2520Clustering%2520Algorithms%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2007%26volume%3D3%26spage%3D2312%26epage%3D2334%26doi%3D10.1021%2Fct700119m" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roe, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span> <span> </span><span class="NLM_article-title">PTRAJ and CPPTRAJ: Software for Processing and Analysis of Molecular Dynamics Trajectory Data</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3084</span>â <span class="NLM_lpage">3095</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct400341p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400341p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2013&amp;pages=3084-3095&amp;author=D.+R.+Roeauthor=T.+E.+Cheatham&amp;title=PTRAJ+and+CPPTRAJ%3A+Software+for+Processing+and+Analysis+of+Molecular+Dynamics+Trajectory+Data&amp;doi=10.1021%2Fct400341p" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">PTRAJ and CPPTRAJ: Software for Processing and Analysis of Molecular Dynamics Trajectory Data</span></div><div class="casAuthors">Roe, Daniel R.; Cheatham, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3084-3095</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe PTRAJ and its successor CPPTRAJ, two complementary, portable, and freely available computer programs for the anal. and processing of time series of three-dimensional at. positions (i.e., coordinate trajectories) and the data therein derived.  Common tools include the ability to manipulate the data to convert among trajectory formats, process groups of trajectories generated with ensemble methods (e.g., replica exchange mol. dynamics), image with periodic boundary conditions, create av. structures, strip subsets of the system, and perform calcns. such as RMS fitting, measuring distances, B-factors, radii of gyration, radial distribution functions, and time correlations, among other actions and analyses.  Both the PTRAJ and CPPTRAJ programs and source code are freely available under the GNU General Public License version 3 and are currently distributed within the AmberTools 12 suite of support programs that make up part of the Amber package of computer programs (see http://ambermd.org).  This overview describes the general design, features, and history of these two programs, as well as algorithmic improvements and new features available in CPPTRAJ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq97UsRfUy65bVg90H21EOLACvtfcHk0lhytYliYp1E4g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D&amp;md5=6f1bee934f13f180bd7e1feb6b78036d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fct400341p&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400341p%26sid%3Dliteratum%253Aachs%26aulast%3DRoe%26aufirst%3DD.%2BR.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26atitle%3DPTRAJ%2520and%2520CPPTRAJ%253A%2520Software%2520for%2520Processing%2520and%2520Analysis%2520of%2520Molecular%2520Dynamics%2520Trajectory%2520Data%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3084%26epage%3D3095%26doi%3D10.1021%2Fct400341p" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span>; <span class="NLM_string-name">Babin, V.</span>; <span class="NLM_string-name">Berryman, J. T.</span>; <span class="NLM_string-name">Betz, R. M.</span>; <span class="NLM_string-name">Cai, Q.</span>; <span class="NLM_string-name">Cerutti, D. S.</span>; <span class="NLM_string-name">Cheatham, T. E.,  III</span>; <span class="NLM_string-name">Darden, T. A.</span>; <span class="NLM_string-name">Duke, R. E.</span>; <span class="NLM_string-name">Gohlke, H.</span>; <span class="NLM_string-name">Goetz, A. W.</span>; <span class="NLM_string-name">Gusarov, S.</span>; <span class="NLM_string-name">Homeyer, N.</span>; <span class="NLM_string-name">Janowski, P.</span>; <span class="NLM_string-name">Kaus, J.</span>; <span class="NLM_string-name">KolossvÃ¡ry, I.</span>; <span class="NLM_string-name">Kovalenko, A.</span>; <span class="NLM_string-name">Lee, T. S.</span>; <span class="NLM_string-name">LeGrand, S.</span>; <span class="NLM_string-name">Luchko, T.</span>; <span class="NLM_string-name">Luo, R. B.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Kollaman, P. A.</span></span> <i>AMBER 14</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco, CA</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2014&amp;author=D.+A.+Case&amp;author=V.+Babin&amp;author=J.+T.+Berryman&amp;author=R.+M.+Betz&amp;author=Q.+Cai&amp;author=D.+S.+Cerutti&amp;author=T.+E.+Cheatham&amp;author=T.+A.+Darden&amp;author=R.+E.+Duke&amp;author=H.+Gohlke&amp;author=A.+W.+Goetz&amp;author=S.+Gusarov&amp;author=N.+Homeyer&amp;author=P.+Janowski&amp;author=J.+Kaus&amp;author=I.+Kolossv%C3%A1ry&amp;author=A.+Kovalenko&amp;author=T.+S.+Lee&amp;author=S.+LeGrand&amp;author=T.+Luchko&amp;author=R.+B.+Luo&amp;author=X.+Wu&amp;author=P.+A.+Kollaman&amp;title=AMBER+14" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26btitle%3DAMBER%252014%26pub%3DUniversity%2520of%2520California%26date%3D2014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brandl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jabs, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">SÃ¼hnel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">C-HÂ·Â·Â·Ï-Interactions in Proteins</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>307</i></span>,  <span class="NLM_fpage">357</span>â <span class="NLM_lpage">377</span>, <span class="refDoi">&nbsp;DOI: 10.1006/jmbi.2000.4473</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1006%2Fjmbi.2000.4473" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=11243825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhslSnsb4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=307&amp;publication_year=2001&amp;pages=357-377&amp;author=M.+Brandlauthor=M.+S.+Weissauthor=A.+Jabsauthor=J.+S%C3%BChnelauthor=R.+Hilgenfeld&amp;title=C-H%C2%B7%C2%B7%C2%B7%CF%80-Interactions+in+Proteins&amp;doi=10.1006%2Fjmbi.2000.4473" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">C-HÂ·Â·Â·Ï-Interactions in Proteins</span></div><div class="casAuthors">Brandl, Maria; Weiss, Manfred S.; Jabs, Andreas; Suhnel, Jurgen; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">307</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">357-377</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A non-redundant set of 1154 protein structures from the Protein Data Bank was examd. with respect to close interactions between C-H-donor and Ï-acceptor groups.  A total of 31,087 interactions were found to satisfy our selection criteria.  Their geometric parameters suggest that these interactions can be classified as weak hydrogen bonds.  A set of 12 interaction classes were defined based on the division of the donors into three groups and the acceptors into four groups.  These classes were examd. sep., and the resp. interactions described in detail in each class.  Most prominent were interactions between aliph. C-H donors and arom. Ï-acceptors and interactions between arom. C-H donors and arom. Ï-acceptors.  About three-quarters of the Trp-rings, half of all Phe and Tyr-rings and a quarter of all His-rings were found to be involved as acceptors in C-HÂ·Â·Â·Ï-interactions.  On the donor side, a preference for arom. C-H groups was obsd., but also for the aliph. side-chains of the long, extended amino acid residues Lys, Arg and Met, and the Pro ring.  The av. distance between the C-donor and the center-of-mass of the Ï-acceptor was obsd. to be significantly longer in the 174 protein structures detd. at &gt;2.5 Ã resoln.  Also, the distribution is significantly wider.  This resoln. dependence suggests that the force fields commonly used for the refinement of protein structures may not be adequate.  C-HÂ·Â·Â·Ï-interactions involving arom. groups either as donor or as acceptor groups are found mostly in the interior of the protein.  The more hydrophilic the participating groups are, the closer to the surface are the interactions located.  About 40 % of all C-HÂ·Â·Â·Ï-interactions occur between amino acid residue side-chains that are sepd. by nine or less residues in sequence.  Dependent on the interaction class, different preferences for secondary structure, residue type and side-chain conformation were obsd.  It is likely that the C-HÂ·Â·Â·Ï-interactions contribute significantly to the overall stability of a protein.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFu6opUqoQ67Vg90H21EOLACvtfcHk0lhxxH6qqFKzXg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3MXhslSnsb4%253D&amp;md5=0aaa18f920eaafb06283b8a88db5c072</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2000.4473&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2000.4473%26sid%3Dliteratum%253Aachs%26aulast%3DBrandl%26aufirst%3DM.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DJabs%26aufirst%3DA.%26aulast%3DS%25C3%25BChnel%26aufirst%3DJ.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DC-H%25C2%25B7%25C2%25B7%25C2%25B7%25CF%2580-Interactions%2520in%2520Proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2001%26volume%3D307%26spage%3D357%26epage%3D377%26doi%3D10.1006%2Fjmbi.2000.4473" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kottur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, D. T.</span></span> <span> </span><span class="NLM_article-title">A Strategically Located Serine Residue Is Critical for the Mutator Activity of DNA Polymerase IV from Escherichia Coli</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">5104</span>â <span class="NLM_lpage">5114</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gkt146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1093%2Fnar%2Fgkt146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=23525461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVyjsr0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=41&amp;publication_year=2013&amp;pages=5104-5114&amp;author=A.+Sharmaauthor=J.+Kotturauthor=N.+Narayananauthor=D.+T.+Nair&amp;title=A+Strategically+Located+Serine+Residue+Is+Critical+for+the+Mutator+Activity+of+DNA+Polymerase+IV+from+Escherichia+Coli&amp;doi=10.1093%2Fnar%2Fgkt146" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A strategically located serine residue is critical for the mutator activity of DNA polymerase IV from Escherichia coli</span></div><div class="casAuthors">Sharma, Amit; Kottur, Jithesh; Narayanan, Naveen; Nair, Deepak T.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">5104-5114</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Y-family DNA polymerase IV or Pol IV (Escherichia coli) is the founding member of the DinB family and is known to play an important role in stress-induced mutagenesis.  We have detd. four crystal structures of this enzyme in its pre-catalytic state in complex with substrate DNA presenting the four possible template nucleotides that are paired with the corresponding incoming nucleotide triphosphates.  In all four structures, the Ser42 residue in the active site forms interactions with the base moieties of the incipient Watson-Crick base pair.  This residue is located close to the center of the nascent base pair towards the minor groove.  In vitro and in vivo assays show that the fidelity of the Pol IV enzyme increases drastically when this Ser residue was mutated to Ala.  In addn., the structure of Pol IV with the mismatch A:C in the active site shows that the Ser42 residue plays an important role in stabilizing dCTP in a conformation compatible with catalysis.  Overall, the structural, biochem. and functional data presented here show that the Ser42 residue is present at a strategic location to stabilize mismatches in the Pol IV active site, and thus facilitate the appearance of transition and transversion mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof6J51geZVq7Vg90H21EOLACvtfcHk0lhxxH6qqFKzXg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXntVyjsr0%253D&amp;md5=5790994ef6713be7e379ad73899c7d5f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt146&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt146%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DKottur%26aufirst%3DJ.%26aulast%3DNarayanan%26aufirst%3DN.%26aulast%3DNair%26aufirst%3DD.%2BT.%26atitle%3DA%2520Strategically%2520Located%2520Serine%2520Residue%2520Is%2520Critical%2520for%2520the%2520Mutator%2520Activity%2520of%2520DNA%2520Polymerase%2520IV%2520from%2520Escherichia%2520Coli%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D5104%26epage%3D5114%26doi%3D10.1093%2Fnar%2Fgkt146" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanquelef, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquant, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klimerak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delepine, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupradeau, F. Y.</span></span> <span> </span><span class="NLM_article-title">R.E.D. Server: A Web Service for Deriving RESP and ESP Charges and Building Force Field Libraries for New Molecules and Molecular Fragments</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">W511</span>â <span class="NLM_lpage">7</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gkr288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1093%2Fnar%2Fgkr288" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=21609950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosVOmsbY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=2011&amp;pages=W511-7&amp;author=E.+Vanquelefauthor=S.+Simonauthor=G.+Marquantauthor=E.+Garciaauthor=G.+Klimerakauthor=J.+C.+Delepineauthor=P.+Cieplakauthor=F.+Y.+Dupradeau&amp;title=R.E.D.+Server%3A+A+Web+Service+for+Deriving+RESP+and+ESP+Charges+and+Building+Force+Field+Libraries+for+New+Molecules+and+Molecular+Fragments&amp;doi=10.1093%2Fnar%2Fgkr288" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">R.E.D. Server: a web service for deriving RESP and ESP charges and building force field libraries for new molecules and molecular fragments</span></div><div class="casAuthors">Vanquelef, Enguerran; Simon, Sabrina; Marquant, Gaelle; Garcia, Elodie; Klimerak, Geoffroy; Delepine, Jean Charles; Cieplak, Piotr; Dupradeau, Francois-Yves</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W511-W517</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">R.E.D. Server is a unique, open web service, designed to derive non-polarizable RESP and ESP charges and to build force field libraries for new mols./mol. fragments.  It provides to computational biologists the means to derive rigorously mol. electrostatic potential-based charges embedded in force field libraries that are ready to be used in force field development, charge validation and mol. dynamics simulations.  R.E.D. Server interfaces quantum mechanics programs, the RESP program and the latest version of the R.E.D. tools.  A two step approach has been developed.  The first one consists of prepg. P2N file(s) to rigorously define key elements such as atom names, topol. and chem. equivalencing needed when building a force field library.  Then, P2N files are used to derive RESP or ESP charges embedded in force field libraries in the Tripos mol2 format.  In complex cases an entire set of force field libraries or force field topol. database is generated.  Other features developed in R.E.D.  Server include help services, a demonstration, tutorials, frequently asked questions, Jmol-based tools useful to construct PDB input files and parse R.E.D.  Server outputs as well as a graphical queuing system allowing any user to check the status of R.E.D. Server jobs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEOBBuQu4rcbVg90H21EOLACvtfcHk0lgUb3Q4Jh2lPg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXosVOmsbY%253D&amp;md5=1d224a2941d8634b6ed2159530d9e529</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr288&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr288%26sid%3Dliteratum%253Aachs%26aulast%3DVanquelef%26aufirst%3DE.%26aulast%3DSimon%26aufirst%3DS.%26aulast%3DMarquant%26aufirst%3DG.%26aulast%3DGarcia%26aufirst%3DE.%26aulast%3DKlimerak%26aufirst%3DG.%26aulast%3DDelepine%26aufirst%3DJ.%2BC.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DDupradeau%26aufirst%3DF.%2BY.%26atitle%3DR.E.D.%2520Server%253A%2520A%2520Web%2520Service%2520for%2520Deriving%2520RESP%2520and%2520ESP%2520Charges%2520and%2520Building%2520Force%2520Field%2520Libraries%2520for%2520New%2520Molecules%2520and%2520Molecular%2520Fragments%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DW511%26epage%3D7%26doi%3D10.1093%2Fnar%2Fgkr288" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dupradeau, F.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pigache, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaffran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savineau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grivel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosanski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span> <span> </span><span class="NLM_article-title">The R.E.D. Tools: Advances in RESP and ESP Charge Derivation and Force Field Library Building</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">7821</span>, <span class="refDoi">&nbsp;DOI: 10.1039/c0cp00111b</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1039%2Fc0cp00111b" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=20574571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFaqtrk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2010&amp;pages=7821&amp;author=F.-Y.+Dupradeauauthor=A.+Pigacheauthor=T.+Zaffranauthor=C.+Savineauauthor=R.+Lelongauthor=N.+Grivelauthor=D.+Lelongauthor=W.+Rosanskiauthor=P.+Cieplak&amp;title=The+R.E.D.+Tools%3A+Advances+in+RESP+and+ESP+Charge+Derivation+and+Force+Field+Library+Building&amp;doi=10.1039%2Fc0cp00111b" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The R.E.D. tools: advances in RESP and ESP charge derivation and force field library building</span></div><div class="casAuthors">Dupradeau, Francois-Yves; Pigache, Adrien; Zaffran, Thomas; Savineau, Corentin; Lelong, Rodolphe; Grivel, Nicolas; Lelong, Dimitri; Rosanski, Wilfried; Cieplak, Piotr</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">7821-7839</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Deriving at. charges and building a force field library for a new mol. are key steps when developing a force field required for conducting structural and energy-based anal. using mol. mechanics.  Derivation of popular RESP charges for a set of residues is a complex and error prone procedure because it depends on numerous input parameters.  To overcome these problems, the R.E.D.  Tools (RESP and ESP charge Derive, http://q4md-forcefieldtools.org/RED/) have been developed to perform charge derivation in an automatic and straightforward way.  The R.E.D. program handles chem. elements up to bromine in the periodic table.  It interfaces different quantum mech. programs employed for geometry optimization and computing mol. electrostatic potential(s), and performs charge fitting using the RESP program.  By defining tight optimization criteria and by controlling the mol. orientation of each optimized geometry, charge values are reproduced at any computer platform with an accuracy of 0.0001 e.  The charges can be fitted using multiple conformations, making them suitable for mol. dynamics simulations.  R.E.D. allows also for defining charge constraints during multiple mol. charge fitting, which are used to derive charges for mol. fragments.  Finally, R.E.D. incorporates charges into a force field library, readily usable in mol. dynamics computer packages.  For complex cases, such as a set of homologous mols. belonging to a common family, an entire force field topol. database is generated.  Currently, the at. charges and force field libraries have been developed for more than fifty model systems and stored in the RESP ESP charge DDataBase.  Selected results related to non-polarizable charge models are presented and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxiXPn3YLNM7Vg90H21EOLACvtfcHk0ljWY51t6tYVJw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXosFaqtrk%253D&amp;md5=845cc10ea7ed2f0222361db82bd80e39</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1039%2Fc0cp00111b&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cp00111b%26sid%3Dliteratum%253Aachs%26aulast%3DDupradeau%26aufirst%3DF.-Y.%26aulast%3DPigache%26aufirst%3DA.%26aulast%3DZaffran%26aufirst%3DT.%26aulast%3DSavineau%26aufirst%3DC.%26aulast%3DLelong%26aufirst%3DR.%26aulast%3DGrivel%26aufirst%3DN.%26aulast%3DLelong%26aufirst%3DD.%26aulast%3DRosanski%26aufirst%3DW.%26aulast%3DCieplak%26aufirst%3DP.%26atitle%3DThe%2520R.E.D.%2520Tools%253A%2520Advances%2520in%2520RESP%2520and%2520ESP%2520Charge%2520Derivation%2520and%2520Force%2520Field%2520Library%2520Building%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2010%26volume%3D12%26spage%3D7821%26doi%3D10.1039%2Fc0cp00111b" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornell, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P.</span></span> <span> </span><span class="NLM_article-title">A Well-Behaved Electrostatic Potential Based Method Using Charge Restraints for Deriving Atomic</span>. <i>J. Phys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">10269</span>â <span class="NLM_lpage">10280</span>, <span class="refDoi">&nbsp;DOI: 10.1021/j100142a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/j100142a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=97&amp;publication_year=1993&amp;pages=10269-10280&amp;author=C.+Baylyauthor=P.+Cieplakauthor=W.+Cornellauthor=P.+Kollman&amp;title=A+Well-Behaved+Electrostatic+Potential+Based+Method+Using+Charge+Restraints+for+Deriving+Atomic&amp;doi=10.1021%2Fj100142a004" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fj100142a004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100142a004%26sid%3Dliteratum%253Aachs%26aulast%3DBayly%26aufirst%3DC.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DCornell%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%26atitle%3DA%2520Well-Behaved%2520Electrostatic%2520Potential%2520Based%2520Method%2520Using%2520Charge%2520Restraints%2520for%2520Deriving%2520Atomic%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1993%26volume%3D97%26spage%3D10269%26epage%3D10280%26doi%3D10.1021%2Fj100142a004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J.</span>; <span class="NLM_string-name">Trucks, G. W.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Petersson, G. A.</span>; <span class="NLM_string-name">Nakatsuji, H.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Caricato, M.</span>; <span class="NLM_string-name">Marenich, A.</span>; <span class="NLM_string-name">Bloino, J.</span>; <span class="NLM_string-name">Janesko, B. G.</span>; <span class="NLM_string-name">Gomperts, R.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Hratchian, H. P.</span>; <span class="NLM_string-name">Ortiz, J. V.</span>; <span class="NLM_string-name">Izmaylov, A. F.</span>; <span class="NLM_string-name">Sonnenberg, J. L.</span>; <span class="NLM_string-name">Williams-Young, D.</span>; <span class="NLM_string-name">Ding, F.</span>; <span class="NLM_string-name">Lipparini, F.</span>; <span class="NLM_string-name">Egidi, F.</span>; <span class="NLM_string-name">Goings, J.</span>; <span class="NLM_string-name">Peng, B.</span>; <span class="NLM_string-name">Petrone, A.</span>; <span class="NLM_string-name">Henderson, T.</span>; <span class="NLM_string-name">Ranasinghe, D.</span>; <span class="NLM_string-name">Zakrzewski, V. G.</span>; <span class="NLM_string-name">Gao, J.</span>; <span class="NLM_string-name">Rega, N.</span>; <span class="NLM_string-name">Zheng, G.</span>; <span class="NLM_string-name">Liang, W.</span>; <span class="NLM_string-name">Hada, M.</span>; <span class="NLM_string-name">Ehara, M.</span>; <span class="NLM_string-name">Toyota, K.</span>; <span class="NLM_string-name">Fukuda, R.</span>; <span class="NLM_string-name">Hasegawa, J.</span>; <span class="NLM_string-name">Ishida, M.</span>; <span class="NLM_string-name">Nakajima, T.</span>; <span class="NLM_string-name">Honda, Y.</span>; <span class="NLM_string-name">Kitao, O.</span>; <span class="NLM_string-name">Nakai, H.</span>; <span class="NLM_string-name">Vreven, T.</span>; <span class="NLM_string-name">Throssell, K.</span>; <span class="NLM_string-name">Montgomery, J. A.,  Jr.</span>; <span class="NLM_string-name">Peralta, P. E.</span>; <span class="NLM_string-name">Ogliaro, F.</span>; <span class="NLM_string-name">Bearpark, M.</span>; <span class="NLM_string-name">Heyd, J.
J.</span>; <span class="NLM_string-name">Brothers, E.</span>; <span class="NLM_string-name">Kudin, K.
N.</span>; <span class="NLM_string-name">Staroverov, V. N.</span>; <span class="NLM_string-name">Keith, T.</span>; <span class="NLM_string-name">Kobayashi, R.</span>; <span class="NLM_string-name">Normand, J.</span>; <span class="NLM_string-name">Raghavachari, K.</span>; <span class="NLM_string-name">Rendell, A.</span>; <span class="NLM_string-name">Burant, J. C.</span>; <span class="NLM_string-name">Iyengar, S. S.</span>; <span class="NLM_string-name">Tomasi, J.</span>; <span class="NLM_string-name">Cossi, M.</span>; <span class="NLM_string-name">Millam, J. M.</span>; <span class="NLM_string-name">Klene, M.</span>; <span class="NLM_string-name">Adamo, C.</span>; <span class="NLM_string-name">Cammi, R.</span>; <span class="NLM_string-name">Ochterski, J. W.</span>; <span class="NLM_string-name">Martin, R. L.</span>; <span class="NLM_string-name">Morokuma, K.</span>; <span class="NLM_string-name">Farkas, Ã.</span>; <span class="NLM_string-name">Foresman, J. B.</span>; <span class="NLM_string-name">Fox, D. J.</span></span> <i>Gaussian 09</i>, Revision B.01; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford, CT</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2013&amp;author=M.+J.+Frisch&amp;author=G.+W.+Trucks&amp;author=H.+B.+Schlegel&amp;author=G.+E.+Scuseria&amp;author=M.+A.+Robb&amp;author=J.+R.+Cheeseman&amp;author=G.+Scalmani&amp;author=V.+Barone&amp;author=G.+A.+Petersson&amp;author=H.+Nakatsuji&amp;author=X.+Li&amp;author=M.+Caricato&amp;author=A.+Marenich&amp;author=J.+Bloino&amp;author=B.+G.+Janesko&amp;author=R.+Gomperts&amp;author=B.+Mennucci&amp;author=H.+P.+Hratchian&amp;author=J.+V.+Ortiz&amp;author=A.+F.+Izmaylov&amp;author=J.+L.+Sonnenberg&amp;author=D.+Williams-Young&amp;author=F.+Ding&amp;author=F.+Lipparini&amp;author=F.+Egidi&amp;author=J.+Goings&amp;author=B.+Peng&amp;author=A.+Petrone&amp;author=T.+Henderson&amp;author=D.+Ranasinghe&amp;author=V.+G.+Zakrzewski&amp;author=J.+Gao&amp;author=N.+Rega&amp;author=G.+Zheng&amp;author=W.+Liang&amp;author=M.+Hada&amp;author=M.+Ehara&amp;author=K.+Toyota&amp;author=R.+Fukuda&amp;author=J.+Hasegawa&amp;author=M.+Ishida&amp;author=T.+Nakajima&amp;author=Y.+Honda&amp;author=O.+Kitao&amp;author=H.+Nakai&amp;author=T.+Vreven&amp;author=K.+Throssell&amp;author=J.+A.+Montgomery&amp;author=P.+E.+Peralta&amp;author=F.+Ogliaro&amp;author=M.+Bearpark&amp;author=J.%0AJ.+Heyd&amp;author=E.+Brothers&amp;author=K.%0AN.+Kudin&amp;author=V.+N.+Staroverov&amp;author=T.+Keith&amp;author=R.+Kobayashi&amp;author=J.+Normand&amp;author=K.+Raghavachari&amp;author=A.+Rendell&amp;author=J.+C.+Burant&amp;author=S.+S.+Iyengar&amp;author=J.+Tomasi&amp;author=M.+Cossi&amp;author=J.+M.+Millam&amp;author=M.+Klene&amp;author=C.+Adamo&amp;author=R.+Cammi&amp;author=J.+W.+Ochterski&amp;author=R.+L.+Martin&amp;author=K.+Morokuma&amp;author=%C3%96.+Farkas&amp;author=J.+B.+Foresman&amp;author=D.+J.+Fox&amp;title=Gaussian+09" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26btitle%3DGaussian%252009%26pub%3DGaussian%252C%2520Inc%26date%3D2013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hornak, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strockbine, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roitberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span> <span> </span><span class="NLM_article-title">Comparison of Multiple Amber Force Fields and Development of Improved Protein Backbone Parameters</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">712</span>â <span class="NLM_lpage">725</span>, <span class="refDoi">&nbsp;DOI: 10.1002/prot.21123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1002%2Fprot.21123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=16981200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=65&amp;publication_year=2006&amp;pages=712-725&amp;author=V.+Hornakauthor=R.+Abelauthor=A.+Okurauthor=B.+Strockbineauthor=A.+Roitbergauthor=C.+Simmerling&amp;title=Comparison+of+Multiple+Amber+Force+Fields+and+Development+of+Improved+Protein+Backbone+Parameters&amp;doi=10.1002%2Fprot.21123" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span></div><div class="casAuthors">Hornak, Viktor; Abel, Robert; Okur, Asim; Strockbine, Bentley; Roitberg, Adrian; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">712-725</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The ff94 force field that is commonly assocd. with the Amber simulation package is one of the most widely used parameter sets for biomol. simulation.  After a decade of extensive use and testing, limitations in this force field, such as over-stabilization of Î±-helixes, were reported by the authors and other researchers.  This led to a no. of attempts to improve these parameters, resulting in a variety of "Amber" force fields and significant difficulty in detg. which should be used for a particular application.  The authors show that several of these continue to suffer from inadequate balance between different secondary structure elements.  In addn., the approach used in most of these studies neglected to account for the existence in Amber of two sets of backbone .vphi./Ï dihedral terms.  This led to parameter sets that provide unreasonable conformational preferences for glycine.  The authors report here an effort to improve the .vphi./Ï dihedral terms in the ff99 energy function.  Dihedral term parameters are based on fitting the energies of multiple conformations of glycine and alanine tetrapeptides from high level ab initio quantum mech. calcns.  The new parameters for backbone dihedrals replace those in the existing ff99 force field.  This parameter set, which the authors denote ff99SB, achieves a better balance of secondary structure elements as judged by improved distribution of backbone dihedrals for glycine and alanine with respect to PDB survey data.  It also accomplishes improved agreement with published exptl. data for conformational preferences of short alanine peptides and better accord with exptl. NMR relaxation data of test protein systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmoASzza-Gd7Vg90H21EOLACvtfcHk0ljI16twnHzTwQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM&amp;md5=de683a26eca9e83ae524726e97ac22fa</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fprot.21123&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21123%26sid%3Dliteratum%253Aachs%26aulast%3DHornak%26aufirst%3DV.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DOkur%26aufirst%3DA.%26aulast%3DStrockbine%26aufirst%3DB.%26aulast%3DRoitberg%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3DComparison%2520of%2520Multiple%2520Amber%2520Force%2520Fields%2520and%2520Development%2520of%2520Improved%2520Protein%2520Backbone%2520Parameters%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2006%26volume%3D65%26spage%3D712%26epage%3D725%26doi%3D10.1002%2Fprot.21123" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">PÃ©rez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MarchÃ¡n, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svozil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sponer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laughton, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orozco, M.</span></span> <span> </span><span class="NLM_article-title">Refinement of the AMBER Force Field for Nucleic Acids: Improving the Description of Î±/Î³ Conformers</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">3817</span>â <span class="NLM_lpage">3829</span>, <span class="refDoi">&nbsp;DOI: 10.1529/biophysj.106.097782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1529%2Fbiophysj.106.097782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=17351000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXls1ygu7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=92&amp;publication_year=2007&amp;pages=3817-3829&amp;author=A.+P%C3%A9rezauthor=I.+March%C3%A1nauthor=D.+Svozilauthor=J.+Sponerauthor=T.+E.+Cheathamauthor=C.+A.+Laughtonauthor=M.+Orozco&amp;title=Refinement+of+the+AMBER+Force+Field+for+Nucleic+Acids%3A+Improving+the+Description+of+%CE%B1%2F%CE%B3+Conformers&amp;doi=10.1529%2Fbiophysj.106.097782" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of the AMBER force field for nucleic acids: improving the description of Î±/Î³ conformers</span></div><div class="casAuthors">Perez, Alberto; Marchan, Ivan; Svozil, Daniel; Sponer, Jiri; Cheatham, Thomas E., III; Laughton, Charles A.; Orozco, Modesto</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3817-3829</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">We present here the parmbsc0 force field, a refinement of the AMBER parm99 force field, where emphasis has been made on the correct representation of the Î±/Î³ concerted rotation in nucleic acids (NAs).  The modified force field corrects overpopulations of the Î±/Î³ = (g+,t) backbone that were seen in long (more than 10 ns) simulations with previous AMBER parameter sets (parm94-99).  The force field has been derived by fitting to high-level quantum mech. data and verified by comparison with very high-level quantum mech. calcns. and by a very extensive comparison between simulations and exptl. data.  The set of validation simulations includes two of the longest trajectories published to date for the DNA duplex (200 ns each) and the largest variety of NA structures studied to date (15 different NA families and 97 individual structures).  The total simulation time used to validate the force field includes near 1 Î¼s of state-of-the-art mol. dynamics simulations in aq. soln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriZsH_9T_yB7Vg90H21EOLACvtfcHk0ljI16twnHzTwQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXls1ygu7Y%253D&amp;md5=e0f8d87e5fe7710e59cc41f5e8c25f7b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.097782&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.097782%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez%26aufirst%3DA.%26aulast%3DMarch%25C3%25A1n%26aufirst%3DI.%26aulast%3DSvozil%26aufirst%3DD.%26aulast%3DSponer%26aufirst%3DJ.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DLaughton%26aufirst%3DC.%2BA.%26aulast%3DOrozco%26aufirst%3DM.%26atitle%3DRefinement%2520of%2520the%2520AMBER%2520Force%2520Field%2520for%2520Nucleic%2520Acids%253A%2520Improving%2520the%2520Description%2520of%2520%25CE%25B1%252F%25CE%25B3%2520Conformers%26jtitle%3DBiophys.%2520J.%26date%3D2007%26volume%3D92%26spage%3D3817%26epage%3D3829%26doi%3D10.1529%2Fbiophysj.106.097782" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salomon-Ferrer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GÃ¶tz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 2. Explicit Solvent Particle Mesh Ewald</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3878</span>â <span class="NLM_lpage">3888</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct400314y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400314y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2013&amp;pages=3878-3888&amp;author=R.+Salomon-Ferrerauthor=A.+W.+G%C3%B6tzauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&amp;title=Routine+Microsecond+Molecular+Dynamics+Simulations+with+AMBER+on+GPUs.+2.+Explicit+Solvent+Particle+Mesh+Ewald&amp;doi=10.1021%2Fct400314y" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 2. Explicit Solvent Particle Mesh Ewald</span></div><div class="casAuthors">Salomon-Ferrer, Romelia; Gotz, Andreas W.; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3878-3888</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of explicit solvent all atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA-enabled GPUs.  First released publicly in Apr. 2010 as part of version 11 of the AMBER MD package and further improved and optimized over the last two years, this implementation supports the three most widely used statistical mech. ensembles (NVE, NVT, and NPT), uses particle mesh Ewald (PME) for the long-range electrostatics, and runs entirely on CUDA-enabled NVIDIA graphics processing units (GPUs), providing results that are statistically indistinguishable from the traditional CPU version of the software and with performance that exceeds that achievable by the CPU version of AMBER software running on all conventional CPU-based clusters and supercomputers.  We briefly discuss three different precision models developed specifically for this work (SPDP, SPFP, and DPDP) and highlight the tech. details of the approach as it extends beyond previously reported work [Goetz et al., J.  Chem.  Theory Comput.2012, DOI: 10.1021/ct200909j; Le Grand et al., Comp.  Phys.  Comm.2013, DOI: 10.1016/j.cpc.2012.09.022].We highlight the substantial improvements in performance that are seen over traditional CPU-only machines and provide validation of our implementation and precision models.  We also provide evidence supporting our decision to deprecate the previously described fully single precision (SPSP) model from the latest release of the AMBER software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY-M2E_jgUmLVg90H21EOLACvtfcHk0lgSLPSBZy4Zeg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP&amp;md5=57e18d35e8b81e9a79788ef5af17d19f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fct400314y&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400314y%26sid%3Dliteratum%253Aachs%26aulast%3DSalomon-Ferrer%26aufirst%3DR.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520Microsecond%2520Molecular%2520Dynamics%2520Simulations%2520with%2520AMBER%2520on%2520GPUs.%25202.%2520Explicit%2520Solvent%2520Particle%2520Mesh%2520Ewald%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3878%26epage%3D3888%26doi%3D10.1021%2Fct400314y" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">GÃ¶tz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 1. Generalized Born</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1542</span>â <span class="NLM_lpage">1555</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct200909j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200909j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC38XksFWns78%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2012&amp;pages=1542-1555&amp;author=A.+W.+G%C3%B6tzauthor=M.+J.+Williamsonauthor=D.+Xuauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&amp;title=Routine+Microsecond+Molecular+Dynamics+Simulations+with+AMBER+on+GPUs.+1.+Generalized+Born&amp;doi=10.1021%2Fct200909j" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 1. Generalized Born</span></div><div class="casAuthors">Gotz, Andreas W.; Williamson, Mark J.; Xu, Dong; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1542-1555</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of generalized Born implicit solvent all-atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA enabled NVIDIA graphics processing units (GPUs).  We discuss the algorithms that are used to exploit the processing power of the GPUs and show the performance that can be achieved in comparison to simulations on conventional CPU clusters.  The implementation supports three different precision models in which the contributions to the forces are calcd. in single precision floating point arithmetic but accumulated in double precision (SPDP), or everything is computed in single precision (SPSP) or double precision (DPDP).  In addn. to performance, we have focused on understanding the implications of the different precision models on the outcome of implicit solvent MD simulations.  We show results for a range of tests including the accuracy of single point force evaluations and energy conservation as well as structural properties pertaining to protein dynamics.  The numerical noise due to rounding errors within the SPSP precision model is sufficiently large to lead to an accumulation of errors which can result in unphys. trajectories for long time scale simulations.  We recommend the use of the mixed-precision SPDP model since the numerical results obtained are comparable with those of the full double precision DPDP model and the ref. double precision CPU implementation but at significantly reduced computational cost.  Our implementation provides performance for GB simulations on a single desktop that is on par with, and in some cases exceeds, that of traditional supercomputers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpshIijeXLcqLVg90H21EOLACvtfcHk0lgSLPSBZy4Zeg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XksFWns78%253D&amp;md5=1e6f570db9cd504bb13706e7c56bc356</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fct200909j&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200909j%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWilliamson%26aufirst%3DM.%2BJ.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520Microsecond%2520Molecular%2520Dynamics%2520Simulations%2520with%2520AMBER%2520on%2520GPUs.%25201.%2520Generalized%2520Born%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D1542%26epage%3D1555%26doi%3D10.1021%2Fct200909j" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le
Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GÃ¶tz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">SPFP: Speed without CompromiseâA Mixed Precision Model for GPU Accelerated Molecular Dynamics Simulations</span>. <i>Comput. Phys. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>184</i></span>,  <span class="NLM_fpage">374</span>â <span class="NLM_lpage">380</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cpc.2012.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1016%2Fj.cpc.2012.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=184&amp;publication_year=2013&amp;pages=374-380&amp;author=S.+Le%0AGrandauthor=A.+W.+G%C3%B6tzauthor=R.+C.+Walker&amp;title=SPFP%3A+Speed+without+Compromise%E2%80%94A+Mixed+Precision+Model+for+GPU+Accelerated+Molecular+Dynamics+Simulations&amp;doi=10.1016%2Fj.cpc.2012.09.022" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">SPFP: Speed without compromise-A mixed precision model for GPU accelerated molecular dynamics simulations</span></div><div class="casAuthors">Le Grand, Scott; Gotz, Andreas W.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">374-380</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A new precision model is proposed for the acceleration of all-atom classical mol. dynamics (MD) simulations on graphics processing units (GPUs).  This precision model replaces double precision arithmetic with fixed point integer arithmetic for the accumulation of force components as compared to a previously introduced model that uses mixed single/double precision arithmetic.  This significantly boosts performance on modern GPU hardware without sacrificing numerical accuracy.  We present an implementation for NVIDIA GPUs of both generalized Born implicit solvent simulations as well as explicit solvent simulations using the particle mesh Ewald (PME) algorithm for long-range electrostatics using this precision model.  Tests demonstrate both the performance of this implementation as well as its numerical stability for const. energy and const. temp. biomol. MD as compared to a double precision CPU implementation and double and mixed single/double precision GPU implementations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJB_spY7SrSbVg90H21EOLACvtfcHk0lj8RS1TsOYUJw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO&amp;md5=2d9f102cb17f6e9bcbff3b5057d8c587</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cpc.2012.09.022&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cpc.2012.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DSPFP%253A%2520Speed%2520without%2520Compromise%25E2%2580%2594A%2520Mixed%2520Precision%2520Model%2520for%2520GPU%2520Accelerated%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D2013%26volume%3D184%26spage%3D374%26epage%3D380%26doi%3D10.1016%2Fj.cpc.2012.09.022" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera - A Visualization System for Exploratory Research and Analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>â <span class="NLM_lpage">1612</span>, <span class="refDoi">&nbsp;DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=25&amp;publication_year=2004&amp;pages=1605-1612&amp;author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&amp;title=UCSF+Chimera+-+A+Visualization+System+for+Exploratory+Research+and+Analysis&amp;doi=10.1002%2Fjcc.20084" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley &amp; Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lj8RS1TsOYUJw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&amp;md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%2520-%2520A%2520Visualization%2520System%2520for%2520Exploratory%2520Research%2520and%2520Analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Calculating Structures and Free Energies of Complex Molecules: Combining Molecular Mechanics and Continuum Models</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">889</span>â <span class="NLM_lpage">897</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar000033j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar000033j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmvFGiu7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=33&amp;publication_year=2000&amp;pages=889-897&amp;author=P.%0AA.+Kollmanauthor=I.+Massovaauthor=C.+Reyesauthor=B.+Kuhnauthor=S.+Huoauthor=L.+Chongauthor=M.+Leeauthor=T.+Leeauthor=Y.+Duanauthor=W.+Wang&amp;title=Calculating+Structures+and+Free+Energies+of+Complex+Molecules%3A+Combining+Molecular+Mechanics+and+Continuum+Models&amp;doi=10.1021%2Far000033j" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Calculating Structures and Free Energies of Complex Molecules: Combining Molecular Mechanics and Continuum Models</span></div><div class="casAuthors">Kollman, Peter A.; Massova, Irina; Reyes, Carolina; Kuhn, Bernd; Huo, Shuanghong; Chong, Lillian; Lee, Matthew; Lee, Taisung; Duan, Yong; Wang, Wei; Donini, Oreola; Cieplak, Piotr; Srinivasan, Jaysharee; Case, David A.; Cheatham, Thomas E., III</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">889-897</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, with 63 refs.  A historical perspective on the application of mol. dynamics (MD) to biol. macromols. is presented.  Recent developments combining state-of-the-art force fields with continuum solvation calcns. have allowed us to reach the fourth era of MD applications in which one can often derive both accurate structure and accurate relative free energies from mol. dynamics trajectories.  We illustrate such applications on nucleic acid duplexes, RNA hairpins, protein folding trajectories, and protein-ligand, protein-protein, and protein-nucleic acid interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXGhwlfA3kvLVg90H21EOLACvtfcHk0ljf0pAPDmHC2Q"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3cXmvFGiu7g%253D&amp;md5=8436ee610ae145894428db1a1deff73c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Far000033j&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far000033j%26sid%3Dliteratum%253Aachs%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DMassova%26aufirst%3DI.%26aulast%3DReyes%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DHuo%26aufirst%3DS.%26aulast%3DChong%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DCalculating%2520Structures%2520and%2520Free%2520Energies%2520of%2520Complex%2520Molecules%253A%2520Combining%2520Molecular%2520Mechanics%2520and%2520Continuum%2520Models%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2000%26volume%3D33%26spage%3D889%26epage%3D897%26doi%3D10.1021%2Far000033j" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGee, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swails, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homeyer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roitberg, A. E.</span></span> <span> </span><span class="NLM_article-title">MMPBSA.Py: An Efficient Program for End-State Free Energy Calculations</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3314</span>â <span class="NLM_lpage">3321</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct300418h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300418h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2gtrzP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2012&amp;pages=3314-3321&amp;author=B.+R.+Millerauthor=T.+D.+McGeeauthor=J.+M.+Swailsauthor=N.+Homeyerauthor=H.+Gohlkeauthor=A.+E.+Roitberg&amp;title=MMPBSA.Py%3A+An+Efficient+Program+for+End-State+Free+Energy+Calculations&amp;doi=10.1021%2Fct300418h" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">MMPBSA.py: An Efficient Program for End-State Free Energy Calculations</span></div><div class="casAuthors">Miller, Bill R., III; McGee, T. Dwight, Jr.; Swails, Jason M.; Homeyer, Nadine; Gohlke, Holger; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3314-3321</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MM-PBSA is a post-processing end-state method to calc. free energies of mols. in soln.  MMPBSA.py is a program written in Python for streamlining end-state free energy calcns. using ensembles derived from mol. dynamics (MD) or Monte Carlo (MC) simulations.  Several implicit solvation models are available with MMPBSA.py, including the Poisson-Boltzmann Model, the Generalized Born Model, and the Ref. Interaction Site Model.  Vibrational frequencies may be calcd. using normal mode or quasi-harmonic anal. to approx. the solute entropy.  Specific interactions can also be dissected using free energy decompn. or alanine scanning.  A parallel implementation significantly speeds up the calcn. by dividing frames evenly across available processors.  MMPBSA.py is an efficient, user-friendly program with the flexibility to accommodate the needs of users performing end-state free energy calcns.  The source code can be downloaded at http://ambermd.org/ with AmberTools, released under the GNU General Public License.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZK3C1jqiSULVg90H21EOLACvtfcHk0ljf0pAPDmHC2Q"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhtV2gtrzP&amp;md5=cc4148bd8f70c7cad94fd3ec6f580e52</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fct300418h&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300418h%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DB.%2BR.%26aulast%3DMcGee%26aufirst%3DT.%2BD.%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DHomeyer%26aufirst%3DN.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DMMPBSA.Py%253A%2520An%2520Efficient%2520Program%2520for%2520End-State%2520Free%2520Energy%2520Calculations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3314%26epage%3D3321%26doi%3D10.1021%2Fct300418h" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></div><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Elisabeth Mairhofer, Laurin Flemmich, Christoph Kreutz, <span class="NLM_string-name hlFld-ContribAuthor">Ronald Micura</span>. </span><span class="cited-content_cbyCitation_article-title">Access to 3-Deazaguanosine Building Blocks for RNA Solid-Phase Synthesis Involving HartwigâBuchwald CâN Cross-Coupling. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2019,</strong> <em>21 </em>
                                    (11)
                                     , 3900-3903. <a href="https://doi.org/10.1021/acs.orglett.9b00855" title="DOI URL">https://doi.org/10.1021/acs.orglett.9b00855</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.9b00855&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.9b00855%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DAccess%252Bto%252B3-Deazaguanosine%252BBuilding%252BBlocks%252Bfor%252BRNA%252BSolid-Phase%252BSynthesis%252BInvolving%252BHartwig%2525E2%252580%252593Buchwald%252BC%2525E2%252580%252593N%252BCross-Coupling%26aulast%3DMairhofer%26aufirst%3DElisabeth%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D08032019%26date%3D13052019%26volume%3D21%26issue%3D11%26spage%3D3900%26epage%3D3903" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gandarvakottai Senthilkumar  Arumugam</span>, <span class="hlFld-ContribAuthor ">Asmita  Sen</span>, <span class="hlFld-ContribAuthor ">Swati S.  Dash</span>, <span class="hlFld-ContribAuthor ">Kartik  Mitra</span>, <span class="hlFld-ContribAuthor ">Mukesh  Doble</span>, <span class="hlFld-ContribAuthor ">Gopalan  Rajaraman</span>, <span class="hlFld-ContribAuthor ">Sathyanarayana N.  Gummadi</span>. </span><span class="cited-content_cbyCitation_article-title">Arjunetin as a promising drug candidate against SARS-CoV-2: molecular dynamics simulation studies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2021,</strong> <em>20 </em>, 1-22. <a href="https://doi.org/10.1080/07391102.2021.1970627" title="DOI URL">https://doi.org/10.1080/07391102.2021.1970627</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2021.1970627&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2021.1970627%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DArjunetin%252Bas%252Ba%252Bpromising%252Bdrug%252Bcandidate%252Bagainst%252BSARS-CoV-2%25253A%252Bmolecular%252Bdynamics%252Bsimulation%252Bstudies%26aulast%3DArumugam%26aufirst%3DGandarvakottai%2BSenthilkumar%26date%3D2021%26date%3D2021%26volume%3D20%26spage%3D1%26epage%3D22" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pratibha P.  Ghodke</span>, <span class="hlFld-ContribAuthor ">Pushpangadan I.  Pradeepkumar</span>. </span><span class="cited-content_cbyCitation_article-title">SiteâSpecific
              N
              2
              âdG DNA Adducts: Formation, Synthesis, and TLS PolymeraseâMediated Bypass. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>2020 </em>
                                    (44)
                                     , 6831-6844. <a href="https://doi.org/10.1002/ejoc.202000298" title="DOI URL">https://doi.org/10.1002/ejoc.202000298</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.202000298&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.202000298%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSite%2525E2%252580%252590Specific%252BN%252B2%252B%2525E2%252580%252590dG%252BDNA%252BAdducts%25253A%252BFormation%25252C%252BSynthesis%25252C%252Band%252BTLS%252BPolymerase%2525E2%252580%252590Mediated%252BBypass%26aulast%3DGhodke%26aufirst%3DPratibha%2BP.%26date%3D2020%26date%3D2020%26volume%3D2020%26issue%3D44%26spage%3D6831%26epage%3D6844" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pratibha P.  Ghodke</span>, <span class="hlFld-ContribAuthor ">P. I.  Pradeepkumar</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of
              N
              2
              âArylâ2â²âDeoxyguanosine Modified Phosphoramidites and Oligonucleotides. </span><span class="cited-content_cbyCitation_journal-name">Current Protocols in Nucleic Acid Chemistry</span><span> <strong>2019,</strong> <em>78 </em>
                                    (1)
                                     <a href="https://doi.org/10.1002/cpnc.93" title="DOI URL">https://doi.org/10.1002/cpnc.93</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cpnc.93&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcpnc.93%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Protocols%2520in%2520Nucleic%2520Acid%2520Chemistry%26atitle%3DSynthesis%252Bof%252BN%252B2%252B%2525E2%252580%252590Aryl%2525E2%252580%2525902%2525E2%252580%2525B2%2525E2%252580%252590Deoxyguanosine%252BModified%252BPhosphoramidites%252Band%252BOligonucleotides%26aulast%3DGhodke%26aufirst%3DPratibha%2BP.%26date%3D2019%26date%3D2019%26volume%3D78%26issue%3D1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ankita  Gupta</span>, <span class="hlFld-ContribAuthor ">Shailesh B  Lad</span>, <span class="hlFld-ContribAuthor ">Pratibha P  Ghodke</span>, <span class="hlFld-ContribAuthor ">P I  Pradeepkumar</span>, <span class="hlFld-ContribAuthor ">Kiran  Kondabagil</span>. </span><span class="cited-content_cbyCitation_article-title">Mimivirus encodes a multifunctional primase with DNA/RNA polymerase, terminal transferase and translesion synthesis activities. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2019,</strong> <em>47 </em>
                                    (13)
                                     , 6932-6945. <a href="https://doi.org/10.1093/nar/gkz236" title="DOI URL">https://doi.org/10.1093/nar/gkz236</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkz236&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkz236%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DMimivirus%252Bencodes%252Ba%252Bmultifunctional%252Bprimase%252Bwith%252BDNA%25252FRNA%252Bpolymerase%25252C%252Bterminal%252Btransferase%252Band%252Btranslesion%252Bsynthesis%252Bactivities%26aulast%3DGupta%26aufirst%3DAnkita%26date%3D2019%26date%3D2019%26volume%3D47%26issue%3D13%26spage%3D6932%26epage%3D6945" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li></ol></div></div></div></div><div class="ui-resizable-handle ui-resizable-e" style="z-index: 90;"></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class=""><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0011.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0001.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of <i>N</i><sup>2</sup>-dG adducts which originate from polycyclic aromatic hydrocarbons.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0008.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <i>N</i><sup>2</sup>-Bn, <i>N</i><sup>2</sup>-Naph, <i>N</i><sup>2</sup>-Anth, and <i>N</i><sup>2</sup>-Pyre-dG Modified Phosphoramidites <b>5a</b>â<b>d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"></p><p class="last"><span class="fn-label"><sup>a</sup></span>Reagent and conditions: (i) Pd(OAc)<sub>2</sub>, (<i>R</i>)-BINAP, respective amines, Cs<sub>2</sub>CO<sub>3</sub>, toluene, 85 Â°C; (ii) 33% MeNH<sub>2</sub> in EtOH, rt; (iii) DMT-Cl, pyridine, rt; (iv) CEP-Cl, DIPEA, DCM, rt.</p><p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0002.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. PAGE (20%, 7 M urea) of primer extension reactions with single and with the mixture of dNTPs using unmodified (dG) and modified (<i>N</i><sup>2</sup>-dG adducts) templates employing <i>E. coli</i>. Pol IV. (A) complete sequence of the primer, P (15-mer) and the template, T (50-mer); (B) reactions with dG template (X = dG; lanes: P, primer; C, dCTP; A, dATP; G, dGTP; T, dTTP; N, mixture of dNTPs); (C) reactions with <i>N</i><sup>2</sup>-Bn-dG template; (D) reactions with <i>N</i><sup>2</sup>-Naph-dG template; (E) reactions with <i>N</i><sup>2</sup>-Anth-dG template; and (F) reactions with <i>N</i><sup>2</sup>-Pyre-dG template. All reactions were carried out at 37 Â°C for 1.5 h time course.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0003.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. PAGE (20%, 7 M urea) of full-length extension reactions employing Pol IV from <i>E. coli</i> with all dNTPs. (A) Complete sequence of template, T and primer, P (15-mer); (B) reactions with dG template (X = dG); (C) reactions with <i>N</i><sup>2</sup>-Bn-dG template; (D) reactions with <i>N</i><sup>2</sup>-Naph-dG template; (E) reactions with <i>N</i><sup>2</sup>-Anth-dG template; (F) reactions with <i>N</i><sup>2</sup>-Pyre-dG template. Lane P, primer; lanes 1 to 6: primer extension reactions with 250 Î¼M of mixture of dNTPs in different time course from 30 s, 30 min, 1, 2, 4, and 8 h; lane C, 50-mer size standard. All of the reactions were carried out at 37 Â°C.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0004.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. DNA-Pol IV ternary complex used for MD simulations. The initial structure of <i>N</i><sup>2</sup>-Bn complex prepared from fdG containing crystal structure (PDB ID: <a href="http://doi.org/10.2210/pdb4Q43/pdb">4Q43</a>). The red colored rectangle is used to highlight the active site and zoomed view of the active site. Other complexes are shown in Figure S9, <a href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf" class="ext-link">Supporting Information</a>. Cyan, magenta, green, and yellow are used for polymerase associated domain (PAD) and finger, palm, and thumb domains, respectively. DNA backbone is represented in orange-red color. The <i>N</i><sup>2</sup>-modified residue is highlighted in desire red color. Carbon atoms are represented in white (desire red for <i>N</i><sup>2</sup>-Bn-dG), nitrogen atoms in blue, oxygen atoms in red, and magnesium ions in orange color.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0005.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Superposition of representative structures of ensembles 1 and 2 of <i>N</i><sup>2</sup>-dG-modified DNA-Pol IV ternary complexes. Only residues of interest are highlighted. (A) <i>N</i><sup>2</sup>-Bn complex; (B) <i>N</i><sup>2</sup>-Naph complex; (C) <i>N</i><sup>2</sup>-Anth complex;  and (D) <i>N</i><sup>2</sup>-Pyre complex. Lime green colored cartoon representation is used for the ensemble 1 and light pink colored cartoon representation is used for the ensemble 2. DNA backbone is represented in white for both the representative structures. Pale green colored carbon framework is used for <i>N</i><sup>2</sup>-adducts for ensemble 1 and pale pink colored carbon framework for ensemble 2. Nitrogen atoms are represented in blue and oxygen atoms in red. All of the hydrogens are removed for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0006.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Plot of dihedral Xi (Î¾, shown above using red arrow) vs time for all the complexes. Black, red, blue, and pink colors were used to represent <i>N</i><sup>2</sup>-Bn, -Naph, -Anth, and -Pyre complexes, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/cms/10.1021/acs.joc.8b02082/asset/images/medium/jo-2018-02082j_0007.gif" alt="" data-lg-src="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. van der Waals interactions between AAs of Pol IV and <i>N</i><sup>2</sup>-adducts found in DNA-Pol IV ternary complexes. (A) <i>N</i><sup>2</sup>-Bn complex; (B) <i>N</i><sup>2</sup>-Naph complex; (C) <i>N</i><sup>2</sup>-Anth complex;  and (D) <i>N</i><sup>2</sup>-Pyre complex. Phe76 interacts with the <i>N</i><sup>2</sup>-adduct through T-shaped CâH Â·Â·Â·Ï interactions in the Bn, Naph, Anth, and Pyre complexes. Carbon atom linked to OâH of Ser42 interacts with Ï cloud of Naph, Anth, and Pyre moieties. The backbone atom of Gly32 as well interacts with adducts through CâHÂ·Â·Â· Ï interactions in Naph and Pyre complexes. Phe13 as well forms T-shaped interactions with Anth moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.joc.8b02082/asset/images/large/jo-2018-02082j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.joc.8b02082&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i39">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21255" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21255" class="dropBlock__holder"></div></div></div></div></div><hr></div><p class="rightTab-references-count">
            This article references 53 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moorthy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlin, D. J.</span></span> <span> </span><span class="NLM_article-title">Polycyclic Aromatic Hydrocarbons: From Metabolism to Lung Cancer</span>. <i>Toxicol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>145</i></span>,  <span class="NLM_fpage">5</span>â <span class="NLM_lpage">15</span>, <span class="refDoi">&nbsp;DOI: 10.1093/toxsci/kfv040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1093%2Ftoxsci%2Fkfv040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=25911656" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFSrurzN" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=145&amp;publication_year=2015&amp;pages=5-15&amp;author=B.+Moorthyauthor=C.+Chuauthor=D.+J.+Carlin&amp;title=Polycyclic+Aromatic+Hydrocarbons%3A+From+Metabolism+to+Lung+Cancer&amp;doi=10.1093%2Ftoxsci%2Fkfv040" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Polycyclic aromatic hydrocarbons: from metabolism to lung cancer</span></div><div class="casAuthors">Moorthy, Bhagavatula; Chu, Chun; Carlin, Danielle J.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">145</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-15</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-0929</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Excessive exposure to polycyclic arom. hydrocarbons (PAHs) often results in lung cancer, a disease with the highest cancer mortality in the United States.  After entry into the lung, PAHs induce phase I metabolic enzymes such as cytochrome P 450 (CYP) monooxygenases, i.e.  CYP1A1/2 and 1B1, and phase II enzymes such as glutathione S-transferases, UDP glucuronyl transferases, NADPH quinone oxidoreductases (NQOs), aldo-keto reductases (AKRs), and epoxide hydrolases (EHs), via the aryl hydrocarbon receptor (AhR)-dependent and independent pathways.  Humans can also be exposed to PAHs through diet, via consumption of charcoal broiled foods.  Metab. of PAHs through the CYP1A1/1B1/EH pathway, CYP peroxidase pathway, and AKR pathway leads to the formation of the active carcinogens diol-epoxides, radical cations, and o-quinones.  These reactive metabolites produce DNA adducts, resulting in DNA mutations, alteration of gene expression profiles, and tumorigenesis.  Mutations in xenobiotic metabolic enzymes, as well as polymorphisms of tumor suppressor genes (e.g. p53) and/or genes involved in gene expression (e.g. X-ray repair cross-complementing proteins), are assocd. with lung cancer susceptibility in human populations from different ethnicities, gender, and age groups.  Although various metabolic activation/inactivation pathways, AhR signaling, and genetic susceptibilities contribute to lung cancer, the precise points at which PAHs induce tumor initiation remain unknown.  The goal of this review is to provide a current state-of-the-science of the mechanisms of human lung carcinogenesis mediated by PAHs, the exptl. approaches used to study this complex class of compds., and future directions for research of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoELfgYOijvfLVg90H21EOLACvtfcHk0lhGexoQ0cQQPg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXhsFSrurzN&amp;md5=b1b4dcbfa4b6c37239ddbd96224bb9fd</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfv040&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfv040%26sid%3Dliteratum%253Aachs%26aulast%3DMoorthy%26aufirst%3DB.%26aulast%3DChu%26aufirst%3DC.%26aulast%3DCarlin%26aufirst%3DD.%2BJ.%26atitle%3DPolycyclic%2520Aromatic%2520Hydrocarbons%253A%2520From%2520Metabolism%2520to%2520Lung%2520Cancer%26jtitle%3DToxicol.%2520Sci.%26date%3D2015%26volume%3D145%26spage%3D5%26epage%3D15%26doi%3D10.1093%2Ftoxsci%2Fkfv040" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dipple, A.</span></span> <span> </span><span class="NLM_article-title">DNA Adducts of Chemical Carcinogens</span>. <i>Carcinogenesis</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">437</span>â <span class="NLM_lpage">441</span>, <span class="refDoi">&nbsp;DOI: 10.1093/carcin/16.3.437</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1093%2Fcarcin%2F16.3.437" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=7697795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADyaK2MXkvVyqur0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=1995&amp;pages=437-441&amp;author=A.+Dipple&amp;title=DNA+Adducts+of+Chemical+Carcinogens&amp;doi=10.1093%2Fcarcin%2F16.3.437" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">DNA adducts of chemical carcinogens</span></div><div class="casAuthors">Dipple, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">437-41</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review and discussion with 25 refs. on metabolic activation, types of metabolic reactions (oxidn., redn., conjugation), arylamine-DNA adducts, alkylation, arylamination, aralkylation, and carcinogens on guanine residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpja7lYAMpt97Vg90H21EOLACvtfcHk0lhGexoQ0cQQPg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXkvVyqur0%253D&amp;md5=087d09c1c810d0d27daab8884db1c919</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2F16.3.437&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252F16.3.437%26sid%3Dliteratum%253Aachs%26aulast%3DDipple%26aufirst%3DA.%26atitle%3DDNA%2520Adducts%2520of%2520Chemical%2520Carcinogens%26jtitle%3DCarcinogenesis%26date%3D1995%26volume%3D16%26spage%3D437%26epage%3D441%26doi%3D10.1093%2Fcarcin%2F16.3.437" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hemminki, K.</span></span> <span> </span><span class="NLM_article-title">Nucleic Acid Adducts of Chemical Carcinogens and Mutagens</span>. <i>Arch. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">249</span>â <span class="NLM_lpage">285</span>, <span class="refDoi">&nbsp;DOI: 10.1007/BF00316495</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1007%2FBF00316495" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=6347126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADyaL3sXitFSru7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=1983&amp;pages=249-285&amp;author=K.+Hemminki&amp;title=Nucleic+Acid+Adducts+of+Chemical+Carcinogens+and+Mutagens&amp;doi=10.1007%2FBF00316495" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleic acid adducts of chemical carcinogens and mutagens</span></div><div class="casAuthors">Hemminki, Kari</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Toxicology</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">249-85</span>CODEN:
                <span class="NLM_cas:coden">ARTODN</span>;
        ISSN:<span class="NLM_cas:issn">0340-5761</span>.
    </div><div class="casAbstract">A review with several refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaKEd-g6561LVg90H21EOLACvtfcHk0liETExUWEb56g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL3sXitFSru7Y%253D&amp;md5=b622b3e11fb8bc6977a5d683914393ed</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1007%2FBF00316495&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00316495%26sid%3Dliteratum%253Aachs%26aulast%3DHemminki%26aufirst%3DK.%26atitle%3DNucleic%2520Acid%2520Adducts%2520of%2520Chemical%2520Carcinogens%2520and%2520Mutagens%26jtitle%3DArch.%2520Toxicol.%26date%3D1983%26volume%3D52%26spage%3D249%26epage%3D285%26doi%3D10.1007%2FBF00316495" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sale, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehmann, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodgate, R.</span></span> <span> </span><span class="NLM_article-title">Y-Family DNA Polymerases and their Role in Tolerance of Cellular DNA Damage</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">141</span>â <span class="NLM_lpage">152</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nrm3289</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1038%2Fnrm3289" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=22358330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC38Xis1ekurs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=13&amp;publication_year=2012&amp;pages=141-152&amp;author=J.+E.+Saleauthor=A.+R.+Lehmannauthor=R.+Woodgate&amp;title=Y-Family+DNA+Polymerases+and+their+Role+in+Tolerance+of+Cellular+DNA+Damage&amp;doi=10.1038%2Fnrm3289" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Y-family DNA polymerases and their role in tolerance of cellular DNA damage</span></div><div class="casAuthors">Sale, Julian E.; Lehmann, Alan R.; Woodgate, Roger</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">141-152</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The past 15 years have seen an explosion in our understanding of how cells replicate damaged DNA and how this can lead to mutagenesis.  The Y-family DNA polymerases lie at the heart of this process, which is commonly known as translesion synthesis.  This family of polymerases has unique features that enable them to synthesize DNA past damaged bases.  However, as they exhibit low fidelity when copying undamaged DNA, it is essential that they are only called into play when they are absolutely required.  Several layers of regulation ensure that this is achieved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJf4Ctv0l1jLVg90H21EOLACvtfcHk0liETExUWEb56g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38Xis1ekurs%253D&amp;md5=0bfb44c9b222edae8a1779422cf32cd1</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrm3289&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3289%26sid%3Dliteratum%253Aachs%26aulast%3DSale%26aufirst%3DJ.%2BE.%26aulast%3DLehmann%26aufirst%3DA.%2BR.%26aulast%3DWoodgate%26aufirst%3DR.%26atitle%3DY-Family%2520DNA%2520Polymerases%2520and%2520their%2520Role%2520in%2520Tolerance%2520of%2520Cellular%2520DNA%2520Damage%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D141%26epage%3D152%26doi%3D10.1038%2Fnrm3289" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jarosz, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godoy, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essigmann, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. C.</span></span> <span> </span><span class="NLM_article-title">A Single Amino Acid Governs Enhanced Activity of DinB DNA Polymerases on Damaged Templates</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>439</i></span>,  <span class="NLM_fpage">225</span>â <span class="NLM_lpage">228</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature04318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1038%2Fnature04318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=16407906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A280%3ADC%252BD28%252FitVyrtw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=439&amp;publication_year=2006&amp;pages=225-228&amp;author=D.+F.+Jaroszauthor=V.+G.+Godoyauthor=J.+C.+Delaneyauthor=J.+M.+Essigmannauthor=G.+C.+Walker&amp;title=A+Single+Amino+Acid+Governs+Enhanced+Activity+of+DinB+DNA+Polymerases+on+Damaged+Templates&amp;doi=10.1038%2Fnature04318" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A single amino acid governs enhanced activity of DinB DNA polymerases on damaged templates</span></div><div class="casAuthors">Jarosz Daniel F; Godoy Veronica G; Delaney James C; Essigmann John M; Walker Graham C</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">439</span>
        (<span class="NLM_cas:issue">7073</span>),
    <span class="NLM_cas:pages">225-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Translesion synthesis (TLS) by Y-family DNA polymerases is a chief mechanism of DNA damage tolerance.  Such TLS can be accurate or error-prone, as it is for bypass of a cyclobutane pyrimidine dimer by DNA polymerase eta (XP-V or Rad30) or bypass of a (6-4) TT photoproduct by DNA polymerase V (UmuD'2C), respectively.  Although DinB is the only Y-family DNA polymerase conserved among all domains of life, the biological rationale for this striking conservation has remained enigmatic.  Here we report that the Escherichia coli dinB gene is required for resistance to some DNA-damaging agents that form adducts at the N2-position of deoxyguanosine (dG).  We show that DinB (DNA polymerase IV) catalyses accurate TLS over one such N2-dG adduct (N2-furfuryl-dG), and that DinB and its mammalian orthologue, DNA polymerase kappa, insert deoxycytidine (dC) opposite N2-furfuryl-dG with 10-15-fold greater catalytic proficiency than opposite undamaged dG.  We also show that mutating a single amino acid, the 'steric gate' residue of DinB (Phe13 --&gt; Val) and that of its archaeal homologue Dbh (Phe12 --&gt; Ala), separates the abilities of these enzymes to perform TLS over N2-dG adducts from their abilities to replicate an undamaged template.  We propose that DinB and its orthologues are specialized to catalyse relatively accurate TLS over some N2-dG adducts that are ubiquitous in nature, that lesion bypass occurs more efficiently than synthesis on undamaged DNA, and that this specificity may be achieved at least in part through a lesion-induced conformational change.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSygKbT0LHAudTiMd2aNQVUfW6udTcc2eZWLkQA8m3KGrntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BD28%252FitVyrtw%253D%253D&amp;md5=d38a3179bb2c3e01de936f5b52d37ab1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature04318&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04318%26sid%3Dliteratum%253Aachs%26aulast%3DJarosz%26aufirst%3DD.%2BF.%26aulast%3DGodoy%26aufirst%3DV.%2BG.%26aulast%3DDelaney%26aufirst%3DJ.%2BC.%26aulast%3DEssigmann%26aufirst%3DJ.%2BM.%26aulast%3DWalker%26aufirst%3DG.%2BC.%26atitle%3DA%2520Single%2520Amino%2520Acid%2520Governs%2520Enhanced%2520Activity%2520of%2520DinB%2520DNA%2520Polymerases%2520on%2520Damaged%2520Templates%26jtitle%3DNature%26date%3D2006%26volume%3D439%26spage%3D225%26epage%3D228%26doi%3D10.1038%2Fnature04318" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kottur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gore, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeepkumar, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, D. T.</span></span> <span> </span><span class="NLM_article-title">Unique Structural Features in DNA Polymerase IV Enable Efficient Bypass of the N2 Adduct Induced by the Nitrofurazone Antibiotic</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">56</span>â <span class="NLM_lpage">67</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.str.2014.10.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1016%2Fj.str.2014.10.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=25497730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFCrtLjM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2015&amp;pages=56-67&amp;author=J.+Kotturauthor=A.+Sharmaauthor=K.+R.+Goreauthor=N.+Narayananauthor=B.+Samantaauthor=P.+I.+Pradeepkumarauthor=D.+T.+Nair&amp;title=Unique+Structural+Features+in+DNA+Polymerase+IV+Enable+Efficient+Bypass+of+the+N2+Adduct+Induced+by+the+Nitrofurazone+Antibiotic&amp;doi=10.1016%2Fj.str.2014.10.019" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Unique Structural Features in DNA Polymerase IV Enable Efficient Bypass of the N2 Adduct Induced by the Nitrofurazone Antibiotic</span></div><div class="casAuthors">Kottur, Jithesh; Sharma, Amit; Gore, Kiran R.; Narayanan, Naveen; Samanta, Biswajit; Pradeepkumar, Pushpangadan I.; Nair, Deepak T.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-67</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The redn. in the efficacy of therapeutic antibiotics represents a global problem of increasing intensity and concern.  Nitrofuran antibiotics act primarily through the formation of covalent adducts at the N2 atom of the deoxyguanosine nucleotide in genomic DNA.  These adducts inhibit replicative DNA polymerases (dPols), leading to the death of the prokaryote.  N2-furfuryl-deoxyguanosine (fdG) represents a stable structural analog of the nitrofuran-induced adducts.  Unlike other known dPols, DNA polymerase IV (PolIV) from E. coli can bypass the fdG adduct accurately with high catalytic efficiency.  This property of PolIV is central to its role in reducing the sensitivity of E. coli toward nitrofuran antibiotics such as nitrofurazone (NFZ).  We present the mechanism used by PolIV to bypass NFZ-induced adducts and thus improve viability of E. coli in the presence of NFZ.  Our results can be used to develop specific inhibitors of PolIV that may potentiate the activity of nitrofuran antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1UosfmyW8ZLVg90H21EOLACvtfcHk0lhlbM1A_-RHpQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXitFCrtLjM&amp;md5=4cfe91793eb48666059432db8b735c38</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.10.019&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.10.019%26sid%3Dliteratum%253Aachs%26aulast%3DKottur%26aufirst%3DJ.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DGore%26aufirst%3DK.%2BR.%26aulast%3DNarayanan%26aufirst%3DN.%26aulast%3DSamanta%26aufirst%3DB.%26aulast%3DPradeepkumar%26aufirst%3DP.%2BI.%26aulast%3DNair%26aufirst%3DD.%2BT.%26atitle%3DUnique%2520Structural%2520Features%2520in%2520DNA%2520Polymerase%2520IV%2520Enable%2520Efficient%2520Bypass%2520of%2520the%2520N2%2520Adduct%2520Induced%2520by%2520the%2520Nitrofurazone%2520Antibiotic%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D56%26epage%3D67%26doi%3D10.1016%2Fj.str.2014.10.019" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furukohri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philippin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loechler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akiyama, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katayama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maki, H.</span></span> <span> </span><span class="NLM_article-title">DNA Polymerase IV Mediates Efficient and Quick Recovery of Replication Forks Stalled at N2-dG Adducts</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">8461</span>â <span class="NLM_lpage">8472</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gku547</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1093%2Fnar%2Fgku547" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=42&amp;publication_year=2014&amp;pages=8461-8472&amp;author=M.+Ikedaauthor=A.+Furukohriauthor=G.+Philippinauthor=E.+Loechlerauthor=M.+T.+Akiyamaauthor=T.+Katayamaauthor=R.+P.+Fuchsauthor=H.+Maki&amp;title=DNA+Polymerase+IV+Mediates+Efficient+and+Quick+Recovery+of+Replication+Forks+Stalled+at+N2-dG+Adducts&amp;doi=10.1093%2Fnar%2Fgku547" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku547&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku547%26sid%3Dliteratum%253Aachs%26aulast%3DIkeda%26aufirst%3DM.%26aulast%3DFurukohri%26aufirst%3DA.%26aulast%3DPhilippin%26aufirst%3DG.%26aulast%3DLoechler%26aufirst%3DE.%26aulast%3DAkiyama%26aufirst%3DM.%2BT.%26aulast%3DKatayama%26aufirst%3DT.%26aulast%3DFuchs%26aufirst%3DR.%2BP.%26aulast%3DMaki%26aufirst%3DH.%26atitle%3DDNA%2520Polymerase%2520IV%2520Mediates%2520Efficient%2520and%2520Quick%2520Recovery%2520of%2520Replication%2520Forks%2520Stalled%2520at%2520N2-dG%2520Adducts%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D8461%26epage%3D8472%26doi%3D10.1093%2Fnar%2Fgku547" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moschel, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudgins, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dipple, A.</span></span> <span> </span><span class="NLM_article-title">Aralkylation of Guanosine by the Carcinogen N-Nitroso-N-Benzylurea</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">533</span>â <span class="NLM_lpage">535</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jo01291a037</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo01291a037" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=1980&amp;pages=533-535&amp;author=R.+C.+Moschelauthor=W.+R.+Hudginsauthor=A.+Dipple&amp;title=Aralkylation+of+Guanosine+by+the+Carcinogen+N-Nitroso-N-Benzylurea&amp;doi=10.1021%2Fjo01291a037" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjo01291a037&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo01291a037%26sid%3Dliteratum%253Aachs%26aulast%3DMoschel%26aufirst%3DR.%2BC.%26aulast%3DHudgins%26aufirst%3DW.%2BR.%26aulast%3DDipple%26aufirst%3DA.%26atitle%3DAralkylation%2520of%2520Guanosine%2520by%2520the%2520Carcinogen%2520N-Nitroso-N-Benzylurea%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1980%26volume%3D45%26spage%3D533%26epage%3D535%26doi%3D10.1021%2Fjo01291a037" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, L. A.</span></span> <span> </span><span class="NLM_article-title">N-Nitrosobenzylmethylamine Is Activated to a DNA Benzylating Agent in Rats</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">19</span>â <span class="NLM_lpage">26</span>, <span class="refDoi">&nbsp;DOI: 10.1021/tx9601014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx9601014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=1997&amp;pages=19-26&amp;author=L.+A.+Peterson&amp;title=N-Nitrosobenzylmethylamine+Is+Activated+to+a+DNA+Benzylating+Agent+in+Rats&amp;doi=10.1021%2Ftx9601014" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Ftx9601014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx9601014%26sid%3Dliteratum%253Aachs%26aulast%3DPeterson%26aufirst%3DL.%2BA.%26atitle%3DN-Nitrosobenzylmethylamine%2520Is%2520Activated%2520to%2520a%2520DNA%2520Benzylating%2520Agent%2520in%2520Rats%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D1997%26volume%3D10%26spage%3D19%26epage%3D26%26doi%3D10.1021%2Ftx9601014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moon, K.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. S.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Characterization of Oligonucleotides Containing Site-Specific Bulky N2-Aralkylated Guanines and N6-Aralkylated Adenines</span>. <i>Arch. Pharmacal Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">139</span>â <span class="NLM_lpage">146</span>, <span class="refDoi">&nbsp;DOI: 10.1007/BF02975502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1007%2FBF02975502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10836739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjs12hsbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2000&amp;pages=139-146&amp;author=K.-Y.+Moonauthor=Y.+S.+Kim&amp;title=Synthesis+and+Characterization+of+Oligonucleotides+Containing+Site-Specific+Bulky+N2-Aralkylated+Guanines+and+N6-Aralkylated+Adenines&amp;doi=10.1007%2FBF02975502" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and characterization of oligonucleotides containing site-specific bulky N2-aralkylated guanines and N6-aralkylated adenines</span></div><div class="casAuthors">Moon, Ki-Young; Kim, Yeong Shik</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Pharmacal Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">139-146</span>CODEN:
                <span class="NLM_cas:coden">APHRDQ</span>;
        ISSN:<span class="NLM_cas:issn">0253-6269</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Korea</span>)
        </div><div class="casAbstract">7 - Bromomethylbenz[a]anthracene is a known mutagen and carcinogen.  The two major DNA adducts produced by this carcinogen, i.e., N2-(benz[a]anthracen-7-ylmethyl)-2'-deoxyguanosine (b[a]a2G) and N6-(benz[a]anthracen-7-ylmethyl)-2'-deoxyadenosine (b[a]a6A), as well as the simpler benzylated analogs, N2-benzyl-2'-deoxyguanosine (bn2G) and N6-benzyl-2'-deoxyadenosine (bn6A), were prepd. by direct aralkylation of 2'-deoxyguanosine and 2'-deoxyadenosine.  To det. the site-specific mutagenicity of these bulky exocyclic amino-substituted adducts, the suitably protected nucleosides were incorporated into 16-base oligodeoxyribonucleotides in place of a normal guanine or adenine residues which resp. are part of the ATG initiation codon for the lac Z' Î±-complementation gene by using an in situ activation approach and automated phosphite triester synthetic methods.  The base compn. and the incorporation of the bulky adducts into synthetic oligonucleotides were characterized after purifn. of the modified oligonucleotides by enzymic digestion and HPLC anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJFCEGfotRbLVg90H21EOLACvtfcHk0lhlbM1A_-RHpQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3cXjs12hsbc%253D&amp;md5=a692e97c0abd6380d292e7cbad7cebf2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2FBF02975502&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02975502%26sid%3Dliteratum%253Aachs%26aulast%3DMoon%26aufirst%3DK.-Y.%26aulast%3DKim%26aufirst%3DY.%2BS.%26atitle%3DSynthesis%2520and%2520Characterization%2520of%2520Oligonucleotides%2520Containing%2520Site-Specific%2520Bulky%2520N2-Aralkylated%2520Guanines%2520and%2520N6-Aralkylated%2520Adenines%26jtitle%3DArch.%2520Pharmacal%2520Res.%26date%3D2000%26volume%3D23%26spage%3D139%26epage%3D146%26doi%3D10.1007%2FBF02975502" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koag, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span> <span> </span><span class="NLM_article-title">Structural and Kinetic Studies of the Effect of Guanine-N7 Alkylation and Metal Cofactors on DNA Replication</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">5105</span>â <span class="NLM_lpage">5116</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.biochem.8b00331</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00331" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ejsrjK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=57&amp;publication_year=2018&amp;pages=5105-5116&amp;author=Y.+Kouauthor=M.-C.+Koagauthor=S.+Lee&amp;title=Structural+and+Kinetic+Studies+of+the+Effect+of+Guanine-N7+Alkylation+and+Metal+Cofactors+on+DNA+Replication&amp;doi=10.1021%2Facs.biochem.8b00331" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Kinetic Studies of the Effect of Guanine N7 Alkylation and Metal Cofactors on DNA Replication</span></div><div class="casAuthors">Kou, Yi; Koag, Myong-Chul; Lee, Seongmin</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">5105-5116</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A wide variety of endogenous and exogenous alkylating agents attack DNA to preferentially generate N7-alkylguanine (N7-alkylG) adducts.  Studies on the effect of N7-alkylG lesions on biol. processes have been difficult due in part to complications arising from the chem. lability of the pos. charged N7-alkylG, which can readily produce secondary lesions.  To assess the effect of bulky N7-alkylG on DNA replication, we prepd. chem. stable N7-benzylguanine(N7bnG)-contg. DNA and evaluated nucleotide incorporation opposite the lesion by human DNA polymerase Î² (polÎ²), a model enzyme for high-fidelity DNA polymerases.  Kinetic studies showed that the N7-benzyl-G lesion greatly inhibited dCTP incorporation by polÎ².  The crystal structure of polÎ² incorporating dCTP opposite N7bnG showed a Watson-Crick N7bnG:dCTP.  The polÎ²-N7bnG:dCTP structure showed an open protein conformation, a relatively disordered dCTP, and lack of catalytic metal, which explained the inefficient nucleotide incorporation opposite N7bnG.  This indicates that polÎ² is sensitive to major groove adducts in the templating-base side and deters nucleotide incorporation opposite bulky N7-alkylG adducts by adopting a catalytically incompetent conformation.  Substituting Mg2+ for Mn2+ induced an open-to-closed conformational change due to the presence of catalytic metal and stably bound dCTP and increased the catalytic efficiency by â¼10-fold, highlighting the effect of binding of incoming nucleotide and catalytic metal on protein conformation and nucleotidyl transfer reaction.  Overall, these results suggest that, although bulky alkyl groups at guanine-N7 may not alter base-pairing properties of guanine, the major-groove-positioned lesions in the template could impede nucleotidyl transfer by some DNA polymerases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKx0GLtkJBIrVg90H21EOLACvtfcHk0lhO_4B9J-L9tA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1cXht1ejsrjK&amp;md5=09f63033f165ccca4030d6b993b7d3c1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00331&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00331%26sid%3Dliteratum%253Aachs%26aulast%3DKou%26aufirst%3DY.%26aulast%3DKoag%26aufirst%3DM.-C.%26aulast%3DLee%26aufirst%3DS.%26atitle%3DStructural%2520and%2520Kinetic%2520Studies%2520of%2520the%2520Effect%2520of%2520Guanine-N7%2520Alkylation%2520and%2520Metal%2520Cofactors%2520on%2520DNA%2520Replication%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26spage%3D5105%26epage%3D5116%26doi%3D10.1021%2Facs.biochem.8b00331" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flesher, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehner, A. F.</span></span> <span> </span><span class="NLM_article-title">9-Sulfooxymethylanthracene is an ultimate electrophilic and carcinogenic form of 9-hydroxymethylanthracene</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>251</i></span>,  <span class="NLM_fpage">239</span>â <span class="NLM_lpage">243</span>, <span class="refDoi">&nbsp;DOI: 10.1006/bbrc.1998.9448</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1006%2Fbbrc.1998.9448" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=251&amp;publication_year=1998&amp;pages=239-243&amp;author=J.+W.+Flesherauthor=J.+Hornauthor=A.+F.+Lehner&amp;title=9-Sulfooxymethylanthracene+is+an+ultimate+electrophilic+and+carcinogenic+form+of+9-hydroxymethylanthracene&amp;doi=10.1006%2Fbbrc.1998.9448" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1998.9448&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1998.9448%26sid%3Dliteratum%253Aachs%26aulast%3DFlesher%26aufirst%3DJ.%2BW.%26aulast%3DHorn%26aufirst%3DJ.%26aulast%3DLehner%26aufirst%3DA.%2BF.%26atitle%3D9-Sulfooxymethylanthracene%2520is%2520an%2520ultimate%2520electrophilic%2520and%2520carcinogenic%2520form%2520of%25209-hydroxymethylanthracene%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1998%26volume%3D251%26spage%3D239%26epage%3D243%26doi%3D10.1006%2Fbbrc.1998.9448" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Casale, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, L. W.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Properties of an Oligodeoxynucleotide Containing a Polycyclic Aromatic Hydrocarbon Site Specifically Bound to the N2-Amino Group of a 2â²-Deoxyguanosine Residue</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">5264</span>â <span class="NLM_lpage">5271</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja00169a039</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00169a039" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=112&amp;publication_year=1990&amp;pages=5264-5271&amp;author=R.+Casaleauthor=L.+W.+McLaughlin&amp;title=Synthesis+and+Properties+of+an+Oligodeoxynucleotide+Containing+a+Polycyclic+Aromatic+Hydrocarbon+Site+Specifically+Bound+to+the+N2-Amino+Group+of+a+2%E2%80%B2-Deoxyguanosine+Residue&amp;doi=10.1021%2Fja00169a039" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fja00169a039&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00169a039%26sid%3Dliteratum%253Aachs%26aulast%3DCasale%26aufirst%3DR.%26aulast%3DMcLaughlin%26aufirst%3DL.%2BW.%26atitle%3DSynthesis%2520and%2520Properties%2520of%2520an%2520Oligodeoxynucleotide%2520Containing%2520a%2520Polycyclic%2520Aromatic%2520Hydrocarbon%2520Site%2520Specifically%2520Bound%2520to%2520the%2520N2-Amino%2520Group%2520of%2520a%25202%25E2%2580%25B2-Deoxyguanosine%2520Residue%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1990%26volume%3D112%26spage%3D5264%26epage%3D5271%26doi%3D10.1021%2Fja00169a039" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bendadani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monien, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobbernack, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glatt, H.</span></span> <span> </span><span class="NLM_article-title">The Carcinogen 1-Methylpyrene Forms Benzylic DNA Adducts in Mouse and Rat Tissues in Vivo via a Reactive Sulphuric Acid Ester</span>. <i>Arch. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">815</span>â <span class="NLM_lpage">821</span>, <span class="refDoi">&nbsp;DOI: 10.1007/s00204-013-1182-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1007%2Fs00204-013-1182-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=24337722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFajsb%252FE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=88&amp;publication_year=2014&amp;pages=815-821&amp;author=C.+Bendadaniauthor=W.+Meinlauthor=B.+H.+Monienauthor=G.+Dobbernackauthor=H.+Glatt&amp;title=The+Carcinogen+1-Methylpyrene+Forms+Benzylic+DNA+Adducts+in+Mouse+and+Rat+Tissues+in+Vivo+via+a+Reactive+Sulphuric+Acid+Ester&amp;doi=10.1007%2Fs00204-013-1182-6" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The carcinogen 1-methylpyrene forms benzylic DNA adducts in mouse and rat tissues in vivo via a reactive sulphuric acid ester</span></div><div class="casAuthors">Bendadani, Carolin; Meinl, Walter; Monien, Bernhard H.; Dobbernack, Gisela; Glatt, Hansruedi</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Toxicology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">815-821</span>CODEN:
                <span class="NLM_cas:coden">ARTODN</span>;
        ISSN:<span class="NLM_cas:issn">0340-5761</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The common polycyclic arom. hydrocarbon 1-methylpyrene is hepatocarcinogenic in the newborn mouse assay.  In vitro studies showed that it is metabolically activated via benzylic hydroxylation and sulphation to a reactive ester, which forms benzylic DNA adducts, N2-(1-methylpyrenyl)-2'-deoxyguanosine (MPdG) and N6-(1-methylpyrenyl)-2'-deoxyadenosine (MPdA).  Formation of these adducts was also obsd. in animals treated with the metabolites, 1-hydroxymethylpyrene and 1-sulfooxymethylpyrene (1-SMP), whereas corresponding data are missing for 1-methylpyrene.  In the present study, we treated mice with 1-methylpyrene and subsequently analyzed blood serum for the presence of the reactive metabolite 1-SMP and tissue DNA for the presence of MPdG and MPdA adducts.  We used wild-type mice and a mouse line transgenic for human sulfotransferases (SULT) 1A1 and 1A2, males and females.  All analyses were conducted using ultra-performance liq. chromatog. coupled with tandem mass spectrometry, for the adducts with isotope-labeled internal stds. 1-SMP was detected in all treated animals.  Its serum level was higher in transgenic mice than in the wild-type (p &lt; 0.001).  Likewise, both adducts were detected in liver, kidney and lung DNA of all exposed animals.  The transgene significantly enhanced the level of each adduct in each tissue of both sexes (p &lt; 0.01-0.001).  Adduct levels were highest in the liver, the target tissue of carcinogenesis, in each animal model used.  MPdG and MPdA adducts were also obsd. in rats treated with 1-methylpyrene.  Our findings corroborate the hypothesis that 1-SMP is indeed the ultimate carcinogen of 1-methylpyrene and that human SULT are able to mediate the terminal activation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-_bKPLkvUnbVg90H21EOLACvtfcHk0lhO_4B9J-L9tA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhvFajsb%252FE&amp;md5=fb85b6ac15fade532bb93abb32b51de1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2Fs00204-013-1182-6&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00204-013-1182-6%26sid%3Dliteratum%253Aachs%26aulast%3DBendadani%26aufirst%3DC.%26aulast%3DMeinl%26aufirst%3DW.%26aulast%3DMonien%26aufirst%3DB.%2BH.%26aulast%3DDobbernack%26aufirst%3DG.%26aulast%3DGlatt%26aufirst%3DH.%26atitle%3DThe%2520Carcinogen%25201-Methylpyrene%2520Forms%2520Benzylic%2520DNA%2520Adducts%2520in%2520Mouse%2520and%2520Rat%2520Tissues%2520in%2520Vivo%2520via%2520a%2520Reactive%2520Sulphuric%2520Acid%2520Ester%26jtitle%3DArch.%2520Toxicol.%26date%3D2014%26volume%3D88%26spage%3D815%26epage%3D821%26doi%3D10.1007%2Fs00204-013-1182-6" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bendadani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monien, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobbernack, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engst, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Himmelbauer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glatt, H.</span></span> <span> </span><span class="NLM_article-title">Determination of Sulfotransferase Forms Involved in the Metabolic Activation of the Genotoxicant 1-Hydroxymethylpyrene Using Bacterially Expressed Enzymes and Genetically Modified Mouse Models</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1060</span>â <span class="NLM_lpage">1069</span>, <span class="refDoi">&nbsp;DOI: 10.1021/tx500129g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx500129g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=27&amp;publication_year=2014&amp;pages=1060-1069&amp;author=C.+Bendadaniauthor=W.+Meinlauthor=B.+Monienauthor=G.+Dobbernackauthor=S.+Florianauthor=W.+Engstauthor=T.+Noldenauthor=H.+Himmelbauerauthor=H.+Glatt&amp;title=Determination+of+Sulfotransferase+Forms+Involved+in+the+Metabolic+Activation+of+the+Genotoxicant+1-Hydroxymethylpyrene+Using+Bacterially+Expressed+Enzymes+and+Genetically+Modified+Mouse+Models&amp;doi=10.1021%2Ftx500129g" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Ftx500129g&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx500129g%26sid%3Dliteratum%253Aachs%26aulast%3DBendadani%26aufirst%3DC.%26aulast%3DMeinl%26aufirst%3DW.%26aulast%3DMonien%26aufirst%3DB.%26aulast%3DDobbernack%26aufirst%3DG.%26aulast%3DFlorian%26aufirst%3DS.%26aulast%3DEngst%26aufirst%3DW.%26aulast%3DNolden%26aufirst%3DT.%26aulast%3DHimmelbauer%26aufirst%3DH.%26aulast%3DGlatt%26aufirst%3DH.%26atitle%3DDetermination%2520of%2520Sulfotransferase%2520Forms%2520Involved%2520in%2520the%2520Metabolic%2520Activation%2520of%2520the%2520Genotoxicant%25201-Hydroxymethylpyrene%2520Using%2520Bacterially%2520Expressed%2520Enzymes%2520and%2520Genetically%2520Modified%2520Mouse%2520Models%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2014%26volume%3D27%26spage%3D1060%26epage%3D1069%26doi%3D10.1021%2Ftx500129g" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guengerich, F. P.</span></span> <span> </span><span class="NLM_article-title">Interactions of Carcinogen-Bound DNA with Individual DNA Polymerases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">420</span>â <span class="NLM_lpage">452</span>, <span class="refDoi">&nbsp;DOI: 10.1021/cr0404693</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr0404693" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD28XitVyksw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=106&amp;publication_year=2006&amp;pages=420-452&amp;author=F.+P.+Guengerich&amp;title=Interactions+of+Carcinogen-Bound+DNA+with+Individual+DNA+Polymerases&amp;doi=10.1021%2Fcr0404693" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Interactions of Carcinogen-Bound DNA with Individual DNA Polymerases</span></div><div class="casAuthors">Guengerich, F. Peter</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">420-452</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Several types of studies with DNA polymerases and carcinogen-modified DNA will be presented and discussed, along with some of the author's own interpretations about relevance.  Although the interactions within cells are most important and many studies have been done, the focus of this review will be studies of individual, isolated DNA polymerases with oligonucleotides bearing chem. defined adducts at specific sites.  The most detailed understanding of systems is possible at this level, albeit reductionist, particularly using kinetic anal. and x-ray crystallog. to study structure and function.  Some discussion of site-specific mutagenesis and mutation spectra will be included.  The point can be made that reliance on only a single approach will never yield a particularly insightful understanding of mechanisms of chem. induced mutagenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps1kyQQ7A0LrVg90H21EOLACvtfcHk0lgE4wRP2gSkHA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28XitVyksw%253D%253D&amp;md5=9ba54dbba504a057056a59dffe3e23ca</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fcr0404693&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr0404693%26sid%3Dliteratum%253Aachs%26aulast%3DGuengerich%26aufirst%3DF.%2BP.%26atitle%3DInteractions%2520of%2520Carcinogen-Bound%2520DNA%2520with%2520Individual%2520DNA%2520Polymerases%26jtitle%3DChem.%2520Rev.%26date%3D2006%26volume%3D106%26spage%3D420%26epage%3D452%26doi%3D10.1021%2Fcr0404693" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Essigmann, J. M.</span></span> <span> </span><span class="NLM_article-title">Biological Properties of Single Chemical-DNA Adducts: A Twenty Year Perspective</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">232</span>â <span class="NLM_lpage">252</span>, <span class="refDoi">&nbsp;DOI: 10.1021/tx700292a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx700292a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVWqs7bO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2008&amp;pages=232-252&amp;author=J.+C.+Delaneyauthor=J.+M.+Essigmann&amp;title=Biological+Properties+of+Single+Chemical-DNA+Adducts%3A+A+Twenty+Year+Perspective&amp;doi=10.1021%2Ftx700292a" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Biological Properties of Single Chemical-DNA Adducts: A Twenty Year Perspective</span></div><div class="casAuthors">Delaney, James C.; Essigmann, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">232-252</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The genome and its nucleotide precursor pool are under sustained attack by radiation, reactive oxygen and nitrogen species, chem. carcinogens, hydrolytic reactions, and certain drugs.  As a result, a large and heterogeneous population of damaged nucleotides forms in all cells.  Some of the lesions are repaired, but for those that remain, there can be serious biol. consequences.  For example, lesions that form in DNA can lead to altered gene expression, mutation, and death.  This perspective examines systems developed over the past 20 years to study the biol. properties of single DNA lesions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPu0MDivE9nLVg90H21EOLACvtfcHk0lgE4wRP2gSkHA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXhsVWqs7bO&amp;md5=42900335a8dd1208cf8716272bc021df</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Ftx700292a&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx700292a%26sid%3Dliteratum%253Aachs%26aulast%3DDelaney%26aufirst%3DJ.%2BC.%26aulast%3DEssigmann%26aufirst%3DJ.%2BM.%26atitle%3DBiological%2520Properties%2520of%2520Single%2520Chemical-DNA%2520Adducts%253A%2520A%2520Twenty%2520Year%2520Perspective%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26spage%3D232%26epage%3D252%26doi%3D10.1021%2Ftx700292a" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeCorte, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsarouhtsis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuchimanchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, T. M.</span></span> <span> </span><span class="NLM_article-title">Improved Strategies for Postoligomerization Synthesis of Oligodeoxynucleotides Bearing Structurally Defined Adducts at the N2 Position of Deoxyguanosine</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">630</span>â <span class="NLM_lpage">637</span>, <span class="refDoi">&nbsp;DOI: 10.1021/tx9501795</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx9501795" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=1996&amp;pages=630-637&amp;author=B.+L.+DeCorteauthor=D.+Tsarouhtsisauthor=S.+Kuchimanchiauthor=M.+D.+Cooperauthor=P.+Hortonauthor=C.+M.+Harrisauthor=T.+M.+Harris&amp;title=Improved+Strategies+for+Postoligomerization+Synthesis+of+Oligodeoxynucleotides+Bearing+Structurally+Defined+Adducts+at+the+N2+Position+of+Deoxyguanosine&amp;doi=10.1021%2Ftx9501795" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Ftx9501795&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx9501795%26sid%3Dliteratum%253Aachs%26aulast%3DDeCorte%26aufirst%3DB.%2BL.%26aulast%3DTsarouhtsis%26aufirst%3DD.%26aulast%3DKuchimanchi%26aufirst%3DS.%26aulast%3DCooper%26aufirst%3DM.%2BD.%26aulast%3DHorton%26aufirst%3DP.%26aulast%3DHarris%26aufirst%3DC.%2BM.%26aulast%3DHarris%26aufirst%3DT.%2BM.%26atitle%3DImproved%2520Strategies%2520for%2520Postoligomerization%2520Synthesis%2520of%2520Oligodeoxynucleotides%2520Bearing%2520Structurally%2520Defined%2520Adducts%2520at%2520the%2520N2%2520Position%2520of%2520Deoxyguanosine%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D1996%26volume%3D9%26spage%3D630%26epage%3D637%26doi%3D10.1021%2Ftx9501795" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steinbrecher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wameling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oesch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidel, A.</span></span> <span> </span><span class="NLM_article-title">Activation of the C2 Position of Purine by the Trifluoromethanesulfonate Group: Synthesis of N2-Alkylated Deoxyguanosines</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">404</span>â <span class="NLM_lpage">406</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.199304041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1002%2Fanie.199304041" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=32&amp;publication_year=1993&amp;pages=404-406&amp;author=T.+Steinbrecherauthor=C.+Wamelingauthor=F.+Oeschauthor=A.+Seidel&amp;title=Activation+of+the+C2+Position+of+Purine+by+the+Trifluoromethanesulfonate+Group%3A+Synthesis+of+N2-Alkylated+Deoxyguanosines&amp;doi=10.1002%2Fanie.199304041" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fanie.199304041&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.199304041%26sid%3Dliteratum%253Aachs%26aulast%3DSteinbrecher%26aufirst%3DT.%26aulast%3DWameling%26aufirst%3DC.%26aulast%3DOesch%26aufirst%3DF.%26aulast%3DSeidel%26aufirst%3DA.%26atitle%3DActivation%2520of%2520the%2520C2%2520Position%2520of%2520Purine%2520by%2520the%2520Trifluoromethanesulfonate%2520Group%253A%2520Synthesis%2520of%2520N2-Alkylated%2520Deoxyguanosines%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D1993%26volume%3D32%26spage%3D404%26epage%3D406%26doi%3D10.1002%2Fanie.199304041" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, G. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krawczyk, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terhorst, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Law, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">CoutrÃ©, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bischofberger, N.</span></span> <span> </span><span class="NLM_article-title">N2- and C8-Substituted Oligodeoxynucleotides with Enhanced Thrombin Inhibitory Activity in Vitro and in Vivo</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">2234</span>â <span class="NLM_lpage">2242</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jm970434d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970434d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=41&amp;publication_year=1998&amp;pages=2234-2242&amp;author=G.+X.+Heauthor=S.+H.+Krawczykauthor=S.+Swaminathanauthor=R.+G.+Sheaauthor=J.+P.+Doughertyauthor=T.+Terhorstauthor=V.+S.+Lawauthor=L.+C.+Griffinauthor=S.+Coutr%C3%A9author=N.+Bischofberger&amp;title=N2-+and+C8-Substituted+Oligodeoxynucleotides+with+Enhanced+Thrombin+Inhibitory+Activity+in+Vitro+and+in+Vivo&amp;doi=10.1021%2Fjm970434d" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm970434d&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970434d%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DG.%2BX.%26aulast%3DKrawczyk%26aufirst%3DS.%2BH.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DShea%26aufirst%3DR.%2BG.%26aulast%3DDougherty%26aufirst%3DJ.%2BP.%26aulast%3DTerhorst%26aufirst%3DT.%26aulast%3DLaw%26aufirst%3DV.%2BS.%26aulast%3DGriffin%26aufirst%3DL.%2BC.%26aulast%3DCoutr%25C3%25A9%26aufirst%3DS.%26aulast%3DBischofberger%26aufirst%3DN.%26atitle%3DN2-%2520and%2520C8-Substituted%2520Oligodeoxynucleotides%2520with%2520Enhanced%2520Thrombin%2520Inhibitory%2520Activity%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D2234%26epage%3D2242%26doi%3D10.1021%2Fjm970434d" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moon, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moschel, R. C.</span></span> <span> </span><span class="NLM_article-title">Effect of Ionic State of 2â²-Deoxyguanosine and Solvent on Its Aralkylation by Benzyl Bromide</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">696</span>â <span class="NLM_lpage">702</span>, <span class="refDoi">&nbsp;DOI: 10.1021/tx980012m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx980012m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=1998&amp;pages=696-702&amp;author=K.+Y.+Moonauthor=R.+C.+Moschel&amp;title=Effect+of+Ionic+State+of+2%E2%80%B2-Deoxyguanosine+and+Solvent+on+Its+Aralkylation+by+Benzyl+Bromide&amp;doi=10.1021%2Ftx980012m" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Ftx980012m&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx980012m%26sid%3Dliteratum%253Aachs%26aulast%3DMoon%26aufirst%3DK.%2BY.%26aulast%3DMoschel%26aufirst%3DR.%2BC.%26atitle%3DEffect%2520of%2520Ionic%2520State%2520of%25202%25E2%2580%25B2-Deoxyguanosine%2520and%2520Solvent%2520on%2520Its%2520Aralkylation%2520by%2520Benzyl%2520Bromide%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D1998%26volume%3D11%26spage%3D696%26epage%3D702%26doi%3D10.1021%2Ftx980012m" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sergueeva, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sergueev, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, B. R.</span></span> <span> </span><span class="NLM_article-title">Rapid and Selective Reduction of Amide Group by Borane-Amine Complexes in Acyl Protected Nucleosides</span>. <i>Nucleos. Nucleot. Nucl.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">275</span>â <span class="NLM_lpage">282</span>, <span class="refDoi">&nbsp;DOI: 10.1080/15257770008033009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1080%2F15257770008033009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10772715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVagur4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2000&amp;pages=275-282&amp;author=Z.+A.+Sergueevaauthor=D.+S.+Sergueevauthor=B.+R.+Shaw&amp;title=Rapid+and+Selective+Reduction+of+Amide+Group+by+Borane-Amine+Complexes+in+Acyl+Protected+Nucleosides&amp;doi=10.1080%2F15257770008033009" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid and selective reduction of amide group by borane-amine complexes in acyl protected nucleosides</span></div><div class="casAuthors">Sergueeva, Zinaida A.; Sergueev, Dmitri S.; Shaw, Barbara Ramsay</div><div class="citationInfo"><span class="NLM_cas:title">Nucleosides, Nucleotides &amp; Nucleic Acids</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1 &amp; 2</span>),
    <span class="NLM_cas:pages">275-282</span>CODEN:
                <span class="NLM_cas:coden">NNNAFY</span>;
        ISSN:<span class="NLM_cas:issn">1525-7770</span>.
    
            (<span class="NLM_cas:orgname">Marcel Dekker, Inc.</span>)
        </div><div class="casAbstract">Borane-amine complexes provide an unusually fast and selective redn. of a deoxynucleoside N-acyl group to a corresponding N-alkyl group.  Three different nucleosides (dG, dA, and dC) each having one of three N-protecting groups (benzoyl, isobutyryl, or acetyl) were used to prep. N-alkylated nucleosides in good yields under mild conditions.  Deoxyribose O-acyl protecting groups remain intact at the conditions of N-acyl group redn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIlOUZJbRkbbVg90H21EOLACvtfcHk0ljVbMhJVlOnww"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3cXhvVagur4%253D&amp;md5=31ba42c5e466f5421e4fcc86b76df13e</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1080%2F15257770008033009&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15257770008033009%26sid%3Dliteratum%253Aachs%26aulast%3DSergueeva%26aufirst%3DZ.%2BA.%26aulast%3DSergueev%26aufirst%3DD.%2BS.%26aulast%3DShaw%26aufirst%3DB.%2BR.%26atitle%3DRapid%2520and%2520Selective%2520Reduction%2520of%2520Amide%2520Group%2520by%2520Borane-Amine%2520Complexes%2520in%2520Acyl%2520Protected%2520Nucleosides%26jtitle%3DNucleos.%2520Nucleot.%2520Nucl.%26date%3D2000%26volume%3D19%26spage%3D275%26epage%3D282%26doi%3D10.1080%2F15257770008033009" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guengerich, F. P.</span></span> <span> </span><span class="NLM_article-title">Analysis of the Effect of Bulk at N2-Alkylguanine DNA Adducts on Catalytic Efficiency and Fidelity of the Processive DNA Polymerases Bacteriophage T7 Exonuclease- and HIV-1 Reverse Transcriptase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">19217</span>â <span class="NLM_lpage">19229</span>, <span class="refDoi">&nbsp;DOI: 10.1074/jbc.M313759200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1074%2Fjbc.M313759200" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=279&amp;publication_year=2004&amp;pages=19217-19229&amp;author=J.+Choiauthor=F.+P.+Guengerich&amp;title=Analysis+of+the+Effect+of+Bulk+at+N2-Alkylguanine+DNA+Adducts+on+Catalytic+Efficiency+and+Fidelity+of+the+Processive+DNA+Polymerases+Bacteriophage+T7+Exonuclease-+and+HIV-1+Reverse+Transcriptase&amp;doi=10.1074%2Fjbc.M313759200" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M313759200&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M313759200%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%26aulast%3DGuengerich%26aufirst%3DF.%2BP.%26atitle%3DAnalysis%2520of%2520the%2520Effect%2520of%2520Bulk%2520at%2520N2-Alkylguanine%2520DNA%2520Adducts%2520on%2520Catalytic%2520Efficiency%2520and%2520Fidelity%2520of%2520the%2520Processive%2520DNA%2520Polymerases%2520Bacteriophage%2520T7%2520Exonuclease-%2520and%2520HIV-1%2520Reverse%2520Transcriptase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D19217%26epage%3D19229%26doi%3D10.1074%2Fjbc.M313759200" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonala, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shishkina, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, F.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Biologically Active N2-Amine Adducts of 2â²-Deoxyguanosine</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">7281</span>â <span class="NLM_lpage">7284</span>, <span class="refDoi">&nbsp;DOI: 10.1016/S0040-4039(00)01196-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1016%2FS0040-4039%2800%2901196-5" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=41&amp;publication_year=2000&amp;pages=7281-7284&amp;author=R.+R.+Bonalaauthor=I.+G.+Shishkinaauthor=F.+Johnson&amp;title=Synthesis+of+Biologically+Active+N2-Amine+Adducts+of+2%E2%80%B2-Deoxyguanosine&amp;doi=10.1016%2FS0040-4039%2800%2901196-5" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2800%2901196-5&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252800%252901196-5%26sid%3Dliteratum%253Aachs%26aulast%3DBonala%26aufirst%3DR.%2BR.%26aulast%3DShishkina%26aufirst%3DI.%2BG.%26aulast%3DJohnson%26aufirst%3DF.%26atitle%3DSynthesis%2520of%2520Biologically%2520Active%2520N2-Amine%2520Adducts%2520of%25202%25E2%2580%25B2-Deoxyguanosine%26jtitle%3DTetrahedron%2520Lett.%26date%3D2000%26volume%3D41%26spage%3D7281%26epage%3D7284%26doi%3D10.1016%2FS0040-4039%2800%2901196-5" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luna, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stezowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiselyo, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, R. G.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Oligonucleotide Adducts of the Bay Region Diol Epoxide Metabolites of Carcinogenic Polycyclic Aromatic Hydrocarbons</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5604</span>â <span class="NLM_lpage">5613</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jo00122a048</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00122a048" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=60&amp;publication_year=1995&amp;pages=5604-5613&amp;author=H.+Leeauthor=E.+Lunaauthor=M.+Hinzauthor=J.+J.+Stezowskiauthor=A.+S.+Kiselyoauthor=R.+G.+Harvey&amp;title=Synthesis+of+Oligonucleotide+Adducts+of+the+Bay+Region+Diol+Epoxide+Metabolites+of+Carcinogenic+Polycyclic+Aromatic+Hydrocarbons&amp;doi=10.1021%2Fjo00122a048" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjo00122a048&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00122a048%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%26aulast%3DLuna%26aufirst%3DE.%26aulast%3DHinz%26aufirst%3DM.%26aulast%3DStezowski%26aufirst%3DJ.%2BJ.%26aulast%3DKiselyo%26aufirst%3DA.%2BS.%26aulast%3DHarvey%26aufirst%3DR.%2BG.%26atitle%3DSynthesis%2520of%2520Oligonucleotide%2520Adducts%2520of%2520the%2520Bay%2520Region%2520Diol%2520Epoxide%2520Metabolites%2520of%2520Carcinogenic%2520Polycyclic%2520Aromatic%2520Hydrocarbons%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1995%26volume%3D60%26spage%3D5604%26epage%3D5613%26doi%3D10.1021%2Fjo00122a048" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kath, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jergic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heltzel, J. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loparo, J. J.</span></span> <span> </span><span class="NLM_article-title">Polymerase Exchange on Single DNA Molecules Reveals Processivity Clamp Control of Translesion Synthesis</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">7647</span>â <span class="NLM_lpage">7652</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1321076111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1073%2Fpnas.1321076111" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=111&amp;publication_year=2014&amp;pages=7647-7652&amp;author=J.+E.+Kathauthor=S.+Jergicauthor=J.+M.+H.+Heltzelauthor=D.+T.+Jacobauthor=N.+E.+Dixonauthor=M.+D.+Suttonauthor=G.+C.+Walkerauthor=J.+J.+Loparo&amp;title=Polymerase+Exchange+on+Single+DNA+Molecules+Reveals+Processivity+Clamp+Control+of+Translesion+Synthesis&amp;doi=10.1073%2Fpnas.1321076111" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1321076111&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1321076111%26sid%3Dliteratum%253Aachs%26aulast%3DKath%26aufirst%3DJ.%2BE.%26aulast%3DJergic%26aufirst%3DS.%26aulast%3DHeltzel%26aufirst%3DJ.%2BM.%2BH.%26aulast%3DJacob%26aufirst%3DD.%2BT.%26aulast%3DDixon%26aufirst%3DN.%2BE.%26aulast%3DSutton%26aufirst%3DM.%2BD.%26aulast%3DWalker%26aufirst%3DG.%2BC.%26aulast%3DLoparo%26aufirst%3DJ.%2BJ.%26atitle%3DPolymerase%2520Exchange%2520on%2520Single%2520DNA%2520Molecules%2520Reveals%2520Processivity%2520Clamp%2520Control%2520of%2520Translesion%2520Synthesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D7647%26epage%3D7652%26doi%3D10.1073%2Fpnas.1321076111" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghodke, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harikrishna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeepkumar, P. I.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Polymerase-Mediated Bypass Studies of the N2-Deoxyguanosine DNA Damage Caused by a Lucidin Analogue</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">2128</span>â <span class="NLM_lpage">2138</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jo502627b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo502627b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=80&amp;publication_year=2015&amp;pages=2128-2138&amp;author=P.+P.+Ghodkeauthor=S.+Harikrishnaauthor=P.+I.+Pradeepkumar&amp;title=Synthesis+and+Polymerase-Mediated+Bypass+Studies+of+the+N2-Deoxyguanosine+DNA+Damage+Caused+by+a+Lucidin+Analogue&amp;doi=10.1021%2Fjo502627b" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjo502627b&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo502627b%26sid%3Dliteratum%253Aachs%26aulast%3DGhodke%26aufirst%3DP.%2BP.%26aulast%3DHarikrishna%26aufirst%3DS.%26aulast%3DPradeepkumar%26aufirst%3DP.%2BI.%26atitle%3DSynthesis%2520and%2520Polymerase-Mediated%2520Bypass%2520Studies%2520of%2520the%2520N2-Deoxyguanosine%2520DNA%2520Damage%2520Caused%2520by%2520a%2520Lucidin%2520Analogue%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2015%26volume%3D80%26spage%3D2128%26epage%3D2138%26doi%3D10.1021%2Fjo502627b" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Champeil, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradhan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakshman, M. K.</span></span> <span> </span><span class="NLM_article-title">Palladium-Catalyzed Synthesis of Nucleoside Adducts from Bay- and Fjord-Region Diol Epoxides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">5035</span>â <span class="NLM_lpage">5045</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jo070204z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo070204z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt12hu7o%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=72&amp;publication_year=2007&amp;pages=5035-5045&amp;author=E.+Champeilauthor=P.+Pradhanauthor=M.+K.+Lakshman&amp;title=Palladium-Catalyzed+Synthesis+of+Nucleoside+Adducts+from+Bay-+and+Fjord-Region+Diol+Epoxides&amp;doi=10.1021%2Fjo070204z" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-Catalyzed Synthesis of Nucleoside Adducts from Bay- and Fjord-Region Diol Epoxides</span></div><div class="casAuthors">Champeil, Elise; Pradhan, Padmanava; Lakshman, Mahesh K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5035-5045</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Palladium-catalyzed C-N bond formation has been utilized to synthesize covalent 2'-deoxyadenosine (dA) and 2'-deoxyguanosine (dG) adducts of benzo[a]pyrene (BaP) series 1 (syn) and benzo[c]phenanthrene (BcPh) series 2 (anti) diol epoxides.  For this, (Â±)-10Î±-amino-7Î²,8Î±,9Î²-tris-benzoyl-oxy-7,8,9,10-tetrahydro (BaP) and (Â±)-1Î²-amino-2Î±,3Î±,4Î²-tris-benzoyl-oxy-1,2,3,4-tetrahydro (BcPh) were coupled with 6-halo-9-[3,5-bis-O-(tert-butyldimethylsilyl)-Î²-D-erythro-pentofuranosyl]purine and O6-benzyl-3',5'-bis-O-(tert-butyldimethylsilyl)-2-bromo-2'-deoxyinosine, using a (Â±)-BINAP-Pd complex and Cs2CO3.  For the synthesis of the dA adducts, both the 6-chloro- as well as the 6-bromopurine nucleoside derivs. were analyzed for the C-N coupling reaction with the hydrocarbon amino tri-benzoates.  With the BaP amino tri-benzoate, the 6-chloro-nucleoside provided satisfactory results, whereas the 6-bromo analog proved to be superior with the BcPh amino tri-benzoate.  Overall, lower yields of the dA adducts were obtained with the more hindered fjord-region BcPh amino tri-benzoate as compared to the bay-region BaP amino tri-benzoate.  In contrast to reactions leading to the dA adducts, the C-N reactions of both BaP and BcPh amino tri-benzoates with the 2-bromo-2'-deoxyinosine deriv. proceeded in comparable yields.  This seems to indicate that such Pd-catalyzed adduct forming reactions at the C-6 position may be influenced by steric constraints of the amine component, whereas those at the C-2 position are less sensitive.  Diastereomeric adduct pairs were sepd. and characterized by spectral methods and by comparisons to adducts produced by direct displacement reactions as well as those formed from DNA alkylation by diol epoxides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz2CwZQEPKzLVg90H21EOLACvtfcHk0ljdfXxjr2ev2w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXmt12hu7o%253D&amp;md5=99df3462308844fa54f7e921cf3bcf65</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjo070204z&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo070204z%26sid%3Dliteratum%253Aachs%26aulast%3DChampeil%26aufirst%3DE.%26aulast%3DPradhan%26aufirst%3DP.%26aulast%3DLakshman%26aufirst%3DM.%2BK.%26atitle%3DPalladium-Catalyzed%2520Synthesis%2520of%2520Nucleoside%2520Adducts%2520from%2520Bay-%2520and%2520Fjord-Region%2520Diol%2520Epoxides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2007%26volume%3D72%26spage%3D5035%26epage%3D5045%26doi%3D10.1021%2Fjo070204z" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Cu<sup>2+</sup> Fluorescent Sensor Based on Mesoporous Silica Nanosphere</span>. <i>Dyes Pigm.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">239</span>â <span class="NLM_lpage">246</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.dyepig.2012.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1016%2Fj.dyepig.2012.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsFGit70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=94&amp;publication_year=2012&amp;pages=239-246&amp;author=D.+Luauthor=J.+Leiauthor=Z.+Tianauthor=L.+Wangauthor=J.+Zhang&amp;title=Cu2%2B+Fluorescent+Sensor+Based+on+Mesoporous+Silica+Nanosphere&amp;doi=10.1016%2Fj.dyepig.2012.01.004" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Cu2+ fluorescent sensor based on mesoporous silica nanosphere</span></div><div class="casAuthors">Lu, Deli; Lei, Juying; Tian, Zhidan; Wang, Lingzhi; Zhang, Jinlong</div><div class="citationInfo"><span class="NLM_cas:title">Dyes and Pigments</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">239-246</span>CODEN:
                <span class="NLM_cas:coden">DYPIDX</span>;
        ISSN:<span class="NLM_cas:issn">0143-7208</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Monodisperse mesoporous silica nanosphere (MSN) modified with anthracene deriv., 2-(3-(triethoxysilyl) propylamino)-N-(9-anthrylMe) acetamide (SGAAn) was fabricated as a fluorescent sensor (SGAAn-MSN) for the detection of metal ions, which shows an exclusive selectivity to Cu2+.  Compared with SGAAn, SGAAn-MSN presents a higher sensitivity to Cu2+.  The loading amt. of SGAAn in MSN has a great influence on the detection sensitivity of SGAAn-MSN, which varies with the local concn. and accessibility of SGAAn in the pore channel of MSN.  Interference studies indicate that the addn. of other metal ions such as Ag+, Cd2+ and Pb2+ has a negligible effect on the selectivity of SGAAn-MSN to Cu2+.  The possible mechanism for the sensing behavior of SGAAn-MSN to Cu2+ is proposed based on the relation between fluorescence quench degree and Cu2+ concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopuymFCLOUWLVg90H21EOLACvtfcHk0ljdfXxjr2ev2w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XjsFGit70%253D&amp;md5=2a71800d6a25bd7f72c702c1940ee465</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.dyepig.2012.01.004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dyepig.2012.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DD.%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DCu2%252B%2520Fluorescent%2520Sensor%2520Based%2520on%2520Mesoporous%2520Silica%2520Nanosphere%26jtitle%3DDyes%2520Pigm.%26date%3D2012%26volume%3D94%26spage%3D239%26epage%3D246%26doi%3D10.1016%2Fj.dyepig.2012.01.004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abiraj, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gowda, D. C.</span></span> <span> </span><span class="NLM_article-title">Zinc/Ammonium Formate: A New Facile System for the Rapid and Selective Reduction of Oximes to Amines</span>. <i>J. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2003</i></span>,  <span class="NLM_fpage">332</span>â <span class="NLM_lpage">334</span>, <span class="refDoi">&nbsp;DOI: 10.3184/030823403103174281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.3184%2F030823403103174281" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2003&amp;publication_year=2003&amp;pages=332-334&amp;author=K.+Abirajauthor=D.+C.+Gowda&amp;title=Zinc%2FAmmonium+Formate%3A+A+New+Facile+System+for+the+Rapid+and+Selective+Reduction+of+Oximes+to+Amines&amp;doi=10.3184%2F030823403103174281" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.3184%2F030823403103174281&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.3184%252F030823403103174281%26sid%3Dliteratum%253Aachs%26aulast%3DAbiraj%26aufirst%3DK.%26aulast%3DGowda%26aufirst%3DD.%2BC.%26atitle%3DZinc%252FAmmonium%2520Formate%253A%2520A%2520New%2520Facile%2520System%2520for%2520the%2520Rapid%2520and%2520Selective%2520Reduction%2520of%2520Oximes%2520to%2520Amines%26jtitle%3DJ.%2520Chem.%2520Res.%26date%3D2003%26volume%3D2003%26spage%3D332%26epage%3D334%26doi%3D10.3184%2F030823403103174281" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, T.</span></span> <span> </span><span class="NLM_article-title">Selective Enzymatic Demethylation of <i>N</i><sup>2</sup>,<i>N</i><sup>2</sup>-Dimethylguanosine in RNA and Its Application in High-Throughput tRNA Sequencing</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5017</span>â <span class="NLM_lpage">5020</span>, <span class="refDoi">&nbsp;DOI: 10.1002/anie.201700537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1002%2Fanie.201700537" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=56&amp;publication_year=2017&amp;pages=5017-5020&amp;author=Q.+Daiauthor=G.+Zhengauthor=M.+H.+Schwartzauthor=W.+C.+Clarkauthor=T.+Pan&amp;title=Selective+Enzymatic+Demethylation+of+N2%2CN2-Dimethylguanosine+in+RNA+and+Its+Application+in+High-Throughput+tRNA+Sequencing&amp;doi=10.1002%2Fanie.201700537" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.201700537&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201700537%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DG.%26aulast%3DSchwartz%26aufirst%3DM.%2BH.%26aulast%3DClark%26aufirst%3DW.%2BC.%26aulast%3DPan%26aufirst%3DT.%26atitle%3DSelective%2520Enzymatic%2520Demethylation%2520of%2520N2%252CN2-Dimethylguanosine%2520in%2520RNA%2520and%2520Its%2520Application%2520in%2520High-Throughput%2520tRNA%2520Sequencing%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5017%26epage%3D5020%26doi%3D10.1002%2Fanie.201700537" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Himmelsbach, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trichtinger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charubala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfleiderer, W.</span></span> <span> </span><span class="NLM_article-title">The p-nitrophenylethyl (NPE) group: A versatile new blocking group for phosphate and aglycone protection in nucleosides and nucleotides</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">59</span>â <span class="NLM_lpage">72</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0040-4020(84)85103-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1016%2F0040-4020%2884%2985103-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADyaL2cXltFWgsLo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=40&amp;publication_year=1984&amp;pages=59-72&amp;author=F.+Himmelsbachauthor=B.+S.+Schulzauthor=T.+Trichtingerauthor=R.+Charubalaauthor=W.+Pfleiderer&amp;title=The+p-nitrophenylethyl+%28NPE%29+group%3A+A+versatile+new+blocking+group+for+phosphate+and+aglycone+protection+in+nucleosides+and+nucleotides&amp;doi=10.1016%2F0040-4020%2884%2985103-0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The p-nitrophenylethyl (NPE) group.  A versatile new blocking group for phosphate and aglycone protection in nucleosides and nucleotides</span></div><div class="casAuthors">Himmelsbach, Frank; Schulz, Bernd S.; Trichtinger, Thomas; Charubala, Ramamurthy; Pfleiderer, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">59-72</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    </div><div class="casAbstract">The syntheses of new monomeric building blocks for oligonucleotide synthesis via the phosphotriester approach contg. the p-nitrophenylethyl group for phosphate and aglycon protection are described.  Blocking of the amide function in guanosines at O6 can be achieved by the Mitsunobu reaction forming the corresponding O6-p-nitrophenylethyl derivs.  Sugar-protected thymidine and uridine have been alkylated at O4 in an SN1-type reaction by p-nitrophenylethyl iodide-Ag2CO3 in C6H6 to form the O4-p-nitrophenylethyl derivs.  Protection of the NH2 group in 2'-deoxycytidine and cytidine can be performed directly by 1-(p-nitrophenylethoxycarbonyl)benzotriazole in DMF to obtain the corresponding carbamates as a new type of N4-acylated cytidine deriv.  p-Nitrophenylethoxycarbonylation of the NH2 group in 2'-deoxyadenosine and adenosine requires previous sugar protection by acyl or silyl groups and can then be achieved by p-nitrophenylethyl chloroformate or better by 1-methyl-3-p-nitrophenylethoxycarbonylimidazolium chloride to form N6-p-nitrophenylethoxycarbonyladenosines.  The various p-nitrophenylethyl blocking groups are stable under mild hydrolytic conditions (e.g., NH3 and Et3N) but can be cleaved selectively by DBU or DBN in aprotic solvents.  5'-O-Monomethoxytritylation as well as phosphorylation at the 3'-OH group can be effected to give the corresponding 3'-(2,5-dichlorophenyl p-nitrophenylethyl) phosphotriesters also in high yields.  Oximate cleavage of the latter compds. to the phosphodiesters and detritylation to the 5'-unblocked phosphotriesters do not affect the aglycon protecting groups, thereby demonstrating their general versatility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraD2QuKCEtrbVg90H21EOLACvtfcHk0lgRX38lotE78g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL2cXltFWgsLo%253D&amp;md5=eab017017027631de697686f0c76bdaf</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2F0040-4020%2884%2985103-0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0040-4020%252884%252985103-0%26sid%3Dliteratum%253Aachs%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DSchulz%26aufirst%3DB.%2BS.%26aulast%3DTrichtinger%26aufirst%3DT.%26aulast%3DCharubala%26aufirst%3DR.%26aulast%3DPfleiderer%26aufirst%3DW.%26atitle%3DThe%2520p-nitrophenylethyl%2520%2528NPE%2529%2520group%253A%2520A%2520versatile%2520new%2520blocking%2520group%2520for%2520phosphate%2520and%2520aglycone%2520protection%2520in%2520nucleosides%2520and%2520nucleotides%26jtitle%3DTetrahedron%26date%3D1984%26volume%3D40%26spage%3D59%26epage%3D72%26doi%3D10.1016%2F0040-4020%2884%2985103-0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kottur, J.</span>; <span class="NLM_string-name">Ghodke, P. P.</span>; <span class="NLM_string-name">Nair, D. T.</span>; <span class="NLM_string-name">Pradeepkumar, P. I.</span></span> Manuscript in preparation.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=Kottur%2C+J.%3B+Ghodke%2C+P.+P.%3B+Nair%2C+D.+T.%3B+Pradeepkumar%2C+P.+I.+Manuscript+in+preparation." target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DKottur%26aufirst%3DJ." title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixit, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trejo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juliano, R. L.</span></span> <span> </span><span class="NLM_article-title">Intracellular Delivery of an Anionic Antisense Oligonucleotide via Receptor-Mediated Endocytosis</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2764</span>â <span class="NLM_lpage">2776</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gkn115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1093%2Fnar%2Fgkn115" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=2008&amp;pages=2764-2776&amp;author=R.+Alamauthor=V.+Dixitauthor=H.+Kangauthor=Z.+B.+Liauthor=X.+Chenauthor=J.+Trejoauthor=M.+Fisherauthor=R.+L.+Juliano&amp;title=Intracellular+Delivery+of+an+Anionic+Antisense+Oligonucleotide+via+Receptor-Mediated+Endocytosis&amp;doi=10.1093%2Fnar%2Fgkn115" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn115&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn115%26sid%3Dliteratum%253Aachs%26aulast%3DAlam%26aufirst%3DR.%26aulast%3DDixit%26aufirst%3DV.%26aulast%3DKang%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DZ.%2BB.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DTrejo%26aufirst%3DJ.%26aulast%3DFisher%26aufirst%3DM.%26aulast%3DJuliano%26aufirst%3DR.%2BL.%26atitle%3DIntracellular%2520Delivery%2520of%2520an%2520Anionic%2520Antisense%2520Oligonucleotide%2520via%2520Receptor-Mediated%2520Endocytosis%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D2764%26epage%3D2776%26doi%3D10.1093%2Fnar%2Fgkn115" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sholder, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creech, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loechler, E. L.</span></span> <span> </span><span class="NLM_article-title">How Y-Family DNA Polymerase IV Is More Accurate than Dpo4 at dCTP Insertion Opposite an <i>N</i><sup>2</sup>-dG Adduct of Benzo[<i>a</i>]Pyrene</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">144</span>â <span class="NLM_lpage">153</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.dnarep.2015.09.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1016%2Fj.dnarep.2015.09.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=26523515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKqsLfF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=35&amp;publication_year=2015&amp;pages=144-153&amp;author=G.+Sholderauthor=A.+Creechauthor=E.+L.+Loechler&amp;title=How+Y-Family+DNA+Polymerase+IV+Is+More+Accurate+than+Dpo4+at+dCTP+Insertion+Opposite+an+N2-dG+Adduct+of+Benzo%5Ba%5DPyrene&amp;doi=10.1016%2Fj.dnarep.2015.09.020" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">How Y-Family DNA polymerase IV is more accurate than Dpo4 at dCTP insertion opposite an N2-dG adduct of benzo[a]pyrene</span></div><div class="casAuthors">Sholder, Gabriel; Creech, Amanda; Loechler, Edward L.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">144-153</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">To bypass DNA damage, cells have Y-Family DNA polymerases (DNAPs).  One Y-Family-class includes DNAP Îº and DNAP IV, which accurately insert dCTP opposite N2-dG adducts, including from the carcinogen benzo[a]pyrene (BP).  Another class includes DNAP Î· and DNAP V, which insert accurately opposite UV-damage, but inaccurately opposite BP-N2-dG.  To investigate structural differences between Y-Family-classes, regions are swapped between DNAP IV (a Îº/IV-class-member) and Dpo4 (a Î·/V-class-member); the kinetic consequences are evaluated via primer-extension studies with a BP-N2-dG-contg. template.  Four key structural elements are revealed.  (1) Y-Family DNAPs have discreet non-covalent contacts between their little finger-domain (LF-Domain) and their catalytic core-domain (CC-Domain), which we call "non-covalent bridges" (NCBs).  Arg37 and Arg38 in DNAP IV's CC-Domain near the active site form a non-covalent bridge (AS-NCB) by interacting with Glu251 and Asp252, resp., in DNAP IV's LF-Domain.  Without these interactions dATP/dGTP/dTTP misinsertions increase.  DNAP IV's AS-NCB suppresses misinsertions better than Dpo4's equiv. AS-NCB.  (2) DNAP IV also suppresses dATP/dGTP/dTTP misinsertions via a second non-covalent bridge, which is â¼8 Ã from the active site (Distal-NCB).  Dpo4 has no Distal-NCB, rendering it inferior at dATP/dGTP/dTTP suppression.  (3) dCTP insertion is facilitated by the larger minor groove opening near the active site in DNAP IV vs. Dpo4, which is sensible given that Watson/Crick-like [dCTP:BP-N2-dG] pairing requires the BP-moiety to be in the minor groove.  (4) Compared to Dpo4, DNAP IV has a smaller major groove opening, which suppresses dGTP misinsertion, implying BP-N2-dG bulk in the major groove during Hoogsteen syn-adduct-dG:dGTP pairing.  In summary, DNAP IV has a large minor groove opening to enhance dCTP insertion, a plugged major groove opening to suppress dGTP misinsertion, and two non-covalent bridges (near and distal to the active site) to suppress dATP/dGTP/dTTP misinsertions; collectively these four structural features enhance DNAP IV's dNTP insertion fidelity opposite a BP-N2-dG adduct compared to Dpo4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK6l92GpV4m7Vg90H21EOLACvtfcHk0lgRX38lotE78g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKqsLfF&amp;md5=5a432c52b2c362b4b003d35c2b3d366c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2015.09.020&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2015.09.020%26sid%3Dliteratum%253Aachs%26aulast%3DSholder%26aufirst%3DG.%26aulast%3DCreech%26aufirst%3DA.%26aulast%3DLoechler%26aufirst%3DE.%2BL.%26atitle%3DHow%2520Y-Family%2520DNA%2520Polymerase%2520IV%2520Is%2520More%2520Accurate%2520than%2520Dpo4%2520at%2520dCTP%2520Insertion%2520Opposite%2520an%2520N2-dG%2520Adduct%2520of%2520Benzo%255Ba%255DPyrene%26jtitle%3DDNA%2520Repair%26date%3D2015%26volume%3D35%26spage%3D144%26epage%3D153%26doi%3D10.1016%2Fj.dnarep.2015.09.020" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghodke, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gore, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harikrishna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kottur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeepkumar, P. I.</span></span> <span> </span><span class="NLM_article-title">The N2-Furfuryl-Deoxyguanosine Adduct Does Not Alter the Structure of B-DNA</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">502</span>â <span class="NLM_lpage">511</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.joc.5b02341</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.joc.5b02341" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=81&amp;publication_year=2016&amp;pages=502-511&amp;author=P.+P.+Ghodkeauthor=K.+R.+Goreauthor=S.+Harikrishnaauthor=B.+Samantaauthor=J.+Kotturauthor=D.+T.+Nairauthor=P.+I.+Pradeepkumar&amp;title=The+N2-Furfuryl-Deoxyguanosine+Adduct+Does+Not+Alter+the+Structure+of+B-DNA&amp;doi=10.1021%2Facs.joc.5b02341" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.5b02341&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.5b02341%26sid%3Dliteratum%253Aachs%26aulast%3DGhodke%26aufirst%3DP.%2BP.%26aulast%3DGore%26aufirst%3DK.%2BR.%26aulast%3DHarikrishna%26aufirst%3DS.%26aulast%3DSamanta%26aufirst%3DB.%26aulast%3DKottur%26aufirst%3DJ.%26aulast%3DNair%26aufirst%3DD.%2BT.%26aulast%3DPradeepkumar%26aufirst%3DP.%2BI.%26atitle%3DThe%2520N2-Furfuryl-Deoxyguanosine%2520Adduct%2520Does%2520Not%2520Alter%2520the%2520Structure%2520of%2520B-DNA%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2016%26volume%3D81%26spage%3D502%26epage%3D511%26doi%3D10.1021%2Facs.joc.5b02341" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanner, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span> <span> </span><span class="NLM_article-title">Clustering Molecular Dynamics Trajectories: 1. Characterizing the Performance of Different Clustering Algorithms</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">2312</span>â <span class="NLM_lpage">2334</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct700119m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700119m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFaqsLzE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2007&amp;pages=2312-2334&amp;author=J.+Shaoauthor=S.+W.+Tannerauthor=N.+Thompsonauthor=T.+E.+Cheatham&amp;title=Clustering+Molecular+Dynamics+Trajectories%3A+1.+Characterizing+the+Performance+of+Different+Clustering+Algorithms&amp;doi=10.1021%2Fct700119m" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Clustering Molecular Dynamics Trajectories: 1. Characterizing the Performance of Different Clustering Algorithms</span></div><div class="casAuthors">Shao, Jianyin; Tanner, Stephen W.; Thompson, Nephi; Cheatham, Thomas E., III</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2334</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics simulation methods produce trajectories of at. positions (and optionally velocities and energies) as a function of time and provide a representation of the sampling of a given mol.'s energetically accessible conformational ensemble.  As simulations on the 10-100 ns time scale become routine, with sampled configurations stored on the picosecond time scale, such trajectories contain large amts. of data.  Data-mining techniques, like clustering, provide one means to group and make sense of the information in the trajectory.  In this work, several clustering algorithms were implemented, compared, and utilized to understand MD trajectory data.  The development of the algorithms into a freely available C code library, and their application to a simple test example of random (or systematically placed) points in a 2D plane (where the pairwise metric is the distance between points) provide a means to understand the relative performance.  Eleven different clustering algorithms were developed, ranging from top-down splitting (hierarchical) and bottom-up aggregating (including single-linkage edge joining, centroid-linkage, av.-linkage, complete-linkage, centripetal, and centripetal-complete) to various refinement (means, Bayesian, and self-organizing maps) and tree (COBWEB) algorithms.  Systematic testing in the context of MD simulation of various DNA systems (including DNA single strands and the interaction of a minor groove binding drug DB226 with a DNA hairpin) allows a more direct assessment of the relative merits of the distinct clustering algorithms.  Addnl., means to assess the relative performance and differences between the algorithms, to dynamically select the initial cluster count, and to achieve faster data mining by "sieved clustering" were evaluated.  Overall, it was found that there is no one perfect "one size fits all" algorithm for clustering MD trajectories and that the results strongly depend on the choice of atoms for the pairwise comparison.  Some algorithms tend to produce homogeneously sized clusters, whereas others have a tendency to produce singleton clusters.  Issues related to the choice of a pairwise metric, clustering metrics, which atom selection is used for the comparison, and about the relative performance are discussed.  Overall, the best performance was obsd. with the av.-linkage, means, and SOM algorithms.  If the cluster count is not known in advance, the hierarchical or av.-linkage clustering algorithms are recommended.  Although these algorithms perform well, it is important to be aware of the limitations or weaknesses of each algorithm, specifically the high sensitivity to outliers with hierarchical, the tendency to generate homogenously sized clusters with means, and the tendency to produce small or singleton clusters with av.-linkage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzr3sLW2dkQbVg90H21EOLACvtfcHk0liCXW5i3t2CKA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXhtFaqsLzE&amp;md5=668cff11c29736325a4788379cc43def</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fct700119m&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700119m%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DTanner%26aufirst%3DS.%2BW.%26aulast%3DThompson%26aufirst%3DN.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26atitle%3DClustering%2520Molecular%2520Dynamics%2520Trajectories%253A%25201.%2520Characterizing%2520the%2520Performance%2520of%2520Different%2520Clustering%2520Algorithms%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2007%26volume%3D3%26spage%3D2312%26epage%3D2334%26doi%3D10.1021%2Fct700119m" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roe, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span> <span> </span><span class="NLM_article-title">PTRAJ and CPPTRAJ: Software for Processing and Analysis of Molecular Dynamics Trajectory Data</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3084</span>â <span class="NLM_lpage">3095</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct400341p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400341p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2013&amp;pages=3084-3095&amp;author=D.+R.+Roeauthor=T.+E.+Cheatham&amp;title=PTRAJ+and+CPPTRAJ%3A+Software+for+Processing+and+Analysis+of+Molecular+Dynamics+Trajectory+Data&amp;doi=10.1021%2Fct400341p" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">PTRAJ and CPPTRAJ: Software for Processing and Analysis of Molecular Dynamics Trajectory Data</span></div><div class="casAuthors">Roe, Daniel R.; Cheatham, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3084-3095</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe PTRAJ and its successor CPPTRAJ, two complementary, portable, and freely available computer programs for the anal. and processing of time series of three-dimensional at. positions (i.e., coordinate trajectories) and the data therein derived.  Common tools include the ability to manipulate the data to convert among trajectory formats, process groups of trajectories generated with ensemble methods (e.g., replica exchange mol. dynamics), image with periodic boundary conditions, create av. structures, strip subsets of the system, and perform calcns. such as RMS fitting, measuring distances, B-factors, radii of gyration, radial distribution functions, and time correlations, among other actions and analyses.  Both the PTRAJ and CPPTRAJ programs and source code are freely available under the GNU General Public License version 3 and are currently distributed within the AmberTools 12 suite of support programs that make up part of the Amber package of computer programs (see http://ambermd.org).  This overview describes the general design, features, and history of these two programs, as well as algorithmic improvements and new features available in CPPTRAJ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq97UsRfUy65bVg90H21EOLACvtfcHk0liCXW5i3t2CKA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D&amp;md5=6f1bee934f13f180bd7e1feb6b78036d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fct400341p&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400341p%26sid%3Dliteratum%253Aachs%26aulast%3DRoe%26aufirst%3DD.%2BR.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26atitle%3DPTRAJ%2520and%2520CPPTRAJ%253A%2520Software%2520for%2520Processing%2520and%2520Analysis%2520of%2520Molecular%2520Dynamics%2520Trajectory%2520Data%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3084%26epage%3D3095%26doi%3D10.1021%2Fct400341p" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span>; <span class="NLM_string-name">Babin, V.</span>; <span class="NLM_string-name">Berryman, J. T.</span>; <span class="NLM_string-name">Betz, R. M.</span>; <span class="NLM_string-name">Cai, Q.</span>; <span class="NLM_string-name">Cerutti, D. S.</span>; <span class="NLM_string-name">Cheatham, T. E.,  III</span>; <span class="NLM_string-name">Darden, T. A.</span>; <span class="NLM_string-name">Duke, R. E.</span>; <span class="NLM_string-name">Gohlke, H.</span>; <span class="NLM_string-name">Goetz, A. W.</span>; <span class="NLM_string-name">Gusarov, S.</span>; <span class="NLM_string-name">Homeyer, N.</span>; <span class="NLM_string-name">Janowski, P.</span>; <span class="NLM_string-name">Kaus, J.</span>; <span class="NLM_string-name">KolossvÃ¡ry, I.</span>; <span class="NLM_string-name">Kovalenko, A.</span>; <span class="NLM_string-name">Lee, T. S.</span>; <span class="NLM_string-name">LeGrand, S.</span>; <span class="NLM_string-name">Luchko, T.</span>; <span class="NLM_string-name">Luo, R. B.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Kollaman, P. A.</span></span> <i>AMBER 14</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco, CA</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2014&amp;author=D.+A.+Case&amp;author=V.+Babin&amp;author=J.+T.+Berryman&amp;author=R.+M.+Betz&amp;author=Q.+Cai&amp;author=D.+S.+Cerutti&amp;author=T.+E.+Cheatham&amp;author=T.+A.+Darden&amp;author=R.+E.+Duke&amp;author=H.+Gohlke&amp;author=A.+W.+Goetz&amp;author=S.+Gusarov&amp;author=N.+Homeyer&amp;author=P.+Janowski&amp;author=J.+Kaus&amp;author=I.+Kolossv%C3%A1ry&amp;author=A.+Kovalenko&amp;author=T.+S.+Lee&amp;author=S.+LeGrand&amp;author=T.+Luchko&amp;author=R.+B.+Luo&amp;author=X.+Wu&amp;author=P.+A.+Kollaman&amp;title=AMBER+14" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26btitle%3DAMBER%252014%26pub%3DUniversity%2520of%2520California%26date%3D2014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brandl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jabs, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">SÃ¼hnel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">C-HÂ·Â·Â·Ï-Interactions in Proteins</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>307</i></span>,  <span class="NLM_fpage">357</span>â <span class="NLM_lpage">377</span>, <span class="refDoi">&nbsp;DOI: 10.1006/jmbi.2000.4473</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1006%2Fjmbi.2000.4473" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=11243825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhslSnsb4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=307&amp;publication_year=2001&amp;pages=357-377&amp;author=M.+Brandlauthor=M.+S.+Weissauthor=A.+Jabsauthor=J.+S%C3%BChnelauthor=R.+Hilgenfeld&amp;title=C-H%C2%B7%C2%B7%C2%B7%CF%80-Interactions+in+Proteins&amp;doi=10.1006%2Fjmbi.2000.4473" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">C-HÂ·Â·Â·Ï-Interactions in Proteins</span></div><div class="casAuthors">Brandl, Maria; Weiss, Manfred S.; Jabs, Andreas; Suhnel, Jurgen; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">307</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">357-377</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A non-redundant set of 1154 protein structures from the Protein Data Bank was examd. with respect to close interactions between C-H-donor and Ï-acceptor groups.  A total of 31,087 interactions were found to satisfy our selection criteria.  Their geometric parameters suggest that these interactions can be classified as weak hydrogen bonds.  A set of 12 interaction classes were defined based on the division of the donors into three groups and the acceptors into four groups.  These classes were examd. sep., and the resp. interactions described in detail in each class.  Most prominent were interactions between aliph. C-H donors and arom. Ï-acceptors and interactions between arom. C-H donors and arom. Ï-acceptors.  About three-quarters of the Trp-rings, half of all Phe and Tyr-rings and a quarter of all His-rings were found to be involved as acceptors in C-HÂ·Â·Â·Ï-interactions.  On the donor side, a preference for arom. C-H groups was obsd., but also for the aliph. side-chains of the long, extended amino acid residues Lys, Arg and Met, and the Pro ring.  The av. distance between the C-donor and the center-of-mass of the Ï-acceptor was obsd. to be significantly longer in the 174 protein structures detd. at &gt;2.5 Ã resoln.  Also, the distribution is significantly wider.  This resoln. dependence suggests that the force fields commonly used for the refinement of protein structures may not be adequate.  C-HÂ·Â·Â·Ï-interactions involving arom. groups either as donor or as acceptor groups are found mostly in the interior of the protein.  The more hydrophilic the participating groups are, the closer to the surface are the interactions located.  About 40 % of all C-HÂ·Â·Â·Ï-interactions occur between amino acid residue side-chains that are sepd. by nine or less residues in sequence.  Dependent on the interaction class, different preferences for secondary structure, residue type and side-chain conformation were obsd.  It is likely that the C-HÂ·Â·Â·Ï-interactions contribute significantly to the overall stability of a protein.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFu6opUqoQ67Vg90H21EOLACvtfcHk0liCXW5i3t2CKA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3MXhslSnsb4%253D&amp;md5=0aaa18f920eaafb06283b8a88db5c072</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2000.4473&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2000.4473%26sid%3Dliteratum%253Aachs%26aulast%3DBrandl%26aufirst%3DM.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DJabs%26aufirst%3DA.%26aulast%3DS%25C3%25BChnel%26aufirst%3DJ.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DC-H%25C2%25B7%25C2%25B7%25C2%25B7%25CF%2580-Interactions%2520in%2520Proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2001%26volume%3D307%26spage%3D357%26epage%3D377%26doi%3D10.1006%2Fjmbi.2000.4473" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kottur, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, D. T.</span></span> <span> </span><span class="NLM_article-title">A Strategically Located Serine Residue Is Critical for the Mutator Activity of DNA Polymerase IV from Escherichia Coli</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">5104</span>â <span class="NLM_lpage">5114</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gkt146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1093%2Fnar%2Fgkt146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=23525461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVyjsr0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=41&amp;publication_year=2013&amp;pages=5104-5114&amp;author=A.+Sharmaauthor=J.+Kotturauthor=N.+Narayananauthor=D.+T.+Nair&amp;title=A+Strategically+Located+Serine+Residue+Is+Critical+for+the+Mutator+Activity+of+DNA+Polymerase+IV+from+Escherichia+Coli&amp;doi=10.1093%2Fnar%2Fgkt146" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A strategically located serine residue is critical for the mutator activity of DNA polymerase IV from Escherichia coli</span></div><div class="casAuthors">Sharma, Amit; Kottur, Jithesh; Narayanan, Naveen; Nair, Deepak T.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">5104-5114</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Y-family DNA polymerase IV or Pol IV (Escherichia coli) is the founding member of the DinB family and is known to play an important role in stress-induced mutagenesis.  We have detd. four crystal structures of this enzyme in its pre-catalytic state in complex with substrate DNA presenting the four possible template nucleotides that are paired with the corresponding incoming nucleotide triphosphates.  In all four structures, the Ser42 residue in the active site forms interactions with the base moieties of the incipient Watson-Crick base pair.  This residue is located close to the center of the nascent base pair towards the minor groove.  In vitro and in vivo assays show that the fidelity of the Pol IV enzyme increases drastically when this Ser residue was mutated to Ala.  In addn., the structure of Pol IV with the mismatch A:C in the active site shows that the Ser42 residue plays an important role in stabilizing dCTP in a conformation compatible with catalysis.  Overall, the structural, biochem. and functional data presented here show that the Ser42 residue is present at a strategic location to stabilize mismatches in the Pol IV active site, and thus facilitate the appearance of transition and transversion mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof6J51geZVq7Vg90H21EOLACvtfcHk0liA7uZXd9Joig"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXntVyjsr0%253D&amp;md5=5790994ef6713be7e379ad73899c7d5f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt146&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt146%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DKottur%26aufirst%3DJ.%26aulast%3DNarayanan%26aufirst%3DN.%26aulast%3DNair%26aufirst%3DD.%2BT.%26atitle%3DA%2520Strategically%2520Located%2520Serine%2520Residue%2520Is%2520Critical%2520for%2520the%2520Mutator%2520Activity%2520of%2520DNA%2520Polymerase%2520IV%2520from%2520Escherichia%2520Coli%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D5104%26epage%3D5114%26doi%3D10.1093%2Fnar%2Fgkt146" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanquelef, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquant, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klimerak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delepine, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupradeau, F. Y.</span></span> <span> </span><span class="NLM_article-title">R.E.D. Server: A Web Service for Deriving RESP and ESP Charges and Building Force Field Libraries for New Molecules and Molecular Fragments</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">W511</span>â <span class="NLM_lpage">7</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gkr288</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1093%2Fnar%2Fgkr288" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=21609950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosVOmsbY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=2011&amp;pages=W511-7&amp;author=E.+Vanquelefauthor=S.+Simonauthor=G.+Marquantauthor=E.+Garciaauthor=G.+Klimerakauthor=J.+C.+Delepineauthor=P.+Cieplakauthor=F.+Y.+Dupradeau&amp;title=R.E.D.+Server%3A+A+Web+Service+for+Deriving+RESP+and+ESP+Charges+and+Building+Force+Field+Libraries+for+New+Molecules+and+Molecular+Fragments&amp;doi=10.1093%2Fnar%2Fgkr288" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">R.E.D. Server: a web service for deriving RESP and ESP charges and building force field libraries for new molecules and molecular fragments</span></div><div class="casAuthors">Vanquelef, Enguerran; Simon, Sabrina; Marquant, Gaelle; Garcia, Elodie; Klimerak, Geoffroy; Delepine, Jean Charles; Cieplak, Piotr; Dupradeau, Francois-Yves</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W511-W517</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">R.E.D. Server is a unique, open web service, designed to derive non-polarizable RESP and ESP charges and to build force field libraries for new mols./mol. fragments.  It provides to computational biologists the means to derive rigorously mol. electrostatic potential-based charges embedded in force field libraries that are ready to be used in force field development, charge validation and mol. dynamics simulations.  R.E.D. Server interfaces quantum mechanics programs, the RESP program and the latest version of the R.E.D. tools.  A two step approach has been developed.  The first one consists of prepg. P2N file(s) to rigorously define key elements such as atom names, topol. and chem. equivalencing needed when building a force field library.  Then, P2N files are used to derive RESP or ESP charges embedded in force field libraries in the Tripos mol2 format.  In complex cases an entire set of force field libraries or force field topol. database is generated.  Other features developed in R.E.D.  Server include help services, a demonstration, tutorials, frequently asked questions, Jmol-based tools useful to construct PDB input files and parse R.E.D.  Server outputs as well as a graphical queuing system allowing any user to check the status of R.E.D. Server jobs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEOBBuQu4rcbVg90H21EOLACvtfcHk0liA7uZXd9Joig"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXosVOmsbY%253D&amp;md5=1d224a2941d8634b6ed2159530d9e529</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr288&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr288%26sid%3Dliteratum%253Aachs%26aulast%3DVanquelef%26aufirst%3DE.%26aulast%3DSimon%26aufirst%3DS.%26aulast%3DMarquant%26aufirst%3DG.%26aulast%3DGarcia%26aufirst%3DE.%26aulast%3DKlimerak%26aufirst%3DG.%26aulast%3DDelepine%26aufirst%3DJ.%2BC.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DDupradeau%26aufirst%3DF.%2BY.%26atitle%3DR.E.D.%2520Server%253A%2520A%2520Web%2520Service%2520for%2520Deriving%2520RESP%2520and%2520ESP%2520Charges%2520and%2520Building%2520Force%2520Field%2520Libraries%2520for%2520New%2520Molecules%2520and%2520Molecular%2520Fragments%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DW511%26epage%3D7%26doi%3D10.1093%2Fnar%2Fgkr288" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dupradeau, F.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pigache, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaffran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savineau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grivel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosanski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span> <span> </span><span class="NLM_article-title">The R.E.D. Tools: Advances in RESP and ESP Charge Derivation and Force Field Library Building</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">7821</span>, <span class="refDoi">&nbsp;DOI: 10.1039/c0cp00111b</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1039%2Fc0cp00111b" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=20574571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFaqtrk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2010&amp;pages=7821&amp;author=F.-Y.+Dupradeauauthor=A.+Pigacheauthor=T.+Zaffranauthor=C.+Savineauauthor=R.+Lelongauthor=N.+Grivelauthor=D.+Lelongauthor=W.+Rosanskiauthor=P.+Cieplak&amp;title=The+R.E.D.+Tools%3A+Advances+in+RESP+and+ESP+Charge+Derivation+and+Force+Field+Library+Building&amp;doi=10.1039%2Fc0cp00111b" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The R.E.D. tools: advances in RESP and ESP charge derivation and force field library building</span></div><div class="casAuthors">Dupradeau, Francois-Yves; Pigache, Adrien; Zaffran, Thomas; Savineau, Corentin; Lelong, Rodolphe; Grivel, Nicolas; Lelong, Dimitri; Rosanski, Wilfried; Cieplak, Piotr</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">7821-7839</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Deriving at. charges and building a force field library for a new mol. are key steps when developing a force field required for conducting structural and energy-based anal. using mol. mechanics.  Derivation of popular RESP charges for a set of residues is a complex and error prone procedure because it depends on numerous input parameters.  To overcome these problems, the R.E.D.  Tools (RESP and ESP charge Derive, http://q4md-forcefieldtools.org/RED/) have been developed to perform charge derivation in an automatic and straightforward way.  The R.E.D. program handles chem. elements up to bromine in the periodic table.  It interfaces different quantum mech. programs employed for geometry optimization and computing mol. electrostatic potential(s), and performs charge fitting using the RESP program.  By defining tight optimization criteria and by controlling the mol. orientation of each optimized geometry, charge values are reproduced at any computer platform with an accuracy of 0.0001 e.  The charges can be fitted using multiple conformations, making them suitable for mol. dynamics simulations.  R.E.D. allows also for defining charge constraints during multiple mol. charge fitting, which are used to derive charges for mol. fragments.  Finally, R.E.D. incorporates charges into a force field library, readily usable in mol. dynamics computer packages.  For complex cases, such as a set of homologous mols. belonging to a common family, an entire force field topol. database is generated.  Currently, the at. charges and force field libraries have been developed for more than fifty model systems and stored in the RESP ESP charge DDataBase.  Selected results related to non-polarizable charge models are presented and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxiXPn3YLNM7Vg90H21EOLACvtfcHk0liA7uZXd9Joig"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXosFaqtrk%253D&amp;md5=845cc10ea7ed2f0222361db82bd80e39</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1039%2Fc0cp00111b&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cp00111b%26sid%3Dliteratum%253Aachs%26aulast%3DDupradeau%26aufirst%3DF.-Y.%26aulast%3DPigache%26aufirst%3DA.%26aulast%3DZaffran%26aufirst%3DT.%26aulast%3DSavineau%26aufirst%3DC.%26aulast%3DLelong%26aufirst%3DR.%26aulast%3DGrivel%26aufirst%3DN.%26aulast%3DLelong%26aufirst%3DD.%26aulast%3DRosanski%26aufirst%3DW.%26aulast%3DCieplak%26aufirst%3DP.%26atitle%3DThe%2520R.E.D.%2520Tools%253A%2520Advances%2520in%2520RESP%2520and%2520ESP%2520Charge%2520Derivation%2520and%2520Force%2520Field%2520Library%2520Building%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2010%26volume%3D12%26spage%3D7821%26doi%3D10.1039%2Fc0cp00111b" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornell, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P.</span></span> <span> </span><span class="NLM_article-title">A Well-Behaved Electrostatic Potential Based Method Using Charge Restraints for Deriving Atomic</span>. <i>J. Phys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">10269</span>â <span class="NLM_lpage">10280</span>, <span class="refDoi">&nbsp;DOI: 10.1021/j100142a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/j100142a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=97&amp;publication_year=1993&amp;pages=10269-10280&amp;author=C.+Baylyauthor=P.+Cieplakauthor=W.+Cornellauthor=P.+Kollman&amp;title=A+Well-Behaved+Electrostatic+Potential+Based+Method+Using+Charge+Restraints+for+Deriving+Atomic&amp;doi=10.1021%2Fj100142a004" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fj100142a004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100142a004%26sid%3Dliteratum%253Aachs%26aulast%3DBayly%26aufirst%3DC.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DCornell%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%26atitle%3DA%2520Well-Behaved%2520Electrostatic%2520Potential%2520Based%2520Method%2520Using%2520Charge%2520Restraints%2520for%2520Deriving%2520Atomic%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1993%26volume%3D97%26spage%3D10269%26epage%3D10280%26doi%3D10.1021%2Fj100142a004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J.</span>; <span class="NLM_string-name">Trucks, G. W.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Petersson, G. A.</span>; <span class="NLM_string-name">Nakatsuji, H.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Caricato, M.</span>; <span class="NLM_string-name">Marenich, A.</span>; <span class="NLM_string-name">Bloino, J.</span>; <span class="NLM_string-name">Janesko, B. G.</span>; <span class="NLM_string-name">Gomperts, R.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Hratchian, H. P.</span>; <span class="NLM_string-name">Ortiz, J. V.</span>; <span class="NLM_string-name">Izmaylov, A. F.</span>; <span class="NLM_string-name">Sonnenberg, J. L.</span>; <span class="NLM_string-name">Williams-Young, D.</span>; <span class="NLM_string-name">Ding, F.</span>; <span class="NLM_string-name">Lipparini, F.</span>; <span class="NLM_string-name">Egidi, F.</span>; <span class="NLM_string-name">Goings, J.</span>; <span class="NLM_string-name">Peng, B.</span>; <span class="NLM_string-name">Petrone, A.</span>; <span class="NLM_string-name">Henderson, T.</span>; <span class="NLM_string-name">Ranasinghe, D.</span>; <span class="NLM_string-name">Zakrzewski, V. G.</span>; <span class="NLM_string-name">Gao, J.</span>; <span class="NLM_string-name">Rega, N.</span>; <span class="NLM_string-name">Zheng, G.</span>; <span class="NLM_string-name">Liang, W.</span>; <span class="NLM_string-name">Hada, M.</span>; <span class="NLM_string-name">Ehara, M.</span>; <span class="NLM_string-name">Toyota, K.</span>; <span class="NLM_string-name">Fukuda, R.</span>; <span class="NLM_string-name">Hasegawa, J.</span>; <span class="NLM_string-name">Ishida, M.</span>; <span class="NLM_string-name">Nakajima, T.</span>; <span class="NLM_string-name">Honda, Y.</span>; <span class="NLM_string-name">Kitao, O.</span>; <span class="NLM_string-name">Nakai, H.</span>; <span class="NLM_string-name">Vreven, T.</span>; <span class="NLM_string-name">Throssell, K.</span>; <span class="NLM_string-name">Montgomery, J. A.,  Jr.</span>; <span class="NLM_string-name">Peralta, P. E.</span>; <span class="NLM_string-name">Ogliaro, F.</span>; <span class="NLM_string-name">Bearpark, M.</span>; <span class="NLM_string-name">Heyd, J.
J.</span>; <span class="NLM_string-name">Brothers, E.</span>; <span class="NLM_string-name">Kudin, K.
N.</span>; <span class="NLM_string-name">Staroverov, V. N.</span>; <span class="NLM_string-name">Keith, T.</span>; <span class="NLM_string-name">Kobayashi, R.</span>; <span class="NLM_string-name">Normand, J.</span>; <span class="NLM_string-name">Raghavachari, K.</span>; <span class="NLM_string-name">Rendell, A.</span>; <span class="NLM_string-name">Burant, J. C.</span>; <span class="NLM_string-name">Iyengar, S. S.</span>; <span class="NLM_string-name">Tomasi, J.</span>; <span class="NLM_string-name">Cossi, M.</span>; <span class="NLM_string-name">Millam, J. M.</span>; <span class="NLM_string-name">Klene, M.</span>; <span class="NLM_string-name">Adamo, C.</span>; <span class="NLM_string-name">Cammi, R.</span>; <span class="NLM_string-name">Ochterski, J. W.</span>; <span class="NLM_string-name">Martin, R. L.</span>; <span class="NLM_string-name">Morokuma, K.</span>; <span class="NLM_string-name">Farkas, Ã.</span>; <span class="NLM_string-name">Foresman, J. B.</span>; <span class="NLM_string-name">Fox, D. J.</span></span> <i>Gaussian 09</i>, Revision B.01; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford, CT</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2013&amp;author=M.+J.+Frisch&amp;author=G.+W.+Trucks&amp;author=H.+B.+Schlegel&amp;author=G.+E.+Scuseria&amp;author=M.+A.+Robb&amp;author=J.+R.+Cheeseman&amp;author=G.+Scalmani&amp;author=V.+Barone&amp;author=G.+A.+Petersson&amp;author=H.+Nakatsuji&amp;author=X.+Li&amp;author=M.+Caricato&amp;author=A.+Marenich&amp;author=J.+Bloino&amp;author=B.+G.+Janesko&amp;author=R.+Gomperts&amp;author=B.+Mennucci&amp;author=H.+P.+Hratchian&amp;author=J.+V.+Ortiz&amp;author=A.+F.+Izmaylov&amp;author=J.+L.+Sonnenberg&amp;author=D.+Williams-Young&amp;author=F.+Ding&amp;author=F.+Lipparini&amp;author=F.+Egidi&amp;author=J.+Goings&amp;author=B.+Peng&amp;author=A.+Petrone&amp;author=T.+Henderson&amp;author=D.+Ranasinghe&amp;author=V.+G.+Zakrzewski&amp;author=J.+Gao&amp;author=N.+Rega&amp;author=G.+Zheng&amp;author=W.+Liang&amp;author=M.+Hada&amp;author=M.+Ehara&amp;author=K.+Toyota&amp;author=R.+Fukuda&amp;author=J.+Hasegawa&amp;author=M.+Ishida&amp;author=T.+Nakajima&amp;author=Y.+Honda&amp;author=O.+Kitao&amp;author=H.+Nakai&amp;author=T.+Vreven&amp;author=K.+Throssell&amp;author=J.+A.+Montgomery&amp;author=P.+E.+Peralta&amp;author=F.+Ogliaro&amp;author=M.+Bearpark&amp;author=J.%0AJ.+Heyd&amp;author=E.+Brothers&amp;author=K.%0AN.+Kudin&amp;author=V.+N.+Staroverov&amp;author=T.+Keith&amp;author=R.+Kobayashi&amp;author=J.+Normand&amp;author=K.+Raghavachari&amp;author=A.+Rendell&amp;author=J.+C.+Burant&amp;author=S.+S.+Iyengar&amp;author=J.+Tomasi&amp;author=M.+Cossi&amp;author=J.+M.+Millam&amp;author=M.+Klene&amp;author=C.+Adamo&amp;author=R.+Cammi&amp;author=J.+W.+Ochterski&amp;author=R.+L.+Martin&amp;author=K.+Morokuma&amp;author=%C3%96.+Farkas&amp;author=J.+B.+Foresman&amp;author=D.+J.+Fox&amp;title=Gaussian+09" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26btitle%3DGaussian%252009%26pub%3DGaussian%252C%2520Inc%26date%3D2013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hornak, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strockbine, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roitberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span> <span> </span><span class="NLM_article-title">Comparison of Multiple Amber Force Fields and Development of Improved Protein Backbone Parameters</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">712</span>â <span class="NLM_lpage">725</span>, <span class="refDoi">&nbsp;DOI: 10.1002/prot.21123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1002%2Fprot.21123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=16981200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=65&amp;publication_year=2006&amp;pages=712-725&amp;author=V.+Hornakauthor=R.+Abelauthor=A.+Okurauthor=B.+Strockbineauthor=A.+Roitbergauthor=C.+Simmerling&amp;title=Comparison+of+Multiple+Amber+Force+Fields+and+Development+of+Improved+Protein+Backbone+Parameters&amp;doi=10.1002%2Fprot.21123" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of multiple Amber force fields and development of improved protein backbone parameters</span></div><div class="casAuthors">Hornak, Viktor; Abel, Robert; Okur, Asim; Strockbine, Bentley; Roitberg, Adrian; Simmerling, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">712-725</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The ff94 force field that is commonly assocd. with the Amber simulation package is one of the most widely used parameter sets for biomol. simulation.  After a decade of extensive use and testing, limitations in this force field, such as over-stabilization of Î±-helixes, were reported by the authors and other researchers.  This led to a no. of attempts to improve these parameters, resulting in a variety of "Amber" force fields and significant difficulty in detg. which should be used for a particular application.  The authors show that several of these continue to suffer from inadequate balance between different secondary structure elements.  In addn., the approach used in most of these studies neglected to account for the existence in Amber of two sets of backbone .vphi./Ï dihedral terms.  This led to parameter sets that provide unreasonable conformational preferences for glycine.  The authors report here an effort to improve the .vphi./Ï dihedral terms in the ff99 energy function.  Dihedral term parameters are based on fitting the energies of multiple conformations of glycine and alanine tetrapeptides from high level ab initio quantum mech. calcns.  The new parameters for backbone dihedrals replace those in the existing ff99 force field.  This parameter set, which the authors denote ff99SB, achieves a better balance of secondary structure elements as judged by improved distribution of backbone dihedrals for glycine and alanine with respect to PDB survey data.  It also accomplishes improved agreement with published exptl. data for conformational preferences of short alanine peptides and better accord with exptl. NMR relaxation data of test protein systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmoASzza-Gd7Vg90H21EOLACvtfcHk0lhYa75bCvZaKQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD28XhtFWqt7fM&amp;md5=de683a26eca9e83ae524726e97ac22fa</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fprot.21123&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21123%26sid%3Dliteratum%253Aachs%26aulast%3DHornak%26aufirst%3DV.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DOkur%26aufirst%3DA.%26aulast%3DStrockbine%26aufirst%3DB.%26aulast%3DRoitberg%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26atitle%3DComparison%2520of%2520Multiple%2520Amber%2520Force%2520Fields%2520and%2520Development%2520of%2520Improved%2520Protein%2520Backbone%2520Parameters%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2006%26volume%3D65%26spage%3D712%26epage%3D725%26doi%3D10.1002%2Fprot.21123" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">PÃ©rez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MarchÃ¡n, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svozil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sponer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laughton, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orozco, M.</span></span> <span> </span><span class="NLM_article-title">Refinement of the AMBER Force Field for Nucleic Acids: Improving the Description of Î±/Î³ Conformers</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">3817</span>â <span class="NLM_lpage">3829</span>, <span class="refDoi">&nbsp;DOI: 10.1529/biophysj.106.097782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1529%2Fbiophysj.106.097782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=17351000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD2sXls1ygu7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=92&amp;publication_year=2007&amp;pages=3817-3829&amp;author=A.+P%C3%A9rezauthor=I.+March%C3%A1nauthor=D.+Svozilauthor=J.+Sponerauthor=T.+E.+Cheathamauthor=C.+A.+Laughtonauthor=M.+Orozco&amp;title=Refinement+of+the+AMBER+Force+Field+for+Nucleic+Acids%3A+Improving+the+Description+of+%CE%B1%2F%CE%B3+Conformers&amp;doi=10.1529%2Fbiophysj.106.097782" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of the AMBER force field for nucleic acids: improving the description of Î±/Î³ conformers</span></div><div class="casAuthors">Perez, Alberto; Marchan, Ivan; Svozil, Daniel; Sponer, Jiri; Cheatham, Thomas E., III; Laughton, Charles A.; Orozco, Modesto</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3817-3829</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">We present here the parmbsc0 force field, a refinement of the AMBER parm99 force field, where emphasis has been made on the correct representation of the Î±/Î³ concerted rotation in nucleic acids (NAs).  The modified force field corrects overpopulations of the Î±/Î³ = (g+,t) backbone that were seen in long (more than 10 ns) simulations with previous AMBER parameter sets (parm94-99).  The force field has been derived by fitting to high-level quantum mech. data and verified by comparison with very high-level quantum mech. calcns. and by a very extensive comparison between simulations and exptl. data.  The set of validation simulations includes two of the longest trajectories published to date for the DNA duplex (200 ns each) and the largest variety of NA structures studied to date (15 different NA families and 97 individual structures).  The total simulation time used to validate the force field includes near 1 Î¼s of state-of-the-art mol. dynamics simulations in aq. soln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriZsH_9T_yB7Vg90H21EOLACvtfcHk0lhYn4T9gmaQ6w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXls1ygu7Y%253D&amp;md5=e0f8d87e5fe7710e59cc41f5e8c25f7b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.097782&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.097782%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A9rez%26aufirst%3DA.%26aulast%3DMarch%25C3%25A1n%26aufirst%3DI.%26aulast%3DSvozil%26aufirst%3DD.%26aulast%3DSponer%26aufirst%3DJ.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DLaughton%26aufirst%3DC.%2BA.%26aulast%3DOrozco%26aufirst%3DM.%26atitle%3DRefinement%2520of%2520the%2520AMBER%2520Force%2520Field%2520for%2520Nucleic%2520Acids%253A%2520Improving%2520the%2520Description%2520of%2520%25CE%25B1%252F%25CE%25B3%2520Conformers%26jtitle%3DBiophys.%2520J.%26date%3D2007%26volume%3D92%26spage%3D3817%26epage%3D3829%26doi%3D10.1529%2Fbiophysj.106.097782" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salomon-Ferrer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GÃ¶tz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 2. Explicit Solvent Particle Mesh Ewald</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3878</span>â <span class="NLM_lpage">3888</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct400314y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400314y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2013&amp;pages=3878-3888&amp;author=R.+Salomon-Ferrerauthor=A.+W.+G%C3%B6tzauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&amp;title=Routine+Microsecond+Molecular+Dynamics+Simulations+with+AMBER+on+GPUs.+2.+Explicit+Solvent+Particle+Mesh+Ewald&amp;doi=10.1021%2Fct400314y" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 2. Explicit Solvent Particle Mesh Ewald</span></div><div class="casAuthors">Salomon-Ferrer, Romelia; Gotz, Andreas W.; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3878-3888</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of explicit solvent all atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA-enabled GPUs.  First released publicly in Apr. 2010 as part of version 11 of the AMBER MD package and further improved and optimized over the last two years, this implementation supports the three most widely used statistical mech. ensembles (NVE, NVT, and NPT), uses particle mesh Ewald (PME) for the long-range electrostatics, and runs entirely on CUDA-enabled NVIDIA graphics processing units (GPUs), providing results that are statistically indistinguishable from the traditional CPU version of the software and with performance that exceeds that achievable by the CPU version of AMBER software running on all conventional CPU-based clusters and supercomputers.  We briefly discuss three different precision models developed specifically for this work (SPDP, SPFP, and DPDP) and highlight the tech. details of the approach as it extends beyond previously reported work [Goetz et al., J.  Chem.  Theory Comput.2012, DOI: 10.1021/ct200909j; Le Grand et al., Comp.  Phys.  Comm.2013, DOI: 10.1016/j.cpc.2012.09.022].We highlight the substantial improvements in performance that are seen over traditional CPU-only machines and provide validation of our implementation and precision models.  We also provide evidence supporting our decision to deprecate the previously described fully single precision (SPSP) model from the latest release of the AMBER software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY-M2E_jgUmLVg90H21EOLACvtfcHk0lhYn4T9gmaQ6w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXht1arsrzP&amp;md5=57e18d35e8b81e9a79788ef5af17d19f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fct400314y&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400314y%26sid%3Dliteratum%253Aachs%26aulast%3DSalomon-Ferrer%26aufirst%3DR.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520Microsecond%2520Molecular%2520Dynamics%2520Simulations%2520with%2520AMBER%2520on%2520GPUs.%25202.%2520Explicit%2520Solvent%2520Particle%2520Mesh%2520Ewald%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3878%26epage%3D3888%26doi%3D10.1021%2Fct400314y" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">GÃ¶tz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 1. Generalized Born</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1542</span>â <span class="NLM_lpage">1555</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct200909j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct200909j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC38XksFWns78%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2012&amp;pages=1542-1555&amp;author=A.+W.+G%C3%B6tzauthor=M.+J.+Williamsonauthor=D.+Xuauthor=D.+Pooleauthor=S.+Le+Grandauthor=R.+C.+Walker&amp;title=Routine+Microsecond+Molecular+Dynamics+Simulations+with+AMBER+on+GPUs.+1.+Generalized+Born&amp;doi=10.1021%2Fct200909j" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Routine Microsecond Molecular Dynamics Simulations with AMBER on GPUs. 1. Generalized Born</span></div><div class="casAuthors">Gotz, Andreas W.; Williamson, Mark J.; Xu, Dong; Poole, Duncan; Le Grand, Scott; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1542-1555</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an implementation of generalized Born implicit solvent all-atom classical mol. dynamics (MD) within the AMBER program package that runs entirely on CUDA enabled NVIDIA graphics processing units (GPUs).  We discuss the algorithms that are used to exploit the processing power of the GPUs and show the performance that can be achieved in comparison to simulations on conventional CPU clusters.  The implementation supports three different precision models in which the contributions to the forces are calcd. in single precision floating point arithmetic but accumulated in double precision (SPDP), or everything is computed in single precision (SPSP) or double precision (DPDP).  In addn. to performance, we have focused on understanding the implications of the different precision models on the outcome of implicit solvent MD simulations.  We show results for a range of tests including the accuracy of single point force evaluations and energy conservation as well as structural properties pertaining to protein dynamics.  The numerical noise due to rounding errors within the SPSP precision model is sufficiently large to lead to an accumulation of errors which can result in unphys. trajectories for long time scale simulations.  We recommend the use of the mixed-precision SPDP model since the numerical results obtained are comparable with those of the full double precision DPDP model and the ref. double precision CPU implementation but at significantly reduced computational cost.  Our implementation provides performance for GB simulations on a single desktop that is on par with, and in some cases exceeds, that of traditional supercomputers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpshIijeXLcqLVg90H21EOLACvtfcHk0lgt7nQG0gzjQw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XksFWns78%253D&amp;md5=1e6f570db9cd504bb13706e7c56bc356</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fct200909j&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct200909j%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWilliamson%26aufirst%3DM.%2BJ.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DPoole%26aufirst%3DD.%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DRoutine%2520Microsecond%2520Molecular%2520Dynamics%2520Simulations%2520with%2520AMBER%2520on%2520GPUs.%25201.%2520Generalized%2520Born%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D1542%26epage%3D1555%26doi%3D10.1021%2Fct200909j" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le
Grand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GÃ¶tz, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R. C.</span></span> <span> </span><span class="NLM_article-title">SPFP: Speed without CompromiseâA Mixed Precision Model for GPU Accelerated Molecular Dynamics Simulations</span>. <i>Comput. Phys. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>184</i></span>,  <span class="NLM_fpage">374</span>â <span class="NLM_lpage">380</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cpc.2012.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1016%2Fj.cpc.2012.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=184&amp;publication_year=2013&amp;pages=374-380&amp;author=S.+Le%0AGrandauthor=A.+W.+G%C3%B6tzauthor=R.+C.+Walker&amp;title=SPFP%3A+Speed+without+Compromise%E2%80%94A+Mixed+Precision+Model+for+GPU+Accelerated+Molecular+Dynamics+Simulations&amp;doi=10.1016%2Fj.cpc.2012.09.022" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">SPFP: Speed without compromise-A mixed precision model for GPU accelerated molecular dynamics simulations</span></div><div class="casAuthors">Le Grand, Scott; Gotz, Andreas W.; Walker, Ross C.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">374-380</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A new precision model is proposed for the acceleration of all-atom classical mol. dynamics (MD) simulations on graphics processing units (GPUs).  This precision model replaces double precision arithmetic with fixed point integer arithmetic for the accumulation of force components as compared to a previously introduced model that uses mixed single/double precision arithmetic.  This significantly boosts performance on modern GPU hardware without sacrificing numerical accuracy.  We present an implementation for NVIDIA GPUs of both generalized Born implicit solvent simulations as well as explicit solvent simulations using the particle mesh Ewald (PME) algorithm for long-range electrostatics using this precision model.  Tests demonstrate both the performance of this implementation as well as its numerical stability for const. energy and const. temp. biomol. MD as compared to a double precision CPU implementation and double and mixed single/double precision GPU implementations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJB_spY7SrSbVg90H21EOLACvtfcHk0lgt7nQG0gzjQw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhsV2jsrzO&amp;md5=2d9f102cb17f6e9bcbff3b5057d8c587</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cpc.2012.09.022&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cpc.2012.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BGrand%26aufirst%3DS.%26aulast%3DG%25C3%25B6tz%26aufirst%3DA.%2BW.%26aulast%3DWalker%26aufirst%3DR.%2BC.%26atitle%3DSPFP%253A%2520Speed%2520without%2520Compromise%25E2%2580%2594A%2520Mixed%2520Precision%2520Model%2520for%2520GPU%2520Accelerated%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D2013%26volume%3D184%26spage%3D374%26epage%3D380%26doi%3D10.1016%2Fj.cpc.2012.09.022" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera - A Visualization System for Exploratory Research and Analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>â <span class="NLM_lpage">1612</span>, <span class="refDoi">&nbsp;DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=25&amp;publication_year=2004&amp;pages=1605-1612&amp;author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&amp;title=UCSF+Chimera+-+A+Visualization+System+for+Exploratory+Research+and+Analysis&amp;doi=10.1002%2Fjcc.20084" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley &amp; Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lgt7nQG0gzjQw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&amp;md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%2520-%2520A%2520Visualization%2520System%2520for%2520Exploratory%2520Research%2520and%2520Analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Calculating Structures and Free Energies of Complex Molecules: Combining Molecular Mechanics and Continuum Models</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">889</span>â <span class="NLM_lpage">897</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar000033j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar000033j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmvFGiu7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=33&amp;publication_year=2000&amp;pages=889-897&amp;author=P.%0AA.+Kollmanauthor=I.+Massovaauthor=C.+Reyesauthor=B.+Kuhnauthor=S.+Huoauthor=L.+Chongauthor=M.+Leeauthor=T.+Leeauthor=Y.+Duanauthor=W.+Wang&amp;title=Calculating+Structures+and+Free+Energies+of+Complex+Molecules%3A+Combining+Molecular+Mechanics+and+Continuum+Models&amp;doi=10.1021%2Far000033j" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Calculating Structures and Free Energies of Complex Molecules: Combining Molecular Mechanics and Continuum Models</span></div><div class="casAuthors">Kollman, Peter A.; Massova, Irina; Reyes, Carolina; Kuhn, Bernd; Huo, Shuanghong; Chong, Lillian; Lee, Matthew; Lee, Taisung; Duan, Yong; Wang, Wei; Donini, Oreola; Cieplak, Piotr; Srinivasan, Jaysharee; Case, David A.; Cheatham, Thomas E., III</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">889-897</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, with 63 refs.  A historical perspective on the application of mol. dynamics (MD) to biol. macromols. is presented.  Recent developments combining state-of-the-art force fields with continuum solvation calcns. have allowed us to reach the fourth era of MD applications in which one can often derive both accurate structure and accurate relative free energies from mol. dynamics trajectories.  We illustrate such applications on nucleic acid duplexes, RNA hairpins, protein folding trajectories, and protein-ligand, protein-protein, and protein-nucleic acid interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXGhwlfA3kvLVg90H21EOLACvtfcHk0lj-iV_aBl5X5g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3cXmvFGiu7g%253D&amp;md5=8436ee610ae145894428db1a1deff73c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Far000033j&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far000033j%26sid%3Dliteratum%253Aachs%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DMassova%26aufirst%3DI.%26aulast%3DReyes%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DHuo%26aufirst%3DS.%26aulast%3DChong%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26atitle%3DCalculating%2520Structures%2520and%2520Free%2520Energies%2520of%2520Complex%2520Molecules%253A%2520Combining%2520Molecular%2520Mechanics%2520and%2520Continuum%2520Models%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2000%26volume%3D33%26spage%3D889%26epage%3D897%26doi%3D10.1021%2Far000033j" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGee, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swails, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homeyer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roitberg, A. E.</span></span> <span> </span><span class="NLM_article-title">MMPBSA.Py: An Efficient Program for End-State Free Energy Calculations</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3314</span>â <span class="NLM_lpage">3321</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ct300418h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300418h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.joc.8b02082&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2gtrzP" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2012&amp;pages=3314-3321&amp;author=B.+R.+Millerauthor=T.+D.+McGeeauthor=J.+M.+Swailsauthor=N.+Homeyerauthor=H.+Gohlkeauthor=A.+E.+Roitberg&amp;title=MMPBSA.Py%3A+An+Efficient+Program+for+End-State+Free+Energy+Calculations&amp;doi=10.1021%2Fct300418h" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">MMPBSA.py: An Efficient Program for End-State Free Energy Calculations</span></div><div class="casAuthors">Miller, Bill R., III; McGee, T. Dwight, Jr.; Swails, Jason M.; Homeyer, Nadine; Gohlke, Holger; Roitberg, Adrian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3314-3321</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MM-PBSA is a post-processing end-state method to calc. free energies of mols. in soln.  MMPBSA.py is a program written in Python for streamlining end-state free energy calcns. using ensembles derived from mol. dynamics (MD) or Monte Carlo (MC) simulations.  Several implicit solvation models are available with MMPBSA.py, including the Poisson-Boltzmann Model, the Generalized Born Model, and the Ref. Interaction Site Model.  Vibrational frequencies may be calcd. using normal mode or quasi-harmonic anal. to approx. the solute entropy.  Specific interactions can also be dissected using free energy decompn. or alanine scanning.  A parallel implementation significantly speeds up the calcn. by dividing frames evenly across available processors.  MMPBSA.py is an efficient, user-friendly program with the flexibility to accommodate the needs of users performing end-state free energy calcns.  The source code can be downloaded at http://ambermd.org/ with AmberTools, released under the GNU General Public License.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZK3C1jqiSULVg90H21EOLACvtfcHk0lj-iV_aBl5X5g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhtV2gtrzP&amp;md5=cc4148bd8f70c7cad94fd3ec6f580e52</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fct300418h&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300418h%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DB.%2BR.%26aulast%3DMcGee%26aufirst%3DT.%2BD.%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DHomeyer%26aufirst%3DN.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DRoitberg%26aufirst%3DA.%2BE.%26atitle%3DMMPBSA.Py%253A%2520An%2520Efficient%2520Program%2520for%2520End-State%2520Free%2520Energy%2520Calculations%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3314%26epage%3D3321%26doi%3D10.1021%2Fct300418h" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://doi.org/10.2210/pdb4Q43/pdb" onclick="trackOutboundLink('http://doi.org/10.2210/pdb4Q43/pdb','PDB','4Q43'); return false;">PDB: 4Q43</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i36">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_12564"></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.joc.8b02082">10.1021/acs.joc.8b02082</a>.</p><ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">PAGEs of nucleotide incorporation and full length extension assays; additional figures and tables from molecular modeling and dynamics studies; <sup>1</sup>H, <sup>13</sup>C, and <sup>31</sup>P NMR of all of the new compounds; and representative MALDI spectra of <i>N</i><sup>2</sup>-dG modified oligonucleotides (<a class="ext-link" href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf">PDF</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.joc.8b02082/suppl_file/jo8b02082_si_001.pdf">jo8b02082_si_001.pdf (5.35 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms &amp; Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7" style="width: 457.7px;"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.joc.8b02082&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjoceah.2019.84.issue-4%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Ajoceah%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.joc.8b02082%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3BrequestedJournal%3Ajournal%3Ajoceah" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self" style="display: none;"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.joc.8b02082" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article><div class="w-slide"><div class="w-slide_head"><a href="#" class="w-slide__back"><i class=" icon-arrow_l" aria-hidden="true"></i>back</a><span class="w-slide__title"></span></div><div class="w-slide__content"></div></div></main><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2022<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS &amp; Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support &amp; Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"><input type="hidden" name="appID" value="literatum" class="appID"><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onclick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        <!-- <div id="gdpr-message-container">
			<a id="closebtn" href="#"><i class="icon-times-circle-o"></i></a> -->
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        <!-- </div> -->
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        <!-- AdRoll Pixel -->
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>

<!-- Twitter universal website tag code -->
<!-- Commented out for ACHS-6152
<script>
    !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe.apply(s,arguments):s.queue.push(arguments);
    },s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',
    a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');
    // Insert Twitter Pixel ID and Standard Event data below
    twq('init','o3njp');
    twq('track','PageView');
</script>
-->
<!-- Facebook Pixel Code -->
<!-- Commented out for ACHS-6152
<script>
    !function(f,b,e,v,n,t,s)
    {if(f.fbq)return;n=f.fbq=function(){n.callMethod?
    n.callMethod.apply(n,arguments):n.queue.push(arguments)};
    if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
    n.queue=[];t=b.createElement(e);t.async=!0;
    t.src=v;s=b.getElementsByTagName(e)[0];
    s.parentNode.insertBefore(t,s)}(window, document,'script',
    'https://connect.facebook.net/en_US/fbevents.js');
    fbq('init', '529840281033320');
    fbq('track', 'PageView');
</script>
<noscript><img height="1" width="1" style="display:none" src="https://www.facebook.com/tr?id=529840281033320&amp;ev=PageView&amp;noscript=1"
/></noscript>
-->





        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div><div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/acs.joc.8b02624">Solid-Phase Approaches for Labeling Targeting Peptides with Far-Red Emitting Coumarin Fluorophores</a></li><li><a href="/doi/full/10.1021/acs.joc.8b03144">[6Ï] Photocyclization to cis-Hexahydrocarbazol-4-ones: Substrate Modification, Mechanism, and Scope</a></li><li><a href="/doi/full/10.1021/acs.joc.8b02305">Mononuclear Rearrangement of the Z-Phenylhydrazones of Some 3-Acyl-1,2,4-oxadiazoles: Effect of Substituents on the Nucleophilic Character of the &gt;CâNâNHâC6 H5 Chain and on the Charge Density of N-2 of the 1,2,4-Oxadiazole Ring (Electrophilic Counterpart)</a></li><li><a href="/doi/full/10.1021/acs.joc.8b03145">Organotrifluoroborate Salts as Complexation Reagents for Synthesizing BODIPY Dyes Containing Both Fluoride and an Organo Substituent at the Boron Center</a></li><li><a href="/doi/full/10.1021/acs.joc.8b02978">Extended Triangulenium Ions: Syntheses and Characterization of Benzo-Bridged Dioxa- and Diazatriangulenium Dyes</a></li></ul></div></div></div>




        
        <script>
var currentJournal = "The Journal of Organic Chemistry";
$(".citationLinks > a").click(function(e){
    if($(this).text() === "ACS Full Text ") {
        ga('send', 'event', 'Reference _Click', 'ACS Full Text', currentJournal);
    }
    else if($(this).text() === "Crossref") {
        ga('send', 'event', 'Reference _Click', 'Crossref', currentJournal);
    }
    else if($(this).text() === "CAS") {
        ga('send', 'event', 'Reference _Click', 'CAS', currentJournal);
    }
    else if($(this).text() === "PubMed") {
        ga('send', 'event', 'Reference _Click', 'PubMed', currentJournal);
    }
    else if($(this).text() === "Google Scholar") {
        ga('send', 'event', 'Reference _Click', 'Google Scholar', currentJournal);
    }
    else if($(this).hasClass("issue-item_open-url") === true) {
        ga('send', 'event', 'Reference _Click', 'OpenURL', currentJournal);
    }
});
</script>






        
        



        
    





        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        <div tabindex="0" class="showMenuAlertMsg-popup"><div class="pairing-affiliations-popup-content"><div class="pairing-affiliations-header"><button type="button" class="close paringAffClose"><i aria-hidden="true" class="icon-close_thin"></i></button></div><div class="responsrMsg"></div></div></div>




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	dartInstitutionType = 'education';
</script>
        </div>
    





        
        



        
    





        
        <script>
// LIT-455776 workaround
// Moving the popup before the header content for tab accessibility
document.addEventListener('DOMContentLoaded', () => {
    const headerNode = $('.header_content');
    const popupNode = $('.pairing-affiliations-popup');

    if (!headerNode.length || !popupNode.length)
        return;

    popupNode.insertBefore(headerNode);  
});
</script> 
        </div>
    </div>





	
    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-d93b9088814c7fcd2f1c.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-9dee82777d6af4ace557.js"></script>
<script type="text/javascript" src="/wro/l6j1~product.js"></script>



















<div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh376" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.7545402036267569&amp;iit=1650473253453&amp;tmr=load%3D1650473253031%26core%3D1650473253066%26main%3D1650473253448%26ifr%3D1650473253458&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=pubs.acs.org&amp;dr=&amp;du=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facs.joc.8b02082&amp;href=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facs.joc.8b02082&amp;dt=Synthesis%20of%20N2-Deoxyguanosine%20Modified%20DNAs%20and%20the%20Studies%20on%20Their%20Translesion%20Synthesis%20by%20the%20E.%20coli%20DNA%20Polymerase%20IV&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Ctype%3DArticle%2Cdescription%2Cimage%2Ctitle%2Curl&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=62603925394fb6f3&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=url%3Dhttps%253A%252F%252Fpubs.acs.org%252Fdoi%252Ffull%252F10.1021%252Facs.joc.8b02082%26title%3DSynthesis%2520of%2520N2-Deoxyguanosine%2520Modified%2520DNAs%2520and%2520the%2520Studies%2520on%2520Their%2520Translesion%2520Synthesis%2520by%2520the%2520E.%2520coli%2520DNA%2520Polymerase%2520IV%26image%3Dhttps%253A%252F%252Fpubs.acs.org%252Fcms%252F10.1021%252Facs.joc.8b02082%252Fasset%252Fimages%252Facs.joc.8b02082.social.jpeg_v03%26description%3DWe%2520report%2520the%2520synthesis%2520of%2520N2-aryl%2520(benzyl%252C%2520naphthyl%252C%2520anthracenyl%252C%2520and%2520pyrenyl)-deoxyguanosine%2520(dG)%2520modified%2520phosphoramidite%2520building%2520blocks%2520and%2520the%2520corresponding%2520damaged%2520DNAs.%2520Primer%2520extension%2520studies%2520using%2520E.%2520coli%2520Pol%2520IV%252C%2520a%2520translesion%2520polymerase%252C%2520demonstrate%2520that%2520translesion%2520synthesis%2520(TLS)%2520across%2520these%2520N2-dG%2520adducts%2520is%2520error%2520free.%2520However%252C%2520the%2520efficiency%2520of%2520TLS%2520activity%2520decreases%2520with%2520increase%2520in%2520the%2520steric%2520bulkiness%2520of%2520the%2520adducts.%2520Molecular%2520dynamics%2520simulations%2520of%2520damaged%2520DNA-Pol%2520IV%2520complexes%2520reveal%2520the%2520van%2520der%2520Waals%2520interactions%2520between%2520key%2520amino%2520acid%2520residues%2520(Phe13%252C%2520Ile31%252C%2520Gly32%252C%2520Gly33%252C%2520Ser42%252C%2520Pro73%252C%2520Gly74%252C%2520Phe76%252C%2520and%2520Tyr79)%2520of%2520the%2520enzyme%2520and%2520adduct%2520that%2520help%2520to%2520accommodate%2520the%2520bulky%2520damages%2520in%2520a%2520hydrophobic%2520pocket%2520to%2520facilitate%2520TLS.%2520Overall%252C%2520the%2520results%2520presented%2520here%2520provide%2520insights%2520into%2520the%2520TLS%2520across%2520N2-aryl-dG%2520damaged%2520DNAs%2520by%2520Pol%2520IV.%26type%3DArticle%26site_name%3DACS%2520Publications&amp;csi=undefined&amp;rev=v8.28.8-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div></body></html>