<html class="pb-page" data-request-id="b5d4257d-da72-41c2-a48c-883025dfda92" lang="en"><head data-pb-dropzone="head"><script type="text/javascript" src="https://m.addthis.com/live/red_lojson/300lo.json?si=6269cccdea2c32e4&amp;bkl=0&amp;bl=1&amp;pdt=524&amp;sid=6269cccdea2c32e4&amp;pub=xa-4faab26f2cff13a7&amp;rev=v8.28.8-wp&amp;ln=en&amp;pc=men&amp;cb=0&amp;ab=-&amp;dp=pubs.acs.org&amp;fp=doi%2F10.1021%2Facs.jpcb.8b11151&amp;fr=&amp;of=2&amp;pd=0&amp;irt=1&amp;vcl=2&amp;md=0&amp;ct=1&amp;tct=0&amp;abt=0&amp;cdn=0&amp;pi=1&amp;rb=0&amp;gen=100&amp;chr=UTF-8&amp;colc=1651100877911&amp;jsl=1&amp;uvs=6269cc301226c77104c&amp;skipb=1&amp;callback=addthis.cbs.jsonp__62213468546867050"></script><script type="text/javascript" src="https://v1.addthisedge.com/live/boost/xa-4faab26f2cff13a7/_ate.track.config_resp"></script><script src="https://widgets.figshare.com/static/figshare.js?_=1651100877388"></script><script type="text/javascript" src="https://z.moatads.com/addthismoatframe568911941483/moatframe.js"></script><meta name="pbContext" content=";journal:journal:jpcbfk;issue:issue:10.1021/jpcbfk.2019.123.issue-6;page:string:Article/Chapter View;ctype:string:Journal Content;requestedJournal:journal:jpcbfk;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jpcb.8b11151">




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="The Journal of Physical Chemistry B"><meta name="dc.Title" content="Slow Motion Protein Dance Visualized Using Red-Edge Excitation Shift of a Buried Fluorophore"><meta name="dc.Creator" content="Prajna  Mishra"><meta name="dc.Creator" content="Santosh Kumar  Jha"><meta name="dc.Description" content="It has been extremely challenging to detect protein structures with a dynamic core, such as dry molten globules, that remain in equilibrium with the tightly packed native (N) state and that are imp..."><meta name="Description" content="It has been extremely challenging to detect protein structures with a dynamic core, such as dry molten globules, that remain in equilibrium with the tightly packed native (N) state and that are imp..."><meta name="dc.Publisher" content="American Chemical Society"><meta name="dc.Date" scheme="WTN8601" content="January 14, 2019"><meta name="dc.Type" content="research-article"><meta name="dc.Format" content="text/HTML"><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jpcb.8b11151"><meta name="dc.Language" content="en"><meta name="dc.Coverage" content="world"><meta name="dc.Rights" content="Copyright Â© 2019 American Chemical Society">
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jpcb.8b11151">
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jpcb.8b11151">
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jpcb.8b11151">
        
    
    

<title>Slow Motion Protein Dance Visualized Using Red-Edge Excitation Shift of a Buried Fluorophore | The Journal of Physical Chemistry B</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive">





    






        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jpcb.8b11151"><meta property="og:title" content="Slow Motion Protein Dance Visualized Using Red-Edge Excitation Shift of a Buried Fluorophore"><meta property="og:image" content="https://pubs.acs.org/cms/10.1021/acs.jpcb.8b11151/asset/images/acs.jpcb.8b11151.social.jpeg_v03"><meta property="og:description" content="It has been extremely challenging to detect protein structures with a dynamic core, such as dry molten globules, that remain in equilibrium with the tightly packed native (N) state and that are important for a myriad of entropy-driven protein functions. Here, we detect the higher entropy conformations of a human serum protein, using red-edge excitation shift experiments. We covalently introduced a fluorophore inside the protein core and observed that in a subset of native population, the side chains of the polar and buried residues have different spatial arrangements than the mean population and that they solvate the fluorophore on a timescale much slower than the nanosecond timescale of fluorescence. Our results provide direct evidence for the dense fluidity of protein core and show that alternate side-chain packing arrangements exist in the core that might be important for multiple binding functions of this protein."><meta property="og:type" content="Article"><meta property="og:site_name" content="ACS Publications"><meta name="twitter:card" content="summary_large_image"><meta name="twitter:site" content="@ACSPublications"><meta name="twitter:image" content="https://pubs.acs.org/cms/10.1021/acs.jpcb.8b11151/asset/images/acs.jpcb.8b11151.social.jpeg_v03">
    





    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0">















    

    <meta name="dc.identifier" content="10.1021/acs.jpcb.8b11151">



<meta name="format-detection" content="telephone=no">




<link rel="stylesheet" type="text/css" href="/wro/l6j1~product.css">

<link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-c7d2282eca032afdc856.css"><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css">




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1640088552890.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1648583148410.css">
<script type="text/javascript" async="" src="//img.en25.com/i/elqCfg.min.js"></script><script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script><script async="" src="//www.google-analytics.com/analytics.js"></script><script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer="" src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon="{&quot;rayId&quot;:&quot;5200035e6e4d5180&quot;,&quot;startTime&quot;:1570117212010}"></script>

            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    , 'auto'
                    );
                
                ga('send', 'pageview');
            </script>
            











    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jpcb.8b11151">
            
        







    <!-- loading Javascript for using the persistence Service -->
    <script src="/templates/jsp/pb2/js/plugin/thiss-ds.js"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            alt_context: window.location.origin,
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if (ms == undefined || ms === '') {
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            if (results.length == 0) {
                                persistService.entities(self.alt_context).then((res) => {
                                    return resolve(res.data);
                                }, function (err) {
                                    // failed
                                    reject('failed to retrieve entity from local storage (publisher domain): ' + err);
                                });
                            } else {
                                return resolve(results);
                            }
                        }, function (err) {
                            // failed
                            reject('failed to retrieve entity from local storage: ' + err);
                        });
                    // Set up the timeout
                    setTimeout(function () {
                        reject('Promise timed out ' + ms + ' ms');
                    }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                    persistService.update(self.alt_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage (publisher domain) duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            var results = res.data;
                            if (!results) {
                                persistService.remove(self.alt_context, entityId).then((res) => {
                                    return resolve(res);
                                }, function (err) {
                                    // failed
                                    console.log('failed to remove from local storage (publisher domain) duo to: ' + err);
                                });
                            }
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId, entityTitle) {
                return new Promise(function (resolve, reject) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {
                            doi: doi,
                            entityid: entityId,
                            entityTitle: entityTitle,
                            eventGroupKey: '22f519ad-71c8-4b43-b15d-176bc5d1b75b'
                        }

                    }).done(function (status) {
                        return resolve(status);
                    }).fail(function (status) {
                        return reject(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID, ssoInstitution.entity.title)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + ssoInstitution.entity.entityID,err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            }).catch(function (err) {
                                reject(err);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();

                    if ($('.institution-preference-userconsent-checkbox').length > 0) {
                        let institutionPreferenceUserConsentCheckbox = $('.institution-preference-userconsent-checkbox').get(0);
                        if (!$(institutionPreferenceUserConsentCheckbox).is(":checked")) {
                            return;
                        }
                    }

                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err) {
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>







    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY">


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html">

<link rel="shortcut icon" href="/pb-assets/ux3/favicon-1635812676023.ico"><link type="text/css" rel="stylesheet" href="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/CcdcWidget.min.css"><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><meta http-equiv="origin-trial" content="AxujKG9INjsZ8/gUq8+dTruNvk7RjZQ1oFhhgQbcTJKDnZfbzSTE81wvC2Hzaf3TW4avA76LTZEMdiedF1vIbA4AAABueyJvcmlnaW4iOiJodHRwczovL2ltYXNkay5nb29nbGVhcGlzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzVGhpcmRQYXJ0eSI6dHJ1ZX0="><meta http-equiv="origin-trial" content="Azuce85ORtSnWe1MZDTv68qpaW3iHyfL9YbLRy0cwcCZwVnePnOmkUJlG8HGikmOwhZU22dElCcfrfX2HhrBPAkAAAB7eyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A16nvcdeoOAqrJcmjLRpl1I6f3McDD8EfofAYTt/P/H4/AWwB99nxiPp6kA0fXoiZav908Z8etuL16laFPUdfQsAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="AxBHdr0J44vFBQtZUqX9sjiqf5yWZ/OcHRcRMN3H9TH+t90V/j3ENW6C8+igBZFXMJ7G3Pr8Dd13632aLng42wgAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A88BWHFjcawUfKU3lIejLoryXoyjooBXLgWmGh+hNcqMK44cugvsI5YZbNarYvi3roc1fYbHA1AVbhAtuHZflgEAAAB2eyJvcmlnaW4iOiJodHRwczovL2dvb2dsZS5jb206NDQzIiwiZmVhdHVyZSI6IlRydXN0VG9rZW5zIiwiZXhwaXJ5IjoxNjUyNzc0NDAwLCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="AzoawhTRDevLR66Y6MROu167EDncFPBvcKOaQispTo9ouEt5LvcBjnRFqiAByRT+2cDHG1Yj4dXwpLeIhc98/gIAAACFeyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A6+nc62kbJgC46ypOwRsNW6RkDn2x7tgRh0wp7jb3DtFF7oEhu1hhm4rdZHZ6zXvnKZLlYcBlQUImC4d3kKihAcAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A/9La288e7MDEU2ifusFnMg1C2Ij6uoa/Z/ylwJIXSsWfK37oESIPbxbt4IU86OGqDEPnNVruUiMjfKo65H/CQwAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_2022042101.js" async=""></script><style></style></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'physical_chemistry_b';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/physical_chemistry_b/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/physical_chemistry_b/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
					console.log("ad display: " + event.creativeId + " " + event.size[0] + "x" + event.size[1]);
				}
				else {
				    console.log("no ad found");
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>

<style>
.articleHeaderDropzone5 .advertisement { display: none; }
.article_header .advertisement {
  margin: -25px auto 25px auto;
}
</style>




        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&amp;EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Indiana University Libraries - Bloomington</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"></a>




        
        <div class="pj-badge"></div><a href="/journal/jpcbfk" title="The Journal of Physical Chemistry B" class="header_article-logo"><img src="/cms/10.1021/jpcbfk/asset/title.png" alt="The Journal of Physical Chemistry B"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="/doi/10.1021/acs.jpcb.8b09694" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span><span class="hlFld-Title">Slow Motion Protein Dance Visualized Using Red-Edge Excitation Shift of a Buried Fluorophore</span></span>
        </div>
         <a href="/doi/10.1021/acs.jpcb.8b11456" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete ui-autocomplete-input"><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jpcbfk" name="SeriesKey" checked="checked" class="all-content"><span>J. Phys. Chem. B</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"><span>All Publications/Website</span></label></div><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" style="display: none;"></ul></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science &amp; Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials &amp; Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio &amp; Med Chem Au</option><option value="abseba">ACS Biomaterials Science &amp; Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health &amp; Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&amp;T Engineering</option><option value="aewcaa">ACS ES&amp;T Water</option><option value="afsthl">ACS Food Science &amp; Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic &amp; Inorganic Au</option><option value="aptsfn">ACS Pharmacology &amp; Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry &amp; Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&amp;EN Global Enterprise</option><option value="cenear">Chemical &amp; Engineering News Archive</option><option value="chlseg">Chemical Health &amp; Safety</option><option value="chlseg0">Chemical Health &amp; Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth &amp; Design</option><option value="enfuem">Energy &amp; Fuels</option><option value="esthag">Environmental Science &amp; Technology</option><option value="estlcu">Environmental Science &amp; Technology Letters</option><option value="iepra6.2">I&amp;EC Product Research and Development</option><option value="iechad">Industrial &amp; Engineering Chemistry</option><option value="iecac0">Industrial &amp; Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series</option><option value="iecfa7">Industrial &amp; Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial &amp; Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial &amp; Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial &amp; Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical &amp; Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health &amp; Safety</option><option value="jchsc20">Journal of Chemical Health &amp; Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial &amp; Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research &amp; Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&amp;D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autocomplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autocomplete="false" placeholder="Page" required="required" class="quick-search_page-input"><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden="âtrueâ"></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/acs.jpcc.8b11685">Ternary Composites with PbS Quantum Dots for Hybrid Photovoltaics</a></li><li><a href="/doi/full/10.1021/acs.jpcb.8b09487">Length-Dependent Melting Kinetics of Short DNA Oligonucleotides Using Temperature-Jump IR Spectroscopy</a></li><li><a href="/doi/full/10.1021/acs.jpcc.8b11201">Theoretical Insights into Propene Epoxidation on Au7/Anatase TiO2âx(001) Catalysts: Effect of the Interface and Reaction Atmosphere</a></li><li><a href="/doi/full/10.1021/acs.jpcc.8b07734">Ultrafast Excited-State Dynamics of Partially and Fully Functionalized Silsesquioxanes</a></li><li><a href="/doi/full/10.1021/acs.jpcc.8b11324">Reactive Molecular Dynamics Simulations to Investigate the Shock Response of Liquid Nitromethane</a></li></ul></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="âtrueâ"></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="âtrueâ"></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden="âtrueâ"></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors &amp; Reviewers');">Authors &amp;
                        Reviewers</a></li>
                <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
				<li><a href="https://pubs.acs.org/page/virtual-collections.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Virtual Issues');">Virtual Issues</a></li>
                <li><a href="https://preferences.acs.org/journals" onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS &amp; Mobile');">RSS &amp; Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products &amp; Services');">Products &amp; Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos &amp; Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#" onclick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');" class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/who-we-serve/librarians/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians &amp; Account Managers');">For
                        Librarians &amp; Account Managers</a></li>
            </ul>
        </li>
        <li class="menu-pair-section hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a></li>
                
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage" style="display: block;">Login</a>
            <a class="header_burger-menu_logout-link" href="#" style="display: none;">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS &amp; Open Access');">ACS &amp; Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="/events" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i><div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/acs.jpcc.8b11685">Ternary Composites with PbS Quantum Dots for Hybrid Photovoltaics</a></li><li><a href="/doi/full/10.1021/acs.jpcb.8b09487">Length-Dependent Melting Kinetics of Short DNA Oligonucleotides Using Temperature-Jump IR Spectroscopy</a></li><li><a href="/doi/full/10.1021/acs.jpcc.8b11201">Theoretical Insights into Propene Epoxidation on Au7/Anatase TiO2âx(001) Catalysts: Effect of the Interface and Reaction Atmosphere</a></li><li><a href="/doi/full/10.1021/acs.jpcc.8b07734">Ultrafast Excited-State Dynamics of Partially and Fully Functionalized Silsesquioxanes</a></li><li><a href="/doi/full/10.1021/acs.jpcc.8b11324">Reactive Molecular Dynamics Simulations to Investigate the Shock Response of Liquid Nitromethane</a></li></ul></div></div>


</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!-- Types--><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><!-- Subjects--><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!--div.pub-modal_right-cover-gallery.hidden-lg.hidden-md//a(href="/coverGallery") Browse Our Cover Art Gallery
--><!-- Publications list--><!-- 'pub-modal_right--inner' adds styling required for IE--><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><!-- The widget data--><input type="hidden" id="ajaxUrl" name="ajaxUrl" value="/pb/widgets/achs-publication-list?widgetId=cabcb15c-6f7a-494b-a19c-eba4fafaee5c&amp;pbContext=%3Bjournal%3Ajournal%3Ajpcbfk%3Bissue%3Aissue%3A10.1021%2Fjpcbfk.2019.123.issue-6%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3BrequestedJournal%3Ajournal%3Ajpcbfk%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jpcb.8b11151&amp;key=c7ca46a97ce077568c7371697bcd489c0&amp;ajaxRequest=true"><input type="hidden" name="pubModalData" value=""></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article" style="min-height: 754px;"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jpcb.8b11151&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jpcb.8b11151&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right" style="display: inline;"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left" style="display: inline;"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jpcb.8b11151&amp;href=/doi/10.1021/acs.jpcb.8b11151" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Phys. Chem. B</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 123</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 1256-1264</span></span></span></div></div></div></div></div><input type="hidden" name="jpcbfk" id="copyRights" value="/pb-assets/json/journal-info-1641590078917.json"><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jpcbfk/123/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jpcb.8b09694" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jpcb.8b11456" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jpcbfk" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/cms/10.1021/jpcbfk/asset/title.png" alt="Journal Logo"></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="jpcbfk" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info" href="#"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/jpcbfk?ifr=1"></div></div></div><h1 class="article_header-title"><span class="hlFld-Title">Slow Motion Protein Dance Visualized Using Red-Edge Excitation Shift of a Buried Fluorophore</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Prajna Mishra</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Prajna Mishra</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Physical and Materials Chemistry Division, Academy of Scientific and Innovative Research (AcSIR), CSIR-National Chemical Laboratory, Dr. Homi Bhabha Road, Pune 411008, Maharashtra, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Prajna++Mishra">Prajna Mishra</a></span></div></div></span></li><span class="comma-separator">&nbsp;and&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Santosh Kumar Jha</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Santosh Kumar Jha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Physical and Materials Chemistry Division, Academy of Scientific and Innovative Research (AcSIR), CSIR-National Chemical Laboratory, Dr. Homi Bhabha Road, Pune 411008, Maharashtra, India</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="mailto:sk.jha@ncl.res.in">sk.jha@ncl.res.in</a>. Phone: 91-20-25902588. Fax: 91-20-25902615.</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Santosh+Kumar++Jha">Santosh Kumar Jha</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0003-1339-7409" title="Orcid link">http://orcid.org/0000-0003-1339-7409</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jpcbfk"><input type="hidden" name="journalNameForjhpLink" value="The Journal of Physical Chemistry B"><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jpcb.8b11151&amp;href=/doi/10.1021%2Facs.jpcb.8b11151" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Phys. Chem. B</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 123</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 1256â1264</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 14, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>17 November 2018</li><li><span class="item_label"><b>Revised</b></span>7 January 2019</li><li><span class="item_label"><b>Published</b> online</span>14 January 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 February 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jpcb.8b11151" title="DOI URL">https://doi.org/10.1021/acs.jpcb.8b11151</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1256%26pageCount%3D9%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPrajna%2BMishra%252C%2BSantosh%2BKumar%2BJha%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D123%26issueNum%3D6%26contentID%3Dacs.jpcb.8b11151%26title%3DSlow%2BMotion%2BProtein%2BDance%2BVisualized%2BUsing%2BRed-Edge%2BExcitation%2BShift%2Bof%2Ba%2BBuried%2BFluorophore%26numPages%3D9%26pa%3D%26issn%3D1520-6106%26publisherName%3Dacs%26publication%3Djpcbfk%26rpt%3Dn%26endPage%3D1264%26publicationDate%3DFebruary%2B2019">RIGHTS &amp; PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access"></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jpcb.8b11151"><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">414</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">1</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">9</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" class="doi" value="10.1021/acs.jpcb.8b11151"><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="add-full-document achs-mendeley_button"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Slow Motion Protein Dance Visualized Using Red-Edge Excitation Shift of a Buried Fluorophore&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Prajna&quot;,&quot;last_name&quot;:&quot;Mishra&quot;},{&quot;first_name&quot;:&quot;Santosh&quot;,&quot;last_name&quot;:&quot;Kumar Jha&quot;}],&quot;source&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;14&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;123&quot;,&quot;pages&quot;:&quot;1256-1264&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jpcb.8b11151&quot;},&quot;abstract&quot;:&quot;It has been extremely challenging to detect protein structures with a dynamic core, such as dry molten globules, that remain in equilibrium with the tightly packed native (N) state and that are important for a myriad of entropy-driven protein functions. Here, we detect the higher entropy conformations of a human serum protein, using red-edge excitation shift experiments. We covalently introduced a fluorophore inside the protein core and observed that in a subset of native population, the side chains of the polar and buried residues have different spatial arrangements than the mean population and that they solvate the fluorophore on a timescale much slower than the nanosecond timescale of fluorescence. Our results provide direct evidence for the dense fluidity of protein core and show that alternate side-chain packing arrangements exist in the core that might be important for multiple binding functions of this protein.&quot;}" class="meta-data"><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"><input type="hidden" name="pub-access" value="full" class="pub-access"><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"><input type="hidden" name="selected-share-option" class="selected-share-option"></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jpcb.8b11151&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jpcb.8b11151" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jpcb.8b11151&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jpcb.8b11151" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jpcb.8b11151&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jpcb.8b11151" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jpcb.8b11151&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jpcb.8b11151&amp;href=/doi/10.1021/acs.jpcb.8b11151" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jpcb.8b11151"><a href="/doi/pdf/10.1021/acs.jpcb.8b11151" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.8b11151&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fid%3Ddoi%3A10.1021%252Facs.jpcb.8b11151%26sid%3Dliteratum%253Aachs%26pmid%3D30640479%26genre%3Darticle%26aulast%3DMishra%26date%3D2019%26atitle%3DSlow%2BMotion%2BProtein%2BDance%2BVisualized%2BUsing%2BRed-Edge%2BExcitation%2BShift%2Bof%2Ba%2BBuried%2BFluorophore%26jtitle%3DThe%2BJournal%2Bof%2BPhysical%2BChemistry%2BB%26title%3DThe%2BJournal%2Bof%2BPhysical%2BChemistry%2BB%26volume%3D123%26issue%3D6%26spage%3D1256%26epage%3D1264%26issn%3D1520-6106" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="jpcbfk" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info" href="#"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/jpcbfk?ifr=1"></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy" style="padding-left: 337.6px;"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291912" title="Fluorescence">Fluorescence</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292606" title="Imaging probes">Imaging probes</a>,</li><li><a href="/action/doSearch?ConceptID=292017" title="Solvents">Solvents</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="external-alerts-management" style="margin-left: -59px;"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="jpcbfk" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info" href="#"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/jpcbfk?ifr=1"></div><div class="cover-image"><div class="cover-image__image"><a href="/toc/jpcbfk/123/6" title="Go to The Journal of Physical Chemistry B "><img src="/cms/10.1021/jpcbfk.2019.123.issue-6/asset/jpcbfk.2019.123.issue-6.largecover.jpg" alt="Go to The Journal of Physical Chemistry B "></a></div></div>




        
        <!--Article Page E-Alerts-->
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jpcbfk');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jpcbfk');
	  }
  });
</script>
<!--End Article Script-->




        
        <div class="pj-badge"><input type="hidden" id="partnerJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json"></div>




        
        <div class="aJhp_link"><a href="https://pubs.acs.org/jpcb">The Journal of Physical Chemistry B</a></div>
<input type="hidden" id="aJhpJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json">
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jpcbfk" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_0006.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jpcb.8b11151&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">It has been extremely challenging to detect protein structures with a dynamic core, such as dry molten globules, that remain in equilibrium with the tightly packed native (N) state and that are important for a myriad of entropy-driven protein functions. Here, we detect the higher entropy conformations of a human serum protein, using red-edge excitation shift experiments. We covalently introduced a fluorophore inside the protein core and observed that in a subset of native population, the side chains of the polar and buried residues have different spatial arrangements than the mean population and that they solvate the fluorophore on a timescale much slower than the nanosecond timescale of fluorescence. Our results provide direct evidence for the dense fluidity of protein core and show that alternate side-chain packing arrangements exist in the core that might be important for multiple binding functions of this protein.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left ui-resizable" style="width: 730px;"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17833" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17833" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i16" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i28" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div class="NLM_p">The extent and degree of conformational heterogeneity in side-chain packing inside the hydrophobic interior of proteins are poorly understood. A body of theoretical work predicts a spectrum of possibilities for the dynamic properties of the protein core, including: (i) proteins could be like aperiodic solids which lack the periodic regularity of crystals; <a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> (ii) proteins may behave like surface-molten solids with liquid-like surface, whereas interior could be solid-like; <a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> and (iii) the side chains in the core of proteins could undergo substantial structural fluctuations and the core could be like a dense fluid. <a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> In contrast, crystal structures of proteins project a static image of the native (N) state, where the interior of proteins is tightly packed with packing densities as high as 0.7â0.8, <a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5â7)</a> which is very similar to the solid crystals of small organic compounds. Recent studies propose that the N state ensemble contains loosely packed forms termed dry molten globules (DMG) <a onclick="showRef(event, 'ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11">(8â11)</a> in which tight side-chain packing is ruptured but the core remains dehydrated. DMG-like states have expanded and dynamic cores with high conformational entropy and are important for entropy-driven protein function such as dynamic allostery. <a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> Experimentally, DMG-like states, however, have been majorly observed as kinetic intermediates, <a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14â16)</a> and it has remained elusive to detect and characterize them under equilibrium conditions in the N state ensemble. <a onclick="showRef(event, 'ref8 ref11 ref17'); return false;" href="javascript:void(0);" class="ref ref8 ref11 ref17">(8,11,17)</a> Thermodynamically, it is possible for proteins to undergo large fluctuations around the mean values of their physical properties such as internal energy, enthalpy, and molecular volume to give rise to DMG-like protein structures because of their small size. <a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> However, it has remained extremely challenging to understand the precise molecular and temporal nature of fluctuations and how small and rapid movements of protein side chains in the hydrophobic core may combine to give an overall change in the mean configuration.</div><div class="NLM_p">There is a scarcity of experimental and spectroscopic methods which can yield dynamic information (on microsecond to nanosecond timescale) on side-chain packing in the native state ensemble of proteins. NMR methods such as dynamic order parameter of methyl side chains and relaxation dispersion experiments have been employed for a few proteins <a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> including calmodulin <a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> and cyclophilin A, <a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> and side-chain dynamics in picosecond to microsecond timescale have been measured. However, NMR methods are limited to small, soluble proteins and cannot be applied to large, multidomain, and aggregation-prone proteins because of the requirement of millimolar concentrations for NMR studies. Thiolâdisulfide exchange is an elegant alternative method that has provided temporal information on the side-chain packing of proteins such as apmyoglobin, <a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> barstar, <a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> monellin, <a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and RNAse H. <a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> A major disadvantage of this method, however, is that undesirable side reactions such as disulfide scrambling and air oxidation of thiols to disulfide compete with the thiolâdisulfide exchange reaction.</div><div class="NLM_p">Fluorescence-based methods are highly sensitive (require micromolar concentrations of proteins) and are methods of choice for large multidomain proteins. Red-edge excitation shift (REES) studies of a fluorophore buried in the core of a protein can potentially give dynamic information on dipoles created by polar and polarizable side chains of the hydrophobic core, if they solvate the fluorophore on nanosecond and slower timescale. <a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> The phenomenon of REES is marked by a red shift in the wavelength of the maximum fluorescence emission (Î»<sub class="stack">max</sub><sup class="stack">em</sup>), when excited at the red edge of the excitation spectrum of the fluorophore <a onclick="showRef(event, 'ref28 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref28 ref30 ref31">(28,30,31)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). In general, the Î»<sub class="stack">max</sub><sup class="stack">em</sup> for a fluorophore is independent of the wavelength of excitation (Î»<sup>ex</sup>). <a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> However, in the cases where the environment surrounding the fluorophore is heterogeneous and motionally restricted, this principle is not followed and REES is observed. The molecular basis of REES is governed by the heterogeneity of the fluorophoreâsolvent interactions and the rate at which the solvent dipoles reorient themselves around the fluorophore in the excited state, as schematically illustrated in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>. For the illustration, a model <a onclick="showRef(event, 'ref30 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref32">(30,32)</a> is considered in which there exists a wide distribution of the fluorophoreâsolvent interaction energy in the ground state and after a vertical transition to the FranckâCondon (F) excited state, the fluorescence emission with a decay rate Ï<sub>f</sub> can occur from two distinct solvation states: (i) a completely solvent-relaxed R state, where reorientation of solvent dipoles around the fluorophores with a time constant of Ï<sub>s</sub> is complete, and (ii) a partially solvent-relaxed Râ² state, where solvent-dipole reorientation around the fluorophore has not completely occurred. Because of the heterogeneous distribution of the fluorophoreâsolvent interaction energy in the ground state, photoexcitation of the ensemble at the center (Î»<sub>C</sub>) or the red edge (Î»<sub>R</sub>) of the absorbance spectrum photoselects a different subpopulation of the molecules to the excited state. In the case of a dynamically solvated fluorophore (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A), the rate of the solvent relaxation is faster than that of the fluorescence timescale (Ï<sub>s</sub> &lt; Ï<sub>f</sub>) which allows the solvent dipoles to realign around the excited fluorophore before the emission. As a result, irrespective of the excitation at the Î»<sub>C</sub> or the Î»<sub>R</sub>, the fluorescence emission takes place from the completely solvent-relaxed R state. Thus, the emission spectra become independent of the Î»<sup>ex</sup>. On the other hand, if the fluorophore is placed in a heterogeneous and restricted solvation environment, as in the core of a protein, the solvent relaxation time becomes slower than the rate of the fluorescence emission, that is, Ï<sub>s</sub> &gt; Ï<sub>f</sub> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B). This does not permit the solvent dipolar realignment before the fluorescence emission, and in this case, the emission occurs from a partially solvent-relaxed Râ² state. For the low-energy Î»<sub>R</sub> excitation, the orientations of the solvent dipoles are such that the energy difference between the ground and the Râ² state is smaller than that for the high-energy Î»<sub>C</sub> excitation. As a result, the fluorescence spectra exhibit red spectral shift for the Î»<sub>R</sub> excitation compared to the Î»<sub>C</sub> excitation.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_0001.gif" alt="" data-lg-src="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0001.jpeg" id="gr1" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic description of REES. Comparative effects of the central and the red-edge excitations (top) on the emission spectra (bottom) of a fluorophore are shown. (A) Timescale of the solvent relaxation, Ï<sub>s</sub> âª the fluorescence timescale, Ï<sub>f</sub>, (B) Ï<sub>s</sub> â« Ï<sub>f</sub>. In both the panels, the inner yellow circle and the thick gray arrow inside it represent the fluorophore and its dipole moment vector, respectively. The gray hexagons and their directions denote the orientations of the solvent molecules. The shaded gray distribution with the red tail represents the distribution of the soluteâsolvent interaction energy in the ground singlet state (S<sub>0</sub>). In the excited singlet state (S<sub>1</sub>), the dotted and solid cyan lines show the FranckâCondon (F) states for the center and the red-edge excitations, respectively. The dashed pink lines indicate the complete solvent-relaxed (R) states. The incomplete solvent-relaxed states (Râ²) for the center and the red-edge excitations are presented by the dotted and the solid pink lines, respectively. The vertical upward blue arrows represent the excitation energies (absorbance), and the vertically downward dashed green and solid red arrows represent the fluorescence emission energies. In the bottom panels, the green dashed lines and the red solid lines, respectively, represent the emission spectra when excited at the center (Î»<sub>C</sub>) and the red edge (Î»<sub>R</sub>) of the absorbance spectrum (the blue solid lines). In the panel (B) (top), the crooked line around the gray hexagons denotes a restricted solvation environment, as inside the core of a protein.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jpcb.8b11151&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">One of the unique features of REES is that it provides information about the heterogeneity and the dynamics of the environment, in contrast to other fluorescence techniques that report about the fluorophore itself. This unique feature of REES makes it a reliable method to study the dense fluidity of the side-chain residues inside the protein core. Proteins with heterogeneous side-chain packing can give rise to an ensemble of DMG-like conformations in equilibrium with the N state. <a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> If the polar and buried side-chain residues inside the core of the DMG solvate a suitably placed fluorophore on a timescale much slower than the fluorescence timescale, REES can provide useful information on the equilibrium conformational states of a protein. REES has been applied mainly to study the solvation dynamics of water molecules in complex solvation environment. <a onclick="showRef(event, 'ref28 ref29 ref33'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref33">(28,29,33)</a> The dynamics of interfacial water molecules in model membranes is one such example. <a onclick="showRef(event, 'ref32 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref32 ref34 ref35">(32,34,35)</a> In the context of proteins, REES has been utilized to understand the conformational changes in the native, <a onclick="showRef(event, 'ref30 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref30 ref36 ref37">(30,36,37)</a> the unfolded (U), <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and the molten globule <a onclick="showRef(event, 'ref11 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref11 ref39 ref40">(11,39,40)</a> states. Extrinsic fluorescent probes such as 2-(<i>p</i>-toluidinylnaphthalene)-6-sulfonate have been employed to monitor REES in multiple proteins including Î²-lactoglobulin, Î²-casein, and serum albumins. <a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> In the case of erythroid spectrin, <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> application of REES revealed the presence of residual structure in the unfolded state. However, its application to study the dynamics of the protein core is highly limited. <a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a></div><div class="NLM_p last">In this study, we have utilized REES of a buried fluorophore to study the dynamics and conformational heterogeneity of core packing in the native state ensemble of a multidomain protein, human serum albumin (HSA). We observed that the N state of HSA is heterogeneous, and in a subpopulation of protein molecules, some of the side-chain residues in the protein core have different orientations and packings than the mean population. Our results indicate that the N state of HSA is an ensemble of DMG-like states.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81616" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81616" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i16" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i28" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Spectroscopic Methods and Instruments</h3><div class="NLM_p last">We used a PerkinElmer LAMBDA 650UV/Vis spectrophotometer and a PerkinElmer fluorescence spectrometer LS 55, respectively, to measure all of the absorption and fluorescence emission spectra. In all of the cases, cuvette of path length 1 cm was used. All of the circular dichroism (CD) spectra were obtained using a JASCO J-815 CD spectrometer. Here, cuvettes of path length 1 mm and 1 cm were used for far- and near-UV CD, respectively. For all of the spectra, background signal corrections were done by subtracting the buffer signals.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemicals, Reagents, Buffers, and Experimental Conditions</h3><div class="NLM_p">HSA (99% pure, fatty acid and globulin free) and urea (ultrapure grade) were procured from Alfa Aesar. 5-((((2-iodoacetyl)amino)ethyl)amino)naphthalene-1-sulfonic acid (1,5-IAEDANS) was purchased from Life Technologies. Guanidine hydrochloride (GdmCl) was obtained from Sigma. All other chemicals of highest purity grade were procured from HiMedia and used as received without any further purification. We determined the concentration of HSA by measuring the absorbance at 280 nm and using a molar extinction coefficient <a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> of 36â500 M<sup>â1</sup> cm<sup>â1</sup>.</div><div class="NLM_p last">Acetate (20 mM), phosphate (20 mM), and Tris-HCl (20 mM) were used at pHs 5, 7, and 8 native buffers, respectively. The unfolding buffers for each pH consisted of 9 M urea in their respective native buffers. We determined the concentrations of urea and GdmCl. from refractive index measurements, <a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> using an Abbe refractometer from Rajdhani Scientific Instruments Co. (model: RSR-2).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Preparation of 1,5-IAEDANS-Labeled HSA</h3><div class="NLM_p last">HSA was labeled with 1,5-IAEDANS at the sole-free cysteine, C34, as described earlier. <a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In short, the protein was labeled with 15-fold molar excess of dye in 6 M GdmCl and 20 mM Tris-HCl at pH 8. The reaction mixture was kept in dark for 4 h. The protein was then refolded, and the labeling reaction was quenched by diluting the reaction mixture into 20-fold volume excess of refolding buffer (20 mM phosphate buffer, pH 7) followed by overnight incubation at 4 Â°C. The labeled protein was then separated from the excess free dye by a PD-10 column. The percentage of labeling was estimated as reported previously. <a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The labeled protein (HSA-IAEDANS) was found to be &gt;95% labeled with the dye.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> pH Dependence of Protein Structure Monitored by Fluorescence and CD</h3><div class="NLM_p">For pH dependence studies of the protein structure, all of the samples were incubated overnight at different pHs. For fluorescence measurements, the protein concentration was â¼4â5 Î¼M. The aromatic residues were excited at 280 nm, and the emission was collected from 310 to 420 nm. C34-IAEDANS was excited at 337 nm, and emission spectrum was collected from 350 to 550 nm. Slit widths for excitation and emission were kept nominal. The scan speed was 100 nm/min.</div><div class="NLM_p last">For far-UV and near-UV CD measurements, the protein concentrations were 4â5 and â¼20 Î¼M, respectively. The far-UV CD experiment was monitored in the wavelength range of 190â250 nm, whereas for near-UV CD experiment, it was 250â310 nm. Each spectrum was averaged over three scans. For each CD spectrum, the data pitch, data integration time, bandwidth, and scan speed were 1 nm, 1 s, 2 nm, and 100 nm/min, respectively.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Urea-Induced Equilibrium Unfolding Transitions Monitored by Fluorescence</h3><div class="NLM_p last">For the urea-induced equilibrium unfolding transitions for HSA and HSA-IAEDANS, the protein samples (â¼4â5 Î¼M) were incubated overnight in different concentrations of urea at room temperature at pH 7.0. The equilibrium unfolding was monitored by the change in fluorescence of W214 at 340 nm upon excitation at 295 nm. The transitions were analyzed using a two-state, N â U model. The data were fitted to <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a> <a onclick="showRef(event, 'ref11 ref43'); return false;" href="javascript:void(0);" class="ref ref11 ref43">(11,43)</a><span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_m001.gif" alt="" id="_i9"><span class="labelSpan">(1)</span></span>where <i>y</i><sub>obs</sub> is the observed fluorescence signal, <i>y</i><sub>N</sub> and <i>y</i><sub>U</sub> are the signals of the native and unfolded states, respectively, and Î<i>G</i><sub>NU</sub> is the free energy of unfolding of N to U. It is assumed that Î<i>G</i><sub>NU</sub> is linearly dependent on the denaturant concentration [D] and is given by<span class="NLM_disp-formula" id="eq2"><img class="imgWithoutFigViewer " src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_m002.gif" alt="" id="_i10"><span class="labelSpan">(2)</span></span>where <span class="NLM_inline-formula"><img class="imgWithoutFigViewer inline-formula-styles" src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_m003.gif" alt="" id="_i11"></span> and <i>m</i><sub>NU</sub> are the standard free energy at 0 M urea and slope of the N â U transition, respectively.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Time-Resolved Fluorescence Measurements</h3><div class="NLM_p">An excitation laser source of 370 nm was used to measure the time-resolved decay kinetics of C34-IAEDANS. The emission decays were collected at 480 nm with 10â000 counts at the peak. An aqueous solution of the milk powder was used to measure the instrument response function (IRF), which was found to be â¼196 ps. We used the DAS6 analysis software supplied by HORIBA Scientific and deconvoluted and fitted the fluorescence decays to a sum of two-three exponentials as per the following equation <a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a><span class="NLM_disp-formula" id="eq3"><img class="imgWithoutFigViewer " src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_m004.gif" alt="" id="_i13"><span class="labelSpan">(3)</span></span></div><div class="NLM_p last">In the above equation, <i>I</i>(<i>t</i>) corresponds to the fluorescence intensity at time t and Î±<sub><i>i</i></sub> is the amplitude of the <i>i</i>th fluorescence lifetime, Ï<sub><i>i</i></sub>. The mean fluorescence lifetime (Ï<sub>m</sub>) was calculated using the following equation<span class="NLM_disp-formula" id="eq4"><img class="imgWithoutFigViewer " src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_m005.gif" alt="" id="_i14"><span class="labelSpan">(4)</span></span>such that â<sub><i>i</i></sub>Î±<sub><i>i</i></sub> = 1 and <i>i</i> = 1â3.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> REES Experiments</h3><div class="NLM_p last">For C34-IAEDANS, the fluorescence emission spectra were collected by exciting from 337 to 410 nm, whereas for W214, the excitation wavelengths ranged from 295 to 305 nm. The protein concentration was kept between â¼8 and 15 Î¼M. Nominal slit widths for both excitation and emission were used. The scan speed was 50 nm/min. Each spectrum was averaged over three scans. For all of the samples, buffer signal was corrected for solvent Raman peak. The emission maxima were calculated both by the first derivative as well as the manual inspection of the fluorescence spectrum and were found to be similar. The error bars for the values of emission maximum were estimated from the standard deviations of at least three independent measurements.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36603" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36603" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i16" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i28" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> C34 Is Buried inside the Protein Core and Is Solvated by Polar and Polarizable Residues</h3><div class="NLM_p">HSA is the most abundant protein found in blood plasma having the major function of transportation of fatty acids, hormones, and other compounds through the blood stream. It is an all-helical, three-domain protein where each domain is divided into two subdomains (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A). It has 35 cysteine residues distributed over its three domains. Out of these 35 cysteines, 34 forms disulfide bonds and the sole-free cysteine (C34) is buried in the hydrophobic core of domain I (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A). From the crystal structure, it was revealed that inside the core of domain I, C34 is solvated by the side chains of residues A28, L31, Q33, P35, D38, H39, L42, L74, V77, Y84, Y140, and R144 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B). Out of the 12 amino acid residues solvating C34 in the protein core, two are basic (H39 and R144), one is acidic (D38), and three are polar but uncharged (Q33, Y84, and Y140) in nature. The percentage burial of all of these residues is 95 Â± 5; except Q33, P35, and D38, which are around 60% buried. We also calculated the water accessibility of C34 by ASAview <a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and found it to be â¼99% buried.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_0002.gif" alt="" data-lg-src="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0002.jpeg" id="gr2" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. C34 and C34-IAEDANS are buried in the protein structure in the N state, and HSA and HSA-IAEDANS have similar structure and stability. (A) Location of C34 and W214 is shown in the crystal structure of HSA (drawn from PDB file <a href="http://doi.org/10.2210/pdb1AO6/pdb">1AO6</a> using the program PyMOL). In the three-domain structure of the protein, C34 is completely buried in the core of domain I and W214 is partially buried in the interdomain region. (B) Structure of HSA showing the side-chain residues solvating C34 in the core of domain I. The atoms of C34 are shown in orange spheres. The side chain of the amino acids solvating C34 are shown in blue sticks. The background light gray surface around the amino acid residues shows the water accessible surface area of the residues burying C34. (C) Mean residue ellipticity (MRE) of HSA and HSA-IAEDANS is shown in the far-UV range in the blue dotted and red solid lines, respectively. (D) Change in the fraction of unfolded molecules (FU) as a function of [urea] is shown for HSA (blue circles) and HSA-IAEDANS (red circles). The blue solid line through the circles is a fit to a two-state, N â U model. (E) Fluorescence emission spectra of HSA-IAEDANS in the N and the U states are shown at different pHs (indicated in the figure). Each spectrum was normalized at its maximum intensity. (F) Fluorescence lifetime decay of C34-IAEDANS in the N and the U states at different pHs as indicated in the figure is shown (IRF: Instrument Response Function).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jpcb.8b11151&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Structure and Stability of HSA Does Not Change upon Labeling C34 with 1,5-IAEDANS</h3><div class="NLM_p last">In order to probe the core dynamics of the protein, we labeled C34 with a highly sensitive fluorophore, 1,5-IAEDANS. It is important to determine that labeling does not change the structural properties as well as the thermodynamic stability of HSA. We first compared the global secondary structure of the labeled (HSA-IAEDANS) and the unlabeled proteins by far-UV CD spectroscopy (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C). The mean residual ellipticity (MRE) for HSA and HSA-IAEDANS at pH 7 at 222 nm was â18â670 and â18â262 deg cm<sup>2</sup> dmol<sup>â1</sup>, respectively, which conveyed that both the labeled and unlabeled HSA have similar secondary structure. We next investigated the effect of labeling on the thermodynamic stability of HSA by urea-induced unfolding experiments (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>D). We observed that the standard free energy of unfolding, <span class="NLM_inline-formula"><img class="imgWithoutFigViewer inline-formula-styles" src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_m006.gif" alt="" id="_i19"></span>, for HSA was 5.17 Â± 0.15 kcal mol<sup>â1</sup> and for HSA-IAEDANS was 5.11 Â± 0.18 kcal mol<sup>â1</sup>. The similar values of <span class="NLM_inline-formula"><img class="imgWithoutFigViewer inline-formula-styles" src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_m007.gif" alt="" id="_i20"></span> for both HSA and HSA-IAEDANS indicated that the structural stability of the N state did not change upon labeling. Similarly, the <i>m</i>-value, which is a measure of the solvent-exposed surface area of the protein, remained similar for both HSA (1.15 Â± 0.03 kcal mol<sup>â1</sup> M<sup>â1</sup>) and HSA-IAEDANS (1.15 Â± 0.10 kcal mol<sup>â1</sup> M<sup>â1</sup>).</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Burial of C34-IAEDANS in the Core as Revealed by Steady-State and Time-Resolved Fluorescence Studies</h3><div class="NLM_p last">All of the above data revealed that labeling of C34 did not change the global secondary structure and thermodynamic stability of HSA. Hence, C34-IAEDANS can be used as a reporter to probe the core dynamics of the protein in the N state. HSA is known to maintain N-like conformation in the pH range of 4.8â8.5 <a onclick="showRef(event, 'ref11 ref40'); return false;" href="javascript:void(0);" class="ref ref11 ref40">(11,40)</a> which forms its N state ensemble. In one of our previous studies, <a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> we have shown that the N state ensemble of HSA contains a DMG-like intermediate with an alternate side-chain packing arrangement and that the pH modulates the N â DMG equilibrium. The population of the DMG increases as the pH of the environment decreases from 7 to 5. In order to further understand the equilibrium dynamics of the N state ensemble, we performed REES experiments at different pHs varying from pH 5 to 8. It is, however, important to ensure that the probe C34-IAEDANS is buried in the protein core at all of the pH. Fluorescence Stokesâ shift and fluorescence lifetimes are sensitive measures of the solvation environment of a fluorophore. We, therefore, monitored the Î»<sub class="stack">max</sub><sup class="stack">em</sup> and fluorescence lifetime as a function of pH in the N- and urea-unfolded forms at pHs 5, 7, and 8 to probe the burial of C34-IAEDANS. The Î»<sub class="stack">max</sub><sup class="stack">em</sup> of C34-IAEDANS in the solvent-exposed U state at all of the pH is 483 nm and that in the N state is 469 nm (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>E). Because around 50% of the residues solvating C34 is polar or charged as revealed by the crystal structure (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B), the blue shift is not expected to be very large as in the case of a complete hydrophobic environment. Nevertheless, the observation that C34-IAEDANS shows a large Stokesâ shift in the U state which significantly blue shifts in the folded state at all of the pH, indicating that C34-IAEDANS is buried in the hydrophobic core in the N state. Similarly, at all of the pH, the N states exhibited a larger fluorescence lifetime (â¼19 ns) compared to the U states (â¼15 ns) in the time-resolved fluorescence measurements (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>F) further supporting the burial of C34-IAEDANS in the pH range 5â8. In previous studies, <a onclick="showRef(event, 'ref40 ref45'); return false;" href="javascript:void(0);" class="ref ref40 ref45">(40,45)</a> we have measured the solvent accessibility of C34-IAEDANS in the pH range 2â11, using dynamic fluorescence quenching experiments with acrylamide as the quencher. The quenching experiments suggested that C34-IAEDANS is similarly buried in the protein structure at all of the pH measured. The results of the Stokesâ shift and the fluorescence lifetime experiments in the current study support our previous observations.</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> REES as a Reporter of Heterogeneity and Dynamics of the Protein Core</h3><div class="NLM_p">In order to understand the dynamics of core residues in the N states, we excited the HSA-IAEDANS at different wavelengths from the center to the red edge of the absorbance spectrum of C34-IAEDANS (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A). We observed that Î»<sub class="stack">max</sub><sup class="stack">em</sup> varies as a function of Î»<sup>ex</sup> in the N state ensemble (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>B). For example, the representative fluorescence emission spectra at pH 7, as shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>B, displayed a Î»<sub class="stack">max</sub><sup class="stack">em</sup> of 469 nm when excited at 337 nm. However, upon excitation at 410 nm, Î»<sub class="stack">max</sub><sup class="stack">em</sup> shifts by 6 nm to 475 nm. Interestingly, this effect was not observed in the U state (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>B: inset) where the observed Î»<sub class="stack">max</sub><sup class="stack">em</sup> remained at â¼483 nm irrespective of any excitation between 337 and 410 nm. When we plotted the Î»<sub class="stack">max</sub><sup class="stack">em</sup> as a function of Î»<sup>ex</sup> at different pHs, we observed that the N state ensemble showed a typical parabolic curve (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>C). When excited from the center up to the red edge (in the wavelength range of 337â370 nm) of the absorbance spectrum, Î»<sub class="stack">max</sub><sup class="stack">em</sup> remained unchanged. Upon exciting at different wavelengths on the red edge (in the wavelength range of 371â410 nm), the Î»<sub class="stack">max</sub><sup class="stack">em</sup> steeply shifted toward red. However, this phenomenon was absent in the U states at different pHs (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>C) where only a very small change in Î»<sub class="stack">max</sub><sup class="stack">em</sup> (1â2 nm) as a function of Î»<sup>ex</sup> was observed (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>D). In the U forms, the protein structure is disrupted and C34-IAEDANS is solvated by bulk water molecules (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>E,F). Because the solvation environment is highly dynamic in this case and the timescale of solvent relaxation (â¼picosecond) is faster than the fluorescence timescale (â¼nanosecond), very little REES is observed (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). In contrast, in the N forms, C34-IAEDANS is buried in the protein core and is solvated by the dipolar side chains of the protein matrix (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B). The high extent of the REES observed in N forms (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>C,D) indicates that the photoexcitation at the red edge selectively excites a subpopulation of protein molecules in which the fluorophore has a different solvation environment than the mean of the population distribution (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>), and the side chains solvating the fluorophore reorient themselves on a timescale slower than the â¼19 ns fluorescence lifetime of C34-IAEDANS. As C34-IAEDANS is majorly solvated by the side chains of buried polar residues (Q33, D38, H39, Y84, Y140, and R144) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B), these results indicate that in the N forms, the orientations of these side-chain residues are different in the two population. Hence, our results suggest that the core packing of HSA is heterogeneous and the side chains solvating C34-IAEDANS have relatively restricted motions, similar to a viscous dense liquid.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_0003.gif" alt="" data-lg-src="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0003.jpeg" id="gr3" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Core dynamics and heterogeneity of HSA as revealed by REES. (A) Molar absorptivity coefficient spectrum, Îµ (Î»), of C34-IAEDANS is shown in panel (A). The absorption spectrum has been divided by the molar protein concentration to obtain Îµ (Î»). (B) Fluorescence emission spectra of C34-IAEDANS when excited at different wavelengths in the red-edge regions in the N and the U states at pH 7 are shown in panel (B) and its inset, respectively. (C) Panel (C) shows the dependence of the wavelength of maximum fluorescence emission (Î»<sub class="stack">max</sub><sup class="stack">em</sup>) on the wavelength of excitation (Î»<sup>ex</sup>) of C34-IAEDANS in the N and the U states at pHs 5, 7, and 8 and in the presence of 50% glycerol at pH 7. (D) Extent of REES as observed in C34-IAEDANS at all of the U and N states at pHs 5, 7, and 8 and in the presence of 50% glycerol at pH 7.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jpcb.8b11151&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Observed REES of C34-IAEDANS Is Independent of the Viscosity of the Exterior Solvent</h3><div class="NLM_p last">One possible reason for the observed REES of C34-IAEDANS is that it might be measuring the slow dynamics of one or few ordered water molecules around it. This is, however, not the case because of the following two reasons: (i) in the crystal structure of HSA, no water molecules could be seen near C34 which is present in the core and (ii) most importantly, if the REES is due to the interfacial water molecules, then the extent of REES will depend upon the nature and viscosity of the exterior solvent (water in this case), as has been shown previously. <a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> We examined this possibility by changing the viscosity and polarity of the exterior solvent with 50% glycerol/pH 7 buffer mixture in the N state. We observed that the Î»<sub class="stack">max</sub><sup class="stack">em</sup> as a function of Î»<sup>ex</sup> showed a similar parabolic trend in both the presence and the absence of 50% glycerol in the external solvent (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>C). The extent of REES remained similar (â¼6 nm) in the absence and the presence of 50% glycerol (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>D). This result strongly indicates that the observed REES is due to the constrained environment of the protein matrix and not because of the dynamics of the interfacial water molecules around C34-IAEDANS.</div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> REES of a Partially Solvent Exposed Residue, W214, Depends upon the Viscosity of the Exterior Solvent</h3><div class="NLM_p">The sole tryptophan residue, W214, of HSA is located at the interface of domain I and domain II and is partially exposed (â¼45%) to the exterior solvent as revealed by the crystal structure (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A). In previous studies, we <a onclick="showRef(event, 'ref11 ref40'); return false;" href="javascript:void(0);" class="ref ref11 ref40">(11,40)</a> and others <a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> have shown that W214 shows a small but distinct amount of REES because of the presence of ordered water molecules around it at the proteinâwater interface. We took the case of W214 as a positive control to check whether the REES due to interfacial water molecules could be modulated by the change in the viscosity and polarity of the solvent. For W214, in the N state at pH 7, the extent of REES was â¼2 nm in response to the excitation from 295 to 305 nm (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A,B). However, upon changing the exterior solvent to the 50% glycerol/pH 7 buffer mixture, the extent of REES dramatically increased to â¼6 nm for the same excitation range (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>C,D). These results show that the amount of REES is markedly perturbed by changing the solvent to 50% glycerol/pH 7 buffer mixture for a fluorophore near the solvent-exposed surface of the protein. Hence, the absence of any modulation of the extent of REES in the case of C34-IAEDANS, by increasing the viscosity of exterior solvent, indicates that the solvation environment of C34-IAEDANS is the protein matrix and not water, and the observed REES is due to the heterogeneity in the orientations of the protein side chains and their slower dipolar relaxation compared to the fluorescence timescale.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_0004.gif" alt="" data-lg-src="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0004.jpeg" id="gr4" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Change in viscosity of exterior solvent modulates the REES of W214 (A) the molar absorptivity coefficient spectrum, Îµ (Î») of W214. (B) Representative fluorescence emission spectra of W214 at pH 7 when excited at different wavelengths of excitation at the red edge of its absorbance spectrum. (C) Panel (C) shows the representative fluorescence emission spectra of W214 in the presence of 50% glycerol at pH 7 upon excitation at different wavelengths of red edge of its absorbance spectrum. For comparison, all of the spectra were normalized to 1 at their emission maximum. (D) Dependence of Î»<sub class="stack">max</sub><sup class="stack">em</sup> on Î»<sup>ex</sup> of W214 in the N state and in the presence of 50% glycerol at pH 7.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jpcb.8b11151&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> pH as a Modulator of Heterogeneity of the Core</h3><div class="NLM_p last">Although the extent of REES for C34-IAEDANS remained same at all of the pH for the N forms (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>D), the functional dependence with respect to pH appeared to be marginally different (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>C). One possible reason for this could be the heterogeneity in the N state ensemble. As discussed above and reported earlier, <a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> pH is known to modulate the equilibrium between different DMG-like near-native states which together form the N state ensemble. The changes in structural heterogeneity due to the side-chain packing of neighboring residues around the fluorophores, at different pHs, could give rise to this marginally different functional dependence. Another factor that might contribute here is the p<i>K</i><sub>a</sub> of neighboring polar and charged residues. The C34-IAEDANS which is the reporter of the protein core is being solvated mostly by the side chains of charged and polar residues, namely, Q33, D38, H39, Y84, Y140, and R144. These residues might have a different p<i>K</i><sub>a</sub> in the buried form in the N state than that in the water-exposed unfolded state. D38 and H39 could possibly titrate in the pH range 5â8 and that could be another reason for the marginally different trends in REES observed for the N state ensemble at the different pHs.</div></div><div id="sec3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> HSA Has a Compact and Highly Helical Secondary Structure but a Flexible and Loosely Packed Tertiary Structure</h3><div class="NLM_p">Above results indicate that side-chain residues inside the hydrophobic core of HSA are heterogeneously oriented with possibly alternate packing arrangements and that the population distribution might be modulated by pH. <a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> We observed that this heterogeneous distribution of side-chain packing does not loosen and expand the N forms enough to allow the penetration of water molecules inside the protein core. HSA has 18 tyrosine residues distributed all over its 3 domains along with the sole tryptophan residue W214. The absorbance spectrum of tryptophan overlaps with the emission spectrum of tyrosine, and they form a fluorescence resonance energy transfer (FRET) pair. <a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A shows the fluorescence spectra of HSA in the N and the U forms in the pH range 5â8, after excitation at 280 nm that excites both the tyrosine and tryptophan residues. The U form, at each pH, exhibited similar fluorescence spectra with two peaks with Î»<sub class="stack">max</sub><sup class="stack">em</sup> at 300 and 350 nm, corresponding to the emission because of the tyrosine and tryptophan residues, respectively. In contrast, the N forms at each pH exhibited similar fluorescence spectra, with the single peak with Î»<sub class="stack">max</sub><sup class="stack">em</sup> at 340 nm because of the emission of W214 buried in the protein structure. The peak at 300 nm due to the tyrosine residues is absent as the fluorescence emission of tyrosine residues is heavily quenched due to the spatial proximity to W214 in the N form. <a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> These results suggest that at all of the pH the structure in the N form is compact. Site-specific FRET experiments in a previous study also showed similar results. <a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The observation that the Î»<sub class="stack">max</sub><sup class="stack">em</sup> of W214 in the N form is similar at each pH and shifted to blue by â¼10 nm compared to the U form indicate that the core is not substantially hydrated in the alternatively packed N forms and it behaves like a DMG.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_0005.gif" alt="" data-lg-src="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0005.jpeg" id="gr5" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Global structural properties of HSA. (A) Fluorescence emission spectra of the protein at pHs 5, 7, and 8 in both the N and the U forms, when excited at 280 nm is shown. (B) Panel (B) and its inset show changes in MRE with respect to wavelength as a measure of global secondary and tertiary structures, respectively, in the N and the U forms at pHs 5, 7, and 8.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jpcb.8b11151&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We observed that HSA has a highly helical but a loosely packed three-dimensional structure. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B shows the far-UV CD spectrum of HSA in the N and U forms in the pH range 5â8. The N forms at pH each have similar far-UV CD spectra with two minima at 208 and 222 nm. The value of MRE at 208 nm is â22149 Â± 913 deg cm<sup>2</sup> dmol<sup>â1</sup> and that at 222 nm is â20331 Â± 833 deg cm<sup>2</sup> dmol<sup>â1</sup>, indicating a highly helical structure. <a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The inset in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B shows the near-UV CD spectra of HSA in the N and the U forms in the pH range 5â8. HSA absorbs weakly in the near-UV region, and its near-UV CD spectrum, at each pH, is similar and shows absorbance bands mainly in the 255â270 nm region because of the asymmetric packing of its 31 phenylalanine residues in the hydrophobic core. The absorbance in the 275â295 nm region due to the tyrosine and tryptophan residues is almost negligible. The value of MRE at 261 nm in the N form is â121 Â± 3 deg cm<sup>2</sup> dmol<sup>â1</sup> that is only slightly higher than that in the U form (â72 Â± 10 deg cm<sup>2</sup> dmol<sup>â1</sup>). The low values of MRE in the near-UV CD spectrum indicate that the three-dimensional structure of HSA is loosely packed. Hence, these results indicate that HSA has a highly helical structure but a flexible and loose tertiary structure. The loosely packed structure of HSA is likely to give rise to the heterogeneous distribution of side-chain packing indicated by the REES experiments.</div></div><div id="sec3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> N State Ensemble of HSA Contains a Number of DMG-Like States</h3><div class="NLM_p last">The N state ensemble of HSA has a flexible structure (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B: inset), and it contains sub-population of protein molecules in which the orientation and packing of some of the side-chain residues in the protein core are different than the mean population (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> and <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). These results indicate that the N state ensemble of HSA contains multiple DMG-like states. Although DMG-like states have been originally postulated to be hypothetical transition states, <a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> they have been observed as partially unfolded intermediate states on the native side of the free-energy barrier of some proteins, both in simulations <a onclick="showRef(event, 'ref10 ref50'); return false;" href="javascript:void(0);" class="ref ref10 ref50">(10,50)</a> and in experiments. <a onclick="showRef(event, 'ref14 ref15 ref16 ref40 ref45 ref51 ref52 ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref40 ref45 ref51 ref52 ref53 ref54">(14â16,40,45,51â54)</a> It has, however, been challenging to detect them in the N-state ensemble. Notably, DMG-like states have been observed in the N-state ensemble of the small protein HP35 by the tripletâtriplet energy transfer method. <a onclick="showRef(event, 'ref8 ref17'); return false;" href="javascript:void(0);" class="ref ref8 ref17">(8,17)</a> Our observation that N state remains in equilibrium with DMG-like conformations with heterogeneous side-chain orientations might be useful for multiple functions of HSA. <a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> For example, HSA performs a multitude of functions in which binding and transportation of ligands hold maximum significance. The ligands are of a variety of chemical nature, shape, and size. The presence of multiple high energy binding sites could facilitate the binding by shifting the N â DMG equilibrium by preferentially stabilizing one particular high-energy state with respect to others. We observed that the side chains solvating C34-IAEDANS relax on a nanoscale or slower timescale. It is possible that many of these small fluctuations of side chains could couple to give large thermodynamic fluctuations required for conformational conversion and subsequent allosteric and entropy-driven functions. <a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> These results also indicate that the core of HSA is like a molten dense liquid.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78156" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78156" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i16" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i28" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><div class="NLM_p last">In summary, in this study, we explored the heterogeneity and dynamics of side-chain packing in the hydrophobic core of the native state ensemble of HSA using REES, a spectroscopic method that is well developed to study the dynamics of solvent molecules. Crystal structure and solvent accessibility calculations showed that C34 is buried in the protein structure and surrounded by side chains of polar and charged amino acid residues. We labeled C34 with a highly sensitive fluorescent dye 1,5-IAEDANS. The immediate solvation environment of C34-IAEDANS is the side chains of the polar and charged residues buried in the protein core that is well supported by fluorescence Stokesâ shift and lifetime studies. REES experiments on HSA-IAEDANS revealed that the Î»<sub class="stack">max</sub><sup class="stack">em</sup> varies in a seemingly parabolic fashion as a function of Î»<sup>ex</sup> in the N state ensemble, whereas no such functional dependence was observed in the U forms. The extent of REES for C34-IAEDANS in the N state ensemble was observed to be independent of the viscosity of the exterior solvent. Global structural probes showed that the protein maintains a highly helical structure, but the aromatic side-chain residues in the core are loosely packed and the core is devoid of water molecules. All of these observations indicated that the side-chain residues inside the core have a heterogeneous distribution of spatial arrangement in the N state ensemble. These side-chain residues fluctuate much slower than the fluorescence timescale of C34-IAEDANS, and hence the core behaves like a molten dense liquid. These results together suggest that the N state of HSA is an ensemble of DMG-like states. HSA binds to a variety of ligands of different molecular structures and interconversion between different DMG-like states in the N state ensemble could be crucial to perform its functions.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jpcb.8b11151" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget" style="width: 876.3px; display: block;"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15243" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15243" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i16" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i28" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Santosh Kumar Jha</span> - <span class="hlFld-Affiliation affiliation">Physical and Materials Chemistry Division,
Academy of Scientific and Innovative Research (AcSIR), CSIR-National Chemical Laboratory, Dr. Homi Bhabha Road, Pune 411008, Maharashtra, India</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0003-1339-7409" title="Orcid link">http://orcid.org/0000-0003-1339-7409</a>;&nbsp;
    </span><span class="author-email">
                Email: <a href="mailto:sk.jha@ncl.res.in">sk.jha@ncl.res.in</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Prajna Mishra</span> - <span class="hlFld-Affiliation affiliation">Physical and Materials Chemistry Division,
Academy of Scientific and Innovative Research (AcSIR), CSIR-National Chemical Laboratory, Dr. Homi Bhabha Road, Pune 411008, Maharashtra, India</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00787" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00787" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i16" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i28" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr><p class="last">This work was funded by DST-SERB early career research award (project # ECR/2015/000027) to S.K.J. P.M. is a recipient of Senior Research Fellowship by University Grants Commission, India.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47663" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47663" class="dropBlock__holder"><ul><li><a href="#Abstract" class="internalNav" title="Abstract"><span>Abstract</span></a></li><li><a href="#_i2" class="internalNav" title="Introduction"><span>Introduction</span></a></li><li><a href="#_i3" class="internalNav" title="Materials and Methods"><span>Materials and Methods</span></a></li><li><a href="#_i16" class="internalNav" title="Results and Discussion"><span>Results and Discussion</span></a></li><li><a href="#_i28" class="internalNav" title="Conclusions"><span>Conclusions</span></a></li><li><a href="#authorInformationSection" class="internalNav" title="Author Information"><span>Author Information</span></a></li><li><a href="#_i29" class="internalNav" title="Acknowledgments"><span>Acknowledgments</span></a></li><li><a href="#_i30" class="internalNav" title="References"><span>References</span></a></li></ul></div></div></div></div></div><hr></div><p class="references-count">
            This article references 54 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">SchrÃ¶dinger, E.</span></span> <span> </span><span class="NLM_article-title">What is Life?</span> <i>The Physical Aspect of the Living Cell</i>; <span class="NLM_publisher-name">The University press; The Macmillan company</span>: <span class="NLM_publisher-loc">Cambridge England</span>, <span class="NLM_year">1945</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1945&amp;author=E.+Schr%C3%B6dinger&amp;title=The+Physical+Aspect+of+the+Living+Cell" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6dinger%26aufirst%3DE.%26atitle%3DWhat%2520is%2520Life%253F%26btitle%3DThe%2520Physical%2520Aspect%2520of%2520the%2520Living%2520Cell%26pub%3DThe%2520University%2520press%253B%2520The%2520Macmillan%2520company%26date%3D1945" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitkup, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, M.</span></span> <span> </span><span class="NLM_article-title">Native proteins are surface-molten solids: application of the lindemann criterion for the solid versus liquid state 1 1Edited by A. R. Fersht</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">1371</span>â <span class="NLM_lpage">1375</span>, <span class="refDoi">&nbsp;DOI: 10.1006/jmbi.1998.2374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1006%2Fjmbi.1998.2374" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=285&amp;publication_year=1999&amp;pages=1371-1375&amp;author=Y.+Zhouauthor=D.+Vitkupauthor=M.+Karplus&amp;title=Native+proteins+are+surface-molten+solids%3A+application+of+the+lindemann+criterion+for+the+solid+versus+liquid+state+1+1Edited+by+A.+R.+Fersht&amp;doi=10.1006%2Fjmbi.1998.2374" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1998.2374&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1998.2374%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DVitkup%26aufirst%3DD.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DNative%2520proteins%2520are%2520surface-molten%2520solids%253A%2520application%2520of%2520the%2520lindemann%2520criterion%2520for%2520the%2520solid%2520versus%2520liquid%2520state%25201%25201Edited%2520by%2520A.%2520R.%2520Fersht%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1999%26volume%3D285%26spage%3D1371%26epage%3D1375%26doi%3D10.1006%2Fjmbi.1998.2374" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geissler, P. L.</span></span> <span> </span><span class="NLM_article-title">Extensive Conformational Heterogeneity within Protein Cores</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">6417</span>â <span class="NLM_lpage">6423</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jp4105823</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp4105823" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFylurs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=118&amp;publication_year=2014&amp;pages=6417-6423&amp;author=G.+R.+Bowmanauthor=P.+L.+Geissler&amp;title=Extensive+Conformational+Heterogeneity+within+Protein+Cores&amp;doi=10.1021%2Fjp4105823" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Extensive conformational heterogeneity within protein cores</span></div><div class="casAuthors">Bowman, Gregory R.; Geissler, Phillip L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6417-6423</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Basic principles of statistical mechanics require that proteins sample an ensemble of conformations at any non-zero temp.  However, it is still common to treat the crystallog. structure of a protein as the structure of its native state, largely because high-resoln. structural characterization of protein flexibility remains a profound challenge.  To assess the typical degree of conformational heterogeneity within folded proteins, the authors constructed Markov state models describing the thermodn. and kinetics of proteins ranging from 72 to 263 residues in length.  Each of these models was built from hundreds of microseconds of atomically detailed mol. dynamics simulations.  Examn. of the side-chain degrees of freedom revealed that almost every residue visited at least 2 rotameric states over this time frame, with rotamer transition rates spanning a wide range of time scales (from nanoseconds to tens of microseconds).  The authors also reported substantial backbone dynamics on time scales longer than were typically addressed by exptl. measures of protein flexibility, such as NMR order parameters.  Finally, the authors demonstrated that these extensive rearrangements were consistent with NMR and crystallog. data, which supported the validity of their models.  Altogether, these results depict the interior of proteins not as well-ordered solids, as is often imagined, but instead as dense fluids, which undergo substantial structural fluctuations despite their high packing fraction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe_1No4CoVqrVg90H21EOLACvtfcHk0lixcmo-5ztuig"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXivFylurs%253D&amp;md5=d7e368b268bd1406f126d77728c33dd3</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjp4105823&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp4105823%26sid%3Dliteratum%253Aachs%26aulast%3DBowman%26aufirst%3DG.%2BR.%26aulast%3DGeissler%26aufirst%3DP.%2BL.%26atitle%3DExtensive%2520Conformational%2520Heterogeneity%2520within%2520Protein%2520Cores%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2014%26volume%3D118%26spage%3D6417%26epage%3D6423%26doi%3D10.1021%2Fjp4105823" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DuBay, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geissler, P. L.</span></span> <span> </span><span class="NLM_article-title">Fluctuations within Folded Proteins: Implications for Thermodynamic and Allosteric Regulation</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1098</span>â <span class="NLM_lpage">1105</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar500351b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar500351b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFSqt7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=48&amp;publication_year=2015&amp;pages=1098-1105&amp;author=K.+H.+DuBayauthor=G.+R.+Bowmanauthor=P.+L.+Geissler&amp;title=Fluctuations+within+Folded+Proteins%3A+Implications+for+Thermodynamic+and+Allosteric+Regulation&amp;doi=10.1021%2Far500351b" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Fluctuations within folded proteins: Implications for thermodynamic and allosteric regulation</span></div><div class="casAuthors">DuBay, Kateri H.; Bowman, Gregory R.; Geissler, Phillip L.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1098-1105</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Folded protein structures are both stable and dynamic.  Historically, the clearest window into these structures came from x-ray crystallog., which generally provided a static image of each protein's singular "folded state", highlighting its stability.  Deviations away from that crystallog. structure were difficult to quantify, and as a result, their potential functional consequences were often neglected.  However, several dynamical and statistical studies now highlight the structural variability that is present within the protein's folded state.  Here, the authors review mounting evidence of the importance of these structural rearrangements; both expt. and computation indicate that folded proteins undergo substantial fluctuations that can greatly influence their function.  Crucially, recent studies have shown that structural elements of proteins, esp. their side-chain degrees of freedom, fluctuate in ways that generate significant conformational heterogeneity.  The entropy assocd. with these motions contributes to the folded structure's thermodn. stability.  In addn., since these fluctuations can shift in response to perturbations such as ligand binding, they may play an important role in the protein's capacity to respond to environmental cues.  In one compelling example, the entropy assocd. with side-chain fluctuations contributes significantly to regulating the binding of calmodulin to a set of peptide ligands.  The neglect of fluctuations within protein native states was often justified by the dense packing within folded proteins, which has inspired comparisons with cryst. solids.  Many liqs., however, can achieve similarly dense packing yet fluidity is maintained through correlated mol. motions.  Indeed, the studies discussed here favor comparison of folded proteins not with solids but instead with dense liqs., where the internal side-chain fluidity is facilitated by collective motions that are correlated over long distances.  These correlated rearrangements can enable allosteric communication between different parts of a protein, through subtle and varied channels.  Such long-range correlations appear to be an innate feature of proteins in general, manifest even in mols. lacking known allosteric regulators and arising robustly from the phys. nature of their internal environment.  Given their ubiquity, it is only to be expected that, over time, Nature has refined some subset of these correlated motions and put them to use.  Native state fluctuations increasingly appear to be vital for proteins' natural functions.  Understanding the diversity, origin, and range of these rearrangements may provide novel routes for rationally manipulating biomol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJMobE8ZDwRLVg90H21EOLACvtfcHk0ljC43121qo3Jw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXivFSqt7w%253D&amp;md5=cbe612ab9f1f4fdf930d916f05531076</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Far500351b&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far500351b%26sid%3Dliteratum%253Aachs%26aulast%3DDuBay%26aufirst%3DK.%2BH.%26aulast%3DBowman%26aufirst%3DG.%2BR.%26aulast%3DGeissler%26aufirst%3DP.%2BL.%26atitle%3DFluctuations%2520within%2520Folded%2520Proteins%253A%2520Implications%2520for%2520Thermodynamic%2520and%2520Allosteric%2520Regulation%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26spage%3D1098%26epage%3D1105%26doi%3D10.1021%2Far500351b" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richards, F.
M.</span></span> <span> </span><span class="NLM_article-title">The Interpretation of Protein Structures: Total Volume, Group Volume Distributions and Packing Density</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">14</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0022-2836(74)90570-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2F0022-2836%2874%2990570-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=4818482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaE2cXkvVKqsL4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=82&amp;publication_year=1974&amp;pages=1-14&amp;author=F.%0AM.+Richards&amp;title=The+Interpretation+of+Protein+Structures%3A+Total+Volume%2C+Group+Volume+Distributions+and+Packing+Density&amp;doi=10.1016%2F0022-2836%2874%2990570-1" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Interpretation of protein structures.  Total volume, group volume distributions, and packing density</span></div><div class="casAuthors">Richards, Frederic M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1974</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-14</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Vols. occupied by all the atoms or groups of atoms in lysozyme [9001-63-2] and RNase S [9001-99-4] were estd. from at. coordinates provided by crystal structure studies.  The av. packing d. for the interior of both proteins was â¼0.75 and the mean packing d. for all at. types was 0.7-0.8.  Packing d. averaged over 5-15 atoms varied in different parts of the same protein; low d. representing packing defects may facilitate mobility, e.g. at active sites, whereas surrounding high d. areas may be involved in transmission or correlation of motions over large mol. areas.  The total vol. of the 2 enzymes as derived from the coordinate list were 7-10% larger than those estd. from partial sp. vols.  Attributing this vol. difference to a change in packing of H2O in a monolayer surrounding the protein corresponds to an av. decrease of 1-2 Ã3/H2O mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfkJ9sxXcPJbVg90H21EOLACvtfcHk0ljC43121qo3Jw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaE2cXkvVKqsL4%253D&amp;md5=e4c9e4c66cbee1e390b9ea52851c1a82</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2874%2990570-1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252874%252990570-1%26sid%3Dliteratum%253Aachs%26aulast%3DRichards%26aufirst%3DF.%2BM.%26atitle%3DThe%2520Interpretation%2520of%2520Protein%2520Structures%253A%2520Total%2520Volume%252C%2520Group%2520Volume%2520Distributions%2520and%2520Packing%2520Density%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1974%26volume%3D82%26spage%3D1%26epage%3D14%26doi%3D10.1016%2F0022-2836%2874%2990570-1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chothia, C.</span></span> <span> </span><span class="NLM_article-title">Structural Invariants in Protein Folding</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1975</span>,  <span class="NLM_volume"><i>254</i></span>,  <span class="NLM_fpage">304</span>â <span class="NLM_lpage">308</span>, <span class="refDoi">&nbsp;DOI: 10.1038/254304a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2F254304a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1118010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaE2MXktl2js7s%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=254&amp;publication_year=1975&amp;pages=304-308&amp;author=C.+Chothia&amp;title=Structural+Invariants+in+Protein+Folding&amp;doi=10.1038%2F254304a0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structural invariants in protein folding</span></div><div class="casAuthors">Chothia, Cyrus</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">254</span>
        (<span class="NLM_cas:issue">5498</span>),
    <span class="NLM_cas:pages">304-8</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Anal. for 15 proteins of structural descriptions directly related to energy terms showed that protein structures can be described by simple equations or expressions.  The loss of accessible surface area by monomeric proteins on folding was a simple function of mol. wt., the proportion of polar groups forming intramol. H-bonds was const., and protein interiors were closely packed, each residue occupying the same vol. as its does in crystals of amino acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa_GFhyFCinLVg90H21EOLACvtfcHk0ljC43121qo3Jw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaE2MXktl2js7s%253D&amp;md5=aa04ff07cd5f1612d50cdb5cc8430afc</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2F254304a0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F254304a0%26sid%3Dliteratum%253Aachs%26aulast%3DChothia%26aufirst%3DC.%26atitle%3DStructural%2520Invariants%2520in%2520Protein%2520Folding%26jtitle%3DNature%26date%3D1975%26volume%3D254%26spage%3D304%26epage%3D308%26doi%3D10.1038%2F254304a0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richards, F. M.</span></span> <span> </span><span class="NLM_article-title">Areas, Volumes, Packing, and Protein Structure</span>. <i>Annu. Rev. Biophys. Bioeng.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">151</span>â <span class="NLM_lpage">176</span>, <span class="refDoi">&nbsp;DOI: 10.1146/annurev.bb.06.060177.001055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1146%2Fannurev.bb.06.060177.001055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=326146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaE2sXks1CisrY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=1977&amp;pages=151-176&amp;author=F.+M.+Richards&amp;title=Areas%2C+Volumes%2C+Packing%2C+and+Protein+Structure&amp;doi=10.1146%2Fannurev.bb.06.060177.001055" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Areas, volumes, packing, and protein structure</span></div><div class="casAuthors">Richards, Frederic M.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Bioengineering</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">151-76</span>CODEN:
                <span class="NLM_cas:coden">ABPBBK</span>;
        ISSN:<span class="NLM_cas:issn">0084-6589</span>.
    </div><div class="casAbstract">A review with 148 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvHGphzlx3JbVg90H21EOLACvtfcHk0lgH09akl3T77w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaE2sXks1CisrY%253D&amp;md5=13c917ccbda3f36f1c6390a42c712818</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1146%2Fannurev.bb.06.060177.001055&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.bb.06.060177.001055%26sid%3Dliteratum%253Aachs%26aulast%3DRichards%26aufirst%3DF.%2BM.%26atitle%3DAreas%252C%2520Volumes%252C%2520Packing%252C%2520and%2520Protein%2520Structure%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Bioeng.%26date%3D1977%26volume%3D6%26spage%3D151%26epage%3D176%26doi%3D10.1146%2Fannurev.bb.06.060177.001055" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reiner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henklein, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefhaber, T.</span></span> <span> </span><span class="NLM_article-title">An Unlocking/Relocking Barrier in Conformational Fluctuations of Villin Headpiece Subdomain</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">4955</span>â <span class="NLM_lpage">4960</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.0910001107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1073%2Fpnas.0910001107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=20194774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFGntrg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=107&amp;publication_year=2010&amp;pages=4955-4960&amp;author=A.+Reinerauthor=P.+Henkleinauthor=T.+Kiefhaber&amp;title=An+Unlocking%2FRelocking+Barrier+in+Conformational+Fluctuations+of+Villin+Headpiece+Subdomain&amp;doi=10.1073%2Fpnas.0910001107" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">An unlocking/relocking barrier in conformational fluctuations of villin headpiece subdomain</span></div><div class="casAuthors">Reiner, Andreas; Henklein, Peter; Kiefhaber, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4955-4960, S4955/1-S4955/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A reversible structural unlocking reaction, in which the close-packed van der Waals interactions break cooperatively, has been found for the villin headpiece subdomain (HP35) using triplet-triplet-energy transfer to monitor conformational fluctuations from equil.  Unlocking is assocd. with an unfavorable enthalpy change (ÎH0 = 35 Â± 4 kJ/mol) which is nearly compensated in free energy by the entropy change (ÎS0 = 112 Â± 20 J/mol-1/K-1).  The unlocking reaction has a time const. of about 1 Î¼s at 5Â° and is enthalpy-limited with an activation energy of 32 Â± 1 kJ/mol and a large Arrhenius preexponential factor of A = 7.5 Ã 1011 s-1.  In the unlocked state a fast local conformational fluctuation with a time const. of 170 ns and a low activation barrier of 17 Â± 1 kJ/mol leads to unfolding of the C-terminal helix and to its undocking from the core.  On a much slower time scale, global unfolding occurs from the unlocked state.  These results suggest that native protein structures are locked into conformations with low amplitude motions.  Large scale motions and global unfolding require an initial structural unlocking step leading to a state with properties of a dry molten globule.  The expts. addnl. yielded information on the dynamics of loop formation between different positions in unfolded HP35.  Comparison of the results with dynamics in unstructured model peptides indicates slightly decelerated kinetics of local loop formation in the C-terminal region which points at residual, nonrandom structure.  Dynamics of long-range loop formation, in contrast, are not influenced by residual structure, which argues against unfolded state properties as mol. origin for ultrafast folding of HP35.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSNXIYzC31_7Vg90H21EOLACvtfcHk0lgH09akl3T77w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXjvFGntrg%253D&amp;md5=09e9c15dac4b26c12147179c34fbba9f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0910001107&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0910001107%26sid%3Dliteratum%253Aachs%26aulast%3DReiner%26aufirst%3DA.%26aulast%3DHenklein%26aufirst%3DP.%26aulast%3DKiefhaber%26aufirst%3DT.%26atitle%3DAn%2520Unlocking%252FRelocking%2520Barrier%2520in%2520Conformational%2520Fluctuations%2520of%2520Villin%2520Headpiece%2520Subdomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D4955%26epage%3D4960%26doi%3D10.1073%2Fpnas.0910001107" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frieden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, G. D.</span></span> <span> </span><span class="NLM_article-title">Dry Molten Globule Intermediates and the Mechanism of Protein Unfolding</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">2725</span>â <span class="NLM_lpage">2737</span>, <span class="refDoi">&nbsp;DOI: 10.1002/prot.22803</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1002%2Fprot.22803" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=78&amp;publication_year=2010&amp;pages=2725-2737&amp;author=R.+L.+Baldwinauthor=C.+Friedenauthor=G.+D.+Rose&amp;title=Dry+Molten+Globule+Intermediates+and+the+Mechanism+of+Protein+Unfolding&amp;doi=10.1002%2Fprot.22803" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fprot.22803&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22803%26sid%3Dliteratum%253Aachs%26aulast%3DBaldwin%26aufirst%3DR.%2BL.%26aulast%3DFrieden%26aufirst%3DC.%26aulast%3DRose%26aufirst%3DG.%2BD.%26atitle%3DDry%2520Molten%2520Globule%2520Intermediates%2520and%2520the%2520Mechanism%2520of%2520Protein%2520Unfolding%26jtitle%3DProteins%26date%3D2010%26volume%3D78%26spage%3D2725%26epage%3D2737%26doi%3D10.1002%2Fprot.22803" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thirumalai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span> <span> </span><span class="NLM_article-title">Protein Folding: From Theory to Practice</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">22</span>â <span class="NLM_lpage">29</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.sbi.2012.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2Fj.sbi.2012.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=23266001" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2ktb7M" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2013&amp;pages=22-29&amp;author=D.+Thirumalaiauthor=Z.+Liuauthor=E.+P.+O%E2%80%99Brienauthor=G.+Reddy&amp;title=Protein+Folding%3A+From+Theory+to+Practice&amp;doi=10.1016%2Fj.sbi.2012.11.010" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Protein folding: from theory to practice</span></div><div class="casAuthors">Thirumalai, D.; Liu, Zhenxing; O'Brien, Edward P.; Reddy, Govardhan</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-29</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  A quant. theory of protein folding should make testable predictions using theor. models and simulations performed under conditions that closely mimic those used in expts.  Typically, in lab. expts. folding or unfolding is initiated using denaturants or external mech. force, whereas theories and simulations use temp. as the control parameter, thus making it difficult to make direct comparisons with expts.  The mol. transfer model (MTM), which incorporates environmental changes using measured quantities in mol. simulations, overcomes these difficulties.  Predictions of the folding thermodn. and kinetics of a no. of proteins using MTM simulations are in remarkable agreement with expts.  The MTM and all atom simulations demonstrating the presence of dry globules represent major advances in the proteins folding field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4w-0X831BibVg90H21EOLACvtfcHk0lgH09akl3T77w"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhvV2ktb7M&amp;md5=05c84df577495f29de7ef2c4610d8e4f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2012.11.010&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2012.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DThirumalai%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DE.%2BP.%26aulast%3DReddy%26aufirst%3DG.%26atitle%3DProtein%2520Folding%253A%2520From%2520Theory%2520to%2520Practice%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2013%26volume%3D23%26spage%3D22%26epage%3D29%26doi%3D10.1016%2Fj.sbi.2012.11.010" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span> <span> </span><span class="NLM_article-title">An Alternatively Packed Dry Molten Globule-Like Intermediate in the Native State Ensemble of a Multidomain Protein</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">9336</span>â <span class="NLM_lpage">9347</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.jpcb.7b07032</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jpcb.7b07032" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVyis7bF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=121&amp;publication_year=2017&amp;pages=9336-9347&amp;author=P.+Mishraauthor=S.+K.+Jha&amp;title=An+Alternatively+Packed+Dry+Molten+Globule-Like+Intermediate+in+the+Native+State+Ensemble+of+a+Multidomain+Protein&amp;doi=10.1021%2Facs.jpcb.7b07032" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">An Alternatively Packed Dry Molten Globule-like Intermediate in the Native State Ensemble of a Multidomain Protein</span></div><div class="casAuthors">Mishra, Prajna; Jha, Santosh Kumar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">9336-9347</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">It has been difficult to quantify the degree of side-chain conformational heterogeneity in the native (N) state ensemble of proteins and the relative energetic contributions of the side-chain packing and the hydrophobic effect in protein stability.  Here, we show using multiple site-specific spectroscopic probes and tools of thermodn. that the N state ensemble of the multi-domain protein, human serum albumin, contains an equil. intermediate (I) whose inter-domain region resembles a dry molten globule (DMG).  In the I state, a Trp residue (Trp-214) in the inter-domain region was alternatively packed but its secondary structure and intra-domain packing were N-like.  The I state also had a larger inter-domain distance, but the domain-domain interface was dry and molten.  These results indicated that hydrophobic desolvation and side-chain packing are decoupled during protein folding and that inter-domain packing interactions have an important energetic contribution in protein stability.  Dynamic inter-conversion between alternatively packed N-like states could be important for multiple allosteric and ligand binding functions of this protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXjR8Vs542aLVg90H21EOLACvtfcHk0ljF529yqB7u8A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhsVyis7bF&amp;md5=8b1803a2deff90ca08b7e3c44f20a5dd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jpcb.7b07032&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jpcb.7b07032%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DP.%26aulast%3DJha%26aufirst%3DS.%2BK.%26atitle%3DAn%2520Alternatively%2520Packed%2520Dry%2520Molten%2520Globule-Like%2520Intermediate%2520in%2520the%2520Native%2520State%2520Ensemble%2520of%2520a%2520Multidomain%2520Protein%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2017%26volume%3D121%26spage%3D9336%26epage%3D9347%26doi%3D10.1021%2Facs.jpcb.7b07032" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, G. D.</span></span> <span> </span><span class="NLM_article-title">Molten Globules, Entropy-Driven Conformational Change and Protein Folding</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">4</span>â <span class="NLM_lpage">10</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.sbi.2012.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2Fj.sbi.2012.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=23237704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWgtLnN" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2013&amp;pages=4-10&amp;author=R.+L.+Baldwinauthor=G.+D.+Rose&amp;title=Molten+Globules%2C+Entropy-Driven+Conformational+Change+and+Protein+Folding&amp;doi=10.1016%2Fj.sbi.2012.11.004" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Molten globules, entropy-driven conformational change and protein folding</span></div><div class="casAuthors">Baldwin, Robert L.; Rose, George D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-10</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The exquisite side chain close-packing in the protein core and at binding interfaces has prompted a conviction that packing selectivity is the primary mechanism for mol. recognition in folding and/or binding reactions.  Contrary to this view, molten globule proteins can adopt native topol. and bind targets tightly and specifically in the absence of side chain close-packing.  The molten globule is a highly dynamic form with native-like secondary structure and a loose protein core that admits solvent.  The related (but still controversial) dry molten globule is an expanded form of the native protein with largely intact topol. but a tighter protein core that excludes solvent.  Neither form retains side chain close-packing, and therefore both structure and function must result from other factors, assuming that the reality of the dry molten globule is accepted.  This simplifying realization calls for a re-evaluation of established models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNV9HY3Veq-rVg90H21EOLACvtfcHk0ljF529yqB7u8A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhvVWgtLnN&amp;md5=0c958500dc87edd17bad7e04c0a36a0e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2012.11.004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2012.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DBaldwin%26aufirst%3DR.%2BL.%26aulast%3DRose%26aufirst%3DG.%2BD.%26atitle%3DMolten%2520Globules%252C%2520Entropy-Driven%2520Conformational%2520Change%2520and%2520Protein%2520Folding%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2013%26volume%3D23%26spage%3D4%26epage%3D10%26doi%3D10.1016%2Fj.sbi.2012.11.004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapienza, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petit, C. M.</span></span> <span> </span><span class="NLM_article-title">Native State Volume Fluctuations in Proteins as a Mechanism for Dynamic Allostery</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">3599</span>â <span class="NLM_lpage">3602</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.6b12058</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b12058" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKitLY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=139&amp;publication_year=2017&amp;pages=3599-3602&amp;author=A.+B.+Lawauthor=P.+J.+Sapienzaauthor=J.+Zhangauthor=X.+Zuoauthor=C.+M.+Petit&amp;title=Native+State+Volume+Fluctuations+in+Proteins+as+a+Mechanism+for+Dynamic+Allostery&amp;doi=10.1021%2Fjacs.6b12058" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Native State Volume Fluctuations in Proteins as a Mechanism for Dynamic Allostery</span></div><div class="casAuthors">Law, Anthony B.; Sapienza, Paul J.; Zhang, Jun; Zuo, Xiaobing; Petit, Chad M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3599-3602</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allostery enables tight regulation of protein function in the cellular environment.  While existing models of allostery are firmly rooted in the current structure-function paradigm, the mechanistic basis for allostery in the absence of structural change remains unclear.  In this study, we show that a typical globular protein is able to undergo significant changes in vol. under native conditions while exhibiting no addnl. changes in protein structure.  These native state vol. fluctuations were found to correlate with changes in internal motions that were previously recognized as a source of allosteric entropy.  This finding offers a novel mechanistic basis for allostery in the absence of canonical structural change.  The unexpected observation that function can be derived from expanded, low d. protein states has broad implications for our understanding of allostery and suggests that the general concept of the native state be expanded to allow for more variable phys. dimensions with looser packing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5SsKYemFuR7Vg90H21EOLACvtfcHk0ljF529yqB7u8A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKitLY%253D&amp;md5=07b2b88933154f96fe19cfaac28d5faa</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b12058&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b12058%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DA.%2BB.%26aulast%3DSapienza%26aufirst%3DP.%2BJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZuo%26aufirst%3DX.%26aulast%3DPetit%26aufirst%3DC.%2BM.%26atitle%3DNative%2520State%2520Volume%2520Fluctuations%2520in%2520Proteins%2520as%2520a%2520Mechanism%2520for%2520Dynamic%2520Allostery%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D3599%26epage%3D3602%26doi%3D10.1021%2Fjacs.6b12058" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kiefhaber, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labhardt, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, R. L.</span></span> <span> </span><span class="NLM_article-title">Direct NMR Evidence for an Intermediate Preceding the Rate-Limiting Step in the Unfolding of Ribonuclease A</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>375</i></span>,  <span class="NLM_fpage">513</span>â <span class="NLM_lpage">515</span>, <span class="refDoi">&nbsp;DOI: 10.1038/375513a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2F375513a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=7777063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaK2MXmtFCgsbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=375&amp;publication_year=1995&amp;pages=513-515&amp;author=T.+Kiefhaberauthor=A.+M.+Labhardtauthor=R.+L.+Baldwin&amp;title=Direct+NMR+Evidence+for+an+Intermediate+Preceding+the+Rate-Limiting+Step+in+the+Unfolding+of+Ribonuclease+A&amp;doi=10.1038%2F375513a0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Direct NMR evidence for an intermediate preceding the rate-limiting step in the unfolding of ribonuclease A</span></div><div class="casAuthors">Kiefhaber, Thomas; Labhardt, Alexander M.; Baldwin, Robert L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">375</span>
        (<span class="NLM_cas:issue">6531</span>),
    <span class="NLM_cas:pages">513-15</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">It is commonly believed that there are no detectable intermediates in the kinetic unfolding reactions of small proteins.  If such intermediates could be found, they would give important information about the nature of the transition state for unfolding, which is thought to occur close to the native state.  The authors report here that one-dimensional proton magnetic resonance spectra recorded during the unfolding of RNase A provide direct evidence for at least one unfolding intermediate in which side chains are free to rotate.  This intermediate appears to be a 'dry molten globule' of the kind hypothesized by Shakhnovich and Finkelstein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIHBAF0z1taLVg90H21EOLACvtfcHk0ljnCqLz_KNTeg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXmtFCgsbs%253D&amp;md5=552b2a31424bf7915dc8e8d779bd2856</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2F375513a0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F375513a0%26sid%3Dliteratum%253Aachs%26aulast%3DKiefhaber%26aufirst%3DT.%26aulast%3DLabhardt%26aufirst%3DA.%2BM.%26aulast%3DBaldwin%26aufirst%3DR.%2BL.%26atitle%3DDirect%2520NMR%2520Evidence%2520for%2520an%2520Intermediate%2520Preceding%2520the%2520Rate-Limiting%2520Step%2520in%2520the%2520Unfolding%2520of%2520Ribonuclease%2520A%26jtitle%3DNature%26date%3D1995%26volume%3D375%26spage%3D513%26epage%3D515%26doi%3D10.1038%2F375513a0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udgaonkar, J. B.</span></span> <span> </span><span class="NLM_article-title">Direct Evidence for a Dry Molten Globule Intermediate During the Unfolding of a Small Protein</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">12289</span>â <span class="NLM_lpage">12294</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.0905744106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1073%2Fpnas.0905744106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=19617531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpslCjsrY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=106&amp;publication_year=2009&amp;pages=12289-12294&amp;author=S.+K.+Jhaauthor=J.+B.+Udgaonkar&amp;title=Direct+Evidence+for+a+Dry+Molten+Globule+Intermediate+During+the+Unfolding+of+a+Small+Protein&amp;doi=10.1073%2Fpnas.0905744106" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Direct evidence for a dry molten globule intermediate during the unfolding of a small protein</span></div><div class="casAuthors">Jha, Santosh Kumar; Udgaonkar, Jayant B.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">12289-12294, S12289/1-S12289/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Little is known about how proteins begin to unfold.  In particular, how and when water mols. penetrate into the protein interior during unfolding, thereby enabling the dissoln. of specific structure, is poorly understood.  The hypothesis that the native state expands initially into a dry molten globule (MG), in which tight packing interactions are broken, but whose hydrophobic core has not expanded sufficiently to be able to absorb water mols., has very little exptl. support.  Here, the authors report their anal. of the earliest observable events during the unfolding of single-chain monellin (MNEI), a small plant protein.  Far- and near-UV CD measurements of guanidine-HCl (GdnHCl)-induced unfolding indicated that a MG intermediate forms initially, before the major slow unfolding reaction commences.  Steady-state FRET measurements showed that the C-terminal end of the single helix of MNEI initially moves rapidly away from the single Trp residue that is close to the N-terminal end of the helix.  The av. end-to-end distance of the protein also expanded during unfolding to the MG intermediate.  At this time, water has yet to penetrate the protein core, according to the evidence from intrinsic Trp fluorescence and 8-anilino-1-naphthalenesulfonic acid (ANS) fluorescence-monitored kinetic unfolding measurements.  These results therefore provide direct evidence for a dry MG intermediate at the initial stage of unfolding.  The results further suggest that the structural transition between the native and dry MG states could be an all-or-none transition, whereas further swelling of the globule appears to occur gradually.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvqMIwUtJwkLVg90H21EOLACvtfcHk0ljnCqLz_KNTeg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXpslCjsrY%253D&amp;md5=46d83ec8fc80c09753972cfda8e8830f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0905744106&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0905744106%26sid%3Dliteratum%253Aachs%26aulast%3DJha%26aufirst%3DS.%2BK.%26aulast%3DUdgaonkar%26aufirst%3DJ.%2BB.%26atitle%3DDirect%2520Evidence%2520for%2520a%2520Dry%2520Molten%2520Globule%2520Intermediate%2520During%2520the%2520Unfolding%2520of%2520a%2520Small%2520Protein%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D12289%26epage%3D12294%26doi%3D10.1073%2Fpnas.0905744106" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udgaonkar, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamoorthy, G.</span></span> <span> </span><span class="NLM_article-title">Unfolding of a Small Protein Proceeds via Dry and Wet Globules and a Solvated Transition State</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">2392</span>â <span class="NLM_lpage">2402</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bpj.2013.09.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2Fj.bpj.2013.09.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=24268151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVWktr%252FM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=105&amp;publication_year=2013&amp;pages=2392-2402&amp;author=S.+S.+Sarkarauthor=J.+B.+Udgaonkarauthor=G.+Krishnamoorthy&amp;title=Unfolding+of+a+Small+Protein+Proceeds+via+Dry+and+Wet+Globules+and+a+Solvated+Transition+State&amp;doi=10.1016%2Fj.bpj.2013.09.048" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Unfolding of a Small Protein Proceeds via Dry and Wet Globules and a Solvated Transition State</span></div><div class="casAuthors">Sarkar, Saswata Sankar; Udgaonkar, Jayant B.; Krishnamoorthy, Guruswamy</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2392-2402</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Dissecting a protein unfolding process into individual steps can provide valuable information on the forces that maintain the integrity of the folded structure.  Solvation of the protein core dets. stability, but it is not clear when such solvation occurs during unfolding.  In this study, far-UV CD measurements suggest a simplistic two-state view of the unfolding of barstar, but the use of multiple other probes brings out the complexity of the unfolding reaction.  Near-UV CD measurements show that unfolding commences with the loosening of tertiary interactions in a native-like intermediate, N*.  Fluorescence resonance energy transfer measurements show that N* then expands rapidly but partially to form an early unfolding intermediate IE.  Fluorescence spectral measurements indicate that both N* and IE have retained native-like solvent accessibility of the core, suggesting that they are dry molten globules (DMGs).  Dynamic quenching measurements at the single tryptophan buried in the core suggest that the core becomes solvated only later in a late wet molten globule (WMG), IL, which precedes the unfolded form.  Fluorescence anisotropy decay measurements show that tight packing around the core tryptophan is lost when IL forms.  Of importance, the slowest step is unfolding of the wet molten globule and involves a solvated transition state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgiCdKU2ypLbVg90H21EOLACvtfcHk0ljnCqLz_KNTeg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhvVWktr%252FM&amp;md5=4d8499eaaad75c354d39993947b1a7dc</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2013.09.048&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2013.09.048%26sid%3Dliteratum%253Aachs%26aulast%3DSarkar%26aufirst%3DS.%2BS.%26aulast%3DUdgaonkar%26aufirst%3DJ.%2BB.%26aulast%3DKrishnamoorthy%26aufirst%3DG.%26atitle%3DUnfolding%2520of%2520a%2520Small%2520Protein%2520Proceeds%2520via%2520Dry%2520and%2520Wet%2520Globules%2520and%2520a%2520Solvated%2520Transition%2520State%26jtitle%3DBiophys.%2520J.%26date%3D2013%26volume%3D105%26spage%3D2392%26epage%3D2402%26doi%3D10.1016%2Fj.bpj.2013.09.048" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neumaier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefhaber, T.</span></span> <span> </span><span class="NLM_article-title">Redefining the Dry Molten Globule State of Proteins</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>426</i></span>,  <span class="NLM_fpage">2520</span>â <span class="NLM_lpage">2528</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.jmb.2014.04.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2Fj.jmb.2014.04.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=24792909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvFSrtbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=426&amp;publication_year=2014&amp;pages=2520-2528&amp;author=S.+Neumaierauthor=T.+Kiefhaber&amp;title=Redefining+the+Dry+Molten+Globule+State+of+Proteins&amp;doi=10.1016%2Fj.jmb.2014.04.022" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Redefining the dry molten globule state of proteins</span></div><div class="casAuthors">Neumaier, Sabine; Kiefhaber, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2520-2528</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The dynamics and function of proteins are governed by the structural and energetic properties of the different states they adopt and the barriers sepg. them.  In earlier work, native-state triplet-triplet energy transfer (TTET) on the villin headpiece subdomain (HP35) revealed an equil. between a locked native state and an unlocked native state, which were structurally similar but had different dynamic properties.  The locked state was restricted to low amplitude motions, whereas the unlocked state showed increased conformational flexibility and underwent local unfolding reactions.  This classified the unlocked state as a dry molten globule (DMG), which was proposed to represent an expanded native state with loosened side-chain interactions and a solvent-shielded core.  To test whether the unlocked state of HP35 was actually expanded compared to the locked state, the authors performed high-pressure TTET measurements.  Increasing pressure shifted the equil. from the locked toward the unlocked state, with a small neg. reaction vol. for unlocking (ÎV0 = -1.6 cm3/mol).  Therefore, rather than being expanded, the unlocked state represents an alternatively packed, compact state, demonstrating that native proteins can exist in several compact folded states, an observation with implications for protein function.  The transition state for unlocking/locking, in contrast, had a largely increased vol. relative to the locked and unlocked state, with resp. activation vols. of 7.1 and 8.7 cm3/mol, indicating an expansion of the protein during the locking/unlocking transition.  These results demonstrated the existence of both compact, low-energy and expanded, high-energy DMGs, prompting a broader definition of this state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPZoRM_O5Gd7Vg90H21EOLACvtfcHk0lh5N1ZZ79JS7A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXnvFSrtbc%253D&amp;md5=131afdf155648945519507d8430a31b9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2014.04.022&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2014.04.022%26sid%3Dliteratum%253Aachs%26aulast%3DNeumaier%26aufirst%3DS.%26aulast%3DKiefhaber%26aufirst%3DT.%26atitle%3DRedefining%2520the%2520Dry%2520Molten%2520Globule%2520State%2520of%2520Proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D426%26spage%3D2520%26epage%3D2528%26doi%3D10.1016%2Fj.jmb.2014.04.022" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, A.</span></span> <span> </span><span class="NLM_article-title">Thermodynamic Fluctuations in Protein Molecules</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">2740</span>â <span class="NLM_lpage">2741</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.73.8.2740</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1073%2Fpnas.73.8.2740" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1066687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaE28Xltlyrsb0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=73&amp;publication_year=1976&amp;pages=2740-2741&amp;author=A.+Cooper&amp;title=Thermodynamic+Fluctuations+in+Protein+Molecules&amp;doi=10.1073%2Fpnas.73.8.2740" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamic fluctuations in protein molecules</span></div><div class="casAuthors">Cooper, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2740-1</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Apparently conflicting views of the phys. nature of globular proteins, and other macromols., may be reconciled by consideration of the inevitable thermodn. fluctuations inherent in microscopic systems.  Discrete protein mols., considered singly, undergo sizeable fluctuations in thermodn. properties which are manifest in their stochastic properties.  This is not incompatible with time-averaged studies of ensembles of proteins from which a more compact, rigid, and static view of these mols. may be obtained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp81u_i9pHXRbVg90H21EOLACvtfcHk0lh5N1ZZ79JS7A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaE28Xltlyrsb0%253D&amp;md5=77038ea27f97e4a947b761a3627862f3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.73.8.2740&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.73.8.2740%26sid%3Dliteratum%253Aachs%26aulast%3DCooper%26aufirst%3DA.%26atitle%3DThermodynamic%2520Fluctuations%2520in%2520Protein%2520Molecules%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1976%26volume%3D73%26spage%3D2740%26epage%3D2741%26doi%3D10.1073%2Fpnas.73.8.2740" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henzler-Wildman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, D.</span></span> <span> </span><span class="NLM_article-title">Dynamic Personalities of Proteins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>450</i></span>,  <span class="NLM_fpage">964</span>â <span class="NLM_lpage">972</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature06522</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2Fnature06522" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=18075575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr%252FL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=450&amp;publication_year=2007&amp;pages=964-972&amp;author=K.+Henzler-Wildmanauthor=D.+Kern&amp;title=Dynamic+Personalities+of+Proteins&amp;doi=10.1038%2Fnature06522" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic personalities of proteins</span></div><div class="casAuthors">Henzler-Wildman, Katherine; Kern, Dorothee</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7172</span>),
    <span class="NLM_cas:pages">964-972</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Because proteins are central to cellular function, researchers have sought to uncover the secrets of how these complex macromols. execute such a fascinating variety of functions.  Although static structures are known for many proteins, the functions of proteins are governed ultimately by their dynamic character (or 'personality').  The dream is to 'watch' proteins in action in real time at at. resoln.  This requires the addn. of a fourth dimension, time, to structural biol. so that the positions in space and time of all atoms in a protein can be described in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4dGYWDS9O37Vg90H21EOLACvtfcHk0lh5N1ZZ79JS7A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr%252FL&amp;md5=27ca36106100f487683d714de5335dc2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnature06522&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06522%26sid%3Dliteratum%253Aachs%26aulast%3DHenzler-Wildman%26aufirst%3DK.%26aulast%3DKern%26aufirst%3DD.%26atitle%3DDynamic%2520Personalities%2520of%2520Proteins%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D964%26epage%3D972%26doi%3D10.1038%2Fnature06522" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frederick, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentine, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Conformational Entropy in Molecular Recognition by Proteins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>448</i></span>,  <span class="NLM_fpage">325</span>â <span class="NLM_lpage">329</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature05959</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2Fnature05959" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=17637663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnvVeqsbk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=448&amp;publication_year=2007&amp;pages=325-329&amp;author=K.+K.+Frederickauthor=M.+S.+Marlowauthor=K.+G.+Valentineauthor=A.+J.+Wand&amp;title=Conformational+Entropy+in+Molecular+Recognition+by+Proteins&amp;doi=10.1038%2Fnature05959" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational entropy in molecular recognition by proteins</span></div><div class="casAuthors">Frederick, Kendra King; Marlow, Michael S.; Valentine, Kathleen G.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">448</span>
        (<span class="NLM_cas:issue">7151</span>),
    <span class="NLM_cas:pages">325-329</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mol. recognition by proteins is fundamental to almost every biol. process, particularly the protein assocns. underlying cellular signal transduction.  Understanding the basis for protein-protein interactions requires the full characterization of the thermodn. of their assocn.  Historically, it has been virtually impossible to exptl. est. changes in protein conformational entropy, a potentially important component of the free energy of protein assocn.  However, NMR spectroscopy has emerged as a powerful tool for characterizing the dynamics of proteins.  Here, the authors employed changes in conformational dynamics as a proxy for corresponding changes in conformational entropy.  It was found that the change in internal dynamics of the protein, calmodulin, varied significantly on binding a variety of target domains.  Surprisingly, the apparent change in the corresponding conformational entropy was linearly related to the change in the overall binding entropy.  This indicated that changes in protein conformational entropy can contribute significantly to the free energy of protein-ligand assocn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohP3eEb0Q6srVg90H21EOLACvtfcHk0lh5N1ZZ79JS7A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXnvVeqsbk%253D&amp;md5=99cb195dbee8c4e67e348d353451dc1b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature05959&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05959%26sid%3Dliteratum%253Aachs%26aulast%3DFrederick%26aufirst%3DK.%2BK.%26aulast%3DMarlow%26aufirst%3DM.%2BS.%26aulast%3DValentine%26aufirst%3DK.%2BG.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DConformational%2520Entropy%2520in%2520Molecular%2520Recognition%2520by%2520Proteins%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D325%26epage%3D329%26doi%3D10.1038%2Fnature05959" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dogan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederick, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentine, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">The Role of Conformational Entropy in Molecular Recognition by Calmodulin</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">352</span>â <span class="NLM_lpage">358</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.347</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2Fnchembio.347" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=20383153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFakt78%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2010&amp;pages=352-358&amp;author=M.+S.+Marlowauthor=J.+Doganauthor=K.+K.+Frederickauthor=K.+G.+Valentineauthor=A.+J.+Wand&amp;title=The+Role+of+Conformational+Entropy+in+Molecular+Recognition+by+Calmodulin&amp;doi=10.1038%2Fnchembio.347" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The role of conformational entropy in molecular recognition by calmodulin</span></div><div class="casAuthors">Marlow, Michael S.; Dogan, Jakob; Frederick, Kendra K.; Valentine, Kathleen G.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">352-358</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The phys. basis for high-affinity interactions involving proteins is complex and potentially involves a range of energetic contributions.  Among these are changes in protein conformational entropy, which cannot yet be reliably computed from mol. structures.  We have recently used changes in conformational dynamics as a proxy for changes in conformational entropy of calmodulin upon assocn. with domains from regulated proteins.  The apparent change in conformational entropy was linearly related to the overall binding entropy.  This view warrants a more quant. foundation.  Here we calibrate an 'entropy meter' using an exptl. dynamical proxy based on NMR relaxation and show that changes in the conformational entropy of calmodulin are a significant component of the energetics of binding.  Furthermore, the distribution of motion at the interface between the target domain and calmodulin is surprisingly noncomplementary.  These observations promote modification of our understanding of the energetics of protein-ligand interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmK5VBOAHwT7Vg90H21EOLACvtfcHk0lhi0yek-EbDYw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXksFakt78%253D&amp;md5=e9ee34fffa91f612441a77a7385015ff</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.347&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.347%26sid%3Dliteratum%253Aachs%26aulast%3DMarlow%26aufirst%3DM.%2BS.%26aulast%3DDogan%26aufirst%3DJ.%26aulast%3DFrederick%26aufirst%3DK.%2BK.%26aulast%3DValentine%26aufirst%3DK.%2BG.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DThe%2520Role%2520of%2520Conformational%2520Entropy%2520in%2520Molecular%2520Recognition%2520by%2520Calmodulin%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D352%26epage%3D358%26doi%3D10.1038%2Fnchembio.347" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eisenmesser, E. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosco, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, D.</span></span> <span> </span><span class="NLM_article-title">Enzyme Dynamics During Catalysis</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">1520</span>â <span class="NLM_lpage">1523</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.1066176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1126%2Fscience.1066176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=11859194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD38XhsFyqtbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=295&amp;publication_year=2002&amp;pages=1520-1523&amp;author=E.+Z.+Eisenmesserauthor=D.+A.+Boscoauthor=M.+Akkeauthor=D.+Kern&amp;title=Enzyme+Dynamics+During+Catalysis&amp;doi=10.1126%2Fscience.1066176" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme dynamics during catalysis</span></div><div class="casAuthors">Eisenmesser, Elan Zohar; Bosco, Daryl A.; Akke, Mikael; Kern, Dorothee</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">295</span>
        (<span class="NLM_cas:issue">5559</span>),
    <span class="NLM_cas:pages">1520-1523</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Internal protein dynamics are intimately connected to enzymic catalysis.  However, enzyme motions linked to substrate turnover remain largely unknown.  Here, the authors studied the dynamics of an enzyme during catalysis at at. resoln. using NMR relaxation methods.  During catalytic action of cyclophilin A/peptidylprolyl cis-trans isomerase, the authors detected conformational fluctuations of the active site that occurred on a time scale of hundreds of microseconds.  The rates of conformational dynamics of the enzyme strongly correlated with the microscopic rates of substrate turnover.  The present results, together with available structural data, allowed a prediction of the reaction trajectory.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaYeoF3RbSabVg90H21EOLACvtfcHk0lhi0yek-EbDYw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD38XhsFyqtbw%253D&amp;md5=834528acfd39865d81b5341e43dc1ab1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1126%2Fscience.1066176&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1066176%26sid%3Dliteratum%253Aachs%26aulast%3DEisenmesser%26aufirst%3DE.%2BZ.%26aulast%3DBosco%26aufirst%3DD.%2BA.%26aulast%3DAkke%26aufirst%3DM.%26aulast%3DKern%26aufirst%3DD.%26atitle%3DEnzyme%2520Dynamics%2520During%2520Catalysis%26jtitle%3DScience%26date%3D2002%26volume%3D295%26spage%3D1520%26epage%3D1523%26doi%3D10.1126%2Fscience.1066176" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eisenmesser, E. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labeikovsky, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korzhnev, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf-Watz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosco, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skalicky, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, D.</span></span> <span> </span><span class="NLM_article-title">Intrinsic Dynamics of an Enzyme Underlies Catalysis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>438</i></span>,  <span class="NLM_fpage">117</span>â <span class="NLM_lpage">121</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature04105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2Fnature04105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=16267559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFOjurzK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=438&amp;publication_year=2005&amp;pages=117-121&amp;author=E.+Z.+Eisenmesserauthor=O.+Milletauthor=W.+Labeikovskyauthor=D.+M.+Korzhnevauthor=M.+Wolf-Watzauthor=D.+A.+Boscoauthor=J.+J.+Skalickyauthor=L.+E.+Kayauthor=D.+Kern&amp;title=Intrinsic+Dynamics+of+an+Enzyme+Underlies+Catalysis&amp;doi=10.1038%2Fnature04105" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Intrinsic dynamics of an enzyme underlies catalysis</span></div><div class="casAuthors">Eisenmesser, Elan Z.; Millet, Oscar; Labeikovsky, Wladimir; Korzhnev, Dmitry M.; Wolf-Watz, Magnus; Bosco, Daryl A.; Skalicky, Jack J.; Kay, Lewis E.; Kern, Dorothee</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">438</span>
        (<span class="NLM_cas:issue">7064</span>),
    <span class="NLM_cas:pages">117-121</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A unique feature of chem. catalysis mediated by enzymes is that the catalytically reactive atoms are embedded within a folded protein.  Although the current understanding of enzyme function has been focused on the chem. reactions and static three-dimensional structures, the dynamic nature of proteins has been proposed to have a function in catalysis.  The concept of conformational substates has been described; however, the challenge is to unravel the intimate linkage between protein flexibility and enzymic function.  Here, the authors show that the intrinsic plasticity of the protein is a key characteristic of catalysis.  The dynamics of the peptidylprolyl cis-trans isomerase, cyclophilin A (CypA), in its substrate-free state and during catalysis were characterized with NMR relaxation expts.  The characteristic enzyme motions detected during catalysis were already present in the free enzyme with frequencies corresponding to the catalytic turnover rates.  This correlation suggests that the protein motions necessary for catalysis are an intrinsic property of the enzyme and may even limit the overall turnover rate.  Motion was localized not only to the active site but also to a wider dynamic network.  Whereas coupled networks in proteins have been proposed previously, the authors exptl. measured the collective nature of motions with the use of mutant forms of CypA.  The authors propose that the pre-existence of collective dynamics in enzymes before catalysis is a common feature of biocatalysts and that proteins have evolved under synergistic pressure between structure and dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ2tDtQkHDfrVg90H21EOLACvtfcHk0lhi0yek-EbDYw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXhtFOjurzK&amp;md5=b6796cf6768ad0de0a04f419f06349e8</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnature04105&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04105%26sid%3Dliteratum%253Aachs%26aulast%3DEisenmesser%26aufirst%3DE.%2BZ.%26aulast%3DMillet%26aufirst%3DO.%26aulast%3DLabeikovsky%26aufirst%3DW.%26aulast%3DKorzhnev%26aufirst%3DD.%2BM.%26aulast%3DWolf-Watz%26aufirst%3DM.%26aulast%3DBosco%26aufirst%3DD.%2BA.%26aulast%3DSkalicky%26aufirst%3DJ.%2BJ.%26aulast%3DKay%26aufirst%3DL.%2BE.%26aulast%3DKern%26aufirst%3DD.%26atitle%3DIntrinsic%2520Dynamics%2520of%2520an%2520Enzyme%2520Underlies%2520Catalysis%26jtitle%3DNature%26date%3D2005%26volume%3D438%26spage%3D117%26epage%3D121%26doi%3D10.1038%2Fnature04105" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ha, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, S. N.</span></span> <span> </span><span class="NLM_article-title">Changes in Side Chain Packing During Apomyoglobin Folding Characterized by Pulsed Thiol-Disulfide Exchange</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">730</span>â <span class="NLM_lpage">737</span>, <span class="refDoi">&nbsp;DOI: 10.1038/1436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2F1436" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=1998&amp;pages=730-737&amp;author=J.-H.+Haauthor=S.+N.+Loh&amp;title=Changes+in+Side+Chain+Packing+During+Apomyoglobin+Folding+Characterized+by+Pulsed+Thiol-Disulfide+Exchange&amp;doi=10.1038%2F1436" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2F1436&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F1436%26sid%3Dliteratum%253Aachs%26aulast%3DHa%26aufirst%3DJ.-H.%26aulast%3DLoh%26aufirst%3DS.%2BN.%26atitle%3DChanges%2520in%2520Side%2520Chain%2520Packing%2520During%2520Apomyoglobin%2520Folding%2520Characterized%2520by%2520Pulsed%2520Thiol-Disulfide%2520Exchange%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1998%26volume%3D5%26spage%3D730%26epage%3D737%26doi%3D10.1038%2F1436" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udgaonkar, J. B.</span></span> <span> </span><span class="NLM_article-title">Exploring the Cooperativity of the Fast Folding Reaction of a Small Protein Using Pulsed Thiol Labeling and Mass Spectrometry</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">37479</span>â <span class="NLM_lpage">37491</span>, <span class="refDoi">&nbsp;DOI: 10.1074/jbc.m706714200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1074%2Fjbc.m706714200" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=282&amp;publication_year=2007&amp;pages=37479-37491&amp;author=S.+K.+Jhaauthor=J.+B.+Udgaonkar&amp;title=Exploring+the+Cooperativity+of+the+Fast+Folding+Reaction+of+a+Small+Protein+Using+Pulsed+Thiol+Labeling+and+Mass+Spectrometry&amp;doi=10.1074%2Fjbc.m706714200" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m706714200&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m706714200%26sid%3Dliteratum%253Aachs%26aulast%3DJha%26aufirst%3DS.%2BK.%26aulast%3DUdgaonkar%26aufirst%3DJ.%2BB.%26atitle%3DExploring%2520the%2520Cooperativity%2520of%2520the%2520Fast%2520Folding%2520Reaction%2520of%2520a%2520Small%2520Protein%2520Using%2520Pulsed%2520Thiol%2520Labeling%2520and%2520Mass%2520Spectrometry%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D37479%26epage%3D37491%26doi%3D10.1074%2Fjbc.m706714200" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dasgupta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udgaonkar, J. B.</span></span> <span> </span><span class="NLM_article-title">Identification of Multiple Folding Pathways of Monellin Using Pulsed Thiol Labeling and Mass Spectrometry</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">3062</span>â <span class="NLM_lpage">3074</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi1006332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi1006332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjs1ChtLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=50&amp;publication_year=2011&amp;pages=3062-3074&amp;author=S.+K.+Jhaauthor=A.+Dasguptaauthor=P.+Malhotraauthor=J.+B.+Udgaonkar&amp;title=Identification+of+Multiple+Folding+Pathways+of+Monellin+Using+Pulsed+Thiol+Labeling+and+Mass+Spectrometry&amp;doi=10.1021%2Fbi1006332" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Multiple Folding Pathways of Monellin Using Pulsed Thiol Labeling and Mass Spectrometry</span></div><div class="casAuthors">Jha, Santosh Kumar; Dasgupta, Amrita; Malhotra, Pooja; Udgaonkar, Jayant B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3062-3074</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein folding reactions often display multiexponential kinetics of changes in intrinsic optical signals, as a manifestation of heterogeneity, either on one folding pathway or on multiple folding pathways.  Delineating the origin of this heterogeneity is difficult because different coexisting structural forms of a protein cannot be easily distinguished by optical probes.  The complex folding reaction of single-chain monellin has been studied using a pulsed thiol labeling (SX) methodol. in conjunction with mass spectrometry, which measures the kinetics of burial of a cysteine side chain thiol during folding.  Because it can directly distinguish between unfolded and folded mols. and can measure the disappearance of the former during folding, the pulsed SX methodol. is an ideal method for studying whether multiple pathways are operative during folding.  The kinetics of burial of the C42 thiol of monellin was obsd. to follow biexponential kinetics.  To det. whether this was because the fast phase leads to the partial protection of the thiol group in all the mols. or to complete protection in only a fraction of the mols., the duration and intensity of the labeling pulse were varied.  The observation that the extent of labeling did not vary with the duration of the pulse cannot be explained by a simple sequential folding mechanism.  Two parallel folding pathways are operative, with one giving thiol-protective structure more rapidly than the other.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ8StXDphcZbVg90H21EOLACvtfcHk0lid8Y4b7uXnfA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXjs1ChtLw%253D&amp;md5=1e67ba4e40b616b1fd6a3b20edabd84d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fbi1006332&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi1006332%26sid%3Dliteratum%253Aachs%26aulast%3DJha%26aufirst%3DS.%2BK.%26aulast%3DDasgupta%26aufirst%3DA.%26aulast%3DMalhotra%26aufirst%3DP.%26aulast%3DUdgaonkar%26aufirst%3DJ.%2BB.%26atitle%3DIdentification%2520of%2520Multiple%2520Folding%2520Pathways%2520of%2520Monellin%2520Using%2520Pulsed%2520Thiol%2520Labeling%2520and%2520Mass%2520Spectrometry%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D3062%26epage%3D3074%26doi%3D10.1021%2Fbi1006332" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbury, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marqusee, S.</span></span> <span> </span><span class="NLM_article-title">Structural and Kinetic Mapping of Side-Chain Exposure onto the Protein Energy Landscape</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">10532</span>â <span class="NLM_lpage">10537</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1103629108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1073%2Fpnas.1103629108" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=108&amp;publication_year=2011&amp;pages=10532-10537&amp;author=R.+Bernsteinauthor=K.+L.+Schmidtauthor=P.+B.+Harburyauthor=S.+Marqusee&amp;title=Structural+and+Kinetic+Mapping+of+Side-Chain+Exposure+onto+the+Protein+Energy+Landscape&amp;doi=10.1073%2Fpnas.1103629108" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1103629108&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1103629108%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DR.%26aulast%3DSchmidt%26aufirst%3DK.%2BL.%26aulast%3DHarbury%26aufirst%3DP.%2BB.%26aulast%3DMarqusee%26aufirst%3DS.%26atitle%3DStructural%2520and%2520Kinetic%2520Mapping%2520of%2520Side-Chain%2520Exposure%2520onto%2520the%2520Protein%2520Energy%2520Landscape%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D10532%26epage%3D10537%26doi%3D10.1073%2Fpnas.1103629108" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demchenko, A. P.</span></span> <span> </span><span class="NLM_article-title">The Red-Edge Effects: 30 Years of Exploration</span>. <i>Luminescence</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">19</span>â <span class="NLM_lpage">42</span>, <span class="refDoi">&nbsp;DOI: 10.1002/bio.671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1002%2Fbio.671" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=11816059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD38XhvFGhu7s%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=2002&amp;pages=19-42&amp;author=A.+P.+Demchenko&amp;title=The+Red-Edge+Effects%3A+30+Years+of+Exploration&amp;doi=10.1002%2Fbio.671" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The red-edge effects: 30 years of exploration</span></div><div class="casAuthors">Demchenko, Alexander P.</div><div class="citationInfo"><span class="NLM_cas:title">Luminescence</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-42</span>CODEN:
                <span class="NLM_cas:coden">LUMIFC</span>;
        ISSN:<span class="NLM_cas:issn">1522-7235</span>.
    
            (<span class="NLM_cas:orgname">John Wiley &amp; Sons Ltd.</span>)
        </div><div class="casAbstract">A review.  In 1970, 3 labs. independently made a discovery that, for arom. fluorophores embedded into different rigid and highly viscous media, the spectroscopic properties do not conform to classical rules.  The fluorescence spectra can depend on excitation wavelength, and the excited-state energy transfer, if present, fails at the red excitation edge.  These red-edge effects were related to the existence of excited-state distribution of fluorophores on their interaction energy with the environment and the slow rate of dielec. relaxation of this environment.  In these conditions the site-selection can be provided by variation of the energy of illuminating light quanta, and the behavior of selected species can be followed as a function of time and other variables.  These observations found extensive application in different areas of research: colloid and polymer science, mol. biophysics, photochem. and photobiol.  In particular, they led to the development of very productive methods of studying the dynamics of dielec. relaxations in protein and membranes, using the tryptophan emission and the emission of a variety of probes.  These studies were extended to the time domain with the observation of new site-selective effects in emission intensity and anisotropy decays.  They stimulated the emergence and development of cryogenic energy-selective and single-mol. techniques that became valuable tools in their own right in chem. and biophysics research.  Site-selection effects were discovered for electron-transfer and proton-transfer reactions if they depended on the dynamics of the environment.  This review is focused on the progress in the field of red-edge effects, their applications and prospects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBEYGc9Y2nOLVg90H21EOLACvtfcHk0lid8Y4b7uXnfA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD38XhvFGhu7s%253D&amp;md5=ec02d96136bb65dc02005dd04ccfc4aa</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fbio.671&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbio.671%26sid%3Dliteratum%253Aachs%26aulast%3DDemchenko%26aufirst%3DA.%2BP.%26atitle%3DThe%2520Red-Edge%2520Effects%253A%252030%2520Years%2520of%2520Exploration%26jtitle%3DLuminescence%26date%3D2002%26volume%3D17%26spage%3D19%26epage%3D42%26doi%3D10.1002%2Fbio.671" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, S.</span></span> <span> </span><span class="NLM_article-title">Dynamic Insight into Protein Structure Utilizing Red Edge Excitation Shift</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">12</span>â <span class="NLM_lpage">19</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar400006z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar400006z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlChtLnK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=2014&amp;pages=12-19&amp;author=A.+Chattopadhyayauthor=S.+Haldar&amp;title=Dynamic+Insight+into+Protein+Structure+Utilizing+Red+Edge+Excitation+Shift&amp;doi=10.1021%2Far400006z" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic Insight into Protein Structure Utilizing Red Edge Excitation Shift</span></div><div class="casAuthors">Chattopadhyay, Amitabha; Haldar, Sourav</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-19</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Proteins are considered the workhorses in the cellular machinery.  They are often organized in a highly ordered conformation in the crowded cellular environment.  These conformations display characteristic dynamics over a range of time scales.  An emerging consensus is that protein function is critically dependent on its dynamics.  The subtle interplay between structure and dynamics is a hallmark of protein organization and is essential for its function.  Depending on the environmental context, proteins can adopt a range of conformations such as native, molten globule, unfolded (denatured), and misfolded states.  Although protein crystallog. is a well established technique, it is not always possible to characterize various protein conformations by X-ray crystallog. due to transient nature of these states.  Even in cases where structural characterization is possible, the information obtained lacks dynamic component, which is needed to understand protein function.  In this overall scenario, approaches that reveal information on protein dynamics are much appreciated.  Dynamics of confined water has interesting implications in protein folding.  Interfacial hydration combines the motion of water mols. with the slow moving protein mols.  The red edge excitation shift (REES) approach becomes relevant in this context.  REES is defined as the shift in the wavelength of max. fluorescence emission toward higher wavelengths, caused by a shift in the excitation wavelength toward the red edge of absorption spectrum.  REES arises due to slow rates (relative to fluorescence lifetime) of solvent relaxation (reorientation) around an excited state fluorophore in organized assemblies such as proteins.  Consequently, REES depends on the environment-induced motional restriction imposed on the solvent mols. in the immediate vicinity of the fluorophore.  In the case of a protein, the confined water in the protein creates a dipolar field that acts as the solvent for a fluorophore in the protein.  In this Account, we focus on REES to monitor organization and dynamics of sol. and membrane proteins utilizing intrinsic protein fluorescence.  We discuss here the application of REES in various conformations of proteins.  While application of REES to proteins in native conformation has been in use for a long time, our work highlights the potential of this approach in case of molten globule and denatured conformations.  For example, we have demonstrated the presence of residual structure, that could not be detected using other methods, by REES of denatured spectrin.  Given the functional relevance of such residual structures, these results are very far reaching.  We discuss here the application of REES to molten globule conformation and to the green fluorescent protein (GFP).  The case of GFP is particularly interesting since the dipolar field in this case is provided by the protein matrix itself and not confined water.  We envision that future applications of REES in proteins will involve generating a dynamic hydration map of the protein, which would allow us to explore protein function in terms of local dynamics and hydration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQSXfHF15j-bVg90H21EOLACvtfcHk0lgqtkT8qTqLdg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhtlChtLnK&amp;md5=ea51eafb84eb25da2aa7ad123662bcac</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Far400006z&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far400006z%26sid%3Dliteratum%253Aachs%26aulast%3DChattopadhyay%26aufirst%3DA.%26aulast%3DHaldar%26aufirst%3DS.%26atitle%3DDynamic%2520Insight%2520into%2520Protein%2520Structure%2520Utilizing%2520Red%2520Edge%2520Excitation%2520Shift%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2014%26volume%3D47%26spage%3D12%26epage%3D19%26doi%3D10.1021%2Far400006z" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demchenko, A. P.</span></span> <span> </span><span class="NLM_article-title">On the nanosecond mobility in proteins: edge excitation fluorescence red shift of protein-bound 2-(p-toluidinylnaphthalene)-6-sulfonate</span>. <i>Biophys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">101</span>â <span class="NLM_lpage">109</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0301-4622(82)80022-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2F0301-4622%2882%2980022-7" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=1982&amp;pages=101-109&amp;author=A.+P.+Demchenko&amp;title=On+the+nanosecond+mobility+in+proteins%3A+edge+excitation+fluorescence+red+shift+of+protein-bound+2-%28p-toluidinylnaphthalene%29-6-sulfonate&amp;doi=10.1016%2F0301-4622%2882%2980022-7" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2F0301-4622%2882%2980022-7&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0301-4622%252882%252980022-7%26sid%3Dliteratum%253Aachs%26aulast%3DDemchenko%26aufirst%3DA.%2BP.%26atitle%3DOn%2520the%2520nanosecond%2520mobility%2520in%2520proteins%253A%2520edge%2520excitation%2520fluorescence%2520red%2520shift%2520of%2520protein-bound%25202-%2528p-toluidinylnaphthalene%2529-6-sulfonate%26jtitle%3DBiophys.%2520Chem.%26date%3D1982%26volume%3D15%26spage%3D101%26epage%3D109%26doi%3D10.1016%2F0301-4622%2882%2980022-7" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lakowicz, J. R.</span></span> <i>Principles of Fluorescence Spectroscopy</i>; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Singapore</span>, <span class="NLM_year">2006</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1007%2F978-0-387-46312-4" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2006&amp;author=J.+R.+Lakowicz&amp;title=Principles+of+Fluorescence+Spectroscopy" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1007%2F978-0-387-46312-4&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-0-387-46312-4%26sid%3Dliteratum%253Aachs%26aulast%3DLakowicz%26aufirst%3DJ.%2BR.%26btitle%3DPrinciples%2520of%2520Fluorescence%2520Spectroscopy%26pub%3DSpringer%26date%3D2006" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lakowicz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating-Nakamoto, S.</span></span> <span> </span><span class="NLM_article-title">Red-Edge Excitation of Fluorescence and Dynamic Properties of Proteins and Membranes</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3013</span>â <span class="NLM_lpage">3021</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00308a026</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00308a026" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaL2cXktVegtLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=1984&amp;pages=3013-3021&amp;author=J.+R.+Lakowiczauthor=S.+Keating-Nakamoto&amp;title=Red-Edge+Excitation+of+Fluorescence+and+Dynamic+Properties+of+Proteins+and+Membranes&amp;doi=10.1021%2Fbi00308a026" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Red-edge excitation of fluorescence and dynamic properties of proteins and membranes</span></div><div class="casAuthors">Lakowicz, Joseph R.; Keating-Nakamoto, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3013-21</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">The phenomenon of the dependence of the fluorescence emission spectra and lifetimes upon the excitation wavelength was used to investigate the dynamic properties of ref. solvents, model membranes, and the protein apomyoglobin labeled with 6-(p-toluidinyl)-2-naphthalenesulfonic acid (TNS).  The spectral shifts and lifetime data indicated that red-edge excitation results in the selective excitation of solvent-relaxed fluorophores.  By comparison of the data obtained for TNS in solvents and bound to the macromols., one may est. the relaxation rate of the environment.  This comparison indicates rapid spectral relaxation of TNS bound to lipid vesicles, and a somewhat slower relaxation around TNS bound to the heme site of apomyoglobin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjrOXKDxlcNbVg90H21EOLACvtfcHk0lgqtkT8qTqLdg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL2cXktVegtLw%253D&amp;md5=adb55d6a9ee29dcea1b7af5c061690af</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fbi00308a026&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00308a026%26sid%3Dliteratum%253Aachs%26aulast%3DLakowicz%26aufirst%3DJ.%2BR.%26aulast%3DKeating-Nakamoto%26aufirst%3DS.%26atitle%3DRed-Edge%2520Excitation%2520of%2520Fluorescence%2520and%2520Dynamic%2520Properties%2520of%2520Proteins%2520and%2520Membranes%26jtitle%3DBiochemistry%26date%3D1984%26volume%3D23%26spage%3D3013%26epage%3D3021%26doi%3D10.1021%2Fbi00308a026" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukhopadhyay, S.</span></span> <span> </span><span class="NLM_article-title">Ordered Water within the Collapsed Globules of an Amyloidogenic Intrinsically Disordered Protein</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">9191</span>â <span class="NLM_lpage">9198</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jp504076a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp504076a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFOqtL7N" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=118&amp;publication_year=2014&amp;pages=9191-9198&amp;author=S.+Aryaauthor=S.+Mukhopadhyay&amp;title=Ordered+Water+within+the+Collapsed+Globules+of+an+Amyloidogenic+Intrinsically+Disordered+Protein&amp;doi=10.1021%2Fjp504076a" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Ordered water within the collapsed globules of an amyloidogenic intrinsically disordered protein</span></div><div class="casAuthors">Arya, Shruti; Mukhopadhyay, Samrat</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9191-9198</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intrinsically disordered proteins (IDPs) confront the traditional sequence-structure-function paradigm and are assocd. with important functions and amyloid disorders.  Water mols. residing in the vicinity of the polypeptide chain play potentially important roles in directing the course of binding-induced folding and amyloid aggregation of IDPs.  Here, the authors characterized the nature of water mols. entrapped within the collapsed globules of the amyloidogenic IDP, Îº-casein.  These globules could undergo further compaction in the presence of an anionic detergent that was capable of diminishing the intrachain repulsion from the pos. charged Gln/Asn-rich amyloidogenic N-terminal domain comprising 100 residues.  Using time-resolved fluorescence spectroscopy, the authors estd. the longer component of the solvation time to be â¼1.4 ns, which was 3 orders of magnitude slower than that in bulk water and more than an order of magnitude slower than the "biol. water" present at the protein surface.  Profoundly restrained water within the collapsed IDP globules resembled the ordered water cluster found under nanoconfinement.  The authors suggest that the assocn. of these globules would result in the release of ordered water mols. into the bulk milieu causing an entropic gain that would eventually drive the formation of the key (obligatory) oligomeric intermediates on the pathway to amyloids via nucleation-dependent polymn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSf08tPiCxN7Vg90H21EOLACvtfcHk0lgqtkT8qTqLdg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXhtFOqtL7N&amp;md5=573cee2f7971645e7f35f18074e49cd2</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjp504076a&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp504076a%26sid%3Dliteratum%253Aachs%26aulast%3DArya%26aufirst%3DS.%26aulast%3DMukhopadhyay%26aufirst%3DS.%26atitle%3DOrdered%2520Water%2520within%2520the%2520Collapsed%2520Globules%2520of%2520an%2520Amyloidogenic%2520Intrinsically%2520Disordered%2520Protein%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2014%26volume%3D118%26spage%3D9191%26epage%3D9198%26doi%3D10.1021%2Fjp504076a" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span> <span> </span><span class="NLM_article-title">Red Edge Excitation Shift of a Deeply Embedded Membrane Probe: Implications in Water Penetration in the Bilayer</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">8180</span>â <span class="NLM_lpage">8185</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jp991303m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp991303m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaK1MXls1Ojt70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=103&amp;publication_year=1999&amp;pages=8180-8185&amp;author=A.+Chattopadhyayauthor=S.+Mukherjee&amp;title=Red+Edge+Excitation+Shift+of+a+Deeply+Embedded+Membrane+Probe%3A+Implications+in+Water+Penetration+in+the+Bilayer&amp;doi=10.1021%2Fjp991303m" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Red Edge Excitation Shift of a Deeply Embedded Membrane Probe: Implications in Water Penetration in the Bilayer</span></div><div class="casAuthors">Chattopadhyay, Amitabha; Mukherjee, Sushmita</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">8180-8185</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. membrane is a highly organized anisotropic mol. assembly.  While the center of the bilayer is nearly isotropic, the upper portion, only a few angstroms away toward the membrane surface, is highly ordered.  How this organization correlates with the degree of water penetration into the bilayer interior is not clear.  In general, it is believed that there is not much water in the deeper hydrocarbon regions of the bilayer.  In this study, we have utilized the phenomenon of wavelength-selective fluorescence to address this question.  We show here that when the same fluorescent group (i.e., 7-nitrobenz-2-oxa-1,3-diazol-4-yl or NBD) is localized at different depths within the bilayer (viz., near the membrane interface in case of the headgroup-labeled NBD-phosphatidylethanolamine (NBD-PE) and near the center of the bilayer in NBD-cholesterol), the degrees to which their fluorescence properties exhibit solvent-induced effects are markedly different.  For example, the headgroup-labeled NBD-PE exhibits a much stronger red edge excitation shift (REES) relative to that of NBD-cholesterol.  This indicates lesser restriction to mobility in this region as compared to the polar/hydrocarbon interface.  In the gel phase, however, REES of NBD-PE did not show any significant change while NBD-cholesterol exhibited no REES.  In addn., NBD-cholesterol exhibits a stronger dependence of fluorescence polarization on excitation wavelength in fluid membranes.  We attribute these results to the more compact arrangement of the lipid acyl chains in the gel phase which results in lesser water penetration.  Since the hydrophobic core of the lipid bilayer is made up of Me and methylene groups, the only solvent dipoles capable of any interaction with the dipole of the fluorophore giving rise to the REES effect in the fluid phase have to be water mols. that have penetrated deep into the bilayer close to the NBD moiety of NBD-cholesterol.  Our results indicate that at least in the fluid phase of the membrane, penetration of water in the deep hydrocarbon region of the bilayer does indeed occur.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbH5ci1H_ev7Vg90H21EOLACvtfcHk0lgb2SL5d1dEiQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1MXls1Ojt70%253D&amp;md5=b74bcb02117822c0e869bc69928cbde4</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjp991303m&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp991303m%26sid%3Dliteratum%253Aachs%26aulast%3DChattopadhyay%26aufirst%3DA.%26aulast%3DMukherjee%26aufirst%3DS.%26atitle%3DRed%2520Edge%2520Excitation%2520Shift%2520of%2520a%2520Deeply%2520Embedded%2520Membrane%2520Probe%253A%2520Implications%2520in%2520Water%2520Penetration%2520in%2520the%2520Bilayer%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1999%26volume%3D103%26spage%3D8180%26epage%3D8185%26doi%3D10.1021%2Fjp991303m" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhuri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, A.</span></span> <span> </span><span class="NLM_article-title">Organization and Dynamics of Membrane Probes and Proteins Utilizing the Red Edge Excitation Shift</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">5693</span>â <span class="NLM_lpage">5706</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jp200255e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp200255e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvVaqu7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=115&amp;publication_year=2011&amp;pages=5693-5706&amp;author=S.+Haldarauthor=A.+Chaudhuriauthor=A.+Chattopadhyay&amp;title=Organization+and+Dynamics+of+Membrane+Probes+and+Proteins+Utilizing+the+Red+Edge+Excitation+Shift&amp;doi=10.1021%2Fjp200255e" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Organization and Dynamics of Membrane Probes and Proteins Utilizing the Red Edge Excitation Shift</span></div><div class="casAuthors">Haldar, Sourav; Chaudhuri, Arunima; Chattopadhyay, Amitabha</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5693-5706</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Dynamics of confined water has interesting implications in the organization and function of mol. assemblies such as membranes.  A direct consequence of this type of organization is the restriction imposed on the mobility of the constituent structural units.  Interestingly, this restriction (confinement) of mobility couples the motion of solvent (water) mols. with the slow moving mols. in the assembly.  It is in this context that the red edge excitation shift (REES) represents a sensitive approach to monitor the environment and dynamics around a fluorophore in such organized assemblies.  A shift in the wavelength of max. fluorescence emission toward higher wavelengths, caused by a shift in the excitation wavelength toward the red edge of the absorption band, is termed REES.  REES relies on slow solvent reorientation in the excited state of a fluorophore that can be used to monitor the environment and dynamics around a fluorophore in a host assembly.  In this article, the authors focus on the application of REES to monitor organization and dynamics of membrane probes and proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbFYKxPZrwSrVg90H21EOLACvtfcHk0lgb2SL5d1dEiQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXjvVaqu7g%253D&amp;md5=f785f730602c7aba768229ee25afe1aa</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjp200255e&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp200255e%26sid%3Dliteratum%253Aachs%26aulast%3DHaldar%26aufirst%3DS.%26aulast%3DChaudhuri%26aufirst%3DA.%26aulast%3DChattopadhyay%26aufirst%3DA.%26atitle%3DOrganization%2520and%2520Dynamics%2520of%2520Membrane%2520Probes%2520and%2520Proteins%2520Utilizing%2520the%2520Red%2520Edge%2520Excitation%2520Shift%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2011%26volume%3D115%26spage%3D5693%26epage%3D5706%26doi%3D10.1021%2Fjp200255e" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawat, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharyya, B.</span></span> <span> </span><span class="NLM_article-title">Tubulin Conformation and Dynamics: A Red Edge Excitation Shift Study</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">13426</span>â <span class="NLM_lpage">13433</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi961251g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi961251g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaK28XlslOlurk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=35&amp;publication_year=1996&amp;pages=13426-13433&amp;author=S.+Guhaauthor=S.+S.+Rawatauthor=A.+Chattopadhyayauthor=B.+Bhattacharyya&amp;title=Tubulin+Conformation+and+Dynamics%3A+A+Red+Edge+Excitation+Shift+Study&amp;doi=10.1021%2Fbi961251g" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Tubulin conformation and dynamics: A red edge excitation shift study</span></div><div class="casAuthors">Guha, Suranjana; Rawat, Satinder S.; Chattopadhyay, Amitabha; Bhattacharyya, Bhabatarak</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">13426-13433</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The fluorescence emission max. of a polar fluorophore in viscous medium often shows a dependence on excitation wavelength, a phenomenon which is named red edge excitation shift (REES).  Here, the authors found that the fluorescence spectra of the tubulin Trp residues exhibited a REES of â¼7 nm.  Also, their steady state fluorescence polarization and mean lifetimes showed a dependence on both excitation and emission wavelengths.  These results indicated that the av. Trp environment in tubulin is motionally restricted.  Although the Trp residue(s) responsible for the obsd. REES effect could not be localized, it could be concluded from energy transfer expts. with the tubulin-colchicine complex that the Trp residue(s) participating in energy transfer with bound colchicine probably did not contribute to the REES.  A REES of 7 nm was also obsd. in the case of colchicine complexed with tubulin.  However, such a REES was not seen in similar studies with the B-ring analogs of colchicine, viz., 2-methoxy-5-(2',3',4'-trimethoxyphenyl)tropone (called AC because it lacks the B ring of colchicine) and deacetamidocolchicine (which lacks the acetamido substituent at the C-7 position of the B ring).  There may be 2 possible reasons to explain these data.  (1) Structural differences between colchicine and its analogs may give rise to differences in their excited state dipole moments which will directly affect the extent of REES, and (2) The B-ring substituent, hanging outside the colchicine binding site on the Î² subunit of the tubulin dimer, probably makes contact with the Î± subunit of tubulin and imparts a rigidity to that region of the protein, which facilitates the REES.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRot2M2IKBu7Vg90H21EOLACvtfcHk0lgb2SL5d1dEiQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK28XlslOlurk%253D&amp;md5=7838ad028038740784dade85edf14b46</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fbi961251g&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi961251g%26sid%3Dliteratum%253Aachs%26aulast%3DGuha%26aufirst%3DS.%26aulast%3DRawat%26aufirst%3DS.%2BS.%26aulast%3DChattopadhyay%26aufirst%3DA.%26aulast%3DBhattacharyya%26aufirst%3DB.%26atitle%3DTubulin%2520Conformation%2520and%2520Dynamics%253A%2520A%2520Red%2520Edge%2520Excitation%2520Shift%2520Study%26jtitle%3DBiochemistry%26date%3D1996%26volume%3D35%26spage%3D13426%26epage%3D13433%26doi%3D10.1021%2Fbi961251g" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Catici, D. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amos, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Elsen, J. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pudney, C. R.</span></span> <span> </span><span class="NLM_article-title">The Red Edge Excitation Shift Phenomenon can be Used to Unmask Protein Structural Ensembles: Implications for Nemo-Ubiquitin Interactions</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">2272</span>â <span class="NLM_lpage">2284</span>, <span class="refDoi">&nbsp;DOI: 10.1111/febs.13724</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1111%2Ffebs.13724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=27028374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsVyjt7o%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=283&amp;publication_year=2016&amp;pages=2272-2284&amp;author=D.+A.+M.+Caticiauthor=H.+E.+Amosauthor=Y.+Yangauthor=J.+M.+H.+van+den+Elsenauthor=C.+R.+Pudney&amp;title=The+Red+Edge+Excitation+Shift+Phenomenon+can+be+Used+to+Unmask+Protein+Structural+Ensembles%3A+Implications+for+Nemo-Ubiquitin+Interactions&amp;doi=10.1111%2Ffebs.13724" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The red edge excitation shift phenomenon can be used to unmask protein structural ensembles: implications for NEMO-ubiquitin interactions</span></div><div class="casAuthors">Catici, Dragana A. M.; Amos, Hope E.; Yang, Yi; van den Elsen, Jean M. H.; Pudney, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2272-2284</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">To understand complex mol. interactions, it is necessary to account for mol. flexibility and the available equil. of conformational states.  Only a small no. of exptl. approaches can access such information.  Potentially, steady-state red edge excitation shift (REES) spectroscopy can act as a qual. metric of changes to the protein free energy landscape (FEL) and the equil. of conformational states.  First, the authors validated this hypothesis using a single Trp-contg. protein, NF-ÎºB essential modulator (NEMO).  The authors provided detailed evidence from chem. denaturation studies, macromol. crowding studies, and the 1st report of the pressure dependence of the REES effect.  A combination of these data demonstrated that the REES effect can report on the 'ruggedness' of the FEL and the authors present a phenomenol. model, based on realistic phys. interpretations, for fitting steady-state REES data to allow quantification of this aspect of the REES effect.  The authors tested the conceptual framework they developed by correlating findings from NEMO ligand-binding studies with the REES data in a range of NEMO-ligand binary complexes.  These findings shed light on the nature of the interaction between NEMO and poly-ubiquitin, suggesting that NEMO is differentially regulated by polyubiquitin chain length and that this regulation occurs via a modulation of the available equil. of conformational states, rather than gross structural change.  This study therefore demonstrates the potential of REES as a powerful tool for tackling contemporary issues in structural biol. and biophysics and elucidates novel information on the structure-function relation of NEMO and key interaction partners.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE1xuMXcBAUrVg90H21EOLACvtfcHk0ljMx-haMI87rw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XpsVyjt7o%253D&amp;md5=a657e1411a6e6e7b2e8b7232aa5f7461</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1111%2Ffebs.13724&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.13724%26sid%3Dliteratum%253Aachs%26aulast%3DCatici%26aufirst%3DD.%2BA.%2BM.%26aulast%3DAmos%26aufirst%3DH.%2BE.%26aulast%3DYang%26aufirst%3DY.%26aulast%3Dvan%2Bden%2BElsen%26aufirst%3DJ.%2BM.%2BH.%26aulast%3DPudney%26aufirst%3DC.%2BR.%26atitle%3DThe%2520Red%2520Edge%2520Excitation%2520Shift%2520Phenomenon%2520can%2520be%2520Used%2520to%2520Unmask%2520Protein%2520Structural%2520Ensembles%253A%2520Implications%2520for%2520Nemo-Ubiquitin%2520Interactions%26jtitle%3DFEBS%2520J.%26date%3D2016%26volume%3D283%26spage%3D2272%26epage%3D2284%26doi%3D10.1111%2Ffebs.13724" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawat, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelkar, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabarti, A.</span></span> <span> </span><span class="NLM_article-title">Organization and dynamics of tryptophan residues in erythroid spectrin: novel structural features of denatured spectrin revealed by the wavelength-selective fluorescence approach</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2389</span>â <span class="NLM_lpage">2403</span>, <span class="refDoi">&nbsp;DOI: 10.1110/ps.03302003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1110%2Fps.03302003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=14573853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD3sXosleltbo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2003&amp;pages=2389-2403&amp;author=A.+Chattopadhyayauthor=S.+S.+Rawatauthor=D.+A.+Kelkarauthor=S.+Rayauthor=A.+Chakrabarti&amp;title=Organization+and+dynamics+of+tryptophan+residues+in+erythroid+spectrin%3A+novel+structural+features+of+denatured+spectrin+revealed+by+the+wavelength-selective+fluorescence+approach&amp;doi=10.1110%2Fps.03302003" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Organization and dynamics of tryptophan residues in erythroid spectrin: Novel structural features of denatured spectrin revealed by the wavelength-selective fluorescence approach</span></div><div class="casAuthors">Chattopadhyay, Amitabha; Rawat, Satinder S.; Kelkar, Devaki A.; Ray, Sibnath; Chakrabarti, Abhijit</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2389-2403</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">We have investigated the organization and dynamics of the functionally important tryptophan residues of erythroid spectrin in native and denatured conditions utilizing the wavelength-selective fluorescence approach.  We obsd. a red edge excitation shift (REES) of 4 nm for the tryptophans in the case of spectrin in its native state.  This indicates that tryptophans in spectrin are localized in a microenvironment of restricted mobility, and that the regions surrounding the spectrin tryptophans offer considerable restriction to the reorientational motion of the water dipoles around the excited state tryptophans.  Interestingly, spectrin exhibits a REES of 3 nm even when denatured in 8 M urea.  This represents the first report of a denatured protein displaying REES.  Observation of REES in the denatured state implies that some of the structural and dynamic features of this microenvironment around the spectrin tryptophans are retained even when the protein is denatured.  Fluorescence quenching data of denatured spectrin support this conclusion.  In addn., we have deduced the organization and dynamics of the hydrophobic binding site of the polarity-sensitive fluorescent probe PRODAN that binds erythroid spectrin with high affinity.  When bound to spectrin, PRODAN exhibits a REES of 9 nm.  Because PRODAN binds to a hydrophobic site in spectrin, such a result would directly imply that this region of spectrin offers considerable restriction to the reorientational motion of the solvent dipoles around the excited state fluorophore.  The results of our study could provide vital insight into the role of tryptophans in the stability and folding of spectrin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ0fl0vm2HrLVg90H21EOLACvtfcHk0ljMx-haMI87rw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3sXosleltbo%253D&amp;md5=f1626f0c34f2691556ae835b11944867</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1110%2Fps.03302003&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.03302003%26sid%3Dliteratum%253Aachs%26aulast%3DChattopadhyay%26aufirst%3DA.%26aulast%3DRawat%26aufirst%3DS.%2BS.%26aulast%3DKelkar%26aufirst%3DD.%2BA.%26aulast%3DRay%26aufirst%3DS.%26aulast%3DChakrabarti%26aufirst%3DA.%26atitle%3DOrganization%2520and%2520dynamics%2520of%2520tryptophan%2520residues%2520in%2520erythroid%2520spectrin%253A%2520novel%2520structural%2520features%2520of%2520denatured%2520spectrin%2520revealed%2520by%2520the%2520wavelength-selective%2520fluorescence%2520approach%26jtitle%3DProtein%2520Sci.%26date%3D2003%26volume%3D12%26spage%3D2389%26epage%3D2403%26doi%3D10.1110%2Fps.03302003" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhuri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, A.</span></span> <span> </span><span class="NLM_article-title">Organization and dynamics of tryptophans in the molten globule state of bovine Î±-lactalbumin utilizing wavelength-selective fluorescence approach: Comparisons with native and denatured states</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>394</i></span>,  <span class="NLM_fpage">1082</span>â <span class="NLM_lpage">1086</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bbrc.2010.03.130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2Fj.bbrc.2010.03.130" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=394&amp;publication_year=2010&amp;pages=1082-1086&amp;author=A.+Chaudhuriauthor=S.+Haldarauthor=A.+Chattopadhyay&amp;title=Organization+and+dynamics+of+tryptophans+in+the+molten+globule+state+of+bovine+%CE%B1-lactalbumin+utilizing+wavelength-selective+fluorescence+approach%3A+Comparisons+with+native+and+denatured+states&amp;doi=10.1016%2Fj.bbrc.2010.03.130" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2010.03.130&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2010.03.130%26sid%3Dliteratum%253Aachs%26aulast%3DChaudhuri%26aufirst%3DA.%26aulast%3DHaldar%26aufirst%3DS.%26aulast%3DChattopadhyay%26aufirst%3DA.%26atitle%3DOrganization%2520and%2520dynamics%2520of%2520tryptophans%2520in%2520the%2520molten%2520globule%2520state%2520of%2520bovine%2520%25CE%25B1-lactalbumin%2520utilizing%2520wavelength-selective%2520fluorescence%2520approach%253A%2520Comparisons%2520with%2520native%2520and%2520denatured%2520states%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2010%26volume%3D394%26spage%3D1082%26epage%3D1086%26doi%3D10.1016%2Fj.bbrc.2010.03.130" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span> <span> </span><span class="NLM_article-title">Evidence for Dry Molten Globule-Like Domains in the pH-Induced Equilibrium Folding Intermediate of a Multidomain Protein</span>. <i>J. Phys. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">173</span>â <span class="NLM_lpage">179</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.jpclett.5b02545</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jpclett.5b02545" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVygt73L" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2016&amp;pages=173-179&amp;author=N.+Acharyaauthor=P.+Mishraauthor=S.+K.+Jha&amp;title=Evidence+for+Dry+Molten+Globule-Like+Domains+in+the+pH-Induced+Equilibrium+Folding+Intermediate+of+a+Multidomain+Protein&amp;doi=10.1021%2Facs.jpclett.5b02545" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for Dry Molten Globule-Like Domains in the pH-Induced Equilibrium Folding Intermediate of a Multidomain Protein</span></div><div class="casAuthors">Acharya, Nirbhik; Mishra, Prajna; Jha, Santosh Kumar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">173-179</span>CODEN:
                <span class="NLM_cas:coden">JPCLCD</span>;
        ISSN:<span class="NLM_cas:issn">1948-7185</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The role of van der Waals (vdW) packing interactions compared to the hydrophobic effect in stabilizing the functional structure of proteins is poorly understood.  Here we show, using fluorescence resonance energy transfer, dynamic fluorescence quenching, red-edge excitation shift, and near- and far-UV CD, that the pH-induced structural perturbation of a multidomain protein leads to the formation of a state in which two out of the three domains have characteristics of dry molten globules, i.e., the domains are expanded compared to the native protein with disrupted packing interactions but have dry cores.  We quant. est. the energetic contribution of vdW interactions and show that they play an important role in the stability of the native state and cooperativity of its structural transition, in addn. to the hydrophobic effect.  Our results also indicate that during the pH-induced unfolding, side-chain unlocking and hydrophobic solvation occur in two distinct steps and not in a concerted manner, as commonly believed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-vmLRGPIMUbVg90H21EOLACvtfcHk0lguAENhne7Ogg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXitVygt73L&amp;md5=dc2eecf40f625c12d906949ab55605cf</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jpclett.5b02545&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jpclett.5b02545%26sid%3Dliteratum%253Aachs%26aulast%3DAcharya%26aufirst%3DN.%26aulast%3DMishra%26aufirst%3DP.%26aulast%3DJha%26aufirst%3DS.%2BK.%26atitle%3DEvidence%2520for%2520Dry%2520Molten%2520Globule-Like%2520Domains%2520in%2520the%2520pH-Induced%2520Equilibrium%2520Folding%2520Intermediate%2520of%2520a%2520Multidomain%2520Protein%26jtitle%3DJ.%2520Phys.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D173%26epage%3D179%26doi%3D10.1021%2Facs.jpclett.5b02545" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beeby, P. J.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Interaction with Human Serum Albumin of the First 3,18-Disubstituted Derivative of Bilirubin</span>. <i>J. Chem. Soc. Perkin Trans. 1</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_fpage">3041</span>â <span class="NLM_lpage">3044</span>, <span class="refDoi">&nbsp;DOI: 10.1039/a803429j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1039%2Fa803429j" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1998&amp;pages=3041-3044&amp;author=L.+Painterauthor=M.+M.+Hardingauthor=P.+J.+Beeby&amp;title=Synthesis+and+Interaction+with+Human+Serum+Albumin+of+the+First+3%2C18-Disubstituted+Derivative+of+Bilirubin&amp;doi=10.1039%2Fa803429j" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1039%2Fa803429j&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fa803429j%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DL.%26aulast%3DHarding%26aufirst%3DM.%2BM.%26aulast%3DBeeby%26aufirst%3DP.%2BJ.%26atitle%3DSynthesis%2520and%2520Interaction%2520with%2520Human%2520Serum%2520Albumin%2520of%2520the%2520First%25203%252C18-Disubstituted%2520Derivative%2520of%2520Bilirubin%26jtitle%3DJ.%2520Chem.%2520Soc.%2520Perkin%2520Trans.%25201%26date%3D1998%26spage%3D3041%26epage%3D3044%26doi%3D10.1039%2Fa803429j" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pace, C. N.</span></span> <span> </span><span class="NLM_article-title">Determination and analysis of urea and guanidine hydrochloride denaturation curves</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">266</span>â <span class="NLM_lpage">280</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0076-6879(86)31045-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2F0076-6879%2886%2931045-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=3773761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaL2sXmtlOitw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=131&amp;publication_year=1986&amp;pages=266-280&amp;author=C.+N.+Pace&amp;title=Determination+and+analysis+of+urea+and+guanidine+hydrochloride+denaturation+curves&amp;doi=10.1016%2F0076-6879%2886%2931045-0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Determination and analysis of urea and guanidine hydrochloride denaturation curves</span></div><div class="casAuthors">Pace, C. N.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">Enzyme Struct., Pt. L</span>),
    <span class="NLM_cas:pages">266-80</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">A review, with 38 refs., on practical aspects of detg. and analyzing urea- and guanidine hydrochloride-induced denaturation curves of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps8UxgdhPl77Vg90H21EOLACvtfcHk0lguAENhne7Ogg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL2sXmtlOitw%253D%253D&amp;md5=efc254fd3a598035cf9573702116917f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2886%2931045-0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252886%252931045-0%26sid%3Dliteratum%253Aachs%26aulast%3DPace%26aufirst%3DC.%2BN.%26atitle%3DDetermination%2520and%2520analysis%2520of%2520urea%2520and%2520guanidine%2520hydrochloride%2520denaturation%2520curves%26jtitle%3DMethods%2520Enzymol.%26date%3D1986%26volume%3D131%26spage%3D266%26epage%3D280%26doi%3D10.1016%2F0076-6879%2886%2931045-0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Street, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courtemanche, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrick, D.</span></span> <span> </span><span class="NLM_article-title">Protein Folding and Stability Using Denaturants</span>. <i>Methods Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">295</span>â <span class="NLM_lpage">325</span>, <span class="refDoi">&nbsp;DOI: 10.1016/s0091-679x(07)84011-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2Fs0091-679x%2807%2984011-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=17964936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1amtLY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=84&amp;publication_year=2008&amp;pages=295-325&amp;author=T.+O.+Streetauthor=N.+Courtemancheauthor=D.+Barrick&amp;title=Protein+Folding+and+Stability+Using+Denaturants&amp;doi=10.1016%2Fs0091-679x%2807%2984011-8" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Protein folding and stability using denaturants</span></div><div class="casAuthors">Street, Timothy O.; Courtemanche, Naomi; Barrick, Doug</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">Biophysical Tools for Biologists, Volume One</span>),
    <span class="NLM_cas:pages">295-325</span>CODEN:
                <span class="NLM_cas:coden">MCBLAG</span>;
        ISSN:<span class="NLM_cas:issn">0091-679X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Measurements of protein folding and thermodn. stability provide insight into the forces and energetics that det. structure, and can inform on protein domain organization, interdomain interactions, and effects of mutations on structure.  This chapter describes methods, theory, and data anal. for the most accessible means to det. the thermodn. of protein folding: chem. denaturation.  Topics include overall features of the folding reaction, advances in instrumentation, optimization of reagent purity, mechanistic models for anal., and statistical and structural interpretation of fitted thermodn. parameters.  Examples in which stability measurements have provided insight into structure and function will be taken from studies in the author's lab. on the Notch signaling pathway.  It is hoped that this chapter will enable mol., cell, and structural biologists to make precise measurements of protein stability, and will also provide a strong foundation for biophysics students who wish to undertake exptl. studies of protein folding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplhWYZ7zDdwrVg90H21EOLACvtfcHk0lguAENhne7Ogg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXht1amtLY%253D&amp;md5=f265362505ccf991e79aa386426075b9</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fs0091-679x%2807%2984011-8&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0091-679x%252807%252984011-8%26sid%3Dliteratum%253Aachs%26aulast%3DStreet%26aufirst%3DT.%2BO.%26aulast%3DCourtemanche%26aufirst%3DN.%26aulast%3DBarrick%26aufirst%3DD.%26atitle%3DProtein%2520Folding%2520and%2520Stability%2520Using%2520Denaturants%26jtitle%3DMethods%2520Cell%2520Biol.%26date%3D2008%26volume%3D84%26spage%3D295%26epage%3D325%26doi%3D10.1016%2Fs0091-679x%2807%2984011-8" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gromiha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawareh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarai, A.</span></span> <span> </span><span class="NLM_article-title">Asaview: Database and Tool for Solvent Accessibility Representation in Proteins</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">51</span>, <span class="refDoi">&nbsp;DOI: 10.1186/1471-2105-5-51</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1186%2F1471-2105-5-51" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=15119964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A280%3ADC%252BD2c3otVOqsg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2004&amp;pages=51&amp;author=S.+Ahmadauthor=M.+Gromihaauthor=H.+Fawarehauthor=A.+Sarai&amp;title=Asaview%3A+Database+and+Tool+for+Solvent+Accessibility+Representation+in+Proteins&amp;doi=10.1186%2F1471-2105-5-51" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">ASAView: database and tool for solvent accessibility representation in proteins</span></div><div class="casAuthors">Ahmad Shandar; Gromiha Michael; Fawareh Hamed; Sarai Akinori</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Accessible surface area (ASA) or solvent accessibility of amino acids in a protein has important implications.  Knowledge of surface residues helps in locating potential candidates of active sites.  Therefore, a method to quickly see the surface residues in a two dimensional model would help to immediately understand the population of amino acid residues on the surface and in the inner core of the proteins.  RESULTS:  ASAView is an algorithm, an application and a database of schematic representations of solvent accessibility of amino acid residues within proteins.  A characteristic two-dimensional spiral plot of solvent accessibility provides a convenient graphical view of residues in terms of their exposed surface areas.  In addition, sequential plots in the form of bar charts are also provided.  Online plots of the proteins included in the entire Protein Data Bank (PDB), are provided for the entire protein as well as their chains separately.  CONCLUSIONS:  These graphical plots of solvent accessibility are likely to provide a quick view of the overall topological distribution of residues in proteins.  Chain-wise computation of solvent accessibility is also provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTCaKPeuivdBuRS7coPn-RtfW6udTcc2eYYRkMYlFIYlrntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BD2c3otVOqsg%253D%253D&amp;md5=d253b58936b69be51806e48b66ef8c44</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-5-51&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-5-51%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DS.%26aulast%3DGromiha%26aufirst%3DM.%26aulast%3DFawareh%26aufirst%3DH.%26aulast%3DSarai%26aufirst%3DA.%26atitle%3DAsaview%253A%2520Database%2520and%2520Tool%2520for%2520Solvent%2520Accessibility%2520Representation%2520in%2520Proteins%26jtitle%3DBMC%2520Bioinf.%26date%3D2004%26volume%3D5%26spage%3D51%26doi%3D10.1186%2F1471-2105-5-51" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span> <span> </span><span class="NLM_article-title">A Dry Molten Globule-Like Intermediate During the Base-Induced Unfolding of a Multidomain Protein</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">30207</span>â <span class="NLM_lpage">30216</span>, <span class="refDoi">&nbsp;DOI: 10.1039/c7cp06614g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1039%2Fc7cp06614g" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2017&amp;pages=30207-30216&amp;author=N.+Acharyaauthor=P.+Mishraauthor=S.+K.+Jha&amp;title=A+Dry+Molten+Globule-Like+Intermediate+During+the+Base-Induced+Unfolding+of+a+Multidomain+Protein&amp;doi=10.1039%2Fc7cp06614g" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1039%2Fc7cp06614g&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc7cp06614g%26sid%3Dliteratum%253Aachs%26aulast%3DAcharya%26aufirst%3DN.%26aulast%3DMishra%26aufirst%3DP.%26aulast%3DJha%26aufirst%3DS.%2BK.%26atitle%3DA%2520Dry%2520Molten%2520Globule-Like%2520Intermediate%2520During%2520the%2520Base-Induced%2520Unfolding%2520of%2520a%2520Multidomain%2520Protein%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2017%26volume%3D19%26spage%3D30207%26epage%3D30216%26doi%3D10.1039%2Fc7cp06614g" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demchenko, A. P.</span></span> <span> </span><span class="NLM_article-title">Red-Edge-Excitation Fluorescence Spectroscopy of Single-Tryptophan Proteins</span>. <i>Eur. Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">121</span>â <span class="NLM_lpage">129</span>, <span class="refDoi">&nbsp;DOI: 10.1007/bf00255522</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1007%2Fbf00255522" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=3208709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaL1cXlvVKlsrw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=1988&amp;pages=121-129&amp;author=A.+P.+Demchenko&amp;title=Red-Edge-Excitation+Fluorescence+Spectroscopy+of+Single-Tryptophan+Proteins&amp;doi=10.1007%2Fbf00255522" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Red-edge-excitation fluorescence spectroscopy of single-tryptophan proteins</span></div><div class="casAuthors">Demchenko, A. P.</div><div class="citationInfo"><span class="NLM_cas:title">European Biophysics Journal</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">121-9</span>CODEN:
                <span class="NLM_cas:coden">EBJOE8</span>;
        ISSN:<span class="NLM_cas:issn">0175-7571</span>.
    </div><div class="casAbstract">With the aim of finding nonequil. dipole-relaxational electronic excited states of tryptophan residues in proteins, the dependence of the fluorescence emission max. on excitation wavelength was studied for several proteins contg. a single tryptophan residue per mol.  Spectral shifts upon red-edge excitation are not obsd. for short wavelength-emitting proteins (azurin, 2-Ca form of whiting parvalbumin, RNases C2 and T1).  This may be because of the nonpolar environment of the tryptophan residues in these proteins or because of the absence of dipole-orientational broadening of spectra.  The effect was also not found for proteins emitting at long wavelengths (max. at 341-350 nm): melittin at low ionic strength, IT-Aj1 protease inhibitor, myelin basic protein.  In these proteins, the tryptophan residues are exposed to the rapidly relaxing aq. solvent.  Spectral shifts assocd. with red-edge excitation are obsd. for proteins emitting in the medium spectral range-human serum albumin in the N and F forms, IT-Aj1 protease inhibitor at pH 2.9, melittin at high ionic strength as well as the albumin-SDS complex.  This suggests the existence in these proteins of a distribution of microstates for tryptophan environment with various orientation of dipoles and of slow (on the nanosecond time scale) mobility of the field of these dipoles.  As a result, the emission proceeds from electronic excited states which are not at equil.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqza9BS99B2lLVg90H21EOLACvtfcHk0ljssUMsjPsAqA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL1cXlvVKlsrw%253D&amp;md5=16dd42ecb8bd9b31bdbeb1f86f421926</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1007%2Fbf00255522&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fbf00255522%26sid%3Dliteratum%253Aachs%26aulast%3DDemchenko%26aufirst%3DA.%2BP.%26atitle%3DRed-Edge-Excitation%2520Fluorescence%2520Spectroscopy%2520of%2520Single-Tryptophan%2520Proteins%26jtitle%3DEur.%2520Biophys.%2520J.%26date%3D1988%26volume%3D16%26spage%3D121%26epage%3D129%26doi%3D10.1007%2Fbf00255522" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenfield, N. J.</span></span> <span> </span><span class="NLM_article-title">Using circular dichroism spectra to estimate protein secondary structure</span>. <i>Nat Protoc</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">2876</span>â <span class="NLM_lpage">2890</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nprot.2006.202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2Fnprot.2006.202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=17406547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGjtLrM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1&amp;publication_year=2006&amp;pages=2876-2890&amp;author=N.+J.+Greenfield&amp;title=Using+circular+dichroism+spectra+to+estimate+protein+secondary+structure&amp;doi=10.1038%2Fnprot.2006.202" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Using circular dichroism spectra to estimate protein secondary structure</span></div><div class="casAuthors">Greenfield, Norma J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2876-2890</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">CD (CD) is an excellent tool for rapid detn. of the secondary structure and folding properties of proteins that have been obtained using recombinant techniques or purified from tissues.  The most widely used applications of protein CD are to det. whether an expressed, purified protein is folded, or if a mutation affects its conformation or stability.  In addn., it can be used to study protein interactions.  This protocol details the basic steps of obtaining and interpreting CD data, and methods for analyzing spectra to est. the secondary structural compn. of proteins.  CD has the advantage that measurements may be made on multiple samples contg. â¤20 Î¼g of proteins in physiol. buffers in a few hours.  However, it does not give the residue-specific information that can be obtained by x-ray crystallog. or NMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlsmX4yVdMSbVg90H21EOLACvtfcHk0lj7DYiVyk2SLQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGjtLrM&amp;md5=7a4ce0eeff5915c85be1db2327f845a2</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.202&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.202%26sid%3Dliteratum%253Aachs%26aulast%3DGreenfield%26aufirst%3DN.%2BJ.%26atitle%3DUsing%2520circular%2520dichroism%2520spectra%2520to%2520estimate%2520protein%2520secondary%2520structure%26jtitle%3DNat%2520Protoc%26date%3D2006%26volume%3D1%26spage%3D2876%26epage%3D2890%26doi%3D10.1038%2Fnprot.2006.202" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finkelstein, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakhnovich, E. I.</span></span> <span> </span><span class="NLM_article-title">Theory of Cooperative Transitions in Protein Molecules. II. Phase Diagram for a Protein Molecule in Solution</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1681</span>â <span class="NLM_lpage">1694</span>, <span class="refDoi">&nbsp;DOI: 10.1002/bip.360281004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1002%2Fbip.360281004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=2597724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A280%3ADyaK3c%252FovFGjtA%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=28&amp;publication_year=1989&amp;pages=1681-1694&amp;author=A.+V.+Finkelsteinauthor=E.+I.+Shakhnovich&amp;title=Theory+of+Cooperative+Transitions+in+Protein+Molecules.+II.+Phase+Diagram+for+a+Protein+Molecule+in+Solution&amp;doi=10.1002%2Fbip.360281004" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Theory of cooperative transitions in protein molecules. II. Phase diagram for a protein molecule in solution</span></div><div class="casAuthors">Finkelstein A V; Shakhnovich E I</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1681-94</span>
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    </div><div class="casAbstract">The thermodynamically stable states of denatured protein in solution are investigated.  These states are distinguished from the native state by the absence of tight packing of side chains while the compactness of denatured protein may vary within a wide region.  The following regimes are outlined: 1. the "wet" molten globule, i.e., the compact state with pores occupied by solvent; 2. the swollen globule ("wet," of course); and 3. the coil.  The "dry" molten globule, when solvent does not penetrate inside the protein, is excluded for all experimental conditions.  All the transitions within the denatured globule state are gradual while the denatured globule-coil phase transition is a second order one.  The conditions of protein denaturation as well as conditions of transitions and crossovers within the denatured state are outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQJeb6uujKyfMzK395ZObgVfW6udTcc2ebkdwKudz3a-bntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADyaK3c%252FovFGjtA%253D%253D&amp;md5=8ed55ae24a29b8b4673c47709ee14d9f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fbip.360281004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.360281004%26sid%3Dliteratum%253Aachs%26aulast%3DFinkelstein%26aufirst%3DA.%2BV.%26aulast%3DShakhnovich%26aufirst%3DE.%2BI.%26atitle%3DTheory%2520of%2520Cooperative%2520Transitions%2520in%2520Protein%2520Molecules.%2520II.%2520Phase%2520Diagram%2520for%2520a%2520Protein%2520Molecule%2520in%2520Solution%26jtitle%3DBiopolymers%26date%3D1989%26volume%3D28%26spage%3D1681%26epage%3D1694%26doi%3D10.1002%2Fbip.360281004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shakhnovich, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkelstein, A. V.</span></span> <span> </span><span class="NLM_article-title">Theory of Cooperative Transitions in Protein Molecules. I. Why Denaturation of Globular Protein is a First-Order Phase Transition</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1667</span>â <span class="NLM_lpage">1680</span>, <span class="refDoi">&nbsp;DOI: 10.1002/bip.360281003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1002%2Fbip.360281003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=2597723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaK3cXivVGn" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=28&amp;publication_year=1989&amp;pages=1667-1680&amp;author=E.+I.+Shakhnovichauthor=A.+V.+Finkelstein&amp;title=Theory+of+Cooperative+Transitions+in+Protein+Molecules.+I.+Why+Denaturation+of+Globular+Protein+is+a+First-Order+Phase+Transition&amp;doi=10.1002%2Fbip.360281003" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Theory of cooperative transitions in protein molecules.  I.  Why denaturation of globular protein is a first-order phase transition</span></div><div class="casAuthors">Shakhnovich, E. I.; Finkel'shtein, A. V.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1667-80</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    </div><div class="casAbstract">A theory of equil. denaturation of proteins is suggested.  According to this theory, a cornerstone of protein denaturation is disruption of tight packing of side chains in the protein core.  This disruption was investigated.  This disruption is an all-or-none transition (independent of how compact the denatured state of a protein is and independent of protein-solvent interactions) because expansion of a globule must exceed some threshold to release rotational isomerization of side chains.  Smaller expansion cannot produce entropy compensation of nonbonded energy loss; this is the origin of a free-energy barrier (transition state) between the native and denatured states.  The d. of the transition state is so high that the solvent cannot penetrate into protein in this state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-SThXaIbxxrVg90H21EOLACvtfcHk0lj7DYiVyk2SLQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3cXivVGn&amp;md5=552f410b3f1984eb3c672650a5ae3e75</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fbip.360281003&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.360281003%26sid%3Dliteratum%253Aachs%26aulast%3DShakhnovich%26aufirst%3DE.%2BI.%26aulast%3DFinkelstein%26aufirst%3DA.%2BV.%26atitle%3DTheory%2520of%2520Cooperative%2520Transitions%2520in%2520Protein%2520Molecules.%2520I.%2520Why%2520Denaturation%2520of%2520Globular%2520Protein%2520is%2520a%2520First-Order%2520Phase%2520Transition%26jtitle%3DBiopolymers%26date%3D1989%26volume%3D28%26spage%3D1667%26epage%3D1680%26doi%3D10.1002%2Fbip.360281003" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mountain, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thirumalai, D.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics Simulations of End-to-End Contact Formation in Hydrocarbon Chains in Water and Aqueous Urea Solution</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1950</span>â <span class="NLM_lpage">1957</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja020496f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja020496f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmtVSnsA%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=125&amp;publication_year=2003&amp;pages=1950-1957&amp;author=R.+D.+Mountainauthor=D.+Thirumalai&amp;title=Molecular+Dynamics+Simulations+of+End-to-End+Contact+Formation+in+Hydrocarbon+Chains+in+Water+and+Aqueous+Urea+Solution&amp;doi=10.1021%2Fja020496f" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Dynamics Simulations of End-to-End Contact Formation in Hydrocarbon Chains in Water and Aqueous Urea Solution</span></div><div class="casAuthors">Mountain, Raymond D.; Thirumalai, D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1950-1957</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The urea-denaturation mechanism was studied using mol. dynamics simulations of an elementary "folding" event, namely, the formation of end-to-end contact in the linear hydrocarbon chain (HC) CH3(CH2)18CH3.  Electrostatic effects are examd. using a model HC in which one end of the chain is pos. charged (+0.2e) and the other contains a neg. charge (-0.2e).  For these systems multiple transitions between "folded" (conformations in which the chain ends are in contact) and "unfolded" (end-to-end contact is broken) can be obsd. during 4 ns mol. dynamics simulations.  In water and 6 M aq. urea soln. HC and the charged HC fluctuate between collapsed globular conformations and a set of expanded structures.  The collapsed conformation adopted by the HC in water is slightly destabilized in 6 M urea.  In contrast, the end-to-end contact is disrupted in the charged HC only in aq. urea soln.  Despite the presence of a large hydrophobic patch, on length scales on the order of â¼8-10 Ã "denaturation" (transition to the expanded unfolded state) occurs by a direct interaction of urea with charges on the chain ends.  The contiguous patch of hydrophobic moieties leads to "mild dewetting", which becomes more pronounced in the charged HC in 6 M aq. urea soln.  These simulations establish that the urea denaturation mechanism is most likely electrostatic in origin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp36dXv3EvoBbVg90H21EOLACvtfcHk0lj7DYiVyk2SLQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3sXmtVSnsA%253D%253D&amp;md5=5b08f62f14e7fc33d6f53b8ebd91043b</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fja020496f&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja020496f%26sid%3Dliteratum%253Aachs%26aulast%3DMountain%26aufirst%3DR.%2BD.%26aulast%3DThirumalai%26aufirst%3DD.%26atitle%3DMolecular%2520Dynamics%2520Simulations%2520of%2520End-to-End%2520Contact%2520Formation%2520in%2520Hydrocarbon%2520Chains%2520in%2520Water%2520and%2520Aqueous%2520Urea%2520Solution%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D1950%26epage%3D1957%26doi%3D10.1021%2Fja020496f" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoeltzli, S.
D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frieden, C.</span></span> <span> </span><span class="NLM_article-title">Stopped-Flow NMR Spectroscopy: Real-Time Unfolding Studies of 6-<sup>19</sup>F-Tryptophan-Labeled <i>Escherichia coli</i> Dihydrofolate Reductase</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">9318</span>â <span class="NLM_lpage">9322</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.92.20.9318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1073%2Fpnas.92.20.9318" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=92&amp;publication_year=1995&amp;pages=9318-9322&amp;author=S.%0AD.+Hoeltzliauthor=C.+Frieden&amp;title=Stopped-Flow+NMR+Spectroscopy%3A+Real-Time+Unfolding+Studies+of+6-19F-Tryptophan-Labeled+Escherichia+coli+Dihydrofolate+Reductase&amp;doi=10.1073%2Fpnas.92.20.9318" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1073%2Fpnas.92.20.9318&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.92.20.9318%26sid%3Dliteratum%253Aachs%26aulast%3DHoeltzli%26aufirst%3DS.%2BD.%26aulast%3DFrieden%26aufirst%3DC.%26atitle%3DStopped-Flow%2520NMR%2520Spectroscopy%253A%2520Real-Time%2520Unfolding%2520Studies%2520of%25206-19F-Tryptophan-Labeled%2520Escherichia%2520coli%2520Dihydrofolate%2520Reductase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1995%26volume%3D92%26spage%3D9318%26epage%3D9322%26doi%3D10.1073%2Fpnas.92.20.9318" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rami, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udgaonkar, J. B.</span></span> <span> </span><span class="NLM_article-title">Mechanism of Formation of a Productive Molten Globule Form of Barstarâ </span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1710</span>â <span class="NLM_lpage">1716</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi0120300</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0120300" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD38Xkt1CrtQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=41&amp;publication_year=2002&amp;pages=1710-1716&amp;author=B.+R.+Ramiauthor=J.+B.+Udgaonkar&amp;title=Mechanism+of+Formation+of+a+Productive+Molten+Globule+Form+of+Barstar%E2%80%A0&amp;doi=10.1021%2Fbi0120300" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of Formation of a Productive Molten Globule Form of Barstar</span></div><div class="casAuthors">Rami, Bhadresh R.; Udgaonkar, Jayant B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1710-1716</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structural anal. of the initial steps in protein folding is difficult because of the swiftness with which these steps occur.  Hence, the link between initial polypeptide chain collapse and formation of secondary and other specific structures remains poorly understood.  Here, an equil. model has been developed for characterizing the initial steps of folding of the small protein barstar, which lead to the formation of a productive molten globule in the folding pathway.  In this model, the high-pH-unfolded form (D form) of barstar, which is shown to be as unstructured as the urea-denatured form, is transformed progressively into a molten globule B form by incremental addn. of the salt Na2SO4 at pH 12.  At very low concns. of Na2SO4, the D form collapses into a pre-molten globule (P) form, whose vol. exceeds that of the native (N) state by only 20%, and which lacks any specific structure as detd. by far- and near-UV CD.  At higher concns. of Na2SO4, the P form transforms into the molten globule (B) form in a highly noncooperative transition populated by an ensemble of at least two intermediates.  The B form is a dry molten globule in which water is excluded from the core, and in which secondary structure develops to 65% and tertiary contacts develop to 40%, relative to that of the native protein.  Kinetic refolding expts. carried out at pH 7 and at high Na2SO4 concns., in which the rate of folding of the D form to the N state is compared to that of the B form to the N state, indicate conclusively that the B form is a productive intermediate that forms on the direct pathway of folding from the D form to the N state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLsutbyDvbr7Vg90H21EOLACvtfcHk0lg-wMua9MX24A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD38Xkt1CrtQ%253D%253D&amp;md5=b83f27e35312abc7aa844695395df31b</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fbi0120300&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0120300%26sid%3Dliteratum%253Aachs%26aulast%3DRami%26aufirst%3DB.%2BR.%26aulast%3DUdgaonkar%26aufirst%3DJ.%2BB.%26atitle%3DMechanism%2520of%2520Formation%2520of%2520a%2520Productive%2520Molten%2520Globule%2520Form%2520of%2520Barstar%25E2%2580%25A0%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26spage%3D1710%26epage%3D1716%26doi%3D10.1021%2Fbi0120300" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasinath, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moorman, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nucci, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilser, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Coupled Motion in Proteins Revealed by Pressure Perturbation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">8543</span>â <span class="NLM_lpage">8550</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja3004655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja3004655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC38XksFWgsbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=134&amp;publication_year=2012&amp;pages=8543-8550&amp;author=Y.+Fuauthor=V.+Kasinathauthor=V.+R.+Moormanauthor=N.+V.+Nucciauthor=V.+J.+Hilserauthor=A.+J.+Wand&amp;title=Coupled+Motion+in+Proteins+Revealed+by+Pressure+Perturbation&amp;doi=10.1021%2Fja3004655" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Coupled Motion in Proteins Revealed by Pressure Perturbation</span></div><div class="casAuthors">Fu, Yinan; Kasinath, Vignesh; Moorman, Veronica R.; Nucci, Nathaniel V.; Hilser, Vincent J.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8543-8550</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The cooperative nature of protein substructure and internal motion is a crit. aspect of their functional competence about which little is known exptl.  NMR relaxation is used here to monitor the effects of high pressure on fast internal motion in the protein ubiquitin.  In contrast to the main chain, the motions of the methyl-bearing side chains have a large and variable pressure dependence.  Within the core, this pressure sensitivity correlates with the magnitude of motion at ambient pressure.  Spatial clustering of the dynamic response to applied hydrostatic pressure is also seen, indicating localized cooperativity of motion on the sub-nanosecond time scale and suggesting regions of variable compressibility.  These and other features indicate that the native ensemble contains a significant fraction of members with characteristics ascribed to the recently postulated "dry molten globule".  The accompanying variable side-chain conformational entropy helps complete our view of the thermodn. architecture underlying protein stability, folding, and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa9mgJ5NQpqbVg90H21EOLACvtfcHk0lg-wMua9MX24A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XksFWgsbc%253D&amp;md5=9d8d93a5cd026221cb28e14914512ec6</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fja3004655&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja3004655%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DY.%26aulast%3DKasinath%26aufirst%3DV.%26aulast%3DMoorman%26aufirst%3DV.%2BR.%26aulast%3DNucci%26aufirst%3DN.%2BV.%26aulast%3DHilser%26aufirst%3DV.%2BJ.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DCoupled%2520Motion%2520in%2520Proteins%2520Revealed%2520by%2520Pressure%2520Perturbation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D8543%26epage%3D8550%26doi%3D10.1021%2Fja3004655" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marqusee, S.</span></span> <span> </span><span class="NLM_article-title">Kinetic Evidence for a Two-Stage Mechanism of Protein Denaturation by Guanidinium Chloride</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">4856</span>â <span class="NLM_lpage">4861</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1315453111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1073%2Fpnas.1315453111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=24639503" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksVShtL0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=111&amp;publication_year=2014&amp;pages=4856-4861&amp;author=S.+K.+Jhaauthor=S.+Marqusee&amp;title=Kinetic+Evidence+for+a+Two-Stage+Mechanism+of+Protein+Denaturation+by+Guanidinium+Chloride&amp;doi=10.1073%2Fpnas.1315453111" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic evidence for a two-stage mechanism of protein denaturation by guanidinium chloride</span></div><div class="casAuthors">Jha, Santosh Kumar; Marqusee, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4856-4861</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Dry molten globular (DMG) intermediates, an expanded form of the native protein with a dry core, have been obsd. during denaturant-induced unfolding of many proteins.  These observations are counterintuitive because traditional models of chem. denaturation rely on changes in solvent-accessible surface area, and there is no notable change in solvent-accessible surface area during the formation of the DMG.  Here we show, using multisite fluorescence resonance energy transfer, far-UV CD, and kinetic thiol-labeling expts., that the guanidinium chloride (GdmCl)-induced unfolding of RNase H also begins with the formation of the DMG.  Population of the DMG occurs within the 5-ms dead time of our measurements.  We observe that the size and/or population of the DMG is linearly dependent on [GdmCl], although not as strongly as the second and major step of unfolding, which is accompanied by core solvation and global unfolding.  This rapid GdmCl-dependent population of the DMG indicates that GdmCl can interact with the protein before disrupting the hydrophobic core.  These results imply that the effect of chem. denaturants cannot be interpreted solely as a disruption of the hydrophobic effect and strongly support recent computational studies, which hypothesize that chem. denaturants first interact directly with the protein surface before completely unfolding the protein in the second step (direct interaction mechanism).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsEgN87ohg-LVg90H21EOLACvtfcHk0lg-wMua9MX24A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXksVShtL0%253D&amp;md5=59f168c932bb8620d113b2e02dcf055f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1315453111&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1315453111%26sid%3Dliteratum%253Aachs%26aulast%3DJha%26aufirst%3DS.%2BK.%26aulast%3DMarqusee%26aufirst%3DS.%26atitle%3DKinetic%2520Evidence%2520for%2520a%2520Two-Stage%2520Mechanism%2520of%2520Protein%2520Denaturation%2520by%2520Guanidinium%2520Chloride%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D4856%26epage%3D4861%26doi%3D10.1073%2Fpnas.1315453111" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></div><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 9 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Ravi Kumar Venkatraman, <span class="NLM_string-name hlFld-ContribAuthor">Andrew J. Orr-Ewing</span>. </span><span class="cited-content_cbyCitation_article-title">Photochemistry of Benzophenone in Solution: A Tale of Two Different Solvent Environments. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2019,</strong> <em>141 </em>
                                    (38)
                                     , 15222-15229. <a href="https://doi.org/10.1021/jacs.9b07047" title="DOI URL">https://doi.org/10.1021/jacs.9b07047</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.9b07047&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.9b07047%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DPhotochemistry%252Bof%252BBenzophenone%252Bin%252BSolution%25253A%252BA%252BTale%252Bof%252BTwo%252BDifferent%252BSolvent%252BEnvironments%26aulast%3DVenkatraman%26aufirst%3DRavi%2BKumar%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D03072019%26date%3D10092019%26date%3D03092019%26volume%3D141%26issue%3D38%26spage%3D15222%26epage%3D15229" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Prajna  Mishra</span>, <span class="hlFld-ContribAuthor ">Santosh Kumar  Jha</span>. </span><span class="cited-content_cbyCitation_article-title">The native state conformational heterogeneity in the energy landscape of protein folding. </span><span class="cited-content_cbyCitation_journal-name">Biophysical Chemistry</span><span> <strong>2022,</strong> <em>283 </em>, 106761. <a href="https://doi.org/10.1016/j.bpc.2022.106761" title="DOI URL">https://doi.org/10.1016/j.bpc.2022.106761</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bpc.2022.106761&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bpc.2022.106761%26sid%3Dliteratum%253Aachs%26jtitle%3DBiophysical%2520Chemistry%26atitle%3DThe%252Bnative%252Bstate%252Bconformational%252Bheterogeneity%252Bin%252Bthe%252Benergy%252Blandscape%252Bof%252Bprotein%252Bfolding%26aulast%3DMishra%26aufirst%3DPrajna%26date%3D2022%26volume%3D283%26spage%3D106761" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Heber E.  Andrada</span>, <span class="hlFld-ContribAuthor ">Lisandro  Venosta</span>, <span class="hlFld-ContribAuthor ">Silvia E.  Jacobo</span>, <span class="hlFld-ContribAuthor ">O.  Fernando Silva</span>, <span class="hlFld-ContribAuthor ">R.  DarÃ­o Falcone</span>, <span class="hlFld-ContribAuthor ">Paula G.  Bercoff</span>. </span><span class="cited-content_cbyCitation_article-title">Highly Stable Nanostructured Magnetic Vesicles as Doxorubicin Carriers for Fieldâassisted Therapies. </span><span class="cited-content_cbyCitation_journal-name">ChemNanoMat</span><span> <strong>2022,</strong> <em>8 </em>
                                    (4)
                                     <a href="https://doi.org/10.1002/cnma.202100409" title="DOI URL">https://doi.org/10.1002/cnma.202100409</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cnma.202100409&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcnma.202100409%26sid%3Dliteratum%253Aachs%26jtitle%3DChemNanoMat%26atitle%3DHighly%252BStable%252BNanostructured%252BMagnetic%252BVesicles%252Bas%252BDoxorubicin%252BCarriers%252Bfor%252BField%2525E2%252580%252590assisted%252BTherapies%26aulast%3DAndrada%26aufirst%3DHeber%2BE.%26date%3D2022%26date%3D2022%26volume%3D8%26issue%3D4" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meimei  Huang</span>, <span class="hlFld-ContribAuthor ">Jia  Kong</span>, <span class="hlFld-ContribAuthor ">Jiaxing  Zhang</span>, <span class="hlFld-ContribAuthor ">Zixuan  Wang</span>, <span class="hlFld-ContribAuthor ">Dongzhao  Hao</span>, <span class="hlFld-ContribAuthor ">Jiaojiao  Zhang</span>, <span class="hlFld-ContribAuthor ">Yuefei  Wang</span>, <span class="hlFld-ContribAuthor ">Wei  Qi</span>, <span class="hlFld-ContribAuthor ">Rongxin  Su</span>, <span class="hlFld-ContribAuthor ">Zhimin  He</span>. </span><span class="cited-content_cbyCitation_article-title">Enhanced Polychromatic Luminescence of Bionic Peptidyl Nanoparticles Driven by Hydrogen Bonds. </span><span class="cited-content_cbyCitation_journal-name">Particle &amp; Particle Systems Characterization</span><span> <strong>2022,</strong> <em>39 </em>
                                    (2)
                                     , 2100260. <a href="https://doi.org/10.1002/ppsc.202100260" title="DOI URL">https://doi.org/10.1002/ppsc.202100260</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ppsc.202100260&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fppsc.202100260%26sid%3Dliteratum%253Aachs%26jtitle%3DParticle%2520%2526%2520Particle%2520Systems%2520Characterization%26atitle%3DEnhanced%252BPolychromatic%252BLuminescence%252Bof%252BBionic%252BPeptidyl%252BNanoparticles%252BDriven%252Bby%252BHydrogen%252BBonds%26aulast%3DHuang%26aufirst%3DMeimei%26date%3D2022%26date%3D2022%26volume%3D39%26issue%3D2%26spage%3D2100260" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Satyaki  Chatterjee</span>, <span class="hlFld-ContribAuthor ">Rupasree  Brahma</span>, <span class="hlFld-ContribAuthor ">H.  Raghuraman</span>. </span><span class="cited-content_cbyCitation_article-title">Gating-related Structural Dynamics of the MgtE Magnesium Channel in Membrane-Mimetics Utilizing Site-Directed Tryptophan Fluorescence. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2021,</strong> <em>433 </em>
                                    (17)
                                     , 166691. <a href="https://doi.org/10.1016/j.jmb.2020.10.025" title="DOI URL">https://doi.org/10.1016/j.jmb.2020.10.025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2020.10.025&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2020.10.025%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DGating-related%252BStructural%252BDynamics%252Bof%252Bthe%252BMgtE%252BMagnesium%252BChannel%252Bin%252BMembrane-Mimetics%252BUtilizing%252BSite-Directed%252BTryptophan%252BFluorescence%26aulast%3DChatterjee%26aufirst%3DSatyaki%26date%3D2021%26volume%3D433%26issue%3D17%26spage%3D166691" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Prajna  Mishra</span>, <span class="hlFld-ContribAuthor ">Divya  Patni</span>, <span class="hlFld-ContribAuthor ">Santosh Kumar  Jha</span>. </span><span class="cited-content_cbyCitation_article-title">A pH-dependent protein stability switch coupled to the perturbed pKa of a single ionizable residue. </span><span class="cited-content_cbyCitation_journal-name">Biophysical Chemistry</span><span> <strong>2021,</strong> <em>274 </em>, 106591. <a href="https://doi.org/10.1016/j.bpc.2021.106591" title="DOI URL">https://doi.org/10.1016/j.bpc.2021.106591</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bpc.2021.106591&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bpc.2021.106591%26sid%3Dliteratum%253Aachs%26jtitle%3DBiophysical%2520Chemistry%26atitle%3DA%252BpH-dependent%252Bprotein%252Bstability%252Bswitch%252Bcoupled%252Bto%252Bthe%252Bperturbed%252BpKa%252Bof%252Ba%252Bsingle%252Bionizable%252Bresidue%26aulast%3DMishra%26aufirst%3DPrajna%26date%3D2021%26volume%3D274%26spage%3D106591" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">, <span class="hlFld-ContribAuthor ">Rupasree  Brahma</span>, <span class="hlFld-ContribAuthor ">H.  Raghuraman</span>. </span><span class="cited-content_cbyCitation_article-title">Novel insights in linking solvent relaxation dynamics and protein conformations utilizing red edge excitation shift approach. </span><span class="cited-content_cbyCitation_journal-name">Emerging Topics in Life Sciences</span><span> <strong>2021,</strong> <em>5 </em>
                                    (1)
                                     , 89-101. <a href="https://doi.org/10.1042/ETLS20200256" title="DOI URL">https://doi.org/10.1042/ETLS20200256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/ETLS20200256&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FETLS20200256%26sid%3Dliteratum%253Aachs%26jtitle%3DEmerging%2520Topics%2520in%2520Life%2520Sciences%26atitle%3DNovel%252Binsights%252Bin%252Blinking%252Bsolvent%252Brelaxation%252Bdynamics%252Band%252Bprotein%252Bconformations%252Butilizing%252Bred%252Bedge%252Bexcitation%252Bshift%252Bapproach%26aulast%3DBose%26aufirst%3DKakoli%26date%3D2021%26date%3D2021%26volume%3D5%26issue%3D1%26spage%3D89%26epage%3D101" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weiwei  Su</span>, <span class="hlFld-ContribAuthor ">Juanjuan  Yin</span>, <span class="hlFld-ContribAuthor ">Ran  Wang</span>, <span class="hlFld-ContribAuthor ">Mengqian  Shi</span>, <span class="hlFld-ContribAuthor ">Ping  Liu</span>, <span class="hlFld-ContribAuthor ">Zhihui  Qin</span>, <span class="hlFld-ContribAuthor ">Ruirui  Xing</span>, <span class="hlFld-ContribAuthor ">Tifeng  Jiao</span>. </span><span class="cited-content_cbyCitation_article-title">Self-assembled natural biomacromolecular fluorescent hydrogels with tunable red edge effects. </span><span class="cited-content_cbyCitation_journal-name">Colloids and Surfaces A: Physicochemical and Engineering Aspects</span><span> <strong>2021,</strong> <em>612 </em>, 125993. <a href="https://doi.org/10.1016/j.colsurfa.2020.125993" title="DOI URL">https://doi.org/10.1016/j.colsurfa.2020.125993</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.colsurfa.2020.125993&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.colsurfa.2020.125993%26sid%3Dliteratum%253Aachs%26jtitle%3DColloids%2520and%2520Surfaces%2520A%253A%2520Physicochemical%2520and%2520Engineering%2520Aspects%26atitle%3DSelf-assembled%252Bnatural%252Bbiomacromolecular%252Bfluorescent%252Bhydrogels%252Bwith%252Btunable%252Bred%252Bedge%252Beffects%26aulast%3DSu%26aufirst%3DWeiwei%26date%3D2021%26volume%3D612%26spage%3D125993" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H.  Raghuraman</span>, <span class="hlFld-ContribAuthor ">Satyaki  Chatterjee</span>, <span class="hlFld-ContribAuthor ">Anindita  Das</span>. </span><span class="cited-content_cbyCitation_article-title">Site-Directed Fluorescence Approaches for Dynamic Structural Biology of Membrane Peptides and Proteins. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Molecular Biosciences</span><span> <strong>2019,</strong> <em>6 </em><a href="https://doi.org/10.3389/fmolb.2019.00096" title="DOI URL">https://doi.org/10.3389/fmolb.2019.00096</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmolb.2019.00096&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmolb.2019.00096%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Molecular%2520Biosciences%26atitle%3DSite-Directed%252BFluorescence%252BApproaches%252Bfor%252BDynamic%252BStructural%252BBiology%252Bof%252BMembrane%252BPeptides%252Band%252BProteins%26aulast%3DRaghuraman%26aufirst%3DH.%26date%3D2019%26date%3D2019%26volume%3D6" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li></ol></div></div></div></div><div class="ui-resizable-handle ui-resizable-e" style="z-index: 90;"></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class=""><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_0006.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jpcb.8b11151&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_0001.gif" alt="" data-lg-src="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic description of REES. Comparative effects of the central and the red-edge excitations (top) on the emission spectra (bottom) of a fluorophore are shown. (A) Timescale of the solvent relaxation, Ï<sub>s</sub> âª the fluorescence timescale, Ï<sub>f</sub>, (B) Ï<sub>s</sub> â« Ï<sub>f</sub>. In both the panels, the inner yellow circle and the thick gray arrow inside it represent the fluorophore and its dipole moment vector, respectively. The gray hexagons and their directions denote the orientations of the solvent molecules. The shaded gray distribution with the red tail represents the distribution of the soluteâsolvent interaction energy in the ground singlet state (S<sub>0</sub>). In the excited singlet state (S<sub>1</sub>), the dotted and solid cyan lines show the FranckâCondon (F) states for the center and the red-edge excitations, respectively. The dashed pink lines indicate the complete solvent-relaxed (R) states. The incomplete solvent-relaxed states (Râ²) for the center and the red-edge excitations are presented by the dotted and the solid pink lines, respectively. The vertical upward blue arrows represent the excitation energies (absorbance), and the vertically downward dashed green and solid red arrows represent the fluorescence emission energies. In the bottom panels, the green dashed lines and the red solid lines, respectively, represent the emission spectra when excited at the center (Î»<sub>C</sub>) and the red edge (Î»<sub>R</sub>) of the absorbance spectrum (the blue solid lines). In the panel (B) (top), the crooked line around the gray hexagons denotes a restricted solvation environment, as inside the core of a protein.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jpcb.8b11151&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_0002.gif" alt="" data-lg-src="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. C34 and C34-IAEDANS are buried in the protein structure in the N state, and HSA and HSA-IAEDANS have similar structure and stability. (A) Location of C34 and W214 is shown in the crystal structure of HSA (drawn from PDB file <a href="http://doi.org/10.2210/pdb1AO6/pdb">1AO6</a> using the program PyMOL). In the three-domain structure of the protein, C34 is completely buried in the core of domain I and W214 is partially buried in the interdomain region. (B) Structure of HSA showing the side-chain residues solvating C34 in the core of domain I. The atoms of C34 are shown in orange spheres. The side chain of the amino acids solvating C34 are shown in blue sticks. The background light gray surface around the amino acid residues shows the water accessible surface area of the residues burying C34. (C) Mean residue ellipticity (MRE) of HSA and HSA-IAEDANS is shown in the far-UV range in the blue dotted and red solid lines, respectively. (D) Change in the fraction of unfolded molecules (FU) as a function of [urea] is shown for HSA (blue circles) and HSA-IAEDANS (red circles). The blue solid line through the circles is a fit to a two-state, N â U model. (E) Fluorescence emission spectra of HSA-IAEDANS in the N and the U states are shown at different pHs (indicated in the figure). Each spectrum was normalized at its maximum intensity. (F) Fluorescence lifetime decay of C34-IAEDANS in the N and the U states at different pHs as indicated in the figure is shown (IRF: Instrument Response Function).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jpcb.8b11151&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_0003.gif" alt="" data-lg-src="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Core dynamics and heterogeneity of HSA as revealed by REES. (A) Molar absorptivity coefficient spectrum, Îµ (Î»), of C34-IAEDANS is shown in panel (A). The absorption spectrum has been divided by the molar protein concentration to obtain Îµ (Î»). (B) Fluorescence emission spectra of C34-IAEDANS when excited at different wavelengths in the red-edge regions in the N and the U states at pH 7 are shown in panel (B) and its inset, respectively. (C) Panel (C) shows the dependence of the wavelength of maximum fluorescence emission (Î»<sub class="stack">max</sub><sup class="stack">em</sup>) on the wavelength of excitation (Î»<sup>ex</sup>) of C34-IAEDANS in the N and the U states at pHs 5, 7, and 8 and in the presence of 50% glycerol at pH 7. (D) Extent of REES as observed in C34-IAEDANS at all of the U and N states at pHs 5, 7, and 8 and in the presence of 50% glycerol at pH 7.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jpcb.8b11151&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_0004.gif" alt="" data-lg-src="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Change in viscosity of exterior solvent modulates the REES of W214 (A) the molar absorptivity coefficient spectrum, Îµ (Î») of W214. (B) Representative fluorescence emission spectra of W214 at pH 7 when excited at different wavelengths of excitation at the red edge of its absorbance spectrum. (C) Panel (C) shows the representative fluorescence emission spectra of W214 in the presence of 50% glycerol at pH 7 upon excitation at different wavelengths of red edge of its absorbance spectrum. For comparison, all of the spectra were normalized to 1 at their emission maximum. (D) Dependence of Î»<sub class="stack">max</sub><sup class="stack">em</sup> on Î»<sup>ex</sup> of W214 in the N state and in the presence of 50% glycerol at pH 7.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jpcb.8b11151&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/cms/10.1021/acs.jpcb.8b11151/asset/images/medium/jp-2018-111517_0005.gif" alt="" data-lg-src="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Global structural properties of HSA. (A) Fluorescence emission spectra of the protein at pHs 5, 7, and 8 in both the N and the U forms, when excited at 280 nm is shown. (B) Panel (B) and its inset show changes in MRE with respect to wavelength as a measure of global secondary and tertiary structures, respectively, in the N and the U forms at pHs 5, 7, and 8.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acs.jpcb.8b11151/asset/images/large/jp-2018-111517_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jpcb.8b11151&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i30">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10408" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10408" class="dropBlock__holder"></div></div></div></div></div><hr></div><p class="rightTab-references-count">
            This article references 54 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">SchrÃ¶dinger, E.</span></span> <span> </span><span class="NLM_article-title">What is Life?</span> <i>The Physical Aspect of the Living Cell</i>; <span class="NLM_publisher-name">The University press; The Macmillan company</span>: <span class="NLM_publisher-loc">Cambridge England</span>, <span class="NLM_year">1945</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1945&amp;author=E.+Schr%C3%B6dinger&amp;title=The+Physical+Aspect+of+the+Living+Cell" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6dinger%26aufirst%3DE.%26atitle%3DWhat%2520is%2520Life%253F%26btitle%3DThe%2520Physical%2520Aspect%2520of%2520the%2520Living%2520Cell%26pub%3DThe%2520University%2520press%253B%2520The%2520Macmillan%2520company%26date%3D1945" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitkup, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, M.</span></span> <span> </span><span class="NLM_article-title">Native proteins are surface-molten solids: application of the lindemann criterion for the solid versus liquid state 1 1Edited by A. R. Fersht</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">1371</span>â <span class="NLM_lpage">1375</span>, <span class="refDoi">&nbsp;DOI: 10.1006/jmbi.1998.2374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1006%2Fjmbi.1998.2374" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=285&amp;publication_year=1999&amp;pages=1371-1375&amp;author=Y.+Zhouauthor=D.+Vitkupauthor=M.+Karplus&amp;title=Native+proteins+are+surface-molten+solids%3A+application+of+the+lindemann+criterion+for+the+solid+versus+liquid+state+1+1Edited+by+A.+R.+Fersht&amp;doi=10.1006%2Fjmbi.1998.2374" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1998.2374&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1998.2374%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DVitkup%26aufirst%3DD.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DNative%2520proteins%2520are%2520surface-molten%2520solids%253A%2520application%2520of%2520the%2520lindemann%2520criterion%2520for%2520the%2520solid%2520versus%2520liquid%2520state%25201%25201Edited%2520by%2520A.%2520R.%2520Fersht%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1999%26volume%3D285%26spage%3D1371%26epage%3D1375%26doi%3D10.1006%2Fjmbi.1998.2374" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geissler, P. L.</span></span> <span> </span><span class="NLM_article-title">Extensive Conformational Heterogeneity within Protein Cores</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">6417</span>â <span class="NLM_lpage">6423</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jp4105823</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp4105823" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFylurs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=118&amp;publication_year=2014&amp;pages=6417-6423&amp;author=G.+R.+Bowmanauthor=P.+L.+Geissler&amp;title=Extensive+Conformational+Heterogeneity+within+Protein+Cores&amp;doi=10.1021%2Fjp4105823" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Extensive conformational heterogeneity within protein cores</span></div><div class="casAuthors">Bowman, Gregory R.; Geissler, Phillip L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6417-6423</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Basic principles of statistical mechanics require that proteins sample an ensemble of conformations at any non-zero temp.  However, it is still common to treat the crystallog. structure of a protein as the structure of its native state, largely because high-resoln. structural characterization of protein flexibility remains a profound challenge.  To assess the typical degree of conformational heterogeneity within folded proteins, the authors constructed Markov state models describing the thermodn. and kinetics of proteins ranging from 72 to 263 residues in length.  Each of these models was built from hundreds of microseconds of atomically detailed mol. dynamics simulations.  Examn. of the side-chain degrees of freedom revealed that almost every residue visited at least 2 rotameric states over this time frame, with rotamer transition rates spanning a wide range of time scales (from nanoseconds to tens of microseconds).  The authors also reported substantial backbone dynamics on time scales longer than were typically addressed by exptl. measures of protein flexibility, such as NMR order parameters.  Finally, the authors demonstrated that these extensive rearrangements were consistent with NMR and crystallog. data, which supported the validity of their models.  Altogether, these results depict the interior of proteins not as well-ordered solids, as is often imagined, but instead as dense fluids, which undergo substantial structural fluctuations despite their high packing fraction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe_1No4CoVqrVg90H21EOLACvtfcHk0liSjdFJsnJ9Xg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXivFylurs%253D&amp;md5=d7e368b268bd1406f126d77728c33dd3</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjp4105823&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp4105823%26sid%3Dliteratum%253Aachs%26aulast%3DBowman%26aufirst%3DG.%2BR.%26aulast%3DGeissler%26aufirst%3DP.%2BL.%26atitle%3DExtensive%2520Conformational%2520Heterogeneity%2520within%2520Protein%2520Cores%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2014%26volume%3D118%26spage%3D6417%26epage%3D6423%26doi%3D10.1021%2Fjp4105823" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DuBay, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geissler, P. L.</span></span> <span> </span><span class="NLM_article-title">Fluctuations within Folded Proteins: Implications for Thermodynamic and Allosteric Regulation</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1098</span>â <span class="NLM_lpage">1105</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar500351b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar500351b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFSqt7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=48&amp;publication_year=2015&amp;pages=1098-1105&amp;author=K.+H.+DuBayauthor=G.+R.+Bowmanauthor=P.+L.+Geissler&amp;title=Fluctuations+within+Folded+Proteins%3A+Implications+for+Thermodynamic+and+Allosteric+Regulation&amp;doi=10.1021%2Far500351b" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Fluctuations within folded proteins: Implications for thermodynamic and allosteric regulation</span></div><div class="casAuthors">DuBay, Kateri H.; Bowman, Gregory R.; Geissler, Phillip L.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1098-1105</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Folded protein structures are both stable and dynamic.  Historically, the clearest window into these structures came from x-ray crystallog., which generally provided a static image of each protein's singular "folded state", highlighting its stability.  Deviations away from that crystallog. structure were difficult to quantify, and as a result, their potential functional consequences were often neglected.  However, several dynamical and statistical studies now highlight the structural variability that is present within the protein's folded state.  Here, the authors review mounting evidence of the importance of these structural rearrangements; both expt. and computation indicate that folded proteins undergo substantial fluctuations that can greatly influence their function.  Crucially, recent studies have shown that structural elements of proteins, esp. their side-chain degrees of freedom, fluctuate in ways that generate significant conformational heterogeneity.  The entropy assocd. with these motions contributes to the folded structure's thermodn. stability.  In addn., since these fluctuations can shift in response to perturbations such as ligand binding, they may play an important role in the protein's capacity to respond to environmental cues.  In one compelling example, the entropy assocd. with side-chain fluctuations contributes significantly to regulating the binding of calmodulin to a set of peptide ligands.  The neglect of fluctuations within protein native states was often justified by the dense packing within folded proteins, which has inspired comparisons with cryst. solids.  Many liqs., however, can achieve similarly dense packing yet fluidity is maintained through correlated mol. motions.  Indeed, the studies discussed here favor comparison of folded proteins not with solids but instead with dense liqs., where the internal side-chain fluidity is facilitated by collective motions that are correlated over long distances.  These correlated rearrangements can enable allosteric communication between different parts of a protein, through subtle and varied channels.  Such long-range correlations appear to be an innate feature of proteins in general, manifest even in mols. lacking known allosteric regulators and arising robustly from the phys. nature of their internal environment.  Given their ubiquity, it is only to be expected that, over time, Nature has refined some subset of these correlated motions and put them to use.  Native state fluctuations increasingly appear to be vital for proteins' natural functions.  Understanding the diversity, origin, and range of these rearrangements may provide novel routes for rationally manipulating biomol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJMobE8ZDwRLVg90H21EOLACvtfcHk0liSjdFJsnJ9Xg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXivFSqt7w%253D&amp;md5=cbe612ab9f1f4fdf930d916f05531076</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Far500351b&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far500351b%26sid%3Dliteratum%253Aachs%26aulast%3DDuBay%26aufirst%3DK.%2BH.%26aulast%3DBowman%26aufirst%3DG.%2BR.%26aulast%3DGeissler%26aufirst%3DP.%2BL.%26atitle%3DFluctuations%2520within%2520Folded%2520Proteins%253A%2520Implications%2520for%2520Thermodynamic%2520and%2520Allosteric%2520Regulation%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26spage%3D1098%26epage%3D1105%26doi%3D10.1021%2Far500351b" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richards, F.
M.</span></span> <span> </span><span class="NLM_article-title">The Interpretation of Protein Structures: Total Volume, Group Volume Distributions and Packing Density</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">14</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0022-2836(74)90570-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2F0022-2836%2874%2990570-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=4818482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaE2cXkvVKqsL4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=82&amp;publication_year=1974&amp;pages=1-14&amp;author=F.%0AM.+Richards&amp;title=The+Interpretation+of+Protein+Structures%3A+Total+Volume%2C+Group+Volume+Distributions+and+Packing+Density&amp;doi=10.1016%2F0022-2836%2874%2990570-1" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Interpretation of protein structures.  Total volume, group volume distributions, and packing density</span></div><div class="casAuthors">Richards, Frederic M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1974</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-14</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Vols. occupied by all the atoms or groups of atoms in lysozyme [9001-63-2] and RNase S [9001-99-4] were estd. from at. coordinates provided by crystal structure studies.  The av. packing d. for the interior of both proteins was â¼0.75 and the mean packing d. for all at. types was 0.7-0.8.  Packing d. averaged over 5-15 atoms varied in different parts of the same protein; low d. representing packing defects may facilitate mobility, e.g. at active sites, whereas surrounding high d. areas may be involved in transmission or correlation of motions over large mol. areas.  The total vol. of the 2 enzymes as derived from the coordinate list were 7-10% larger than those estd. from partial sp. vols.  Attributing this vol. difference to a change in packing of H2O in a monolayer surrounding the protein corresponds to an av. decrease of 1-2 Ã3/H2O mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfkJ9sxXcPJbVg90H21EOLACvtfcHk0lgSRW5hZBRCHQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaE2cXkvVKqsL4%253D&amp;md5=e4c9e4c66cbee1e390b9ea52851c1a82</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2874%2990570-1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252874%252990570-1%26sid%3Dliteratum%253Aachs%26aulast%3DRichards%26aufirst%3DF.%2BM.%26atitle%3DThe%2520Interpretation%2520of%2520Protein%2520Structures%253A%2520Total%2520Volume%252C%2520Group%2520Volume%2520Distributions%2520and%2520Packing%2520Density%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1974%26volume%3D82%26spage%3D1%26epage%3D14%26doi%3D10.1016%2F0022-2836%2874%2990570-1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chothia, C.</span></span> <span> </span><span class="NLM_article-title">Structural Invariants in Protein Folding</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1975</span>,  <span class="NLM_volume"><i>254</i></span>,  <span class="NLM_fpage">304</span>â <span class="NLM_lpage">308</span>, <span class="refDoi">&nbsp;DOI: 10.1038/254304a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2F254304a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1118010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaE2MXktl2js7s%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=254&amp;publication_year=1975&amp;pages=304-308&amp;author=C.+Chothia&amp;title=Structural+Invariants+in+Protein+Folding&amp;doi=10.1038%2F254304a0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structural invariants in protein folding</span></div><div class="casAuthors">Chothia, Cyrus</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">254</span>
        (<span class="NLM_cas:issue">5498</span>),
    <span class="NLM_cas:pages">304-8</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Anal. for 15 proteins of structural descriptions directly related to energy terms showed that protein structures can be described by simple equations or expressions.  The loss of accessible surface area by monomeric proteins on folding was a simple function of mol. wt., the proportion of polar groups forming intramol. H-bonds was const., and protein interiors were closely packed, each residue occupying the same vol. as its does in crystals of amino acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa_GFhyFCinLVg90H21EOLACvtfcHk0lgSRW5hZBRCHQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaE2MXktl2js7s%253D&amp;md5=aa04ff07cd5f1612d50cdb5cc8430afc</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2F254304a0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F254304a0%26sid%3Dliteratum%253Aachs%26aulast%3DChothia%26aufirst%3DC.%26atitle%3DStructural%2520Invariants%2520in%2520Protein%2520Folding%26jtitle%3DNature%26date%3D1975%26volume%3D254%26spage%3D304%26epage%3D308%26doi%3D10.1038%2F254304a0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richards, F. M.</span></span> <span> </span><span class="NLM_article-title">Areas, Volumes, Packing, and Protein Structure</span>. <i>Annu. Rev. Biophys. Bioeng.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">151</span>â <span class="NLM_lpage">176</span>, <span class="refDoi">&nbsp;DOI: 10.1146/annurev.bb.06.060177.001055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1146%2Fannurev.bb.06.060177.001055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=326146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaE2sXks1CisrY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=1977&amp;pages=151-176&amp;author=F.+M.+Richards&amp;title=Areas%2C+Volumes%2C+Packing%2C+and+Protein+Structure&amp;doi=10.1146%2Fannurev.bb.06.060177.001055" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Areas, volumes, packing, and protein structure</span></div><div class="casAuthors">Richards, Frederic M.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Bioengineering</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">151-76</span>CODEN:
                <span class="NLM_cas:coden">ABPBBK</span>;
        ISSN:<span class="NLM_cas:issn">0084-6589</span>.
    </div><div class="casAbstract">A review with 148 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvHGphzlx3JbVg90H21EOLACvtfcHk0liFaJKqWt06Pg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaE2sXks1CisrY%253D&amp;md5=13c917ccbda3f36f1c6390a42c712818</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1146%2Fannurev.bb.06.060177.001055&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.bb.06.060177.001055%26sid%3Dliteratum%253Aachs%26aulast%3DRichards%26aufirst%3DF.%2BM.%26atitle%3DAreas%252C%2520Volumes%252C%2520Packing%252C%2520and%2520Protein%2520Structure%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Bioeng.%26date%3D1977%26volume%3D6%26spage%3D151%26epage%3D176%26doi%3D10.1146%2Fannurev.bb.06.060177.001055" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reiner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henklein, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefhaber, T.</span></span> <span> </span><span class="NLM_article-title">An Unlocking/Relocking Barrier in Conformational Fluctuations of Villin Headpiece Subdomain</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">4955</span>â <span class="NLM_lpage">4960</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.0910001107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1073%2Fpnas.0910001107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=20194774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFGntrg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=107&amp;publication_year=2010&amp;pages=4955-4960&amp;author=A.+Reinerauthor=P.+Henkleinauthor=T.+Kiefhaber&amp;title=An+Unlocking%2FRelocking+Barrier+in+Conformational+Fluctuations+of+Villin+Headpiece+Subdomain&amp;doi=10.1073%2Fpnas.0910001107" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">An unlocking/relocking barrier in conformational fluctuations of villin headpiece subdomain</span></div><div class="casAuthors">Reiner, Andreas; Henklein, Peter; Kiefhaber, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4955-4960, S4955/1-S4955/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A reversible structural unlocking reaction, in which the close-packed van der Waals interactions break cooperatively, has been found for the villin headpiece subdomain (HP35) using triplet-triplet-energy transfer to monitor conformational fluctuations from equil.  Unlocking is assocd. with an unfavorable enthalpy change (ÎH0 = 35 Â± 4 kJ/mol) which is nearly compensated in free energy by the entropy change (ÎS0 = 112 Â± 20 J/mol-1/K-1).  The unlocking reaction has a time const. of about 1 Î¼s at 5Â° and is enthalpy-limited with an activation energy of 32 Â± 1 kJ/mol and a large Arrhenius preexponential factor of A = 7.5 Ã 1011 s-1.  In the unlocked state a fast local conformational fluctuation with a time const. of 170 ns and a low activation barrier of 17 Â± 1 kJ/mol leads to unfolding of the C-terminal helix and to its undocking from the core.  On a much slower time scale, global unfolding occurs from the unlocked state.  These results suggest that native protein structures are locked into conformations with low amplitude motions.  Large scale motions and global unfolding require an initial structural unlocking step leading to a state with properties of a dry molten globule.  The expts. addnl. yielded information on the dynamics of loop formation between different positions in unfolded HP35.  Comparison of the results with dynamics in unstructured model peptides indicates slightly decelerated kinetics of local loop formation in the C-terminal region which points at residual, nonrandom structure.  Dynamics of long-range loop formation, in contrast, are not influenced by residual structure, which argues against unfolded state properties as mol. origin for ultrafast folding of HP35.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSNXIYzC31_7Vg90H21EOLACvtfcHk0liFaJKqWt06Pg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXjvFGntrg%253D&amp;md5=09e9c15dac4b26c12147179c34fbba9f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0910001107&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0910001107%26sid%3Dliteratum%253Aachs%26aulast%3DReiner%26aufirst%3DA.%26aulast%3DHenklein%26aufirst%3DP.%26aulast%3DKiefhaber%26aufirst%3DT.%26atitle%3DAn%2520Unlocking%252FRelocking%2520Barrier%2520in%2520Conformational%2520Fluctuations%2520of%2520Villin%2520Headpiece%2520Subdomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D4955%26epage%3D4960%26doi%3D10.1073%2Fpnas.0910001107" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frieden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, G. D.</span></span> <span> </span><span class="NLM_article-title">Dry Molten Globule Intermediates and the Mechanism of Protein Unfolding</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">2725</span>â <span class="NLM_lpage">2737</span>, <span class="refDoi">&nbsp;DOI: 10.1002/prot.22803</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1002%2Fprot.22803" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=78&amp;publication_year=2010&amp;pages=2725-2737&amp;author=R.+L.+Baldwinauthor=C.+Friedenauthor=G.+D.+Rose&amp;title=Dry+Molten+Globule+Intermediates+and+the+Mechanism+of+Protein+Unfolding&amp;doi=10.1002%2Fprot.22803" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fprot.22803&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22803%26sid%3Dliteratum%253Aachs%26aulast%3DBaldwin%26aufirst%3DR.%2BL.%26aulast%3DFrieden%26aufirst%3DC.%26aulast%3DRose%26aufirst%3DG.%2BD.%26atitle%3DDry%2520Molten%2520Globule%2520Intermediates%2520and%2520the%2520Mechanism%2520of%2520Protein%2520Unfolding%26jtitle%3DProteins%26date%3D2010%26volume%3D78%26spage%3D2725%26epage%3D2737%26doi%3D10.1002%2Fprot.22803" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thirumalai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span> <span> </span><span class="NLM_article-title">Protein Folding: From Theory to Practice</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">22</span>â <span class="NLM_lpage">29</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.sbi.2012.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2Fj.sbi.2012.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=23266001" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2ktb7M" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2013&amp;pages=22-29&amp;author=D.+Thirumalaiauthor=Z.+Liuauthor=E.+P.+O%E2%80%99Brienauthor=G.+Reddy&amp;title=Protein+Folding%3A+From+Theory+to+Practice&amp;doi=10.1016%2Fj.sbi.2012.11.010" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Protein folding: from theory to practice</span></div><div class="casAuthors">Thirumalai, D.; Liu, Zhenxing; O'Brien, Edward P.; Reddy, Govardhan</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-29</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  A quant. theory of protein folding should make testable predictions using theor. models and simulations performed under conditions that closely mimic those used in expts.  Typically, in lab. expts. folding or unfolding is initiated using denaturants or external mech. force, whereas theories and simulations use temp. as the control parameter, thus making it difficult to make direct comparisons with expts.  The mol. transfer model (MTM), which incorporates environmental changes using measured quantities in mol. simulations, overcomes these difficulties.  Predictions of the folding thermodn. and kinetics of a no. of proteins using MTM simulations are in remarkable agreement with expts.  The MTM and all atom simulations demonstrating the presence of dry globules represent major advances in the proteins folding field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4w-0X831BibVg90H21EOLACvtfcHk0liFaJKqWt06Pg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhvV2ktb7M&amp;md5=05c84df577495f29de7ef2c4610d8e4f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2012.11.010&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2012.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DThirumalai%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DE.%2BP.%26aulast%3DReddy%26aufirst%3DG.%26atitle%3DProtein%2520Folding%253A%2520From%2520Theory%2520to%2520Practice%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2013%26volume%3D23%26spage%3D22%26epage%3D29%26doi%3D10.1016%2Fj.sbi.2012.11.010" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span> <span> </span><span class="NLM_article-title">An Alternatively Packed Dry Molten Globule-Like Intermediate in the Native State Ensemble of a Multidomain Protein</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">9336</span>â <span class="NLM_lpage">9347</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.jpcb.7b07032</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jpcb.7b07032" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVyis7bF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=121&amp;publication_year=2017&amp;pages=9336-9347&amp;author=P.+Mishraauthor=S.+K.+Jha&amp;title=An+Alternatively+Packed+Dry+Molten+Globule-Like+Intermediate+in+the+Native+State+Ensemble+of+a+Multidomain+Protein&amp;doi=10.1021%2Facs.jpcb.7b07032" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">An Alternatively Packed Dry Molten Globule-like Intermediate in the Native State Ensemble of a Multidomain Protein</span></div><div class="casAuthors">Mishra, Prajna; Jha, Santosh Kumar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">9336-9347</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">It has been difficult to quantify the degree of side-chain conformational heterogeneity in the native (N) state ensemble of proteins and the relative energetic contributions of the side-chain packing and the hydrophobic effect in protein stability.  Here, we show using multiple site-specific spectroscopic probes and tools of thermodn. that the N state ensemble of the multi-domain protein, human serum albumin, contains an equil. intermediate (I) whose inter-domain region resembles a dry molten globule (DMG).  In the I state, a Trp residue (Trp-214) in the inter-domain region was alternatively packed but its secondary structure and intra-domain packing were N-like.  The I state also had a larger inter-domain distance, but the domain-domain interface was dry and molten.  These results indicated that hydrophobic desolvation and side-chain packing are decoupled during protein folding and that inter-domain packing interactions have an important energetic contribution in protein stability.  Dynamic inter-conversion between alternatively packed N-like states could be important for multiple allosteric and ligand binding functions of this protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXjR8Vs542aLVg90H21EOLACvtfcHk0lhA6gNur7PxmQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhsVyis7bF&amp;md5=8b1803a2deff90ca08b7e3c44f20a5dd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jpcb.7b07032&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jpcb.7b07032%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DP.%26aulast%3DJha%26aufirst%3DS.%2BK.%26atitle%3DAn%2520Alternatively%2520Packed%2520Dry%2520Molten%2520Globule-Like%2520Intermediate%2520in%2520the%2520Native%2520State%2520Ensemble%2520of%2520a%2520Multidomain%2520Protein%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2017%26volume%3D121%26spage%3D9336%26epage%3D9347%26doi%3D10.1021%2Facs.jpcb.7b07032" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, G. D.</span></span> <span> </span><span class="NLM_article-title">Molten Globules, Entropy-Driven Conformational Change and Protein Folding</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">4</span>â <span class="NLM_lpage">10</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.sbi.2012.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2Fj.sbi.2012.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=23237704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWgtLnN" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2013&amp;pages=4-10&amp;author=R.+L.+Baldwinauthor=G.+D.+Rose&amp;title=Molten+Globules%2C+Entropy-Driven+Conformational+Change+and+Protein+Folding&amp;doi=10.1016%2Fj.sbi.2012.11.004" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Molten globules, entropy-driven conformational change and protein folding</span></div><div class="casAuthors">Baldwin, Robert L.; Rose, George D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-10</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The exquisite side chain close-packing in the protein core and at binding interfaces has prompted a conviction that packing selectivity is the primary mechanism for mol. recognition in folding and/or binding reactions.  Contrary to this view, molten globule proteins can adopt native topol. and bind targets tightly and specifically in the absence of side chain close-packing.  The molten globule is a highly dynamic form with native-like secondary structure and a loose protein core that admits solvent.  The related (but still controversial) dry molten globule is an expanded form of the native protein with largely intact topol. but a tighter protein core that excludes solvent.  Neither form retains side chain close-packing, and therefore both structure and function must result from other factors, assuming that the reality of the dry molten globule is accepted.  This simplifying realization calls for a re-evaluation of established models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNV9HY3Veq-rVg90H21EOLACvtfcHk0lhA6gNur7PxmQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhvVWgtLnN&amp;md5=0c958500dc87edd17bad7e04c0a36a0e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2012.11.004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2012.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DBaldwin%26aufirst%3DR.%2BL.%26aulast%3DRose%26aufirst%3DG.%2BD.%26atitle%3DMolten%2520Globules%252C%2520Entropy-Driven%2520Conformational%2520Change%2520and%2520Protein%2520Folding%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2013%26volume%3D23%26spage%3D4%26epage%3D10%26doi%3D10.1016%2Fj.sbi.2012.11.004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapienza, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petit, C. M.</span></span> <span> </span><span class="NLM_article-title">Native State Volume Fluctuations in Proteins as a Mechanism for Dynamic Allostery</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">3599</span>â <span class="NLM_lpage">3602</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jacs.6b12058</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b12058" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKitLY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=139&amp;publication_year=2017&amp;pages=3599-3602&amp;author=A.+B.+Lawauthor=P.+J.+Sapienzaauthor=J.+Zhangauthor=X.+Zuoauthor=C.+M.+Petit&amp;title=Native+State+Volume+Fluctuations+in+Proteins+as+a+Mechanism+for+Dynamic+Allostery&amp;doi=10.1021%2Fjacs.6b12058" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Native State Volume Fluctuations in Proteins as a Mechanism for Dynamic Allostery</span></div><div class="casAuthors">Law, Anthony B.; Sapienza, Paul J.; Zhang, Jun; Zuo, Xiaobing; Petit, Chad M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3599-3602</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allostery enables tight regulation of protein function in the cellular environment.  While existing models of allostery are firmly rooted in the current structure-function paradigm, the mechanistic basis for allostery in the absence of structural change remains unclear.  In this study, we show that a typical globular protein is able to undergo significant changes in vol. under native conditions while exhibiting no addnl. changes in protein structure.  These native state vol. fluctuations were found to correlate with changes in internal motions that were previously recognized as a source of allosteric entropy.  This finding offers a novel mechanistic basis for allostery in the absence of canonical structural change.  The unexpected observation that function can be derived from expanded, low d. protein states has broad implications for our understanding of allostery and suggests that the general concept of the native state be expanded to allow for more variable phys. dimensions with looser packing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5SsKYemFuR7Vg90H21EOLACvtfcHk0liBMsMY3VRuAg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKitLY%253D&amp;md5=07b2b88933154f96fe19cfaac28d5faa</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b12058&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b12058%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DA.%2BB.%26aulast%3DSapienza%26aufirst%3DP.%2BJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZuo%26aufirst%3DX.%26aulast%3DPetit%26aufirst%3DC.%2BM.%26atitle%3DNative%2520State%2520Volume%2520Fluctuations%2520in%2520Proteins%2520as%2520a%2520Mechanism%2520for%2520Dynamic%2520Allostery%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D3599%26epage%3D3602%26doi%3D10.1021%2Fjacs.6b12058" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kiefhaber, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labhardt, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, R. L.</span></span> <span> </span><span class="NLM_article-title">Direct NMR Evidence for an Intermediate Preceding the Rate-Limiting Step in the Unfolding of Ribonuclease A</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>375</i></span>,  <span class="NLM_fpage">513</span>â <span class="NLM_lpage">515</span>, <span class="refDoi">&nbsp;DOI: 10.1038/375513a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2F375513a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=7777063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaK2MXmtFCgsbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=375&amp;publication_year=1995&amp;pages=513-515&amp;author=T.+Kiefhaberauthor=A.+M.+Labhardtauthor=R.+L.+Baldwin&amp;title=Direct+NMR+Evidence+for+an+Intermediate+Preceding+the+Rate-Limiting+Step+in+the+Unfolding+of+Ribonuclease+A&amp;doi=10.1038%2F375513a0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Direct NMR evidence for an intermediate preceding the rate-limiting step in the unfolding of ribonuclease A</span></div><div class="casAuthors">Kiefhaber, Thomas; Labhardt, Alexander M.; Baldwin, Robert L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">375</span>
        (<span class="NLM_cas:issue">6531</span>),
    <span class="NLM_cas:pages">513-15</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">It is commonly believed that there are no detectable intermediates in the kinetic unfolding reactions of small proteins.  If such intermediates could be found, they would give important information about the nature of the transition state for unfolding, which is thought to occur close to the native state.  The authors report here that one-dimensional proton magnetic resonance spectra recorded during the unfolding of RNase A provide direct evidence for at least one unfolding intermediate in which side chains are free to rotate.  This intermediate appears to be a 'dry molten globule' of the kind hypothesized by Shakhnovich and Finkelstein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIHBAF0z1taLVg90H21EOLACvtfcHk0liBMsMY3VRuAg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK2MXmtFCgsbs%253D&amp;md5=552b2a31424bf7915dc8e8d779bd2856</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2F375513a0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F375513a0%26sid%3Dliteratum%253Aachs%26aulast%3DKiefhaber%26aufirst%3DT.%26aulast%3DLabhardt%26aufirst%3DA.%2BM.%26aulast%3DBaldwin%26aufirst%3DR.%2BL.%26atitle%3DDirect%2520NMR%2520Evidence%2520for%2520an%2520Intermediate%2520Preceding%2520the%2520Rate-Limiting%2520Step%2520in%2520the%2520Unfolding%2520of%2520Ribonuclease%2520A%26jtitle%3DNature%26date%3D1995%26volume%3D375%26spage%3D513%26epage%3D515%26doi%3D10.1038%2F375513a0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udgaonkar, J. B.</span></span> <span> </span><span class="NLM_article-title">Direct Evidence for a Dry Molten Globule Intermediate During the Unfolding of a Small Protein</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">12289</span>â <span class="NLM_lpage">12294</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.0905744106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1073%2Fpnas.0905744106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=19617531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpslCjsrY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=106&amp;publication_year=2009&amp;pages=12289-12294&amp;author=S.+K.+Jhaauthor=J.+B.+Udgaonkar&amp;title=Direct+Evidence+for+a+Dry+Molten+Globule+Intermediate+During+the+Unfolding+of+a+Small+Protein&amp;doi=10.1073%2Fpnas.0905744106" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Direct evidence for a dry molten globule intermediate during the unfolding of a small protein</span></div><div class="casAuthors">Jha, Santosh Kumar; Udgaonkar, Jayant B.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">12289-12294, S12289/1-S12289/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Little is known about how proteins begin to unfold.  In particular, how and when water mols. penetrate into the protein interior during unfolding, thereby enabling the dissoln. of specific structure, is poorly understood.  The hypothesis that the native state expands initially into a dry molten globule (MG), in which tight packing interactions are broken, but whose hydrophobic core has not expanded sufficiently to be able to absorb water mols., has very little exptl. support.  Here, the authors report their anal. of the earliest observable events during the unfolding of single-chain monellin (MNEI), a small plant protein.  Far- and near-UV CD measurements of guanidine-HCl (GdnHCl)-induced unfolding indicated that a MG intermediate forms initially, before the major slow unfolding reaction commences.  Steady-state FRET measurements showed that the C-terminal end of the single helix of MNEI initially moves rapidly away from the single Trp residue that is close to the N-terminal end of the helix.  The av. end-to-end distance of the protein also expanded during unfolding to the MG intermediate.  At this time, water has yet to penetrate the protein core, according to the evidence from intrinsic Trp fluorescence and 8-anilino-1-naphthalenesulfonic acid (ANS) fluorescence-monitored kinetic unfolding measurements.  These results therefore provide direct evidence for a dry MG intermediate at the initial stage of unfolding.  The results further suggest that the structural transition between the native and dry MG states could be an all-or-none transition, whereas further swelling of the globule appears to occur gradually.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvqMIwUtJwkLVg90H21EOLACvtfcHk0liBMsMY3VRuAg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1MXpslCjsrY%253D&amp;md5=46d83ec8fc80c09753972cfda8e8830f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0905744106&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0905744106%26sid%3Dliteratum%253Aachs%26aulast%3DJha%26aufirst%3DS.%2BK.%26aulast%3DUdgaonkar%26aufirst%3DJ.%2BB.%26atitle%3DDirect%2520Evidence%2520for%2520a%2520Dry%2520Molten%2520Globule%2520Intermediate%2520During%2520the%2520Unfolding%2520of%2520a%2520Small%2520Protein%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D12289%26epage%3D12294%26doi%3D10.1073%2Fpnas.0905744106" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udgaonkar, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamoorthy, G.</span></span> <span> </span><span class="NLM_article-title">Unfolding of a Small Protein Proceeds via Dry and Wet Globules and a Solvated Transition State</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">2392</span>â <span class="NLM_lpage">2402</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bpj.2013.09.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2Fj.bpj.2013.09.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=24268151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVWktr%252FM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=105&amp;publication_year=2013&amp;pages=2392-2402&amp;author=S.+S.+Sarkarauthor=J.+B.+Udgaonkarauthor=G.+Krishnamoorthy&amp;title=Unfolding+of+a+Small+Protein+Proceeds+via+Dry+and+Wet+Globules+and+a+Solvated+Transition+State&amp;doi=10.1016%2Fj.bpj.2013.09.048" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Unfolding of a Small Protein Proceeds via Dry and Wet Globules and a Solvated Transition State</span></div><div class="casAuthors">Sarkar, Saswata Sankar; Udgaonkar, Jayant B.; Krishnamoorthy, Guruswamy</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2392-2402</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Dissecting a protein unfolding process into individual steps can provide valuable information on the forces that maintain the integrity of the folded structure.  Solvation of the protein core dets. stability, but it is not clear when such solvation occurs during unfolding.  In this study, far-UV CD measurements suggest a simplistic two-state view of the unfolding of barstar, but the use of multiple other probes brings out the complexity of the unfolding reaction.  Near-UV CD measurements show that unfolding commences with the loosening of tertiary interactions in a native-like intermediate, N*.  Fluorescence resonance energy transfer measurements show that N* then expands rapidly but partially to form an early unfolding intermediate IE.  Fluorescence spectral measurements indicate that both N* and IE have retained native-like solvent accessibility of the core, suggesting that they are dry molten globules (DMGs).  Dynamic quenching measurements at the single tryptophan buried in the core suggest that the core becomes solvated only later in a late wet molten globule (WMG), IL, which precedes the unfolded form.  Fluorescence anisotropy decay measurements show that tight packing around the core tryptophan is lost when IL forms.  Of importance, the slowest step is unfolding of the wet molten globule and involves a solvated transition state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgiCdKU2ypLbVg90H21EOLACvtfcHk0lisc3MJjWZMTw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhvVWktr%252FM&amp;md5=4d8499eaaad75c354d39993947b1a7dc</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2013.09.048&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2013.09.048%26sid%3Dliteratum%253Aachs%26aulast%3DSarkar%26aufirst%3DS.%2BS.%26aulast%3DUdgaonkar%26aufirst%3DJ.%2BB.%26aulast%3DKrishnamoorthy%26aufirst%3DG.%26atitle%3DUnfolding%2520of%2520a%2520Small%2520Protein%2520Proceeds%2520via%2520Dry%2520and%2520Wet%2520Globules%2520and%2520a%2520Solvated%2520Transition%2520State%26jtitle%3DBiophys.%2520J.%26date%3D2013%26volume%3D105%26spage%3D2392%26epage%3D2402%26doi%3D10.1016%2Fj.bpj.2013.09.048" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neumaier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefhaber, T.</span></span> <span> </span><span class="NLM_article-title">Redefining the Dry Molten Globule State of Proteins</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>426</i></span>,  <span class="NLM_fpage">2520</span>â <span class="NLM_lpage">2528</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.jmb.2014.04.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2Fj.jmb.2014.04.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=24792909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvFSrtbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=426&amp;publication_year=2014&amp;pages=2520-2528&amp;author=S.+Neumaierauthor=T.+Kiefhaber&amp;title=Redefining+the+Dry+Molten+Globule+State+of+Proteins&amp;doi=10.1016%2Fj.jmb.2014.04.022" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Redefining the dry molten globule state of proteins</span></div><div class="casAuthors">Neumaier, Sabine; Kiefhaber, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2520-2528</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The dynamics and function of proteins are governed by the structural and energetic properties of the different states they adopt and the barriers sepg. them.  In earlier work, native-state triplet-triplet energy transfer (TTET) on the villin headpiece subdomain (HP35) revealed an equil. between a locked native state and an unlocked native state, which were structurally similar but had different dynamic properties.  The locked state was restricted to low amplitude motions, whereas the unlocked state showed increased conformational flexibility and underwent local unfolding reactions.  This classified the unlocked state as a dry molten globule (DMG), which was proposed to represent an expanded native state with loosened side-chain interactions and a solvent-shielded core.  To test whether the unlocked state of HP35 was actually expanded compared to the locked state, the authors performed high-pressure TTET measurements.  Increasing pressure shifted the equil. from the locked toward the unlocked state, with a small neg. reaction vol. for unlocking (ÎV0 = -1.6 cm3/mol).  Therefore, rather than being expanded, the unlocked state represents an alternatively packed, compact state, demonstrating that native proteins can exist in several compact folded states, an observation with implications for protein function.  The transition state for unlocking/locking, in contrast, had a largely increased vol. relative to the locked and unlocked state, with resp. activation vols. of 7.1 and 8.7 cm3/mol, indicating an expansion of the protein during the locking/unlocking transition.  These results demonstrated the existence of both compact, low-energy and expanded, high-energy DMGs, prompting a broader definition of this state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPZoRM_O5Gd7Vg90H21EOLACvtfcHk0lisc3MJjWZMTw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXnvFSrtbc%253D&amp;md5=131afdf155648945519507d8430a31b9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2014.04.022&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2014.04.022%26sid%3Dliteratum%253Aachs%26aulast%3DNeumaier%26aufirst%3DS.%26aulast%3DKiefhaber%26aufirst%3DT.%26atitle%3DRedefining%2520the%2520Dry%2520Molten%2520Globule%2520State%2520of%2520Proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D426%26spage%3D2520%26epage%3D2528%26doi%3D10.1016%2Fj.jmb.2014.04.022" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, A.</span></span> <span> </span><span class="NLM_article-title">Thermodynamic Fluctuations in Protein Molecules</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">2740</span>â <span class="NLM_lpage">2741</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.73.8.2740</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1073%2Fpnas.73.8.2740" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1066687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaE28Xltlyrsb0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=73&amp;publication_year=1976&amp;pages=2740-2741&amp;author=A.+Cooper&amp;title=Thermodynamic+Fluctuations+in+Protein+Molecules&amp;doi=10.1073%2Fpnas.73.8.2740" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamic fluctuations in protein molecules</span></div><div class="casAuthors">Cooper, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2740-1</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Apparently conflicting views of the phys. nature of globular proteins, and other macromols., may be reconciled by consideration of the inevitable thermodn. fluctuations inherent in microscopic systems.  Discrete protein mols., considered singly, undergo sizeable fluctuations in thermodn. properties which are manifest in their stochastic properties.  This is not incompatible with time-averaged studies of ensembles of proteins from which a more compact, rigid, and static view of these mols. may be obtained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp81u_i9pHXRbVg90H21EOLACvtfcHk0lisc3MJjWZMTw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaE28Xltlyrsb0%253D&amp;md5=77038ea27f97e4a947b761a3627862f3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.73.8.2740&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.73.8.2740%26sid%3Dliteratum%253Aachs%26aulast%3DCooper%26aufirst%3DA.%26atitle%3DThermodynamic%2520Fluctuations%2520in%2520Protein%2520Molecules%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1976%26volume%3D73%26spage%3D2740%26epage%3D2741%26doi%3D10.1073%2Fpnas.73.8.2740" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henzler-Wildman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, D.</span></span> <span> </span><span class="NLM_article-title">Dynamic Personalities of Proteins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>450</i></span>,  <span class="NLM_fpage">964</span>â <span class="NLM_lpage">972</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature06522</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2Fnature06522" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=18075575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr%252FL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=450&amp;publication_year=2007&amp;pages=964-972&amp;author=K.+Henzler-Wildmanauthor=D.+Kern&amp;title=Dynamic+Personalities+of+Proteins&amp;doi=10.1038%2Fnature06522" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic personalities of proteins</span></div><div class="casAuthors">Henzler-Wildman, Katherine; Kern, Dorothee</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7172</span>),
    <span class="NLM_cas:pages">964-972</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Because proteins are central to cellular function, researchers have sought to uncover the secrets of how these complex macromols. execute such a fascinating variety of functions.  Although static structures are known for many proteins, the functions of proteins are governed ultimately by their dynamic character (or 'personality').  The dream is to 'watch' proteins in action in real time at at. resoln.  This requires the addn. of a fourth dimension, time, to structural biol. so that the positions in space and time of all atoms in a protein can be described in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4dGYWDS9O37Vg90H21EOLACvtfcHk0lieTjCQEltHxA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr%252FL&amp;md5=27ca36106100f487683d714de5335dc2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnature06522&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06522%26sid%3Dliteratum%253Aachs%26aulast%3DHenzler-Wildman%26aufirst%3DK.%26aulast%3DKern%26aufirst%3DD.%26atitle%3DDynamic%2520Personalities%2520of%2520Proteins%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D964%26epage%3D972%26doi%3D10.1038%2Fnature06522" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frederick, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentine, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Conformational Entropy in Molecular Recognition by Proteins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>448</i></span>,  <span class="NLM_fpage">325</span>â <span class="NLM_lpage">329</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature05959</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2Fnature05959" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=17637663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnvVeqsbk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=448&amp;publication_year=2007&amp;pages=325-329&amp;author=K.+K.+Frederickauthor=M.+S.+Marlowauthor=K.+G.+Valentineauthor=A.+J.+Wand&amp;title=Conformational+Entropy+in+Molecular+Recognition+by+Proteins&amp;doi=10.1038%2Fnature05959" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational entropy in molecular recognition by proteins</span></div><div class="casAuthors">Frederick, Kendra King; Marlow, Michael S.; Valentine, Kathleen G.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">448</span>
        (<span class="NLM_cas:issue">7151</span>),
    <span class="NLM_cas:pages">325-329</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mol. recognition by proteins is fundamental to almost every biol. process, particularly the protein assocns. underlying cellular signal transduction.  Understanding the basis for protein-protein interactions requires the full characterization of the thermodn. of their assocn.  Historically, it has been virtually impossible to exptl. est. changes in protein conformational entropy, a potentially important component of the free energy of protein assocn.  However, NMR spectroscopy has emerged as a powerful tool for characterizing the dynamics of proteins.  Here, the authors employed changes in conformational dynamics as a proxy for corresponding changes in conformational entropy.  It was found that the change in internal dynamics of the protein, calmodulin, varied significantly on binding a variety of target domains.  Surprisingly, the apparent change in the corresponding conformational entropy was linearly related to the change in the overall binding entropy.  This indicated that changes in protein conformational entropy can contribute significantly to the free energy of protein-ligand assocn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohP3eEb0Q6srVg90H21EOLACvtfcHk0lieTjCQEltHxA"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXnvVeqsbk%253D&amp;md5=99cb195dbee8c4e67e348d353451dc1b</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature05959&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05959%26sid%3Dliteratum%253Aachs%26aulast%3DFrederick%26aufirst%3DK.%2BK.%26aulast%3DMarlow%26aufirst%3DM.%2BS.%26aulast%3DValentine%26aufirst%3DK.%2BG.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DConformational%2520Entropy%2520in%2520Molecular%2520Recognition%2520by%2520Proteins%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D325%26epage%3D329%26doi%3D10.1038%2Fnature05959" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dogan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederick, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentine, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">The Role of Conformational Entropy in Molecular Recognition by Calmodulin</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">352</span>â <span class="NLM_lpage">358</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nchembio.347</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2Fnchembio.347" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=20383153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFakt78%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2010&amp;pages=352-358&amp;author=M.+S.+Marlowauthor=J.+Doganauthor=K.+K.+Frederickauthor=K.+G.+Valentineauthor=A.+J.+Wand&amp;title=The+Role+of+Conformational+Entropy+in+Molecular+Recognition+by+Calmodulin&amp;doi=10.1038%2Fnchembio.347" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The role of conformational entropy in molecular recognition by calmodulin</span></div><div class="casAuthors">Marlow, Michael S.; Dogan, Jakob; Frederick, Kendra K.; Valentine, Kathleen G.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">352-358</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The phys. basis for high-affinity interactions involving proteins is complex and potentially involves a range of energetic contributions.  Among these are changes in protein conformational entropy, which cannot yet be reliably computed from mol. structures.  We have recently used changes in conformational dynamics as a proxy for changes in conformational entropy of calmodulin upon assocn. with domains from regulated proteins.  The apparent change in conformational entropy was linearly related to the overall binding entropy.  This view warrants a more quant. foundation.  Here we calibrate an 'entropy meter' using an exptl. dynamical proxy based on NMR relaxation and show that changes in the conformational entropy of calmodulin are a significant component of the energetics of binding.  Furthermore, the distribution of motion at the interface between the target domain and calmodulin is surprisingly noncomplementary.  These observations promote modification of our understanding of the energetics of protein-ligand interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmK5VBOAHwT7Vg90H21EOLACvtfcHk0lh5Uyekdc80IQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3cXksFakt78%253D&amp;md5=e9ee34fffa91f612441a77a7385015ff</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.347&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.347%26sid%3Dliteratum%253Aachs%26aulast%3DMarlow%26aufirst%3DM.%2BS.%26aulast%3DDogan%26aufirst%3DJ.%26aulast%3DFrederick%26aufirst%3DK.%2BK.%26aulast%3DValentine%26aufirst%3DK.%2BG.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DThe%2520Role%2520of%2520Conformational%2520Entropy%2520in%2520Molecular%2520Recognition%2520by%2520Calmodulin%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D352%26epage%3D358%26doi%3D10.1038%2Fnchembio.347" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eisenmesser, E. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosco, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, D.</span></span> <span> </span><span class="NLM_article-title">Enzyme Dynamics During Catalysis</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">1520</span>â <span class="NLM_lpage">1523</span>, <span class="refDoi">&nbsp;DOI: 10.1126/science.1066176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1126%2Fscience.1066176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=11859194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD38XhsFyqtbw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=295&amp;publication_year=2002&amp;pages=1520-1523&amp;author=E.+Z.+Eisenmesserauthor=D.+A.+Boscoauthor=M.+Akkeauthor=D.+Kern&amp;title=Enzyme+Dynamics+During+Catalysis&amp;doi=10.1126%2Fscience.1066176" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme dynamics during catalysis</span></div><div class="casAuthors">Eisenmesser, Elan Zohar; Bosco, Daryl A.; Akke, Mikael; Kern, Dorothee</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">295</span>
        (<span class="NLM_cas:issue">5559</span>),
    <span class="NLM_cas:pages">1520-1523</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Internal protein dynamics are intimately connected to enzymic catalysis.  However, enzyme motions linked to substrate turnover remain largely unknown.  Here, the authors studied the dynamics of an enzyme during catalysis at at. resoln. using NMR relaxation methods.  During catalytic action of cyclophilin A/peptidylprolyl cis-trans isomerase, the authors detected conformational fluctuations of the active site that occurred on a time scale of hundreds of microseconds.  The rates of conformational dynamics of the enzyme strongly correlated with the microscopic rates of substrate turnover.  The present results, together with available structural data, allowed a prediction of the reaction trajectory.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaYeoF3RbSabVg90H21EOLACvtfcHk0lh5Uyekdc80IQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD38XhsFyqtbw%253D&amp;md5=834528acfd39865d81b5341e43dc1ab1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1126%2Fscience.1066176&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1066176%26sid%3Dliteratum%253Aachs%26aulast%3DEisenmesser%26aufirst%3DE.%2BZ.%26aulast%3DBosco%26aufirst%3DD.%2BA.%26aulast%3DAkke%26aufirst%3DM.%26aulast%3DKern%26aufirst%3DD.%26atitle%3DEnzyme%2520Dynamics%2520During%2520Catalysis%26jtitle%3DScience%26date%3D2002%26volume%3D295%26spage%3D1520%26epage%3D1523%26doi%3D10.1126%2Fscience.1066176" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eisenmesser, E. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labeikovsky, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korzhnev, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf-Watz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosco, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skalicky, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, D.</span></span> <span> </span><span class="NLM_article-title">Intrinsic Dynamics of an Enzyme Underlies Catalysis</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>438</i></span>,  <span class="NLM_fpage">117</span>â <span class="NLM_lpage">121</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nature04105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2Fnature04105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=16267559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFOjurzK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=438&amp;publication_year=2005&amp;pages=117-121&amp;author=E.+Z.+Eisenmesserauthor=O.+Milletauthor=W.+Labeikovskyauthor=D.+M.+Korzhnevauthor=M.+Wolf-Watzauthor=D.+A.+Boscoauthor=J.+J.+Skalickyauthor=L.+E.+Kayauthor=D.+Kern&amp;title=Intrinsic+Dynamics+of+an+Enzyme+Underlies+Catalysis&amp;doi=10.1038%2Fnature04105" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Intrinsic dynamics of an enzyme underlies catalysis</span></div><div class="casAuthors">Eisenmesser, Elan Z.; Millet, Oscar; Labeikovsky, Wladimir; Korzhnev, Dmitry M.; Wolf-Watz, Magnus; Bosco, Daryl A.; Skalicky, Jack J.; Kay, Lewis E.; Kern, Dorothee</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">438</span>
        (<span class="NLM_cas:issue">7064</span>),
    <span class="NLM_cas:pages">117-121</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A unique feature of chem. catalysis mediated by enzymes is that the catalytically reactive atoms are embedded within a folded protein.  Although the current understanding of enzyme function has been focused on the chem. reactions and static three-dimensional structures, the dynamic nature of proteins has been proposed to have a function in catalysis.  The concept of conformational substates has been described; however, the challenge is to unravel the intimate linkage between protein flexibility and enzymic function.  Here, the authors show that the intrinsic plasticity of the protein is a key characteristic of catalysis.  The dynamics of the peptidylprolyl cis-trans isomerase, cyclophilin A (CypA), in its substrate-free state and during catalysis were characterized with NMR relaxation expts.  The characteristic enzyme motions detected during catalysis were already present in the free enzyme with frequencies corresponding to the catalytic turnover rates.  This correlation suggests that the protein motions necessary for catalysis are an intrinsic property of the enzyme and may even limit the overall turnover rate.  Motion was localized not only to the active site but also to a wider dynamic network.  Whereas coupled networks in proteins have been proposed previously, the authors exptl. measured the collective nature of motions with the use of mutant forms of CypA.  The authors propose that the pre-existence of collective dynamics in enzymes before catalysis is a common feature of biocatalysts and that proteins have evolved under synergistic pressure between structure and dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ2tDtQkHDfrVg90H21EOLACvtfcHk0lh5Uyekdc80IQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXhtFOjurzK&amp;md5=b6796cf6768ad0de0a04f419f06349e8</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnature04105&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04105%26sid%3Dliteratum%253Aachs%26aulast%3DEisenmesser%26aufirst%3DE.%2BZ.%26aulast%3DMillet%26aufirst%3DO.%26aulast%3DLabeikovsky%26aufirst%3DW.%26aulast%3DKorzhnev%26aufirst%3DD.%2BM.%26aulast%3DWolf-Watz%26aufirst%3DM.%26aulast%3DBosco%26aufirst%3DD.%2BA.%26aulast%3DSkalicky%26aufirst%3DJ.%2BJ.%26aulast%3DKay%26aufirst%3DL.%2BE.%26aulast%3DKern%26aufirst%3DD.%26atitle%3DIntrinsic%2520Dynamics%2520of%2520an%2520Enzyme%2520Underlies%2520Catalysis%26jtitle%3DNature%26date%3D2005%26volume%3D438%26spage%3D117%26epage%3D121%26doi%3D10.1038%2Fnature04105" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ha, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, S. N.</span></span> <span> </span><span class="NLM_article-title">Changes in Side Chain Packing During Apomyoglobin Folding Characterized by Pulsed Thiol-Disulfide Exchange</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">730</span>â <span class="NLM_lpage">737</span>, <span class="refDoi">&nbsp;DOI: 10.1038/1436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2F1436" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=1998&amp;pages=730-737&amp;author=J.-H.+Haauthor=S.+N.+Loh&amp;title=Changes+in+Side+Chain+Packing+During+Apomyoglobin+Folding+Characterized+by+Pulsed+Thiol-Disulfide+Exchange&amp;doi=10.1038%2F1436" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2F1436&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F1436%26sid%3Dliteratum%253Aachs%26aulast%3DHa%26aufirst%3DJ.-H.%26aulast%3DLoh%26aufirst%3DS.%2BN.%26atitle%3DChanges%2520in%2520Side%2520Chain%2520Packing%2520During%2520Apomyoglobin%2520Folding%2520Characterized%2520by%2520Pulsed%2520Thiol-Disulfide%2520Exchange%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1998%26volume%3D5%26spage%3D730%26epage%3D737%26doi%3D10.1038%2F1436" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udgaonkar, J. B.</span></span> <span> </span><span class="NLM_article-title">Exploring the Cooperativity of the Fast Folding Reaction of a Small Protein Using Pulsed Thiol Labeling and Mass Spectrometry</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">37479</span>â <span class="NLM_lpage">37491</span>, <span class="refDoi">&nbsp;DOI: 10.1074/jbc.m706714200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1074%2Fjbc.m706714200" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=282&amp;publication_year=2007&amp;pages=37479-37491&amp;author=S.+K.+Jhaauthor=J.+B.+Udgaonkar&amp;title=Exploring+the+Cooperativity+of+the+Fast+Folding+Reaction+of+a+Small+Protein+Using+Pulsed+Thiol+Labeling+and+Mass+Spectrometry&amp;doi=10.1074%2Fjbc.m706714200" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m706714200&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m706714200%26sid%3Dliteratum%253Aachs%26aulast%3DJha%26aufirst%3DS.%2BK.%26aulast%3DUdgaonkar%26aufirst%3DJ.%2BB.%26atitle%3DExploring%2520the%2520Cooperativity%2520of%2520the%2520Fast%2520Folding%2520Reaction%2520of%2520a%2520Small%2520Protein%2520Using%2520Pulsed%2520Thiol%2520Labeling%2520and%2520Mass%2520Spectrometry%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D37479%26epage%3D37491%26doi%3D10.1074%2Fjbc.m706714200" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dasgupta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udgaonkar, J. B.</span></span> <span> </span><span class="NLM_article-title">Identification of Multiple Folding Pathways of Monellin Using Pulsed Thiol Labeling and Mass Spectrometry</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">3062</span>â <span class="NLM_lpage">3074</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi1006332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi1006332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjs1ChtLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=50&amp;publication_year=2011&amp;pages=3062-3074&amp;author=S.+K.+Jhaauthor=A.+Dasguptaauthor=P.+Malhotraauthor=J.+B.+Udgaonkar&amp;title=Identification+of+Multiple+Folding+Pathways+of+Monellin+Using+Pulsed+Thiol+Labeling+and+Mass+Spectrometry&amp;doi=10.1021%2Fbi1006332" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Multiple Folding Pathways of Monellin Using Pulsed Thiol Labeling and Mass Spectrometry</span></div><div class="casAuthors">Jha, Santosh Kumar; Dasgupta, Amrita; Malhotra, Pooja; Udgaonkar, Jayant B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3062-3074</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein folding reactions often display multiexponential kinetics of changes in intrinsic optical signals, as a manifestation of heterogeneity, either on one folding pathway or on multiple folding pathways.  Delineating the origin of this heterogeneity is difficult because different coexisting structural forms of a protein cannot be easily distinguished by optical probes.  The complex folding reaction of single-chain monellin has been studied using a pulsed thiol labeling (SX) methodol. in conjunction with mass spectrometry, which measures the kinetics of burial of a cysteine side chain thiol during folding.  Because it can directly distinguish between unfolded and folded mols. and can measure the disappearance of the former during folding, the pulsed SX methodol. is an ideal method for studying whether multiple pathways are operative during folding.  The kinetics of burial of the C42 thiol of monellin was obsd. to follow biexponential kinetics.  To det. whether this was because the fast phase leads to the partial protection of the thiol group in all the mols. or to complete protection in only a fraction of the mols., the duration and intensity of the labeling pulse were varied.  The observation that the extent of labeling did not vary with the duration of the pulse cannot be explained by a simple sequential folding mechanism.  Two parallel folding pathways are operative, with one giving thiol-protective structure more rapidly than the other.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ8StXDphcZbVg90H21EOLACvtfcHk0lirAm7itmM33g"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXjs1ChtLw%253D&amp;md5=1e67ba4e40b616b1fd6a3b20edabd84d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fbi1006332&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi1006332%26sid%3Dliteratum%253Aachs%26aulast%3DJha%26aufirst%3DS.%2BK.%26aulast%3DDasgupta%26aufirst%3DA.%26aulast%3DMalhotra%26aufirst%3DP.%26aulast%3DUdgaonkar%26aufirst%3DJ.%2BB.%26atitle%3DIdentification%2520of%2520Multiple%2520Folding%2520Pathways%2520of%2520Monellin%2520Using%2520Pulsed%2520Thiol%2520Labeling%2520and%2520Mass%2520Spectrometry%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D3062%26epage%3D3074%26doi%3D10.1021%2Fbi1006332" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbury, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marqusee, S.</span></span> <span> </span><span class="NLM_article-title">Structural and Kinetic Mapping of Side-Chain Exposure onto the Protein Energy Landscape</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">10532</span>â <span class="NLM_lpage">10537</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1103629108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1073%2Fpnas.1103629108" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=108&amp;publication_year=2011&amp;pages=10532-10537&amp;author=R.+Bernsteinauthor=K.+L.+Schmidtauthor=P.+B.+Harburyauthor=S.+Marqusee&amp;title=Structural+and+Kinetic+Mapping+of+Side-Chain+Exposure+onto+the+Protein+Energy+Landscape&amp;doi=10.1073%2Fpnas.1103629108" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1103629108&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1103629108%26sid%3Dliteratum%253Aachs%26aulast%3DBernstein%26aufirst%3DR.%26aulast%3DSchmidt%26aufirst%3DK.%2BL.%26aulast%3DHarbury%26aufirst%3DP.%2BB.%26aulast%3DMarqusee%26aufirst%3DS.%26atitle%3DStructural%2520and%2520Kinetic%2520Mapping%2520of%2520Side-Chain%2520Exposure%2520onto%2520the%2520Protein%2520Energy%2520Landscape%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D10532%26epage%3D10537%26doi%3D10.1073%2Fpnas.1103629108" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demchenko, A. P.</span></span> <span> </span><span class="NLM_article-title">The Red-Edge Effects: 30 Years of Exploration</span>. <i>Luminescence</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">19</span>â <span class="NLM_lpage">42</span>, <span class="refDoi">&nbsp;DOI: 10.1002/bio.671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1002%2Fbio.671" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=11816059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD38XhvFGhu7s%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=2002&amp;pages=19-42&amp;author=A.+P.+Demchenko&amp;title=The+Red-Edge+Effects%3A+30+Years+of+Exploration&amp;doi=10.1002%2Fbio.671" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The red-edge effects: 30 years of exploration</span></div><div class="casAuthors">Demchenko, Alexander P.</div><div class="citationInfo"><span class="NLM_cas:title">Luminescence</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-42</span>CODEN:
                <span class="NLM_cas:coden">LUMIFC</span>;
        ISSN:<span class="NLM_cas:issn">1522-7235</span>.
    
            (<span class="NLM_cas:orgname">John Wiley &amp; Sons Ltd.</span>)
        </div><div class="casAbstract">A review.  In 1970, 3 labs. independently made a discovery that, for arom. fluorophores embedded into different rigid and highly viscous media, the spectroscopic properties do not conform to classical rules.  The fluorescence spectra can depend on excitation wavelength, and the excited-state energy transfer, if present, fails at the red excitation edge.  These red-edge effects were related to the existence of excited-state distribution of fluorophores on their interaction energy with the environment and the slow rate of dielec. relaxation of this environment.  In these conditions the site-selection can be provided by variation of the energy of illuminating light quanta, and the behavior of selected species can be followed as a function of time and other variables.  These observations found extensive application in different areas of research: colloid and polymer science, mol. biophysics, photochem. and photobiol.  In particular, they led to the development of very productive methods of studying the dynamics of dielec. relaxations in protein and membranes, using the tryptophan emission and the emission of a variety of probes.  These studies were extended to the time domain with the observation of new site-selective effects in emission intensity and anisotropy decays.  They stimulated the emergence and development of cryogenic energy-selective and single-mol. techniques that became valuable tools in their own right in chem. and biophysics research.  Site-selection effects were discovered for electron-transfer and proton-transfer reactions if they depended on the dynamics of the environment.  This review is focused on the progress in the field of red-edge effects, their applications and prospects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBEYGc9Y2nOLVg90H21EOLACvtfcHk0lh1GAdfmCqpwQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD38XhvFGhu7s%253D&amp;md5=ec02d96136bb65dc02005dd04ccfc4aa</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fbio.671&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbio.671%26sid%3Dliteratum%253Aachs%26aulast%3DDemchenko%26aufirst%3DA.%2BP.%26atitle%3DThe%2520Red-Edge%2520Effects%253A%252030%2520Years%2520of%2520Exploration%26jtitle%3DLuminescence%26date%3D2002%26volume%3D17%26spage%3D19%26epage%3D42%26doi%3D10.1002%2Fbio.671" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, S.</span></span> <span> </span><span class="NLM_article-title">Dynamic Insight into Protein Structure Utilizing Red Edge Excitation Shift</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">12</span>â <span class="NLM_lpage">19</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ar400006z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar400006z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlChtLnK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=2014&amp;pages=12-19&amp;author=A.+Chattopadhyayauthor=S.+Haldar&amp;title=Dynamic+Insight+into+Protein+Structure+Utilizing+Red+Edge+Excitation+Shift&amp;doi=10.1021%2Far400006z" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic Insight into Protein Structure Utilizing Red Edge Excitation Shift</span></div><div class="casAuthors">Chattopadhyay, Amitabha; Haldar, Sourav</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-19</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Proteins are considered the workhorses in the cellular machinery.  They are often organized in a highly ordered conformation in the crowded cellular environment.  These conformations display characteristic dynamics over a range of time scales.  An emerging consensus is that protein function is critically dependent on its dynamics.  The subtle interplay between structure and dynamics is a hallmark of protein organization and is essential for its function.  Depending on the environmental context, proteins can adopt a range of conformations such as native, molten globule, unfolded (denatured), and misfolded states.  Although protein crystallog. is a well established technique, it is not always possible to characterize various protein conformations by X-ray crystallog. due to transient nature of these states.  Even in cases where structural characterization is possible, the information obtained lacks dynamic component, which is needed to understand protein function.  In this overall scenario, approaches that reveal information on protein dynamics are much appreciated.  Dynamics of confined water has interesting implications in protein folding.  Interfacial hydration combines the motion of water mols. with the slow moving protein mols.  The red edge excitation shift (REES) approach becomes relevant in this context.  REES is defined as the shift in the wavelength of max. fluorescence emission toward higher wavelengths, caused by a shift in the excitation wavelength toward the red edge of absorption spectrum.  REES arises due to slow rates (relative to fluorescence lifetime) of solvent relaxation (reorientation) around an excited state fluorophore in organized assemblies such as proteins.  Consequently, REES depends on the environment-induced motional restriction imposed on the solvent mols. in the immediate vicinity of the fluorophore.  In the case of a protein, the confined water in the protein creates a dipolar field that acts as the solvent for a fluorophore in the protein.  In this Account, we focus on REES to monitor organization and dynamics of sol. and membrane proteins utilizing intrinsic protein fluorescence.  We discuss here the application of REES in various conformations of proteins.  While application of REES to proteins in native conformation has been in use for a long time, our work highlights the potential of this approach in case of molten globule and denatured conformations.  For example, we have demonstrated the presence of residual structure, that could not be detected using other methods, by REES of denatured spectrin.  Given the functional relevance of such residual structures, these results are very far reaching.  We discuss here the application of REES to molten globule conformation and to the green fluorescent protein (GFP).  The case of GFP is particularly interesting since the dipolar field in this case is provided by the protein matrix itself and not confined water.  We envision that future applications of REES in proteins will involve generating a dynamic hydration map of the protein, which would allow us to explore protein function in terms of local dynamics and hydration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQSXfHF15j-bVg90H21EOLACvtfcHk0lh1GAdfmCqpwQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhtlChtLnK&amp;md5=ea51eafb84eb25da2aa7ad123662bcac</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Far400006z&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far400006z%26sid%3Dliteratum%253Aachs%26aulast%3DChattopadhyay%26aufirst%3DA.%26aulast%3DHaldar%26aufirst%3DS.%26atitle%3DDynamic%2520Insight%2520into%2520Protein%2520Structure%2520Utilizing%2520Red%2520Edge%2520Excitation%2520Shift%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2014%26volume%3D47%26spage%3D12%26epage%3D19%26doi%3D10.1021%2Far400006z" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demchenko, A. P.</span></span> <span> </span><span class="NLM_article-title">On the nanosecond mobility in proteins: edge excitation fluorescence red shift of protein-bound 2-(p-toluidinylnaphthalene)-6-sulfonate</span>. <i>Biophys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">101</span>â <span class="NLM_lpage">109</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0301-4622(82)80022-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2F0301-4622%2882%2980022-7" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=1982&amp;pages=101-109&amp;author=A.+P.+Demchenko&amp;title=On+the+nanosecond+mobility+in+proteins%3A+edge+excitation+fluorescence+red+shift+of+protein-bound+2-%28p-toluidinylnaphthalene%29-6-sulfonate&amp;doi=10.1016%2F0301-4622%2882%2980022-7" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2F0301-4622%2882%2980022-7&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0301-4622%252882%252980022-7%26sid%3Dliteratum%253Aachs%26aulast%3DDemchenko%26aufirst%3DA.%2BP.%26atitle%3DOn%2520the%2520nanosecond%2520mobility%2520in%2520proteins%253A%2520edge%2520excitation%2520fluorescence%2520red%2520shift%2520of%2520protein-bound%25202-%2528p-toluidinylnaphthalene%2529-6-sulfonate%26jtitle%3DBiophys.%2520Chem.%26date%3D1982%26volume%3D15%26spage%3D101%26epage%3D109%26doi%3D10.1016%2F0301-4622%2882%2980022-7" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lakowicz, J. R.</span></span> <i>Principles of Fluorescence Spectroscopy</i>; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Singapore</span>, <span class="NLM_year">2006</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1007%2F978-0-387-46312-4" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2006&amp;author=J.+R.+Lakowicz&amp;title=Principles+of+Fluorescence+Spectroscopy" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1007%2F978-0-387-46312-4&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-0-387-46312-4%26sid%3Dliteratum%253Aachs%26aulast%3DLakowicz%26aufirst%3DJ.%2BR.%26btitle%3DPrinciples%2520of%2520Fluorescence%2520Spectroscopy%26pub%3DSpringer%26date%3D2006" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lakowicz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating-Nakamoto, S.</span></span> <span> </span><span class="NLM_article-title">Red-Edge Excitation of Fluorescence and Dynamic Properties of Proteins and Membranes</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3013</span>â <span class="NLM_lpage">3021</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi00308a026</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00308a026" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaL2cXktVegtLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=1984&amp;pages=3013-3021&amp;author=J.+R.+Lakowiczauthor=S.+Keating-Nakamoto&amp;title=Red-Edge+Excitation+of+Fluorescence+and+Dynamic+Properties+of+Proteins+and+Membranes&amp;doi=10.1021%2Fbi00308a026" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Red-edge excitation of fluorescence and dynamic properties of proteins and membranes</span></div><div class="casAuthors">Lakowicz, Joseph R.; Keating-Nakamoto, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3013-21</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">The phenomenon of the dependence of the fluorescence emission spectra and lifetimes upon the excitation wavelength was used to investigate the dynamic properties of ref. solvents, model membranes, and the protein apomyoglobin labeled with 6-(p-toluidinyl)-2-naphthalenesulfonic acid (TNS).  The spectral shifts and lifetime data indicated that red-edge excitation results in the selective excitation of solvent-relaxed fluorophores.  By comparison of the data obtained for TNS in solvents and bound to the macromols., one may est. the relaxation rate of the environment.  This comparison indicates rapid spectral relaxation of TNS bound to lipid vesicles, and a somewhat slower relaxation around TNS bound to the heme site of apomyoglobin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjrOXKDxlcNbVg90H21EOLACvtfcHk0lh1GAdfmCqpwQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL2cXktVegtLw%253D&amp;md5=adb55d6a9ee29dcea1b7af5c061690af</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fbi00308a026&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00308a026%26sid%3Dliteratum%253Aachs%26aulast%3DLakowicz%26aufirst%3DJ.%2BR.%26aulast%3DKeating-Nakamoto%26aufirst%3DS.%26atitle%3DRed-Edge%2520Excitation%2520of%2520Fluorescence%2520and%2520Dynamic%2520Properties%2520of%2520Proteins%2520and%2520Membranes%26jtitle%3DBiochemistry%26date%3D1984%26volume%3D23%26spage%3D3013%26epage%3D3021%26doi%3D10.1021%2Fbi00308a026" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukhopadhyay, S.</span></span> <span> </span><span class="NLM_article-title">Ordered Water within the Collapsed Globules of an Amyloidogenic Intrinsically Disordered Protein</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">9191</span>â <span class="NLM_lpage">9198</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jp504076a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp504076a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFOqtL7N" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=118&amp;publication_year=2014&amp;pages=9191-9198&amp;author=S.+Aryaauthor=S.+Mukhopadhyay&amp;title=Ordered+Water+within+the+Collapsed+Globules+of+an+Amyloidogenic+Intrinsically+Disordered+Protein&amp;doi=10.1021%2Fjp504076a" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Ordered water within the collapsed globules of an amyloidogenic intrinsically disordered protein</span></div><div class="casAuthors">Arya, Shruti; Mukhopadhyay, Samrat</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9191-9198</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intrinsically disordered proteins (IDPs) confront the traditional sequence-structure-function paradigm and are assocd. with important functions and amyloid disorders.  Water mols. residing in the vicinity of the polypeptide chain play potentially important roles in directing the course of binding-induced folding and amyloid aggregation of IDPs.  Here, the authors characterized the nature of water mols. entrapped within the collapsed globules of the amyloidogenic IDP, Îº-casein.  These globules could undergo further compaction in the presence of an anionic detergent that was capable of diminishing the intrachain repulsion from the pos. charged Gln/Asn-rich amyloidogenic N-terminal domain comprising 100 residues.  Using time-resolved fluorescence spectroscopy, the authors estd. the longer component of the solvation time to be â¼1.4 ns, which was 3 orders of magnitude slower than that in bulk water and more than an order of magnitude slower than the "biol. water" present at the protein surface.  Profoundly restrained water within the collapsed IDP globules resembled the ordered water cluster found under nanoconfinement.  The authors suggest that the assocn. of these globules would result in the release of ordered water mols. into the bulk milieu causing an entropic gain that would eventually drive the formation of the key (obligatory) oligomeric intermediates on the pathway to amyloids via nucleation-dependent polymn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSf08tPiCxN7Vg90H21EOLACvtfcHk0lh1GAdfmCqpwQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXhtFOqtL7N&amp;md5=573cee2f7971645e7f35f18074e49cd2</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjp504076a&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp504076a%26sid%3Dliteratum%253Aachs%26aulast%3DArya%26aufirst%3DS.%26aulast%3DMukhopadhyay%26aufirst%3DS.%26atitle%3DOrdered%2520Water%2520within%2520the%2520Collapsed%2520Globules%2520of%2520an%2520Amyloidogenic%2520Intrinsically%2520Disordered%2520Protein%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2014%26volume%3D118%26spage%3D9191%26epage%3D9198%26doi%3D10.1021%2Fjp504076a" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span> <span> </span><span class="NLM_article-title">Red Edge Excitation Shift of a Deeply Embedded Membrane Probe: Implications in Water Penetration in the Bilayer</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">8180</span>â <span class="NLM_lpage">8185</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jp991303m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp991303m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaK1MXls1Ojt70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=103&amp;publication_year=1999&amp;pages=8180-8185&amp;author=A.+Chattopadhyayauthor=S.+Mukherjee&amp;title=Red+Edge+Excitation+Shift+of+a+Deeply+Embedded+Membrane+Probe%3A+Implications+in+Water+Penetration+in+the+Bilayer&amp;doi=10.1021%2Fjp991303m" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Red Edge Excitation Shift of a Deeply Embedded Membrane Probe: Implications in Water Penetration in the Bilayer</span></div><div class="casAuthors">Chattopadhyay, Amitabha; Mukherjee, Sushmita</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">8180-8185</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. membrane is a highly organized anisotropic mol. assembly.  While the center of the bilayer is nearly isotropic, the upper portion, only a few angstroms away toward the membrane surface, is highly ordered.  How this organization correlates with the degree of water penetration into the bilayer interior is not clear.  In general, it is believed that there is not much water in the deeper hydrocarbon regions of the bilayer.  In this study, we have utilized the phenomenon of wavelength-selective fluorescence to address this question.  We show here that when the same fluorescent group (i.e., 7-nitrobenz-2-oxa-1,3-diazol-4-yl or NBD) is localized at different depths within the bilayer (viz., near the membrane interface in case of the headgroup-labeled NBD-phosphatidylethanolamine (NBD-PE) and near the center of the bilayer in NBD-cholesterol), the degrees to which their fluorescence properties exhibit solvent-induced effects are markedly different.  For example, the headgroup-labeled NBD-PE exhibits a much stronger red edge excitation shift (REES) relative to that of NBD-cholesterol.  This indicates lesser restriction to mobility in this region as compared to the polar/hydrocarbon interface.  In the gel phase, however, REES of NBD-PE did not show any significant change while NBD-cholesterol exhibited no REES.  In addn., NBD-cholesterol exhibits a stronger dependence of fluorescence polarization on excitation wavelength in fluid membranes.  We attribute these results to the more compact arrangement of the lipid acyl chains in the gel phase which results in lesser water penetration.  Since the hydrophobic core of the lipid bilayer is made up of Me and methylene groups, the only solvent dipoles capable of any interaction with the dipole of the fluorophore giving rise to the REES effect in the fluid phase have to be water mols. that have penetrated deep into the bilayer close to the NBD moiety of NBD-cholesterol.  Our results indicate that at least in the fluid phase of the membrane, penetration of water in the deep hydrocarbon region of the bilayer does indeed occur.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbH5ci1H_ev7Vg90H21EOLACvtfcHk0ljVlFzXEn0Snw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1MXls1Ojt70%253D&amp;md5=b74bcb02117822c0e869bc69928cbde4</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjp991303m&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp991303m%26sid%3Dliteratum%253Aachs%26aulast%3DChattopadhyay%26aufirst%3DA.%26aulast%3DMukherjee%26aufirst%3DS.%26atitle%3DRed%2520Edge%2520Excitation%2520Shift%2520of%2520a%2520Deeply%2520Embedded%2520Membrane%2520Probe%253A%2520Implications%2520in%2520Water%2520Penetration%2520in%2520the%2520Bilayer%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1999%26volume%3D103%26spage%3D8180%26epage%3D8185%26doi%3D10.1021%2Fjp991303m" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhuri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, A.</span></span> <span> </span><span class="NLM_article-title">Organization and Dynamics of Membrane Probes and Proteins Utilizing the Red Edge Excitation Shift</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">5693</span>â <span class="NLM_lpage">5706</span>, <span class="refDoi">&nbsp;DOI: 10.1021/jp200255e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp200255e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjvVaqu7g%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=115&amp;publication_year=2011&amp;pages=5693-5706&amp;author=S.+Haldarauthor=A.+Chaudhuriauthor=A.+Chattopadhyay&amp;title=Organization+and+Dynamics+of+Membrane+Probes+and+Proteins+Utilizing+the+Red+Edge+Excitation+Shift&amp;doi=10.1021%2Fjp200255e" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Organization and Dynamics of Membrane Probes and Proteins Utilizing the Red Edge Excitation Shift</span></div><div class="casAuthors">Haldar, Sourav; Chaudhuri, Arunima; Chattopadhyay, Amitabha</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5693-5706</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1520-5207</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Dynamics of confined water has interesting implications in the organization and function of mol. assemblies such as membranes.  A direct consequence of this type of organization is the restriction imposed on the mobility of the constituent structural units.  Interestingly, this restriction (confinement) of mobility couples the motion of solvent (water) mols. with the slow moving mols. in the assembly.  It is in this context that the red edge excitation shift (REES) represents a sensitive approach to monitor the environment and dynamics around a fluorophore in such organized assemblies.  A shift in the wavelength of max. fluorescence emission toward higher wavelengths, caused by a shift in the excitation wavelength toward the red edge of the absorption band, is termed REES.  REES relies on slow solvent reorientation in the excited state of a fluorophore that can be used to monitor the environment and dynamics around a fluorophore in a host assembly.  In this article, the authors focus on the application of REES to monitor organization and dynamics of membrane probes and proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbFYKxPZrwSrVg90H21EOLACvtfcHk0ljVlFzXEn0Snw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXjvVaqu7g%253D&amp;md5=f785f730602c7aba768229ee25afe1aa</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjp200255e&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp200255e%26sid%3Dliteratum%253Aachs%26aulast%3DHaldar%26aufirst%3DS.%26aulast%3DChaudhuri%26aufirst%3DA.%26aulast%3DChattopadhyay%26aufirst%3DA.%26atitle%3DOrganization%2520and%2520Dynamics%2520of%2520Membrane%2520Probes%2520and%2520Proteins%2520Utilizing%2520the%2520Red%2520Edge%2520Excitation%2520Shift%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2011%26volume%3D115%26spage%3D5693%26epage%3D5706%26doi%3D10.1021%2Fjp200255e" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawat, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharyya, B.</span></span> <span> </span><span class="NLM_article-title">Tubulin Conformation and Dynamics: A Red Edge Excitation Shift Study</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">13426</span>â <span class="NLM_lpage">13433</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi961251g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi961251g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaK28XlslOlurk%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=35&amp;publication_year=1996&amp;pages=13426-13433&amp;author=S.+Guhaauthor=S.+S.+Rawatauthor=A.+Chattopadhyayauthor=B.+Bhattacharyya&amp;title=Tubulin+Conformation+and+Dynamics%3A+A+Red+Edge+Excitation+Shift+Study&amp;doi=10.1021%2Fbi961251g" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Tubulin conformation and dynamics: A red edge excitation shift study</span></div><div class="casAuthors">Guha, Suranjana; Rawat, Satinder S.; Chattopadhyay, Amitabha; Bhattacharyya, Bhabatarak</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">13426-13433</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The fluorescence emission max. of a polar fluorophore in viscous medium often shows a dependence on excitation wavelength, a phenomenon which is named red edge excitation shift (REES).  Here, the authors found that the fluorescence spectra of the tubulin Trp residues exhibited a REES of â¼7 nm.  Also, their steady state fluorescence polarization and mean lifetimes showed a dependence on both excitation and emission wavelengths.  These results indicated that the av. Trp environment in tubulin is motionally restricted.  Although the Trp residue(s) responsible for the obsd. REES effect could not be localized, it could be concluded from energy transfer expts. with the tubulin-colchicine complex that the Trp residue(s) participating in energy transfer with bound colchicine probably did not contribute to the REES.  A REES of 7 nm was also obsd. in the case of colchicine complexed with tubulin.  However, such a REES was not seen in similar studies with the B-ring analogs of colchicine, viz., 2-methoxy-5-(2',3',4'-trimethoxyphenyl)tropone (called AC because it lacks the B ring of colchicine) and deacetamidocolchicine (which lacks the acetamido substituent at the C-7 position of the B ring).  There may be 2 possible reasons to explain these data.  (1) Structural differences between colchicine and its analogs may give rise to differences in their excited state dipole moments which will directly affect the extent of REES, and (2) The B-ring substituent, hanging outside the colchicine binding site on the Î² subunit of the tubulin dimer, probably makes contact with the Î± subunit of tubulin and imparts a rigidity to that region of the protein, which facilitates the REES.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRot2M2IKBu7Vg90H21EOLACvtfcHk0ljVlFzXEn0Snw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK28XlslOlurk%253D&amp;md5=7838ad028038740784dade85edf14b46</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fbi961251g&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi961251g%26sid%3Dliteratum%253Aachs%26aulast%3DGuha%26aufirst%3DS.%26aulast%3DRawat%26aufirst%3DS.%2BS.%26aulast%3DChattopadhyay%26aufirst%3DA.%26aulast%3DBhattacharyya%26aufirst%3DB.%26atitle%3DTubulin%2520Conformation%2520and%2520Dynamics%253A%2520A%2520Red%2520Edge%2520Excitation%2520Shift%2520Study%26jtitle%3DBiochemistry%26date%3D1996%26volume%3D35%26spage%3D13426%26epage%3D13433%26doi%3D10.1021%2Fbi961251g" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Catici, D. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amos, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Elsen, J. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pudney, C. R.</span></span> <span> </span><span class="NLM_article-title">The Red Edge Excitation Shift Phenomenon can be Used to Unmask Protein Structural Ensembles: Implications for Nemo-Ubiquitin Interactions</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">2272</span>â <span class="NLM_lpage">2284</span>, <span class="refDoi">&nbsp;DOI: 10.1111/febs.13724</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1111%2Ffebs.13724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=27028374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsVyjt7o%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=283&amp;publication_year=2016&amp;pages=2272-2284&amp;author=D.+A.+M.+Caticiauthor=H.+E.+Amosauthor=Y.+Yangauthor=J.+M.+H.+van+den+Elsenauthor=C.+R.+Pudney&amp;title=The+Red+Edge+Excitation+Shift+Phenomenon+can+be+Used+to+Unmask+Protein+Structural+Ensembles%3A+Implications+for+Nemo-Ubiquitin+Interactions&amp;doi=10.1111%2Ffebs.13724" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The red edge excitation shift phenomenon can be used to unmask protein structural ensembles: implications for NEMO-ubiquitin interactions</span></div><div class="casAuthors">Catici, Dragana A. M.; Amos, Hope E.; Yang, Yi; van den Elsen, Jean M. H.; Pudney, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2272-2284</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">To understand complex mol. interactions, it is necessary to account for mol. flexibility and the available equil. of conformational states.  Only a small no. of exptl. approaches can access such information.  Potentially, steady-state red edge excitation shift (REES) spectroscopy can act as a qual. metric of changes to the protein free energy landscape (FEL) and the equil. of conformational states.  First, the authors validated this hypothesis using a single Trp-contg. protein, NF-ÎºB essential modulator (NEMO).  The authors provided detailed evidence from chem. denaturation studies, macromol. crowding studies, and the 1st report of the pressure dependence of the REES effect.  A combination of these data demonstrated that the REES effect can report on the 'ruggedness' of the FEL and the authors present a phenomenol. model, based on realistic phys. interpretations, for fitting steady-state REES data to allow quantification of this aspect of the REES effect.  The authors tested the conceptual framework they developed by correlating findings from NEMO ligand-binding studies with the REES data in a range of NEMO-ligand binary complexes.  These findings shed light on the nature of the interaction between NEMO and poly-ubiquitin, suggesting that NEMO is differentially regulated by polyubiquitin chain length and that this regulation occurs via a modulation of the available equil. of conformational states, rather than gross structural change.  This study therefore demonstrates the potential of REES as a powerful tool for tackling contemporary issues in structural biol. and biophysics and elucidates novel information on the structure-function relation of NEMO and key interaction partners.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE1xuMXcBAUrVg90H21EOLACvtfcHk0ljEwQYC0nyi2A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XpsVyjt7o%253D&amp;md5=a657e1411a6e6e7b2e8b7232aa5f7461</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1111%2Ffebs.13724&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.13724%26sid%3Dliteratum%253Aachs%26aulast%3DCatici%26aufirst%3DD.%2BA.%2BM.%26aulast%3DAmos%26aufirst%3DH.%2BE.%26aulast%3DYang%26aufirst%3DY.%26aulast%3Dvan%2Bden%2BElsen%26aufirst%3DJ.%2BM.%2BH.%26aulast%3DPudney%26aufirst%3DC.%2BR.%26atitle%3DThe%2520Red%2520Edge%2520Excitation%2520Shift%2520Phenomenon%2520can%2520be%2520Used%2520to%2520Unmask%2520Protein%2520Structural%2520Ensembles%253A%2520Implications%2520for%2520Nemo-Ubiquitin%2520Interactions%26jtitle%3DFEBS%2520J.%26date%3D2016%26volume%3D283%26spage%3D2272%26epage%3D2284%26doi%3D10.1111%2Ffebs.13724" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawat, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelkar, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabarti, A.</span></span> <span> </span><span class="NLM_article-title">Organization and dynamics of tryptophan residues in erythroid spectrin: novel structural features of denatured spectrin revealed by the wavelength-selective fluorescence approach</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2389</span>â <span class="NLM_lpage">2403</span>, <span class="refDoi">&nbsp;DOI: 10.1110/ps.03302003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1110%2Fps.03302003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=14573853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD3sXosleltbo%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2003&amp;pages=2389-2403&amp;author=A.+Chattopadhyayauthor=S.+S.+Rawatauthor=D.+A.+Kelkarauthor=S.+Rayauthor=A.+Chakrabarti&amp;title=Organization+and+dynamics+of+tryptophan+residues+in+erythroid+spectrin%3A+novel+structural+features+of+denatured+spectrin+revealed+by+the+wavelength-selective+fluorescence+approach&amp;doi=10.1110%2Fps.03302003" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Organization and dynamics of tryptophan residues in erythroid spectrin: Novel structural features of denatured spectrin revealed by the wavelength-selective fluorescence approach</span></div><div class="casAuthors">Chattopadhyay, Amitabha; Rawat, Satinder S.; Kelkar, Devaki A.; Ray, Sibnath; Chakrabarti, Abhijit</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2389-2403</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">We have investigated the organization and dynamics of the functionally important tryptophan residues of erythroid spectrin in native and denatured conditions utilizing the wavelength-selective fluorescence approach.  We obsd. a red edge excitation shift (REES) of 4 nm for the tryptophans in the case of spectrin in its native state.  This indicates that tryptophans in spectrin are localized in a microenvironment of restricted mobility, and that the regions surrounding the spectrin tryptophans offer considerable restriction to the reorientational motion of the water dipoles around the excited state tryptophans.  Interestingly, spectrin exhibits a REES of 3 nm even when denatured in 8 M urea.  This represents the first report of a denatured protein displaying REES.  Observation of REES in the denatured state implies that some of the structural and dynamic features of this microenvironment around the spectrin tryptophans are retained even when the protein is denatured.  Fluorescence quenching data of denatured spectrin support this conclusion.  In addn., we have deduced the organization and dynamics of the hydrophobic binding site of the polarity-sensitive fluorescent probe PRODAN that binds erythroid spectrin with high affinity.  When bound to spectrin, PRODAN exhibits a REES of 9 nm.  Because PRODAN binds to a hydrophobic site in spectrin, such a result would directly imply that this region of spectrin offers considerable restriction to the reorientational motion of the solvent dipoles around the excited state fluorophore.  The results of our study could provide vital insight into the role of tryptophans in the stability and folding of spectrin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ0fl0vm2HrLVg90H21EOLACvtfcHk0ljEwQYC0nyi2A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3sXosleltbo%253D&amp;md5=f1626f0c34f2691556ae835b11944867</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1110%2Fps.03302003&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.03302003%26sid%3Dliteratum%253Aachs%26aulast%3DChattopadhyay%26aufirst%3DA.%26aulast%3DRawat%26aufirst%3DS.%2BS.%26aulast%3DKelkar%26aufirst%3DD.%2BA.%26aulast%3DRay%26aufirst%3DS.%26aulast%3DChakrabarti%26aufirst%3DA.%26atitle%3DOrganization%2520and%2520dynamics%2520of%2520tryptophan%2520residues%2520in%2520erythroid%2520spectrin%253A%2520novel%2520structural%2520features%2520of%2520denatured%2520spectrin%2520revealed%2520by%2520the%2520wavelength-selective%2520fluorescence%2520approach%26jtitle%3DProtein%2520Sci.%26date%3D2003%26volume%3D12%26spage%3D2389%26epage%3D2403%26doi%3D10.1110%2Fps.03302003" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhuri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chattopadhyay, A.</span></span> <span> </span><span class="NLM_article-title">Organization and dynamics of tryptophans in the molten globule state of bovine Î±-lactalbumin utilizing wavelength-selective fluorescence approach: Comparisons with native and denatured states</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>394</i></span>,  <span class="NLM_fpage">1082</span>â <span class="NLM_lpage">1086</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.bbrc.2010.03.130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2Fj.bbrc.2010.03.130" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=394&amp;publication_year=2010&amp;pages=1082-1086&amp;author=A.+Chaudhuriauthor=S.+Haldarauthor=A.+Chattopadhyay&amp;title=Organization+and+dynamics+of+tryptophans+in+the+molten+globule+state+of+bovine+%CE%B1-lactalbumin+utilizing+wavelength-selective+fluorescence+approach%3A+Comparisons+with+native+and+denatured+states&amp;doi=10.1016%2Fj.bbrc.2010.03.130" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2010.03.130&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2010.03.130%26sid%3Dliteratum%253Aachs%26aulast%3DChaudhuri%26aufirst%3DA.%26aulast%3DHaldar%26aufirst%3DS.%26aulast%3DChattopadhyay%26aufirst%3DA.%26atitle%3DOrganization%2520and%2520dynamics%2520of%2520tryptophans%2520in%2520the%2520molten%2520globule%2520state%2520of%2520bovine%2520%25CE%25B1-lactalbumin%2520utilizing%2520wavelength-selective%2520fluorescence%2520approach%253A%2520Comparisons%2520with%2520native%2520and%2520denatured%2520states%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2010%26volume%3D394%26spage%3D1082%26epage%3D1086%26doi%3D10.1016%2Fj.bbrc.2010.03.130" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span> <span> </span><span class="NLM_article-title">Evidence for Dry Molten Globule-Like Domains in the pH-Induced Equilibrium Folding Intermediate of a Multidomain Protein</span>. <i>J. Phys. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">173</span>â <span class="NLM_lpage">179</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.jpclett.5b02545</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jpclett.5b02545" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVygt73L" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2016&amp;pages=173-179&amp;author=N.+Acharyaauthor=P.+Mishraauthor=S.+K.+Jha&amp;title=Evidence+for+Dry+Molten+Globule-Like+Domains+in+the+pH-Induced+Equilibrium+Folding+Intermediate+of+a+Multidomain+Protein&amp;doi=10.1021%2Facs.jpclett.5b02545" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for Dry Molten Globule-Like Domains in the pH-Induced Equilibrium Folding Intermediate of a Multidomain Protein</span></div><div class="casAuthors">Acharya, Nirbhik; Mishra, Prajna; Jha, Santosh Kumar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">173-179</span>CODEN:
                <span class="NLM_cas:coden">JPCLCD</span>;
        ISSN:<span class="NLM_cas:issn">1948-7185</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The role of van der Waals (vdW) packing interactions compared to the hydrophobic effect in stabilizing the functional structure of proteins is poorly understood.  Here we show, using fluorescence resonance energy transfer, dynamic fluorescence quenching, red-edge excitation shift, and near- and far-UV CD, that the pH-induced structural perturbation of a multidomain protein leads to the formation of a state in which two out of the three domains have characteristics of dry molten globules, i.e., the domains are expanded compared to the native protein with disrupted packing interactions but have dry cores.  We quant. est. the energetic contribution of vdW interactions and show that they play an important role in the stability of the native state and cooperativity of its structural transition, in addn. to the hydrophobic effect.  Our results also indicate that during the pH-induced unfolding, side-chain unlocking and hydrophobic solvation occur in two distinct steps and not in a concerted manner, as commonly believed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-vmLRGPIMUbVg90H21EOLACvtfcHk0ljEwQYC0nyi2A"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2MXitVygt73L&amp;md5=dc2eecf40f625c12d906949ab55605cf</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jpclett.5b02545&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jpclett.5b02545%26sid%3Dliteratum%253Aachs%26aulast%3DAcharya%26aufirst%3DN.%26aulast%3DMishra%26aufirst%3DP.%26aulast%3DJha%26aufirst%3DS.%2BK.%26atitle%3DEvidence%2520for%2520Dry%2520Molten%2520Globule-Like%2520Domains%2520in%2520the%2520pH-Induced%2520Equilibrium%2520Folding%2520Intermediate%2520of%2520a%2520Multidomain%2520Protein%26jtitle%3DJ.%2520Phys.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D173%26epage%3D179%26doi%3D10.1021%2Facs.jpclett.5b02545" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beeby, P. J.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Interaction with Human Serum Albumin of the First 3,18-Disubstituted Derivative of Bilirubin</span>. <i>J. Chem. Soc. Perkin Trans. 1</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_fpage">3041</span>â <span class="NLM_lpage">3044</span>, <span class="refDoi">&nbsp;DOI: 10.1039/a803429j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1039%2Fa803429j" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1998&amp;pages=3041-3044&amp;author=L.+Painterauthor=M.+M.+Hardingauthor=P.+J.+Beeby&amp;title=Synthesis+and+Interaction+with+Human+Serum+Albumin+of+the+First+3%2C18-Disubstituted+Derivative+of+Bilirubin&amp;doi=10.1039%2Fa803429j" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1039%2Fa803429j&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fa803429j%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DL.%26aulast%3DHarding%26aufirst%3DM.%2BM.%26aulast%3DBeeby%26aufirst%3DP.%2BJ.%26atitle%3DSynthesis%2520and%2520Interaction%2520with%2520Human%2520Serum%2520Albumin%2520of%2520the%2520First%25203%252C18-Disubstituted%2520Derivative%2520of%2520Bilirubin%26jtitle%3DJ.%2520Chem.%2520Soc.%2520Perkin%2520Trans.%25201%26date%3D1998%26spage%3D3041%26epage%3D3044%26doi%3D10.1039%2Fa803429j" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pace, C. N.</span></span> <span> </span><span class="NLM_article-title">Determination and analysis of urea and guanidine hydrochloride denaturation curves</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">266</span>â <span class="NLM_lpage">280</span>, <span class="refDoi">&nbsp;DOI: 10.1016/0076-6879(86)31045-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2F0076-6879%2886%2931045-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=3773761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaL2sXmtlOitw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=131&amp;publication_year=1986&amp;pages=266-280&amp;author=C.+N.+Pace&amp;title=Determination+and+analysis+of+urea+and+guanidine+hydrochloride+denaturation+curves&amp;doi=10.1016%2F0076-6879%2886%2931045-0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Determination and analysis of urea and guanidine hydrochloride denaturation curves</span></div><div class="casAuthors">Pace, C. N.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">Enzyme Struct., Pt. L</span>),
    <span class="NLM_cas:pages">266-80</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">A review, with 38 refs., on practical aspects of detg. and analyzing urea- and guanidine hydrochloride-induced denaturation curves of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGps8UxgdhPl77Vg90H21EOLACvtfcHk0li4Zv3UP5UKbQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL2sXmtlOitw%253D%253D&amp;md5=efc254fd3a598035cf9573702116917f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2886%2931045-0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252886%252931045-0%26sid%3Dliteratum%253Aachs%26aulast%3DPace%26aufirst%3DC.%2BN.%26atitle%3DDetermination%2520and%2520analysis%2520of%2520urea%2520and%2520guanidine%2520hydrochloride%2520denaturation%2520curves%26jtitle%3DMethods%2520Enzymol.%26date%3D1986%26volume%3D131%26spage%3D266%26epage%3D280%26doi%3D10.1016%2F0076-6879%2886%2931045-0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Street, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courtemanche, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrick, D.</span></span> <span> </span><span class="NLM_article-title">Protein Folding and Stability Using Denaturants</span>. <i>Methods Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">295</span>â <span class="NLM_lpage">325</span>, <span class="refDoi">&nbsp;DOI: 10.1016/s0091-679x(07)84011-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1016%2Fs0091-679x%2807%2984011-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=17964936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1amtLY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=84&amp;publication_year=2008&amp;pages=295-325&amp;author=T.+O.+Streetauthor=N.+Courtemancheauthor=D.+Barrick&amp;title=Protein+Folding+and+Stability+Using+Denaturants&amp;doi=10.1016%2Fs0091-679x%2807%2984011-8" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Protein folding and stability using denaturants</span></div><div class="casAuthors">Street, Timothy O.; Courtemanche, Naomi; Barrick, Doug</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">Biophysical Tools for Biologists, Volume One</span>),
    <span class="NLM_cas:pages">295-325</span>CODEN:
                <span class="NLM_cas:coden">MCBLAG</span>;
        ISSN:<span class="NLM_cas:issn">0091-679X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Measurements of protein folding and thermodn. stability provide insight into the forces and energetics that det. structure, and can inform on protein domain organization, interdomain interactions, and effects of mutations on structure.  This chapter describes methods, theory, and data anal. for the most accessible means to det. the thermodn. of protein folding: chem. denaturation.  Topics include overall features of the folding reaction, advances in instrumentation, optimization of reagent purity, mechanistic models for anal., and statistical and structural interpretation of fitted thermodn. parameters.  Examples in which stability measurements have provided insight into structure and function will be taken from studies in the author's lab. on the Notch signaling pathway.  It is hoped that this chapter will enable mol., cell, and structural biologists to make precise measurements of protein stability, and will also provide a strong foundation for biophysics students who wish to undertake exptl. studies of protein folding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplhWYZ7zDdwrVg90H21EOLACvtfcHk0li4Zv3UP5UKbQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXht1amtLY%253D&amp;md5=f265362505ccf991e79aa386426075b9</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fs0091-679x%2807%2984011-8&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0091-679x%252807%252984011-8%26sid%3Dliteratum%253Aachs%26aulast%3DStreet%26aufirst%3DT.%2BO.%26aulast%3DCourtemanche%26aufirst%3DN.%26aulast%3DBarrick%26aufirst%3DD.%26atitle%3DProtein%2520Folding%2520and%2520Stability%2520Using%2520Denaturants%26jtitle%3DMethods%2520Cell%2520Biol.%26date%3D2008%26volume%3D84%26spage%3D295%26epage%3D325%26doi%3D10.1016%2Fs0091-679x%2807%2984011-8" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gromiha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawareh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarai, A.</span></span> <span> </span><span class="NLM_article-title">Asaview: Database and Tool for Solvent Accessibility Representation in Proteins</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">51</span>, <span class="refDoi">&nbsp;DOI: 10.1186/1471-2105-5-51</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1186%2F1471-2105-5-51" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=15119964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A280%3ADC%252BD2c3otVOqsg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2004&amp;pages=51&amp;author=S.+Ahmadauthor=M.+Gromihaauthor=H.+Fawarehauthor=A.+Sarai&amp;title=Asaview%3A+Database+and+Tool+for+Solvent+Accessibility+Representation+in+Proteins&amp;doi=10.1186%2F1471-2105-5-51" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">ASAView: database and tool for solvent accessibility representation in proteins</span></div><div class="casAuthors">Ahmad Shandar; Gromiha Michael; Fawareh Hamed; Sarai Akinori</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Accessible surface area (ASA) or solvent accessibility of amino acids in a protein has important implications.  Knowledge of surface residues helps in locating potential candidates of active sites.  Therefore, a method to quickly see the surface residues in a two dimensional model would help to immediately understand the population of amino acid residues on the surface and in the inner core of the proteins.  RESULTS:  ASAView is an algorithm, an application and a database of schematic representations of solvent accessibility of amino acid residues within proteins.  A characteristic two-dimensional spiral plot of solvent accessibility provides a convenient graphical view of residues in terms of their exposed surface areas.  In addition, sequential plots in the form of bar charts are also provided.  Online plots of the proteins included in the entire Protein Data Bank (PDB), are provided for the entire protein as well as their chains separately.  CONCLUSIONS:  These graphical plots of solvent accessibility are likely to provide a quick view of the overall topological distribution of residues in proteins.  Chain-wise computation of solvent accessibility is also provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTCaKPeuivdBuRS7coPn-RtfW6udTcc2eb0CX6vjca7e7ntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BD2c3otVOqsg%253D%253D&amp;md5=d253b58936b69be51806e48b66ef8c44</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-5-51&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-5-51%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DS.%26aulast%3DGromiha%26aufirst%3DM.%26aulast%3DFawareh%26aufirst%3DH.%26aulast%3DSarai%26aufirst%3DA.%26atitle%3DAsaview%253A%2520Database%2520and%2520Tool%2520for%2520Solvent%2520Accessibility%2520Representation%2520in%2520Proteins%26jtitle%3DBMC%2520Bioinf.%26date%3D2004%26volume%3D5%26spage%3D51%26doi%3D10.1186%2F1471-2105-5-51" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span> <span> </span><span class="NLM_article-title">A Dry Molten Globule-Like Intermediate During the Base-Induced Unfolding of a Multidomain Protein</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">30207</span>â <span class="NLM_lpage">30216</span>, <span class="refDoi">&nbsp;DOI: 10.1039/c7cp06614g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1039%2Fc7cp06614g" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2017&amp;pages=30207-30216&amp;author=N.+Acharyaauthor=P.+Mishraauthor=S.+K.+Jha&amp;title=A+Dry+Molten+Globule-Like+Intermediate+During+the+Base-Induced+Unfolding+of+a+Multidomain+Protein&amp;doi=10.1039%2Fc7cp06614g" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1039%2Fc7cp06614g&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc7cp06614g%26sid%3Dliteratum%253Aachs%26aulast%3DAcharya%26aufirst%3DN.%26aulast%3DMishra%26aufirst%3DP.%26aulast%3DJha%26aufirst%3DS.%2BK.%26atitle%3DA%2520Dry%2520Molten%2520Globule-Like%2520Intermediate%2520During%2520the%2520Base-Induced%2520Unfolding%2520of%2520a%2520Multidomain%2520Protein%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2017%26volume%3D19%26spage%3D30207%26epage%3D30216%26doi%3D10.1039%2Fc7cp06614g" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demchenko, A. P.</span></span> <span> </span><span class="NLM_article-title">Red-Edge-Excitation Fluorescence Spectroscopy of Single-Tryptophan Proteins</span>. <i>Eur. Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">121</span>â <span class="NLM_lpage">129</span>, <span class="refDoi">&nbsp;DOI: 10.1007/bf00255522</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1007%2Fbf00255522" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=3208709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaL1cXlvVKlsrw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=1988&amp;pages=121-129&amp;author=A.+P.+Demchenko&amp;title=Red-Edge-Excitation+Fluorescence+Spectroscopy+of+Single-Tryptophan+Proteins&amp;doi=10.1007%2Fbf00255522" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Red-edge-excitation fluorescence spectroscopy of single-tryptophan proteins</span></div><div class="casAuthors">Demchenko, A. P.</div><div class="citationInfo"><span class="NLM_cas:title">European Biophysics Journal</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">121-9</span>CODEN:
                <span class="NLM_cas:coden">EBJOE8</span>;
        ISSN:<span class="NLM_cas:issn">0175-7571</span>.
    </div><div class="casAbstract">With the aim of finding nonequil. dipole-relaxational electronic excited states of tryptophan residues in proteins, the dependence of the fluorescence emission max. on excitation wavelength was studied for several proteins contg. a single tryptophan residue per mol.  Spectral shifts upon red-edge excitation are not obsd. for short wavelength-emitting proteins (azurin, 2-Ca form of whiting parvalbumin, RNases C2 and T1).  This may be because of the nonpolar environment of the tryptophan residues in these proteins or because of the absence of dipole-orientational broadening of spectra.  The effect was also not found for proteins emitting at long wavelengths (max. at 341-350 nm): melittin at low ionic strength, IT-Aj1 protease inhibitor, myelin basic protein.  In these proteins, the tryptophan residues are exposed to the rapidly relaxing aq. solvent.  Spectral shifts assocd. with red-edge excitation are obsd. for proteins emitting in the medium spectral range-human serum albumin in the N and F forms, IT-Aj1 protease inhibitor at pH 2.9, melittin at high ionic strength as well as the albumin-SDS complex.  This suggests the existence in these proteins of a distribution of microstates for tryptophan environment with various orientation of dipoles and of slow (on the nanosecond time scale) mobility of the field of these dipoles.  As a result, the emission proceeds from electronic excited states which are not at equil.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqza9BS99B2lLVg90H21EOLACvtfcHk0lha4ZpcC_vscQ"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaL1cXlvVKlsrw%253D&amp;md5=16dd42ecb8bd9b31bdbeb1f86f421926</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1007%2Fbf00255522&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fbf00255522%26sid%3Dliteratum%253Aachs%26aulast%3DDemchenko%26aufirst%3DA.%2BP.%26atitle%3DRed-Edge-Excitation%2520Fluorescence%2520Spectroscopy%2520of%2520Single-Tryptophan%2520Proteins%26jtitle%3DEur.%2520Biophys.%2520J.%26date%3D1988%26volume%3D16%26spage%3D121%26epage%3D129%26doi%3D10.1007%2Fbf00255522" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenfield, N. J.</span></span> <span> </span><span class="NLM_article-title">Using circular dichroism spectra to estimate protein secondary structure</span>. <i>Nat Protoc</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">2876</span>â <span class="NLM_lpage">2890</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nprot.2006.202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1038%2Fnprot.2006.202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=17406547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGjtLrM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1&amp;publication_year=2006&amp;pages=2876-2890&amp;author=N.+J.+Greenfield&amp;title=Using+circular+dichroism+spectra+to+estimate+protein+secondary+structure&amp;doi=10.1038%2Fnprot.2006.202" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Using circular dichroism spectra to estimate protein secondary structure</span></div><div class="casAuthors">Greenfield, Norma J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2876-2890</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">CD (CD) is an excellent tool for rapid detn. of the secondary structure and folding properties of proteins that have been obtained using recombinant techniques or purified from tissues.  The most widely used applications of protein CD are to det. whether an expressed, purified protein is folded, or if a mutation affects its conformation or stability.  In addn., it can be used to study protein interactions.  This protocol details the basic steps of obtaining and interpreting CD data, and methods for analyzing spectra to est. the secondary structural compn. of proteins.  CD has the advantage that measurements may be made on multiple samples contg. â¤20 Î¼g of proteins in physiol. buffers in a few hours.  However, it does not give the residue-specific information that can be obtained by x-ray crystallog. or NMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlsmX4yVdMSbVg90H21EOLACvtfcHk0ljANdPeNp-0Vg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGjtLrM&amp;md5=7a4ce0eeff5915c85be1db2327f845a2</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.202&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.202%26sid%3Dliteratum%253Aachs%26aulast%3DGreenfield%26aufirst%3DN.%2BJ.%26atitle%3DUsing%2520circular%2520dichroism%2520spectra%2520to%2520estimate%2520protein%2520secondary%2520structure%26jtitle%3DNat%2520Protoc%26date%3D2006%26volume%3D1%26spage%3D2876%26epage%3D2890%26doi%3D10.1038%2Fnprot.2006.202" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finkelstein, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakhnovich, E. I.</span></span> <span> </span><span class="NLM_article-title">Theory of Cooperative Transitions in Protein Molecules. II. Phase Diagram for a Protein Molecule in Solution</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1681</span>â <span class="NLM_lpage">1694</span>, <span class="refDoi">&nbsp;DOI: 10.1002/bip.360281004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1002%2Fbip.360281004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=2597724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A280%3ADyaK3c%252FovFGjtA%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=28&amp;publication_year=1989&amp;pages=1681-1694&amp;author=A.+V.+Finkelsteinauthor=E.+I.+Shakhnovich&amp;title=Theory+of+Cooperative+Transitions+in+Protein+Molecules.+II.+Phase+Diagram+for+a+Protein+Molecule+in+Solution&amp;doi=10.1002%2Fbip.360281004" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Theory of cooperative transitions in protein molecules. II. Phase diagram for a protein molecule in solution</span></div><div class="casAuthors">Finkelstein A V; Shakhnovich E I</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1681-94</span>
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    </div><div class="casAbstract">The thermodynamically stable states of denatured protein in solution are investigated.  These states are distinguished from the native state by the absence of tight packing of side chains while the compactness of denatured protein may vary within a wide region.  The following regimes are outlined: 1. the "wet" molten globule, i.e., the compact state with pores occupied by solvent; 2. the swollen globule ("wet," of course); and 3. the coil.  The "dry" molten globule, when solvent does not penetrate inside the protein, is excluded for all experimental conditions.  All the transitions within the denatured globule state are gradual while the denatured globule-coil phase transition is a second order one.  The conditions of protein denaturation as well as conditions of transitions and crossovers within the denatured state are outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQJeb6uujKyfMzK395ZObgVfW6udTcc2eaPp-EM2DIOwrntkvexoksv"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADyaK3c%252FovFGjtA%253D%253D&amp;md5=8ed55ae24a29b8b4673c47709ee14d9f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fbip.360281004&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.360281004%26sid%3Dliteratum%253Aachs%26aulast%3DFinkelstein%26aufirst%3DA.%2BV.%26aulast%3DShakhnovich%26aufirst%3DE.%2BI.%26atitle%3DTheory%2520of%2520Cooperative%2520Transitions%2520in%2520Protein%2520Molecules.%2520II.%2520Phase%2520Diagram%2520for%2520a%2520Protein%2520Molecule%2520in%2520Solution%26jtitle%3DBiopolymers%26date%3D1989%26volume%3D28%26spage%3D1681%26epage%3D1694%26doi%3D10.1002%2Fbip.360281004" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shakhnovich, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkelstein, A. V.</span></span> <span> </span><span class="NLM_article-title">Theory of Cooperative Transitions in Protein Molecules. I. Why Denaturation of Globular Protein is a First-Order Phase Transition</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1667</span>â <span class="NLM_lpage">1680</span>, <span class="refDoi">&nbsp;DOI: 10.1002/bip.360281003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1002%2Fbip.360281003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=2597723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADyaK3cXivVGn" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=28&amp;publication_year=1989&amp;pages=1667-1680&amp;author=E.+I.+Shakhnovichauthor=A.+V.+Finkelstein&amp;title=Theory+of+Cooperative+Transitions+in+Protein+Molecules.+I.+Why+Denaturation+of+Globular+Protein+is+a+First-Order+Phase+Transition&amp;doi=10.1002%2Fbip.360281003" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Theory of cooperative transitions in protein molecules.  I.  Why denaturation of globular protein is a first-order phase transition</span></div><div class="casAuthors">Shakhnovich, E. I.; Finkel'shtein, A. V.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1667-80</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    </div><div class="casAbstract">A theory of equil. denaturation of proteins is suggested.  According to this theory, a cornerstone of protein denaturation is disruption of tight packing of side chains in the protein core.  This disruption was investigated.  This disruption is an all-or-none transition (independent of how compact the denatured state of a protein is and independent of protein-solvent interactions) because expansion of a globule must exceed some threshold to release rotational isomerization of side chains.  Smaller expansion cannot produce entropy compensation of nonbonded energy loss; this is the origin of a free-energy barrier (transition state) between the native and denatured states.  The d. of the transition state is so high that the solvent cannot penetrate into protein in this state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-SThXaIbxxrVg90H21EOLACvtfcHk0ljANdPeNp-0Vg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK3cXivVGn&amp;md5=552f410b3f1984eb3c672650a5ae3e75</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fbip.360281003&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.360281003%26sid%3Dliteratum%253Aachs%26aulast%3DShakhnovich%26aufirst%3DE.%2BI.%26aulast%3DFinkelstein%26aufirst%3DA.%2BV.%26atitle%3DTheory%2520of%2520Cooperative%2520Transitions%2520in%2520Protein%2520Molecules.%2520I.%2520Why%2520Denaturation%2520of%2520Globular%2520Protein%2520is%2520a%2520First-Order%2520Phase%2520Transition%26jtitle%3DBiopolymers%26date%3D1989%26volume%3D28%26spage%3D1667%26epage%3D1680%26doi%3D10.1002%2Fbip.360281003" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mountain, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thirumalai, D.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics Simulations of End-to-End Contact Formation in Hydrocarbon Chains in Water and Aqueous Urea Solution</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1950</span>â <span class="NLM_lpage">1957</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja020496f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja020496f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmtVSnsA%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=125&amp;publication_year=2003&amp;pages=1950-1957&amp;author=R.+D.+Mountainauthor=D.+Thirumalai&amp;title=Molecular+Dynamics+Simulations+of+End-to-End+Contact+Formation+in+Hydrocarbon+Chains+in+Water+and+Aqueous+Urea+Solution&amp;doi=10.1021%2Fja020496f" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Dynamics Simulations of End-to-End Contact Formation in Hydrocarbon Chains in Water and Aqueous Urea Solution</span></div><div class="casAuthors">Mountain, Raymond D.; Thirumalai, D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1950-1957</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The urea-denaturation mechanism was studied using mol. dynamics simulations of an elementary "folding" event, namely, the formation of end-to-end contact in the linear hydrocarbon chain (HC) CH3(CH2)18CH3.  Electrostatic effects are examd. using a model HC in which one end of the chain is pos. charged (+0.2e) and the other contains a neg. charge (-0.2e).  For these systems multiple transitions between "folded" (conformations in which the chain ends are in contact) and "unfolded" (end-to-end contact is broken) can be obsd. during 4 ns mol. dynamics simulations.  In water and 6 M aq. urea soln. HC and the charged HC fluctuate between collapsed globular conformations and a set of expanded structures.  The collapsed conformation adopted by the HC in water is slightly destabilized in 6 M urea.  In contrast, the end-to-end contact is disrupted in the charged HC only in aq. urea soln.  Despite the presence of a large hydrophobic patch, on length scales on the order of â¼8-10 Ã "denaturation" (transition to the expanded unfolded state) occurs by a direct interaction of urea with charges on the chain ends.  The contiguous patch of hydrophobic moieties leads to "mild dewetting", which becomes more pronounced in the charged HC in 6 M aq. urea soln.  These simulations establish that the urea denaturation mechanism is most likely electrostatic in origin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp36dXv3EvoBbVg90H21EOLACvtfcHk0ljANdPeNp-0Vg"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3sXmtVSnsA%253D%253D&amp;md5=5b08f62f14e7fc33d6f53b8ebd91043b</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fja020496f&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja020496f%26sid%3Dliteratum%253Aachs%26aulast%3DMountain%26aufirst%3DR.%2BD.%26aulast%3DThirumalai%26aufirst%3DD.%26atitle%3DMolecular%2520Dynamics%2520Simulations%2520of%2520End-to-End%2520Contact%2520Formation%2520in%2520Hydrocarbon%2520Chains%2520in%2520Water%2520and%2520Aqueous%2520Urea%2520Solution%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D1950%26epage%3D1957%26doi%3D10.1021%2Fja020496f" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoeltzli, S.
D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frieden, C.</span></span> <span> </span><span class="NLM_article-title">Stopped-Flow NMR Spectroscopy: Real-Time Unfolding Studies of 6-<sup>19</sup>F-Tryptophan-Labeled <i>Escherichia coli</i> Dihydrofolate Reductase</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">9318</span>â <span class="NLM_lpage">9322</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.92.20.9318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1073%2Fpnas.92.20.9318" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=92&amp;publication_year=1995&amp;pages=9318-9322&amp;author=S.%0AD.+Hoeltzliauthor=C.+Frieden&amp;title=Stopped-Flow+NMR+Spectroscopy%3A+Real-Time+Unfolding+Studies+of+6-19F-Tryptophan-Labeled+Escherichia+coli+Dihydrofolate+Reductase&amp;doi=10.1073%2Fpnas.92.20.9318" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1073%2Fpnas.92.20.9318&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.92.20.9318%26sid%3Dliteratum%253Aachs%26aulast%3DHoeltzli%26aufirst%3DS.%2BD.%26aulast%3DFrieden%26aufirst%3DC.%26atitle%3DStopped-Flow%2520NMR%2520Spectroscopy%253A%2520Real-Time%2520Unfolding%2520Studies%2520of%25206-19F-Tryptophan-Labeled%2520Escherichia%2520coli%2520Dihydrofolate%2520Reductase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1995%26volume%3D92%26spage%3D9318%26epage%3D9322%26doi%3D10.1073%2Fpnas.92.20.9318" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rami, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udgaonkar, J. B.</span></span> <span> </span><span class="NLM_article-title">Mechanism of Formation of a Productive Molten Globule Form of Barstarâ </span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1710</span>â <span class="NLM_lpage">1716</span>, <span class="refDoi">&nbsp;DOI: 10.1021/bi0120300</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0120300" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BD38Xkt1CrtQ%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=41&amp;publication_year=2002&amp;pages=1710-1716&amp;author=B.+R.+Ramiauthor=J.+B.+Udgaonkar&amp;title=Mechanism+of+Formation+of+a+Productive+Molten+Globule+Form+of+Barstar%E2%80%A0&amp;doi=10.1021%2Fbi0120300" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of Formation of a Productive Molten Globule Form of Barstar</span></div><div class="casAuthors">Rami, Bhadresh R.; Udgaonkar, Jayant B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1710-1716</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structural anal. of the initial steps in protein folding is difficult because of the swiftness with which these steps occur.  Hence, the link between initial polypeptide chain collapse and formation of secondary and other specific structures remains poorly understood.  Here, an equil. model has been developed for characterizing the initial steps of folding of the small protein barstar, which lead to the formation of a productive molten globule in the folding pathway.  In this model, the high-pH-unfolded form (D form) of barstar, which is shown to be as unstructured as the urea-denatured form, is transformed progressively into a molten globule B form by incremental addn. of the salt Na2SO4 at pH 12.  At very low concns. of Na2SO4, the D form collapses into a pre-molten globule (P) form, whose vol. exceeds that of the native (N) state by only 20%, and which lacks any specific structure as detd. by far- and near-UV CD.  At higher concns. of Na2SO4, the P form transforms into the molten globule (B) form in a highly noncooperative transition populated by an ensemble of at least two intermediates.  The B form is a dry molten globule in which water is excluded from the core, and in which secondary structure develops to 65% and tertiary contacts develop to 40%, relative to that of the native protein.  Kinetic refolding expts. carried out at pH 7 and at high Na2SO4 concns., in which the rate of folding of the D form to the N state is compared to that of the B form to the N state, indicate conclusively that the B form is a productive intermediate that forms on the direct pathway of folding from the D form to the N state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLsutbyDvbr7Vg90H21EOLACvtfcHk0lhUCAOAM7U4Kw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD38Xkt1CrtQ%253D%253D&amp;md5=b83f27e35312abc7aa844695395df31b</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fbi0120300&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0120300%26sid%3Dliteratum%253Aachs%26aulast%3DRami%26aufirst%3DB.%2BR.%26aulast%3DUdgaonkar%26aufirst%3DJ.%2BB.%26atitle%3DMechanism%2520of%2520Formation%2520of%2520a%2520Productive%2520Molten%2520Globule%2520Form%2520of%2520Barstar%25E2%2580%25A0%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26spage%3D1710%26epage%3D1716%26doi%3D10.1021%2Fbi0120300" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasinath, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moorman, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nucci, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilser, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Coupled Motion in Proteins Revealed by Pressure Perturbation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">8543</span>â <span class="NLM_lpage">8550</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ja3004655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja3004655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC38XksFWgsbc%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=134&amp;publication_year=2012&amp;pages=8543-8550&amp;author=Y.+Fuauthor=V.+Kasinathauthor=V.+R.+Moormanauthor=N.+V.+Nucciauthor=V.+J.+Hilserauthor=A.+J.+Wand&amp;title=Coupled+Motion+in+Proteins+Revealed+by+Pressure+Perturbation&amp;doi=10.1021%2Fja3004655" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Coupled Motion in Proteins Revealed by Pressure Perturbation</span></div><div class="casAuthors">Fu, Yinan; Kasinath, Vignesh; Moorman, Veronica R.; Nucci, Nathaniel V.; Hilser, Vincent J.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8543-8550</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The cooperative nature of protein substructure and internal motion is a crit. aspect of their functional competence about which little is known exptl.  NMR relaxation is used here to monitor the effects of high pressure on fast internal motion in the protein ubiquitin.  In contrast to the main chain, the motions of the methyl-bearing side chains have a large and variable pressure dependence.  Within the core, this pressure sensitivity correlates with the magnitude of motion at ambient pressure.  Spatial clustering of the dynamic response to applied hydrostatic pressure is also seen, indicating localized cooperativity of motion on the sub-nanosecond time scale and suggesting regions of variable compressibility.  These and other features indicate that the native ensemble contains a significant fraction of members with characteristics ascribed to the recently postulated "dry molten globule".  The accompanying variable side-chain conformational entropy helps complete our view of the thermodn. architecture underlying protein stability, folding, and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa9mgJ5NQpqbVg90H21EOLACvtfcHk0lhUCAOAM7U4Kw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XksFWgsbc%253D&amp;md5=9d8d93a5cd026221cb28e14914512ec6</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fja3004655&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja3004655%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DY.%26aulast%3DKasinath%26aufirst%3DV.%26aulast%3DMoorman%26aufirst%3DV.%2BR.%26aulast%3DNucci%26aufirst%3DN.%2BV.%26aulast%3DHilser%26aufirst%3DV.%2BJ.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DCoupled%2520Motion%2520in%2520Proteins%2520Revealed%2520by%2520Pressure%2520Perturbation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D8543%26epage%3D8550%26doi%3D10.1021%2Fja3004655" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jha, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marqusee, S.</span></span> <span> </span><span class="NLM_article-title">Kinetic Evidence for a Two-Stage Mechanism of Protein Denaturation by Guanidinium Chloride</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">4856</span>â <span class="NLM_lpage">4861</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1315453111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=10.1073%2Fpnas.1315453111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=24639503" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksVShtL0%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=111&amp;publication_year=2014&amp;pages=4856-4861&amp;author=S.+K.+Jhaauthor=S.+Marqusee&amp;title=Kinetic+Evidence+for+a+Two-Stage+Mechanism+of+Protein+Denaturation+by+Guanidinium+Chloride&amp;doi=10.1073%2Fpnas.1315453111" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic evidence for a two-stage mechanism of protein denaturation by guanidinium chloride</span></div><div class="casAuthors">Jha, Santosh Kumar; Marqusee, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4856-4861</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Dry molten globular (DMG) intermediates, an expanded form of the native protein with a dry core, have been obsd. during denaturant-induced unfolding of many proteins.  These observations are counterintuitive because traditional models of chem. denaturation rely on changes in solvent-accessible surface area, and there is no notable change in solvent-accessible surface area during the formation of the DMG.  Here we show, using multisite fluorescence resonance energy transfer, far-UV CD, and kinetic thiol-labeling expts., that the guanidinium chloride (GdmCl)-induced unfolding of RNase H also begins with the formation of the DMG.  Population of the DMG occurs within the 5-ms dead time of our measurements.  We observe that the size and/or population of the DMG is linearly dependent on [GdmCl], although not as strongly as the second and major step of unfolding, which is accompanied by core solvation and global unfolding.  This rapid GdmCl-dependent population of the DMG indicates that GdmCl can interact with the protein before disrupting the hydrophobic core.  These results imply that the effect of chem. denaturants cannot be interpreted solely as a disruption of the hydrophobic effect and strongly support recent computational studies, which hypothesize that chem. denaturants first interact directly with the protein surface before completely unfolding the protein in the second step (direct interaction mechanism).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder Â®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsEgN87ohg-LVg90H21EOLACvtfcHk0lhUCAOAM7U4Kw"> &gt;&gt; More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXksVShtL0%253D&amp;md5=59f168c932bb8620d113b2e02dcf055f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1315453111&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1315453111%26sid%3Dliteratum%253Aachs%26aulast%3DJha%26aufirst%3DS.%2BK.%26aulast%3DMarqusee%26aufirst%3DS.%26atitle%3DKinetic%2520Evidence%2520for%2520a%2520Two-Stage%2520Mechanism%2520of%2520Protein%2520Denaturation%2520by%2520Guanidinium%2520Chloride%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D4856%26epage%3D4861%26doi%3D10.1073%2Fpnas.1315453111" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://doi.org/10.2210/pdb1AO6/pdb" onclick="trackOutboundLink('http://doi.org/10.2210/pdb1AO6/pdb','PDB','1AO6'); return false;">PDB: 1AO6</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7" style="width: 457.7px;"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jpcb.8b11151&amp;pbContext=%3Bjournal%3Ajournal%3Ajpcbfk%3Bissue%3Aissue%3A10.1021%2Fjpcbfk.2019.123.issue-6%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3BrequestedJournal%3Ajournal%3Ajpcbfk%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jpcb.8b11151" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self" style="display: none;"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jpcb.8b11151" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article><div class="w-slide"><div class="w-slide_head"><a href="#" class="w-slide__back"><i class=" icon-arrow_l" aria-hidden="true"></i>back</a><span class="w-slide__title"></span></div><div class="w-slide__content"></div></div></main><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2022<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS &amp; Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support &amp; Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"><input type="hidden" name="appID" value="literatum" class="appID"><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onclick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        <!-- <div id="gdpr-message-container">
			<a id="closebtn" href="#"><i class="icon-times-circle-o"></i></a> -->
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        <!-- </div> -->
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        <!-- AdRoll Pixel -->
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>

<!-- Twitter universal website tag code -->
<!-- Commented out for ACHS-6152
<script>
    !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe.apply(s,arguments):s.queue.push(arguments);
    },s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',
    a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');
    // Insert Twitter Pixel ID and Standard Event data below
    twq('init','o3njp');
    twq('track','PageView');
</script>
-->
<!-- Facebook Pixel Code -->
<!-- Commented out for ACHS-6152
<script>
    !function(f,b,e,v,n,t,s)
    {if(f.fbq)return;n=f.fbq=function(){n.callMethod?
    n.callMethod.apply(n,arguments):n.queue.push(arguments)};
    if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
    n.queue=[];t=b.createElement(e);t.async=!0;
    t.src=v;s=b.getElementsByTagName(e)[0];
    s.parentNode.insertBefore(t,s)}(window, document,'script',
    'https://connect.facebook.net/en_US/fbevents.js');
    fbq('init', '529840281033320');
    fbq('track', 'PageView');
</script>
<noscript><img height="1" width="1" style="display:none" src="https://www.facebook.com/tr?id=529840281033320&amp;ev=PageView&amp;noscript=1"
/></noscript>
-->





        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div><div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/acs.jpcc.8b11685">Ternary Composites with PbS Quantum Dots for Hybrid Photovoltaics</a></li><li><a href="/doi/full/10.1021/acs.jpcb.8b09487">Length-Dependent Melting Kinetics of Short DNA Oligonucleotides Using Temperature-Jump IR Spectroscopy</a></li><li><a href="/doi/full/10.1021/acs.jpcc.8b11201">Theoretical Insights into Propene Epoxidation on Au7/Anatase TiO2âx(001) Catalysts: Effect of the Interface and Reaction Atmosphere</a></li><li><a href="/doi/full/10.1021/acs.jpcc.8b07734">Ultrafast Excited-State Dynamics of Partially and Fully Functionalized Silsesquioxanes</a></li><li><a href="/doi/full/10.1021/acs.jpcc.8b11324">Reactive Molecular Dynamics Simulations to Investigate the Shock Response of Liquid Nitromethane</a></li></ul></div></div></div>




        
        <script>
var currentJournal = "The Journal of Physical Chemistry B";
$(".citationLinks > a").click(function(e){
    if($(this).text() === "ACS Full Text ") {
        ga('send', 'event', 'Reference _Click', 'ACS Full Text', currentJournal);
    }
    else if($(this).text() === "Crossref") {
        ga('send', 'event', 'Reference _Click', 'Crossref', currentJournal);
    }
    else if($(this).text() === "CAS") {
        ga('send', 'event', 'Reference _Click', 'CAS', currentJournal);
    }
    else if($(this).text() === "PubMed") {
        ga('send', 'event', 'Reference _Click', 'PubMed', currentJournal);
    }
    else if($(this).text() === "Google Scholar") {
        ga('send', 'event', 'Reference _Click', 'Google Scholar', currentJournal);
    }
    else if($(this).hasClass("issue-item_open-url") === true) {
        ga('send', 'event', 'Reference _Click', 'OpenURL', currentJournal);
    }
});
</script>






        
        



        
    





        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        <div tabindex="0" class="showMenuAlertMsg-popup"><div class="pairing-affiliations-popup-content"><div class="pairing-affiliations-header"><button type="button" class="close paringAffClose"><i aria-hidden="true" class="icon-close_thin"></i></button></div><div class="responsrMsg"></div></div></div>




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	dartInstitutionType = 'education';
</script>
        </div>
    





        
        



        
    





        
        <script>
// LIT-455776 workaround
// Moving the popup before the header content for tab accessibility
document.addEventListener('DOMContentLoaded', () => {
    const headerNode = $('.header_content');
    const popupNode = $('.pairing-affiliations-popup');

    if (!headerNode.length || !popupNode.length)
        return;

    popupNode.insertBefore(headerNode);  
});
</script> 
        </div>
    </div>





	
    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-d93b9088814c7fcd2f1c.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-9dee82777d6af4ace557.js"></script>
<script type="text/javascript" src="/wro/l6j1~product.js"></script>



















<div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh217" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.4791695076012331&amp;iit=1651100877900&amp;tmr=load%3D1651100877489%26core%3D1651100877523%26main%3D1651100877895%26ifr%3D1651100877905&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=pubs.acs.org&amp;dr=&amp;du=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facs.jpcb.8b11151&amp;href=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facs.jpcb.8b11151&amp;dt=Slow%20Motion%20Protein%20Dance%20Visualized%20Using%20Red-Edge%20Excitation%20Shift%20of%20a%20Buried%20Fluorophore&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Ctype%3DArticle%2Cdescription%2Cimage%2Ctitle%2Curl&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=6269cccdea2c32e4&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=url%3Dhttps%253A%252F%252Fpubs.acs.org%252Fdoi%252Ffull%252F10.1021%252Facs.jpcb.8b11151%26title%3DSlow%2520Motion%2520Protein%2520Dance%2520Visualized%2520Using%2520Red-Edge%2520Excitation%2520Shift%2520of%2520a%2520Buried%2520Fluorophore%26image%3Dhttps%253A%252F%252Fpubs.acs.org%252Fcms%252F10.1021%252Facs.jpcb.8b11151%252Fasset%252Fimages%252Facs.jpcb.8b11151.social.jpeg_v03%26description%3DIt%2520has%2520been%2520extremely%2520challenging%2520to%2520detect%2520protein%2520structures%2520with%2520a%2520dynamic%2520core%252C%2520such%2520as%2520dry%2520molten%2520globules%252C%2520that%2520remain%2520in%2520equilibrium%2520with%2520the%2520tightly%2520packed%2520native%2520(N)%2520state%2520and%2520that%2520are%2520important%2520for%2520a%2520myriad%2520of%2520entropy-driven%2520protein%2520functions.%2520Here%252C%2520we%2520detect%2520the%2520higher%2520entropy%2520conformations%2520of%2520a%2520human%2520serum%2520protein%252C%2520using%2520red-edge%2520excitation%2520shift%2520experiments.%2520We%2520covalently%2520introduced%2520a%2520fluorophore%2520inside%2520the%2520protein%2520core%2520and%2520observed%2520that%2520in%2520a%2520subset%2520of%2520native%2520population%252C%2520the%2520side%2520chains%2520of%2520the%2520polar%2520and%2520buried%2520residues%2520have%2520different%2520spatial%2520arrangements%2520than%2520the%2520mean%2520population%2520and%2520that%2520they%2520solvate%2520the%2520fluorophore%2520on%2520a%2520timescale%2520much%2520slower%2520than%2520the%2520nanosecond%2520timescale%2520of%2520fluorescence.%2520Our%2520results%2520provide%2520direct%2520evidence%2520for%2520the%2520dense%2520fluidity%2520of%2520protein%2520core%2520and%2520show%2520that%2520alternate%2520side-chain%2520packing%2520arrangements%2520exist%2520in%2520the%2520core%2520that%2520might%2520be%2520important%2520for%2520multiple%2520binding%2520functions%2520of%2520this%2520protein.%26type%3DArticle%26site_name%3DACS%2520Publications&amp;csi=undefined&amp;rev=v8.28.8-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div></body></html>