<html class="pb-page" data-request-id="229b4eaa-b93c-4e33-8dbf-42b190035c20" lang="en"><head data-pb-dropzone="head"><script type="text/javascript" src="https://m.addthis.com/live/red_lojson/300lo.json?si=6271939f8adc0cb0&amp;bkl=0&amp;bl=1&amp;pdt=418&amp;sid=6271939f8adc0cb0&amp;pub=xa-4faab26f2cff13a7&amp;rev=v8.28.8-wp&amp;ln=en&amp;pc=men&amp;cb=0&amp;ab=-&amp;dp=pubs.acs.org&amp;fp=doi%2F10.1021%2Facssensors.9b01974&amp;fr=&amp;of=0&amp;pd=0&amp;irt=1&amp;vcl=1&amp;md=0&amp;ct=1&amp;tct=0&amp;abt=0&amp;cdn=0&amp;pi=1&amp;rb=0&amp;gen=100&amp;chr=UTF-8&amp;mk=fractal%20LAMP%2Cdigital%20analysis%2Cnucleic%20acid%20amplification%20test%2CBayesian%20model%2Cbootstrapping&amp;colc=1651610527611&amp;jsl=1&amp;uvs=6271939d987ec819001&amp;skipb=1&amp;callback=addthis.cbs.jsonp__466097056866822460"></script><script type="text/javascript" src="https://v1.addthisedge.com/live/boost/xa-4faab26f2cff13a7/_ate.track.config_resp"></script><script type="text/javascript" src="https://z.moatads.com/addthismoatframe568911941483/moatframe.js"></script><meta name="pbContext" content=";journal:journal:ascefj;page:string:Article/Chapter View;issue:issue:10.1021/ascefj.2020.5.issue-2;ctype:string:Journal Content;requestedJournal:journal:ascefj;wgroup:string:ACHS website Group;article:article:10.1021/acssensors.9b01974;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text">




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="ACS Sensors"><meta name="dc.Title" content="Fractal LAMP: Label-Free Analysis of Fractal Precipitate for Digital Loop-Mediated Isothermal Nucleic Acid Amplification"><meta name="dc.Creator" content="Hector E.  Muñoz"><meta name="dc.Creator" content="Carson T.  Riche"><meta name="dc.Creator" content="Janay E.  Kong"><meta name="dc.Creator" content="Mark  van Zee"><meta name="dc.Creator" content="Omai B.  Garner"><meta name="dc.Creator" content="Aydogan  Ozcan"><meta name="dc.Creator" content="Dino  Di Carlo"><meta name="dc.Subject" content="fractal LAMP; digital analysis; nucleic acid amplification test; Bayesian model; bootstrapping"><meta name="dc.Description" content="Nucleic acid amplification assays including loop-mediated isothermal amplification (LAMP) are routinely used in diagnosing diseases and monitoring water and food quality. The results of amplificati..."><meta name="Description" content="Nucleic acid amplification assays including loop-mediated isothermal amplification (LAMP) are routinely used in diagnosing diseases and monitoring water and food quality. The results of amplificati..."><meta name="dc.Publisher" content="American Chemical Society"><meta name="dc.Date" scheme="WTN8601" content="January 6, 2020"><meta name="dc.Type" content="research-article"><meta name="dc.Format" content="text/HTML"><meta name="dc.Identifier" scheme="doi" content="10.1021/acssensors.9b01974"><meta name="dc.Language" content="en"><meta name="dc.Coverage" content="world"><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society"><meta name="keywords" content="fractal LAMP,digital analysis,nucleic acid amplification test,Bayesian model,bootstrapping">
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facssensors.9b01974">
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facssensors.9b01974">
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facssensors.9b01974">
        
    
    

<title>Fractal LAMP: Label-Free Analysis of Fractal Precipitate for Digital Loop-Mediated Isothermal Nucleic Acid Amplification | ACS Sensors</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive">





    






        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acssensors.9b01974"><meta property="og:title" content="Fractal LAMP: Label-Free Analysis of Fractal Precipitate for Digital Loop-Mediated Isothermal Nucleic Acid Amplification"><meta property="og:image" content="https://pubs.acs.org/cms/10.1021/acssensors.9b01974/asset/images/acssensors.9b01974.social.jpeg_v03"><meta property="og:description" content="Nucleic acid amplification assays including loop-mediated isothermal amplification (LAMP) are routinely used in diagnosing diseases and monitoring water and food quality. The results of amplification in these assays are commonly measured with an analog fluorescence readout, which requires specialized optical equipment and can lack quantitative precision. Digital analysis of amplification in small fluid compartments based on exceeding a threshold fluorescence level can enhance the quantitative precision of nucleic acid assays (i.e., digital nucleic acid amplification assays), but still requires specialized optical systems for fluorescence readout and the inclusion of a fluorescent dye. Here, we report Fractal LAMP, an automated method to detect amplified DNA in subnanoliter scale droplets following LAMP in a label-free manner. Our computer vision algorithm achieves high accuracy detecting DNA amplification in droplets by identifying LAMP byproducts that form fractal structures observable in brightfield microscopy. The capabilities of Fractal LAMP are further realized by developing a Bayesian model to estimate DNA concentrations for unknown samples and a bootstrapping method to estimate the number of droplets required to achieve target limits of detection. This digital, label-free assay has the potential to lower reagent and reader cost for nucleic acid measurement while maintaining high quantitative accuracy over 3 orders of magnitude of concentration."><meta property="og:type" content="Article"><meta property="og:site_name" content="ACS Publications"><meta name="twitter:card" content="summary_large_image"><meta name="twitter:site" content="@ACSPublications"><meta name="twitter:image" content="https://pubs.acs.org/cms/10.1021/acssensors.9b01974/asset/images/acssensors.9b01974.social.jpeg_v03">
    





    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0">















    

    <meta name="dc.identifier" content="10.1021/acssensors.9b01974">



<meta name="format-detection" content="telephone=no">




<link rel="stylesheet" type="text/css" href="/wro/l6j1~product.css">

<link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-c7d2282eca032afdc856.css"><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css">




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1640088552890.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1648583148410.css">
<script type="text/javascript" async="" src="//img.en25.com/i/elqCfg.min.js"></script><script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script><script async="" src="//www.google-analytics.com/analytics.js"></script><script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer="" src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon="{&quot;rayId&quot;:&quot;5200035e6e4d5180&quot;,&quot;startTime&quot;:1570117212010}"></script>

            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    , 'auto'
                    );
                
                ga('send', 'pageview');
            </script>
            











    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acssensors.9b01974">
            
        







    <!-- loading Javascript for using the persistence Service -->
    <script src="/templates/jsp/pb2/js/plugin/thiss-ds.js"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            alt_context: window.location.origin,
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if (ms == undefined || ms === '') {
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            if (results.length == 0) {
                                persistService.entities(self.alt_context).then((res) => {
                                    return resolve(res.data);
                                }, function (err) {
                                    // failed
                                    reject('failed to retrieve entity from local storage (publisher domain): ' + err);
                                });
                            } else {
                                return resolve(results);
                            }
                        }, function (err) {
                            // failed
                            reject('failed to retrieve entity from local storage: ' + err);
                        });
                    // Set up the timeout
                    setTimeout(function () {
                        reject('Promise timed out ' + ms + ' ms');
                    }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                    persistService.update(self.alt_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage (publisher domain) duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            var results = res.data;
                            if (!results) {
                                persistService.remove(self.alt_context, entityId).then((res) => {
                                    return resolve(res);
                                }, function (err) {
                                    // failed
                                    console.log('failed to remove from local storage (publisher domain) duo to: ' + err);
                                });
                            }
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId, entityTitle) {
                return new Promise(function (resolve, reject) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {
                            doi: doi,
                            entityid: entityId,
                            entityTitle: entityTitle,
                            eventGroupKey: 'c27b6bfa-fed9-4fb8-8d85-2b4c1631160f'
                        }

                    }).done(function (status) {
                        return resolve(status);
                    }).fail(function (status) {
                        return reject(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID, ssoInstitution.entity.title)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + ssoInstitution.entity.entityID,err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            }).catch(function (err) {
                                reject(err);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();

                    if ($('.institution-preference-userconsent-checkbox').length > 0) {
                        let institutionPreferenceUserConsentCheckbox = $('.institution-preference-userconsent-checkbox').get(0);
                        if (!$(institutionPreferenceUserConsentCheckbox).is(":checked")) {
                            return;
                        }
                    }

                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err) {
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>







    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY">


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html">

<link rel="shortcut icon" href="/pb-assets/ux3/favicon-1635812676023.ico"><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><meta http-equiv="origin-trial" content="AxujKG9INjsZ8/gUq8+dTruNvk7RjZQ1oFhhgQbcTJKDnZfbzSTE81wvC2Hzaf3TW4avA76LTZEMdiedF1vIbA4AAABueyJvcmlnaW4iOiJodHRwczovL2ltYXNkay5nb29nbGVhcGlzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzVGhpcmRQYXJ0eSI6dHJ1ZX0="><meta http-equiv="origin-trial" content="Azuce85ORtSnWe1MZDTv68qpaW3iHyfL9YbLRy0cwcCZwVnePnOmkUJlG8HGikmOwhZU22dElCcfrfX2HhrBPAkAAAB7eyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A16nvcdeoOAqrJcmjLRpl1I6f3McDD8EfofAYTt/P/H4/AWwB99nxiPp6kA0fXoiZav908Z8etuL16laFPUdfQsAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="AxBHdr0J44vFBQtZUqX9sjiqf5yWZ/OcHRcRMN3H9TH+t90V/j3ENW6C8+igBZFXMJ7G3Pr8Dd13632aLng42wgAAACBeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiVHJ1c3RUb2tlbnMiLCJleHBpcnkiOjE2NTI3NzQ0MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A88BWHFjcawUfKU3lIejLoryXoyjooBXLgWmGh+hNcqMK44cugvsI5YZbNarYvi3roc1fYbHA1AVbhAtuHZflgEAAAB2eyJvcmlnaW4iOiJodHRwczovL2dvb2dsZS5jb206NDQzIiwiZmVhdHVyZSI6IlRydXN0VG9rZW5zIiwiZXhwaXJ5IjoxNjUyNzc0NDAwLCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="AzoawhTRDevLR66Y6MROu167EDncFPBvcKOaQispTo9ouEt5LvcBjnRFqiAByRT+2cDHG1Yj4dXwpLeIhc98/gIAAACFeyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A6+nc62kbJgC46ypOwRsNW6RkDn2x7tgRh0wp7jb3DtFF7oEhu1hhm4rdZHZ6zXvnKZLlYcBlQUImC4d3kKihAcAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><meta http-equiv="origin-trial" content="A/9La288e7MDEU2ifusFnMg1C2Ij6uoa/Z/ylwJIXSsWfK37oESIPbxbt4IU86OGqDEPnNVruUiMjfKo65H/CQwAAACLeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXRhZ3NlcnZpY2VzLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjYxMjk5MTk5LCJpc1N1YmRvbWFpbiI6dHJ1ZSwiaXNUaGlyZFBhcnR5Ijp0cnVlfQ=="><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_2022042801.js" async=""></script><script src="https://widgets.figshare.com/static/figshare.js?_=1651610526877"></script><script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/CcdcJsmolUtilities.min.js?_=1651610526878"></script><script src="https://common.ccdc.cam.ac.uk/jsmol/14.2.7/JSmol.min.nojq.js?_=1651610526879"></script><link type="text/css" rel="stylesheet" href="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/CcdcWidget.min.css"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'sensors';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/sensors/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/sensors/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
					console.log("ad display: " + event.creativeId + " " + event.size[0] + "x" + event.size[1]);
				}
				else {
				    console.log("no ad found");
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>

<style>
.articleHeaderDropzone5 .advertisement { display: none; }
.article_header .advertisement {
  margin: -25px auto 25px auto;
}
</style>




        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.107/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&amp;EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Indiana University Libraries - Bloomington</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"></a>




        
        <div class="pj-badge"></div><a href="/journal1/ascefj" title="ACS Sensors" class="header_article-logo"><img src="/cms/10.1021/ascefj/asset/title.png" alt="ACS Sensors"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="ascefj" name="SeriesKey" checked="checked" class="all-content"><span>ACS Sens.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"><select name="quickLinkJournal" class="quick-search_journals-select"><option value="ascefj">ACS Sensors</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science &amp; Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials &amp; Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio &amp; Med Chem Au</option><option value="abseba">ACS Biomaterials Science &amp; Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health &amp; Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&amp;T Engineering</option><option value="aewcaa">ACS ES&amp;T Water</option><option value="afsthl">ACS Food Science &amp; Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic &amp; Inorganic Au</option><option value="aptsfn">ACS Pharmacology &amp; Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascecg">ACS Sustainable Chemistry &amp; Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&amp;EN Global Enterprise</option><option value="cenear">Chemical &amp; Engineering News Archive</option><option value="chlseg">Chemical Health &amp; Safety</option><option value="chlseg0">Chemical Health &amp; Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth &amp; Design</option><option value="enfuem">Energy &amp; Fuels</option><option value="esthag">Environmental Science &amp; Technology</option><option value="estlcu">Environmental Science &amp; Technology Letters</option><option value="iepra6.2">I&amp;EC Product Research and Development</option><option value="iechad">Industrial &amp; Engineering Chemistry</option><option value="iecac0">Industrial &amp; Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series</option><option value="iecfa7">Industrial &amp; Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial &amp; Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial &amp; Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial &amp; Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical &amp; Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health &amp; Safety</option><option value="jchsc20">Journal of Chemical Health &amp; Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial &amp; Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research &amp; Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&amp;D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autocomplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autocomplete="false" placeholder="Page" required="required" class="quick-search_page-input"><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden="”true”"></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><ul><li><a href="/doi/full/10.1021/acssensors.9b01928">Improved Measurement of Proteins Using a Solid-State Nanopore Coupled with a Hydrogel</a></li></ul></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="”true”"></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden="”true”"></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden="”true”"></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors &amp; Reviewers');">Authors &amp;
                        Reviewers</a></li>
                <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
				<li><a href="https://pubs.acs.org/page/virtual-collections.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Virtual Issues');">Virtual Issues</a></li>
                <li><a href="https://preferences.acs.org/journals" onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS &amp; Mobile');">RSS &amp; Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products &amp; Services');">Products &amp; Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos &amp; Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#" onclick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');" class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/who-we-serve/librarians/" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians &amp; Account Managers');">For
                        Librarians &amp; Account Managers</a></li>
            </ul>
        </li>
        <li class="menu-pair-section hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a></li>
                
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pb/widgets/affiliateDevice">Pair a device</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS &amp; Open Access');">ACS &amp; Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="/events" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!-- Types--><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><!-- Subjects--><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><!--div.pub-modal_right-cover-gallery.hidden-lg.hidden-md//a(href="/coverGallery") Browse Our Cover Art Gallery
--><!-- Publications list--><!-- 'pub-modal_right--inner' adds styling required for IE--><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><!-- The widget data--><input type="hidden" id="ajaxUrl" name="ajaxUrl" value="/pb/widgets/achs-publication-list?widgetId=cabcb15c-6f7a-494b-a19c-eba4fafaee5c&amp;pbContext=%3Bjournal%3Ajournal%3Aascefj%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fascefj.2020.5.issue-2%3Bctype%3Astring%3AJournal+Content%3BrequestedJournal%3Ajournal%3Aascefj%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facssensors.9b01974%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;key=c7ca46a97ce077568c7371697bcd489c0&amp;ajaxRequest=true"><input type="hidden" name="pubModalData" value=""></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facssensors.9b01974&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facssensors.9b01974&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acssensors.9b01974&amp;href=/doi/10.1021/acssensors.9b01974" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>ACS Sens.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 5</span><span class="cit-fg-issue">, 2</span><span class="cit-fg-pageRange">, 385-394</span></span></span></div></div></div></div></div><input type="hidden" name="ascefj" id="copyRights" value="/pb-assets/json/journal-info-1641590078917.json"><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/ascefj/5/2" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acssensors.9b01968" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acssensors.9b02001" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/ascefj" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/cms/10.1021/ascefj/asset/title.png" alt="Journal Logo"></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Fractal LAMP: Label-Free Analysis of Fractal Precipitate for Digital Loop-Mediated Isothermal Nucleic Acid Amplification</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Hector E. Muñoz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hector E. Muñoz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Bioengineering, University of California, Los Angeles, California 90095, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hector+E.++Mu%C3%B1oz">Hector E. Muñoz</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-7851-2549" title="Orcid link">http://orcid.org/0000-0001-7851-2549</a></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Carson T. Riche</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carson T. Riche</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Bioengineering, University of California, Los Angeles, California 90095, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carson+T.++Riche">Carson T. Riche</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Janay E. Kong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Janay E. Kong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Bioengineering, University of California, Los Angeles, California 90095, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Janay+E.++Kong">Janay E. Kong</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-5911-8116" title="Orcid link">http://orcid.org/0000-0001-5911-8116</a></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Mark van Zee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mark van Zee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Bioengineering, University of California, Los Angeles, California 90095, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mark++van+Zee">Mark van Zee</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Omai B. Garner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Omai B. Garner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology &amp; Laboratory Medicine, University of California, Los Angeles, California 90095, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Omai+B.++Garner">Omai B. Garner</a></span></div></div></span></li><span class="comma-separator">,&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Aydogan Ozcan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Aydogan Ozcan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Bioengineering, University of California, Los Angeles, California 90095, United States</div><div class="loa-info-affiliations-info">Department of Electrical and Computer Engineering, University of California, Los Angeles, California 90095, United States</div><div class="loa-info-affiliations-info">California NanoSystems Institute, University of California, Los Angeles, California 90095, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Aydogan++Ozcan">Aydogan Ozcan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0002-0717-683X" title="Orcid link">http://orcid.org/0000-0002-0717-683X</a></div></div></span></li><span class="comma-separator">,&nbsp;and&nbsp;</span><li><span><span class="hlFld-ContribAuthor">Dino Di Carlo</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dino Di Carlo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Bioengineering, University of California, Los Angeles, California 90095, United States</div><div class="loa-info-affiliations-info">California NanoSystems Institute, University of California, Los Angeles, California 90095, United States</div><div class="loa-info-affiliations-info">Jonsson Comprehensive Cancer Center, University of California, Los Angeles, California 90095, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="mailto:dicarlo@ucla.edu">dicarlo@ucla.edu</a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dino++Di+Carlo">Dino Di Carlo</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0003-3942-4284" title="Orcid link">http://orcid.org/0000-0003-3942-4284</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="ascefj"><input type="hidden" name="journalNameForjhpLink" value="ACS Sensors"><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facssensors.9b01974&amp;href=/doi/10.1021%2Facssensors.9b01974" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>ACS Sens.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 5</span><span class="cit-issue">, 2</span><span class="cit-pageRange">, 385–394</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 6, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 October 2019</li><li><span class="item_label"><b>Accepted</b></span>6 January 2020</li><li><span class="item_label"><b>Published</b> online</span>6 January 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 February 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acssensors.9b01974" title="DOI URL">https://doi.org/10.1021/acssensors.9b01974</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D385%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DHector%2BE.%2BMu%25C3%25B1oz%252C%2BCarson%2BT.%2BRiche%252C%2BJanay%2BE.%2BKong%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D5%26issueNum%3D2%26contentID%3Dacssensors.9b01974%26title%3DFractal%2BLAMP%253A%2BLabel-Free%2BAnalysis%2Bof%2BFractal%2BPrecipitate%2Bfor%2BDigital%2BLoop-Mediated%2BIsothermal%2BNucleic%2BAcid%2BAmplification%26numPages%3D10%26pa%3D%26issn%3D2379-3694%26publisherName%3Dacs%26publication%3Dascefj%26rpt%3Dn%26endPage%3D394%26publicationDate%3DFebruary%2B2020">RIGHTS &amp; PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access"></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acssensors.9b01974"><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1399</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">15</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><input type="hidden" class="doi" value="10.1021/acssensors.9b01974"><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Fractal LAMP: Label-Free Analysis of Fractal Precipitate for Digital Loop-Mediated Isothermal Nucleic Acid Amplification&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Hector&quot;,&quot;last_name&quot;:&quot;E. Muñoz&quot;},{&quot;first_name&quot;:&quot;Carson&quot;,&quot;last_name&quot;:&quot;T. Riche&quot;},{&quot;first_name&quot;:&quot;Janay&quot;,&quot;last_name&quot;:&quot;E. Kong&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;van Zee&quot;},{&quot;first_name&quot;:&quot;Omai&quot;,&quot;last_name&quot;:&quot;B. Garner&quot;},{&quot;first_name&quot;:&quot;Aydogan&quot;,&quot;last_name&quot;:&quot;Ozcan&quot;},{&quot;first_name&quot;:&quot;Dino&quot;,&quot;last_name&quot;:&quot;Di Carlo&quot;}],&quot;source&quot;:&quot;ACS Sensors&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;06&quot;,&quot;issue&quot;:&quot;2&quot;,&quot;volume&quot;:&quot;5&quot;,&quot;pages&quot;:&quot;385-394&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acssensors.9b01974&quot;},&quot;abstract&quot;:&quot;Nucleic acid amplification assays including loop-mediated isothermal amplification (LAMP) are routinely used in diagnosing diseases and monitoring water and food quality. The results of amplification in these assays are commonly measured with an analog fluorescence readout, which requires specialized optical equipment and can lack quantitative precision. Digital analysis of amplification in small fluid compartments based on exceeding a threshold fluorescence level can enhance the quantitative precision of nucleic acid assays (i.e., digital nucleic acid amplification assays), but still requires specialized optical systems for fluorescence readout and the inclusion of a fluorescent dye. Here, we report Fractal LAMP, an automated method to detect amplified DNA in subnanoliter scale droplets following LAMP in a label-free manner. Our computer vision algorithm achieves high accuracy detecting DNA amplification in droplets by identifying LAMP byproducts that form fractal structures observable in brightfield microsc&quot;}" class="meta-data"><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"><input type="hidden" name="pub-access" value="full" class="pub-access"><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"><input type="hidden" name="selected-share-option" class="selected-share-option"></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facssensors.9b01974&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acssensors.9b01974" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facssensors.9b01974&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acssensors.9b01974" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facssensors.9b01974&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acssensors.9b01974" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facssensors.9b01974&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acssensors.9b01974&amp;href=/doi/10.1021/acssensors.9b01974" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acssensors.9b01974"><a href="/doi/pdf/10.1021/acssensors.9b01974" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acssensors.9b01974&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fid%3Ddoi%3A10.1021%252Facssensors.9b01974%26sid%3Dliteratum%253Aachs%26pmid%3D31902202%26genre%3Darticle%26aulast%3DMu%25C3%25B1oz%26date%3D2020%26atitle%3DFractal%2BLAMP%253A%2BLabel-Free%2BAnalysis%2Bof%2BFractal%2BPrecipitate%2Bfor%2BDigital%2BLoop-Mediated%2BIsothermal%2BNucleic%2BAcid%2BAmplification%26jtitle%3DACS%2BSensors%26title%3DACS%2BSensors%26volume%3D5%26issue%3D2%26spage%3D385%26epage%3D394" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"><input type="hidden" name="is-enabled" value="true" class="is-enabled"><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"><!-- TODO: comment out below line:--><!--input.acs-id(type="hidden" name="acs-id" value="31838729")--><input type="hidden" name="pub-code" value="ascefj" class="pub-code"><input type="hidden" name="linkout-url" value="https://preferences.acs.org/journals" class="linkout-url"><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acs.org/ascefj?ifr=1"></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollduration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291912" title="Fluorescence">Fluorescence</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291400" title="Fluorescence imaging">Fluorescence imaging</a>,</li><li><a href="/action/doSearch?ConceptID=292492" title="Liquids">Liquids</a>,</li><li><a href="/action/doSearch?ConceptID=291685" title="Dyes and pigments">Dyes and pigments</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/ascefj/5/2" title="Go to ACS Sensors "><img src="/cms/10.1021/ascefj.2020.5.issue-2/asset/ascefj.2020.5.issue-2.largecover.jpg" alt="Go to ACS Sensors "></a></div></div>




        
        <!--Article Page E-Alerts-->
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','ascefj');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','ascefj');
	  }
  });
</script>
<!--End Article Script-->




        
        <div class="pj-badge"><input type="hidden" id="partnerJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json"></div>




        
        <div class="aJhp_link"></div>
<input type="hidden" id="aJhpJournalInfo" value="/pb-assets/json/journal-info-1641590078917.json">
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=ascefj" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/cms/10.1021/acssensors.9b01974/asset/images/medium/se9b01974_0004.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acssensors.9b01974&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Nucleic acid amplification assays including loop-mediated isothermal amplification (LAMP) are routinely used in diagnosing diseases and monitoring water and food quality. The results of amplification in these assays are commonly measured with an analog fluorescence readout, which requires specialized optical equipment and can lack quantitative precision. Digital analysis of amplification in small fluid compartments based on exceeding a threshold fluorescence level can enhance the quantitative precision of nucleic acid assays (i.e., digital nucleic acid amplification assays), but still requires specialized optical systems for fluorescence readout and the inclusion of a fluorescent dye. Here, we report Fractal LAMP, an automated method to detect amplified DNA in subnanoliter scale droplets following LAMP in a label-free manner. Our computer vision algorithm achieves high accuracy detecting DNA amplification in droplets by identifying LAMP byproducts that form fractal structures observable in brightfield microscopy. The capabilities of Fractal LAMP are further realized by developing a Bayesian model to estimate DNA concentrations for unknown samples and a bootstrapping method to estimate the number of droplets required to achieve target limits of detection. This digital, label-free assay has the potential to lower reagent and reader cost for nucleic acid measurement while maintaining high quantitative accuracy over 3 orders of magnitude of concentration.</p></div></div></div></div><div class="article_keywords hlFld-Abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_keywords-container"><span>KEYWORDS:</span><ul class="rlist--inline"><li><a class="keyword" href="https://pubs.acs.org/action/doSearch?action=search&amp;AllField=Fractal+LAMP&amp;qsSearchArea=AllField">fractal LAMP</a></li><li> <a class="keyword" href="https://pubs.acs.org/action/doSearch?action=search&amp;AllField=Digital+Analysis&amp;qsSearchArea=AllField">digital analysis</a></li><li> <a class="keyword" href="https://pubs.acs.org/action/doSearch?action=search&amp;AllField=Nucleic+Acid+Amplification+Test&amp;qsSearchArea=AllField">nucleic acid amplification test</a></li><li> <a class="keyword" href="https://pubs.acs.org/action/doSearch?action=search&amp;AllField=Bayesian+Model&amp;qsSearchArea=AllField">Bayesian model</a></li><li> <a class="keyword" href="https://pubs.acs.org/action/doSearch?action=search&amp;AllField=Bootstrapping&amp;qsSearchArea=AllField">bootstrapping</a></li></ul></div></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="NLM_p first">The detection of nucleic acids is widely used in the diagnosis of disease, including identifying mutations in tumors <a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> or microbial and viral pathogens obtained from patient samples. <a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Nucleic acid detection has also become vital in assessing food and water quality by detecting pathogens, allergens, or genetic identity of food. <a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Ongoing research is aimed at detecting nucleic acids, such as DNA and RNA, at lower concentrations, more cheaply, or in a more portable, accessible manner. <a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">Innovations in nucleic analysis include approaches to isolate and amplify target DNA, quantify the amount of target, as well as instrumentation to detect the isolated/amplified targets. When target DNA is present at sufficiently low concentrations, it must first be amplified by any number of methods including polymerase chain reaction (PCR), <a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> rolling circle amplification, <a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> or loop-mediated isothermal amplification (LAMP). <a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Once the target DNA has been sufficiently amplified, its presence is often measured using fluorescent dyes, whose intensity is correlated to the amount of DNA present, or through electrical signals. <a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> Improved quantification and limits of detection can be achieved by partitioning the sample into microcompartments, each independently amplified and read, <a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> i.e., digital nucleic acid amplification. Finally, readout instruments that leverage low-cost consumer electronic devices can lower costs and increase the portability of these assays. <a onclick="showRef(event, 'ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15">(13−15)</a></div><div class="NLM_p">LAMP is an attractive method for amplifying DNA for its thermal simplicity and speed. Unlike some other DNA amplification methods, LAMP proceeds at a single temperature, simplifying the equipment needed to run it. It can also create a large quantity of amplified DNA on short timescales, with more than 50 times more product than PCR at comparable scales. <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> While many DNA amplification tests use fluorescent dyes directly on the main product, LAMP progress can be assessed by one of its byproducts. During the LAMP reaction, pyrophosphate is formed as a byproduct and can precipitate with magnesium ions in solution to form magnesium pyrophosphate (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>a), increasing solution turbidity as a function of DNA amplification. <a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> Solution turbidity has been shown to be a valid indicator for DNA amplification with LAMP and possesses particular advantages. In contrast, one class of fluorescent dyes, intercalating dyes, which are usually incorporated into reaction solutions prior to reaction, has been shown to interfere with DNA reaction kinetics, <a onclick="showRef(event, 'ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22">(19−22)</a> although there have been advances to counter this effect. <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Previous work also showed that intercalating dyes can be variable and heat sensitive. <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Other advantages of using intrinsic reaction products are that nonfluorescence measures such as turbidity in LAMP can avoid the need for extra dyes, fluorescence excitation and reading equipment, and improve amplification times.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acssensors.9b01974/asset/images/medium/se9b01974_0005.gif" alt="" data-lg-src="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0005.jpeg" id="gr1" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Fractal LAMP: Detection of DNA via LAMP byproducts in droplets imaged using brightfield microscopy. (a) During DNA amplification by DNA polymerase, pyrophosphate (blue) is cleaved from the deoxynucleotide (dNTP) as a byproduct. Pyrophosphate reacts with magnesium ions in the LAMP solution mix and precipitates at sufficiently high concentrations. (b) DNA amplification can be assessed via fluorescent dyes, or by turbidity in the case of LAMP. In bulk solution, the fluorescence or turbidity increases with DNA concentration but may be difficult to discern at low concentrations or across small differences in concentration. Compartmentalization of volume creates a digital readout, where fractal-like precipitate or fluorescence is not diluted by a large volume. The percentage of negative compartments are directly correlated to compartment size and the original bulk concentration. (c) LAMP reaction mix and DNA are compartmentalized with a three-dimensional (3D) printed droplet generator and are incubated at 67 °C for 2 h before imaging. (d) After incubation, the droplets are transferred to a 120 μm slit chamber for brightfield and fluorescence imaging. Magnesium pyrophosphate precipitate is visible in brightfield and corresponds to droplets with increased fluorescent intercalating dye signal.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acssensors.9b01974&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Digitization of DNA amplification provides improved limits of detection and quantitative accuracy. DNA amplification has traditionally been carried out and measured in bulk solution. In that case, DNA content is correlated to a single analog measure describing the amplification in the bulk fluid. In a digital assay, the bulk solution is partitioned into many small volumes, such as wells or aqueous droplets. <a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Compartmentalization binarizes the initial readout, where the analog signal in each compartment can be thresholded to yield either a positive or negative event, which at sufficiently low DNA counts can be related to a single starting DNA molecule. Finally, the percentage of negative compartments is directly correlated to the original bulk concentration. Previous work has demonstrated this digital assay concept using LAMP whereby fluorescent intercalating dyes led to amplified compartment-specific fluorescent signals. <a onclick="showRef(event, 'ref23 ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26 ref27">(23−27)</a> The use of the magnesium pyrophosphate product of LAMP (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>b) has not been used to our knowledge to detect amplified signals in a digital format.</div><div class="NLM_p last">Here, we report Fractal LAMP, a droplet-based method to perform label-free, digital DNA quantification. This is achieved by encapsulating target DNA samples and the LAMP reaction mix in thousands of subnanoliter scale droplets. Following the reaction, droplets with target DNA molecules accumulate magnesium pyrophosphate precipitate inside. This coalesced precipitate, with a unique fractal-like structure, is tens of microns in size and is clearly visible using brightfield imaging. We detect the precipitate with relatively low magnification brightfield microscopic imaging followed by image analysis with a customized computer vision algorithm. Utilizing partitioning statistics, the percentage of droplets containing precipitate is correlated with the original bulk DNA concentration, enabling automated detection of precipitate droplets and calculation of bulk DNA concentration. Combined with the reduced instrument complexity required for isothermal LAMP amplification, this new approach to digital detection of amplification should further reduce final instrument complexity and cost by employing cost-effective portable brightfield microscopy systems. <a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37206" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37206" class="dropBlock__holder"></div></div></div></div></div><hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Overview</h3><div class="NLM_p">We have implemented a label-free, brightfield digital DNA amplification detection method using droplets. A droplet-based approach is preferred for Fractal LAMP given that precipitate observed in wells after performing LAMP is not concentrated and may variably adhere to well walls (Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S1</a>). We hypothesize that performing Fractal LAMP in droplets promotes the accumulation of precipitate at a single point, due to droplet shape and reduced volume, further facilitating imaging and image analysis. Target DNA solution is combined with LAMP reaction mix and primers, and introduced into droplets using a 3D printed droplet generator. Droplets are heated to initiate the LAMP reaction and accumulate reaction products. The precipitate that accumulates within droplets that amplify target DNA is imaged and analyzed and compared to the standard intercalating dye-based readout. We characterize how precipitate correlates with fluorescence at the single droplet level and how the number of droplets with precipitate correlates with overall differences in DNA target concentration. Finally, we develop image analysis tools based on a Bag of Visual Words (BOVW) <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> to identify precipitate in droplets in an automated fashion and report on our automated label-free results for predicting target DNA concentration.</div><div class="NLM_p last">Solutions of the LAMP reaction mix, fluorescent nucleic acid intercalating dye, and target bacteriophage DNA were encapsulated in droplets via a 3D droplet generator <a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>c) and incubated at 67 °C for 2 h. After incubation, the droplets were transferred to a 120 μm slit chamber for brightfield and fluorescence imaging at 10× magnification (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>d). Multiple fields of view across multiple sample chambers were imaged for each DNA concentration (5.7 × 10<sup>1</sup> copies DNA μL<sup>–1</sup>, 5.7 × 10<sup>2</sup> copies DNA μL<sup>–1</sup>, 5.7 × 10<sup>3</sup> copies DNA μL<sup>–1</sup>, 5.7 × 10<sup>4</sup> copies DNA μL<sup>–1</sup>, and 5.7 × 10<sup>5</sup> copies DNA μL<sup>–1</sup>). Droplets with coalesced magnesium pyrophosphate are also positive with the nucleic acid intercalating dye. The precipitate aggregates into clumps that are typically found in the center of droplets. Imaged droplets with diameters 65–96 μm (76.9 ± 6.9 μm) are detected and extracted from both brightfield and fluorescence images using a custom MATLAB script. A total of 30 138 droplet images are extracted in total, with at least 2700 droplets from each DNA concentration. Each droplet image is manually inspected to determine the presence of precipitate.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Characterization of Precipitate in Droplets</h3><div class="NLM_p">We find that the presence of magnesium pyrophosphate precipitate is correlated with nucleic acid intercalating dye fluorescence in a droplet. The presence of precipitate in a droplet is compared with the integrated fluorescence of the droplet in corresponding fluorescence images. The precipitate-positive droplets (<i>n</i> = 5390, median = 5.25, SD = 1.44), have a clear increase in fluorescence over precipitate-negative droplets (<i>n</i> = 19 399, median = 2.33, SD = 0.65) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>a). A point biserial correlation is used to determine the relationship between fluorescence and precipitate presence. There is a positive correlation between fluorescence and precipitate, which is statistically significant (<i>r</i><sub>pb</sub> = 0.81, <i>n</i> = 24 789, <i>p</i> &lt; 0.001).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acssensors.9b01974/asset/images/medium/se9b01974_0001.gif" alt="" data-lg-src="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0001.jpeg" id="gr2" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Presence of precipitate in LAMP droplets is correlated to the integrated droplet fluorescence and the bulk DNA concentration. (a) The presence of magnesium pyrophosphate precipitate in droplets where LAMP was performed is correlated to integrated droplet fluorescence. Images where droplets moved between brightfield and fluorescence imaging or with fluorescence aberrations are excluded. (b) The percentage of droplets containing precipitate increases as the target DNA concentration in a sample increases. (c) The percentage of droplets without precipitate is used to calculate λ (blue), the average number of target DNA molecules per droplet, which is linearly related to the original DNA concentration (λ = <i>ω</i><sup>–1</sup>·Conc<sub>DNA</sub>·Vol<sub>drop</sub>). Bayesian inference yields a median inefficiency factor ω of 127.14 and confidence interval (gray band, 99% highest posterior density (HPD), indicating that there is an ∼127× inefficiency factor when performing LAMP in droplets. Inference also yields 99% HPD intervals (black) for each sample’s λ estimate that are drawn back to linearity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acssensors.9b01974&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Precipitate presence in droplets is correlated to bulk target DNA concentration. As bulk DNA concentration increases, the percentage of empty droplets decreases (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>b). Partitioning of original target DNA in droplets is governed by the binomial and Poisson distributions. <a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> As a result, the average number of target DNA molecules per droplet, λ is defined as λ = Conc<sub>DNA</sub>·Vol<sub>drop</sub>, where Conc<sub>DNA</sub> is the bulk DNA concentration and Vol<sub>drop</sub> is the droplet volume. λ can also be related to the percentage of empty droplets (<i>E</i>), through the relationship λ = −ln(<i>E</i>). The extent to which our observed λ values deviate from the ideal λ for each DNA concentration is defined as the inefficiency factor ω (λ<sub>ideal</sub> = ω·λ<sub>observed</sub>) and is determined with hierarchical Bayesian inference. <a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The posterior of ω (median = 127.14, 99% CI [121.81, 132.65]) indicates that there is an ∼127× inefficiency with either partitioning the DNA molecules, or the LAMP reaction (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>c). Nonetheless, there is a clear linear relationship between our measured λ values, based on counting droplets positive for precipitate and DNA concentration. Despite deviations in observed λ from the regression, moderately informed priors in the Bayesian inference can produce 99% highest posterior density (HPD) intervals that intersect the regression confidence band.</div><div class="NLM_p last">When performing the same Bayesian inference using thresholded fluorescence values instead of precipitate presence (Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S2</a>) ω is larger and has a wider posterior (median = 204.48, 99% CI [195.09, 214.47]). Improved efficiency for reading precipitate presence vs fluorescence can be attributed to the loss of some fluorescent droplets that are removed due to fluorescence photobleaching or droplet movement. Additionally, the fluorescence threshold used is determined by finding a local minimum in the intensity distribution, which may be imprecise or difficult to determine with an imbalanced distribution.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Automated Classification of Precipitate Images</h3><div class="NLM_p">Automated classification of droplet images is enabled by image processing. To process and analyze droplet images for the presence of precipitate, some form of image processing is required. Detecting the precipitate is a challenge because although the fractal-like precipitate structures look similar, they vary in their size, morphology, location, and sharpness. A simple approach to droplet classification entails generating intensity-based statistics for each image, including mean and median intensity, sharpness, and measuring the result of automated thresholding. This approach is useful but does not account for differences in lighting, or small or out-of-focus precipitates. Convolutional neural nets are another viable approach to automated classification. <a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> This solution would likely be able to recognize the fractal subunits of the precipitate in a few different presentations. However, there is a degree of reduction of the interpretability of neural nets, which would make it more difficult to understand why an image was misclassified. In this paper, we have chosen to implement the previously mentioned BOVW algorithm, which builds a dictionary of image patches (visual words) and then matches and counts the occurrence of image patches in each image. <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Given a dictionary of a specified size, a feature vector of word counts is generated for each image, which can be utilized by a machine learning algorithm for automated classification of droplets. This method excels by recognizing a set of visually similar image patches, such as the fractal subunits, which appear a variable number of times per image. We are able to diagnose the BOVW model by seeing which image patches are extracted from a droplet image and whether they are indicative of precipitate-positive or negative droplets. The entirety of the 30 138 extracted images is randomly assigned to Training, Validation, and Testing sets according to a 64/16/20 split.</div><div class="NLM_p">We build our BOVW model by automatically extracting Speeded Up Robust Features (SURF) keypoints <a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> from regions of high contrast (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>a). SURF keypoint extraction is scale and rotation invariant, yielding the keypoints with the highest contrast in a small region. Each SURF keypoint is then described by dividing the region around the keypoint into a 4 × 4 grid, where four horizontal and vertical wavelet responses are summed in each cell, resulting in 64-dimensional feature space for each keypoint. Additionally, the sign of Laplacian (SoL), which indicates if the keypoint represents a dark blob on a light background (positive) or a light blob on dark background (negative) is also calculated. SURF keypoints from the Training set droplets are clustered based on their 64 features into a predefined number of words to form the BOVW dictionary (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>b). SURF keypoints with positive or negative SoL are clustered separately to form Dark and Light words, respectively (Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S3</a>). Additionally, SURF keypoints can be filtered by a metric representing their contrast. Low-contrast features are likely image artifacts, but there are potential benefits to creating low-contrast words that do not dilute the predictive power of high-contrast keypoints with a similar appearance. Therefore, we also generate a set of low-contrast (Fuzzy) visual words that map to the same 64-dimensional feature space as corresponding high-contrast (Sharp) visual words. The number of SURF keypoints extracted from precipitate-negative and precipitate-positive droplets is Poisson-distributed, with an average of 0.4 and 15.3 keypoints, respectively. The traditional BOVW model can be further extended by generating meta-features, such as the number of total words per image, number of Sharp words, likely-precipitate words (based on the premise that precipitate is normally found near the center of the droplet), in addition to image statistics.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acssensors.9b01974/asset/images/medium/se9b01974_0002.gif" alt="" data-lg-src="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0002.jpeg" id="gr3" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Creation of Bag of Visual Words model from SURF keypoints. (a) SURF keypoints, highlighted in colored boxes, are extracted from previously isolated images of droplets. SURF keypoints are scale and rotation invariant and are described with a 64-dimensional feature space. (b) The BOVW model is created by clustering SURF keypoints within their feature space to define words. Dark and Light words are clustered separately, according to the positive and negative Sign of Laplacian, respectively. SURF keypoints closest to the cluster centers are shown for some word examples. Words are further divided into Sharp and Fuzzy words, which overlap in the same feature space, but have different levels of contrast. (c) Example images of droplets with and without precipitate are processed in a BOVW model containing 32 words, with eight words each of Dark/Sharp, Dark/Fuzzy, Light/Sharp, and Light/Fuzzy. The frequency of each type of word appearing in the images (circles), is compared to mean frequencies in the entire dataset (bars).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acssensors.9b01974&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Several competing BOVW models are created that vary in the dictionary size, inclusion of low-contrast keypoints, contrast cutoffs, meta-features, or image statistics. Each of these models is trained using a random forest classifier <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> using the set of Training droplet images and compared against the Validation set. The ideal model has a high precision-recall Area Under the Curve (AUC) and receiver operating characteristic curve (ROC) AUC. The base-selected BOVW model maximizes both measures and features 8 Dark words (and corresponding Fuzzy), 8 Light words (and corresponding Fuzzy) for a total of 32 words (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>c, Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S4</a>). This model is augmented with BOVW meta-features and image statistics features. Feature importance calculation reveals that the tendency of a visual word to be found in the middle of the droplet (“Likely Precip Words”), droplet image sharpness (99th percentile of the Laplacian of Gaussian), and total number of visual words in a droplet are most useful in classifying precipitate-positive droplets (see Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S5</a> for the list of features, and relative importance via permutation importance <a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a>).</div><div class="NLM_p">The augmented BOVW model achieves high levels of precision (positive predictive value) and recall (sensitivity) on the droplet Testing set. The default setting of the random forest classifier yields a precision of 0.988 and a recall of 0.982 (Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S6</a>). The random forest classifier yields prediction scores of whether a droplet is a precipitate-positive droplet, which is shown for a set of droplets from the Training and Testing sets (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>a, see Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S7</a> for representative droplet images across the range of prediction scores). Note that blurry/smaller precipitate with a lower prediction score is still classifiable.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acssensors.9b01974/asset/images/medium/se9b01974_0003.gif" alt="" data-lg-src="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0003.jpeg" id="gr4" class="inline-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Classification of droplets predicts Testing set λ and DNA concentration. (a) Random forest classification of droplets. Testing set droplets indicated with black circle markers, all others are from Training and Validation sets. Color on outlining circles indicates prediction score of a precipitate-positive droplet. The threshold to classify a droplet as precipitate-positive can be adjusted but is 0.5 by default. (b) λ Values for droplet predictions (blue, horizontal) and ground truth (yellow, horizontal) are computed as before (with 0.5 threshold) and related to bulk DNA concentration. The predicted λs with this classifier score threshold are consistently higher than the ground truth, indicating a systematic propensity to classify false positives, and limitations of low droplet count. Bayesian estimates of the λ HPD intervals (vertical) illustrate the ability to recover valid ranges of λ values for unknown DNA concentration but can be susceptible to false positives at low concentrations. (c) Bootstrapping across a range of λs and droplet sample sizes enables the calculation of the minimum number of droplets needed to distinguish a sample from 0 DNA at 95 or 99% confidence levels (Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S8</a>, see Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Note 1</a> for more detail).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acssensors.9b01974&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">At the default random forest setting, the classifier produces estimates for the average number of DNA molecules, λ, per sample that closely approaches the known values. There is a higher deviation from the known λ at the lowest concentration (on a log scale), due to a tendency towards false positives, as seen in the precision (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>b, horizontal lines). Using the previously estimated inefficiency factor, ω, new Bayesian models generate posterior λ values for the Testing set ground truth and machine learning results as if the DNA concentration is unknown. The HPD intervals of the ground truth can recover linearity, but the HPD intervals of the machine learning results of the lowest DNA concentration are affected by false positives. This overestimation in λ would result in an overestimation of predicted bulk DNA concentrations. Overestimation for low numbers of precipitate-positive droplets could be mitigated by training a classifier that includes multiple classes, including one for droplets that may contain debris or image artifacts, which contribute to false positives.</div><div class="NLM_p last">Limits of detection are determined as a function of a number of droplets and machine learning performance and can approach one copy per microliter with a million droplets. The average DNA per droplet (λ) for a given Poisson distribution can be more precisely estimated with a larger number of sampled droplets (Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S8a</a>). With the previously established linear relationship between λ and DNA concentration, we generate curves of required droplets to distinguish a sample from 0 DNA at 95 and 99% confidence levels (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure </a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>c, see Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Note 1</a> for more detail). In this study, 9185 droplets are imaged for the lowest DNA concentration (57 copies DNA μL<sup>–1</sup>), which is less than the 15 617 droplets needed to differentiate from a 0 DNA sample with 95% confidence level. Distinguishing a sample with 11.3 copies DNA μL<sup>–1</sup> would require 83 000 and 138 000 droplets to achieve 95 and 99% confidence levels, respectively. On average, approximately two times as many droplets are required to move from 95 to 99% confidence levels. Achieving this level of performance can be made more feasible by improving the machine learning model, or decreasing the droplet size, which would speed up imaging.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42186" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42186" class="dropBlock__holder"></div></div></div></div></div><hr><div class="NLM_p">In this paper, we have presented Fractal LAMP, a method to perform label-free DNA detection and quantification using a digital assay. Performing LAMP in droplets allows us to detect DNA amplification via magnesium pyrophosphate precipitate accumulation. This precipitate is visible in droplets in brightfield and is correlated to overall DNA intercalating dye fluorescence. We were able to develop an automated computer vision algorithm utilizing BOVW to accurately detect the presence of precipitate that varied in the presentation. The practical limit of detection is affected by the performance of our computer vision algorithm, as well as the number of droplets imaged; however, high quantitative accuracy in predicting DNA concentration is achieved over at least 3 orders of magnitude.</div><div class="NLM_p">Digital, label-free LAMP is an attractive method to detect low concentrations of DNA and quantify the amount of target DNA in a sample. A label-free approach eliminates the need for extra optical equipment and dyes to generate and read fluorescence. Brightfield microscopic imaging systems can be more cost-effective, with a larger field of view, and a longer depth of focus than fluorescence systems. <a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> Furthermore, by only requiring brightfield imaging, we can image droplets on cell phone imaging platforms, which are more portable than conventional microscopes. In addition, common intercalating dyes have been shown to delay the speed of DNA amplification and may be unstable. <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> We have previously observed the instability of EvaGreen fluorescence over time, with changing temperatures, and witnessed photobleaching in this paper. Independence from fluorescence dyes also obviates the need to determine fluorescence value cutoffs, as the presence or absence of precipitate is a clear binary state.</div><div class="NLM_p">Droplets are an advantageous method to partition bulk solution for digital assays. Whether using serial generation with 3D printed devices, <a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> or massively parallelized implementations, <a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> droplet generators can quickly partition bulk solution into monodisperse volumes. Engineered carrier microparticles can even be used to emulsify a bulk solution into controlled volumes without the need for microfluidics or droplet generators for the end user. The imaging of static droplets in this paper can be accelerated using wide-field brightfield imagers based on consumer electronic devices, which have been previously demonstrated by us <a onclick="showRef(event, 'ref35 ref36 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref39 ref40">(35,36,39,40)</a> and others and may be compatible with this system.</div><div class="NLM_p">While this paper imaged static droplets, there is the potential for droplets to also be imaged in a continuous flow, enabling high-throughput potential, especially if detecting precipitate presence without the need for fluorescence analysis, which has been previously required. <a onclick="showRef(event, 'ref27 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref27 ref41 ref42">(27,41,42)</a> The recent work by Yelleswarapu et al. demonstrates multichannel high-throughput imaging of droplets in flow. Our droplets and fractal precipitates are likely compatible with this imaging setup and eliminate the need for fluorescence readout, which requires longer exposure time. In combination with the system detailed in Yelleswarapu et al., extremely intense illumination and submicrosecond exposure times could potentially eliminate the need for streak imaging. We anticipate measurement in flow is also possible using high-speed cameras that intrinsically have high sensitivity and short exposure times. For example, we have used high-speed cameras with frame rates of hundreds of thousands of frames per second in previous work analyzing the deformation of cells. <a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> For droplets, we expect that the internal motion of precipitates in moving droplets in a channel could be frozen in space at sufficiently high frame rates, and therefore analyzed without motion blur. The localization of precipitate at the bottom of the droplet may be disturbed in this case unless the clever design of the channels is used to minimize disturbing recirculating flows in droplets, which would be a concern for any method of imaging in flow. Nevertheless, imaging droplets in flow would be an exciting next step for analysis of precipitate-producing reactions in droplets, reducing practical limits of detection that our current imaging imposes.</div><div class="NLM_p">Our computer vision algorithm excelled at detecting the magnesium pyrophosphate fractal precipitate in droplets. This precipitate varied in both size, shape, and focus, but with sufficiently high numbers of example images, our algorithm was able to learn to distinguish precipitate from debris or small incorporated oil drops. Precipitate detection from brightfield images could be improved by restricting the imaging chamber height, to reduce out-of-focus precipitate, and collecting more example images. Alternative algorithms such as convolutional neural nets <a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> may also provide a boost in performance that allows us to achieve lower limits of detection while imaging fewer droplets. Imaging and computer vision algorithms may not be needed entirely if digital LAMP shows enhanced turbidity or light scattering for individual drops, and precise fiber optics can be aligned to detect alterations in light transmission.</div><div class="NLM_p last">There are several important future directions that can further enhance Fractal LAMP as a label-free, digital assay. To start, there are some limitations to the amplification reporting signals. Neither the intercalating dyes nor magnesium pyrophosphate precipitate generated is sequence-specific, which is a general limitation of LAMP. They can report that the amplification occurred, but not guarantee which DNA was amplified. That will rely on the DNA primers, which are also susceptible to self-amplification. Using brightfield imaging alone, we also cannot chemically confirm that the identified particles are in fact magnesium pyrophosphate precipitate, which at low quantities can resemble debris. We have not investigated the long-term stability of the precipitate in droplets, but it is expected that precipitate will remain stable over longer periods of time than intercalating dyes, which can photobleach. Performing Fractal LAMP without any intercalating dyes should also reduce reaction time. This hints at the exciting possibility to collect and run Fractal LAMP continuously with low-cost instrumentation but analyze results of several experiments at a future date at a central location with higher complexity instrumentation. If we maintain the use of images to observe precipitate, we can use more robust methods such as wide-field imaging or holographic imaging augmented by neural networks, which would allow detecting precipitate on more planes of focus <a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> at extreme throughputs, leading to lower limits of detection and higher quantitative accuracy.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83047" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83047" class="dropBlock__holder"></div></div></div></div></div><hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> LAMP/DNA Mix</h3><div class="NLM_p last">The LAMP reaction was performed as described in previous work, <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> with a few key changes. The polymerase concentration was doubled, and the EvaGreen (Biotium) concentration used was 2.5 μM. Studies were performed by serially diluting bacteriophage lambda DNA (Thermo Fisher) to yield concentrations of 5.7 × 10<sup>1</sup> copies DNA μL<sup>–1</sup>, 5.7 × 10<sup>2</sup> copies DNA μL<sup>–1</sup>, 5.7 × 10<sup>3</sup> copies DNA μL<sup>–1</sup>, 5.7 × 10<sup>4</sup> copies DNA μL<sup>–1</sup>, and 5.7 × 10<sup>5</sup> copies DNA μL<sup>–1</sup>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Droplet Formation and Incubation</h3><div class="NLM_p last">The prepared LAMP solution and DNA were coinjected into a microfluidic 3D droplet generator <a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> with FC-40 and RAN fluorosurfactant to produce 0.56 ± 0.1 nL (diameter: 102.03 ± 6.02 μm) droplets. Generating 1000 droplets took approximately 1.12 s. The emulsified samples were transferred from the droplet generator via tubing into 0.5 mL microcentrifuge tubes and incubated at 67 °C for 2 h, followed by 2 min at 80 °C.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Droplet Imaging</h3><div class="NLM_p last">After incubation, the droplets were transferred via pipette to a 120 μm slit chamber for brightfield and fluorescence (fluorescein isothiocyanate—FITC [535/45 nm]) imaging at 10× magnification. Images of several fields of view are captured for each sample chamber. Imaging 1000 took approximately 200 s.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Image Processing</h3><div class="NLM_p last">Brightfield microscopy images are processed by a MATLAB script that identifies droplets (<i>n</i> = 30 138, captured over 31 imaging sessions on the same day) with diameters 65–96 μm using a circle Hough Transform. <a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Brightfield images of droplets and corresponding fluorescence images are cropped and resized for each detected droplet. Fluorescence intensity is calculated by integrating total fluorescence intensity of the image. When comparing droplet fluorescence values, 7% of the field of views across all DNA concentrations are discarded due to droplet movement between acquiring brightfield and fluorescence images, photobleaching of the intercalating dye, or inconsistent illumination across the fluorescence images. This results in an 18% reduction to the total number of droplets considered (<i>n</i> = 24 789) in our comparative analysis.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Dataset Creation</h3><div class="NLM_p last">All droplet images are manually inspected and marked for the presence of precipitate. Droplet images are divided into Training, Validation, and Testing sets following a 64/16/20 split. Competing models are trained on the Training set and compared on the Validation set. Finally, the most performant model is run with the Testing set.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Bayesian Regression</h3><div class="NLM_p last">Bayesian inference is conducted with PyMC3 <a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and No U-Turn MCMC Sampler <a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> to perform regression and calculate an inefficiency factor (slope) from which the observed λ deviates from the ideal λ. This is performed by creating a hierarchical Bayesian model where the likelihood for each <i>j</i>th DNA concentration is a binomial distribution (∑<sub class="stack"><i>i</i></sub><sup class="stack"><i>N</i></sup>1{<i>X</i><sub><i>i</i></sub> &gt; 0}) = <i>n</i><sub><i>j</i></sub> ∼ Binomial (<i>N</i><sub><i>j</i></sub>,θ<sub><i>j</i></sub>) based on the number of precipitate-positive droplets <i>n</i><sub><i>j</i></sub> in its Training set <i>N</i><sub><i>j</i></sub>, and θ<sub><i>j</i></sub> = 1 – e<sup><i>–</i>λ<sub><i>j</i></sub></sup>. λ<sub><i>j</i></sub> is distributed as a product of a gamma prior and inefficiency factor, ω, which is shared across all DNA concentrations (λ<sub><i>j</i></sub> ∼ gamma (α<sub><i>j</i></sub>, β)1/ω). ω has a log-normal prior (ω ∼ LogNormal (μ = 0, σ = 4)), which is centered at 1, with 0.01 and 100 being equally as likely. The gamma prior for each concentration is moderately informed by its ideal λ value, and the confidence we would have after measuring this λ value from 5000 droplets (α<sub><i>j</i></sub> = λ<sub><i>j</i>,&nbsp;ideal</sub>·5000; β = 5000). Three chains of 2000 steps and 2000 discarded tuning steps are used by the sampler (Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S9</a>).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Bag of Visual Words Creation</h3><div class="NLM_p last">A BOVW model is constructed from Training brightfield droplet images. Image patches of interest are identified via SURF keypoint extraction, which yields image patch centroids, orientation, and 64 wavelet responses. Visual words are created by clustering 64-dimensional wavelet features. SURF keypoints with negative (Light) and positive SoL (Dark) are clustered separately. SURF keypoints are further divided according to a sharpness metric (Fuzzy/Sharp), yielding four sets of visual words with eight words each. The frequency of visual words appearing in droplet images is used to construct features used to classify images for the presence of precipitate.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Random Forest Classification</h3><div class="NLM_p last">BOVW features are combined with meta-features and image statistics to create a final feature set. A random forest classifier uses this feature set to identify precipitate-positive droplets. Feature generation and classification per 1000 droplets took approximately 13 s on a standard computer.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Bayesian Inference of Testing Set</h3><div class="NLM_p last">Two Bayesian models are constructed similarly to the previous model used on the Training set using either the Testing ground truth or the predicted machine learning results. The posterior of the previously estimated ω is used instead of the relatively uninformed prior used before. In this case, we want to evaluate the models if the actual DNA concentration is unknown, so an uninformed gamma prior is used where α = 0.001, β = 0.001 (Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S10</a>).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Limit of Detection Calculation</h3><div class="NLM_p last">Repeated sampling of a specified number of droplets from a Poisson distribution can be used to generate a confidence interval for the estimate of λ. True positive and false positive rates from the machine learning algorithm are applied to simulated repeat sampling of droplets to generate confidence intervals for estimates of λ (Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S8b</a>). Confidence intervals for a given λ across a range of droplet sample sizes are compared to confidence intervals for λ = 0 (DNA concentration = 0). The point at which the confidence intervals for the two λs no longer overlap, tells us the minimum number of droplets needed to distinguish a sample from a 0 DNA condition (Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S8c,d</a>, see Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Note 1</a> for more detail).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68550" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68550" class="dropBlock__holder"></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acssensors.9b01974?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acssensors.9b01974</a>.</p><ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">LAMP precipitate in microwells, fluorescence-thresholded results, visualization of image patch clustering, examples of visual words, importance of machine learning features, machine learning model comparison, examples of ranges of machine learning prediction scores, demonstration of determining minimum droplets required, illustration of Bayesian model for Training cases, illustration of Bayesian model for Testing cases, and an explanation of statistical methods used to determine minimum droplets required in digital assays (<a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf">se9b01974_si_001.pdf (1.34 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acssensors.9b01974" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms &amp; Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63804" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63804" class="dropBlock__holder"></div></div></div></div></div><hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dino Di Carlo</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering, University of California, Los Angeles, California 90095, United States</span>;&nbsp;
    <span class="hlFld-Affiliation affiliation">California
NanoSystems Institute, University of California, Los Angeles, California 90095, United States</span>;&nbsp;
    <span class="hlFld-Affiliation affiliation">Jonsson
Comprehensive Cancer Center, University
of California, Los Angeles, California 90095, United States</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0003-3942-4284" title="Orcid link">http://orcid.org/0000-0003-3942-4284</a>;&nbsp;
    </span><span class="author-email">
                Email: <a href="mailto:dicarlo@ucla.edu">dicarlo@ucla.edu</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hector E. Muñoz</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering, University of California, Los Angeles, California 90095, United States</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-7851-2549" title="Orcid link">http://orcid.org/0000-0001-7851-2549</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carson T. Riche</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering, University of California, Los Angeles, California 90095, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Janay E. Kong</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering, University of California, Los Angeles, California 90095, United States</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0001-5911-8116" title="Orcid link">http://orcid.org/0000-0001-5911-8116</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark van Zee</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering, University of California, Los Angeles, California 90095, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Omai B. Garner</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology &amp; Laboratory Medicine, University of California, Los
Angeles, California 90095, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aydogan Ozcan</span> - <span class="hlFld-Affiliation affiliation">Department
of Bioengineering, University of California, Los Angeles, California 90095, United States</span>;&nbsp;
    <span class="hlFld-Affiliation affiliation">Department
of Electrical and Computer Engineering, University of California, Los
Angeles, California 90095, United States</span>;&nbsp;
    <span class="hlFld-Affiliation affiliation">California
NanoSystems Institute, University of California, Los Angeles, California 90095, United States</span>;&nbsp;
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid"><a href="http://orcid.org/0000-0002-0717-683X" title="Orcid link">http://orcid.org/0000-0002-0717-683X</a></span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br></div></li></ul></div><div class="ack" id="ACK-d7e875-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33375" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33375" class="dropBlock__holder"></div></div></div></div></div><hr><p class="last">This work is supported by NSF grant 1332275.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30884" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30884" class="dropBlock__holder"></div></div></div></div></div><hr></div><p class="references-count">
            This article references 48 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tate, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamford, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubb, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sondka, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beare, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bindal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutselakis, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creatore, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, E.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">COSMIC: The Catalogue Of Somatic Mutations In Cancer</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">D941</span>– <span class="NLM_lpage">D947</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gky1015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1093%2Fnar%2Fgky1015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=30371878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7bM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=2019&amp;pages=D941-D947&amp;author=J.+G.+Tateauthor=S.+Bamfordauthor=H.+C.+Jubbauthor=Z.+Sondkaauthor=D.+M.+Beareauthor=N.+Bindalauthor=H.+Boutselakisauthor=C.+G.+Coleauthor=C.+Creatoreauthor=E.+Dawson&amp;title=COSMIC%3A+The+Catalogue+Of+Somatic+Mutations+In+Cancer&amp;doi=10.1093%2Fnar%2Fgky1015" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">COSMIC: the Catalogue Of Somatic Mutations In Cancer</span></div><div class="casAuthors">Tate, John G.; Bamford, Sally; Jubb, Harry C.; Sondka, Zbyslaw; Beare, David M.; Bindal, Nidhi; Boutselakis, Harry; Cole, Charlotte G.; Creatore, Celestino; Dawson, Elisabeth; Fish, Peter; Harsha, Bhavana; Hathaway, Charlie; Jupe, Steve C.; Kok, Chai Yin; Noble, Kate; Ponting, Laura; Ramshaw, Christopher C.; Rye, Claire E.; Speedy, Helen E.; Stefancsik, Ray; Thompson, Sam L.; Wang, Shicai; Ward, Sari; Campbell, Peter J.; Forbes, Simon A.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D941-D947</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">COSMIC, the Catalog Of Somatic Mutations In Cancer (https://cancer.sanger.ac.uk) is the most detailed and comprehensive resource for exploring the effect of somatic mutations in human cancer.  The latest release, COSMIC v86 (August 2018), includes almost 6 million coding mutations across 1.4 million tumor samples, curated from over 26,000 publications.  In addn. to coding mutations, COSMIC covers all the genetic mechanisms by which somatic mutations promote cancer, including non-coding mutations, gene fusions, copy-no. variants and drug resistance mutations.  COSMIC is primarily handcurated, ensuring quality, accuracy and descriptive data capture.  Building on our manual curation processes, we are introducing new initiatives that allow us to prioritize key genes and diseases, and to react more quickly and comprehensively to new findings in the literature.  Alongside improvements to the public website and data-download systems, new functionality in COSMIC-3D allows exploration of mutations within three-dimensional protein structures, their protein structural and functional impacts, and implications for druggability.  In parallel with COSMIC's deep and broad variant coverage, the Cancer Gene Census (CGC) describes a curated catalog of genes driving every form of human cancer.  Currently describing 719 genes, the CGC has recently introduced functional descriptions of how each gene drives disease, summarized into the 10 cancer Hallmarks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyWHUT512HAbVg90H21EOLACvtfcHk0lhqhBe26pSoWQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7bM&amp;md5=56574cedbe9adc54c097a5af5fe41eb8</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1015&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1015%26sid%3Dliteratum%253Aachs%26aulast%3DTate%26aufirst%3DJ.%2BG.%26aulast%3DBamford%26aufirst%3DS.%26aulast%3DJubb%26aufirst%3DH.%2BC.%26aulast%3DSondka%26aufirst%3DZ.%26aulast%3DBeare%26aufirst%3DD.%2BM.%26aulast%3DBindal%26aufirst%3DN.%26aulast%3DBoutselakis%26aufirst%3DH.%26aulast%3DCole%26aufirst%3DC.%2BG.%26aulast%3DCreatore%26aufirst%3DC.%26aulast%3DDawson%26aufirst%3DE.%26atitle%3DCOSMIC%253A%2520The%2520Catalogue%2520Of%2520Somatic%2520Mutations%2520In%2520Cancer%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3DD941%26epage%3DD947%26doi%3D10.1093%2Fnar%2Fgky1015" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benesova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belsanova, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suchanek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopeckova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minarikova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipska, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Visokai, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zavoral, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minarik, M.</span></span> <span> </span><span class="NLM_article-title">Mutation-Based Detection and Monitoring of Cell-Free Tumor DNA in Peripheral Blood of Cancer Patients</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>433</i></span>,  <span class="NLM_fpage">227</span>– <span class="NLM_lpage">234</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.ab.2012.06.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.ab.2012.06.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=22750103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslymsLjF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=433&amp;publication_year=2013&amp;pages=227-234&amp;author=L.+Benesovaauthor=B.+Belsanovaauthor=S.+Suchanekauthor=M.+Kopeckovaauthor=P.+Minarikovaauthor=L.+Lipskaauthor=M.+Levyauthor=V.+Visokaiauthor=M.+Zavoralauthor=M.+Minarik&amp;title=Mutation-Based+Detection+and+Monitoring+of+Cell-Free+Tumor+DNA+in+Peripheral+Blood+of+Cancer+Patients&amp;doi=10.1016%2Fj.ab.2012.06.018" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mutation-based detection and monitoring of cell-free tumor DNA in peripheral blood of cancer patients</span></div><div class="casAuthors">Benesova, L.; Belsanova, B.; Suchanek, S.; Kopeckova, M.; Minarikova, P.; Lipska, L.; Levy, M.; Visokai, V.; Zavoral, M.; Minarik, M.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">433</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">227-234</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Prognosis of solid cancers is generally more favorable if the disease is treated early and efficiently.  A key to long cancer survival is in radical surgical therapy directed at the primary tumor followed by early detection of possible progression, with swift application of subsequent therapeutic intervention reducing the risk of disease generalization.  The conventional follow-up care is based on regular observation of tumor markers in combination with computed tomog./endoscopic ultrasound/magnetic resonance/positron emission tomog. imaging to monitor potential tumor progression.  A recent development in methodologies allowing screening for a presence of cell-free DNA (cfDNA) brings a new viable tool in early detection and management of major cancers.  It is believed that cfDNA is released from tumors primarily due to necrotization, whereas the origin of nontumorous cfDNA is mostly apoptotic.  The process of cfDNA detection starts with proper collection and treatment of blood and isolation and storage of blood plasma.  The next important steps include cfDNA extn. from plasma and its detection and/or quantification.  To distinguish tumor cfDNA from nontumorous cfDNA, specific somatic DNA mutations, previously localized in the primary tumor tissue, are identified in the extd. cfDNA.  Apart from conventional mutation detection approaches, several dedicated techniques have been presented to detect low levels of cfDNA in an excess of nontumorous (nonmutated) DNA, including real-time polymerase chain reaction (PCR), BEAMing (beads, emulsion, amplification, and magnetics), and denaturing capillary electrophoresis.  Techniques to facilitate the mutant detection, such as mutant-enriched PCR and COLD-PCR (coamplification at lower denaturation temp. PCR), are also applicable.  Finally, a no. of newly developed miniaturized approaches, such as single-mol. sequencing, are promising for the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwqMGPX59UnLVg90H21EOLACvtfcHk0lhqhBe26pSoWQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhslymsLjF&amp;md5=a38a12e042e438ddf13f4bf66dd6d7a6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2012.06.018&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2012.06.018%26sid%3Dliteratum%253Aachs%26aulast%3DBenesova%26aufirst%3DL.%26aulast%3DBelsanova%26aufirst%3DB.%26aulast%3DSuchanek%26aufirst%3DS.%26aulast%3DKopeckova%26aufirst%3DM.%26aulast%3DMinarikova%26aufirst%3DP.%26aulast%3DLipska%26aufirst%3DL.%26aulast%3DLevy%26aufirst%3DM.%26aulast%3DVisokai%26aufirst%3DV.%26aulast%3DZavoral%26aufirst%3DM.%26aulast%3DMinarik%26aufirst%3DM.%26atitle%3DMutation-Based%2520Detection%2520and%2520Monitoring%2520of%2520Cell-Free%2520Tumor%2520DNA%2520in%2520Peripheral%2520Blood%2520of%2520Cancer%2520Patients%26jtitle%3DAnal.%2520Biochem.%26date%3D2013%26volume%3D433%26spage%3D227%26epage%3D234%26doi%3D10.1016%2Fj.ab.2012.06.018" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niemz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyle, D. S.</span></span> <span> </span><span class="NLM_article-title">Point-of-Care Nucleic Acid Testing for Infectious Diseases</span>. <i>Trends Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">250</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.tibtech.2011.01.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.tibtech.2011.01.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=21377748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFegt7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=2011&amp;pages=240-250&amp;author=A.+Niemzauthor=T.+M.+Fergusonauthor=D.+S.+Boyle&amp;title=Point-of-Care+Nucleic+Acid+Testing+for+Infectious+Diseases&amp;doi=10.1016%2Fj.tibtech.2011.01.007" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Point-of-care nucleic acid testing for infectious diseases</span></div><div class="casAuthors">Niemz, Angelika; Ferguson, Tanya M.; Boyle, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">240-250</span>CODEN:
                <span class="NLM_cas:coden">TRBIDM</span>;
        ISSN:<span class="NLM_cas:issn">0167-7799</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Nucleic acid testing for infectious diseases at the point of care is beginning to enter clin. practice in developed and developing countries; esp. for applications requiring fast turnaround times, and in settings where a centralized lab. approach faces limitations.  Current systems for clin. diagnostic applications are mainly PCR-based, can only be used in hospitals, and are still relatively complex and expensive.  Integrating sample prepn. with nucleic acid amplification and detection in a cost-effective, robust, and user-friendly format remains challenging.  This review describes recent tech. advances that might be able to address these limitations, with a focus on isothermal nucleic acid amplification methods.  It briefly discusses selected applications related to the diagnosis and management of tuberculosis, HIV, and perinatal and nosocomial infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonIJ_GhFP2sbVg90H21EOLACvtfcHk0lhmEUdP3fr9gw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXltFegt7w%253D&amp;md5=a5e0c6beb0e2d18a71c51f4673e0ec6a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tibtech.2011.01.007&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibtech.2011.01.007%26sid%3Dliteratum%253Aachs%26aulast%3DNiemz%26aufirst%3DA.%26aulast%3DFerguson%26aufirst%3DT.%2BM.%26aulast%3DBoyle%26aufirst%3DD.%2BS.%26atitle%3DPoint-of-Care%2520Nucleic%2520Acid%2520Testing%2520for%2520Infectious%2520Diseases%26jtitle%3DTrends%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D240%26epage%3D250%26doi%3D10.1016%2Fj.tibtech.2011.01.007" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martzy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krska, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mach, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnleitner, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reischer, G. H.</span></span> <span> </span><span class="NLM_article-title">Challenges and Perspectives in the Application of Isothermal DNA Amplification Methods for Food and Water Analysis</span>. <i>Anal. Bioanal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>411</i></span>,  <span class="NLM_fpage">1695</span>– <span class="NLM_lpage">1702</span>, <span class="refDoi">&nbsp;DOI: 10.1007/s00216-018-1553-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1007%2Fs00216-018-1553-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=30617408" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvVGhs74%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=411&amp;publication_year=2019&amp;pages=1695-1702&amp;author=R.+Martzyauthor=C.+Kolmauthor=R.+Krskaauthor=R.+L.+Machauthor=A.+H.+Farnleitnerauthor=G.+H.+Reischer&amp;title=Challenges+and+Perspectives+in+the+Application+of+Isothermal+DNA+Amplification+Methods+for+Food+and+Water+Analysis&amp;doi=10.1007%2Fs00216-018-1553-1" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges and perspectives in the application of isothermal DNA amplification methods for food and water analysis</span></div><div class="casAuthors">Martzy, Roland; Kolm, Claudia; Krska, Rudolf; Mach, Robert L.; Farnleitner, Andreas H.; Reischer, Georg H.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical and Bioanalytical Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">411</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1695-1702</span>CODEN:
                <span class="NLM_cas:coden">ABCNBP</span>;
        ISSN:<span class="NLM_cas:issn">1618-2642</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Mol. diagnostic tools in the field of food and water quality anal. are becoming increasingly widespread.  Usually, based on DNA amplification techniques such as polymerase chain reaction (PCR), these methods are highly sensitive and versatile but require well-equipped labs. and trained personnel.  To reduce anal. time and avoid expensive equipment, isothermal DNA amplification methods for detecting various target organisms have been developed.  However, to make mol. diagnostics suitable for low-resource settings and in-field applications, it is crucial to continuously adapt the working steps assocd. with DNA amplification, namely sample prepn., DNA extn., and visualization of the results.  Many novel approaches have been evaluated in recent years to tackle these challenges, e.g., the use of ionic liqs. for the rapid isolation of nucleic acids from organisms relevant for food and water anal. or the integration of entire anal. workflows on microfluidic chips.  In any event, the future of applications in the field of isothermal amplification will probably lie in ready-to-use cartridges combined with affordable handheld devices for on-site anal.  This trend article aims to make prospective users more familiar with this technol. and its potential for moving mol. diagnostics from the lab. to the field. [Figure not available: see fulltext.].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfbXyIP7Zm9LVg90H21EOLACvtfcHk0lhmEUdP3fr9gw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXlvVGhs74%253D&amp;md5=dc5f55bd5e4049d3f90c81500dc1c407</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs00216-018-1553-1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00216-018-1553-1%26sid%3Dliteratum%253Aachs%26aulast%3DMartzy%26aufirst%3DR.%26aulast%3DKolm%26aufirst%3DC.%26aulast%3DKrska%26aufirst%3DR.%26aulast%3DMach%26aufirst%3DR.%2BL.%26aulast%3DFarnleitner%26aufirst%3DA.%2BH.%26aulast%3DReischer%26aufirst%3DG.%2BH.%26atitle%3DChallenges%2520and%2520Perspectives%2520in%2520the%2520Application%2520of%2520Isothermal%2520DNA%2520Amplification%2520Methods%2520for%2520Food%2520and%2520Water%2520Analysis%26jtitle%3DAnal.%2520Bioanal.%2520Chem.%26date%3D2019%26volume%3D411%26spage%3D1695%26epage%3D1702%26doi%3D10.1007%2Fs00216-018-1553-1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span> <span> </span><span class="NLM_article-title">Point-of-Care-Testing of Nucleic Acids by Microfluidics</span>. <i>TrAC, Trends Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">106</span>– <span class="NLM_lpage">116</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.trac.2017.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.trac.2017.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1WlsbvM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=94&amp;publication_year=2017&amp;pages=106-116&amp;author=L.+Zhangauthor=B.+Dingauthor=Q.+Chenauthor=Q.+Fengauthor=L.+Linauthor=J.+Sun&amp;title=Point-of-Care-Testing+of+Nucleic+Acids+by+Microfluidics&amp;doi=10.1016%2Fj.trac.2017.07.013" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Point-of-care-testing of nucleic acids by microfluidics</span></div><div class="casAuthors">Zhang, Lu; Ding, Baozhi; Chen, Qinghua; Feng, Qiang; Lin, Ling; Sun, Jiashu</div><div class="citationInfo"><span class="NLM_cas:title">TrAC, Trends in Analytical Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">106-116</span>CODEN:
                <span class="NLM_cas:coden">TTAEDJ</span>;
        ISSN:<span class="NLM_cas:issn">0165-9936</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nucleic acid detection offers crit. information in diagnosing pathogen infection and monitoring disease treatment.  Point-of-care testing (POCT), with ideal characteristics such as rapidness, user-friendly, sensitivity, specificity, and robustness, has drawn wide attention, esp. in resource-poor settings where lack the healthcare infrastructure.  Accompanied by the development of microfluidic technologies and new amplification methods, POCT of nucleic acids greatly improves health monitoring and prevents outbreak of infectious diseases.  This review summarizes recent advances in diagnostics of nucleic acids in POC formats, and discusses challenges assocd. with POCT of nucleic acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogvQCdYQKH5rVg90H21EOLACvtfcHk0lhmEUdP3fr9gw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXht1WlsbvM&amp;md5=9369e79766718869dc787e6ebb437ab2</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.trac.2017.07.013&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trac.2017.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DDing%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DJ.%26atitle%3DPoint-of-Care-Testing%2520of%2520Nucleic%2520Acids%2520by%2520Microfluidics%26jtitle%3DTrAC%252C%2520Trends%2520Anal.%2520Chem.%26date%3D2017%26volume%3D94%26spage%3D106%26epage%3D116%26doi%3D10.1016%2Fj.trac.2017.07.013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garibyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avashia, N.</span></span> <span> </span><span class="NLM_article-title">Polymerase Chain Reaction</span>. <i>J. Invest. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">4</span>, <span class="refDoi">&nbsp;DOI: 10.1038/jid.2013.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1038%2Fjid.2013.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=23399825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A280%3ADC%252BC3szntlyksw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=133&amp;publication_year=2013&amp;pages=1-4&amp;author=L.+Garibyanauthor=N.+Avashia&amp;title=Polymerase+Chain+Reaction&amp;doi=10.1038%2Fjid.2013.1" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Polymerase chain reaction</span></div><div class="casAuthors">Garibyan Lilit; Avashia Nidhi</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of investigative dermatology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1-4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR6S-kR0Z69e0PI-fCgwosgfW6udTcc2eajNAMp6RaDdbntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC3szntlyksw%253D%253D&amp;md5=a2b4ed72e813d938dd74567970d8079b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fjid.2013.1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fjid.2013.1%26sid%3Dliteratum%253Aachs%26aulast%3DGaribyan%26aufirst%3DL.%26aulast%3DAvashia%26aufirst%3DN.%26atitle%3DPolymerase%2520Chain%2520Reaction%26jtitle%3DJ.%2520Invest.%2520Dermatol.%26date%3D2013%26volume%3D133%26spage%3D1%26epage%3D4%26doi%3D10.1038%2Fjid.2013.1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ali, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, D.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ankrum, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span> <span> </span><span class="NLM_article-title">Rolling Circle Amplification: A Versatile Tool for Chemical Biology, Materials Science and Medicine</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">3324</span>, <span class="refDoi">&nbsp;DOI: 10.1039/c3cs60439j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1039%2Fc3cs60439j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=24643375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmvFGhsb8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=43&amp;publication_year=2014&amp;pages=3324&amp;author=M.+M.+Aliauthor=F.+Liauthor=Z.+Zhangauthor=K.+Zhangauthor=D.-K.+Kangauthor=J.+A.+Ankrumauthor=X.+C.+Leauthor=W.+Zhao&amp;title=Rolling+Circle+Amplification%3A+A+Versatile+Tool+for+Chemical+Biology%2C+Materials+Science+and+Medicine&amp;doi=10.1039%2Fc3cs60439j" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Rolling circle amplification: a versatile tool for chemical biology, materials science and medicine</span></div><div class="casAuthors">Ali, M. Monsur; Li, Feng; Zhang, Zhiqing; Zhang, Kaixiang; Kang, Dong-Ku; Ankrum, James A.; Le, X. Chris; Zhao, Weian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3324-3341</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Rolling circle amplification (RCA) is an isothermal enzymic process where a short DNA or RNA primer is amplified to form a long single stranded DNA or RNA using a circular DNA template and special DNA or RNA polymerases.  The RCA product is a concatemer contg. tens to hundreds of tandem repeats that are complementary to the circular template.  The power, simplicity, and versatility of the DNA amplification technique have made it an attractive tool for biomedical research and nanobiotechnol.  Traditionally, RCA has been used to develop sensitive diagnostic methods for a variety of targets including nucleic acids (DNA, RNA), small mols., proteins, and cells.  RCA has also attracted significant attention in the field of nanotechnol. and nanobiotechnol.  The RCA-produced long, single-stranded DNA with repeating units has been used as template for the periodic assembly of nanospecies.  Moreover, since RCA products can be tailor-designed by manipulating the circular template, RCA has been employed to generate complex DNA nanostructures such as DNA origami, nanotubes, nanoribbons and DNA based metamaterials.  These functional RCA based nanotechnologies have been utilized for biodetection, drug delivery, designing bioelectronic circuits and biosepn.  In this review, we introduce the fundamental engineering principles used to design RCA nanotechnologies, discuss recently developed RCA-based diagnostics and bioanal. tools, and summarize the use of RCA to construct multivalent mol. scaffolds and nanostructures for applications in biol., diagnostics and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlPH9u97qAabVg90H21EOLACvtfcHk0lgwk-KJAPQQ4g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXmvFGhsb8%253D&amp;md5=3a50f54645d8e6221e5a3e4383bdbc31</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1039%2Fc3cs60439j&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3cs60439j%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DM.%2BM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DKang%26aufirst%3DD.-K.%26aulast%3DAnkrum%26aufirst%3DJ.%2BA.%26aulast%3DLe%26aufirst%3DX.%2BC.%26aulast%3DZhao%26aufirst%3DW.%26atitle%3DRolling%2520Circle%2520Amplification%253A%2520A%2520Versatile%2520Tool%2520for%2520Chemical%2520Biology%252C%2520Materials%2520Science%2520and%2520Medicine%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2014%26volume%3D43%26spage%3D3324%26doi%3D10.1039%2Fc3cs60439j" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mori, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagamine, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Notomi, T.</span></span> <span> </span><span class="NLM_article-title">Detection of Loop-Mediated Isothermal Amplification Reaction by Turbidity Derived from Magnesium Pyrophosphate Formation</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">154</span>, <span class="refDoi">&nbsp;DOI: 10.1006/bbrc.2001.5921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1006%2Fbbrc.2001.5921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=11708792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BD3MXot12ht7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=289&amp;publication_year=2001&amp;pages=150-154&amp;author=Y.+Moriauthor=K.+Nagamineauthor=N.+Tomitaauthor=T.+Notomi&amp;title=Detection+of+Loop-Mediated+Isothermal+Amplification+Reaction+by+Turbidity+Derived+from+Magnesium+Pyrophosphate+Formation&amp;doi=10.1006%2Fbbrc.2001.5921" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of Loop-Mediated Isothermal Amplification Reaction by Turbidity Derived from Magnesium Pyrophosphate Formation</span></div><div class="casAuthors">Mori, Yasuyoshi; Nagamine, Kentaro; Tomita, Norihiro; Notomi, Tsugunori</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">150-154</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">The loop-mediated isothermal amplification (LAMP) is a novel nucleic acid amplification method that uses only one type of enzyme.  One of the characteristics of the LAMP method is its ability to synthesize extremely large amt. of DNA.  Accordingly, a large amt. of byproduct, pyrophosphate ion, is produced, yielding white ppt. of magnesium pyrophosphate in the reaction mixt.  Judging the presence or absence of this white ppt. allows easy distinction of whether nucleic acid was amplified by the LAMP method.  Since an increase in the turbidity of the reaction mixt. according to the prodn. of ppt. correlates with the amt. of DNA synthesized, real-time monitoring of the LAMP reaction was achieved by real-time measurement of turbidity.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1yNhZIhbXd7Vg90H21EOLACvtfcHk0lgwk-KJAPQQ4g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3MXot12ht7Y%253D&amp;md5=2f8786b3d548afb6ec7f49cc9d5c8db5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.2001.5921&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.2001.5921%26sid%3Dliteratum%253Aachs%26aulast%3DMori%26aufirst%3DY.%26aulast%3DNagamine%26aufirst%3DK.%26aulast%3DTomita%26aufirst%3DN.%26aulast%3DNotomi%26aufirst%3DT.%26atitle%3DDetection%2520of%2520Loop-Mediated%2520Isothermal%2520Amplification%2520Reaction%2520by%2520Turbidity%2520Derived%2520from%2520Magnesium%2520Pyrophosphate%2520Formation%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2001%26volume%3D289%26spage%3D150%26epage%3D154%26doi%3D10.1006%2Fbbrc.2001.5921" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duarte-Guevara, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bashir, R.</span></span> <span> </span><span class="NLM_article-title">On-Chip Electrical Detection of Parallel Loop-Mediated Isothermal Amplification with DG-BioFETs for the Detection of Foodborne Bacterial Pathogens</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">103872</span>– <span class="NLM_lpage">103887</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C6RA19685C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1039%2FC6RA19685C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs12ksrnJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2016&amp;pages=103872-103887&amp;author=C.+Duarte-Guevaraauthor=V.+V.+Swaminathanauthor=B.+Reddyauthor=J.-C.+Huangauthor=Y.-S.+Liuauthor=R.+Bashir&amp;title=On-Chip+Electrical+Detection+of+Parallel+Loop-Mediated+Isothermal+Amplification+with+DG-BioFETs+for+the+Detection+of+Foodborne+Bacterial+Pathogens&amp;doi=10.1039%2FC6RA19685C" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">On-chip electrical detection of parallel loop-mediated isothermal amplification with DG-BioFETs for the detection of foodborne bacterial pathogens</span></div><div class="casAuthors">Duarte-Guevara, Carlos; Swaminathan, Vikhram V.; Reddy, Bobby; Huang, Jui-Cheng; Liu, Yi-Shao; Bashir, Rashid</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">106</span>),
    <span class="NLM_cas:pages">103872-103887</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The use of field effect transistors (FETs) as the transduction element for the detection of DNA amplification reactions will enable portable and inexpensive nucleic acid anal.  Transistors used as biol. sensors, or BioFETs, minimize the cost and size of detection platforms by leveraging fabrication methods already well developed for electronics.  Here, we report a dual-gate BioFET (DG-BioFET) array platform with 1024×1024 sensors that is used for on-chip elec. detection of loop-mediated isothermal amplification (LAMP) reactions that target food borne bacterial pathogens.  The DG-BioFETs of our 7 × 7 mm2 array are able to elec. detect pH changes that are triggered by nucleotide incorporation during LAMP elongation.  Multiple 250 nL reactions can be simultaneously elec. monitored in our array that is divided in 30 micro-chambers with gold-coated anisotropically etched silicon wells that act both as ref. electrode and confinement element.  Our characterization results show that the gold-biased DG-BioFETs have a sensitivity of 32 mV pH-1 (equiv. to 2 μA pH-1) and an av. resoln. of 0.5 pH units.  This sensitivity is high enough to detect the pH changes triggered by the amplification reaction, but to maximize our signal-to-noise ratio and improve our quant. conclusions we use a group of data anal. techniques that are available in our high-d. platform that monitors each reaction with ∼3500 independent BioFETs.  We use redundancy techniques to minimize the overall std. deviation of our measurements, the Grubbs test to eliminate measurements outside the expected normal distribution, and ref. micro-chambers to subtract the common noise.  With these techniques we are capable of reducing the P value, of a t-test comparing pos. and neg. readings, from a typical 0.17 to 0.03.  The platform that we present along with the anal. techniques that we developed allow the on-chip elec. detection and identification of E. coli O157 and S. typhi with parallel LAMP assays targeting eae and invA genes.  The LAMP reactions are highly specific, without false positives, and our titrn. assays demonstrate a limit of detection of 23 CFU per reaction on chip.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeNf2SFHu0QrVg90H21EOLACvtfcHk0lgwk-KJAPQQ4g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28Xhs12ksrnJ&amp;md5=56b13ff8ee55404c57e1993d2c7e15f2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1039%2FC6RA19685C&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6RA19685C%26sid%3Dliteratum%253Aachs%26aulast%3DDuarte-Guevara%26aufirst%3DC.%26aulast%3DSwaminathan%26aufirst%3DV.%2BV.%26aulast%3DReddy%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DJ.-C.%26aulast%3DLiu%26aufirst%3DY.-S.%26aulast%3DBashir%26aufirst%3DR.%26atitle%3DOn-Chip%2520Electrical%2520Detection%2520of%2520Parallel%2520Loop-Mediated%2520Isothermal%2520Amplification%2520with%2520DG-BioFETs%2520for%2520the%2520Detection%2520of%2520Foodborne%2520Bacterial%2520Pathogens%26jtitle%3DRSC%2520Adv.%26date%3D2016%26volume%3D6%26spage%3D103872%26epage%3D103887%26doi%3D10.1039%2FC6RA19685C" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salm, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte-Guevara, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bashir, R.</span></span> <span> </span><span class="NLM_article-title">Electrical Detection of Nucleic Acid Amplification Using an On-Chip Quasi-Reference Electrode and a PVC REFET</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">6968</span>– <span class="NLM_lpage">6975</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ac500897t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac500897t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvFCltLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=86&amp;publication_year=2014&amp;pages=6968-6975&amp;author=E.+Salmauthor=Y.+Zhongauthor=B.+Reddyauthor=C.+Duarte-Guevaraauthor=V.+Swaminathanauthor=Y.-S.+Liuauthor=R.+Bashir&amp;title=Electrical+Detection+of+Nucleic+Acid+Amplification+Using+an+On-Chip+Quasi-Reference+Electrode+and+a+PVC+REFET&amp;doi=10.1021%2Fac500897t" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Electrical detection of nucleic acid amplification using an on-chip quasi-reference electrode and a PVC REFET</span></div><div class="casAuthors">Salm, Eric; Zhong, Yu; Reddy, Bobby; Duarte-Guevara, Carlos; Swaminathan, Vikhram; Liu, Yi-Shao; Bashir, Rashid</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6968-6975</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Elec. detection of nucleic acid amplification through pH changes assocd. with nucleotide addn. enables miniaturization, greater portability of testing app., and reduced costs.  However, current ion-sensitive field effect transistor methods for sensing nucleic acid amplification rely on establishing the fluid gate potential with a bulky, difficult to microfabricate ref. electrode that limits the potential for massively parallel reaction detection.  Here we demonstrate a novel method of utilizing a microfabricated solid-state quasi-ref. electrode (QRE) paired with a pH-insensitive ref. field effect transistor (REFET) for detection of real-time pH changes.  The end result is a 0.18 μm, silicon-on-insulator, foundry-fabricated sensor that utilizes a platinum QRE to establish a pH-sensitive fluid gate potential and a PVC membrane REFET to enable pH detection of loop mediated isothermal amplification (LAMP).  This technique is highly amendable to com. scale-up, reduces the packaging and fabrication requirements for ISFET pH detection, and enables massively parallel droplet interrogation for applications, such as monitoring reaction progression in digital PCR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjFip-YxJsGbVg90H21EOLACvtfcHk0ljXxrZoUNqSZg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXpvFCltLg%253D&amp;md5=13304ddff83cd5d6e4477810ebc3fa5b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fac500897t&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac500897t%26sid%3Dliteratum%253Aachs%26aulast%3DSalm%26aufirst%3DE.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DReddy%26aufirst%3DB.%26aulast%3DDuarte-Guevara%26aufirst%3DC.%26aulast%3DSwaminathan%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DY.-S.%26aulast%3DBashir%26aufirst%3DR.%26atitle%3DElectrical%2520Detection%2520of%2520Nucleic%2520Acid%2520Amplification%2520Using%2520an%2520On-Chip%2520Quasi-Reference%2520Electrode%2520and%2520a%2520PVC%2520REFET%26jtitle%3DAnal.%2520Chem.%26date%3D2014%26volume%3D86%26spage%3D6968%26epage%3D6975%26doi%3D10.1021%2Fac500897t" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tabata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katayama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seichi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyahara, Y.</span></span> <span> </span><span class="NLM_article-title">Label-Free and Electrochemical Detection of Nucleic Acids Based on Isothermal Amplification in Combination with Solid-State PH Sensor</span>. <i>Procedia Eng.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>168</i></span>,  <span class="NLM_fpage">419</span>– <span class="NLM_lpage">422</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.proeng.2016.11.534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.proeng.2016.11.534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1yrur4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=168&amp;publication_year=2016&amp;pages=419-422&amp;author=M.+Tabataauthor=Y.+Katayamaauthor=F.+Mannanauthor=A.+Seichiauthor=K.+Suzukiauthor=T.+Godaauthor=A.+Matsumotoauthor=Y.+Miyahara&amp;title=Label-Free+and+Electrochemical+Detection+of+Nucleic+Acids+Based+on+Isothermal+Amplification+in+Combination+with+Solid-State+PH+Sensor&amp;doi=10.1016%2Fj.proeng.2016.11.534" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Label-free and Electrochemical Detection of Nucleic Acids Based on Isothermal Amplification in Combination with Solid-state pH Sensor</span></div><div class="casAuthors">Tabata, Miyuki; Katayama, Yurika; Mannan, Fahmida; Seichi, Ayaka; Suzuki, Koji; Goda, Tatsuro; Matsumoto, Akira; Miyahara, Yuji</div><div class="citationInfo"><span class="NLM_cas:title">Procedia Engineering</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">419-422</span>CODEN:
                <span class="NLM_cas:coden">PERNBE</span>;
        ISSN:<span class="NLM_cas:issn">1877-7058</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Label-free quantification methods for nucleic acids are attracting attention, since they provide potential tools in clin. applications such as disease diagnosis or prognosis observation.  In this research, we developed a simpler and cost-effective elec. monitoring device of nucleic acid amplification, combining an isothermal amplification method and an Iridium/Iridium oxide (Ir/IrOx) electrode without labeling.  The fabricated Ir/IrOx electrode showed ideal Nernstian response under the various pH buffer solns., and its potential response was less affected even in the presence of charged proteins.  Moreover, we successfully real-time monitored primer generation-rolling circle amplification (PG-RCA) at the 0, 10, 100, 1000 pM of target DNA by detection of released proton during amplification reaction at the const. reaction temp.  This label-free and portable device might be received considerable attention as a useful platform for a point-of-care testing in clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFrrKjUzXnpbVg90H21EOLACvtfcHk0ljXxrZoUNqSZg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXht1yrur4%253D&amp;md5=001ef83181211bf526c0d969cdc8ae38</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.proeng.2016.11.534&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.proeng.2016.11.534%26sid%3Dliteratum%253Aachs%26aulast%3DTabata%26aufirst%3DM.%26aulast%3DKatayama%26aufirst%3DY.%26aulast%3DMannan%26aufirst%3DF.%26aulast%3DSeichi%26aufirst%3DA.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DGoda%26aufirst%3DT.%26aulast%3DMatsumoto%26aufirst%3DA.%26aulast%3DMiyahara%26aufirst%3DY.%26atitle%3DLabel-Free%2520and%2520Electrochemical%2520Detection%2520of%2520Nucleic%2520Acids%2520Based%2520on%2520Isothermal%2520Amplification%2520in%2520Combination%2520with%2520Solid-State%2520PH%2520Sensor%26jtitle%3DProcedia%2520Eng.%26date%3D2016%26volume%3D168%26spage%3D419%26epage%3D422%26doi%3D10.1016%2Fj.proeng.2016.11.534" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Basu, A. S.</span></span> <span> </span><span class="NLM_article-title">Digital Assays Part I: Partitioning Statistics and Digital PCR</span>. <i>SLAS Technol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">369</span>– <span class="NLM_lpage">386</span>, <span class="refDoi">&nbsp;DOI: 10.1177/2472630317705680</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1177%2F2472630317705680" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=28448765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A280%3ADC%252BC1crhslCitg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=2017&amp;pages=369-386&amp;author=A.+S.+Basu&amp;title=Digital+Assays+Part+I%3A+Partitioning+Statistics+and+Digital+PCR&amp;doi=10.1177%2F2472630317705680" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Digital Assays Part I: Partitioning Statistics and Digital PCR</span></div><div class="casAuthors">Basu Amar S</div><div class="citationInfo"><span class="NLM_cas:title">SLAS technology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">369-386</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A digital assay is one in which the sample is partitioned into many small containers such that each partition contains a discrete number of biological entities (0, 1, 2, 3, ...).  A powerful technique in the biologist's toolkit, digital assays bring a new level of precision in quantifying nucleic acids, measuring proteins and their enzymatic activity, and probing single-cell genotypes and phenotypes.  Part I of this review begins with the benefits and Poisson statistics of partitioning, including sources of error.  The remainder focuses on digital PCR (dPCR) for quantification of nucleic acids.  We discuss five commercial instruments that partition samples into physically isolated chambers (cdPCR) or droplet emulsions (ddPCR).  We compare the strengths of dPCR (absolute quantitation, precision, and ability to detect rare or mutant targets) with those of its predecessor, quantitative real-time PCR (dynamic range, larger sample volumes, and throughput).  Lastly, we describe several promising applications of dPCR, including copy number variation, quantitation of circulating tumor DNA and viral load, RNA/miRNA quantitation with reverse transcription dPCR, and library preparation for next-generation sequencing.  This review is intended to give a broad perspective to scientists interested in adopting digital assays into their workflows.  Part II focuses on digital protein and cell assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRylYUBtyUvJoUSrryheKE6fW6udTcc2eZg_3U6wodHC7ntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC1crhslCitg%253D%253D&amp;md5=25b010effce8a7693d5b82faf8dfb050</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1177%2F2472630317705680&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472630317705680%26sid%3Dliteratum%253Aachs%26aulast%3DBasu%26aufirst%3DA.%2BS.%26atitle%3DDigital%2520Assays%2520Part%2520I%253A%2520Partitioning%2520Statistics%2520and%2520Digital%2520PCR%26jtitle%3DSLAS%2520Technol.%26date%3D2017%26volume%3D22%26spage%3D369%26epage%3D386%26doi%3D10.1177%2F2472630317705680" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vashist, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudanyali, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Cellphone-Based Devices for Bioanalytical Sciences</span>. <i>Anal. Bioanal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>406</i></span>,  <span class="NLM_fpage">3263</span>– <span class="NLM_lpage">3277</span>, <span class="refDoi">&nbsp;DOI: 10.1007/s00216-013-7473-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1007%2Fs00216-013-7473-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=24287630" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVKisbnI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=406&amp;publication_year=2014&amp;pages=3263-3277&amp;author=S.+K.+Vashistauthor=O.+Mudanyaliauthor=E.+M.+Schneiderauthor=R.+Zengerleauthor=A.+Ozcan&amp;title=Cellphone-Based+Devices+for+Bioanalytical+Sciences&amp;doi=10.1007%2Fs00216-013-7473-1" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cellphone-based devices for bioanalytical sciences</span></div><div class="casAuthors">Vashist, Sandeep Kumar; Mudanyali, Onur; Schneider, E. Marion; Zengerle, Roland; Ozcan, Aydogan</div><div class="citationInfo"><span class="NLM_cas:title">Analytical and Bioanalytical Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3263-3277</span>CODEN:
                <span class="NLM_cas:coden">ABCNBP</span>;
        ISSN:<span class="NLM_cas:issn">1618-2642</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  During the last decade, there has been a rapidly growing trend toward the use of cellphone-based devices (CBDs) in bioanal. sciences.  For example, they have been used for digital microscopy, cytometry, read-out of immunoassays and lateral flow tests, electrochem. and surface plasmon resonance based bio-sensing, colorimetric detection and healthcare monitoring, among others.  Cellphone can be considered as one of the most prospective devices for the development of next-generation point-of-care (POC) diagnostics platforms, enabling mobile healthcare delivery and personalized medicine.  With more than 6.5 billion cellphone subscribers worldwide and approx. 1.6 billion new devices being sold each year, cellphone technol. is also creating new business and research opportunities.  Many cellphone-based devices, such as those targeted for diabetic management, wt. management, monitoring of blood pressure and pulse rate, have already become com.-available in recent years.  In addn. to such monitoring platforms, several other CBDs are also being introduced, targeting e.g., microscopic imaging and sensing applications for medical diagnostics using novel computational algorithms and components already embedded on cellphones.  This report aims to review these recent developments in CBDs for bioanal. sciences along with some of the challenges involved and the future opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf2OqKsrjVvLVg90H21EOLACvtfcHk0lgyiFNSrjEviA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhvVKisbnI&amp;md5=31f88a69fa1460341913991b0dee933d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs00216-013-7473-1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00216-013-7473-1%26sid%3Dliteratum%253Aachs%26aulast%3DVashist%26aufirst%3DS.%2BK.%26aulast%3DMudanyali%26aufirst%3DO.%26aulast%3DSchneider%26aufirst%3DE.%2BM.%26aulast%3DZengerle%26aufirst%3DR.%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DCellphone-Based%2520Devices%2520for%2520Bioanalytical%2520Sciences%26jtitle%3DAnal.%2520Bioanal.%2520Chem.%26date%3D2014%26volume%3D406%26spage%3D3263%26epage%3D3277%26doi%3D10.1007%2Fs00216-013-7473-1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Mobile Phones Democratize and Cultivate Next-Generation Imaging, Diagnostics and Measurement Tools</span>. <i>Lab Chip</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">3187</span>– <span class="NLM_lpage">3194</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C4LC00010B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1039%2FC4LC00010B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=24647550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1CgtbrE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=2014&amp;pages=3187-3194&amp;author=A.+Ozcan&amp;title=Mobile+Phones+Democratize+and+Cultivate+Next-Generation+Imaging%2C+Diagnostics+and+Measurement+Tools&amp;doi=10.1039%2FC4LC00010B" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mobile phones democratize and cultivate next-generation imaging, diagnostics and measurement tools</span></div><div class="casAuthors">Ozcan, Aydogan</div><div class="citationInfo"><span class="NLM_cas:title">Lab on a Chip</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3187-3194</span>CODEN:
                <span class="NLM_cas:coden">LCAHAM</span>;
        ISSN:<span class="NLM_cas:issn">1473-0189</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In this article, I discuss some of the emerging applications and the future opportunities and challenges created by the use of mobile phones and their embedded components for the development of next-generation imaging, sensing, diagnostics and measurement tools.  The massive vol. of mobile phone users, which has now reached ∼7 billion, drives the rapid improvements of the hardware, software and high-end imaging and sensing technologies embedded in our phones, transforming the mobile phone into a cost-effective and yet extremely powerful platform to run, e.g., biomedical tests, and perform scientific measurements that would normally require advanced lab. instruments.  This rapidly evolving and continuing trend will help us transform how medicine, engineering and sciences are practiced and taught globally.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNnKufjULYsLVg90H21EOLACvtfcHk0lgyiFNSrjEviA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXht1CgtbrE&amp;md5=cf19e56f8c499eca5cf56884d2f28257</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2FC4LC00010B&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4LC00010B%26sid%3Dliteratum%253Aachs%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DMobile%2520Phones%2520Democratize%2520and%2520Cultivate%2520Next-Generation%2520Imaging%252C%2520Diagnostics%2520and%2520Measurement%2520Tools%26jtitle%3DLab%2520Chip%26date%3D2014%26volume%3D14%26spage%3D3187%26epage%3D3194%26doi%3D10.1039%2FC4LC00010B" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ornob, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brisbin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganguli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vemuri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strzebonski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, K. J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Mobile Platform for Multiplexed Detection and Differentiation of Disease-Specific Nucleic Acid Sequences, Using Microfluidic Loop-Mediated Isothermal Amplification and Smartphone Detection</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">11219</span>– <span class="NLM_lpage">11226</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.analchem.7b02478</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.7b02478" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlGksrjI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=89&amp;publication_year=2017&amp;pages=11219-11226&amp;author=W.+Chenauthor=H.+Yuauthor=F.+Sunauthor=A.+Ornobauthor=R.+Brisbinauthor=A.+Ganguliauthor=V.+Vemuriauthor=P.+Strzebonskiauthor=G.+Cuiauthor=K.+J.+Allen&amp;title=Mobile+Platform+for+Multiplexed+Detection+and+Differentiation+of+Disease-Specific+Nucleic+Acid+Sequences%2C+Using+Microfluidic+Loop-Mediated+Isothermal+Amplification+and+Smartphone+Detection&amp;doi=10.1021%2Facs.analchem.7b02478" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mobile Platform for Multiplexed Detection and Differentiation of Disease-Specific Nucleic Acid Sequences, Using Microfluidic Loop-Mediated Isothermal Amplification and Smartphone Detection</span></div><div class="casAuthors">Chen, Weili; Yu, Hojeong; Sun, Fu; Ornob, Akid; Brisbin, Ryan; Ganguli, Anurup; Vemuri, Vinay; Strzebonski, Piotr; Cui, Guangzhe; Allen, Karen J.; Desai, Smit A.; Lin, Weiran; Nash, David M.; Hirschberg, David L.; Brooks, Ian; Bashir, Rashid; Cunningham, Brian T.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11219-11226</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New tools are needed to enable rapid detection, identification, and reporting of infectious viral and microbial pathogens in a wide variety of point-of-care applications that impact human and animal health.  We report the design, construction, and characterization of a platform for multiplexed anal. of disease-specific DNA sequences that utilizes a smartphone camera as the sensor in conjunction with a handheld "cradle" that interfaces the phone with a silicon-based microfluidic chip embedded within a credit card-sized cartridge.  Utilizing specific nucleic acid sequences for four equine respiratory pathogens as representative examples, we demonstrate the ability of the system to utilize a single 15 μL droplet of test sample to perform selective pos./neg. detn. of target sequences, including integrated exptl. controls, in approx. 30 min.  Our approach utilizes loop mediated isothermal amplification (LAMP) reagents pre-deposited into distinct lanes of the microfluidic chip, which, when exposed to target nucleic acid sequences from the test sample, generates fluorescent products that, when excited by appropriately selected light emitting diodes (LEDs) are visualized and automatically analyzed by a software application running on the smartphone microprocessor.  The system achieves detection limits comparable to those obtained by lab.-based methods and instruments.  Assay information is combined with information from the cartridge and the patient to populate a cloud-based database for epidemiol. reporting of test results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqchhgJ2_pqV7Vg90H21EOLACvtfcHk0lgyiFNSrjEviA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhtlGksrjI&amp;md5=976364717fe65488949dfd802136e3b7</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.7b02478&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.7b02478%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DF.%26aulast%3DOrnob%26aufirst%3DA.%26aulast%3DBrisbin%26aufirst%3DR.%26aulast%3DGanguli%26aufirst%3DA.%26aulast%3DVemuri%26aufirst%3DV.%26aulast%3DStrzebonski%26aufirst%3DP.%26aulast%3DCui%26aufirst%3DG.%26aulast%3DAllen%26aufirst%3DK.%2BJ.%26atitle%3DMobile%2520Platform%2520for%2520Multiplexed%2520Detection%2520and%2520Differentiation%2520of%2520Disease-Specific%2520Nucleic%2520Acid%2520Sequences%252C%2520Using%2520Microfluidic%2520Loop-Mediated%2520Isothermal%2520Amplification%2520and%2520Smartphone%2520Detection%26jtitle%3DAnal.%2520Chem.%26date%3D2017%26volume%3D89%26spage%3D11219%26epage%3D11226%26doi%3D10.1021%2Facs.analchem.7b02478" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sannarangaiah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dash, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morita, K.</span></span> <span> </span><span class="NLM_article-title">Loop Mediated Isothermal Amplification (LAMP): A New Generation of Innovative Gene Amplification Technique; Perspectives in Clinical Diagnosis of Infectious Diseases</span>. <i>Rev. Med. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">421</span>, <span class="refDoi">&nbsp;DOI: 10.1002/rmv.593</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1002%2Frmv.593" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=18716992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFWrt7jK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=18&amp;publication_year=2008&amp;pages=407-421&amp;author=M.+Paridaauthor=S.+Sannarangaiahauthor=P.+K.+Dashauthor=P.+V.+L.+Raoauthor=K.+Morita&amp;title=Loop+Mediated+Isothermal+Amplification+%28LAMP%29%3A+A+New+Generation+of+Innovative+Gene+Amplification+Technique%3B+Perspectives+in+Clinical+Diagnosis+of+Infectious+Diseases&amp;doi=10.1002%2Frmv.593" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Loop mediated isothermal amplication (LAMP): a new generation of innovative gene amplification technique; perspectives in clinical diagnosis of infectious diseases</span></div><div class="casAuthors">Parida, Manmohan; Sannarangaiah, Santhosh; Dash, Paban Kumar; Rao, P. V. L.; Morita, Kouichi</div><div class="citationInfo"><span class="NLM_cas:title">Reviews in Medical Virology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">407-421</span>CODEN:
                <span class="NLM_cas:coden">RMVIEW</span>;
        ISSN:<span class="NLM_cas:issn">1052-9276</span>.
    
            (<span class="NLM_cas:orgname">John Wiley &amp; Sons Ltd.</span>)
        </div><div class="casAbstract">A review.  Loop mediated isothermal amplification (LAMP) is a powerful innovative gene amplification technique emerging as a simple rapid diagnostic tool for early detection and identification of microbial diseases.  The whole procedure is very simple and rapid wherein the amplification can be completed in less than 1 h under isothermal conditions employing a set of six specially designed primers spanning eight distinct sequences of a target gene, by incubating all the reagents in a single tube.  Gene amplification products can be detected by agarose gel electrophoresis as well as by real-time monitoring in an inexpensive turbidimeter.  Gene copy no. can also be quantified with the help of a std. curve generated from different concns. of gene copy no. plotted against time of positivity with the help of a real-time turbidimeter.  Alternatively, gene amplification can be visualized by the naked eye either as turbidity or in the form of a color change when SYBR Green I, a fluorescent dsDNA intercalating dye, is employed.  LAMP does not require a thermal cycler and can be performed simply with a heating block and/or water bath.  Considering the advantages of rapid amplification, simple operation and easy detection, LAMP has potential applications for clin. diagnosis as well as surveillance of infectious diseases in developing countries without requiring sophisticated equipment or skilled personnel.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxY1refChnwbVg90H21EOLACvtfcHk0lgyiFNSrjEviA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXhsFWrt7jK&amp;md5=c01ca23fb35cb2d53fa43e6fd658384b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Frmv.593&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Frmv.593%26sid%3Dliteratum%253Aachs%26aulast%3DParida%26aufirst%3DM.%26aulast%3DSannarangaiah%26aufirst%3DS.%26aulast%3DDash%26aufirst%3DP.%2BK.%26aulast%3DRao%26aufirst%3DP.%2BV.%2BL.%26aulast%3DMorita%26aufirst%3DK.%26atitle%3DLoop%2520Mediated%2520Isothermal%2520Amplification%2520%2528LAMP%2529%253A%2520A%2520New%2520Generation%2520of%2520Innovative%2520Gene%2520Amplification%2520Technique%253B%2520Perspectives%2520in%2520Clinical%2520Diagnosis%2520of%2520Infectious%2520Diseases%26jtitle%3DRev.%2520Med.%2520Virol.%26date%3D2008%26volume%3D18%26spage%3D407%26epage%3D421%26doi%3D10.1002%2Frmv.593" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morton, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shopsowitz, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, P. T.</span></span> <span> </span><span class="NLM_article-title">RNAi-Microsponges Form through Self-Assembly of the Organic and Inorganic Products of Transcription</span>. <i>Small</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1623</span>– <span class="NLM_lpage">1633</span>, <span class="refDoi">&nbsp;DOI: 10.1002/smll.201302676</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1002%2Fsmll.201302676" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2013&amp;pages=1623-1633&amp;author=S.+W.+Mortonauthor=Z.+J.+Dengauthor=Y.+H.+Rohauthor=K.+E.+Shopsowitzauthor=P.+T.+Hammond&amp;title=RNAi-Microsponges+Form+through+Self-Assembly+of+the+Organic+and+Inorganic+Products+of+Transcription&amp;doi=10.1002%2Fsmll.201302676" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fsmll.201302676&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fsmll.201302676%26sid%3Dliteratum%253Aachs%26aulast%3DMorton%26aufirst%3DS.%2BW.%26aulast%3DDeng%26aufirst%3DZ.%2BJ.%26aulast%3DRoh%26aufirst%3DY.%2BH.%26aulast%3DShopsowitz%26aufirst%3DK.%2BE.%26aulast%3DHammond%26aufirst%3DP.%2BT.%26atitle%3DRNAi-Microsponges%2520Form%2520through%2520Self-Assembly%2520of%2520the%2520Organic%2520and%2520Inorganic%2520Products%2520of%2520Transcription%26jtitle%3DSmall%26date%3D2013%26volume%3D10%26spage%3D1623%26epage%3D1633%26doi%3D10.1002%2Fsmll.201302676" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishikawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takakura, Y.</span></span> <span> </span><span class="NLM_article-title">Application of Magnesium Pyrophosphate-Based Sponge-Like Microparticles to Enhance the Delivery Efficiency and Adjuvant Effects of Polyriboinosinic-Polyribocytidylic Acid in Immune Cells</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">766</span>– <span class="NLM_lpage">772</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.xphs.2015.11.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.xphs.2015.11.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=26869429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFegu7fI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=105&amp;publication_year=2016&amp;pages=766-772&amp;author=S.+Takagiauthor=Y.+Takahashiauthor=K.+Sugimuraauthor=M.+Nishikawaauthor=Y.+Takakura&amp;title=Application+of+Magnesium+Pyrophosphate-Based+Sponge-Like+Microparticles+to+Enhance+the+Delivery+Efficiency+and+Adjuvant+Effects+of+Polyriboinosinic-Polyribocytidylic+Acid+in+Immune+Cells&amp;doi=10.1016%2Fj.xphs.2015.11.020" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Magnesium Pyrophosphate-Based Sponge-Like Microparticles to Enhance the Delivery Efficiency and Adjuvant Effects of Polyriboinosinic-Polyribocytidylic Acid in Immune Cells</span></div><div class="casAuthors">Takagi, Shoichi; Takahashi, Yuki; Sugimura, Kanako; Nishikawa, Makiya; Takakura, Yoshinobu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">766-772</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The magnesium pyrophosphate particle (MgPP) is a unique and safe carrier that is prepd. by simply mixing magnesium chloride and sodium pyrophosphate.  In this study, we investigated whether MgPP can be used to deliver nucleic acid-based adjuvants to immune cells.  Polyriboinosinic-polyribocytidylic acid (polyI:C), a ligand for toll-like receptor 3, was selected as a model nucleic acid-based adjuvant.  PolyI:C-loaded MgPP (polyI:C-MgPP) was prepd. by adding polyI:C during the MgPP prepn. process.  Efficient loading of polyI:C into MgPP was confirmed by measuring the absorbance at 260 nm after disruption of polyI:C-MgPP by EDTA.  SEM revealed that both MgPP and polyI:C-MgPP had a unique sponge-like shape with a diam. of approx. 1 μm.  PolyI:C-MgPP was more efficiently taken up by toll-like receptor 3-pos. RAW264.7 cells than naked polyI:C, and its uptake stimulated increased tumor necrosis factor-α prodn.  When the presentation of ovalbumin (OVA), a model antigen, was evaluated after the addn. of OVA along with naked polyI:C or polyI:C-MgPP to mouse dendritic DC2.4 cells, polyI:C-MgPP substantially increased OVA presentation.  These results indicate that MgPP is a useful delivery vehicle for polyI:C and that polyI:C-MgPP is an effective immune cell adjuvant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXYZ7dhZbUh7Vg90H21EOLACvtfcHk0lgY8gOsiRBp7g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XhtFegu7fI&amp;md5=a2363dec306eadd04cf6bfd326f1ac88</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.xphs.2015.11.020&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.xphs.2015.11.020%26sid%3Dliteratum%253Aachs%26aulast%3DTakagi%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DY.%26aulast%3DSugimura%26aufirst%3DK.%26aulast%3DNishikawa%26aufirst%3DM.%26aulast%3DTakakura%26aufirst%3DY.%26atitle%3DApplication%2520of%2520Magnesium%2520Pyrophosphate-Based%2520Sponge-Like%2520Microparticles%2520to%2520Enhance%2520the%2520Delivery%2520Efficiency%2520and%2520Adjuvant%2520Effects%2520of%2520Polyriboinosinic-Polyribocytidylic%2520Acid%2520in%2520Immune%2520Cells%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2016%26volume%3D105%26spage%3D766%26epage%3D772%26doi%3D10.1016%2Fj.xphs.2015.11.020" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monis, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giglio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saint, C. P.</span></span> <span> </span><span class="NLM_article-title">Comparison of SYTO9 and SYBR Green I for Real-Time Polymerase Chain Reaction and Investigation of the Effect of Dye Concentration on Amplification and DNA Melting Curve Analysis</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>340</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">34</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.ab.2005.01.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.ab.2005.01.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=15802126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivV2jsL4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=340&amp;publication_year=2005&amp;pages=24-34&amp;author=P.+T.+Monisauthor=S.+Giglioauthor=C.+P.+Saint&amp;title=Comparison+of+SYTO9+and+SYBR+Green+I+for+Real-Time+Polymerase+Chain+Reaction+and+Investigation+of+the+Effect+of+Dye+Concentration+on+Amplification+and+DNA+Melting+Curve+Analysis&amp;doi=10.1016%2Fj.ab.2005.01.046" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of SYTO9 and SYBR Green I for real-time polymerase chain reaction and investigation of the effect of dye concentration on amplification and DNA melting curve analysis</span></div><div class="casAuthors">Monis, Paul T.; Giglio, Steven; Saint, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">340</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-34</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Following the initial report of the use of SYBR Green I for real-time polymerase chain reaction (PCR) in 1997, little attention has been given to the development of alternative intercalating dyes for this application.  This is surprising considering the reported limitations of SYBR Green I, which include limited dye stability, dye-dependent PCR inhibition, and selective detection of amplicons during DNA melting curve anal. of multiplex PCRs.  We have tested an alternative to SYBR Green I and report the first detailed evaluation of the intercalating dye SYTO9.  Our findings demonstrate that SYTO9 produces highly reproducible DNA melting curves over a broader range of dye concns. than does SYBR Green I, is far less inhibitory to PCR than SYBR Green I, and does not appear to selectively detect particular amplicons.  The low inhibition and high melting curve reproducibility of SYTO9 means that it can be readily incorporated into a conventional PCR at a broad range of concns., allowing closed tube anal. by DNA melting curve anal.  These features simplify the use of intercalating dyes in real-time PCR and the improved reproducibility of DNA melting curve anal. will make SYTO9 useful in a diagnostic context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPDPuKLUL1BrVg90H21EOLACvtfcHk0lgY8gOsiRBp7g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXivV2jsL4%253D&amp;md5=97c6fdb3aff06fdf97f8064907b99f40</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2005.01.046&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2005.01.046%26sid%3Dliteratum%253Aachs%26aulast%3DMonis%26aufirst%3DP.%2BT.%26aulast%3DGiglio%26aufirst%3DS.%26aulast%3DSaint%26aufirst%3DC.%2BP.%26atitle%3DComparison%2520of%2520SYTO9%2520and%2520SYBR%2520Green%2520I%2520for%2520Real-Time%2520Polymerase%2520Chain%2520Reaction%2520and%2520Investigation%2520of%2520the%2520Effect%2520of%2520Dye%2520Concentration%2520on%2520Amplification%2520and%2520DNA%2520Melting%2520Curve%2520Analysis%26jtitle%3DAnal.%2520Biochem.%26date%3D2005%26volume%3D340%26spage%3D24%26epage%3D34%26doi%3D10.1016%2Fj.ab.2005.01.046" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gudnason, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bang, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolff, A.</span></span> <span> </span><span class="NLM_article-title">Comparison of Multiple DNA Dyes for Real-Time PCR: Effects of Dye Concentration and Sequence Composition on DNA Amplification and Melting Temperature</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>, <span class="NLM_elocation-id">e127</span> <span class="refDoi">&nbsp;DOI: 10.1093/nar/gkm671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1093%2Fnar%2Fgkm671" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=17897966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1yktbvJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=35&amp;publication_year=2007&amp;author=H.+Gudnasonauthor=M.+Dufvaauthor=D.+D.+Bangauthor=A.+Wolff&amp;title=Comparison+of+Multiple+DNA+Dyes+for+Real-Time+PCR%3A+Effects+of+Dye+Concentration+and+Sequence+Composition+on+DNA+Amplification+and+Melting+Temperature&amp;doi=10.1093%2Fnar%2Fgkm671" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of multiple DNA dyes for real-time PCR: Effects of dye concentration and sequence composition on DNA amplification and melting temperature</span></div><div class="casAuthors">Gudnason, Haukur; Dufva, Martin; Bang, D. D.; Wolff, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">e127/1-e127/8</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The importance of real-time polymerase chain reaction (PCR) has increased steadily in clin. applications over the last decade.  Many applications utilize SYBR Green I dye to follow the accumulation of amplicons in real time.  SYBR Green I has, however, a no. of limitations that include the inhibition of PCR, preferential binding to GC-rich sequences and effects on melting curve anal.  Although a few alternative dyes without some of these limitations have been recently proposed, no large-scale investigation into the properties of intercalating dyes has been performed.  In this study, we investigate 15 different intercalating DNA dyes for their inhibitory effects on PCR, effects on DNA melting temp. and possible preferential binding to GC-rich sequences.  Our results demonstrated that in contrast to the results of SYBR Green I, two intercalating dyes SYTO-13 and SYTO-82 do not inhibit PCR, show no preferential binding to GC rich sequences and do not influence melting temp., Tm, even at high concns.  In addn., SYTO-82 demonstrated a 50-fold lower detection limit in a diln. series assay.  In conclusion, the properties of SYTO-82 and SYTO-13 will simplify the development of multiplex assays and increase the sensitivity of real-time PCR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpALrHgnh2robVg90H21EOLACvtfcHk0lgY8gOsiRBp7g"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXht1yktbvJ&amp;md5=fd0e885f9ec6957e8165bcf31b2504a5</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm671&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm671%26sid%3Dliteratum%253Aachs%26aulast%3DGudnason%26aufirst%3DH.%26aulast%3DDufva%26aufirst%3DM.%26aulast%3DBang%26aufirst%3DD.%2BD.%26aulast%3DWolff%26aufirst%3DA.%26atitle%3DComparison%2520of%2520Multiple%2520DNA%2520Dyes%2520for%2520Real-Time%2520PCR%253A%2520Effects%2520of%2520Dye%2520Concentration%2520and%2520Sequence%2520Composition%2520on%2520DNA%2520Amplification%2520and%2520Melting%2520Temperature%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26doi%3D10.1093%2Fnar%2Fgkm671" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, X.</span></span> <span> </span><span class="NLM_article-title">Characterization of EvaGreen and the Implication of Its Physicochemical Properties for QPCR Applications</span>. <i>BMC Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">76</span> <span class="refDoi">&nbsp;DOI: 10.1186/1472-6750-7-76</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1186%2F1472-6750-7-76" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=17996102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A280%3ADC%252BD1c%252FmtVSksw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2007&amp;author=F.+Maoauthor=W.-Y.+Leungauthor=X.+Xin&amp;title=Characterization+of+EvaGreen+and+the+Implication+of+Its+Physicochemical+Properties+for+QPCR+Applications&amp;doi=10.1186%2F1472-6750-7-76" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of EvaGreen and the implication of its physicochemical properties for qPCR applications</span></div><div class="casAuthors">Mao Fei; Leung Wai-Yee; Xin Xing</div><div class="citationInfo"><span class="NLM_cas:title">BMC biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">76</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  EvaGreen (EG) is a newly developed DNA-binding dye that has recently been used in quantitative real-time PCR (qPCR), post-PCR DNA melt curve analysis and several other applications.  However, very little is known about the physicochemical properties of the dye and their relevance to the applications, particularly to qPCR and post PCR DNA melt curve analysis.  In this paper, we characterized EG along with a widely used qPCR dye, SYBR Green I (SG), for their DNA-binding properties and stability, and compared their performance in qPCR under a variety of conditions.  RESULTS:  This study systematically compared the DNA binding profiles of the two dyes under different conditions and had these findings: a) EG had a lower binding affinity for both double-stranded DNA (dsDNA) and single-stranded DNA (ssDNA) than SG; b) EG showed no apparent preference for either GC- or AT-rich sequence while SG had a slight preference for AT-rich sequence; c) both dyes showed substantially lower affinity toward ssDNA than toward dsDNA and even lower affinity toward shorter ssDNA fragments except that this trend was more pronounced for EG.  Our results also demonstrated that EG was stable both under PCR condition and during routine storage and handling.  In the comparative qPCR study, both EG and SG exhibited PCR interference when used at high dye concentration, as evident from delayed Ct and/or nonspecific product formation.  The problem worsened when the chain extension time was shortened or when the amplicon size was relatively long (&gt;500 bp).  However, qPCR using EG tolerated a significantly higher dye concentration, thus permitting a more robust PCR signal as well as a sharper and stronger DNA melt peak.  These differences in qPCR performance between the two dyes are believed to be attributable to their differences in DNA binding profiles.  CONCLUSION:  These findings suggest that an ideal qPCR dye should possess several DNA-binding characteristics, including a "just right" affinity for dsDNA and low or no affinity for ssDNA and short DNA fragments.  The favorable DNA-binding profile of EG, coupled with its good stability and instrument-compatibility, should make EG a promising dye for qPCR and related applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZzfmbgDKOB7qxQ2mb9Pu8fW6udTcc2ebvNg5EwlgHg7ntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BD1c%252FmtVSksw%253D%253D&amp;md5=0fd4d90f0c27051a4d9bed6c12bfc7fb</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1186%2F1472-6750-7-76&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6750-7-76%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DF.%26aulast%3DLeung%26aufirst%3DW.-Y.%26aulast%3DXin%26aufirst%3DX.%26atitle%3DCharacterization%2520of%2520EvaGreen%2520and%2520the%2520Implication%2520of%2520Its%2520Physicochemical%2520Properties%2520for%2520QPCR%2520Applications%26jtitle%3DBMC%2520Biotechnol.%26date%3D2007%26volume%3D7%26doi%3D10.1186%2F1472-6750-7-76" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nath, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarosy, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Como, C. J.</span></span> <span> </span><span class="NLM_article-title">Effects of Ethidium Bromide and SYBR Green I on Different Polymerase Chain Reaction Systems</span>. <i>J. Biochem. Biophys. Methods</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">29</span>, <span class="refDoi">&nbsp;DOI: 10.1016/S0165-022X(99)00033-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2FS0165-022X%2899%2900033-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10647811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADyaK1MXns1ahsLY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=42&amp;publication_year=2000&amp;pages=15-29&amp;author=K.+Nathauthor=J.+W.+Sarosyauthor=J.+Hahnauthor=C.+J.+Di+Como&amp;title=Effects+of+Ethidium+Bromide+and+SYBR+Green+I+on+Different+Polymerase+Chain+Reaction+Systems&amp;doi=10.1016%2FS0165-022X%2899%2900033-0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of ethidium bromide and SYBR Green I on different polymerase chain reaction systems</span></div><div class="casAuthors">Nath, K.; Sarosy, J. W.; Hahn, J.; Di Como, C. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemical and Biophysical Methods</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">15-29</span>CODEN:
                <span class="NLM_cas:coden">JBBMDG</span>;
        ISSN:<span class="NLM_cas:issn">0165-022X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">In an in-gel polymerase chain reaction (PCR), the generation of a 1750-bp yeast DNA fragment was inhibited when yeast DNA gel-stabs or gel-slices stained with ethidium bromide (EtBr) or SYBR Green I were used.  Similar inhibition occurred to a varying degree in the reamplification of PCR fragments in prokaryotic systems.  Inclusion of the dyes in PCR resulted in an inhibition at about 10 μg/mL EtBr and at 10,000-20,000-fold diln. of SYBR Green I in all systems.  The effect remained unchanged despite increasing the PCR cycles to 40.  However, increasing the magnesium chloride concn. did reverse the inhibitory actions, although the PCR specificity was lost.  In an unusual observation, we find that, at higher dye concns. (50 μg/mL EtBr, or thousand fold diln. of SYBR Green I), the input yeast DNA electrophoretic profile is maintained following 25 PCR cycles (despite a denaturation temp. of 94°C).  It varied significantly in different DNA systems and was readily reversed by high Mg++ concns.  It is concluded that, at low Mg++ concns., different PCR systems are inhibited to varying extents by intercalating dyes and, in some PCR systems, intercalating dyes at unusually high concns. maintain input DNA electrophoretic profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUYIh2z_UNZbVg90H21EOLACvtfcHk0lgphB-bpaMV3Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1MXns1ahsLY%253D&amp;md5=7f96a617ff4f57f8978150201d8da69d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS0165-022X%2899%2900033-0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0165-022X%252899%252900033-0%26sid%3Dliteratum%253Aachs%26aulast%3DNath%26aufirst%3DK.%26aulast%3DSarosy%26aufirst%3DJ.%2BW.%26aulast%3DHahn%26aufirst%3DJ.%26aulast%3DDi%2BComo%26aufirst%3DC.%2BJ.%26atitle%3DEffects%2520of%2520Ethidium%2520Bromide%2520and%2520SYBR%2520Green%2520I%2520on%2520Different%2520Polymerase%2520Chain%2520Reaction%2520Systems%26jtitle%3DJ.%2520Biochem.%2520Biophys.%2520Methods%26date%3D2000%26volume%3D42%26spage%3D15%26epage%3D29%26doi%3D10.1016%2FS0165-022X%2899%2900033-0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, O. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Carlo, D.</span></span> <span> </span><span class="NLM_article-title">Highly Stable and Sensitive Nucleic Acid Amplification and Cell-Phone-Based Readout</span>. <i>ACS Nano</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2934</span>– <span class="NLM_lpage">2943</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acsnano.6b08274</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsnano.6b08274" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlCnu70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2017&amp;pages=2934-2943&amp;author=J.+E.+Kongauthor=Q.+Weiauthor=D.+Tsengauthor=J.+Zhangauthor=E.+Panauthor=M.+Lewinskiauthor=O.+B.+Garnerauthor=A.+Ozcanauthor=D.+Di+Carlo&amp;title=Highly+Stable+and+Sensitive+Nucleic+Acid+Amplification+and+Cell-Phone-Based+Readout&amp;doi=10.1021%2Facsnano.6b08274" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Highly stable and sensitive nucleic acid amplification and cell-phone-based readout</span></div><div class="casAuthors">Kong, Janay E.; Wei, Qingshan; Tseng, Derek; Zhang, Jingzi; Pan, Eric; Lewinski, Michael; Garner, Omai B.; Ozcan, Aydogan; Di Carlo, Dino</div><div class="citationInfo"><span class="NLM_cas:title">ACS Nano</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">2934-2943</span>CODEN:
                <span class="NLM_cas:coden">ANCAC3</span>;
        ISSN:<span class="NLM_cas:issn">1936-0851</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Key challenges with point-of-care (POC) nucleic acid tests include achieving a low-cost, portable form factor, and stable readout, while also retaining the same robust stds. of benchtop lab-based tests.  We addressed two crucial aspects of this problem, identifying a chem. additive, hydroxynaphthol blue, that both stabilizes and significantly enhances intercalator-based fluorescence readout of nucleic acid concn., and developing a cost-effective fiber-optic bundle-based fluorescence microplate reader integrated onto a mobile phone.  Using loop-mediated isothermal amplification on lambda DNA we achieve a 69-fold increase in signal above background, 20-fold higher than the gold std., yielding an overall limit of detection of 25 copies/μL within an hour using our mobile-phone-based platform.  Crit. for a point-of-care system, we achieve a &gt;60% increase in fluorescence stability as a function of temp. and time, obviating the need for manual baseline correction or secondary calibration dyes.  This field-portable and cost-effective mobile-phone-based nucleic acid amplification and readout platform is broadly applicable to other real-time nucleic acid amplification tests by similarly modulating intercalating dye performance and is compatible with any fluorescence-based assay that can be run in a 96-well microplate format, making it esp. valuable for POC and resource-limited settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHpYm7oXzpoLVg90H21EOLACvtfcHk0lgphB-bpaMV3Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXjtlCnu70%253D&amp;md5=0d86e01ccb93d7680e9f97a31bacb09f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facsnano.6b08274&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsnano.6b08274%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DJ.%2BE.%26aulast%3DWei%26aufirst%3DQ.%26aulast%3DTseng%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DPan%26aufirst%3DE.%26aulast%3DLewinski%26aufirst%3DM.%26aulast%3DGarner%26aufirst%3DO.%2BB.%26aulast%3DOzcan%26aufirst%3DA.%26aulast%3DDi%2BCarlo%26aufirst%3DD.%26atitle%3DHighly%2520Stable%2520and%2520Sensitive%2520Nucleic%2520Acid%2520Amplification%2520and%2520Cell-Phone-Based%2520Readout%26jtitle%3DACS%2520Nano%26date%3D2017%26volume%3D11%26spage%3D2934%26epage%3D2943%26doi%3D10.1021%2Facsnano.6b08274" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azizi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaferani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheong, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbaspourrad, A.</span></span> <span> </span><span class="NLM_article-title">Pathogenic Bacteria Detection Using RNA-Based Loop-Mediated Isothermal-Amplification-Assisted Nucleic Acid Amplification via Droplet Microfluidics</span>. <i>ACS Sens.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">848</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acssensors.8b01206</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acssensors.8b01206" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC1MXls1OgsLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2019&amp;pages=841-848&amp;author=M.+Aziziauthor=M.+Zaferaniauthor=S.+H.+Cheongauthor=A.+Abbaspourrad&amp;title=Pathogenic+Bacteria+Detection+Using+RNA-Based+Loop-Mediated+Isothermal-Amplification-Assisted+Nucleic+Acid+Amplification+via+Droplet+Microfluidics&amp;doi=10.1021%2Facssensors.8b01206" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Pathogenic Bacteria Detection Using RNA-Based Loop-Mediated Isothermal-Amplification-Assisted Nucleic Acid Amplification via Droplet Microfluidics</span></div><div class="casAuthors">Azizi, Morteza; Zaferani, Meisam; Cheong, Soon Hon; Abbaspourrad, Alireza</div><div class="citationInfo"><span class="NLM_cas:title">ACS Sensors</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">841-848</span>CODEN:
                <span class="NLM_cas:coden">ASCEFJ</span>;
        ISSN:<span class="NLM_cas:issn">2379-3694</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nucleic acid amplifications, such as PCR, are very beneficial for diagnostic applications, esp. in the context of bacterial or viral outbreaks due to their high specificity and sensitivity.  However, the need for bulky instrumentation and complicated protocols makes these methods expensive and slow, particularly for low nos. of RNA or DNA templates.  In addn., implementing conventional nucleic acid amplification in a high-throughput manner is both reagent- and time-consuming.  We bring droplet-based microfluidics and loop-mediated isothermal amplification (LAMP) together in an optimized operational condition to provide a sensitive biosensor for amplifying extd. RNA templates for the detection of Salmonella typhimurium (targeting the invA gene).  By simultaneously performing ∼106 LAMP-assisted amplification reactions in picoliter-sized droplets and applying a new math. model for the no. of droplets necessary to screen for the first pos. droplet, we study the detection limit of our platform with pure culture and real samples (bacterial contaminated milk samples).  Our LAMP-assisted droplet-based microfluidic technique was simple in operation, sensitive, specific, and rapid for the detection of pathogenic bacteria Salmonella typhimurium in comparison with well-established conventional methods.  More importantly, the high-throughput nature of this technique makes it suitable for many applications in biol. assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnVJ7oJxUmcbVg90H21EOLACvtfcHk0lhKntf6L2IOCw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXls1OgsLw%253D&amp;md5=240c687d807abc0860416d16cb5d6c79</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facssensors.8b01206&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facssensors.8b01206%26sid%3Dliteratum%253Aachs%26aulast%3DAzizi%26aufirst%3DM.%26aulast%3DZaferani%26aufirst%3DM.%26aulast%3DCheong%26aufirst%3DS.%2BH.%26aulast%3DAbbaspourrad%26aufirst%3DA.%26atitle%3DPathogenic%2520Bacteria%2520Detection%2520Using%2520RNA-Based%2520Loop-Mediated%2520Isothermal-Amplification-Assisted%2520Nucleic%2520Acid%2520Amplification%2520via%2520Droplet%2520Microfluidics%26jtitle%3DACS%2520Sens.%26date%3D2019%26volume%3D4%26spage%3D841%26epage%3D848%26doi%3D10.1021%2Facssensors.8b01206" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, W.</span></span> <span> </span><span class="NLM_article-title">Absolute Quantification of H5-Subtype Avian Influenza Viruses Using Droplet Digital Loop-Mediated Isothermal Amplification</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">745</span>– <span class="NLM_lpage">750</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.analchem.6b03328</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.6b03328" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFKnsrjI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=89&amp;publication_year=2017&amp;pages=745-750&amp;author=Y.+Huauthor=P.+Xuauthor=J.+Luoauthor=H.+Heauthor=W.+Du&amp;title=Absolute+Quantification+of+H5-Subtype+Avian+Influenza+Viruses+Using+Droplet+Digital+Loop-Mediated+Isothermal+Amplification&amp;doi=10.1021%2Facs.analchem.6b03328" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute Quantification of H5-Subtype Avian Influenza Viruses Using Droplet Digital Loop-Mediated Isothermal Amplification</span></div><div class="casAuthors">Hu, Yi; Xu, Peng; Luo, Jing; He, Hongxuan; Du, Wenbin</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">745-750</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human infection with avian influenza A H5N1 viruses can cause severe diseases with high mortality rate and continues to pose a significant threat to global public health.  Rapid diagnosis is needed for identifying the types of influenza viruses for making timely treatment decisions.  Here, we demonstrate abs. quantification of H5-subtype influenza viruses by digital loop-mediated isothermal amplification (dLAMP) on our recently developed cross-interface emulsification (XiE) method.  Our results show that XiE-based dLAMP is highly specific and displays comparable sensitivity to real-time PCR (qPCR) and digital PCR (dPCR).  Notably, dLAMP is more tolerant to inhibitory substances than PCR methods and demonstrated similar detection efficiency to qPCR for real H5N1 samples.  Therefore, it can serve as a robust and precise alternative to qPCR or dPCR and is esp. suitable for environmental and clin. samples with hard-to-remove contaminants.  We believe that our dLAMP method offers great potential for rapid and accurate diagnosis of influenza and other infectious diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgspF_1xoxdrVg90H21EOLACvtfcHk0lhKntf6L2IOCw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XhvFKnsrjI&amp;md5=cd3a4afc1fb1c4339c4fbbe5dafa2fcc</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.6b03328&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.6b03328%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DP.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DDu%26aufirst%3DW.%26atitle%3DAbsolute%2520Quantification%2520of%2520H5-Subtype%2520Avian%2520Influenza%2520Viruses%2520Using%2520Droplet%2520Digital%2520Loop-Mediated%2520Isothermal%2520Amplification%26jtitle%3DAnal.%2520Chem.%26date%3D2017%26volume%3D89%26spage%3D745%26epage%3D750%26doi%3D10.1021%2Facs.analchem.6b03328" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, M. R.</span></span> <span> </span><span class="NLM_article-title">Digital Loop-Mediated Isothermal Amplification on a Commercial Membrane</span>. <i>ACS Sens.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">242</span>– <span class="NLM_lpage">249</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acssensors.8b01419</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acssensors.8b01419" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlymtg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2019&amp;pages=242-249&amp;author=X.+Linauthor=X.+Huangauthor=K.+Urmannauthor=X.+Xieauthor=M.+R.+Hoffmann&amp;title=Digital+Loop-Mediated+Isothermal+Amplification+on+a+Commercial+Membrane&amp;doi=10.1021%2Facssensors.8b01419" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Digital Loop-Mediated Isothermal Amplification on a Commercial Membrane</span></div><div class="casAuthors">Lin, Xingyu; Huang, Xiao; Urmann, Katharina; Xie, Xing; Hoffmann, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Sensors</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">242-249</span>CODEN:
                <span class="NLM_cas:coden">ASCEFJ</span>;
        ISSN:<span class="NLM_cas:issn">2379-3694</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this work, we report digital loop-mediated isothermal amplification (LAMP) or reverse-transcription LAMP (RT-LAMP) on a com. membrane, without the need for complex chip fabrication or use of specialized equipment.  Due to the pore size distribution, the theor. error for digital LAMP on these membranes was analyzed, using a combination of Random Distribution Model and Multivolume Theory.  A facile peel-off process was developed for effective droplet formation on the com. track-etched polycarbonate (PCTE) membrane.  Each pore functions as an individual nanoreactor for single DNA amplification.  Abs. quantification of bacteria genomic DNA was realized with a dynamic range from 11 to 1.1 × 105 copies/μL.  One-step digital RT-LAMP was also successfully performed on the membrane for the quantification of MS2 virus in wastewater.  With the introduction of new probes, the pos. pores can be easily distinguished from neg. ones with 100 times difference in fluorescence intensities.  Finally, the cost of a disposable membrane is less than $0.10/piece, which, to the best of our knowledge, is the most inexpensive way to perform digital LAMP.  The membrane system offers opportunities for point-of-care users or common labs. to perform digital quantification, single cell anal., or other bioassays in an inexpensive, flexible, and simplified way.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeWOqs6B7QU7Vg90H21EOLACvtfcHk0lhKntf6L2IOCw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXitlymtg%253D%253D&amp;md5=a3ba57624ee73da474dd546a5ba7d5d3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facssensors.8b01419&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facssensors.8b01419%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DUrmann%26aufirst%3DK.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DHoffmann%26aufirst%3DM.%2BR.%26atitle%3DDigital%2520Loop-Mediated%2520Isothermal%2520Amplification%2520on%2520a%2520Commercial%2520Membrane%26jtitle%3DACS%2520Sens.%26date%3D2019%26volume%3D4%26spage%3D242%26epage%3D249%26doi%3D10.1021%2Facssensors.8b01419" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rane, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zec, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span> <span> </span><span class="NLM_article-title">Microfluidic Continuous Flow Digital Loop-Mediated Isothermal Amplification (LAMP)</span>. <i>Lab Chip</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">776</span>– <span class="NLM_lpage">782</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C4LC01158A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1039%2FC4LC01158A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=25431886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2ksrrJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=2015&amp;pages=776-782&amp;author=T.+D.+Raneauthor=L.+Chenauthor=H.+C.+Zecauthor=T.+Wang&amp;title=Microfluidic+Continuous+Flow+Digital+Loop-Mediated+Isothermal+Amplification+%28LAMP%29&amp;doi=10.1039%2FC4LC01158A" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Microfluidic continuous flow digital loop-mediated isothermal amplification (LAMP)</span></div><div class="casAuthors">Rane, Tushar D.; Chen, Liben; Zec, Helena C.; Wang, Tza-Huei</div><div class="citationInfo"><span class="NLM_cas:title">Lab on a Chip</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">776-782</span>CODEN:
                <span class="NLM_cas:coden">LCAHAM</span>;
        ISSN:<span class="NLM_cas:issn">1473-0189</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Digital nucleic acid detection is rapidly becoming a popular technique for ultra-sensitive and quant. detection of nucleic acid mols. in a wide range of biomedical studies.  Digital polymerase chain reaction (PCR) remains the most popular way of conducting digital nucleic acid detection.  However, due to the need for thermocycling, digital PCR is difficult to implement in a streamlined manner on a single microfluidic device, leading to complex fragmented workflows and multiple sep. devices and instruments.  Loop-mediated isothermal amplification (LAMP) is an excellent isothermal alternative to PCR with potentially better specificity than PCR because of the use of multiple primer sets for a nucleic acid target.  Here we report a microfluidic droplet device implementing all the steps required for digital nucleic acid detection including droplet generation, incubation and in-line detection for digital LAMP.  As compared to microchamber or droplet array-based digital assays, the continuous flow operation of this device eliminates the constraints on the no. of total reactions imposed by the footprint of the device and the anal. throughput caused by the time for lengthy incubation and transfer of materials between instruments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6yA8HesOYebVg90H21EOLACvtfcHk0lhKntf6L2IOCw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2ksrrJ&amp;md5=b4241832af6312d8e2daf3f7ba889999</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC4LC01158A&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4LC01158A%26sid%3Dliteratum%253Aachs%26aulast%3DRane%26aufirst%3DT.%2BD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DZec%26aufirst%3DH.%2BC.%26aulast%3DWang%26aufirst%3DT.%26atitle%3DMicrofluidic%2520Continuous%2520Flow%2520Digital%2520Loop-Mediated%2520Isothermal%2520Amplification%2520%2528LAMP%2529%26jtitle%3DLab%2520Chip%26date%3D2015%26volume%3D15%26spage%3D776%26epage%3D782%26doi%3D10.1039%2FC4LC01158A" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csurka, G.</span>; <span class="NLM_string-name">Dance, C. R.</span>; <span class="NLM_string-name">Fan, L.</span>; <span class="NLM_string-name">Willamowski, J.</span>; <span class="NLM_string-name">Bray, C.</span></span> <span> </span><span class="NLM_article-title">Visual Categorization with Bags of Keypoints</span>. In  <i>Workshop on Statistical Learning in Computer Vision, ECCV</i> <span class="NLM_year">2004</span>; pp  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">22</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2004&amp;pages=1-22&amp;author=G.+Csurka&amp;author=C.+R.+Dance&amp;author=L.+Fan&amp;author=J.+Willamowski&amp;author=C.+Bray&amp;title=Visual+Categorization+with+Bags+of+Keypoints" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DCsurka%26aufirst%3DG.%26atitle%3DVisual%2520Categorization%2520with%2520Bags%2520of%2520Keypoints%26jtitle%3DWorkshop%2520on%2520Statistical%2520Learning%2520in%2520Computer%2520Vision%252C%2520ECCV%26date%3D2004%26spage%3D1%26epage%3D22" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riche, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brutchey, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malmstadt, N.</span></span> <span> </span><span class="NLM_article-title">Flow Invariant Droplet Formation for Stable Parallel Microreactors</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">10780</span> <span class="refDoi">&nbsp;DOI: 10.1038/ncomms10780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1038%2Fncomms10780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=26902825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjt1ehtb8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2016&amp;author=C.+T.+Richeauthor=E.+J.+Robertsauthor=M.+Guptaauthor=R.+L.+Brutcheyauthor=N.+Malmstadt&amp;title=Flow+Invariant+Droplet+Formation+for+Stable+Parallel+Microreactors&amp;doi=10.1038%2Fncomms10780" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Flow invariant droplet formation for stable parallel microreactors</span></div><div class="casAuthors">Riche, Carson T.; Roberts, Emily J.; Gupta, Malancha; Brutchey, Richard L.; Malmstadt, Noah</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10780</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The translation of batch chemistries onto continuous flow platforms requires addressing the issues of consistent fluidic behavior, channel fouling and high-throughput processing.  Droplet microfluidic technologies reduce channel fouling and provide an improved level of control over heat and mass transfer to control reaction kinetics.  However, in conventional geometries, the droplet size is sensitive to changes in flow rates.  Here we report a three-dimensional droplet generating device that exhibits flow invariant behavior and is robust to fluctuations in flow rate.  In addn., the droplet generator is capable of producing droplet vols. spanning four orders of magnitude.  We apply this device in a parallel network to synthesize platinum nanoparticles using an ionic liq. solvent, demonstrate reproducible synthesis after recycling the ionic liq., and double the reaction yield compared with an analogous batch synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7KSkRc6kOcbVg90H21EOLACvtfcHk0lgx7ldwiPaqcw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28Xjt1ehtb8%253D&amp;md5=448855cefab1324816d72c3a3f6e31e5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fncomms10780&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10780%26sid%3Dliteratum%253Aachs%26aulast%3DRiche%26aufirst%3DC.%2BT.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DGupta%26aufirst%3DM.%26aulast%3DBrutchey%26aufirst%3DR.%2BL.%26aulast%3DMalmstadt%26aufirst%3DN.%26atitle%3DFlow%2520Invariant%2520Droplet%2520Formation%2520for%2520Stable%2520Parallel%2520Microreactors%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26doi%3D10.1038%2Fncomms10780" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kruschke, J. K.</span></span> <span> </span><span class="NLM_article-title">Doing Bayesian Data Analysis: A Tutorial with R, JAGS, and Stan</span>. <span class="NLM_publisher-name">Academic Press</span>, p 776.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=Kruschke%2C+J.+K.+Doing+Bayesian+Data+Analysis%3A+A+Tutorial+with+R%2C+JAGS%2C+and+Stan.+Academic+Press%2C+p+776." target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DKruschke%26aufirst%3DJ.%2BK.%26atitle%3DDoing%2520Bayesian%2520Data%2520Analysis%253A%2520A%2520Tutorial%2520with%2520R%252C%2520JAGS%252C%2520and%2520Stan%26pub%3DAcademic%2520Press" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krizhevsky, A.</span>; <span class="NLM_string-name">Sutskever, I.</span>; <span class="NLM_string-name">Hinton, G. E.</span></span> In  <i>ImageNet Classification with Deep Convolutional Neural Networks</i>, Proceedings of the 25th International Conference on Neural Information Processing Systems, <span class="NLM_publisher-name">NIPS</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">1097</span>– <span class="NLM_lpage">1105</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2012&amp;pages=1097-1105&amp;author=A.+Krizhevsky&amp;author=I.+Sutskever&amp;author=G.+E.+Hinton&amp;title=ImageNet+Classification+with+Deep+Convolutional+Neural+Networks" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DKrizhevsky%26aufirst%3DA.%26jtitle%3DImageNet%2520Classification%2520with%2520Deep%2520Convolutional%2520Neural%2520Networks%26pub%3DNIPS%26date%3D2012%26spage%3D1097%26epage%3D1105" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bay, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ess, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuytelaars, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gool, L.</span></span> <span> </span><span class="NLM_article-title">Speeded-Up Robust Features (SURF)</span>. <i>Comput. Vis. Image Underst.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">346</span>– <span class="NLM_lpage">359</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cviu.2007.09.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.cviu.2007.09.014" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=110&amp;publication_year=2008&amp;pages=346-359&amp;author=H.+Bayauthor=A.+Essauthor=T.+Tuytelaarsauthor=L.+Van+Gool&amp;title=Speeded-Up+Robust+Features+%28SURF%29&amp;doi=10.1016%2Fj.cviu.2007.09.014" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.cviu.2007.09.014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cviu.2007.09.014%26sid%3Dliteratum%253Aachs%26aulast%3DBay%26aufirst%3DH.%26aulast%3DEss%26aufirst%3DA.%26aulast%3DTuytelaars%26aufirst%3DT.%26aulast%3DVan%2BGool%26aufirst%3DL.%26atitle%3DSpeeded-Up%2520Robust%2520Features%2520%2528SURF%2529%26jtitle%3DComput.%2520Vis.%2520Image%2520Underst.%26date%3D2008%26volume%3D110%26spage%3D346%26epage%3D359%26doi%3D10.1016%2Fj.cviu.2007.09.014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Breiman, L.</span></span> <span> </span><span class="NLM_article-title">Random Forests</span>. <i>Mach. Learn.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">32</span>, <span class="refDoi">&nbsp;DOI: 10.1023/A:1010933404324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1023%2FA%3A1010933404324" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=2001&amp;pages=5-32&amp;author=L.+Breiman&amp;title=Random+Forests&amp;doi=10.1023%2FA%3A1010933404324" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1023%2FA%3A1010933404324&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1010933404324%26sid%3Dliteratum%253Aachs%26aulast%3DBreiman%26aufirst%3DL.%26atitle%3DRandom%2520Forests%26jtitle%3DMach.%2520Learn.%26date%3D2001%26volume%3D45%26spage%3D5%26epage%3D32%26doi%3D10.1023%2FA%3A1010933404324" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strobl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulesteix, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kneib, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeileis, A.</span></span> <span> </span><span class="NLM_article-title">Conditional Variable Importance for Random Forests</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">307</span> <span class="refDoi">&nbsp;DOI: 10.1186/1471-2105-9-307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1186%2F1471-2105-9-307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=18620558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A280%3ADC%252BD1cvos1Crug%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2008&amp;author=C.+Stroblauthor=A.-L.+Boulesteixauthor=T.+Kneibauthor=T.+Augustinauthor=A.+Zeileis&amp;title=Conditional+Variable+Importance+for+Random+Forests&amp;doi=10.1186%2F1471-2105-9-307" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Conditional variable importance for random forests</span></div><div class="casAuthors">Strobl Carolin; Boulesteix Anne-Laure; Kneib Thomas; Augustin Thomas; Zeileis Achim</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">307</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Random forests are becoming increasingly popular in many scientific fields because they can cope with "small n large p" problems, complex interactions and even highly correlated predictor variables.  Their variable importance measures have recently been suggested as screening tools for, e.g., gene expression studies.  However, these variable importance measures show a bias towards correlated predictor variables.  RESULTS:  We identify two mechanisms responsible for this finding: (i) A preference for the selection of correlated predictors in the tree building process and (ii) an additional advantage for correlated predictor variables induced by the unconditional permutation scheme that is employed in the computation of the variable importance measure.  Based on these considerations we develop a new, conditional permutation scheme for the computation of the variable importance measure.  CONCLUSION:  The resulting conditional variable importance reflects the true impact of each predictor variable more reliably than the original marginal approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQNtCwFapSCFwwxfk_AGB-EfW6udTcc2ea28VBURKfX9Lntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BD1cvos1Crug%253D%253D&amp;md5=ce09e6988b2765e0f6d060f54fd766b6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-9-307&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-9-307%26sid%3Dliteratum%253Aachs%26aulast%3DStrobl%26aufirst%3DC.%26aulast%3DBoulesteix%26aufirst%3DA.-L.%26aulast%3DKneib%26aufirst%3DT.%26aulast%3DAugustin%26aufirst%3DT.%26aulast%3DZeileis%26aufirst%3DA.%26atitle%3DConditional%2520Variable%2520Importance%2520for%2520Random%2520Forests%26jtitle%3DBMC%2520Bioinf.%26date%3D2008%26volume%3D9%26doi%3D10.1186%2F1471-2105-9-307" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feizi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, P.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kandukuri, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Wide-Field Computational Imaging of Pathology Slides Using Lens-Free on-Chip Microscopy</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">267ra175</span>, <span class="refDoi">&nbsp;DOI: 10.1126/scitranslmed.3009850</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1126%2Fscitranslmed.3009850" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=25520396" title="">PubMed</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2014&amp;pages=267ra175&amp;author=A.+Greenbaumauthor=Y.+Zhangauthor=A.+Feiziauthor=P.-L.+Chungauthor=W.+Luoauthor=S.+R.+Kandukuriauthor=A.+Ozcan&amp;title=Wide-Field+Computational+Imaging+of+Pathology+Slides+Using+Lens-Free+on-Chip+Microscopy&amp;doi=10.1126%2Fscitranslmed.3009850" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3009850&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3009850%26sid%3Dliteratum%253Aachs%26aulast%3DGreenbaum%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DFeizi%26aufirst%3DA.%26aulast%3DChung%26aufirst%3DP.-L.%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DKandukuri%26aufirst%3DS.%2BR.%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DWide-Field%2520Computational%2520Imaging%2520of%2520Pathology%2520Slides%2520Using%2520Lens-Free%2520on-Chip%2520Microscopy%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2014%26volume%3D6%26spage%3D267ra175%26doi%3D10.1126%2Fscitranslmed.3009850" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, T.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göröcs, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isikman, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coskun, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudanyali, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Imaging without Lenses: Achievements and Remaining Challenges of Wide-Field on-Chip Microscopy</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">889</span>– <span class="NLM_lpage">895</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nmeth.2114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1038%2Fnmeth.2114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=22936170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ynt7rL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2012&amp;pages=889-895&amp;author=A.+Greenbaumauthor=W.+Luoauthor=T.-W.+Suauthor=Z.+G%C3%B6r%C3%B6csauthor=L.+Xueauthor=S.+O.+Isikmanauthor=A.+F.+Coskunauthor=O.+Mudanyaliauthor=A.+Ozcan&amp;title=Imaging+without+Lenses%3A+Achievements+and+Remaining+Challenges+of+Wide-Field+on-Chip+Microscopy&amp;doi=10.1038%2Fnmeth.2114" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Imaging without lenses: achievements and remaining challenges of wide-field on-chip microscopy</span></div><div class="casAuthors">Greenbaum, Alon; Luo, Wei; Su, Ting-Wei; Goeroecs, Zoltan; Xue, Liang; Isikman, Serhan O.; Coskun, Ahmet F.; Mudanyali, Onur; Ozcan, Aydogan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">889-895</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We discuss unique features of lens-free computational imaging tools and report some of their emerging results for wide-field on-chip microscopy, such as the achievement of a numerical aperture (NA) of ∼0.8-0.9 across a field of view (FOV) of more than 20 mm2 or an NA of ∼0.1 across a FOV of ∼18 cm2, which corresponds to an image with more than 1.5 gigapixels.  We also discuss the current challenges that these computational on-chip microscopes face, shedding light on their future directions and applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPnRkgHYMA3bVg90H21EOLACvtfcHk0lhnOHQ6eWG8AA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38Xht1ynt7rL&amp;md5=bf5f3f120d6eb14a5b3619bd599d45d0</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2114&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2114%26sid%3Dliteratum%253Aachs%26aulast%3DGreenbaum%26aufirst%3DA.%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DSu%26aufirst%3DT.-W.%26aulast%3DG%25C3%25B6r%25C3%25B6cs%26aufirst%3DZ.%26aulast%3DXue%26aufirst%3DL.%26aulast%3DIsikman%26aufirst%3DS.%2BO.%26aulast%3DCoskun%26aufirst%3DA.%2BF.%26aulast%3DMudanyali%26aufirst%3DO.%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DImaging%2520without%2520Lenses%253A%2520Achievements%2520and%2520Remaining%2520Challenges%2520of%2520Wide-Field%2520on-Chip%2520Microscopy%26jtitle%3DNat.%2520Methods%26date%3D2012%26volume%3D9%26spage%3D889%26epage%3D895%26doi%3D10.1038%2Fnmeth.2114" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Zee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riche, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tofig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallaher, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damoiseaux, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Carlo, D.</span></span> <span> </span><span class="NLM_article-title">A Gelatin Microdroplet Platform for High-Throughput Sorting of Hyperproducing Single-Cell-Derived Microalgal Clones</span>. <i>Small</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>, <span class="NLM_elocation-id">1803315</span> <span class="refDoi">&nbsp;DOI: 10.1002/smll.201803315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1002%2Fsmll.201803315" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=2018&amp;author=M.+Liauthor=M.+van+Zeeauthor=C.+T.+Richeauthor=B.+Tofigauthor=S.+D.+Gallaherauthor=S.+S.+Merchantauthor=R.+Damoiseauxauthor=K.+Godaauthor=D.+Di+Carlo&amp;title=A+Gelatin+Microdroplet+Platform+for+High-Throughput+Sorting+of+Hyperproducing+Single-Cell-Derived+Microalgal+Clones&amp;doi=10.1002%2Fsmll.201803315" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fsmll.201803315&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fsmll.201803315%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DM.%26aulast%3Dvan%2BZee%26aufirst%3DM.%26aulast%3DRiche%26aufirst%3DC.%2BT.%26aulast%3DTofig%26aufirst%3DB.%26aulast%3DGallaher%26aufirst%3DS.%2BD.%26aulast%3DMerchant%26aufirst%3DS.%2BS.%26aulast%3DDamoiseaux%26aufirst%3DR.%26aulast%3DGoda%26aufirst%3DK.%26aulast%3DDi%2BCarlo%26aufirst%3DD.%26atitle%3DA%2520Gelatin%2520Microdroplet%2520Platform%2520for%2520High-Throughput%2520Sorting%2520of%2520Hyperproducing%2520Single-Cell-Derived%2520Microalgal%2520Clones%26jtitle%3DSmall%26date%3D2018%26volume%3D14%26doi%3D10.1002%2Fsmll.201803315" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Rutte, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Carlo, D.</span></span> <span> </span><span class="NLM_article-title">Scalable High-Throughput Production of Modular Microgels for In Situ Assembly of Microporous Tissue Scaffolds</span>. <i>Adv. Funct. Mater.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>, <span class="NLM_elocation-id">1900071</span> <span class="refDoi">&nbsp;DOI: 10.1002/adfm.201900071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1002%2Fadfm.201900071" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=2019&amp;author=J.+M.+de%0ARutteauthor=J.+Kohauthor=D.+Di+Carlo&amp;title=Scalable+High-Throughput+Production+of+Modular+Microgels+for+In+Situ+Assembly+of+Microporous+Tissue+Scaffolds&amp;doi=10.1002%2Fadfm.201900071" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fadfm.201900071&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fadfm.201900071%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BRutte%26aufirst%3DJ.%2BM.%26aulast%3DKoh%26aufirst%3DJ.%26aulast%3DDi%2BCarlo%26aufirst%3DD.%26atitle%3DScalable%2520High-Throughput%2520Production%2520of%2520Modular%2520Microgels%2520for%2520In%2520Situ%2520Assembly%2520of%2520Microporous%2520Tissue%2520Scaffolds%26jtitle%3DAdv.%2520Funct.%2520Mater.%26date%3D2019%26volume%3D29%26doi%3D10.1002%2Fadfm.201900071" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sikora, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Field-Portable Wide-Field Microscopy of Dense Samples Using Multi-Height Pixel Super-Resolution Based Lensfree Imaging</span>. <i>Lab Chip</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1242</span>– <span class="NLM_lpage">1245</span>, <span class="refDoi">&nbsp;DOI: 10.1039/c2lc21072j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1039%2Fc2lc21072j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=22334329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlKrsbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2012&amp;pages=1242-1245&amp;author=A.+Greenbaumauthor=U.+Sikoraauthor=A.+Ozcan&amp;title=Field-Portable+Wide-Field+Microscopy+of+Dense+Samples+Using+Multi-Height+Pixel+Super-Resolution+Based+Lensfree+Imaging&amp;doi=10.1039%2Fc2lc21072j" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Field-portable wide-field microscopy of dense samples using multi-height pixel super-resolution based lensfree imaging</span></div><div class="casAuthors">Greenbaum, Alon; Sikora, Uzair; Ozcan, Aydogan</div><div class="citationInfo"><span class="NLM_cas:title">Lab on a Chip</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1242-1245</span>CODEN:
                <span class="NLM_cas:coden">LCAHAM</span>;
        ISSN:<span class="NLM_cas:issn">1473-0189</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We report a field-portable lensfree microscope that can image dense and connected specimens with sub-micron resoln. over a large field-of-view of ∼30 mm2 (i.e., ∼6.4 mm × ∼4.6 mm) using pixel super-resoln. and iterative phase recovery techniques.  Weighing ∼122 g with dimensions of 4 cm × 4 cm × 15 cm, this microscope records lensfree in-line holograms of specimens onto an opto-electronic sensor-array using partially coherent illumination.  To reconstruct the phase and amplitude images of dense samples (with &gt;0.3 billion pixels in each image, i.e., &gt;0.6 billion pixels total), we employ a multi-height imaging approach, where by using a mech. interface the sensor-to-sample distance is dynamically changed by random discrete steps of e.g., ∼10 to 80 μm.  By digitally propagating back and forth between these multi-height super-resolved holograms (corresponding to typically 2-5 planes), phase and amplitude images of dense samples can be recovered without the need for any spatial masks or filtering.  We demonstrate the performance of this field-portable multi-height lensfree microscope by imaging Papanicolaou smears (also known as Pap tests).  Our results reveal the promising potential of this multi-height lensfree computational microscopy platform for e.g., pathol. needs in resource limited settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpL9HxR3wM-J7Vg90H21EOLACvtfcHk0lhnOHQ6eWG8AA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XjtlKrsbs%253D&amp;md5=43e58a212d771907c9fec580f24ad6c5</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1039%2Fc2lc21072j&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2lc21072j%26sid%3Dliteratum%253Aachs%26aulast%3DGreenbaum%26aufirst%3DA.%26aulast%3DSikora%26aufirst%3DU.%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DField-Portable%2520Wide-Field%2520Microscopy%2520of%2520Dense%2520Samples%2520Using%2520Multi-Height%2520Pixel%2520Super-Resolution%2520Based%2520Lensfree%2520Imaging%26jtitle%3DLab%2520Chip%26date%3D2012%26volume%3D12%26spage%3D1242%26epage%3D1245%26doi%3D10.1039%2Fc2lc21072j" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aidukas, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konda, P. C.</span></span> <span> </span><span class="NLM_article-title">Low-Cost, Sub-Micron Resolution, Wide-Field Computational Microscopy Using Opensource Hardware</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">7457</span> <span class="refDoi">&nbsp;DOI: 10.1038/s41598-019-43845-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1038%2Fs41598-019-43845-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=31092867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A280%3ADC%252BB3M7mtFKquw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2019&amp;author=T.+Aidukasauthor=R.+Eckertauthor=A.+R.+Harveyauthor=L.+Wallerauthor=P.+C.+Konda&amp;title=Low-Cost%2C+Sub-Micron+Resolution%2C+Wide-Field+Computational+Microscopy+Using+Opensource+Hardware&amp;doi=10.1038%2Fs41598-019-43845-9" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Low-cost, sub-micron resolution, wide-field computational microscopy using opensource hardware</span></div><div class="casAuthors">Aidukas Tomas; Harvey Andrew R; Konda Pavan C; Eckert Regina; Waller Laura</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7457</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The revolution in low-cost consumer photography and computation provides fertile opportunity for a disruptive reduction in the cost of biomedical imaging.  Conventional approaches to low-cost microscopy are fundamentally restricted, however, to modest field of view (FOV) and/or resolution.  We report a low-cost microscopy technique, implemented with a Raspberry Pi single-board computer and color camera combined with Fourier ptychography (FP), to computationally construct 25-megapixel images with sub-micron resolution.  New image-construction techniques were developed to enable the use of the low-cost Bayer color sensor, to compensate for the highly aberrated re-used camera lens and to compensate for misalignments associated with the 3D-printed microscope structure.  This high ratio of performance to cost is of particular interest to high-throughput microscopy applications, ranging from drug discovery and digital pathology to health screening in low-income countries. 3D models and assembly instructions of our microscope are made available for open source use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTK4j42ZllqepPq5LRczm8ffW6udTcc2eZZ1_PyPRmlGbntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BB3M7mtFKquw%253D%253D&amp;md5=a60ba19fb88ab6c8d1a8379aca2d25a3</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-43845-9&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-43845-9%26sid%3Dliteratum%253Aachs%26aulast%3DAidukas%26aufirst%3DT.%26aulast%3DEckert%26aufirst%3DR.%26aulast%3DHarvey%26aufirst%3DA.%2BR.%26aulast%3DWaller%26aufirst%3DL.%26aulast%3DKonda%26aufirst%3DP.%2BC.%26atitle%3DLow-Cost%252C%2520Sub-Micron%2520Resolution%252C%2520Wide-Field%2520Computational%2520Microscopy%2520Using%2520Opensource%2520Hardware%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26doi%3D10.1038%2Fs41598-019-43845-9" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yelleswarapu, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, H.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadavali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issadore, D.</span></span> <span> </span><span class="NLM_article-title">Ultra-High Throughput Detection (1 Million Droplets per Second) of Fluorescent Droplets Using a Cell Phone Camera and Time Domain Encoded Optofluidics</span>. <i>Lab Chip</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1083</span>– <span class="NLM_lpage">1094</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C6LC01489E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1039%2FC6LC01489E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=28225099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisF2qsrs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=2017&amp;pages=1083-1094&amp;author=V.+R.+Yelleswarapuauthor=H.-H.+Jeongauthor=S.+Yadavaliauthor=D.+Issadore&amp;title=Ultra-High+Throughput+Detection+%281+Million+Droplets+per+Second%29+of+Fluorescent+Droplets+Using+a+Cell+Phone+Camera+and+Time+Domain+Encoded+Optofluidics&amp;doi=10.1039%2FC6LC01489E" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Ultra-high throughput detection (1 million droplets per second) of fluorescent droplets using a cell phone camera and time domain encoded optofluidics</span></div><div class="casAuthors">Yelleswarapu, Venkata R.; Jeong, Heon-Ho; Yadavali, Sagar; Issadore, David</div><div class="citationInfo"><span class="NLM_cas:title">Lab on a Chip</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1083-1094</span>CODEN:
                <span class="NLM_cas:coden">LCAHAM</span>;
        ISSN:<span class="NLM_cas:issn">1473-0189</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Droplet-based assays-in which ultra-sensitive mol. measurements are made by performing millions of parallel expts. in picoliter droplets-have generated enormous enthusiasm due to their single mol. resoln. and robustness to reaction conditions.  These assays have great untapped potential for point of care diagnostics but are currently confined to lab. settings due to the instrumentation necessary to serially generate, control, and measure tens of millions of droplets.  To address this challenge, we have developed the microdroplet megascale detector (μMD) that can generate and detect the fluorescence of millions of droplets per s (1000× faster than conventional approaches) using only a conventional cell phone camera.  The key innovation of our approach is borrowed from the telecommunications industry, wherein we modulate the excitation light with a pseudorandom sequence that enables individual droplets to be resolved that would otherwise overlap due to the limited frame rate of digital cameras.  Using this approach, the μMD measures droplets at a rate of 106 droplets per s (.vphi. = 166 mL h-1) in 120 parallel microfluidic channels and achieves a limit of detection LOD = 1 μM Rhodamine dye, sufficient for typical droplet based assays.  We incorporate this new droplet detection technol. with our previously reported parallelized droplet prodn. strategy, incorporating 200 parallel droplet makers and only one set of continuous and droplet phase inputs and one output line.  By miniaturizing and integrating droplet based diagnostics into a handheld format, the μMD platform can translate ultra-sensitive droplet based assays into a self-contained platform for practical use in clin. and industrial settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJSqAgLzjGnLVg90H21EOLACvtfcHk0lgeD5gbdSyz_Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXisF2qsrs%253D&amp;md5=b6f7dc4f3afde3739f89c6c2c47911e5</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1039%2FC6LC01489E&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6LC01489E%26sid%3Dliteratum%253Aachs%26aulast%3DYelleswarapu%26aufirst%3DV.%2BR.%26aulast%3DJeong%26aufirst%3DH.-H.%26aulast%3DYadavali%26aufirst%3DS.%26aulast%3DIssadore%26aufirst%3DD.%26atitle%3DUltra-High%2520Throughput%2520Detection%2520%25281%2520Million%2520Droplets%2520per%2520Second%2529%2520of%2520Fluorescent%2520Droplets%2520Using%2520a%2520Cell%2520Phone%2520Camera%2520and%2520Time%2520Domain%2520Encoded%2520Optofluidics%26jtitle%3DLab%2520Chip%26date%3D2017%26volume%3D17%26spage%3D1083%26epage%3D1094%26doi%3D10.1039%2FC6LC01489E" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yelleswarapu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haber, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inapuri, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issadore, D.</span></span> <span> </span><span class="NLM_article-title">Mobile Platform for Rapid Sub–Picogram-per-Milliliter, Multiplexed, Digital Droplet Detection of Proteins</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">4489</span>– <span class="NLM_lpage">4495</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1814110116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1073%2Fpnas.1814110116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=30765530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC1MXktFejur4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=116&amp;publication_year=2019&amp;pages=4489-4495&amp;author=V.+Yelleswarapuauthor=J.+R.+Buserauthor=M.+Haberauthor=J.+Baronauthor=E.+Inapuriauthor=D.+Issadore&amp;title=Mobile+Platform+for+Rapid+Sub%E2%80%93Picogram-per-Milliliter%2C+Multiplexed%2C+Digital+Droplet+Detection+of+Proteins&amp;doi=10.1073%2Fpnas.1814110116" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Mobile platform for rapid subpicogram-per-milliliter, multiplexed, digital droplet detection of proteins</span></div><div class="casAuthors">Yelleswarapu, Venkata; Buser, Joshua R.; Haber, Margalit; Baron, Jonathan; Inapuri, Eshwar; Issadore, David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4489-4495</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Digital droplet assays in which biol. samples are compartmentalized into millions of femtoliter-vol. droplets and interrogated individually have generated enormous enthusiasm for their ability to detect biomarkers with single-mol. sensitivity.  These assays have untapped potential for point-of-care diagnostics but are currently mainly confined to lab. settings, due to the instrumentation necessary to serially generate, control, and measure tens of millions of droplets/compartments.  To address this challenge, the authors developed an optofluidic platform that miniaturizes digital assays into a mobile format by parallelizing their operation.  This technol. is based on three key innovations: (1) the integration and parallel operation of a hundred droplet generators onto a single chip that operates and gt;100A~, faster than a single droplet generator, (2) the fluorescence detection of droplets at and gt;100A~, faster than conventional in-flow detection using time domain-encoded mobile phone imaging, and (3) the integration of on-chip delay lines and sample processing to allow serum-to-answer device operation.  To demonstrate the power of this approach, the authors performed a duplex digital ELISA.  The authors characterized the performance of this assay by first using spiked recombinant proteins in a complex media (FBS) and measured a limit of detection, 0.004 pg/mL (300 aM), a 1.000 A~, improvement over std. ELISA and matching that of the existing lab.-based gold std. digital ELISA system.  The authors addnl. measured endogenous GM-CSF and IL6 in human serum from n = 14 human subjects using the authors' mobile duplex assay, and showed excellent agreement with the gold std. system (R2 = 0.96).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5NsgeNcFLmrVg90H21EOLACvtfcHk0lgeD5gbdSyz_Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXktFejur4%253D&amp;md5=56b615d3f856bfb6fa2016223f83d333</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1814110116&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1814110116%26sid%3Dliteratum%253Aachs%26aulast%3DYelleswarapu%26aufirst%3DV.%26aulast%3DBuser%26aufirst%3DJ.%2BR.%26aulast%3DHaber%26aufirst%3DM.%26aulast%3DBaron%26aufirst%3DJ.%26aulast%3DInapuri%26aufirst%3DE.%26aulast%3DIssadore%26aufirst%3DD.%26atitle%3DMobile%2520Platform%2520for%2520Rapid%2520Sub%25E2%2580%2593Picogram-per-Milliliter%252C%2520Multiplexed%252C%2520Digital%2520Droplet%2520Detection%2520of%2520Proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2019%26volume%3D116%26spage%3D4489%26epage%3D4495%26doi%3D10.1073%2Fpnas.1814110116" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gossett, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, H. T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindgren, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Carlo, D.</span></span> <span> </span><span class="NLM_article-title">Hydrodynamic Stretching of Single Cells for Large Population Mechanical Phenotyping</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">7630</span>– <span class="NLM_lpage">7635</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1200107109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1073%2Fpnas.1200107109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=22547795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC38Xot1KnsL4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=109&amp;publication_year=2012&amp;pages=7630-7635&amp;author=D.+R.+Gossettauthor=H.+T.+K.+Tseauthor=S.+Leeauthor=Y.+Yingauthor=A.+G.+Lindgrenauthor=O.+O.+Yangauthor=J.+Raoauthor=A.+Clarkauthor=D.+Di+Carlo&amp;title=Hydrodynamic+Stretching+of+Single+Cells+for+Large+Population+Mechanical+Phenotyping&amp;doi=10.1073%2Fpnas.1200107109" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrodynamic stretching of single cells for large population mechanical phenotyping</span></div><div class="casAuthors">Gossett, Daniel R.; Tse, Henry T. K.; Lee, Serena A.; Ying, Yong; Lindgren, Anne G.; Yang, Otto O.; Rao, Jianyu; Clark, Amander T.; Di Carlo, Dino</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7630-7635, S7630/1-S7630/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Cell state is often assayed through measurement of biochem. and biophys. markers.  Although biochem. markers were widely used, intrinsic biophys. markers, such as the ability to mech. deform under a load, are advantageous in that they do not require costly labeling or sample prepn.  However, current techniques that assay cell mech. properties have had limited adoption in clin. and cell biol. research applications.  Here, we demonstrate an automated microfluidic technol. capable of probing single-cell deformability at approx. 2,000 cells/s.  The method uses inertial focusing to uniformly deliver cells to a stretching extensional flow where cells are deformed at high strain rates, imaged with a high-speed camera, and computationally analyzed to ext. quant. parameters.  This approach allows us to analyze cells at throughputs orders of magnitude faster than previously reported biophys. flow cytometers and single-cell mechanics tools, while creating easily observable larger strains and limiting user time commitment and bias through automation.  Using this approach we rapidly assay the deformability of native populations of leukocytes and malignant cells in pleural effusions and accurately predict disease state in patients with cancer and immune activation with a sensitivity of 91% and a specificity of 86%.  As a tool for biol. research, we show the deformability we measure is an early biomarker for pluripotent stem cell differentiation and is likely linked to nuclear structural changes.  Microfluidic deformability cytometry brings the statistical accuracy of traditional flow cytometric techniques to label-free biophys. biomarkers, enabling applications in clin. diagnostics, stem cell characterization, and single-cell biophysics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqL31SCuHJC7Vg90H21EOLACvtfcHk0lixiyku-YmWNw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38Xot1KnsL4%253D&amp;md5=460b3fe2fae06b6ec42e8557c0b5be51</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1200107109&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1200107109%26sid%3Dliteratum%253Aachs%26aulast%3DGossett%26aufirst%3DD.%2BR.%26aulast%3DTse%26aufirst%3DH.%2BT.%2BK.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DYing%26aufirst%3DY.%26aulast%3DLindgren%26aufirst%3DA.%2BG.%26aulast%3DYang%26aufirst%3DO.%2BO.%26aulast%3DRao%26aufirst%3DJ.%26aulast%3DClark%26aufirst%3DA.%26aulast%3DDi%2BCarlo%26aufirst%3DD.%26atitle%3DHydrodynamic%2520Stretching%2520of%2520Single%2520Cells%2520for%2520Large%2520Population%2520Mechanical%2520Phenotyping%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D7630%26epage%3D7635%26doi%3D10.1073%2Fpnas.1200107109" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivenson, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Haan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Bright-Field Holography: Cross-Modality Deep Learning Enables Snapshot 3D Imaging with Bright-Field Contrast Using a Single Hologram</span>. <i>Light: Sci. Appl.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">25</span> <span class="refDoi">&nbsp;DOI: 10.1038/s41377-019-0139-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1038%2Fs41377-019-0139-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=30854197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A280%3ADC%252BB3cbisVCnsw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2019&amp;author=Y.+Wuauthor=Y.+Luoauthor=G.+Chaudhariauthor=Y.+Rivensonauthor=A.+Calisauthor=K.+de+Haanauthor=A.+Ozcan&amp;title=Bright-Field+Holography%3A+Cross-Modality+Deep+Learning+Enables+Snapshot+3D+Imaging+with+Bright-Field+Contrast+Using+a+Single+Hologram&amp;doi=10.1038%2Fs41377-019-0139-9" target="_blank">Google Scholar</a><div class="casRecord" id="
                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Bright-field holography: cross-modality deep learning enables snapshot 3D imaging with bright-field contrast using a single hologram</span></div><div class="casAuthors">Wu Yichen; Luo Yilin; Rivenson Yair; Calis Ayfer; de Haan Kevin; Ozcan Aydogan; Wu Yichen; Luo Yilin; Rivenson Yair; Calis Ayfer; de Haan Kevin; Ozcan Aydogan; Wu Yichen; Luo Yilin; Rivenson Yair; Calis Ayfer; de Haan Kevin; Ozcan Aydogan; Chaudhari Gunvant; Ozcan Aydogan</div><div class="citationInfo"><span class="NLM_cas:title">Light, science &amp; applications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">25</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Digital holographic microscopy enables the 3D reconstruction of volumetric samples from a single-snapshot hologram.  However, unlike a conventional bright-field microscopy image, the quality of holographic reconstructions is compromised by interference fringes as a result of twin images and out-of-plane objects.  Here, we demonstrate that cross-modality deep learning using a generative adversarial network (GAN) can endow holographic images of a sample volume with bright-field microscopy contrast, combining the volumetric imaging capability of holography with the speckle- and artifact-free image contrast of incoherent bright-field microscopy.  We illustrate the performance of this "bright-field holography" method through the snapshot imaging of bioaerosols distributed in 3D, matching the artifact-free image contrast and axial sectioning performance of a high-NA bright-field microscope.  This data-driven deep-learning-based imaging method bridges the contrast gap between coherent and incoherent imaging, and enables the snapshot 3D imaging of objects with bright-field contrast from a single hologram, benefiting from the wave-propagation framework of holography.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFk2dAVf44K1QKX3ooHxWjfW6udTcc2eYtuM_VqLZSrbntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BB3cbisVCnsw%253D%253D&amp;md5=95ef8ccf478a409fe579c753138b6e44</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fs41377-019-0139-9&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41377-019-0139-9%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DChaudhari%26aufirst%3DG.%26aulast%3DRivenson%26aufirst%3DY.%26aulast%3DCalis%26aufirst%3DA.%26aulast%3Dde%2BHaan%26aufirst%3DK.%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DBright-Field%2520Holography%253A%2520Cross-Modality%2520Deep%2520Learning%2520Enables%2520Snapshot%25203D%2520Imaging%2520with%2520Bright-Field%2520Contrast%2520Using%2520a%2520Single%2520Hologram%26jtitle%3DLight%253A%2520Sci.%2520Appl.%26date%3D2019%26volume%3D8%26doi%3D10.1038%2Fs41377-019-0139-9" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivenson, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günaydin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Extended Depth-of-Field in Holographic Imaging Using Deep-Learning-Based Autofocusing and Phase Recovery</span>. <i>Optica</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">704</span> <span class="refDoi">&nbsp;DOI: 10.1364/OPTICA.5.000704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1364%2FOPTICA.5.000704" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2018&amp;author=Y.+Wuauthor=Y.+Rivensonauthor=Y.+Zhangauthor=Z.+Weiauthor=H.+G%C3%BCnaydinauthor=X.+Linauthor=A.+Ozcan&amp;title=Extended+Depth-of-Field+in+Holographic+Imaging+Using+Deep-Learning-Based+Autofocusing+and+Phase+Recovery&amp;doi=10.1364%2FOPTICA.5.000704" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1364%2FOPTICA.5.000704&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1364%252FOPTICA.5.000704%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DRivenson%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DZ.%26aulast%3DG%25C3%25BCnaydin%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DExtended%2520Depth-of-Field%2520in%2520Holographic%2520Imaging%2520Using%2520Deep-Learning-Based%2520Autofocusing%2520and%2520Phase%2520Recovery%26jtitle%3DOptica%26date%3D2018%26volume%3D5%26doi%3D10.1364%2FOPTICA.5.000704" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atherton, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerbyson, D. J.</span></span> <span> </span><span class="NLM_article-title">Size Invariant Circle Detection</span>. <i>Image Vis. Comput.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">795</span>– <span class="NLM_lpage">803</span>, <span class="refDoi">&nbsp;DOI: 10.1016/S0262-8856(98)00160-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2FS0262-8856%2898%2900160-7" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=1999&amp;pages=795-803&amp;author=T.+J.+Athertonauthor=D.+J.+Kerbyson&amp;title=Size+Invariant+Circle+Detection&amp;doi=10.1016%2FS0262-8856%2898%2900160-7" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2FS0262-8856%2898%2900160-7&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0262-8856%252898%252900160-7%26sid%3Dliteratum%253Aachs%26aulast%3DAtherton%26aufirst%3DT.%2BJ.%26aulast%3DKerbyson%26aufirst%3DD.%2BJ.%26atitle%3DSize%2520Invariant%2520Circle%2520Detection%26jtitle%3DImage%2520Vis.%2520Comput.%26date%3D1999%26volume%3D17%26spage%3D795%26epage%3D803%26doi%3D10.1016%2FS0262-8856%2898%2900160-7" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salvatier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiecki, T. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonnesbeck, C.</span></span> <span> </span><span class="NLM_article-title">Probabilistic Programming in Python Using PyMC3</span>. <i>PeerJ Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e55</span> <span class="refDoi">&nbsp;DOI: 10.7717/peerj-cs.55</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.7717%2Fpeerj-cs.55" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2&amp;publication_year=2016&amp;author=J.+Salvatierauthor=T.+V.+Wieckiauthor=C.+Fonnesbeck&amp;title=Probabilistic+Programming+in+Python+Using+PyMC3&amp;doi=10.7717%2Fpeerj-cs.55" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.7717%2Fpeerj-cs.55&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7717%252Fpeerj-cs.55%26sid%3Dliteratum%253Aachs%26aulast%3DSalvatier%26aufirst%3DJ.%26aulast%3DWiecki%26aufirst%3DT.%2BV.%26aulast%3DFonnesbeck%26aufirst%3DC.%26atitle%3DProbabilistic%2520Programming%2520in%2520Python%2520Using%2520PyMC3%26jtitle%3DPeerJ%2520Comput.%2520Sci.%26date%3D2016%26volume%3D2%26doi%3D10.7717%2Fpeerj-cs.55" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelman, A.</span></span> <span> </span><span class="NLM_article-title">The No-U-Turn Sampler: Adaptively Setting Path Lengths in Hamiltonian Monte Carlo</span>. <i>J. Mach. Learn. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1593</span>– <span class="NLM_lpage">1623</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=2014&amp;pages=1593-1623&amp;author=M.+D.+Hoffmanauthor=A.+Gelman&amp;title=The+No-U-Turn+Sampler%3A+Adaptively+Setting+Path+Lengths+in+Hamiltonian+Monte+Carlo" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DM.%2BD.%26aulast%3DGelman%26aufirst%3DA.%26atitle%3DThe%2520No-U-Turn%2520Sampler%253A%2520Adaptively%2520Setting%2520Path%2520Lengths%2520in%2520Hamiltonian%2520Monte%2520Carlo%26jtitle%3DJ.%2520Mach.%2520Learn.%2520Res.%26date%3D2014%26volume%3D15%26spage%3D1593%26epage%3D1623" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></div><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 15 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Rongxin Fu, Wenli Du, Xiangyu Jin, Ruliang Wang, Xue Lin, Ya Su, Han Yang, Xiaohui Shan, Wenqi Lv, Zhi Zheng, <span class="NLM_string-name hlFld-ContribAuthor">Guoliang Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Microfluidic Biosensor for Rapid Nucleic Acid Quantitation Based on Hyperspectral Interferometric Amplicon-Complex Analysis. </span><span class="cited-content_cbyCitation_journal-name">ACS Sensors</span><span> <strong>2021,</strong> <em>6 </em>
                                    (11)
                                     , 4057-4066. <a href="https://doi.org/10.1021/acssensors.1c01491" title="DOI URL">https://doi.org/10.1021/acssensors.1c01491</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acssensors.1c01491&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facssensors.1c01491%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Sensors%26atitle%3DMicrofluidic%252BBiosensor%252Bfor%252BRapid%252BNucleic%252BAcid%252BQuantitation%252BBased%252Bon%252BHyperspectral%252BInterferometric%252BAmplicon-Complex%252BAnalysis%26aulast%3DFu%26aufirst%3DRongxin%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D14072021%26date%3D11102021%26date%3D25102021%26volume%3D6%26issue%3D11%26spage%3D4057%26epage%3D4066" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ilada Choopara, Akkapol Suea-Ngam, Yothin Teethaisong, Philip D. Howes, Mathias Schmelcher, Asada Leelahavanichkul, Sudaluck Thunyaharn, Doonyapong Wongsawaeng, Andrew J. deMello, Deborah Dean, <span class="NLM_string-name hlFld-ContribAuthor">Naraporn Somboonna</span>. </span><span class="cited-content_cbyCitation_article-title">Fluorometric Paper-Based, Loop-Mediated Isothermal Amplification Devices for Quantitative Point-of-Care Detection of Methicillin-Resistant Staphylococcus aureus (MRSA). </span><span class="cited-content_cbyCitation_journal-name">ACS Sensors</span><span> <strong>2021,</strong> <em>6 </em>
                                    (3)
                                     , 742-751. <a href="https://doi.org/10.1021/acssensors.0c01405" title="DOI URL">https://doi.org/10.1021/acssensors.0c01405</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acssensors.0c01405&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facssensors.0c01405%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Sensors%26atitle%3DFluorometric%252BPaper-Based%25252C%252BLoop-Mediated%252BIsothermal%252BAmplification%252BDevices%252Bfor%252BQuantitative%252BPoint-of-Care%252BDetection%252Bof%252BMethicillin-Resistant%252BStaphylococcus%252Baureus%252B%252528MRSA%252529%26aulast%3DChoopara%26aufirst%3DIlada%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D09072020%26date%3D04012021%26date%3D13012021%26volume%3D6%26issue%3D3%26spage%3D742%26epage%3D751" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yannick Rondelez, <span class="NLM_string-name hlFld-ContribAuthor">Guillaume Gines</span>. </span><span class="cited-content_cbyCitation_article-title">Multiplex Digital MicroRNA Detection Using Cross-Inhibitory DNA Circuits. </span><span class="cited-content_cbyCitation_journal-name">ACS Sensors</span><span> <strong>2020,</strong> <em>5 </em>
                                    (8)
                                     , 2430-2437. <a href="https://doi.org/10.1021/acssensors.0c00593" title="DOI URL">https://doi.org/10.1021/acssensors.0c00593</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acssensors.0c00593&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facssensors.0c00593%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Sensors%26atitle%3DMultiplex%252BDigital%252BMicroRNA%252BDetection%252BUsing%252BCross-Inhibitory%252BDNA%252BCircuits%26aulast%3DRondelez%26aufirst%3DYannick%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D24032020%26date%3D30062020%26date%3D25072020%26date%3D30062020%26volume%3D5%26issue%3D8%26spage%3D2430%26epage%3D2437" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xi Yuan, Chengming Yang, Qian He, Junhu Chen, Dongmei Yu, Jie Li, Shiyao Zhai, Zhifeng Qin, Ke Du, Zhenhai Chu, <span class="NLM_string-name hlFld-ContribAuthor">Peiwu Qin</span>. </span><span class="cited-content_cbyCitation_article-title">Current and Perspective Diagnostic Techniques for COVID-19. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2020,</strong> <em>6 </em>
                                    (8)
                                     , 1998-2016. <a href="https://doi.org/10.1021/acsinfecdis.0c00365" title="DOI URL">https://doi.org/10.1021/acsinfecdis.0c00365</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.0c00365&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.0c00365%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DCurrent%252Band%252BPerspective%252BDiagnostic%252BTechniques%252Bfor%252BCOVID-19%26aulast%3DYuan%26aufirst%3DXi%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29052020%26date%3D04082020%26date%3D17072020%26volume%3D6%26issue%3D8%26spage%3D1998%26epage%3D2016" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Linzhe  Chen</span>, <span class="hlFld-ContribAuthor ">Jingyi  Ding</span>, <span class="hlFld-ContribAuthor ">Hao  Yuan</span>, <span class="hlFld-ContribAuthor ">Chi  Chen</span>, <span class="hlFld-ContribAuthor ">Zida  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Deep‐dLAMP: Deep Learning‐Enabled Polydisperse Emulsion‐Based Digital Loop‐Mediated Isothermal Amplification. </span><span class="cited-content_cbyCitation_journal-name">Advanced Science</span><span> <strong>2022,</strong> <em>9 </em>
                                    (9)
                                     , 2105450. <a href="https://doi.org/10.1002/advs.202105450" title="DOI URL">https://doi.org/10.1002/advs.202105450</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/advs.202105450&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadvs.202105450%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Science%26atitle%3DDeep%2525E2%252580%252590dLAMP%25253A%252BDeep%252BLearning%2525E2%252580%252590Enabled%252BPolydisperse%252BEmulsion%2525E2%252580%252590Based%252BDigital%252BLoop%2525E2%252580%252590Mediated%252BIsothermal%252BAmplification%26aulast%3DChen%26aufirst%3DLinzhe%26date%3D2022%26date%3D2022%26volume%3D9%26issue%3D9%26spage%3D2105450" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mark  van Zee</span>, <span class="hlFld-ContribAuthor ">Joseph  de Rutte</span>, <span class="hlFld-ContribAuthor ">Rose  Rumyan</span>, <span class="hlFld-ContribAuthor ">Cayden  Williamson</span>, <span class="hlFld-ContribAuthor ">Trevor  Burnes</span>, <span class="hlFld-ContribAuthor ">Randor  Radakovits</span>, <span class="hlFld-ContribAuthor ">Andrew  Sonico Eugenio</span>, <span class="hlFld-ContribAuthor ">Sara  Badih</span>, <span class="hlFld-ContribAuthor ">Sohyung  Lee</span>, <span class="hlFld-ContribAuthor ">Dong-Hyun  Lee</span>, <span class="hlFld-ContribAuthor ">Maani  Archang</span>, <span class="hlFld-ContribAuthor ">Dino  Di Carlo</span>. </span><span class="cited-content_cbyCitation_article-title">High-throughput selection of cells based on accumulated growth and division using PicoShell particles. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2022,</strong> <em>119 </em>
                                    (4)
                                     <a href="https://doi.org/10.1073/pnas.2109430119" title="DOI URL">https://doi.org/10.1073/pnas.2109430119</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.2109430119&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.2109430119%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DHigh-throughput%252Bselection%252Bof%252Bcells%252Bbased%252Bon%252Baccumulated%252Bgrowth%252Band%252Bdivision%252Busing%252BPicoShell%252Bparticles%26aulast%3Dvan%2BZee%26aufirst%3DMark%26date%3D2022%26date%3D2022%26volume%3D119%26issue%3D4" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yun  Luo</span>, <span class="hlFld-ContribAuthor ">Fenglan  Lv</span>, <span class="hlFld-ContribAuthor ">Minhui  Wang</span>, <span class="hlFld-ContribAuthor ">Lixia  Lu</span>, <span class="hlFld-ContribAuthor ">Yuanjian  Liu</span>, <span class="hlFld-ContribAuthor ">Xiaohui  Xiong</span>. </span><span class="cited-content_cbyCitation_article-title">A multicolor electrochemiluminescence device based on closed bipolar electrode for rapid visual screening of Salmonella typhimurium. </span><span class="cited-content_cbyCitation_journal-name">Sensors and Actuators B: Chemical</span><span> <strong>2021,</strong> <em>349 </em>, 130761. <a href="https://doi.org/10.1016/j.snb.2021.130761" title="DOI URL">https://doi.org/10.1016/j.snb.2021.130761</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.snb.2021.130761&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.snb.2021.130761%26sid%3Dliteratum%253Aachs%26jtitle%3DSensors%2520and%2520Actuators%2520B%253A%2520Chemical%26atitle%3DA%252Bmulticolor%252Belectrochemiluminescence%252Bdevice%252Bbased%252Bon%252Bclosed%252Bbipolar%252Belectrode%252Bfor%252Brapid%252Bvisual%252Bscreening%252Bof%252BSalmonella%252Btyphimurium%26aulast%3DLuo%26aufirst%3DYun%26date%3D2021%26volume%3D349%26spage%3D130761" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiahao  Zheng</span>, <span class="hlFld-ContribAuthor ">Tim  Cole</span>, <span class="hlFld-ContribAuthor ">Yuxin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jeeson  Kim</span>, <span class="hlFld-ContribAuthor ">Shi-Yang  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Exploiting machine learning for bestowing intelligence to microfluidics. </span><span class="cited-content_cbyCitation_journal-name">Biosensors and Bioelectronics</span><span> <strong>2021,</strong> <em>194 </em>, 113666. <a href="https://doi.org/10.1016/j.bios.2021.113666" title="DOI URL">https://doi.org/10.1016/j.bios.2021.113666</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bios.2021.113666&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bios.2021.113666%26sid%3Dliteratum%253Aachs%26jtitle%3DBiosensors%2520and%2520Bioelectronics%26atitle%3DExploiting%252Bmachine%252Blearning%252Bfor%252Bbestowing%252Bintelligence%252Bto%252Bmicrofluidics%26aulast%3DZheng%26aufirst%3DJiahao%26date%3D2021%26volume%3D194%26spage%3D113666" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tingting  Han</span>, <span class="hlFld-ContribAuthor ">Hailin  Cong</span>, <span class="hlFld-ContribAuthor ">Youqing  Shen</span>, <span class="hlFld-ContribAuthor ">Bing  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in detection technologies for COVID-19. </span><span class="cited-content_cbyCitation_journal-name">Talanta</span><span> <strong>2021,</strong> <em>233 </em>, 122609. <a href="https://doi.org/10.1016/j.talanta.2021.122609" title="DOI URL">https://doi.org/10.1016/j.talanta.2021.122609</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.talanta.2021.122609&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.talanta.2021.122609%26sid%3Dliteratum%253Aachs%26jtitle%3DTalanta%26atitle%3DRecent%252Badvances%252Bin%252Bdetection%252Btechnologies%252Bfor%252BCOVID-19%26aulast%3DHan%26aufirst%3DTingting%26date%3D2021%26volume%3D233%26spage%3D122609" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yilian  Wang</span>, <span class="hlFld-ContribAuthor ">Vishwesh  Shah</span>, <span class="hlFld-ContribAuthor ">Angela  Lu</span>, <span class="hlFld-ContribAuthor ">Ella  Pachler</span>, <span class="hlFld-ContribAuthor ">Brian  Cheng</span>, <span class="hlFld-ContribAuthor ">Dino  Di Carlo</span>. </span><span class="cited-content_cbyCitation_article-title">Counting of enzymatically amplified affinity reactions in hydrogel particle-templated drops. </span><span class="cited-content_cbyCitation_journal-name">Lab on a Chip</span><span> <strong>2021,</strong> <em>21 </em>
                                    (18)
                                     , 3438-3448. <a href="https://doi.org/10.1039/D1LC00344E" title="DOI URL">https://doi.org/10.1039/D1LC00344E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1LC00344E&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1LC00344E%26sid%3Dliteratum%253Aachs%26jtitle%3DLab%2520on%2520a%2520Chip%26atitle%3DCounting%252Bof%252Benzymatically%252Bamplified%252Baffinity%252Breactions%252Bin%252Bhydrogel%252Bparticle-templated%252Bdrops%26aulast%3DWang%26aufirst%3DYilian%26date%3D2021%26date%3D2021%26volume%3D21%26issue%3D18%26spage%3D3438%26epage%3D3448" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ryan Shijie  Du</span>, <span class="hlFld-ContribAuthor ">Lily  Liu</span>, <span class="hlFld-ContribAuthor ">Simon  Ng</span>, <span class="hlFld-ContribAuthor ">Sneha  Sambandam</span>, <span class="hlFld-ContribAuthor ">Bernardo  Hernandez Adame</span>, <span class="hlFld-ContribAuthor ">Hansell  Perez</span>, <span class="hlFld-ContribAuthor ">Kyung  Ha</span>, <span class="hlFld-ContribAuthor ">Claudia  Falcon</span>, <span class="hlFld-ContribAuthor ">Joseph  de Rutte</span>, <span class="hlFld-ContribAuthor ">Dino  Di Carlo</span>, <span class="hlFld-ContribAuthor ">Andrea L.  Bertozzi</span>. </span><span class="cited-content_cbyCitation_article-title">Statistical energy minimization theory for systems of drop-carrier particles. </span><span class="cited-content_cbyCitation_journal-name">Physical Review E</span><span> <strong>2021,</strong> <em>104 </em>
                                    (1)
                                     <a href="https://doi.org/10.1103/PhysRevE.104.015109" title="DOI URL">https://doi.org/10.1103/PhysRevE.104.015109</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1103/PhysRevE.104.015109&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1103%2FPhysRevE.104.015109%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Review%2520E%26atitle%3DStatistical%252Benergy%252Bminimization%252Btheory%252Bfor%252Bsystems%252Bof%252Bdrop-carrier%252Bparticles%26aulast%3DDu%26aufirst%3DRyan%2BShijie%26date%3D2021%26date%3D2021%26volume%3D104%26issue%3D1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Idorenyin A.  Iwe</span>, <span class="hlFld-ContribAuthor ">Wenqin  Li</span>, <span class="hlFld-ContribAuthor ">Zhigang  Li</span>, <span class="hlFld-ContribAuthor ">Jiahao  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Hairpin DNA-Mediated isothermal amplification (HDMIA) techniques for nucleic acid testing. </span><span class="cited-content_cbyCitation_journal-name">Talanta</span><span> <strong>2021,</strong> <em>226 </em>, 122146. <a href="https://doi.org/10.1016/j.talanta.2021.122146" title="DOI URL">https://doi.org/10.1016/j.talanta.2021.122146</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.talanta.2021.122146&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.talanta.2021.122146%26sid%3Dliteratum%253Aachs%26jtitle%3DTalanta%26atitle%3DHairpin%252BDNA-Mediated%252Bisothermal%252Bamplification%252B%252528HDMIA%252529%252Btechniques%252Bfor%252Bnucleic%252Bacid%252Btesting%26aulast%3DIwe%26aufirst%3DIdorenyin%2BA.%26date%3D2021%26volume%3D226%26spage%3D122146" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meng  Chen</span>, <span class="hlFld-ContribAuthor ">Qidi  He</span>, <span class="hlFld-ContribAuthor ">Yanli  Tong</span>, <span class="hlFld-ContribAuthor ">Zuanguang  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">A universal fluorescent sensing system for pathogen determination based on loop-mediated isothermal amplification triggering dual-primer rolling circle extension. </span><span class="cited-content_cbyCitation_journal-name">Sensors and Actuators B: Chemical</span><span> <strong>2021,</strong> <em>331 </em>, 129436. <a href="https://doi.org/10.1016/j.snb.2021.129436" title="DOI URL">https://doi.org/10.1016/j.snb.2021.129436</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.snb.2021.129436&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.snb.2021.129436%26sid%3Dliteratum%253Aachs%26jtitle%3DSensors%2520and%2520Actuators%2520B%253A%2520Chemical%26atitle%3DA%252Buniversal%252Bfluorescent%252Bsensing%252Bsystem%252Bfor%252Bpathogen%252Bdetermination%252Bbased%252Bon%252Bloop-mediated%252Bisothermal%252Bamplification%252Btriggering%252Bdual-primer%252Brolling%252Bcircle%252Bextension%26aulast%3DChen%26aufirst%3DMeng%26date%3D2021%26volume%3D331%26spage%3D129436" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ming  Wang</span>, <span class="hlFld-ContribAuthor ">Mui Hoon  Nai</span>, <span class="hlFld-ContribAuthor ">Ruby Yun-Ju  Huang</span>, <span class="hlFld-ContribAuthor ">Hwa Liang  Leo</span>, <span class="hlFld-ContribAuthor ">Chwee Teck  Lim</span>, <span class="hlFld-ContribAuthor ">Chia-Hung  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">High-throughput functional profiling of single adherent cells
              via
              hydrogel drop-screen. </span><span class="cited-content_cbyCitation_journal-name">Lab on a Chip</span><span> <strong>2021,</strong> <em>21 </em>
                                    (4)
                                     , 764-774. <a href="https://doi.org/10.1039/D0LC01294G" title="DOI URL">https://doi.org/10.1039/D0LC01294G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0LC01294G&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0LC01294G%26sid%3Dliteratum%253Aachs%26jtitle%3DLab%2520on%2520a%2520Chip%26atitle%3DHigh-throughput%252Bfunctional%252Bprofiling%252Bof%252Bsingle%252Badherent%252Bcells%252Bvia%252Bhydrogel%252Bdrop-screen%26aulast%3DWang%26aufirst%3DMing%26date%3D2021%26date%3D2021%26volume%3D21%26issue%3D4%26spage%3D764%26epage%3D774" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ghulam  Destgeer</span>, <span class="hlFld-ContribAuthor ">Mengxing  Ouyang</span>, <span class="hlFld-ContribAuthor ">Chueh-Yu  Wu</span>, <span class="hlFld-ContribAuthor ">Dino  Di Carlo</span>. </span><span class="cited-content_cbyCitation_article-title">Fabrication of 3D concentric amphiphilic microparticles to form uniform nanoliter reaction volumes for amplified affinity assays. </span><span class="cited-content_cbyCitation_journal-name">Lab on a Chip</span><span> <strong>2020,</strong> <em>20 </em>
                                    (19)
                                     , 3503-3514. <a href="https://doi.org/10.1039/D0LC00698J" title="DOI URL">https://doi.org/10.1039/D0LC00698J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0LC00698J&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0LC00698J%26sid%3Dliteratum%253Aachs%26jtitle%3DLab%2520on%2520a%2520Chip%26atitle%3DFabrication%252Bof%252B3D%252Bconcentric%252Bamphiphilic%252Bmicroparticles%252Bto%252Bform%252Buniform%252Bnanoliter%252Breaction%252Bvolumes%252Bfor%252Bamplified%252Baffinity%252Bassays%26aulast%3DDestgeer%26aufirst%3DGhulam%26date%3D2020%26date%3D2020%26volume%3D20%26issue%3D19%26spage%3D3503%26epage%3D3514" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/cms/10.1021/acssensors.9b01974/asset/images/medium/se9b01974_0004.gif" alt="" id="tgr1"><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acssensors.9b01974&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/cms/10.1021/acssensors.9b01974/asset/images/medium/se9b01974_0005.gif" alt="" data-lg-src="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0005.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Fractal LAMP: Detection of DNA via LAMP byproducts in droplets imaged using brightfield microscopy. (a) During DNA amplification by DNA polymerase, pyrophosphate (blue) is cleaved from the deoxynucleotide (dNTP) as a byproduct. Pyrophosphate reacts with magnesium ions in the LAMP solution mix and precipitates at sufficiently high concentrations. (b) DNA amplification can be assessed via fluorescent dyes, or by turbidity in the case of LAMP. In bulk solution, the fluorescence or turbidity increases with DNA concentration but may be difficult to discern at low concentrations or across small differences in concentration. Compartmentalization of volume creates a digital readout, where fractal-like precipitate or fluorescence is not diluted by a large volume. The percentage of negative compartments are directly correlated to compartment size and the original bulk concentration. (c) LAMP reaction mix and DNA are compartmentalized with a three-dimensional (3D) printed droplet generator and are incubated at 67 °C for 2 h before imaging. (d) After incubation, the droplets are transferred to a 120 μm slit chamber for brightfield and fluorescence imaging. Magnesium pyrophosphate precipitate is visible in brightfield and corresponds to droplets with increased fluorescent intercalating dye signal.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acssensors.9b01974&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/cms/10.1021/acssensors.9b01974/asset/images/medium/se9b01974_0001.gif" alt="" data-lg-src="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Presence of precipitate in LAMP droplets is correlated to the integrated droplet fluorescence and the bulk DNA concentration. (a) The presence of magnesium pyrophosphate precipitate in droplets where LAMP was performed is correlated to integrated droplet fluorescence. Images where droplets moved between brightfield and fluorescence imaging or with fluorescence aberrations are excluded. (b) The percentage of droplets containing precipitate increases as the target DNA concentration in a sample increases. (c) The percentage of droplets without precipitate is used to calculate λ (blue), the average number of target DNA molecules per droplet, which is linearly related to the original DNA concentration (λ = <i>ω</i><sup>–1</sup>·Conc<sub>DNA</sub>·Vol<sub>drop</sub>). Bayesian inference yields a median inefficiency factor ω of 127.14 and confidence interval (gray band, 99% highest posterior density (HPD), indicating that there is an ∼127× inefficiency factor when performing LAMP in droplets. Inference also yields 99% HPD intervals (black) for each sample’s λ estimate that are drawn back to linearity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acssensors.9b01974&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/cms/10.1021/acssensors.9b01974/asset/images/medium/se9b01974_0002.gif" alt="" data-lg-src="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Creation of Bag of Visual Words model from SURF keypoints. (a) SURF keypoints, highlighted in colored boxes, are extracted from previously isolated images of droplets. SURF keypoints are scale and rotation invariant and are described with a 64-dimensional feature space. (b) The BOVW model is created by clustering SURF keypoints within their feature space to define words. Dark and Light words are clustered separately, according to the positive and negative Sign of Laplacian, respectively. SURF keypoints closest to the cluster centers are shown for some word examples. Words are further divided into Sharp and Fuzzy words, which overlap in the same feature space, but have different levels of contrast. (c) Example images of droplets with and without precipitate are processed in a BOVW model containing 32 words, with eight words each of Dark/Sharp, Dark/Fuzzy, Light/Sharp, and Light/Fuzzy. The frequency of each type of word appearing in the images (circles), is compared to mean frequencies in the entire dataset (bars).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acssensors.9b01974&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/cms/10.1021/acssensors.9b01974/asset/images/medium/se9b01974_0003.gif" alt="" data-lg-src="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav"><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Classification of droplets predicts Testing set λ and DNA concentration. (a) Random forest classification of droplets. Testing set droplets indicated with black circle markers, all others are from Training and Validation sets. Color on outlining circles indicates prediction score of a precipitate-positive droplet. The threshold to classify a droplet as precipitate-positive can be adjusted but is 0.5 by default. (b) λ Values for droplet predictions (blue, horizontal) and ground truth (yellow, horizontal) are computed as before (with 0.5 threshold) and related to bulk DNA concentration. The predicted λs with this classifier score threshold are consistently higher than the ground truth, indicating a systematic propensity to classify false positives, and limitations of low droplet count. Bayesian estimates of the λ HPD intervals (vertical) illustrate the ability to recover valid ranges of λ values for unknown DNA concentration but can be susceptible to false positives at low concentrations. (c) Bootstrapping across a range of λs and droplet sample sizes enables the calculation of the minimum number of droplets needed to distinguish a sample from 0 DNA at 95 or 99% confidence levels (Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Figure S8</a>, see Supporting Information <a href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf" class="ext-link">Note 1</a> for more detail).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/cms/10.1021/acssensors.9b01974/asset/images/large/se9b01974_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acssensors.9b01974&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i20">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38522" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38522" class="dropBlock__holder"></div></div></div></div></div><hr></div><p class="rightTab-references-count">
            This article references 48 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tate, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamford, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubb, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sondka, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beare, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bindal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutselakis, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creatore, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, E.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">COSMIC: The Catalogue Of Somatic Mutations In Cancer</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">D941</span>– <span class="NLM_lpage">D947</span>, <span class="refDoi">&nbsp;DOI: 10.1093/nar/gky1015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1093%2Fnar%2Fgky1015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=30371878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7bM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=2019&amp;pages=D941-D947&amp;author=J.+G.+Tateauthor=S.+Bamfordauthor=H.+C.+Jubbauthor=Z.+Sondkaauthor=D.+M.+Beareauthor=N.+Bindalauthor=H.+Boutselakisauthor=C.+G.+Coleauthor=C.+Creatoreauthor=E.+Dawson&amp;title=COSMIC%3A+The+Catalogue+Of+Somatic+Mutations+In+Cancer&amp;doi=10.1093%2Fnar%2Fgky1015" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">COSMIC: the Catalogue Of Somatic Mutations In Cancer</span></div><div class="casAuthors">Tate, John G.; Bamford, Sally; Jubb, Harry C.; Sondka, Zbyslaw; Beare, David M.; Bindal, Nidhi; Boutselakis, Harry; Cole, Charlotte G.; Creatore, Celestino; Dawson, Elisabeth; Fish, Peter; Harsha, Bhavana; Hathaway, Charlie; Jupe, Steve C.; Kok, Chai Yin; Noble, Kate; Ponting, Laura; Ramshaw, Christopher C.; Rye, Claire E.; Speedy, Helen E.; Stefancsik, Ray; Thompson, Sam L.; Wang, Shicai; Ward, Sari; Campbell, Peter J.; Forbes, Simon A.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D941-D947</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">COSMIC, the Catalog Of Somatic Mutations In Cancer (https://cancer.sanger.ac.uk) is the most detailed and comprehensive resource for exploring the effect of somatic mutations in human cancer.  The latest release, COSMIC v86 (August 2018), includes almost 6 million coding mutations across 1.4 million tumor samples, curated from over 26,000 publications.  In addn. to coding mutations, COSMIC covers all the genetic mechanisms by which somatic mutations promote cancer, including non-coding mutations, gene fusions, copy-no. variants and drug resistance mutations.  COSMIC is primarily handcurated, ensuring quality, accuracy and descriptive data capture.  Building on our manual curation processes, we are introducing new initiatives that allow us to prioritize key genes and diseases, and to react more quickly and comprehensively to new findings in the literature.  Alongside improvements to the public website and data-download systems, new functionality in COSMIC-3D allows exploration of mutations within three-dimensional protein structures, their protein structural and functional impacts, and implications for druggability.  In parallel with COSMIC's deep and broad variant coverage, the Cancer Gene Census (CGC) describes a curated catalog of genes driving every form of human cancer.  Currently describing 719 genes, the CGC has recently introduced functional descriptions of how each gene drives disease, summarized into the 10 cancer Hallmarks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyWHUT512HAbVg90H21EOLACvtfcHk0li2N1wFUOolYQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7bM&amp;md5=56574cedbe9adc54c097a5af5fe41eb8</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1015&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1015%26sid%3Dliteratum%253Aachs%26aulast%3DTate%26aufirst%3DJ.%2BG.%26aulast%3DBamford%26aufirst%3DS.%26aulast%3DJubb%26aufirst%3DH.%2BC.%26aulast%3DSondka%26aufirst%3DZ.%26aulast%3DBeare%26aufirst%3DD.%2BM.%26aulast%3DBindal%26aufirst%3DN.%26aulast%3DBoutselakis%26aufirst%3DH.%26aulast%3DCole%26aufirst%3DC.%2BG.%26aulast%3DCreatore%26aufirst%3DC.%26aulast%3DDawson%26aufirst%3DE.%26atitle%3DCOSMIC%253A%2520The%2520Catalogue%2520Of%2520Somatic%2520Mutations%2520In%2520Cancer%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3DD941%26epage%3DD947%26doi%3D10.1093%2Fnar%2Fgky1015" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benesova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belsanova, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suchanek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopeckova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minarikova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipska, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Visokai, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zavoral, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minarik, M.</span></span> <span> </span><span class="NLM_article-title">Mutation-Based Detection and Monitoring of Cell-Free Tumor DNA in Peripheral Blood of Cancer Patients</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>433</i></span>,  <span class="NLM_fpage">227</span>– <span class="NLM_lpage">234</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.ab.2012.06.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.ab.2012.06.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=22750103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslymsLjF" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=433&amp;publication_year=2013&amp;pages=227-234&amp;author=L.+Benesovaauthor=B.+Belsanovaauthor=S.+Suchanekauthor=M.+Kopeckovaauthor=P.+Minarikovaauthor=L.+Lipskaauthor=M.+Levyauthor=V.+Visokaiauthor=M.+Zavoralauthor=M.+Minarik&amp;title=Mutation-Based+Detection+and+Monitoring+of+Cell-Free+Tumor+DNA+in+Peripheral+Blood+of+Cancer+Patients&amp;doi=10.1016%2Fj.ab.2012.06.018" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mutation-based detection and monitoring of cell-free tumor DNA in peripheral blood of cancer patients</span></div><div class="casAuthors">Benesova, L.; Belsanova, B.; Suchanek, S.; Kopeckova, M.; Minarikova, P.; Lipska, L.; Levy, M.; Visokai, V.; Zavoral, M.; Minarik, M.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">433</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">227-234</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Prognosis of solid cancers is generally more favorable if the disease is treated early and efficiently.  A key to long cancer survival is in radical surgical therapy directed at the primary tumor followed by early detection of possible progression, with swift application of subsequent therapeutic intervention reducing the risk of disease generalization.  The conventional follow-up care is based on regular observation of tumor markers in combination with computed tomog./endoscopic ultrasound/magnetic resonance/positron emission tomog. imaging to monitor potential tumor progression.  A recent development in methodologies allowing screening for a presence of cell-free DNA (cfDNA) brings a new viable tool in early detection and management of major cancers.  It is believed that cfDNA is released from tumors primarily due to necrotization, whereas the origin of nontumorous cfDNA is mostly apoptotic.  The process of cfDNA detection starts with proper collection and treatment of blood and isolation and storage of blood plasma.  The next important steps include cfDNA extn. from plasma and its detection and/or quantification.  To distinguish tumor cfDNA from nontumorous cfDNA, specific somatic DNA mutations, previously localized in the primary tumor tissue, are identified in the extd. cfDNA.  Apart from conventional mutation detection approaches, several dedicated techniques have been presented to detect low levels of cfDNA in an excess of nontumorous (nonmutated) DNA, including real-time polymerase chain reaction (PCR), BEAMing (beads, emulsion, amplification, and magnetics), and denaturing capillary electrophoresis.  Techniques to facilitate the mutant detection, such as mutant-enriched PCR and COLD-PCR (coamplification at lower denaturation temp. PCR), are also applicable.  Finally, a no. of newly developed miniaturized approaches, such as single-mol. sequencing, are promising for the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwqMGPX59UnLVg90H21EOLACvtfcHk0lgoPLqntOGJ0Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XhslymsLjF&amp;md5=a38a12e042e438ddf13f4bf66dd6d7a6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2012.06.018&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2012.06.018%26sid%3Dliteratum%253Aachs%26aulast%3DBenesova%26aufirst%3DL.%26aulast%3DBelsanova%26aufirst%3DB.%26aulast%3DSuchanek%26aufirst%3DS.%26aulast%3DKopeckova%26aufirst%3DM.%26aulast%3DMinarikova%26aufirst%3DP.%26aulast%3DLipska%26aufirst%3DL.%26aulast%3DLevy%26aufirst%3DM.%26aulast%3DVisokai%26aufirst%3DV.%26aulast%3DZavoral%26aufirst%3DM.%26aulast%3DMinarik%26aufirst%3DM.%26atitle%3DMutation-Based%2520Detection%2520and%2520Monitoring%2520of%2520Cell-Free%2520Tumor%2520DNA%2520in%2520Peripheral%2520Blood%2520of%2520Cancer%2520Patients%26jtitle%3DAnal.%2520Biochem.%26date%3D2013%26volume%3D433%26spage%3D227%26epage%3D234%26doi%3D10.1016%2Fj.ab.2012.06.018" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niemz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyle, D. S.</span></span> <span> </span><span class="NLM_article-title">Point-of-Care Nucleic Acid Testing for Infectious Diseases</span>. <i>Trends Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">250</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.tibtech.2011.01.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.tibtech.2011.01.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=21377748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFegt7w%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=2011&amp;pages=240-250&amp;author=A.+Niemzauthor=T.+M.+Fergusonauthor=D.+S.+Boyle&amp;title=Point-of-Care+Nucleic+Acid+Testing+for+Infectious+Diseases&amp;doi=10.1016%2Fj.tibtech.2011.01.007" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Point-of-care nucleic acid testing for infectious diseases</span></div><div class="casAuthors">Niemz, Angelika; Ferguson, Tanya M.; Boyle, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">240-250</span>CODEN:
                <span class="NLM_cas:coden">TRBIDM</span>;
        ISSN:<span class="NLM_cas:issn">0167-7799</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Nucleic acid testing for infectious diseases at the point of care is beginning to enter clin. practice in developed and developing countries; esp. for applications requiring fast turnaround times, and in settings where a centralized lab. approach faces limitations.  Current systems for clin. diagnostic applications are mainly PCR-based, can only be used in hospitals, and are still relatively complex and expensive.  Integrating sample prepn. with nucleic acid amplification and detection in a cost-effective, robust, and user-friendly format remains challenging.  This review describes recent tech. advances that might be able to address these limitations, with a focus on isothermal nucleic acid amplification methods.  It briefly discusses selected applications related to the diagnosis and management of tuberculosis, HIV, and perinatal and nosocomial infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonIJ_GhFP2sbVg90H21EOLACvtfcHk0lgoPLqntOGJ0Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3MXltFegt7w%253D&amp;md5=a5e0c6beb0e2d18a71c51f4673e0ec6a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tibtech.2011.01.007&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibtech.2011.01.007%26sid%3Dliteratum%253Aachs%26aulast%3DNiemz%26aufirst%3DA.%26aulast%3DFerguson%26aufirst%3DT.%2BM.%26aulast%3DBoyle%26aufirst%3DD.%2BS.%26atitle%3DPoint-of-Care%2520Nucleic%2520Acid%2520Testing%2520for%2520Infectious%2520Diseases%26jtitle%3DTrends%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D240%26epage%3D250%26doi%3D10.1016%2Fj.tibtech.2011.01.007" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martzy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krska, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mach, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnleitner, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reischer, G. H.</span></span> <span> </span><span class="NLM_article-title">Challenges and Perspectives in the Application of Isothermal DNA Amplification Methods for Food and Water Analysis</span>. <i>Anal. Bioanal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>411</i></span>,  <span class="NLM_fpage">1695</span>– <span class="NLM_lpage">1702</span>, <span class="refDoi">&nbsp;DOI: 10.1007/s00216-018-1553-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1007%2Fs00216-018-1553-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=30617408" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvVGhs74%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=411&amp;publication_year=2019&amp;pages=1695-1702&amp;author=R.+Martzyauthor=C.+Kolmauthor=R.+Krskaauthor=R.+L.+Machauthor=A.+H.+Farnleitnerauthor=G.+H.+Reischer&amp;title=Challenges+and+Perspectives+in+the+Application+of+Isothermal+DNA+Amplification+Methods+for+Food+and+Water+Analysis&amp;doi=10.1007%2Fs00216-018-1553-1" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges and perspectives in the application of isothermal DNA amplification methods for food and water analysis</span></div><div class="casAuthors">Martzy, Roland; Kolm, Claudia; Krska, Rudolf; Mach, Robert L.; Farnleitner, Andreas H.; Reischer, Georg H.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical and Bioanalytical Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">411</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1695-1702</span>CODEN:
                <span class="NLM_cas:coden">ABCNBP</span>;
        ISSN:<span class="NLM_cas:issn">1618-2642</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Mol. diagnostic tools in the field of food and water quality anal. are becoming increasingly widespread.  Usually, based on DNA amplification techniques such as polymerase chain reaction (PCR), these methods are highly sensitive and versatile but require well-equipped labs. and trained personnel.  To reduce anal. time and avoid expensive equipment, isothermal DNA amplification methods for detecting various target organisms have been developed.  However, to make mol. diagnostics suitable for low-resource settings and in-field applications, it is crucial to continuously adapt the working steps assocd. with DNA amplification, namely sample prepn., DNA extn., and visualization of the results.  Many novel approaches have been evaluated in recent years to tackle these challenges, e.g., the use of ionic liqs. for the rapid isolation of nucleic acids from organisms relevant for food and water anal. or the integration of entire anal. workflows on microfluidic chips.  In any event, the future of applications in the field of isothermal amplification will probably lie in ready-to-use cartridges combined with affordable handheld devices for on-site anal.  This trend article aims to make prospective users more familiar with this technol. and its potential for moving mol. diagnostics from the lab. to the field. [Figure not available: see fulltext.].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfbXyIP7Zm9LVg90H21EOLACvtfcHk0lgE2LhcXbyYpA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXlvVGhs74%253D&amp;md5=dc5f55bd5e4049d3f90c81500dc1c407</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs00216-018-1553-1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00216-018-1553-1%26sid%3Dliteratum%253Aachs%26aulast%3DMartzy%26aufirst%3DR.%26aulast%3DKolm%26aufirst%3DC.%26aulast%3DKrska%26aufirst%3DR.%26aulast%3DMach%26aufirst%3DR.%2BL.%26aulast%3DFarnleitner%26aufirst%3DA.%2BH.%26aulast%3DReischer%26aufirst%3DG.%2BH.%26atitle%3DChallenges%2520and%2520Perspectives%2520in%2520the%2520Application%2520of%2520Isothermal%2520DNA%2520Amplification%2520Methods%2520for%2520Food%2520and%2520Water%2520Analysis%26jtitle%3DAnal.%2520Bioanal.%2520Chem.%26date%3D2019%26volume%3D411%26spage%3D1695%26epage%3D1702%26doi%3D10.1007%2Fs00216-018-1553-1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span> <span> </span><span class="NLM_article-title">Point-of-Care-Testing of Nucleic Acids by Microfluidics</span>. <i>TrAC, Trends Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">106</span>– <span class="NLM_lpage">116</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.trac.2017.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.trac.2017.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1WlsbvM" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=94&amp;publication_year=2017&amp;pages=106-116&amp;author=L.+Zhangauthor=B.+Dingauthor=Q.+Chenauthor=Q.+Fengauthor=L.+Linauthor=J.+Sun&amp;title=Point-of-Care-Testing+of+Nucleic+Acids+by+Microfluidics&amp;doi=10.1016%2Fj.trac.2017.07.013" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Point-of-care-testing of nucleic acids by microfluidics</span></div><div class="casAuthors">Zhang, Lu; Ding, Baozhi; Chen, Qinghua; Feng, Qiang; Lin, Ling; Sun, Jiashu</div><div class="citationInfo"><span class="NLM_cas:title">TrAC, Trends in Analytical Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">106-116</span>CODEN:
                <span class="NLM_cas:coden">TTAEDJ</span>;
        ISSN:<span class="NLM_cas:issn">0165-9936</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nucleic acid detection offers crit. information in diagnosing pathogen infection and monitoring disease treatment.  Point-of-care testing (POCT), with ideal characteristics such as rapidness, user-friendly, sensitivity, specificity, and robustness, has drawn wide attention, esp. in resource-poor settings where lack the healthcare infrastructure.  Accompanied by the development of microfluidic technologies and new amplification methods, POCT of nucleic acids greatly improves health monitoring and prevents outbreak of infectious diseases.  This review summarizes recent advances in diagnostics of nucleic acids in POC formats, and discusses challenges assocd. with POCT of nucleic acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogvQCdYQKH5rVg90H21EOLACvtfcHk0lgE2LhcXbyYpA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXht1WlsbvM&amp;md5=9369e79766718869dc787e6ebb437ab2</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.trac.2017.07.013&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trac.2017.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DDing%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DJ.%26atitle%3DPoint-of-Care-Testing%2520of%2520Nucleic%2520Acids%2520by%2520Microfluidics%26jtitle%3DTrAC%252C%2520Trends%2520Anal.%2520Chem.%26date%3D2017%26volume%3D94%26spage%3D106%26epage%3D116%26doi%3D10.1016%2Fj.trac.2017.07.013" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garibyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avashia, N.</span></span> <span> </span><span class="NLM_article-title">Polymerase Chain Reaction</span>. <i>J. Invest. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">4</span>, <span class="refDoi">&nbsp;DOI: 10.1038/jid.2013.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1038%2Fjid.2013.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=23399825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A280%3ADC%252BC3szntlyksw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=133&amp;publication_year=2013&amp;pages=1-4&amp;author=L.+Garibyanauthor=N.+Avashia&amp;title=Polymerase+Chain+Reaction&amp;doi=10.1038%2Fjid.2013.1" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Polymerase chain reaction</span></div><div class="casAuthors">Garibyan Lilit; Avashia Nidhi</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of investigative dermatology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1-4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR6S-kR0Z69e0PI-fCgwosgfW6udTcc2eYxD_l58wZHsbntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC3szntlyksw%253D%253D&amp;md5=a2b4ed72e813d938dd74567970d8079b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fjid.2013.1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fjid.2013.1%26sid%3Dliteratum%253Aachs%26aulast%3DGaribyan%26aufirst%3DL.%26aulast%3DAvashia%26aufirst%3DN.%26atitle%3DPolymerase%2520Chain%2520Reaction%26jtitle%3DJ.%2520Invest.%2520Dermatol.%26date%3D2013%26volume%3D133%26spage%3D1%26epage%3D4%26doi%3D10.1038%2Fjid.2013.1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ali, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, D.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ankrum, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span> <span> </span><span class="NLM_article-title">Rolling Circle Amplification: A Versatile Tool for Chemical Biology, Materials Science and Medicine</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">3324</span>, <span class="refDoi">&nbsp;DOI: 10.1039/c3cs60439j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1039%2Fc3cs60439j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=24643375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmvFGhsb8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=43&amp;publication_year=2014&amp;pages=3324&amp;author=M.+M.+Aliauthor=F.+Liauthor=Z.+Zhangauthor=K.+Zhangauthor=D.-K.+Kangauthor=J.+A.+Ankrumauthor=X.+C.+Leauthor=W.+Zhao&amp;title=Rolling+Circle+Amplification%3A+A+Versatile+Tool+for+Chemical+Biology%2C+Materials+Science+and+Medicine&amp;doi=10.1039%2Fc3cs60439j" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Rolling circle amplification: a versatile tool for chemical biology, materials science and medicine</span></div><div class="casAuthors">Ali, M. Monsur; Li, Feng; Zhang, Zhiqing; Zhang, Kaixiang; Kang, Dong-Ku; Ankrum, James A.; Le, X. Chris; Zhao, Weian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3324-3341</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Rolling circle amplification (RCA) is an isothermal enzymic process where a short DNA or RNA primer is amplified to form a long single stranded DNA or RNA using a circular DNA template and special DNA or RNA polymerases.  The RCA product is a concatemer contg. tens to hundreds of tandem repeats that are complementary to the circular template.  The power, simplicity, and versatility of the DNA amplification technique have made it an attractive tool for biomedical research and nanobiotechnol.  Traditionally, RCA has been used to develop sensitive diagnostic methods for a variety of targets including nucleic acids (DNA, RNA), small mols., proteins, and cells.  RCA has also attracted significant attention in the field of nanotechnol. and nanobiotechnol.  The RCA-produced long, single-stranded DNA with repeating units has been used as template for the periodic assembly of nanospecies.  Moreover, since RCA products can be tailor-designed by manipulating the circular template, RCA has been employed to generate complex DNA nanostructures such as DNA origami, nanotubes, nanoribbons and DNA based metamaterials.  These functional RCA based nanotechnologies have been utilized for biodetection, drug delivery, designing bioelectronic circuits and biosepn.  In this review, we introduce the fundamental engineering principles used to design RCA nanotechnologies, discuss recently developed RCA-based diagnostics and bioanal. tools, and summarize the use of RCA to construct multivalent mol. scaffolds and nanostructures for applications in biol., diagnostics and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlPH9u97qAabVg90H21EOLACvtfcHk0lgbTdZOPUXHGQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXmvFGhsb8%253D&amp;md5=3a50f54645d8e6221e5a3e4383bdbc31</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1039%2Fc3cs60439j&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3cs60439j%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DM.%2BM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DKang%26aufirst%3DD.-K.%26aulast%3DAnkrum%26aufirst%3DJ.%2BA.%26aulast%3DLe%26aufirst%3DX.%2BC.%26aulast%3DZhao%26aufirst%3DW.%26atitle%3DRolling%2520Circle%2520Amplification%253A%2520A%2520Versatile%2520Tool%2520for%2520Chemical%2520Biology%252C%2520Materials%2520Science%2520and%2520Medicine%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2014%26volume%3D43%26spage%3D3324%26doi%3D10.1039%2Fc3cs60439j" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mori, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagamine, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Notomi, T.</span></span> <span> </span><span class="NLM_article-title">Detection of Loop-Mediated Isothermal Amplification Reaction by Turbidity Derived from Magnesium Pyrophosphate Formation</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">154</span>, <span class="refDoi">&nbsp;DOI: 10.1006/bbrc.2001.5921</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1006%2Fbbrc.2001.5921" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=11708792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BD3MXot12ht7Y%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=289&amp;publication_year=2001&amp;pages=150-154&amp;author=Y.+Moriauthor=K.+Nagamineauthor=N.+Tomitaauthor=T.+Notomi&amp;title=Detection+of+Loop-Mediated+Isothermal+Amplification+Reaction+by+Turbidity+Derived+from+Magnesium+Pyrophosphate+Formation&amp;doi=10.1006%2Fbbrc.2001.5921" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of Loop-Mediated Isothermal Amplification Reaction by Turbidity Derived from Magnesium Pyrophosphate Formation</span></div><div class="casAuthors">Mori, Yasuyoshi; Nagamine, Kentaro; Tomita, Norihiro; Notomi, Tsugunori</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">150-154</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">The loop-mediated isothermal amplification (LAMP) is a novel nucleic acid amplification method that uses only one type of enzyme.  One of the characteristics of the LAMP method is its ability to synthesize extremely large amt. of DNA.  Accordingly, a large amt. of byproduct, pyrophosphate ion, is produced, yielding white ppt. of magnesium pyrophosphate in the reaction mixt.  Judging the presence or absence of this white ppt. allows easy distinction of whether nucleic acid was amplified by the LAMP method.  Since an increase in the turbidity of the reaction mixt. according to the prodn. of ppt. correlates with the amt. of DNA synthesized, real-time monitoring of the LAMP reaction was achieved by real-time measurement of turbidity.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1yNhZIhbXd7Vg90H21EOLACvtfcHk0lgbTdZOPUXHGQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD3MXot12ht7Y%253D&amp;md5=2f8786b3d548afb6ec7f49cc9d5c8db5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.2001.5921&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.2001.5921%26sid%3Dliteratum%253Aachs%26aulast%3DMori%26aufirst%3DY.%26aulast%3DNagamine%26aufirst%3DK.%26aulast%3DTomita%26aufirst%3DN.%26aulast%3DNotomi%26aufirst%3DT.%26atitle%3DDetection%2520of%2520Loop-Mediated%2520Isothermal%2520Amplification%2520Reaction%2520by%2520Turbidity%2520Derived%2520from%2520Magnesium%2520Pyrophosphate%2520Formation%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2001%26volume%3D289%26spage%3D150%26epage%3D154%26doi%3D10.1006%2Fbbrc.2001.5921" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duarte-Guevara, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bashir, R.</span></span> <span> </span><span class="NLM_article-title">On-Chip Electrical Detection of Parallel Loop-Mediated Isothermal Amplification with DG-BioFETs for the Detection of Foodborne Bacterial Pathogens</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">103872</span>– <span class="NLM_lpage">103887</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C6RA19685C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1039%2FC6RA19685C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs12ksrnJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2016&amp;pages=103872-103887&amp;author=C.+Duarte-Guevaraauthor=V.+V.+Swaminathanauthor=B.+Reddyauthor=J.-C.+Huangauthor=Y.-S.+Liuauthor=R.+Bashir&amp;title=On-Chip+Electrical+Detection+of+Parallel+Loop-Mediated+Isothermal+Amplification+with+DG-BioFETs+for+the+Detection+of+Foodborne+Bacterial+Pathogens&amp;doi=10.1039%2FC6RA19685C" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">On-chip electrical detection of parallel loop-mediated isothermal amplification with DG-BioFETs for the detection of foodborne bacterial pathogens</span></div><div class="casAuthors">Duarte-Guevara, Carlos; Swaminathan, Vikhram V.; Reddy, Bobby; Huang, Jui-Cheng; Liu, Yi-Shao; Bashir, Rashid</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">106</span>),
    <span class="NLM_cas:pages">103872-103887</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The use of field effect transistors (FETs) as the transduction element for the detection of DNA amplification reactions will enable portable and inexpensive nucleic acid anal.  Transistors used as biol. sensors, or BioFETs, minimize the cost and size of detection platforms by leveraging fabrication methods already well developed for electronics.  Here, we report a dual-gate BioFET (DG-BioFET) array platform with 1024×1024 sensors that is used for on-chip elec. detection of loop-mediated isothermal amplification (LAMP) reactions that target food borne bacterial pathogens.  The DG-BioFETs of our 7 × 7 mm2 array are able to elec. detect pH changes that are triggered by nucleotide incorporation during LAMP elongation.  Multiple 250 nL reactions can be simultaneously elec. monitored in our array that is divided in 30 micro-chambers with gold-coated anisotropically etched silicon wells that act both as ref. electrode and confinement element.  Our characterization results show that the gold-biased DG-BioFETs have a sensitivity of 32 mV pH-1 (equiv. to 2 μA pH-1) and an av. resoln. of 0.5 pH units.  This sensitivity is high enough to detect the pH changes triggered by the amplification reaction, but to maximize our signal-to-noise ratio and improve our quant. conclusions we use a group of data anal. techniques that are available in our high-d. platform that monitors each reaction with ∼3500 independent BioFETs.  We use redundancy techniques to minimize the overall std. deviation of our measurements, the Grubbs test to eliminate measurements outside the expected normal distribution, and ref. micro-chambers to subtract the common noise.  With these techniques we are capable of reducing the P value, of a t-test comparing pos. and neg. readings, from a typical 0.17 to 0.03.  The platform that we present along with the anal. techniques that we developed allow the on-chip elec. detection and identification of E. coli O157 and S. typhi with parallel LAMP assays targeting eae and invA genes.  The LAMP reactions are highly specific, without false positives, and our titrn. assays demonstrate a limit of detection of 23 CFU per reaction on chip.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeNf2SFHu0QrVg90H21EOLACvtfcHk0lgbTdZOPUXHGQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28Xhs12ksrnJ&amp;md5=56b13ff8ee55404c57e1993d2c7e15f2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1039%2FC6RA19685C&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6RA19685C%26sid%3Dliteratum%253Aachs%26aulast%3DDuarte-Guevara%26aufirst%3DC.%26aulast%3DSwaminathan%26aufirst%3DV.%2BV.%26aulast%3DReddy%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DJ.-C.%26aulast%3DLiu%26aufirst%3DY.-S.%26aulast%3DBashir%26aufirst%3DR.%26atitle%3DOn-Chip%2520Electrical%2520Detection%2520of%2520Parallel%2520Loop-Mediated%2520Isothermal%2520Amplification%2520with%2520DG-BioFETs%2520for%2520the%2520Detection%2520of%2520Foodborne%2520Bacterial%2520Pathogens%26jtitle%3DRSC%2520Adv.%26date%3D2016%26volume%3D6%26spage%3D103872%26epage%3D103887%26doi%3D10.1039%2FC6RA19685C" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salm, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte-Guevara, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bashir, R.</span></span> <span> </span><span class="NLM_article-title">Electrical Detection of Nucleic Acid Amplification Using an On-Chip Quasi-Reference Electrode and a PVC REFET</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">6968</span>– <span class="NLM_lpage">6975</span>, <span class="refDoi">&nbsp;DOI: 10.1021/ac500897t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac500897t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvFCltLg%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=86&amp;publication_year=2014&amp;pages=6968-6975&amp;author=E.+Salmauthor=Y.+Zhongauthor=B.+Reddyauthor=C.+Duarte-Guevaraauthor=V.+Swaminathanauthor=Y.-S.+Liuauthor=R.+Bashir&amp;title=Electrical+Detection+of+Nucleic+Acid+Amplification+Using+an+On-Chip+Quasi-Reference+Electrode+and+a+PVC+REFET&amp;doi=10.1021%2Fac500897t" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Electrical detection of nucleic acid amplification using an on-chip quasi-reference electrode and a PVC REFET</span></div><div class="casAuthors">Salm, Eric; Zhong, Yu; Reddy, Bobby; Duarte-Guevara, Carlos; Swaminathan, Vikhram; Liu, Yi-Shao; Bashir, Rashid</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6968-6975</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Elec. detection of nucleic acid amplification through pH changes assocd. with nucleotide addn. enables miniaturization, greater portability of testing app., and reduced costs.  However, current ion-sensitive field effect transistor methods for sensing nucleic acid amplification rely on establishing the fluid gate potential with a bulky, difficult to microfabricate ref. electrode that limits the potential for massively parallel reaction detection.  Here we demonstrate a novel method of utilizing a microfabricated solid-state quasi-ref. electrode (QRE) paired with a pH-insensitive ref. field effect transistor (REFET) for detection of real-time pH changes.  The end result is a 0.18 μm, silicon-on-insulator, foundry-fabricated sensor that utilizes a platinum QRE to establish a pH-sensitive fluid gate potential and a PVC membrane REFET to enable pH detection of loop mediated isothermal amplification (LAMP).  This technique is highly amendable to com. scale-up, reduces the packaging and fabrication requirements for ISFET pH detection, and enables massively parallel droplet interrogation for applications, such as monitoring reaction progression in digital PCR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjFip-YxJsGbVg90H21EOLACvtfcHk0lji-LzhJBxBuw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXpvFCltLg%253D&amp;md5=13304ddff83cd5d6e4477810ebc3fa5b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fac500897t&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac500897t%26sid%3Dliteratum%253Aachs%26aulast%3DSalm%26aufirst%3DE.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DReddy%26aufirst%3DB.%26aulast%3DDuarte-Guevara%26aufirst%3DC.%26aulast%3DSwaminathan%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DY.-S.%26aulast%3DBashir%26aufirst%3DR.%26atitle%3DElectrical%2520Detection%2520of%2520Nucleic%2520Acid%2520Amplification%2520Using%2520an%2520On-Chip%2520Quasi-Reference%2520Electrode%2520and%2520a%2520PVC%2520REFET%26jtitle%3DAnal.%2520Chem.%26date%3D2014%26volume%3D86%26spage%3D6968%26epage%3D6975%26doi%3D10.1021%2Fac500897t" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tabata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katayama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seichi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyahara, Y.</span></span> <span> </span><span class="NLM_article-title">Label-Free and Electrochemical Detection of Nucleic Acids Based on Isothermal Amplification in Combination with Solid-State PH Sensor</span>. <i>Procedia Eng.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>168</i></span>,  <span class="NLM_fpage">419</span>– <span class="NLM_lpage">422</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.proeng.2016.11.534</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.proeng.2016.11.534" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1yrur4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=168&amp;publication_year=2016&amp;pages=419-422&amp;author=M.+Tabataauthor=Y.+Katayamaauthor=F.+Mannanauthor=A.+Seichiauthor=K.+Suzukiauthor=T.+Godaauthor=A.+Matsumotoauthor=Y.+Miyahara&amp;title=Label-Free+and+Electrochemical+Detection+of+Nucleic+Acids+Based+on+Isothermal+Amplification+in+Combination+with+Solid-State+PH+Sensor&amp;doi=10.1016%2Fj.proeng.2016.11.534" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Label-free and Electrochemical Detection of Nucleic Acids Based on Isothermal Amplification in Combination with Solid-state pH Sensor</span></div><div class="casAuthors">Tabata, Miyuki; Katayama, Yurika; Mannan, Fahmida; Seichi, Ayaka; Suzuki, Koji; Goda, Tatsuro; Matsumoto, Akira; Miyahara, Yuji</div><div class="citationInfo"><span class="NLM_cas:title">Procedia Engineering</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">419-422</span>CODEN:
                <span class="NLM_cas:coden">PERNBE</span>;
        ISSN:<span class="NLM_cas:issn">1877-7058</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Label-free quantification methods for nucleic acids are attracting attention, since they provide potential tools in clin. applications such as disease diagnosis or prognosis observation.  In this research, we developed a simpler and cost-effective elec. monitoring device of nucleic acid amplification, combining an isothermal amplification method and an Iridium/Iridium oxide (Ir/IrOx) electrode without labeling.  The fabricated Ir/IrOx electrode showed ideal Nernstian response under the various pH buffer solns., and its potential response was less affected even in the presence of charged proteins.  Moreover, we successfully real-time monitored primer generation-rolling circle amplification (PG-RCA) at the 0, 10, 100, 1000 pM of target DNA by detection of released proton during amplification reaction at the const. reaction temp.  This label-free and portable device might be received considerable attention as a useful platform for a point-of-care testing in clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFrrKjUzXnpbVg90H21EOLACvtfcHk0lji-LzhJBxBuw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXht1yrur4%253D&amp;md5=001ef83181211bf526c0d969cdc8ae38</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.proeng.2016.11.534&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.proeng.2016.11.534%26sid%3Dliteratum%253Aachs%26aulast%3DTabata%26aufirst%3DM.%26aulast%3DKatayama%26aufirst%3DY.%26aulast%3DMannan%26aufirst%3DF.%26aulast%3DSeichi%26aufirst%3DA.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DGoda%26aufirst%3DT.%26aulast%3DMatsumoto%26aufirst%3DA.%26aulast%3DMiyahara%26aufirst%3DY.%26atitle%3DLabel-Free%2520and%2520Electrochemical%2520Detection%2520of%2520Nucleic%2520Acids%2520Based%2520on%2520Isothermal%2520Amplification%2520in%2520Combination%2520with%2520Solid-State%2520PH%2520Sensor%26jtitle%3DProcedia%2520Eng.%26date%3D2016%26volume%3D168%26spage%3D419%26epage%3D422%26doi%3D10.1016%2Fj.proeng.2016.11.534" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Basu, A. S.</span></span> <span> </span><span class="NLM_article-title">Digital Assays Part I: Partitioning Statistics and Digital PCR</span>. <i>SLAS Technol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">369</span>– <span class="NLM_lpage">386</span>, <span class="refDoi">&nbsp;DOI: 10.1177/2472630317705680</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1177%2F2472630317705680" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=28448765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A280%3ADC%252BC1crhslCitg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=2017&amp;pages=369-386&amp;author=A.+S.+Basu&amp;title=Digital+Assays+Part+I%3A+Partitioning+Statistics+and+Digital+PCR&amp;doi=10.1177%2F2472630317705680" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Digital Assays Part I: Partitioning Statistics and Digital PCR</span></div><div class="casAuthors">Basu Amar S</div><div class="citationInfo"><span class="NLM_cas:title">SLAS technology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">369-386</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A digital assay is one in which the sample is partitioned into many small containers such that each partition contains a discrete number of biological entities (0, 1, 2, 3, ...).  A powerful technique in the biologist's toolkit, digital assays bring a new level of precision in quantifying nucleic acids, measuring proteins and their enzymatic activity, and probing single-cell genotypes and phenotypes.  Part I of this review begins with the benefits and Poisson statistics of partitioning, including sources of error.  The remainder focuses on digital PCR (dPCR) for quantification of nucleic acids.  We discuss five commercial instruments that partition samples into physically isolated chambers (cdPCR) or droplet emulsions (ddPCR).  We compare the strengths of dPCR (absolute quantitation, precision, and ability to detect rare or mutant targets) with those of its predecessor, quantitative real-time PCR (dynamic range, larger sample volumes, and throughput).  Lastly, we describe several promising applications of dPCR, including copy number variation, quantitation of circulating tumor DNA and viral load, RNA/miRNA quantitation with reverse transcription dPCR, and library preparation for next-generation sequencing.  This review is intended to give a broad perspective to scientists interested in adopting digital assays into their workflows.  Part II focuses on digital protein and cell assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRylYUBtyUvJoUSrryheKE6fW6udTcc2eYMdb9Je3xaJrntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BC1crhslCitg%253D%253D&amp;md5=25b010effce8a7693d5b82faf8dfb050</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1177%2F2472630317705680&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472630317705680%26sid%3Dliteratum%253Aachs%26aulast%3DBasu%26aufirst%3DA.%2BS.%26atitle%3DDigital%2520Assays%2520Part%2520I%253A%2520Partitioning%2520Statistics%2520and%2520Digital%2520PCR%26jtitle%3DSLAS%2520Technol.%26date%3D2017%26volume%3D22%26spage%3D369%26epage%3D386%26doi%3D10.1177%2F2472630317705680" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vashist, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudanyali, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Cellphone-Based Devices for Bioanalytical Sciences</span>. <i>Anal. Bioanal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>406</i></span>,  <span class="NLM_fpage">3263</span>– <span class="NLM_lpage">3277</span>, <span class="refDoi">&nbsp;DOI: 10.1007/s00216-013-7473-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1007%2Fs00216-013-7473-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=24287630" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVKisbnI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=406&amp;publication_year=2014&amp;pages=3263-3277&amp;author=S.+K.+Vashistauthor=O.+Mudanyaliauthor=E.+M.+Schneiderauthor=R.+Zengerleauthor=A.+Ozcan&amp;title=Cellphone-Based+Devices+for+Bioanalytical+Sciences&amp;doi=10.1007%2Fs00216-013-7473-1" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cellphone-based devices for bioanalytical sciences</span></div><div class="casAuthors">Vashist, Sandeep Kumar; Mudanyali, Onur; Schneider, E. Marion; Zengerle, Roland; Ozcan, Aydogan</div><div class="citationInfo"><span class="NLM_cas:title">Analytical and Bioanalytical Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">406</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3263-3277</span>CODEN:
                <span class="NLM_cas:coden">ABCNBP</span>;
        ISSN:<span class="NLM_cas:issn">1618-2642</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  During the last decade, there has been a rapidly growing trend toward the use of cellphone-based devices (CBDs) in bioanal. sciences.  For example, they have been used for digital microscopy, cytometry, read-out of immunoassays and lateral flow tests, electrochem. and surface plasmon resonance based bio-sensing, colorimetric detection and healthcare monitoring, among others.  Cellphone can be considered as one of the most prospective devices for the development of next-generation point-of-care (POC) diagnostics platforms, enabling mobile healthcare delivery and personalized medicine.  With more than 6.5 billion cellphone subscribers worldwide and approx. 1.6 billion new devices being sold each year, cellphone technol. is also creating new business and research opportunities.  Many cellphone-based devices, such as those targeted for diabetic management, wt. management, monitoring of blood pressure and pulse rate, have already become com.-available in recent years.  In addn. to such monitoring platforms, several other CBDs are also being introduced, targeting e.g., microscopic imaging and sensing applications for medical diagnostics using novel computational algorithms and components already embedded on cellphones.  This report aims to review these recent developments in CBDs for bioanal. sciences along with some of the challenges involved and the future opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf2OqKsrjVvLVg90H21EOLACvtfcHk0lji-LzhJBxBuw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC3sXhvVKisbnI&amp;md5=31f88a69fa1460341913991b0dee933d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs00216-013-7473-1&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00216-013-7473-1%26sid%3Dliteratum%253Aachs%26aulast%3DVashist%26aufirst%3DS.%2BK.%26aulast%3DMudanyali%26aufirst%3DO.%26aulast%3DSchneider%26aufirst%3DE.%2BM.%26aulast%3DZengerle%26aufirst%3DR.%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DCellphone-Based%2520Devices%2520for%2520Bioanalytical%2520Sciences%26jtitle%3DAnal.%2520Bioanal.%2520Chem.%26date%3D2014%26volume%3D406%26spage%3D3263%26epage%3D3277%26doi%3D10.1007%2Fs00216-013-7473-1" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Mobile Phones Democratize and Cultivate Next-Generation Imaging, Diagnostics and Measurement Tools</span>. <i>Lab Chip</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">3187</span>– <span class="NLM_lpage">3194</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C4LC00010B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1039%2FC4LC00010B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=24647550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1CgtbrE" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=2014&amp;pages=3187-3194&amp;author=A.+Ozcan&amp;title=Mobile+Phones+Democratize+and+Cultivate+Next-Generation+Imaging%2C+Diagnostics+and+Measurement+Tools&amp;doi=10.1039%2FC4LC00010B" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mobile phones democratize and cultivate next-generation imaging, diagnostics and measurement tools</span></div><div class="casAuthors">Ozcan, Aydogan</div><div class="citationInfo"><span class="NLM_cas:title">Lab on a Chip</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3187-3194</span>CODEN:
                <span class="NLM_cas:coden">LCAHAM</span>;
        ISSN:<span class="NLM_cas:issn">1473-0189</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In this article, I discuss some of the emerging applications and the future opportunities and challenges created by the use of mobile phones and their embedded components for the development of next-generation imaging, sensing, diagnostics and measurement tools.  The massive vol. of mobile phone users, which has now reached ∼7 billion, drives the rapid improvements of the hardware, software and high-end imaging and sensing technologies embedded in our phones, transforming the mobile phone into a cost-effective and yet extremely powerful platform to run, e.g., biomedical tests, and perform scientific measurements that would normally require advanced lab. instruments.  This rapidly evolving and continuing trend will help us transform how medicine, engineering and sciences are practiced and taught globally.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNnKufjULYsLVg90H21EOLACvtfcHk0lgK2ZBPlW_WyQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXht1CgtbrE&amp;md5=cf19e56f8c499eca5cf56884d2f28257</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2FC4LC00010B&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4LC00010B%26sid%3Dliteratum%253Aachs%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DMobile%2520Phones%2520Democratize%2520and%2520Cultivate%2520Next-Generation%2520Imaging%252C%2520Diagnostics%2520and%2520Measurement%2520Tools%26jtitle%3DLab%2520Chip%26date%3D2014%26volume%3D14%26spage%3D3187%26epage%3D3194%26doi%3D10.1039%2FC4LC00010B" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ornob, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brisbin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganguli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vemuri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strzebonski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, K. J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Mobile Platform for Multiplexed Detection and Differentiation of Disease-Specific Nucleic Acid Sequences, Using Microfluidic Loop-Mediated Isothermal Amplification and Smartphone Detection</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">11219</span>– <span class="NLM_lpage">11226</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.analchem.7b02478</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.7b02478" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlGksrjI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=89&amp;publication_year=2017&amp;pages=11219-11226&amp;author=W.+Chenauthor=H.+Yuauthor=F.+Sunauthor=A.+Ornobauthor=R.+Brisbinauthor=A.+Ganguliauthor=V.+Vemuriauthor=P.+Strzebonskiauthor=G.+Cuiauthor=K.+J.+Allen&amp;title=Mobile+Platform+for+Multiplexed+Detection+and+Differentiation+of+Disease-Specific+Nucleic+Acid+Sequences%2C+Using+Microfluidic+Loop-Mediated+Isothermal+Amplification+and+Smartphone+Detection&amp;doi=10.1021%2Facs.analchem.7b02478" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mobile Platform for Multiplexed Detection and Differentiation of Disease-Specific Nucleic Acid Sequences, Using Microfluidic Loop-Mediated Isothermal Amplification and Smartphone Detection</span></div><div class="casAuthors">Chen, Weili; Yu, Hojeong; Sun, Fu; Ornob, Akid; Brisbin, Ryan; Ganguli, Anurup; Vemuri, Vinay; Strzebonski, Piotr; Cui, Guangzhe; Allen, Karen J.; Desai, Smit A.; Lin, Weiran; Nash, David M.; Hirschberg, David L.; Brooks, Ian; Bashir, Rashid; Cunningham, Brian T.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11219-11226</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New tools are needed to enable rapid detection, identification, and reporting of infectious viral and microbial pathogens in a wide variety of point-of-care applications that impact human and animal health.  We report the design, construction, and characterization of a platform for multiplexed anal. of disease-specific DNA sequences that utilizes a smartphone camera as the sensor in conjunction with a handheld "cradle" that interfaces the phone with a silicon-based microfluidic chip embedded within a credit card-sized cartridge.  Utilizing specific nucleic acid sequences for four equine respiratory pathogens as representative examples, we demonstrate the ability of the system to utilize a single 15 μL droplet of test sample to perform selective pos./neg. detn. of target sequences, including integrated exptl. controls, in approx. 30 min.  Our approach utilizes loop mediated isothermal amplification (LAMP) reagents pre-deposited into distinct lanes of the microfluidic chip, which, when exposed to target nucleic acid sequences from the test sample, generates fluorescent products that, when excited by appropriately selected light emitting diodes (LEDs) are visualized and automatically analyzed by a software application running on the smartphone microprocessor.  The system achieves detection limits comparable to those obtained by lab.-based methods and instruments.  Assay information is combined with information from the cartridge and the patient to populate a cloud-based database for epidemiol. reporting of test results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqchhgJ2_pqV7Vg90H21EOLACvtfcHk0lgK2ZBPlW_WyQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXhtlGksrjI&amp;md5=976364717fe65488949dfd802136e3b7</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.7b02478&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.7b02478%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DF.%26aulast%3DOrnob%26aufirst%3DA.%26aulast%3DBrisbin%26aufirst%3DR.%26aulast%3DGanguli%26aufirst%3DA.%26aulast%3DVemuri%26aufirst%3DV.%26aulast%3DStrzebonski%26aufirst%3DP.%26aulast%3DCui%26aufirst%3DG.%26aulast%3DAllen%26aufirst%3DK.%2BJ.%26atitle%3DMobile%2520Platform%2520for%2520Multiplexed%2520Detection%2520and%2520Differentiation%2520of%2520Disease-Specific%2520Nucleic%2520Acid%2520Sequences%252C%2520Using%2520Microfluidic%2520Loop-Mediated%2520Isothermal%2520Amplification%2520and%2520Smartphone%2520Detection%26jtitle%3DAnal.%2520Chem.%26date%3D2017%26volume%3D89%26spage%3D11219%26epage%3D11226%26doi%3D10.1021%2Facs.analchem.7b02478" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sannarangaiah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dash, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morita, K.</span></span> <span> </span><span class="NLM_article-title">Loop Mediated Isothermal Amplification (LAMP): A New Generation of Innovative Gene Amplification Technique; Perspectives in Clinical Diagnosis of Infectious Diseases</span>. <i>Rev. Med. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">421</span>, <span class="refDoi">&nbsp;DOI: 10.1002/rmv.593</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1002%2Frmv.593" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=18716992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFWrt7jK" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=18&amp;publication_year=2008&amp;pages=407-421&amp;author=M.+Paridaauthor=S.+Sannarangaiahauthor=P.+K.+Dashauthor=P.+V.+L.+Raoauthor=K.+Morita&amp;title=Loop+Mediated+Isothermal+Amplification+%28LAMP%29%3A+A+New+Generation+of+Innovative+Gene+Amplification+Technique%3B+Perspectives+in+Clinical+Diagnosis+of+Infectious+Diseases&amp;doi=10.1002%2Frmv.593" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Loop mediated isothermal amplication (LAMP): a new generation of innovative gene amplification technique; perspectives in clinical diagnosis of infectious diseases</span></div><div class="casAuthors">Parida, Manmohan; Sannarangaiah, Santhosh; Dash, Paban Kumar; Rao, P. V. L.; Morita, Kouichi</div><div class="citationInfo"><span class="NLM_cas:title">Reviews in Medical Virology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">407-421</span>CODEN:
                <span class="NLM_cas:coden">RMVIEW</span>;
        ISSN:<span class="NLM_cas:issn">1052-9276</span>.
    
            (<span class="NLM_cas:orgname">John Wiley &amp; Sons Ltd.</span>)
        </div><div class="casAbstract">A review.  Loop mediated isothermal amplification (LAMP) is a powerful innovative gene amplification technique emerging as a simple rapid diagnostic tool for early detection and identification of microbial diseases.  The whole procedure is very simple and rapid wherein the amplification can be completed in less than 1 h under isothermal conditions employing a set of six specially designed primers spanning eight distinct sequences of a target gene, by incubating all the reagents in a single tube.  Gene amplification products can be detected by agarose gel electrophoresis as well as by real-time monitoring in an inexpensive turbidimeter.  Gene copy no. can also be quantified with the help of a std. curve generated from different concns. of gene copy no. plotted against time of positivity with the help of a real-time turbidimeter.  Alternatively, gene amplification can be visualized by the naked eye either as turbidity or in the form of a color change when SYBR Green I, a fluorescent dsDNA intercalating dye, is employed.  LAMP does not require a thermal cycler and can be performed simply with a heating block and/or water bath.  Considering the advantages of rapid amplification, simple operation and easy detection, LAMP has potential applications for clin. diagnosis as well as surveillance of infectious diseases in developing countries without requiring sophisticated equipment or skilled personnel.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxY1refChnwbVg90H21EOLACvtfcHk0lgK2ZBPlW_WyQ"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD1cXhsFWrt7jK&amp;md5=c01ca23fb35cb2d53fa43e6fd658384b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Frmv.593&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Frmv.593%26sid%3Dliteratum%253Aachs%26aulast%3DParida%26aufirst%3DM.%26aulast%3DSannarangaiah%26aufirst%3DS.%26aulast%3DDash%26aufirst%3DP.%2BK.%26aulast%3DRao%26aufirst%3DP.%2BV.%2BL.%26aulast%3DMorita%26aufirst%3DK.%26atitle%3DLoop%2520Mediated%2520Isothermal%2520Amplification%2520%2528LAMP%2529%253A%2520A%2520New%2520Generation%2520of%2520Innovative%2520Gene%2520Amplification%2520Technique%253B%2520Perspectives%2520in%2520Clinical%2520Diagnosis%2520of%2520Infectious%2520Diseases%26jtitle%3DRev.%2520Med.%2520Virol.%26date%3D2008%26volume%3D18%26spage%3D407%26epage%3D421%26doi%3D10.1002%2Frmv.593" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morton, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shopsowitz, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, P. T.</span></span> <span> </span><span class="NLM_article-title">RNAi-Microsponges Form through Self-Assembly of the Organic and Inorganic Products of Transcription</span>. <i>Small</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1623</span>– <span class="NLM_lpage">1633</span>, <span class="refDoi">&nbsp;DOI: 10.1002/smll.201302676</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1002%2Fsmll.201302676" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2013&amp;pages=1623-1633&amp;author=S.+W.+Mortonauthor=Z.+J.+Dengauthor=Y.+H.+Rohauthor=K.+E.+Shopsowitzauthor=P.+T.+Hammond&amp;title=RNAi-Microsponges+Form+through+Self-Assembly+of+the+Organic+and+Inorganic+Products+of+Transcription&amp;doi=10.1002%2Fsmll.201302676" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fsmll.201302676&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fsmll.201302676%26sid%3Dliteratum%253Aachs%26aulast%3DMorton%26aufirst%3DS.%2BW.%26aulast%3DDeng%26aufirst%3DZ.%2BJ.%26aulast%3DRoh%26aufirst%3DY.%2BH.%26aulast%3DShopsowitz%26aufirst%3DK.%2BE.%26aulast%3DHammond%26aufirst%3DP.%2BT.%26atitle%3DRNAi-Microsponges%2520Form%2520through%2520Self-Assembly%2520of%2520the%2520Organic%2520and%2520Inorganic%2520Products%2520of%2520Transcription%26jtitle%3DSmall%26date%3D2013%26volume%3D10%26spage%3D1623%26epage%3D1633%26doi%3D10.1002%2Fsmll.201302676" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishikawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takakura, Y.</span></span> <span> </span><span class="NLM_article-title">Application of Magnesium Pyrophosphate-Based Sponge-Like Microparticles to Enhance the Delivery Efficiency and Adjuvant Effects of Polyriboinosinic-Polyribocytidylic Acid in Immune Cells</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">766</span>– <span class="NLM_lpage">772</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.xphs.2015.11.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.xphs.2015.11.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=26869429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFegu7fI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=105&amp;publication_year=2016&amp;pages=766-772&amp;author=S.+Takagiauthor=Y.+Takahashiauthor=K.+Sugimuraauthor=M.+Nishikawaauthor=Y.+Takakura&amp;title=Application+of+Magnesium+Pyrophosphate-Based+Sponge-Like+Microparticles+to+Enhance+the+Delivery+Efficiency+and+Adjuvant+Effects+of+Polyriboinosinic-Polyribocytidylic+Acid+in+Immune+Cells&amp;doi=10.1016%2Fj.xphs.2015.11.020" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Magnesium Pyrophosphate-Based Sponge-Like Microparticles to Enhance the Delivery Efficiency and Adjuvant Effects of Polyriboinosinic-Polyribocytidylic Acid in Immune Cells</span></div><div class="casAuthors">Takagi, Shoichi; Takahashi, Yuki; Sugimura, Kanako; Nishikawa, Makiya; Takakura, Yoshinobu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">766-772</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The magnesium pyrophosphate particle (MgPP) is a unique and safe carrier that is prepd. by simply mixing magnesium chloride and sodium pyrophosphate.  In this study, we investigated whether MgPP can be used to deliver nucleic acid-based adjuvants to immune cells.  Polyriboinosinic-polyribocytidylic acid (polyI:C), a ligand for toll-like receptor 3, was selected as a model nucleic acid-based adjuvant.  PolyI:C-loaded MgPP (polyI:C-MgPP) was prepd. by adding polyI:C during the MgPP prepn. process.  Efficient loading of polyI:C into MgPP was confirmed by measuring the absorbance at 260 nm after disruption of polyI:C-MgPP by EDTA.  SEM revealed that both MgPP and polyI:C-MgPP had a unique sponge-like shape with a diam. of approx. 1 μm.  PolyI:C-MgPP was more efficiently taken up by toll-like receptor 3-pos. RAW264.7 cells than naked polyI:C, and its uptake stimulated increased tumor necrosis factor-α prodn.  When the presentation of ovalbumin (OVA), a model antigen, was evaluated after the addn. of OVA along with naked polyI:C or polyI:C-MgPP to mouse dendritic DC2.4 cells, polyI:C-MgPP substantially increased OVA presentation.  These results indicate that MgPP is a useful delivery vehicle for polyI:C and that polyI:C-MgPP is an effective immune cell adjuvant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXYZ7dhZbUh7Vg90H21EOLACvtfcHk0li3hOCrSUAKGg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XhtFegu7fI&amp;md5=a2363dec306eadd04cf6bfd326f1ac88</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.xphs.2015.11.020&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.xphs.2015.11.020%26sid%3Dliteratum%253Aachs%26aulast%3DTakagi%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DY.%26aulast%3DSugimura%26aufirst%3DK.%26aulast%3DNishikawa%26aufirst%3DM.%26aulast%3DTakakura%26aufirst%3DY.%26atitle%3DApplication%2520of%2520Magnesium%2520Pyrophosphate-Based%2520Sponge-Like%2520Microparticles%2520to%2520Enhance%2520the%2520Delivery%2520Efficiency%2520and%2520Adjuvant%2520Effects%2520of%2520Polyriboinosinic-Polyribocytidylic%2520Acid%2520in%2520Immune%2520Cells%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2016%26volume%3D105%26spage%3D766%26epage%3D772%26doi%3D10.1016%2Fj.xphs.2015.11.020" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monis, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giglio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saint, C. P.</span></span> <span> </span><span class="NLM_article-title">Comparison of SYTO9 and SYBR Green I for Real-Time Polymerase Chain Reaction and Investigation of the Effect of Dye Concentration on Amplification and DNA Melting Curve Analysis</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>340</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">34</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.ab.2005.01.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.ab.2005.01.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=15802126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivV2jsL4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=340&amp;publication_year=2005&amp;pages=24-34&amp;author=P.+T.+Monisauthor=S.+Giglioauthor=C.+P.+Saint&amp;title=Comparison+of+SYTO9+and+SYBR+Green+I+for+Real-Time+Polymerase+Chain+Reaction+and+Investigation+of+the+Effect+of+Dye+Concentration+on+Amplification+and+DNA+Melting+Curve+Analysis&amp;doi=10.1016%2Fj.ab.2005.01.046" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of SYTO9 and SYBR Green I for real-time polymerase chain reaction and investigation of the effect of dye concentration on amplification and DNA melting curve analysis</span></div><div class="casAuthors">Monis, Paul T.; Giglio, Steven; Saint, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">340</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-34</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Following the initial report of the use of SYBR Green I for real-time polymerase chain reaction (PCR) in 1997, little attention has been given to the development of alternative intercalating dyes for this application.  This is surprising considering the reported limitations of SYBR Green I, which include limited dye stability, dye-dependent PCR inhibition, and selective detection of amplicons during DNA melting curve anal. of multiplex PCRs.  We have tested an alternative to SYBR Green I and report the first detailed evaluation of the intercalating dye SYTO9.  Our findings demonstrate that SYTO9 produces highly reproducible DNA melting curves over a broader range of dye concns. than does SYBR Green I, is far less inhibitory to PCR than SYBR Green I, and does not appear to selectively detect particular amplicons.  The low inhibition and high melting curve reproducibility of SYTO9 means that it can be readily incorporated into a conventional PCR at a broad range of concns., allowing closed tube anal. by DNA melting curve anal.  These features simplify the use of intercalating dyes in real-time PCR and the improved reproducibility of DNA melting curve anal. will make SYTO9 useful in a diagnostic context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPDPuKLUL1BrVg90H21EOLACvtfcHk0li3hOCrSUAKGg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2MXivV2jsL4%253D&amp;md5=97c6fdb3aff06fdf97f8064907b99f40</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2005.01.046&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2005.01.046%26sid%3Dliteratum%253Aachs%26aulast%3DMonis%26aufirst%3DP.%2BT.%26aulast%3DGiglio%26aufirst%3DS.%26aulast%3DSaint%26aufirst%3DC.%2BP.%26atitle%3DComparison%2520of%2520SYTO9%2520and%2520SYBR%2520Green%2520I%2520for%2520Real-Time%2520Polymerase%2520Chain%2520Reaction%2520and%2520Investigation%2520of%2520the%2520Effect%2520of%2520Dye%2520Concentration%2520on%2520Amplification%2520and%2520DNA%2520Melting%2520Curve%2520Analysis%26jtitle%3DAnal.%2520Biochem.%26date%3D2005%26volume%3D340%26spage%3D24%26epage%3D34%26doi%3D10.1016%2Fj.ab.2005.01.046" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gudnason, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bang, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolff, A.</span></span> <span> </span><span class="NLM_article-title">Comparison of Multiple DNA Dyes for Real-Time PCR: Effects of Dye Concentration and Sequence Composition on DNA Amplification and Melting Temperature</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>, <span class="NLM_elocation-id">e127</span> <span class="refDoi">&nbsp;DOI: 10.1093/nar/gkm671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1093%2Fnar%2Fgkm671" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=17897966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1yktbvJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=35&amp;publication_year=2007&amp;author=H.+Gudnasonauthor=M.+Dufvaauthor=D.+D.+Bangauthor=A.+Wolff&amp;title=Comparison+of+Multiple+DNA+Dyes+for+Real-Time+PCR%3A+Effects+of+Dye+Concentration+and+Sequence+Composition+on+DNA+Amplification+and+Melting+Temperature&amp;doi=10.1093%2Fnar%2Fgkm671" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of multiple DNA dyes for real-time PCR: Effects of dye concentration and sequence composition on DNA amplification and melting temperature</span></div><div class="casAuthors">Gudnason, Haukur; Dufva, Martin; Bang, D. D.; Wolff, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">e127/1-e127/8</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The importance of real-time polymerase chain reaction (PCR) has increased steadily in clin. applications over the last decade.  Many applications utilize SYBR Green I dye to follow the accumulation of amplicons in real time.  SYBR Green I has, however, a no. of limitations that include the inhibition of PCR, preferential binding to GC-rich sequences and effects on melting curve anal.  Although a few alternative dyes without some of these limitations have been recently proposed, no large-scale investigation into the properties of intercalating dyes has been performed.  In this study, we investigate 15 different intercalating DNA dyes for their inhibitory effects on PCR, effects on DNA melting temp. and possible preferential binding to GC-rich sequences.  Our results demonstrated that in contrast to the results of SYBR Green I, two intercalating dyes SYTO-13 and SYTO-82 do not inhibit PCR, show no preferential binding to GC rich sequences and do not influence melting temp., Tm, even at high concns.  In addn., SYTO-82 demonstrated a 50-fold lower detection limit in a diln. series assay.  In conclusion, the properties of SYTO-82 and SYTO-13 will simplify the development of multiplex assays and increase the sensitivity of real-time PCR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpALrHgnh2robVg90H21EOLACvtfcHk0li3hOCrSUAKGg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BD2sXht1yktbvJ&amp;md5=fd0e885f9ec6957e8165bcf31b2504a5</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm671&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm671%26sid%3Dliteratum%253Aachs%26aulast%3DGudnason%26aufirst%3DH.%26aulast%3DDufva%26aufirst%3DM.%26aulast%3DBang%26aufirst%3DD.%2BD.%26aulast%3DWolff%26aufirst%3DA.%26atitle%3DComparison%2520of%2520Multiple%2520DNA%2520Dyes%2520for%2520Real-Time%2520PCR%253A%2520Effects%2520of%2520Dye%2520Concentration%2520and%2520Sequence%2520Composition%2520on%2520DNA%2520Amplification%2520and%2520Melting%2520Temperature%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26doi%3D10.1093%2Fnar%2Fgkm671" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, X.</span></span> <span> </span><span class="NLM_article-title">Characterization of EvaGreen and the Implication of Its Physicochemical Properties for QPCR Applications</span>. <i>BMC Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">76</span> <span class="refDoi">&nbsp;DOI: 10.1186/1472-6750-7-76</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1186%2F1472-6750-7-76" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=17996102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A280%3ADC%252BD1c%252FmtVSksw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2007&amp;author=F.+Maoauthor=W.-Y.+Leungauthor=X.+Xin&amp;title=Characterization+of+EvaGreen+and+the+Implication+of+Its+Physicochemical+Properties+for+QPCR+Applications&amp;doi=10.1186%2F1472-6750-7-76" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of EvaGreen and the implication of its physicochemical properties for qPCR applications</span></div><div class="casAuthors">Mao Fei; Leung Wai-Yee; Xin Xing</div><div class="citationInfo"><span class="NLM_cas:title">BMC biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">76</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  EvaGreen (EG) is a newly developed DNA-binding dye that has recently been used in quantitative real-time PCR (qPCR), post-PCR DNA melt curve analysis and several other applications.  However, very little is known about the physicochemical properties of the dye and their relevance to the applications, particularly to qPCR and post PCR DNA melt curve analysis.  In this paper, we characterized EG along with a widely used qPCR dye, SYBR Green I (SG), for their DNA-binding properties and stability, and compared their performance in qPCR under a variety of conditions.  RESULTS:  This study systematically compared the DNA binding profiles of the two dyes under different conditions and had these findings: a) EG had a lower binding affinity for both double-stranded DNA (dsDNA) and single-stranded DNA (ssDNA) than SG; b) EG showed no apparent preference for either GC- or AT-rich sequence while SG had a slight preference for AT-rich sequence; c) both dyes showed substantially lower affinity toward ssDNA than toward dsDNA and even lower affinity toward shorter ssDNA fragments except that this trend was more pronounced for EG.  Our results also demonstrated that EG was stable both under PCR condition and during routine storage and handling.  In the comparative qPCR study, both EG and SG exhibited PCR interference when used at high dye concentration, as evident from delayed Ct and/or nonspecific product formation.  The problem worsened when the chain extension time was shortened or when the amplicon size was relatively long (&gt;500 bp).  However, qPCR using EG tolerated a significantly higher dye concentration, thus permitting a more robust PCR signal as well as a sharper and stronger DNA melt peak.  These differences in qPCR performance between the two dyes are believed to be attributable to their differences in DNA binding profiles.  CONCLUSION:  These findings suggest that an ideal qPCR dye should possess several DNA-binding characteristics, including a "just right" affinity for dsDNA and low or no affinity for ssDNA and short DNA fragments.  The favorable DNA-binding profile of EG, coupled with its good stability and instrument-compatibility, should make EG a promising dye for qPCR and related applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZzfmbgDKOB7qxQ2mb9Pu8fW6udTcc2eYxkawTmAbMK7ntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BD1c%252FmtVSksw%253D%253D&amp;md5=0fd4d90f0c27051a4d9bed6c12bfc7fb</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1186%2F1472-6750-7-76&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6750-7-76%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DF.%26aulast%3DLeung%26aufirst%3DW.-Y.%26aulast%3DXin%26aufirst%3DX.%26atitle%3DCharacterization%2520of%2520EvaGreen%2520and%2520the%2520Implication%2520of%2520Its%2520Physicochemical%2520Properties%2520for%2520QPCR%2520Applications%26jtitle%3DBMC%2520Biotechnol.%26date%3D2007%26volume%3D7%26doi%3D10.1186%2F1472-6750-7-76" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nath, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarosy, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Como, C. J.</span></span> <span> </span><span class="NLM_article-title">Effects of Ethidium Bromide and SYBR Green I on Different Polymerase Chain Reaction Systems</span>. <i>J. Biochem. Biophys. Methods</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">29</span>, <span class="refDoi">&nbsp;DOI: 10.1016/S0165-022X(99)00033-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2FS0165-022X%2899%2900033-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10647811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADyaK1MXns1ahsLY%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=42&amp;publication_year=2000&amp;pages=15-29&amp;author=K.+Nathauthor=J.+W.+Sarosyauthor=J.+Hahnauthor=C.+J.+Di+Como&amp;title=Effects+of+Ethidium+Bromide+and+SYBR+Green+I+on+Different+Polymerase+Chain+Reaction+Systems&amp;doi=10.1016%2FS0165-022X%2899%2900033-0" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of ethidium bromide and SYBR Green I on different polymerase chain reaction systems</span></div><div class="casAuthors">Nath, K.; Sarosy, J. W.; Hahn, J.; Di Como, C. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemical and Biophysical Methods</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">15-29</span>CODEN:
                <span class="NLM_cas:coden">JBBMDG</span>;
        ISSN:<span class="NLM_cas:issn">0165-022X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">In an in-gel polymerase chain reaction (PCR), the generation of a 1750-bp yeast DNA fragment was inhibited when yeast DNA gel-stabs or gel-slices stained with ethidium bromide (EtBr) or SYBR Green I were used.  Similar inhibition occurred to a varying degree in the reamplification of PCR fragments in prokaryotic systems.  Inclusion of the dyes in PCR resulted in an inhibition at about 10 μg/mL EtBr and at 10,000-20,000-fold diln. of SYBR Green I in all systems.  The effect remained unchanged despite increasing the PCR cycles to 40.  However, increasing the magnesium chloride concn. did reverse the inhibitory actions, although the PCR specificity was lost.  In an unusual observation, we find that, at higher dye concns. (50 μg/mL EtBr, or thousand fold diln. of SYBR Green I), the input yeast DNA electrophoretic profile is maintained following 25 PCR cycles (despite a denaturation temp. of 94°C).  It varied significantly in different DNA systems and was readily reversed by high Mg++ concns.  It is concluded that, at low Mg++ concns., different PCR systems are inhibited to varying extents by intercalating dyes and, in some PCR systems, intercalating dyes at unusually high concns. maintain input DNA electrophoretic profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUYIh2z_UNZbVg90H21EOLACvtfcHk0lj2RWKXHhgIEw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADyaK1MXns1ahsLY%253D&amp;md5=7f96a617ff4f57f8978150201d8da69d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS0165-022X%2899%2900033-0&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0165-022X%252899%252900033-0%26sid%3Dliteratum%253Aachs%26aulast%3DNath%26aufirst%3DK.%26aulast%3DSarosy%26aufirst%3DJ.%2BW.%26aulast%3DHahn%26aufirst%3DJ.%26aulast%3DDi%2BComo%26aufirst%3DC.%2BJ.%26atitle%3DEffects%2520of%2520Ethidium%2520Bromide%2520and%2520SYBR%2520Green%2520I%2520on%2520Different%2520Polymerase%2520Chain%2520Reaction%2520Systems%26jtitle%3DJ.%2520Biochem.%2520Biophys.%2520Methods%26date%3D2000%26volume%3D42%26spage%3D15%26epage%3D29%26doi%3D10.1016%2FS0165-022X%2899%2900033-0" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, O. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Carlo, D.</span></span> <span> </span><span class="NLM_article-title">Highly Stable and Sensitive Nucleic Acid Amplification and Cell-Phone-Based Readout</span>. <i>ACS Nano</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2934</span>– <span class="NLM_lpage">2943</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acsnano.6b08274</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsnano.6b08274" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlCnu70%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2017&amp;pages=2934-2943&amp;author=J.+E.+Kongauthor=Q.+Weiauthor=D.+Tsengauthor=J.+Zhangauthor=E.+Panauthor=M.+Lewinskiauthor=O.+B.+Garnerauthor=A.+Ozcanauthor=D.+Di+Carlo&amp;title=Highly+Stable+and+Sensitive+Nucleic+Acid+Amplification+and+Cell-Phone-Based+Readout&amp;doi=10.1021%2Facsnano.6b08274" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Highly stable and sensitive nucleic acid amplification and cell-phone-based readout</span></div><div class="casAuthors">Kong, Janay E.; Wei, Qingshan; Tseng, Derek; Zhang, Jingzi; Pan, Eric; Lewinski, Michael; Garner, Omai B.; Ozcan, Aydogan; Di Carlo, Dino</div><div class="citationInfo"><span class="NLM_cas:title">ACS Nano</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">2934-2943</span>CODEN:
                <span class="NLM_cas:coden">ANCAC3</span>;
        ISSN:<span class="NLM_cas:issn">1936-0851</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Key challenges with point-of-care (POC) nucleic acid tests include achieving a low-cost, portable form factor, and stable readout, while also retaining the same robust stds. of benchtop lab-based tests.  We addressed two crucial aspects of this problem, identifying a chem. additive, hydroxynaphthol blue, that both stabilizes and significantly enhances intercalator-based fluorescence readout of nucleic acid concn., and developing a cost-effective fiber-optic bundle-based fluorescence microplate reader integrated onto a mobile phone.  Using loop-mediated isothermal amplification on lambda DNA we achieve a 69-fold increase in signal above background, 20-fold higher than the gold std., yielding an overall limit of detection of 25 copies/μL within an hour using our mobile-phone-based platform.  Crit. for a point-of-care system, we achieve a &gt;60% increase in fluorescence stability as a function of temp. and time, obviating the need for manual baseline correction or secondary calibration dyes.  This field-portable and cost-effective mobile-phone-based nucleic acid amplification and readout platform is broadly applicable to other real-time nucleic acid amplification tests by similarly modulating intercalating dye performance and is compatible with any fluorescence-based assay that can be run in a 96-well microplate format, making it esp. valuable for POC and resource-limited settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHpYm7oXzpoLVg90H21EOLACvtfcHk0lj2RWKXHhgIEw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXjtlCnu70%253D&amp;md5=0d86e01ccb93d7680e9f97a31bacb09f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facsnano.6b08274&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsnano.6b08274%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DJ.%2BE.%26aulast%3DWei%26aufirst%3DQ.%26aulast%3DTseng%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DPan%26aufirst%3DE.%26aulast%3DLewinski%26aufirst%3DM.%26aulast%3DGarner%26aufirst%3DO.%2BB.%26aulast%3DOzcan%26aufirst%3DA.%26aulast%3DDi%2BCarlo%26aufirst%3DD.%26atitle%3DHighly%2520Stable%2520and%2520Sensitive%2520Nucleic%2520Acid%2520Amplification%2520and%2520Cell-Phone-Based%2520Readout%26jtitle%3DACS%2520Nano%26date%3D2017%26volume%3D11%26spage%3D2934%26epage%3D2943%26doi%3D10.1021%2Facsnano.6b08274" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azizi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaferani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheong, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbaspourrad, A.</span></span> <span> </span><span class="NLM_article-title">Pathogenic Bacteria Detection Using RNA-Based Loop-Mediated Isothermal-Amplification-Assisted Nucleic Acid Amplification via Droplet Microfluidics</span>. <i>ACS Sens.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">848</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acssensors.8b01206</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acssensors.8b01206" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC1MXls1OgsLw%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2019&amp;pages=841-848&amp;author=M.+Aziziauthor=M.+Zaferaniauthor=S.+H.+Cheongauthor=A.+Abbaspourrad&amp;title=Pathogenic+Bacteria+Detection+Using+RNA-Based+Loop-Mediated+Isothermal-Amplification-Assisted+Nucleic+Acid+Amplification+via+Droplet+Microfluidics&amp;doi=10.1021%2Facssensors.8b01206" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Pathogenic Bacteria Detection Using RNA-Based Loop-Mediated Isothermal-Amplification-Assisted Nucleic Acid Amplification via Droplet Microfluidics</span></div><div class="casAuthors">Azizi, Morteza; Zaferani, Meisam; Cheong, Soon Hon; Abbaspourrad, Alireza</div><div class="citationInfo"><span class="NLM_cas:title">ACS Sensors</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">841-848</span>CODEN:
                <span class="NLM_cas:coden">ASCEFJ</span>;
        ISSN:<span class="NLM_cas:issn">2379-3694</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nucleic acid amplifications, such as PCR, are very beneficial for diagnostic applications, esp. in the context of bacterial or viral outbreaks due to their high specificity and sensitivity.  However, the need for bulky instrumentation and complicated protocols makes these methods expensive and slow, particularly for low nos. of RNA or DNA templates.  In addn., implementing conventional nucleic acid amplification in a high-throughput manner is both reagent- and time-consuming.  We bring droplet-based microfluidics and loop-mediated isothermal amplification (LAMP) together in an optimized operational condition to provide a sensitive biosensor for amplifying extd. RNA templates for the detection of Salmonella typhimurium (targeting the invA gene).  By simultaneously performing ∼106 LAMP-assisted amplification reactions in picoliter-sized droplets and applying a new math. model for the no. of droplets necessary to screen for the first pos. droplet, we study the detection limit of our platform with pure culture and real samples (bacterial contaminated milk samples).  Our LAMP-assisted droplet-based microfluidic technique was simple in operation, sensitive, specific, and rapid for the detection of pathogenic bacteria Salmonella typhimurium in comparison with well-established conventional methods.  More importantly, the high-throughput nature of this technique makes it suitable for many applications in biol. assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnVJ7oJxUmcbVg90H21EOLACvtfcHk0lj2RWKXHhgIEw"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXls1OgsLw%253D&amp;md5=240c687d807abc0860416d16cb5d6c79</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facssensors.8b01206&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facssensors.8b01206%26sid%3Dliteratum%253Aachs%26aulast%3DAzizi%26aufirst%3DM.%26aulast%3DZaferani%26aufirst%3DM.%26aulast%3DCheong%26aufirst%3DS.%2BH.%26aulast%3DAbbaspourrad%26aufirst%3DA.%26atitle%3DPathogenic%2520Bacteria%2520Detection%2520Using%2520RNA-Based%2520Loop-Mediated%2520Isothermal-Amplification-Assisted%2520Nucleic%2520Acid%2520Amplification%2520via%2520Droplet%2520Microfluidics%26jtitle%3DACS%2520Sens.%26date%3D2019%26volume%3D4%26spage%3D841%26epage%3D848%26doi%3D10.1021%2Facssensors.8b01206" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, W.</span></span> <span> </span><span class="NLM_article-title">Absolute Quantification of H5-Subtype Avian Influenza Viruses Using Droplet Digital Loop-Mediated Isothermal Amplification</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">745</span>– <span class="NLM_lpage">750</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acs.analchem.6b03328</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.6b03328" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFKnsrjI" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=89&amp;publication_year=2017&amp;pages=745-750&amp;author=Y.+Huauthor=P.+Xuauthor=J.+Luoauthor=H.+Heauthor=W.+Du&amp;title=Absolute+Quantification+of+H5-Subtype+Avian+Influenza+Viruses+Using+Droplet+Digital+Loop-Mediated+Isothermal+Amplification&amp;doi=10.1021%2Facs.analchem.6b03328" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Absolute Quantification of H5-Subtype Avian Influenza Viruses Using Droplet Digital Loop-Mediated Isothermal Amplification</span></div><div class="casAuthors">Hu, Yi; Xu, Peng; Luo, Jing; He, Hongxuan; Du, Wenbin</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">745-750</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human infection with avian influenza A H5N1 viruses can cause severe diseases with high mortality rate and continues to pose a significant threat to global public health.  Rapid diagnosis is needed for identifying the types of influenza viruses for making timely treatment decisions.  Here, we demonstrate abs. quantification of H5-subtype influenza viruses by digital loop-mediated isothermal amplification (dLAMP) on our recently developed cross-interface emulsification (XiE) method.  Our results show that XiE-based dLAMP is highly specific and displays comparable sensitivity to real-time PCR (qPCR) and digital PCR (dPCR).  Notably, dLAMP is more tolerant to inhibitory substances than PCR methods and demonstrated similar detection efficiency to qPCR for real H5N1 samples.  Therefore, it can serve as a robust and precise alternative to qPCR or dPCR and is esp. suitable for environmental and clin. samples with hard-to-remove contaminants.  We believe that our dLAMP method offers great potential for rapid and accurate diagnosis of influenza and other infectious diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgspF_1xoxdrVg90H21EOLACvtfcHk0liU7eEwnqsmWA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28XhvFKnsrjI&amp;md5=cd3a4afc1fb1c4339c4fbbe5dafa2fcc</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.6b03328&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.6b03328%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DP.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DDu%26aufirst%3DW.%26atitle%3DAbsolute%2520Quantification%2520of%2520H5-Subtype%2520Avian%2520Influenza%2520Viruses%2520Using%2520Droplet%2520Digital%2520Loop-Mediated%2520Isothermal%2520Amplification%26jtitle%3DAnal.%2520Chem.%26date%3D2017%26volume%3D89%26spage%3D745%26epage%3D750%26doi%3D10.1021%2Facs.analchem.6b03328" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, M. R.</span></span> <span> </span><span class="NLM_article-title">Digital Loop-Mediated Isothermal Amplification on a Commercial Membrane</span>. <i>ACS Sens.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">242</span>– <span class="NLM_lpage">249</span>, <span class="refDoi">&nbsp;DOI: 10.1021/acssensors.8b01419</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acssensors.8b01419" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text"></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlymtg%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2019&amp;pages=242-249&amp;author=X.+Linauthor=X.+Huangauthor=K.+Urmannauthor=X.+Xieauthor=M.+R.+Hoffmann&amp;title=Digital+Loop-Mediated+Isothermal+Amplification+on+a+Commercial+Membrane&amp;doi=10.1021%2Facssensors.8b01419" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Digital Loop-Mediated Isothermal Amplification on a Commercial Membrane</span></div><div class="casAuthors">Lin, Xingyu; Huang, Xiao; Urmann, Katharina; Xie, Xing; Hoffmann, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Sensors</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">242-249</span>CODEN:
                <span class="NLM_cas:coden">ASCEFJ</span>;
        ISSN:<span class="NLM_cas:issn">2379-3694</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this work, we report digital loop-mediated isothermal amplification (LAMP) or reverse-transcription LAMP (RT-LAMP) on a com. membrane, without the need for complex chip fabrication or use of specialized equipment.  Due to the pore size distribution, the theor. error for digital LAMP on these membranes was analyzed, using a combination of Random Distribution Model and Multivolume Theory.  A facile peel-off process was developed for effective droplet formation on the com. track-etched polycarbonate (PCTE) membrane.  Each pore functions as an individual nanoreactor for single DNA amplification.  Abs. quantification of bacteria genomic DNA was realized with a dynamic range from 11 to 1.1 × 105 copies/μL.  One-step digital RT-LAMP was also successfully performed on the membrane for the quantification of MS2 virus in wastewater.  With the introduction of new probes, the pos. pores can be easily distinguished from neg. ones with 100 times difference in fluorescence intensities.  Finally, the cost of a disposable membrane is less than $0.10/piece, which, to the best of our knowledge, is the most inexpensive way to perform digital LAMP.  The membrane system offers opportunities for point-of-care users or common labs. to perform digital quantification, single cell anal., or other bioassays in an inexpensive, flexible, and simplified way.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeWOqs6B7QU7Vg90H21EOLACvtfcHk0liU7eEwnqsmWA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXitlymtg%253D%253D&amp;md5=a3ba57624ee73da474dd546a5ba7d5d3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facssensors.8b01419&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facssensors.8b01419%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DUrmann%26aufirst%3DK.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DHoffmann%26aufirst%3DM.%2BR.%26atitle%3DDigital%2520Loop-Mediated%2520Isothermal%2520Amplification%2520on%2520a%2520Commercial%2520Membrane%26jtitle%3DACS%2520Sens.%26date%3D2019%26volume%3D4%26spage%3D242%26epage%3D249%26doi%3D10.1021%2Facssensors.8b01419" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rane, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zec, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span> <span> </span><span class="NLM_article-title">Microfluidic Continuous Flow Digital Loop-Mediated Isothermal Amplification (LAMP)</span>. <i>Lab Chip</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">776</span>– <span class="NLM_lpage">782</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C4LC01158A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1039%2FC4LC01158A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=25431886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2ksrrJ" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=2015&amp;pages=776-782&amp;author=T.+D.+Raneauthor=L.+Chenauthor=H.+C.+Zecauthor=T.+Wang&amp;title=Microfluidic+Continuous+Flow+Digital+Loop-Mediated+Isothermal+Amplification+%28LAMP%29&amp;doi=10.1039%2FC4LC01158A" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Microfluidic continuous flow digital loop-mediated isothermal amplification (LAMP)</span></div><div class="casAuthors">Rane, Tushar D.; Chen, Liben; Zec, Helena C.; Wang, Tza-Huei</div><div class="citationInfo"><span class="NLM_cas:title">Lab on a Chip</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">776-782</span>CODEN:
                <span class="NLM_cas:coden">LCAHAM</span>;
        ISSN:<span class="NLM_cas:issn">1473-0189</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Digital nucleic acid detection is rapidly becoming a popular technique for ultra-sensitive and quant. detection of nucleic acid mols. in a wide range of biomedical studies.  Digital polymerase chain reaction (PCR) remains the most popular way of conducting digital nucleic acid detection.  However, due to the need for thermocycling, digital PCR is difficult to implement in a streamlined manner on a single microfluidic device, leading to complex fragmented workflows and multiple sep. devices and instruments.  Loop-mediated isothermal amplification (LAMP) is an excellent isothermal alternative to PCR with potentially better specificity than PCR because of the use of multiple primer sets for a nucleic acid target.  Here we report a microfluidic droplet device implementing all the steps required for digital nucleic acid detection including droplet generation, incubation and in-line detection for digital LAMP.  As compared to microchamber or droplet array-based digital assays, the continuous flow operation of this device eliminates the constraints on the no. of total reactions imposed by the footprint of the device and the anal. throughput caused by the time for lengthy incubation and transfer of materials between instruments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6yA8HesOYebVg90H21EOLACvtfcHk0liU7eEwnqsmWA"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2ksrrJ&amp;md5=b4241832af6312d8e2daf3f7ba889999</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC4LC01158A&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4LC01158A%26sid%3Dliteratum%253Aachs%26aulast%3DRane%26aufirst%3DT.%2BD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DZec%26aufirst%3DH.%2BC.%26aulast%3DWang%26aufirst%3DT.%26atitle%3DMicrofluidic%2520Continuous%2520Flow%2520Digital%2520Loop-Mediated%2520Isothermal%2520Amplification%2520%2528LAMP%2529%26jtitle%3DLab%2520Chip%26date%3D2015%26volume%3D15%26spage%3D776%26epage%3D782%26doi%3D10.1039%2FC4LC01158A" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csurka, G.</span>; <span class="NLM_string-name">Dance, C. R.</span>; <span class="NLM_string-name">Fan, L.</span>; <span class="NLM_string-name">Willamowski, J.</span>; <span class="NLM_string-name">Bray, C.</span></span> <span> </span><span class="NLM_article-title">Visual Categorization with Bags of Keypoints</span>. In  <i>Workshop on Statistical Learning in Computer Vision, ECCV</i> <span class="NLM_year">2004</span>; pp  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">22</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2004&amp;pages=1-22&amp;author=G.+Csurka&amp;author=C.+R.+Dance&amp;author=L.+Fan&amp;author=J.+Willamowski&amp;author=C.+Bray&amp;title=Visual+Categorization+with+Bags+of+Keypoints" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DCsurka%26aufirst%3DG.%26atitle%3DVisual%2520Categorization%2520with%2520Bags%2520of%2520Keypoints%26jtitle%3DWorkshop%2520on%2520Statistical%2520Learning%2520in%2520Computer%2520Vision%252C%2520ECCV%26date%3D2004%26spage%3D1%26epage%3D22" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riche, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brutchey, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malmstadt, N.</span></span> <span> </span><span class="NLM_article-title">Flow Invariant Droplet Formation for Stable Parallel Microreactors</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">10780</span> <span class="refDoi">&nbsp;DOI: 10.1038/ncomms10780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1038%2Fncomms10780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=26902825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjt1ehtb8%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2016&amp;author=C.+T.+Richeauthor=E.+J.+Robertsauthor=M.+Guptaauthor=R.+L.+Brutcheyauthor=N.+Malmstadt&amp;title=Flow+Invariant+Droplet+Formation+for+Stable+Parallel+Microreactors&amp;doi=10.1038%2Fncomms10780" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Flow invariant droplet formation for stable parallel microreactors</span></div><div class="casAuthors">Riche, Carson T.; Roberts, Emily J.; Gupta, Malancha; Brutchey, Richard L.; Malmstadt, Noah</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10780</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The translation of batch chemistries onto continuous flow platforms requires addressing the issues of consistent fluidic behavior, channel fouling and high-throughput processing.  Droplet microfluidic technologies reduce channel fouling and provide an improved level of control over heat and mass transfer to control reaction kinetics.  However, in conventional geometries, the droplet size is sensitive to changes in flow rates.  Here we report a three-dimensional droplet generating device that exhibits flow invariant behavior and is robust to fluctuations in flow rate.  In addn., the droplet generator is capable of producing droplet vols. spanning four orders of magnitude.  We apply this device in a parallel network to synthesize platinum nanoparticles using an ionic liq. solvent, demonstrate reproducible synthesis after recycling the ionic liq., and double the reaction yield compared with an analogous batch synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7KSkRc6kOcbVg90H21EOLACvtfcHk0liliWCk3Fo7Zg"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC28Xjt1ehtb8%253D&amp;md5=448855cefab1324816d72c3a3f6e31e5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fncomms10780&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10780%26sid%3Dliteratum%253Aachs%26aulast%3DRiche%26aufirst%3DC.%2BT.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DGupta%26aufirst%3DM.%26aulast%3DBrutchey%26aufirst%3DR.%2BL.%26aulast%3DMalmstadt%26aufirst%3DN.%26atitle%3DFlow%2520Invariant%2520Droplet%2520Formation%2520for%2520Stable%2520Parallel%2520Microreactors%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26doi%3D10.1038%2Fncomms10780" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kruschke, J. K.</span></span> <span> </span><span class="NLM_article-title">Doing Bayesian Data Analysis: A Tutorial with R, JAGS, and Stan</span>. <span class="NLM_publisher-name">Academic Press</span>, p 776.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=Kruschke%2C+J.+K.+Doing+Bayesian+Data+Analysis%3A+A+Tutorial+with+R%2C+JAGS%2C+and+Stan.+Academic+Press%2C+p+776." target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DKruschke%26aufirst%3DJ.%2BK.%26atitle%3DDoing%2520Bayesian%2520Data%2520Analysis%253A%2520A%2520Tutorial%2520with%2520R%252C%2520JAGS%252C%2520and%2520Stan%26pub%3DAcademic%2520Press" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krizhevsky, A.</span>; <span class="NLM_string-name">Sutskever, I.</span>; <span class="NLM_string-name">Hinton, G. E.</span></span> In  <i>ImageNet Classification with Deep Convolutional Neural Networks</i>, Proceedings of the 25th International Conference on Neural Information Processing Systems, <span class="NLM_publisher-name">NIPS</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">1097</span>– <span class="NLM_lpage">1105</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2012&amp;pages=1097-1105&amp;author=A.+Krizhevsky&amp;author=I.+Sutskever&amp;author=G.+E.+Hinton&amp;title=ImageNet+Classification+with+Deep+Convolutional+Neural+Networks" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DKrizhevsky%26aufirst%3DA.%26jtitle%3DImageNet%2520Classification%2520with%2520Deep%2520Convolutional%2520Neural%2520Networks%26pub%3DNIPS%26date%3D2012%26spage%3D1097%26epage%3D1105" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bay, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ess, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuytelaars, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gool, L.</span></span> <span> </span><span class="NLM_article-title">Speeded-Up Robust Features (SURF)</span>. <i>Comput. Vis. Image Underst.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">346</span>– <span class="NLM_lpage">359</span>, <span class="refDoi">&nbsp;DOI: 10.1016/j.cviu.2007.09.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2Fj.cviu.2007.09.014" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=110&amp;publication_year=2008&amp;pages=346-359&amp;author=H.+Bayauthor=A.+Essauthor=T.+Tuytelaarsauthor=L.+Van+Gool&amp;title=Speeded-Up+Robust+Features+%28SURF%29&amp;doi=10.1016%2Fj.cviu.2007.09.014" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.cviu.2007.09.014&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cviu.2007.09.014%26sid%3Dliteratum%253Aachs%26aulast%3DBay%26aufirst%3DH.%26aulast%3DEss%26aufirst%3DA.%26aulast%3DTuytelaars%26aufirst%3DT.%26aulast%3DVan%2BGool%26aufirst%3DL.%26atitle%3DSpeeded-Up%2520Robust%2520Features%2520%2528SURF%2529%26jtitle%3DComput.%2520Vis.%2520Image%2520Underst.%26date%3D2008%26volume%3D110%26spage%3D346%26epage%3D359%26doi%3D10.1016%2Fj.cviu.2007.09.014" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Breiman, L.</span></span> <span> </span><span class="NLM_article-title">Random Forests</span>. <i>Mach. Learn.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">32</span>, <span class="refDoi">&nbsp;DOI: 10.1023/A:1010933404324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1023%2FA%3A1010933404324" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=2001&amp;pages=5-32&amp;author=L.+Breiman&amp;title=Random+Forests&amp;doi=10.1023%2FA%3A1010933404324" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1023%2FA%3A1010933404324&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1010933404324%26sid%3Dliteratum%253Aachs%26aulast%3DBreiman%26aufirst%3DL.%26atitle%3DRandom%2520Forests%26jtitle%3DMach.%2520Learn.%26date%3D2001%26volume%3D45%26spage%3D5%26epage%3D32%26doi%3D10.1023%2FA%3A1010933404324" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strobl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulesteix, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kneib, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeileis, A.</span></span> <span> </span><span class="NLM_article-title">Conditional Variable Importance for Random Forests</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">307</span> <span class="refDoi">&nbsp;DOI: 10.1186/1471-2105-9-307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1186%2F1471-2105-9-307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=18620558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A280%3ADC%252BD1cvos1Crug%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2008&amp;author=C.+Stroblauthor=A.-L.+Boulesteixauthor=T.+Kneibauthor=T.+Augustinauthor=A.+Zeileis&amp;title=Conditional+Variable+Importance+for+Random+Forests&amp;doi=10.1186%2F1471-2105-9-307" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Conditional variable importance for random forests</span></div><div class="casAuthors">Strobl Carolin; Boulesteix Anne-Laure; Kneib Thomas; Augustin Thomas; Zeileis Achim</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">307</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Random forests are becoming increasingly popular in many scientific fields because they can cope with "small n large p" problems, complex interactions and even highly correlated predictor variables.  Their variable importance measures have recently been suggested as screening tools for, e.g., gene expression studies.  However, these variable importance measures show a bias towards correlated predictor variables.  RESULTS:  We identify two mechanisms responsible for this finding: (i) A preference for the selection of correlated predictors in the tree building process and (ii) an additional advantage for correlated predictor variables induced by the unconditional permutation scheme that is employed in the computation of the variable importance measure.  Based on these considerations we develop a new, conditional permutation scheme for the computation of the variable importance measure.  CONCLUSION:  The resulting conditional variable importance reflects the true impact of each predictor variable more reliably than the original marginal approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQNtCwFapSCFwwxfk_AGB-EfW6udTcc2ebj92NgQQun47ntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BD1cvos1Crug%253D%253D&amp;md5=ce09e6988b2765e0f6d060f54fd766b6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-9-307&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-9-307%26sid%3Dliteratum%253Aachs%26aulast%3DStrobl%26aufirst%3DC.%26aulast%3DBoulesteix%26aufirst%3DA.-L.%26aulast%3DKneib%26aufirst%3DT.%26aulast%3DAugustin%26aufirst%3DT.%26aulast%3DZeileis%26aufirst%3DA.%26atitle%3DConditional%2520Variable%2520Importance%2520for%2520Random%2520Forests%26jtitle%3DBMC%2520Bioinf.%26date%3D2008%26volume%3D9%26doi%3D10.1186%2F1471-2105-9-307" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feizi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, P.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kandukuri, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Wide-Field Computational Imaging of Pathology Slides Using Lens-Free on-Chip Microscopy</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">267ra175</span>, <span class="refDoi">&nbsp;DOI: 10.1126/scitranslmed.3009850</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1126%2Fscitranslmed.3009850" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=25520396" title="">PubMed</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2014&amp;pages=267ra175&amp;author=A.+Greenbaumauthor=Y.+Zhangauthor=A.+Feiziauthor=P.-L.+Chungauthor=W.+Luoauthor=S.+R.+Kandukuriauthor=A.+Ozcan&amp;title=Wide-Field+Computational+Imaging+of+Pathology+Slides+Using+Lens-Free+on-Chip+Microscopy&amp;doi=10.1126%2Fscitranslmed.3009850" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3009850&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3009850%26sid%3Dliteratum%253Aachs%26aulast%3DGreenbaum%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DFeizi%26aufirst%3DA.%26aulast%3DChung%26aufirst%3DP.-L.%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DKandukuri%26aufirst%3DS.%2BR.%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DWide-Field%2520Computational%2520Imaging%2520of%2520Pathology%2520Slides%2520Using%2520Lens-Free%2520on-Chip%2520Microscopy%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2014%26volume%3D6%26spage%3D267ra175%26doi%3D10.1126%2Fscitranslmed.3009850" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, T.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göröcs, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isikman, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coskun, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudanyali, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Imaging without Lenses: Achievements and Remaining Challenges of Wide-Field on-Chip Microscopy</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">889</span>– <span class="NLM_lpage">895</span>, <span class="refDoi">&nbsp;DOI: 10.1038/nmeth.2114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1038%2Fnmeth.2114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=22936170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ynt7rL" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2012&amp;pages=889-895&amp;author=A.+Greenbaumauthor=W.+Luoauthor=T.-W.+Suauthor=Z.+G%C3%B6r%C3%B6csauthor=L.+Xueauthor=S.+O.+Isikmanauthor=A.+F.+Coskunauthor=O.+Mudanyaliauthor=A.+Ozcan&amp;title=Imaging+without+Lenses%3A+Achievements+and+Remaining+Challenges+of+Wide-Field+on-Chip+Microscopy&amp;doi=10.1038%2Fnmeth.2114" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Imaging without lenses: achievements and remaining challenges of wide-field on-chip microscopy</span></div><div class="casAuthors">Greenbaum, Alon; Luo, Wei; Su, Ting-Wei; Goeroecs, Zoltan; Xue, Liang; Isikman, Serhan O.; Coskun, Ahmet F.; Mudanyali, Onur; Ozcan, Aydogan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">889-895</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We discuss unique features of lens-free computational imaging tools and report some of their emerging results for wide-field on-chip microscopy, such as the achievement of a numerical aperture (NA) of ∼0.8-0.9 across a field of view (FOV) of more than 20 mm2 or an NA of ∼0.1 across a FOV of ∼18 cm2, which corresponds to an image with more than 1.5 gigapixels.  We also discuss the current challenges that these computational on-chip microscopes face, shedding light on their future directions and applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPnRkgHYMA3bVg90H21EOLACvtfcHk0lhXvlfQPAJE2w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38Xht1ynt7rL&amp;md5=bf5f3f120d6eb14a5b3619bd599d45d0</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2114&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2114%26sid%3Dliteratum%253Aachs%26aulast%3DGreenbaum%26aufirst%3DA.%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DSu%26aufirst%3DT.-W.%26aulast%3DG%25C3%25B6r%25C3%25B6cs%26aufirst%3DZ.%26aulast%3DXue%26aufirst%3DL.%26aulast%3DIsikman%26aufirst%3DS.%2BO.%26aulast%3DCoskun%26aufirst%3DA.%2BF.%26aulast%3DMudanyali%26aufirst%3DO.%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DImaging%2520without%2520Lenses%253A%2520Achievements%2520and%2520Remaining%2520Challenges%2520of%2520Wide-Field%2520on-Chip%2520Microscopy%26jtitle%3DNat.%2520Methods%26date%3D2012%26volume%3D9%26spage%3D889%26epage%3D895%26doi%3D10.1038%2Fnmeth.2114" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Zee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riche, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tofig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallaher, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damoiseaux, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Carlo, D.</span></span> <span> </span><span class="NLM_article-title">A Gelatin Microdroplet Platform for High-Throughput Sorting of Hyperproducing Single-Cell-Derived Microalgal Clones</span>. <i>Small</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>, <span class="NLM_elocation-id">1803315</span> <span class="refDoi">&nbsp;DOI: 10.1002/smll.201803315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1002%2Fsmll.201803315" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=2018&amp;author=M.+Liauthor=M.+van+Zeeauthor=C.+T.+Richeauthor=B.+Tofigauthor=S.+D.+Gallaherauthor=S.+S.+Merchantauthor=R.+Damoiseauxauthor=K.+Godaauthor=D.+Di+Carlo&amp;title=A+Gelatin+Microdroplet+Platform+for+High-Throughput+Sorting+of+Hyperproducing+Single-Cell-Derived+Microalgal+Clones&amp;doi=10.1002%2Fsmll.201803315" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fsmll.201803315&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fsmll.201803315%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DM.%26aulast%3Dvan%2BZee%26aufirst%3DM.%26aulast%3DRiche%26aufirst%3DC.%2BT.%26aulast%3DTofig%26aufirst%3DB.%26aulast%3DGallaher%26aufirst%3DS.%2BD.%26aulast%3DMerchant%26aufirst%3DS.%2BS.%26aulast%3DDamoiseaux%26aufirst%3DR.%26aulast%3DGoda%26aufirst%3DK.%26aulast%3DDi%2BCarlo%26aufirst%3DD.%26atitle%3DA%2520Gelatin%2520Microdroplet%2520Platform%2520for%2520High-Throughput%2520Sorting%2520of%2520Hyperproducing%2520Single-Cell-Derived%2520Microalgal%2520Clones%26jtitle%3DSmall%26date%3D2018%26volume%3D14%26doi%3D10.1002%2Fsmll.201803315" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Rutte, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Carlo, D.</span></span> <span> </span><span class="NLM_article-title">Scalable High-Throughput Production of Modular Microgels for In Situ Assembly of Microporous Tissue Scaffolds</span>. <i>Adv. Funct. Mater.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>, <span class="NLM_elocation-id">1900071</span> <span class="refDoi">&nbsp;DOI: 10.1002/adfm.201900071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1002%2Fadfm.201900071" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=2019&amp;author=J.+M.+de%0ARutteauthor=J.+Kohauthor=D.+Di+Carlo&amp;title=Scalable+High-Throughput+Production+of+Modular+Microgels+for+In+Situ+Assembly+of+Microporous+Tissue+Scaffolds&amp;doi=10.1002%2Fadfm.201900071" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fadfm.201900071&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fadfm.201900071%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BRutte%26aufirst%3DJ.%2BM.%26aulast%3DKoh%26aufirst%3DJ.%26aulast%3DDi%2BCarlo%26aufirst%3DD.%26atitle%3DScalable%2520High-Throughput%2520Production%2520of%2520Modular%2520Microgels%2520for%2520In%2520Situ%2520Assembly%2520of%2520Microporous%2520Tissue%2520Scaffolds%26jtitle%3DAdv.%2520Funct.%2520Mater.%26date%3D2019%26volume%3D29%26doi%3D10.1002%2Fadfm.201900071" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sikora, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Field-Portable Wide-Field Microscopy of Dense Samples Using Multi-Height Pixel Super-Resolution Based Lensfree Imaging</span>. <i>Lab Chip</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1242</span>– <span class="NLM_lpage">1245</span>, <span class="refDoi">&nbsp;DOI: 10.1039/c2lc21072j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1039%2Fc2lc21072j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=22334329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlKrsbs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2012&amp;pages=1242-1245&amp;author=A.+Greenbaumauthor=U.+Sikoraauthor=A.+Ozcan&amp;title=Field-Portable+Wide-Field+Microscopy+of+Dense+Samples+Using+Multi-Height+Pixel+Super-Resolution+Based+Lensfree+Imaging&amp;doi=10.1039%2Fc2lc21072j" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Field-portable wide-field microscopy of dense samples using multi-height pixel super-resolution based lensfree imaging</span></div><div class="casAuthors">Greenbaum, Alon; Sikora, Uzair; Ozcan, Aydogan</div><div class="citationInfo"><span class="NLM_cas:title">Lab on a Chip</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1242-1245</span>CODEN:
                <span class="NLM_cas:coden">LCAHAM</span>;
        ISSN:<span class="NLM_cas:issn">1473-0189</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We report a field-portable lensfree microscope that can image dense and connected specimens with sub-micron resoln. over a large field-of-view of ∼30 mm2 (i.e., ∼6.4 mm × ∼4.6 mm) using pixel super-resoln. and iterative phase recovery techniques.  Weighing ∼122 g with dimensions of 4 cm × 4 cm × 15 cm, this microscope records lensfree in-line holograms of specimens onto an opto-electronic sensor-array using partially coherent illumination.  To reconstruct the phase and amplitude images of dense samples (with &gt;0.3 billion pixels in each image, i.e., &gt;0.6 billion pixels total), we employ a multi-height imaging approach, where by using a mech. interface the sensor-to-sample distance is dynamically changed by random discrete steps of e.g., ∼10 to 80 μm.  By digitally propagating back and forth between these multi-height super-resolved holograms (corresponding to typically 2-5 planes), phase and amplitude images of dense samples can be recovered without the need for any spatial masks or filtering.  We demonstrate the performance of this field-portable multi-height lensfree microscope by imaging Papanicolaou smears (also known as Pap tests).  Our results reveal the promising potential of this multi-height lensfree computational microscopy platform for e.g., pathol. needs in resource limited settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpL9HxR3wM-J7Vg90H21EOLACvtfcHk0lj0Sw6fS-Kk2Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38XjtlKrsbs%253D&amp;md5=43e58a212d771907c9fec580f24ad6c5</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1039%2Fc2lc21072j&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2lc21072j%26sid%3Dliteratum%253Aachs%26aulast%3DGreenbaum%26aufirst%3DA.%26aulast%3DSikora%26aufirst%3DU.%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DField-Portable%2520Wide-Field%2520Microscopy%2520of%2520Dense%2520Samples%2520Using%2520Multi-Height%2520Pixel%2520Super-Resolution%2520Based%2520Lensfree%2520Imaging%26jtitle%3DLab%2520Chip%26date%3D2012%26volume%3D12%26spage%3D1242%26epage%3D1245%26doi%3D10.1039%2Fc2lc21072j" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aidukas, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konda, P. C.</span></span> <span> </span><span class="NLM_article-title">Low-Cost, Sub-Micron Resolution, Wide-Field Computational Microscopy Using Opensource Hardware</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">7457</span> <span class="refDoi">&nbsp;DOI: 10.1038/s41598-019-43845-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1038%2Fs41598-019-43845-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=31092867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A280%3ADC%252BB3M7mtFKquw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2019&amp;author=T.+Aidukasauthor=R.+Eckertauthor=A.+R.+Harveyauthor=L.+Wallerauthor=P.+C.+Konda&amp;title=Low-Cost%2C+Sub-Micron+Resolution%2C+Wide-Field+Computational+Microscopy+Using+Opensource+Hardware&amp;doi=10.1038%2Fs41598-019-43845-9" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Low-cost, sub-micron resolution, wide-field computational microscopy using opensource hardware</span></div><div class="casAuthors">Aidukas Tomas; Harvey Andrew R; Konda Pavan C; Eckert Regina; Waller Laura</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7457</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The revolution in low-cost consumer photography and computation provides fertile opportunity for a disruptive reduction in the cost of biomedical imaging.  Conventional approaches to low-cost microscopy are fundamentally restricted, however, to modest field of view (FOV) and/or resolution.  We report a low-cost microscopy technique, implemented with a Raspberry Pi single-board computer and color camera combined with Fourier ptychography (FP), to computationally construct 25-megapixel images with sub-micron resolution.  New image-construction techniques were developed to enable the use of the low-cost Bayer color sensor, to compensate for the highly aberrated re-used camera lens and to compensate for misalignments associated with the 3D-printed microscope structure.  This high ratio of performance to cost is of particular interest to high-throughput microscopy applications, ranging from drug discovery and digital pathology to health screening in low-income countries. 3D models and assembly instructions of our microscope are made available for open source use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTK4j42ZllqepPq5LRczm8ffW6udTcc2eavMRVuZaOAOrntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BB3M7mtFKquw%253D%253D&amp;md5=a60ba19fb88ab6c8d1a8379aca2d25a3</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-43845-9&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-43845-9%26sid%3Dliteratum%253Aachs%26aulast%3DAidukas%26aufirst%3DT.%26aulast%3DEckert%26aufirst%3DR.%26aulast%3DHarvey%26aufirst%3DA.%2BR.%26aulast%3DWaller%26aufirst%3DL.%26aulast%3DKonda%26aufirst%3DP.%2BC.%26atitle%3DLow-Cost%252C%2520Sub-Micron%2520Resolution%252C%2520Wide-Field%2520Computational%2520Microscopy%2520Using%2520Opensource%2520Hardware%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26doi%3D10.1038%2Fs41598-019-43845-9" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yelleswarapu, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, H.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadavali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issadore, D.</span></span> <span> </span><span class="NLM_article-title">Ultra-High Throughput Detection (1 Million Droplets per Second) of Fluorescent Droplets Using a Cell Phone Camera and Time Domain Encoded Optofluidics</span>. <i>Lab Chip</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1083</span>– <span class="NLM_lpage">1094</span>, <span class="refDoi">&nbsp;DOI: 10.1039/C6LC01489E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1039%2FC6LC01489E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=28225099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisF2qsrs%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=2017&amp;pages=1083-1094&amp;author=V.+R.+Yelleswarapuauthor=H.-H.+Jeongauthor=S.+Yadavaliauthor=D.+Issadore&amp;title=Ultra-High+Throughput+Detection+%281+Million+Droplets+per+Second%29+of+Fluorescent+Droplets+Using+a+Cell+Phone+Camera+and+Time+Domain+Encoded+Optofluidics&amp;doi=10.1039%2FC6LC01489E" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Ultra-high throughput detection (1 million droplets per second) of fluorescent droplets using a cell phone camera and time domain encoded optofluidics</span></div><div class="casAuthors">Yelleswarapu, Venkata R.; Jeong, Heon-Ho; Yadavali, Sagar; Issadore, David</div><div class="citationInfo"><span class="NLM_cas:title">Lab on a Chip</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1083-1094</span>CODEN:
                <span class="NLM_cas:coden">LCAHAM</span>;
        ISSN:<span class="NLM_cas:issn">1473-0189</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Droplet-based assays-in which ultra-sensitive mol. measurements are made by performing millions of parallel expts. in picoliter droplets-have generated enormous enthusiasm due to their single mol. resoln. and robustness to reaction conditions.  These assays have great untapped potential for point of care diagnostics but are currently confined to lab. settings due to the instrumentation necessary to serially generate, control, and measure tens of millions of droplets.  To address this challenge, we have developed the microdroplet megascale detector (μMD) that can generate and detect the fluorescence of millions of droplets per s (1000× faster than conventional approaches) using only a conventional cell phone camera.  The key innovation of our approach is borrowed from the telecommunications industry, wherein we modulate the excitation light with a pseudorandom sequence that enables individual droplets to be resolved that would otherwise overlap due to the limited frame rate of digital cameras.  Using this approach, the μMD measures droplets at a rate of 106 droplets per s (.vphi. = 166 mL h-1) in 120 parallel microfluidic channels and achieves a limit of detection LOD = 1 μM Rhodamine dye, sufficient for typical droplet based assays.  We incorporate this new droplet detection technol. with our previously reported parallelized droplet prodn. strategy, incorporating 200 parallel droplet makers and only one set of continuous and droplet phase inputs and one output line.  By miniaturizing and integrating droplet based diagnostics into a handheld format, the μMD platform can translate ultra-sensitive droplet based assays into a self-contained platform for practical use in clin. and industrial settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJSqAgLzjGnLVg90H21EOLACvtfcHk0lj0Sw6fS-Kk2Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC2sXisF2qsrs%253D&amp;md5=b6f7dc4f3afde3739f89c6c2c47911e5</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1039%2FC6LC01489E&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6LC01489E%26sid%3Dliteratum%253Aachs%26aulast%3DYelleswarapu%26aufirst%3DV.%2BR.%26aulast%3DJeong%26aufirst%3DH.-H.%26aulast%3DYadavali%26aufirst%3DS.%26aulast%3DIssadore%26aufirst%3DD.%26atitle%3DUltra-High%2520Throughput%2520Detection%2520%25281%2520Million%2520Droplets%2520per%2520Second%2529%2520of%2520Fluorescent%2520Droplets%2520Using%2520a%2520Cell%2520Phone%2520Camera%2520and%2520Time%2520Domain%2520Encoded%2520Optofluidics%26jtitle%3DLab%2520Chip%26date%3D2017%26volume%3D17%26spage%3D1083%26epage%3D1094%26doi%3D10.1039%2FC6LC01489E" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yelleswarapu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haber, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inapuri, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Issadore, D.</span></span> <span> </span><span class="NLM_article-title">Mobile Platform for Rapid Sub–Picogram-per-Milliliter, Multiplexed, Digital Droplet Detection of Proteins</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">4489</span>– <span class="NLM_lpage">4495</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1814110116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1073%2Fpnas.1814110116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=30765530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC1MXktFejur4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=116&amp;publication_year=2019&amp;pages=4489-4495&amp;author=V.+Yelleswarapuauthor=J.+R.+Buserauthor=M.+Haberauthor=J.+Baronauthor=E.+Inapuriauthor=D.+Issadore&amp;title=Mobile+Platform+for+Rapid+Sub%E2%80%93Picogram-per-Milliliter%2C+Multiplexed%2C+Digital+Droplet+Detection+of+Proteins&amp;doi=10.1073%2Fpnas.1814110116" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Mobile platform for rapid subpicogram-per-milliliter, multiplexed, digital droplet detection of proteins</span></div><div class="casAuthors">Yelleswarapu, Venkata; Buser, Joshua R.; Haber, Margalit; Baron, Jonathan; Inapuri, Eshwar; Issadore, David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4489-4495</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Digital droplet assays in which biol. samples are compartmentalized into millions of femtoliter-vol. droplets and interrogated individually have generated enormous enthusiasm for their ability to detect biomarkers with single-mol. sensitivity.  These assays have untapped potential for point-of-care diagnostics but are currently mainly confined to lab. settings, due to the instrumentation necessary to serially generate, control, and measure tens of millions of droplets/compartments.  To address this challenge, the authors developed an optofluidic platform that miniaturizes digital assays into a mobile format by parallelizing their operation.  This technol. is based on three key innovations: (1) the integration and parallel operation of a hundred droplet generators onto a single chip that operates and gt;100A~, faster than a single droplet generator, (2) the fluorescence detection of droplets at and gt;100A~, faster than conventional in-flow detection using time domain-encoded mobile phone imaging, and (3) the integration of on-chip delay lines and sample processing to allow serum-to-answer device operation.  To demonstrate the power of this approach, the authors performed a duplex digital ELISA.  The authors characterized the performance of this assay by first using spiked recombinant proteins in a complex media (FBS) and measured a limit of detection, 0.004 pg/mL (300 aM), a 1.000 A~, improvement over std. ELISA and matching that of the existing lab.-based gold std. digital ELISA system.  The authors addnl. measured endogenous GM-CSF and IL6 in human serum from n = 14 human subjects using the authors' mobile duplex assay, and showed excellent agreement with the gold std. system (R2 = 0.96).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5NsgeNcFLmrVg90H21EOLACvtfcHk0lj0Sw6fS-Kk2Q"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC1MXktFejur4%253D&amp;md5=56b615d3f856bfb6fa2016223f83d333</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1814110116&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1814110116%26sid%3Dliteratum%253Aachs%26aulast%3DYelleswarapu%26aufirst%3DV.%26aulast%3DBuser%26aufirst%3DJ.%2BR.%26aulast%3DHaber%26aufirst%3DM.%26aulast%3DBaron%26aufirst%3DJ.%26aulast%3DInapuri%26aufirst%3DE.%26aulast%3DIssadore%26aufirst%3DD.%26atitle%3DMobile%2520Platform%2520for%2520Rapid%2520Sub%25E2%2580%2593Picogram-per-Milliliter%252C%2520Multiplexed%252C%2520Digital%2520Droplet%2520Detection%2520of%2520Proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2019%26volume%3D116%26spage%3D4489%26epage%3D4495%26doi%3D10.1073%2Fpnas.1814110116" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gossett, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, H. T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindgren, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Carlo, D.</span></span> <span> </span><span class="NLM_article-title">Hydrodynamic Stretching of Single Cells for Large Population Mechanical Phenotyping</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">7630</span>– <span class="NLM_lpage">7635</span>, <span class="refDoi">&nbsp;DOI: 10.1073/pnas.1200107109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1073%2Fpnas.1200107109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=22547795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A528%3ADC%252BC38Xot1KnsL4%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=109&amp;publication_year=2012&amp;pages=7630-7635&amp;author=D.+R.+Gossettauthor=H.+T.+K.+Tseauthor=S.+Leeauthor=Y.+Yingauthor=A.+G.+Lindgrenauthor=O.+O.+Yangauthor=J.+Raoauthor=A.+Clarkauthor=D.+Di+Carlo&amp;title=Hydrodynamic+Stretching+of+Single+Cells+for+Large+Population+Mechanical+Phenotyping&amp;doi=10.1073%2Fpnas.1200107109" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrodynamic stretching of single cells for large population mechanical phenotyping</span></div><div class="casAuthors">Gossett, Daniel R.; Tse, Henry T. K.; Lee, Serena A.; Ying, Yong; Lindgren, Anne G.; Yang, Otto O.; Rao, Jianyu; Clark, Amander T.; Di Carlo, Dino</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7630-7635, S7630/1-S7630/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Cell state is often assayed through measurement of biochem. and biophys. markers.  Although biochem. markers were widely used, intrinsic biophys. markers, such as the ability to mech. deform under a load, are advantageous in that they do not require costly labeling or sample prepn.  However, current techniques that assay cell mech. properties have had limited adoption in clin. and cell biol. research applications.  Here, we demonstrate an automated microfluidic technol. capable of probing single-cell deformability at approx. 2,000 cells/s.  The method uses inertial focusing to uniformly deliver cells to a stretching extensional flow where cells are deformed at high strain rates, imaged with a high-speed camera, and computationally analyzed to ext. quant. parameters.  This approach allows us to analyze cells at throughputs orders of magnitude faster than previously reported biophys. flow cytometers and single-cell mechanics tools, while creating easily observable larger strains and limiting user time commitment and bias through automation.  Using this approach we rapidly assay the deformability of native populations of leukocytes and malignant cells in pleural effusions and accurately predict disease state in patients with cancer and immune activation with a sensitivity of 91% and a specificity of 86%.  As a tool for biol. research, we show the deformability we measure is an early biomarker for pluripotent stem cell differentiation and is likely linked to nuclear structural changes.  Microfluidic deformability cytometry brings the statistical accuracy of traditional flow cytometric techniques to label-free biophys. biomarkers, enabling applications in clin. diagnostics, stem cell characterization, and single-cell biophysics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqL31SCuHJC7Vg90H21EOLACvtfcHk0lj6f_XtzyCl1w"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A528%3ADC%252BC38Xot1KnsL4%253D&amp;md5=460b3fe2fae06b6ec42e8557c0b5be51</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1200107109&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1200107109%26sid%3Dliteratum%253Aachs%26aulast%3DGossett%26aufirst%3DD.%2BR.%26aulast%3DTse%26aufirst%3DH.%2BT.%2BK.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DYing%26aufirst%3DY.%26aulast%3DLindgren%26aufirst%3DA.%2BG.%26aulast%3DYang%26aufirst%3DO.%2BO.%26aulast%3DRao%26aufirst%3DJ.%26aulast%3DClark%26aufirst%3DA.%26aulast%3DDi%2BCarlo%26aufirst%3DD.%26atitle%3DHydrodynamic%2520Stretching%2520of%2520Single%2520Cells%2520for%2520Large%2520Population%2520Mechanical%2520Phenotyping%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D7630%26epage%3D7635%26doi%3D10.1073%2Fpnas.1200107109" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivenson, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Haan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Bright-Field Holography: Cross-Modality Deep Learning Enables Snapshot 3D Imaging with Bright-Field Contrast Using a Single Hologram</span>. <i>Light: Sci. Appl.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">25</span> <span class="refDoi">&nbsp;DOI: 10.1038/s41377-019-0139-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1038%2Fs41377-019-0139-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facssensors.9b01974&amp;key=30854197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facssensors.9b01974&amp;key=1%3ACAS%3A280%3ADC%252BB3cbisVCnsw%253D%253D" title="">CAS</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2019&amp;author=Y.+Wuauthor=Y.+Luoauthor=G.+Chaudhariauthor=Y.+Rivensonauthor=A.+Calisauthor=K.+de+Haanauthor=A.+Ozcan&amp;title=Bright-Field+Holography%3A+Cross-Modality+Deep+Learning+Enables+Snapshot+3D+Imaging+with+Bright-Field+Contrast+Using+a+Single+Hologram&amp;doi=10.1038%2Fs41377-019-0139-9" target="_blank">Google Scholar</a><div class="casRecord" id="
                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Bright-field holography: cross-modality deep learning enables snapshot 3D imaging with bright-field contrast using a single hologram</span></div><div class="casAuthors">Wu Yichen; Luo Yilin; Rivenson Yair; Calis Ayfer; de Haan Kevin; Ozcan Aydogan; Wu Yichen; Luo Yilin; Rivenson Yair; Calis Ayfer; de Haan Kevin; Ozcan Aydogan; Wu Yichen; Luo Yilin; Rivenson Yair; Calis Ayfer; de Haan Kevin; Ozcan Aydogan; Chaudhari Gunvant; Ozcan Aydogan</div><div class="citationInfo"><span class="NLM_cas:title">Light, science &amp; applications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">25</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Digital holographic microscopy enables the 3D reconstruction of volumetric samples from a single-snapshot hologram.  However, unlike a conventional bright-field microscopy image, the quality of holographic reconstructions is compromised by interference fringes as a result of twin images and out-of-plane objects.  Here, we demonstrate that cross-modality deep learning using a generative adversarial network (GAN) can endow holographic images of a sample volume with bright-field microscopy contrast, combining the volumetric imaging capability of holography with the speckle- and artifact-free image contrast of incoherent bright-field microscopy.  We illustrate the performance of this "bright-field holography" method through the snapshot imaging of bioaerosols distributed in 3D, matching the artifact-free image contrast and axial sectioning performance of a high-NA bright-field microscope.  This data-driven deep-learning-based imaging method bridges the contrast gap between coherent and incoherent imaging, and enables the snapshot 3D imaging of objects with bright-field contrast from a single hologram, benefiting from the wave-propagation framework of holography.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder ®" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFk2dAVf44K1QKX3ooHxWjfW6udTcc2eZ1sU6bhqtQ1rntkvexoksv"> &gt;&gt; More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&amp;resolution=options&amp;coi=1%3ACAS%3A280%3ADC%252BB3cbisVCnsw%253D%253D&amp;md5=95ef8ccf478a409fe579c753138b6e44</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fs41377-019-0139-9&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41377-019-0139-9%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DChaudhari%26aufirst%3DG.%26aulast%3DRivenson%26aufirst%3DY.%26aulast%3DCalis%26aufirst%3DA.%26aulast%3Dde%2BHaan%26aufirst%3DK.%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DBright-Field%2520Holography%253A%2520Cross-Modality%2520Deep%2520Learning%2520Enables%2520Snapshot%25203D%2520Imaging%2520with%2520Bright-Field%2520Contrast%2520Using%2520a%2520Single%2520Hologram%26jtitle%3DLight%253A%2520Sci.%2520Appl.%26date%3D2019%26volume%3D8%26doi%3D10.1038%2Fs41377-019-0139-9" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivenson, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günaydin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozcan, A.</span></span> <span> </span><span class="NLM_article-title">Extended Depth-of-Field in Holographic Imaging Using Deep-Learning-Based Autofocusing and Phase Recovery</span>. <i>Optica</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">704</span> <span class="refDoi">&nbsp;DOI: 10.1364/OPTICA.5.000704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1364%2FOPTICA.5.000704" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2018&amp;author=Y.+Wuauthor=Y.+Rivensonauthor=Y.+Zhangauthor=Z.+Weiauthor=H.+G%C3%BCnaydinauthor=X.+Linauthor=A.+Ozcan&amp;title=Extended+Depth-of-Field+in+Holographic+Imaging+Using+Deep-Learning-Based+Autofocusing+and+Phase+Recovery&amp;doi=10.1364%2FOPTICA.5.000704" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1364%2FOPTICA.5.000704&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1364%252FOPTICA.5.000704%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DRivenson%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DZ.%26aulast%3DG%25C3%25BCnaydin%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DOzcan%26aufirst%3DA.%26atitle%3DExtended%2520Depth-of-Field%2520in%2520Holographic%2520Imaging%2520Using%2520Deep-Learning-Based%2520Autofocusing%2520and%2520Phase%2520Recovery%26jtitle%3DOptica%26date%3D2018%26volume%3D5%26doi%3D10.1364%2FOPTICA.5.000704" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atherton, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerbyson, D. J.</span></span> <span> </span><span class="NLM_article-title">Size Invariant Circle Detection</span>. <i>Image Vis. Comput.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">795</span>– <span class="NLM_lpage">803</span>, <span class="refDoi">&nbsp;DOI: 10.1016/S0262-8856(98)00160-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.1016%2FS0262-8856%2898%2900160-7" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=1999&amp;pages=795-803&amp;author=T.+J.+Athertonauthor=D.+J.+Kerbyson&amp;title=Size+Invariant+Circle+Detection&amp;doi=10.1016%2FS0262-8856%2898%2900160-7" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2FS0262-8856%2898%2900160-7&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0262-8856%252898%252900160-7%26sid%3Dliteratum%253Aachs%26aulast%3DAtherton%26aufirst%3DT.%2BJ.%26aulast%3DKerbyson%26aufirst%3DD.%2BJ.%26atitle%3DSize%2520Invariant%2520Circle%2520Detection%26jtitle%3DImage%2520Vis.%2520Comput.%26date%3D1999%26volume%3D17%26spage%3D795%26epage%3D803%26doi%3D10.1016%2FS0262-8856%2898%2900160-7" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salvatier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiecki, T. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonnesbeck, C.</span></span> <span> </span><span class="NLM_article-title">Probabilistic Programming in Python Using PyMC3</span>. <i>PeerJ Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e55</span> <span class="refDoi">&nbsp;DOI: 10.7717/peerj-cs.55</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facssensors.9b01974&amp;key=10.7717%2Fpeerj-cs.55" title="">Crossref</a>],&nbsp;<a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2&amp;publication_year=2016&amp;author=J.+Salvatierauthor=T.+V.+Wieckiauthor=C.+Fonnesbeck&amp;title=Probabilistic+Programming+in+Python+Using+PyMC3&amp;doi=10.7717%2Fpeerj-cs.55" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.7717%2Fpeerj-cs.55&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26id%3Ddoi%3A10.7717%252Fpeerj-cs.55%26sid%3Dliteratum%253Aachs%26aulast%3DSalvatier%26aufirst%3DJ.%26aulast%3DWiecki%26aufirst%3DT.%2BV.%26aulast%3DFonnesbeck%26aufirst%3DC.%26atitle%3DProbabilistic%2520Programming%2520in%2520Python%2520Using%2520PyMC3%26jtitle%3DPeerJ%2520Comput.%2520Sci.%26date%3D2016%26volume%3D2%26doi%3D10.7717%2Fpeerj-cs.55" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelman, A.</span></span> <span> </span><span class="NLM_article-title">The No-U-Turn Sampler: Adaptively Setting Path Lengths in Hamiltonian Monte Carlo</span>. <i>J. Mach. Learn. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1593</span>– <span class="NLM_lpage">1623</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=2014&amp;pages=1593-1623&amp;author=M.+D.+Hoffmanauthor=A.+Gelman&amp;title=The+No-U-Turn+Sampler%3A+Adaptively+Setting+Path+Lengths+in+Hamiltonian+Monte+Carlo" target="_blank">Google Scholar</a><div class="casRecord empty" id="
                            rightTab-cas_R
                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=https%3A%2F%2Flibkey.io%2Flibraries%2F163%2Fopenurl%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DM.%2BD.%26aulast%3DGelman%26aufirst%3DA.%26atitle%3DThe%2520No-U-Turn%2520Sampler%253A%2520Adaptively%2520Setting%2520Path%2520Lengths%2520in%2520Hamiltonian%2520Monte%2520Carlo%26jtitle%3DJ.%2520Mach.%2520Learn.%2520Res.%26date%3D2014%26volume%3D15%26spage%3D1593%26epage%3D1623" title="OpenURL INDIANA UNIV BLOOMINGTON" onclick="newWindow(this.href);return false" class="button_primary"><span>open URL</span></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i18">Supporting Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_86808"></div></div></div></div></div><hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acssensors.9b01974?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acssensors.9b01974</a>.</p><ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">LAMP precipitate in microwells, fluorescence-thresholded results, visualization of image patch clustering, examples of visual words, importance of machine learning features, machine learning model comparison, examples of ranges of machine learning prediction scores, demonstration of determining minimum droplets required, illustration of Bayesian model for Training cases, illustration of Bayesian model for Testing cases, and an explanation of statistical methods used to determine minimum droplets required in digital assays (<a class="ext-link" href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf">PDF</a>)</p></li></ul><p></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acssensors.9b01974/suppl_file/se9b01974_si_001.pdf">se9b01974_si_001.pdf (1.34 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms &amp; Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facssensors.9b01974&amp;pbContext=%3Bjournal%3Ajournal%3Aascefj%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fascefj.2020.5.issue-2%3Bctype%3Astring%3AJournal+Content%3BrequestedJournal%3Ajournal%3Aascefj%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facssensors.9b01974%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acssensors.9b01974" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2022<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS &amp; Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support &amp; Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"><input type="hidden" name="appID" value="literatum" class="appID"><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"><input type="hidden" name="clientID" value="atypon-lives" class="clientID"><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onclick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        <!-- <div id="gdpr-message-container">
			<a id="closebtn" href="#"><i class="icon-times-circle-o"></i></a> -->
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        <!-- </div> -->
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        <!-- AdRoll Pixel -->
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>

<!-- Twitter universal website tag code -->
<!-- Commented out for ACHS-6152
<script>
    !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe.apply(s,arguments):s.queue.push(arguments);
    },s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',
    a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');
    // Insert Twitter Pixel ID and Standard Event data below
    twq('init','o3njp');
    twq('track','PageView');
</script>
-->
<!-- Facebook Pixel Code -->
<!-- Commented out for ACHS-6152
<script>
    !function(f,b,e,v,n,t,s)
    {if(f.fbq)return;n=f.fbq=function(){n.callMethod?
    n.callMethod.apply(n,arguments):n.queue.push(arguments)};
    if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
    n.queue=[];t=b.createElement(e);t.async=!0;
    t.src=v;s=b.getElementsByTagName(e)[0];
    s.parentNode.insertBefore(t,s)}(window, document,'script',
    'https://connect.facebook.net/en_US/fbevents.js');
    fbq('init', '529840281033320');
    fbq('track', 'PageView');
</script>
<noscript><img height="1" width="1" style="display:none" src="https://www.facebook.com/tr?id=529840281033320&amp;ev=PageView&amp;noscript=1"
/></noscript>
-->





        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
var currentJournal = "ACS Sensors";
$(".citationLinks > a").click(function(e){
    if($(this).text() === "ACS Full Text ") {
        ga('send', 'event', 'Reference _Click', 'ACS Full Text', currentJournal);
    }
    else if($(this).text() === "Crossref") {
        ga('send', 'event', 'Reference _Click', 'Crossref', currentJournal);
    }
    else if($(this).text() === "CAS") {
        ga('send', 'event', 'Reference _Click', 'CAS', currentJournal);
    }
    else if($(this).text() === "PubMed") {
        ga('send', 'event', 'Reference _Click', 'PubMed', currentJournal);
    }
    else if($(this).text() === "Google Scholar") {
        ga('send', 'event', 'Reference _Click', 'Google Scholar', currentJournal);
    }
    else if($(this).hasClass("issue-item_open-url") === true) {
        ga('send', 'event', 'Reference _Click', 'OpenURL', currentJournal);
    }
});
</script>






        
        



        
    





        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        <div tabindex="0" class="showMenuAlertMsg-popup"><div class="pairing-affiliations-popup-content"><div class="pairing-affiliations-header"><button type="button" class="close paringAffClose"><i aria-hidden="true" class="icon-close_thin"></i></button></div><div class="responsrMsg"></div></div></div>




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	dartInstitutionType = 'education';
</script>
        </div>
    





        
        



        
    





        
        <script>
// LIT-455776 workaround
// Moving the popup before the header content for tab accessibility
document.addEventListener('DOMContentLoaded', () => {
    const headerNode = $('.header_content');
    const popupNode = $('.pairing-affiliations-popup');

    if (!headerNode.length || !popupNode.length)
        return;

    popupNode.insertBefore(headerNode);  
});
</script> 
        </div>
    </div>





	
    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-d93b9088814c7fcd2f1c.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-9dee82777d6af4ace557.js"></script>
<script type="text/javascript" src="/wro/l6j1~product.js"></script>

















<script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/v652eace1692a40cfa3763df669d7439c1639079717194" integrity="sha512-Gi7xpJR8tSkrpF7aordPZQlW2DLtzUlZcumS8dMQjwDHEnw9I7ZLyiOj/6tZStRBGtGgN6ceN6cMH8z7etPGlw==" data-cf-beacon="{&quot;rayId&quot;:&quot;705bd23a590420e5&quot;,&quot;token&quot;:&quot;0562ec7bcba14d1cae3543aa43d6e749&quot;,&quot;version&quot;:&quot;2021.12.0&quot;,&quot;si&quot;:100}" crossorigin="anonymous"></script>


<div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh962" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.9826615145776444&amp;iit=1651610527595&amp;tmr=load%3D1651610527274%26core%3D1651610527326%26main%3D1651610527588%26ifr%3D1651610527606&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=fractal%20LAMP%2Cdigital%20analysis%2Cnucleic%20acid%20amplification%20test%2CBayesian%20model%2Cbootstrapping&amp;ab=-&amp;dh=pubs.acs.org&amp;dr=&amp;du=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facssensors.9b01974&amp;href=https%3A%2F%2Fpubs.acs.org%2Fdoi%2F10.1021%2Facssensors.9b01974&amp;dt=Fractal%20LAMP%3A%20Label-Free%20Analysis%20of%20Fractal%20Precipitate%20for%20Digital%20Loop-Mediated%20Isothermal%20Nucleic%20Acid%20Amplification&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Ctype%3DArticle%2Cdescription%2Cimage%2Ctitle%2Curl&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=6271939f8adc0cb0&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=url%3Dhttps%253A%252F%252Fpubs.acs.org%252Fdoi%252Ffull%252F10.1021%252Facssensors.9b01974%26title%3DFractal%2520LAMP%253A%2520Label-Free%2520Analysis%2520of%2520Fractal%2520Precipitate%2520for%2520Digital%2520Loop-Mediated%2520Isothermal%2520Nucleic%2520Acid%2520Amplification%26image%3Dhttps%253A%252F%252Fpubs.acs.org%252Fcms%252F10.1021%252Facssensors.9b01974%252Fasset%252Fimages%252Facssensors.9b01974.social.jpeg_v03%26description%3DNucleic%2520acid%2520amplification%2520assays%2520including%2520loop-mediated%2520isothermal%2520amplification%2520(LAMP)%2520are%2520routinely%2520used%2520in%2520diagnosing%2520diseases%2520and%2520monitoring%2520water%2520and%2520food%2520quality.%2520The%2520results%2520of%2520amplification%2520in%2520these%2520assays%2520are%2520commonly%2520measured%2520with%2520an%2520analog%2520fluorescence%2520readout%252C%2520which%2520requires%2520specialized%2520optical%2520equipment%2520and%2520can%2520lack%2520quantitative%2520precision.%2520Digital%2520analysis%2520of%2520amplification%2520in%2520small%2520fluid%2520compartments%2520based%2520on%2520exceeding%2520a%2520threshold%2520fluorescence%2520level%2520can%2520enhance%2520the%2520quantitative%2520precision%2520of%2520nucleic%2520acid%2520assays%2520(i.e.%252C%2520digital%2520nucleic%2520acid%2520amplification%2520assays)%252C%2520but%2520still%2520requires%2520specialized%2520optical%2520systems%2520for%2520fluorescence%2520readout%2520and%2520the%2520inclusion%2520of%2520a%2520fluorescent%2520dye.%2520Here%252C%2520we%2520report%2520Fractal%2520LAMP%252C%2520an%2520automated%2520method%2520to%2520detect%2520amplified%2520DNA%2520in%2520subnanoliter%2520scale%2520droplets%2520following%2520LAMP%2520in%2520a%2520label-free%2520manner.%2520Our%2520computer%2520vision%2520algorithm%2520achieves%2520high%2520accuracy%2520detecting%2520DNA%2520amplification%2520in%2520droplets%2520by%2520identifying%2520LAMP%2520byproducts%2520that%2520form%2520fractal%2520structures%2520observable%2520in%2520brightfield%2520microscopy.%2520The%2520capabilities%2520of%2520Fractal%2520LAMP%2520are%2520further%2520realized%2520by%2520developing%2520a%2520Bayesian%2520model%2520to%2520estimate%2520DNA%2520concentrations%2520for%2520unknown%2520samples%2520and%2520a%2520bootstrapping%2520method%2520to%2520estimate%2520the%2520number%2520of%2520droplets%2520required%2520to%2520achieve%2520target%2520limits%2520of%2520detection.%2520This%2520digital%252C%2520label-free%2520assay%2520has%2520the%2520potential%2520to%2520lower%2520reagent%2520and%2520reader%2520cost%2520for%2520nucleic%2520acid%2520measurement%2520while%2520maintaining%2520high%2520quantitative%2520accuracy%2520over%25203%2520orders%2520of%2520magnitude%2520of%2520concentration.%26type%3DArticle%26site_name%3DACS%2520Publications&amp;csi=undefined&amp;rev=v8.28.8-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style></body></html>